<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.08.28.270652v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.08.28.270652v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.08.28.270652v2.ppt"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/04/14/2020.08.28.270652.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.08.28.270652v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Cellxgene VIP unleashes full power of interactive visualization and integrative analysis of scRNA-seq, spatial transcriptomics, and multiome data"/>
  <meta name="DC.Identifier" content="10.1101/2020.08.28.270652"/>
  <meta name="DC.Date" content="2022-04-14"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="To meet the growing demands from scientists to effectively extract deep insights from single cell RNA sequencing, spatial transcriptomics, and emerging multiome datasets, we developed cellxgene VIP (Visualization In Plugin), a frontend interactive visualization plugin of cellxgene framework, which greatly expanded capabilities of the base tool in the following aspects. First, it generates a comprehensive set of over eighteen commonly used quality control and analytical plots in high resolution with highly customizable settings in real time. Second, it provides more advanced analytical functions to gain insights on cellular compositions and deep biology, such as marker gene identification, differential gene expression analysis, and gene set enrichment analysis. Third, it empowers advanced users to perform analysis in a Jupyter Notebook like environment, dubbed Command Line Interface (CLI) by programming in Python and/or R directly without limiting themselves to functional modules available via graphical user interface (GUI). Finally, it pioneers methods to visualize multi-modal data, such as spatial transcriptomics embedding aligned with histological image on one slice or multiple slices in a grid format, and the latest 10x Genomic Multiome dataset where both DNA accessibility and gene expression in the same cells are measured, under the same framework in an integrative way to fully leverage the functionalities mentioned above. Taken together, the open-source tool makes large scale single cell data visualization and analysis more accessible to biologists in a user-friendly manner and fosters computational reproducibility by simplifying data and code reuse through the CLI. Going forward, it has the potential to become a crowdsourcing ecosystem for the scientific community to contribute even more modules to the Swiss Army knife of single cell data exploration tools.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Kejie Li"/>
  <meta name="DC.Contributor" content="Zhengyu Ouyang"/>
  <meta name="DC.Contributor" content="Yirui Chen"/>
  <meta name="DC.Contributor" content="Jacob Gagnon"/>
  <meta name="DC.Contributor" content="Dongdong Lin"/>
  <meta name="DC.Contributor" content="Michael Mingueneau"/>
  <meta name="DC.Contributor" content="Will Chen"/>
  <meta name="DC.Contributor" content="David Sexton"/>
  <meta name="DC.Contributor" content="Baohong Zhang"/>
  <meta name="article:published_time" content="2022-04-14"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Cellxgene VIP unleashes full power of interactive visualization and integrative analysis of scRNA-seq, spatial transcriptomics, and multiome data"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>To meet the growing demands from scientists to effectively extract deep insights from single cell RNA sequencing, spatial transcriptomics, and emerging multiome datasets, we developed cellxgene VIP (Visualization In Plugin), a frontend interactive visualization plugin of cellxgene framework, which greatly expanded capabilities of the base tool in the following aspects. First, it generates a comprehensive set of over eighteen commonly used quality control and analytical plots in high resolution with highly customizable settings in real time. Second, it provides more advanced analytical functions to gain insights on cellular compositions and deep biology, such as marker gene identification, differential gene expression analysis, and gene set enrichment analysis. Third, it empowers advanced users to perform analysis in a Jupyter Notebook like environment, dubbed Command Line Interface (CLI) by programming in Python and/or R directly without limiting themselves to functional modules available via graphical user interface (GUI). Finally, it pioneers methods to visualize multi-modal data, such as spatial transcriptomics embedding aligned with histological image on one slice or multiple slices in a grid format, and the latest 10x Genomic Multiome dataset where both DNA accessibility and gene expression in the same cells are measured, under the same framework in an integrative way to fully leverage the functionalities mentioned above. Taken together, the open-source tool makes large scale single cell data visualization and analysis more accessible to biologists in a user-friendly manner and fosters computational reproducibility by simplifying data and code reuse through the CLI. Going forward, it has the potential to become a crowdsourcing ecosystem for the scientific community to contribute even more modules to the Swiss Army knife of single cell data exploration tools.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.08.28.270652v2"/>
  <meta name="citation_id" content="2020.08.28.270652v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.08.28.270652v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.08.28.270652v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.08.28.270652v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.08.28.270652"/>
  <meta name="citation_num_pages" content="33"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.08.28.270652"/>
  <meta name="citation_author" content="Kejie Li"/>
  <meta name="citation_author_institution" content="Research Department, Biogen, Inc."/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2134-3857"/>
  <meta name="citation_author" content="Zhengyu Ouyang"/>
  <meta name="citation_author_institution" content="BioInfoRx, Inc."/>
  <meta name="citation_author" content="Yirui Chen"/>
  <meta name="citation_author_institution" content="Research Department, Biogen, Inc."/>
  <meta name="citation_author" content="Jacob Gagnon"/>
  <meta name="citation_author_institution" content="Research Department, Biogen, Inc."/>
  <meta name="citation_author" content="Dongdong Lin"/>
  <meta name="citation_author_institution" content="Research Department, Biogen, Inc."/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5645-4798"/>
  <meta name="citation_author" content="Michael Mingueneau"/>
  <meta name="citation_author_institution" content="Research Department, Biogen, Inc."/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3873-7329"/>
  <meta name="citation_author" content="Will Chen"/>
  <meta name="citation_author_institution" content="Research Department, Biogen, Inc."/>
  <meta name="citation_author" content="David Sexton"/>
  <meta name="citation_author_institution" content="Research Department, Biogen, Inc."/>
  <meta name="citation_author" content="Baohong Zhang"/>
  <meta name="citation_author_institution" content="Research Department, Biogen, Inc."/>
  <meta name="citation_author_email" content="baohong.zhang@biogen.com"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4735-4212"/>
  <meta name="citation_reference" content="Tang F, et al. mRNA-Seq whole-transcriptome analysis of a single cell. Nature Methods 6, 377–382 (2009)."/>
  <meta name="citation_reference" content="Svensson V, da Veiga Beltrame E, Pachter L. A curated database reveals trends in single-cell transcriptomics. Database 2020, (2020)."/>
  <meta name="citation_reference" content="Cao J, et al. A human cell atlas of fetal gene expression. Science 370, eaba7721 (2020)."/>
  <meta name="citation_reference" content="Megill C, et al. cellxgene: a performant, scalable exploration platform for high dimensional sparse matrices. bioRxiv, 2021.2004.2005.438318 (2021)."/>
  <meta name="citation_reference" content="Cakir B, Prete M, Huang N, van Dongen S, Pir P, Kiselev Vladimir Y. Comparison of visualization tools for single-cell RNAseq data. NAR Genomics and Bioinformatics 2, (2020)."/>
  <meta name="citation_reference" content="Asp M, Bergenstråhle J, Lundeberg J. Spatially resolved transcriptomes—next generation tools for tissue exploration. BioEssays 42, 1900221 (2020)."/>
  <meta name="citation_reference" content="Codeluppi S, et al. Spatial organization of the somatosensory cortex revealed by osmFISH. Nature methods 15, 932–935 (2018)."/>
  <meta name="citation_reference" content="Chen KH, Boettiger AN, Moffitt JR, Wang S, Zhuang X. Spatially resolved, highly multiplexed RNA profiling in single cells. Science 348, (2015)."/>
  <meta name="citation_reference" content="Ståhl PL, et al. Visualization and analysis of gene expression in tissue sections by spatial transcriptomics. Science 353, 78–82 (2016)."/>
  <meta name="citation_reference" content="Eng C-HL, et al. Transcriptome-scale super-resolved imaging in tissues by RNA seqFISH+. Nature 568, 235–239 (2019)."/>
  <meta name="citation_reference" content="Toldra J. American Association for Cancer Research (AACR)-110th Annual Meeting. Atlanta, Georgia, USA-March 29-April 3, 2019. In: American Association for Cancer Research (AACR)-110th Annual Meeting. Atlanta) (2019)."/>
  <meta name="citation_reference" content="Rodriques SG, et al. Slide-seq: A scalable technology for measuring genome-wide expression at high spatial resolution. Science 363, 1463–1467 (2019)."/>
  <meta name="citation_reference" content="Palla G, Fischer DS, Regev A, Theis FJ. Spatial components of molecular tissue biology. Nature Biotechnology 40, 308–318 (2022)."/>
  <meta name="citation_reference" content="Moses L, Pachter L. Museum of spatial transcriptomics. Nature Methods, (2022)."/>
  <meta name="citation_reference" content="Palla G, et al. Squidpy: a scalable framework for spatial single cell analysis. bioRxiv, 2021.2002.2019.431994 (2021)."/>
  <meta name="citation_reference" content="Dries R, et al. Giotto: a toolbox for integrative analysis and visualization of spatial expression data. Genome Biology 22, 78 (2021)."/>
  <meta name="citation_reference" content="Hao Y, et al. Integrated analysis of multimodal single-cell data. bioRxiv, 2020.2010.2012.335331 (2020)."/>
  <meta name="citation_reference" content="Stuart T, et al. Comprehensive integration of single-cell data. Cell 177, 1888–1902. e1821 (2019)."/>
  <meta name="citation_reference" content="Bergenstraahle JL, Larsson. STUtility: Visualization and analysis tools for Spatial Transcriptomics data. Available at https://githubcom/jbergenstrahle/STUtility, ([Accessed: 6 April 2021])."/>
  <meta name="citation_reference" content="Stuart T, Srivastava A, Lareau C, Satija R. Multimodal single-cell chromatin analysis with Signac. bioRxiv, 2020.2011.2009.373613 (2020)."/>
  <meta name="citation_reference" content="Wolf FA, Angerer P, Theis FJ. SCANPY: large-scale single-cell gene expression data analysis. Genome Biology 19, 15 (2018)."/>
  <meta name="citation_reference" content="Li K, Hurt J, Whelan CD, Challa R, Lin D, Zhang B. BioInfograph: An Online Tool to Design and Display Multi-Panel Scientific Figure Interactively. Front Genet 12, 784531 (2021)."/>
  <meta name="citation_reference" content="Schirmer L, et al. Neuronal vulnerability and multilineage diversity in multiple sclerosis. Nature 573, 75–82 (2019)."/>
  <meta name="citation_reference" content="David S. Fischer FRH. Fast and scalable differential expression analysis on single-cell RNA-seq data. Available at https://githubcom/theislab/diffxpy, ([Accessed: 8 May 2020])."/>
  <meta name="citation_reference" content="Korotkevich G, Sukhov V, Budin N, Shpak B, Artyomov MN, Sergushichev A. Fast gene set enrichment analysis. bioRxiv, 060012 (2021)."/>
  <meta name="citation_reference" content="Brooks ME, et al. glmmTMB balances speed and flexibility among packages for zero-inflated generalized linear mixed modeling. The R journal 9, 378–400 (2017)."/>
  <meta name="citation_reference" content="He L, Kulminski AM. NEBULA: a fast negative binomial mixed model for differential expression and co-expression analyses of large-scale multi-subject single-cell data. bioRxiv, 2020.2009.2024.311662 (2020)."/>
  <meta name="citation_reference" content="Ochoa D, et al. Open Targets Platform: supporting systematic drug-target identification and prioritisation. Nucleic Acids Research 49, D1302–D1310 (2021)."/>
  <meta name="citation_reference" content="Absinta M, et al. A lymphocyte-microglia-astrocyte axis in chronic active multiple sclerosis. Nature 597, 709–714 (2021)."/>
  <meta name="citation_reference" content="Reich DS, Lucchinetti CF, Calabresi PA. Multiple Sclerosis. New England Journal of Medicine 378, 169–180 (2018)."/>
  <meta name="citation_reference" content="Kuhlmann T, Ludwin S, Prat A, Antel J, Brück W, Lassmann H. An updated histological classification system for multiple sclerosis lesions. Acta neuropathologica 133, 13–24 (2017)."/>
  <meta name="citation_reference" content="Dal-Bianco A, et al. Slow expansion of multiple sclerosis iron rim lesions: pathology and 7 T magnetic resonance imaging. Acta neuropathologica 133, 25–42 (2017)."/>
  <meta name="citation_reference" content="Absinta M, et al. Persistent 7-tesla phase rim predicts poor outcome in new multiple sclerosis patient lesions. The Journal of clinical investigation 126, 2597–2609 (2016)."/>
  <meta name="citation_reference" content="Bagnato F, et al. Tracking iron in multiple sclerosis: a combined imaging and histopathological study at 7 Tesla. Brain 134, 3602–3615 (2011)."/>
  <meta name="citation_reference" content="He L, Davila-Velderrain J, Sumida TS, Hafler DA, Kellis M, Kulminski AM. NEBULA is a fast negative binomial mixed model for differential or co-expression analysis of large-scale multisubject single-cell data. Commun Biol 4, 629 (2021)."/>
  <meta name="citation_reference" content="Mollie E. Brooks KK, Koen J. van Benthem, Arni Magnusson, Casper W. Berg, Anders Nielsen, Hans J. Skaug, Martin Mächler, Benjamin M. Bolker. glmmTMB Balances Speed and Flexibility Among Packages for Zero-inflated Generalized Linear Mixed Modeling. The R Journal 9, 378–400 (2017)."/>
  <meta name="citation_reference" content="SoRelle ED, et al. Single-cell RNA-seq reveals transcriptomic heterogeneity mediated by hostpathogen dynamics in lymphoblastoid cell lines. eLife 10, e62586 (2021)."/>
  <meta name="citation_reference" content="Hoffman JA, Papas BN, Trotter KW, Archer TK. Single-cell RNA sequencing reveals a heterogeneous response to Glucocorticoids in breast cancer cells. Communications Biology 3, 126 (2020)."/>
  <meta name="citation_reference" content="Hwang B, Lee JH, Bang D. Single-cell RNA sequencing technologies and bioinformaticspipelines. Experimental & Molecular Medicine 50, 1–14 (2018)."/>
  <meta name="citation_reference" content="Lareau CA, et al. Droplet-based combinatorial indexing for massive-scale single-cell chromatin accessibility. Nature Biotechnology 37, 916–924 (2019)."/>
  <meta name="citation_reference" content="Buenrostro JD, et al. Single-cell chromatin accessibility reveals principles of regulatory variation. Nature 523, 486–490 (2015)."/>
  <meta name="citation_reference" content="Buenrostro JD, Wu B, Chang HY, Greenleaf WJ. ATAC-seq: A Method for Assaying Chromatin Accessibility Genome-Wide. Curr Protoc Mol Biol 109, 21.29.21–21.29.29 (2015)."/>
  <meta name="citation_reference" content="Ranzoni AM, et al. Integrative Single-Cell RNA-Seq and ATAC-Seq Analysis of Human Developmental Hematopoiesis. Cell stem cell 28, 472–487.e477 (2021)."/>
  <meta name="citation_reference" content="Muto Y, et al. Single cell transcriptional and chromatin accessibility profiling redefine cellular heterogeneity in the adult human kidney. Nature Communications 12, 2190 (2021)."/>
  <meta name="citation_reference" content="Lareau CA, et al. Massively parallel single-cell mitochondrial DNA genotyping and chromatin profiling. Nature Biotechnology 39, 451–461 (2021)."/>
  <meta name="citation_reference" content="Single Cell Mulitome ATAC + Gene Expression Dataset by Cell Ranger ARC 2.0.0, May 3rd) PBMC from a healthy donor - granulocytes removed through cell sorting (10k). Available at https://www10xgenomicscom/resources/datasets/pbmc-from-a-healthy-donor-granulocytes-removed-through-cell-sorting-10-k-1-standard-2-0-0, (2021)."/>
  <meta name="citation_reference" content="Korsunsky I, et al. Fast, sensitive and accurate integration of single-cell data with Harmony. Nature Methods 16, 1289–1296 (2019)."/>
  <meta name="citation_reference" content="Welch JD, Kozareva V, Ferreira A, Vanderburg C, Martin C, Macosko EZ. Single-cell multi-omic integration compares and contrasts features of brain cell identity. Cell 177, 1873–1887. e1817 (2019)."/>
  <meta name="citation_reference" content="Blondel VD, Guillaume J-L, Lambiotte R, Lefebvre E. Fast unfolding of communities in large networks. Journal of statistical mechanics: theory and experiment 2008, P10008 (2008)."/>
  <meta name="citation_reference" content="Maaten Lvd, Hinton G. Visualizing data using t-SNE. Journal of machine learning research 9, 2579–2605 (2008)."/>
  <meta name="citation_reference" content="Becht E, et al. Dimensionality reduction for visualizing single-cell data using UMAP. Nature biotechnology 37, 38–44 (2019)."/>
  <meta name="citation_reference" content="Kanehisa M, Goto S. KEGG: kyoto encyclopedia of genes and genomes. Nucleic acids research 28, 27–30 (2000)."/>
  <meta name="citation_reference" content="Martens M, et al. WikiPathways: connecting communities. Nucleic Acids Research 49, D613–D621 (2021)."/>
  <meta name="citation_reference" content="Frolkis A, et al. SMPDB: the small molecule pathway database. Nucleic acids research 38, D480–D487 (2010)."/>
  <meta name="citation_reference" content="Fabregat A, et al. The reactome pathway knowledgebase. Nucleic acids research 46, D649–D655 (2018)."/>
  <meta name="citation_reference" content="Consortium GO. The gene ontology resource: 20 years and still GOing strong. Nucleic acids research 47, D330–D338 (2019)."/>
  <meta name="citation_reference" content="Liberzon A, Subramanian A, Pinchback R, Thorvaldsdóttir H, Tamayo P, Mesirov JP. Molecular signatures database (MSigDB) 3.0. Bioinformatics 27, 1739–1740 (2011)."/>
  <meta name="citation_reference" content="Subramanian A, et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proceedings of the National Academy of Sciences 102, 15545–15550 (2005)."/>
  <meta name="citation_reference" content="Cotter D, Maer A, Guda C, Saunders B, Subramaniam S. Lmpd: lipid maps proteome database. Nucleic acids research 34, D507–D510 (2006)."/>
  <meta name="citation_reference" content="Traag VA, Waltman L, Van Eck NJ. From Louvain to Leiden: guaranteeing well-connected communities. Scientific reports 9, 1–12 (2019)."/>
  <meta name="citation_reference" content="Umesh P. Image processing in python. CSI Communications 23, 2 (2012)."/>
  <meta name="citation_reference" content="Hafemeister C, Satija R. Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression. Genome Biology 20, 296 (2019)."/>
  <meta name="citation_reference" content="Tim Stuart WWK. Sinto: single-cell analysis tools. Available at https://githubcom/timoast/sinto, ([Accessed: Nov 2021])."/>
  <meta name="citation_reference" content="Ramírez F, et al. deepTools2: a next generation web server for deep-sequencing data analysis. Nucleic acids research 44, W160–W165 (2016)."/>
  <meta name="citation_reference" content="Xie Y. bookdown: Authoring Books and Technical Documents with R Markdown. R package version 0.24. Available at https://githubcom/rstudio/bookdown, (2021)."/>
  <meta name="citation_reference" content="citation_title=Cellxgene VIP snRNA-seq demo dataset for visualization and DE analysis;citation_year=2022"/>
  <meta name="citation_reference" content="citation_title=10X Genomics Human Visium Spatial Transcriptomics Demo Dataset for Cellxgene VIP;citation_year=2021"/>
  <meta name="twitter:title" content="Cellxgene VIP unleashes full power of interactive visualization and integrative analysis of scRNA-seq, spatial transcriptomics, and multiome data"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="To meet the growing demands from scientists to effectively extract deep insights from single cell RNA sequencing, spatial transcriptomics, and emerging multiome datasets, we developed cellxgene VIP (Visualization In Plugin), a frontend interactive visualization plugin of cellxgene framework, which greatly expanded capabilities of the base tool in the following aspects. First, it generates a comprehensive set of over eighteen commonly used quality control and analytical plots in high resolution with highly customizable settings in real time. Second, it provides more advanced analytical functions to gain insights on cellular compositions and deep biology, such as marker gene identification, differential gene expression analysis, and gene set enrichment analysis. Third, it empowers advanced users to perform analysis in a Jupyter Notebook like environment, dubbed Command Line Interface (CLI) by programming in Python and/or R directly without limiting themselves to functional modules available via graphical user interface (GUI). Finally, it pioneers methods to visualize multi-modal data, such as spatial transcriptomics embedding aligned with histological image on one slice or multiple slices in a grid format, and the latest 10x Genomic Multiome dataset where both DNA accessibility and gene expression in the same cells are measured, under the same framework in an integrative way to fully leverage the functionalities mentioned above. Taken together, the open-source tool makes large scale single cell data visualization and analysis more accessible to biologists in a user-friendly manner and fosters computational reproducibility by simplifying data and code reuse through the CLI. Going forward, it has the potential to become a crowdsourcing ecosystem for the scientific community to contribute even more modules to the Swiss Army knife of single cell data exploration tools.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Cellxgene VIP unleashes full power of interactive visualization and integrative analysis of scRNA-seq, spatial transcriptomics, and multiome data"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.08.28.270652v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="To meet the growing demands from scientists to effectively extract deep insights from single cell RNA sequencing, spatial transcriptomics, and emerging multiome datasets, we developed cellxgene VIP (Visualization In Plugin), a frontend interactive visualization plugin of cellxgene framework, which greatly expanded capabilities of the base tool in the following aspects. First, it generates a comprehensive set of over eighteen commonly used quality control and analytical plots in high resolution with highly customizable settings in real time. Second, it provides more advanced analytical functions to gain insights on cellular compositions and deep biology, such as marker gene identification, differential gene expression analysis, and gene set enrichment analysis. Third, it empowers advanced users to perform analysis in a Jupyter Notebook like environment, dubbed Command Line Interface (CLI) by programming in Python and/or R directly without limiting themselves to functional modules available via graphical user interface (GUI). Finally, it pioneers methods to visualize multi-modal data, such as spatial transcriptomics embedding aligned with histological image on one slice or multiple slices in a grid format, and the latest 10x Genomic Multiome dataset where both DNA accessibility and gene expression in the same cells are measured, under the same framework in an integrative way to fully leverage the functionalities mentioned above. Taken together, the open-source tool makes large scale single cell data visualization and analysis more accessible to biologists in a user-friendly manner and fosters computational reproducibility by simplifying data and code reuse through the CLI. Going forward, it has the potential to become a crowdsourcing ecosystem for the scientific community to contribute even more modules to the Swiss Army knife of single cell data exploration tools.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-04-14"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.08.28.270652v2"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2496640"/>
  <title>
   Cellxgene VIP unleashes full power of interactive visualization and integrative analysis of scRNA-seq, spatial transcriptomics, and multiome data | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"aMKmDntLBnnIEx6n92Ib0Qhcbb3dNxC4oiI3wiEtQ14"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2496640","apath":"\/biorxiv\/early\/2022\/04\/14\/2020.08.28.270652.atom","pisa":"biorxiv;2020.08.28.270652v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.08.28.270652v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.08.28.270652v2","title":"Cellxgene VIP unleashes full power of interactive visualization and integrative analysis of scRNA-seq, spatial transcriptomics, and multiome data","identifier":"node\/2496640"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+n8A\\\\\\\/oQlczhciqHpLW4yDvuWfSiq+HY6qFuVwU\\\\\\\/CyePDf\\\\\\\/1CHhdE9TcutOU1peY5MBpoi1YTwpRRWpHH2rmA4l4BFWNKxO5q9Xy2ZimzJN8xuy3JtsvY5KKBVU9h+WmTAdg9ggY+qPB7IZ79BHMh\\\\\\\/LxnZYIAFlL7Yvpk1XWV4+fR23F9dIguSHg\\\\\\\/5QrNgCBSZr7svWkGfDNoAIdLwnMZloh5OrJyUp4TkFaqcEMKvijILMwNfNwYZX9GL0QI4FNZ8Ly0dcDHoQ5UBU3tiJcTBaSQPZtxRzhd\\\\\\\/luAKHhBjDB5fcp8pYNMZZQK23pfP+y7INgZJW4GG98EoSMGWkf4DJOWI0YfAyoLn1bEdKOqJWRt\\\\\\\/VUprq\\\\\\\/JidhYZVSnSsDzunOWsrjK3WCt7PLufjO1Zc7Msa\\\\\\\/qyAP4Ih\\\\\\\/uou+2zJof6\\\\\\\/4rOAFFEj2qFUfQjUurFuV5cRB1eRJbqQDAOnIr51Lwx7KZR1mIqK18bvNRS9XFAJU0C4j38iDkM0CFa0ThzE0ykkgLdC3W5KFFbKZmK5aXeHNlC+ysCJUua8m6dSurkvTiS3Z5bc6GPe9a0qhXTXP\\\\\\\/c4dsP9twABkz1oRKhPXkscFcZqz0eBE\\\\\\\/9vfXE2g2zDsCy994C8g0zQoNu5+UMfz0\\\\\\\/vReK71DB0\\\\\\\/FZ6ZYnDXOmVogGBS\\\\\\\/vVl5uDvvOFN9\\\\\\\/nOqKZW4d1wh9BaLT8FgPdyWa1fx71sUMspastfEoVqEwRf8R5SUikkjwGOOFs8Ejy06XJLy14Ov3c4jhrSeYFABTVBAH4SbFhYjMXjOSutHWdil+52WtSAngnKl3gmShbeUTXVRS8r1KRR5FPtlfw8Jh2kBXo8ZC7DAGqB2+fCCFboMKUblqObyC+kFjihD4lqKlXJSwR1y7Aygk6Nwr6OCEzYW7kdLOZA1k3egi7EZYKo3PQANvdEWItMyECN+zt5IVUrXdU7bMR0irGXpfUjIcpFpbiQYYM6iJIRC2GhswItqO3oKLtziFxY6N+MPENa6bhrN2FUrr8ppDxu8Khpis5tbwSNjveNV5O++NY6rjxg3c5KLh1FoIIf2zwQ3CBMadxfB8dvSZU4ioaGWNXZmGWzMBY3UrQk9LtNf7CXuTZrTi2aCLRqOXf9QO5YLHzTF+upSg7DPuLOvS0JfmdUObjBWrun0sF\\\\\\\/6dNiJnqr8C4PopaJYsPGZ1y8YWIqXteTOVgBU8Cm63+0edI2vTYoFdM7SNGcn65yKvR69Q3tEfac\\\\\\\/vELnO974pNwzElIyYzM4O6Ymy6NuH1rGlqU9HuMCitcERjAHtM29VlXhjbpaUkPpqXtdpJT64KIfqjv+v54TPWYN9XfZXsM9HBy6opxDmA8ETP77W4SQb9MvZ2JF5HcJ2CFi00hSyx2LbTo=\\\u0022,\\\u0022iv\\\u0022:\\\u00221ocrAgA\\\\\\\/C5217wzR0R4ARg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022ee11105a1836541248b772361e501462d7e8d6de62f51feeb79b2964adcb05c2\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.08.28.270652v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2496640 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.08.28.270652v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_993323211">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_993323211" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_362861761" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-9mUmE6bAFvSC6x7Ju6jCEw1Br-oWPaRZMiDSX7kF8ow"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2496640" data-node-nid="2496640" id="node-2496640--2700651758" data-pisa="biorxiv;2020.08.28.270652v2" data-pisa-master="biorxiv;2020.08.28.270652" data-apath="/biorxiv/early/2022/04/14/2020.08.28.270652.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Cellxgene VIP unleashes full power of interactive visualization and integrative analysis of scRNA-seq, spatial transcriptomics, and multiome data
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0003-2134-3857" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Kejie
                      </span>
                      <span class="nlm-surname">
                       Li
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Zhengyu
                      </span>
                      <span class="nlm-surname">
                       Ouyang
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Yirui
                      </span>
                      <span class="nlm-surname">
                       Chen
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Jacob
                      </span>
                      <span class="nlm-surname">
                       Gagnon
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4">
                      <a href="http://orcid.org/0000-0001-5645-4798" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Dongdong
                      </span>
                      <span class="nlm-surname">
                       Lin
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5">
                      <a href="http://orcid.org/0000-0002-3873-7329" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Michael
                      </span>
                      <span class="nlm-surname">
                       Mingueneau
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="6">
                      <span class="nlm-given-names">
                       Will
                      </span>
                      <span class="nlm-surname">
                       Chen
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="7">
                      <span class="nlm-given-names">
                       David
                      </span>
                      <span class="nlm-surname">
                       Sexton
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8">
                      <a href="http://orcid.org/0000-0003-4735-4212" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Baohong
                      </span>
                      <span class="nlm-surname">
                       Zhang
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.08.28.270652
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2496640--2700651758" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Kejie Li
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Research Department, Biogen, Inc.
                       </span>
                       , 225 Binney St, Cambridge, MA 02142,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Kejie%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Li%20K&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AKejie%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-2134-3857" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Kejie Li
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Zhengyu Ouyang
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        BioInfoRx, Inc.
                       </span>
                       , 510 Charmany Dr, Suite 275A, Madison, WI 53719,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Zhengyu%2BOuyang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Ouyang%20Z&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AZhengyu%2BOuyang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Yirui Chen
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Research Department, Biogen, Inc.
                       </span>
                       , 225 Binney St, Cambridge, MA 02142,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Yirui%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Chen%20Y&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AYirui%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Jacob Gagnon
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Research Department, Biogen, Inc.
                       </span>
                       , 225 Binney St, Cambridge, MA 02142,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Jacob%2BGagnon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Gagnon%20J&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AJacob%2BGagnon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Dongdong Lin
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Research Department, Biogen, Inc.
                       </span>
                       , 225 Binney St, Cambridge, MA 02142,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Dongdong%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Lin%20D&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ADongdong%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-5645-4798" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Dongdong Lin
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Michael Mingueneau
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Research Department, Biogen, Inc.
                       </span>
                       , 225 Binney St, Cambridge, MA 02142,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Michael%2BMingueneau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Mingueneau%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AMichael%2BMingueneau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-3873-7329" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Michael Mingueneau
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">
                     Will Chen
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Research Department, Biogen, Inc.
                       </span>
                       , 225 Binney St, Cambridge, MA 02142,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Will%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Chen%20W&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AWill%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-7">
                    <div class="author-tooltip-name">
                     David Sexton
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Research Department, Biogen, Inc.
                       </span>
                       , 225 Binney St, Cambridge, MA 02142,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=David%2BSexton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Sexton%20D&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ADavid%2BSexton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-8">
                    <div class="author-tooltip-name">
                     Baohong Zhang
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Research Department, Biogen, Inc.
                       </span>
                       , 225 Binney St, Cambridge, MA 02142,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Baohong%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Zhang%20B&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ABaohong%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0003-4735-4212" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Baohong Zhang
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:baohong.zhang@biogen.com" class="" data-icon-position="" data-hide-link-title="0">
                        baohong.zhang@biogen.com
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.08.28.270652v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2496640" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2496640/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.08.28.270652v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2496640" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2496640/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.08.28.270652v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2496640" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2496640/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.08.28.270652v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2496640" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2496640/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.08.28.270652v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2496640" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2496640/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.08.28.270652v2.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:2496640" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:2496640/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.08.28.270652v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2496640" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2496640/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-4">
                             To meet the growing demands from scientists to effectively extract deep insights from single cell RNA sequencing, spatial transcriptomics, and emerging multiome datasets, we developed cellxgene VIP (Visualization In Plugin), a frontend interactive visualization plugin of cellxgene framework, which greatly expanded capabilities of the base tool in the following aspects. First, it generates a comprehensive set of over eighteen commonly used quality control and analytical plots in high resolution with highly customizable settings in real time. Second, it provides more advanced analytical functions to gain insights on cellular compositions and deep biology, such as marker gene identification, differential gene expression analysis, and gene set enrichment analysis. Third, it empowers advanced users to perform analysis in a Jupyter Notebook like environment, dubbed Command Line Interface (CLI) by programming in Python and/or R directly without limiting themselves to functional modules available via graphical user interface (GUI). Finally, it pioneers methods to visualize multi-modal data, such as spatial transcriptomics embedding aligned with histological image on one slice or multiple slices in a grid format, and the latest 10x Genomic Multiome dataset where both DNA accessibility and gene expression in the same cells are measured, under the same framework in an integrative way to fully leverage the functionalities mentioned above. Taken together, the open-source tool makes large scale single cell data visualization and analysis more accessible to biologists in a user-friendly manner and fosters computational reproducibility by simplifying data and code reuse through the CLI. Going forward, it has the potential to become a crowdsourcing ecosystem for the scientific community to contribute even more modules to the Swiss Army knife of single cell data exploration tools.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-16">
                             Since the first single-cell RNA sequencing (scRNA-seq) study was debuted in 2009
                             <sup>
                              <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                               1
                              </a>
                             </sup>
                             , over 1600 scRNA-seq studies have been published to date, of which at least 72 studies have reported profiles in 200k or more cells
                             <sup>
                              <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                               2
                              </a>
                             </sup>
                             . The largest scRNA-seq study generated gene expression data of 4 million human cells
                             <sup>
                              <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                             </sup>
                             . It is foreseen that there will be a trend of increasing cell size to 500k or more in scRNA-seq studies. The sheer amount of data had brought challenges in visualizing and exploring such big data set interactively for scientists, even computational biologists.
                            </p>
                            <p id="p-17">
                             Cellxgene
                             <sup>
                              <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                             </sup>
                             is a leading open source scRNA-seq data visualization tool recommended in a recent evaluation
                             <sup>
                              <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                             </sup>
                             , which scales well in millions of cells and scores highly in user experience by leveraging modern web techniques with interactive features. Used by a large community of biologists, cellxgene proves itself in meeting the need of handling data interactively at scale. Except it lacks some essential plotting and analysis functions seen in scRNA-seq related command-line tools and publications that biologists are used to, hindering its utility and limiting scientists from taking advantage of ever accumulating scRNA-seq data sets to their full potential.
                            </p>
                            <p id="p-18">
                             While bringing a more-fine grained assessment of each cell’s transcriptome, scRNA-seq loses spatial information during the cell dissociation process from a tissue. To gain deeper understanding of tissue cellular composition and cell-cell communication simultaneously, spatial transcriptomics methods
                             <sup>
                              <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                               6
                              </a>
                             </sup>
                             , e.g.: smFish
                             <sup>
                              <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                               7
                              </a>
                             </sup>
                             , MERFISH
                             <sup>
                              <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                             </sup>
                             , Spatial Transcriptomics
                             <sup>
                              <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                             </sup>
                             , seqFISH+
                             <sup>
                              <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                               10
                              </a>
                             </sup>
                             , NanoString GeoMx
                             <sup>
                              <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                             </sup>
                             , Slide-seq
                             <sup>
                              <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                             </sup>
                             , and 10X Visium
                             <sup>
                              <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                             </sup>
                             are developed to allow the preservation of organization of tissue context and captures potentially cross talk between a variety of cell types within tissue
                             <sup>
                              <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ,
                              <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                             </sup>
                             . As the data grows in both volume and complexity involving histological imaging data and near single cell level transcriptomic measurements, presenting it in a digestible form for biologists is a daunting and challenging task. Currently, free or open-source spatial transcriptomics data visualization tools are either developed for computational experts comfortable of writing code in R or Python (Squidpy
                             <sup>
                              <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                             </sup>
                             , Giotto
                             <sup>
                              <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                             </sup>
                             , Seurat 4.0
                             <sup>
                              <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                               17
                              </a>
                              ,
                              <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                             </sup>
                             , and STUtility
                             <sup>
                              <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                               19
                              </a>
                             </sup>
                             ) to interact with data, or offered as client end software (Loupe Browser from 10x Genomics) that requires installation on a personal computer that has very limited computing power. Although Loupe Browser and Giotto have user-friendly interactive features to overlay an image of stained tissue and gene expression on spatially distributed spots to explore multiple layers of information, none of these packages supports visualizing more than one samples at a time, which is needed for comparing replicates or selecting transcriptionally similar spots from multiple samples for downstream analysis, such as differential expressed gene (DEG) and spatial variable gene (SVG) analysis. Moreover, even for computational biologists, integrating spatial transcriptomics data with other analytical tools seamlessly poses a great challenge due to the limited capabilities of these tools.
                            </p>
                            <p id="p-19">
                             Recently, 10X Genomics introduced multiome technology to simultaneously profile gene expression and open chromatin from the same cell to leverage gene expression markers to aid interpretation of epigenetic profiles, combine discovery of regulatory elements with gene expression to explore gene regulatory interactions, and obtain transcriptome and open chromatin information simultaneously to save samples and effort. Signac
                             <sup>
                              <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                             </sup>
                             from the Seurat
                             <sup>
                              <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">
                               17
                              </a>
                              ,
                              <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                             </sup>
                             family is the only tool we found to visualize this type of multi-modal data in a meaningful way by jointly plotting gene expression of genes and nearby peaks representing chromatin openness. However, bioinformatics skill is needed to master this command line tool.
                            </p>
                            <p id="p-20">
                             To fill these gaps and to empower scientists to investigate these types of big data more efficiently, we developed a plugin of cellxgene named Visualization in Plugin, in short VIP to address the urgent needs of such essential functions for interactive visual exploration of scRNA-seq, emerging spatial transcriptomics and multiome data, and generation of publication-ready figures in a user-friendly way, especially for non-professional data analysts.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Result
                            </h2>
                            <p id="p-21">
                             Cellxgene VIP’s strength lies in versatility, modularity, and scalability. Notably, it enhanced plotting functions significantly to generate violin, stacked violin, stacked bar, heatmap, volcano, embedding, dot, track, density, 2D density, Sankey and dual-gene plot in high-resolution by calling server-side plotting functions provided by Scanpy
                             <sup>
                              <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                               21
                              </a>
                             </sup>
                             , Seurat
                             <sup>
                              <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">
                               17
                              </a>
                              ,
                              <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                             </sup>
                             and other general Python and R plotting libraries as illustrated in
                             <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                              Figure 1
                             </a>
                             and supplementary tutorial. Innovative split plots in embedding (
                             <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                              Figure 1a
                             </a>
                             ) and density plots (
                             <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                              Figure 1g
                             </a>
                             and Supplementary Fig. 13, see
                             <a href="https://bit.ly/3j4SR0a">
                              https://bit.ly/3j4SR0a
                             </a>
                             ) by annotations, such as cell types, make it easier to see which genes are differentially expressed in which cells. With the goal of becoming a platform to bridge users and various tools, cellxgene VIP not only leverages Python analytical capabilities, but also opens to another popular language R, which is demonstrated by a volcano plot (
                             <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                              Figure 4b
                             </a>
                             ). In addition, interactive Sankey diagram (
                             <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                              Figure 1f
                             </a>
                             ) and stacked barplot (
                             <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                              Figure 1c
                             </a>
                             ) based on yet another language, JavaScript, enable investigation of the relationship between metadata and gene expression more intuitively.
                            </p>
                            <div id="F1" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F1.large.jpg?width=800&height=600&carousel=1" title="cellxgene VIP serves as an ecosystem of analytical modules that provide essential functions for interactive visualization and generation of publication-ready plots. Individual plots were assembled by bioInfograph22 with zoomable feature available at https://bit.ly/2QqdMg3 that is best viewed by Chrome. (a) Multi-tSNE/UMAP plot visually highlights which cells express cell markers on selected embedding, e.g., UMAP based on harmony batch correction in this example. (b) Dual-gene plot highlights cells expressing SYT1 and GAD1 (green SYT1 only, red GAD1 only, yellow co-expression of STY1 and GAD1), expression cutoff 2.2. (c) Stacked barplot demonstrates the fraction of each major cell type across each sample (C are Control and MS are MS patients). (d) Trackplot shows expression of lineage marker genes across individual cells in annotated clusters. (e) Violin plot shows the AQP4 gene expression across cell types. (f) Sankey diagram (a.k.a. Riverplot) provides quick and easy way to explore the inter-dependent relationship of variables in the MS snRNA-seq dataset23. (g) Density plots shows expression of marker genes across annotated clusters and split across cell types. (h) Stacked violin and Dot plot are the key visualizations of selected cell markers across cell types. They highlight their selective expression and validates the scRNA-seq approach and visualization method. (i) CLI exposed by mini Jupyter Notebook to provide maximal flexibility of performing various analytics on the whole or sliced single cell dataset." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-463299482" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">cellxgene VIP serves as an ecosystem of analytical modules that provide essential functions for interactive visualization and generation of publication-ready plots.</span> Individual plots were assembled by bioInfograph<sup>22</sup> with zoomable feature available at https://bit.ly/2QqdMg3 that is best viewed by Chrome. (<strong>a</strong>) <strong>Multi-tSNE/UMAP plot</strong> visually highlights which cells express cell markers on selected embedding, e.g., UMAP based on harmony batch correction in this example. (<strong>b</strong>) <strong>Dual-gene plot</strong> highlights cells expressing SYT1 and GAD1 (green SYT1 only, red GAD1 only, yellow co-expression of STY1 and GAD1), expression cutoff 2.2. (<strong>c</strong>) <strong>Stacked barplot</strong> demonstrates the fraction of each major cell type across each sample (C are Control and MS are MS patients). (<strong>d</strong>) <strong>Trackplot</strong> shows expression of lineage marker genes across individual cells in annotated clusters. (<strong>e</strong>) <strong>Violin plot</strong> shows the AQP4 gene expression across cell types. (<strong>f</strong>) <strong>Sankey diagram</strong> (a.k.a. Riverplot) provides quick and easy way to explore the inter-dependent relationship of variables in the MS snRNA-seq dataset<sup>23</sup>. (<strong>g</strong>) <strong>Density plots</strong> shows expression of marker genes across annotated clusters and split across cell types. (<strong>h</strong>) <strong>Stacked violin</strong> and <strong>Dot plot</strong> are the key visualizations of selected cell markers across cell types. They highlight their selective expression and validates the scRNA-seq approach and visualization method. (<strong>i</strong>) <strong>CLI</strong> exposed by mini Jupyter Notebook to provide maximal flexibility of performing various analytics on the whole or sliced single cell dataset.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F1.medium.gif" width="440" height="383"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F1.medium.gif" width="440" height="383"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                              <span class="fig-label">
                               Figure 1
                              </span>
                              <span class="caption-title">
                               cellxgene VIP serves as an ecosystem of analytical modules that provide essential functions for interactive visualization and generation of publication-ready plots.
                              </span>
                              <p id="p-22" class="first-child">
                               Individual plots were assembled by bioInfograph
                               <sup>
                                <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                                 22
                                </a>
                               </sup>
                               with zoomable feature available at
                               <a href="https://bit.ly/2QqdMg3">
                                https://bit.ly/2QqdMg3
                               </a>
                               that is best viewed by Chrome. (
                               <strong>
                                a
                               </strong>
                               )
                               <strong>
                                Multi-tSNE/UMAP plot
                               </strong>
                               visually highlights which cells express cell markers on selected embedding, e.g., UMAP based on harmony batch correction in this example. (
                               <strong>
                                b
                               </strong>
                               )
                               <strong>
                                Dual-gene plot
                               </strong>
                               highlights cells expressing SYT1 and GAD1 (green SYT1 only, red GAD1 only, yellow co-expression of STY1 and GAD1), expression cutoff 2.2. (
                               <strong>
                                c
                               </strong>
                               )
                               <strong>
                                Stacked barplot
                               </strong>
                               demonstrates the fraction of each major cell type across each sample (C are Control and MS are MS patients). (
                               <strong>
                                d
                               </strong>
                               )
                               <strong>
                                Trackplot
                               </strong>
                               shows expression of lineage marker genes across individual cells in annotated clusters. (
                               <strong>
                                e
                               </strong>
                               )
                               <strong>
                                Violin plot
                               </strong>
                               shows the AQP4 gene expression across cell types. (
                               <strong>
                                f
                               </strong>
                               )
                               <strong>
                                Sankey diagram
                               </strong>
                               (a.k.a. Riverplot) provides quick and easy way to explore the inter-dependent relationship of variables in the MS snRNA-seq dataset
                               <sup>
                                <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                                 23
                                </a>
                               </sup>
                               . (
                               <strong>
                                g
                               </strong>
                               )
                               <strong>
                                Density plots
                               </strong>
                               shows expression of marker genes across annotated clusters and split across cell types. (
                               <strong>
                                h
                               </strong>
                               )
                               <strong>
                                Stacked violin
                               </strong>
                               and
                               <strong>
                                Dot plot
                               </strong>
                               are the key visualizations of selected cell markers across cell types. They highlight their selective expression and validates the scRNA-seq approach and visualization method. (
                               <strong>
                                i
                               </strong>
                               )
                               <strong>
                                CLI
                               </strong>
                               exposed by mini Jupyter Notebook to provide maximal flexibility of performing various analytics on the whole or sliced single cell dataset.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-23">
                             To emphasize simplicity and flexibility, we paid great attention to user-friendliness by crafting the plugin to allow users to customize analysis and plotting parameters, set figure options like font size, image format and resolution, and pick a color palette for embedding as shown in
                             <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                              Figure 2
                             </a>
                             . To further strengthen the framework’s usability, the following convenient features are implemented such as displaying numbers of a brushed range, increasing the width of the central canvas to enlarge the working space, showing the description of the data set on the main window, and initializing the look-n-feel by a remote session file. Often, it is a challenge if not impossible to select interested cells by the cellxgene lasso tool in some situations, such as in ‘Spatial Transcriptomics’ where cells in a cluster could be separated in the space embedding, here the user can enable the cumulative selections in the global setting to select spots segment by segment. In another case, the user can turn on the scaling option to transform data to unit variance per gene (not zero mean due to the efficiency to handle sparse matrix) by calculating z-scores of the expression data using Scanpy’s scale function when it is appropriate. Further options of including genes with no variance and/or clipping at a threshold are provided for data scaling.
                            </p>
                            <div id="F2" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F2.large.jpg?width=800&height=600&carousel=1" title="Main Window, VIP Window and setting parameters for figure plotting in Global Setting. Notable operations and features are highlighted in red call-out boxes. VIP windows can be fired up, minimized, moved, resized, and brought into scene by corresponding mouse actions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-463299482" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Main Window, VIP Window and setting parameters for figure plotting in Global Setting.</span> Notable operations and features are highlighted in red call-out boxes. VIP windows can be fired up, minimized, moved, resized, and brought into scene by corresponding mouse actions.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F2.medium.gif" width="440" height="249"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F2.medium.gif" width="440" height="249"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure 2
                              </span>
                              <span class="caption-title">
                               Main Window, VIP Window and setting parameters for figure plotting in Global Setting.
                              </span>
                              <p id="p-24" class="first-child">
                               Notable operations and features are highlighted in red call-out boxes. VIP windows can be fired up, minimized, moved, resized, and brought into scene by corresponding mouse actions.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-25">
                             To help biologists visualize and analyze 10X Visium Spatial Transcriptomics data, especially multiple samples/slices dataset, we developed a module in VIP to overlay the spatial spots on top of the Hematoxylin and Eosin (H&E) stained image (
                             <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                              Figure 3a
                             </a>
                             ) and display gene expression of one or multiple genes on the spatial coordinates while other existing single cell visualizations are only applicable to spatial coordinate data. In VIP, the transparency of the H&E image or spatial spots can be adjusted to improve the visualization of the histology image, to zoom into specific regions of interest to allow better molecular insights under spatial context, to view derived layers/classification of glass slide spots with tSNE/UMAP, to allow manual cumulative selection of spots (
                             <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                              Figure 3b
                             </a>
                             ) within regions of interest as groups or clusters by lasso tool, and to find differentially expressed genes between 2 regions occupied by selected spots. Besides overlaid visualization, viewing embedding image and gene expression profile side-by-side (
                             <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                              Figure 3c
                             </a>
                             ) allows scientists to discover subtle histological features that usually are hidden in an overlaid setting. In addition, users can design the layout of sample slides to be merged by interactively changing grid location, rotation, and flipping of individual slides (
                             <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                              Figure 3d
                             </a>
                             ) and then save the newly customized layout file for merging h5ad files by a Python script detailed in Method section.
                            </p>
                            <div id="F3" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F3.large.jpg?width=800&height=600&carousel=1" title="Advanced features of visualizing and processing of spatial transcriptomics data accessed by “Spatial transcriptomics” menu. (a) Overlapped view of merged spatial embeddings colored by clusters and histological images of multiple samples. The image can be loaded or removed by clicking on “Get Spatial Image” or “Remove Image” button, respectively. Overlapping order, opacity of embedding and image can be adjusted to obtain the optimal view. (b) Cumulative selection option is chosen in “Global Setting” to allow selections of cells from non-adjacent regions or multiple samples as shown by the red and blue irregular shapes drawn by lasso selection tool. Please note that numbers of cells in the main window refer to the current selection while numbers of cells in VIP window denote accumulated selections by multiple lasso operations. Downstream analysis such as differential gene expression can be performed between the accumulated cell groups. (c) Embedding and image could be reviewed separately by selecting “No” to “Move embedding and image together”. This way, subtle histological features obstructed in overlaid setting can be seen easily. (d) “Layout” menu provides a flexible way to re-arrange a slide by moving, rotating, flipping on X or Y coordinates as the original setting of the slide might not be the desired one, e.g., the third slide on the top row is rotated by 90 degrees and flipped on Y. A slide is activated by clicking on it or removed from the layout by clicking “X” on the top. In the end, the layout can be saved and used to create the custom merged h5ad file based on the on-screen design." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-463299482" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Advanced features of visualizing and processing of spatial transcriptomics data accessed by “Spatial transcriptomics” menu.</span> (a) Overlapped view of merged spatial embeddings colored by clusters and histological images of multiple samples. The image can be loaded or removed by clicking on “Get Spatial Image” or “Remove Image” button, respectively. Overlapping order, opacity of embedding and image can be adjusted to obtain the optimal view. (b) Cumulative selection option is chosen in “Global Setting” to allow selections of cells from non-adjacent regions or multiple samples as shown by the red and blue irregular shapes drawn by lasso selection tool. Please note that numbers of cells in the main window refer to the current selection while numbers of cells in VIP window denote accumulated selections by multiple lasso operations. Downstream analysis such as differential gene expression can be performed between the accumulated cell groups. (c) Embedding and image could be reviewed separately by selecting “No” to “Move embedding and image together”. This way, subtle histological features obstructed in overlaid setting can be seen easily. (d) “Layout” menu provides a flexible way to re-arrange a slide by moving, rotating, flipping on X or Y coordinates as the original setting of the slide might not be the desired one, e.g., the third slide on the top row is rotated by 90 degrees and flipped on Y. A slide is activated by clicking on it or removed from the layout by clicking “X” on the top. In the end, the layout can be saved and used to create the custom merged h5ad file based on the on-screen design.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F3.medium.gif" width="440" height="273"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F3.medium.gif" width="440" height="273"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure 3
                              </span>
                              <span class="caption-title">
                               Advanced features of visualizing and processing of spatial transcriptomics data accessed by “Spatial transcriptomics” menu.
                              </span>
                              <p id="p-26" class="first-child">
                               (a) Overlapped view of merged spatial embeddings colored by clusters and histological images of multiple samples. The image can be loaded or removed by clicking on “Get Spatial Image” or “Remove Image” button, respectively. Overlapping order, opacity of embedding and image can be adjusted to obtain the optimal view. (b) Cumulative selection option is chosen in “Global Setting” to allow selections of cells from non-adjacent regions or multiple samples as shown by the red and blue irregular shapes drawn by lasso selection tool. Please note that numbers of cells in the main window refer to the current selection while numbers of cells in VIP window denote accumulated selections by multiple lasso operations. Downstream analysis such as differential gene expression can be performed between the accumulated cell groups. (c) Embedding and image could be reviewed separately by selecting “No” to “Move embedding and image together”. This way, subtle histological features obstructed in overlaid setting can be seen easily. (d) “Layout” menu provides a flexible way to re-arrange a slide by moving, rotating, flipping on X or Y coordinates as the original setting of the slide might not be the desired one, e.g., the third slide on the top row is rotated by 90 degrees and flipped on Y. A slide is activated by clicking on it or removed from the layout by clicking “X” on the top. In the end, the layout can be saved and used to create the custom merged h5ad file based on the on-screen design.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-27">
                             Beyond the visualization functions, we integrated diffxpy
                             <sup>
                              <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                               24
                              </a>
                             </sup>
                             to provide more analytical capabilities in VIP. In addition to simple t-test and returning only a limited number of differentially expressed (DE) genes in cellxgene’s DE analysis, users can get the full list of DE genes by calling any of the methods implemented in diffxpy, namely Welch’s t-test, Wilcoxon rank-sum test, and Wald’s test. Volcano plots in both interactive and static forms with fold change and p-value are also generated to enable global view of biological perturbation (
                             <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                              Figure 4b
                             </a>
                             ). Gene Set Enrichment Analysis (GSEA) can be elected to run for the user in DE analysis as well. Since the response speed is one of the critical concerns of the web application, we used fgsea implementation
                             <sup>
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                             </sup>
                             . Users can also download the pre-defined gene sets (MSigDB: symbol version) from Broad Institute or generate their own. Cellxgene VIP also allows the user to load and visualize pre-computed DE gene lists by any other single cell differential analysis algorithms requiring long computational run time, such as glmmTMB
                             <sup>
                              <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                               26
                              </a>
                             </sup>
                             or NEBULA
                             <sup>
                              <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                               27
                              </a>
                             </sup>
                             .
                            </p>
                            <div id="F4" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F4.large.jpg?width=800&height=600&carousel=1" title="Differential Gene Expression analysis. (a) User could select microglia cells and define selection 1 as MS (1186 cells) and selection 2 as Control (124 cells). DEG analysis could be performed by using Welch’s t-test from diffxpy implementation. GSEA could be enabled too. All with default parameters except showing full DEG result and select hall mark gene set (h.all.v7.3) from GSEA. (b) Result of the simple DEG analysis. 1. Interactive volcano plot, 2. Searchable DEG table, 3. Static volcano plot. (c) GSEA results. (d) Bubble plot to explore known list of gene DE results. Here we would like to see how a list of genes “FTL, MBP, CD74 and CD68” DE results across a defined set of pre-computed comparisons. From the bubble plot, user can easily see FTL is statistically significantly differential expressed in Microglia, Layer 2 and 3 Excitatory Neurons (EN.L2.2), Oligodendrocytes (OL), OPC and VIP+ inhibitory neurons (IN.VIP) with fold change > 1.5 and FDR < 0.05. While CD74 is up regulated in Microglia and MBP is up regulated in VIP+ inhibitory neurons." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-463299482" data-figure-caption='<div class="highwire-markup">Differential Gene Expression analysis. (a) User could select microglia cells and define selection 1 as MS (1186 cells) and selection 2 as Control (124 cells). DEG analysis could be performed by using Welch’s t-test from diffxpy implementation. GSEA could be enabled too. All with default parameters except showing full DEG result and select hall mark gene set (h.all.v7.3) from GSEA. (b) Result of the simple DEG analysis. 1. Interactive volcano plot, 2. Searchable DEG table, 3. Static volcano plot. (c) GSEA results. (d) Bubble plot to explore known list of gene DE results. Here we would like to see how a list of genes “FTL, MBP, CD74 and CD68” DE results across a defined set of pre-computed comparisons. From the bubble plot, user can easily see FTL is statistically significantly differential expressed in Microglia, Layer 2 and 3 Excitatory Neurons (EN.L2.2), Oligodendrocytes (OL), OPC and VIP+ inhibitory neurons (IN.VIP) with fold change > 1.5 and FDR < 0.05. While CD74 is up regulated in Microglia and MBP is up regulated in VIP+ inhibitory neurons.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F4.medium.gif" width="346" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F4.medium.gif" width="346" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure 4
                              </span>
                              <p id="p-28" class="first-child">
                               Differential Gene Expression analysis. (a) User could select microglia cells and define selection 1 as MS (1186 cells) and selection 2 as Control (124 cells). DEG analysis could be performed by using Welch’s t-test from diffxpy implementation. GSEA could be enabled too. All with default parameters except showing full DEG result and select hall mark gene set (h.all.v7.3) from GSEA. (b) Result of the simple DEG analysis. 1. Interactive volcano plot, 2. Searchable DEG table, 3. Static volcano plot. (c) GSEA results. (d) Bubble plot to explore known list of gene DE results. Here we would like to see how a list of genes “FTL, MBP, CD74 and CD68” DE results across a defined set of pre-computed comparisons. From the bubble plot, user can easily see FTL is statistically significantly differential expressed in Microglia, Layer 2 and 3 Excitatory Neurons (EN.L2.2), Oligodendrocytes (OL), OPC and VIP+ inhibitory neurons (IN.VIP) with fold change > 1.5 and FDR < 0.05. While CD74 is up regulated in Microglia and MBP is up regulated in VIP+ inhibitory neurons.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-29">
                             For demo purpose, Schirmer et al.
                             <sup>
                              <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                             </sup>
                             data was re-processed from fastq files for cellxgene VIP visualization (details of the reprocessing in iPython notebook format could be found from our GitHub repository at
                             <a href="https://bit.ly/2CeUHtO">
                              https://bit.ly/2CeUHtO
                             </a>
                             , while the demo site is available at
                             <a href="https://cellxgenevipms.bxgenomics.com">
                              https://cellxgenevipms.bxgenomics.com
                             </a>
                             ). In this study, Schirmer et al. acquired brain blocks from 12 MS and 9 control individuals and generated snRNA-seq data to study the disease mechanism of Multiple Sclerosis. From the example below (
                             <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                              Figure 4
                             </a>
                             ), a user would be able to quickly look for trends by performing fast statistical tests online as shown in
                             <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                              Figure 4a
                             </a>
                             and then confirm by more sophisticated and timeconsuming statistical tests ran offline (
                             <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                              Figure 4b
                             </a>
                             ). To illustrate the importance of using a statistically rigorous test after the quick investigation, we use the finding of FTL (ferritin light chain) gene as an example. FTL is responsible for storing iron in a soluble, non-toxic, readily available form, which is important for iron homeostasis. Iron is taken up in the ferrous form and deposited as ferric hydroxides after oxidation. The OpenTargets platform
                             <sup>
                              <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                               28
                              </a>
                             </sup>
                             shows a connection from FTL to Multiple Sclerosis (
                             <a href="https://bit.ly/3Krq6qN">
                              https://bit.ly/3Krq6qN
                             </a>
                             ). From MS disease pathology, accumulated phagocytosed iron in a brain lesion is associated with clinical deterioration
                             <sup>
                              <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                              ,
                              <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ,
                              <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              ,
                              <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                               32
                              </a>
                              ,
                              <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                               33
                              </a>
                              ,
                              <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                             </sup>
                             . In this dataset, FTL gene shows weak upregulation in MS vs Control in microglia from t-test with log2 fold change of 0.49. Since a generalized linear model (glm) with “age”, “sex”, “mitochondrial fraction”, “capture batch” as covariates is more appropriate, NEBULA
                             <sup>
                              <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                               35
                              </a>
                             </sup>
                             and glmmTMB
                             <sup>
                              <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                               36
                              </a>
                             </sup>
                             methods were applied, and FTL is shown to be strongly upregulated in MS microglia cells with log2 fold change of 2.46. Further, gene set enrichment analysis (GSEA) is also available in VIP for understanding of biological pathway context (
                             <a id="xref-fig-4-6" class="xref-fig" href="#F4">
                              Figure 4c
                             </a>
                             ). Finally, a bubble plot visualization provides the user the ability to directly query a list of genes from the DE analysis results. It allows quick global overview of differential expression patterns for a list of genes across many comparisons from the pre-computed DE results as shown in
                             <a id="xref-fig-4-7" class="xref-fig" href="#F4">
                              Figure 4d
                             </a>
                             .
                            </p>
                            <div id="F5" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F5.large.jpg?width=800&height=600&carousel=1" title="Visualization of multiome (scRNA-seq and scATAC-seq) data. (a) Add genes to be investigated first under “Add Genes” menu. (b) In “Multiome” module, user can select cell type specific tracks of DNA accessibility, regions of DNA, and added genes before “Plot” action. (c) Joint plot of genome browser track of DNA accessibility peaks from bigwig files and violin plot of gene expression. The percentage of cells expressing a gene above the cutoff in log scale is shown in the up-right corner of a violin plot. Pre-computed Pearson correlation between the expression of a gene and the accessibility of each peak is displayed as a curved line color-coded by the score scale in the legend." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-463299482" data-figure-caption='<div class="highwire-markup">Visualization of multiome (scRNA-seq and scATAC-seq) data. (a) Add genes to be investigated first under “Add Genes” menu. (b) In “Multiome” module, user can select cell type specific tracks of DNA accessibility, regions of DNA, and added genes before “Plot” action. (c) Joint plot of genome browser track of DNA accessibility peaks from bigwig files and violin plot of gene expression. The percentage of cells expressing a gene above the cutoff in log scale is shown in the up-right corner of a violin plot. Pre-computed Pearson correlation between the expression of a gene and the accessibility of each peak is displayed as a curved line color-coded by the score scale in the legend.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F5.medium.gif" width="440" height="293"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F5.medium.gif" width="440" height="293"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure 5
                              </span>
                              <p id="p-30" class="first-child">
                               Visualization of multiome (scRNA-seq and scATAC-seq) data. (a) Add genes to be investigated first under “Add Genes” menu. (b) In “Multiome” module, user can select cell type specific tracks of DNA accessibility, regions of DNA, and added genes before “Plot” action. (c) Joint plot of genome browser track of DNA accessibility peaks from bigwig files and violin plot of gene expression. The percentage of cells expressing a gene above the cutoff in log scale is shown in the up-right corner of a violin plot. Pre-computed Pearson correlation between the expression of a gene and the accessibility of each peak is displayed as a curved line color-coded by the score scale in the legend.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-31">
                             scRNA-seq provides insight into transcriptomic heterogeneity and unveils unknown cell types in cellular populations
                             <sup>
                              <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                               37
                              </a>
                              ,
                              <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                               38
                              </a>
                              ,
                              <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                               39
                              </a>
                             </sup>
                             , while scATAC-seq dissects the transcriptional regulation network in mixed cell populations
                             <sup>
                              <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                               40
                              </a>
                              ,
                              <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                               41
                              </a>
                              ,
                              <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                               42
                              </a>
                             </sup>
                             . The integrative analysis of multimodal single-cell data including RNA transcription information and chromatin profiles provides an unprecedented opportunity to investigate the cell type-specific gene regulation and cell-to-cell variance in gene expression
                             <sup>
                              <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                               43
                              </a>
                              ,
                              <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                               44
                              </a>
                              ,
                              <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                               45
                              </a>
                             </sup>
                             . Using a cryopreserved human peripheral blood mononuclear cells (PBMCs) dataset
                             <sup>
                              <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                               46
                              </a>
                             </sup>
                             , we illustrated how cellxgene VIP allows users to explore this multiome dataset interactively (
                             <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                              Figure 5
                             </a>
                             ). IGV styled genome browser visualization of combined DNA accessibility and gene expression data at one of the B cell marker genes, MS4A1 locus, shows the differential DNA accessibility among B cells and other cell types. The displayed link shows how DNA accessibility at the locus correlates with the expression of nearby genes within a given distance from the gene transcriptional start site, where peaks representing DNA chromatin openness in all B cell classes are highly correlated with the expression level of MS4A1 gene but not the other gene, MS4A5.
                            </p>
                            <p id="p-32">
                             To further empower advanced users with programming skills, a Command Line Interface (CLI) is provided to offer unlimited analytical capabilities by directly utilizing any Python and/or R packages such as Seurat or any other R packages installed on the server (sample vignettes in CLI) through a mini Jupyter notebook environment as shown in
                             <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                              Figure 6
                             </a>
                             . All cells are used in vignette 1 (V1) while only T cells are selected in V2 and 3 for generating the result quickly. Computationally intensive and timeconsuming jobs are not recommended to run in CLI. It is mainly to prototype and perform bioinformatics analysis in a small subset of cells or generate plots that is not provided by other interactive functional modules. Moreover, by sharing a collection of well-test vignettes with other analysts or even biologists, use cases of data reuse will be greatly expanded, e.g., general biologists could just change gene names in vignette 1 to generate a joint plot for genes of interest without seeking assistance form a computational biologist. Even for computational biologists, it is a great learning platform to share code snippets and workflows with each other.
                            </p>
                            <div id="F6" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F6.large.jpg?width=800&height=600&carousel=1" title="Command Line Interface (CLI) and demonstration of code examples by vignettes. (a) Pure Python code provided by clicking on “Vignette 1” operates on all cells by “Run” button to generate a joint plot in SVG format that can be readily saved. Through GUI, a user can select a limited number of annotations and layouts to minimize data trafficking between the server and the in-memory h5ad file. Other functional buttons are for additional convenience, such as “Hide input” will increase the space for viewing the result. (b) Partial output of “Vignette 2” that combines both Python and R code blocks in a Jupyter notebook like environment. Since the huge in-memory data needs to be converted to a file that R code will operate on, only a subset of cells is selected for the demo purpose. “Vignette 3” will generate a similar report in R markdown form that is not shown here." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-463299482" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Command Line Interface (CLI) and demonstration of code examples by vignettes.</span> (a) Pure Python code provided by clicking on “Vignette 1” operates on all cells by “Run” button to generate a joint plot in SVG format that can be readily saved. Through GUI, a user can select a limited number of annotations and layouts to minimize data trafficking between the server and the in-memory h5ad file. Other functional buttons are for additional convenience, such as “Hide input” will increase the space for viewing the result. (b) Partial output of “Vignette 2” that combines both Python and R code blocks in a Jupyter notebook like environment. Since the huge in-memory data needs to be converted to a file that R code will operate on, only a subset of cells is selected for the demo purpose. “Vignette 3” will generate a similar report in R markdown form that is not shown here.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F6.medium.gif" width="440" height="274"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F6.medium.gif" width="440" height="274"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/14/2020.08.28.270652/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure 6
                              </span>
                              <span class="caption-title">
                               Command Line Interface (CLI) and demonstration of code examples by vignettes.
                              </span>
                              <p id="p-33" class="first-child">
                               (a) Pure Python code provided by clicking on “Vignette 1” operates on all cells by “Run” button to generate a joint plot in SVG format that can be readily saved. Through GUI, a user can select a limited number of annotations and layouts to minimize data trafficking between the server and the in-memory h5ad file. Other functional buttons are for additional convenience, such as “Hide input” will increase the space for viewing the result. (b) Partial output of “Vignette 2” that combines both Python and R code blocks in a Jupyter notebook like environment. Since the huge in-memory data needs to be converted to a file that R code will operate on, only a subset of cells is selected for the demo purpose. “Vignette 3” will generate a similar report in R markdown form that is not shown here.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-34">
                             Loading half-million-cell dataset even with sparse matrix compression applied would take up to approximately 20G of memory. Standard Mac/PC laptop would not be able to load such a dataset into memory, not to mention doing additional interactive exploration and plotting. Within cellxgene VIP, all heavy lifting functions such as differential gene expression analysis are executed on the server side that in turn greatly reduces the burden on the client browser side.
                            </p>
                            <p id="p-35">
                             To facilitate reproducible research, sessions can be saved including all settings in the VIP window and cell selections in the main window, which can be loaded later to restore settings from previous analysis, with options to load from a URL or local session file and follow by reproducing plots in the previous session in one click (Supplementary Materials for more details).
                            </p>
                            <p id="p-36">
                             Cellxgene VIP also initializes cellxgene embedding, coloring and brief description of the data (Supplementary Methods), sync cell selection, gene input from the main window to VIP automatically, save and load sessions, and create combinatorial annotations and abbreviations (Supplementary Fig. 24,
                             <a href="https://bit.ly/376k28w">
                              https://bit.ly/376k28w
                             </a>
                             ). Further, we made cellxgene VIP open source and free of charge to facilitate broader scientific collaborations on large scale scRNA-seq data sets.
                            </p>
                            <p id="p-37">
                             In summary, our web-based interactive tool is built upon cellxgene but greatly extended its plotting and analytical capabilities by integrating state-of-the-art tools in this field. It allows users with no programming skills to rapidly explore scRNA-seq, Visium spatial transcriptomics, and multiome data interactively and create high-resolution figures commonly seen in high-profile publications. Furthermore, it is the first tool to the author’s best knowledge to allow computational biologists to write their own code to communicate with the hosting server via a mini Jupyter notebook interface to perform advanced analysis or provide code snippets for others to reuse in the same environment. It stimulates unlimited capabilities even beyond the rich set of plotting and analytical functions already provided in the tool.
                            </p>
                           </div>
                           <div class="section" id="sec-3">
                            <h2 class="">
                             Methods
                            </h2>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Software architecture
                             </h3>
                             <p id="p-38">
                              Cellxgene VIP uses a server-client architecture that enables users to take full advantage of an interactive web browser-based interface to set parameters for analysis, plot figures and program in CLI while the heavy lifting computational tasks are performed on the server side. On the server side, the most popular scRNA-seq and spatial transcriptomics data processing tools, Scanpy and Seurat, are integrated for data analysis and visualization. Various JavaScript libraries including D3, jQuery, Crossfilter, and Ace were used to provide the interactive user experience.
                             </p>
                             <p id="p-39">
                              VIP makes minimal modifications of cellxgene source code. It takes advantage of the cached data by cellxgene server to avoid data reloading which is time consuming for a large data set in gigabytes. On the client-side, a non-invasive JavaScript panel (Supplementary Method 1,
                              <a href="https://bit.ly/3rftBZA">
                               https://bit.ly/3rftBZA
                              </a>
                              ) was elegantly plugged in to bridge the communications between the user and server through the web browser store (Supplementary Method 3,
                              <a href="https://bit.ly/37zXeOq">
                               https://bit.ly/37zXeOq
                              </a>
                              ). With extensibility in mind, we decoupled the development of the plugin and cellxgene as much as possible, so it is easy to incorporate additional tools in Python or any other languages like R on server side while taking full benefits of the interactive user experience offered by both cellxgene and the plugin. Located on the same web page, the VIP works seamlessly with cellxgene main window through real time data exchange by JavaScript, such as synchronizing cell selections and showing ranges of brushing on histograms of continuous annotations. With such design, developers can easily add new functions to the plugin without touching the cellxgene code base at all. In the end, we provide easy-to-use scripts to make necessary modifications mentioned above and install cellxgene VIP components, R packages and Python modules on the server side.
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Input data format
                             </h3>
                             <p id="p-40">
                              AnnData (
                              <a href="https://anndata.readthedocs.io">
                               https://anndata.readthedocs.io
                              </a>
                              ) format developed for Scanpy is used along with a text file with custom description and initial settings. We follow the same input requirements as cellxgene
                              <sup>
                               <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
                                4
                               </a>
                              </sup>
                              . To start cellxgene, a h5ad file meeting the following AnnData specifications is required: 1. anndata.X is the raw/normalized expression matrix; 2. annadata.obsm contain at least one embedding (with prefix “X_”, such as “X_tSNE”); 3. Cell barcode or gene IDs must be unique. To provide more details about the data set and set initial options of visualization, we create an optional description file to add a study dataset description at the bottom of the categorical annotations on the left panel. The description file is required to be the same name as the h5ad file with ‘txt’ as extension within the same directory. The content of an example description file is as follows:
                             </p>
                             <p id="p-41">
                              Description: Human brain snRNA-seq 46k cells (MS Nature 2019 Schirmer et al.)
                             </p>
                             <p id="p-42">
                              Platform: 10X 3′ v2 chemistry nuclear sequencing
                             </p>
                             <p id="p-43">
                              Data: normalized, log transformed UMI
                             </p>
                             <p id="p-44">
                              Initial Embedding: umap_harmony
                             </p>
                             <p id="p-45">
                              Initial Coloring: cell_type
                             </p>
                             <p id="p-46">
                              Initial Session:
                              <a href="https://interactivereport.github.io/cellxgene_VIP/static/MS.vip.session.txt">
                               https://interactivereport.github.io/cellxgene_VIP/static/MS.vip.session.txt
                              </a>
                             </p>
                             <p id="p-47">
                              Colon is used as the delimiter for VIP to parse the initial settings for embedding and coloring. In our demo, the default layout is “umap_harmony” while cells will be colored based on “cell_type” categories. If keyword “scale” is not found in “Data” line, VIP will set the figure option “Scale data to unit variance for plotting” to “Yes”. The value of “Initial Session” will populate the “Load from a URL” input in “Load Session” interface.
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Sample data used for demonstration of cellxgene VIP functionality
                             </h3>
                             <p id="p-48">
                              Human MS brain snRNA-seq dataset raw data (fastq, sample metadata and cell type identification files) from Schirmer et al.
                              <sup>
                               <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">
                                23
                               </a>
                              </sup>
                              was downloaded and re-processed with 10x cellranger 3.0.2 (count) by mapping to human pre-mRNA genome reference (GRCh38 reference - 3.1.0, July 24, 2019). Quality control criteria were applied as follows: filtered out cells that have less than 1000 counts, or more than 40000 counts, and kept cells with < 0.2 mitochondria gene counts. Harmony
                              <sup>
                               <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                                47
                               </a>
                              </sup>
                              and Liger
                              <sup>
                               <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                                48
                               </a>
                              </sup>
                              were used to correct for batch effects.
                             </p>
                             <p id="p-49">
                              Louvain clustering method
                              <sup>
                               <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                                49
                               </a>
                              </sup>
                              was used to cluster cells into different subgroups (clusters) on both batch corrected results. Both tSNE
                              <sup>
                               <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                                50
                               </a>
                              </sup>
                              and UMAP
                              <sup>
                               <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                                51
                               </a>
                              </sup>
                              coordinates are provided. Authors’ cell type annotation was in great concordance with the Louvain clustering result, except we decided to merge OL-A, OL-B and OL-C clusters into OL (Oligodendrocyte) and EN-L2-3-A and EN-L2-3-B clusters into EN-L2-3 (Excitatory Neuron Layer).
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Integrated gene set query
                             </h3>
                             <p id="p-50">
                              Cellxgene VIP equips a helper tool, xGenesets that provides an Application Programming Interface (API) for web-based applications to select a list of genes in a gene set. The gene sets are defined based on pathways, gene sets, and categories from various sources, including KEGG Pathways
                              <sup>
                               <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">
                                52
                               </a>
                              </sup>
                              , WikiPathways
                              <sup>
                               <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">
                                53
                               </a>
                              </sup>
                              , Small Molecule Pathway Database
                              <sup>
                               <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">
                                54
                               </a>
                              </sup>
                              , Reactome
                              <sup>
                               <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">
                                55
                               </a>
                              </sup>
                              , Gene Ontology
                              <sup>
                               <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">
                                56
                               </a>
                              </sup>
                              , Molecular Signatures Database
                              <sup>
                               <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">
                                57
                               </a>
                              </sup>
                              from Gene Set Enrichment Analysis (GSEA
                              <sup>
                               <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">
                                58
                               </a>
                              </sup>
                              ), and LIPID MAPS Proteome Database
                              <sup>
                               <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">
                                59
                               </a>
                              </sup>
                              . xGenesets currently has 175,537 gene sets stored in a MySQL database, covering gene sets for species including human (115,497 records), mouse (30,554 records) and rat (29,486 records). xGenesets features auto-completion for quick gene set selection, gene set database browsing, and the full member gene list of any gene set. It can be easily embedded into an online application by leveraging jQuery plugin, DataTables, and bootstrap (either v3 or v4). An example of using xGenesets API can be found at
                              <a href="https://bxaf.net/genesets.xGenesets">
                               https://bxaf.net/genesets.xGenesets
                              </a>
                              API is freely available for all usages.
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              Generation of combo annotations
                             </h3>
                             <p id="p-51">
                              Celllxgene VIP also provides a fast annotation combination tool to label cells across selected categories. A new category, “Custom_combine”, will be automatically created to annotate cells across different categories, such as cell type and diagnosis. The user can also re-order the annotations (e.g., EN-L2-3:MS, EN-L2-3:Control) in this category which can be used in VIP plotting functions. Cells not belonging to any combined annotations will not be considered in plotting.
                             </p>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              Differential gene expression
                             </h3>
                             <p id="p-52">
                              Three statistical test methods of differential gene expression analysis are provided including Welch’s t-test (both cellxgene and diffxpy implementations), Wilcoxon rank test, and Wald’s test. The statistical test results are presented in a table format including log2 fold change, p-value, and multi-testing adjusted p-value by Bonferroni in cellxgene or Benjamini-Hochberg in diffxpy. Notably, we provide users with simple test methods owing to quick response time within the interactive framework. However, more advanced statistical methods developed for differential gene expression analysis of scRNA-seq data such as glmmTMB
                              <sup>
                               <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">
                                26
                               </a>
                              </sup>
                              or NEBULA
                              <sup>
                               <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">
                                27
                               </a>
                              </sup>
                              are recommended even if it takes much longer computational time to get results considering that there would be covariates that need to be modeled in a proper statistical test. To accommodate data exploration of offline results, we developed a loading script for users to load and explore pre-computed results interactively inside the VIP environment in table, interactive volcano, and bubble heatmap forms.
                             </p>
                            </div>
                            <div id="sec-10" class="subsection">
                             <h3>
                              glmmTMB differential analysis
                             </h3>
                             <p id="p-53">
                              As an example of an advanced testing method, we applied the glmmTMB algorithm
                              <sup>
                               <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">
                                26
                               </a>
                              </sup>
                              to the data of Schirmer et al
                              <sup>
                               <a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">
                                23
                               </a>
                              </sup>
                              . The glmmTMB method uses a generalized linear mixed model framework combined with the template model builder to determine differentially expressed genes. This method is very flexible allowing the user to adjust for covariate effects and model subject to subject or sample to sample variance components. The user can choose a variety of response families, dispersion models, zero inflation models, and variance/covariance structures. Unfortunately, this approach is very computationally demanding so its DEG results were pre-computed and then visualized using cellxgene VIP.
                             </p>
                            </div>
                            <div id="sec-11" class="subsection">
                             <h3>
                              NEBULA differential analysis
                             </h3>
                             <p id="p-54">
                              As a second example of an advanced testing method, we applied the NEBULA algorithm
                              <sup>
                               <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">
                                27
                               </a>
                              </sup>
                              (both NEBULA-LN and NEBULA-HL) to the data of Schirmer et al
                              <sup>
                               <a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">
                                23
                               </a>
                              </sup>
                              . The NEBULA method uses a generalized linear mixed model framework to determine differentially expressed genes. This method is flexible allowing the user to adjust for covariate effects, to account for subject-level overdispersions, and to account for cell-level overdispersions. A large sample approximation in NEBULA-LN and the h-likelihood method of NEBULA-HL allows NEBULA to be computationally efficient and to be much faster than other single-cell level DEG tools. After calculating the DEGs for the data of Schirmer et al
                              <sup>
                               <a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">
                                23
                               </a>
                              </sup>
                              , the results were visualized using cellxgene VIP.
                             </p>
                            </div>
                            <div id="sec-12" class="subsection">
                             <h3>
                              Filtering criteria of DE analysis
                             </h3>
                             <p id="p-55">
                              For the NEBULA and glmmTMB differential analysis, the counts data of Schirmer et al
                              <sup>
                               <a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">
                                23
                               </a>
                              </sup>
                              was imported into R and two rounds of QC filtering were applied. In the first round, filtering was applied to the entire count matrix. We require: 1) a library size between 200 to 20M, 2) genes must be expressed in at least 3 cells, and 3) cells must have at least 250 genes expressed. Additionally, mitochondrial and ribosomal genes were filtered out at this stage. During the second round, the filters focused on the cell subset corresponding to a cell type of interest. With this cell subset, we required a minimum of 10% of the cells to be expressed in either group for the contrast of interest, a minimum of 3 cells per subject, and a minimum of 2 subjects per group.
                             </p>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              Loading of pre-computed DEGs
                             </h3>
                             <p id="p-56">
                              The pre-computed DEGs can be visualized in cellxgene VIP as shown in
                              <a id="xref-fig-4-8" class="xref-fig" href="#F4">
                               Figure 4b
                              </a>
                              and
                              <a id="xref-fig-4-9" class="xref-fig" href="#F4">
                               4d
                              </a>
                              . The user needs to create a sqlite database file which must be in the same folder as the h5ad file and share the same name but with ‘db’ as the file extension, e.g., “ms_nature_2019_Schirmer.db” and “ms_nature_2019_Schirmer.h5ad” are in the same folder to be loaded. If the database file has changed, the server doesn’t need to restart.
                             </p>
                             <p id="p-57">
                              There is a script named “DEG2sqlite3.py” under the bin folder, which saves all pre-computed DEG results in csv format to sqlite3 file. It takes 2 parameters where the first parameter is the full path of all csv files, and the second parameter is the name of the database file that will be saved in the same folder as csv files, e.g.,
                             </p>
                             <p id="p-58">
                              python DEG2sqlite3.py ./csv Ms_nature_2019_Schirmer
                             </p>
                             <p id="p-59">
                              The csv files need to contain the 4 columns with headers: gene name, log2FC, Pvalue, FDR. The script is hard coded to pull the first 4 columns in that order. The names of the csv files must be A.vs.B_tag1_tag2.csv, where the contrast is between A and B, and tags can be cell types, e.g., NAWM.vs.NWM_Astrocytes.csv.
                             </p>
                            </div>
                            <div id="sec-14" class="subsection">
                             <h3>
                              Visualization of spatial transcriptomics data
                             </h3>
                             <p id="p-60">
                              For pre-processing, we used Scanpy “read_visium” function to read data of each sample from 10x genomics Space Ranger output directory. To note, in order to have the right orientation (Visium capture image border marker on the bottom right should be an empty circle) in the user interface, spatial coordinate Y axis needs to have negative values to fit coordinates design of cellxgene. Therefore, in Python environment such as Jupyter notebook, when plotting the H&E image and coordinates together, they do NOT overlap. On the other hand, when loaded into cellxgene VIP, they would. We also added an additional capability to merge multiple spatial transcriptomics samples into one spatial embedding in a grid format. Our demo example is a 2 x 2 image merged by 4 spatial transcriptomics samples. We apply AnnData.concatenate function to merge 4 multiple AnnData objects into a single AnnData object, then add back individual spatial coordinates stored in AnnData.obsm variable with name starting as “X_spatial_” and original spatial images. After merging, we perform basic QC, apply normalization, log1p transformation, select highly variable genes (HVGs), run PCA dimension reduction, generate UMAP layout and cluster spots by Leiden
                              <sup>
                               <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">
                                60
                               </a>
                              </sup>
                              . At the same time, H&E images were stitched by PIL
                              <sup>
                               <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">
                                61
                               </a>
                              </sup>
                              Python module and then saved back to Anndata.uns[‘spatial’] as the merged spatial embedding. In summary, all these steps are performed by the below command.
                             </p>
                             <p id="p-61">
                              python3 st_sample_merge.py -i <inputfile of data folders= -o <outputfile= [-d <grid dimension=] [-s <grid cell size=]
                             </p>
                             <p id="p-62">
                              Where, each line in the input file holds one directory in which Space Ranger outputs including aligned images of one Visium slide are stored. Both grid dimension and cell size parameters are optional. Grid dimension is defined in a row by column format (e.g., 2×3 without space) and calculated by default to fit slides in a rectangular grid. The default grid cell size is 700 that is big enough to contain a 600-pixel low resolution image, which is used to minimize the size of the merged image.
                             </p>
                             <p id="p-63">
                              Spatial coordinates are visualized by cellxgene embedding layout function, which is normally used for PCA, tSNE or UMAP. To accommodate individual samples in the merged h5ad file, we need to use the same number of spatial spots from the merged samples for each individual sample in the spatial layout, so that any spots that do not belong to an individual sample are assigned (0,0) spatial coordinates, which are displayed as a single dot on the top left corner of the spatial layout with minimum visibility. The user could use a selecting subset cell function from cellxgene to exclude spots not belonging to the sample of interest before plotting or performing analysis. VIP provides a function to retrieve the H&E images and overlay with spatial coordinates. The user can choose either the image or embedding to be on the top layer, adjust transparency, zoom in and out with image and embedding together or separately, realign image and embedding. Further, to facilitate the user to compare and analyze multiple samples together, we allow one to design the desired orientation of samples by arranging, flipping, rotating images of interest into a N x M grid. Once the design is done, the user can download the layout file in JSON format. This JSON file can be supplied to an analyst to generate a new h5ad file with the desired layout of samples by calling the Python script that is provided in the bin directory of the GitHub repository.
                             </p>
                             <p id="p-64">
                              python3 st_h5ad_image_operation.py -j design.json -i merge.h5ad -o final.h5ad
                             </p>
                             <p id="p-65">
                              Once a new h5ad file is created, an analyst can spin up another cellxgene VIP instance of the finalized data set for the user to explore. This workflow encourages close collaboration between experimentalists and analysts and prevents unintended large h5ad files from proliferating if an end user is given an option to generate such files on the fly through the user interface.
                             </p>
                            </div>
                            <div id="sec-15" class="subsection">
                             <h3>
                              Command line interface (CLI)
                             </h3>
                             <p id="p-66">
                              On client side, we embed the Ace editor (
                              <a href="https://ace.c9.io">
                               https://ace.c9.io
                              </a>
                              ) in VIP for advanced users to write analysis code to operate on either the whole dataset or a smaller sliced data set based on selections of cells and genes. The editor offers syntax coloring, automated indent, highlighting matching parentheses and other advanced features to match performance of native editors such as Vim. For Python programmers, Jupyter notebook syntax is fully supported, including built-in magic commands, e.g., to set the output image format to SVG.
                             </p>
                             <p id="p-67">
                              %config InlineBackend.figure_formats = [‘svg’]
                             </p>
                             <p id="p-68">
                              The browser then passes the code written in the CLI editor to the server side, where the following one-line command firstly converts the Python script submitted from the client to Jupyter notebook document by “jupytext” (
                              <a href="https://jupytext.readthedocs.io">
                               https://jupytext.readthedocs.io
                              </a>
                              ) and secondly runs “jupyter nbconvert” (
                              <a href="https://nbconvert.readthedocs.io">
                               https://nbconvert.readthedocs.io
                              </a>
                              ) to execute the notebook and generate output in HTML format that will be returned to the client side.
                             </p>
                             <p id="p-69">
                              jupytext --to notebook --output - <input script< | jupyter nbconvert --ExecutePreprocessor.timeout=1800--to html --execute --stdin –stdout
                             </p>
                             <p id="p-70">
                              To communicate with big data hosted on server side in CLI, two preserved variables are made visible to both the client and the server side. One is “adata” that is a Python AnnData object holding gene expression matrix of the whole dataset or selected cells and metadata while the other is “strPath” that is the file name including path but leaving out the file extension. Thus, concatenation of strPath and “.h5ad” is the name of the file storing “adata” that is derived from the whole data based on user selections and stored on the server temporally for use in a session executing the script submitted from the interface.
                             </p>
                             <p id="p-71">
                              To be flexible, there are two ways of programming in R. One is using R magic enabled by rpy2 module that is already installed on server side. As shown in CLI Vignette 2, the following code snippet shows how to use apache Arrow technique to share “adata” object in feather file format (
                              <a href="https://bit.ly/3dOi0Lu">
                               https://bit.ly/3dOi0Lu
                              </a>
                              ) between Python and R code blocks.
                             </p>
                             <p id="p-72">
                              from pyarrow import feather
                             </p>
                             <p id="p-73">
                              import pandas as pd
                             </p>
                             <p id="p-74">
                              X = adata.to_df()
                             </p>
                             <p id="p-75">
                              meta = pd.concat([adata.obs,adata.obsm.to_df()],axis=1)
                             </p>
                             <p id="p-76">
                              feather.write_feather(X,strPath+‘.feather’)
                             </p>
                             <p id="p-77">
                              feather.write_feather(meta,strPath+“.obs.feather”)
                             </p>
                             <p id="p-78">
                              %%R
                             </p>
                             <p id="p-79">
                              D <- t(arrow::read_feather(paste(strPath,“.feather”,sep=“”)))
                             </p>
                             <p id="p-80">
                              meta <- as.data.frame(arrow::read_feather(paste(strPath,“.obs.feather”,sep=“”)))
                             </p>
                             <p id="p-81">
                              colnames(D) <- meta[,“name_0”]
                             </p>
                             <p id="p-82">
                              X <;- CreateSeuratObject(counts=D)
                             </p>
                             <p id="p-83">
                              X@meta.data <- cbind(X@meta.data,meta)
                             </p>
                             <p id="p-84">
                              As illustrated in CLI Vignette 3, R markdown is another way to program in R. The reticulate package is used to run Python code in R markdown to prepare the data file the same way as above. On the server side, the following command is executed to generate the returned html report.
                             </p>
                             <p id="p-85">
                              Rscript -e ‘rmarkdown::render(“<input R markdown>”, output_file=“output.html”)’
                             </p>
                             <p id="p-86">
                              All Python modules and R packages used in CLI environment need to be installed beforehand by following the installation instruction in the GitHub.
                             </p>
                            </div>
                            <div id="sec-16" class="subsection">
                             <h3>
                              Visualization of multiome data
                             </h3>
                             <p id="p-87">
                              Cellxgene VIP supports the visualization of multiome data to illustrate the relationship of DNA accessibility and expression of nearby genes at a locus. To exemplify this functionality, we used a PBMC multiome dataset from 10x Genomics
                              <sup>
                               <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">
                                46
                               </a>
                              </sup>
                              . For pre-processing, we utilized Signac
                              <sup>
                               <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">
                                20
                               </a>
                              </sup>
                              and Seurat to make a joint RNA and ATAC analysis with the hg38 reference genome. DNA accessibility data was used to compute per-cell quality control metrics and filter out the low-quality cells. After quality control, gene expression data was processed by SCTranfrom
                              <sup>
                               <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">
                                62
                               </a>
                              </sup>
                              and PCA while DNA accessibility data was processed by peak-calling, latent semantic indexing, and single value decomposition. After normalization and dimension reduction, we used a curated Azimuth human PBMC reference
                              <sup>
                               <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">
                                17
                               </a>
                              </sup>
                              to annotate cell types through Seurat reference mapping. To visualize the joint embedding plot, the weighted nearest neighbor method and UMAP were applied.
                             </p>
                             <p id="p-88">
                              Moreover, with scATAC-seq data, VIP allows users to visualize track plot in a mini genome browser. In Signac, we can calculate the correlation between gene expression and accessibility at the nearby peaks by using LinkPeaks to compute the GC content, and linkage between peaks and genes. The frequency plot over genomic region, coordinates, peaks distribution and links can be visualized by the CoveragePlot in Signac. However, fetching peaks and links annotation from the Seurat object which contains the whole genome information and displaying them in real-time on VIP interface are challenging. As a viable alternative, we used the annotated cell types to split scATAC bam files by Sinto
                              <sup>
                               <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">
                                63
                               </a>
                              </sup>
                              and made the BIGWIG files separately by bamCoverage in deepTools
                              <sup>
                               <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">
                                64
                               </a>
                              </sup>
                              . Links, peaks, and genomic range information from Signac session were saved as ‘links.rds’, ‘annotation.rds’ and ‘peaks.rds’. All files with ‘rds’ and ‘bw’ extension are required to be in a folder with the same names as the corresponding h5ad file holding scRNA-seq data. In addition, it also requires a file named “bw.cluster” in the folder. The first row in “bw.cluster” is the metadata category which BIGWIG files were split on; for example, we split on ‘CellType’ category with PBMC multiome dataset. The following two columns should indicate the filenames of bigwigs, and corresponding categorical names in h5ad. Upon the availability of these files, “Multiome” menu in VIP will be activated when such dataset is loaded in cellxgene VIP. The detailed reproductible process of the demo data is available at “Data Availability” section.
                             </p>
                            </div>
                            <div id="sec-17" class="subsection">
                             <h3>
                              Tutorial
                             </h3>
                             <p id="p-89">
                              A comprehensive user guide to illustrate detailed functionality of each module with step-by-step instruction is available as supplementary information and also accessible conveniently as an online HTML Bookdown
                              <sup>
                               <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">
                                65
                               </a>
                              </sup>
                              document at
                              <a href="https://interactivereport.github.io/cellxgene_VIP/tutorial/docs">
                               https://interactivereport.github.io/cellxgene_VIP/tutorial/docs
                              </a>
                              . Inside the application, a question mark behind the title of each function module provides a direct way to reach the corresponding section of the HTML document for help.
                             </p>
                            </div>
                            <div id="sec-18" class="subsection">
                             <h3>
                              Demos
                             </h3>
                             <p id="p-90">
                              sWe demonstrate the functionalities and capabilities of VIP on visualizing scRNA-seq, spatial transcriptomics, and multiome datasets by the following links,
                              <a href="https://cellxgenevip-ms.bxgenomics.com">
                               https://cellxgenevip-ms.bxgenomics.com
                              </a>
                              ,
                              <a href="https://cellxgenevip-spatial.bxgenomics.com">
                               https://cellxgenevip-spatial.bxgenomics.com
                              </a>
                              , and
                              <a href="https://cellxgenevip-multiome.bxgenomics.comrespectively">
                               https://cellxgenevip-multiome.bxgenomics.comrespectively
                              </a>
                              .
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-20">
                            <h2 class="">
                             Code availability
                            </h2>
                            <p id="p-91">
                             The source code and detailed installation guide are provided at
                             <a href="https://github.com/interactivereport/cellxgene_VIP">
                              https://github.com/interactivereport/cellxgene_VIP
                             </a>
                             . Cellxgene VIP is released under the MIT License.
                            </p>
                           </div>
                           <div class="section" id="sec-21">
                            <h2 class="">
                             Data availability
                            </h2>
                            <p id="p-92">
                             snRNA-seq. The demo MS snRNA-seq data in h5ad format can be reproduced by following the Jupyter notebook
                             <a href="https://bit.ly/2CeUHtO">
                              https://bit.ly/2CeUHtO
                             </a>
                             or downloaded from
                             <a href="https://bit.ly/3JjQqSp">
                              https://bit.ly/3JjQqSp
                             </a>
                             <sup>
                              <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">
                               66
                              </a>
                             </sup>
                             .
                            </p>
                            <p id="p-93">
                             Spatial transcriptomics. 4 Visium Spatial Transcriptomics datasets were downloaded from the 10X Genomics data site and were reformatted as spaceranger outputs for the convenience of reproducible downstream analysis. Data
                             <sup>
                              <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">
                               67
                              </a>
                             </sup>
                             can be downloaded from
                             <a href="https://bit.ly/3uut88d">
                              https://bit.ly/3uut88d
                             </a>
                             . The demo h5ad file can be reproduced by following the notebook
                             <a href="https://bit.ly/3O1fyRh">
                              https://bit.ly/3O1fyRh
                             </a>
                             or by this command, “python3 st_sample_merge.py -i inputfiles.txt -o merged_demo.h5ad -d 2×2 -s 600”.
                            </p>
                            <p id="p-94">
                             Multiome. The publicly available 10x Genomic Multiome dataset can be downloaded from
                             <a href="https://bit.ly/3urczd4">
                              https://bit.ly/3urczd4
                             </a>
                             . The demo multiome h5ad and bigwig files can be reproduced by following
                             <a href="https://bit.ly/3LXnUaD">
                              https://bit.ly/3LXnUaD
                             </a>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-22">
                            <h2 class="">
                             Author contributions
                            </h2>
                            <p id="p-96">
                             K.L, Z.O. and B.Z. designed and implemented the software. All authors contributed to testing of the tool. K.L., Y.C., J.G., D.L., M.M. and B.Z. wrote the manuscript with input from all authors. All authors read and approved the manuscript.
                            </p>
                           </div>
                           <div class="section" id="sec-23">
                            <h2 class="">
                             Competing interests
                            </h2>
                            <p id="p-97">
                             The authors declare no competing interests.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-95">
                             The authors are grateful and indebted to the cellxgene team from Chan Zuckerberg Initiative for the development of the base framework. We would also like to thank Dr. L. Schirmer and Prof. D.H. Rowitch for allowing us to use scRNA-seq data on Multiple Sclerosis for the demonstration of cellxgene VIP.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-others" id="fn-1">
                              <p id="p-1">
                               Emails: KL:
                               <span class="em-link">
                                <span class="em-addr">
                                 kejie.li{at}biogen.com
                                </span>
                               </span>
                               , ZO:
                               <span class="em-link">
                                <span class="em-addr">
                                 oyoung{at}bioinforx.com
                                </span>
                               </span>
                               , YC:
                               <span class="em-link">
                                <span class="em-addr">
                                 Yirui.chen{at}biogen.com
                                </span>
                               </span>
                               , JG:
                               <span class="em-link">
                                <span class="em-addr">
                                 jake.gagnon{at}biogen.com
                                </span>
                               </span>
                               , DL:
                               <span class="em-link">
                                <span class="em-addr">
                                 dongdong.lin{at}biogen.com
                                </span>
                               </span>
                               , MM:
                               <span class="em-link">
                                <span class="em-addr">
                                 michael.mingueneau{at}biogen.com
                                </span>
                               </span>
                               , WC:
                               <span class="em-link">
                                <span class="em-addr">
                                 wwchen{at}post.harvard.edu
                                </span>
                               </span>
                               , DS:
                               <span class="em-link">
                                <span class="em-addr">
                                 david.sexton{at}biogen.com
                                </span>
                               </span>
                               , BZ:
                               <span class="em-link">
                                <span class="em-addr">
                                 baohong.zhang{at}biogen.com
                                </span>
                               </span>
                              </p>
                             </li>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-3">
                              <p id="p-6">
                               In the new version, we extended Cellxgene VIP to visualize Spatial transcriptomics and Multiome data types which was not covered before.
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-4">
                              <p id="p-7">
                               <a href="https://cellxgenevip-ms.bxgenomics.com">
                                https://cellxgenevip-ms.bxgenomics.com
                               </a>
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-5">
                              <p id="p-8">
                               <a href="https://github.com/interactivereport/cellxgene_VIP">
                                https://github.com/interactivereport/cellxgene_VIP
                               </a>
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-6">
                              <p id="p-9">
                               <a href="https://bit.ly/2CeUHtO">
                                https://bit.ly/2CeUHtO
                               </a>
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-7">
                              <p id="p-10">
                               <a href="https://bit.ly/3JjQqSp">
                                https://bit.ly/3JjQqSp
                               </a>
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-8">
                              <p id="p-11">
                               <a href="https://bit.ly/3uut88d">
                                https://bit.ly/3uut88d
                               </a>
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-9">
                              <p id="p-12">
                               <a href="https://bit.ly/3O1fyRh">
                                https://bit.ly/3O1fyRh
                               </a>
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-10">
                              <p id="p-13">
                               <a href="https://bit.ly/3urczd4">
                                https://bit.ly/3urczd4
                               </a>
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-11">
                              <p id="p-14">
                               <a href="https://bit.ly/3LXnUaD">
                                https://bit.ly/3LXnUaD
                               </a>
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-12">
                              <p id="p-15">
                               <a href="https://interactivereport.github.io/cellxgene_VIP/tutorial/docs/">
                                https://interactivereport.github.io/cellxgene_VIP/tutorial/docs/
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2>
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tang
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  mRNA-Seq whole-transcriptome analysis of a single cell
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Methods
                                 </abbr>
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  377
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  382
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D6%26rft.spage%253D377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Svensson
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   da Veiga Beltrame
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pachter
                                  </span>
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  A curated database reveals trends in single-cell transcriptomics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Database
                                 </abbr>
                                 <span class="cit-vol">
                                  2020
                                 </span>
                                 , (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.3" data-doi="10.1126/science.aba7721">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cao
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  A human cell atlas of fetal gene expression
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  370
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  eaba7721
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCao%26rft.auinit1%253DJ.%26rft.volume%253D370%26rft.issue%253D6518%26rft.spage%253Deaba7721%26rft.epage%253Deaba7721%26rft.atitle%253DA%2Bhuman%2Bcell%2Batlas%2Bof%2Bfetal%2Bgene%2Bexpression%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aba7721%26rft_id%253Dinfo%253Apmid%252F33184181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE3OiIzNzAvNjUxOC9lYWJhNzcyMSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA0LzE0LzIwMjAuMDguMjguMjcwNjUyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Megill
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  cellxgene: a performant, scalable exploration platform for high dimensional sparse matrices
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 , 2021.2004.2005.438318 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cakir
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Prete
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Huang
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   van Dongen
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pir
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kiselev Vladimir
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Comparison of visualization tools for single-cell RNAseq data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  NAR Genomics and Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 , (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.6" data-doi="10.1002/bies.201900221">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Asp
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bergenstråhle
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lundeberg
                                  </span>
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Spatially resolved transcriptomes—next generation tools for tissue exploration
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BioEssays
                                 </abbr>
                                 <span class="cit-vol">
                                  42
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1900221
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioEssays%26rft.volume%253D42%26rft.spage%253D1900221%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.201900221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/bies.201900221&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Codeluppi
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Spatial organization of the somatosensory cortex revealed by osmFISH
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature methods
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  932
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  935
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D15%26rft.spage%253D932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  <span class="cit-name-given-names">
                                   KH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Boettiger
                                  </span>
                                  <span class="cit-name-given-names">
                                   AN
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moffitt
                                  </span>
                                  <span class="cit-name-given-names">
                                   JR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhuang
                                  </span>
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Spatially resolved, highly multiplexed RNA profiling in single cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  348
                                 </span>
                                 , (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.9" data-doi="10.1126/science.aaf2403">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ståhl
                                  </span>
                                  <span class="cit-name-given-names">
                                   PL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Visualization and analysis of gene expression in tissue sections by spatial transcriptomics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  353
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  78
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  82
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DStahl%26rft.auinit1%253DP.%2BL.%26rft.volume%253D353%26rft.issue%253D6294%26rft.spage%253D78%26rft.epage%253D82%26rft.atitle%253DVisualization%2Band%2Banalysis%2Bof%2Bgene%2Bexpression%2Bin%2Btissue%2Bsections%2Bby%2Bspatial%2Btranscriptomics%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf2403%26rft_id%253Dinfo%253Apmid%252F27365449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzNTMvNjI5NC83OCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA0LzE0LzIwMjAuMDguMjguMjcwNjUyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eng
                                  </span>
                                  <span class="cit-name-given-names">
                                   C-HL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Transcriptome-scale super-resolved imaging in tissues by RNA seqFISH+
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  568
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  235
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  239
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D568%26rft.spage%253D235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-confproc" id="cit-2020.08.28.270652v2.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Toldra
                                  </span>
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  American Association for Cancer Research (AACR)-110th Annual Meeting
                                 </span>
                                 .
                                 <span class="cit-conf-loc">
                                  Atlanta, Georgia, USA
                                 </span>
                                 -
                                 <span class="cit-conf-date">
                                  March 29-April 3, 2019
                                 </span>
                                 . In:
                                 <span class="cit-conf-name">
                                  American Association for Cancer Research (AACR)-110th Annual Meeting. Atlanta)
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.12" data-doi="10.1126/science.aaw1219">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rodriques
                                  </span>
                                  <span class="cit-name-given-names">
                                   SG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Slide-seq: A scalable technology for measuring genome-wide expression at high spatial resolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  363
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1463
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1467
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRodriques%26rft.auinit1%253DS.%2BG.%26rft.volume%253D363%26rft.issue%253D6434%26rft.spage%253D1463%26rft.epage%253D1467%26rft.atitle%253DSlide-seq%253A%2BA%2Bscalable%2Btechnology%2Bfor%2Bmeasuring%2Bgenome-wide%2Bexpression%2Bat%2Bhigh%2Bspatial%2Bresolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaw1219%26rft_id%253Dinfo%253Apmid%252F30923225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNjMvNjQzNC8xNDYzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDQvMTQvMjAyMC4wOC4yOC4yNzA2NTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Palla
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fischer
                                  </span>
                                  <span class="cit-name-given-names">
                                   DS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Regev
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Theis
                                  </span>
                                  <span class="cit-name-given-names">
                                   FJ.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Spatial components of molecular tissue biology
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Biotechnology
                                 </abbr>
                                 <span class="cit-vol">
                                  40
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  308
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  318
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BBiotechnology%26rft.volume%253D40%26rft.spage%253D308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moses
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pachter
                                  </span>
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Museum of spatial transcriptomics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Methods
                                 </abbr>
                                 , (
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.15">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Palla
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Squidpy: a scalable framework for spatial single cell analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 , 2021.2002.2019.431994 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.16" data-doi="10.1186/s13059-021-02286-2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dries
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Giotto: a toolbox for integrative analysis and visualization of spatial expression data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  78
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D22%26rft.spage%253D78%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-021-02286-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-021-02286-2&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.17">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hao
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Integrated analysis of multimodal single-cell data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 , 2020.2010.2012.335331 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.18" data-doi="10.1016/j.cell.2019.05.031">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stuart
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Comprehensive integration of single-cell data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  177
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1888
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1902
                                 </span>
                                 .
                                 <span class="cit-page-range">
                                  e1821
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D177%26rft.spage%253D1888%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2019.05.031%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2019.05.031&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.08.28.270652v2.19">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bergenstraahle JL
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Larsson
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  STUtility: Visualization and analysis tools for Spatial Transcriptomics data
                                 </span>
                                 . Available at
                                 <a href="https://githubcom/jbergenstrahle/STUtility">
                                  https://githubcom/jbergenstrahle/STUtility
                                 </a>
                                 , ([Accessed:
                                 <span class="cit-date-in-citation">
                                  6 April 2021
                                 </span>
                                 ]).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.20">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stuart
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Srivastava
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lareau
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Satija
                                  </span>
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Multimodal single-cell chromatin analysis with Signac
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 , 2020.2011.2009.373613 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.21" data-doi="10.1186/s13059-017-1382-0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wolf
                                  </span>
                                  <span class="cit-name-given-names">
                                   FA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Angerer
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Theis
                                  </span>
                                  <span class="cit-name-given-names">
                                   FJ.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  SCANPY: large-scale single-cell gene expression data analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  15
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D19%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1382-0%26rft_id%253Dinfo%253Apmid%252F29409532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-017-1382-0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29409532&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hurt
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Whelan
                                  </span>
                                  <span class="cit-name-given-names">
                                   CD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Challa
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lin
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  BioInfograph: An Online Tool to Design and Display Multi-Panel Scientific Figure Interactively
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Front Genet
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  784531
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFront%2BGenet%26rft.volume%253D12%26rft.spage%253D784531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.23" data-doi="10.1038/s41586-019-1404-z">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schirmer
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Neuronal vulnerability and multilineage diversity in multiple sclerosis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  573
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  75
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  82
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D573%26rft.spage%253D75%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-019-1404-z%26rft_id%253Dinfo%253Apmid%252F31316211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-019-1404-z&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=31316211&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.08.28.270652v2.24">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   David
                                  </span>
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Fischer FRH. Fast and scalable differential expression analysis on single-cell RNA-seq data
                                 </span>
                                 . Available at
                                 <a href="https://githubcom/theislab/diffxpy">
                                  https://githubcom/theislab/diffxpy
                                 </a>
                                 , ([Accessed:
                                 <span class="cit-date-in-citation">
                                  8 May 2020
                                 </span>
                                 ]).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               25.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.25">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Korotkevich
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sukhov
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Budin
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shpak
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Artyomov
                                  </span>
                                  <span class="cit-name-given-names">
                                   MN
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sergushichev
                                  </span>
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Fast gene set enrichment analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 ,
                                 <span class="cit-fpage">
                                  060012
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               26.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brooks
                                  </span>
                                  <span class="cit-name-given-names">
                                   ME
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  glmmTMB balances speed and flexibility among packages for zero-inflated generalized linear mixed modeling
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The R journal
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  378
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  400
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BR%2Bjournal%26rft.volume%253D9%26rft.spage%253D378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               27.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.27">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   He
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kulminski
                                  </span>
                                  <span class="cit-name-given-names">
                                   AM.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  NEBULA: a fast negative binomial mixed model for differential expression and co-expression analyses of large-scale multi-subject single-cell data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 , 2020.2009.2024.311662 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               28.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.28">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ochoa
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Open Targets Platform: supporting systematic drug-target identification and prioritisation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Research
                                 </abbr>
                                 <span class="cit-vol">
                                  49
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  D1302
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D1310
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D49%26rft.spage%253DD1302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               29.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.29" data-doi="10.1038/s41586-021-03892-7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Absinta
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  A lymphocyte-microglia-astrocyte axis in chronic active multiple sclerosis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  597
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  709
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  714
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D597%26rft.spage%253D709%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-021-03892-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-021-03892-7&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               30.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.30" data-doi="10.1056/NEJMra1401483">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reich
                                  </span>
                                  <span class="cit-name-given-names">
                                   DS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lucchinetti
                                  </span>
                                  <span class="cit-name-given-names">
                                   CF
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Calabresi
                                  </span>
                                  <span class="cit-name-given-names">
                                   PA.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Multiple Sclerosis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  New England Journal of Medicine
                                 </abbr>
                                 <span class="cit-vol">
                                  378
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  169
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  180
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.volume%253D378%26rft.spage%253D169%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMra1401483%26rft_id%253Dinfo%253Apmid%252F29320652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1056/NEJMra1401483&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29320652&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               31.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.31" data-doi="10.1007/s00401-016-1653-y">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kuhlmann
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ludwin
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Prat
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Antel
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brück
                                  </span>
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lassmann
                                  </span>
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  An updated histological classification system for multiple sclerosis lesions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Acta neuropathologica
                                 </abbr>
                                 <span class="cit-vol">
                                  133
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  13
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  24
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DActa%2Bneuropathologica%26rft.volume%253D133%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00401-016-1653-y%26rft_id%253Dinfo%253Apmid%252F27988845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s00401-016-1653-y&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27988845&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               32.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.32" data-doi="10.1007/s00401-016-1636-z">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dal-Bianco
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Slow expansion of multiple sclerosis iron rim lesions: pathology and 7 T magnetic resonance imaging
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Acta neuropathologica
                                 </abbr>
                                 <span class="cit-vol">
                                  133
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  25
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  42
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DActa%2Bneuropathologica%26rft.volume%253D133%26rft.spage%253D25%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00401-016-1636-z%26rft_id%253Dinfo%253Apmid%252F27796537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s00401-016-1636-z&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27796537&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               33.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.33" data-doi="10.1172/JCI86198">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Absinta
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Persistent 7-tesla phase rim predicts poor outcome in new multiple sclerosis patient lesions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Journal of clinical investigation
                                 </abbr>
                                 <span class="cit-vol">
                                  126
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2597
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2609
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bclinical%2Binvestigation%26rft.volume%253D126%26rft.spage%253D2597%26rft_id%253Dinfo%253Adoi%252F10.1172%252FJCI86198%26rft_id%253Dinfo%253Apmid%252F27270171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1172/JCI86198&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27270171&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               34.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.34" data-doi="10.1093/brain/awr278">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bagnato
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Tracking iron in multiple sclerosis: a combined imaging and histopathological study at 7 Tesla
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Brain
                                 </abbr>
                                 <span class="cit-vol">
                                  134
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3602
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3615
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBrain%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbrain%252Fawr278%26rft_id%253Dinfo%253Apmid%252F22171355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/brain/awr278&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22171355&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               35.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.35">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   He
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Davila-Velderrain
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sumida
                                  </span>
                                  <span class="cit-name-given-names">
                                   TS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hafler
                                  </span>
                                  <span class="cit-name-given-names">
                                   DA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kellis
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kulminski
                                  </span>
                                  <span class="cit-name-given-names">
                                   AM.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  NEBULA is a fast negative binomial mixed model for differential or co-expression analysis of large-scale multisubject single-cell data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Commun Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  629
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCommun%2BBiol%26rft.volume%253D4%26rft.spage%253D629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               36.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.36">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mollie E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Brooks KK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Koen J.
                                  </span>
                                  <span class="cit-name-surname">
                                   van Benthem
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Arni
                                  </span>
                                  <span class="cit-name-surname">
                                   Magnusson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Casper W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Berg
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Anders
                                  </span>
                                  <span class="cit-name-surname">
                                   Nielsen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Hans J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Skaug
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Martin
                                  </span>
                                  <span class="cit-name-surname">
                                   Mächler
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Benjamin M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bolker
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  glmmTMB Balances Speed and Flexibility Among Packages for Zero-inflated Generalized Linear Mixed Modeling
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The R Journal
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  378
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  400
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BR%2BJournal%26rft.volume%253D9%26rft.spage%253D378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               37.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.37">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   SoRelle
                                  </span>
                                  <span class="cit-name-given-names">
                                   ED
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Single-cell RNA-seq reveals transcriptomic heterogeneity mediated by hostpathogen dynamics in lymphoblastoid cell lines
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  eLife
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e62586
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D10%26rft.spage%253De62586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               38.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.38">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hoffman
                                  </span>
                                  <span class="cit-name-given-names">
                                   JA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Papas
                                  </span>
                                  <span class="cit-name-given-names">
                                   BN
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Trotter
                                  </span>
                                  <span class="cit-name-given-names">
                                   KW
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Archer
                                  </span>
                                  <span class="cit-name-given-names">
                                   TK.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Single-cell RNA sequencing reveals a heterogeneous response to Glucocorticoids in breast cancer cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Communications Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  126
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCommunications%2BBiology%26rft.volume%253D3%26rft.spage%253D126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               39.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.39" data-doi="10.1038/s12276-018-0048-7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hwang
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                  <span class="cit-name-given-names">
                                   JH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bang
                                  </span>
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Single-cell RNA sequencing technologies and bioinformaticspipelines
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Experimental & Molecular Medicine
                                 </abbr>
                                 <span class="cit-vol">
                                  50
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  14
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DExperimental%2B%2526%2BMolecular%2BMedicine%26rft.volume%253D50%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs12276-018-0048-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s12276-018-0048-7&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               40.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.40">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lareau
                                  </span>
                                  <span class="cit-name-given-names">
                                   CA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Droplet-based combinatorial indexing for massive-scale single-cell chromatin accessibility
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Biotechnology
                                 </abbr>
                                 <span class="cit-vol">
                                  37
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  916
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  924
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BBiotechnology%26rft.volume%253D37%26rft.spage%253D916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               41.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.41" data-doi="10.1038/nature14590">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Buenrostro
                                  </span>
                                  <span class="cit-name-given-names">
                                   JD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Single-cell chromatin accessibility reveals principles of regulatory variation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  523
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  486
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  490
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D523%26rft.spage%253D486%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14590%26rft_id%253Dinfo%253Apmid%252F26083756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature14590&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26083756&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               42.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.42" data-doi="10.1002/047114727.mb2129s109">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Buenrostro
                                  </span>
                                  <span class="cit-name-given-names">
                                   JD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chang
                                  </span>
                                  <span class="cit-name-given-names">
                                   HY
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Greenleaf
                                  </span>
                                  <span class="cit-name-given-names">
                                   WJ.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  ATAC-seq: A Method for Assaying Chromatin Accessibility Genome-Wide
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Curr Protoc Mol Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  109
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  21.29.21
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  21.29.29
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurr%2BProtoc%2BMol%2BBiol%26rft.volume%253D109%26rft.spage%253D212921%26rft_id%253Dinfo%253Adoi%252F10.1002%252F047114727.mb2129s109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/047114727.mb2129s109&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               43.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.43">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ranzoni
                                  </span>
                                  <span class="cit-name-given-names">
                                   AM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Integrative Single-Cell RNA-Seq and ATAC-Seq Analysis of Human Developmental Hematopoiesis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell stem cell
                                 </abbr>
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  472
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  487
                                 </span>
                                 .
                                 <span class="cit-page-range">
                                  e477
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%2Bstem%2Bcell%26rft.volume%253D28%26rft.spage%253D472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               44.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.44">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Muto
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Single cell transcriptional and chromatin accessibility profiling redefine cellular heterogeneity in the adult human kidney
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Communications
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2190
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D12%26rft.spage%253D2190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               45.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.45">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lareau
                                  </span>
                                  <span class="cit-name-given-names">
                                   CA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Massively parallel single-cell mitochondrial DNA genotyping and chromatin profiling
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Biotechnology
                                 </abbr>
                                 <span class="cit-vol">
                                  39
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  451
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  461
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BBiotechnology%26rft.volume%253D39%26rft.spage%253D451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               46.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.08.28.270652v2.46">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  Single Cell Mulitome ATAC + Gene Expression Dataset by Cell Ranger ARC 2.0.0, May 3rd) PBMC from a healthy donor
                                 </span>
                                 -
                                 <span class="cit-article-title">
                                  granulocytes removed through cell sorting (10k)
                                 </span>
                                 . Available at
                                 <a href="https://www10xgenomicscom/resources/datasets/pbmc-from-a-healthy-donor-granulocytes-removed-through-cell-sorting-10-k-1-standard-2-0-0">
                                  https://www10xgenomicscom/resources/datasets/pbmc-from-a-healthy-donor-granulocytes-removed-through-cell-sorting-10-k-1-standard-2-0-0
                                 </a>
                                 , (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               47.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.47">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Korsunsky
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Fast, sensitive and accurate integration of single-cell data with Harmony
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Methods
                                 </abbr>
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1289
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1296
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D16%26rft.spage%253D1289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               48.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.48" data-doi="10.1016/J.CELL.2019.05.006">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Welch
                                  </span>
                                  <span class="cit-name-given-names">
                                   JD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kozareva
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ferreira
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vanderburg
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Martin
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Macosko
                                  </span>
                                  <span class="cit-name-given-names">
                                   EZ.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Single-cell multi-omic integration compares and contrasts features of brain cell identity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  177
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1873
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1887
                                 </span>
                                 .
                                 <span class="cit-page-range">
                                  e1817
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D177%26rft.spage%253D1873%26rft_id%253Dinfo%253Adoi%252F10.1016%252FJ.CELL.2019.05.006%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/J.CELL.2019.05.006&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               49.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.49" data-doi="10.1088/1742-5468/2008/10/P10008">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blondel
                                  </span>
                                  <span class="cit-name-given-names">
                                   VD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guillaume
                                  </span>
                                  <span class="cit-name-given-names">
                                   J-L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lambiotte
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lefebvre
                                  </span>
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Fast unfolding of communities in large networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of statistical mechanics: theory and experiment
                                 </abbr>
                                 <span class="cit-vol">
                                  2008
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  P10008
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstatistical%2Bmechanics%253A%2Btheory%2Band%2Bexperiment%26rft.volume%253D2008%26rft.spage%253D10008P%26rft_id%253Dinfo%253Adoi%252F10.1088%252F1742-5468%252F2008%252F10%252FP10008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1088/1742-5468/2008/10/P10008&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               50.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.50">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maaten
                                  </span>
                                  <span class="cit-name-given-names">
                                   Lvd
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hinton
                                  </span>
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Visualizing data using t-SNE
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of machine learning research
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2579
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2605
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmachine%2Blearning%2Bresearch%26rft.volume%253D9%26rft.spage%253D2579%26rft_id%253Dinfo%253Apmid%252FWOS%253A00026263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=WOS:00026263&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               51.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.51" data-doi="10.1038/nbt.4314">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Becht
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Dimensionality reduction for visualizing single-cell data using UMAP
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature biotechnology
                                 </abbr>
                                 <span class="cit-vol">
                                  37
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  38
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  44
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.volume%253D37%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.4314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nbt.4314&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               52.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.52" data-doi="10.1093/nar/28.1.27">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kanehisa
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goto
                                  </span>
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  KEGG: kyoto encyclopedia of genes and genomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic acids research
                                 </abbr>
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  27
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  30
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKanehisa%26rft.auinit1%253DM.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D30%26rft.atitle%253DKEGG%253A%2BKyoto%2BEncyclopedia%2Bof%2BGenes%2Band%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.27%26rft_id%253Dinfo%253Apmid%252F10592173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/28.1.27&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10592173&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000084896300007&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               53.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.53" data-doi="10.1093/nar/gkaa1024">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Martens
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  WikiPathways: connecting communities
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Research
                                 </abbr>
                                 <span class="cit-vol">
                                  49
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  D613
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D621
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D49%26rft.spage%253DD613%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkaa1024%26rft_id%253Dinfo%253Apmid%252F33211851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkaa1024&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=33211851&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               54.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.54" data-doi="10.1093/nar/gkp1002">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Frolkis
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  SMPDB: the small molecule pathway database
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic acids research
                                 </abbr>
                                 <span class="cit-vol">
                                  38
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  D480
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D487
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp1002%26rft_id%253Dinfo%253Apmid%252F19948758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkp1002&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19948758&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000276399100077&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               55.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.55" data-doi="10.1093/nar/gkx1132">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fabregat
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The reactome pathway knowledgebase
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic acids research
                                 </abbr>
                                 <span class="cit-vol">
                                  46
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  D649
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D655
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft.volume%253D46%26rft.spage%253DD649%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1132%26rft_id%253Dinfo%253Apmid%252F29145629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkx1132&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29145629&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               56.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.56" data-doi="10.1093/nar/gky1055">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  Consortium GO
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The gene ontology resource: 20 years and still GOing strong
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic acids research
                                 </abbr>
                                 <span class="cit-vol">
                                  47
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  D330
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D338
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft.volume%253D47%26rft.spage%253DD330%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky1055%26rft_id%253Dinfo%253Apmid%252F30395331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gky1055&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30395331&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               57.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.57" data-doi="10.1093/bioinformatics/btr260">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liberzon
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Subramanian
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pinchback
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thorvaldsdóttir
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tamayo
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mesirov
                                  </span>
                                  <span class="cit-name-given-names">
                                   JP.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Molecular signatures database (MSigDB) 3.0
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1739
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1740
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr260%26rft_id%253Dinfo%253Apmid%252F21546393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr260&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21546393&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000291261300036&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               58.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.58" data-doi="10.1073/pnas.0506580102">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Subramanian
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  102
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  15545
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  15550
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0506580102%26rft_id%253Dinfo%253Apmid%252F16199517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzQzLzE1NTQ1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDQvMTQvMjAyMC4wOC4yOC4yNzA2NTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               59.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.59" data-doi="10.1093/nar/gkj122">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cotter
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maer
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guda
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Saunders
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Subramaniam
                                  </span>
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Lmpd: lipid maps proteome database
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic acids research
                                 </abbr>
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  D507
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D510
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DCotter%26rft.auinit1%253DD.%26rft.volume%253D34%26rft.issue%253Dsuppl_1%26rft.spage%253DD507%26rft.epage%253DD510%26rft.atitle%253DLMPD%253A%2BLIPID%2BMAPS%2Bproteome%2Bdatabase%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkj122%26rft_id%253Dinfo%253Apmid%252F16381922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkj122&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16381922&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000239307700111&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               60.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.60">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Traag
                                  </span>
                                  <span class="cit-name-given-names">
                                   VA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Waltman
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Van Eck
                                  </span>
                                  <span class="cit-name-given-names">
                                   NJ.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  From Louvain to Leiden: guaranteeing well-connected communities
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Scientific reports
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  12
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScientific%2Breports%26rft.volume%253D9%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               61.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.61">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Umesh
                                  </span>
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Image processing in python
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  CSI Communications
                                 </abbr>
                                 <span class="cit-vol">
                                  23
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCSI%2BCommunications%26rft.volume%253D23%26rft.spage%253D2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               62.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.62" data-doi="10.1186/s13059-019-1874-1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hafemeister
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Satija
                                  </span>
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  296
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D20%26rft.spage%253D296%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-019-1874-1%26rft_id%253Dinfo%253Apmid%252F31870423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-019-1874-1&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=31870423&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               63.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.08.28.270652v2.63">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tim Stuart
                                  </span>
                                  <span class="cit-name-given-names">
                                   WWK.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Sinto: single-cell analysis tools
                                 </span>
                                 . Available at
                                 <a href="https://githubcom/timoast/sinto">
                                  https://githubcom/timoast/sinto
                                 </a>
                                 , ([Accessed:
                                 <span class="cit-date-in-citation">
                                  Nov 2021
                                 </span>
                                 ]).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               64.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.64" data-doi="10.1093/nar/gkw257">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ramírez
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  deepTools2: a next generation web server for deep-sequencing data analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic acids research
                                 </abbr>
                                 <span class="cit-vol">
                                  44
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  W160
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  W165
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw257%26rft_id%253Dinfo%253Apmid%252F27079975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkw257&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27079975&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F14%2F2020.08.28.270652.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               65.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.08.28.270652v2.65">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xie
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  bookdown: Authoring Books and Technical Documents with R Markdown
                                 </span>
                                 .
                                 <span class="cit-source">
                                  R package version 0.24
                                 </span>
                                 . Available at
                                 <a href="https://githubcom/rstudio/bookdown">
                                  https://githubcom/rstudio/bookdown
                                 </a>
                                 , (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               66.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.66">
                               <div class="cit-metadata">
                                <ol class="duplicate">
                                 <li>
                                  <span class="cit-ed cit-collab">
                                   Zenodo
                                  </span>
                                 </li>
                                </ol>
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ouyang
                                  </span>
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Cellxgene VIP snRNA-seq demo dataset for visualization and DE analysis
                                 </span>
                                 . (ed
                                 <span class="cit-ed cit-collab">
                                  Zenodo
                                 </span>
                                 ) (
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               67.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.28.270652v2.67">
                               <div class="cit-metadata">
                                <ol class="duplicate">
                                 <li>
                                  <span class="cit-ed cit-collab">
                                   Zenodo
                                  </span>
                                 </li>
                                </ol>
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  10X Genomics Human Visium Spatial Transcriptomics Demo Dataset for Cellxgene VIP
                                 </span>
                                 . (ed
                                 <span class="cit-ed cit-collab">
                                  Zenodo
                                 </span>
                                 ) (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.08.28.270652v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.07.29.454194v4" title="Interfacial tension driven open droplet microfluidics" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2021.06.26.450025v2" title="Risk modelling of single cell populations revealed the heterogeneity of immune infiltration in hepatocellular carcinoma" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted April 14, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.08.28.270652v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.08.28.270652v2.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/04/14/2020.08.28.270652.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.08.28.270652v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.08.28.270652v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   Cellxgene VIP unleashes full power of interactive visualization and integrative analysis of scRNA-seq, spatial transcriptomics, and multiome data
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2496640"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-f0V_KY1g7oh6d6V9TlFX2rXwH32ye4HU7pazmUDJkHs"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484477420"/>
                                   <input type="hidden" name="captcha_token" value="665b0977a1854ff3d085cda071d048ee"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2496640--3" data-node-nid="2496640" id="node-2496640--41458910467" data-pisa="biorxiv;2020.08.28.270652v2" data-pisa-master="biorxiv;2020.08.28.270652" data-seqnum="2496640" data-apath="/biorxiv/early/2022/04/14/2020.08.28.270652.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Cellxgene VIP unleashes full power of interactive visualization and integrative analysis of scRNA-seq, spatial transcriptomics, and multiome data
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Kejie
                                  </span>
                                  <span class="nlm-surname">
                                   Li
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Zhengyu
                                  </span>
                                  <span class="nlm-surname">
                                   Ouyang
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Yirui
                                  </span>
                                  <span class="nlm-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Jacob
                                  </span>
                                  <span class="nlm-surname">
                                   Gagnon
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Dongdong
                                  </span>
                                  <span class="nlm-surname">
                                   Lin
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Michael
                                  </span>
                                  <span class="nlm-surname">
                                   Mingueneau
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">
                                   Will
                                  </span>
                                  <span class="nlm-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="7">
                                  <span class="nlm-given-names">
                                   David
                                  </span>
                                  <span class="nlm-surname">
                                   Sexton
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="8">
                                  <span class="nlm-given-names">
                                   Baohong
                                  </span>
                                  <span class="nlm-surname">
                                   Zhang
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.08.28.270652;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.08.28.270652
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.08.28.270652v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.08.28.270652v2&title=Cellxgene%20VIP%20unleashes%20full%20power%20of%20interactive%20visualization%20and%20integrative%20analysis%20of%20scRNA-seq%2C%20spatial%20transcriptomics%2C%20and%20multiome%20data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.08.28.270652v2&title=Cellxgene%20VIP%20unleashes%20full%20power%20of%20interactive%20visualization%20and%20integrative%20analysis%20of%20scRNA-seq%2C%20spatial%20transcriptomics%2C%20and%20multiome%20data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.08.28.270652v2&text=Cellxgene%20VIP%20unleashes%20full%20power%20of%20interactive%20visualization%20and%20integrative%20analysis%20of%20scRNA-seq%2C%20spatial%20transcriptomics%2C%20and%20multiome%20data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.08.28.270652v2&t=Cellxgene%20VIP%20unleashes%20full%20power%20of%20interactive%20visualization%20and%20integrative%20analysis%20of%20scRNA-seq%2C%20spatial%20transcriptomics%2C%20and%20multiome%20data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.08.28.270652v2&title=Cellxgene%20VIP%20unleashes%20full%20power%20of%20interactive%20visualization%20and%20integrative%20analysis%20of%20scRNA-seq%2C%20spatial%20transcriptomics%2C%20and%20multiome%20data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.08.28.270652v2&title=Cellxgene%20VIP%20unleashes%20full%20power%20of%20interactive%20visualization%20and%20integrative%20analysis%20of%20scRNA-seq%2C%20spatial%20transcriptomics%2C%20and%20multiome%20data&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.08.28.270652v2&title=Cellxgene%20VIP%20unleashes%20full%20power%20of%20interactive%20visualization%20and%20integrative%20analysis%20of%20scRNA-seq%2C%20spatial%20transcriptomics%2C%20and%20multiome%20data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2496640--5" data-node-nid="2496640" id="citation-node-2496640--61097485300" data-pisa="biorxiv;2020.08.28.270652v2" data-pisa-master="biorxiv;2020.08.28.270652" data-seqnum="2496640" data-apath="/biorxiv/early/2022/04/14/2020.08.28.270652.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Cellxgene VIP unleashes full power of interactive visualization and integrative analysis of scRNA-seq, spatial transcriptomics, and multiome data
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Kejie
                                      </span>
                                      <span class="nlm-surname">
                                       Li
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Zhengyu
                                      </span>
                                      <span class="nlm-surname">
                                       Ouyang
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Yirui
                                      </span>
                                      <span class="nlm-surname">
                                       Chen
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Jacob
                                      </span>
                                      <span class="nlm-surname">
                                       Gagnon
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Dongdong
                                      </span>
                                      <span class="nlm-surname">
                                       Lin
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Michael
                                      </span>
                                      <span class="nlm-surname">
                                       Mingueneau
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">
                                       Will
                                      </span>
                                      <span class="nlm-surname">
                                       Chen
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="7">
                                      <span class="nlm-given-names">
                                       David
                                      </span>
                                      <span class="nlm-surname">
                                       Sexton
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="8">
                                      <span class="nlm-given-names">
                                       Baohong
                                      </span>
                                      <span class="nlm-surname">
                                       Zhang
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.08.28.270652;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.08.28.270652
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Cellxgene%20VIP%20unleashes%20full%20power%20of%20interactive%20visualization%20and%20integrative%20analysis%20of%20scRNA-seq%2C%20spatial%20transcriptomics%2C%20and%20multiome%20data&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Li%2C+Kejie&rft.au=Ouyang%2C+Zhengyu&rft.au=Chen%2C+Yirui&rft.au=Gagnon%2C+Jacob&rft.au=Lin%2C+Dongdong&rft.au=Mingueneau%2C+Michael&rft.au=Chen%2C+Will&rft.au=Sexton%2C+David&rft.au=Zhang%2C+Baohong">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2496640/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2496640/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2496640/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2496640/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2496640/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2496640/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2496640/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2496640/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2496640/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2496640/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2496640/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2496640/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.08.28.270652v2&count=horizontal&via=&text=Cellxgene%20VIP%20unleashes%20full%20power%20of%20interactive%20visualization%20and%20integrative%20analysis%20of%20scRNA-seq%2C%20spatial%20transcriptomics%2C%20and%20multiome%20data&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.08.28.270652v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.08.28.270652v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.08.28.270652v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Bioinformatics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15187)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
