<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.08.29.273698v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.08.29.273698v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.08.29.273698v2.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/embed/inline-graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/08/08/2020.08.29.273698.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.08.29.273698v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="DiffGR: Detecting Differentially Interacting Genomic Regions from Hi-C Contact Maps"/>
  <meta name="DC.Identifier" content="10.1101/2020.08.29.273698"/>
  <meta name="DC.Date" content="2022-08-08"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Recent advances in Hi-C techniques have allowed us to map genome-wide chromatin interactions and uncover higher-order chromatin structures, thereby shedding light on the principles of genome architecture and functions. However, statistical methods for detecting changes in large-scale chromatin organization such as topologically-associating domains (TADs) are still lacking. We proposed a new statistical method, DiffGR, for detecting differentially interacting genomic regions at the TAD level between Hi-C contact maps. We utilized the stratum-adjusted correlation coefficient to measure similarity of local TAD regions. We then developed a non-parametric approach to identify statistically significant changes of genomic interacting regions. Through simulation studies, we demonstrated that DiffGR can robustly and effectively discover differential genomic regions under various conditions. Furthermore, we successfully revealed cell type-specific changes in genomic interacting regions in both human and mouse Hi-C datasets, and illustrated that DiffGR yielded consistent and advantageous results compared with state-of-the-art differential TAD detection methods. The DiffGR R code is published under the GNU GPL ≥ 2 license and is publicly available at <https://github.com/wmalab/DiffGR>.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Huiling Liu"/>
  <meta name="DC.Contributor" content="Wenxiu Ma"/>
  <meta name="article:published_time" content="2022-08-08"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="DiffGR: Detecting Differentially Interacting Genomic Regions from Hi-C Contact Maps"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Recent advances in Hi-C techniques have allowed us to map genome-wide chromatin interactions and uncover higher-order chromatin structures, thereby shedding light on the principles of genome architecture and functions. However, statistical methods for detecting changes in large-scale chromatin organization such as topologically-associating domains (TADs) are still lacking. We proposed a new statistical method, DiffGR, for detecting differentially interacting genomic regions at the TAD level between Hi-C contact maps. We utilized the stratum-adjusted correlation coefficient to measure similarity of local TAD regions. We then developed a non-parametric approach to identify statistically significant changes of genomic interacting regions. Through simulation studies, we demonstrated that DiffGR can robustly and effectively discover differential genomic regions under various conditions. Furthermore, we successfully revealed cell type-specific changes in genomic interacting regions in both human and mouse Hi-C datasets, and illustrated that DiffGR yielded consistent and advantageous results compared with state-of-the-art differential TAD detection methods. The DiffGR R code is published under the GNU GPL ≥ 2 license and is publicly available at https://github.com/wmalab/DiffGR.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.08.29.273698v2"/>
  <meta name="citation_id" content="2020.08.29.273698v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.08.29.273698v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.08.29.273698v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.08.29.273698v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.08.29.273698"/>
  <meta name="citation_num_pages" content="23"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.08.29.273698"/>
  <meta name="citation_author" content="Huiling Liu"/>
  <meta name="citation_author_institution" content="Department of Statistics, University of California Riverside"/>
  <meta name="citation_author" content="Wenxiu Ma"/>
  <meta name="citation_author_institution" content="Department of Statistics, University of California Riverside"/>
  <meta name="citation_author_email" content="wenxiu.ma@ucr.edu"/>
  <meta name="citation_reference" content="Marieke Simonis, Petra Klous, Erik Splinter, Yuri Moshkin, Rob Willemsen, Elzo De Wit, Bas Van Steensel, and Wouter De Laat. Nuclear organization of active and inactive chromatin domains uncovered by chromosome conformation capture–on-chip (4c). Nature genetics, 38 (11):1348–1354, 2006."/>
  <meta name="citation_reference" content="Josée Dostie, Todd A Richmond, Ramy A Arnaout, Rebecca R Selzer, William L Lee, Tracey A Honan, Eric D Rubio, Anton Krumm, Justin Lamb, Chad Nusbaum, et al. Chromosome conformation capture carbon copy (5c): a massively parallel solution for mapping interactions between genomic elements. Genome research, 16(10):1299–1309, 2006."/>
  <meta name="citation_reference" content="Erez Lieberman-Aiden, Nynke L Van Berkum, Louise Williams, Maxim Imakaev, Tobias Ragoczy, Agnes Telling, Ido Amit, Bryan R Lajoie, Peter J Sabo, Michael O Dorschner, et al. Comprehensive mapping of long-range interactions reveals folding principles of the human genome. science, 326(5950):289–293, 2009."/>
  <meta name="citation_reference" content="Z. Duan, M. Andronescu, K. Schutz, S. McIlwain, Y. J. Kim, C. Lee, J. Shendure, S. Fields, C. A. Blau, and W. S. Noble. A three-dimensional model of the yeast genome. Nature, 465 (7296):363–367, 2010."/>
  <meta name="citation_reference" content="R. Kalhor, H. Tjong, N. Jayathilaka, F. Alber, and L. Chen. Genome architectures revealed by tethered chromosome conformation capture and population-based modeling. Nature biotechnology, 30(1):90–98, 2012."/>
  <meta name="citation_reference" content="Guoliang Li, Melissa J Fullwood, Han Xu, Fabianus Hendriyan Mulawadi, Stoyan Velkov, Vinsensius Vega, Pramila Nuwantha Ariyaratne, Yusoff Bin Mohamed, Hong-Sain Ooi, Chandana Tennakoon, et al. Chia-pet tool for comprehensive chromatin interaction analysis with paired-end tag sequencing. Genome biology, 11(2):R22, 2010."/>
  <meta name="citation_reference" content="Maxwell R Mumbach, Adam J Rubin, Ryan A Flynn, Chao Dai, Paul A Khavari, William J Greenleaf, and Howard Y Chang. Hichip: efficient and sensitive analysis of protein-directed genome architecture. Nature methods, 13(11):919–922, 2016."/>
  <meta name="citation_reference" content="Wenxiu Ma, Ferhat Ay, Choli Lee, Gunhan Gulsoy, Xinxian Deng, Savannah Cook, Jennifer Hesson, Christopher Cavanaugh, Carol B Ware, Anton Krumm, et al. Fine-scale chromatin interaction maps reveal the cis-regulatory landscape of human lincrna genes. Nature methods, 12(1):71–78, 2015."/>
  <meta name="citation_reference" content="Suhas SP Rao, Miriam H Huntley, Neva C Durand, Elena K Stamenova, Ivan D Bochkov, James T Robinson, Adrian L Sanborn, Ido Machol, Arina D Omer, Eric S Lander, et al. A 3d map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell, 159(7):1665–1680, 2014."/>
  <meta name="citation_reference" content="V. Ramani, D. A. Cusanovich, R. J. Hause, W. Ma, R. Qiu, X. Deng, C. A. Blau, C. M. Disteche, W. S. Noble, J. Shendure, and Z. Duan. Mapping 3D genome architecture through in situ DNase Hi-C. Nature protocols, 11(11):2104–2121, 2016."/>
  <meta name="citation_reference" content="Jesse R Dixon, Siddarth Selvaraj, Feng Yue, Audrey Kim, Yan Li, Yin Shen, Ming Hu, Jun S Liu, and Bing Ren. Topological domains in mammalian genomes identified by analysis of chromatin interactions. Nature, 485(7398):376–380, 2012."/>
  <meta name="citation_reference" content="Geoffrey Fudenberg, Maxim Imakaev, Carolyn Lu, Anton Goloborodko, Nezar Abdennur, and Leonid A Mirny. Formation of chromosomal domains by loop extrusion. Cell reports, 15(9): 2038–2049, 2016."/>
  <meta name="citation_reference" content="Fengling Chen, Guipeng Li, Michael Q Zhang, and Yang Chen. Hicdb: a sensitive and robust method for detecting contact domain boundaries. Nucleic acids research, 46(21):11239–11250, 2018."/>
  <meta name="citation_reference" content="Emily Crane, Qian Bian, Rachel Patton McCord, Bryan R Lajoie, Bayly S Wheeler, Edward J Ralston, Satoru Uzawa, Job Dekker, and Barbara J Meyer. Condensin-driven remodelling of x chromosome topology during dosage compensation. Nature, 523(7559):240, 2015."/>
  <meta name="citation_reference" content="Darya Filippova, Rob Patro, Geet Duggal, and Carl Kingsford. Identification of alternative topological domains in chromatin. Algorithms for Molecular Biology, 9(1):14, 2014."/>
  <meta name="citation_reference" content="Celine Lévy-Leduc, Maud Delattre, Tristan Mary-Huard, and Stephane Robin. Two-dimensional segmentation for analyzing hi-c data. Bioinformatics, 30(17):i386–i392, 2014."/>
  <meta name="citation_reference" content="Mattia Forcato, Chiara Nicoletti, Koustav Pal, Carmen Maria Livi, Francesco Ferrari, and Silvio Bicciato. Comparison of computational methods for hi-c data analysis. Nature methods, 14(7):679, 2017."/>
  <meta name="citation_reference" content="Zhijun Han and Gang Wei. Computational tools for hi-c data analysis. Quantitative Biology, 5(3):215–225, 2017."/>
  <meta name="citation_reference" content="Junbai Wang, Xun Lan, Pei-Yin Hsu, Hang-Kai Hsu, Kun Huang, Jeffrey Parvin, Tim HM Huang, and Victor X Jin. Genome-wide analysis uncovers high frequency, strong differential chromosomal interactions and their associated epigenetic patterns in e2-mediated gene regulation. BMC genomics, 14(1):70, 2013."/>
  <meta name="citation_reference" content="Jesse R Dixon, Inkyung Jung, Siddarth Selvaraj, Yin Shen, Jessica E Antosiewicz-Bourget, Ah Young Lee, Zhen Ye, Audrey Kim, Nisha Rajagopal, Wei Xie, et al. Chromatin architecture reorganization during stem cell differentiation. Nature, 518(7539):331, 2015."/>
  <meta name="citation_reference" content="Sven Heinz, Christopher Benner, Nathanael Spann, Eric Bertolino, Yin C Lin, Peter Laslo, Jason X Cheng, Cornelis Murre, Harinder Singh, and Christopher K Glass. Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and b cell identities. Molecular cell, 38(4):576–589, 2010."/>
  <meta name="citation_reference" content="Mark D Robinson, Davis J McCarthy, and Gordon K Smyth. edger: a bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics, 26(1):139–140, 2010."/>
  <meta name="citation_reference" content="Michael I Love, Wolfgang Huber, and Simon Anders. Moderated estimation of fold change and dispersion for rna-seq data with deseq2. Genome biology, 15(12):550, 2014."/>
  <meta name="citation_reference" content="Aaron TL Lun and Gordon K Smyth. diffhic: a bioconductor package to detect differential genomic interactions in hi-c data. BMC bioinformatics, 16(1):258, 2015."/>
  <meta name="citation_reference" content="John C Stansfield, Kellen G Cresswell, Vladimir I Vladimirov, and Mikhail G Dozmorov. Hiccompare: an r-package for joint normalization and comparison of hi-c datasets. BMC bioinformatics, 19(1):1–10, 2018."/>
  <meta name="citation_reference" content="Mohamed Nadhir Djekidel, Yang Chen, and Michael Q Zhang. Find: differential chromatin interactions detection using a spatial poisson process. Genome research, 28(3):412–422, 2018."/>
  <meta name="citation_reference" content="Kate B Cook, Borislav H Hristov, Karine G Le Roch, Jean Philippe Vert, and William Stafford Noble. Measuring significant changes in chromatin conformation with accost. Nucleic acids research, 48(5):2303–2311, 2020."/>
  <meta name="citation_reference" content="Phillippa C Taberlay, Joanna Achinger-Kawecka, Aaron TL Lun, Fabian A Buske, Kenneth Sabir, Cathryn M Gould, Elena Zotenko, Saul A Bert, Katherine A Giles, Denis C Bauer, et al. Three-dimensional disorganization of the cancer genome occurs coincident with long-range genetic and epigenetic alterations. Genome research, 26(6):719–731, 2016."/>
  <meta name="citation_reference" content="Kellen G Cresswell and Mikhail G Dozmorov. Tadcompare: An r package for differential and temporal analysis of topologically associated domains. Frontiers in Genetics, 11:158, 2020."/>
  <meta name="citation_reference" content="Raphaël Mourad. Tadreg: a versatile regression framework for tad identification, differential analysis and rearranged 3d genome prediction. BMC bioinformatics, 23(1):1–14, 2022."/>
  <meta name="citation_reference" content="Joachim Wolff, Leily Rabbani, Ralf Gilsbach, Gautier Richard, Thomas Manke, Rolf Backofen, and Björn A Grüning. Galaxy hicexplorer 3: a web server for reproducible hi-c, capture hi-c and single-cell hi-c data analysis, quality control and visualization. Nucleic Acids Research, 48(W1):W177–W184, 2020."/>
  <meta name="citation_reference" content="Joachim Wolff, Vivek Bhardwaj, Stephan Nothjunge, Gautier Richard, Gina Renschler, Ralf Gilsbach, Thomas Manke, Rolf Backofen, Fidel Ramírez, and Björn A Grüning. Galaxy hicexplorer: a web server for reproducible hi-c data analysis, quality control and visualization. Nucleic acids research, 46(W1):W11–W16, 2018."/>
  <meta name="citation_reference" content="Fidel Ramírez, Vivek Bhardwaj, Laura Arrigoni, Kin Chung Lam, Björn A Grüning, José Villaveces, Bianca Habermann, Asifa Akhtar, and Thomas Manke. High-resolution tads reveal dna sequences underlying genome organization in flies. Nature communications, 9(1):1–15, 2018."/>
  <meta name="citation_reference" content="Marie Zufferey, Daniele Tavernari, Elisa Oricchio, and Giovanni Ciriello. Comparison of computational methods for the identification of topologically associating domains. Genome biology, 19(1):1–18, 2018."/>
  <meta name="citation_reference" content="Tao Yang, Feipeng Zhang, Galip Gürkan Yardimci, Fan Song, Ross C Hardison, William Stafford Noble, Feng Yue, and Qunhua Li. Hicrep: assessing the reproducibility of hi-c data using a stratum-adjusted correlation coefficient. Genome research, 27(11):1939–1949, 2017."/>
  <meta name="citation_reference" content="Alan Agresti and Maria Kateri. Categorical data analysis. Springer, 2011."/>
  <meta name="citation_reference" content="Philip A Knight and Daniel Ruiz. A fast algorithm for matrix balancing. IMA Journal of Numerical Analysis, 33(3):1029–1047, 2013."/>
  <meta name="citation_reference" content="Yoav Benjamini and Yosef Hochberg. Controlling the false discovery rate: a practical and powerful approach to multiple testing. Journal of the Royal statistical society: series B (Methodological), 57(1):289–300, 1995."/>
  <meta name="citation_reference" content="Galip Gürkan Yardimci, Hakan Ozadam, Michael EG Sauria, Oana Ursu, Koon-Kiu Yan, Tao Yang, Abhijit Chakraborty, Arya Kaul, Bryan R Lajoie, Fan Song, et al. Measuring the reproducibility and quality of hi-c data. Genome biology, 20(1):1–19, 2019."/>
  <meta name="citation_reference" content="ENCODE Project Consortium et al. The encode (encyclopedia of dna elements) project. Science, 306(5696):636–640, 2004."/>
  <meta name="citation_reference" content="ENCODE Project Consortium et al. An integrated encyclopedia of dna elements in the human genome. Nature, 489(7414):57, 2012."/>
  <meta name="citation_reference" content="Mihaela Pertea, Daehwan Kim, Geo M Pertea, Jeffrey T Leek, and Steven L Salzberg. Transcript-level expression analysis of rna-seq experiments with hisat, stringtie and ballgown. Nature protocols, 11(9):1650–1667, 2016."/>
  <meta name="citation_reference" content="Brad T Sherman, Richard A Lempicki, et al. Systematic and integrative analysis of large gene lists using david bioinformatics resources. Nature protocols, 4(1):44, 2009."/>
  <meta name="citation_reference" content="Natalie Sauerwald, Akshat Singhal, and Carl Kingsford. Analysis of the structural variability of topologically associated domains as revealed by hi-c. NAR genomics and bioinformatics, 2 (1):qz008, 2020."/>
  <meta name="citation_reference" content="Evonne McArthur and John A Capra. Topologically associating domain boundaries that are stable across diverse cell types are evolutionarily constrained and enriched for heritability. The American Journal of Human Genetics, 108(2):269–283, 2021."/>
  <meta name="twitter:title" content="DiffGR: Detecting Differentially Interacting Genomic Regions from Hi-C Contact Maps"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Recent advances in Hi-C techniques have allowed us to map genome-wide chromatin interactions and uncover higher-order chromatin structures, thereby shedding light on the principles of genome architecture and functions. However, statistical methods for detecting changes in large-scale chromatin organization such as topologically-associating domains (TADs) are still lacking. We proposed a new statistical method, DiffGR, for detecting differentially interacting genomic regions at the TAD level between Hi-C contact maps. We utilized the stratum-adjusted correlation coefficient to measure similarity of local TAD regions. We then developed a non-parametric approach to identify statistically significant changes of genomic interacting regions. Through simulation studies, we demonstrated that DiffGR can robustly and effectively discover differential genomic regions under various conditions. Furthermore, we successfully revealed cell type-specific changes in genomic interacting regions in both human and mouse Hi-C datasets, and illustrated that DiffGR yielded consistent and advantageous results compared with state-of-the-art differential TAD detection methods. The DiffGR R code is published under the GNU GPL ≥ 2 license and is publicly available at <https://github.com/wmalab/DiffGR>.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="DiffGR: Detecting Differentially Interacting Genomic Regions from Hi-C Contact Maps"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.08.29.273698v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Recent advances in Hi-C techniques have allowed us to map genome-wide chromatin interactions and uncover higher-order chromatin structures, thereby shedding light on the principles of genome architecture and functions. However, statistical methods for detecting changes in large-scale chromatin organization such as topologically-associating domains (TADs) are still lacking. We proposed a new statistical method, DiffGR, for detecting differentially interacting genomic regions at the TAD level between Hi-C contact maps. We utilized the stratum-adjusted correlation coefficient to measure similarity of local TAD regions. We then developed a non-parametric approach to identify statistically significant changes of genomic interacting regions. Through simulation studies, we demonstrated that DiffGR can robustly and effectively discover differential genomic regions under various conditions. Furthermore, we successfully revealed cell type-specific changes in genomic interacting regions in both human and mouse Hi-C datasets, and illustrated that DiffGR yielded consistent and advantageous results compared with state-of-the-art differential TAD detection methods. The DiffGR R code is published under the GNU GPL ≥ 2 license and is publicly available at <https://github.com/wmalab/DiffGR>.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-08-08"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.08.29.273698v2"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2684813"/>
  <title>
   DiffGR: Detecting Differentially Interacting Genomic Regions from Hi-C Contact Maps | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"jwSsOEeP5EPTGm1efifUgjn5OWhCYMSNkcHC8qFH7os"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.08.29.273698v2"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.08.29.273698v2"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.08.29.273698v2"}],"nid":"2684813","apath":"\/biorxiv\/early\/2022\/08\/08\/2020.08.29.273698.atom","pisa":"biorxiv;2020.08.29.273698v2","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.08.29.273698v2","title":"DiffGR: Detecting Differentially Interacting Genomic Regions from Hi-C Contact Maps","identifier":"node\/2684813"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dFF8Z6bLSetrlXeoQXt\\\\\\\/EFrHYV1\\\\\\\/+JG+E5IxxyfQo8CeskC6EVQLvimNh18AMqH2VGljS2lIgOaTYV\\\\\\\/imyPuEtkkVsmYSPE9RyEUD3c+Ui6xaZkw3Bbs0LO94F4TE6rVZ85imhryzNR3Jhkmlh1eV+YC+O84CPvJf43tE74HBLHtVf1YhZACoyxDoh3CyVebfEXN\\\\\\\/wvauLuWwofFYe27oBdbfAUDHuqotmYJarduv4rlQAzc00Tr\\\\\\\/\\\\\\\/fl\\\\\\\/3Y4sZjO1hx0ghIa06trDOkty3ok\\\\\\\/fU2cxAPBgnIwBH9OtyPTohM31UQD+VIj8BEj7ud7BhPBTg1EdihP3lCeFGIKQoEegFVRMjxiKx+SyL0fhYg9NReSFC2a1+3\\\\\\\/H20Dxet681sdRyqfmJhI5zIP7Wyq65tfX9KHKu\\\\\\\/cYKapH5pWasNGMs4KcCW5zHUQPV8hPxYdAmo8tJvSQxysj+ZmyyzJ7vXCx6HBNDUSaDuyAInL306ouUxz7y8Nb+zFyQK+VKoL5ZMjObXvEeeBHzZIXmZ79F4SFKN9DOnWzIeoSbrP3XAoPLYXIFRtOLxxo1SH1OvllmCcfJZeLFxyPOkEmjpC47vl35WJr3nUNFOP8JeNBQ\\\\\\\/VSX7qqkldGy0yitoBdxOQ8DlSF62HvobJ2tbqsUpvYJVgGlTeSL2YFUBx1+D7ZYOaczH3lx836QCtN8rniGeGBamG+VTgwWjOFIfNqNaMefakUQRGg4TImdwGvA9lX60Qy\\\\\\\/RGm3tiUmvCbjPXO6k03rgUMvQeZKxDSnE7OoUyhXX8W7dhIHN6qwzNUx8i1XLnX\\\\\\\/Z\\\\\\\/aGKR4owv9ImprJKOV\\\\\\\/K09QTlMRFNCtc5YPE2Whxx9+2jpJi2x7iHVhZbCLtM4pl+CCBfuCM01R3fOarhxFHyYpls80XFm8cRqdoqVBP1qN9dDEh5B\\\\\\\/PXJFxm4PGuMnDDK7BKTeCawS7JQRmxqFKt\\\\\\\/icm6rDh13ID7dd8TSCMtFRicLliayKW80ulJBOAeF0FhGIpoI17JfHhzrMRj0ZrEquZxxGhqDI7ZwEO3mEIYXtxeWwn8n4MmN1CHtZw+ZjMrNjYaYTs9Bv1gQhTsEnQYOHNzW55tjlkexo3jDBkVPlKAxOKyIfimEWQtY3BFngb05MTF5DkJfl5tStBE0R4xaes0EMoxE7qykCl4+c0Z5OR2ZEqerPY+ZCSF\\\\\\\/vZJz9q3zzVsAUQvTjMO2cwEI9xcRHSDu+LUuJiLQNW6qXaB7XHESf7P2CFprvv7Cnmh4dhVIoqrffbzCF\\\\\\\/nxzlf8miiwuavJ9O2wITxmUWrmXlCgQdLO5atCme6cSvI1WTgYtNpXLxCycH1GfnjM+WzxrFThp3Ap543ArdBQEkciW7bsgafkNXtpt694fEsb8W1U=\\\u0022,\\\u0022iv\\\u0022:\\\u0022bZf1UPe52R28EPg2eQbS0A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00226f53526bd059af1f6da3cffa50d43151bedd3a3e2ae5d6d9eff448143fa7a5c1\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.08.29.273698v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2684813 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.08.29.273698v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_2144110889">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_2144110889" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_804149213" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-YXPqMqEDICCY6mQuIaDdzNzW7h9nMGIJp_do9SVFW6s"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2684813" data-node-nid="2684813" id="node-2684813--21411944962" data-pisa="biorxiv;2020.08.29.273698v2" data-pisa-master="biorxiv;2020.08.29.273698" data-apath="/biorxiv/early/2022/08/08/2020.08.29.273698.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    DiffGR: Detecting Differentially Interacting Genomic Regions from Hi-C Contact Maps
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first" data-delta="0">
                      <span class="nlm-given-names">
                       Huiling
                      </span>
                      <span class="nlm-surname">
                       Liu
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Wenxiu
                      </span>
                      <span class="nlm-surname">
                       Ma
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.08.29.273698
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2684813--21411944962" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Huiling Liu
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">
                        Department of Statistics, University of California Riverside
                       </span>
                       , Riverside, CA 92521,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Huiling%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Liu%20H&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AHuiling%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Wenxiu Ma
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">
                        Department of Statistics, University of California Riverside
                       </span>
                       , Riverside, CA 92521,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Wenxiu%2BMa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Ma%20W&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AWenxiu%2BMa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:wenxiu.ma@ucr.edu" class="" data-icon-position="" data-hide-link-title="0">
                        wenxiu.ma@ucr.edu
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.08.29.273698v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2684813" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2684813/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.08.29.273698v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2684813" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2684813/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.08.29.273698v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2684813" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2684813/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.08.29.273698v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2684813" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2684813/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.08.29.273698v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2684813" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2684813/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-2">
                             Recent advances in Hi-C techniques have allowed us to map genome-wide chromatin interactions and uncover higher-order chromatin structures, thereby shedding light on the principles of genome architecture and functions. However, statistical methods for detecting changes in large-scale chromatin organization such as topologically-associating domains (TADs) are still lacking. We proposed a new statistical method, DiffGR, for detecting differentially interacting genomic regions at the TAD level between Hi-C contact maps. We utilized the stratum-adjusted correlation coefficient to measure similarity of local TAD regions. We then developed a non-parametric approach to identify statistically significant changes of genomic interacting regions. Through simulation studies, we demonstrated that DiffGR can robustly and effectively discover differential genomic regions under various conditions. Furthermore, we successfully revealed cell type-specific changes in genomic interacting regions in both human and mouse Hi-C datasets, and illustrated that DiffGR yielded consistent and advantageous results compared with state-of-the-art differential TAD detection methods. The DiffGR R code is published under the GNU GPL ≥ 2 license and is publicly available at
                             <a href="https://github.com/wmalab/DiffGR">
                              https://github.com/wmalab/DiffGR
                             </a>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             1 Introduction
                            </h2>
                            <p id="p-5">
                             Recent developments of chromatin conformation capture (3C)-based techniques—including 4C [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ], 5C [
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ], Hi-C [
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             –
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ], ChIA-PET [
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ], and Hi-ChIP [
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             ]—have allowed high-throughput characterization of pairwise chromatin interactions in the cell nucleus, and provided an unprecedented opportunity to investigate the three-dimensional (3D) chromatin structures and to elucidate their roles in nuclear organization and gene expression regulation. Among these techniques, Hi-C and its variants [
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             –
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ] are of particular interest because of their ability to map chromatin interactions at a genome-wide scale.
                            </p>
                            <p id="p-6">
                             A Hi-C experiment yields a symmetric contact matrix in which each entry represents the chromatin contact frequency between the corresponding pair of genomic loci. A particularly important characteristic of Hi-C contact matrices is the presence of the topologically-associating domains (TADs), which are functional units of chromatin with higher tendency of intra-domain interactions [
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ]. TADs are largely conserved across cell types and species. Moreover, CTCF and other chromatin binding proteins are enriched at the TAD boundaries, indicating that TAD boundary regions form chromatin loops and play an essential role in gene expression regulation [
                             <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ,
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ].
                            </p>
                            <p id="p-7">
                             Several computational methods have been developed to detect TADs in Hi-C contact maps. These methods can be categorized into two groups: one-dimensional (1D) statistic-based methods and two-dimensional (2D) contact matrix-based methods [
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ]. Of these, 1D statistic-based methods often take a sliding window approach along the diagonal of Hi-C contact matrix and compute a 1D statistic for each diagonal bin to detect TADs and/or TAD boundaries. For instance, Dixon et al. [
                             <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ] introduced a statistic named directionality index (DI) to quantify whether a genomic locus preferentially interacts with upstream or downstream loci and developed a hidden Markov model to call TADs from DIs. Later, Crane et al. [
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ] proposed a novel TAD detection method, which computes an insulation score (IS) for each genomic bin by aggregating chromatin interactions within a square sliding through the diagonal and then searches for the minima along the IS profile as TAD boundaries. Unlike the 1D statistic-based methods which calculate statistics using local information, the 2D contact matrix-based methods utilize global information on the contact matrix to capture TAD structures. For example, the Armatus algorithm [
                             <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ] identifies consistent TAD patterns across different resolutions by maximizing a quality scoring function of domain partition using dynamic programming. In addition, Lévy-Leduc et al. [
                             <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             ] proposed a TAD boundary detection method named HiCseg, which performs a 2D block-wise segmentation via a maximum likelihood approach to partition each chromosome into its constituent TADs. Recently, several review papers have quantitatively compared the performances of the aforementioned TAD-calling methods and demonstrated that HiCseg detects a stable number of TADs against changes of sequencing coverage and maintains the highest reproducibility among Hi-C replicates across all resolutions when compared with other TAD-calling methods [
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             ,
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ].
                            </p>
                            <p id="p-8">
                             With the fast accumulation of Hi-C datasets, there has been a growing interest in performing differential analysis of Hi-C contact matrices. To date, several computational tools have been developed for comparative Hi-C analysis, but the majority of them focused on the identification of differential chromatin interactions (DCIs), which represent different chromatin looping events between two Hi-C contact maps. In early studies, the most common strategy for DCI detection was to use the fold change values between two Hi-C contact maps. For instance, Wang et al. [
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              19
                             </a>
                             ] used a simple fold-change strategy to detect the influence of estrogen treatment on chromatin interactions in MCF-7 Hi-C samples. Additionally, Dixon et al. [
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              20
                             </a>
                             ] utilized the fold change values of chromatin interactions to train a random forest model to discover the epigenetic signals that were more predictive of changes in interaction frequencies. In addition to these fold change-based approaches, another commonly utilized method for detecting DCIs was the binomial model implemented by the HOMER software [
                             <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                              21
                             </a>
                             ]. In contrast, in more recent studies, count-based statistical methods, such as edgeR [
                             <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                              22
                             </a>
                             ] and DESeq [
                             <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ], have been adopted to identify pairwise chromatin interactions that show significant changes in contact frequencies. Among them, Lun and Smyth [
                             <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                              24
                             </a>
                             ] presented a tool named diffHic for rigorous detection of differential interactions by leveraging the generalized linear model (negative binomial regression) of edgeR, and demonstrated that edgeR outperformed the binomial model. Later, Stansfield et al. [
                             <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                              25
                             </a>
                             ] introduced MD normalization and performed Z-tests to detect statistically significant DCIs. While all these methods assumed independence among pairwise interactions, which holds true only in coarse-resolution Hi-C maps, Djekidel et al. [
                             <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                              26
                             </a>
                             ] presented a novel method, named FIND, that takes into account the dependency of adjacent loci at finer resolutions. Briefly, FIND utilizes a spatial Poisson process model to detect DCIs that show significant changes in interaction frequencies of both themselves and their neighborhood bins. Lastly, Cook et al. [
                             <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                              27
                             </a>
                             ] introduced ACCOST to identify differential chromatin contacts by extending the DESeq model used in RNA-seq analysis and repurposing the “size factor” to account for the notable genomic-distance effect in Hi-C contact matrices.
                            </p>
                            <p id="p-9">
                             In the cell nucleus, chromatin is organized at multiple levels, ranging from active and inactive chromosomal compartments and sub-compartments (on a multi-Mb scale) [
                             <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ,
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ], TADs (0.5–2 Mb on average) [
                             <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ], to fine-scale chromatin interacting loops [
                             <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ,
                             <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ]. Chromatin structures also exhibit multi-scale differences among different cell types in their compartments, TADs, and chromatin loops. Among these, changes in TAD organizations are of particular interest as TADs are strongly linked to cell type-specific gene expression [
                             <a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ]. For example, Taberlay et al. [
                             <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                              28
                             </a>
                             ] have shown that genomic rearrangements in cancer cells are partly guided by changes in higher-order chromatin structures, such as TADs. They discovered that some large TADs in normal cells are further segmented into several smaller TADs in cancer cells, and these changes are tightly correlated with oncogene expression levels. Current differential analyses of TAD structures between different cell types and conditions are limited to the detection of TAD boundary changes. Recently, Chen et al. [
                             <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ] proposed a TAD boundary detection approach named HiCDB, which is constructed based on local measures of relative insulation and multi-scale aggregation. In addition to calling TAD boundaries in single Hi-C sample, HiCDB also provides differential TAD boundary detection using the average values of relative insulation across multiple samples. Later, Cresswell and Dozmorov [
                             <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                              29
                             </a>
                             ] developed TADCompare, which uses a spectral clustering-derived metric named eigenvector gap to identify differential and consensus TAD boundaries and track TAD boundary changes over time. Lastly, TADreg [
                             <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                              30
                             </a>
                             ] introduced a versatile regression framework which generalizes the insulation score by estimating the relative insulating effects of genomic loci and adding a sparsity constraint. The TADreg framework was designed for TAD boundary detection, but also allowed differential TAD analysis across various conditions. The HiCDB, TADCompare and TADreg methods focused on detecting changes in TAD boundaries rather than changes in chromatin organization within TADs. However, differential TAD boundaries do not necessarily indicate differential chromatin conformation within those regions. First, Hi-C contact matrices are often sparse and noisy, which might lead to unstable detection of TAD boundaries. Second, chromatin interactions within a TAD could be strengthened or weakened in another Hi-C sample, which would suggest different patterns of chromatin organization within the same TAD region. Unfortunately, few methods have been developed to detect differential TAD regions instead of boundaries. Recently, the Hi-C preprocessing and analysis tool HiCExplorer [
                             <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                              31
                             </a>
                             –
                             <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                              33
                             </a>
                             ] expanded its functions to capture differential TAD regions by comparing the precomputed TAD regions on the target Hi-C map with the same regions on the control map by accounting for the information in both intra-TAD and inter-TAD regions. However, such comparison was only limited to the precomputed genomic regions in only one of the Hi-C conditions. Thus, appropriate statistical methods for detecting differentially interacting regions by considering TAD regions across both conditions are still lacking.
                            </p>
                            <p id="p-10">
                             To tackle this problem, we developed a novel statistical method, DiffGR, for detecting differential genomic regions at TAD level between two Hi-C contact maps. Briefly, DiffGR utilizes the stratum-adjusted correlation coefficient (SCC), which effectively eliminates the genomic-distance effect in Hi-C data, to measure the similarity of local genomic regions between two contact matrices. Subsequently, DiffGR applies a nonparametric permutation test on those SCC values to detect genomic regions with statistically significant differential interactions. We demonstrated, through simulation studies and real data analyses, that DiffGR can effectively and robustly identify differentially interacting genomic regions at TAD level.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             2 Methods
                            </h2>
                            <p id="p-11">
                             The DiffGR method detects differentially interacting genomic regions in three steps, as shown in
                             <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                              Figure 1A
                             </a>
                             and described below in
                             <a id="xref-sec-3-1" class="xref-sec" href="#sec-3">
                              Sections 2.1
                             </a>
                             -
                             <a id="xref-sec-5-1" class="xref-sec" href="#sec-5">
                              2.3
                             </a>
                             . In addition, the simulation settings are outlined in
                             <a id="xref-sec-6-1" class="xref-sec" href="#sec-6">
                              Section 2.4
                             </a>
                             and real data preprocessing and analyses are described in
                             <a id="xref-sec-10-1" class="xref-sec" href="#sec-10">
                              Section 2.5
                             </a>
                             .
                            </p>
                            <div id="F1" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F1.large.jpg?width=800&height=600&carousel=1" title="Overview of DiffGR. A. Workflow of the DiffGR algorithm. B. Illustration of three candidate types of differential genomic regions. The gray vertical bars represent the common TAD boundaries between two conditions, which partition the genome into three types of candidate regions. The blue points stand for unique TAD boundaries in only one of the two conditions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-794947281" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Overview of DiffGR.</span> A. Workflow of the DiffGR algorithm. B. Illustration of three candidate types of differential genomic regions. The gray vertical bars represent the common TAD boundaries between two conditions, which partition the genome into three types of candidate regions. The blue points stand for unique TAD boundaries in only one of the two conditions.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F1.medium.gif" width="440" height="108"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F1.medium.gif" width="440" height="108"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                              <span class="fig-label">
                               Figure 1:
                              </span>
                              <span class="caption-title">
                               Overview of DiffGR.
                              </span>
                              <p id="p-12" class="first-child">
                               A. Workflow of the DiffGR algorithm. B. Illustration of three candidate types of differential genomic regions. The gray vertical bars represent the common TAD boundaries between two conditions, which partition the genome into three types of candidate regions. The blue points stand for unique TAD boundaries in only one of the two conditions.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="sec-3" class="subsection">
                             <h3>
                              2.1 Identifying candidate genomic regions
                             </h3>
                             <p id="p-13">
                              Suppose we have two sets of Hi-C data and their corresponding contact frequency matrices as the input. First, we detect the TAD boundaries in each Hi-C data, separately. Specifically, we apply HiCseg [
                              <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              ] to the raw contact matrices and obtain the corresponding TAD boundaries. Note that in this step one can change HiCseg with any other TAD caller, whose detected TADs satisfy the non-overlapping and continuous properties. We choose HiCseg because it has been shown that HiCseg produces more robust and reliable TAD boundaries than other TAD-calling methods [
                              <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">
                               17
                              </a>
                              ,
                              <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ]. We next combine the TAD boundaries from both Hi-C contact maps to identify the candidate genomic regions for subsequent analyses. TAD boundaries within two-bin distance are considered to be a common boundary shared by both Hi-C datasets and replaced by the middle bin locus. We then partition the genome into non-overlapping candidate regions using the common TAD boundaries, and categorize these candidate regions into the following three groups: (1) single-TAD candidate regions, (2) hierarchical-TAD candidate regions, and (3) complex-TAD candidate regions, as illustrated in
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Figure 1B
                              </a>
                              .
                             </p>
                             <p id="p-14">
                              We expected different patterns of differential features in these three kinds of candidate genomic regions. As to the differential single-TAD region, we would expect strength changes occurred in such areas. For differential hierarchical-TAD regions, one large interacting domain could be evidently split into two or more sub-domains, or vice versa, boundaries between TADs disappeared and thus the corresponding domains merged in one of the contact maps. Lastly, domains might be split, merged, or shifted in a more complicated manner thereby constructing an entirely new structure, which would be defined as differential complex-TAD regions. Unlike differential single-TAD regions, the differential hierarchical-TAD and complex-TAD regions represent more disruptive changes in the 3D structure of the chromatin.
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              2.2 Measuring similarity of candidate regions between two Hi-C contact maps
                             </h3>
                             <p id="p-15">
                              In the second step, we evaluate the similarity of each candidate region between the two samples. Suppose a candidate genomic region is bounded by two common TAD boundaries shared by both Hi-C maps, and contains k unique TAD boundaries in either one of the two Hi-C maps (shown as blue points in
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Figure 1B
                              </a>
                              ). In the single-TAD candidate region, k = 0; in the hierarchical-TAD or complex-TAD candidate regions, k >= 1. For each candidate region, we consider all
                              <span class="inline-formula" id="inline-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/inline-graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/inline-graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              possible (sub)TADs, which are separated by any pair of TAD boundaries within that region, as potential differential TADs. For each potential differential TAD, we calculate the stratum-adjusted correlation coefficient (SCC) [
                              <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                               35
                              </a>
                              ] rather than the standard Pearson or Spearman correlation coefficients (CCs) to measure the similarity of intra-TAD chromatin interactions between two Hi-C samples. The advantages of using SCC instead of standard CCs are shown in Supplementary Results in File S1.
                             </p>
                             <p id="p-16">
                              The SCC metric was introduced by Yang et al. [
                              <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">
                               35
                              </a>
                              ] as a measure of similarity and reproducibility between two Hi-C contact matrices. To account for the pronounced distance-dependence effect in Hi-C contact maps, chromatin contacts are first stratified into K stratum according to the genomic distances of the contacting loci pairs, and the correlation coefficients of contacts within each stratum are calculated between two samples. These stratum-specific correlation coefficients are then aggregated to compute the SCC value using a weighted average approach, where the weights are derived from the Cochran-Mantel-Haenszel (CMH) statistic [
                              <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                               36
                              </a>
                              ]. That is, the SCC ρ is calculated as
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where N
                              <sub>
                               k
                              </sub>
                              is the number of elements in the k-th stratum, r
                              <sub>
                               2k
                              </sub>
                              is the product of standard deviations of the elements in the k-th stratum of both samples, and ρ
                              <sub>
                               k
                              </sub>
                              denotes the correlation coefficient of the k-th stratum between two samples.
                             </p>
                             <p id="p-17">
                              The original SCC metric is computed using the intra-chromosomal contact matrices with a predefined genomic distance limit. The resulting value has a range of [−1, 1] and can be interpreted in a way similar to the standard correlation coefficient. Here we use SCC as a local similarity measurement to evaluate each potential differential TAD between two Hi-C samples. In the SCC calculation, an upper limit of genomic distance is set to 10 Mb because TADs are commonly smaller than 10 Mb and distal interactions over a genomic distance larger than 10 Mb are often sparse and highly stochastic. In addition, as the sparsity of Hi-C matrices might affect the precision of SCC values, the loci pairs with zero contact frequencies in both samples are excluded from the calculation.
                             </p>
                             <p id="p-18">
                              Hi-C contact maps are often sparse due to sequencing coverage limits and contain various systematic biases. To solve these issues, when preprocessing the Hi-C contact matrices, we first smooth each contact map by a 2D mean filter [
                              <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">
                               35
                              </a>
                              ], which substitutes the contact count observed between each bin pair by the average contact count in its neighborhood. This smoothing process improves the contiguity of the TAD regions with elevated contact frequencies, thereby enhancing the domain structures. Next, we utilize the Knight-Ruiz (KR) normalization [
                              <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                               37
                              </a>
                              ] on the smoothed matrices to remove potential biases.
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              2.3 Detecting statistically significant differential regions
                             </h3>
                             <p id="p-19">
                              In the third step, we identify differential genomic regions by first finding differential TADs within these candidate regions. In each candidate genomic region, we calculate the SCC values for all potential differential TADs as described above. Then we develop a nonparametric permutation test to estimate the p-values for these local SCC values. Additionally, we propose a quantile regression strategy to speed up the permutation test (see details in Supplementary Method in File S1). Finally, we consider a candidate region to be a differentially interacting genomic region, if at least one TAD within that region exhibits a statistically significant difference between the two samples and the size of the largest differential TAD meeting this criterion is greater than one third of the length of the entire candidate region. The longest differential TADs within the detected differentially interacting genomic regions are defined as the noticeable differential areas.
                             </p>
                             <p id="p-20">
                              Specifically, we perform the following nonparametric permutation test for each unique TAD size, as the local SCC values are calculated for all potential differential TADs of various sizes.
                             </p>
                             <p id="p-21">
                              Suppose s is a potential differential TAD whose length is l
                              <sub>
                               s
                              </sub>
                              and SCC value between two Hi-C samples is ρ
                              <sub>
                               s
                              </sub>
                              . To assess the statistical significance of the observed SCC value ρ
                              <sub>
                               s
                              </sub>
                              , the null distribution of SCC values for TADs of the same size is estimated via the following permutation procedure. To generate a random TAD with length l
                              <sub>
                               s
                              </sub>
                              , we first randomly select l
                              <sub>
                               s
                              </sub>
                              positions from main diagonal of Hi-C contact matrix, then l
                              <sub>
                               s
                              </sub>
                              − 1 position from the first off-diagonal, …, and lastly 1 position from the (l
                              <sub>
                               s
                              </sub>
                              − 1)-th off-diagonal. We subsequently extract contact counts of these randomly selected positions from the two Hi-C contact matrices to construct the permuted TAD pair and calculate its SCC value. We repeat the above random TAD generation step N times (N = 2000) and obtain the corresponding SCC values
                              <span class="inline-formula" id="inline-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/inline-graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/inline-graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Then the p-value of the observed SCC value ρ
                              <sub>
                               s
                              </sub>
                              can be computed as:
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where I(·) is the indicator function. Lastly, we compare the p-values with a pre-defined significance level α (by default α = 0.05) to determine differential TADs meeting the significance threshold. Note that the permutation framework accounts for the multiple testing correction using the Benjamini-Hochberg procedure [
                              <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                               38
                              </a>
                              ].
                             </p>
                             <p id="p-22">
                              One potential issue of this permutation framework is the false detection of significantly differential TADs when the two samples are highly similar (e.g., biological replicates from same experiment). This is because the high similarity between biological replicates would lead to high SCC values of the corresponding random TAD patterns. As a result, some non-differential TADs with relatively low SCC values would be falsely detected as differential ones. In order to reduce the number of false positives, we provide an option to filter the p-values p
                              <sub>
                               s
                              </sub>
                              by an empirical or automatically calculated threshold. This optional filtering step allows us to pre-specify the meaningful SCC between the two Hi-C datasets that should be reached in order to call a differential TAD truly significant.
                              <span class="disp-formula" id="disp-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-23">
                              The threshold θ can normally be defined as 0.85, which corresponds to a clear margin separating non-replicates from biological/pseudo-replicates in the whole-chromosome similarity comparison between multiple cell lines [
                              <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                               39
                              </a>
                              ]. Alternatively, θ can be calculated automatically as
                              <span class="inline-formula" id="inline-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/inline-graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/inline-graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <span class="inline-formula" id="inline-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/inline-graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/inline-graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              represents the mean α quantile of SCCs between non-replicate data and
                              <span class="inline-formula" id="inline-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/inline-graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/inline-graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the mean α quantile of SCCs between their corresponding biological/pseudo-replicate data. Here, we call matrices from different cell lines as non-replicates, matrices from the same cell type as biological replicates, and matrices sampled from pooled biological replicates as pseudo-replicates.
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              2.4 Simulation settings
                             </h3>
                             <p id="p-24">
                              To evaluate the performance of the DiffGR method, we conducted a series of simulation experiments by varying the proportion of altered TADs, proportion of TAD alternation, noise level, and sequencing coverage level. Specifically, we utilized the published chromosome 1 contact matrix of K562 cells at 50-kb resolution [
                              <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ] as the original Hi-C data and simulated the altered Hi-C contact matrices as described below.
                             </p>
                             <div id="sec-7" class="subsection">
                              <h4>
                               2.4.1 Single-TAD alternation
                              </h4>
                              <p id="p-25">
                               Since TADs are conserved genomic patterns and TAD boundaries are relatively stable across cell types and even across species [
                               <a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">
                                11
                               </a>
                               ], our simulations primarily focused on the scenarios of single-TAD alternations. Suppose we had an original Hi-C contact matrix M and its identified TAD boundaries. Each of our simulated Hi-C matrices contained two components: the signal matrix S and the noise matrix N, with a certain signal-to-noise ratio.
                              </p>
                              <p id="p-26">
                               First, to construct the signal matrix S, we randomly selected a subset of TADs from original contact matrix to serve as the true differential TADs. Then we replaced a certain portion of contact counts in each selected TAD by randomly sampling contact counts from the corresponding diagonals of the contact matrix. Second, we simulated the noise matrix N which represents the random ligation events in Hi-C experiments. Briefly, we generated these contacts by randomly choosing two bins, i and j, and adding one to the entry N
                               <sub>
                                ij
                               </sub>
                               in the noise matrix. The probability of sampling each bin in the bin pair was set proportional to the marginal count of that bin in the original matrix. The sampling process was repeated C times, where C was the total number of contacts in the original Hi-C contact matrix M. The resulting random ligation noise matrix N contained the same number of contacts as the original contact matrix M.
                              </p>
                              <p id="p-27">
                               To summarize, we had the following parameters in our single-TAD simulations.
                              </p>
                              <ul class="list-unord" id="list-1">
                               <li id="list-item-1">
                                <p id="p-28">
                                 proportion of altered TADs. Using HiCseg, we detected 189 TADs with a mean size of 1.2 Mb in the original K562 chromosome 1 contact matrix (Supplementary Figure S1). By default, we set the proportion of altered TADs to be 50%, which can vary from 20% to 70%.
                                </p>
                               </li>
                               <li id="list-item-2">
                                <p id="p-29">
                                 proportion of TAD alternation. In the default setting, we substituted all contact counts in the selected TADs by random counts permuted from the matching diagonals in Hi-C maps. To reduce the degree of intra-TAD alternation, we gradually decreased the proportion of randomly substituted intra-TAD contacts from 100% to 10%.
                                </p>
                               </li>
                               <li id="list-item-3">
                                <p id="p-30">
                                 noise level, i.e., the ratio between the noise and signal matrices. The noise level was set to 10% by default, and varied from 1% to 80%.
                                </p>
                               </li>
                              </ul>
                              <p id="p-31">
                               For each simulation parameter setting, we generated 100 altered Hi-C contact matrices to compare against the original contact matrix. To evaluate the accuracy of the detection results, we used the false detection rate which defines as inaccurate percentage and is computed as
                               <span class="inline-formula" id="inline-formula-6">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/inline-graphic-6.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/inline-graphic-6.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , where FP denotes the falsely detected differential regions, FN represents the the falsely detected non-differential regions, and N is the total number of candidate regions being tested.
                              </p>
                             </div>
                             <div id="sec-8" class="subsection">
                              <h4>
                               2.4.2 Hierarchical-TAD alternation
                              </h4>
                              <p id="p-32">
                               In addition to single-TAD alternation, we also simulated the alternation pattern of hierarchical TADs. We randomly selected 50% of the large TADs whose size was greater than 10 bins in the signal matrix to serve as the true differential TADs. For each of the selected large TAD, we chose a random subTAD boundary to split it into two smaller subTADs (each with size > 5 bins). We then replaced all inter-subTAD contact counts by randomly sampled counts in Hi-C maps. Next, we validated the performance of DiffGR under the hierarchical-TAD condition with respect to different noise levels similar to the single-TAD simulations. Because the complex-TAD condition has complicated TAD boundaries between two samples and occurs less frequently in real data, we did not generate simulation data for this condition.
                              </p>
                             </div>
                             <div id="sec-9" class="subsection">
                              <h4>
                               2.4.3 Simulating low-coverage contact matrices
                              </h4>
                              <p id="p-33">
                               Low sequencing depth of Hi-C experiments would lead to low-coverage and sparse contact matrices, thus it could potentially affect the performance of the detection of differentially interacting regions. To simulate low-coverage contact matrices, we started with a deep-sequenced Hi-C contact map obtained from human GM12878 cells [
                               <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">
                                9
                               </a>
                               ], and down-sampled the contact counts to generate lower-coverage matrices. Specifically, for each non-zero contact count M
                               <sub>
                                ij
                               </sub>
                               in the original matrix, we assumed that the simulated contact count follows a binomial distribution
                               <span class="inline-formula" id="inline-formula-7">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/inline-graphic-7.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/08/2020.08.29.273698/embed/inline-graphic-7.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , where the binomial parameter p = {0.2, 0.4, 0.6, 0.8, 1.0} represents the relative coverage level of the down-sampled contact matrix M′. In addition, 10% noise were added to the down-sampled matrices.
                              </p>
                             </div>
                            </div>
                            <div id="sec-10" class="subsection">
                             <h3>
                              2.5 Real data preprocessing steps
                             </h3>
                             <p id="p-34">
                              In our real data analysis, we used two published Hi-C datasets by Rao et al. [
                              <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ] (GEO accession GSE63525) and Dixon et al. [
                              <a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                              ](GEO accession GSE35156). The Rao et al. [
                              <a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ] dataset include five human cell types: B-lymphoblastoid cells (GM12878), mammary epithelial cells (HMEC), umbilical vein endothelial cells (HUVEC), erythrocytic leukemia cells (K562), and epidermal keratinocytes (NHEK). The GM12878 dataset contains two replicates, which were also pooled together in cell type-specific comparison. The Dixon et al. [
                              <a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                              ] dataset are from mouse embryonic stem (ES) and cortex cells. Two replicates from mouse ES cells were merged together in cell type-specific comparison. We applied DiffGR to detect differential genomic regions between each pair of cell types at 25-kb, 50-kb, and 100-kb resolutions. Since some of these Hi-C datasets were not deeply sequenced, the local variations introduced by low sequencing coverage made it challenging to capture large domain structures, especially in fine-resolution analyses. Therefore, to enhance the domain structures, all contact matrices were first preprocessed by a 2D mean filter smoothing and then normalized by the KR method to eliminate potential biases.
                             </p>
                             <p id="p-35">
                              In addition to Hi-C contact maps, ChIP-seq and RNA-seq data from the same cell lines were also included in real data analyses. For ChIP-seq analysis, CTCF and histone modification (H3K4me1, H3K4me2, H3K27me3, and H3K36me3) datasets from five human cell lines in Rao et al. [
                              <a id="xref-ref-9-7" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ], and CTCF, Polr2a, and histone modification (H3K4me1, H3K4me3, and H3K27ac) datasets from mouse cell lines in Dixon et al. [
                              <a id="xref-ref-11-9" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                              ] were obtained from the ENCODE project [
                              <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                               40
                              </a>
                              ,
                              <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                               41
                              </a>
                              ] (
                              <a href="https://www.encodeproject.org/">
                               https://www.encodeproject.org/
                              </a>
                              ). The ChIP-seq peak files were in narrowpeak/broadpeak BED format. The ChIP-seq peaks were aggregated into fixed-size bins with the same resolution as the Hi-C data, and the bin-wise peak counts were normalized by the total number of peaks in each ChIP-seq dataset. The absolute mean differences of the normalized bin-wise peak counts were calculated for each pair of cell lines for the subsequent analyses. In addition, RNA-seq datasets were also obtained from the ENCODE project [
                              <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">
                               41
                              </a>
                              ] for human GM12878 and K562 cells (GEO accession GSE78552 and GSE78625) in read count format, and for mouse ES and cortex cells (GEO accession GSM723776 and GSM723769) in FPKM format.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-11">
                            <h2 class="">
                             3 Results
                            </h2>
                            <div id="sec-12" class="subsection">
                             <h3>
                              3.1 DiffGR accurately detected single-TAD differences in simulated datasets
                             </h3>
                             <p id="p-36">
                              To validate the accuracy and efficiency of our DiffGR method, we first generated pairs of original and simulated Hi-C contact matrices, where a given proportion of TADs in the simulated contact matrices were altered (see Methods). We used the intra-chromosomal contact matrix of chromosome 1 in K562 cells at 50-kb resolution to serve as the original contact matrix. At the default setting, we altered 50% of the original TADs by completely replacing the intra-TAD contact counts by randomly sampled counts outside the TAD regions. In addition, we added 10% random-ligation noise into the altered contact matrices.
                             </p>
                             <p id="p-37">
                              We first simulated Hi-C matrices with various proportions of altered TADs (20%, 30%, 40%, 50%, 60%, and 70%). With each proportion setting, we completely mutated the intra-TAD counts and added 10% noise, and repeated this simulation procedure 100 times. As expected, the performance of the DiffGR method depended on the proportion of altered TADs. As shown in
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Figure 2A
                              </a>
                              and Supplementary Table S1, when the proportion of altered TADs changed from 20% to 70%, the false detection rate increased from 0.01 to 0.21. One possible explanation of this observed trend is that when the majority of TADs were altered, the large differences between the original and altered matrices would affect the permutation test and therefore lead to inaccurate detection. However, differential TADs rarely exist in large proportion in real data. The false detection rates of our method remained below 0.07 when the proportion of altered TADs was smaller than or equal to 50%, which demonstrated that our method can accurately and reliably detect single-TAD differences under these conditions.
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F2.large.jpg?width=800&height=600&carousel=1" title="Performance of single-TAD simulations. The curves display the mean false detection rates at different levels of A. proportion of altered TADs, B. proportion of TAD alternation, C. noise, and D. sequencing coverage. Vertical bars represent 95% confidence intervals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-794947281" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Performance of single-TAD simulations.</span> The curves display the mean false detection rates at different levels of A. proportion of altered TADs, B. proportion of TAD alternation, C. noise, and D. sequencing coverage. Vertical bars represent 95% confidence intervals.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F2.medium.gif" width="440" height="405"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F2.medium.gif" width="440" height="405"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2:
                               </span>
                               <span class="caption-title">
                                Performance of single-TAD simulations.
                               </span>
                               <p id="p-38" class="first-child">
                                The curves display the mean false detection rates at different levels of A. proportion of altered TADs, B. proportion of TAD alternation, C. noise, and D. sequencing coverage. Vertical bars represent 95% confidence intervals.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-39">
                              In the default simulation setting, we completely altered the selected TADs by substituting all intra-TAD contact counts by randomly sampled counts from the matching diagonals outside the TADs. To investigate the influence of the degree of TAD alternation on the DiffGR performance, we generated a series of simulated contact matrices, in which half of original TADs were altered and the proportion of intra-TAD alternation varied from 10%, to 20%, 30%, 50%, 80%, and 100%. In theory, TADs with higher degrees of alternation are easier to identify, whereas TADs with minor changes remain difficult to be detected. As illustrated in
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Figure 2B
                              </a>
                              and Supplementary Table S2, the performance of DiffGR improved resulting in higher accuracy as the percentage of randomly substituted counts in altered TADs increased. Even with the most challenging case where only 10% of the intra-TAD counts were altered, the accuracy of our method was 0.73, suggesting that DiffGR can effectively detect subtle TAD differences.
                             </p>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              3.2 DiffGR performed stably against changes in noise and coverage levels
                             </h3>
                             <p id="p-40">
                              Next we sought to evaluate the robustness of our method under various noise levels and sequencing coverage conditions.
                             </p>
                             <p id="p-41">
                              In the earlier simulations, we added 10% noise to the simulated differential contact matrices. To evaluate the performance of our method under different noise levels, we fixed the proportion of altered TADs at 50% and the proportion of intra-TAD alternation at 100%, and simulated the differential contact matrices with a wide range of noise levels (1%, 5%, 10%, 20%, 50%, and 80%). Intuitively, a good detection method should easily discover the differential regions in the less noisy matrices, and it becomes more challenging to detect the differential regions in the noisier cases. Our results demonstrated that DiffGR was able to correctly rank the simulated datasets. We observed a monotonic increasing trend of the false detection rate and a decreasing tendency of other precision measures as the noise levels raised (
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Figure 2C
                              </a>
                              and Supplementary Table S3). With moderate noise levels that were not greater than 20%, the accuracy of DiffGR remained above 0.93, indicating that our method can correctly detect differential TAD regions in such noisy cases.
                             </p>
                             <p id="p-42">
                              The sequencing coverage of the Hi-C contact maps is another major factor that could affect the performance of our method. Considering two Hi-C replicates that have the same underlying TAD structures but different sequencing coverage levels, we questioned whether our DiffGR method can correctly categorize them as non-differential. In other words, we intended to estimate the false positive rates caused by low-coverage and sparse Hi-C data. To directly investigate the influence of the sequencing coverage on the detection of differential regions, we utilized the GM12878 chromosome 1 contact matrix as the original matrix, and generated a series of down-sampled contact matrices with lower coverage levels (20%, 40%, 60%, 80%, and 100%).
                              <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                               Figure 2D
                              </a>
                              and Supplementary Table S4 show that the average false detection rates remained below 0.05 for most coverage levels, except for the lowest coverage level of 20%, demonstrating the robustness of our DiffGR method under low-coverage conditions.
                             </p>
                            </div>
                            <div id="sec-14" class="subsection">
                             <h3>
                              3.3 DiffGR successfully detected hierarchical-TAD changes
                             </h3>
                             <p id="p-43">
                              In addition to single-TAD differences, hierarchical-TAD changes also exist in some genomic regions between different cell types. In these regions, one of the Hi-C contact maps exhibits a single dominant TAD structure, while the other Hi-C contact map presents two or more subTADs separated by additional boundaries in between. Hierarchical TADs are computationally challenging to detect. Although the two Hi-C maps have different TAD boundaries, the chromatin interaction patterns within the subTADs could be very similar. Consequently, the correlation coefficients (CCs) for the strata with small genomic distances might still remain high between two contact maps. In addition, as the genomic distance increases, the weight of the corresponding stratum in the SCC calculation gradually declines. As a result, the SCC values are primarily contributed by CC values from strata with smaller genomic distances, which makes it difficult to detect differential regions in the hierarchical-TAD cases.
                             </p>
                             <p id="p-44">
                              To evaluate the performance of DiffGR in this more challenging situation, we simulated contact matrices containing hierarchical-TAD structures with respect to varying noise levels (see Methods) and then computed the false detection rate in a similar manner as in the single-TAD simulations. As demonstrated in
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Figure 3
                              </a>
                              and Supplementary Table S5, the trend of the false detection rates and other measure statistics across various noise levels under the hierarchical-TAD setting was similar to the pattern observed in the single-TAD case (
                              <a id="xref-fig-2-5" class="xref-fig" href="#F2">
                               Figure 2C
                              </a>
                              and Supplementary Table S3). Furthermore, the false detection rates remained lower than 0.05 when the noise level was within 50%. Taken together, these results indicated that DiffGR can reliably detect the differentially interacting genomic regions with hierarchical-TAD patterns.
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F3.large.jpg?width=800&height=600&carousel=1" title="Performance of hierarchical-TAD simulations. The curve shows the mean false detection rates at various noise levels. Vertical bars represent 95% confidence intervals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-794947281" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Performance of hierarchical-TAD simulations.</span> The curve shows the mean false detection rates at various noise levels. Vertical bars represent 95% confidence intervals.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F3.medium.gif" width="440" height="284"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F3.medium.gif" width="440" height="284"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3:
                               </span>
                               <span class="caption-title">
                                Performance of hierarchical-TAD simulations.
                               </span>
                               <p id="p-45" class="first-child">
                                The curve shows the mean false detection rates at various noise levels. Vertical bars represent 95% confidence intervals.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-15" class="subsection">
                             <h3>
                              3.4 DiffGR revealed cell type-specific genomic interacting regions
                             </h3>
                             <p id="p-46">
                              Besides validating our method on simulated datasets, we further applied DiffGR to detect cell type-specific differences in five human cell types (GM12878, HMEC, HUVEC, K562, and NHEK) [
                              <a id="xref-ref-9-8" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ] and in two mouse cell types (ES and cortex cells) [
                              <a id="xref-ref-11-10" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                              ]. In total, we conducted two comparisons between biological replicates in human GM12878 and mouse ES cells, and eleven pairwise comparisons between different cell types (ten pairs among five human cell types and one pair between two mouse cell types). In each pairwise comparison, we first applied HiCseg to identify TAD boundaries from the 50-kb contact matrix for each data and then partitioned the genome into three types of candidate regions: single-TAD candidate regions, hierarchical-TAD candidate regions, and complex-TAD candidate regions. Statistically significant differential genomic regions were identified between each comparison with FDR cutoff 0.05.
                             </p>
                             <p id="p-47">
                              We first sought to evaluate the performance of our method on biological replicates of Hi-C data. Previous studies have shown that the high degree of similarity between biological replicates and dominant consistence between TAD boundaries in replicate data [
                              <a id="xref-ref-9-9" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ,
                              <a id="xref-ref-11-11" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                              ,
                              <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">
                               39
                              </a>
                              ]. For the comparison between human GM12878 replicates, consistent with our expectations, the majority (89.55%) of the 2325 candidate genomic across the genome regions belonged to single-TAD type and very few (2.45%) candidate genomic regions were detected as differential by our method (Supplementary Figure S2). Specifically, only 1.97% of single-TADs were identified as differential, whereas 6.17% and 4.94% were detected in hierarchical-TAD and complex-TAD cases respectively. Similar results were also witnessed in the comparison between replicates in mouse ES cells: 83.42% candidate genomic regions were classified as single-TAD type and few (6.02%) were identified as differential (Supplementary Table S6). Overall, our DiffGR results confirmed that these biological replicates displayed highly consistent chromatin structures with minor biological variations.
                             </p>
                             <p id="p-48">
                              Next, we applied DiffGR to detect cell-type-specific differences and the results are illustrated in
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Figure 4
                              </a>
                              and Supplementary Table S7. For the ten pairwise comparisons among human cell types, 55.57% of all candidate genomic regions belonged to the single-TAD category (consistent with previous observations indicating that TAD boundaries are stable across cell types [
                              <a id="xref-ref-11-12" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                              ]), 31.88% to the hierarchical-TAD category, and 12.55% to the complex-TAD category. Our DiffGR analyses showed that only 24.26% of the single-TAD candidate regions showed statistically significant differences between two samples; 59.24% of the hierarchical-TAD candidate regions were determined to be differential; while the differential proportion of the complex-TAD category was as high as 89.82%. In addition, we found that the proportion of detected differential regions varied largely across chromosomes, ranging from 0.14 to 0.76 (Supplementary Figure S3). For the comparison between mouse ES and cortex cells, 20.22% of the candidate genomic regions in the single-TAD category were identified as differential, while the proportion increased to 75.94% in the complex-TAD category. These observations indicated that candidate genomic regions with more distinct patterns of TAD boundaries are more likely to be detected as differential between two Hi-C samples.
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F4.large.jpg?width=800&height=600&carousel=1" title="Piecharts of DiffGR results obtained from human Hi-C datasets. The center piechart presents the proportions of three categories of candidate regions. The three outer piecharts display the proportions of DiffGR-detected differential genomic regions, one for each candidate category." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-794947281" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Piecharts of DiffGR results obtained from human Hi-C datasets.</span> The center piechart presents the proportions of three categories of candidate regions. The three outer piecharts display the proportions of DiffGR-detected differential genomic regions, one for each candidate category.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F4.medium.gif" width="440" height="436"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F4.medium.gif" width="440" height="436"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4:
                               </span>
                               <span class="caption-title">
                                Piecharts of DiffGR results obtained from human Hi-C datasets.
                               </span>
                               <p id="p-49" class="first-child">
                                The center piechart presents the proportions of three categories of candidate regions. The three outer piecharts display the proportions of DiffGR-detected differential genomic regions, one for each candidate category.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-50">
                              In addition to partitioning the genome at 50-kb resolution, we also performed differential analyses on the five human Hi-C datasets at 25-kb and 100-kb resolutions, separately. We calculated the overlapping rate (that is, the proportion of the genome that was classified into the same differential or non-differential status) between different resolutions. Overall, we observed a high consistency between the detected differential regions across different resolutions, where the overlapping rate was 0.9856 between the detection results at 50-kb and 100-kb resolutions, and 0.9480 between those at 25-kb and 50-kb resolutions. These results demonstrated that DiffGR can robustly and consistently detect cell type-specific differential genomic regions across various resolutions.
                             </p>
                            </div>
                            <div id="sec-16" class="subsection">
                             <h3>
                              3.5 Changes in CTCF and histone modification patterns were consistent with DiffGR detection results
                             </h3>
                             <p id="p-51">
                              As there is no ground truth of differential chromatin interacting regions in real data, we sought to evaluate the performance of our method by investigating the association between the changes in 1D epigenomic features and 3D genomic interaction regions. The chromatin architectural protein CTCF plays an essential role in establishing higher-order chromatin structures such as TADs. In addition, it has been shown that transcription factors and histone marks are enriched or depleted at TAD boundaries. Therefore, we hypothesized that differential bindings of transcription factors such as CTCF and histone modifications would also be present at the TAD boundaries in differential genomic interacting regions.
                             </p>
                             <p id="p-52">
                              To test this hypothesis, we first combined TAD boundaries from both Hi-C datasets and classified them into two categories: those within the DiffGR-detected differential regions and those outside the differential regions. We then utilized the ChIP-seq datasets of transcription factors like CTCF and histone modifications from the ENCODE project [
                              <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">
                               40
                              </a>
                              ]. For each ChIP-seq dataset, we calculated the mean absolute difference of ChIP-seq peaks between the two cell types within the neighborhood (+/- 1 bin) of each TAD boundary. We expected that if two cell lines have highly different chromatin structures in certain genomic regions, different patterns of CTCF bindings and histone modifications in these regions would be observed. Therefore, we performed the following t-test for each ChIP-seq dataset using the DiffGR detection results. In each chromosome, we evaluated whether the mean absolute differences of the ChIP-seq signal at the TAD boundaries in differential regions were significantly different from those in non-differential regions. If the ChIP-seq signal differences at the TAD boundaries in differential regions were significantly higher (with a significant level 0.1) than those in non-differential regions, we considered the ChIP-seq changes to be consistent with our DiffGR differential detection results.
                             </p>
                             <p id="p-53">
                              <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              and Supplementary Table S8 summarize the ChIP-seq analyses on the DiffGR detection results obtained from five human Hi-C datasets [
                              <a id="xref-ref-9-10" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ] and two mouse Hi-C datasets [
                              <a id="xref-ref-11-13" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                              ]. For each human ChIP-seq dataset, we performed 220 t-tests (ten pairwise comparisons between cell types, 22 chromosome-wide tests one for each autosome) at 100-kb, 50-kb, and 25-kb resolutions; for each mouse ChIP-seq dataset, we conducted 19 t-tests one for each autosome at 50-kb resolution. Overall, DiffGR-detected differential genomic regions were supported by 1D epigenomic features in both human and mouse data. Furthermore, we observed that the agreement between the changes in ChIP-seq signal and chromatin structures was improved in finer-resolution analyses. As shown in
                              <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              , 76 out of 220 (34.55%) tests showed significantly higher absolute mean differences of CTCF values at the TAD boundaries in DiffGR-detected differential genomic regions than those in non-differential regions at 100-kb resolution. Whereas in the results at 25-kb resolution, 142 (64.55%) tests exhibited significantly larger changes in CTCF bindings in differential regions than non-differential ones. In addition, the histone modification datasets showed similar results in agreement with the detection results of differentially interacting regions in Hi-C contact maps. At 25-kb resolution, the majority of the t-tests showed significantly larger changes of ChIP-seq signal in differentially interacting regions for all four histone modification datasets, including H3K4me1, H3K4me2, H3K27me3, and H3K26me3. Collectively, these results indicated that the changes in CTCF bindings and histone modifications were in good agreements with the differences in genomic interacting regions. Furthermore, at finer resolution our DiffGR method produced more accurate identification of differentially interacting genomic regions in higher agreement with the CTCF and histone modification data.
                             </p>
                             <div id="T1" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2686945/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2686945/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2686945" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 1:
                               </span>
                               <span class="caption-title">
                                Agreements between ChIP-seq data and DiffGR-detected differential genomic regions in human Hi-C datasets.
                               </span>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-54">
                              We would like to point out that for those cases where the changes in CTCF or histone modifications are not in significant agreement with the detection results of differentially interacting genomic regions, it does not necessarily suggest that these epigenomic features are inconsistent with 3D genome organization nor DiffGR detection results are inaccurate. Due to the resolution limit of Hi-C contact maps, the boundaries of differential regions are usually identified with a resolution of tens to hundreds of kilobases. Aggregating ChIP-seq data with such a large bin size dilutes the signal, thereby yielding less statistical power to detect significant changes. Moreover, CTCF and histone modifications play fundamental roles in regulating chromatin structures and gene expression; their functions are not limited to TAD formations. Therefore, changes in CTCF bindings or histone modifications exist in many genomic loci other than TAD boundaries, thus may not be represented in our analyses.
                             </p>
                            </div>
                            <div id="sec-17" class="subsection">
                             <h3>
                              3.6 Differential RNA-seq analysis results were consistent with DiffGR detection
                             </h3>
                             <p id="p-55">
                              In addition to investigating the changes in 1D epigenomic features, we further studied the relationship between quantitative changes in gene expression levels and 3D genomic interaction regions. Previous studies have showed that topological changes of 3D genome organization have a large effect on the cross-talk between enhancers and promoters therefore can alter gene expression [
                              <a id="xref-ref-9-11" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ,
                              <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              ]. Thus, we expected to observe an enrichment of differential expressed genes in DiffGR-detected differential genomic regions.
                             </p>
                             <p id="p-56">
                              To evaluate this assumption, we first detected significant changes in gene expression levels between human GM12878 and K562 cells using DESeq2 [
                              <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ] and those between mouse ES and cortex cells using ballgown [
                              <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                               42
                              </a>
                              ]. Then we calculated the percentage of differentially expressed genes that were located inside the DiffGR-identified differential genomic regions. To calculate the enrichment of differentially expressed genes, we randomly chose a set of genes, whose number is equivalent to the number of the DESeq2-detected differentially expressed genes, with 200 times, computed their corresponding proportions located in differential genomic regions, and then performed t-test for comparison. In summary, a total number of 8781 differentially expressed genes were detected between human GM12878 and K562 cells and 79.54% of them were located in DiffGR-detected differential genomic regions (p-value = 3.72 × 10
                              <sup>
                               −5
                              </sup>
                              , permutation test); whereas 2124 differentially expressed genes were identified between mouse ES and cortex cells and 61.66% were within DiffGR-detected differential genomic regions (p-value < 2.2 × 10
                              <sup>
                               −16
                              </sup>
                              ). Taken together, these results demonstrated that the changes of gene expression in RNA-seq data were highly consistent with the DiffGR detection results.
                             </p>
                             <p id="p-57">
                              To further explore the potential functional roles of the differentially expressed genes located in differential genomic regions, we performed Gene Ontology (GO) enrichment analysis on the top 2000 genes using DAVID [
                              <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                               43
                              </a>
                              ]. As show in
                              <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              , we observed a high enrichment of GO terms related to the immune responses, which is consistent with the immunological nature of GM12878 lymphoblastoid B-cells.
                             </p>
                             <div id="T2" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2686950/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2686950/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2686950" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 2:
                               </span>
                               <span class="caption-title">
                                Functional enrichment of differentially expressed genes located in differential genomic regions between GM12878 and K562.
                               </span>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-18" class="subsection">
                             <h3>
                              3.7 DiffGR detection was supported by differential chromatin interactions
                             </h3>
                             <p id="p-58">
                              Several Hi-C comparative studies have demonstrated that the majority of the chromatin structural changes tend to couple with the formation/disappearance of topologically associated domains (TADs) [
                              <a id="xref-ref-9-12" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ,
                              <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              ], implying that changes in Hi-C interaction counts are likely to be observed within genomic regions at TAD level. Hence, we checked differential chromatin interactions (DCIs) between GM12878 and K562 cells at 50-kb resolution by FIND [
                              <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">
                               26
                              </a>
                              ] and compared FIND results with our DiffGR results. As shown in
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Figure 5
                              </a>
                              , the percentages of DCIs detected by FIND located within candidate genomic regions were dominant in the majority of chromosomes and with 55.43% across the whole genome. In addition, 82.80% of the DCIs located in candidate genomic regions are classified into differential regions, demonstrating that DiffGR effectively detected the regions with significant changes in chromatin contacts.
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F5.large.jpg?width=800&height=600&carousel=1" title="Comparison between FIND and DiffGR. Barchart of the proportions of FIND-detected DCIs located in candidate genomic regions (GRs) and differential GRs for all autosomes between GM12878 and K562. The light gray bars denote the proportions of DCIs located in candidate GRs; the dark gray bars represent the proportions of DCIs located in differential GRs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-794947281" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Comparison between FIND and DiffGR.</span> Barchart of the proportions of FIND-detected DCIs located in candidate genomic regions (GRs) and differential GRs for all autosomes between GM12878 and K562. The light gray bars denote the proportions of DCIs located in candidate GRs; the dark gray bars represent the proportions of DCIs located in differential GRs.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F5.medium.gif" width="440" height="349"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F5.medium.gif" width="440" height="349"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/08/2020.08.29.273698/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5:
                               </span>
                               <span class="caption-title">
                                Comparison between FIND and DiffGR.
                               </span>
                               <p id="p-59" class="first-child">
                                Barchart of the proportions of FIND-detected DCIs located in candidate genomic regions (GRs) and differential GRs for all autosomes between GM12878 and K562. The light gray bars denote the proportions of DCIs located in candidate GRs; the dark gray bars represent the proportions of DCIs located in differential GRs.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-19" class="subsection">
                             <h3>
                              3.8 Performance and comparison with state-of-the-art differential TAD detection tools
                             </h3>
                             <p id="p-60">
                              Next, we compared the DiffGR results with three differential TAD boundaries detection methods (HiCDB [
                              <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ], TADCompare [
                              <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                              ], and TADreg [
                              <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ]) and one differential TAD regions detection tool (provided by HiCExplorer [
                              <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              –
                              <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">
                               33
                              </a>
                              ]) on the five human Hi-C datasets by Rao et al. [
                              <a id="xref-ref-9-13" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ] and the two mouse datasets by Dixon et al. [
                              <a id="xref-ref-11-14" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                              ] at 50-kb resolution. Overall, the differential TAD boundaries/regions identified by HiCDB, TADCompare, TADreg, or HiCExplorer were highly concordant with DiffGR-detected differentially interacting genomic regions. Notably, 73.86% of the HiCDB-detected, 76.25% of the TADCompare-detected, and 71.90% of the TADreg-detected differential TAD boundaries displayed consistent results with our DiffGR detection in the human datasets. In addition, highly concordant rates were also witnessed in the mouse dataset with 59.56%, 62.01%, and 60.32% consistency rate with HiCDB, TADCompare, and TADreg, respectively. Furthermore, 60.62% of the 2877 HiCExplorer-identified differential regions from the five human cell lines overlapped with DiffGR-detected differential regions.
                             </p>
                             <p id="p-61">
                              To investigate the advantages of DiffGR over TADCompare, we further performed tests on changes in CTCF and histone modification patterns for the TADCompare-detected differential TAD boundaries within DiffGR-detected differential and non-differential genomic regions in human datasets. From
                              <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">
                               Table 3
                              </a>
                              , we observed that 155 out of 220 (70.45%) contrasts showed higher absolute mean differences of CTCF values at TADCompare-detected differential TAD boundaries in DiffGR-detected differential genomic regions than those in non-differential regions. In addition 98 (44.55%) CTCF tests exhibited significantly larger changes of CTCF bindings with a significant level of 0.1 at differential TAD boundaries in differential regions than those in non-differential regions. Furthermore, the histone modification datasets (including H3K4me1, H3K4me2, H3K27me3, and H3K36me3) showed similar results that were in agreement with the advantageous results of differential TAD boundaries in differentially interacting regions. Collectively, these results indicated that DiffGR-detected differential genomic regions had a better agreement with 1D epigenomic features than TADCompare-detected differential TAD bounds.
                             </p>
                             <div id="T3" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2686948/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2686948/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2686948" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 3:
                               </span>
                               <span class="caption-title">
                                Differential TAD boundaries detected by TADcompare in DiffGR-detected differential genomic regions showed higher agreement with ChIP-seq signals than those in non-differential regions in human Hi-C datasets.
                               </span>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-20">
                            <h2 class="">
                             4 Discussion and Conclusions
                            </h2>
                            <p id="p-62">
                             With the fast accumulation of Hi-C datasets, there has been a dramatically increasing interest in comparative analysis of Hi-C contact maps. However, most existing methods for comparative Hi-C analysis focused on the identification of differential chromatin interactions, while few studies addressed the detection of differential chromatin organization at TAD scale. To tackle this problem, we developed a novel method, DiffGR, for calling differentially interacting genomic regions between two Hi-C contact maps. Taking genomic distance features of Hi-C data into consideration, our algorithm utilized the SCC metric instead of the standard Pearson CC to measure the similarity of local genomic regions between Hi-C contact maps. Furthermore, we proposed a nonparametric permutation test to assess the statistical significance of the local SCC values. In contrast to the parametric approaches that were used by most Hi-C data analysis methods, our nonparametric approach does not have a set of predefined assumptions about the nature of the null distribution and, therefore, is more robust and can be applied to more diverse data from real cases. Additionally, we utilized a non-parametric smoothing spline regression to speed up the permutation test and showed that the speed-up algorithm can steadily produce consistent outputs. Through empirical evaluations, we have demonstrated that DiffGR can effectively discover differential regions in both simulated data and real Hi-C data from different cell types. That is, DiffGR produced robust and stable detection results under various noise and coverage levels in simulated data; DiffGR detection results in real data were effectively validated by the ChIP-seq and RNA-seq data; DiffGR produced consistent and advantageous results compared with state-of-the-art differential TAD boundaries/regions detection tools. To summarize, DiffGR provides a statistically rigorous method for the detection of differentially interacting genomic regions in Hi-C contact maps from different cells and conditions, therefore would facilitate the investigation of their biological functions.
                            </p>
                            <p id="p-63">
                             We envision a few possible extensions and future directions based on this work. First, our method performs pairwise comparison between Hi-C contact maps. One potential future direction is to design a more general statistical framework for differential analyses among three or more samples. Then we could further assign the differentially interacting genomic regions to cell type-specific or condition-specific changing areas. Second, we currently pool biological replicates together in our analyses. Extending DiffGR to incorporate multiple biological replicates to detect reproducible differences would enhance the reliability of the detection results. Third, in our algorithm, we use the shared TAD boundaries between two samples to segment the genome into candidate genomic regions and then detect differential regions. Recently, the notion of TADs being highly conserved across cell types has been questioned [
                             <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                              44
                             </a>
                             ,
                             <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                              45
                             </a>
                             ]. Therefore, a more general approach to define and classify the candidate genomic regions would be beneficial to better characterize the variability of chromatin interactions between different conditions. Lastly, our method is specifically designed for bulk Hi-C data. Given the high sparsity and variability of single-cell Hi-C contact matrices, identifying differential genomic regions at single-cell level remains a significant challenge.
                            </p>
                           </div>
                           <div class="section" id="sec-21">
                            <h2 class="">
                             5 Code Availability
                            </h2>
                            <p id="p-64">
                             The DiffGR R Code (both algorithm and simulation) is publicly available at
                             <a href="https://github.com/wmalab/DiffGR">
                              https://github.com/wmalab/DiffGR
                             </a>
                             under the GNU GPL ≥ 2 license. The source code is also available at BioCode
                             <a href="https://ngdc.cncb.ac.cn/biocode/tools/BT007313">
                              https://ngdc.cncb.ac.cn/biocode/tools/BT007313
                             </a>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-22">
                            <h2 class="">
                             6 CRediT author statement
                            </h2>
                            <p id="p-65">
                             <strong>
                              Huiling Liu
                             </strong>
                             : Conceptualization, Methodology, Software, Formal analysis, Writing - Original Draft.
                             <strong>
                              Wenxiu Ma
                             </strong>
                             : Conceptualization, Supervision, Writing - Review & Editing, Funding acquisition. All authors read and approved the final manuscript.
                            </p>
                           </div>
                           <div class="section" id="sec-23">
                            <h2 class="">
                             7 Competing interests
                            </h2>
                            <p id="p-66">
                             The authors have declared that no competing interests exist.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             8 Acknowledgments
                            </h2>
                            <p id="p-67">
                             The authors would like to thank Tiantian Ye, Yangyang Hu, and Luke Klein for helpful discussions and feedback, and the editor and reviewers for their insightful comments and suggestions. This work was supported by the National Science Foundation [DBI-1751317]; and the National Institute of Health [R35GM133678].
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-4">
                               Results Section
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               [1].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.1" data-doi="10.1038/ng1896">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Marieke
                                  </span>
                                  <span class="cit-name-surname">
                                   Simonis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Petra
                                  </span>
                                  <span class="cit-name-surname">
                                   Klous
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Erik
                                  </span>
                                  <span class="cit-name-surname">
                                   Splinter
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yuri
                                  </span>
                                  <span class="cit-name-surname">
                                   Moshkin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Rob
                                  </span>
                                  <span class="cit-name-surname">
                                   Willemsen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Elzo
                                  </span>
                                  <span class="cit-name-surname">
                                   De Wit
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Bas
                                  </span>
                                  <span class="cit-name-surname">
                                   Van Steensel
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Wouter
                                  </span>
                                  <span class="cit-name-surname">
                                   De Laat
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Nuclear organization of active and inactive chromatin domains uncovered by chromosome conformation capture–on-chip (4c)
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  38
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1348
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1354
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSimonis%26rft.auinit1%253DM.%26rft.volume%253D38%26rft.issue%253D11%26rft.spage%253D1348%26rft.epage%253D1354%26rft.atitle%253DNuclear%2Borganization%2Bof%2Bactive%2Band%2Binactive%2Bchromatin%2Bdomains%2Buncovered%2Bby%2Bchromosome%2Bconformation%2Bcapture-on-chip%2B%25284C%2529.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1896%26rft_id%253Dinfo%253Apmid%252F17033623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ng1896&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17033623&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F08%2F2020.08.29.273698.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000241592700026&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [2].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.2" data-doi="10.1101/gr.5571506">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Josée
                                  </span>
                                  <span class="cit-name-surname">
                                   Dostie
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Todd A
                                  </span>
                                  <span class="cit-name-surname">
                                   Richmond
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ramy A
                                  </span>
                                  <span class="cit-name-surname">
                                   Arnaout
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Rebecca R
                                  </span>
                                  <span class="cit-name-surname">
                                   Selzer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   William L
                                  </span>
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tracey A
                                  </span>
                                  <span class="cit-name-surname">
                                   Honan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Eric D
                                  </span>
                                  <span class="cit-name-surname">
                                   Rubio
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Anton
                                  </span>
                                  <span class="cit-name-surname">
                                   Krumm
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Justin
                                  </span>
                                  <span class="cit-name-surname">
                                   Lamb
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Chad
                                  </span>
                                  <span class="cit-name-surname">
                                   Nusbaum
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Chromosome conformation capture carbon copy (5c): a massively parallel solution for mapping interactions between genomic elements
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1299
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1309
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5571506%26rft_id%253Dinfo%253Apmid%252F16954542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNi8xMC8xMjk5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDgvMDgvMjAyMC4wOC4yOS4yNzM2OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [3].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.3" data-doi="10.1126/science.1181369">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Erez
                                  </span>
                                  <span class="cit-name-surname">
                                   Lieberman-Aiden
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nynke L
                                  </span>
                                  <span class="cit-name-surname">
                                   Van Berkum
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Louise
                                  </span>
                                  <span class="cit-name-surname">
                                   Williams
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Maxim
                                  </span>
                                  <span class="cit-name-surname">
                                   Imakaev
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tobias
                                  </span>
                                  <span class="cit-name-surname">
                                   Ragoczy
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Agnes
                                  </span>
                                  <span class="cit-name-surname">
                                   Telling
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ido
                                  </span>
                                  <span class="cit-name-surname">
                                   Amit
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Bryan R
                                  </span>
                                  <span class="cit-name-surname">
                                   Lajoie
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Peter J
                                  </span>
                                  <span class="cit-name-surname">
                                   Sabo
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael O
                                  </span>
                                  <span class="cit-name-surname">
                                   Dorschner
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Comprehensive mapping of long-range interactions reveals folding principles of the human genome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  science
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  326
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5950
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  289
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  293
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLieberman-Aiden%26rft.auinit1%253DE.%26rft.volume%253D326%26rft.issue%253D5950%26rft.spage%253D289%26rft.epage%253D293%26rft.atitle%253DComprehensive%2BMapping%2Bof%2BLong-Range%2BInteractions%2BReveals%2BFolding%2BPrinciples%2Bof%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1181369%26rft_id%253Dinfo%253Apmid%252F19815776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjYvNTk1MC8yODkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wOC8wOC8yMDIwLjA4LjI5LjI3MzY5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [4].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.29.273698v2.4" data-doi="10.1038/nature08973">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                  <span class="cit-name-surname">
                                   Duan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Andronescu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Schutz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   McIlwain
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Shendure
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Fields
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Blau
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Noble
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A three-dimensional model of the yeast genome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  465
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7296
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  363
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  367
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDuan%26rft.auinit1%253DZ.%26rft.volume%253D465%26rft.issue%253D7296%26rft.spage%253D363%26rft.epage%253D367%26rft.atitle%253DA%2Bthree-dimensional%2Bmodel%2Bof%2Bthe%2Byeast%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08973%26rft_id%253Dinfo%253Apmid%252F20436457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature08973&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20436457&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F08%2F2020.08.29.273698.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000277829200044&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [5].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.5" data-doi="10.1038/nbt.2057">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kalhor
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tjong
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jayathilaka
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Alber
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Genome architectures revealed by tethered chromosome conformation capture and population-based modeling
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature biotechnology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  90
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  98
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DKalhor%26rft.auinit1%253DR.%26rft.volume%253D30%26rft.issue%253D1%26rft.spage%253D90%26rft.epage%253D98%26rft.atitle%253DGenome%2Barchitectures%2Brevealed%2Bby%2Btethered%2Bchromosome%2Bconformation%2Bcapture%2Band%2Bpopulation-based%2Bmodeling.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2057%26rft_id%253Dinfo%253Apmid%252F22198700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nbt.2057&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22198700&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F08%2F2020.08.29.273698.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [6].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.6" data-doi="10.1186/gb-2010-11-2-r22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Guoliang
                                  </span>
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Melissa J
                                  </span>
                                  <span class="cit-name-surname">
                                   Fullwood
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Han
                                  </span>
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Fabianus Hendriyan
                                  </span>
                                  <span class="cit-name-surname">
                                   Mulawadi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Stoyan
                                  </span>
                                  <span class="cit-name-surname">
                                   Velkov
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Vinsensius
                                  </span>
                                  <span class="cit-name-surname">
                                   Vega
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Pramila Nuwantha
                                  </span>
                                  <span class="cit-name-surname">
                                   Ariyaratne
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yusoff
                                  </span>
                                  <span class="cit-name-surname">
                                   Bin Mohamed
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Hong-Sain
                                  </span>
                                  <span class="cit-name-surname">
                                   Ooi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Chandana
                                  </span>
                                  <span class="cit-name-surname">
                                   Tennakoon
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Chia-pet tool for comprehensive chromatin interaction analysis with paired-end tag sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  R22
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLi%26rft.auinit1%253DG.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253DR22%26rft.epage%253DR22%26rft.atitle%253DChIA-PET%2Btool%2Bfor%2Bcomprehensive%2Bchromatin%2Binteraction%2Banalysis%2Bwith%2Bpaired-end%2Btag%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-2-r22%26rft_id%253Dinfo%253Apmid%252F20181287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-2-r22&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20181287&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F08%2F2020.08.29.273698.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [7].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Maxwell R
                                  </span>
                                  <span class="cit-name-surname">
                                   Mumbach
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Adam J
                                  </span>
                                  <span class="cit-name-surname">
                                   Rubin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ryan A
                                  </span>
                                  <span class="cit-name-surname">
                                   Flynn
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Chao
                                  </span>
                                  <span class="cit-name-surname">
                                   Dai
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Paul A
                                  </span>
                                  <span class="cit-name-surname">
                                   Khavari
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   William J
                                  </span>
                                  <span class="cit-name-surname">
                                   Greenleaf
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Howard Y
                                  </span>
                                  <span class="cit-name-surname">
                                   Chang
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Hichip: efficient and sensitive analysis of protein-directed genome architecture
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  919
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  922
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D13%26rft.spage%253D919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [8].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Wenxiu
                                  </span>
                                  <span class="cit-name-surname">
                                   Ma
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ferhat
                                  </span>
                                  <span class="cit-name-surname">
                                   Ay
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Choli
                                  </span>
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Gunhan
                                  </span>
                                  <span class="cit-name-surname">
                                   Gulsoy
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Xinxian
                                  </span>
                                  <span class="cit-name-surname">
                                   Deng
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Savannah
                                  </span>
                                  <span class="cit-name-surname">
                                   Cook
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jennifer
                                  </span>
                                  <span class="cit-name-surname">
                                   Hesson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christopher
                                  </span>
                                  <span class="cit-name-surname">
                                   Cavanaugh
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Carol B
                                  </span>
                                  <span class="cit-name-surname">
                                   Ware
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Anton
                                  </span>
                                  <span class="cit-name-surname">
                                   Krumm
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Fine-scale chromatin interaction maps reveal the cis-regulatory landscape of human lincrna genes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  71
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  78
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D12%26rft.spage%253D71%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [9].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.9" data-doi="10.1016/j.cell.2014.11.021">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Suhas SP
                                  </span>
                                  <span class="cit-name-surname">
                                   Rao
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Miriam H
                                  </span>
                                  <span class="cit-name-surname">
                                   Huntley
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Neva C
                                  </span>
                                  <span class="cit-name-surname">
                                   Durand
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Elena K
                                  </span>
                                  <span class="cit-name-surname">
                                   Stamenova
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ivan D
                                  </span>
                                  <span class="cit-name-surname">
                                   Bochkov
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   James T
                                  </span>
                                  <span class="cit-name-surname">
                                   Robinson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Adrian L
                                  </span>
                                  <span class="cit-name-surname">
                                   Sanborn
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ido
                                  </span>
                                  <span class="cit-name-surname">
                                   Machol
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Arina D
                                  </span>
                                  <span class="cit-name-surname">
                                   Omer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Eric S
                                  </span>
                                  <span class="cit-name-surname">
                                   Lander
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  A 3d map of the human genome at kilobase resolution reveals principles of chromatin looping
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  159
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1665
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1680
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D159%26rft.spage%253D1665%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.11.021%26rft_id%253Dinfo%253Apmid%252F25497547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.11.021&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25497547&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F08%2F2020.08.29.273698.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000347922500019&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [10].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ramani
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cusanovich
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hause
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ma
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Qiu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                  <span class="cit-name-surname">
                                   Deng
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Blau
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Disteche
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Noble
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Shendure
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                  <span class="cit-name-surname">
                                   Duan
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Mapping 3D genome architecture through in situ DNase Hi-C
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature protocols
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2104
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2121
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bprotocols%26rft.volume%253D11%26rft.spage%253D2104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [11].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.11" data-doi="10.1038/nature11082">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jesse R
                                  </span>
                                  <span class="cit-name-surname">
                                   Dixon
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Siddarth
                                  </span>
                                  <span class="cit-name-surname">
                                   Selvaraj
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Feng
                                  </span>
                                  <span class="cit-name-surname">
                                   Yue
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Audrey
                                  </span>
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yan
                                  </span>
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yin
                                  </span>
                                  <span class="cit-name-surname">
                                   Shen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ming
                                  </span>
                                  <span class="cit-name-surname">
                                   Hu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jun S
                                  </span>
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Bing
                                  </span>
                                  <span class="cit-name-surname">
                                   Ren
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Topological domains in mammalian genomes identified by analysis of chromatin interactions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  485
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7398
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  376
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  380
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDixon%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D485%26rft.issue%253D7398%26rft.spage%253D376%26rft.epage%253D380%26rft.atitle%253DTopological%2Bdomains%2Bin%2Bmammalian%2Bgenomes%2Bidentified%2Bby%2Banalysis%2Bof%2Bchromatin%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11082%26rft_id%253Dinfo%253Apmid%252F22495300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature11082&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22495300&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F08%2F2020.08.29.273698.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000304099100044&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [12].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Geoffrey
                                  </span>
                                  <span class="cit-name-surname">
                                   Fudenberg
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Maxim
                                  </span>
                                  <span class="cit-name-surname">
                                   Imakaev
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Carolyn
                                  </span>
                                  <span class="cit-name-surname">
                                   Lu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Anton
                                  </span>
                                  <span class="cit-name-surname">
                                   Goloborodko
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nezar
                                  </span>
                                  <span class="cit-name-surname">
                                   Abdennur
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Leonid A
                                  </span>
                                  <span class="cit-name-surname">
                                   Mirny
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Formation of chromosomal domains by loop extrusion
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell reports
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2038
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2049
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%2Breports%26rft.volume%253D15%26rft.spage%253D2038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [13].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.13" data-doi="10.1093/nar/gky789">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Fengling
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Guipeng
                                  </span>
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael Q
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yang
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Hicdb: a sensitive and robust method for detecting contact domain boundaries
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic acids research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  46
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  21
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  11239
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  11250
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft.volume%253D46%26rft.spage%253D11239%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gky789&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [14].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.14" data-doi="10.1038/nature14450">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Emily
                                  </span>
                                  <span class="cit-name-surname">
                                   Crane
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Qian
                                  </span>
                                  <span class="cit-name-surname">
                                   Bian
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Rachel Patton
                                  </span>
                                  <span class="cit-name-surname">
                                   McCord
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Bryan R
                                  </span>
                                  <span class="cit-name-surname">
                                   Lajoie
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Bayly S
                                  </span>
                                  <span class="cit-name-surname">
                                   Wheeler
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Edward J
                                  </span>
                                  <span class="cit-name-surname">
                                   Ralston
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Satoru
                                  </span>
                                  <span class="cit-name-surname">
                                   Uzawa
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Job
                                  </span>
                                  <span class="cit-name-surname">
                                   Dekker
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Barbara J
                                  </span>
                                  <span class="cit-name-surname">
                                   Meyer
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Condensin-driven remodelling of x chromosome topology during dosage compensation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  523
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7559
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  240
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D523%26rft.spage%253D240%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14450%26rft_id%253Dinfo%253Apmid%252F26030525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature14450&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26030525&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F08%2F2020.08.29.273698.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [15].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.15">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Darya
                                  </span>
                                  <span class="cit-name-surname">
                                   Filippova
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Rob
                                  </span>
                                  <span class="cit-name-surname">
                                   Patro
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Geet
                                  </span>
                                  <span class="cit-name-surname">
                                   Duggal
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Carl
                                  </span>
                                  <span class="cit-name-surname">
                                   Kingsford
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Identification of alternative topological domains in chromatin
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Algorithms for Molecular Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAlgorithms%2Bfor%2BMolecular%2BBiology%26rft.volume%253D9%26rft.spage%253D14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [16].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.16" data-doi="10.1093/bioinformatics/btu443">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Celine
                                  </span>
                                  <span class="cit-name-surname">
                                   Lévy-Leduc
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Maud
                                  </span>
                                  <span class="cit-name-surname">
                                   Delattre
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tristan
                                  </span>
                                  <span class="cit-name-surname">
                                   Mary-Huard
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Stephane
                                  </span>
                                  <span class="cit-name-surname">
                                   Robin
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Two-dimensional segmentation for analyzing hi-c data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  17
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  i386
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  i392
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu443%26rft_id%253Dinfo%253Apmid%252F25161224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu443&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25161224&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F08%2F2020.08.29.273698.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [17].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.17">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mattia
                                  </span>
                                  <span class="cit-name-surname">
                                   Forcato
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Chiara
                                  </span>
                                  <span class="cit-name-surname">
                                   Nicoletti
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Koustav
                                  </span>
                                  <span class="cit-name-surname">
                                   Pal
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Carmen Maria
                                  </span>
                                  <span class="cit-name-surname">
                                   Livi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Francesco
                                  </span>
                                  <span class="cit-name-surname">
                                   Ferrari
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Silvio
                                  </span>
                                  <span class="cit-name-surname">
                                   Bicciato
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Comparison of computational methods for hi-c data analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  679
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D14%26rft.spage%253D679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [18].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.18">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Zhijun
                                  </span>
                                  <span class="cit-name-surname">
                                   Han
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Gang
                                  </span>
                                  <span class="cit-name-surname">
                                   Wei
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Computational tools for hi-c data analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Quantitative Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  215
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  225
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DQuantitative%2BBiology%26rft.volume%253D5%26rft.spage%253D215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [19].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.19" data-doi="10.1186/1471-2164-14-70">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Junbai
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Xun
                                  </span>
                                  <span class="cit-name-surname">
                                   Lan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Pei-Yin
                                  </span>
                                  <span class="cit-name-surname">
                                   Hsu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Hang-Kai
                                  </span>
                                  <span class="cit-name-surname">
                                   Hsu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kun
                                  </span>
                                  <span class="cit-name-surname">
                                   Huang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jeffrey
                                  </span>
                                  <span class="cit-name-surname">
                                   Parvin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tim HM
                                  </span>
                                  <span class="cit-name-surname">
                                   Huang
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Victor X
                                  </span>
                                  <span class="cit-name-surname">
                                   Jin
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Genome-wide analysis uncovers high frequency, strong differential chromosomal interactions and their associated epigenetic patterns in e2-mediated gene regulation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC genomics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  70
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%26rft.volume%253D14%26rft.spage%253D70%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-70%26rft_id%253Dinfo%253Apmid%252F23368971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-70&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23368971&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F08%2F2020.08.29.273698.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [20].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.20" data-doi="10.1038/nature14222">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jesse R
                                  </span>
                                  <span class="cit-name-surname">
                                   Dixon
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Inkyung
                                  </span>
                                  <span class="cit-name-surname">
                                   Jung
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Siddarth
                                  </span>
                                  <span class="cit-name-surname">
                                   Selvaraj
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yin
                                  </span>
                                  <span class="cit-name-surname">
                                   Shen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jessica E
                                  </span>
                                  <span class="cit-name-surname">
                                   Antosiewicz-Bourget
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ah Young
                                  </span>
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Zhen
                                  </span>
                                  <span class="cit-name-surname">
                                   Ye
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Audrey
                                  </span>
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nisha
                                  </span>
                                  <span class="cit-name-surname">
                                   Rajagopal
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Wei
                                  </span>
                                  <span class="cit-name-surname">
                                   Xie
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Chromatin architecture reorganization during stem cell differentiation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  518
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7539
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  331
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D518%26rft.spage%253D331%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14222%26rft_id%253Dinfo%253Apmid%252F25693564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature14222&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25693564&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F08%2F2020.08.29.273698.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [21].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.21" data-doi="10.1016/j.molcel.2010.05.004">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sven
                                  </span>
                                  <span class="cit-name-surname">
                                   Heinz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christopher
                                  </span>
                                  <span class="cit-name-surname">
                                   Benner
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nathanael
                                  </span>
                                  <span class="cit-name-surname">
                                   Spann
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Eric
                                  </span>
                                  <span class="cit-name-surname">
                                   Bertolino
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yin C
                                  </span>
                                  <span class="cit-name-surname">
                                   Lin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Peter
                                  </span>
                                  <span class="cit-name-surname">
                                   Laslo
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jason X
                                  </span>
                                  <span class="cit-name-surname">
                                   Cheng
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Cornelis
                                  </span>
                                  <span class="cit-name-surname">
                                   Murre
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Harinder
                                  </span>
                                  <span class="cit-name-surname">
                                   Singh
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christopher K
                                  </span>
                                  <span class="cit-name-surname">
                                   Glass
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and b cell identities
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular cell
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  38
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  576
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  589
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DHeinz%26rft.auinit1%253DS.%26rft.volume%253D38%26rft.issue%253D4%26rft.spage%253D576%26rft.epage%253D589%26rft.atitle%253DSimple%2Bcombinations%2Bof%2Blineage-determining%2Btranscription%2Bfactors%2Bprime%2Bcis-regulatory%2Belements%2Brequired%2Bfor%2Bmacrophage%2Band%2BB%2Bcell%2Bidentities.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.05.004%26rft_id%253Dinfo%253Apmid%252F20513432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.05.004&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20513432&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F08%2F2020.08.29.273698.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000278448100012&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [22].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.22" data-doi="10.1093/bioinformatics/btp616">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mark D
                                  </span>
                                  <span class="cit-name-surname">
                                   Robinson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Davis J
                                  </span>
                                  <span class="cit-name-surname">
                                   McCarthy
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Gordon K
                                  </span>
                                  <span class="cit-name-surname">
                                   Smyth
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  edger: a bioconductor package for differential expression analysis of digital gene expression data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  139
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  140
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp616%26rft_id%253Dinfo%253Apmid%252F19910308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp616&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19910308&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F08%2F2020.08.29.273698.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000273116100025&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [23].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.23" data-doi="10.1186/s13059-014-0550-8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael I
                                  </span>
                                  <span class="cit-name-surname">
                                   Love
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Wolfgang
                                  </span>
                                  <span class="cit-name-surname">
                                   Huber
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Simon
                                  </span>
                                  <span class="cit-name-surname">
                                   Anders
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Moderated estimation of fold change and dispersion for rna-seq data with deseq2
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  550
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.volume%253D15%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0550-8%26rft_id%253Dinfo%253Apmid%252F25516281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-014-0550-8&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25516281&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F08%2F2020.08.29.273698.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [24].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.24" data-doi="10.1186/s12859-015-0683-0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Aaron TL
                                  </span>
                                  <span class="cit-name-surname">
                                   Lun
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Gordon K
                                  </span>
                                  <span class="cit-name-surname">
                                   Smyth
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  diffhic: a bioconductor package to detect differential genomic interactions in hi-c data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC bioinformatics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  258
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%26rft.volume%253D16%26rft.spage%253D258%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-015-0683-0%26rft_id%253Dinfo%253Apmid%252F26283514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s12859-015-0683-0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26283514&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F08%2F2020.08.29.273698.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [25].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.25" data-doi="10.1186/s12859-018-2201-7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John C
                                  </span>
                                  <span class="cit-name-surname">
                                   Stansfield
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kellen G
                                  </span>
                                  <span class="cit-name-surname">
                                   Cresswell
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Vladimir I
                                  </span>
                                  <span class="cit-name-surname">
                                   Vladimirov
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mikhail G
                                  </span>
                                  <span class="cit-name-surname">
                                   Dozmorov
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Hiccompare: an r-package for joint normalization and comparison of hi-c datasets
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC bioinformatics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%26rft.volume%253D19%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-018-2201-7%26rft_id%253Dinfo%253Apmid%252F29458351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s12859-018-2201-7&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29458351&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F08%2F2020.08.29.273698.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [26].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference [26] in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.26" data-doi="10.1101/gr.212241.116">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mohamed Nadhir
                                  </span>
                                  <span class="cit-name-surname">
                                   Djekidel
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yang
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael Q
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Find: differential chromatin interactions detection using a spatial poisson process
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  412
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  422
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.212241.116%26rft_id%253Dinfo%253Apmid%252F29440282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI4LzMvNDEyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDgvMDgvMjAyMC4wOC4yOS4yNzM2OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [27].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference [27] in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.27" data-doi="10.1093/nar/gkaa069">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kate B
                                  </span>
                                  <span class="cit-name-surname">
                                   Cook
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Borislav H
                                  </span>
                                  <span class="cit-name-surname">
                                   Hristov
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Karine G
                                  </span>
                                  <span class="cit-name-surname">
                                   Le Roch
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jean Philippe
                                  </span>
                                  <span class="cit-name-surname">
                                   Vert
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   William Stafford
                                  </span>
                                  <span class="cit-name-surname">
                                   Noble
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Measuring significant changes in chromatin conformation with accost
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic acids research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  48
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2303
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2311
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft.volume%253D48%26rft.spage%253D2303%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkaa069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkaa069&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [28].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.28" data-doi="10.1101/gr.201517.115">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Phillippa C
                                  </span>
                                  <span class="cit-name-surname">
                                   Taberlay
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joanna
                                  </span>
                                  <span class="cit-name-surname">
                                   Achinger-Kawecka
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Aaron TL
                                  </span>
                                  <span class="cit-name-surname">
                                   Lun
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Fabian A
                                  </span>
                                  <span class="cit-name-surname">
                                   Buske
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kenneth
                                  </span>
                                  <span class="cit-name-surname">
                                   Sabir
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Cathryn M
                                  </span>
                                  <span class="cit-name-surname">
                                   Gould
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Elena
                                  </span>
                                  <span class="cit-name-surname">
                                   Zotenko
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Saul A
                                  </span>
                                  <span class="cit-name-surname">
                                   Bert
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Katherine A
                                  </span>
                                  <span class="cit-name-surname">
                                   Giles
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Denis C
                                  </span>
                                  <span class="cit-name-surname">
                                   Bauer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Three-dimensional disorganization of the cancer genome occurs coincident with long-range genetic and epigenetic alterations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  719
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  731
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.201517.115%26rft_id%253Dinfo%253Apmid%252F27053337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI2LzYvNzE5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDgvMDgvMjAyMC4wOC4yOS4yNzM2OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [29].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kellen G
                                  </span>
                                  <span class="cit-name-surname">
                                   Cresswell
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mikhail G
                                  </span>
                                  <span class="cit-name-surname">
                                   Dozmorov
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Tadcompare: An r package for differential and temporal analysis of topologically associated domains
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Frontiers in Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  158
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BGenetics%26rft.volume%253D11%26rft.spage%253D158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [30].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Raphaël
                                  </span>
                                  <span class="cit-name-surname">
                                   Mourad
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Tadreg: a versatile regression framework for tad identification, differential analysis and rearranged 3d genome prediction
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC bioinformatics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  23
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%26rft.volume%253D23%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [31].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.31" data-doi="10.1093/nar/gkaa220">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joachim
                                  </span>
                                  <span class="cit-name-surname">
                                   Wolff
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Leily
                                  </span>
                                  <span class="cit-name-surname">
                                   Rabbani
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ralf
                                  </span>
                                  <span class="cit-name-surname">
                                   Gilsbach
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Gautier
                                  </span>
                                  <span class="cit-name-surname">
                                   Richard
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Thomas
                                  </span>
                                  <span class="cit-name-surname">
                                   Manke
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Rolf
                                  </span>
                                  <span class="cit-name-surname">
                                   Backofen
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Björn A
                                  </span>
                                  <span class="cit-name-surname">
                                   Grüning
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Galaxy hicexplorer 3: a web server for reproducible hi-c, capture hi-c and single-cell hi-c data analysis, quality control and visualization
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  48
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  W1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  W177
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  W184
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D48%26rft.spage%253DW177%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkaa220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkaa220&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [32].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.29.273698v2.32" data-doi="10.1093/nar/gky504">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joachim
                                  </span>
                                  <span class="cit-name-surname">
                                   Wolff
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Vivek
                                  </span>
                                  <span class="cit-name-surname">
                                   Bhardwaj
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Stephan
                                  </span>
                                  <span class="cit-name-surname">
                                   Nothjunge
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Gautier
                                  </span>
                                  <span class="cit-name-surname">
                                   Richard
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Gina
                                  </span>
                                  <span class="cit-name-surname">
                                   Renschler
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ralf
                                  </span>
                                  <span class="cit-name-surname">
                                   Gilsbach
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Thomas
                                  </span>
                                  <span class="cit-name-surname">
                                   Manke
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Rolf
                                  </span>
                                  <span class="cit-name-surname">
                                   Backofen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Fidel
                                  </span>
                                  <span class="cit-name-surname">
                                   Ramírez
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Björn A
                                  </span>
                                  <span class="cit-name-surname">
                                   Grüning
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Galaxy hicexplorer: a web server for reproducible hi-c data analysis, quality control and visualization
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic acids research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  46
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  W1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  W11
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  W16
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft.volume%253D46%26rft.spage%253DW11%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gky504&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [33].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference [33] in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.33">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Fidel
                                  </span>
                                  <span class="cit-name-surname">
                                   Ramírez
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Vivek
                                  </span>
                                  <span class="cit-name-surname">
                                   Bhardwaj
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Laura
                                  </span>
                                  <span class="cit-name-surname">
                                   Arrigoni
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kin Chung
                                  </span>
                                  <span class="cit-name-surname">
                                   Lam
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Björn A
                                  </span>
                                  <span class="cit-name-surname">
                                   Grüning
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   José
                                  </span>
                                  <span class="cit-name-surname">
                                   Villaveces
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Bianca
                                  </span>
                                  <span class="cit-name-surname">
                                   Habermann
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Asifa
                                  </span>
                                  <span class="cit-name-surname">
                                   Akhtar
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Thomas
                                  </span>
                                  <span class="cit-name-surname">
                                   Manke
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  High-resolution tads reveal dna sequences underlying genome organization in flies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature communications
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  15
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D9%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [34].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [34] in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.34" data-doi="10.1186/s13059-017-1381-1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Marie
                                  </span>
                                  <span class="cit-name-surname">
                                   Zufferey
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Daniele
                                  </span>
                                  <span class="cit-name-surname">
                                   Tavernari
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Elisa
                                  </span>
                                  <span class="cit-name-surname">
                                   Oricchio
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Giovanni
                                  </span>
                                  <span class="cit-name-surname">
                                   Ciriello
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Comparison of computational methods for the identification of topologically associating domains
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  18
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.volume%253D19%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1381-1%26rft_id%253Dinfo%253Apmid%252F29301551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-017-1381-1&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29301551&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F08%2F2020.08.29.273698.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [35].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [35] in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.35" data-doi="10.1101/gr.220640.117">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tao
                                  </span>
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Feipeng
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Galip Gürkan
                                  </span>
                                  <span class="cit-name-surname">
                                   Yardimci
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Fan
                                  </span>
                                  <span class="cit-name-surname">
                                   Song
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ross C
                                  </span>
                                  <span class="cit-name-surname">
                                   Hardison
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   William Stafford
                                  </span>
                                  <span class="cit-name-surname">
                                   Noble
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Feng
                                  </span>
                                  <span class="cit-name-surname">
                                   Yue
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Qunhua
                                  </span>
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Hicrep: assessing the reproducibility of hi-c data using a stratum-adjusted correlation coefficient
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1939
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1949
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.220640.117%26rft_id%253Dinfo%253Apmid%252F28855260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8xMS8xOTM5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDgvMDgvMjAyMC4wOC4yOS4yNzM2OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [36].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference [36] in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.08.29.273698v2.36">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alan
                                  </span>
                                  <span class="cit-name-surname">
                                   Agresti
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Maria
                                  </span>
                                  <span class="cit-name-surname">
                                   Kateri
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Categorical data analysis
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [37].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference [37] in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.37" data-doi="10.1093/imanum/drs019">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Philip A
                                  </span>
                                  <span class="cit-name-surname">
                                   Knight
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Daniel
                                  </span>
                                  <span class="cit-name-surname">
                                   Ruiz
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A fast algorithm for matrix balancing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  IMA Journal of Numerical Analysis
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  33
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1029
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1047
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIMA%2BJournal%2Bof%2BNumerical%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fimanum%252Fdrs019%26rft_id%253Dinfo%253Apmid%252F19029915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/imanum/drs019&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19029915&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F08%2F2020.08.29.273698.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [38].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference [38] in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.38" data-doi="10.2307/2346101">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yoav
                                  </span>
                                  <span class="cit-name-surname">
                                   Benjamini
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yosef
                                  </span>
                                  <span class="cit-name-surname">
                                   Hochberg
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Controlling the false discovery rate: a practical and powerful approach to multiple testing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of the Royal statistical society: series B (Methodological)
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  57
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  289
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  300
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1995
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BRoyal%2Bstatistical%2Bsociety%253A%2Bseries%2BB%2B%2528Methodological%2529%26rft.volume%253D57%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2346101%26rft_id%253Dinfo%253Apmid%252FWOS%253AA1995QE4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.2307/2346101&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=WOS:A1995QE4&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F08%2F2020.08.29.273698.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1995QE45300017&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [39].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference [39] in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.39" data-doi="10.1186/s13059-019-1727-y">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Galip Gürkan
                                  </span>
                                  <span class="cit-name-surname">
                                   Yardimci
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Hakan
                                  </span>
                                  <span class="cit-name-surname">
                                   Ozadam
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael EG
                                  </span>
                                  <span class="cit-name-surname">
                                   Sauria
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Oana
                                  </span>
                                  <span class="cit-name-surname">
                                   Ursu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Koon-Kiu
                                  </span>
                                  <span class="cit-name-surname">
                                   Yan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tao
                                  </span>
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Abhijit
                                  </span>
                                  <span class="cit-name-surname">
                                   Chakraborty
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Arya
                                  </span>
                                  <span class="cit-name-surname">
                                   Kaul
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Bryan R
                                  </span>
                                  <span class="cit-name-surname">
                                   Lajoie
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Fan
                                  </span>
                                  <span class="cit-name-surname">
                                   Song
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Measuring the reproducibility and quality of hi-c data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  19
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.volume%253D20%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-019-1727-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-019-1727-y&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [40].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference [40] in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.40" data-doi="10.1126/science.1105136">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   ENCODE Project
                                  </span>
                                  <span class="cit-name-surname">
                                   Consortium
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The encode (encyclopedia of dna elements) project
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  306
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5696
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  636
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  640
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DThe%2BENCODE%2BProject%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D306%26rft.issue%253D5696%26rft.spage%253D636%26rft.epage%253D640%26rft.atitle%253DThe%2BENCODE%2B%2528ENCyclopedia%2BOf%2BDNA%2BElements%2529%2BProject%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1105136%26rft_id%253Dinfo%253Apmid%252F15499007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDYvNTY5Ni82MzYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wOC8wOC8yMDIwLjA4LjI5LjI3MzY5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [41].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [41] in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.41" data-doi="10.1038/nature11247">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   ENCODE Project
                                  </span>
                                  <span class="cit-name-surname">
                                   Consortium
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  An integrated encyclopedia of dna elements in the human genome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  489
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7414
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  57
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBernstein%26rft.auinit1%253DB.%2BE.%26rft.volume%253D489%26rft.issue%253D7414%26rft.spage%253D57%26rft.epage%253D74%26rft.atitle%253DAn%2Bintegrated%2Bencyclopedia%2Bof%2BDNA%2Belements%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11247%26rft_id%253Dinfo%253Apmid%252F22955616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature11247&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22955616&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F08%2F2020.08.29.273698.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000308347000039&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [42].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference [42] in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.42">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mihaela
                                  </span>
                                  <span class="cit-name-surname">
                                   Pertea
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Daehwan
                                  </span>
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Geo M
                                  </span>
                                  <span class="cit-name-surname">
                                   Pertea
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jeffrey T
                                  </span>
                                  <span class="cit-name-surname">
                                   Leek
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Steven L
                                  </span>
                                  <span class="cit-name-surname">
                                   Salzberg
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Transcript-level expression analysis of rna-seq experiments with hisat, stringtie and ballgown
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature protocols
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1650
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1667
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bprotocols%26rft.volume%253D11%26rft.spage%253D1650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [43].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference [43] in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.43">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Brad T
                                  </span>
                                  <span class="cit-name-surname">
                                   Sherman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Richard A
                                  </span>
                                  <span class="cit-name-surname">
                                   Lempicki
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Systematic and integrative analysis of large gene lists using david bioinformatics resources
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature protocols
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  44
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bprotocols%26rft.volume%253D4%26rft.spage%253D44%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [44].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference [44] in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.44">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Natalie
                                  </span>
                                  <span class="cit-name-surname">
                                   Sauerwald
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Akshat
                                  </span>
                                  <span class="cit-name-surname">
                                   Singhal
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Carl
                                  </span>
                                  <span class="cit-name-surname">
                                   Kingsford
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Analysis of the structural variability of topologically associated domains as revealed by hi-c
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  NAR genomics and bioinformatics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  qz008
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNAR%2Bgenomics%2Band%2Bbioinformatics%26rft.volume%253D2%26rft.spage%253D008qz%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [45].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference [45] in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.29.273698v2.45" data-doi="10.1016/j.ajhg.2021.01.001">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Evonne
                                  </span>
                                  <span class="cit-name-surname">
                                   McArthur
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John A
                                  </span>
                                  <span class="cit-name-surname">
                                   Capra
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Topologically associating domain boundaries that are stable across diverse cell types are evolutionarily constrained and enriched for heritability
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The American Journal of Human Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  108
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  269
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  283
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BJournal%2Bof%2BHuman%2BGenetics%26rft.volume%253D108%26rft.spage%253D269%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2021.01.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2021.01.001&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.08.29.273698v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.12.08.471774v2" title="Testosterone promotes intestinal replication and dissemination of Coxsackievirus B3 in an oral inoculation mouse model" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.03.10.483885v1" title="Cancer related subarachnoid hemorrhage: a multicenter retrospective study using propensity score matching analysis" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted August 08, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.08.29.273698v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.08.29.273698v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.08.29.273698v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   DiffGR: Detecting Differentially Interacting Genomic Regions from Hi-C Contact Maps
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2684813"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-BHyWg4jGmwNaMvIyM4fFqoG7SKc8ErKCosbHMsiLO2M"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484477442"/>
                                   <input type="hidden" name="captcha_token" value="36a133df65829b0a6410c952f700ea6d"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2684813--3" data-node-nid="2684813" id="node-2684813--41068305405" data-pisa="biorxiv;2020.08.29.273698v2" data-pisa-master="biorxiv;2020.08.29.273698" data-seqnum="2684813" data-apath="/biorxiv/early/2022/08/08/2020.08.29.273698.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 DiffGR: Detecting Differentially Interacting Genomic Regions from Hi-C Contact Maps
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Huiling
                                  </span>
                                  <span class="nlm-surname">
                                   Liu
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Wenxiu
                                  </span>
                                  <span class="nlm-surname">
                                   Ma
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.08.29.273698;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.08.29.273698
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.08.29.273698v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.08.29.273698v2&title=DiffGR%3A%20Detecting%20Differentially%20Interacting%20Genomic%20Regions%20from%20Hi-C%20Contact%20Maps" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.08.29.273698v2&title=DiffGR%3A%20Detecting%20Differentially%20Interacting%20Genomic%20Regions%20from%20Hi-C%20Contact%20Maps" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.08.29.273698v2&text=DiffGR%3A%20Detecting%20Differentially%20Interacting%20Genomic%20Regions%20from%20Hi-C%20Contact%20Maps" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.08.29.273698v2&t=DiffGR%3A%20Detecting%20Differentially%20Interacting%20Genomic%20Regions%20from%20Hi-C%20Contact%20Maps" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.08.29.273698v2&title=DiffGR%3A%20Detecting%20Differentially%20Interacting%20Genomic%20Regions%20from%20Hi-C%20Contact%20Maps" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.08.29.273698v2&title=DiffGR%3A%20Detecting%20Differentially%20Interacting%20Genomic%20Regions%20from%20Hi-C%20Contact%20Maps&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.08.29.273698v2&title=DiffGR%3A%20Detecting%20Differentially%20Interacting%20Genomic%20Regions%20from%20Hi-C%20Contact%20Maps" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2684813--5" data-node-nid="2684813" id="citation-node-2684813--6696553197" data-pisa="biorxiv;2020.08.29.273698v2" data-pisa-master="biorxiv;2020.08.29.273698" data-seqnum="2684813" data-apath="/biorxiv/early/2022/08/08/2020.08.29.273698.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     DiffGR: Detecting Differentially Interacting Genomic Regions from Hi-C Contact Maps
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Huiling
                                      </span>
                                      <span class="nlm-surname">
                                       Liu
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Wenxiu
                                      </span>
                                      <span class="nlm-surname">
                                       Ma
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.08.29.273698;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.08.29.273698
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=DiffGR%3A%20Detecting%20Differentially%20Interacting%20Genomic%20Regions%20from%20Hi-C%20Contact%20Maps&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Liu%2C+Huiling&rft.au=Ma%2C+Wenxiu">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2684813/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2684813/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2684813/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2684813/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2684813/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2684813/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2684813/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2684813/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2684813/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2684813/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2684813/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2684813/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.08.29.273698v2&count=horizontal&via=&text=DiffGR%3A%20Detecting%20Differentially%20Interacting%20Genomic%20Regions%20from%20Hi-C%20Contact%20Maps&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.08.29.273698v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.08.29.273698v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.08.29.273698v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Bioinformatics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15187)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
