<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.11.17.386524v3.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.11.17.386524v3.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.11.17.386524v3.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/03/22/2020.11.17.386524.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.11.17.386524v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="A generalised mechano-kinetic model for use in multiscale simulation protocols"/>
  <meta name="DC.Identifier" content="10.1101/2020.11.17.386524"/>
  <meta name="DC.Date" content="2022-03-22"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Many biophysical systems and proteins undergo mesoscopic conformational changes in order to perform their biological function. However, these conformational changes often result from a cascade of atomistic interactions within a much larger overall object. For simulations of such systems, the computational cost of retaining high-resolution structural and dynamical information whilst at the same time observing large scale motions over long times is high. Furthermore, this cost is only going to increase as ever larger biological objects are observed experimentally at high resolution.

We derive a generalised mechano-kinetic simulation framework which aims to compensate for these disparate time scales, capable of dynamically exploring a defined mechanical energy landscape whilst at the same time performing kinetic transitions between discretely defined states. With insight from the theory of Markov state models and transition state theory, the framework models continuous dynamical motion at coarse-grained length scales whilst simultaneously including fine-detail, chemical reactions or conformational changes implicitly using a kinetic model. The kinetic model is coupled to the dynamic model in a highly generalised manner, such that kinetic transition rates are continuously modulated by the dynamic simulation. Further, it can be applied to any defined continuous energy landscape, and hence, any existing dynamic simulation framework. We present a series of analytical examples to validate the framework, and showcase its capabilities for studying more complex systems by simulating protein unfolding via single-molecule force spectroscopy on an atomic force microscope.

Author summary Our intention with this work is to provide a generalised, highly coarse-grained model to allow kinetic processes (conformational changes, protein unfolding, chemical reactions etc) to occur within the context of a dynamic simulation. Performing computationally intensive dynamic simulations to obtain kinetic information can be unjustifiably costly and scientifically inefficient, and so we instead want to emphasise that experimentally available kinetic information can be used to infer the underlying dynamics they result from. We hope our work can begin a discussion on the topic of computationally efficient science, and continue the drive towards collaborative science between theory and experimentation.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Benjamin S. Hanson"/>
  <meta name="DC.Contributor" content="Lorna Dougan"/>
  <meta name="DC.Contributor" content="Oliver G. Harlen"/>
  <meta name="DC.Contributor" content="Sarah A. Harris"/>
  <meta name="DC.Contributor" content="Daniel J. Read"/>
  <meta name="article:published_time" content="2022-03-22"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="A generalised mechano-kinetic model for use in multiscale simulation protocols"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Many biophysical systems and proteins undergo mesoscopic conformational changes in order to perform their biological function. However, these conformational changes often result from a cascade of atomistic interactions within a much larger overall object. For simulations of such systems, the computational cost of retaining high-resolution structural and dynamical information whilst at the same time observing large scale motions over long times is high. Furthermore, this cost is only going to increase as ever larger biological objects are observed experimentally at high resolution.</p><p>We derive a generalised mechano-kinetic simulation framework which aims to compensate for these disparate time scales, capable of dynamically exploring a defined mechanical energy landscape whilst at the same time performing kinetic transitions between discretely defined states. With insight from the theory of Markov state models and transition state theory, the framework models continuous dynamical motion at coarse-grained length scales whilst simultaneously including fine-detail, chemical reactions or conformational changes implicitly using a kinetic model. The kinetic model is coupled to the dynamic model in a highly generalised manner, such that kinetic transition rates are continuously modulated by the dynamic simulation. Further, it can be applied to any defined continuous energy landscape, and hence, any existing dynamic simulation framework. We present a series of analytical examples to validate the framework, and showcase its capabilities for studying more complex systems by simulating protein unfolding via single-molecule force spectroscopy on an atomic force microscope.</p><h3>Author summary</h3>
<p>Our intention with this work is to provide a generalised, highly coarse-grained model to allow kinetic processes (conformational changes, protein unfolding, chemical reactions etc) to occur within the context of a dynamic simulation. Performing computationally intensive dynamic simulations to obtain kinetic information can be unjustifiably costly and scientifically inefficient, and so we instead want to emphasise that experimentally available kinetic information can be used to infer the underlying dynamics they result from. We hope our work can begin a discussion on the topic of computationally efficient science, and continue the drive towards collaborative science between theory and experimentation.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.11.17.386524v3"/>
  <meta name="citation_id" content="2020.11.17.386524v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.11.17.386524v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.11.17.386524v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.11.17.386524v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.11.17.386524"/>
  <meta name="citation_num_pages" content="29"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.11.17.386524"/>
  <meta name="citation_author" content="Benjamin S. Hanson"/>
  <meta name="citation_author_institution" content="School of Physics & Astronomy, University of Leeds"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6079-4506"/>
  <meta name="citation_author" content="Lorna Dougan"/>
  <meta name="citation_author_institution" content="School of Physics & Astronomy, University of Leeds"/>
  <meta name="citation_author_institution" content="Astbury Centre for Structural Molecular Biology, University of Leeds"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2620-5827"/>
  <meta name="citation_author" content="Oliver G. Harlen"/>
  <meta name="citation_author_institution" content="School of Mathematics, University of Leeds"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4593-3547"/>
  <meta name="citation_author" content="Sarah A. Harris"/>
  <meta name="citation_author_institution" content="School of Physics & Astronomy, University of Leeds"/>
  <meta name="citation_author_institution" content="Astbury Centre for Structural Molecular Biology, University of Leeds"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2812-1651"/>
  <meta name="citation_author" content="Daniel J. Read"/>
  <meta name="citation_author_institution" content="School of Mathematics, University of Leeds"/>
  <meta name="citation_reference" content="Bates M, Huang B, Rust MJ, Dempsey GT, Wang W, Zhuang X. Sub-Diffraction-Limit Imaging with Stochastic Optical Reconstruction Microscopy. In: A G, R R, J W, editors. Single Molecule Spectroscopy in Chemistry, Physics and Biology. Springer Berlin Heidelberg; 2010."/>
  <meta name="citation_reference" content="Cheng Y. Single-particle Cryo-EM at crystallographic resolution. Cell. 2015;161(3):450–457."/>
  <meta name="citation_reference" content="Kühlbrandt W. The Resolution Revolution. Science. 2014;343(6178):1443–1444."/>
  <meta name="citation_reference" content="Frank J. Time-resolved Cryo-Electron Microscopy: Recent Progress. Journal of Structural Biology. 2017;200(3):303–306."/>
  <meta name="citation_reference" content="Schermelleh L, Ferrand A, Huser T, Eggeling C, Sauer M, Biehlmaier O, et al. Super-resolution microscopy demystified. Nature Cell Biology. 2019;21(1):72–84."/>
  <meta name="citation_reference" content="Lawson CL, Baker ML, Best C, Bi C, Dougherty M, Feng P, et al. EMDataBank.org: Unified data resource for CryoEM. Nucleic Acids Research. 2011;39:456–464."/>
  <meta name="citation_reference" content="Solernou A, Hanson BS, Richardson RA, Welch R, Read DJ, Harlen OG, et al. Fluctuating Finite Element Analysis (FFEA): A continuum mechanics software tool for mesoscale simulation of biomolecules. PLoS Computational Biology. 2018;14(3):1–6."/>
  <meta name="citation_reference" content="Leng J, Shoura M, McLeish TCB, Real AN, Hardey M, McCafferty J, et al. Securing the future of research computing in the biosciences. PLoS Computational Biology. 2019;15(5):1–15."/>
  <meta name="citation_reference" content="Richardson RA, Hanson BS, Read DJ, Harlen OG, Harris SA. Exploring the dynamics of flagellar dynein within the axoneme with Fluctuating Finite Element Analysis. Quarterly Reviews of Biophysics. 2020;53."/>
  <meta name="citation_reference" content="Hanson BS, Iida S, Read DJ, Harlen OG, Kurisu G, Nakamura H, et al. Continuum mechanical parameterisation of cytoplasmic dynein from atomistic simulation. Methods. 2021;185:39–48."/>
  <meta name="citation_reference" content="Kubo S, Li W, Takada S. Allosteric conformational change cascade in cytoplasmic dynein revealed by structure-based molecular simulations. PLoS Computational Biology. 2017;13(9):1–27."/>
  <meta name="citation_reference" content="Reck-peterson SL, Yildiz A, Carter AP, Gennerich A, Vale RD, Mcintosh JR, et al. Single-Molecule Analysis of Dynein Processivity and Stepping Behavior. Cell. 2006;126(2):335–348."/>
  <meta name="citation_reference" content="Li L, Alper J, Alexov E. Cytoplasmic dynein binding, run length, and velocity are guided by long-range electrostatic interactions. Scientific Reports. 2016;6(1):1–12."/>
  <meta name="citation_reference" content="Kinoshita Y, Kambara T, Nishikawa K, Kaya M, Higuchi H. Step Sizes and Rate Constants of Single-headed Cytoplasmic Dynein Measured with Optical Tweezers. Scientific Reports. 2018;8(1):1–14."/>
  <meta name="citation_reference" content="Sasaki K, Kaya M, Higuchi H. A Unified Walking Model for Dimeric Motor Proteins. Biophysical Journal. 2018;115(10):1981–1992."/>
  <meta name="citation_reference" content="Dudko OK, Hummer G, Szabo A. Intrinsic rates and activation free energies from single-molecule pulling experiments. Physical Review Letters. 2006;96(10):1–4."/>
  <meta name="citation_reference" content="Na YR, Park C. Investigating protein unfolding kinetics by pulse proteolysis. Protein Science. 2009;18(2):268–276."/>
  <meta name="citation_reference" content="West DK, Olmsted PD, Paci E. Mechanical unfolding revisited through a simple but realistic model. Journal of Chemical Physics. 2006;124(15)."/>
  <meta name="citation_reference" content="Ohashi KG, Han L, Mentley B, Wang J, Fricks J, Hancock WO. Load-dependent detachment kinetics plays a key role in bidirectional cargo transport by kinesin and dynein. Traffic. 2019;20(4):284–294."/>
  <meta name="citation_reference" content="Belyy V, Schlager MA, Foster H, Reimer AE, Carter AP, Yildiz A. The mammalian dynein-dynactin complex is a strong opponent to kinesin in a tug-of-war competition. Nature Cell Biology. 2016;18(9):1018–1024."/>
  <meta name="citation_reference" content="Linke WA, Ivemeyer M, Mundel P, Stockmeier MR, Kolmerer B. Nature of PEVK-titin elasticity in skeletal muscle. Proceedings of the National Academy of Sciences of the United States of America. 1998;95(14):8052–8057."/>
  <meta name="citation_reference" content="Hughes MDG, Cussons S, Mahmoudi N, Brockwell DJ, Dougan L. Single molecule protein stabilisation translates to macromolecular mechanics of a protein network. Soft Matter. 2020;16(27):6389–6399."/>
  <meta name="citation_reference" content="Sarlah A, Vilfan A. The winch model can explain both coordinated and uncoordinated stepping of cytoplasmic dynein. Biophysical Journal. 2014;107(3):662–671."/>
  <meta name="citation_reference" content="Sarlah A, Vilfan A. Minimum requirements for motility of a processive motor protein. PLoS ONE. 2017;12(10):1–13."/>
  <meta name="citation_reference" content="West DK, Olmsted PD, Paci E. Free energy for protein folding from nonequilibrium simulations using the Jarzynski equality. Journal of Chemical Physics. 2006;125(20):1–7."/>
  <meta name="citation_reference" content="West DK, Paci E, Olmsted PD. Internal protein dynamics shifts the distance to the mechanical transition state. Physical Review E. 2006;74(6):061912."/>
  <meta name="citation_reference" content="Staple DB, Payne SH, Reddin ALC, Kreuzer HJ. Model for stretching and unfolding the giant multidomain muscle protein using single-molecule force spectroscopy. Physical Review Letters. 2008;101(24):1–4."/>
  <meta name="citation_reference" content="Sikora M, Sulkowska JI, Witkowski BS, Cieplak M. BSDB: The biomolecule stretching database. Nucleic Acids Research. 2011;39(SUPPL. 1)."/>
  <meta name="citation_reference" content="Booth JJ, Shalashilin DV. Fully Atomistic Simulations of Protein Unfolding in Low Speed Atomic Force Microscope and Force Clamp Experiments with the Help of Boxed Molecular Dynamics. Journal of Physical Chemistry B. 2016;120(4):700–708."/>
  <meta name="citation_reference" content="Takada S. Go model revisited. Biophysics and Physicobiology. 2019;16(0):248–255."/>
  <meta name="citation_reference" content="Bell GI. Models for the specific adhesion of cells to cells. Science. 1978;200(4342):618–627."/>
  <meta name="citation_reference" content="Hughes ML, Dougan L. The physics of pulling polyproteins: A review of single molecule force spectroscopy using the AFM to study protein unfolding. Reports on Progress in Physics. 2016;79(7):076601."/>
  <meta name="citation_reference" content="Hughes MDG, Hanson BS, Cussons S, Mahmoudi N, Brockwell DJ, Dougan L. Control of Nanoscale in Situ Protein Unfolding Defines Network Architecture and Mechanics of Protein Hydrogels. ACS Nano. 2021;15(7):11296–11308."/>
  <meta name="citation_reference" content="Imai H, Shima T, Sutoh K, Walker ML, Knight PJ, Kon T, et al. Direct observation shows superposition and large scale flexibility within cytoplasmic dynein motors moving along microtubules. Nature Communications. 2015;6(1):1–11."/>
  <meta name="citation_reference" content="Kamiya N, Mashimo T, Takano Y, Kon T, Kurisu G, Nakamura H. Elastic properties of dynein motor domain obtained from all-atom molecular dynamics simulations. Protein Engineering, Design and Selection. 2016;29(8):317–326."/>
  <meta name="citation_reference" content="Niekamp S, Stuurman N, Zhang N, Vale RD. Three-color single-molecule imaging reveals conformational dynamics of dynein undergoing motility. Proceedings of the National Academy of Sciences of the United States of America. 2021;118(31)."/>
  <meta name="citation_reference" content="Bertz M, Rief M. Mechanical Unfoldons as Building Blocks of Maltose-binding Protein. Journal of Molecular Biology. 2008;378(2):447–458."/>
  <meta name="citation_reference" content="Wood K, Frölich A, Paciaroni A, Moulin M, Härtlein M, Zaccai G, et al. Coincidence of dynamical transitions in a soluble protein and its hydration water: Direct measurements by neutron scattering and MD simulations. Journal of the American Chemical Society. 2008;130(14):4586–4587."/>
  <meta name="citation_reference" content="Telmer PG, Shilton BH. Insights into the Conformational Equilibria of Maltose-binding Protein by Analysis of High Affinity Mutants. Journal of Biological Chemistry. 2003;278(36):34555–34567."/>
  <meta name="citation_reference" content="Hoffmann T, Dougan L. Single molecule force spectroscopy using polyproteins. Chemical Society Reviews. 2012;41(14):4781–4796."/>
  <meta name="citation_reference" content="Hoffmann T, Tych KM, Hughes ML, Brockwell DJ, Dougan L. Towards design principles for determining the mechanical stability of proteins. Physical Chemistry Chemical Physics. 2013;15(38):15767–15780."/>
  <meta name="citation_reference" content="Tych KM, Hughes ML, Bourke J, Taniguchi Y, Kawakami M, Brockwell DJ, et al. Optimizing the calculation of energy landscape parameters from single-molecule protein unfolding experiments. Physical Review E - Statistical, Nonlinear, and Soft Matter Physics. 2015;91(1):1–9."/>
  <meta name="citation_reference" content="Da Silva MA, Lenton S, Hughes M, Brockwell DJ, Dougan L. Assessing the Potential of Folded Globular Polyproteins As Hydrogel Building Blocks. Biomacromolecules. 2017;18(2):636–646."/>
  <meta name="citation_reference" content="Stirnemann G, Giganti D, Fernandez JM, Berne BJ. Elasticity, structure, and relaxation of extended proteins under force. Proceedings of the National Academy of Sciences of the United States of America. 2013;110(10):3847–3852."/>
  <meta name="citation_reference" content="Phillips JC, Hardy DJ, Maia JDC, Stone JE, Ribeiro JV, Bernardi RC, et al. Scalable molecular dynamics on CPU and GPU architectures with NAMD. Journal of Chemical Physics. 2020;153(4)."/>
  <meta name="citation_reference" content="Mücksch C, Urbassek HM. Accelerating Steered Molecular Dynamics: Toward Smaller Velocities in Forced Unfolding Simulations. Journal of Chemical Theory and Computation. 2016;12(3):1380–1384."/>
  <meta name="citation_reference" content="Zinober RC, Brockwell DJ, Beddard GS, Blake AW, Olmsted PD, Radford SE, et al. Mechanically unfolding proteins: The effect of unfolding history and the supramolecular scaffold. Protein Science. 2009;11(12):2759–2765."/>
  <meta name="citation_reference" content="Uversky VN. Intrinsically disordered proteins and their ‘‘Mysterious” (meta)physics. Frontiers in Physics. 2019;7:8–23."/>
  <meta name="citation_reference" content="Feller G. Protein stability and enzyme activity at extreme biological temperatures. Journal of Physics Condensed Matter. 2010;22(32)."/>
  <meta name="citation_reference" content="Reed CJ, Lewis H, Trejo E, Winston V, Evilia C. Protein adaptations in archaeal extremophiles. Archaea. 2013;."/>
  <meta name="citation_reference" content="Brininger C, Spradlin S, Cobani L, Evilia C. The more adaptive to change, the more likely you are to survive: Protein adaptation in extremophiles. Seminars in Cell and Developmental Biology. 2018;84:158–169."/>
  <meta name="citation_reference" content="Evans E. Probing the Relation Between Force, Lifetime and Chemistry in Single Molecular Bonds. Annual Review of Biophysics and Biomolecular Structure. 2001;30(1):105–128."/>
  <meta name="citation_reference" content="Isojima H, Iino R, Niitani Y, Noji H, Tomishige M. Direct observation of intermediate states during the stepping motion of kinesin-1. Nature Chemical Biology. 2016;12(4):290–297."/>
  <meta name="citation_reference" content="Bowman GR, Pande VS, Noe F. An Introduction to Markov State Models and Their Application to Long Timescale Molecular Simulation. Springer; 2014."/>
  <meta name="citation_reference" content="Husic BE, Pande VS. Markov State Models: From an Art to a Science. Journal of the American Chemical Society. 2018;140(7):2386–2396."/>
  <meta name="citation_reference" content="Bialek W, Onuchic JN. Protein dynamics and reaction rates: mode-specific chemistry in large molecules? Proceedings of the National Academy of Sciences of the United States of America. 1988;85(16):5908–5912."/>
  <meta name="citation_reference" content="Brockwell DJ, Paci E, Zinober RC, Beddard GS, Olmsted PD, Smith DA, et al. Pulling geometry defines the mechanical resistance of a beta-sheet protein. Nature Structural Biology. 2003;10(9):731–737."/>
  <meta name="citation_reference" content="Carrion-Vazquez M, Li H, Lu H, Marszalek PE, Oberhauser AF, Fernandez JM. The mechanical stability of ubiquitin is linkage dependent. Nature Structural Biology. 2003;10(9):738–743."/>
  <meta name="citation_reference" content="Dietz H, Berkemeier F, Bertz M, Rief M. Anisotropic deformation response of single protein molecules. Proceedings of the National Academy of Sciences of the United States of America. 2006;103(34):12724–12728."/>
  <meta name="citation_reference" content="Trott LE, Hafezparast M, Madzvamuse A. A mathematical understanding of how cytoplasmic dynein walks on microtubules. Royal Society Open Science. 2018;5(8):171568."/>
  <meta name="twitter:title" content="A generalised mechano-kinetic model for use in multiscale simulation protocols"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Many biophysical systems and proteins undergo mesoscopic conformational changes in order to perform their biological function. However, these conformational changes often result from a cascade of atomistic interactions within a much larger overall object. For simulations of such systems, the computational cost of retaining high-resolution structural and dynamical information whilst at the same time observing large scale motions over long times is high. Furthermore, this cost is only going to increase as ever larger biological objects are observed experimentally at high resolution.

We derive a generalised mechano-kinetic simulation framework which aims to compensate for these disparate time scales, capable of dynamically exploring a defined mechanical energy landscape whilst at the same time performing kinetic transitions between discretely defined states. With insight from the theory of Markov state models and transition state theory, the framework models continuous dynamical motion at coarse-grained length scales whilst simultaneously including fine-detail, chemical reactions or conformational changes implicitly using a kinetic model. The kinetic model is coupled to the dynamic model in a highly generalised manner, such that kinetic transition rates are continuously modulated by the dynamic simulation. Further, it can be applied to any defined continuous energy landscape, and hence, any existing dynamic simulation framework. We present a series of analytical examples to validate the framework, and showcase its capabilities for studying more complex systems by simulating protein unfolding via single-molecule force spectroscopy on an atomic force microscope.

Author summary Our intention with this work is to provide a generalised, highly coarse-grained model to allow kinetic processes (conformational changes, protein unfolding, chemical reactions etc) to occur within the context of a dynamic simulation. Performing computationally intensive dynamic simulations to obtain kinetic information can be unjustifiably costly and scientifically inefficient, and so we instead want to emphasise that experimentally available kinetic information can be used to infer the underlying dynamics they result from. We hope our work can begin a discussion on the topic of computationally efficient science, and continue the drive towards collaborative science between theory and experimentation.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="A generalised mechano-kinetic model for use in multiscale simulation protocols"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.11.17.386524v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Many biophysical systems and proteins undergo mesoscopic conformational changes in order to perform their biological function. However, these conformational changes often result from a cascade of atomistic interactions within a much larger overall object. For simulations of such systems, the computational cost of retaining high-resolution structural and dynamical information whilst at the same time observing large scale motions over long times is high. Furthermore, this cost is only going to increase as ever larger biological objects are observed experimentally at high resolution.

We derive a generalised mechano-kinetic simulation framework which aims to compensate for these disparate time scales, capable of dynamically exploring a defined mechanical energy landscape whilst at the same time performing kinetic transitions between discretely defined states. With insight from the theory of Markov state models and transition state theory, the framework models continuous dynamical motion at coarse-grained length scales whilst simultaneously including fine-detail, chemical reactions or conformational changes implicitly using a kinetic model. The kinetic model is coupled to the dynamic model in a highly generalised manner, such that kinetic transition rates are continuously modulated by the dynamic simulation. Further, it can be applied to any defined continuous energy landscape, and hence, any existing dynamic simulation framework. We present a series of analytical examples to validate the framework, and showcase its capabilities for studying more complex systems by simulating protein unfolding via single-molecule force spectroscopy on an atomic force microscope.

Author summary Our intention with this work is to provide a generalised, highly coarse-grained model to allow kinetic processes (conformational changes, protein unfolding, chemical reactions etc) to occur within the context of a dynamic simulation. Performing computationally intensive dynamic simulations to obtain kinetic information can be unjustifiably costly and scientifically inefficient, and so we instead want to emphasise that experimentally available kinetic information can be used to infer the underlying dynamics they result from. We hope our work can begin a discussion on the topic of computationally efficient science, and continue the drive towards collaborative science between theory and experimentation.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-03-22"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.11.17.386524v3"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2460219"/>
  <title>
   A generalised mechano-kinetic model for use in multiscale simulation protocols | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"oFvyNeyr6vCaVFvC6pN6QKHukMHrY9GRGa4EtCAtULU"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2460219","apath":"\/biorxiv\/early\/2022\/03\/22\/2020.11.17.386524.atom","pisa":"biorxiv;2020.11.17.386524v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.11.17.386524v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.11.17.386524v3","title":"A generalised mechano-kinetic model for use in multiscale simulation protocols","identifier":"node\/2460219"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022JhwAUIKEma2Opg4Q0NCyeyKb8\\\\\\\/RRxS3hmFt1zEV9E2cHtazBqRHaMLW6DwDzn5TkPRKQzLJRrVrIrGschmpCReDqPUcmpo0NACevqXkZGd8fTeFTH6\\\\\\\/G1OUVzUE0tpTkYtTe+ARKz\\\\\\\/goSC7+DRRUwkTWgbdvhjW60oAtXvcoBrNiPeajyvv\\\\\\\/83ovHPU2Xz\\\\\\\/N0BKrM8t45LKV32hcZLxbpIGSLp81xcwKNh+pIS1lb11MEbQesqiHG0iSTml7KF7TS85Fqf5YcKNjGPQSy+bir1eSTDHM\\\\\\\/uqT23z9ju4IsS6vAjEjJFLSyzlVn3DqdaJ3t+1Tb2HGSnpNsRNpcpgfvBK5XwI4YFiDm1K4U9Wo+VrpvwtAfdVcLUEy\\\\\\\/fvqaGA1sa63AperhJ5l80VPZjptezvEsJCb5+oMrTYqms3XAC74rB65xIqfYEkEBncuPtETcDrguVkb46fH3wPmnCdrlI9RkzHN354h0cohaec1NJv9aGFCGa0\\\\\\\/YSqml3ISWCXIRbR6V0Aq55J+SKmMiba1h+SfSP8kfi0EYO8OfNTrcvXV\\\\\\\/c9P9L6p8pUa2HRfzQWy459RcCLh13nnP\\\\\\\/PUk6YQjXrZLMed+0QLsdH\\\\\\\/ZWBMaHH8ndvU8M1gQ27Yjg+qQht29w1k82ripzQLtxeMCeyb8X0VmdNIovbdqJJPahvJxYYOTyKbP6\\\\\\\/2Vtw3lBFp9xEk1n5arL+cT97V+syUX\\\\\\\/pQMB+g6VK68mkZsoLXC3RKLoe1OkrHZ9ZngX33gKJBNCCHq1kymv81oWFU5289\\\\\\\/ZhZRsrcXvDEGcKPyxH6OAAf\\\\\\\/Lf318Acayaci0oq9QDYSFClSlSGfxXDkw38gpDF71oroFD7DEKy3migC5vT+zCHqcKXZeAPWLX\\\\\\\/NMu71PhsI72u0mGuHv6fPhp4o1F8+0MR6Y68tJDcAutCw\\\\\\\/zI9gmabEe+lVKQfrvkLjXyIYweWyuKxsBxGccMvK4U40icIb7qPkFGBeJiJ5KI1PHuNSlVfEZLnuCvKaUjQHLd12MXkwMVh\\\\\\\/t3HMkG2fmlWBbA3oVs1\\\\\\\/UJev9B9WC3haY\\\\\\\/1THSjDZmGedq5gO0WbhXACE4KT1fRcN6V\\\\\\\/\\\\\\\/5+\\\\\\\/28kqGOaaONdS8zTXJrUGV304PEQ37x+LTbtZrRvl7bsh1n1x4vi7jK\\\\\\\/a0EEVTLuobYNGT+klq47mgM0+9ajH\\\\\\\/U9bcAAdZnDdBMmu+dKdFk6n4PweJPIocGQUf+xjMV9iY0RIy6Ej3H3v0oSksPLXPu6qnQgIePNANrof7jQjdsQWaNvcdanD00nOfGJWC2lvPFu9N+iJ9K08uqTBqKzKVMFDeyJTBQLFj6Xa\\\\\\\/wf1aRUIVNFNvwG9sCfcMwmCsJwR\\\\\\\/w8E6gnCe69Ss2ORcNqdderqq\\\\\\\/euk3JnI=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Zr+Wwl\\\\\\\/Z9alMWmAYM0HAMw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00224582aecaf2177bfee7c7304b3f2ada10d06da4274eb92fec637d044f0a6d8560\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.11.17.386524v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2460219 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.11.17.386524v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_1219395910">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_1219395910" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1624444170" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-Z8hEvF57Lg68QV7Z91DXJAVSNmM7BuOpGv66qsyB10c"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2460219" data-node-nid="2460219" id="node-2460219--2107793310" data-pisa="biorxiv;2020.11.17.386524v3" data-pisa-master="biorxiv;2020.11.17.386524" data-apath="/biorxiv/early/2022/03/22/2020.11.17.386524.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    A generalised mechano-kinetic model for use in multiscale simulation protocols
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0002-6079-4506" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Benjamin S.
                      </span>
                      <span class="nlm-surname">
                       Hanson
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0002-2620-5827" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Lorna
                      </span>
                      <span class="nlm-surname">
                       Dougan
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2">
                      <a href="http://orcid.org/0000-0002-4593-3547" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Oliver G.
                      </span>
                      <span class="nlm-surname">
                       Harlen
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3">
                      <a href="http://orcid.org/0000-0002-2812-1651" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Sarah A.
                      </span>
                      <span class="nlm-surname">
                       Harris
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="4">
                      <span class="nlm-given-names">
                       Daniel J.
                      </span>
                      <span class="nlm-surname">
                       Read
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.11.17.386524
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2460219--2107793310" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Benjamin S. Hanson
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Physics & Astronomy, University of Leeds
                       </span>
                       , Leeds,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Benjamin%2BS.%2BHanson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Hanson%20BS&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ABenjamin%2BS.%2BHanson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-6079-4506" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Benjamin S. Hanson
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Lorna Dougan
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Physics & Astronomy, University of Leeds
                       </span>
                       , Leeds,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Astbury Centre for Structural Molecular Biology, University of Leeds
                       </span>
                       , Leeds,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Lorna%2BDougan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Dougan%20L&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ALorna%2BDougan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-2620-5827" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Lorna Dougan
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Oliver G. Harlen
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        School of Mathematics, University of Leeds
                       </span>
                       , Leeds,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Oliver%2BG.%2BHarlen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Harlen%20OG&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AOliver%2BG.%2BHarlen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-4593-3547" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Oliver G. Harlen
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Sarah A. Harris
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Physics & Astronomy, University of Leeds
                       </span>
                       , Leeds,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Astbury Centre for Structural Molecular Biology, University of Leeds
                       </span>
                       , Leeds,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Sarah%2BA.%2BHarris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Harris%20SA&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ASarah%2BA.%2BHarris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-2812-1651" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Sarah A. Harris
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Daniel J. Read
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        School of Mathematics, University of Leeds
                       </span>
                       , Leeds,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Daniel%2BJ.%2BRead%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Read%20DJ&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ADaniel%2BJ.%2BRead%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.11.17.386524v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2460219" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2460219/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.17.386524v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2460219" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2460219/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.17.386524v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2460219" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2460219/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.17.386524v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2460219" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2460219/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.17.386524v3.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2460219" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2460219/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.11.17.386524v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2460219" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2460219/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-3">
                             Many biophysical systems and proteins undergo mesoscopic conformational changes in order to perform their biological function. However, these conformational changes often result from a cascade of atomistic interactions within a much larger overall object. For simulations of such systems, the computational cost of retaining high-resolution structural and dynamical information whilst at the same time observing large scale motions over long times is high. Furthermore, this cost is only going to increase as ever larger biological objects are observed experimentally at high resolution.
                            </p>
                            <p id="p-4">
                             We derive a generalised mechano-kinetic simulation framework which aims to compensate for these disparate time scales, capable of dynamically exploring a defined mechanical energy landscape whilst at the same time performing kinetic transitions between discretely defined states. With insight from the theory of Markov state models and transition state theory, the framework models continuous dynamical motion at coarse-grained length scales whilst simultaneously including fine-detail, chemical reactions or conformational changes implicitly using a kinetic model. The kinetic model is coupled to the dynamic model in a highly generalised manner, such that kinetic transition rates are continuously modulated by the dynamic simulation. Further, it can be applied to any defined continuous energy landscape, and hence, any existing dynamic simulation framework. We present a series of analytical examples to validate the framework, and showcase its capabilities for studying more complex systems by simulating protein unfolding via single-molecule force spectroscopy on an atomic force microscope.
                            </p>
                            <div id="sec-1" class="subsection">
                             <p id="p-5">
                              <strong>
                               Author summary
                              </strong>
                              Our intention with this work is to provide a generalised, highly coarse-grained model to allow kinetic processes (conformational changes, protein unfolding, chemical reactions etc) to occur within the context of a dynamic simulation. Performing computationally intensive dynamic simulations to obtain kinetic information can be unjustifiably costly and scientifically inefficient, and so we instead want to emphasise that experimentally available kinetic information can be used to infer the underlying dynamics they result from. We hope our work can begin a discussion on the topic of computationally efficient science, and continue the drive towards collaborative science between theory and experimentation.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-8">
                             Over the past decade, experimental techniques have matured to such an extent that we can observe biological systems on length scales never before seen. Cryo-electron microscopy is enabling us to determine the atomistic structures of macromolecular complexes without the need for crystallisation, providing us with significantly more realistic instances of biological functional pathways [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             –
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ]. Additionally, the continued development of various forms of super resolution microscopy and high throughput data analysis allow us to observe motions over ever shorter time scales [
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ,
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ]. Many of these structures are being deposited into the Electron Microscopy Data Bank (EMDB), which has seen an almost exponential growth rate over the past decade [
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ]. Simulation techniques are being developed which utilise this data, especially at coarse-grained scales [
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             ]. However, further consideration is needed with respect to the meaning of physical models at these newly explored spatio-temporal scales [
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ], and to the practicalities of performing computer simulations of these almost microscopic objects.
                            </p>
                            <p id="p-9">
                             We may take as an illustrative example the molecular motor dynein, a system we have previously investigated [
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ,
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ]. We consider that the features we outline now for dynein are common to many biomolecules, and to attempts to capture their operation in a multiscale simulation framework. Hanson et al. recently used existing all-atom molecular dynamics simulations of a single monomer of the motor to parameterise a much coarser continuum mechanical model of the same object [
                             <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ], and found that a significantly coarser model can be used successfully for simulating the equilibrium dynamics. However, the biological function of the motor is in its so-called “powerstroke”, a change of equilibrium shape driven by cascading atomistic processes [
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ] which generates the directed force required for the motor to do the useful physical work [
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             –
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ]. Experiments show that the time between individual powerstroke events is typically much longer than either the mechanical relaxation time of the molecule or the time over which the powerstroke itself occurs. Hence the change in equilibrium configuration associated with the powerstroke is often considered as an effectively instantaneous chemical kinetic event [
                             <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ]. However, given that the underlying structure of the motor [
                             <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ] and its physical environment [
                             <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ] affect its equilibrium, these kinetic events may not be completely independent of the underlying dynamics from which they emerge. The question then arises as to how one should include the powerstroke event into coarse-grained dynamical models of dynein, and reconcile these disparate timescales.
                            </p>
                            <p id="p-10">
                             Dynein is not the only example of such disparate timescales in biological physics. We believe that the issues encountered above are sufficiently general to warrant consideration. Any coarse-grained biomolecular simulation might need to model transitions between different states, in which the configuration of a molecule at mechanical equilibrium is changed. Two particular examples of this are (i) molecular motors (as in dynein above) in which the configurational change between pre- and post-powerstroke states is driven by reactions within chemical-kinetic cycle, or (ii) protein unfolding, either at equilibrium [
                             <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             –
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ] or driven by application of external force. External force can play a role in both of these examples. Translational molecular motors such as dynein have measurable stall forces [
                             <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ,
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              19
                             </a>
                             ], and these forces can be applied by other molecular motors acting in the opposite direction [
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              20
                             </a>
                             ]. Proteins are subjected to large forces in the artificial setting of single-molecule force spectroscopy experiments using an atomic force microscope (SMFS-AFM), but also in natural settings such as titin within muscle fibres [
                             <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                              21
                             </a>
                             ], and engineered biological matrices such as protein hydrogels [
                             <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                              22
                             </a>
                             ]. These external factors change the effective activation energy of the kinetic transition, and hence change the rate and manner by which thermal fluctuations and other dynamical effects drive transitions in these types of system.
                            </p>
                            <p id="p-11">
                             The above discussion shows a need for a framework in which the disparity between dynamic and kinetic timescales are somehow reconciled, and their coupling explicitly modelled. This has been previously attempted specifically with respect to both molecular motors [
                             <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ,
                             <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                              24
                             </a>
                             ] and much more commonly to protein unfolding [
                             <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ,
                             <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                              25
                             </a>
                             –
                             <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                              30
                             </a>
                             ], but to our knowledge, a fully general approach has yet to be discussed. To address this considerable challenge in mesoscale modelling, we have developed a generalised ‘mechano-kinetic’ simulation framework which utilises both explicit mechanical modelling at mesoscopic length scales, whilst at the same time accounting for chemical-kinetic events beyond the scope of the dynamic model with a coupled kinetic model similar to a Markov state model. This coupling is such that the kinetic rates themselves dynamically alter as the system moves through the dynamic energy landscape and hence, using only the principle of detailed balance, our simulation framework enables the mesoscopic dynamical simulation to instantaneously contribute towards kinetic events such as conformational changes and chemical reactions, and thus implicitly affect dynamics at smaller length scales and vice versa.
                            </p>
                           </div>
                           <div class="section" id="sec-3">
                            <h2 class="">
                             Theoretical Development
                            </h2>
                            <p id="p-12">
                             Our goal is to derive a ‘mechano-kinetic’ simulation framework; a simulation that is able to dynamically explore a defined mechanical energy landscape whilst at the same time perform kinetic transitions between discretely defined states. Here, and in what follows, we use the word dynamics to refer to changes in shape and conformation of a molecule as it explores a mechanical energy landscape; we use the word kinetics to refer to (discrete) changes of state which would alter that landscape (e.g. due to a chemical change such as a binding or unbinding event). We recognise it is usually not possible to cleanly distinguish these. Nevertheless, such a distinction is forced on us in coarse-grained modelling: “dynamical” changes refer to changes in molecular configuration explicitly resolved in the model, whilst “kinetic” changes are driven by processes at smaller, unresolved, scales.
                            </p>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Mesostates
                             </h3>
                             <p id="p-13">
                              Importantly, we want the location in dynamical phase space to affect the kinetic processes in a thermodynamically consistent manner, such that we gain insight into how kinetic processes interact with different mechanical energy environments. To emphasise the target application of the biological mesoscale, and also to distinguish the discrete kinetic regime from continuous mechanical regime, we will refer to the discrete kinetic states as “mesostates”.
                             </p>
                             <p id="p-14">
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Fig.1
                              </a>
                              shows a schematic of a kinetic network, where each mesostate has some associated occupation probability P
                              <sub>
                               i
                              </sub>
                              . For a physical system, these occupation probabilities can be written in terms of a free energy, F
                              <sub>
                               i
                              </sub>
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where Z
                              <sub>
                               T
                              </sub>
                              is the total partition function of the entire network, here acting as a normalisation constant. These probabilities, at equilibrium, must be related to the transition rates between mesostates, R
                              <sub>
                               ij
                              </sub>
                              , by the principle of detailed balance
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              In principle, these probabilites and rates can be measured, either by experimentation or by a Markov State Model (MSM) calculation from a higher resolution simulation.
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F1.large.jpg?width=800&height=600&carousel=1" title="A dimensionless representation of a kinetic network with the number of states N = 5. Each state has an associated probability Pi which is determined by the number of available microstates within it. Transitions between states occur explicitly between microstates with rates rij, as shown by the dashed arrows, which then determine the relative transition rates Rij between each pair of states i, j, as given by the principle of detailed balance." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-843933992" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">A dimensionless representation of a kinetic network with the number of states <em>N</em> = 5. Each state has an associated probability <em>P<sub>i</sub></em> which is determined by the number of available microstates within it. Transitions between states occur explicitly between microstates with rates <em>r<sub>ij</sub></em>, as shown by the dashed arrows, which then determine the relative transition rates <em>R<sub>ij</sub></em> between each pair of states <em>i, j</em>, as given by the principle of detailed balance.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F1.medium.gif" width="440" height="309"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F1.medium.gif" width="440" height="309"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Fig 1.
                               </span>
                               <p id="p-15" class="first-child">
                                A dimensionless representation of a kinetic network with the number of states N = 5. Each state has an associated probability P
                                <sub>
                                 i
                                </sub>
                                which is determined by the number of available microstates within it. Transitions between states occur explicitly between microstates with rates r
                                <sub>
                                 ij
                                </sub>
                                , as shown by the dashed arrows, which then determine the relative transition rates R
                                <sub>
                                 ij
                                </sub>
                                between each pair of states i, j, as given by the principle of detailed balance.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Microstates and Free Energy
                             </h3>
                             <p id="p-16">
                              As shown for mesostates 2 and 3 in
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Fig.1
                              </a>
                              , each mesostate contains within it a (potentially large number) of microstates. What these microstates are, and how they are parameterised, will depend on the system in question. In the examples considered here, the microstates are considered to be the set of (mechanical) shapes or configurations that a protein can take: we denote the microstate by the multidimensional vector
                              <span class="inline-formula" id="inline-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . So, our physical picture is that the molecule exists within some mesostate i for some time, dynamically exploring changes of shape represented by changes in
                              <span class="inline-formula" id="inline-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <p id="p-17">
                              The total partition function for the system is:
                              <span class="disp-formula" id="disp-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where the partition function for mesostate i is:
                              <span class="disp-formula" id="disp-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-18">
                              Here,
                              <span class="inline-formula" id="inline-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the energy as a function of
                              <span class="inline-formula" id="inline-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (it is, in fact, a free energy if
                              <span class="inline-formula" id="inline-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              are coarse-grained variables). g
                              <sub>
                               i
                              </sub>
                              is the density of states within mesostate i. Any dependence of density of state on
                              <span class="inline-formula" id="inline-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-6.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-6.gif"/>
                                </noscript>
                               </span>
                              </span>
                              can be included as an entropic term within
                              <span class="inline-formula" id="inline-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (and should be, since it will affect the dynamics of
                              <span class="inline-formula" id="inline-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ). Then, the equilibrium probability density of system being at microstate
                              <span class="inline-formula" id="inline-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                              within mesostate i is:
                              <span class="disp-formula" id="disp-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-6.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-6.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-10.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-10.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . All dynamics and kinetics in the system must be consistent with these equilibrium distributions.
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Kinetic transitions between microstates
                             </h3>
                             <p id="p-19">
                              At some instant, a kinetic event may occur such that the molecule transitions from mesostate i to mesostate j. But, that transition must actually occur from a microstate within i, to a microstate within j, as illustrated explicitly for mesostates 2 and 3 in
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Fig.1
                              </a>
                              . Detailed balance must also apply to such transitions between microstates.
                             </p>
                             <p id="p-20">
                              We define
                              <span class="inline-formula" id="inline-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-11.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-11.gif"/>
                                </noscript>
                               </span>
                              </span>
                              to be the transition rate from a single microstate at
                              <span class="inline-formula" id="inline-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-12.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-12.gif"/>
                                </noscript>
                               </span>
                              </span>
                              within i to a single microstate at
                              <span class="inline-formula" id="inline-formula-13">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-13.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-13.gif"/>
                                </noscript>
                               </span>
                              </span>
                              within j. Then, the equilibrium rate of transition from volume
                              <span class="inline-formula" id="inline-formula-14">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-14.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-14.gif"/>
                                </noscript>
                               </span>
                              </span>
                              to volume
                              <span class="inline-formula" id="inline-formula-15">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-15.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-15.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the probability of being in
                              <span class="inline-formula" id="inline-formula-16">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-16.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-16.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , multiplied by the rate of transition per state, multiplied by the number of states in
                              <span class="inline-formula" id="inline-formula-17">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-17.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-17.gif"/>
                                </noscript>
                               </span>
                              </span>
                              :
                              <span class="disp-formula" id="disp-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-21">
                              A similar equation applies to the equilibrium rate of reverse transition from
                              <span class="inline-formula" id="inline-formula-18">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-18.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-18.gif"/>
                                </noscript>
                               </span>
                              </span>
                              to volume
                              <span class="inline-formula" id="inline-formula-19">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-19.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-19.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , and hence (equating these two rates) we recover the expected detailed balance condition:
                              <span class="disp-formula" id="disp-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <span class="inline-formula" id="inline-formula-20">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-20.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-20.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <p id="p-22">
                              Then, the total equilibrium rate of transition from mesostate i to mesostate j is obtained by integrating over all such pairs of volumes:
                              <span class="disp-formula" id="disp-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-23">
                              A similar equation applies to the reverse transition rate P
                              <sub>
                               j
                              </sub>
                              R
                              <sub>
                               ji
                              </sub>
                              . Applying
                              <a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">
                               Eq.6
                              </a>
                              thus ensures that
                              <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">
                               Eq.2
                              </a>
                              is fulfilled.
                              <a id="xref-disp-formula-8-1" class="xref-disp-formula" href="#disp-formula-8">
                               Eq.7
                              </a>
                              is useful to us in that it relates transition rates between microstates,
                              <span class="inline-formula" id="inline-formula-21">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-21.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-21.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , to the total transition rate between mesostates R
                              <sub>
                               ij
                              </sub>
                              .
                             </p>
                             <p id="p-24">
                              A special case of the above is where the spaces
                              <span class="inline-formula" id="inline-formula-22">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-22.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-22.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-23">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-23.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-23.gif"/>
                                </noscript>
                               </span>
                              </span>
                              are identical, density of states g
                              <sub>
                               i
                              </sub>
                              = g
                              <sub>
                               j
                              </sub>
                              = g, and transitions occur only between the same point in phase space, i.e.
                              <span class="inline-formula" id="inline-formula-24">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-24.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-24.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for all transitions. In this case we may write:
                              <span class="disp-formula" id="disp-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-10.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-10.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-25">
                              Then the microscopic detailed balance becomes:
                              <span class="disp-formula" id="disp-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-11.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-11.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <a id="xref-disp-formula-8-2" class="xref-disp-formula" href="#disp-formula-8">
                               Eq.7
                              </a>
                              simplifies as may be expected:
                              <span class="disp-formula" id="disp-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-12.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-12.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-26">
                              All the specific examples pursued in this paper are of this simpler form. The more general
                              <a id="xref-disp-formula-8-3" class="xref-disp-formula" href="#disp-formula-8">
                               Eq.7
                              </a>
                              becomes necessary if transitions occur between different points of phase space within mesostates i and j, or if the phase spaces of i and j are not identical. The former may occur, for example, in models where a dynamic transition through phase space is explicitly defined as a part of a kinetic transition, and the latter in coarse grained modelling where the two mesostates are represented by different coarse grained models.
                             </p>
                             <p id="p-27">
                              The above equations are extremely general. They relate forward and reverse transition rates (via detailed balance), and mesoscopic transition rates to microsopic transition rates. These equations place constraints on what is “allowed” in a model. However, they are not sufficient to fully constrain a model: one needs to make further physical assumptions. In what follows, we explore some possible assumptions that are consistent with the preceding work.
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Candidate microscopic models
                             </h3>
                             <p id="p-28">
                              We find it convenient to separate the energy of a microstate as
                              <span class="inline-formula" id="inline-formula-25">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-25.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-25.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , where
                              <span class="inline-formula" id="inline-formula-26">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-26.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-26.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the “mechanical” energy used to compute dynamics of
                              <span class="inline-formula" id="inline-formula-27">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-27.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-27.gif"/>
                                </noscript>
                               </span>
                              </span>
                              within mesostate i, and
                              <span class="inline-formula" id="inline-formula-28">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-28.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-28.gif"/>
                                </noscript>
                               </span>
                              </span>
                              represents all other contributions to the (free) energy of mesostate i that are independent of
                              <span class="inline-formula" id="inline-formula-29">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-29.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-29.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Correspondingly, energy differences can be written as
                              <span class="inline-formula" id="inline-formula-30">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-30.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-30.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <p id="p-29">
                              This then allows us to write
                              <a id="xref-disp-formula-10-1" class="xref-disp-formula" href="#disp-formula-10">
                               Eq.8
                              </a>
                              in the form
                              <span class="disp-formula" id="disp-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-13.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-13.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where the ‘intrinsic’ rates
                              <span class="inline-formula" id="inline-formula-31">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-31.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-31.gif"/>
                                </noscript>
                               </span>
                              </span>
                              are related as
                              <span class="disp-formula" id="disp-formula-13">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-14.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-14.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-30">
                              We note that these intrinsic rates are not necessarily independent of
                              <span class="inline-formula" id="inline-formula-32">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-32.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-32.gif"/>
                                </noscript>
                               </span>
                              </span>
                              : it may be, for example, that transitions are only possible within some region of phase space. However, their ratio is independent of
                              <span class="inline-formula" id="inline-formula-33">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-33.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-33.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . For the purposes of this work we will assume them to be constant, but recognise that so long as
                              <a id="xref-disp-formula-13-1" class="xref-disp-formula" href="#disp-formula-13">
                               Eq.11
                              </a>
                              is satisfied, they can take any functional form.
                             </p>
                             <p>
                              We now indicate two candidate forms of model that are consistent with
                              <a id="xref-disp-formula-12-1" class="xref-disp-formula" href="#disp-formula-12">
                               Eq.10
                              </a>
                              .
                             </p>
                             <ol class="list-ord" id="list-1">
                              <li id="list-item-1">
                               <p id="p-32">
                                <strong>
                                 Mechanical energy difference model.
                                </strong>
                                With insight from the work of Sarlah and Vilfan [
                                <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">
                                 23
                                </a>
                                ,
                                <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">
                                 24
                                </a>
                                ], we separate the detailed balance condition of
                                <a id="xref-disp-formula-12-2" class="xref-disp-formula" href="#disp-formula-12">
                                 Eq.10
                                </a>
                                into individual rates as
                                <span class="disp-formula" id="disp-formula-14">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-15.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-15.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-33">
                                The factor
                                <span class="inline-formula" id="inline-formula-34">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-34.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-34.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                indicates the fraction of the mechanical energy difference
                                <span class="inline-formula" id="inline-formula-35">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-35.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-35.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                that influences the forward transition rate
                                <span class="inline-formula" id="inline-formula-36">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-36.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-36.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , the remainder being attributed to the reverse transition.
                                <a id="xref-disp-formula-12-3" class="xref-disp-formula" href="#disp-formula-12">
                                 Eq.10
                                </a>
                                then requires the constraint
                                <span class="inline-formula" id="inline-formula-37">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-37.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-37.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                . The implicit assumption is of an energy barrier to the transition, located at some position along the reaction co-ordinate between mesostates i and j. The mechanical energy alters the height of that barrier in an Arrhenius-like fashion.
                                <span class="inline-formula" id="inline-formula-38">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-38.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-38.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                suggests the barrier is located close to state j along the reaction co-ordinate, whilst
                                <span class="inline-formula" id="inline-formula-39">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-39.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-39.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                indicates the barrier is close to i.
                               </p>
                               <p id="p-34">
                                We can now relate
                                <a id="xref-disp-formula-14-1" class="xref-disp-formula" href="#disp-formula-14">
                                 Eq.12
                                </a>
                                to equilibrium transition rates at the mesoscale. Substitution into
                                <a id="xref-disp-formula-11-1" class="xref-disp-formula" href="#disp-formula-11">
                                 Eq.9
                                </a>
                                , and using
                                <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">
                                 Eqs.1
                                </a>
                                and
                                <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">
                                 5
                                </a>
                                with g
                                <sub>
                                 i
                                </sub>
                                = g gives
                                <span class="disp-formula" id="disp-formula-15">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-16.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-16.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                and, with the assumption of constant
                                <span class="inline-formula" id="inline-formula-40">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-40.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-40.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , we finally obtain
                                <span class="disp-formula" id="disp-formula-16">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-17.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-17.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-35">
                                This equation determines the value of
                                <span class="inline-formula" id="inline-formula-41">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-41.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-41.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                required in the model to obtain a given value of R
                                <sub>
                                 ij
                                </sub>
                                when the system is at thermal equilibrium. So, for example, R
                                <sub>
                                 ij
                                </sub>
                                may be an experimentally observed transition rate between two mesostates. Having now determined the appropriate choice of
                                <span class="inline-formula" id="inline-formula-42">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-42.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-42.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , we can continue to use
                                <a id="xref-disp-formula-14-2" class="xref-disp-formula" href="#disp-formula-14">
                                 Eq.12
                                </a>
                                , for example in a non-equilibrium situation where an additional external force is applied to the system, which would then result in a change in the overall average transition rate.
                               </p>
                              </li>
                              <li id="list-item-2">
                               <p id="p-36">
                                <strong>
                                 Force-activated transition model.
                                </strong>
                                A reasonable assumption in some situations is to consider that the transition rate from i to j is driven by some component of a force
                                <span class="inline-formula" id="inline-formula-43">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-43.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-43.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , which depends upon the configuration
                                <span class="inline-formula" id="inline-formula-44">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-44.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-44.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                . So, for example,
                                <span class="inline-formula" id="inline-formula-45">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-45.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-45.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                could be a force acting across a bond, which may dissociate to drive a change of state (e.g. a protein unfolding). This force might (in many cases) be obtained from the gradient of the mechanical energy
                                <span class="inline-formula" id="inline-formula-46">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-46.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-46.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                with respect to one of the components of
                                <span class="inline-formula" id="inline-formula-47">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-47.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-47.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                . This distinguishes the force-activated transition from the mechanical energy difference model (where the transition rate is related only to the difference in energy between mesostate i and j rather than gradient of the mechanical energy within a mesostate). Models explicitly considering external force have long been used ubiquitously, most relevant to us in the case of protein unfolding [
                                <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                                 31
                                </a>
                                ,
                                <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                                 32
                                </a>
                                ]. However, these models are often used in isolation at the free-energy (mesostate) level, whereas we show that external forces can be explicitly included at the mechanical (microstate) level in the same manner.
                               </p>
                               <p id="p-37">
                                If the critical extension of the bond for dissociation is a small distance b, where b = b
                                <sub>
                                 ij
                                </sub>
                                = b
                                <sub>
                                 ji
                                </sub>
                                , the energy barrier towards the transition is then reduced by
                                <span class="inline-formula" id="inline-formula-48">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-48.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-48.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                giving transition rate:
                                <span class="disp-formula" id="disp-formula-17">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-18.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-18.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-38">
                                It may be that the rate of the reverse transition is negligible and can be ignored. However, if re-association is to be considered a possibility then
                                <a id="xref-disp-formula-12-4" class="xref-disp-formula" href="#disp-formula-12">
                                 Eq.10
                                </a>
                                requires that:
                                <span class="disp-formula" id="disp-formula-18">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-19.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-19.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-39">
                                We can again relate these microscopic transition rates to equilibrium transition rates at the mesoscale. Substitution into
                                <a id="xref-disp-formula-11-2" class="xref-disp-formula" href="#disp-formula-11">
                                 Eq.9
                                </a>
                                , and using
                                <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">
                                 Eqs.1
                                </a>
                                and
                                <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">
                                 5
                                </a>
                                with g
                                <sub>
                                 i
                                </sub>
                                = g gives, for the forward rates:
                                <span class="disp-formula" id="disp-formula-19">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-20.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-20.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-40">
                                However, for the reverse rates:
                                <span class="disp-formula" id="disp-formula-20">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-21.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-21.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-41">
                                In an unbinding transition, for which this model is most appropriate, it would typically be the case that mesostate i (representing bound state) would be mechanically stiffer than mesostate j. Indeed, many proteins are held in their folded conformation by strong disulphide bonds in a variety of configurations [
                                <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                                 33
                                </a>
                                ], and it may be the extension of these bond complexes that lead to the value b
                                <sub>
                                 ij
                                </sub>
                                . Hence, integrals over
                                <span class="inline-formula" id="inline-formula-49">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-49.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-49.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                are dominated by relatively small region of phase space V
                                <sub>
                                 i
                                </sub>
                                , whilst integrals over
                                <span class="inline-formula" id="inline-formula-50">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-50.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-50.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                are dominated by a larger region of phase space V
                                <sub>
                                 j
                                </sub>
                                ≫ V
                                <sub>
                                 i
                                </sub>
                                . Thus, the reverse transition rate in
                                <a id="xref-disp-formula-20-1" class="xref-disp-formula" href="#disp-formula-20">
                                 Eq. 18
                                </a>
                                contains an implicit factor of V
                                <sub>
                                 i
                                </sub>
                                /V
                                <sub>
                                 j
                                </sub>
                                , representing the fact that rebinding can only occur from state j to state i if the system is found in the correct region of phase space, close to the favourable region for state i. If V
                                <sub>
                                 j
                                </sub>
                                ≫ V
                                <sub>
                                 i
                                </sub>
                                then such occasions will be rare, through random thermal fluctuation.
                               </p>
                               <p id="p-42">
                                We note here that the factor
                                <span class="inline-formula" id="inline-formula-51">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-51.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-51.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                occurring in these equations is a linearised energy contribution, which will typically only be appropriate in the region V
                                <sub>
                                 i
                                </sub>
                                of phase space where mesostate i undergoes typical fluctuations. It may be that it is appropriate to set
                                <span class="inline-formula" id="inline-formula-52">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-52.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-52.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                and
                                <span class="inline-formula" id="inline-formula-53">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-53.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-53.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                to zero outside this region (and taking it back inside the integrals), so as to prevent the factor
                                <span class="inline-formula" id="inline-formula-54">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-54.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-54.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                in
                                <a id="xref-disp-formula-19-1" class="xref-disp-formula" href="#disp-formula-19">
                                 equations 17
                                </a>
                                and
                                <a id="xref-disp-formula-20-2" class="xref-disp-formula" href="#disp-formula-20">
                                 18
                                </a>
                                from creating spurious, unphysical effects.
                               </p>
                              </li>
                             </ol>
                             <p id="p-43">
                              The following section contains various examples which show the capabilities of the method, and hint at its generality and wide field of potential applicability.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-8">
                            <h2 class="">
                             Results
                            </h2>
                            <p id="p-44">
                             <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                              Fig.2
                             </a>
                             shows the various systems investigated in this section. Before discussion of those, however, we begin by showing how
                             <a id="xref-disp-formula-16-1" class="xref-disp-formula" href="#disp-formula-16">
                              Eq.14
                             </a>
                             can be calculated for a specific system, and how
                             <a id="xref-disp-formula-14-3" class="xref-disp-formula" href="#disp-formula-14">
                              Eq.12
                             </a>
                             is applied as part of a dynamic simulation protocol. This is done using the mechanical energy difference model, although an equivalent set of calculations could be performed for the force-activated transition model using
                             <a id="xref-disp-formula-17-1" class="xref-disp-formula" href="#disp-formula-17">
                              Eqs.15
                             </a>
                             –
                             <a id="xref-disp-formula-20-3" class="xref-disp-formula" href="#disp-formula-20">
                              18
                             </a>
                             .
                            </p>
                            <div id="F2" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F2.large.jpg?width=800&height=600&carousel=1" title="The structural models in each of our example systems. a) Four mesostates, defined by their different equilibrium lengths and spring constants and coloured to correspond to the associated mesostate graphs throughout this article. Here there is only a single transition pathway from mesostate 1 to mesostate 4, representing a process akin to (but distinct from) a reaction co-ordinate. b) Similar systems to a), but this time every possible kinetic transition is allowed. c) By representing “proteins” as Hookean springs which can transition to springs with a modified finitely extensible non-linear elastic (FENE) potential, i.e. a polypeptide chain [32], we obtain a representation of protein unfolding by mimicking SMFS-AFM experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-843933992" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">The structural models in each of our example systems. <strong>a)</strong> Four mesostates, defined by their different equilibrium lengths and spring constants and coloured to correspond to the associated mesostate graphs throughout this article. Here there is only a single transition pathway from mesostate 1 to mesostate 4, representing a process akin to (but distinct from) a reaction co-ordinate. <strong>b)</strong> Similar systems to <strong>a)</strong>, but this time every possible kinetic transition is allowed. <strong>c)</strong> By representing “proteins” as Hookean springs which can transition to springs with a modified finitely extensible non-linear elastic (FENE) potential, i.e. a polypeptide chain [32], we obtain a representation of protein unfolding by mimicking SMFS-AFM experiments.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Fig 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F2.medium.gif" width="440" height="217"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Fig 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F2.medium.gif" width="440" height="217"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 2." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Fig 2.
                              </span>
                              <p id="p-45" class="first-child">
                               The structural models in each of our example systems.
                               <strong>
                                a)
                               </strong>
                               Four mesostates, defined by their different equilibrium lengths and spring constants and coloured to correspond to the associated mesostate graphs throughout this article. Here there is only a single transition pathway from mesostate 1 to mesostate 4, representing a process akin to (but distinct from) a reaction co-ordinate.
                               <strong>
                                b)
                               </strong>
                               Similar systems to
                               <strong>
                                a)
                               </strong>
                               , but this time every possible kinetic transition is allowed.
                               <strong>
                                c)
                               </strong>
                               By representing “proteins” as Hookean springs which can transition to springs with a modified finitely extensible non-linear elastic (FENE) potential, i.e. a polypeptide chain [
                               <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">
                                32
                               </a>
                               ], we obtain a representation of protein unfolding by mimicking SMFS-AFM experiments.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             Applying the mechano-kinetic framework
                            </h2>
                            <p id="p-46">
                             To investigate the capabilities of the model, we begin with a simple 1D dumbbell model; two beads connected by a Hookean spring with spring constant k
                             <sub>
                              i
                             </sub>
                             and equilibrium length l
                             <sub>
                              i
                             </sub>
                             . The dynamic simulation framework we use is a Brownian dynamics protocol, numerical integration of the Langevin equation where inertia, and hence velocity degrees of freedom, are neglected. For the two dumbbell coordinates x
                             <sub>
                              1
                             </sub>
                             and x
                             <sub>
                              2
                             </sub>
                             , the Langevin equations are
                             <span class="disp-formula" id="disp-formula-21">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-23.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-23.gif"/>
                               </noscript>
                              </span>
                             </span>
                             <span class="disp-formula" id="disp-formula-22">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-24.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-24.gif"/>
                               </noscript>
                              </span>
                             </span>
                            </p>
                            <p id="p-47">
                             Each bead is subject to a Stokes drag λ and an associated stochastic thermal noise
                             <span class="inline-formula" id="inline-formula-55">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-55.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-55.gif"/>
                               </noscript>
                              </span>
                             </span>
                             calculated via the fluctuation-dissipation theorem, with associated statistics
                             <span class="disp-formula" id="disp-formula-23">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-25.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-25.gif"/>
                               </noscript>
                              </span>
                             </span>
                             <span class="disp-formula" id="disp-formula-24">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-26.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-26.gif"/>
                               </noscript>
                              </span>
                             </span>
                            </p>
                            <p id="p-48">
                             The corresponding mechanical energy for the system is given by
                             <span class="disp-formula" id="disp-formula-25">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-27.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-27.gif"/>
                               </noscript>
                              </span>
                             </span>
                             where x = x
                             <sub>
                              2
                             </sub>
                             − x
                             <sub>
                              1
                             </sub>
                             defines the single elastic degree of freedom within the system.
                            </p>
                            <p id="p-49">
                             The kinetic simulation component utilises the sets of internal spring parameters, k
                             <sub>
                              i
                             </sub>
                             and l
                             <sub>
                              i
                             </sub>
                             . These two parameters define the mesostate, i.e. k
                             <sub>
                              i
                             </sub>
                             and l
                             <sub>
                              i
                             </sub>
                             are the parameters for mesostate i, and will vary between mesostates. When altered, these parameters change the energy landscape independently of the location of the dumbbell nodes within phase space, setting x = x′ in
                             <a id="xref-disp-formula-16-2" class="xref-disp-formula" href="#disp-formula-16">
                              Eq.14
                             </a>
                             . In principle all parameters (e.g. the drag) could vary between mesostates, but for clarity we consider only kinetic alterations to the elastic energy landscape. In the simulations reported below, we set λ = 6πpN.ns/nm and remains constant across all mesostates.
                            </p>
                            <p id="p-50">
                             With these independent definitions of the dynamic and kinetic simulation components, we can now couple them together. For this 1D system and in the absence of intertia, the mechanical energy term within
                             <a id="xref-disp-formula-16-3" class="xref-disp-formula" href="#disp-formula-16">
                              Eq.14
                             </a>
                             is greatly simplified. Introducing
                             <a id="xref-disp-formula-25-1" class="xref-disp-formula" href="#disp-formula-25">
                              Eq.23
                             </a>
                             into
                             <a id="xref-disp-formula-16-4" class="xref-disp-formula" href="#disp-formula-16">
                              Eq.14
                             </a>
                             and expressing the denominator integral exponent as a polynomial in x gives only Gaussian integrals in x and so can be solved exactly, yielding
                             <span class="disp-formula" id="disp-formula-26">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-28.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-28.gif"/>
                               </noscript>
                              </span>
                             </span>
                            </p>
                            <p id="p-51">
                             <a id="xref-disp-formula-26-1" class="xref-disp-formula" href="#disp-formula-26">
                              Eq.24
                             </a>
                             completes the coupling between dynamic and kinetic simulation components within the mechano-kinetic framework. From a set of R
                             <sub>
                              ij
                             </sub>
                             together with a dynamic model, we can calculate the set
                             <span class="inline-formula" id="inline-formula-56">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-56.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-56.gif"/>
                               </noscript>
                              </span>
                             </span>
                             . Now, following each dynamic simulation timestep, we include the probability of a mesostate transition as per
                             <a id="xref-disp-formula-14-4" class="xref-disp-formula" href="#disp-formula-14">
                              Eq.12
                             </a>
                             , where we convert the rates to transition probabilities by multiplying by the simulation timestep.
                             <a id="xref-disp-formula-14-5" class="xref-disp-formula" href="#disp-formula-14">
                              Eq.12
                             </a>
                             therefore makes these transition probabilities explicitly dependent on the current microstate mechanical energy. If a transition occurs, we keep the two node positions constant, change the spring parameters, and allow the dynamic simulation to continue. If at any point a probability exceeds 1, we stop the simulation and lower the timestep to allow the correct resolution of kinetic events.
                            </p>
                            <p id="p-52">
                             The introduction of instantaneous kinetic transitions within a dynamic simulation introduces the potential for non-convergence of statistical properties. Physical validation of each simulation component is provided as Supporting Information.
                            </p>
                            <div id="sec-10" class="subsection">
                             <h3>
                              A four-state system
                             </h3>
                             <p id="p-53">
                              To investigate the behaviour of the model when applied to a molecule undergoing conformational changes, we have designed a series of kinetic transitions using the dumbbell model corresponding to a four state system, in which the average length of the dumbbell increases throught the states. We define two alternative systems of this set of kinetic mesostates, given by their mechanical parameterisations in
                              <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              , together with an associated equilibrium kinetic transition rate matrix corresponding to both systems given in
                              <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              . The schematic of this system is shown in
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig.2a
                              </a>
                              .
                             </p>
                             <div id="T1" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2462559/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2462559/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2462559" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 1.
                               </span>
                               <p id="p-54" class="first-child">
                                The two families of material parameters used to define the mesostates in the four-state kinetic networks of the dumbbell model. The first (left) set contains equal and relatively low values of k
                                <sub>
                                 i
                                </sub>
                                whereas the second (right) set of k
                                <sub>
                                 i
                                </sub>
                                are much higher. The values of l
                                <sub>
                                 i
                                </sub>
                                linearly increase with the mesostate index.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="T2" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2462553/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2462553/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2462553" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 2.
                               </span>
                               <p id="p-55" class="first-child">
                                The average transition rates between mesostates used in the four-state kinetic networks of the dumbbell model. The mesostate occupation probabilities have been analytically calculated for comparison with simulation.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-56">
                              Since
                              <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              defines all of the expected transition rates, which in principle may have been experimentally measured, it also implicitly defines the mesoscale free energy landscape and all of the resultant occupation probabilities. Thus our investigation will tell us nothing new at the mesoscale level at this stage. Instead, our aim is to analyse the modifications to the kinetic rates made by the continuous underlying dynamics, and verify that the measured, average kinetic behaviour is reproduced. Note that many of the transition rates shown in
                              <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              are zero, indicating that direct transitions cannot occur between these states. However, transitions via other intermediate states can still occur (i.e. from mesostate 1 to mesostate 3 via mesostate 2).
                             </p>
                             <p id="p-57">
                              The energy landscapes implied by this kinetic system are illustrated in
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Fig.3
                              </a>
                              . We see in
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Fig.3a
                              </a>
                              ) that as all of the stiffness parameters k
                              <sub>
                               i
                              </sub>
                              are equal in the first parameterisation, no particular state is preferable over long periods of time with respect to mechanical energy. On the other hand, in the second parameterisation we include two high stiffness states (k
                              <sub>
                               1
                              </sub>
                              and k
                              <sub>
                               4
                              </sub>
                              ) and two low stiffness states (k
                              <sub>
                               2
                              </sub>
                              and k
                              <sub>
                               3
                              </sub>
                              ). However, irrespective of the mechanical energy environment, the transition rates in
                              <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              imply certain characteristics of the free energy landscape along a “pseudo-reaction co-ordinate” between mesostates. If R
                              <sub>
                               ij
                              </sub>
                              > R
                              <sub>
                               ji
                              </sub>
                              , then this implies (via
                              <a id="xref-disp-formula-1-3" class="xref-disp-formula" href="#disp-formula-1">
                               Eqs.1
                              </a>
                              and
                              <a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">
                               2
                              </a>
                              ) that the free energy of state i is higher than state j. Smaller overall values of R
                              <sub>
                               ij
                              </sub>
                              and R
                              <sub>
                               ji
                              </sub>
                              typically imply larger energy barriers between the two states. We have set
                              <span class="inline-formula" id="inline-formula-57">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-57.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-57.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for all i, j in the simulations, which implies the energy barrier lies half-way between the two states along the reaction co-ordinate. Given this, the transition rates in
                              <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              imply a one-dimensional reaction co-ordinate of the form shown in
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Fig.3
                              </a>
                              . We emphasise that this “pseudo-reaction co-ordinate” is not the extension x of the dumbbell, but rather some underlying reaction variable along which lie our implicitly coarse-grained kinetic states.
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F3.large.jpg?width=800&height=600&carousel=1" title="The two “levels” of energy landscape within the four-state kinetic networks of the dumbbell model. a) The mechanical energy landscapes given in Table 1, where the occupation of each state is given a different colour. Top: Set 1 - All values of ki are equal. Bottom: Set 2 - Values of ki vary between mesostates. b) A possible representation of the free energy as a function of some hypothetical “pseudo-reaction co-ordinate”, with barriers implied by the rates given in Table 2. The “center” of each mesostate is shown as a vertical dashed line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-843933992" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">The two “levels” of energy landscape within the four-state kinetic networks of the dumbbell model. a) The mechanical energy landscapes given in Table 1, where the occupation of each state is given a different colour. Top: Set 1 - All values of <em>k<sub>i</sub></em> are equal. Bottom: Set 2 - Values of <em>k<sub>i</sub></em> vary between mesostates. b) A possible representation of the free energy as a function of some hypothetical “pseudo-reaction co-ordinate”, with barriers implied by the rates given in Table 2. The “center” of each mesostate is shown as a vertical dashed line.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F3.medium.gif" width="440" height="186"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F3.medium.gif" width="440" height="186"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig 3.
                               </span>
                               <p id="p-58" class="first-child">
                                The two “levels” of energy landscape within the four-state kinetic networks of the dumbbell model. a) The mechanical energy landscapes given in
                                <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">
                                 Table 1
                                </a>
                                , where the occupation of each state is given a different colour. Top: Set 1 - All values of k
                                <sub>
                                 i
                                </sub>
                                are equal. Bottom: Set 2 - Values of k
                                <sub>
                                 i
                                </sub>
                                vary between mesostates. b) A possible representation of the free energy as a function of some hypothetical “pseudo-reaction co-ordinate”, with barriers implied by the rates given in
                                <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">
                                 Table 2
                                </a>
                                . The “center” of each mesostate is shown as a vertical dashed line.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="sec-11" class="subsection">
                              <h4>
                               Four state system - Results
                              </h4>
                              <p id="p-59">
                               For each mechanical simulation parameterisation,
                               <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">
                                Table 3
                               </a>
                               shows the rates
                               <span class="inline-formula" id="inline-formula-58">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-58.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-58.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               which are modified by exploration of the mechanical energy landscape to obtain the R
                               <sub>
                                ij
                               </sub>
                               values shown in
                               <a id="xref-table-wrap-2-7" class="xref-table" href="#T2">
                                Table 2
                               </a>
                               . Let us briefly investigate some of the properties of these values. For the set of
                               <span class="inline-formula" id="inline-formula-59">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-59.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-59.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where all the k
                               <sub>
                                i
                               </sub>
                               are equal for the four mesostates, each
                               <span class="inline-formula" id="inline-formula-60">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-60.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-60.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is increased from its corresponding R
                               <sub>
                                ij
                               </sub>
                               value by the same multiplicative factor. This is because the mechanical energy environments for each state are identical, albeit translated in space, and so with the application of
                               <a id="xref-disp-formula-26-2" class="xref-disp-formula" href="#disp-formula-26">
                                Eq.24
                               </a>
                               , all integral results are the same for every state pair. The presence of a mechanical energy landscape here, then, acts as a universal barrier to kinetic transition, such that an extra amount of energy is required to make the kinetic transition. For the other set, corresponding to the mesostates with differing k
                               <sub>
                                i
                               </sub>
                               values, we see a similar effect for
                               <span class="inline-formula" id="inline-formula-61">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-61.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-61.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-62">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-62.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-62.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               which, having the same mechanical energy landscape, are increased from their corresponding mesoscopic values by the same factor. The increase here is greater than in the previous (equal k
                               <sub>
                                i
                               </sub>
                               ) network, as the values of k
                               <sub>
                                i
                               </sub>
                               are significantly higher. Importantly, because the values are increased by the same factor, the detailed balance condition between the pair of states is unchanged by the presence of a mechanical energy landscape and thus, while that rates are slower, the contributions to the equilibrium probabilities will be the same. However, this is not the case for any of the other pairs of rates in this network. As the mechanical energy environment changes between mesostates 1 & 2, and 3 & 4, the relative increase of
                               <span class="inline-formula" id="inline-formula-63">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-63.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-63.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               from the associated R
                               <sub>
                                ij
                               </sub>
                               value is different, showing that the detailed balance condition of the
                               <span class="inline-formula" id="inline-formula-64">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-64.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-64.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               values are not the same as the mesoscopic R
                               <sub>
                                ij
                               </sub>
                               values. In other words, dynamic motion through a mechanical energy landscape, in the general case, modifies the detailed balance conditions between the pre-defined mesostates in our model. In reality the reverse is true, in that the presence of a mechanical energy landscape alters the implicit detailed balance conditions between microstates, leading to a different detailed balance condition at the free energy level.
                              </p>
                              <div id="T3" class="table pos-float">
                               <div class="table-inline table-callout-links">
                                <div class="callout">
                                 <span>
                                  View this table:
                                 </span>
                                 <ul class="callout-links">
                                  <li class="view-inline first">
                                   <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2462556/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                    View inline
                                   </a>
                                  </li>
                                  <li class="view-popup">
                                   <a href="/highwire/markup/2462556/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                    View popup
                                   </a>
                                  </li>
                                  <li class="download-ppt last">
                                   <a href="/highwire/powerpoint/2462556" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                    Download powerpoint
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                               </div>
                               <div class="table-caption">
                                <span class="table-label">
                                 Table 3.
                                </span>
                                <p id="p-60" class="first-child">
                                 The instantaneous transition rates between mesostates used in the four-state kinetic networks of the dumbbell model corresponding to the two sets of mesostates given in
                                 <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">
                                  Table 1
                                 </a>
                                 . Top: Equal, low k
                                 <sub>
                                  i
                                 </sub>
                                 mesostates. Bottom: Differing, large k
                                 <sub>
                                  i
                                 </sub>
                                 values.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-61">
                               Although the
                               <span class="inline-formula" id="inline-formula-65">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-65.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-65.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               values calculated and used in each simulation are different,
                               <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">
                                Table 4
                               </a>
                               shows the equilibrium mesostate occupation probabilities for both sets of mechanical material parameterisations, calculated by averaging the expected occupation probabilities from a single, long-time simulation over many repeats beginning in different initial states. We can see that the mesostate occupation probabilities for the two simulations are equivalent to the analytically calculated values given in
                               <a id="xref-table-wrap-2-8" class="xref-table" href="#T2">
                                Table 2
                               </a>
                               , despite the large differences in the values of k
                               <sub>
                                i
                               </sub>
                               . This clearly shows that the mesostate occupation probabilities within our model do indeed remain constant as required, being fixed by the pre-defined R
                               <sub>
                                ij
                               </sub>
                               values, whilst the underlying mechanical energy landscape is modified.
                              </p>
                              <div id="T4" class="table pos-float">
                               <div class="table-inline table-callout-links">
                                <div class="callout">
                                 <span>
                                  View this table:
                                 </span>
                                 <ul class="callout-links">
                                  <li class="view-inline first">
                                   <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2462564/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                    View inline
                                   </a>
                                  </li>
                                  <li class="view-popup">
                                   <a href="/highwire/markup/2462564/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                    View popup
                                   </a>
                                  </li>
                                  <li class="download-ppt last">
                                   <a href="/highwire/powerpoint/2462564" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                    Download powerpoint
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                               </div>
                               <div class="table-caption">
                                <span class="table-label">
                                 Table 4.
                                </span>
                                <p id="p-62" class="first-child">
                                 The final expected mesostate occupation probabilities within the four-state kinetic networks of the dumbbell model, which accurately reproduce the analytically calculated rates given in
                                 <a id="xref-table-wrap-2-9" class="xref-table" href="#T2">
                                  Table 2
                                 </a>
                                 . Each set corresponds to a different set of material parameters as defined in
                                 <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">
                                  Table 1
                                 </a>
                                 . Uncertainties in the last digit are shown in brackets.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-63">
                               <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                                Fig.4
                               </a>
                               shows the probability density of length fluctuations at equilibrium from a single simulation of each set of mesostates. We see that whilst the kinetic rate modifications within the model take the mechanical energies into account in order to return the average rates R
                               <sub>
                                ij
                               </sub>
                               , the mechanical energy landscape is still fully realised within the framework. In particular, we see that the low mechanical stiffness of individual states in
                               <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                                Fig.4a
                               </a>
                               ) means that length fluctuations within each state are relatively large compared to the associated equilibrium lengths, resulting in a unimodal (albeit skewed) overall distribution centred around, but not exactly on, the most likely mesostate. Within a standard Markov state model, each of these states would likely be indistinguishable from one another, yet our implied knowledge of both the mesostate kinetics and mechanical energy landscape allows us to include them anyway, such that their indistinguishability is effectively a deduced property. On the other hand, for the second parameterisation in
                               <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                                Fig.4b
                               </a>
                               ), the higher mechanical stiffness suppresses the length fluctuations within each mesostate so that the individual mesostates are distinguishable in the overall length distributions, with clear Gaussian distributions centered on each equilibrium length. This is less apparent for mesostates 1 and 2, however, as their associated free energies are significantly higher, and hence their occupation probabilities lower.
                              </p>
                              <div id="F4" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F4.large.jpg?width=800&height=600&carousel=1" title="The microstate probability densities from simulation of the four-state kinetic networks of the dumbbell model. The mechanical equilibrium positions corresponding to each state are shown as dashed green lines. a) Mesostate set 1 b) Mesostate set 2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-843933992" data-figure-caption='<div class="highwire-markup">The microstate probability densities from simulation of the four-state kinetic networks of the dumbbell model. The mechanical equilibrium positions corresponding to each state are shown as dashed green lines. a) Mesostate set 1 b) Mesostate set 2.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Fig 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F4.medium.gif" width="440" height="209"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Fig 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F4.medium.gif" width="440" height="209"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 4." data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Fig 4.
                                </span>
                                <p id="p-64" class="first-child">
                                 The microstate probability densities from simulation of the four-state kinetic networks of the dumbbell model. The mechanical equilibrium positions corresponding to each state are shown as dashed green lines. a) Mesostate set 1 b) Mesostate set 2.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-65">
                               Overall we see that within the mechano-kinetic framework, no matter the structure of the mechanical energy landscape, the introduction of
                               <a id="xref-disp-formula-26-3" class="xref-disp-formula" href="#disp-formula-26">
                                Eq.24
                               </a>
                               into
                               <a id="xref-disp-formula-14-6" class="xref-disp-formula" href="#disp-formula-14">
                                Eq.12
                               </a>
                               enables the simulation to reproduce the R
                               <sub>
                                ij
                               </sub>
                               values used as input parameters. As such, we have used the measured kinetic rates given in
                               <a id="xref-table-wrap-2-10" class="xref-table" href="#T2">
                                Table 2
                               </a>
                               together with the spring parameters in
                               <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">
                                Table 1
                               </a>
                               to infer the specific form of the mechanical energy landscape. However, we will see in the next section that we can go further than the simple reproduction of existing measurements, and consider how direct external modifications to the mechanical energy landscape affect the resultant kinetic behaviour.
                              </p>
                             </div>
                            </div>
                            <div id="sec-12" class="subsection">
                             <h3>
                              External Forces
                             </h3>
                             <p id="p-66">
                              We now investigate how this framework behaves when an external potential is added to the equilibrium system. As previously discussed, we may wish to determine the effect of applying external mechanical forces to biological objects such as proteins or molecular motors. Consider the case where either experiments or high-resolution simulations have been performed and both the free energy and mechanical energy landscapes of the object in isolation are well known. For the case of dynein, the illustrative motor protein discussed in the introduction, both mechanical [
                              <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ,
                              <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                               35
                              </a>
                              ] and kinetic [
                              <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                              ,
                              <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                               36
                              </a>
                              ] information is available to parameterise the system. For proteins and unfolding in general, Hughes et al. provide a detailed review [
                              <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">
                               32
                              </a>
                              ]. Using this information, we can use the mechano-kinetic simulation framework to make predictions based on these experimental systems.
                             </p>
                             <p id="p-67">
                              To explore this type of problem in a general sense, we have designed two kinetic networks, both of which use the dumbbell model together with set of mesostates defined in
                              <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">
                               Table 5
                              </a>
                              . The first network is similar to the previous four-state example, whereas the second network is a fully connected kinetic network, where direct transitions between all states are possible. The schematic of this second system is shown in
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Fig.2d
                              </a>
                              . The associated transition rates defining these networks are given in
                              <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">
                               Table 6
                              </a>
                              , top and bottom tables respectively. In contrast to the previous unforced four-state example, we here observe that because all of the rates are the same, when unperturbed, this system initially ought to have an equal probability of being in each of the mesostates. However, under the subsequent application of external forces we would expect the forces, as a source of mechanical energy, to bias the states in some manner. Within the dynamic simulation component, we apply a constant external force f
                              <sup>
                               ex
                              </sup>
                              to both nodes of the dumbbell but in opposite directions, therefore biasing the dumbbell towards higher extensions. To clarify, we use the values of
                              <span class="inline-formula" id="inline-formula-66">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-66.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-66.gif"/>
                                </noscript>
                               </span>
                              </span>
                              as given at equilibrium in the absence of external forces, and modify the reaction rates via
                              <a id="xref-disp-formula-14-7" class="xref-disp-formula" href="#disp-formula-14">
                               Eq.12
                              </a>
                              to account for additional contributions to the mechanical energy from the external forces. With this approach we are able to use the mechano-kinetic model to predict not only how the application of this external force within the dynamical simulation affects the overall kinetic transition rates between mesostates, but also determine where those transitions are likely to occur in the mechanical energy landscape.
                             </p>
                             <div id="T5" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2462563/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2462563/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2462563" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 5.
                               </span>
                               <p id="p-68" class="first-child">
                                The material parameters used to define the mesostates used in the kinetic network of the dumbbell model with external forces applied.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="T6" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2462569/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2462569/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2462569" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 6.
                               </span>
                               <p id="p-69" class="first-child">
                                The matrices defining the two sets of average transition rates between mesostates, together with the expected mesostate occupation probabilities, used in the kinetic network of the dumbbell model with external forces applied. Top: A single path through the mesostate network. Bottom: A fully connected kinetic network. These rates were chosen such that the total rate of leaving any given mesostate is equivalent to that in the single pathway model.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="sec-13" class="subsection">
                              <h4>
                               External Forces - Results
                              </h4>
                              <p id="p-70">
                               <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                                Fig.5
                               </a>
                               shows the steady state occupation probabilities of each mesostate as a function of applied force, for both sets of rates. These were calculated by averaging the expected occupation probabilities from a single, long-time simulation over three repeats, with each simulation beginning in mesostate 1 at mechanical equilibrium. Although all of the values of k
                               <sub>
                                i
                               </sub>
                               in
                               <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">
                                Table 5
                               </a>
                               and initial transition rates in
                               <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">
                                Table 6
                               </a>
                               are equal, indicating no native energetic preference for any of the mesostates (as can be seen when zero external force is applied), the application of force immediately biases the system towards larger extensions. We also see from
                               <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                                Fig.5b
                               </a>
                               ) that the single pathway and fully connected networks are almost identical at the kinetic scale. For each mesostate in ascending order, the root-mean-square deviations of the residuals of one graph with respect to the other are 0.006, 0.004, 0.006 and 0.006 respectively. This indicates that the steady-state occupation probabilities as a function of applied force are independent of the connectivity of the kinetic network, given simply by the Boltzmann distribution modified to account for the extra energy contribution from the externally applied force. Detailed balance implies that this change is brought about within the model via a change in the resulting values of R
                               <sub>
                                ij
                               </sub>
                               , but key is that this is propagated from the dynamic component of the model into the kinetic component.
                              </p>
                              <div id="F5" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F5.large.jpg?width=800&height=600&carousel=1" title="The expected occupation probability of each mesostate of the dumbbell model as a function of applied force. a) The single pathway network. b) The difference between the single pathway and fully connected networks. Note that, as expected, these traces are therefore identical for the two networks." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-843933992" data-figure-caption='<div class="highwire-markup">The expected occupation probability of each mesostate of the dumbbell model as a function of applied force. a) The single pathway network. b) The difference between the single pathway and fully connected networks. Note that, as expected, these traces are therefore identical for the two networks.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Fig 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F5.medium.gif" width="440" height="211"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Fig 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F5.medium.gif" width="440" height="211"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 5." data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Fig 5.
                                </span>
                                <p id="p-71" class="first-child">
                                 The expected occupation probability of each mesostate of the dumbbell model as a function of applied force. a) The single pathway network. b) The difference between the single pathway and fully connected networks. Note that, as expected, these traces are therefore identical for the two networks.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-72">
                               The application of force across all mesostates effectively generates an additional contribution to the free energy. As such, these forces can be included in pure kinetic models as free energies,
                               <span class="inline-formula" id="inline-formula-67">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-67.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-67.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , using a term such as
                               <span class="inline-formula" id="inline-formula-68">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-68.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-68.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where Δx
                               <sub>
                                ij
                               </sub>
                               is some approximation of the expected extension. This is similar to the force-activated transition model, except that the force is introduced at the kinetic (free-energy) level. However, with the explicit inclusion of the mechanical energy environment, we can observe the subtleties occurring at the microscale.
                               <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                                Fig.6
                               </a>
                               shows the probability distributions, in the single pathway system, of dumbbell extension for zero external force and 10pN respectively, separated by mesostate. Overlapping these in a lighter colour are the distributions of dumbbell extension at the moment of transition into another mesostate. We see Gaussian distributions in the extensions, indicating thermodynamic equilibrium and reinforcing the results of
                               <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                                Fig.5
                               </a>
                               . In the
                               <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                                Fig.6a
                               </a>
                               ) transition distributions, we see either single or bimodal overlapping Gaussian distributions centred on the the half-way points between the each neighbouring pair of states, a direct consequence of the fact that the difference between two quadratic energy potentials is itself quadratic. Under external force, however, in
                               <a id="xref-fig-6-3" class="xref-fig" href="#F6">
                                Fig.6b
                               </a>
                               ), we observe all of these probability distributions have effectively been pulled over to higher extensions, with the bimodal distributions of mesostates 2 and 3 forming a single mode centred at higher extensions, and mesostate 1 having a much smaller occupation and number of transitions occurring.
                              </p>
                              <div id="F6" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F6.large.jpg?width=800&height=600&carousel=1" title="The normalised probability distributions, in the single network pathway system, of the dumbbell node separations and the separations specifically at the moment of transition into another mesostate. a) 0pN force applied. b) 10pN force applied." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-843933992" data-figure-caption='<div class="highwire-markup">The normalised probability distributions, in the single network pathway system, of the dumbbell node separations and the separations specifically at the moment of transition into another mesostate. a) 0pN force applied. b) 10pN force applied.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Fig 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F6.medium.gif" width="440" height="198"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Fig 6." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F6.medium.gif" width="440" height="198"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 6." data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Fig 6.
                                </span>
                                <p id="p-73" class="first-child">
                                 The normalised probability distributions, in the single network pathway system, of the dumbbell node separations and the separations specifically at the moment of transition into another mesostate. a) 0pN force applied. b) 10pN force applied.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-74">
                               <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                                Fig.7
                               </a>
                               shows the same information but for the fully connected network system. We observe approximately the same occupation probability distributions as in
                               <a id="xref-fig-6-4" class="xref-fig" href="#F6">
                                Fig.6
                               </a>
                               , as may be expected from
                               <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                                Fig.5
                               </a>
                               , but the transition probability distributions are slightly more spread out. This is especially apparent if we compare mesostate 1 of
                               <a id="xref-fig-6-5" class="xref-fig" href="#F6">
                                Fig.6a
                               </a>
                               ) and
                               <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                                Fig.7a
                               </a>
                               ), and mesostate 4, and is due to the fact that the fully connected network has more possible transition pathways. Indeed, as defined by the rates (in the absence of external force), mesostate 1 has an equal probability of transitioning straight to state 4 on average as it does mesostate 2. We therefore see that while the two systems shown yield the same occupation probabilities per mesostate, and change similarly under the application of force external to the initial parameterisation using
                               <a id="xref-disp-formula-26-4" class="xref-disp-formula" href="#disp-formula-26">
                                Eq.24
                               </a>
                               , the dynamical motion through phase space that we infer from the application of force at the continuous microstate level is dependent upon the structure and connectivity of the the kinetic landscape, i.e. which kinetic transitions are permitted.
                              </p>
                              <div id="F7" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F7.large.jpg?width=800&height=600&carousel=1" title="The normalised probability distributions, in the fully connected network system, of the dumbbell node separations and the separations specifically at the moment of transition into another mesostate. a) 0pN force applied. b) 10pN force applied." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-843933992" data-figure-caption='<div class="highwire-markup">The normalised probability distributions, in the fully connected network system, of the dumbbell node separations and the separations specifically at the moment of transition into another mesostate. a) 0pN force applied. b) 10pN force applied.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Fig 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F7.medium.gif" width="440" height="199"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Fig 7." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F7.medium.gif" width="440" height="199"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 7." data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Fig 7.
                                </span>
                                <p id="p-75" class="first-child">
                                 The normalised probability distributions, in the fully connected network system, of the dumbbell node separations and the separations specifically at the moment of transition into another mesostate. a) 0pN force applied. b) 10pN force applied.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-76">
                               The above is suggestive of alternative methods to parameterise the model from experiments. It may be the case that it is not possible to obtain R
                               <sub>
                                ij
                               </sub>
                               from equilibrium experimentation (e.g. if the rates are too slow, or too fast). A viable alternative, then, is to measure transition rates accelerated under the application of external forces. Such forces modify the energy landscape, but if these modifications are included in 14 and used to derive an alternate version of
                               <a id="xref-disp-formula-26-5" class="xref-disp-formula" href="#disp-formula-26">
                                Eq.24
                               </a>
                               , then the necessary
                               <span class="inline-formula" id="inline-formula-69">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-69.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-69.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               values can be obtained from the measured R
                               <sub>
                                ij
                               </sub>
                               under external force. Then, on removal of the force from the model, the equilibrium R
                               <sub>
                                ij
                               </sub>
                               (in the absence of force) may be inferred. This is a further example of how this mechano-kinetic framework can go beyond simply reproducing the initial parameterisation from experiment, extending the simulations to experimentally difficult and potentially infeasible scenarios for predictive purposes. The next section shows this formulation in the context of protein unfolding.
                              </p>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-14">
                            <h2 class="">
                             Protein Unfolding
                            </h2>
                            <p id="p-77">
                             We now apply the framework to the problem of protein unfolding by mimicking SMFS-AFM experiments on polyproteins. At the same time, we will use protein unfolding as an example problem to investigate the differences between the two types of microstate model discussed in the Theoretical Development Section: the mechanical energy difference (MED) model and the force-activated transition (FAT) model. We note that conventional analysis for protein unfolding experiments is similar to the FAT model [
                             <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">
                              32
                             </a>
                             ]: this makes sense as we might typically envisage that a force applied across a bond within the protein is what drives the unfolding. The kinetic schematic of this system, irrespective of model, is shown in
                             <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                              Fig.2c
                             </a>
                             , and the structural schematic is shown in
                             <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                              Fig.8
                             </a>
                             .
                            </p>
                            <div id="F8" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F8.large.jpg?width=800&height=600&carousel=1" title="The structural model of our protein unfolding simulations for both the MED and FAT models. a) Each (one-dimensional) protein is modelled as a Hookean spring connecting two dynamic nodes. Upon a kinetic transition to the unfolded state, this spring transforms into a FENE spring, representing the unfolded amino-acid chain. Each node provides half the Stokes drag for each protein. The first node is frozen in place, as indicated by the vertical black line, to mimic the surface binding in SMFS-AFM experiments. b) The FENE spring representing the linker domain between protein subunits. c) Force is applied to the system by a bead attached to a spring, representing a cantilever. The bead moves at constant velocity and is independent of thermal noise, giving a well-characterised force-loading rate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-843933992" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">The structural model of our protein unfolding simulations for both the MED and FAT models. <strong>a)</strong> Each (one-dimensional) protein is modelled as a Hookean spring connecting two dynamic nodes. Upon a kinetic transition to the unfolded state, this spring transforms into a FENE spring, representing the unfolded amino-acid chain. Each node provides half the Stokes drag for each protein. The first node is frozen in place, as indicated by the vertical black line, to mimic the surface binding in SMFS-AFM experiments. <strong>b)</strong> The FENE spring representing the linker domain between protein subunits. <strong>c)</strong> Force is applied to the system by a bead attached to a spring, representing a cantilever. The bead moves at constant velocity and is independent of thermal noise, giving a well-characterised force-loading rate.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Fig 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F8.medium.gif" width="440" height="114"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Fig 8." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F8.medium.gif" width="440" height="114"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 8." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Fig 8.
                              </span>
                              <p id="p-78" class="first-child">
                               The structural model of our protein unfolding simulations for both the MED and FAT models.
                               <strong>
                                a)
                               </strong>
                               Each (one-dimensional) protein is modelled as a Hookean spring connecting two dynamic nodes. Upon a kinetic transition to the unfolded state, this spring transforms into a FENE spring, representing the unfolded amino-acid chain. Each node provides half the Stokes drag for each protein. The first node is frozen in place, as indicated by the vertical black line, to mimic the surface binding in SMFS-AFM experiments.
                               <strong>
                                b)
                               </strong>
                               The FENE spring representing the linker domain between protein subunits.
                               <strong>
                                c)
                               </strong>
                               Force is applied to the system by a bead attached to a spring, representing a cantilever. The bead moves at constant velocity and is independent of thermal noise, giving a well-characterised force-loading rate.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-79">
                             To model protein unfolding, we choose maltose binding protein (MBP) as a model protein due to the availability of all the required information to parameterise our simple 1D model. Rief et al. have studied MBP in great detail using SMFS-AFM [
                             <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                              37
                             </a>
                             ] and further, equilibrium mechanical properties have been measured to a high accuracy using elastic incoherent neutron scattering (EINS) [
                             <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                              38
                             </a>
                             ]. In addition, equilibrium unfolding behaviour has been experimentally inferred by Na et al. [
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             ]. Finally, MBP has recently been used in cross-lengthscale mechanical studies of globular protein-based hydrogels [
                             <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">
                              22
                             </a>
                             ], and so the model developed here may be extended to assist in the future understanding of unfolding in these systems.
                            </p>
                            <p id="p-80">
                             We define two mesostates for each protein; one for the folded state and one for the unfolded. The folded state in a 1D model can be represented as a Hookean spring with an approximate spring constant k
                             <sub>
                              1
                             </sub>
                             = 30.3pN/nm [
                             <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">
                              38
                             </a>
                             ] and equilibrium length l
                             <sub>
                              1
                             </sub>
                             = 5nm, approximately representing MBP [
                             <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                              39
                             </a>
                             ]. The unfolded state can then be represented using a modified FENE potential, with associated force
                             <span class="disp-formula" id="disp-formula-27">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-41.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-41.gif"/>
                               </noscript>
                              </span>
                             </span>
                            </p>
                            <p id="p-81">
                             <a id="xref-disp-formula-27-1" class="xref-disp-formula" href="#disp-formula-27">
                              Eq.25
                             </a>
                             represents a non-interacting amino acid chain with only an entropic restoring force between the N- and C-termini at small extension. For x ≤ x
                             <sub>
                              lim
                             </sub>
                             , both the force and effective stiffness of the chain tend to infinity as x − l
                             <sub>
                              2
                             </sub>
                             → l
                             <sub>
                              c
                             </sub>
                             , as the amino acid chain straightens and the large enthalpic contributions become apparent. l
                             <sub>
                              c
                             </sub>
                             , then, is the contour length of the polypeptide. This potential reproduces some of the core behaviour of the worm-like chain model, a more common model for polypeptides used in SMFS-AFM experimental analysis [
                             <a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">
                              32
                             </a>
                             ]. Unfortunately, the effective linear (Hookean) stiffness tending to infinity presents significant numerical problems, severely limiting the integration timestep we can take at large extensions. To compensate for this, we define the effective differential stiffness at each x, k
                             <sub>
                              eff
                             </sub>
                             , as
                             <span class="disp-formula" id="disp-formula-28">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-42.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-42.gif"/>
                               </noscript>
                              </span>
                             </span>
                            </p>
                            <p id="p-82">
                             We relax the infinite barrier in the standard FENE potential by choosing a limiting stiffness k
                             <sub>
                              lim
                             </sub>
                             , such that when k
                             <sub>
                              eff
                             </sub>
                             > k
                             <sub>
                              lim
                             </sub>
                             , the potential becomes Hookean again with k
                             <sub>
                              lim
                             </sub>
                             being the new stiffness. This transition occurs at a value x
                             <sub>
                              lim
                             </sub>
                             (where k
                             <sub>
                              eff
                             </sub>
                             = k
                             <sub>
                              lim
                             </sub>
                             ), and requires a new effective equilibrium length l
                             <sub>
                              lim
                             </sub>
                             chosen such that the force is continuous at x = x
                             <sub>
                              lim
                             </sub>
                             . Both x
                             <sub>
                              lim
                             </sub>
                             and l
                             <sub>
                              lim
                             </sub>
                             can be numerically calculated prior to the start of the simulation. We effectively use the FENE potential to transition from a linear stiffness k
                             <sub>
                              2
                             </sub>
                             to a maximum linear stiffness k
                             <sub>
                              lim
                             </sub>
                             . We set k
                             <sub>
                              lim
                             </sub>
                             = 411pN/nm so that for our chosen k
                             <sub>
                              B
                             </sub>
                             T = 4.11pN.nm, fluctuations in the modified FENE chains are well-defined and negligible at large extension.
                            </p>
                            <p id="p-83">
                             The average kinetic rates R
                             <sub>
                              ij
                             </sub>
                             in the absence of force are given in
                             <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">
                              Table 7
                             </a>
                             . The native unfolding rate of MBP was sourced from pulse proteolysis experiments [
                             <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             ], whereas the refolding rate was approximated using simple diffusion calculations i.e. the time taken for half the amino-acid chain to diffuse its own length. In practise, however, refolding almost never occurs in the simulation presented. These rates were used with
                             <a id="xref-disp-formula-16-5" class="xref-disp-formula" href="#disp-formula-16">
                              Eq.14
                             </a>
                             , and the relevant integrals were calculated numerically to account for the more complex modified FENE potential.
                            </p>
                            <div id="T7" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2462554/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup">
                                 <a href="/highwire/markup/2462554/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                                <li class="download-ppt last">
                                 <a href="/highwire/powerpoint/2462554" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                  Download powerpoint
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Table 7.
                              </span>
                              <p id="p-84" class="first-child">
                               The initial set of transition rates (in the absence of external force) used to investigate protein unfolding.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-85">
                             As illustrated in
                             <a id="xref-fig-8-2" class="xref-fig" href="#F8">
                              Fig.8
                             </a>
                             , we connect five proteins in series with shorter segments of modified FENE potential amino-acid chain with only 1 mesostate defined, representing the linker domain of an engineered polyprotein [
                             <a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">
                              32
                             </a>
                             ]. We use five primarily because relatively short polyproteins have previously been under independent investigation by the authors [
                             <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                              40
                             </a>
                             –
                             <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                              43
                             </a>
                             ]. FENE parameters were calculated by using the limiting, soft case of the WLC model together with the number of amino acids in each chain, assuming that the persistence length of the chain L
                             <sub>
                              p
                             </sub>
                             = 0.4nm [
                             <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                              44
                             </a>
                             ]. All parameters are given in
                             <a id="xref-table-wrap-8-1" class="xref-table" href="#T8">
                              Table 8
                             </a>
                             .
                            </p>
                            <div id="T8" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2462568/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup">
                                 <a href="/highwire/markup/2462568/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                                <li class="download-ppt last">
                                 <a href="/highwire/powerpoint/2462568" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                  Download powerpoint
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Table 8.
                              </span>
                              <p id="p-86" class="first-child">
                               The energy function and material parameters used to define the protein mesostates used in the kinetic network of the polyprotein unfolding model.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-87">
                             In addition, the final protein in the series is connected to a linear spring, with spring constant k
                             <sub>
                              c
                             </sub>
                             = 1pN/nm, representing the cantilever of the atomic force microscope. This spring is in turn connected to a bead that is moved at a constant velocity v
                             <sub>
                              c
                             </sub>
                             in the +ve x direction. This progressively varies the force over the course of the simulation in a manner analogous to the stretching protocol employed in experiments [
                             <a id="xref-ref-32-7" class="xref-bibr" href="#ref-32">
                              32
                             </a>
                             ,
                             <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">
                              40
                             </a>
                             ,
                             <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                              41
                             </a>
                             ], in which the base of the cantilever is moved with constant velocity, and identical to the constant-velocity steered MD protocol implemented in the NAMD MD package [
                             <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                              45
                             </a>
                             ,
                             <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                              46
                             </a>
                             ]. All protein unfolding simulations were performed by also fixing the position of the left-most particle in the polyprotein. As such, for each particle α other than the first and last particles, each particle is subject to the a Langevin equation
                             <span class="disp-formula" id="disp-formula-29">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-45.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-45.gif"/>
                               </noscript>
                              </span>
                             </span>
                             where F
                             <sub>
                              α,β
                             </sub>
                             is the elastic force applied to particle a by the connection to particle β, and is either a Hookean force (representing a folded protein), or a FENE force (representing either a linker or unfolded protein) depending on the spring object each pair of particles are connected by as the simulation progresses. The connecting object will change as part of the kinetic scheme. The initial particle is unique in that it never moves, and so isn’t subject to Langevin dynamics. The particle connected to the cantilever component is also unique, and has an extra force
                             <span class="inline-formula" id="inline-formula-70">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-70.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-70.gif"/>
                               </noscript>
                              </span>
                             </span>
                             included instead of a further polyprotein component
                             <span class="disp-formula" id="disp-formula-30">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-46.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-46.gif"/>
                               </noscript>
                              </span>
                             </span>
                             where λ = 6πpN.ns/nm and
                             <span class="disp-formula" id="disp-formula-31">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-47.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-47.gif"/>
                               </noscript>
                              </span>
                             </span>
                             and leads to a well-characterised force-loading rate R
                             <sub>
                              L
                             </sub>
                             = k
                             <sub>
                              c
                             </sub>
                             v
                             <sub>
                              c
                             </sub>
                             as the protein assembly reaches maximal extension (in practice the loading rate is slightly smaller due to the increasing extension of the proteins and linkers). We emphasise that the constant-velocity bead in our cantilever object is independent of thermal noise in our simulations, which is not the case experimentally.
                            </p>
                            <p id="p-88">
                             Finally, when implementing the mechanical-energy difference model, we proceed as we have so far with the integrals in
                             <a id="xref-disp-formula-16-6" class="xref-disp-formula" href="#disp-formula-16">
                              Eq.14
                             </a>
                             performed to determine each
                             <span class="inline-formula" id="inline-formula-71">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-71.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-71.gif"/>
                               </noscript>
                              </span>
                             </span>
                             . For the force-activated transition model, we require an additional parameter b
                             <sub>
                              ij
                             </sub>
                             for each integral in
                             <a id="xref-disp-formula-19-2" class="xref-disp-formula" href="#disp-formula-19">
                              Eqs.17
                             </a>
                             and
                             <a id="xref-disp-formula-20-4" class="xref-disp-formula" href="#disp-formula-20">
                              18
                             </a>
                             . In the first investigations we parametrise as best we can from available experimental data [
                             <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">
                              37
                             </a>
                             ,
                             <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">
                              40
                             </a>
                             ], using b
                             <sub>
                              ij
                             </sub>
                             = 1nm as representative, but we later investigate the behaviour of this parameter as a part of the protein-unfolding simulation suite.
                            </p>
                            <div id="sec-15" class="subsection">
                             <h3>
                              Protein Unfolding - Results
                             </h3>
                             <p id="p-89">
                              We begin our investigation with an investigation of the effect of thermal noise on the measured unfolding force of the model proteins, implementing both the MED and FAT models. Thermal fluctuations, while small in comparison to work done by the externally applied force, are nevertheless included in the dynamic portion of the simulation, and therefore ought to directly affect the emergent kinetics. We set the stiffness of the constant-velocity spring component, k
                              <sub>
                               c
                              </sub>
                              = 1pN/nm, and the pulling speed, v
                              <sub>
                               c
                              </sub>
                              = 5 × 10
                              <sup>
                               -4
                              </sup>
                              nm/ns, to minimise the signal-noise ratio and so that the force increases at a reasonable rate. In the FAT implementation, we set b
                              <sub>
                               ij
                              </sub>
                              = b = 1.0nm for each protein along the chain, a somewhat arbitrary choice of unity but of the approximately the correct order of magnitude for protein unfolding [
                              <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">
                               41
                              </a>
                              ] and representative of MBP itself [
                              <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">
                               37
                              </a>
                              ].
                             </p>
                             <p id="p-90">
                              <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                               Fig.9
                              </a>
                              shows four characteristic SMFS-AFM curves extracted from mechano-kinetic simulations performed with and without thermal noise for both model implementations. We emphasise, the force measured here on the y-axis is the external tension within the simulated cantilever components, not the internal force within any of the subunits. The extension on the x-axis is measured from the N-terminus of the first protein to the C-terminus of the last protein, not the position of the attached constant-velocity bead. This choice best represents what would be the tip of the cantilever in a real experiment.
                             </p>
                             <div id="F9" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F9.large.jpg?width=800&height=600&carousel=1" title="A set of polyprotein unfolding curves from associated protein unfolding simulations showing how the presence of thermal noise affects the unfolding kinetics." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-843933992" data-figure-caption='<div class="highwire-markup">A set of polyprotein unfolding curves from associated protein unfolding simulations showing how the presence of thermal noise affects the unfolding kinetics.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F9.medium.gif" width="440" height="439"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 9." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F9.medium.gif" width="440" height="439"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 9." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig 9.
                               </span>
                               <p id="p-91" class="first-child">
                                A set of polyprotein unfolding curves from associated protein unfolding simulations showing how the presence of thermal noise affects the unfolding kinetics.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-92">
                              We can see clearly that thermal noise has a significant effect on the unfolding force in both cases. By extracting the five peak forces (and their associated extensions), we can gain statistics for the unfolding kinetics. It has been previously observed that each specific peak has its own kinetic characteristics [
                              <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                               47
                              </a>
                              ] but for now, we take the average of the these values from each simulation to gain single representative values for each system. For the MED model, in the absence of thermal noise, we measure an unfolding force, F
                              <sub>
                               un
                              </sub>
                              = 111.6 ± 1.4pN, and a peak-to-peak distance, x
                              <sub>
                               pp
                              </sub>
                              = 136.3 ± 0.3nm, and in the presence of thermal noise, we measure F
                              <sub>
                               un
                              </sub>
                              = 85.9 ± 1.0pN and x
                              <sub>
                               pp
                              </sub>
                              = 135.9 ± 0.7nm. For the FAT model, in the absence of thermal noise, we measure an unfolding force, F
                              <sub>
                               un
                              </sub>
                              = 113.5 ± 3.7pN, and a peak-to-peak distance, x
                              <sub>
                               pp
                              </sub>
                              = 136.3 ± 1.2nm, and in the presence of thermal noise, we measure F
                              <sub>
                               un
                              </sub>
                              = 99.5 ± 2.8pN and x
                              <sub>
                               pp
                              </sub>
                              = 136.3 ± 1.6nm. In all cases the measured x
                              <sub>
                               pp
                              </sub>
                              value is remarkably consistent, and the curves overlap well, implying physical consistency between simulations with and without noise. However, the measured x
                              <sub>
                               pp
                              </sub>
                              values are lower than l
                              <sub>
                               c
                              </sub>
                              − (l
                              <sub>
                               1
                              </sub>
                              + F
                              <sub>
                               un
                              </sub>
                              /k
                              <sub>
                               1
                              </sub>
                              ), which is the value we would expect if the FENE chain were to fully extend under the externally applied force. This implies that the proteins gain enough energy to unfold at slightly lower extensions. Nevertheless, this difference is relatively small. More importantly, the presence of thermal noise lowers the required unfolding force by approximately 25pN in the MED model, and 14pN in the FAT model. This difference in the reduction in F
                              <sub>
                               un
                              </sub>
                              upon the inclusion of thermal noise is outside the uncertainty, suggesting a real difference between the models, but may be due to the difference in the value of b used and the average distance to the unfolding transition in the MED model. In both cases however, such a reduction in F
                              <sub>
                               un
                              </sub>
                              was quite unexpected as thermal fluctuations contribute a modest 0.5k
                              <sub>
                               B
                              </sub>
                              T of mechanical energy on average at zero external force, a negligible impact on the stability of most proteins, and MBP in particular. On reflection, however, due to the exponential nature of Boltzmann’s equation, we see that the same amount of energy at high force has a relatively large impact, pushing the protein over the energy barrier required to unfold. This is a significant validation of the relevance of thermal noise in biological systems beyond being simply a tool for diffusion and entropic exploration of conformational space. This will be the case especially in non-equilibrium simulations and in models of proteins with low barriers to unfolding, such as intrinsically disordered proteins [
                              <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                               48
                              </a>
                              ] and some extremophilic proteins [
                              <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                               49
                              </a>
                              –
                              <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                               51
                              </a>
                              ]. All simulations from this point forward will be performed with thermal noise included.
                             </p>
                             <p id="p-93">
                              We now investigate the effect on this system of variations in the (folded) protein spring constant k
                              <sub>
                               1
                              </sub>
                              , and the pulling speed v
                              <sub>
                               c
                              </sub>
                              . It is well known that changes in the pulling speed ought to alter the measured unfolding force due to the unfolding mechanism itself, but interestingly, West et al. also predicted that the existence of internal dynamics ought to alter the unfolding transition kinetics [
                              <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                               26
                              </a>
                              ], and varying k1 ought to allow a systematic analysis of this. First using the MED model, we performed a single unfolding simulation for each of the parameter pairs defined in
                              <a id="xref-table-wrap-9-1" class="xref-table" href="#T9">
                               Table 9
                              </a>
                              , and again extracted the unfolding forces, F
                              <sub>
                               un
                              </sub>
                              , and and peak-to-peak distances, xpp. All other parameters were kept constant between simulations, including the average rates R
                              <sub>
                               ij
                              </sub>
                              .
                             </p>
                             <div id="T9" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2462558/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2462558/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2462558" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 9.
                               </span>
                               <p id="p-94" class="first-child">
                                The parameter space investigated to determine the effect on the unfolding forces and peak-to-peak separations of protein unfolding curves.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-95">
                              From
                              <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                               Fig.10
                              </a>
                              , in the MED model we observe that the protein stiffness has the most significant effect on both F
                              <sub>
                               un
                              </sub>
                              for all values of v
                              <sub>
                               c
                              </sub>
                              (for a given value of R
                              <sub>
                               ij
                              </sub>
                              ). Rewriting the linear protein energy explicitly in terms of force,
                              <span class="inline-formula" id="inline-formula-72">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-72.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-72.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , we see that for a higher stiffness, a higher force is required to reach the same energy. Thus, we see that the unfolding force increases with the internal stiffness of the protein. Interestingly, the peak-to-peak separation x
                              <sub>
                               pp
                              </sub>
                              also increases with internal protein stiffness. As a function of the cantilever pulling speed v
                              <sub>
                               c
                              </sub>
                              , however, we see too much variation between peak-to-peak values to draw any conclusions. However, it is clear that the unfolding force does slightly increase with v
                              <sub>
                               c
                              </sub>
                              , as expected.
                             </p>
                             <div id="F10" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F10.large.jpg?width=800&height=600&carousel=1" title="A series of protein unfolding simulations using both the MED model (crosses) and the FAT model (circles) showing how the cantilever pulling speed and protein stiffness affects the unfolding kinetics. a) Unfolding forces. b) Peak-to-peak distances." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-843933992" data-figure-caption='<div class="highwire-markup">A series of protein unfolding simulations using both the MED model (crosses) and the FAT model (circles) showing how the cantilever pulling speed and protein stiffness affects the unfolding kinetics. a) Unfolding forces. b) Peak-to-peak distances.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 10." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F10.medium.gif" width="440" height="209"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 10." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F10.medium.gif" width="440" height="209"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 10." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig 10.
                               </span>
                               <p id="p-96" class="first-child">
                                A series of protein unfolding simulations using both the MED model (crosses) and the FAT model (circles) showing how the cantilever pulling speed and protein stiffness affects the unfolding kinetics. a) Unfolding forces. b) Peak-to-peak distances.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-97">
                              We performed an identical set of simulations within the FAT model, setting the parameter b = 0.5nm, again of the correct order of magnitude for (MBP) protein unfolding [
                              <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">
                               37
                              </a>
                              ,
                              <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">
                               41
                              </a>
                              ], but also to more clearly show the difference between these models in the parameter sweep. We again see that the peak-to-peak separations are most dominated by the protein stiffness, with the most pronounced difference from the MED model occurring for k
                              <sub>
                               1
                              </sub>
                              = 10pN/nm. We also see that the unfolding forces are much greater than those in the MED model in each case. This will later be shown to be a simple matter of effective parameterisation differences between models. More importantly, though, we observe the unfolding forces in each case are, as expected from
                              <a id="xref-disp-formula-17-2" class="xref-disp-formula" href="#disp-formula-17">
                               Eq.15
                              </a>
                              , approximately independent of protein stiffness. The two models are therefore clearly unique, showing that our choice of mechanical model in the framework is inseparable from the results we obtain. Thus, in the absence of knowledge of the “true” underlying mechanical model, the full mechano-kinetic framework acts as a tool to investigate what we may expect to see, given a certain set of assumptions and experimental data.
                             </p>
                             <p id="p-98">
                              To emphasis this point, we will now use the protein unfolding simulations to investigate two parameters unique to each model. First, we observe the effect of the set of f
                              <sub>
                               ij
                              </sub>
                              in the MED model. These values were (to our knowledge) first used conceptually by Sarlah et al. to imply a “mechanical activation energy”, and modify how far towards the activation energy we need to be to perform the discretised kinetic transition [
                              <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ]. A value of f
                              <sub>
                               ij
                              </sub>
                              close to 1 implies that the mechanical energy barrier is close to the target state, and a value close to zero implies that the mechanical energy barrier is close to the initial state.
                              <a id="xref-fig-11-1" class="xref-fig" href="#F11">
                               Fig.11
                              </a>
                              shows the results of altering f
                              <sub>
                               ij
                              </sub>
                              on the protein unfolding simulations with k
                              <sub>
                               1
                              </sub>
                              = 30.3 pN/nm and v
                              <sub>
                               c
                              </sub>
                              = 10
                              <sup>
                               -3
                              </sup>
                              nm/ns, where f
                              <sub>
                               01
                              </sub>
                              on the x-axis represents the value used for the unfolding transition.
                             </p>
                             <div id="F11" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F11.large.jpg?width=800&height=600&carousel=1" title="A series of protein unfolding simulations, varying fij and k1 a) Unfolding forces. b) Peak-to-peak distances." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-843933992" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">A series of protein unfolding simulations, varying <em>f<sub>ij</sub></em> and <em>k</em><sub>1</sub> a) Unfolding forces. b) Peak-to-peak distances.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 11." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F11.medium.gif" width="440" height="206"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 11." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F11.medium.gif" width="440" height="206"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 11." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig 11.
                               </span>
                               <p id="p-99" class="first-child">
                                A series of protein unfolding simulations, varying f
                                <sub>
                                 ij
                                </sub>
                                and k
                                <sub>
                                 1
                                </sub>
                                a) Unfolding forces. b) Peak-to-peak distances.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-100">
                              We see that the effect of varying k
                              <sub>
                               1
                              </sub>
                              on both F
                              <sub>
                               un
                              </sub>
                              and x
                              <sub>
                               pp
                              </sub>
                              is qualitatively the same across all values of f
                              <sub>
                               01
                              </sub>
                              , increasing both the unfolding force and peak-to-peak distance as previously observed, showing that the f
                              <sub>
                               ij
                              </sub>
                              parameters do act as scaling factors with respect to the energy term. Again, x
                              <sub>
                               pp
                              </sub>
                              is not much affected by the variations in f
                              <sub>
                               01
                              </sub>
                              , but the unfolding force F
                              <sub>
                               un
                              </sub>
                              shows a significant decrease. It is intuitive to see that, because the multiplication of f
                              <sub>
                               ij
                              </sub>
                              and
                              <span class="inline-formula" id="inline-formula-73">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-73.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-73.gif"/>
                                </noscript>
                               </span>
                              </span>
                              together are what determine the transition rate, an increase in f
                              <sub>
                               01
                              </sub>
                              would mean that a smaller
                              <span class="inline-formula" id="inline-formula-74">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-74.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-74.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is required for the same transition probability, and thus a lower unfolding force is observed. However, a more interesting perspective emerges from the observation that f
                              <sub>
                               ij
                              </sub>
                              relates to something like a mechanical activation energy. As
                              <span class="inline-formula" id="inline-formula-75">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-75.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-75.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , then increasing f
                              <sub>
                               ij
                              </sub>
                              reduces the effective mechanical activation energy required to make the transition, thus reducing the unfolding force.
                             </p>
                             <p id="p-101">
                              To conclude this investigation, we now investigate the effect of the value b
                              <sub>
                               ij
                              </sub>
                              on the FAT model. In this model, we recall that the protein internal force is used continuously to modulate the kinetic transition rates, together with a distance parameter b
                              <sub>
                               ij
                              </sub>
                              . b
                              <sub>
                               ij
                              </sub>
                              is a parameter we set in advance, and so we again set b
                              <sub>
                               ij
                              </sub>
                              = b the same for all proteins along the chain. We set k
                              <sub>
                               1
                              </sub>
                              = 30.3 pN/nm and performed a series of simulations for various values of b and v
                              <sub>
                               c
                              </sub>
                              . As each protein unfolded, we recorded the unfolding force within the cantilever object (as would be measured experimentally),
                              <span class="inline-formula" id="inline-formula-76">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-76.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-76.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , the unfolding force within the protein itself, f
                              <sub>
                               un
                              </sub>
                              (x) and therefore also the specific mechanical extension of the protein, Δx.
                             </p>
                             <p id="p-102">
                              In standard theoretical interpretations of protein unfolding experiments, the Bell-Evans-Ritchie model is used to calculate the effective unfolding rates as a function of external force, R
                              <sub>
                               ij
                              </sub>
                              (F
                              <sup>
                               ex
                              </sup>
                              ) [
                              <a id="xref-ref-32-8" class="xref-bibr" href="#ref-32">
                               32
                              </a>
                              ]:
                              <span class="disp-formula" id="disp-formula-32">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-52.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-52.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where F
                              <sup>
                               ex
                              </sup>
                              is some externally applied force and ΔX
                              <sub>
                               un
                              </sub>
                              is the “distance” through the free-energy landscape to the transition barrier. R
                              <sub>
                               ij
                              </sub>
                              (0), then, is the same R
                              <sub>
                               ij
                              </sub>
                              we have been using throughout this work, the experimentally measured equilibrium unfolding rate at the mesoscale. This equation is similar in form to
                              <a id="xref-disp-formula-17-3" class="xref-disp-formula" href="#disp-formula-17">
                               Eq.15
                              </a>
                              , but with important differences. First, although the value b is included at the microscale, we first show that it is equivalent to value ΔX
                              <sub>
                               un
                              </sub>
                              used in
                              <a id="xref-disp-formula-32-1" class="xref-disp-formula" href="#disp-formula-32">
                               Eq.30
                              </a>
                              . From the work of Evans [
                              <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">
                               52
                              </a>
                              ], Hughes et al. [
                              <a id="xref-ref-32-9" class="xref-bibr" href="#ref-32">
                               32
                              </a>
                              ] were able to show that the expected unfolding force as a function of relevant variables,
                              <span class="inline-formula" id="inline-formula-77">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-77.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/inline-graphic-77.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , is:
                              <span class="disp-formula" id="disp-formula-33">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-53.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-53.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-103">
                              Using b = ΔX
                              <sub>
                               un
                              </sub>
                              we fit
                              <a id="xref-disp-formula-33-1" class="xref-disp-formula" href="#disp-formula-33">
                               Eq.31
                              </a>
                              to our simulation data, shown in
                              <a id="xref-fig-12-1" class="xref-fig" href="#F12">
                               Fig.12a
                              </a>
                              ). We find good agreement between our simulations and theoretical predictions, indicating that b is indeed likely equivalent to ΔX
                              <sub>
                               un
                              </sub>
                              . This is shown more explicitly in
                              <a id="xref-fig-12-2" class="xref-fig" href="#F12">
                               Fig.12b
                              </a>
                              ), where we have plotted a characteristic energy E
                              <sub>
                               ch
                              </sub>
                              for both the theoretical predictions and our measured results. By removing the b dependence for the RHS of
                              <a id="xref-disp-formula-33-2" class="xref-disp-formula" href="#disp-formula-33">
                               Eq.31
                              </a>
                              , we define:
                              <span class="disp-formula" id="disp-formula-34">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-54.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-54.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-35">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-55.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/embed/graphic-55.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where R
                              <sub>
                               L
                              </sub>
                              = k
                              <sub>
                               c
                              </sub>
                              v
                              <sub>
                               c
                              </sub>
                              is the force-loading rate. We enforce a gradient of k
                              <sub>
                               B
                              </sub>
                              T in our linear fit of
                              <a id="xref-disp-formula-34-1" class="xref-disp-formula" href="#disp-formula-34">
                               Eq.32
                              </a>
                              to the data, thus implicitly assuming that any errors are due to differences in the effective transition rates, and find an approximate difference of 6pN.nm between our results and the theoretically expected values. This is a minor deviation on the order of thermal fluctuations at equilibrium. The E
                              <sub>
                               ch
                              </sub>
                              values in the
                              <a id="xref-fig-12-3" class="xref-fig" href="#F12">
                               Fig.12b
                              </a>
                              ) legend correspond to extrapolations to the experimentally used loading rates, assuming v
                              <sub>
                               c
                              </sub>
                              = 6 × 10
                              <sup>
                               -7
                              </sup>
                              nm/ns and k
                              <sub>
                               c
                              </sub>
                              = 50pN/nm. Interestingly, Hoffmann et al. also calculated a related form of characteristic energy in their study of many naturally occurring proteins with differing measured values of ΔX
                              <sub>
                               un
                              </sub>
                              and F
                              <sub>
                               un
                              </sub>
                              . From this collection they determined a typical E
                              <sub>
                               ch
                              </sub>
                              = 39.4 ± 3.1pN.nm for each protein in their survey. However, in an earlier work [
                              <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">
                               40
                              </a>
                              ], they also found MBP to be a protein which significantly differs from all others, suggesting that the initial unfolding rate Rij may be responsible for this huge difference between MBP and other natural proteins.
                             </p>
                             <div id="F12" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F12.large.jpg?width=800&height=600&carousel=1" title="A series of protein unfolding simulations, varying b, for a variety of cantilever speeds vc. We observe the approximately correct theoretical dependence of b and the unfolding force, . a) Unfolding force traces. b) Fit and extrapolation of characteristic unfolding energies Ech to the experimental loading rates RL = kc vc." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-843933992" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">A series of protein unfolding simulations, varying <em>b</em>, for a variety of cantilever speeds vc. We observe the approximately correct theoretical dependence of <em>b</em> and the unfolding force, . a) Unfolding force traces. b) Fit and extrapolation of characteristic unfolding energies <em>E<sub>ch</sub></em> to the experimental loading rates <em>R<sub>L</sub></em> = <em>k<sub>c</sub> v<sub>c</sub></em>.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 12." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F12.medium.gif" width="440" height="209"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 12." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F12.medium.gif" width="440" height="209"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 12." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/22/2020.11.17.386524/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig 12.
                               </span>
                               <p id="p-104" class="first-child">
                                A series of protein unfolding simulations, varying b, for a variety of cantilever speeds vc. We observe the approximately correct theoretical dependence of b and the unfolding force,
                                <span class="inline-formula" id="inline-formula-78">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/F12/embed/inline-graphic-78.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/22/2020.11.17.386524/F12/embed/inline-graphic-78.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                . a) Unfolding force traces. b) Fit and extrapolation of characteristic unfolding energies E
                                <sub>
                                 ch
                                </sub>
                                to the experimental loading rates R
                                <sub>
                                 L
                                </sub>
                                = k
                                <sub>
                                 c
                                </sub>
                                v
                                <sub>
                                 c
                                </sub>
                                .
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-105">
                              It is clear that the FAT model is likely a more appropriate model for the problem of protein unfolding than the MED model, given the success of reproduction of existing theoretical predictions.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-16">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-106">
                             We have developed a “mechano-kinetic” simulation framework which enables instantaneous kinetic transitions to occur within, and be explicitly affected by, an underlying dynamic simulation framework. By integrating out the desired mechanical contributions to the (experimentally measured) free energies, expressed as transition rates between discrete kinetic states, we can infer the dynamics present at the microscale for a given energy landscape. We applied this framework to a simple dumbbell simulation and showed that our model correctly reproduces the free energy landscape used to parametrise the model, while at the same time showing exactly where in phase space kinetic transitions are likely to occur, again given a set of experimentally observed mesostate and an appropriate mechanical model. We further showed that external potentials can be introduces to the model, and found that under the action of external forces, while the connectivity of the kinetic states do not affect the measured mesostate occupation probabilities, they strongly affect the microstate populations and, importantly, the distribution of transition microstates. We finally applied the framework to the case of protein unfolding, and showed that characteristic SFMS-AFM unfolding traces can be produced. Utilising both the “Mechanical energy difference” model and the “Force-activated transition” model, we observe that our selection of the underlying mechanical energy landscape and state transition model are inseparable from the interpretations of the results of any simulations.
                            </p>
                            <p id="p-107">
                             The original intention of this project was to develop an empirical framework, enabling extremely complex biophysical events (i.e. protein unfolding, conformational changes, binding / unbinding of ligands etc) to be implicitly simulated in parallel with a dynamic model without the need for extremely high resolution and the large associated amounts of computing power. We have achieved this goal, and even though our protein unfolding simulations show slight deviation from theoretical predictions, we can still calibrate the model to give the behaviour we desire.
                            </p>
                            <p id="p-108">
                             Going forward, we note that there also exist systems in which the energy barrier between kinetic states is naturally quite low as compared to typical thermal energies [
                             <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">
                              48
                             </a>
                             –
                             <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">
                              51
                             </a>
                             ]; in such cases it is difficult to make a clear distinction between states because transitions between them happen readily and frequently. Intermediate states have been observed in our example systems [
                             <a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">
                              37
                             </a>
                             ,
                             <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">
                              53
                             </a>
                             ]. The development of Markov state models (MSMs) [
                             <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">
                              54
                             </a>
                             ,
                             <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">
                              55
                             </a>
                             ] shows that discrete kinetic states can be defined somewhat arbitrarily, and as such energy barriers can be arbitrarily low. Finally, Bialek et al. noted early on that protein dynamics and their reaction rates are coupled [
                             <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">
                              56
                             </a>
                             ], such that a majority of the reactive events in some proteins can be coupled to specific protein modes of motion, indicating dynamic specificity. Multiple groups discovered almost in parallel that SMFS-AFM yields different unfolding forces depending on the directions in which you apply force, indicating dynamic anisotropy [
                             <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">
                              57
                             </a>
                             –
                             <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">
                              59
                             </a>
                             ]. In these cases, then, the instantaneous location in physical space and associated mechanical energies can have a non-negligible effect on the overall free energies, and thus the transition rates between states. The inspiring work of Trott et al. emphasises this, introducing a highly detailed mechanistic model of cytoplasmic dynein which combines a wealth of experimental evidence to deduce the continuous function of the motor [
                             <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">
                              60
                             </a>
                             ]. Examination of their computational insights indicates that from an atomistic statistical mechanics perspective, the mesoscopic transition rates emerge from the underlying dynamical behaviour.
                            </p>
                            <p id="p-109">
                             Our model can account for all of these possibilities. With an appropriate definition of a mechanical energy landscape, whether high-resolution molecular dynamics or bespoke, including velocity degrees of freedom if necessary, mesostate definitions and the transition mechanisms between them, new insights can be drawn into the coupling between microstate dynamic behaviour and mesoscale kinetic behaviour in systems traditionally thought to be too far separated in characteristic timescale to simulate both in parallel.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgments
                            </h2>
                            <p id="p-110">
                             We would like to thank Matt D.G. Hughes, Christa Brown, Harrison Laurent, Kalila Cook and Mazin Nasralla for discussions of the model in the context of protein unfolding in complex systems. Ben Hanson was initially supported by the EPSRC through a DTA, and completed the work supported again by the EPSRC with grant EP/P02288X/1 awarded to Professor Lorna Dougan.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-others" id="fn-1">
                              <p id="p-1">
                               <a class="rev-xref" href="#xref-fn-1-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                *
                               </span>
                               <span class="em-link">
                                <span class="em-addr">
                                 d.j.read{at}leeds.ac.uk
                                </span>
                               </span>
                              </p>
                             </li>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-2">
                              <p id="p-7">
                               Additional mathematical formulations and examples included on the topic of protein unfolding
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.11.17.386524v3.1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bates
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Huang
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rust
                                  </span>
                                  <span class="cit-name-given-names">
                                   MJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dempsey
                                  </span>
                                  <span class="cit-name-given-names">
                                   GT
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhuang
                                  </span>
                                  <span class="cit-name-given-names">
                                   X
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-chapter-title">
                                  Sub-Diffraction-Limit Imaging with Stochastic Optical Reconstruction Microscopy
                                 </span>
                                 . In:
                                 <span class="cit-auth cit-collab">
                                  A G, R R, J W
                                 </span>
                                 , editors.
                                 <span class="cit-source">
                                  Single Molecule Spectroscopy in Chemistry, Physics and Biology
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer Berlin Heidelberg
                                 </span>
                                 ;
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.11.17.386524v3.2" data-doi="10.1016/j.cell.2015.03.049">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cheng
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Single-particle Cryo-EM at crystallographic resolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  161
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  450
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  457
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D161%26rft.spage%253D450%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.03.049%26rft_id%253Dinfo%253Apmid%252F25910205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.03.049&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25910205&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F22%2F2020.11.17.386524.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.3" data-doi="10.1126/science.1251652">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kühlbrandt
                                  </span>
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The Resolution Revolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  343
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6178
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1443
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1444
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKuhlbrandt%26rft.auinit1%253DW.%26rft.volume%253D343%26rft.issue%253D6178%26rft.spage%253D1443%26rft.epage%253D1444%26rft.atitle%253DThe%2BResolution%2BRevolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1251652%26rft_id%253Dinfo%253Apmid%252F24675944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDMvNjE3OC8xNDQzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMjIvMjAyMC4xMS4xNy4zODY1MjQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.4" data-doi="10.1016/j.jsb.2017.06.005">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Frank
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Time-resolved Cryo-Electron Microscopy: Recent Progress
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Structural Biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  200
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  303
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  306
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStructural%2BBiology%26rft.volume%253D200%26rft.spage%253D303%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jsb.2017.06.005%26rft_id%253Dinfo%253Apmid%252F28625887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.jsb.2017.06.005&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28625887&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F22%2F2020.11.17.386524.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.5" data-doi="10.1038/s41556-018-0251-8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schermelleh
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ferrand
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Huser
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eggeling
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sauer
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Biehlmaier
                                  </span>
                                  <span class="cit-name-given-names">
                                   O
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Super-resolution microscopy demystified
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Cell Biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  21
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  72
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  84
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.volume%253D21%26rft.spage%253D72%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41556-018-0251-8%26rft_id%253Dinfo%253Apmid%252F30602772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41556-018-0251-8&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30602772&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F22%2F2020.11.17.386524.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.6" data-doi="10.1093/nar/gkq880">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lawson
                                  </span>
                                  <span class="cit-name-given-names">
                                   CL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Baker
                                  </span>
                                  <span class="cit-name-given-names">
                                   ML
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Best
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bi
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dougherty
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Feng
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  EMDataBank.org: Unified data resource for CryoEM
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Research
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  39
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  456
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  464
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D39%26rft.spage%253D456%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkq880&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.7" data-doi="10.1371/journal.pcbi.1006359">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Solernou
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hanson
                                  </span>
                                  <span class="cit-name-given-names">
                                   BS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Richardson
                                  </span>
                                  <span class="cit-name-given-names">
                                   RA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Welch
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Read
                                  </span>
                                  <span class="cit-name-given-names">
                                   DJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harlen
                                  </span>
                                  <span class="cit-name-given-names">
                                   OG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Fluctuating Finite Element Analysis (FFEA): A continuum mechanics software tool for mesoscale simulation of biomolecules
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Computational Biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BComputational%2BBiology%26rft.volume%253D14%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1006359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1006359&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leng
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shoura
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McLeish
                                  </span>
                                  <span class="cit-name-given-names">
                                   TCB
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Real
                                  </span>
                                  <span class="cit-name-given-names">
                                   AN
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hardey
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McCafferty
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Securing the future of research computing in the biosciences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Computational Biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  15
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BComputational%2BBiology%26rft.volume%253D15%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Richardson
                                  </span>
                                  <span class="cit-name-given-names">
                                   RA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hanson
                                  </span>
                                  <span class="cit-name-given-names">
                                   BS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Read
                                  </span>
                                  <span class="cit-name-given-names">
                                   DJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harlen
                                  </span>
                                  <span class="cit-name-given-names">
                                   OG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harris
                                  </span>
                                  <span class="cit-name-given-names">
                                   SA
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Exploring the dynamics of flagellar dynein within the axoneme with Fluctuating Finite Element Analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Quarterly Reviews of Biophysics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-fpage">
                                  53
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.10" data-doi="10.1016/j.ymeth.2020.01.021">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hanson
                                  </span>
                                  <span class="cit-name-given-names">
                                   BS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Iida
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Read
                                  </span>
                                  <span class="cit-name-given-names">
                                   DJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harlen
                                  </span>
                                  <span class="cit-name-given-names">
                                   OG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kurisu
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nakamura
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Continuum mechanical parameterisation of cytoplasmic dynein from atomistic simulation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Methods
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  185
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  39
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  48
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMethods%26rft.volume%253D185%26rft.spage%253D39%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2020.01.021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2020.01.021&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.11" data-doi="10.1371/journal.pcbi.1005584">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kubo
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Takada
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Allosteric conformational change cascade in cytoplasmic dynein revealed by structure-based molecular simulations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Computational Biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  27
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BComputational%2BBiology%26rft.volume%253D13%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1005584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1005584&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.12" data-doi="10.1016/j.cell.2006.05.046">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reck-peterson
                                  </span>
                                  <span class="cit-name-given-names">
                                   SL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yildiz
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carter
                                  </span>
                                  <span class="cit-name-given-names">
                                   AP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gennerich
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vale
                                  </span>
                                  <span class="cit-name-given-names">
                                   RD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mcintosh
                                  </span>
                                  <span class="cit-name-given-names">
                                   JR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Single-Molecule Analysis of Dynein Processivity and Stepping Behavior
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  126
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  335
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  348
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DReck-Peterson%26rft.auinit1%253DS.%2BL.%26rft.volume%253D126%26rft.issue%253D2%26rft.spage%253D335%26rft.epage%253D348%26rft.atitle%253DSingle-molecule%2Banalysis%2Bof%2Bdynein%2Bprocessivity%2Band%2Bstepping%2Bbehavior.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.05.046%26rft_id%253Dinfo%253Apmid%252F16873064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.05.046&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16873064&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F22%2F2020.11.17.386524.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000239552600019&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.11.17.386524v3.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alper
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alexov
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Cytoplasmic dynein binding, run length, and velocity are guided by long-range electrostatic interactions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Scientific Reports
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  12
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D6%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kinoshita
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kambara
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nishikawa
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kaya
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Higuchi
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Step Sizes and Rate Constants of Single-headed Cytoplasmic Dynein Measured with Optical Tweezers
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Scientific Reports
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  14
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D8%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.15" data-doi="10.1016/j.bpj.2018.09.032">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sasaki
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kaya
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Higuchi
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A Unified Walking Model for Dimeric Motor Proteins
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biophysical Journal
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  115
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1981
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1992
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiophysical%2BJournal%26rft.volume%253D115%26rft.spage%253D1981%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bpj.2018.09.032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.bpj.2018.09.032&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.16" data-doi="10.1103/PhysRevLett.96.224501">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dudko
                                  </span>
                                  <span class="cit-name-given-names">
                                   OK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hummer
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Szabo
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Intrinsic rates and activation free energies from single-molecule pulling experiments
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Physical Review Letters
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  96
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhysical%2BReview%2BLetters%26rft.volume%253D96%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1103%252FPhysRevLett.96.224501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1103/PhysRevLett.96.224501&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.17" data-doi="10.1002/pro.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Na
                                  </span>
                                  <span class="cit-name-given-names">
                                   YR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Park
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Investigating protein unfolding kinetics by pulse proteolysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Protein Science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  268
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  276
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProtein%2BScience%26rft.stitle%253DProtein%2BSci.%26rft.aulast%253DNa%26rft.auinit1%253DY.%2BR.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D268%26rft.epage%253D276%26rft.atitle%253DInvestigating%2Bprotein%2Bunfolding%2Bkinetics%2Bby%2Bpulse%2Bproteolysis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpro.29%26rft_id%253Dinfo%253Apmid%252F19177560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/pro.29&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19177560&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F22%2F2020.11.17.386524.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.18">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   West
                                  </span>
                                  <span class="cit-name-given-names">
                                   DK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Olmsted
                                  </span>
                                  <span class="cit-name-given-names">
                                   PD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Paci
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Mechanical unfolding revisited through a simple but realistic model
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Chemical Physics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  124
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  15
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.19" data-doi="10.1111/tra.12639">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ohashi
                                  </span>
                                  <span class="cit-name-given-names">
                                   KG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Han
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mentley
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fricks
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hancock
                                  </span>
                                  <span class="cit-name-given-names">
                                   WO
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Load-dependent detachment kinetics plays a key role in bidirectional cargo transport by kinesin and dynein
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Traffic
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  284
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  294
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DTraffic%26rft.volume%253D20%26rft.spage%253D284%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftra.12639%26rft_id%253Dinfo%253Apmid%252F30809891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/tra.12639&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30809891&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F22%2F2020.11.17.386524.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.20" data-doi="10.1038/ncb3393">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Belyy
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schlager
                                  </span>
                                  <span class="cit-name-given-names">
                                   MA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Foster
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reimer
                                  </span>
                                  <span class="cit-name-given-names">
                                   AE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carter
                                  </span>
                                  <span class="cit-name-given-names">
                                   AP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yildiz
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The mammalian dynein-dynactin complex is a strong opponent to kinesin in a tug-of-war competition
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Cell Biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1018
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1024
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.volume%253D18%26rft.spage%253D1018%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb3393%26rft_id%253Dinfo%253Apmid%252F27454819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ncb3393&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27454819&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F22%2F2020.11.17.386524.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.21" data-doi="10.1073/pnas.95.14.8052">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Linke
                                  </span>
                                  <span class="cit-name-given-names">
                                   WA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ivemeyer
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mundel
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stockmeier
                                  </span>
                                  <span class="cit-name-given-names">
                                   MR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kolmerer
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Nature of PEVK-titin elasticity in skeletal muscle
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences of the United States of America
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1998
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  95
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  14
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  8052
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  8057
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLinke%26rft.auinit1%253DW.%2BA.%26rft.volume%253D95%26rft.issue%253D14%26rft.spage%253D8052%26rft.epage%253D8057%26rft.atitle%253DNature%2Bof%2BPEVK-titin%2Belasticity%2Bin%2Bskeletal%2Bmuscle%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.14.8052%26rft_id%253Dinfo%253Apmid%252F9653138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTUvMTQvODA1MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAzLzIyLzIwMjAuMTEuMTcuMzg2NTI0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.22" data-doi="10.1039/c9sm02484k">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hughes
                                  </span>
                                  <span class="cit-name-given-names">
                                   MDG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cussons
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mahmoudi
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brockwell
                                  </span>
                                  <span class="cit-name-given-names">
                                   DJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dougan
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Single molecule protein stabilisation translates to macromolecular mechanics of a protein network
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Soft Matter
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  27
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  6389
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6399
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSoft%2BMatter%26rft.volume%253D16%26rft.spage%253D6389%26rft_id%253Dinfo%253Adoi%252F10.1039%252Fc9sm02484k%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1039/c9sm02484k&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.23" data-doi="10.1016/j.bpj.2014.06.022">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sarlah
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vilfan
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The winch model can explain both coordinated and uncoordinated stepping of cytoplasmic dynein
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biophysical Journal
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  107
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  662
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  671
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiophysical%2BJournal%26rft.volume%253D107%26rft.spage%253D662%26rft.atitle%253DTHE%2BWINCH%2BMODEL%2BCAN%2BEXPLAIN%2BBOTH%2BCOORDINATED%2BAN%2BUNCOORDINATED%2BSTEPPING%2BOF%2BCYTOPLASMIC%2BDYNEIN%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bpj.2014.06.022%26rft_id%253Dinfo%253Apmid%252F25099805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.bpj.2014.06.022&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25099805&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F22%2F2020.11.17.386524.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.24" data-doi="10.1371/journal.pone.0179159">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sarlah
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vilfan
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Minimum requirements for motility of a processive motor protein
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS ONE
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  13
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D12%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0179159%26rft_id%253Dinfo%253Apmid%252F28957441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0179159&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28957441&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F22%2F2020.11.17.386524.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               25.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.25" data-doi="10.1063/1.2393232">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   West
                                  </span>
                                  <span class="cit-name-given-names">
                                   DK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Olmsted
                                  </span>
                                  <span class="cit-name-given-names">
                                   PD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Paci
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Free energy for protein folding from nonequilibrium simulations using the Jarzynski equality
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Chemical Physics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  125
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  20
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  7
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BChemical%2BPhysics%26rft.volume%253D125%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1063%252F1.2393232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1063/1.2393232&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               26.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   West
                                  </span>
                                  <span class="cit-name-given-names">
                                   DK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Paci
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Olmsted
                                  </span>
                                  <span class="cit-name-given-names">
                                   PD
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Internal protein dynamics shifts the distance to the mechanical transition state
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Physical Review E
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  74
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  061912
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhysical%2BReview%2BE%26rft.volume%253D74%26rft.spage%253D061912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               27.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.11.17.386524v3.27" data-doi="10.1103/PhysRevLett.101.248301">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Staple
                                  </span>
                                  <span class="cit-name-given-names">
                                   DB
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Payne
                                  </span>
                                  <span class="cit-name-given-names">
                                   SH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reddin
                                  </span>
                                  <span class="cit-name-given-names">
                                   ALC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kreuzer
                                  </span>
                                  <span class="cit-name-given-names">
                                   HJ
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Model for stretching and unfolding the giant multidomain muscle protein using single-molecule force spectroscopy
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Physical Review Letters
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  101
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  24
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhysical%2BReview%2BLetters%26rft.volume%253D101%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1103%252FPhysRevLett.101.248301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1103/PhysRevLett.101.248301&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               28.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.11.17.386524v3.28">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sikora
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sulkowska
                                  </span>
                                  <span class="cit-name-given-names">
                                   JI
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Witkowski
                                  </span>
                                  <span class="cit-name-given-names">
                                   BS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cieplak
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  BSDB: The biomolecule stretching database
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Research
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  39
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  SUPPL. 1
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               29.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.11.17.386524v3.29" data-doi="10.1021/acs.jpcb.5b11519">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Booth
                                  </span>
                                  <span class="cit-name-given-names">
                                   JJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shalashilin
                                  </span>
                                  <span class="cit-name-given-names">
                                   DV
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Fully Atomistic Simulations of Protein Unfolding in Low Speed Atomic Force Microscope and Force Clamp Experiments with the Help of Boxed Molecular Dynamics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Physical Chemistry B
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  120
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  700
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  708
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BPhysical%2BChemistry%2BB%26rft.volume%253D120%26rft.spage%253D700%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facs.jpcb.5b11519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1021/acs.jpcb.5b11519&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               30.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Takada
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Go model revisited
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biophysics and Physicobiology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  0
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  248
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  255
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiophysics%2Band%2BPhysicobiology%26rft.volume%253D16%26rft.spage%253D248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               31.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.31" data-doi="10.1126/science.347575">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bell
                                  </span>
                                  <span class="cit-name-given-names">
                                   GI
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Models for the specific adhesion of cells to cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1978
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  200
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4342
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  618
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  627
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBell%26rft.auinit1%253DG.%26rft.volume%253D200%26rft.issue%253D4342%26rft.spage%253D618%26rft.epage%253D627%26rft.atitle%253DModels%2Bfor%2Bthe%2Bspecific%2Badhesion%2Bof%2Bcells%2Bto%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.347575%26rft_id%253Dinfo%253Apmid%252F347575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyMDAvNDM0Mi82MTgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8yMi8yMDIwLjExLjE3LjM4NjUyNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               32.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.32">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hughes
                                  </span>
                                  <span class="cit-name-given-names">
                                   ML
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dougan
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The physics of pulling polyproteins: A review of single molecule force spectroscopy using the AFM to study protein unfolding
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Reports on Progress in Physics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  79
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  076601
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DReports%2Bon%2BProgress%2Bin%2BPhysics%26rft.volume%253D79%26rft.spage%253D076601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               33.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.33">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hughes
                                  </span>
                                  <span class="cit-name-given-names">
                                   MDG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hanson
                                  </span>
                                  <span class="cit-name-given-names">
                                   BS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cussons
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mahmoudi
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brockwell
                                  </span>
                                  <span class="cit-name-given-names">
                                   DJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dougan
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Control of Nanoscale in Situ Protein Unfolding Defines Network Architecture and Mechanics of Protein Hydrogels
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  ACS Nano
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  11296
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  11308
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DACS%2BNano%26rft.volume%253D15%26rft.spage%253D11296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               34.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.34">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Imai
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shima
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sutoh
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Walker
                                  </span>
                                  <span class="cit-name-given-names">
                                   ML
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Knight
                                  </span>
                                  <span class="cit-name-given-names">
                                   PJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kon
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Direct observation shows superposition and large scale flexibility within cytoplasmic dynein motors moving along microtubules
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Communications
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  11
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D6%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               35.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.35" data-doi="10.1093/protein/gzw022">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kamiya
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mashimo
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Takano
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kon
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kurisu
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nakamura
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Elastic properties of dynein motor domain obtained from all-atom molecular dynamics simulations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Protein Engineering, Design and Selection
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  29
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  317
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  326
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProtein%2BEngineering%252C%2BDesign%2Band%2BSelection%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fprotein%252Fgzw022%26rft_id%253Dinfo%253Apmid%252F27334455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/protein/gzw022&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27334455&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F22%2F2020.11.17.386524.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               36.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.36">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Niekamp
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stuurman
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vale
                                  </span>
                                  <span class="cit-name-given-names">
                                   RD
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Three-color single-molecule imaging reveals conformational dynamics of dynein undergoing motility
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences of the United States of America
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  118
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  31
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               37.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.37" data-doi="10.1016/j.jmb.2008.02.025">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bertz
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rief
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Mechanical Unfoldons as Building Blocks of Maltose-binding Protein
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Molecular Biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  378
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  447
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  458
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBertz%26rft.auinit1%253DM.%26rft.volume%253D378%26rft.issue%253D2%26rft.spage%253D447%26rft.epage%253D458%26rft.atitle%253DMechanical%2Bunfoldons%2Bas%2Bbuilding%2Bblocks%2Bof%2Bmaltose-binding%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2008.02.025%26rft_id%253Dinfo%253Apmid%252F18355837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.jmb.2008.02.025&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18355837&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F22%2F2020.11.17.386524.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000255368400014&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               38.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.38" data-doi="10.1021/ja710526r">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wood
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Frölich
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Paciaroni
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moulin
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Härtlein
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zaccai
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Coincidence of dynamical transitions in a soluble protein and its hydration water: Direct measurements by neutron scattering and MD simulations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of the American Chemical Society
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  130
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  14
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  4586
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4587
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.stitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.aulast%253DWood%26rft.auinit1%253DK.%26rft.volume%253D130%26rft.issue%253D14%26rft.spage%253D4586%26rft.epage%253D4587%26rft.atitle%253DCoincidence%2Bof%2Bdynamical%2Btransitions%2Bin%2Ba%2Bsoluble%2Bprotein%2Band%2Bits%2Bhydration%2Bwater%253A%2Bdirect%2Bmeasurements%2Bby%2Bneutron%2Bscattering%2Band%2BMD%2Bsimulations.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fja710526r%26rft_id%253Dinfo%253Apmid%252F18338890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1021/ja710526r&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18338890&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F22%2F2020.11.17.386524.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000254643900007&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               39.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.39" data-doi="10.1074/jbc.M301004200">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Telmer
                                  </span>
                                  <span class="cit-name-given-names">
                                   PG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shilton
                                  </span>
                                  <span class="cit-name-given-names">
                                   BH
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Insights into the Conformational Equilibria of Maltose-binding Protein by Analysis of High Affinity Mutants
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Biological Chemistry
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  278
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  36
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  34555
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  34567
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M301004200%26rft_id%253Dinfo%253Apmid%252F12794084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzgvMzYvMzQ1NTUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8yMi8yMDIwLjExLjE3LjM4NjUyNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               40.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.40">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hoffmann
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dougan
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Single molecule force spectroscopy using polyproteins
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Chemical Society Reviews
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  41
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  14
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  4781
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4796
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DChemical%2BSociety%2BReviews%26rft.volume%253D41%26rft.spage%253D4781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               41.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.41" data-doi="10.1039/c3cp52142g">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hoffmann
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tych
                                  </span>
                                  <span class="cit-name-given-names">
                                   KM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hughes
                                  </span>
                                  <span class="cit-name-given-names">
                                   ML
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brockwell
                                  </span>
                                  <span class="cit-name-given-names">
                                   DJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dougan
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Towards design principles for determining the mechanical stability of proteins
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Physical Chemistry Chemical Physics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  38
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  15767
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  15780
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhysical%2BChemistry%2BChemical%2BPhysics%26rft.volume%253D15%26rft.spage%253D15767%26rft_id%253Dinfo%253Adoi%252F10.1039%252Fc3cp52142g%26rft_id%253Dinfo%253Apmid%252F23989058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1039/c3cp52142g&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23989058&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F22%2F2020.11.17.386524.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               42.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.11.17.386524v3.42" data-doi="10.1103/PhysRevE.91.022710">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tych
                                  </span>
                                  <span class="cit-name-given-names">
                                   KM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hughes
                                  </span>
                                  <span class="cit-name-given-names">
                                   ML
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bourke
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Taniguchi
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kawakami
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brockwell
                                  </span>
                                  <span class="cit-name-given-names">
                                   DJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Optimizing the calculation of energy landscape parameters from single-molecule protein unfolding experiments
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Physical Review E - Statistical, Nonlinear, and Soft Matter Physics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  91
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  9
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhysical%2BReview%2BE%2B-%2BStatistical%252C%2BNonlinear%252C%2Band%2BSoft%2BMatter%2BPhysics%26rft.volume%253D91%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1103%252FPhysRevE.91.022710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1103/PhysRevE.91.022710&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               43.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.43" data-doi="10.1021/acs.biomac.6b01877">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Da Silva
                                  </span>
                                  <span class="cit-name-given-names">
                                   MA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenton
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hughes
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brockwell
                                  </span>
                                  <span class="cit-name-given-names">
                                   DJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dougan
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Assessing the Potential of Folded Globular Polyproteins As Hydrogel Building Blocks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biomacromolecules
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  636
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  646
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiomacromolecules%26rft.volume%253D18%26rft.spage%253D636%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facs.biomac.6b01877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1021/acs.biomac.6b01877&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               44.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.44" data-doi="10.1073/pnas.1300596110">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stirnemann
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Giganti
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fernandez
                                  </span>
                                  <span class="cit-name-given-names">
                                   JM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Berne
                                  </span>
                                  <span class="cit-name-given-names">
                                   BJ
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Elasticity, structure, and relaxation of extended proteins under force
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences of the United States of America
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  110
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  3847
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3852
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1300596110%26rft_id%253Dinfo%253Apmid%252F23407163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzEwLzM4NDciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8yMi8yMDIwLjExLjE3LjM4NjUyNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               45.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.45">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Phillips
                                  </span>
                                  <span class="cit-name-given-names">
                                   JC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hardy
                                  </span>
                                  <span class="cit-name-given-names">
                                   DJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maia
                                  </span>
                                  <span class="cit-name-given-names">
                                   JDC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stone
                                  </span>
                                  <span class="cit-name-given-names">
                                   JE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ribeiro
                                  </span>
                                  <span class="cit-name-given-names">
                                   JV
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bernardi
                                  </span>
                                  <span class="cit-name-given-names">
                                   RC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Scalable molecular dynamics on CPU and GPU architectures with NAMD
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Chemical Physics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  153
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               46.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.46">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mücksch
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Urbassek
                                  </span>
                                  <span class="cit-name-given-names">
                                   HM
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Accelerating Steered Molecular Dynamics: Toward Smaller Velocities in Forced Unfolding Simulations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Chemical Theory and Computation
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1380
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1384
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BChemical%2BTheory%2Band%2BComputation%26rft.volume%253D12%26rft.spage%253D1380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               47.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.47" data-doi="10.1110/ps.0224602">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zinober
                                  </span>
                                  <span class="cit-name-given-names">
                                   RC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brockwell
                                  </span>
                                  <span class="cit-name-given-names">
                                   DJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Beddard
                                  </span>
                                  <span class="cit-name-given-names">
                                   GS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blake
                                  </span>
                                  <span class="cit-name-given-names">
                                   AW
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Olmsted
                                  </span>
                                  <span class="cit-name-given-names">
                                   PD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Radford
                                  </span>
                                  <span class="cit-name-given-names">
                                   SE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Mechanically unfolding proteins: The effect of unfolding history and the supramolecular scaffold
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Protein Science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2759
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2765
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProtein%2BScience%26rft.volume%253D11%26rft.spage%253D2759%26rft_id%253Dinfo%253Adoi%252F10.1110%252Fps.0224602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1110/ps.0224602&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               48.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.48">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Uversky
                                  </span>
                                  <span class="cit-name-given-names">
                                   VN
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Intrinsically disordered proteins and their ‘‘Mysterious” (meta)physics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Frontiers in Physics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  8
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  23
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BPhysics%26rft.volume%253D7%26rft.spage%253D8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               49.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.49">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Feller
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Protein stability and enzyme activity at extreme biological temperatures
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Physics Condensed Matter
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  32
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               50.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.11.17.386524v3.50">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reed
                                  </span>
                                  <span class="cit-name-given-names">
                                   CJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lewis
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Trejo
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Winston
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Evilia
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Protein adaptations in archaeal extremophiles
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Archaea
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;.
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               51.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.51">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brininger
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Spradlin
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cobani
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Evilia
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The more adaptive to change, the more likely you are to survive: Protein adaptation in extremophiles
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Seminars in Cell and Developmental Biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  84
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  158
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  169
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSeminars%2Bin%2BCell%2Band%2BDevelopmental%2BBiology%26rft.volume%253D84%26rft.spage%253D158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               52.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.52" data-doi="10.1146/annurev.biophys.30.1.105">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Evans
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Probing the Relation Between Force, Lifetime and Chemistry in Single Molecular Bonds
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Annual Review of Biophysics and Biomolecular Structure
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  105
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  128
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiophysics%2Band%2Bbiomolecular%2Bstructure%26rft.stitle%253DAnnu%2BRev%2BBiophys%2BBiomol%2BStruct%26rft.aulast%253DEvans%26rft.auinit1%253DE.%26rft.volume%253D30%26rft.issue%253D1%26rft.spage%253D105%26rft.epage%253D128%26rft.atitle%253DPROBING%2BTHE%2BRELATION%2BBETWEEN%2BFORCE--LIFETIME--AND%2BCHEMISTRY%2BIN%2BSINGLE%2BMOLECULAR%2BBONDS%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biophys.30.1.105%26rft_id%253Dinfo%253Apmid%252F11340054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1146/annurev.biophys.30.1.105&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=11340054&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F22%2F2020.11.17.386524.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000169531600006&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               53.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.53">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Isojima
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Iino
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Niitani
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Noji
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tomishige
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Direct observation of intermediate states during the stepping motion of kinesin-1
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Chemical Biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  290
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  297
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BChemical%2BBiology%26rft.volume%253D12%26rft.spage%253D290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               54.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.11.17.386524v3.54">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bowman
                                  </span>
                                  <span class="cit-name-given-names">
                                   GR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pande
                                  </span>
                                  <span class="cit-name-given-names">
                                   VS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Noe
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  An Introduction to Markov State Models and Their Application to Long Timescale Molecular Simulation
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 ;
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               55.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.55" data-doi="10.1021/jacs.7b12191">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Husic
                                  </span>
                                  <span class="cit-name-given-names">
                                   BE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pande
                                  </span>
                                  <span class="cit-name-given-names">
                                   VS
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Markov State Models: From an Art to a Science
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of the American Chemical Society
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  140
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2386
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2396
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.volume%253D140%26rft.spage%253D2386%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjacs.7b12191%26rft_id%253Dinfo%253Apmid%252F29323881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1021/jacs.7b12191&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29323881&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F22%2F2020.11.17.386524.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               56.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.56" data-doi="10.1073/pnas.85.16.5908">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bialek
                                  </span>
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Onuchic
                                  </span>
                                  <span class="cit-name-given-names">
                                   JN
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Protein dynamics and reaction rates: mode-specific chemistry in large molecules?
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences of the United States of America
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1988
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  85
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  16
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  5908
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  5912
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBialek%26rft.auinit1%253DW.%26rft.volume%253D85%26rft.issue%253D16%26rft.spage%253D5908%26rft.epage%253D5912%26rft.atitle%253DProtein%2Bdynamics%2Band%2Breaction%2Brates%253A%2Bmode-specific%2Bchemistry%2Bin%2Blarge%2Bmolecules%253F%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.85.16.5908%26rft_id%253Dinfo%253Apmid%252F3413064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODUvMTYvNTkwOCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAzLzIyLzIwMjAuMTEuMTcuMzg2NTI0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               57.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.57" data-doi="10.1038/nsb968">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brockwell
                                  </span>
                                  <span class="cit-name-given-names">
                                   DJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Paci
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zinober
                                  </span>
                                  <span class="cit-name-given-names">
                                   RC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Beddard
                                  </span>
                                  <span class="cit-name-given-names">
                                   GS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Olmsted
                                  </span>
                                  <span class="cit-name-given-names">
                                   PD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                  <span class="cit-name-given-names">
                                   DA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Pulling geometry defines the mechanical resistance of a beta-sheet protein
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Structural Biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  731
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  737
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2Bbiology%26rft.stitle%253DNat%2BStruct%2BBiol%26rft.aulast%253DBrockwell%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D10%26rft.issue%253D9%26rft.spage%253D731%26rft.epage%253D737%26rft.atitle%253DPulling%2Bgeometry%2Bdefines%2Bthe%2Bmechanical%2Bresistance%2Bof%2Ba%2Bbeta-sheet%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsb968%26rft_id%253Dinfo%253Apmid%252F12923573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nsb968&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=12923573&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F22%2F2020.11.17.386524.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000184970700015&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               58.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.11.17.386524v3.58" data-doi="10.1038/nsb965">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carrion-Vazquez
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lu
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marszalek
                                  </span>
                                  <span class="cit-name-given-names">
                                   PE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Oberhauser
                                  </span>
                                  <span class="cit-name-given-names">
                                   AF
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fernandez
                                  </span>
                                  <span class="cit-name-given-names">
                                   JM
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The mechanical stability of ubiquitin is linkage dependent
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Structural Biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  738
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  743
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2Bbiology%26rft.stitle%253DNat%2BStruct%2BBiol%26rft.aulast%253DCarrion-Vazquez%26rft.auinit1%253DM.%26rft.volume%253D10%26rft.issue%253D9%26rft.spage%253D738%26rft.epage%253D743%26rft.atitle%253DThe%2Bmechanical%2Bstability%2Bof%2Bubiquitin%2Bis%2Blinkage%2Bdependent.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsb965%26rft_id%253Dinfo%253Apmid%252F12923571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nsb965&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=12923571&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F22%2F2020.11.17.386524.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000184970700016&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               59.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.59" data-doi="10.1073/pnas.0602995103">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dietz
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Berkemeier
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bertz
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rief
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Anisotropic deformation response of single protein molecules
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences of the United States of America
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  103
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  34
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  12724
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  12728
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0602995103%26rft_id%253Dinfo%253Apmid%252F16908850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzM0LzEyNzI0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMjIvMjAyMC4xMS4xNy4zODY1MjQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               60.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.17.386524v3.60" data-doi="10.1098/rsos.171568">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Trott
                                  </span>
                                  <span class="cit-name-given-names">
                                   LE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hafezparast
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Madzvamuse
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A mathematical understanding of how cytoplasmic dynein walks on microtubules
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Royal Society Open Science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  171568
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DRoyal%2BSociety%2BOpen%2BScience%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsos.171568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1098/rsos.171568&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.11.17.386524v3" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.03.21.484343v1" title="Interacting active surfaces: a model for three-dimensional cell aggregates" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.03.20.485043v1" title="Quantitation, base identity and distribution of stably incorporated ribonucleotides in nuclear and mitochondrial DNA from murine tissues" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted March 22, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.11.17.386524v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.11.17.386524v3.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.11.17.386524v3.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.11.17.386524v3" class="active" data-icon-position="" data-hide-link-title="0">
                                   A generalised mechano-kinetic model for use in multiscale simulation protocols
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2460219"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-qoAzQ_8o8vmKYCuJFnZlxpvJucu8XeD2WlBC4BBASaQ"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484486327"/>
                                   <input type="hidden" name="captcha_token" value="df5ef9b3c4810bf17016e4dddb2f86a1"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2460219--3" data-node-nid="2460219" id="node-2460219--41390657482" data-pisa="biorxiv;2020.11.17.386524v3" data-pisa-master="biorxiv;2020.11.17.386524" data-seqnum="2460219" data-apath="/biorxiv/early/2022/03/22/2020.11.17.386524.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 A generalised mechano-kinetic model for use in multiscale simulation protocols
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Benjamin S.
                                  </span>
                                  <span class="nlm-surname">
                                   Hanson
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Lorna
                                  </span>
                                  <span class="nlm-surname">
                                   Dougan
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Oliver G.
                                  </span>
                                  <span class="nlm-surname">
                                   Harlen
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Sarah A.
                                  </span>
                                  <span class="nlm-surname">
                                   Harris
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Daniel J.
                                  </span>
                                  <span class="nlm-surname">
                                   Read
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.11.17.386524;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.11.17.386524
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.11.17.386524v3" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.11.17.386524v3&title=A%20generalised%20mechano-kinetic%20model%20for%20use%20in%20multiscale%20simulation%20protocols" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.17.386524v3&title=A%20generalised%20mechano-kinetic%20model%20for%20use%20in%20multiscale%20simulation%20protocols" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.17.386524v3&text=A%20generalised%20mechano-kinetic%20model%20for%20use%20in%20multiscale%20simulation%20protocols" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.11.17.386524v3&t=A%20generalised%20mechano-kinetic%20model%20for%20use%20in%20multiscale%20simulation%20protocols" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.11.17.386524v3&title=A%20generalised%20mechano-kinetic%20model%20for%20use%20in%20multiscale%20simulation%20protocols" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.11.17.386524v3&title=A%20generalised%20mechano-kinetic%20model%20for%20use%20in%20multiscale%20simulation%20protocols&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.17.386524v3&title=A%20generalised%20mechano-kinetic%20model%20for%20use%20in%20multiscale%20simulation%20protocols" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2460219--5" data-node-nid="2460219" id="citation-node-2460219--6458719482" data-pisa="biorxiv;2020.11.17.386524v3" data-pisa-master="biorxiv;2020.11.17.386524" data-seqnum="2460219" data-apath="/biorxiv/early/2022/03/22/2020.11.17.386524.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     A generalised mechano-kinetic model for use in multiscale simulation protocols
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Benjamin S.
                                      </span>
                                      <span class="nlm-surname">
                                       Hanson
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Lorna
                                      </span>
                                      <span class="nlm-surname">
                                       Dougan
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Oliver G.
                                      </span>
                                      <span class="nlm-surname">
                                       Harlen
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Sarah A.
                                      </span>
                                      <span class="nlm-surname">
                                       Harris
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Daniel J.
                                      </span>
                                      <span class="nlm-surname">
                                       Read
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.11.17.386524;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.11.17.386524
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=A%20generalised%20mechano-kinetic%20model%20for%20use%20in%20multiscale%20simulation%20protocols&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Hanson%2C+Benjamin+S.&rft.au=Dougan%2C+Lorna&rft.au=Harlen%2C+Oliver+G.&rft.au=Harris%2C+Sarah+A.&rft.au=Read%2C+Daniel+J.">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2460219/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2460219/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2460219/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2460219/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2460219/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2460219/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2460219/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2460219/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2460219/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2460219/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2460219/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2460219/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.17.386524v3&count=horizontal&via=&text=A%20generalised%20mechano-kinetic%20model%20for%20use%20in%20multiscale%20simulation%20protocols&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.11.17.386524v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.11.17.386524v3&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.11.17.386524v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/biophysics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Biophysics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23507)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11824)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11440)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15189)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47703)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
