<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.07.14.201475v4.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.07.14.201475v4.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.07.14.201475v4.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/01/02/2020.07.14.201475.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.07.14.201475v4"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="MAVE-NN: learning genotype-phenotype maps from multiplex assays of variant effect"/>
  <meta name="DC.Identifier" content="10.1101/2020.07.14.201475"/>
  <meta name="DC.Date" content="2022-01-02"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Multiplex assays of variant effect (MAVEs) are a family of methods that includes deep mutational scanning (DMS) experiments on proteins and massively parallel reporter assays (MPRAs) on gene regulatory sequences. However, a general strategy for inferring quantitative models of genotype-phenotype (G-P) maps from MAVE data is lacking. Here we introduce MAVE-NN, a neural-network-based Python package that implements a broadly applicable information-theoretic framework for learning G-P maps—including biophysically interpretable models—from MAVE datasets. We demonstrate MAVE-NN in multiple biological contexts, and highlight the ability of our approach to deconvolve mutational effects from otherwise confounding experimental nonlinearities and noise.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Ammar Tareen"/>
  <meta name="DC.Contributor" content="Mahdi Kooshkbaghi"/>
  <meta name="DC.Contributor" content="Anna Posfai"/>
  <meta name="DC.Contributor" content="William T. Ireland"/>
  <meta name="DC.Contributor" content="David M. McCandlish"/>
  <meta name="DC.Contributor" content="Justin B. Kinney"/>
  <meta name="article:published_time" content="2022-01-02"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="MAVE-NN: learning genotype-phenotype maps from multiplex assays of variant effect"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Multiplex assays of variant effect (MAVEs) are a family of methods that includes deep mutational scanning (DMS) experiments on proteins and massively parallel reporter assays (MPRAs) on gene regulatory sequences. However, a general strategy for inferring quantitative models of genotype-phenotype (G-P) maps from MAVE data is lacking. Here we introduce MAVE-NN, a neural-network-based Python package that implements a broadly applicable information-theoretic framework for learning G-P maps—including biophysically interpretable models—from MAVE datasets. We demonstrate MAVE-NN in multiple biological contexts, and highlight the ability of our approach to deconvolve mutational effects from otherwise confounding experimental nonlinearities and noise.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.07.14.201475v4"/>
  <meta name="citation_id" content="2020.07.14.201475v4"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.07.14.201475v4"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.07.14.201475v4.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.07.14.201475v4.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.07.14.201475"/>
  <meta name="citation_num_pages" content="48"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.07.14.201475"/>
  <meta name="citation_author" content="Ammar Tareen"/>
  <meta name="citation_author_institution" content="Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory"/>
  <meta name="citation_author" content="Mahdi Kooshkbaghi"/>
  <meta name="citation_author_institution" content="Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory"/>
  <meta name="citation_author" content="Anna Posfai"/>
  <meta name="citation_author_institution" content="Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory"/>
  <meta name="citation_author" content="William T. Ireland"/>
  <meta name="citation_author_institution" content="Department of Physics, California Institute of Technology"/>
  <meta name="citation_author_institution" content="Department of Applied Physics, Harvard University"/>
  <meta name="citation_author" content="David M. McCandlish"/>
  <meta name="citation_author_institution" content="Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory"/>
  <meta name="citation_author" content="Justin B. Kinney"/>
  <meta name="citation_author_institution" content="Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory"/>
  <meta name="citation_author_email" content="jkinney@cshl.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1897-3778"/>
  <meta name="citation_reference" content="Kinney, J. B. & McCandlish, D. M. Massively parallel assays and quantitative sequence-function relationships. Annu Rev Genom Hum G 20, 99–127 (2019)."/>
  <meta name="citation_reference" content="Starita, L. M. et al. Variant interpretation: functional assays to the rescue. Am J Hum Genetics 101, 315–325 (2017)."/>
  <meta name="citation_reference" content="Fowler, D. M. & Fields, S. Deep mutational scanning: a new style of protein science. Nat Methods 11, 801–807 (2014)."/>
  <meta name="citation_reference" content="Levo, M. & Segal, E. In pursuit of design principles of regulatory sequences. Nat Rev Genet 15, 453–468 (2014)."/>
  <meta name="citation_reference" content="White, M. A. Understanding how cis-regulatory function is encoded in DNA sequence using massively parallel reporter assays and designed sequences. Genomics 106, 165–170 (2015)."/>
  <meta name="citation_reference" content="Inoue, F. & Ahituv, N. Decoding enhancers using massively parallel reporter assays. Genomics 106, 159–164 (2015)."/>
  <meta name="citation_reference" content="Peterman, N. & Levine, E. Sort-seq under the hood: implications of design choices on large-scale characterization of sequence-function relations. BMC Genomics 17, 206 (2016)."/>
  <meta name="citation_reference" content="Fowler, D. M., Araya, C. L., Gerard, W. & Fields, S. Enrich: software for analysis of protein function by enrichment and depletion of variants. Bioinformatics 27, 3430–3431 (2011)."/>
  <meta name="citation_reference" content="Alam, K. K., Chang, J. L. & Burke, D. H. FASTAptamer: A bioinformatic toolkit for high-throughput sequence analysis of combinatorial selections. Mol Ther-Nucleic Acids 4, e230 (2015)."/>
  <meta name="citation_reference" content="Bloom, J. D. Software for the analysis and visualization of deep mutational scanning data. BMC Bioinformatics 16, 168 (2015)."/>
  <meta name="citation_reference" content="Rubin, A. F. et al. A statistical framework for analyzing deep mutational scanning data. Genome Biol 18, 1–15 (2017)."/>
  <meta name="citation_reference" content="Ashuach, T. et al. MPRAnalyze: statistical framework for massively parallel reporter assays. Genome Biol 20, 183 (2019)."/>
  <meta name="citation_reference" content="Niroula, A., Ajore, R. & Nilsson, B. MPRAscore: robust and non-parametric analysis of massively parallel reporter assays. Bioinformatics 35, 5351–5353 (2019)."/>
  <meta name="citation_reference" content="Faure, A. J., Schmiedel, J. M., Baeza-Centurion, P. & Lehner, B. DiMSum: an error model and pipeline for analyzing deep mutational scanning data and diagnosing common experimental pathologies. Genome Biol 21, 207 (2020)."/>
  <meta name="citation_reference" content="Atwal, G. S. & Kinney, J. B. Learning quantitative sequence-function relationships from massively parallel experiments. J Stat Phys 162, 1203–1243 (2016)."/>
  <meta name="citation_reference" content="Kinney, J. B., Murugan, A., Callan, C. G. & Cox, E. C. Using deep sequencing to characterize the biophysical mechanism of a transcriptional regulatory sequence. Proc Natl Acad Sci USA 107, 9158–9163 (2010)."/>
  <meta name="citation_reference" content="Melnikov, A. et al. Systematic dissection and optimization of inducible enhancers in human cells using a massively parallel reporter assay. Nat Biotechnol 30, 271–277 (2012)."/>
  <meta name="citation_reference" content="Mogno, I., Kwasnieski, J. C. & Cohen, B. A. Massively parallel synthetic promoter assays reveal the in vivo effects of binding site variants. Genome Res 23, 1908–1915 (2013)."/>
  <meta name="citation_reference" content="Abadi, M. et al. TensorFlow: A Systems for Large-Scale Machine Learning. in Proceedings of the 12th USENIX Symposium on Operating Systems Design and Implementation (OSDI ‘16) (2016)."/>
  <meta name="citation_reference" content="Kinney, J. B. & Atwal, G. S. Parametric inference in the large data limit using maximally informative models. Neural Comput 26, 637–653 (2014)."/>
  <meta name="citation_reference" content="Kinney, J. B., Tkačik, G. & Callan, C. G. Precise physical models of protein-DNA interaction from high-throughput data. Proc Natl Acad Sci USA 104, 501–506 (2007)."/>
  <meta name="citation_reference" content="Otwinowski, J. & Nemenman, I. Genotype to phenotype mapping and the fitness landscape of the E. coli lac promoter. PLoS ONE 8, e61570 (2013)."/>
  <meta name="citation_reference" content="Sarkisyan, K. S. et al. Local fitness landscape of the green fluorescent protein. Nature 533, 397–401 (2016)."/>
  <meta name="citation_reference" content="Sailer, Z. R. & Harms, M. J. Detecting high-order epistasis in nonlinear genotype-phenotype maps. Genetics 205, 1079–1088 (2017)."/>
  <meta name="citation_reference" content="Otwinowski, J., McCandlish, D. M. & Plotkin, J. B. Inferring the shape of global epistasis. Proc Natl Acad Sci USA 115, E7550–E7558 (2018)."/>
  <meta name="citation_reference" content="Gelman, S., Fahlberg, S. A., Heinzelman, P., Romero, P. A. & Gitter, A. Neural networks to learn protein sequence-function relationships from deep mutational scanning data. Proc Natl Acad Sci USA 118, e2104878118 (2021)."/>
  <meta name="citation_reference" content="Faure, A. J. et al. Global mapping of the energetic and allosteric landscapes of protein binding domains. bioRxiv doi:10.1101/2021.09.14.460249 (2021)"/>
  <meta name="citation_reference" content="Tonner, P. D., Pressman, A. & Ross, D. Interpretable modeling of genotype-phenotype landscapes with state-of-the-art predictive power. bioRxiv doi:10.1101/2021.06.11.448129 (2021)."/>
  <meta name="citation_reference" content="Jones, M. C. & Faddy, M. J. A skew extension of the t-distribution, with applications. J Roy Stat Soc B 65, 159–174 (2003)."/>
  <meta name="citation_reference" content="Kinney, J. B. & Atwal, G. S. Equitability, mutual information, and the maximal information coefficient. Proc Natl Acad Sci USA 111, 3354–3359 (2014)."/>
  <meta name="citation_reference" content="Cover, T. M. & Thomas, J. A. Elements of information theory. (Wiley, 2006)."/>
  <meta name="citation_reference" content="Barber, D. & Agakov, F. The IM algorithm: a variational approach to information maximization. Advances in neural information processing systems 16. (2004)."/>
  <meta name="citation_reference" content="Olson, C. A., Wu, N. C. & Sun, R. A comprehensive biophysical description of pairwise epistasis throughout an entire protein domain. Curr Biol 24, 2643–2651 (2014)."/>
  <meta name="citation_reference" content="Seuma, M., Faure, A., Badia, M., Lehner, B. & Bolognesi, B. The genetic landscape for amyloid beta fibril nucleation accurately discriminates familial Alzheimer’s disease mutations. eLife 10, e63364 (2021)."/>
  <meta name="citation_reference" content="Bolognesi, B. et al. The mutational landscape of a prion-like domain. Nat Commun 10, 4162 (2019)."/>
  <meta name="citation_reference" content="Wong, M. S., Kinney, J. B. & Krainer, A. R. Quantitative activity profile and context dependence of all human 5’ splice sites. Mol Cell 71, 1012-1026.e3 (2018)."/>
  <meta name="citation_reference" content="Bintu, L. et al. Transcriptional regulation by the numbers: models. Curr Opin Genet Dev 15, 116–124 (2005)."/>
  <meta name="citation_reference" content="Sherman, M. S. & Cohen, B. A. Thermodynamic state ensemble models of cis-regulation. PLoS Comput Biol 8, e1002407 (2012)."/>
  <meta name="citation_reference" content="Wong, F. & Gunawardena, J. Gene Regulation in and out of equilibrium. Annu Rev Biophys 49, 199–226 (2020)."/>
  <meta name="citation_reference" content="Otwinowski, J. Biophysical inference of epistasis and the effects of mutations on protein stability and function. Mol Biol Evol 35, 2345–2354 (2018)."/>
  <meta name="citation_reference" content="Manhart, M. & Morozov, A. V. Protein folding and binding can emerge as evolutionary spandrels through structural coupling. Proc Natl Acad Sci USA 112, 1797–1802 (2015)."/>
  <meta name="citation_reference" content="Nisthal, A., Wang, C. Y., Ary, M. L. & Mayo, S. L. Protein stability engineering insights revealed by domain-wide comprehensive mutagenesis. Proc Natl Acad Sci USA 116, 16367–16377 (2019)."/>
  <meta name="citation_reference" content="Pokusaeva, V. O. et al. An experimental assay of the interactions of amino acids from orthologous sequences shaping a complex fitness landscape. PLoS Genet 15, e1008079 (2019)."/>
  <meta name="citation_reference" content="Tareen, A. & Kinney, J. B. Logomaker: beautiful sequence logos in Python. Bioinformatics 36, 2272–2274 (2020)."/>
  <meta name="citation_reference" content="Steeg, G. V. Non-Parametric Entropy Estimation Toolbox (NPEET). https://www.isi.edu/~gregv/npeet.html (2014)."/>
  <meta name="citation_reference" content="Vasicek, O. A test for normality based on sample entropy. J Roy Stat Soc B 38, 54–59 (1976)."/>
  <meta name="citation_reference" content="Kraskov, A., Stögbauer, H. & Grassberger, P. Estimating mutual information. Phys Rev E 69, 066138 (2004)."/>
  <meta name="citation_reference" content="Sharpee, T., Rust, N. C. & Bialek, W. Analyzing neural responses to natural signals: maximally informative dimensions. Neural Comput 16, 223–250 (2004)."/>
  <meta name="citation_reference" content="Efron, B. Bootstrap methods: another look at the jackknife. Ann Stat 7, 1–26 (1979)."/>
  <meta name="citation_reference" content="Efron, B. & Tibshirani, R. Bootstrap methods for standard errors, confidence intervals, and other measures of statistical accuracy. Stat Sci 1, 54–75 (1986)."/>
  <meta name="twitter:title" content="MAVE-NN: learning genotype-phenotype maps from multiplex assays of variant effect"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Multiplex assays of variant effect (MAVEs) are a family of methods that includes deep mutational scanning (DMS) experiments on proteins and massively parallel reporter assays (MPRAs) on gene regulatory sequences. However, a general strategy for inferring quantitative models of genotype-phenotype (G-P) maps from MAVE data is lacking. Here we introduce MAVE-NN, a neural-network-based Python package that implements a broadly applicable information-theoretic framework for learning G-P maps—including biophysically interpretable models—from MAVE datasets. We demonstrate MAVE-NN in multiple biological contexts, and highlight the ability of our approach to deconvolve mutational effects from otherwise confounding experimental nonlinearities and noise.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="MAVE-NN: learning genotype-phenotype maps from multiplex assays of variant effect"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.07.14.201475v4"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Multiplex assays of variant effect (MAVEs) are a family of methods that includes deep mutational scanning (DMS) experiments on proteins and massively parallel reporter assays (MPRAs) on gene regulatory sequences. However, a general strategy for inferring quantitative models of genotype-phenotype (G-P) maps from MAVE data is lacking. Here we introduce MAVE-NN, a neural-network-based Python package that implements a broadly applicable information-theoretic framework for learning G-P maps—including biophysically interpretable models—from MAVE datasets. We demonstrate MAVE-NN in multiple biological contexts, and highlight the ability of our approach to deconvolve mutational effects from otherwise confounding experimental nonlinearities and noise.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-01-02"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.07.14.201475v4"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2330666"/>
  <title>
   MAVE-NN: learning genotype-phenotype maps from multiplex assays of variant effect | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"VYO44HbsgrdWUlOvQ4pjbYETSf-cd8Wz2oYcYHLXDNQ"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2330666","apath":"\/biorxiv\/early\/2022\/01\/02\/2020.07.14.201475.atom","pisa":"biorxiv;2020.07.14.201475v4","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.07.14.201475v4"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.07.14.201475v4","title":"MAVE-NN: learning genotype-phenotype maps from multiplex assays of variant effect","identifier":"node\/2330666"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HPD+JekR\\\\\\\/Rz7MUcYkb\\\\\\\/6ElcyqxOKNUZZc5ZjE\\\\\\\/DhYuxtbV2s768lnwiOqJt\\\\\\\/3gaty8cJXSQny5LJ8kxzMXW92Bu8LXtEcJrl04gV\\\\\\\/5DNLXj6yhwnYKEkropQX+Nd+3qkARTL3ymN9kXAbJzeCSYw6P9z5jW5wnaCK5PSXrBmlBwHTkGrxuJ1E8rGD7xwSn2w\\\\\\\/BDYGh47Mv4kMjl50zSwa8V8vMaspd7XIJZ+mg9iRLBXJ5iAJ0344MNOFRtlyuUbTSSrUCY2PKhClt2MR4mzzLMytZWwnwXJklGUzDP0yvqrQ8nwtB1PF\\\\\\\/imV0BnvAX37FqXdW20B1xAf5e278bmeJgos\\\\\\\/2JpM6hSjBCH6BhZb+cl4GvO\\\\\\\/yG3GMfihxztv\\\\\\\/qioAXVZ4GnH+bGdJrgm+Gi9sp5ICZNJTZx1pNixhNfEZ3IgZnq+eeVOyK7w06Qn4x0B02XSq8r9u4LcV+4YmXJstT8LI56LDr9DkaW4b2ANJKW0LeVTDi2Ub5RrESUzyFlF2wDFmRaUcmbwMitJKjcyn2nj7GntxWXzTWarw2esZR\\\\\\\/SPo06HiqWYKtC4yBUUB5rNUKvrCjWxswT\\\\\\\/kChbh3HrA3O89SbVYRKra+L8pGbVHY8Ih1k0D3Zvtssrn+Gs1bR+llVQWM+5Rl0Zh9v7bD6yxJfkPV+QvO565cLtKaYLVKS1NgrlRNJsBRAQMX++AxqpkM6jSzMt6lG61574kFz8CwxjebzDqYqxyLbMEKYd1NZoWkWkkwb8OLJTSPG5oOx971aULhf6nPgurIiqGj87befIpp6OCw7hhIhw7O9B6UFfITcXNqHQUXsZz1RLHCUd3CE6KUSt+I+0xm+fE3OY015TH6Yb8sPFY+U1OVj0NtRezaWhmSjDKqiO5bJUzoNlXZgsp50hTfw1JSbI2skbLnlkOyXVVN+y1rKOHiHLC20sYvwlTreTJS\\\\\\\/\\\\\\\/5K3P8geVsNI5Od20MQexNgrfOmAYp26oNadRoqddsE+FLAzrB3CYAWq2qtrIsU2PkjGUNeQMI5x1pnvt1cfYH6LZ5jP+srTdk5CnyuV0P4WF7xMOLB1p7WyfOZ5nYRst6W+tLCgymbtw9s5cgBRBuP1nu6NU8JkHndvmi1TBaKHDwDsgcTa7\\\\\\\/OkhagXqPX\\\\\\\/GDC21OTVxCNCeqY+O0C18I4cF6oQOx2\\\\\\\/3GpefNFR7imOxSgMANNsXv1fxaRD9ezqizE4Nu1r8CvozcZ4bmuwa1PaaNDs0Dz8+55luJro8XksmPjEOCd+UfXZiQQm+6fu5Yie5EtrMUvb8MImbtqiWzUYTiDjONXpaMp27NXypvkc\\\\\\\/eMTC8JQQpZOLoI2LBrcrcBwq98ixrzIDJ+BGMn1JdylW3C4NE3siNFZIq87xPCn8XWUk=\\\u0022,\\\u0022iv\\\u0022:\\\u00222LTg5jz2gGLih2D\\\\\\\/+QYadQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022b2abae03c626a5c317bb1bbc8f02b79f518044691407d44f98461a843438115c\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.07.14.201475v4.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2330666 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.07.14.201475v4.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_342088808">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_342088808" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1990648123" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-zcC2u9Jl71wC6OSaT7r--d5QpD5Zo29gwq1metinuPE"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2330666" data-node-nid="2330666" id="node-2330666--2264782450" data-pisa="biorxiv;2020.07.14.201475v4" data-pisa-master="biorxiv;2020.07.14.201475" data-apath="/biorxiv/early/2022/01/02/2020.07.14.201475.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    MAVE-NN: learning genotype-phenotype maps from multiplex assays of variant effect
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first" data-delta="0">
                      <span class="nlm-given-names">
                       Ammar
                      </span>
                      <span class="nlm-surname">
                       Tareen
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Mahdi
                      </span>
                      <span class="nlm-surname">
                       Kooshkbaghi
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Anna
                      </span>
                      <span class="nlm-surname">
                       Posfai
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       William T.
                      </span>
                      <span class="nlm-surname">
                       Ireland
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="4">
                      <span class="nlm-given-names">
                       David M.
                      </span>
                      <span class="nlm-surname">
                       McCandlish
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5">
                      <a href="http://orcid.org/0000-0003-1897-3778" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Justin B.
                      </span>
                      <span class="nlm-surname">
                       Kinney
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.07.14.201475
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2330666--2264782450" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Ammar Tareen
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory
                       </span>
                       , Cold Spring Harbor, NY, 11724
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Ammar%2BTareen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Tareen%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAmmar%2BTareen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Mahdi Kooshkbaghi
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory
                       </span>
                       , Cold Spring Harbor, NY, 11724
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Mahdi%2BKooshkbaghi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Kooshkbaghi%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AMahdi%2BKooshkbaghi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Anna Posfai
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory
                       </span>
                       , Cold Spring Harbor, NY, 11724
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Anna%2BPosfai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Posfai%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAnna%2BPosfai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     William T. Ireland
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Physics, California Institute of Technology
                       </span>
                       , Pasadena, CA, 91125
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Applied Physics, Harvard University
                       </span>
                       , Cambridge, MA, 02134
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=William%2BT.%2BIreland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Ireland%20WT&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AWilliam%2BT.%2BIreland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     David M. McCandlish
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory
                       </span>
                       , Cold Spring Harbor, NY, 11724
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=David%2BM.%2BMcCandlish%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=McCandlish%20DM&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ADavid%2BM.%2BMcCandlish%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Justin B. Kinney
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory
                       </span>
                       , Cold Spring Harbor, NY, 11724
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Justin%2BB.%2BKinney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Kinney%20JB&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AJustin%2BB.%2BKinney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0003-1897-3778" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Justin B. Kinney
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:jkinney@cshl.edu" class="" data-icon-position="" data-hide-link-title="0">
                        jkinney@cshl.edu
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.07.14.201475v4" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2330666" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2330666/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.14.201475v4.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2330666" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2330666/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.14.201475v4.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2330666" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2330666/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.14.201475v4.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2330666" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2330666/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.14.201475v4.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2330666" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2330666/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.14.201475v4.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:2330666" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:2330666/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.07.14.201475v4.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2330666" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2330666/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-2">
                             Multiplex assays of variant effect (MAVEs) are a family of methods that includes deep mutational scanning (DMS) experiments on proteins and massively parallel reporter assays (MPRAs) on gene regulatory sequences. However, a general strategy for inferring quantitative models of genotype-phenotype (G-P) maps from MAVE data is lacking. Here we introduce MAVE-NN, a neural-network-based Python package that implements a broadly applicable information-theoretic framework for learning G-P maps—including biophysically interpretable models—from MAVE datasets. We demonstrate MAVE-NN in multiple biological contexts, and highlight the ability of our approach to deconvolve mutational effects from otherwise confounding experimental nonlinearities and noise.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-6">
                             Over the last decade, the ability to quantitatively study genotype-phenotype (G-P) maps has been revolutionized by the development of multiplex assays of variant effect (MAVEs), which can measure molecular phenotypes for thousands to millions of genotypic variants in parallel.
                             <sup>
                              <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                               1
                              </a>
                              ,
                              <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                               2
                              </a>
                             </sup>
                             MAVE is an umbrella term that describes a diverse set of experimental methods, three examples of which are illustrated in
                             <strong>
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Fig. 1
                              </a>
                             </strong>
                             . Deep mutational scanning (DMS) experiments
                             <sup>
                              <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                             </sup>
                             are a type of MAVE commonly used to study protein sequence-function relationships. These assays work by linking variant proteins to their coding sequences, either directly or indirectly, then using deep sequencing to assay which variants survive a process of activity-dependent selection (e.g.,
                             <strong>
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Fig. 1a
                              </a>
                             </strong>
                             ). Massively parallel reporter assays (MPRAs) are another major class of MAVE, and are commonly used to study DNA or RNA sequences that regulate gene expression at a variety of steps, including transcription, mRNA splicing, cleavage and polyadenylation, translation, and mRNA decay.
                             <sup>
                              <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                              –
                              <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                               7
                              </a>
                             </sup>
                             MPRAs typically rely on either an RNA-seq readout of barcode abundances (
                             <strong>
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Fig. 1c
                              </a>
                             </strong>
                             ) or the sorting of cells expressing a fluorescent reporter gene (
                             <strong>
                              <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                               Fig. 1e
                              </a>
                             </strong>
                             ).
                            </p>
                            <div id="F1" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F1.large.jpg?width=800&height=600&carousel=1" title="Examples illustrating the diversity of MAVEs. (a) DMS assays using either affinity purification or selective growth. (i) The DMS assay of Olson et al.35 used a library of variant GB1 proteins covalently linked to their coding mRNAs via mRNA display. Functional GB1 proteins were then enriched using IgG beads. (ii) The DMS studies of Seuma et al.36 and Bolognesi et al.37 used selective growth in genetically modified Saccharomyces cerevisiae cells to respectively assay the functionality of variant Aβ and TDP-43 proteins. In all three experiments, deep sequencing was used to determine an enrichment ratio for each protein variant. (b) The resulting DMS dataset consists of variant protein sequences and their corresponding log enrichment values. (c) The MPSA of Wong et al..38 A library of 3-exon minigenes was constructed from exons 16, 17, and 18 of BRCA2, with each minigene having a variant 5’ss at exon 17 and a random 20 nt barcode in the 3’ UTR. This library was transfected into HeLa cells, and deep sequencing was used to quantify mRNA isoform abundance. (d) The resulting MPSA dataset comprises variant 5’ss with (noisy) PSI values. (e) The sort-seq MPRA of Kinney et al..16 A plasmid library was generated in which randomly mutagenized versions of the Escherichia coli lac promoter drove the expression of GFP. Cells carrying these plasmids were sorted using FACS, and the variant promoters in each bin of sorted cells as well as the initial library were sequenced. (f) The resulting dataset comprises a list of variant promoter sequences, as well as a matrix of counts for each variant in each FACS bin. MAVE: multiplex assay of variant effect; DMS: deep mutational scanning; MPSA: massively parallel splicing assay; 5’ss: 5’ splice site(s); PSI: percent spliced in; GFP: green fluorescent protein; FACS: fluorescence-activated cell sorting." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-418940397" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Examples illustrating the diversity of MAVEs. (<strong>a</strong>) DMS assays using either affinity purification or selective growth. <strong>(i)</strong> The DMS assay of Olson et al.<sup>35</sup> used a library of variant GB1 proteins covalently linked to their coding mRNAs via mRNA display. Functional GB1 proteins were then enriched using IgG beads. <strong>(ii)</strong> The DMS studies of Seuma et al.<sup>36</sup> and Bolognesi et al.<sup>37</sup> used selective growth in genetically modified <em>Saccharomyces cerevisiae</em> cells to respectively assay the functionality of variant Aβ and TDP-43 proteins. In all three experiments, deep sequencing was used to determine an enrichment ratio for each protein variant. (<strong>b</strong>) The resulting DMS dataset consists of variant protein sequences and their corresponding log enrichment values. (<strong>c</strong>) The MPSA of Wong et al..<sup>38</sup> A library of 3-exon minigenes was constructed from exons 16, 17, and 18 of <em>BRCA2</em>, with each minigene having a variant 5’ss at exon 17 and a random 20 nt barcode in the 3’ UTR. This library was transfected into HeLa cells, and deep sequencing was used to quantify mRNA isoform abundance. (<strong>d</strong>) The resulting MPSA dataset comprises variant 5’ss with (noisy) PSI values. (<strong>e</strong>) The sort-seq MPRA of Kinney et al..<sup>16</sup> A plasmid library was generated in which randomly mutagenized versions of the <em>Escherichia coli lac</em> promoter drove the expression of GFP. Cells carrying these plasmids were sorted using FACS, and the variant promoters in each bin of sorted cells as well as the initial library were sequenced. (<strong>f</strong>) The resulting dataset comprises a list of variant promoter sequences, as well as a matrix of counts for each variant in each FACS bin. MAVE: multiplex assay of variant effect; DMS: deep mutational scanning; MPSA: massively parallel splicing assay; 5’ss: 5’ splice site(s); PSI: percent spliced in; GFP: green fluorescent protein; FACS: fluorescence-activated cell sorting.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F1.medium.gif" width="435" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F1.medium.gif" width="435" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                              <span class="fig-label">
                               Figure 1.
                              </span>
                              <p id="p-7" class="first-child">
                               Examples illustrating the diversity of MAVEs. (
                               <strong>
                                a
                               </strong>
                               ) DMS assays using either affinity purification or selective growth.
                               <strong>
                                (i)
                               </strong>
                               The DMS assay of Olson et al.
                               <sup>
                                <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                                 35
                                </a>
                               </sup>
                               used a library of variant GB1 proteins covalently linked to their coding mRNAs via mRNA display. Functional GB1 proteins were then enriched using IgG beads.
                               <strong>
                                (ii)
                               </strong>
                               The DMS studies of Seuma et al.
                               <sup>
                                <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                                 36
                                </a>
                               </sup>
                               and Bolognesi et al.
                               <sup>
                                <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                                 37
                                </a>
                               </sup>
                               used selective growth in genetically modified Saccharomyces cerevisiae cells to respectively assay the functionality of variant Aβ and TDP-43 proteins. In all three experiments, deep sequencing was used to determine an enrichment ratio for each protein variant. (
                               <strong>
                                b
                               </strong>
                               ) The resulting DMS dataset consists of variant protein sequences and their corresponding log enrichment values. (
                               <strong>
                                c
                               </strong>
                               ) The MPSA of Wong et al..
                               <sup>
                                <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                                 38
                                </a>
                               </sup>
                               A library of 3-exon minigenes was constructed from exons 16, 17, and 18 of BRCA2, with each minigene having a variant 5’ss at exon 17 and a random 20 nt barcode in the 3’ UTR. This library was transfected into HeLa cells, and deep sequencing was used to quantify mRNA isoform abundance. (
                               <strong>
                                d
                               </strong>
                               ) The resulting MPSA dataset comprises variant 5’ss with (noisy) PSI values. (
                               <strong>
                                e
                               </strong>
                               ) The sort-seq MPRA of Kinney et al..
                               <sup>
                                <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                                 16
                                </a>
                               </sup>
                               A plasmid library was generated in which randomly mutagenized versions of the Escherichia coli lac promoter drove the expression of GFP. Cells carrying these plasmids were sorted using FACS, and the variant promoters in each bin of sorted cells as well as the initial library were sequenced. (
                               <strong>
                                f
                               </strong>
                               ) The resulting dataset comprises a list of variant promoter sequences, as well as a matrix of counts for each variant in each FACS bin. MAVE: multiplex assay of variant effect; DMS: deep mutational scanning; MPSA: massively parallel splicing assay; 5’ss: 5’ splice site(s); PSI: percent spliced in; GFP: green fluorescent protein; FACS: fluorescence-activated cell sorting.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-8">
                             Most computational methods for analyzing MAVE data have focused on accurately quantifying the activity of individual assayed sequences.
                             <sup>
                              <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                              –
                              <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                             </sup>
                             However, MAVE measurements like enrichment ratios or cellular fluorescence levels usually cannot be interpreted as providing direct quantification of biologically meaningful activities, due to the presence of experiment-specific nonlinearities and noise. Moreover, MAVE data is usually incomplete, as one often wishes to understand G-P maps over vastly larger regions of sequence space than can be exhaustively assayed. The explicit quantitative modeling of G-P maps can address both the indirectness and incompleteness of MAVE measurements.
                             <sup>
                              <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">
                               1
                              </a>
                              ,
                              <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                             </sup>
                             The goal here is to determine a mathematical function that, given a sequence as input, will return a quantitative value for that sequence’s molecular phenotype. Such quantitative modeling has been of great interest since the earliest MAVE methods were developed,
                             <sup>
                              <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              –
                              <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                             </sup>
                             but no general-use software has yet been described for inferring G-P maps of arbitrary functional form from MAVE data.
                            </p>
                            <p id="p-9">
                             Here we introduce a unified conceptual framework for the quantitative modeling of MAVE data. This framework is based on the use of latent phenotype models, which assume that each assayed sequence has a well-defined latent phenotype (specified by the G-P map), of which the MAVE experiment provides an indirect readout (described by the measurement process). The quantitative forms of both the G-P map and the measurement process are then inferred from MAVE data simultaneously. We further introduce an information-theoretic approach for separately assessing the performance of the G-P map and the measurement process components of latent phenotype models. This strategy is implemented in an easy-to-use open-source Python package called MAVE-NN, which represents latent phenotype models as neural networks and infers the parameters of these models from MAVE data using a TensorFlow 2 backend.
                             <sup>
                              <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                               19
                              </a>
                             </sup>
                            </p>
                            <p id="p-10">
                             In what follows, we expand on this unified MAVE modeling strategy and apply it to a diverse array of DMS and MPRA datasets. Doing so, we find that MAVE-NN provides substantial advantages over other MAVE modeling approaches. Our results also highlight the substantial benefits of including sequence variants with multiple mutations in assayed sequence libraries, as doing so allows MAVE-NN to deconvolve the features of the G-P map from potentially confounding effects of experimental nonlinearities and noise. Importantly, we find that including just a modest number of multiple-mutation variants in a MAVE experiment can be beneficial even when one is primarily interested in the effects of single mutations. Finally, we illustrate how the ability of MAVE-NN to train custom G-P maps can shed light on the biophysical mechanisms of gene regulation.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Results
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Latent phenotype modeling strategy
                             </h3>
                             <p id="p-11">
                              MAVE-NN supports the analysis of MAVE data on DNA, RNA, and protein sequences, and can accommodate either continuous or discrete measurement values. Given a set of sequence-measurement pairs, MAVE-NN aims to infer a probabilistic mapping from sequence to measurement. Our primary enabling assumption, which is encoded in the structure of the latent phenotype model (
                              <strong>
                               <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                                Fig. 2a
                               </a>
                              </strong>
                              ), is that this mapping occurs in two stages. Each sequence is first mapped to a latent phenotype by a deterministic G-P map, then this latent phenotype is mapped to possible measurement values via a stochastic measurement process. During training, the G-P map and measurement process are simultaneously learned by maximizing a regularized form of likelihood.
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F2.large.jpg?width=800&height=600&carousel=1" title="MAVE-NN quantitative modeling strategy. (a) Structure of latent phenotype models. A G-P map f(x) maps each sequence x to a latent phenotype ϕ, after which a measurement process p(y|ϕ) determines the measurement y. (b) Example of an MPA measurement process inferred from the sort-seq MPRA data of Kinney et al..16 MPA measurement processes are used when y values are discrete. (c) Structure of a GE regression model, which is used when y is continuous. A GE measurement process assumes that the mode of p(y|ϕ), called the prediction ŷ, is given by a nonlinear function g(ϕ), and the scatter about this mode is described by a noise model p(y|ŷ). (d) Example of a GE measurement process inferred from the DMS data of Olson et al..35 Shown are the nonlinearity, the 68% CI, and the 95% CI. (e) Information-theoretic quantities used to assess model performance. Intrinsic information, Iint, is the mutual information between sequences x and measurements y. Predictive information, Ipre, is the mutual information between measurements y and the latent phenotype values ϕ assigned by a model. Variational information, Ivar, is a linear transformation of log likelihood. The inequality Iint ≥ Ipre ≥ Ivar always holds on test data (modulo finite data uncertainties), with Iint = Ipre when the G-P map is correct, and Ipre = Ivar when the measurement process correctly describes the distribution of y conditioned on ϕ. G-P: genotype-phenotype; MPA: measurement process agnostic; GE: global epistasis; CI: confidence interval." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-418940397" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">MAVE-NN quantitative modeling strategy. (<strong>a</strong>) Structure of latent phenotype models. A G-P map <em>f</em>(<em>x</em>) maps each sequence <em>x</em> to a latent phenotype <em>ϕ</em>, after which a measurement process <em>p</em>(<em>y</em>|<em>ϕ</em>) determines the measurement <em>y</em>. (<strong>b</strong>) Example of an MPA measurement process inferred from the sort-seq MPRA data of Kinney et al..<sup>16</sup> MPA measurement processes are used when <em>y</em> values are discrete. (<strong>c</strong>) Structure of a GE regression model, which is used when <em>y</em> is continuous. A GE measurement process assumes that the mode of <em>p</em>(<em>y</em>|<em>ϕ</em>), called the prediction <em>ŷ</em>, is given by a nonlinear function <em>g</em>(<em>ϕ</em>), and the scatter about this mode is described by a noise model <em>p</em>(<em>y</em>|<em>ŷ</em>). (<strong>d</strong>) Example of a GE measurement process inferred from the DMS data of Olson et al..<sup>35</sup> Shown are the nonlinearity, the 68% CI, and the 95% CI. (<strong>e</strong>) Information-theoretic quantities used to assess model performance. Intrinsic information, <em>I</em><sub>int</sub>, is the mutual information between sequences <em>x</em> and measurements <em>y</em>. Predictive information, <em>I</em><sub>pre</sub>, is the mutual information between measurements <em>y</em> and the latent phenotype values <em>ϕ</em> assigned by a model. Variational information, <em>I</em><sub>var</sub>, is a linear transformation of log likelihood. The inequality <em>I</em><sub>int</sub> ≥ <em>I</em><sub>pre</sub> ≥ <em>I</em><sub>var</sub> always holds on test data (modulo finite data uncertainties), with <em>I</em><sub>int</sub> = <em>I</em><sub>pre</sub> when the G-P map is correct, and <em>I</em><sub>pre</sub> = <em>I</em><sub>var</sub> when the measurement process correctly describes the distribution of <em>y</em> conditioned on <em>ϕ</em>. G-P: genotype-phenotype; MPA: measurement process agnostic; GE: global epistasis; CI: confidence interval.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F2.medium.gif" width="440" height="235"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F2.medium.gif" width="440" height="235"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2.
                               </span>
                               <p id="p-12" class="first-child">
                                MAVE-NN quantitative modeling strategy. (
                                <strong>
                                 a
                                </strong>
                                ) Structure of latent phenotype models. A G-P map f(x) maps each sequence x to a latent phenotype ϕ, after which a measurement process p(y|ϕ) determines the measurement y. (
                                <strong>
                                 b
                                </strong>
                                ) Example of an MPA measurement process inferred from the sort-seq MPRA data of Kinney et al..
                                <sup>
                                 <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">
                                  16
                                 </a>
                                </sup>
                                MPA measurement processes are used when y values are discrete. (
                                <strong>
                                 c
                                </strong>
                                ) Structure of a GE regression model, which is used when y is continuous. A GE measurement process assumes that the mode of p(y|ϕ), called the prediction ŷ, is given by a nonlinear function g(ϕ), and the scatter about this mode is described by a noise model p(y|ŷ). (
                                <strong>
                                 d
                                </strong>
                                ) Example of a GE measurement process inferred from the DMS data of Olson et al..
                                <sup>
                                 <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">
                                  35
                                 </a>
                                </sup>
                                Shown are the nonlinearity, the 68% CI, and the 95% CI. (
                                <strong>
                                 e
                                </strong>
                                ) Information-theoretic quantities used to assess model performance. Intrinsic information, I
                                <sub>
                                 int
                                </sub>
                                , is the mutual information between sequences x and measurements y. Predictive information, I
                                <sub>
                                 pre
                                </sub>
                                , is the mutual information between measurements y and the latent phenotype values ϕ assigned by a model. Variational information, I
                                <sub>
                                 var
                                </sub>
                                , is a linear transformation of log likelihood. The inequality I
                                <sub>
                                 int
                                </sub>
                                ≥ I
                                <sub>
                                 pre
                                </sub>
                                ≥ I
                                <sub>
                                 var
                                </sub>
                                always holds on test data (modulo finite data uncertainties), with I
                                <sub>
                                 int
                                </sub>
                                = I
                                <sub>
                                 pre
                                </sub>
                                when the G-P map is correct, and I
                                <sub>
                                 pre
                                </sub>
                                = I
                                <sub>
                                 var
                                </sub>
                                when the measurement process correctly describes the distribution of y conditioned on ϕ. G-P: genotype-phenotype; MPA: measurement process agnostic; GE: global epistasis; CI: confidence interval.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-13">
                              MAVE-NN includes four types of built-in G-P maps: additive, neighbor, pairwise, and black box. Additive G-P maps assume that each character at each position within a sequence contributes independently to the latent phenotype. Neighbor G-P maps incorporate interactions between adjacent (i.e., nearest-neighbor) characters in a sequence, while pairwise G-P maps include interactions between all pairs of characters in a sequence regardless of the distance separating the characters in each pair. Black box G-P maps have the form of a densely connected multilayer perceptron, the specific architecture of which can be controlled by the user. MAVE-NN also supports custom G-P maps that can be used, e.g., to represent specific biophysical hypotheses about the mechanisms of sequence function.
                             </p>
                             <p id="p-14">
                              To handle both discrete and continuous measurement values, two different strategies for modeling measurement processes are provided. Measurement process agnostic (MPA) regression uses techniques from the biophysics literature
                              <sup>
                               <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                                15
                               </a>
                               ,
                               <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">
                                16
                               </a>
                               ,
                               <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                                20
                               </a>
                               ,
                               <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                                21
                               </a>
                              </sup>
                              to analyze MAVE datasets that report discrete measurements. Here the measurement process is represented by an overparameterized neural network that takes the latent phenotype value as input and outputs the probability of each possible measurement value (
                              <strong>
                               <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                                Fig. 2b
                               </a>
                              </strong>
                              ). Global epistasis (GE) regression (
                              <strong>
                               <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                                Fig. 2c
                               </a>
                              </strong>
                              ), by contrast, leverages ideas previously developed in the evolution literature for analyzing datasets that contain continuous measurements,
                              <sup>
                               <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                                22
                               </a>
                               –
                               <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                                25
                               </a>
                              </sup>
                              and is becoming an increasingly popular strategy for modeling DMS data.
                              <sup>
                               <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                                26
                               </a>
                               –
                               <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                                28
                               </a>
                              </sup>
                              Here, the latent phenotype is nonlinearly mapped to a prediction that represents the most probable measurement value. A noise model is then used to describe the distribution of likely deviations from this prediction. MAVE-NN supports both homoscedastic and heteroscedastic noise models based on three different classes of probability distribution: Gaussian, Cauchy, and skewed-t. We note that the skewed-t distribution, introduced by Jones and Faddy,
                              <sup>
                               <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                                29
                               </a>
                              </sup>
                              reduces to Gaussian and Cauchy distributions in certain limits while also accommodating asymmetric experimental noise.
                              <strong>
                               <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                                Fig. 2d
                               </a>
                              </strong>
                              shows an example of a GE measurement process with a heteroscedastic skewed-t noise model.
                             </p>
                             <p id="p-15">
                              Readers should note that the current implementation of MAVE-NN places certain constraints on input data and model architecture. Input sequences must be the same length, and when analyzing continuous data, only scalar measurements (as opposed to vectors of multiple measurements) can be used to train models. In addition, because our method for learning the form of experimental nonlinearities depends on observing how multiple mutations combine, MAVE-NN’s functionality is more limited when analyzing MAVE libraries that comprise only single-mutation variants. More information on these constraints and the reasons behind them can be found below in the section “Constraints on datasets and models”.
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Information-theoretic measures of model performance
                             </h3>
                             <p id="p-16">
                              We further propose three distinct quantities for assessing the performance of latent phenotype models: intrinsic information, predictive information, and variational information (
                              <strong>
                               <a id="xref-fig-2-5" class="xref-fig" href="#F2">
                                Fig. 2e
                               </a>
                              </strong>
                              ). These quantities come from information theory and are motivated by thinking of G-P maps in terms of information compression. In information theory, a quantity called mutual information quantifies the amount of information that the value of one variable communicates about the value of another.
                              <sup>
                               <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                                30
                               </a>
                               ,
                               <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                                31
                               </a>
                              </sup>
                              Mutual information is symmetric, nonnegative, is measured in units of “bits”, and is equal to 0 bits only if the two variables are independent. Alternatively, if knowing the value of one variable allows you to narrow down the value of the other variable to one of two possibilities that would otherwise be equally likely, the mutual information between these two variables will be 1.0 bits. If the value is narrowed down to one of four otherwise equally likely possibilities, the mutual information will be 2.0 bits. Narrowing down to one of with eight possibilities will yield 3.0 bits, and so on. But importantly, mutual information does not require that the relationship between two variables in question be so clean cut, and mutual information can in fact be computed between any two types of variables—discrete, continuous, multi-dimensional, etc.. This property makes the information-based quantities we propose applicable to all MAVE datasets, regardless of the specific type of experimental readout used. By contrast, many of the standard model performance metrics have restricted domains of applicability: accuracy can only be applied to data with discrete labels, R
                              <sup>
                               2
                              </sup>
                              can only be applied to data with univariate continuous labels, and so on. We note, however, that estimating mutual information and related quantities from finite data is nontrivial and that MAVE-NN uses a variety of approaches to do this.
                             </p>
                             <p id="p-17">
                              Intrinsic information, I
                              <sub>
                               int
                              </sub>
                              , is the mutual information between the sequences and measurements contained within a MAVE dataset. This quantity provides a benchmark against which to compare the performance of inferred G-P maps. Predictive information, I
                              <sub>
                               pre
                              </sub>
                              , is the mutual information between MAVE measurements and the latent phenotype values predicted by a G-P map of interest. This quantifies how well the G-P map preserves sequence-encoded information that is determinative of experimental measurements. When evaluated on test data, I
                              <sub>
                               pre
                              </sub>
                              is bounded above by I
                              <sub>
                               int
                              </sub>
                              , and equality is realized only when the latent phenotype losslessly encodes relevant sequence-encoded information. Variational information, I
                              <sub>
                               var
                              </sub>
                              , is a linear transformation of log likelihood that provides a variational lower bound on I
                              <sub>
                               pre
                              </sub>
                              .
                              <sup>
                               <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                                32
                               </a>
                               –
                               <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                                34
                               </a>
                              </sup>
                              The difference between I
                              <sub>
                               pre
                              </sub>
                              and I
                              <sub>
                               var
                              </sub>
                              quantifies how accurately the inferred measurement process matches the observed distribution of measurements and latent phenotypes (see
                              <strong>
                               Supplemental Information
                              </strong>
                              ).
                             </p>
                             <p id="p-18">
                              MAVE-NN infers model parameters by maximizing a (lightly) regularized form of likelihood. These computations are performed using the standard backpropagation-based training algorithms provided within the TensorFlow 2 backend. With certain caveats noted (see
                              <strong>
                               Methods
                              </strong>
                              ), this optimization procedure maximizes I
                              <sub>
                               pre
                              </sub>
                              while avoiding the costly estimates of mutual information at each iteration that have hindered the adoption of previous mutual-information-based modeling strategies.
                              <sup>
                               <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">
                                16
                               </a>
                              </sup>
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Application: deep mutational scanning assays
                             </h3>
                             <p id="p-19">
                              We now demonstrate the capabilities of MAVE-NN on three DMS datasets, starting with the study of Olson et al.
                              <sup>
                               <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">
                                35
                               </a>
                              </sup>
                              on pairwise epistasis in protein G. Here the authors measured the effects of all single and nearly all double mutations to residues 2-56 of the IgG binding domain. This domain, called GB1, has long served as a model system for studying protein sequence-function relationships. To assay the binding of GB1 variants to IgG, the authors combined mRNA display with ultra-high-throughput DNA sequencing (
                              <strong>
                               <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                                Fig. 1a
                               </a>
                              </strong>
                              ). The resulting dataset reports log enrichment values for all 1,045 single- and 530,737 double-mutant GB1 variants (
                              <strong>
                               <a id="xref-fig-1-6" class="xref-fig" href="#F1">
                                Fig. 1b
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-20">
                              Inspired by the work of Otwinowski et al.,
                              <sup>
                               <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">
                                25
                               </a>
                              </sup>
                              we used MAVE-NN to infer a latent phenotype model comprising an additive G-P map and a GE measurement process. This inference procedure required only about 5 minutes on a single node of a computer cluster (
                              <strong>
                               Supplemental Fig. S1
                              </strong>
                              ).
                              <strong>
                               <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                                Fig. 3a
                               </a>
                              </strong>
                              illustrates the inferred additive G-P map via the effects that every possible single-residue mutation has on the latent phenotype. From this heatmap of additive effects, we can immediately identify all of the critical GB1 residues, including the IgG interacting residues at 27, 31, and 43.
                              <sup>
                               <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">
                                35
                               </a>
                              </sup>
                              We also observe that missense mutations to proline throughout the GB1 domain tend to negatively impact IgG binding, as expected due to this amino acid’s exceptional conformational rigidity.
                              <strong>
                               <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                                Fig. 3b
                               </a>
                              </strong>
                              illustrates the corresponding GE measurement process, revealing a sigmoidal relationship between log enrichment measurements and the latent phenotype values predicted by the G-P map. Nonlinearities like this are ubiquitous in DMS data due to the presence of background and saturation effects. Unless they are explicitly accounted for in one’s quantitative modeling efforts, as they are here, these nonlinearities can greatly distort the parameters of inferred G-P maps.
                              <strong>
                               <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                                Fig. 3c
                               </a>
                              </strong>
                              shows that accounting for this nonlinearity yields predictions that correlate quite well with measurement values. Moreover, every latent phenotype model inferred by MAVE-NN can be used as a MAVE dataset simulator (see
                              <strong>
                               Methods
                              </strong>
                              ). By analyzing simulated data generated by our inferred model for this GB1 experiment, we further observed that MAVE-NN can accurately and robustly recover the GE nonlinearity and ground-truth G-P map parameters (
                              <strong>
                               Supplementary Fig. S1
                              </strong>
                              ).
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F3.large.jpg?width=800&height=600&carousel=1" title="Analysis of DMS data for protein GB1. MAVE-NN was used to infer a latent phenotype model, consisting of an additive G-P map and a GE measurement process having a heteroskedastic skewed-t noise model, from the DMS data of Olson et al..35 All 530,737 pairwise variants reported for positions 2 to 56 of the GB1 domain were analyzed. Data were split 90:5:5 into training, validation, and test sets. (a) The G-P map parameters inferred from all pairwise variants. Gray dots indicate wildtype residues. Amino acids are ordered as in Olson et al..35 (b) GE plot showing measurements versus predicted latent phenotype values for 5,000 randomly selected test-set sequences (blue dots), alongside the inferred nonlinearity (solid orange line) and the 95% CI (dashed lines) of the noise model. Gray line indicates the latent phenotype value of the wildtype sequence. (c) Measurements plotted against ŷ predictions for these same sequences. Dashed lines indicate the 95% CI of the noise model. Gray line indicates the wildtype sequence ŷ. (d) Corresponding information metrics computed during model training (using training data) or for the final model (using test data); uncertainties in these estimates are roughly the width of the plotted lines. Gray shaded area indicates allowed values for intrinsic information based on upper and lower bounds estimated as described in Methods. (e-g) Test set predictions (blue dots) and GE nonlinearities (orange lines) for models trained using subsets of the GB1 data containing all single mutants and 50,000 (e), 5,000 (f), or 500 (g) double mutants. The GE nonlinearity from panel b is shown for reference (yellow-green lines). Uncertainties reflect standard errors. GE: global epistasis; G-P: genotype-phenotype; CI: confidence interval." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-418940397" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Analysis of DMS data for protein GB1. MAVE-NN was used to infer a latent phenotype model, consisting of an additive G-P map and a GE measurement process having a heteroskedastic skewed-t noise model, from the DMS data of Olson et al..<sup>35</sup> All 530,737 pairwise variants reported for positions 2 to 56 of the GB1 domain were analyzed. Data were split 90:5:5 into training, validation, and test sets. (<strong>a</strong>) The G-P map parameters inferred from all pairwise variants. Gray dots indicate wildtype residues. Amino acids are ordered as in Olson et al..<sup>35</sup> (<strong>b</strong>) GE plot showing measurements versus predicted latent phenotype values for 5,000 randomly selected test-set sequences (blue dots), alongside the inferred nonlinearity (solid orange line) and the 95% CI (dashed lines) of the noise model. Gray line indicates the latent phenotype value of the wildtype sequence. (<strong>c</strong>) Measurements plotted against <em>ŷ</em> predictions for these same sequences. Dashed lines indicate the 95% CI of the noise model. Gray line indicates the wildtype sequence <em>ŷ</em>. (<strong>d</strong>) Corresponding information metrics computed during model training (using training data) or for the final model (using test data); uncertainties in these estimates are roughly the width of the plotted lines. Gray shaded area indicates allowed values for intrinsic information based on upper and lower bounds estimated as described in <strong>Methods</strong>. (<strong>e-g</strong>) Test set predictions (blue dots) and GE nonlinearities (orange lines) for models trained using subsets of the GB1 data containing all single mutants and 50,000 (<strong>e</strong>), 5,000 (<strong>f</strong>), or 500 (<strong>g</strong>) double mutants. The GE nonlinearity from panel <strong>b</strong> is shown for reference (yellow-green lines). Uncertainties reflect standard errors. GE: global epistasis; G-P: genotype-phenotype; CI: confidence interval.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F3.medium.gif" width="428" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F3.medium.gif" width="428" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3.
                               </span>
                               <p id="p-21" class="first-child">
                                Analysis of DMS data for protein GB1. MAVE-NN was used to infer a latent phenotype model, consisting of an additive G-P map and a GE measurement process having a heteroskedastic skewed-t noise model, from the DMS data of Olson et al..
                                <sup>
                                 <a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">
                                  35
                                 </a>
                                </sup>
                                All 530,737 pairwise variants reported for positions 2 to 56 of the GB1 domain were analyzed. Data were split 90:5:5 into training, validation, and test sets. (
                                <strong>
                                 a
                                </strong>
                                ) The G-P map parameters inferred from all pairwise variants. Gray dots indicate wildtype residues. Amino acids are ordered as in Olson et al..
                                <sup>
                                 <a id="xref-ref-35-6" class="xref-bibr" href="#ref-35">
                                  35
                                 </a>
                                </sup>
                                (
                                <strong>
                                 b
                                </strong>
                                ) GE plot showing measurements versus predicted latent phenotype values for 5,000 randomly selected test-set sequences (blue dots), alongside the inferred nonlinearity (solid orange line) and the 95% CI (dashed lines) of the noise model. Gray line indicates the latent phenotype value of the wildtype sequence. (
                                <strong>
                                 c
                                </strong>
                                ) Measurements plotted against ŷ predictions for these same sequences. Dashed lines indicate the 95% CI of the noise model. Gray line indicates the wildtype sequence ŷ. (
                                <strong>
                                 d
                                </strong>
                                ) Corresponding information metrics computed during model training (using training data) or for the final model (using test data); uncertainties in these estimates are roughly the width of the plotted lines. Gray shaded area indicates allowed values for intrinsic information based on upper and lower bounds estimated as described in
                                <strong>
                                 Methods
                                </strong>
                                . (
                                <strong>
                                 e-g
                                </strong>
                                ) Test set predictions (blue dots) and GE nonlinearities (orange lines) for models trained using subsets of the GB1 data containing all single mutants and 50,000 (
                                <strong>
                                 e
                                </strong>
                                ), 5,000 (
                                <strong>
                                 f
                                </strong>
                                ), or 500 (
                                <strong>
                                 g
                                </strong>
                                ) double mutants. The GE nonlinearity from panel
                                <strong>
                                 b
                                </strong>
                                is shown for reference (yellow-green lines). Uncertainties reflect standard errors. GE: global epistasis; G-P: genotype-phenotype; CI: confidence interval.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-22">
                              <strong>
                               <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                                Fig. 3d
                               </a>
                              </strong>
                              summarizes the values of our information-theoretic metrics for model performance. On held-out test data, we find that I
                              <sub>
                               var
                              </sub>
                              = 2.178 ± 0.027 bits and I
                              <sub>
                               pre
                              </sub>
                              = 2.225 ± 0.017 bits. The similarity of these two values suggests that the inferred GE measurement process, which includes a heteroscedastic skewed-t noise model, has nearly sufficient accuracy to fully describe the distribution of residuals. We further find that 2.741 ± 0.013 bits ≤ I
                              <sub>
                               int
                              </sub>
                              ≤ 3.215 ± 0.007 bits (see
                              <strong>
                               Methods
                              </strong>
                              ), meaning that the inferred G-P map accounts for 69%-81% of the total sequence-dependent information in the dataset. While this performance is impressive, the additive G-P map evidently misses some relevant sequence features. This observation motivates the more complex biophysical model for GB1 discussed later in
                              <strong>
                               Results
                              </strong>
                              .
                             </p>
                             <p id="p-23">
                              The ability of MAVE-NN to deconvolve experimental nonlinearities from additive G-P maps requires that some of the assayed sequences contain multiple mutations. This is because such nonlinearities are inferred by reconciling the effects of single mutations with the effects observed for combinations of two or more mutations. To investigate how many multiple-mutation variants are required, we performed GE inference on subsets of the GB1 dataset containing all 1,045 single-mutation sequences and either 50,000, 5,000, or 500 double-mutation sequences (see
                              <strong>
                               Methods
                              </strong>
                              ). The shapes of the resulting GE nonlinearities are illustrated in
                              <strong>
                               <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                                Figs. 3e-g
                               </a>
                              </strong>
                              . Remarkably, MAVE-NN is able to recover the underlying nonlinearity using only about 500 randomly selected double mutants, which represent only ∼0.1% of all possible double mutants. The analysis of simulated data also supports the ability to accurately recover ground-truth model predictions using highly reduced datasets (
                              <strong>
                               Supplemental Fig. S1
                              </strong>
                              ). These findings have important implications for the design of DMS experiments: even if one only wants to determine an additive G-P map, including a modest number of multiple-mutation sequences in the assayed library is often advisable because it may allow the removal of artifactual nonlinearities.
                             </p>
                             <p id="p-24">
                              To test the capabilities of MAVE-NN on less complete DMS datasets, we analyzed recent experiments on amyloid beta (Aβ)
                              <sup>
                               <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">
                                36
                               </a>
                              </sup>
                              and TDP-43,
                              <sup>
                               <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                              </sup>
                              both of which exhibit aggregation behavior in the context of neurodegenerative diseases. In these experiments, protein functionality was assayed using selective growth in genetically modified Saccaromyces cerevisiae: Seuma et al.
                              <sup>
                               <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">
                                36
                               </a>
                              </sup>
                              performed a selection against Aβ toxicity, whereas Bolognesi et al.
                              <sup>
                               <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                              </sup>
                              positively selected for TDP-43 aggregation. Like with GB1, the variant libraries used in these two experiments included a substantial number of multiple-mutation sequences: 499 single- and 15,567 double-mutation sequences for Aβ; 1,266 single- and 56,730 double-mutation sequences for TDP-43. But unlike with GB1, these datasets are highly incomplete due to the use of mutagenic PCR (for Aβ) or doped oligo synthesis (TDP-43) for variant library construction.
                             </p>
                             <p id="p-25">
                              We used MAVE-NN to infer additive G-P maps from these two datasets, adopting the same type of latent phenotype model used for GB1.
                              <strong>
                               <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                                Fig. 4a
                               </a>
                              </strong>
                              illustrates the additive G-P map inferred from aggregation measurements of Aβ variants. In agreement with the original study, we see that most amino acid mutations between positions 30-40 have a negative effect on nucleation, suggesting that this region plays a major role in nucleation behavior.
                              <strong>
                               <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                                Fig. 4b
                               </a>
                              </strong>
                              shows the corresponding measurement process (see also
                              <strong>
                               Supplemental Information Fig. S2
                              </strong>
                              ). Even though these data are much sparser than the GB1 data, the inferred model performs well on held-out test data (I
                              <sub>
                               var
                              </sub>
                              = 1.142 ± 0.065 bits, I
                              <sub>
                               pre
                              </sub>
                              = 1.187 ± 0.050 bits, R
                              <sup>
                               2
                              </sup>
                              = 0.763 ± 0.024). Similarly,
                              <strong>
                               <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                                Figs. 4c-d
                               </a>
                              </strong>
                              show the G-P map parameters and GE measurement process inferred from toxicity measurements of TDP-43 variants, revealing among other things the toxicity-determining hot-spot observed by Bolognesi et al.
                              <sup>
                               <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                              </sup>
                              at positions 310-340. The resulting latent phenotype model performs well on held-out test data (I
                              <sub>
                               var
                              </sub>
                              = 1.834 ± 0.035 bits, I
                              <sub>
                               pre
                              </sub>
                              = 1.994 ± 0.023 bits, R
                              <sup>
                               2
                              </sup>
                              = 0.914 ± 0.007).
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F4.large.jpg?width=800&height=600&carousel=1" title="Analysis of DMS data for Aβ and TDP-43. (a,b) Seuma et al.36 measured nucleation scores for 499 single mutants and 15,567 double mutants of Aβ. These data were used to train a latent phenotype model comprising (a) an additive G-P map and (b) a GE measurement process with a heteroskedastic skewed-t noise model. (c,d) Bolognesi et al.37 measured toxicity scores for 1,266 single mutants and 56,730 double mutants of TDP-43. The resulting data were used to train (c) an additive G-P map and (d) a GE measurement process of the same form as in panel b. In both cases, data were split 90:5:5 into training, validation, and test sets. In (a,c), gray dots indicate the wildtype sequence, amino acids are ordered as in the original publications, and * indicates a stop codon. White squares [355/882 (40.2%) for Aβ; 433/1764 (24.5%) for TDP-43] indicate residues that were not observed in the training set and thus could not be assigned values for their additive effects. In (b,d), blue dots indicate latent phenotype values versus measurements for held-out test data, gray line indicates the latent phenotype value of the wildtype sequence, solid orange line indicates the GE nonlinearity, and dashed orange lines indicate a corresponding 95% CI for the inferred noise model. Values for Ivar, Ipre, and R2 (between y and ŷ) are also shown. Uncertainties reflect standard errors. Supplemental Fig. S2 shows measurements plotted against the ŷ predictions of these models. Aβ: amyloid beta; TDP-43: TAR DNA-binding protein 43; G-P: genotype-phenotype; GE: global epistasis; CI: confidence interval." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-418940397" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Analysis of DMS data for Aβ and TDP-43. (<strong>a</strong>,<strong>b</strong>) Seuma et al.<sup>36</sup> measured nucleation scores for 499 single mutants and 15,567 double mutants of Aβ. These data were used to train a latent phenotype model comprising (<strong>a</strong>) an additive G-P map and (<strong>b</strong>) a GE measurement process with a heteroskedastic skewed-t noise model. (<strong>c</strong>,<strong>d</strong>) Bolognesi et al.<sup>37</sup> measured toxicity scores for 1,266 single mutants and 56,730 double mutants of TDP-43. The resulting data were used to train (<strong>c</strong>) an additive G-P map and (<strong>d</strong>) a GE measurement process of the same form as in panel <strong>b</strong>. In both cases, data were split 90:5:5 into training, validation, and test sets. In (<strong>a</strong>,<strong>c</strong>), gray dots indicate the wildtype sequence, amino acids are ordered as in the original publications, and * indicates a stop codon. White squares [355/882 (40.2%) for Aβ; 433/1764 (24.5%) for TDP-43] indicate residues that were not observed in the training set and thus could not be assigned values for their additive effects. In (<strong>b</strong>,<strong>d</strong>), blue dots indicate latent phenotype values versus measurements for held-out test data, gray line indicates the latent phenotype value of the wildtype sequence, solid orange line indicates the GE nonlinearity, and dashed orange lines indicate a corresponding 95% CI for the inferred noise model. Values for <em>I</em><sub>var</sub>, <em>I</em><sub>pre</sub>, and <em>R</em><sup>2</sup> (between <em>y</em> and <em>ŷ</em>) are also shown. Uncertainties reflect standard errors. <strong>Supplemental Fig. S2</strong> shows measurements plotted against the <em>ŷ</em> predictions of these models. Aβ: amyloid beta; TDP-43: TAR DNA-binding protein 43; G-P: genotype-phenotype; GE: global epistasis; CI: confidence interval.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F4.medium.gif" width="440" height="260"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F4.medium.gif" width="440" height="260"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4.
                               </span>
                               <p id="p-26" class="first-child">
                                Analysis of DMS data for Aβ and TDP-43. (
                                <strong>
                                 a
                                </strong>
                                ,
                                <strong>
                                 b
                                </strong>
                                ) Seuma et al.
                                <sup>
                                 <a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">
                                  36
                                 </a>
                                </sup>
                                measured nucleation scores for 499 single mutants and 15,567 double mutants of Aβ. These data were used to train a latent phenotype model comprising (
                                <strong>
                                 a
                                </strong>
                                ) an additive G-P map and (
                                <strong>
                                 b
                                </strong>
                                ) a GE measurement process with a heteroskedastic skewed-t noise model. (
                                <strong>
                                 c
                                </strong>
                                ,
                                <strong>
                                 d
                                </strong>
                                ) Bolognesi et al.
                                <sup>
                                 <a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">
                                  37
                                 </a>
                                </sup>
                                measured toxicity scores for 1,266 single mutants and 56,730 double mutants of TDP-43. The resulting data were used to train (
                                <strong>
                                 c
                                </strong>
                                ) an additive G-P map and (
                                <strong>
                                 d
                                </strong>
                                ) a GE measurement process of the same form as in panel
                                <strong>
                                 b
                                </strong>
                                . In both cases, data were split 90:5:5 into training, validation, and test sets. In (
                                <strong>
                                 a
                                </strong>
                                ,
                                <strong>
                                 c
                                </strong>
                                ), gray dots indicate the wildtype sequence, amino acids are ordered as in the original publications, and * indicates a stop codon. White squares [355/882 (40.2%) for Aβ; 433/1764 (24.5%) for TDP-43] indicate residues that were not observed in the training set and thus could not be assigned values for their additive effects. In (
                                <strong>
                                 b
                                </strong>
                                ,
                                <strong>
                                 d
                                </strong>
                                ), blue dots indicate latent phenotype values versus measurements for held-out test data, gray line indicates the latent phenotype value of the wildtype sequence, solid orange line indicates the GE nonlinearity, and dashed orange lines indicate a corresponding 95% CI for the inferred noise model. Values for I
                                <sub>
                                 var
                                </sub>
                                , I
                                <sub>
                                 pre
                                </sub>
                                , and R
                                <sup>
                                 2
                                </sup>
                                (between y and ŷ) are also shown. Uncertainties reflect standard errors.
                                <strong>
                                 Supplemental Fig. S2
                                </strong>
                                shows measurements plotted against the ŷ predictions of these models. Aβ: amyloid beta; TDP-43: TAR DNA-binding protein 43; G-P: genotype-phenotype; GE: global epistasis; CI: confidence interval.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Application: a massively parallel splicing assay
                             </h3>
                             <p id="p-27">
                              Exon/intron boundaries are defined by 5’ splice sites (5’ss), which bind the U1 snRNP during the initial stages of spliceosome assembly. To investigate how 5’ss sequence quantitatively controls alternative mRNA splicing, Wong et al.
                              <sup>
                               <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">
                                38
                               </a>
                              </sup>
                              used a massively parallel splicing assay (MPSA) to measure percent-spliced-in (PSI) values for nearly all 32,768 possible 5’ss of the form NNN/GYNNNN in three different genetic contexts (
                              <strong>
                               <a id="xref-fig-1-7" class="xref-fig" href="#F1">
                                Fig. 1c
                               </a>
                              </strong>
                              ,
                              <strong>
                               <a id="xref-fig-1-8" class="xref-fig" href="#F1">
                                d
                               </a>
                              </strong>
                              ). Applying MAVE-NN to data from the BRCA2 exon 17 context, we inferred four different types of G-P maps: additive, neighbor, pairwise, and black box. As with GB1, these G-P maps were each inferred using GE regression with a heteroscedastic skewed-t noise model. For comparison, we also inferred an additive G-P map using the epistasis package of Sailer and Harms.
                              <sup>
                               <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                                24
                               </a>
                              </sup>
                             </p>
                             <p id="p-28">
                              <strong>
                               <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                                Fig. 5a
                               </a>
                              </strong>
                              compares the performance of these G-P map models on held-out test data, while
                              <strong>
                               <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                                Figs. 5b-d
                               </a>
                              </strong>
                              illustrate the corresponding inferred measurement processes. We observe that the additive G-P map inferred using the epistasis package
                              <sup>
                               <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">
                                24
                               </a>
                              </sup>
                              exhibits less predictive information (I
                              <sub>
                               pre
                              </sub>
                              = 0.180 ± 0.011 bits) than the additive G-P map found using MAVE-NN (P =3.8 × 10
                              <sup>
                               −6
                              </sup>
                              , two-sided z-test). This is likely because the epistasis package estimates the parameters of the additive G-P map prior to estimating the GE nonlinearity. We also note that, while the epistasis package provides a variety of options for modeling the GE nonlinearity, none of these options appear to work as well as our mixture-of-sigmoids approach (compare
                              <strong>
                               <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                                Figs. 5b
                               </a>
                              </strong>
                              ,
                              <strong>
                               <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                                c
                               </a>
                              </strong>
                              ). This finding again demonstrates that the accurate inference of G-P maps requires the explicit and simultaneous modeling of experimental nonlinearities.
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F5.large.jpg?width=800&height=600&carousel=1" title="Analysis of MPSA data from Wong et al..38 This dataset reports PSI values, measured in the BRCA2 exon 17 context, for nearly all 32,768 variant 5’ss of the form NNN/GYNNNN. Data were split 60:20:20 into training, validation, and test sets. Latent phenotype models with one of four types of G-P map (additive, neighbor, pairwise, or black box), as well as a GE measurement process with a heteroscedastic skewed-t noise model, were inferred. The epistasis package of Sailer and Harms24 was also used to infer an additive G-P map and GE nonlinearity. (a) Performance of trained models as quantified by Ivar and Ipre, computed on test data. The lower bound on Iint was estimated from experimental replicates (see Methods). p-value reflects a two-sided z-test. Ivar was not computed for the additive (epistasis package) model because that package does not infer an explicit noise model. (b-d) Measurement values versus latent phenotype values, computed on test data, using the additive (epistasis package) model (b), the additive model (c), and the pairwise model (d). The corresponding GE measurement processes are also shown. (e) Sequence logo46 illustrating the additive effects component of the pairwise G-P map. Dashed line indicates the exon/intron boundary. G at +1 serves as a placeholder because no other bases were assayed at this position. Only values for U and C at +2 were inferred. (f) Heatmap showing the pairwise effects component of the pairwise G-P map. White diagonals correspond to unobserved bases. Error bars indicate standard errors. Supplemental Information Fig. S3 shows the uncertainties in these parameters. MPSA: massively parallel splicing assay; PSI: percent spliced in; G-P: genotype-phenotype; GE: global epistasis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-418940397" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Analysis of MPSA data from Wong et al..<sup>38</sup> This dataset reports PSI values, measured in the <em>BRCA2</em> exon 17 context, for nearly all 32,768 variant 5’ss of the form NNN/GYNNNN. Data were split 60:20:20 into training, validation, and test sets. Latent phenotype models with one of four types of G-P map (additive, neighbor, pairwise, or black box), as well as a GE measurement process with a heteroscedastic skewed-t noise model, were inferred. The epistasis package of Sailer and Harms<sup>24</sup> was also used to infer an additive G-P map and GE nonlinearity. (<strong>a</strong>) Performance of trained models as quantified by <em>I</em><sub>var</sub> and <em>I</em><sub>pre</sub>, computed on test data. The lower bound on <em>I</em><sub>int</sub> was estimated from experimental replicates (see <strong>Methods</strong>). p-value reflects a two-sided z-test. <em>I</em><sub>var</sub> was not computed for the additive (epistasis package) model because that package does not infer an explicit noise model. (<strong>b-d</strong>) Measurement values versus latent phenotype values, computed on test data, using the additive (epistasis package) model (<strong>b</strong>), the additive model (<strong>c</strong>), and the pairwise model (<strong>d</strong>). The corresponding GE measurement processes are also shown. (<strong>e</strong>) Sequence logo<sup>46</sup> illustrating the additive effects component of the pairwise G-P map. Dashed line indicates the exon/intron boundary. G at +1 serves as a placeholder because no other bases were assayed at this position. Only values for U and C at +2 were inferred. (<strong>f</strong>) Heatmap showing the pairwise effects component of the pairwise G-P map. White diagonals correspond to unobserved bases. Error bars indicate standard errors. <strong>Supplemental Information Fig. S3</strong> shows the uncertainties in these parameters. MPSA: massively parallel splicing assay; PSI: percent spliced in; G-P: genotype-phenotype; GE: global epistasis.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F5.medium.gif" width="397" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F5.medium.gif" width="397" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5.
                               </span>
                               <p id="p-29" class="first-child">
                                Analysis of MPSA data from Wong et al..
                                <sup>
                                 <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">
                                  38
                                 </a>
                                </sup>
                                This dataset reports PSI values, measured in the BRCA2 exon 17 context, for nearly all 32,768 variant 5’ss of the form NNN/GYNNNN. Data were split 60:20:20 into training, validation, and test sets. Latent phenotype models with one of four types of G-P map (additive, neighbor, pairwise, or black box), as well as a GE measurement process with a heteroscedastic skewed-t noise model, were inferred. The epistasis package of Sailer and Harms
                                <sup>
                                 <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">
                                  24
                                 </a>
                                </sup>
                                was also used to infer an additive G-P map and GE nonlinearity. (
                                <strong>
                                 a
                                </strong>
                                ) Performance of trained models as quantified by I
                                <sub>
                                 var
                                </sub>
                                and I
                                <sub>
                                 pre
                                </sub>
                                , computed on test data. The lower bound on I
                                <sub>
                                 int
                                </sub>
                                was estimated from experimental replicates (see
                                <strong>
                                 Methods
                                </strong>
                                ). p-value reflects a two-sided z-test. I
                                <sub>
                                 var
                                </sub>
                                was not computed for the additive (epistasis package) model because that package does not infer an explicit noise model. (
                                <strong>
                                 b-d
                                </strong>
                                ) Measurement values versus latent phenotype values, computed on test data, using the additive (epistasis package) model (
                                <strong>
                                 b
                                </strong>
                                ), the additive model (
                                <strong>
                                 c
                                </strong>
                                ), and the pairwise model (
                                <strong>
                                 d
                                </strong>
                                ). The corresponding GE measurement processes are also shown. (
                                <strong>
                                 e
                                </strong>
                                ) Sequence logo
                                <sup>
                                 <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                                  46
                                 </a>
                                </sup>
                                illustrating the additive effects component of the pairwise G-P map. Dashed line indicates the exon/intron boundary. G at +1 serves as a placeholder because no other bases were assayed at this position. Only values for U and C at +2 were inferred. (
                                <strong>
                                 f
                                </strong>
                                ) Heatmap showing the pairwise effects component of the pairwise G-P map. White diagonals correspond to unobserved bases. Error bars indicate standard errors.
                                <strong>
                                 Supplemental Information Fig. S3
                                </strong>
                                shows the uncertainties in these parameters. MPSA: massively parallel splicing assay; PSI: percent spliced in; G-P: genotype-phenotype; GE: global epistasis.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-30">
                              We also observe that increasingly complex G-P maps exhibit increased accuracy. For example, the additive G-P map gives I
                              <sub>
                               pre
                              </sub>
                              = 0.257 ± 0.013 bits, whereas the pairwise G-P map (
                              <strong>
                               <a id="xref-fig-5-5" class="xref-fig" href="#F5">
                                Figs. 5e
                               </a>
                              </strong>
                              ,
                              <strong>
                               <a id="xref-fig-5-6" class="xref-fig" href="#F5">
                                f
                               </a>
                              </strong>
                              ) attains I
                              <sub>
                               pre
                              </sub>
                              = 0.374 ± 0.014 bits. Using MAVE-NN’s built-in parametric bootstrap approach for quantifying parameter uncertainty, we find that both the additive and pairwise G-P map parameters are very precisely determined (see
                              <strong>
                               Supplemental Information Fig. S3
                              </strong>
                              ). The black box G-P map, which is comprised of 5 densely connected hidden layers of 10 nodes each, performed the best of all four G-P maps, achieving I
                              <sub>
                               pre
                              </sub>
                              = 0.458 ± 0.015 bits. Remarkably, this last predictive information value exceeds the lower bound of I
                              <sub>
                               int
                              </sub>
                              ≥ 0.462 ± 0.009 bits, which was estimated from replicate experiments (see
                              <strong>
                               Methods
                              </strong>
                              ). We thus conclude that pairwise interaction models are not flexible enough to fully account for how 5’ss sequences control splicing. More generally, these results underscore the need for software that is capable of inferring and assessing a variety of different G-P maps through a uniform interface.
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Application: biophysically interpretable G-P maps
                             </h3>
                             <p id="p-31">
                              Biophysical models, unlike the phenomenological models considered thus far, have mathematical structures that reflect specific hypotheses about how sequence-dependent interactions between macromolecules mechanistically define G-P maps. Thermodynamic models, which rely on a quasi-equilibrium assumption, are the most commonly used type of biophysical model.
                              <sup>
                               <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                                39
                               </a>
                               –
                               <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                                41
                               </a>
                              </sup>
                              Previous studies have shown that precise thermodynamic models can be inferred from MAVE datasets,
                              <sup>
                               <a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">
                                16
                               </a>
                              </sup>
                              but no software intended for use by the broader MAVE community has yet been developed for doing this. MAVE-NN meets this need by enabling the inference of custom G-P maps. We now demonstrate this biophysical modeling capability in the contexts of protein-ligand binding (using DMS data;
                              <strong>
                               <a id="xref-fig-1-9" class="xref-fig" href="#F1">
                                Fig. 1a
                               </a>
                              </strong>
                              ) and bacterial transcriptional regulation (using sort-seq MPRA data;
                              <strong>
                               <a id="xref-fig-1-10" class="xref-fig" href="#F1">
                                Fig. 1e
                               </a>
                              </strong>
                              ). An expanded discussion of how these models are mathematically formulated and specified within MAVE-NN is provided in the “Biophysical modeling” section of
                              <strong>
                               Supplemental Information
                              </strong>
                              .
                             </p>
                             <p id="p-32">
                              Otwinowski
                              <sup>
                               <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                                42
                               </a>
                              </sup>
                              showed that a three-state thermodynamic G-P map (
                              <strong>
                               <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                                Fig. 6a
                               </a>
                              </strong>
                              ), one that accounts for GB1 folding energy in addition to GB1-IgG binding energy,
                              <sup>
                               <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                                43
                               </a>
                              </sup>
                              can explain the DMS data of Olson et al.
                              <sup>
                               <a id="xref-ref-35-7" class="xref-bibr" href="#ref-35">
                                35
                               </a>
                              </sup>
                              better than a simple additive G-P map does. This biophysical model subsequently received impressive confirmation in the work of Nisthal et al.,
                              <sup>
                               <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                                44
                               </a>
                              </sup>
                              who measured the thermostability of 812 single-mutation GB1 variants. We tested the ability of MAVE-NN to recover the same type of thermodynamic model that Otwinowski had inferred using custom analysis scripts. Our analysis yielded a G-P map with significantly improved performance on the data of Olson et al. (I
                              <sub>
                               var
                              </sub>
                              = 2.303 ± 0.013 bits, I
                              <sub>
                               pre
                              </sub>
                              = 2.357 ± 0.007 bits, R
                              <sup>
                               2
                              </sup>
                              = 0.947 ± 0.001) relative to the additive G-P map of
                              <strong>
                               <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                                Fig. 3
                               </a>
                               .
                               <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                                Fig. 6b
                               </a>
                              </strong>
                              shows the two inferred energy matrices that respectively describe the effects of every possible single-residue mutation on the Gibbs free energies of protein folding and protein-ligand binding. The folding energy predictions of our model also correlate as well with the data of Nisthal et al. (R
                              <sup>
                               2
                              </sup>
                              = 0.570 ± 0.049) as the predictions of Otwinowski’s model does (R
                              <sup>
                               2
                              </sup>
                              = 0.515 ± 0.056). This demonstrates that MAVE-NN can infer accurate and interpretable quantitative models of protein biophysics.
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F6.large.jpg?width=800&height=600&carousel=1" title="Biophysical models inferred from DMS and MPRA data. (a) Thermodynamic model for IgG binding by GB1. This model comprises three GB1 microstates (unfolded, folded-unbound, and folded-bound). The Gibbs free energies of folding (ΔGF) and binding (ΔGB) are computed from sequence using additive models called energy matrices. The latent phenotype is given by the fraction of time GB1 is in the folded-bound state. (b) The ΔΔG parameters of the energy matrices for folding and binding, inferred from the data of Olson et al.35 using GE regression. Supplemental Fig. S5 plots folding energy predictions against the measurements of Nisthal et al..44 (c) A four-state thermodynamic model for transcriptional activation at the E. coli lac promoter. The Gibbs free energies of RNAP-DNA binding (ΔGR) and CRP-DNA binding (ΔGC) are computed using energy matrices, whereas the CRP-RNAP interaction energy ΔGI is a scalar. The latent phenotype is the fraction of time a promoter is bound by RNAP. (d,e) The latent phenotype model inferred from the sort-seq MPRA of Kinney et al.,16 including both the MPA measurement process (d) and the parameters of the thermodynamic G-P map (e). Supplemental Fig. S4 provides detailed definitions of the thermodynamic models in panels a and c. Sequence logos in panel e were generated using Logomaker,46 and standard errors for the protein-protein interaction energy were determined using the built-in parametric bootstrapping approach described in Methods. GE: global epistasis. RNAP: RNA polymerase. MPA: measurement-process agnostic. G-P: genotype-phenotype." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-418940397" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Biophysical models inferred from DMS and MPRA data. (<strong>a</strong>) Thermodynamic model for IgG binding by GB1. This model comprises three GB1 microstates (unfolded, folded-unbound, and folded-bound). The Gibbs free energies of folding (Δ<em>G</em><sub><em>F</em></sub>) and binding (Δ<em>G</em><sub><em>B</em></sub>) are computed from sequence using additive models called energy matrices. The latent phenotype is given by the fraction of time GB1 is in the folded-bound state. (<strong>b</strong>) The ΔΔ<em>G</em> parameters of the energy matrices for folding and binding, inferred from the data of Olson et al.<sup>35</sup> using GE regression. <strong>Supplemental Fig. S5</strong> plots folding energy predictions against the measurements of Nisthal et al..<sup>44</sup> (<strong>c</strong>) A four-state thermodynamic model for transcriptional activation at the <em>E. coli lac</em> promoter. The Gibbs free energies of RNAP-DNA binding (Δ<em>G</em><sub>R</sub>) and CRP-DNA binding (Δ<em>G</em><sub>C</sub>) are computed using energy matrices, whereas the CRP-RNAP interaction energy Δ<em>G</em><sub>I</sub> is a scalar. The latent phenotype is the fraction of time a promoter is bound by RNAP. (<strong>d</strong>,<strong>e</strong>) The latent phenotype model inferred from the sort-seq MPRA of Kinney et al.,<sup>16</sup> including both the MPA measurement process (<strong>d</strong>) and the parameters of the thermodynamic G-P map (<strong>e</strong>). Supplemental <strong>Fig. S4</strong> provides detailed definitions of the thermodynamic models in panels <strong>a</strong> and <strong>c</strong>. Sequence logos in panel <strong>e</strong> were generated using Logomaker,<sup>46</sup> and standard errors for the protein-protein interaction energy were determined using the built-in parametric bootstrapping approach described in Methods. GE: global epistasis. RNAP: RNA polymerase. MPA: measurement-process agnostic. G-P: genotype-phenotype.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F6.medium.gif" width="440" height="293"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F6.medium.gif" width="440" height="293"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/02/2020.07.14.201475/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6.
                               </span>
                               <p id="p-33" class="first-child">
                                Biophysical models inferred from DMS and MPRA data. (
                                <strong>
                                 a
                                </strong>
                                ) Thermodynamic model for IgG binding by GB1. This model comprises three GB1 microstates (unfolded, folded-unbound, and folded-bound). The Gibbs free energies of folding (ΔG
                                <sub>
                                 F
                                </sub>
                                ) and binding (ΔG
                                <sub>
                                 B
                                </sub>
                                ) are computed from sequence using additive models called energy matrices. The latent phenotype is given by the fraction of time GB1 is in the folded-bound state. (
                                <strong>
                                 b
                                </strong>
                                ) The ΔΔG parameters of the energy matrices for folding and binding, inferred from the data of Olson et al.
                                <sup>
                                 <a id="xref-ref-35-8" class="xref-bibr" href="#ref-35">
                                  35
                                 </a>
                                </sup>
                                using GE regression.
                                <strong>
                                 Supplemental Fig. S5
                                </strong>
                                plots folding energy predictions against the measurements of Nisthal et al..
                                <sup>
                                 <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">
                                  44
                                 </a>
                                </sup>
                                (
                                <strong>
                                 c
                                </strong>
                                ) A four-state thermodynamic model for transcriptional activation at the E. coli lac promoter. The Gibbs free energies of RNAP-DNA binding (ΔG
                                <sub>
                                 R
                                </sub>
                                ) and CRP-DNA binding (ΔG
                                <sub>
                                 C
                                </sub>
                                ) are computed using energy matrices, whereas the CRP-RNAP interaction energy ΔG
                                <sub>
                                 I
                                </sub>
                                is a scalar. The latent phenotype is the fraction of time a promoter is bound by RNAP. (
                                <strong>
                                 d
                                </strong>
                                ,
                                <strong>
                                 e
                                </strong>
                                ) The latent phenotype model inferred from the sort-seq MPRA of Kinney et al.,
                                <sup>
                                 <a id="xref-ref-16-7" class="xref-bibr" href="#ref-16">
                                  16
                                 </a>
                                </sup>
                                including both the MPA measurement process (
                                <strong>
                                 d
                                </strong>
                                ) and the parameters of the thermodynamic G-P map (
                                <strong>
                                 e
                                </strong>
                                ). Supplemental
                                <strong>
                                 Fig. S4
                                </strong>
                                provides detailed definitions of the thermodynamic models in panels
                                <strong>
                                 a
                                </strong>
                                and
                                <strong>
                                 c
                                </strong>
                                . Sequence logos in panel
                                <strong>
                                 e
                                </strong>
                                were generated using Logomaker,
                                <sup>
                                 <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">
                                  46
                                 </a>
                                </sup>
                                and standard errors for the protein-protein interaction energy were determined using the built-in parametric bootstrapping approach described in Methods. GE: global epistasis. RNAP: RNA polymerase. MPA: measurement-process agnostic. G-P: genotype-phenotype.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-34">
                              To test MAVE-NN’s ability to infer thermodynamic models of transcriptional regulation, we re-analyzed the MPRA data of Kinney et al.,
                              <sup>
                               <a id="xref-ref-16-8" class="xref-bibr" href="#ref-16">
                                16
                               </a>
                              </sup>
                              in which random mutations to a 75 bp region of the Escherichia coli lac promoter were assayed. This promoter region binds two regulatory proteins, σ
                              <sup>
                               70
                              </sup>
                              RNA polymerase (RNAP) and the transcription factor CRP. As in Kinney et al.,
                              <sup>
                               <a id="xref-ref-16-9" class="xref-bibr" href="#ref-16">
                                16
                               </a>
                              </sup>
                              we proposed a four-state thermodynamic model that quantitatively explains how promoter sequences control transcription rate (
                              <strong>
                               <a id="xref-fig-6-3" class="xref-fig" href="#F6">
                                Fig. 6c
                               </a>
                              </strong>
                              ). The parameters of this G-P map include the Gibbs free energy of interaction between CRP and RNAP (ΔG
                              <sub>
                               I
                              </sub>
                              ), as well as energy matrices that describe the CRP-DNA and RNAP-DNA interaction energies. Because the sort-seq MPRA of Kinney et al. yielded discrete measurement values (
                              <strong>
                               <a id="xref-fig-1-11" class="xref-fig" href="#F1">
                                Figs. 1e
                               </a>
                              </strong>
                              ,
                              <strong>
                               <a id="xref-fig-1-12" class="xref-fig" href="#F1">
                                f
                               </a>
                              </strong>
                              ), we used an MPA measurement process in our latent phenotype model (
                              <strong>
                               <a id="xref-fig-6-4" class="xref-fig" href="#F6">
                                Fig. 6d
                               </a>
                              </strong>
                              ). The biophysical parameter values we thus inferred (
                              <strong>
                               <a id="xref-fig-6-5" class="xref-fig" href="#F6">
                                Fig. 6e
                               </a>
                              </strong>
                              ), including a CRP-RNAP interaction energy of ΔG
                              <sub>
                               I
                              </sub>
                              = −2.598 ± 0.018 kcal/mol, largely match those of Kinney et al., but were obtained far more rapidly (in ∼10 min versus multiple days) thanks to the use of stochastic gradient descent rather than Metropolis Monte Carlo.
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              Constraints on datasets and models
                             </h3>
                             <p id="p-35">
                              As stated above, MAVE-NN places certain limitations on both input datasets and latent phenotype models. Some of these constraints have been adopted to simplify the initial release of MAVE-NN and can potentially be relaxed in future updates. Others reflect fundamental mathematical properties of latent phenotype models. Here we summarize the primary constraints users should be aware of.
                             </p>
                             <p id="p-36">
                              MAVE-NN currently requires that all input sequences be the same length. This constraint has been adopted because a large fraction of MAVE datasets have this form, and all of the built-in G-P maps operate only on fixed-length sequences. Users who wish to analyze variable length sequences can still do so by padding the ends of sequences with dummy characters. Alternatively, users can provide a multiple-sequence alignment as input and include the gap character as one of the characters to consider when training models.
                             </p>
                             <p id="p-37">
                              As stated above, MAVE-NN can analyze MAVE datasets that have either continuous or discrete measurements. At present, both types of measurements must be one-dimensional, i.e., users cannot fit a single model to vectors of multiple measurements (e.g., joint measurements of protein binding affinity and protein stability, as in Ref.
                              <sup>
                               <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                                27
                               </a>
                              </sup>
                              ). This constraint has been adopted only to simplify the user interface of the initial release. It is not a fundamental limitation of latent phenotype models and is scheduled to be relaxed in upcoming versions of MAVE-NN.
                             </p>
                             <p id="p-38">
                              The current implementation of MAVE-NN also supports only one-dimensional latent phenotypes (though the latent phenotype of custom G-P maps can depend on multiple precursor phenotypes, e.g., binding energy or folding energy). This restriction was made because accurately interpreting multi-dimensional latent phenotypes is substantially more fraught than interpreting one-dimensional latent phenotypes, and we believe that additional computational tools need to be developed to facilitate such interpretation. That being said, the mathematical form of latent phenotype models is fully compatible with multi-dimensional latent phenotypes. Indeed, this modeling strategy has been used in other work,
                              <sup>
                               <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">
                                20
                               </a>
                               ,
                               <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">
                                27
                               </a>
                               ,
                               <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">
                                28
                               </a>
                               ,
                               <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                                45
                               </a>
                              </sup>
                              and we plan to enable this functionality in future updates to MAVE-NN.
                             </p>
                             <p id="p-39">
                              More fundamental constraints come into play when analyzing MAVE data that contains only single-mutation variants. In such experiments, the underlying effects of individual mutations are hopelessly confounded by the biophysical, physiological, and experimental nonlinearities that may be present. By contrast, when the same mutation is observed in multiple genetic backgrounds, MAVE-NN can use systematic differences in the mutational effects observed between stronger and weaker backgrounds to remove these confounding influences. Thus, for datasets that comprise only single-mutant effects, we limit MAVE-NN to inferring only additive G-P maps using GE regression, and while the noise model in the GE measurement process is allowed to be heteroscedastic, the nonlinearity is constrained to be linear.
                             </p>
                             <p id="p-40">
                              We emphasize that, in practice, only a modest number of multiple-mutant variants are required for MAVE-NN to learn the form of a non-linear measurement process (see
                              <a id="xref-fig-3-7" class="xref-fig" href="#F3">
                               Fig. 3e-g
                              </a>
                              ). In this way, including a small fraction of the possible double-mutation variants in MAVE libraries can be beneficial even just for determining the effects of single mutations. Adding such non-comprehensive sets of double mutants to MAVE libraries is experimentally straight-forward, and our numerical experiments suggest that assaying roughly the same number of double-mutation variants as single-mutation variants should often suffice. We therefore recommend that experimentalists—even those primarily interested in the effects of single mutations—consider augmenting their MAVE libraries with a small subset of double-mutation variants.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-41">
                             In this work we have presented a unified strategy for inferring quantitative models of G-P maps from diverse MAVE datasets. At the core of our approach is the conceptualization of G-P maps as a form of information compression, i.e., the G-P map first compresses an input sequence into a latent phenotype value, which the MAVE then reads out indirectly via a noisy nonlinear measurement process. By explicitly modeling this measurement process, one can remove potentially confounding effects from the G-P map, as well as accommodate diverse experimental designs. We have also introduced three information-theoretic metrics for assessing the performance of the resulting models. These capabilities have been implemented within an easy-to-use Python package called MAVE-NN.
                            </p>
                            <p id="p-42">
                             We have demonstrated the capabilities of MAVE-NN in diverse biological contexts, including in the analysis of both DMS and MPRA data. We have also demonstrated the superior performance of MAVE-NN relative to the epistasis package of Sailer and Harms.
                             <sup>
                              <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">
                               24
                              </a>
                             </sup>
                             Along the way, we observed that MAVE-NN can deconvolve experimental nonlinearities from additive G-P maps when a relatively small number of sequences containing multiple mutations are included in the assayed libraries. This capability provides a compelling reason for experimentalists to include such sequences in their MAVE libraries, even if they are primarily interested in the effects of single mutations. Finally, we showed how MAVE-NN can learn biophysically interpretable G-P maps from both DMS and MPRA data.
                            </p>
                            <p id="p-43">
                             MAVE-NN thus fills a critical need in the MAVE community, providing user-friendly software capable of learning quantitative models of G-P maps from diverse MAVE datasets. MAVE-NN has a streamlined user interface and is readily installed from PyPI by executing “pip install mavenn” at the command line. Comprehensive documentation and step-by-step tutorials are available at
                             <a href="http://mavenn.readthedocs.io">
                              http://mavenn.readthedocs.io
                             </a>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-11">
                            <h2 class="">
                             Author Contributions
                            </h2>
                            <p id="p-45">
                             AT, WTI, DMM, and JBK conceived the project. AT and JBK wrote the software with assistance from AP and MK. WTI and JBK wrote a preliminary version of the software. AT, MK, and JBK performed the data analysis. AT, DMM, and JBK wrote the manuscript with contributions from MK and AP.
                            </p>
                           </div>
                           <div class="section" id="sec-12">
                            <h2 class="">
                             Availability of Data and Materials
                            </h2>
                            <p id="p-46">
                             MAVE-NN can be installed from PyPI by executing “pip install mavenn” at the POSIX command line. Comprehensive documentation, including step-by-step tutorials, is available at
                             <a href="http://mavenn.readthedocs.io">
                              http://mavenn.readthedocs.io
                             </a>
                             . Source code, the data sets analyzed in this paper, and the scripts used for training the models and making the figures presented herein, are available under an MIT open-source license at
                             <a href="https://github.com/jbkinney/mavenn">
                              https://github.com/jbkinney/mavenn
                             </a>
                             . MAVE-NN version 1.0.0 was used for all of the analysis described in this manuscript and is archived on Zenodo at
                             <a href="https://doi.org/10.5281/zenodo.5812439">
                              https://doi.org/10.5281/zenodo.5812439
                             </a>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-13">
                            <h2 class="">
                             Conflicts of Interest
                            </h2>
                            <p id="p-47">
                             The authors declare that they have no known conflicts of interest.
                            </p>
                           </div>
                           <div class="section" id="sec-14">
                            <h2 class="">
                             Online Methods
                            </h2>
                            <div class="section" id="sec-15">
                             <h3>
                              Notation
                             </h3>
                             <p id="p-48">
                              We represent each MAVE dataset as a set of N observations,
                              <span class="inline-formula" id="inline-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , where each observation consists of a sequence x
                              <sub>
                               n
                              </sub>
                              and a measurement y
                              <sub>
                               n
                              </sub>
                              . Here, y
                              <sub>
                               n
                              </sub>
                              can be either a continuous real-valued number, or a nonnegative integer representing the “bin” in which the nth sequence was found. Note that, in this representation the same sequence x can be observed multiple times, potentially with different values for y due to experimental noise.
                             </p>
                            </div>
                            <div class="section" id="sec-16">
                             <h3>
                              G-P maps
                             </h3>
                             <p id="p-49">
                              We assume that all sequences have the same length L, and that at each of the L positions in each sequence there is one of C possible characters. MAVE-NN represents sequences using a vector of one-hot encoded features of the form
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where l = 0,1, …, L − 1 indexes positions within the sequence, and c indexes the C distinct characters. MAVE-NN supports built-in alphabets for DNA, RNA and protein (with or without stop codons), as well as user-defined sequence alphabets.
                             </p>
                             <p id="p-50">
                              We assume that the latent phenotype is given by a linear function ϕ(x; θ) that depends on a set of G-P map parameters θ. As mentioned in the main text, MAVE-NN supports four types of G-P map models, all of which can be inferred using either GE regression or MPA regression. The additive model is given by,
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and thus each position in x contributes independently to the latent phenotype. The neighbor model is given by,
                              <span class="disp-formula" id="disp-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and further accounts for potential epistatic interactions between neighboring positions. The pairwise model is given by,
                              <span class="disp-formula" id="disp-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-10.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-10.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and includes interactions between all pairs of positions. Note our convention of requiring l′ > l in the pairwise parameters θ
                              <sub>
                               l:c,l′ :c′
                              </sub>
                              .
                             </p>
                             <p id="p-51">
                              Unlike these three parametric models, the black box G-P map does not have a fixed functional form. Rather, it is given by a multilayer perceptron that takes a vector of sequence features (additive, neighbor, or pairwise) as input, contains multiple fully-connected hidden layers with nonlinear activations, and has a single node output with a linear activation. Users are able to specify the number of hidden layers, the number of nodes in each hidden layer, and the activation function used by these nodes.
                             </p>
                             <p id="p-52">
                              MAVE-NN further supports custom G-P maps that users can define by subclassing the G-P map base class. These G-P maps can have arbitrary functional form, e.g., representing specific biophysical hypotheses of sequence function. This feature of MAVE-NN is showcased in the analyses of
                              <strong>
                               <a id="xref-fig-6-6" class="xref-fig" href="#F6">
                                Fig. 6
                               </a>
                              </strong>
                              .
                             </p>
                            </div>
                            <div class="section" id="sec-17">
                             <h3>
                              Gauge modes and diffeomorphic modes
                             </h3>
                             <p id="p-53">
                              G-P maps typically have non-identifiable degrees of freedom that must be fixed, i.e., pinned down, before the values of individual parameters can be meaningfully interpreted or compared between models. These degrees of freedom come in two flavors: gauge modes and diffeomorphic modes. Gauge modes are changes to θ that do not alter the values of the latent phenotype ϕ. Diffeomorphic modes
                              <sup>
                               <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">
                                15
                               </a>
                               ,
                               <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">
                                20
                               </a>
                              </sup>
                              are changes to θ that do alter ϕ, but do so in ways that can be undone by transformations of the measurement process p(y|ϕ). As shown by Kinney and Atwal,
                              <sup>
                               <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">
                                15
                               </a>
                               ,
                               <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">
                                20
                               </a>
                              </sup>
                              the diffeomorphic modes of linear G-P maps like those considered here will in general correspond to affine transformations of ϕ, although additional unconstrained modes can occur in special situations.
                             </p>
                             <p id="p-54">
                              MAVE-NN fixes both gauge modes and diffeomorphic modes of inferred models (except when using custom G-P maps). The diffeomorphic modes of G-P maps are fixed by transforming θ via
                              <span class="disp-formula" id="disp-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-11.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-11.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and then
                              <span class="disp-formula" id="disp-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-12.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-12.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where a = mean({ϕ
                              <sub>
                               n
                              </sub>
                              }) and b = std({ϕ
                              <sub>
                               n
                              </sub>
                              }) are the mean and standard deviation of ϕ values computed on the training data. This produces a corresponding change in latent phenotype values ϕ → (ϕ − a)/b. To avoid altering likelihood values, MAVE-NN makes a corresponding transformation to the measurement process p(y|ϕ). In GE regression this is done by adjusting the GE nonlinearity via
                              <span class="disp-formula" id="disp-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-13.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-13.gif"/>
                                </noscript>
                               </span>
                              </span>
                              while keeping the noise model p(y|ŷ) fixed. In MPA regression MAVE-NN transforms the full measurement process via
                              <span class="disp-formula" id="disp-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-14.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-14.gif"/>
                                </noscript>
                               </span>
                              </span>
                              For the three parametric G-P maps, gauge modes are fixed using what we call the “hierarchical gauge.” Here, the parameters θ are adjusted so that the lower-order terms in ϕ(x; θ) account for the highest possible fraction of variance in ϕ. This procedure requires a probability distribution on sequence space with respect to which these variances are computed. MAVE-NN assumes that such distributions factorize by position, and can thus be represented by a probability matrix with elements p
                              <sub>
                               l:c
                              </sub>
                              , denoting the probability of character c at position l. MAVE-NN provides three built-in choices for this distribution: uniform, empirical, or wildtype. The corresponding values of p
                              <sub>
                               l:c
                              </sub>
                              are given by
                              <span class="disp-formula" id="disp-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-15.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-15.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where n
                              <sub>
                               l:c
                              </sub>
                              denotes the number of sequences (out of N total) that have character c at position l, and
                              <span class="inline-formula" id="inline-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the one-hot encoding of a user-specified wildtype sequence. In particular, the wildtype gauge was used for illustrating the additive G-P maps in
                              <strong>
                               <a id="xref-fig-3-8" class="xref-fig" href="#F3">
                                Fig. 3
                               </a>
                              </strong>
                              and
                              <strong>
                               <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                                Fig. 4
                               </a>
                              </strong>
                              , while the uniform gauge was used for illustrating the pairwise G-P map in
                              <strong>
                               <a id="xref-fig-5-7" class="xref-fig" href="#F5">
                                Fig. 5
                               </a>
                              </strong>
                              and the energy matrices in
                              <strong>
                               <a id="xref-fig-6-7" class="xref-fig" href="#F6">
                                Fig. 6
                               </a>
                              </strong>
                              . After a sequence distribution is chosen, MAVE-NN fixes the gauge of the pairwise G-P map by transforming
                              <span class="disp-formula" id="disp-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-16.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-16.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-17.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-17.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="disp-formula" id="disp-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-18.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-18.gif"/>
                                </noscript>
                               </span>
                              </span>
                              This transformation is also used for the additive and neighbor G-P maps, but with θ
                              <sub>
                               l:c,l′ :c′
                              </sub>
                              = 0 for all l, l′ (additive) or whenever l′ ≠ l + 1 (neighbor).
                             </p>
                            </div>
                            <div class="section" id="sec-18">
                             <h3>
                              GE nonlinearities
                             </h3>
                             <p id="p-55">
                              GE models assume that each measurement y is a nonlinear function of the latent phenotype g(ϕ) plus some noise. In MAVE-NN, this nonlinearity is represented as a sum of tanh sigmoids:
                              <span class="disp-formula" id="disp-formula-13">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-19.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-19.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Here, K specifies the number of hidden nodes contributing to the sum, and α = {a, b
                              <sub>
                               k
                              </sub>
                              , c
                              <sub>
                               k
                              </sub>
                              , d
                              <sub>
                               k
                              </sub>
                              } are trainable parameters. We note that this mathematical form is an example of the bottleneck architecture previously used by
                              <sup>
                               <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                                23
                               </a>
                               ,
                               <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">
                                45
                               </a>
                              </sup>
                              for modeling GE nonlinearities. By default, MAVE-NN constrains g(ϕ; α) to be monotonic in ϕ by requiring all b
                              <sub>
                               k
                              </sub>
                              ≥ 0 and c
                              <sub>
                               k
                              </sub>
                              ≥ 0, but this constraint can be relaxed.
                             </p>
                            </div>
                            <div class="section" id="sec-19">
                             <h3>
                              GE noise models
                             </h3>
                             <p id="p-56">
                              MAVE-NN supports three types of GE noise model: Gaussian, Cauchy, and skew-t.
                             </p>
                             <p id="p-57">
                              These all support the analytic computation of quantiles and confidence intervals, as well as the rapid sampling of simulated measurement values. The Gaussian noise model is given by
                              <span class="disp-formula" id="disp-formula-14">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-20.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-20.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where s denotes the standard deviation. Importantly, MAVE-NN allows this noise model to be heteroskedastic by representing s as an exponentiated polynomial in ŷ, i.e.,
                              <span class="disp-formula" id="disp-formula-15">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-21.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-21.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where K is the order of the polynomial and {a
                              <sub>
                               k
                              </sub>
                              } are trainable parameters. The user has the option to set K, and setting K = 0 renders this noise model homoscedastic. Quantiles are computed using y
                              <sub>
                               q
                              </sub>
                              = ŷ + s √2 erf
                              <sup>
                               −1
                              </sup>
                              (2q − 1) for user-specified values of q ∈ [0,1]. Similarly, the Cauchy noise model is given by
                              <span class="disp-formula" id="disp-formula-16">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-22.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-22.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where the scale parameter s is an exponentiated K’th order polynomial in ŷ, and quantiles are computed using
                              <span class="inline-formula" id="inline-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <p id="p-58">
                              The skew-t noise model is of the form described by Jones and Faddy,
                              <sup>
                               <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">
                                29
                               </a>
                              </sup>
                              and is given by
                              <span class="disp-formula" id="disp-formula-17">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-23.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-23.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <span class="disp-formula" id="disp-formula-18">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-24.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-24.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="disp-formula" id="disp-formula-19">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-25.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-25.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Note that the t statistic here is an affine function of y chosen so that the distribution’s mode (corresponding to t*) is positioned at ŷ. The three parameters of this noise model, {s, a, b}, are each represented using K-th order exponentiated polynomials with trainable coefficients.
                             </p>
                             <p id="p-59">
                              Quantiles are computed using
                              <span class="disp-formula" id="disp-formula-20">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-26.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-26.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <span class="disp-formula" id="disp-formula-21">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-27.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-27.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and I
                              <sup>
                               −1
                              </sup>
                              denotes the inverse of the regularized incomplete Beta function I
                              <sub>
                               x
                              </sub>
                              (a, b).
                             </p>
                            </div>
                            <div class="section" id="sec-20">
                             <h3>
                              Empirical noise models
                             </h3>
                             <p id="p-60">
                              MAVE-NN further supports the inference of GE regression models that account for user-specified measurement noise. In such cases, the user provides a set of measurement-specific uncertainties
                              <span class="inline-formula" id="inline-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              along with the corresponding sequences and measurements. These uncertainties can, for example, be estimated by using a software package like Enrich2
                              <sup>
                               <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                                11
                               </a>
                              </sup>
                              or DiMSum
                              <sup>
                               <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">
                                14
                               </a>
                              </sup>
                              . MAVE-NN then trains the parameters of latent phenotype models by assuming a Gaussian noise model of the form
                              <span class="disp-formula" id="disp-formula-22">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-28.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-28.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where ŷ
                              <sub>
                               n
                              </sub>
                              = g(f(x
                              <sub>
                               n
                              </sub>
                              ; θ); α) is the expected measurement for sequence x
                              <sub>
                               n
                              </sub>
                              , θ denotes G-P map parameters, and α denotes the parameters of the GE nonlinearity. This noise model thus has the advantage of having no free parameters, but it may be problematically mis-specified if the true error distribution is heavy-tailed or skewed.
                             </p>
                            </div>
                            <div class="section" id="sec-21">
                             <h3>
                              MPA measurement process
                             </h3>
                             <p id="p-61">
                              In MPA regression, MAVE-NN directly models the measurement process p(y|ϕ). At present, MAVE-NN only supports MPA regression for discrete values of y indexed using nonnegative integers. MAVE-NN supports two alternative forms of input for MPA regression. One is a set of sequence-measurement pairs
                              <span class="inline-formula" id="inline-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , where N is the total number of reads, {x
                              <sub>
                               n
                              </sub>
                              } is a set of (typically) non-unique sequences, each y
                              <sub>
                               n
                              </sub>
                              ∈ {0,1, …, Y − 1} is a bin number, and Y is the total number of bins. The other is a set of sequence-count-vector pairs
                              <span class="inline-formula" id="inline-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-6.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-6.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , where M is the total number of unique sequences and c
                              <sub>
                               m
                              </sub>
                              = (c
                              <sub>
                               m0
                              </sub>
                              , c
                              <sub>
                               m1
                              </sub>
                              , …, c
                              <sub>
                               m(Y−1)
                              </sub>
                              ) is a vector that lists the number of times c
                              <sub>
                               my
                              </sub>
                              that the sequence x
                              <sub>
                               m
                              </sub>
                              was observed in each bin y. MPA measurement processes are represented as multilayer perceptron with one hidden layer (having tanh activations) and a softmax output layer. Specifically,
                              <span class="disp-formula" id="disp-formula-23">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-29.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-29.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <span class="disp-formula" id="disp-formula-24">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-30.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-30.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and K is the number of hidden nodes per value of y. The trainable parameters of this measurement process are η = {a
                              <sub>
                               y
                              </sub>
                              , b
                              <sub>
                               yk
                              </sub>
                              , c
                              <sub>
                               yk
                              </sub>
                              , d
                              <sub>
                               yk
                              </sub>
                              }.
                             </p>
                            </div>
                            <div class="section" id="sec-22">
                             <h3>
                              Loss function
                             </h3>
                             <p id="p-62">
                              Let θ denote the G-P map parameters, and η denote the parameters of the measurement process. MAVE-NN optimizes these parameters using stochastic gradient descent on a loss function given by
                              <span class="disp-formula" id="disp-formula-25">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-31.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-31.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where ℒ
                              <sub>
                               like
                              </sub>
                              is the negative log likelihood of the model, given by
                              <span class="disp-formula" id="disp-formula-26">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-32.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-32.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where ϕ
                              <sub>
                               n
                              </sub>
                              = ϕ(x
                              <sub>
                               n
                              </sub>
                              ; θ), and ℒ
                              <sub>
                               reg
                              </sub>
                              provides for regularization of the model parameters.
                             </p>
                             <p id="p-63">
                              In the context of GE regression, we can write η = (α, β) where α represents the parameters of the GE nonlinearity g(ϕ; α), and β denotes the parameters of the noise model p(y|ŷ; β). The likelihood contribution from each observation n then becomes p(y
                              <sub>
                               n
                              </sub>
                              |ϕ
                              <sub>
                               n
                              </sub>
                              ; η) = p(y
                              <sub>
                               n
                              </sub>
                              |ŷ
                              <sub>
                               n
                              </sub>
                              ; β) where ŷ
                              <sub>
                               n
                              </sub>
                              = g(ϕ
                              <sub>
                               n
                              </sub>
                              ; α). In the context of MPA regression with a dataset of the form
                              <span class="inline-formula" id="inline-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , the loss function simplifies to
                              <span class="disp-formula" id="disp-formula-27">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-33.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-33.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where ϕ
                              <sub>
                               m
                              </sub>
                              = ϕ(x
                              <sub>
                               m
                              </sub>
                              ; θ). For the regularization term, MAVE-NN uses an L
                              <sub>
                               2
                              </sub>
                              penalty of the form
                              <span class="disp-formula" id="disp-formula-28">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-34.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-34.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where the user-adjusted parameters λ
                              <sub>
                               θ
                              </sub>
                              and λ
                              <sub>
                               η
                              </sub>
                              respectively control the strength of regularization for the G-P map and measurement process parameters.
                             </p>
                            </div>
                            <div class="section" id="sec-23">
                             <h3>
                              Predictive information
                             </h3>
                             <p id="p-64">
                              In what follows, we use p
                              <sub>
                               model
                              </sub>
                              (y|ϕ) to denote a measurement process inferred by MAVE-NN, whereas p
                              <sub>
                               true
                              </sub>
                              (y|ϕ) denotes the empirical conditional distribution of y and ϕ values that would be observed in the limit of infinite test data.
                             </p>
                             <p id="p-65">
                              Predictive information I
                              <sub>
                               pre
                              </sub>
                              = I[y; ϕ], where I[·;·] represents mutual information computed on data not used for training (i.e., a held-out test set or data from a different experiment), I
                              <sub>
                               pre
                              </sub>
                              provides a measure of how strongly a G-P map predicts experimental measurements. Importantly, this quantity does not depend on the corresponding measurement process p
                              <sub>
                               model
                              </sub>
                              (y|ϕ). To estimate I
                              <sub>
                               pre
                              </sub>
                              , we use k’th nearest neighbor (kNN) estimators of entropy and mutual information adapted from the NPEET Python package.
                              <sup>
                               <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                                47
                               </a>
                              </sup>
                              Here, the user has the option of adjusting k, which controls a variance/bias tradeoff. When y is discrete (MPA regression), I
                              <sub>
                               pre
                              </sub>
                              is computed using the classic kNN entropy estimator
                              <sup>
                               <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                                48
                               </a>
                               ,
                               <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                                49
                               </a>
                              </sup>
                              via the decomposition I[y; ϕ] = H[ϕ] − ∑
                              <sub>
                               y
                              </sub>
                              p (y)H
                              <sub>
                               y
                              </sub>
                              [ϕ], where H
                              <sub>
                               y
                              </sub>
                              [ϕ] denotes the entropy of p
                              <sub>
                               true
                              </sub>
                              (ϕ|y). When y is continuous (GE regression), I[y; ϕ] is estimated using the kNN-based Kraskov Stögbauer Grassberger (KSG) algorithm.
                              <sup>
                               <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">
                                49
                               </a>
                              </sup>
                              This approach optionally supports the local nonuniformity correction of Gao et al., which is important when y and ϕ exhibit strong dependencies, but which also requires substantially more time to compute.
                             </p>
                            </div>
                            <div class="section" id="sec-24">
                             <h3>
                              Variational information
                             </h3>
                             <p id="p-66">
                              We define variational information as an affine transformation of ℒ
                              <sub>
                               like
                              </sub>
                              ,
                              <span class="disp-formula" id="disp-formula-29">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-35.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-35.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Here, H[y] is the entropy of the data {y
                              <sub>
                               n
                              </sub>
                              }, which is estimated using the k’th nearest neighbor (kNN) estimator from the NPEET package.
                              <sup>
                               <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">
                                47
                               </a>
                              </sup>
                              Noting that this quantity can also be written as I
                              <sub>
                               var
                              </sub>
                              = H[y] − mean({Q
                              <sub>
                               n
                              </sub>
                              }), where Q
                              <sub>
                               n
                              </sub>
                              = −log
                              <sub>
                               2
                              </sub>
                              p(y
                              <sub>
                               n
                              </sub>
                              |ϕ
                              <sub>
                               n
                              </sub>
                              ), we estimate the associated uncertainty using
                              <span class="disp-formula" id="disp-formula-30">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-36.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-36.gif"/>
                                </noscript>
                               </span>
                              </span>
                              The inference strategy used by MAVE-NN is based on the fact that I
                              <sub>
                               var
                              </sub>
                              provides a tight variational lower bound on I
                              <sub>
                               pre
                              </sub>
                              . Indeed, in the large data limit,
                              <span class="disp-formula" id="disp-formula-31">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-37.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-37.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where D
                              <sub>
                               KL
                              </sub>
                              (·) ≥ 0 is the Kullback-Leibler divergence, and thus quantifies the accuracy of the inferred measurement process. From
                              <strong>
                               <a id="xref-disp-formula-31-1" class="xref-disp-formula" href="#disp-formula-31">
                                Eq. 30
                               </a>
                              </strong>
                              one can see that, with appropriate caveats, maximizing I
                              <sub>
                               var
                              </sub>
                              (or equivalently, ℒ
                              <sub>
                               like
                              </sub>
                              ) will also maximize I
                              <sub>
                               pre
                              </sub>
                              .
                              <sup>
                               <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">
                                20
                               </a>
                              </sup>
                              But unlike I
                              <sub>
                               pre
                              </sub>
                              , I
                              <sub>
                               var
                              </sub>
                              is readily compatible with backpropagation and stochastic gradient descent. See Supplemental Information for a derivation of
                              <strong>
                               <a id="xref-disp-formula-31-2" class="xref-disp-formula" href="#disp-formula-31">
                                Eq. 30
                               </a>
                              </strong>
                              and an expanded discussion of this key point. Note: Sharpee et al.
                              <sup>
                               <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                                50
                               </a>
                              </sup>
                              cleverly showed that I
                              <sub>
                               pre
                              </sub>
                              can, in fact, be optimized using stochastic gradient descent. Computing gradients of I
                              <sub>
                               pre
                              </sub>
                              , however, requires a time-consuming density estimation step. Optimizing I
                              <sub>
                               var
                              </sub>
                              , on the other hand, can be done using standard per-datum backpropagation.
                             </p>
                            </div>
                            <div class="section" id="sec-25">
                             <h3>
                              Intrinsic information
                             </h3>
                             <p id="p-67">
                              Intrinsic information, I
                              <sub>
                               int
                              </sub>
                              = I[x; y], is the mutual information between the sequences x and measurements y in a dataset. This quantity is somewhat tricky to estimate due to the high-dimensional nature of sequence space. We instead used three different methods to obtain the upper and lower bounds on I
                              <sub>
                               int
                              </sub>
                              shown in
                              <strong>
                               <a id="xref-fig-3-9" class="xref-fig" href="#F3">
                                Fig. 3d
                               </a>
                              </strong>
                              and
                              <strong>
                               <a id="xref-fig-5-8" class="xref-fig" href="#F5">
                                Fig. 5a
                               </a>
                              </strong>
                              . More generally, we believe the development of both computational and experimental methods for estimating I
                              <sub>
                               int
                              </sub>
                              is be an important avenue for future research.
                             </p>
                             <p id="p-68">
                              To compute the upper bound on I
                              <sub>
                               int
                              </sub>
                              for GB1 data (in
                              <strong>
                               <a id="xref-fig-3-10" class="xref-fig" href="#F3">
                                Fig. 3d
                               </a>
                              </strong>
                              ), we used the fact that
                              <span class="disp-formula" id="disp-formula-32">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-38.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-38.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where H[y] is the entropy of all measurements y, H
                              <sub>
                               x
                              </sub>
                              [y] is the entropy of p(y|x) for a specific choice of sequence x, and ⟨·⟩
                              <sub>
                               x
                              </sub>
                              indicates averaging over all sequences x. In this dataset, the measurement values were computed using
                              <span class="disp-formula" id="disp-formula-33">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-39.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-39.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where c
                              <sub>
                               i
                              </sub>
                              is the input read count and c
                              <sub>
                               s
                              </sub>
                              is the selected read count. H[y] was estimated using the KNN estimator.
                              <sup>
                               <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">
                                48
                               </a>
                              </sup>
                              We estimated the uncertainty in y by propagating errors expected due to Poisson fluctuations in read counts, which gives
                              <span class="disp-formula" id="disp-formula-34">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-40.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-40.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Then, assuming p(y|x) to be approximately Gaussian, we find the corresponding conditional entropy to be
                              <span class="disp-formula" id="disp-formula-35">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-41.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-41.gif"/>
                                </noscript>
                               </span>
                              </span>
                              These H[y] and H
                              <sub>
                               x
                              </sub>
                              [y] values were then used in
                              <strong>
                               <a id="xref-disp-formula-32-1" class="xref-disp-formula" href="#disp-formula-32">
                                Eq. 31
                               </a>
                              </strong>
                              to estimate I
                              <sub>
                               int
                              </sub>
                              . This should provide an upper bound on the true value of I
                              <sub>
                               int
                              </sub>
                              because uncertainty in y must be at least that expected under Poisson sampling of reads. We note, however, that the use of linear error propagation and the assumption that p(y|x) is approximately Gaussian complicate this conclusion. Also, when applied to MPSA data, this method yielded an upper bound of 0.96 bits. We believe this value is likely to be far higher than the true value of I
                              <sub>
                               int
                              </sub>
                              , and that this mismatch probably resulted from read counts in the MPSA data being over-dispersed.
                             </p>
                             <p id="p-69">
                              To compute the lower bound on I
                              <sub>
                               int
                              </sub>
                              for GB1 data (
                              <strong>
                               <a id="xref-fig-3-11" class="xref-fig" href="#F3">
                                Fig. 3d
                               </a>
                              </strong>
                              ) we used the predictive information I
                              <sub>
                               pre
                              </sub>
                              (on test data) of a GE regression model having a blackbox G-P map. This provides a lower bound because I
                              <sub>
                               int
                              </sub>
                              ≥ I
                              <sub>
                               pre
                              </sub>
                              for any model (when evaluated on test data) due to the Data Processing Inequality and the Markov Chain nature of the dependencies y ← x → ϕ in
                              <strong>
                               <a id="xref-fig-2-6" class="xref-fig" href="#F2">
                                Fig. 2e
                               </a>
                              </strong>
                              .
                              <sup>
                               <a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">
                                20
                               </a>
                               ,
                               <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">
                                31
                               </a>
                              </sup>
                             </p>
                             <p id="p-70">
                              To compute a lower bound on I
                              <sub>
                               int
                              </sub>
                              for MPSA data (
                              <a id="xref-fig-5-9" class="xref-fig" href="#F5">
                               <strong>
                                Fig. 5c
                               </strong>
                              </a>
                              ), we leveraged the availability of replicate data in Wong et al..
                              <sup>
                               <a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">
                                38
                               </a>
                              </sup>
                              Let y and y′ represent the original and replicate measurements obtained for a sequence x. Because y ← x → y′ forms a Markov chain, I[x; y] ≥ I[y; y′].
                              <sup>
                               <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">
                                31
                               </a>
                              </sup>
                              We therefore used an estimate of I[y; y′], computed using the KSG method,
                              <sup>
                               <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">
                                47
                               </a>
                               ,
                               <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">
                                49
                               </a>
                              </sup>
                              as the lower bound for I
                              <sub>
                               int
                              </sub>
                              .
                             </p>
                            </div>
                            <div class="section" id="sec-26">
                             <h3>
                              Uncertainties in kNN estimates of mutual information
                             </h3>
                             <p id="p-71">
                              MAVE-NN quantifies uncertainties in H[y] and I[y; ϕ] using multiple random samples of half the data. Let 𝒟
                              <sub>
                               100%
                              </sub>
                              denote a full dataset, and let 𝒟
                              <sub>
                               50%,r
                              </sub>
                              denote a 50% subsample (indexed by r) of this dataset. Given an estimator E(·) of either entropy or mutual information, as well as the number of subsamples R to use, the uncertainty in E(𝒟
                              <sub>
                               100%
                              </sub>
                              ) is estimated as
                              <span class="disp-formula" id="disp-formula-36">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-42.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-42.gif"/>
                                </noscript>
                               </span>
                              </span>
                              MAVE-NN uses R = 25 by default. We note that computing such uncertainty estimates substantially increases computation time, as E(·) needs to be evaluated R + 1 times instead of just once. We also note that bootstrap resampling
                              <sup>
                               <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                                51
                               </a>
                               ,
                               <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">
                                52
                               </a>
                              </sup>
                              is often inadvisable in this context, as it systematically underestimates H[y] and overestimates I[y; z].
                             </p>
                            </div>
                            <div class="section" id="sec-27">
                             <h3>
                              Uncertainties in G-P map parameters
                             </h3>
                             <p id="p-72">
                              Given a trained latent phenotype model, having G-P map parameters θ* and measurement process parameters η*, MAVE-NN can optionally assess model uncertainty using the following parametric bootstrap approach. Using the trained model with parameters (θ*, η*) as “ground truth”, MAVE-NN simulates R (chosen by the user) different MAVE datasets
                              <span class="inline-formula" id="inline-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , where r = 0,1, …, R − 1 indexes the different simulations. Note that the sequences in these simulated datasets are the same as the original training sequence, but the measurements are different. For each simulated dataset 𝒟
                              <sub>
                               r
                              </sub>
                              , MAVE-NN then trains a new model, by default using the same hyperparameters as were used for the ground truth model. This procedure yields a set
                              <span class="inline-formula" id="inline-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                              of simulation-inferred G-P map parameters and corresponding measurement process parameters. Users can then use this sampling of G-P map parameters to estimate uncertainties, e.g., by reporting
                              <span class="inline-formula" id="inline-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-10.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-10.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <p id="p-73">
                              An important detail when assessing parameter uncertainty is to ensure that both the gauge modes and the diffeomorphic modes of each model are fixed. This is necessary so that differences in the parameters that cannot result in changes to model predictions do not inflate the uncertainty estimates. For additive, neighbor, and pairwise G-P maps, MAVE-NN automatically implements the procedure described in the “Gauge modes and diffeomorphic modes” section above, thereby removing these extra degrees of freedom. However, for more complex models such as those implemented by MAVE-NN’s custom G-P map functionality (e.g., representing biophysical models) different gauge freedoms and diffeomorphic modes may arise depending on the details of the model, and users must take care to determine and fix these prior to assessing parameter uncertainty. We also note that no meaningful computation of individual parameter uncertainties is likely to be possible for highly over-parameterized models, such as the “black box” deep neural network models supported by MAVE-NN.
                             </p>
                            </div>
                            <div class="section" id="sec-28">
                             <h3>
                              Datasets
                             </h3>
                             <p id="p-74">
                              For the GB1 DMS dataset of Olson et al.,
                              <sup>
                               <a id="xref-ref-35-9" class="xref-bibr" href="#ref-35">
                                35
                               </a>
                              </sup>
                              measurements were computed using
                              <span class="disp-formula" id="disp-formula-37">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-43.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-43.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <span class="inline-formula" id="inline-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-11.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-11.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-12.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-12.gif"/>
                                </noscript>
                               </span>
                              </span>
                              respectively represent the number of reads from the input and output samples (i.e., pre-selection and post-selection libraries), and n = WT represents the 55 aa wildtype sequence, corresponding to positions 2-56 of the GB1 domain. To infer the model in
                              <strong>
                               <a id="xref-fig-3-12" class="xref-fig" href="#F3">
                                Fig. 3b
                               </a>
                              </strong>
                              and to compute the information metrics in
                              <strong>
                               <a id="xref-fig-3-13" class="xref-fig" href="#F3">
                                Fig. 3c
                               </a>
                              </strong>
                              , only double-mutant sequences with
                              <span class="inline-formula" id="inline-formula-13">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-13.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-13.gif"/>
                                </noscript>
                               </span>
                              </span>
                              were used; these represent 530,737 out of the 536,085 possible double mutants. For the models in
                              <strong>
                               <a id="xref-fig-3-14" class="xref-fig" href="#F3">
                                Figs. 3d-f
                               </a>
                              </strong>
                              , y
                              <sub>
                               n
                              </sub>
                              values for the 1045 single-mutant were also used in the inference procedure.
                             </p>
                             <p id="p-75">
                              For the Aβ DMS data of Seuma et al.
                              <sup>
                               <a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">
                                36
                               </a>
                              </sup>
                              and TDP-43 DMS data of Bolognesi et al.,
                              <sup>
                               <a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                              </sup>
                              y
                              <sub>
                               n
                              </sub>
                              values respectively represent nucleation scores and toxicity scores reported by the authors.
                             </p>
                             <p id="p-76">
                              For the MPSA data of Wong et al.,
                              <sup>
                               <a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">
                                38
                               </a>
                              </sup>
                              we used the data of library 1 replicate 1 obtained for the BRCA2 minigene data. Measurements were computed as
                              <span class="disp-formula" id="disp-formula-38">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-44.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/graphic-44.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <span class="inline-formula" id="inline-formula-14">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-14.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-14.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-15">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-15.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-15.gif"/>
                                </noscript>
                               </span>
                              </span>
                              respectively represent the number of barcode reads obtained from exon inclusion isoforms and from total mRNA, and n = CONS corresponds to the consensus 5’ss sequence CAG/GUAAGU. Corresponding PSI values were computed as
                              <span class="inline-formula" id="inline-formula-16">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-16.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-16.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Only sequences with
                              <span class="inline-formula" id="inline-formula-17">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-17.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/02/2020.07.14.201475/embed/inline-graphic-17.gif"/>
                                </noscript>
                               </span>
                              </span>
                              were used, representing 30,483 of the 32,768 possible sequences of the form NNN/GYNNNN.
                             </p>
                             <p id="p-77">
                              For the lac promoter sort-seq MPRA data of Kinney et al.,
                              <sup>
                               <a id="xref-ref-16-10" class="xref-bibr" href="#ref-16">
                                16
                               </a>
                              </sup>
                              we used data from the “full-wt” experiment (available at
                              <a href="https://github.com/jbkinney/09_sortseq">
                               https://github.com/jbkinney/09_sortseq
                              </a>
                              ).
                             </p>
                            </div>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-44">
                             This work was supported by NIH grant 1R35GM133777 (awarded to JBK), NIH Grant 1R35GM133613 (awarded to DMM), an Alfred P. Sloan Research Fellowship (awarded to DMM), a grant from the CSHL/Northwell Health partnership, and funding from the Simons Center for Quantitative Biology at Cold Spring Harbor Laboratory.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-4">
                               Moderate revisions throughout.
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-2">
                              <p id="p-5">
                               <a href="https://mavenn.readthedocs.io/">
                                https://mavenn.readthedocs.io/
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kinney
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. B.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McCandlish
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Massively parallel assays and quantitative sequence-function relationships
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Annu Rev Genom Hum G
                                 </abbr>
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  99
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  127
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BGenom%2BHum%2BG%26rft.volume%253D20%26rft.spage%253D99%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.2" data-doi="10.1016/j.ajhg.2017.07.014">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Starita
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L. M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Variant interpretation: functional assays to the rescue
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Am J Hum Genetics
                                 </abbr>
                                 <span class="cit-vol">
                                  101
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  315
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  325
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAm%2BJ%2BHum%2BGenetics%26rft.volume%253D101%26rft.spage%253D315%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2017.07.014%26rft_id%253Dinfo%253Apmid%252F28886340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2017.07.014&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28886340&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F02%2F2020.07.14.201475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.3" data-doi="10.1038/nmeth.3027">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fowler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fields
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Deep mutational scanning: a new style of protein science
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Methods
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  801
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  807
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D11%26rft.spage%253D801%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3027%26rft_id%253Dinfo%253Apmid%252F25075907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nmeth.3027&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25075907&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F02%2F2020.07.14.201475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000340075600018&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.4" data-doi="10.1038/nrg3684">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Levo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Segal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  In pursuit of design principles of regulatory sequences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Rev Genet
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  453
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  468
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BGenet%26rft.volume%253D15%26rft.spage%253D453%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3684%26rft_id%253Dinfo%253Apmid%252F24913666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrg3684&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24913666&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F02%2F2020.07.14.201475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.14.201475v4.5" data-doi="10.1016/j.ygeno.2015.06.003">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   White
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Understanding how cis-regulatory function is encoded in DNA sequence using massively parallel reporter assays and designed sequences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genomics
                                 </abbr>
                                 <span class="cit-vol">
                                  106
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  165
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  170
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.volume%253D106%26rft.spage%253D165%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2015.06.003%26rft_id%253Dinfo%253Apmid%252F26072432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2015.06.003&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26072432&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F02%2F2020.07.14.201475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.14.201475v4.6" data-doi="10.1016/j.ygeno.2015.06.005">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Inoue
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ahituv
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Decoding enhancers using massively parallel reporter assays
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genomics
                                 </abbr>
                                 <span class="cit-vol">
                                  106
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  159
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  164
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.volume%253D106%26rft.spage%253D159%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2015.06.005%26rft_id%253Dinfo%253Apmid%252F26072433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2015.06.005&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26072433&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F02%2F2020.07.14.201475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.7" data-doi="10.1186/s12864-016-2533-5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Peterman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Levine
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Sort-seq under the hood: implications of design choices on large-scale characterization of sequence-function relations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Genomics
                                 </abbr>
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  206
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D206%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-2533-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s12864-016-2533-5&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.8" data-doi="10.1093/bioinformatics/btr577">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fowler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Araya
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gerard
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fields
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Enrich: software for analysis of protein function by enrichment and depletion of variants
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3430
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3431
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr577%26rft_id%253Dinfo%253Apmid%252F22006916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr577&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22006916&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F02%2F2020.07.14.201475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000297860000017&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.14.201475v4.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alam
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. L.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burke
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. H.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  FASTAptamer: A bioinformatic toolkit for high-throughput sequence analysis of combinatorial selections
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol Ther-Nucleic Acids
                                 </abbr>
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e230
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol%2BTher-Nucleic%2BAcids%26rft.volume%253D4%26rft.spage%253D230e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.14.201475v4.10" data-doi="10.1186/s12859-015-0590-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bloom
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. D.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Software for the analysis and visualization of deep mutational scanning data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  168
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D16%26rft.spage%253D168%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-015-0590-4%26rft_id%253Dinfo%253Apmid%252F25990960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s12859-015-0590-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25990960&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F02%2F2020.07.14.201475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.11" data-doi="10.1186/s13059-016-1145-3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rubin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. F.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  A statistical framework for analyzing deep mutational scanning data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  15
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D18%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-1145-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-016-1145-3&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.14.201475v4.12" data-doi="10.1186/s13059-019-1787-z">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ashuach
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  MPRAnalyze: statistical framework for massively parallel reporter assays
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  183
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D20%26rft.spage%253D183%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-019-1787-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-019-1787-z&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.14.201475v4.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Niroula
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ajore
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nilsson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  MPRAscore: robust and non-parametric analysis of massively parallel reporter assays
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  35
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  5351
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  5353
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D35%26rft.spage%253D5351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.14" data-doi="10.1186/s13059-020-02091-3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Faure
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schmiedel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Baeza-Centurion
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lehner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  DiMSum: an error model and pipeline for analyzing deep mutational scanning data and diagnosing common experimental pathologies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  21
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  207
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D21%26rft.spage%253D207%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-020-02091-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-020-02091-3&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.15">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Atwal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kinney
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. B.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Learning quantitative sequence-function relationships from massively parallel experiments
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Stat Phys
                                 </abbr>
                                 <span class="cit-vol">
                                  162
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1203
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1243
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BStat%2BPhys%26rft.volume%253D162%26rft.spage%253D1203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.16" data-doi="10.1073/pnas.1004290107">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kinney
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Murugan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Callan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. G.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cox
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. C.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Using deep sequencing to characterize the biophysical mechanism of a transcriptional regulatory sequence
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc Natl Acad Sci USA
                                 </abbr>
                                 <span class="cit-vol">
                                  107
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  9158
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  9163
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1004290107%26rft_id%253Dinfo%253Apmid%252F20439748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA3LzIwLzkxNTgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMS8wMi8yMDIwLjA3LjE0LjIwMTQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.14.201475v4.17" data-doi="10.1038/nbt.2137">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Melnikov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Systematic dissection and optimization of inducible enhancers in human cells using a massively parallel reporter assay
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Biotechnol
                                 </abbr>
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  271
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  277
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DMelnikov%26rft.auinit1%253DA.%26rft.volume%253D30%26rft.issue%253D3%26rft.spage%253D271%26rft.epage%253D277%26rft.atitle%253DSystematic%2Bdissection%2Band%2Boptimization%2Bof%2Binducible%2Benhancers%2Bin%2Bhuman%2Bcells%2Busing%2Ba%2Bmassively%2Bparallel%2Breporter%2Bassay.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2137%26rft_id%253Dinfo%253Apmid%252F22371084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nbt.2137&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22371084&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F02%2F2020.07.14.201475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.18" data-doi="10.1101/gr.157891.113">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mogno
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kwasnieski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cohen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Massively parallel synthetic promoter assays reveal the in vivo effects of binding site variants
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Res
                                 </abbr>
                                 <span class="cit-vol">
                                  23
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1908
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1915
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.157891.113%26rft_id%253Dinfo%253Apmid%252F23921661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xMS8xOTA4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMDIvMjAyMC4wNy4xNC4yMDE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.19">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Abadi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  TensorFlow: A Systems for Large-Scale Machine Learning
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  in Proceedings of the 12th USENIX Symposium on Operating Systems Design and Implementation (OSDI ‘16)
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.20" data-doi="10.1162/NECO_a_00568">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kinney
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. B.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Atwal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Parametric inference in the large data limit using maximally informative models
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neural Comput
                                 </abbr>
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  637
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  653
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeural%2BComput%26rft.volume%253D26%26rft.spage%253D637%26rft_id%253Dinfo%253Adoi%252F10.1162%252FNECO_a_00568%26rft_id%253Dinfo%253Apmid%252F24479782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1162/NECO_a_00568&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24479782&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F02%2F2020.07.14.201475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.21" data-doi="10.1073/pnas.0609908104">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kinney
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tkačik
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Callan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. G.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Precise physical models of protein-DNA interaction from high-throughput data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc Natl Acad Sci USA
                                 </abbr>
                                 <span class="cit-vol">
                                  104
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  501
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  506
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0609908104%26rft_id%253Dinfo%253Apmid%252F17197415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDQvMi81MDEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMS8wMi8yMDIwLjA3LjE0LjIwMTQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.22" data-doi="10.1371/journal.pone.0061570">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Otwinowski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nemenman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Genotype to phenotype mapping and the fitness landscape of the E. coli lac promoter
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS ONE
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e61570
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De61570%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0061570%26rft_id%253Dinfo%253Apmid%252F23650500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0061570&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23650500&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F02%2F2020.07.14.201475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.23" data-doi="10.1038/nature17995">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sarkisyan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Local fitness landscape of the green fluorescent protein
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  533
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  397
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  401
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D533%26rft.spage%253D397%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature17995%26rft_id%253Dinfo%253Apmid%252F27193686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature17995&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27193686&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F02%2F2020.07.14.201475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.24" data-doi="10.1534/genetics.116.195214">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sailer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z. R.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harms
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Detecting high-order epistasis in nonlinear genotype-phenotype maps
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics
                                 </abbr>
                                 <span class="cit-vol">
                                  205
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1079
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1088
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.195214%26rft_id%253Dinfo%253Apmid%252F28100592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNS8zLzEwNzkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMS8wMi8yMDIwLjA3LjE0LjIwMTQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               25.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.25" data-doi="10.1073/pnas.1804015115">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Otwinowski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McCandlish
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Plotkin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. B.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Inferring the shape of global epistasis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc Natl Acad Sci USA
                                 </abbr>
                                 <span class="cit-vol">
                                  115
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  E7550
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  E7558
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1804015115%26rft_id%253Dinfo%253Apmid%252F30037990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE1LzMyL0U3NTUwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMDIvMjAyMC4wNy4xNC4yMDE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               26.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.26" data-doi="10.1073/pnas.2104878118">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gelman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fahlberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Heinzelman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Romero
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gitter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Neural networks to learn protein sequence-function relationships from deep mutational scanning data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc Natl Acad Sci USA
                                 </abbr>
                                 <span class="cit-vol">
                                  118
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e2104878118
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.2104878118%26rft_id%253Dinfo%253Apmid%252F34815338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxODoiMTE4LzQ4L2UyMTA0ODc4MTE4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMDIvMjAyMC4wNy4xNC4yMDE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               27.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.27" data-doi="10.1101/2021.09.14.460249">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Faure
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Global mapping of the energetic and allosteric landscapes of protein binding domains
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/2021.09.14.460249
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2021.09.14.460249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMS4wOS4xNC40NjAyNDl2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAxLzAyLzIwMjAuMDcuMTQuMjAxNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               28.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.28" data-doi="10.1101/2021.06.11.448129">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tonner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pressman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ross
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Interpretable modeling of genotype-phenotype landscapes with state-of-the-art predictive power
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/2021.06.11.448129
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2021.06.11.448129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMS4wNi4xMS40NDgxMjl2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAxLzAyLzIwMjAuMDcuMTQuMjAxNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               29.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jones
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Faddy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  A skew extension of the t-distribution, with applications
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Roy Stat Soc B
                                 </abbr>
                                 <span class="cit-vol">
                                  65
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  159
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  174
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BRoy%2BStat%2BSoc%2BB%26rft.volume%253D65%26rft.spage%253D159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               30.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.30" data-doi="10.1073/pnas.1309933111">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kinney
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. B.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Atwal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Equitability, mutual information, and the maximal information coefficient
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc Natl Acad Sci USA
                                 </abbr>
                                 <span class="cit-vol">
                                  111
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3354
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3359
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1309933111%26rft_id%253Dinfo%253Apmid%252F24550517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTExLzkvMzM1NCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAxLzAyLzIwMjAuMDcuMTQuMjAxNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               31.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.07.14.201475v4.31">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cover
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thomas
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. A.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  Elements of information theory
                                 </span>
                                 . (
                                 <span class="cit-publ-name">
                                  Wiley
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               32.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.32">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barber
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Agakov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The IM algorithm: a variational approach to information maximization
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Advances in neural information processing systems
                                 </abbr>
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               33.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.14.201475v4.33">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alemi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fischer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dillon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. V.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Murphy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Deep variational information bottleneck
                                 </abbr>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  <span class="cit-pub-id-scheme-arxiv">
                                   arxiv:
                                  </span>
                                  1612.00410
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               34.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.34">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chalk
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marre
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tkačik
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Relevant sparse codes with variational information bottleneck
                                 </abbr>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  arXiv:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  1605.07332
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               35.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.35" data-doi="10.1016/j.cub.2014.09.072">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Olson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N. C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sun
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  A comprehensive biophysical description of pairwise epistasis throughout an entire protein domain
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Curr Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  24
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2643
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2651
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurr%2BBiol%26rft.volume%253D24%26rft.spage%253D2643%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2014.09.072%26rft_id%253Dinfo%253Apmid%252F25455030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cub.2014.09.072&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25455030&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F02%2F2020.07.14.201475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               36.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.36">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Seuma
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Faure
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Badia
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lehner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bolognesi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The genetic landscape for amyloid beta fibril nucleation accurately discriminates familial Alzheimer’s disease mutations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  eLife
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e63364
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D10%26rft.spage%253De63364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               37.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.37" data-doi="10.1038/s41467-019-12101-z">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bolognesi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The mutational landscape of a prion-like domain
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Commun
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  4162
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D10%26rft.spage%253D4162%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-019-12101-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41467-019-12101-z&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               38.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.38" data-doi="10.1016/j.molcel.2018.07.033">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kinney
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. B.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Krainer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Quantitative activity profile and context dependence of all human 5’ splice sites
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  71
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1012
                                 </span>
                                 -
                                 <span class="cit-lpage">
                                  1026
                                 </span>
                                 .e3 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol%2BCell%26rft.volume%253D71%26rft.spage%253D1012%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2018.07.033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.molcel.2018.07.033&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               39.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.39" data-doi="10.1016/j.gde.2005.02.007">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bintu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Transcriptional regulation by the numbers: models
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Curr Opin Genet Dev
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  116
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  124
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DBintu%26rft.auinit1%253DL.%26rft.volume%253D15%26rft.issue%253D2%26rft.spage%253D116%26rft.epage%253D124%26rft.atitle%253DTranscriptional%2Bregulation%2Bby%2Bthe%2Bnumbers%253A%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2005.02.007%26rft_id%253Dinfo%253Apmid%252F15797194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.gde.2005.02.007&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15797194&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F02%2F2020.07.14.201475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000228496100002&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               40.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.14.201475v4.40" data-doi="10.1371/journal.pcbi.1002407">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sherman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cohen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Thermodynamic state ensemble models of cis-regulation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Comput Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e1002407
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DSherman%26rft.auinit1%253DM.%2BS.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253De1002407%26rft.epage%253De1002407%26rft.atitle%253DThermodynamic%2Bstate%2Bensemble%2Bmodels%2Bof%2Bcis-regulation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002407%26rft_id%253Dinfo%253Apmid%252F22479169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002407&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22479169&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F02%2F2020.07.14.201475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               41.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.41">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gunawardena
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Gene Regulation in and out of equilibrium
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Annu Rev Biophys
                                 </abbr>
                                 <span class="cit-vol">
                                  49
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  199
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  226
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BBiophys%26rft.volume%253D49%26rft.spage%253D199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               42.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.42">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Otwinowski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Biophysical inference of epistasis and the effects of mutations on protein stability and function
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol Biol Evol
                                 </abbr>
                                 <span class="cit-vol">
                                  35
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2345
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2354
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft.volume%253D35%26rft.spage%253D2345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               43.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.43" data-doi="10.1073/pnas.1415895112">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Manhart
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Morozov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. V.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Protein folding and binding can emerge as evolutionary spandrels through structural coupling
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc Natl Acad Sci USA
                                 </abbr>
                                 <span class="cit-vol">
                                  112
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1797
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1802
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1415895112%26rft_id%253Dinfo%253Apmid%252F25624494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEyLzYvMTc5NyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAxLzAyLzIwMjAuMDcuMTQuMjAxNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               44.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.44" data-doi="10.1073/pnas.1903888116">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nisthal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ary
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. L.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mayo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. L.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Protein stability engineering insights revealed by domain-wide comprehensive mutagenesis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc Natl Acad Sci USA
                                 </abbr>
                                 <span class="cit-vol">
                                  116
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  16367
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  16377
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1903888116%26rft_id%253Dinfo%253Apmid%252F31371509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE2LzMzLzE2MzY3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMDIvMjAyMC4wNy4xNC4yMDE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               45.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.45">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pokusaeva
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V. O.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  An experimental assay of the interactions of amino acids from orthologous sequences shaping a complex fitness landscape
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Genet
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e1008079
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D15%26rft.spage%253De1008079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               46.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.46">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tareen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kinney
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. B.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Logomaker: beautiful sequence logos in Python
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  36
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2272
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2274
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D36%26rft.spage%253D2272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               47.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.07.14.201475v4.47">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Steeg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. V.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  Non-Parametric Entropy Estimation Toolbox (NPEET)
                                 </span>
                                 .
                                 <a href="https://www.isi.edu/~gregv/npeet.html">
                                  https://www.isi.edu/~gregv/npeet.html
                                 </a>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               48.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.48">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vasicek
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  A test for normality based on sample entropy
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Roy Stat Soc B
                                 </abbr>
                                 <span class="cit-vol">
                                  38
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  54
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  59
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1976
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BRoy%2BStat%2BSoc%2BB%26rft.volume%253D38%26rft.spage%253D54%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               49.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.49">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kraskov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stögbauer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grassberger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Estimating mutual information
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Phys Rev E
                                 </abbr>
                                 <span class="cit-vol">
                                  69
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  066138
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhys%2BRev%2BE%26rft.volume%253D69%26rft.spage%253D066138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               50.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.50" data-doi="10.1162/089976604322742010">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sharpee
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rust
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N. C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bialek
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Analyzing neural responses to natural signals: maximally informative dimensions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neural Comput
                                 </abbr>
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  223
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  250
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeural%2BComputation%26rft.stitle%253DNeural%2BComput.%26rft.aulast%253DSharpee%26rft.auinit1%253DT.%26rft.volume%253D16%26rft.issue%253D2%26rft.spage%253D223%26rft.epage%253D250%26rft.atitle%253DAnalyzing%2BNeural%2BResponses%2Bto%2BNatural%2BSignals%253A%2BMaximally%2BInformative%2BDimensions%26rft_id%253Dinfo%253Adoi%252F10.1162%252F089976604322742010%26rft_id%253Dinfo%253Apmid%252F15006095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1162/089976604322742010&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15006095&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F02%2F2020.07.14.201475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000187907400001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               51.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.51" data-doi="10.1214/aos/1176344552">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Efron
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Bootstrap methods: another look at the jackknife
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Ann Stat
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  26
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1979
                                 </span>
                                 )..
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnn%2BStat%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1176344552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1214/aos/1176344552&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1979GL00300001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               52.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.14.201475v4.52" data-doi="10.1214/ss/1177013815">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Efron
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tibshirani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Bootstrap methods for standard errors, confidence intervals, and other measures of statistical accuracy
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Stat Sci
                                 </abbr>
                                 <span class="cit-vol">
                                  1
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  54
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  75
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1986
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DStat%2BSci%26rft.volume%253D1%26rft.spage%253D54%26rft_id%253Dinfo%253Adoi%252F10.1214%252Fss%252F1177013815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1214/ss/1177013815&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.07.14.201475v4" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.12.30.474574v1" title="THE RHIZOSPHERE BACTERIAL COMMUNITIES DIFFER AMONG DOMESTICATED MAIZE LANDRACES – AN EXPERIMENTAL CONFIRMATION" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2021.06.29.449134v4" title="PCaDB - a comprehensive and interactive database for transcriptomes from prostate cancer population cohorts" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted January 02, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.07.14.201475v4.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.07.14.201475v4.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/01/02/2020.07.14.201475.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.07.14.201475v4.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.07.14.201475v4" class="active" data-icon-position="" data-hide-link-title="0">
                                   MAVE-NN: learning genotype-phenotype maps from multiplex assays of variant effect
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2330666"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-M1CJm636hwRkNWxRBfp-RW3rIFuwAo8cgJNqyNVZ47s"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484473856"/>
                                   <input type="hidden" name="captcha_token" value="7a6aeb55d70682bb2ecb8a21a03fda11"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2330666--3" data-node-nid="2330666" id="node-2330666--41037561867" data-pisa="biorxiv;2020.07.14.201475v4" data-pisa-master="biorxiv;2020.07.14.201475" data-seqnum="2330666" data-apath="/biorxiv/early/2022/01/02/2020.07.14.201475.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 MAVE-NN: learning genotype-phenotype maps from multiplex assays of variant effect
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Ammar
                                  </span>
                                  <span class="nlm-surname">
                                   Tareen
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Mahdi
                                  </span>
                                  <span class="nlm-surname">
                                   Kooshkbaghi
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Anna
                                  </span>
                                  <span class="nlm-surname">
                                   Posfai
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   William T.
                                  </span>
                                  <span class="nlm-surname">
                                   Ireland
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   David M.
                                  </span>
                                  <span class="nlm-surname">
                                   McCandlish
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Justin B.
                                  </span>
                                  <span class="nlm-surname">
                                   Kinney
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.07.14.201475;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.07.14.201475
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.07.14.201475v4" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.07.14.201475v4&title=MAVE-NN%3A%20learning%20genotype-phenotype%20maps%20from%20multiplex%20assays%20of%20variant%20effect" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.14.201475v4&title=MAVE-NN%3A%20learning%20genotype-phenotype%20maps%20from%20multiplex%20assays%20of%20variant%20effect" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.14.201475v4&text=MAVE-NN%3A%20learning%20genotype-phenotype%20maps%20from%20multiplex%20assays%20of%20variant%20effect" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.07.14.201475v4&t=MAVE-NN%3A%20learning%20genotype-phenotype%20maps%20from%20multiplex%20assays%20of%20variant%20effect" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.07.14.201475v4&title=MAVE-NN%3A%20learning%20genotype-phenotype%20maps%20from%20multiplex%20assays%20of%20variant%20effect" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.07.14.201475v4&title=MAVE-NN%3A%20learning%20genotype-phenotype%20maps%20from%20multiplex%20assays%20of%20variant%20effect&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.14.201475v4&title=MAVE-NN%3A%20learning%20genotype-phenotype%20maps%20from%20multiplex%20assays%20of%20variant%20effect" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2330666--3" data-node-nid="2330666" id="citation-node-2330666--4691448935" data-pisa="biorxiv;2020.07.14.201475v4" data-pisa-master="biorxiv;2020.07.14.201475" data-seqnum="2330666" data-apath="/biorxiv/early/2022/01/02/2020.07.14.201475.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     MAVE-NN: learning genotype-phenotype maps from multiplex assays of variant effect
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Ammar
                                      </span>
                                      <span class="nlm-surname">
                                       Tareen
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Mahdi
                                      </span>
                                      <span class="nlm-surname">
                                       Kooshkbaghi
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Anna
                                      </span>
                                      <span class="nlm-surname">
                                       Posfai
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       William T.
                                      </span>
                                      <span class="nlm-surname">
                                       Ireland
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       David M.
                                      </span>
                                      <span class="nlm-surname">
                                       McCandlish
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Justin B.
                                      </span>
                                      <span class="nlm-surname">
                                       Kinney
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.07.14.201475;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.07.14.201475
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=MAVE-NN%3A%20learning%20genotype-phenotype%20maps%20from%20multiplex%20assays%20of%20variant%20effect&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Tareen%2C+Ammar&rft.au=Kooshkbaghi%2C+Mahdi&rft.au=Posfai%2C+Anna&rft.au=Ireland%2C+William+T.&rft.au=McCandlish%2C+David+M.&rft.au=Kinney%2C+Justin+B.">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2330666/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2330666/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2330666/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2330666/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2330666/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2330666/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2330666/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2330666/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2330666/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2330666/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2330666/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2330666/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.14.201475v4&count=horizontal&via=&text=MAVE-NN%3A%20learning%20genotype-phenotype%20maps%20from%20multiplex%20assays%20of%20variant%20effect&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.07.14.201475v4" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.07.14.201475v4&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.07.14.201475v4" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Bioinformatics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15187)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
