<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.09.11.292896v3.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.09.11.292896v3.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.09.11.292896v3.ppt"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/05/19/2020.09.11.292896.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.09.11.292896v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Species- and strain-level assessment using rrn long-amplicons suggests donor’s influence on gut microbial transference via fecal transplants in metabolic syndrome subjects"/>
  <meta name="DC.Identifier" content="10.1101/2020.09.11.292896"/>
  <meta name="DC.Date" content="2022-05-19"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Fecal microbiota transplantation (FMT) is currently used for treating Clostridium difficile infection and explored for other clinical applications in experimental trials. However, the effectiveness of this therapy could vary, and partly depend on the donor’s bacterial species engraftment, whose evaluation is challenging because there are no cost-effective strategies for accurately tracking the microbe transference. In this regard, the precise identification of bacterial species inhabiting the human gut is essential to define their role in human health unambiguously. We used Nanopore-based device to sequence bacterial rrn operons (16S-ITS-23S) and to reveal species-level abundance changes in the human gut microbiota of a FMT trial. By assessing the donor and recipient microbiota before and after FMT, we further evaluated whether this molecular approach reveals strain-level genetic variation to demonstrate microbe transfer and engraftment. Strict control over sequencing data quality and major microbiota covariates was critical for accurately estimating the changes in gut microbial species abundance in the recipients after FMT. We detected strain-level variation via single-nucleotide variants (SNVs) at rrn regions in a species-specific manner. We showed that it was possible to explore successfully the donor-bacterial strain (e.g., Parabacteroides merdae) engraftment in recipients of the FMT by assessing the nucleotide frequencies at rrn-associated SNVs. Our findings indicate that the engraftment of donors’ microbiota is to some extent correlated with the improvement of metabolic health in recipients and that parameters such as the baseline gut microbiota configuration, sex, and age of donors should be considered to ensure the success of FMT in humans.

Trial registration The study was prospectively registered at the Dutch Trial registry - NTR4488 (<https://www.trialregister.nl/trial/4488>).

### Competing Interest Statement

The authors have declared no competing interest.

*   BMI
    :   body mass index
    FDR
    :   false discovery rate
    FMT
    :   fecal microbiota transplantation
    LMM
    :   linear mixed model
    NCBI
    :   National Center for Biotechnology Information
    MAG
    :   metagenome-assembled genome
    PCoA
    :   principal coordinate analysis
    PCR
    :   polymerase chain reaction
    RDP
    :   Ribosomal Database Project
    rrn
    :   bacterial ribosome RNA operon (16S-ITS-23S)
    SNP
    :   single-nucleotide polymorphism
    SNV
    :   single-nucleotide variation
    UC
    :   ulcerative colitis"/>
  <meta name="DC.Contributor" content="Alfonso Benítez-Páez"/>
  <meta name="DC.Contributor" content="Annick V. Hartstra"/>
  <meta name="DC.Contributor" content="Max Nieuwdorp"/>
  <meta name="DC.Contributor" content="Yolanda Sanz"/>
  <meta name="article:published_time" content="2022-05-19"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Species- and strain-level assessment using rrn long-amplicons suggests donor’s influence on gut microbial transference via fecal transplants in metabolic syndrome subjects"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Fecal microbiota transplantation (FMT) is currently used for treating <i>Clostridium difficile</i> infection and explored for other clinical applications in experimental trials. However, the effectiveness of this therapy could vary, and partly depend on the donor’s bacterial species engraftment, whose evaluation is challenging because there are no cost-effective strategies for accurately tracking the microbe transference. In this regard, the precise identification of bacterial species inhabiting the human gut is essential to define their role in human health unambiguously. We used Nanopore-based device to sequence bacterial <i>rrn</i> operons (16S-ITS-23S) and to reveal species-level abundance changes in the human gut microbiota of a FMT trial. By assessing the donor and recipient microbiota before and after FMT, we further evaluated whether this molecular approach reveals strain-level genetic variation to demonstrate microbe transfer and engraftment. Strict control over sequencing data quality and major microbiota covariates was critical for accurately estimating the changes in gut microbial species abundance in the recipients after FMT. We detected strain-level variation via single-nucleotide variants (SNVs) at <i>rrn</i> regions in a species-specific manner. We showed that it was possible to explore successfully the donor-bacterial strain (e.g., <i>Parabacteroides merdae)</i> engraftment in recipients of the FMT by assessing the nucleotide frequencies at rrn-associated SNVs. Our findings indicate that the engraftment of donors’ microbiota is to some extent correlated with the improvement of metabolic health in recipients and that parameters such as the baseline gut microbiota configuration, sex, and age of donors should be considered to ensure the success of FMT in humans.</p><h3>Trial registration</h3>
<p>The study was prospectively registered at the Dutch Trial registry - NTR4488 (https://www.trialregister.nl/trial/4488).</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.09.11.292896v3"/>
  <meta name="citation_id" content="2020.09.11.292896v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.09.11.292896v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.09.11.292896v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.09.11.292896v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.09.11.292896"/>
  <meta name="citation_num_pages" content="33"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.09.11.292896"/>
  <meta name="citation_author" content="Alfonso Benítez-Páez"/>
  <meta name="citation_author_institution" content="Microbial Ecology, Nutrition & Health Research Unit. Institute of Agrochemistry and Food Technology, Spanish National Research Council (IATA-CSIC)"/>
  <meta name="citation_author_institution" content="Host-Microbe Interactions in Metabolic Health Laboratory. Principe Felipe Research Center (CIPF)"/>
  <meta name="citation_author_email" content="abenitez@cipf.es"/>
  <meta name="citation_author_email" content="yolsanz@iata.csic.es"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5707-4340"/>
  <meta name="citation_author" content="Annick V. Hartstra"/>
  <meta name="citation_author_institution" content="Department of Internal and Vascular Medicine, Amsterdam University Medical Centres"/>
  <meta name="citation_author" content="Max Nieuwdorp"/>
  <meta name="citation_author_institution" content="Department of Internal and Vascular Medicine, Amsterdam University Medical Centres"/>
  <meta name="citation_author" content="Yolanda Sanz"/>
  <meta name="citation_author_institution" content="Microbial Ecology, Nutrition & Health Research Unit. Institute of Agrochemistry and Food Technology, Spanish National Research Council (IATA-CSIC)"/>
  <meta name="citation_author_email" content="abenitez@cipf.es"/>
  <meta name="citation_author_email" content="yolsanz@iata.csic.es"/>
  <meta name="citation_reference" content="Cammarota G, Ianiro G, Gasbarrini A. Fecal microbiota transplantation for the treatment of Clostridium difficile infection: a systematic review. J Clin Gastroenterol 2014; 48:693–702."/>
  <meta name="citation_reference" content="Chu ND, Crothers JW, Nguyen LTT, Kearney SM, Smith MB, Kassam Z, et al. Dynamic Colonization of Microbes and Their Functions after Fecal Microbiota Transplantation for Inflammatory Bowel Disease. mBio 2021; 12:e0097521."/>
  <meta name="citation_reference" content="Hartstra AV, Schuppel V, Imangaliyev S, Schrantee A, Prodan A, Collard D, et al. Infusion of donor feces affects the gut-brain axis in humans with metabolic syndrome. Mol Metab 2020; 42:101076."/>
  <meta name="citation_reference" content="Sokol H, Landman C, Seksik P, Berard L, Montil M, Nion-Larmurier I, et al. Fecal microbiota transplantation to maintain remission in Crohn’s disease: a pilot randomized controlled study. Microbiome 2020; 8:12."/>
  <meta name="citation_reference" content="Ott SJ, Waetzig GH, Rehman A, Moltzau-Anderson J, Bharti R, Grasis JA, et al. Efficacy of Sterile Fecal Filtrate Transfer for Treating Patients With Clostridium difficile Infection. Gastroenterology 2017; 152:799–811 e7."/>
  <meta name="citation_reference" content="Aggarwala V, Mogno I, Li Z, Yang C, Britton GJ, Chen-Liaw A, et al. Precise quantification of bacterial strains after fecal microbiota transplantation delineates long-term engraftment and explains outcomes. Nat Microbiol 2021; 6:1309–18."/>
  <meta name="citation_reference" content="Verma S, Dutta SK, Firnberg E, Phillips L, Vinayek R, Nair PP. Identification and engraftment of new bacterial strains by shotgun metagenomic sequence analysis in patients with recurrent Clostridioides difficile infection before and after fecal microbiota transplantation and in healthy human subjects. PLoS One 2021; 16:e0251590."/>
  <meta name="citation_reference" content="Wilson BC, Vatanen T, Jayasinghe TN, Leong KSW, Derraik JGB, Albert BB, et al. Strain engraftment competition and functional augmentation in a multi-donor fecal microbiota transplantation trial for obesity. Microbiome 2021; 9:107."/>
  <meta name="citation_reference" content="Rinott E, Youngster I, Yaskolka Meir A, Tsaban G, Zelicha H, Kaplan A, et al. Effects of Diet-Modulated Autologous Fecal Microbiota Transplantation on Weight Regain. Gastroenterology 2021; 160:158–73 e10."/>
  <meta name="citation_reference" content="Yu EW, Gao L, Stastka P, Cheney MC, Mahabamunuge J, Torres Soto M, et al. Fecal microbiota transplantation for the improvement of metabolism in obesity: The FMT-TRIM double-blind placebo-controlled pilot trial. PLoS Med 2020; 17:e1003051."/>
  <meta name="citation_reference" content="Benitez-Paez A, Portune KJ, Sanz Y. Species-level resolution of 16S rRNA gene amplicons sequenced through the MinION portable nanopore sequencer. Gigascience 2016; 5:4."/>
  <meta name="citation_reference" content="Cusco A, Catozzi C, Vines J, Sanchez A, Francino O. Microbiota profiling with long amplicons using Nanopore sequencing: full-length 16S rRNA gene and whole rrn operon. F1000Res 2018; 7:1755."/>
  <meta name="citation_reference" content="Sakai J, Tarumoto N, Kodana M, Ashikawa S, Imai K, Kawamura T, et al. An identification protocol for ESBL-producing Gram-negative bacteria bloodstream infections using a MinION nanopore sequencer. J Med Microbiol 2019; 68:1219–26."/>
  <meta name="citation_reference" content="Shin H, Lee E, Shin J, Ko SR, Oh HS, Ahn CY, et al. Elucidation of the bacterial communities associated with the harmful microalgae Alexandrium tamarense and Cochlodinium polykrikoides using nanopore sequencing. Sci Rep 2018; 8:5323."/>
  <meta name="citation_reference" content="Shin J, Lee S, Go MJ, Lee SY, Kim SC, Lee CH, et al. Analysis of the mouse gut microbiome using full-length 16S rRNA amplicon sequencing. Sci Rep 2016; 6:29681."/>
  <meta name="citation_reference" content="Benitez-Paez A, Sanz Y. Multi-locus and long amplicon sequencing approach to study microbial diversity at species level using the MinION portable nanopore sequencer. Gigascience 2017; 6:1–12."/>
  <meta name="citation_reference" content="Kerkhof LJ, Dillon KP, Haggblom MM, McGuinness LR. Profiling bacterial communities by MinION sequencing of ribosomal operons. Microbiome 2017; 5:116."/>
  <meta name="citation_reference" content="Ciuffreda L, Rodriguez-Perez H, Flores C. Nanopore sequencing and its application to the study of microbial communities. Comput Struct Biotechnol J 2021; 19:1497–511."/>
  <meta name="citation_reference" content="Kerkhof LJ. Is Oxford Nanopore sequencing ready for analyzing complex microbiomes? FEMS Microbiol Ecol 2021; 97."/>
  <meta name="citation_reference" content="Pavlovic J, Cavalieri D, Mastromei G, Pangallo D, Perito B, Marvasi M. MinION technology for microbiome sequencing applications for the conservation of cultural heritage. Microbiol Res 2021; 247:126727."/>
  <meta name="citation_reference" content="Peker N, Garcia-Croes S, Dijkhuizen B, Wiersma HH, van Zanten E, Wisselink G, et al. A Comparison of Three Different Bioinformatics Analyses of the 16S-23S rRNA Encoding Region for Bacterial Identification. Front Microbiol 2019; 10:620."/>
  <meta name="citation_reference" content="Krehenwinkel H, Pomerantz A, Henderson JB, Kennedy SR, Lim JY, Swamy V, et al. Nanopore sequencing of long ribosomal DNA amplicons enables portable and simple biodiversity assessments with high phylogenetic resolution across broad taxonomic scale. Gigascience 2019; 8."/>
  <meta name="citation_reference" content="Brewer TE, Albertsen M, Edwards A, Kirkegaard RH, Rocha EPC, Fierer N. Unlinked rRNA genes are widespread among bacteria and archaea. ISME J 2020; 14:597–608."/>
  <meta name="citation_reference" content="Dowden RA, McGuinness LR, Wisniewski PJ, Campbell SC, Guers JJ, Oydanich M, et al. Host genotype and exercise exhibit species-level selection for members of the gut bacterial communities in the mouse digestive system. Sci Rep 2020; 10:8984."/>
  <meta name="citation_reference" content="Ibironke O, McGuinness LR, Lu SE, Wang Y, Hussain S, Weisel CP, et al. Species-level evaluation of the human respiratory microbiome. Gigascience 2020; 9."/>
  <meta name="citation_reference" content="Gibbons SM, Duvallet C, Alm EJ. Correcting for batch effects in case-control microbiome studies. PLoS Comput Biol 2018; 14:e1006102."/>
  <meta name="citation_reference" content="Schloss PD, Gevers D, Westcott SL. Reducing the effects of PCR amplification and sequencing artifacts on 16S rRNA-based studies. PLoS One 2011; 6:e27310."/>
  <meta name="citation_reference" content="Paramsothy S, Nielsen S, Kamm MA, Deshpande NP, Faith JJ, Clemente JC, et al. Specific Bacteria and Metabolites Associated With Response to Fecal Microbiota Transplantation in Patients With Ulcerative Colitis. Gastroenterology 2019; 156:1440–54 e2."/>
  <meta name="citation_reference" content="Bilinski J, Grzesiowski P, Sorensen N, Madry K, Muszynski J, Robak K, et al. Fecal Microbiota Transplantation in Patients With Blood Disorders Inhibits Gut Colonization With Antibiotic-Resistant Bacteria: Results of a Prospective, Single-Center Study. Clin Infect Dis 2017; 65:364–70."/>
  <meta name="citation_reference" content="Benitez-Paez A, Hess AL, Krautbauer S, Liebisch G, Christensen L, Hjorth MF, et al. Sex, Food, and the Gut Microbiota: Disparate Response to Caloric Restriction Diet with Fiber Supplementation in Women and Men. Mol Nutr Food Res 2021; 65:e2000996."/>
  <meta name="citation_reference" content="El-Salhy M, Casen C, Valeur J, Hausken T, Hatlebakk JG. Responses to faecal microbiota transplantation in female and male patients with irritable bowel syndrome. World J Gastroenterol 2021; 27:2219–37."/>
  <meta name="citation_reference" content="Kielbasa SM, Wan R, Sato K, Horton P, Frith MC. Adaptive seeds tame genomic sequence comparison. Genome Res 2011; 21:487–93."/>
  <meta name="citation_reference" content="Katoh K, Standley DM. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol Biol Evol 2013; 30:772–80."/>
  <meta name="citation_reference" content="Eddy SR. Accelerated Profile HMM Searches. PLoS Comput Biol 2011; 7:e1002195."/>
  <meta name="citation_reference" content="Quast C, Pruesse E, Yilmaz P, Gerken J, Schweer T, Yarza P, et al. The SILVA ribosomal RNA gene database project: improved data processing and web-based tools. Nucleic Acids Res 2013; 41:D590–6."/>
  <meta name="citation_reference" content="Pruesse E, Peplies J, Glockner FO. SINA: accurate high-throughput multiple sequence alignment of ribosomal RNA genes. Bioinformatics 2012; 28:1823–9."/>
  <meta name="citation_reference" content="Caporaso JG, Kuczynski J, Stombaugh J, Bittinger K, Bushman FD, Costello EK, et al. QIIME allows analysis of high-throughput community sequencing data. Nat Methods 2010; 7:335–6."/>
  <meta name="twitter:title" content="Species- and strain-level assessment using rrn long-amplicons suggests donor’s influence on gut microbial transference via fecal transplants in metabolic syndrome subjects"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Fecal microbiota transplantation (FMT) is currently used for treating Clostridium difficile infection and explored for other clinical applications in experimental trials. However, the effectiveness of this therapy could vary, and partly depend on the donor’s bacterial species engraftment, whose evaluation is challenging because there are no cost-effective strategies for accurately tracking the microbe transference. In this regard, the precise identification of bacterial species inhabiting the human gut is essential to define their role in human health unambiguously. We used Nanopore-based device to sequence bacterial rrn operons (16S-ITS-23S) and to reveal species-level abundance changes in the human gut microbiota of a FMT trial. By assessing the donor and recipient microbiota before and after FMT, we further evaluated whether this molecular approach reveals strain-level genetic variation to demonstrate microbe transfer and engraftment. Strict control over sequencing data quality and major microbiota covariates was critical for accurately estimating the changes in gut microbial species abundance in the recipients after FMT. We detected strain-level variation via single-nucleotide variants (SNVs) at rrn regions in a species-specific manner. We showed that it was possible to explore successfully the donor-bacterial strain (e.g., Parabacteroides merdae) engraftment in recipients of the FMT by assessing the nucleotide frequencies at rrn-associated SNVs. Our findings indicate that the engraftment of donors’ microbiota is to some extent correlated with the improvement of metabolic health in recipients and that parameters such as the baseline gut microbiota configuration, sex, and age of donors should be considered to ensure the success of FMT in humans.

Trial registration The study was prospectively registered at the Dutch Trial registry - NTR4488 (<https://www.trialregister.nl/trial/4488>).

### Competing Interest Statement

The authors have declared no competing interest.

*   BMI
    :   body mass index
    FDR
    :   false discovery rate
    FMT
    :   fecal microbiota transplantation
    LMM
    :   linear mixed model
    NCBI
    :   National Center for Biotechnology Information
    MAG
    :   metagenome-assembled genome
    PCoA
    :   principal coordinate analysis
    PCR
    :   polymerase chain reaction
    RDP
    :   Ribosomal Database Project
    rrn
    :   bacterial ribosome RNA operon (16S-ITS-23S)
    SNP
    :   single-nucleotide polymorphism
    SNV
    :   single-nucleotide variation
    UC
    :   ulcerative colitis"/>
  <meta name="og-title" property="og:title" content="Species- and strain-level assessment using rrn long-amplicons suggests donor’s influence on gut microbial transference via fecal transplants in metabolic syndrome subjects"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.09.11.292896v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Fecal microbiota transplantation (FMT) is currently used for treating Clostridium difficile infection and explored for other clinical applications in experimental trials. However, the effectiveness of this therapy could vary, and partly depend on the donor’s bacterial species engraftment, whose evaluation is challenging because there are no cost-effective strategies for accurately tracking the microbe transference. In this regard, the precise identification of bacterial species inhabiting the human gut is essential to define their role in human health unambiguously. We used Nanopore-based device to sequence bacterial rrn operons (16S-ITS-23S) and to reveal species-level abundance changes in the human gut microbiota of a FMT trial. By assessing the donor and recipient microbiota before and after FMT, we further evaluated whether this molecular approach reveals strain-level genetic variation to demonstrate microbe transfer and engraftment. Strict control over sequencing data quality and major microbiota covariates was critical for accurately estimating the changes in gut microbial species abundance in the recipients after FMT. We detected strain-level variation via single-nucleotide variants (SNVs) at rrn regions in a species-specific manner. We showed that it was possible to explore successfully the donor-bacterial strain (e.g., Parabacteroides merdae) engraftment in recipients of the FMT by assessing the nucleotide frequencies at rrn-associated SNVs. Our findings indicate that the engraftment of donors’ microbiota is to some extent correlated with the improvement of metabolic health in recipients and that parameters such as the baseline gut microbiota configuration, sex, and age of donors should be considered to ensure the success of FMT in humans.

Trial registration The study was prospectively registered at the Dutch Trial registry - NTR4488 (<https://www.trialregister.nl/trial/4488>).

### Competing Interest Statement

The authors have declared no competing interest.

*   BMI
    :   body mass index
    FDR
    :   false discovery rate
    FMT
    :   fecal microbiota transplantation
    LMM
    :   linear mixed model
    NCBI
    :   National Center for Biotechnology Information
    MAG
    :   metagenome-assembled genome
    PCoA
    :   principal coordinate analysis
    PCR
    :   polymerase chain reaction
    RDP
    :   Ribosomal Database Project
    rrn
    :   bacterial ribosome RNA operon (16S-ITS-23S)
    SNP
    :   single-nucleotide polymorphism
    SNV
    :   single-nucleotide variation
    UC
    :   ulcerative colitis"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-05-19"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.09.11.292896v3"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2552735"/>
  <title>
   Species- and strain-level assessment using rrn long-amplicons suggests donor’s influence on gut microbial transference via fecal transplants in metabolic syndrome subjects | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"b3XaP7KMaGITzjeF3fcKpoqAVm7H2JDwDtVhuE4_GSY"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2552735","apath":"\/biorxiv\/early\/2022\/05\/19\/2020.09.11.292896.atom","pisa":"biorxiv;2020.09.11.292896v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.09.11.292896v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.09.11.292896v3","title":"Species- and strain-level assessment using rrn long-amplicons suggests donor\u2019s influence on gut microbial transference via fecal transplants in metabolic syndrome subjects","identifier":"node\/2552735"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lHZUl0eN7nBSM7eh7NlCmWMhQwudq1ZarkNA4MEhS5\\\\\\\/ie6\\\\\\\/WxhXKn+tjfjxSX6RSF77m6WGVpZgGqJEJCsL5Or0YtcCvO+JtJ0mbIpJtSGYLmNECopxr+ptTLoRx564vu8xuRO1j32mnuA9FbQAWhh3ENJQ4A41ex2cbNrAJuOGp96+EkSrNqbU98q9UUM5sRl58CGl1k+mepuW9ZBn9CHEfnTrg797vSQW3L4ogDfedbqWSpMvFGBnVDO2C8iuwRHsSW6BukePhDv9ZDyIteCjCnG7RQZo+oKZvJzqK7JvGr2C1zVNCSCff527\\\\\\\/vJTx2EklIjVeq2tdl4wznI6B+DcprSX7p7Y7eiSsdolWH+\\\\\\\/nvvuFk09E2zS9hddLNOa775dSeOXRs3LMrL\\\\\\\/aH1xqrsANHgEEKr9Jr6zXRDgEuBp3QmC9Tc0K4KzITicyqmo9P2GJch4QhJ88e\\\\\\\/+d7tIGxGjvXfqNw01DqmEeiK\\\\\\\/1Uk1FVtRiuenzz5q1pNSeTJNdfLqIzgeuCkKaF3ZqWGfmEjg+GX+0v7vorNVtZg1w1gPwCFbStpxLe3c1ddHv+iN8HvPhVqz4JKQHbghUWCKxU6jdyPmqGqhbddwj8WB4WrEnyV7sLnj5dFu4+JQzMrprWw\\\\\\\/XjMfAGn6GycqormT8PUzM5IgGDpLjDtAq6xz8GvQO54fiiaBzlzdm5xTThvRIvg7jMX5e0ZejbZLbiMS1olIx8Wh2rQDw3elM+xKfgnvYJs6Z9B\\\\\\\/Pp6WqdAMHzXHJILpMq9Bg44T1pkPLf1RQhV0td8DWk6opEDzzA8u3O7yqGYqTLNo2H5qvucHMQ5+n2jpFRaJN5dj2LhCsEZVE7Szy45HiRcrUfypTkKu4ABdsy8aMbBD5zvs5oynxgAz7ayI5W89234+Mu7f53Nwko6q4jYG8ILpbFyHcwNBjPWn9Z6ucoyOlLrwNCG+2vH2vP4zSKa\\\\\\\/ONxmgupnsR5X6uiShJjs9xTixLW2+30eVdM2H2N0ZCRWBNEJuwiexwB\\\\\\\/SyZQSzLZQuqx6OnU\\\\\\\/29tx5Vs6ico0jNFTVh4xxPRa1I5YibA0XYayTzYp0jXzgJ8yYmNU\\\\\\\/KKOUOKFh3fUjoNoMdWon7oyfuYr6A18TmUluLFH6iC1n\\\\\\\/bYfRFYtl8Gj+u8OH8x\\\\\\\/aLLsaWGi3KV+qeOId+9L5NMzRNFCpUBU9DEOKDE4j0e3U+wr8lDjXlUQUBdCbOsEOVt4BB5zahtWZflhzIDUeKiB7PoeKUq9ldRdvOuikLOKtOWESa6vhkOqSyTiRfSZwzO2X6A6y6OyuFG43aME+W12ZYVlMS0Wr6onXz2ak0X4rYdNYCD+aSp1Mzz1nDsWCODvkf5MkUSlxDYgLjvyqEeYsEbbFTvM2EuCKs=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Uyj+8a+jM5EBB8w9mjbFOA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022b95f0611b2bb361933c6c2e217e394f540d3ebebd165106dcc4f71368a6f126c\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.09.11.292896v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2552735 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.09.11.292896v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_466686542">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_466686542" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1625738487" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-Evx69GhkmTlsHClxHQ7tcUXkk1PkXZSEXHczbmpiWZ8"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2552735" data-node-nid="2552735" id="node-2552735--2555267458" data-pisa="biorxiv;2020.09.11.292896v3" data-pisa-master="biorxiv;2020.09.11.292896" data-apath="/biorxiv/early/2022/05/19/2020.09.11.292896.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Species- and strain-level assessment using rrn long-amplicons suggests donor’s influence on gut microbial transference via fecal transplants in metabolic syndrome subjects
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0001-5707-4340" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Alfonso
                      </span>
                      <span class="nlm-surname">
                       Benítez-Páez
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Annick V.
                      </span>
                      <span class="nlm-surname">
                       Hartstra
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Max
                      </span>
                      <span class="nlm-surname">
                       Nieuwdorp
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Yolanda
                      </span>
                      <span class="nlm-surname">
                       Sanz
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.09.11.292896
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2552735--2555267458" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Alfonso Benítez-Páez
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Microbial Ecology, Nutrition & Health Research Unit. Institute of Agrochemistry and Food Technology, Spanish National Research Council (IATA-CSIC)
                       </span>
                       . 46980 Paterna-Valencia,
                       <span class="nlm-country">
                        Spain
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Host-Microbe Interactions in Metabolic Health Laboratory. Principe Felipe Research Center (CIPF)
                       </span>
                       . 46012 Valencia,
                       <span class="nlm-country">
                        Spain
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Alfonso%2BBen%C3%ADtez-P%C3%A1ez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Ben%C3%ADtez-P%C3%A1ez%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAlfonso%2BBen%25C3%25ADtez-P%25C3%25A1ez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0001-5707-4340" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Alfonso Benítez-Páez
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:abenitez@cipf.es" class="" data-icon-position="" data-hide-link-title="0">
                        abenitez@cipf.es
                       </a>
                       <a href="mailto:yolsanz@iata.csic.es" class="" data-icon-position="" data-hide-link-title="0">
                        yolsanz@iata.csic.es
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Annick V. Hartstra
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Internal and Vascular Medicine, Amsterdam University Medical Centres
                       </span>
                       . 1105 AZ Amsterdam,
                       <span class="nlm-country">
                        The Netherlands
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Annick%2BV.%2BHartstra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Hartstra%20AV&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAnnick%2BV.%2BHartstra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Max Nieuwdorp
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Internal and Vascular Medicine, Amsterdam University Medical Centres
                       </span>
                       . 1105 AZ Amsterdam,
                       <span class="nlm-country">
                        The Netherlands
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Max%2BNieuwdorp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Nieuwdorp%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AMax%2BNieuwdorp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Yolanda Sanz
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Microbial Ecology, Nutrition & Health Research Unit. Institute of Agrochemistry and Food Technology, Spanish National Research Council (IATA-CSIC)
                       </span>
                       . 46980 Paterna-Valencia,
                       <span class="nlm-country">
                        Spain
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Yolanda%2BSanz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Sanz%20Y&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AYolanda%2BSanz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:abenitez@cipf.es" class="" data-icon-position="" data-hide-link-title="0">
                        abenitez@cipf.es
                       </a>
                       <a href="mailto:yolsanz@iata.csic.es" class="" data-icon-position="" data-hide-link-title="0">
                        yolsanz@iata.csic.es
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.09.11.292896v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2552735" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2552735/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.09.11.292896v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2552735" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2552735/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.09.11.292896v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2552735" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2552735/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.09.11.292896v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2552735" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2552735/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.09.11.292896v3.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2552735" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2552735/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.09.11.292896v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2552735" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2552735/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-2">
                             Fecal microbiota transplantation (FMT) is currently used for treating Clostridium difficile infection and explored for other clinical applications in experimental trials. However, the effectiveness of this therapy could vary, and partly depend on the donor’s bacterial species engraftment, whose evaluation is challenging because there are no cost-effective strategies for accurately tracking the microbe transference. In this regard, the precise identification of bacterial species inhabiting the human gut is essential to define their role in human health unambiguously. We used Nanopore-based device to sequence bacterial rrn operons (16S-ITS-23S) and to reveal species-level abundance changes in the human gut microbiota of a FMT trial. By assessing the donor and recipient microbiota before and after FMT, we further evaluated whether this molecular approach reveals strain-level genetic variation to demonstrate microbe transfer and engraftment. Strict control over sequencing data quality and major microbiota covariates was critical for accurately estimating the changes in gut microbial species abundance in the recipients after FMT. We detected strain-level variation via single-nucleotide variants (SNVs) at rrn regions in a species-specific manner. We showed that it was possible to explore successfully the donor-bacterial strain (e.g., Parabacteroides merdae) engraftment in recipients of the FMT by assessing the nucleotide frequencies at rrn-associated SNVs. Our findings indicate that the engraftment of donors’ microbiota is to some extent correlated with the improvement of metabolic health in recipients and that parameters such as the baseline gut microbiota configuration, sex, and age of donors should be considered to ensure the success of FMT in humans.
                            </p>
                            <div id="sec-1" class="subsection">
                             <p id="p-3">
                              <strong>
                               Trial registration
                              </strong>
                              The study was prospectively registered at the Dutch Trial registry - NTR4488 (
                              <a href="https://www.trialregister.nl/trial/4488">
                               https://www.trialregister.nl/trial/4488
                              </a>
                              ).
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-6">
                             Microbiome research advances rapidly and the study of microbial communities demands the implementation of sequencing strategies permitting the identification of microbial species and strains of complex ecosystems and allowing tracing of microbes engraftment in clinical trials evaluating microbiome replacement strategies. Among them, fecal microbiota transplantation (FMT) is widely recognized as an efficient therapy for Clostridioides difficile infection that outperforms antibiotic treatment
                             <sup>
                              <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                               1
                              </a>
                             </sup>
                             . Despite the potential risk for the transmission of infectious diseases by FMT, this approach has shown efficacy in ameliorating intestinal inflammation and metabolic disorders, such as insulin resistance
                             <sup>
                              <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                               2
                              </a>
                              -
                              <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                             </sup>
                             . The mode of FMT action is thought to depend on the engraftment of live microbes from donors to recipients, although some clinical trials leave the door open for hypotheses about alternative mechanisms
                             <sup>
                              <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                             </sup>
                             . The demonstration that the disease amelioration is linked to the ability of donor’s microbes to thrive in the recipient gut is challenging but necessary to ascertain whether particular microbes play a therapeutic role in FMT trials. Metagenomics based on DNA shotgun sequencing is currently the only strategy that enables us to reach such a resolution level, but it is still very expensive, and few strain profiling results adopting such protocols have been provided in FMT studies
                             <sup>
                              <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                               6
                              </a>
                              -
                              <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                             </sup>
                             . Most of FMT microbiome surveys have been performed through the targeted amplification of universal gene markers, mainly including a few hypervariable regions (e.g., V3 and/or V4) of the bacterial 16S rRNA gene
                             <sup>
                              <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ,
                              <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                               10
                              </a>
                             </sup>
                             . This technique shows limitations since it only allows changes affecting the global microbiome structure to be captured and does not permit precise taxonomic identifications at the species level, mostly because of the limited resolution of this gene marker in discriminating closely related microbes. This issue is of particular relevance to accurately trace species engraftment in FMT or demonstrate microbe colonization in probiotic-based clinical trials. We previously developed a Nanopore-based amplicon sequencing method to improve the resolution of taxonomic identifications by studying the microbial genetic variability of nearly complete 16S rRNA genes
                             <sup>
                              <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                             </sup>
                             . This approach has been demonstrated to perform well in a wide variety of microbiota studies
                             <sup>
                              <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                              -
                              <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                             </sup>
                             . More recently, we have also pioneered a new methodology combining the sequencing of an extremely variable multilocus region with sample multiplexing
                             <sup>
                              <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                             </sup>
                             to improve the species-level characterization of complex microbial communities
                             <sup>
                              <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                              ,
                              <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                               17
                              </a>
                             </sup>
                             . The implementations mentioned above of nanopore-based technology and their multiple applications to assess microbial communities allow us to gain knowledge in the configuration of different analytical pipelines improving the processing of such data as updated protocols are delivered
                             <sup>
                              <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                              -
                              <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                             </sup>
                             .
                            </p>
                            <p id="p-7">
                             We have made a great effort to compile a great deal of genetic information on rrn sequences from thousands of bacterial species (rrn_db, freely available at
                             <a href="https://github.com/alfbenpa/rrn_db">
                              https://github.com/alfbenpa/rrn_db
                             </a>
                             )
                             <sup>
                              <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                             </sup>
                             . Nevertheless, the limited number of properly annotated DNA sequences available in public repositories of bacteria from multiple ecosystems restricts the utilization of this approach in less explored habitats
                             <sup>
                              <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                              ,
                              <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                               21
                              </a>
                             </sup>
                             . Notwithstanding, the utility of rrn-based analysis for surveying biological diversity has been explored across kingdoms, showing promising results in the identification of metazoans by metabarcoding approaches
                             <sup>
                              <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                             </sup>
                             .
                            </p>
                            <p id="p-8">
                             Despite the issues concerning unlinked 16S rRNA and 23S rRNA markers
                             <sup>
                              <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                             </sup>
                             that can be addressed in certain bacterial groups and environmental samples via the study of individual regions, we think the long-read amplicon sequencing of bacterial rrn regions could help fill the existing gap to demonstrate the transfer and stable colonization of microbes under FMT strategies in a cost-effective manner.
                            </p>
                            <p id="p-9">
                             In this study, we aimed to evaluate the donor’s influence on the transference of specific intestinal microbes to FMT recipients using a unique-donor-to-multiple-recipients approach and amplicon sequencing of the nearly complete rrn regions using Nanopore-based sequencing in a trial evaluating the ability of FMT to alleviate metabolic syndrome. We obtained de novo information on the rrn region from predominant bacterial species inhabiting the human gut, enriching the rrn existing database with the annotations of dozens of microbial species and strains. We assessed to what extent the FMT changed the gut microbiota composition of recipients at the species level and, more importantly, tracked the strain-level variation to investigate microbiota engraftment. Overall, our results demonstrated a specific microbial transference between donors and recipients, which likely mediated health effects in metabolic syndrome patients, and that this transference partly depends on the donor-recipient pair, their baseline gut microbiota configuration, and certain donor-associated demographic variables.
                            </p>
                           </div>
                           <div class="section" id="sec-3">
                            <h2 class="">
                             Results
                            </h2>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Assembly of human gut microbiota-derived rrn sequences
                             </h3>
                             <p id="p-10">
                              The evaluation of the gut microbiota in subjects with metabolic syndrome submitted to FMT aimed at revealing microbiota engraftment; it required the comprehensive analysis of Nanopore sequencing data to generate reliable genetic information supporting species- and strain-level variation additionally to that compiled in our previous rrn repository
                              <sup>
                               <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">
                                16
                               </a>
                              </sup>
                              . During the first step of rrn assembly (
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Figure 1
                              </a>
                              ), based on mapping forward reads against the nonredundant NCBI 16S database and the selection of high-quality alignments, we detected the presence of 381 different species. However, preliminary rrn assembly was initiated only for those species with at least 10X coverage (250 in total), supporting high-quality consensus rrn sequences in downstream analyses. The mapping of reverse reads against the preliminary rrn assemblies permitted us to confirm the detection of 229 microbial species. Full rrn sequences assembled with reliable taxonomy annotations were retrieved disregarding their individual coverage level (e.g. low, intermediate, or high- Figure S1).
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F1.large.jpg?width=800&height=600&carousel=1" title="Graphical description of the study. Data acquisition and processing steps, including sample selection, amplicon sequencing, and the general pipeline for the de novo assembly of rrn regions from human gut microbiota, are depicted." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-667907561" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Graphical description of the study. Data acquisition and processing steps, including sample selection, amplicon sequencing, and the general pipeline for the <em>de novo</em> assembly of rrn regions from human gut microbiota, are depicted.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F1.medium.gif" width="440" height="272"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F1.medium.gif" width="440" height="272"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1.
                               </span>
                               <p id="p-11" class="first-child">
                                Graphical description of the study. Data acquisition and processing steps, including sample selection, amplicon sequencing, and the general pipeline for the de novo assembly of rrn regions from human gut microbiota, are depicted.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Species abundance shifts associated with FMT
                             </h3>
                             <p id="p-12">
                              Sample-derived forward and reverse reads were mapped to assess shifts in diversity and taxonomic features. Among all reads, only 43% supported alignments with the highest quality (according to identity and length) and were used in downstream analyses. After taxonomic assignment, we found no large changes in any of the alpha diversity descriptors analyzed. Nevertheless, we found that FMT increased the richness (Chao’s index) of the microbiota in six out of the ten recipients (Figure S3), which showed a gain of 31 species on average. In the remaining four recipients, there was a reduction in richness resulting in an average loss of 16 species as a result of FMT. The beta diversity evaluation based on the Bray-Curtis dissimilarity index indicated no major shifts in the microbial structure of recipients as a consequence of FMT (PERMANOVA = 1.02, p = 0.403). Nevertheless, the gut microbiota composition of the recipients was strongly influenced from greater to lesser extents by the sequencing run (PERMANOVA = 3.69, p = 0.001), sex (PERMANOVA = 1.70, p = 0.032), and donor (PERMANOVA = 1.69, p = 0.004). The results of this multivariate analysis are shown in
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Figure 2A
                              </a>
                              . Globally, we observed that POST-FMT samples tended to be mapped closer to those from donors. To further assess this hypothesis, we compared the distances (Bray-Curtis weighted metrics) between the respective donors and the PRE-FMT microbiota or the POST-FMT microbiota. As a result, we noted that donor microbiota-PRE-FMT pairs had a trend to be more dissimilar than the donor microbiota-POST-FMT pairs, as indicated by a decrease in the Bray-Curtis distance (PRE-FMT vs Donor median = 0.79, POST-FMT vs Donor median = 0.71, average delta = 0.09, p = 0.075) (
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Figure 2B
                              </a>
                              ).
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F2.large.jpg?width=800&height=600&carousel=1" title="Beta diversity of the microbial communities assessed by rrn sequencing. A - Scatter plot compiling data from the multivariate analysis (principal coordinate analysis - PCoA) of the microbiota from recipients and donors participating in FMT. The donor and recipient samples and the sampling time points are defined according to the legend at the top. PCo; principal coordinate (the two most informative coordinates are shown). Color legend corresponds with the distribution of the different PRE-FMT and POST-FMT samples into the groups linked to the four donor samples evaluated (see legend into the plot). B - A Genetic distance-based approach for evaluating microbiota transfer between donor and recipient pairs. The microbial community structures of the PRE-FMT and POST-FMT samples were compared with those of the donors through the calculation of the Bray-Curtis dissimilarity index and are represented as boxplots. The Wilcoxon signed-rank test for paired samples was used to compare differences in genetic distances. C – Microbiota abundance shifts after FMT. Abundance data distribution for three species selected from those described in Table 1 with meaningful increase after FMT is shown in boxplot fashion." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-667907561" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Beta diversity of the microbial communities assessed by <em>rrn</em> sequencing. A - Scatter plot compiling data from the multivariate analysis (principal coordinate analysis - PCoA) of the microbiota from recipients and donors participating in FMT. The donor and recipient samples and the sampling time points are defined according to the legend at the top. PCo; principal coordinate (the two most informative coordinates are shown). Color legend corresponds with the distribution of the different PRE-FMT and POST-FMT samples into the groups linked to the four donor samples evaluated (see legend into the plot). B - A Genetic distance-based approach for evaluating microbiota transfer between donor and recipient pairs. The microbial community structures of the PRE-FMT and POST-FMT samples were compared with those of the donors through the calculation of the Bray-Curtis dissimilarity index and are represented as boxplots. The Wilcoxon signed-rank test for paired samples was used to compare differences in genetic distances. C – Microbiota abundance shifts after FMT. Abundance data distribution for three species selected from those described in Table 1 with meaningful increase after FMT is shown in boxplot fashion.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F2.medium.gif" width="440" height="265"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F2.medium.gif" width="440" height="265"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2.
                               </span>
                               <p id="p-13" class="first-child">
                                Beta diversity of the microbial communities assessed by rrn sequencing. A - Scatter plot compiling data from the multivariate analysis (principal coordinate analysis - PCoA) of the microbiota from recipients and donors participating in FMT. The donor and recipient samples and the sampling time points are defined according to the legend at the top. PCo; principal coordinate (the two most informative coordinates are shown). Color legend corresponds with the distribution of the different PRE-FMT and POST-FMT samples into the groups linked to the four donor samples evaluated (see legend into the plot). B - A Genetic distance-based approach for evaluating microbiota transfer between donor and recipient pairs. The microbial community structures of the PRE-FMT and POST-FMT samples were compared with those of the donors through the calculation of the Bray-Curtis dissimilarity index and are represented as boxplots. The Wilcoxon signed-rank test for paired samples was used to compare differences in genetic distances. C – Microbiota abundance shifts after FMT. Abundance data distribution for three species selected from those described in
                                <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                                 Table 1
                                </a>
                                with meaningful increase after FMT is shown in boxplot fashion.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-14">
                              We performed an LMM analysis, which provided similar results to the beta diversity evaluation. We found that sequencing batch, sex, and donor were the main covariates influencing the microbiota data. Furthermore, we found that age and baseline BMI also explained the gut microbiota variation among the subjects of this study to some extent. After including the variables mentioned above as random effects in the model, a list of microbial species showing alterations as a consequence of FMT was retrieved (
                              <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              ), and some of them exhibiting an increased abundance as a consequence of the FMT are plotted in
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Figure 2C
                              </a>
                              . Ten different microbial species were found to be differentially increased when paired samples obtained before and after FMT were compared, and only three of them (e.g. Bifidobacterium adolescentis) seemed to show a decline because of transplantation (
                              <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              ).
                             </p>
                             <div id="T1" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2556492/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2556492/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2556492" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 1.
                               </span>
                               <p id="p-15" class="first-child">
                                Changes in abundance and prevalence of gut microbial species after FMT.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Transferred and engrafted gut microbial species
                             </h3>
                             <p id="p-16">
                              To assess species engraftment, we next designed a proof of concept analysis to discriminate strain-level variation and thus trace microbes engraftment more accurately. For this purpose, we selected Parabacteroides merdae, a predominant species in the samples assessed that showed the greatest increase as a result of FMT (
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Figure 3A
                              </a>
                              ), and Faecalibacterium prausnitzii, a highly abundant species in the analyzed samples for which evident transfer between the donor-recipient pairs was not observed (
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Figure 3B
                              </a>
                              ), to detect single-nucleotide variations (SNVs) indicative of strains identity/transference. After a massive analysis of the total dataset merging the species-specific rrn reads derived from all subjects (see methods), we identified three potential informative SNV sites in the P. merdae rrn, whereas six were found in the F. prausnitzii counterpart (
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Figure 3C-D
                              </a>
                              ). In an attempt to explore the natural existence of such genotypic variation inferred from our analysis in a predictive manner, we looked for the presence of similar variants in the NCBI non-redundant nucleotide database using the 100 nt surrounding region as a query for all P. merdae SNVs. The inferred genetic variability was replicated at least for SNV-3527, where multiple matches presented A and G nucleotides at this position (Figure S4). When we studied the nucleotide frequencies of these particular SNVs across samples, a clear transfer pattern of the P. merdae genotype from donor to recipient was detected, as indicated by the decreased genetic distance (Bray-Curtis index for assessing the abundance of the four nucleotides in all SNVs at once, one-sided Wilcoxon Signed-Rank test = 11, p = 0.051) between donor samples and POST-FMT compared to that of POST-FMT and PRE-FMT pairs (
                              <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                               Figure 3E
                              </a>
                              ). By contrast, the genetic distances retrieved after the comparison of F. prausnitzii genotypes between POST-FMT or PRE-FMT samples and their donors did not indicate the transfer of strains belonging to this species from donors to recipients (
                              <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                               Figure 3F
                              </a>
                              ). Comparative analysis of the delta Bray-Curtis distance (POST vs DONOR minus PRE vs DONOR samples) between both species also suggests that only P. merdae was transferred from donors to recipients, since the distance between POST-FMT and donor samples tended to decrease (
                              <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                               Figure 3G
                              </a>
                              , p = 0.056). The direct Sanger sequencing of P. merdae SNV-1254, SNV-2293, and SNV-3527 in the two recipients and their common donor supported the results of our long-read-based assessment of FMT (
                              <a id="xref-fig-3-7" class="xref-fig" href="#F3">
                               Figure 3H
                              </a>
                              ). Globally, we observed the presence of a mixture of strains in some of the samples analyzed, given the basecalling profile visualized in the electropherograms (e.g., SNV-3527 of Donor1 and 06-Pre samples). This pattern of strain coexistence was more evident in the POST-FMT samples of both analyzed recipients (see SNV-2293 and SNV-3527 in
                              <a id="xref-fig-3-8" class="xref-fig" href="#F3">
                               Figure 3H
                              </a>
                              ). The predominant haplotype observed in Donor1 (A1254-C2293-A3527) was transferred to the two recipients explored and became dominant after FMT. Similar patterns of transfer were observed in other donor-recipient pairs. Globally, our results demonstrated that the increased abundance of P. merdae after the FMT was a direct consequence of donor strain transmission. This is likely the case for other bacterial species that increased as a result of FMT.
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F3.large.jpg?width=800&height=600&carousel=1" title="Single-nucleotide variation (SNV) analysis to detect species transfer and engraftment. The species Parabacteroides merdae and Faecalibacterium prausnitzii rrns were used to reveal SNVs within each species and their abundance as a consequence of FMT (A and B panels, respectively). Variation in one unit of clr-normalized counts is roughly equal to a doubling in raw DNA read counts according to data distribution in both cases. C and D - The mapping, indexing, and pileup of thousands of reads of P. merdae and F. prausnitzii rrns allowed us to identify polymorphic sites exhibiting an even frequency for at least two nucleotides. These potential SNVs are highlighted inside open circles, and the position number in the rrn sequence is also indicated. The structural arrangement of rrn is illustrated on the x-axis. The black lines indicate the relative frequency of the dominant allele, whereas the gray lines indicate the relative coverage per site related to the average across the rrn. E and F - The distribution of the Bray-Curtis dissimilarity index of the microbiota between POST-FMT and PRE-FMT samples and the corresponding donors, using the combined nucleotide frequencies of SNVs detected in P. merdae and F. prausnitzii, respectively. The Wilcoxon signed-rank test for paired samples was used to compare the genetic distances of the microbiotas. G – Assessment of delta Bray-Curtis distances computed as POST-FMT vs DONOR samples minus PRE-FMT vs DONOR samples for Faecalibacterium prasunitzii and Parabacteroides merdae species. H - Electropherograms obtained from Sanger sequencing for the strains of P. merdae SNV-3527, SNV-2293, and SNV-1254. The predominant alleles inferred for every sample (based on the Q-score of base calling) supported the hypothesis that the strains were transferred between donor and recipient pairs, as anticipated during the Nanopore-based assessment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-667907561" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Single-nucleotide variation (SNV) analysis to detect species transfer and engraftment. The species <em>Parabacteroides merdae</em> and <em>Faecalibacterium prausnitzii rrn</em>s were used to reveal SNVs within each species and their abundance as a consequence of FMT (A and B panels, respectively). Variation in one unit of clr-normalized counts is roughly equal to a doubling in raw DNA read counts according to data distribution in both cases. C and D - The mapping, indexing, and pileup of thousands of reads of <em>P. merdae</em> and <em>F. prausnitzii rrns</em> allowed us to identify polymorphic sites exhibiting an even frequency for at least two nucleotides. These potential SNVs are highlighted inside open circles, and the position number in the <em>rrn</em> sequence is also indicated. The structural arrangement of <em>rrn</em> is illustrated on the x-axis. The black lines indicate the relative frequency of the dominant allele, whereas the gray lines indicate the relative coverage per site related to the average across the <em>rrn</em>. E and F - The distribution of the Bray-Curtis dissimilarity index of the microbiota between POST-FMT and PRE-FMT samples and the corresponding donors, using the combined nucleotide frequencies of SNVs detected in <em>P. merdae</em> and <em>F. prausnitzii</em>, respectively. The Wilcoxon signed-rank test for paired samples was used to compare the genetic distances of the microbiotas. G – Assessment of delta Bray-Curtis distances computed as POST-FMT vs DONOR samples minus PRE-FMT vs DONOR samples for <em>Faecalibacterium prasunitzii</em> and <em>Parabacteroides merdae</em> species. H - Electropherograms obtained from Sanger sequencing for the strains of <em>P. merdae</em> SNV-3527, SNV-2293, and SNV-1254. The predominant alleles inferred for every sample (based on the Q-score of base calling) supported the hypothesis that the strains were transferred between donor and recipient pairs, as anticipated during the Nanopore-based assessment.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F3.medium.gif" width="440" height="270"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F3.medium.gif" width="440" height="270"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3.
                               </span>
                               <p id="p-17" class="first-child">
                                Single-nucleotide variation (SNV) analysis to detect species transfer and engraftment. The species Parabacteroides merdae and Faecalibacterium prausnitzii rrns were used to reveal SNVs within each species and their abundance as a consequence of FMT (A and B panels, respectively). Variation in one unit of clr-normalized counts is roughly equal to a doubling in raw DNA read counts according to data distribution in both cases. C and D - The mapping, indexing, and pileup of thousands of reads of P. merdae and F. prausnitzii rrns allowed us to identify polymorphic sites exhibiting an even frequency for at least two nucleotides. These potential SNVs are highlighted inside open circles, and the position number in the rrn sequence is also indicated. The structural arrangement of rrn is illustrated on the x-axis. The black lines indicate the relative frequency of the dominant allele, whereas the gray lines indicate the relative coverage per site related to the average across the rrn. E and F - The distribution of the Bray-Curtis dissimilarity index of the microbiota between POST-FMT and PRE-FMT samples and the corresponding donors, using the combined nucleotide frequencies of SNVs detected in P. merdae and F. prausnitzii, respectively. The Wilcoxon signed-rank test for paired samples was used to compare the genetic distances of the microbiotas. G – Assessment of delta Bray-Curtis distances computed as POST-FMT vs DONOR samples minus PRE-FMT vs DONOR samples for Faecalibacterium prasunitzii and Parabacteroides merdae species. H - Electropherograms obtained from Sanger sequencing for the strains of P. merdae SNV-3527, SNV-2293, and SNV-1254. The predominant alleles inferred for every sample (based on the Q-score of base calling) supported the hypothesis that the strains were transferred between donor and recipient pairs, as anticipated during the Nanopore-based assessment.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Correlation between clinical variables and species abundance after FMT
                             </h3>
                             <p id="p-18">
                              Among the multiple clinical variables evaluated in these subjects with metabolic syndrome, FMT tended to induced positive changes in the markers of glucose metabolism and blood pressure in the recipients (
                              <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              ). Fasting insulin (p = 0.030), fasting glucose (p = 0.074), and consequently HOMA-IR (p = 0.005) tended to improved in the recipients after FMT. In line with these findings, there was an apparent decrease in the glycosylated hemoglobin concentration after FMT (p = 0.060). On the other hand, systolic and diastolic blood pressures were lowered as a consequence of donor FMT treatment, being reduced by 10% (p = 0.028) and 19% (p = 0.0008), respectively (
                              <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              ). The concentrations of fecal SCFAs, such as fecal butyrate (p = 0.018) and acetate (p = 0.033), were decreased after FMT, whereas that of propionate was not (p = 0.280). Regarding markers of lipid metabolism, HDL cholesterol levels in plasma were reduced (p = 0.032). By computing rank-based Kendall’s τ (tau) parameter, we established associations between these clinical variables and the alterations in the abundance of microbial species as a consequence of FMT (see
                              <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              ). Species such as P. merdae, which was increased after FMT, were negatively correlated with diastolic blood pressure (τ = -0.44 and FDR = 0.062), whereas A. shahii was negatively correlated with the abundance of fecal acetate and butyrate (τ = -0.50, -0.42 and FDR = 0.017, 0.042, respectively). The above correlations were not detected for other closely related and abundant species (e.g., Parabacteroides johnsonii, Parabacteroides distasonis, Parabacteroides goldsteinii, Alistipes finegoldii, Alistipes onderdonkii, Alistipes putredinis or Alistipes indistinctus).
                             </p>
                             <div id="T2" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2556482/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2556482/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2556482" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 2.
                               </span>
                               <p id="p-19" class="first-child">
                                Clinical variables altered after FMT.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              Donors’ influence on species engraftment and amelioration of metabolic dysfunction
                             </h3>
                             <p id="p-20">
                              After showing that microbiota engraftment took place between donor and recipients, and that reversion of certain metabolic alterations may have been guided by particular microbial species transferred from donors to recipients, we wanted to explore to what extent the donors and their microbiota configurations can explain the FMT success. For such an aim, we ranked the metabolic improvement individually (from 1 to 10, from the lowest to the highest), including the reduction of blood pressure, fasting glucose and insulin levels, glycosylated hemoglobin, HOMA-IR, weight loss, and BMI of the ten subjects of the FMT. Then, the health score resulting from the sum of all scores for every subject was estimated. This score was related to t the dissimilarity index (Bray-Curtis) between the baseline microbiota configuration of donor-recipient pairs and the community structural shift resulting from the FMT in every recipient, taking into account their donor pair (
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Figure 4
                              </a>
                              ). Globally, we observed that metabolic health scores correlate to some extent with the baseline dissimilarity between donors and recipients microbiotas and the community structural changes after FMT. Consequently, the more dissimilarity in the baseline gut microbiota of donors and recipients, the lower the benefits to revert the metabolic dysfunction in recipients and less impact on their gut microbiota structure after FMT. The above observation applies to almost all donor-recipient pairs, except for those including the Donor2. Although we observed a linear relationship between the averaged dissimilarity between donor-recipient pairs and the respective shift in the community structure after FMT (adjusted R-squared: 0.956, p = 0.013), a rank-based approach (Kendall’s tau) showed those parameters were not well correlated with the metabolic health score (τ ≥ -0.24, p ≥ 0.381). Therefore, additional traits yet to explore in donor-recipient pairs should help in the selection of subjects for succeeding in FMT. In line with this hypothesis, we finally evaluated the relationship between the metabolic health score and the recipients sex, showing that women exhibited a larger improvement in metabolic parameters than men (women [N = 5] = 49.6 ± 13.8 health score, men [N = 5] = 33.6 ± 11.3 health score, t-test = 6.62, p = 0.003). Moreover, we found a strong rank-based correlation between the age difference of donor-recipient pairs (donor age - recipient age) and the metabolic health score (Kendall’s τ = -0.51, p = 0.046), suggesting that the youngest the donor (with respect to the recipient), the greater reversion of metabolic dysfunction.
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F4.large.jpg?width=800&height=600&carousel=1" title="Donors’ influence on FMT success. A scatter plot showing the comparison of gut microbiota between donor-recipient pairs at baseline (x-axis, Bray-Curtis dissimilarity index computed) versus the shift in the microbiota community structure of every recipient before and after the FMT (y-axis). The size of filled dots is determined by the sum of ranks linked to the improvement of all metabolic parameters evaluated (blood pressure, fasting glucose and insulin, glycosylated hemoglobin, HOMA-IR, weight loss, and BMI loss) (metabolic health score). Color legend is associated with donor information. Asterisks and size represent the average values of all recipients linked to every donor. Dashed grey line indicates linear correlation based on averaged values (asterisks)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-667907561" data-figure-caption='<div class="highwire-markup">Donors’ influence on FMT success. A scatter plot showing the comparison of gut microbiota between donor-recipient pairs at baseline (x-axis, Bray-Curtis dissimilarity index computed) versus the shift in the microbiota community structure of every recipient before and after the FMT (y-axis). The size of filled dots is determined by the sum of ranks linked to the improvement of all metabolic parameters evaluated (blood pressure, fasting glucose and insulin, glycosylated hemoglobin, HOMA-IR, weight loss, and BMI loss) (metabolic health score). Color legend is associated with donor information. Asterisks and size represent the average values of all recipients linked to every donor. Dashed grey line indicates linear correlation based on averaged values (asterisks).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F4.medium.gif" width="440" height="295"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F4.medium.gif" width="440" height="295"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/19/2020.09.11.292896/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4.
                               </span>
                               <p id="p-21" class="first-child">
                                Donors’ influence on FMT success. A scatter plot showing the comparison of gut microbiota between donor-recipient pairs at baseline (x-axis, Bray-Curtis dissimilarity index computed) versus the shift in the microbiota community structure of every recipient before and after the FMT (y-axis). The size of filled dots is determined by the sum of ranks linked to the improvement of all metabolic parameters evaluated (blood pressure, fasting glucose and insulin, glycosylated hemoglobin, HOMA-IR, weight loss, and BMI loss) (metabolic health score). Color legend is associated with donor information. Asterisks and size represent the average values of all recipients linked to every donor. Dashed grey line indicates linear correlation based on averaged values (asterisks).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-22">
                             Here, we explored the rrn amplicon sequencing of the human gut microbiota through Nanopore sequencing and the de novo assembly of sequences of this multilocus hypervariable region to gain insights into the corresponding genetic markers and to demonstrate their potential use for deeply characterizing complex communities at the species and strain levels, permitting the assessment of microbe engraftment between donors and recipients in an FMT clinical trial. The utility of rrn operon sequencing via the MinION™ platform has already demonstrated a notable performance in resolving microbial communities at the species level, with the potential to define strain variability for particular species
                             <sup>
                              <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                               24
                              </a>
                              ,
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                             </sup>
                             .
                            </p>
                            <p id="p-23">
                             We provided de novo reliable assemblies for more than two hundred rrn region sequences of human gut microbes. We were able to reconstruct a large proportion of rrn regions from approximately 40 new microbial species, and more than 150 new strains were absent in the first database release
                             <sup>
                              <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                             </sup>
                             . These numbers, limited mainly by sequencing coverage, could also be reduced by the DNA extraction method used, in which a physical disruption step was omitted to maintain DNA integrity and ensure the successful amplification of the 5 kbp amplicons. However, the homogeneous processing of all samples resulted in comparable microbial profiles before and after FMT and did not affect the species-specific evaluation conducted in this study.
                            </p>
                            <p id="p-24">
                             By using a dataset consisting of approximately 400 K Nanopore reads derived from 24 samples, employing the updated rrn database, and controlling the covariates influencing the microbiota, we revealed changes in the human gut microbiota at the species and strain levels as a consequence of an FMT intervention. This long amplicon-based approach enabled us to detect increases in species such as Parabacteroides merdae, Butyricimonas virosa, Prevotella copri, and Sutterella massiliensis as a result of FMT. In addition to age, sex, sequencing runs (recognized stochastic run-to-run variation confounding the interpretation of microbiota data
                             <sup>
                              <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                               26
                              </a>
                              ,
                              <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                               27
                              </a>
                             </sup>
                             ), and baseline BMI, we found that the donor was a critical covariate contributing to the impact of FMT on the recipient microbiota when using a unique donor for multiple recipients. These results are in agreement with those previously published on the basis of short reads from the V4 hypervariable region of the 16S rRNA bacterial gene explored in a larger cohort of subjects
                             <sup>
                              <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                             </sup>
                             , for instance, the significant increase of P. copri after FMT; however, our results outperform the taxonomic resolution achieved previously, thus providing a more accurate gut microbiota survey. Interestingly, a recent FMT study on the treatment of ulcerative colitis (UC) in which the microbiota was analyzed by sequencing short reads also showed increases in Sutterella species as a consequence of the intervention
                             <sup>
                              <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                               28
                              </a>
                             </sup>
                             . Similarly, the abundance of Butyricimonas species was shown to be increased as a consequence of an FMT intervention aimed at eradicating antibiotic-resistant bacteria
                             <sup>
                              <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                             </sup>
                             . These findings suggest that some species may frequently show changes as a result of FMT in humans, regardless of the condition of the recipient. Nevertheless, the presumable transfer of species between donors and recipients and their engraftment could not be confirmed in those studies based on short-read amplicon technology due to the lack of sufficient resolution. Additionally, potential replacement with closely related species in the recipient could be impossibly monitored in conventional microbiota surveys based on short reads. To shed light on the ability of the rrn sequencing approach to assess SNVs that are likely associated with strain diversity, thus demonstrating donors’ microbe engraftment in recipients, we designed a proof of concept analysis selecting P. merdae, a species that exhibited a remarkable increase in the recipients’ gut after FMT, and F. prausnitzii, which seemed to not be affected by FMT intervention. The combined information on different informative SNVs from the P. merdae rrn sequence clearly suggests that the increase in this species observed in recipients after undergoing FMT was likely because of strain transfer from the donors. This inference is further supported by the existence of nucleotide variants inferred from Nanopore data processing in either the samples evaluated in the present study or the genomic data present in public repositories and derived from different sequencing methodologies.
                            </p>
                            <p id="p-25">
                             The taxonomic resolution of our sequencing approach, proving the transference of specific bacterial strains from donors to recipients, helped draw partial community structures at the species level and determine compositional distances and their shifts as a consequence of the FMT. This strategy helped to more firmly support a causal relationship between the changes in the gut microbiota and the improvements in the recipient’s clinical variables. In this regard, we attempted to demonstrate that overall improvement of metabolic health (global score) was associated with baseline dissimilarity between microbial communities of donors and recipients and with their microbiota configuration shifts after FMT. We observed that the more similar is the gut microbiota of the donor-recipient pairs at baseline, the better is the microbiota transference and the improvement in metabolic parameters. However, this correspondence was not proven for all pairs; possibly, additional traits of donors and recipients yet to be determined need to be considered for an optimal selection of donors to ensure the clinical success of the FMT. One of those unknown additional factors could be the sex of the recipient and donor. Indeed, we observed that women recipients exhibited higher metabolic health score after FMT than men. Given that the sex of donors and recipients was matched, further studies with a mixed-sex distribution would be needed to determine i) if women’s fecal material can be better engrafted regardless of the recipient sex, or ii) whether women respond better to FMT irrespective of the sex origin of the FMT material. We have previously unveiled differential responses of women and men gut microbiota to a particular dietary intervention, suggesting that such disparate sex-associated microbial configurations and related traits could plausibly explain the success of FMT
                             <sup>
                              <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                             </sup>
                             . Nonetheless, previous FMT interventions in the context of IBS show no differential response between women and men
                             <sup>
                              <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                             </sup>
                             . Finally, an individual-specific response to FMTs cannot be disregarded as in our trial the material from Donor1 exerted the most positive impact on the metabolic health of their three different recipients. Consequently, the role of the personal features of the donor, particularly those influencing the gut microbiota composition and metabolic potential (e.g. diet, lifestyle) as well as other host covariates (e.g. age) linked to successful FMTs should be further investigated to better guide donor selection and identify microbial signatures improving metabolic health.
                            </p>
                           </div>
                           <div class="section" id="sec-10">
                            <h2 class="">
                             Conclusions
                            </h2>
                            <p id="p-26">
                             The molecular methods and data analyses presented here enabled reliable microbial surveys at the species level and the inference of strain variations, taking into account the most abundant human intestinal bacterial species. To the best of our knowledge, this is the first microbiota assessment of fecal samples derived from donors and recipients involved in a FMT clinical trial to demonstrate transference and engraftment of specific bacterial species by exploring the genetic diversity at the strain level using amplicon sequencing. The genetic and taxonomy information retrieved from advanced sequencing methods used here permitted us to evaluate the donors’ influence on the gut microbiota transference and clinical success of the FMT. The determinants of the success include a high relatedness between the donor-recipient baseline microbiota configurations, the sex and the age of the donor, which seem to influence the engraftment of intestinal microbes and the amelioration of metabolic syndrome markers. Further studies should be warranted to confirm our findings in larger intervention trials and demonstrate their predictive value.
                            </p>
                           </div>
                           <div class="section" id="sec-11">
                            <h2 class="">
                             Materials and Methods
                            </h2>
                            <div id="sec-12" class="subsection">
                             <h3>
                              Subjects, samples, and clinical data
                             </h3>
                             <p id="p-27">
                              Samples were obtained upon receiving informed consent from participants in a previous FMT clinical trial carried out under the framework of the MyNewGut project (
                              <a href="http://www.mynewgut.eu/">
                               http://www.mynewgut.eu/
                              </a>
                              ) between September 2014 and March 2017. The details of the study design are publicly available elsewhere
                              <sup>
                               <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">
                                3
                               </a>
                              </sup>
                              . Briefly, a total of twenty-four fecal samples were analyzed in the present study. Twenty samples were obtained from 10 recipients undergoing allogenic FMT, who provided one sample before (PRE-FMT fecal samples) and one sample 4 weeks after (POST-FMT fecal samples) the intervention. The samples of 4 donors were also analyzed, and the effect of this variable on the recipients was considered in the data analysis. This subset of samples was mainly assessed to explore differential species engraftment in multiple FMT recipients with common donors. The sample size evaluated in this study is still representative according to parameters described previously for this intervention arm (two-sided significance level of 0.05 and power of 80%, minimum sample size = 10)
                              <sup>
                               <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">
                                3
                               </a>
                              </sup>
                              . Normality test was completed prior to execute parametric or non-parametric tests to assess changes in the clinical variables explored.
                             </p>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              DNA extraction, multilocus amplification, and sequencing
                             </h3>
                             <p id="p-28">
                              Microbial DNA was recovered from 100 mg feces samples by using the QIAamp® Fast DNA Stol Mini kit (Qiagen, Hilden, Germany) according to the manufacturer’s instructions and omitting cell disruption by mechanical methods (bead-beating) to preserve DNA with the high molecular weight needed to amplify amplicons of 5 kbp in length. The rrn region comprising the nearly full-length bacterial RNA ribosomal operon (16S, ITS, and 23S) was amplified as previously reported
                              <sup>
                               <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">
                                16
                               </a>
                              </sup>
                              . Dual-barcoded purified PCR products were mixed in equimolar proportions before sequencing library preparation. In total, three different libraries were prepared from ∼1 µg of mixed amplicon DNA (containing 7, 8, or 10 barcoded samples) using the SQK-LSK108 sequencing kit (Oxford Nanopore Technologies, Oxford, UK) following the manufacturer’s instructions to produce 1D reads. Each library was individually loaded into FLO-MIN107 (R9.5) flow cells (Oxford Nanopore Technologies, Oxford, UK), and sequencing was carried out in a portable MinION™ MkIb sequencer (Oxford Nanopore Technologies, Oxford, UK), operated with MINKNOW v1.10.23 software (Oxford Nanopore Technologies, Oxford, UK). Flow cells were primed according to the manufacturer’s instructions, and an ∼18 h run of 1D sequencing (outperforming the 2D approach for this multiplex and amplicon setup, and improving de-multiplexing by using the barcode+primer tags) was then executed for each library and flow cell.
                             </p>
                            </div>
                            <div id="sec-14" class="subsection">
                             <h3>
                              Data preprocessing
                             </h3>
                             <p id="p-29">
                              Fast5 files were processed with the Albacore v2.1.3 basecaller and default configuration (minimum q-score filter = 7), and fasta files were retrieved for downstream analyses. The barcode and primer (forward or reverse) sequence information was used for demultiplexing into the DNA reads generated from forward and reverse primers according to previous procedures
                              <sup>
                               <a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">
                                16
                               </a>
                              </sup>
                              . A size filtering step was configured to retain those reads of least 1,500 nt in length. Barcode and primer sequences were then removed by trimming 50 nucleotides at the 5’ end of forward and reverse reads. The sequencing chemistry and Nanopore data-processing algorithms were the latest available by January 2018, when data was generated. A high-performance basecaller released after the end of this analysis was tested on fast5 data produced getting no significant difference in sequences quality (Figure S4).
                             </p>
                            </div>
                            <div id="sec-15" class="subsection">
                             <h3>
                              Assembly of rrn regions
                             </h3>
                             <p id="p-30">
                              The study design and the assembly of rrn sequences are graphically explained in
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Figure 1
                              </a>
                              . This pipeline carries out what we refer to as the strand-wise and reference-assisted method for rrn region reconstruction via the following steps:
                             </p>
                             <ul class="list-unord" id="list-1">
                              <li id="list-item-1">
                               <p id="p-31">
                                Demultiplexing of Nanopore data using barcode and primer sequence information generated two datasets per sample comprising DNA reads led from the 5’
s end of rrn amplicons (initiated from forward primer mapping at the 16S rRNA gene start site = forward reads), and DNA reads beginning at the 3’ end of rrn amplicons (initiated from reverse primer mapping at the 23S rRNA gene end site = reverse reads).
                               </p>
                              </li>
                              <li id="list-item-2">
                               <p id="p-32">
                                Reads from all 24 samples were merged into forward and reverse read subsets.
                               </p>
                              </li>
                              <li id="list-item-3">
                               <p id="p-33">
                                Forward read binning into precise microbial species via competitive alignment against the nonredundant 16S NCBI database (release January 2018). The LAST aligner
                                <sup>
                                 <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                                  32
                                 </a>
                                </sup>
                                with a -s 2 -q 1 -b 1 -Q 0 -a 1 -r 1 configuration was used for this purpose. The alignment score was the main criterion for selecting top hits. In the case of multiple hits with the same top score, the alignment was discarded for downstream processing. High-quality alignments were retained according to sequence identity (sequence identity above the 33rd percentile ≥ 85%) and length information (alignments ≥ 1500 nt).
                               </p>
                              </li>
                              <li id="list-item-4">
                               <p id="p-34">
                                A maximum of 500 forward reads per species (randomly shuffled), producing high-quality alignments, were selected, and aligned through iterative refinement methods implemented in MAFFT v7.310 with the default parameters
                                <sup>
                                 <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                                  33
                                 </a>
                                </sup>
                                , and the consensus sequence was then obtained by using the hmmbuild and hmmemit algorithms implemented in HMMER3
                                <sup>
                                 <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                                  34
                                 </a>
                                </sup>
                                . No significant improvements in consensus rrn sequences were obtained using more than 500 sequences per species. Assemblers supporting nanopore data, temporary available at the date of analysis, were tested to produce de novo consensus sequences but they failed to produce outputs supporting the expected species-level diversity because of the coverage bias across data.
                               </p>
                              </li>
                              <li id="list-item-5">
                               <p id="p-35">
                                Consensus sequences were annotated according to the NCBI taxonomy and used as a reference to bin reverse reads in a similar manner as that used for forward reads. The thresholds for the selection of high-quality alignments based on reverse reads and the preliminary assembly of rrn regions were 85% sequence identity (upper 45th percentile) and ≥ 3500 nt in length.
                               </p>
                              </li>
                              <li id="list-item-6">
                               <p id="p-36">
                                Reverse reads were compared against the consensus sequences obtained from the forward read assemblies and binned into the respective species according to NCBI taxonomic annotation. A maximum of 500 reverse reads per species (randomly shuffled), producing high-quality alignments, were selected, merged with a maximum of 500 forward reads assigned reciprocally to the same species (randomly shuffled), and aligned through iterative refinement methods implemented in MAFFT, after which a new consensus sequence was obtained by using the hmmbuild and hmmemit algorithms implemented in HMMER3.
                               </p>
                              </li>
                              <li id="list-item-7">
                               <p id="p-37">
                                Final consensus sequences were annotated according to the concordant NCBI taxonomy, obtained from forward and reverse reads, and used as a reference to study the abundance and prevalence of gut microbiota members at the species level.
                               </p>
                              </li>
                              <li id="list-item-8">
                               <p id="p-38">
                                A BLAST-based search of the rrn assemblies against the nonredundant nucleotide and reference 16S NCBI databases was performed to evaluate the representation of the reconstructed and identified operons in public databases. The identification of rrn assemblies was compared against the SILVA database
                                <sup>
                                 <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                                  35
                                 </a>
                                </sup>
                                with SINA aligner
                                <sup>
                                 <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                                  36
                                 </a>
                                </sup>
                                . Top hit selection was based on the taxonomy score, TS = Log
                                <sub>
                                 10
                                </sub>
                                [alignment score*sequence identity*alignment length].
                               </p>
                              </li>
                             </ul>
                            </div>
                            <div id="sec-16" class="subsection">
                             <h3>
                              Long-read mapping and variant calling
                             </h3>
                             <p id="p-39">
                              The rrn database
                              <sup>
                               <a id="xref-ref-16-7" class="xref-bibr" href="#ref-16">
                                16
                               </a>
                              </sup>
                              was updated with more than two hundred new rrn operon sequences assembled in this study (labeled operONT) and re-annotated to different taxonomic levels according to the NCBI taxonomy database. The mapping of sample reads was performed against rrn_DBv2 (
                              <a href="https://github.com/alfbenpa/rrn_DBv2">
                               https://github.com/alfbenpa/rrn_DBv2
                              </a>
                              ) by competitive alignment using the LAST aligner, as stated above. Taxonomic assignment was based on the best hit retrieved by the calculation of TS (see above), and reliable taxonomic assignments were filtered to those based on alignments with at least 85% sequence identity that were longer than 1500 nt. Species read counts were normalized using the compositions::clr R v3.6 function, and a covariate-controlling linear mixed model-based method was used to explore the differential abundance of taxonomic features between conditions as stated in the paragraphs below. Species engraftment was evaluated by the selection of all sample reads mapped to the Parabacteroides merdae and Faecalibacterium prausnitzii species. To detect informative sites, based on single-nucleotide variants that were probably linked to strain variation, we first proceeded to map all sample reads matching these species to the respective reference rrn sequences using LAST aligner and the parameters set across this study (see rrn assembly). The maf-convert algorithm (LAST aligner toolkit) was used to generate the respective SAM files. The algorithms compiled within SAMtools v1.3.1 were used to index, order, and pileup reads as well as to retrieve information regarding the nucleotide frequency per site and coverage (vcf files). Site selection was based on rrn positions showing no nucleotide dominance (<70% frequency for a single nucleotide), without reductions in the coverage (>75% of the relative coverage), thus obtaining sites with a balanced representation of at least the two predominant alleles likely representing strain variation across all samples. Second, the sample reads that were mapped to the reference species were individually assessed to detect meaningful changes in intestinal bacterial genotypes between the recipients PRE-FMT and POST-FMT samples. The changes in nucleotide frequencies at all selected SNV sites (three for P. merdae and six for F. prausnitzii) were assessed by calculating the Bray-Curtis dissimilarity index between pairs of POST-FMT and PRE-FMT samples or POST-FMT and corresponding donor samples following statistical evaluation by using the Wilcoxon signed rank test for paired samples.
                             </p>
                            </div>
                            <div id="sec-17" class="subsection">
                             <h3>
                              Sanger sequencing and validation of Parabacteroides merdae SNVs
                             </h3>
                             <p id="p-40">
                              The reference rrn sequence from P. merdae was submitted to the Primer-BLAST web server (
                              <a href="https://www.ncbi.nlm.nih.gov/tools/primer-blast/">
                               https://www.ncbi.nlm.nih.gov/tools/primer-blast/
                              </a>
                              ) to retrieve species-specific primer pairs to selectively amplify this rrn sequence and primers flanking SNV-1254, SNV-2293, and SNV-3527 for Sanger sequencing. Comparisons against the nonredundant NCBI database and Parabacteroides merdae [taxid:46503] as a reference organism were fixed as checking parameters for primer prediction. The rrn region from P. merdae was amplified via 28 PCR cycles including the following stages: 95°C for 20 s, 61°C for 30 s, and 72°C for 150 s. Phusion High-Fidelity Taq Polymerase (Thermo Scientific) and the pm623 (TGCCGTTGAAACTGGGTTACTTGA) and pm3785 (TTTCGCACAGCCATGTGTTTTGTT) primer pairs were used in the amplification reaction. The PCR products were cleaned with an Illustra GFX PCR DNA and gel band purification kit (GE Healthcare, Chicago, IL, USA) and sequenced by using Sanger technology in an ABI 3730XL sequencer (STAB-VIDA. Caparica, Portugal) using the primers described above (pm623 and pm3785) in addition to the pm1855 (ACCCCTTACGGAGTTTATCGTGGA) and pm2686 (TTCGCGTCTACTCACTCCGACTAT) primers. The sequencing electropherograms from the ab1 files were visualized with FinchTV v1.4.0 (Geospiza Inc.).
                             </p>
                            </div>
                            <div id="sec-18" class="subsection">
                             <h3>
                              Diversity and taxonomic analyses at the species level
                             </h3>
                             <p id="p-41">
                              The prevalence and abundance of a total of 2,519 species contained in the database were evaluated, and diversity analyses were completed taking into account the species with a > 0.01% relative abundance on average (∼250 species in total). Alpha diversity descriptors such as Chao’s index, Shannon’s entropy, Simpson’s reciprocal index, and dominance were obtained by using QIIME v1.9.1
                              <sup>
                               <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                              </sup>
                              . Similarly, QIIME was used to calculate the Bray-Curtis dissimilarity indexes among samples and to perform multivariate exploratory (principal coordinate analysis - PCoA) and statistical (PERMANOVA) analyses. A linear mixed model (LMM – nlme::lme R v3.6 function) analysis was also conducted on clr-transformed data to detect differential features in the microbiota before and after the intervention. The inherent variation due to individual features was set as a random effect for each variable analyzed (fixed effect). The possible covariates of the clinical and fecal microbiota that showed differences between the study groups (p-value ≤ 0.05) were selected. Recognized covariates of microbiota, such as age, sex, baseline BMI, and sequencing batch, were identified as significant in this study and included as random effects in the LMM. To identify microbial species potentially linked to clinical variables that were altered as a consequence of FMT, Kendall’s τ (tau) parameter was estimated between variable pairs and corrected for multiple testing using the false discovery rate (FDR) approach. Associations were selected according to an FDR p-value ≤ 0.1. Graphics were generated in R v3.6 using the ggplot2 package.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-20">
                            <h2 class="">
                             Declarations
                            </h2>
                            <div class="section" id="sec-21">
                             <h3>
                              Ethics approval and consent to participate
                             </h3>
                             <p id="p-55">
                              The study was prospectively registered at the Dutch Trial registry (
                              <a href="https://www.trialregister.nl/trial/4488">
                               https://www.trialregister.nl/trial/4488
                              </a>
                              ) and was conducted according to the guidelines laid down in the Declaration of Helsinki and the ethical standards of the responsible local committee on human experimentation of the Amsterdam UMC (location AMC)
                              <sup>
                               <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">
                                3
                               </a>
                              </sup>
                              . The study was registered on August 1
                              <sup>
                               st
                              </sup>
                              , 2014. The first participant was enrolled on September 1
                              <sup>
                               st
                              </sup>
                              , 2014.
                             </p>
                            </div>
                            <div class="section" id="sec-22">
                             <h3>
                              Availability of data and material
                             </h3>
                             <p id="p-56">
                              The Albacore-basecalled fast5 files obtained from specific runs are publicly available in the European Nucleotide Archive under accession number PRJEB33947. The updated rrn database (rrn_DBv2) is publicly accessible at the GitHub repository
                              <a href="https://github.com/alfbenpa/rrn_DBv2">
                               https://github.com/alfbenpa/rrn_DBv2
                              </a>
                              .
                             </p>
                            </div>
                            <div class="section" id="sec-23">
                             <h3>
                              Competing interests
                             </h3>
                             <p id="p-57">
                              The authors have no conflicts of interest to declare.
                             </p>
                            </div>
                            <div class="section" id="sec-24">
                             <h3>
                              Funding
                             </h3>
                             <p id="p-58">
                              This study was supported by the EU Project MyNewGut (No. 613979) of the European Commission 7th Framework Programme and grant PID2020-119536RB-I00 from the Ministry of Science and Innovation (Spain). This research study was also completed thanks to the CP19/00132 grant from Miguel Servet programe to ABP from the Institute of Health Carlos III (ISCIII) and its co-funding from the European Social Fund (ESF/FSE).
                             </p>
                            </div>
                            <div class="section" id="sec-25">
                             <h3>
                              Authors’ contributions
                             </h3>
                             <p id="p-59">
                              ABP conceived and designed the study. ABP performed the experimental sequencing research and data analysis, and AVH and MN performed the clinical research. ABP and YS directed the study. ABP and YS wrote the manuscript. All authors reviewed and approved the final version of the manuscript.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-26">
                            <h2 class="">
                             Figure legends
                            </h2>
                            <p id="p-61">
                             <strong>
                              Figure S1
                             </strong>
                             . BLAST-based results showing the correct identification of assembled rrns. The top hits supporting the identification are shown for six different rrns assembled with low, intermediate or high coverage (number of reads).
                            </p>
                            <p id="p-62">
                             <strong>
                              Figure S2
                             </strong>
                             . Alpha diversity of the recipients’ microbiota before and after FMT. The distributions of the values obtained for four alpha diversity indicators, including Chao’s index, Shannon’s index, the reciprocal Simpson’s index, and the dominance index, are shown as boxplots. The results of the Wilcoxon signed-rank test applied to establish differences between the two groups of samples (POST-FMT and PRE-FMT) are also shown.
                            </p>
                            <p id="p-63">
                             <strong>
                              Figure S3
                             </strong>
                             . Searching for SNV genotypic variation in the NCBI nonredundant nucleotide database. The nucleotide sequences spanning positions 1180 to 1280, 2250 to 2350, and 3470 to 3570 of the P. merdae rrn operon were used as queries in searches against the nonredundant nucleotide database on the BlastN web server (
                             <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">
                              https://blast.ncbi.nlm.nih.gov/Blast.cgi
                             </a>
                             ). The results showed the intrinsic variation attributable to SNV-3527 (as an example) present in genomic records from the NCBI nonredundant nucleotide database including strains with genotype A3527 and others with G3527, both of which were variants inferred from our Nanopore-based sequencing. Nucleotides highlighted in yellow show the 5-mer surrounding the region of SNV-3527, revealed in detail by Sanger sequencing, as shown in
                             <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                              Figure 4H
                             </a>
                             . The C2293 genotype was retrieved from a unique matching sequence (CL06T03C08 strain), and A1254 was predominantly found in most of the P. merdae strains evaluated (data not shown).
                            </p>
                            <p id="p-64">
                             <strong>
                              Figure S4
                             </strong>
                             . FastQC-based assessment of sequences produced by nanopore device. Results from the main basecaller, Albacore v2.1.3 and the more recently released high-performance basecaller of nanopore data (Guppy v5.0.11) are shown for all three different sequencing runs generated in this study.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-60">
                             The authors thank the Bioinformatics and Biostatistics Unit from Principe Felipe Research Center (CIPF) for providing access to the cluster, co-funded by European Regional Development Funds (ERDF/FEDER) in Valencian Community 2014-2020.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-5">
                               The ordering of figures and supplementary material is updated. The title has changed, and new references are indexed. New scope was set to highlight the clinical meaning if this study.
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section glossary">
                            <h2 class="">
                             List of abbreviations
                            </h2>
                            <dl id="def-list-1">
                             <dt id="def-item-1">
                              BMI
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              body mass index
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-2">
                              FDR
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              false discovery rate
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-3">
                              FMT
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              fecal microbiota transplantation
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-4">
                              LMM
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              linear mixed model
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-5">
                              NCBI
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              National Center for Biotechnology Information
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-6">
                              MAG
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              metagenome-assembled genome
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-7">
                              PCoA
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              principal coordinate analysis
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-8">
                              PCR
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              polymerase chain reaction
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-9">
                              RDP
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              Ribosomal Database Project
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-10">
                              rrn
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              bacterial ribosome RNA operon (16S-ITS-23S)
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-11">
                              SNP
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              single-nucleotide polymorphism
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-12">
                              SNV
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              single-nucleotide variation
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-13">
                              UC
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              ulcerative colitis
                             </dd>
                            </dl>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.1" data-doi="10.1097/MCG.0000000000000046">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cammarota
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ianiro
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gasbarrini
                                  </span>
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Fecal microbiota transplantation for the treatment of Clostridium difficile infection: a systematic review
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Clin Gastroenterol
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  48
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  693
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  702
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BClin%2BGastroenterol%26rft.volume%253D48%26rft.spage%253D693%26rft_id%253Dinfo%253Adoi%252F10.1097%252FMCG.0000000000000046%26rft_id%253Dinfo%253Apmid%252F24440934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1097/MCG.0000000000000046&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24440934&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F19%2F2020.09.11.292896.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chu
                                  </span>
                                  <span class="cit-name-given-names">
                                   ND
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Crothers
                                  </span>
                                  <span class="cit-name-given-names">
                                   JW
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nguyen
                                  </span>
                                  <span class="cit-name-given-names">
                                   LTT
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kearney
                                  </span>
                                  <span class="cit-name-given-names">
                                   SM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                  <span class="cit-name-given-names">
                                   MB
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kassam
                                  </span>
                                  <span class="cit-name-given-names">
                                   Z
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Dynamic Colonization of Microbes and Their Functions after Fecal Microbiota Transplantation for Inflammatory Bowel Disease
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  mBio
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  e0097521
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DmBio%26rft.volume%253D12%26rft.spage%253De0097521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hartstra
                                  </span>
                                  <span class="cit-name-given-names">
                                   AV
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schuppel
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Imangaliyev
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schrantee
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Prodan
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Collard
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Infusion of donor feces affects the gut-brain axis in humans with metabolic syndrome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol Metab
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  42
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  101076
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol%2BMetab%26rft.volume%253D42%26rft.spage%253D101076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sokol
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Landman
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Seksik
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Berard
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Montil
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nion-Larmurier
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Fecal microbiota transplantation to maintain remission in Crohn’s disease: a pilot randomized controlled study
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Microbiome
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  12
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMicrobiome%26rft.volume%253D8%26rft.spage%253D12%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.5" data-doi="10.1053/j.gastro.2016.11.010">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ott
                                  </span>
                                  <span class="cit-name-given-names">
                                   SJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Waetzig
                                  </span>
                                  <span class="cit-name-given-names">
                                   GH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rehman
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moltzau-Anderson
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bharti
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grasis
                                  </span>
                                  <span class="cit-name-given-names">
                                   JA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Efficacy of Sterile Fecal Filtrate Transfer for Treating Patients With Clostridium difficile Infection
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Gastroenterology
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  152
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  799
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  811 e7
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGastroenterology%26rft.volume%253D152%26rft.spage%253D799%26rft_id%253Dinfo%253Adoi%252F10.1053%252Fj.gastro.2016.11.010%26rft_id%253Dinfo%253Apmid%252F27866880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1053/j.gastro.2016.11.010&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27866880&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F19%2F2020.09.11.292896.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Aggarwala
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mogno
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   Z
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Britton
                                  </span>
                                  <span class="cit-name-given-names">
                                   GJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen-Liaw
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Precise quantification of bacterial strains after fecal microbiota transplantation delineates long-term engraftment and explains outcomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Microbiol
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1309
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  18
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BMicrobiol%26rft.volume%253D6%26rft.spage%253D1309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.09.11.292896v3.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Verma
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dutta
                                  </span>
                                  <span class="cit-name-given-names">
                                   SK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Firnberg
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Phillips
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vinayek
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nair
                                  </span>
                                  <span class="cit-name-given-names">
                                   PP
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Identification and engraftment of new bacterial strains by shotgun metagenomic sequence analysis in patients with recurrent Clostridioides difficile infection before and after fecal microbiota transplantation and in healthy human subjects
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS One
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  e0251590
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D16%26rft.spage%253De0251590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wilson
                                  </span>
                                  <span class="cit-name-given-names">
                                   BC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vatanen
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jayasinghe
                                  </span>
                                  <span class="cit-name-given-names">
                                   TN
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leong
                                  </span>
                                  <span class="cit-name-given-names">
                                   KSW
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Derraik
                                  </span>
                                  <span class="cit-name-given-names">
                                   JGB
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Albert
                                  </span>
                                  <span class="cit-name-given-names">
                                   BB
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Strain engraftment competition and functional augmentation in a multi-donor fecal microbiota transplantation trial for obesity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Microbiome
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  107
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMicrobiome%26rft.volume%253D9%26rft.spage%253D107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rinott
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Youngster
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yaskolka Meir
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tsaban
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zelicha
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kaplan
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Effects of Diet-Modulated Autologous Fecal Microbiota Transplantation on Weight Regain
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Gastroenterology
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  160
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  158
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  73 e10
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGastroenterology%26rft.volume%253D160%26rft.spage%253D158%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F19%2F2020.09.11.292896.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yu
                                  </span>
                                  <span class="cit-name-given-names">
                                   EW
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gao
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stastka
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cheney
                                  </span>
                                  <span class="cit-name-given-names">
                                   MC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mahabamunuge
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Torres Soto
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Fecal microbiota transplantation for the improvement of metabolism in obesity: The FMT-TRIM double-blind placebo-controlled pilot trial
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Med
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  e1003051
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BMed%26rft.volume%253D17%26rft.spage%253De1003051%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F19%2F2020.09.11.292896.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.11" data-doi="10.1186/s13742-016-0111-z">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Benitez-Paez
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Portune
                                  </span>
                                  <span class="cit-name-given-names">
                                   KJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sanz
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Species-level resolution of 16S rRNA gene amplicons sequenced through the MinION portable nanopore sequencer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Gigascience
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  4
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D5%26rft.spage%253D4%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13742-016-0111-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13742-016-0111-z&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cusco
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Catozzi
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vines
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sanchez
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Francino
                                  </span>
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Microbiota profiling with long amplicons using Nanopore sequencing: full-length 16S rRNA gene and whole rrn operon
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  F1000Res
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1755
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DF1000Res%26rft.volume%253D7%26rft.spage%253D1755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.09.11.292896v3.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sakai
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tarumoto
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kodana
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ashikawa
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Imai
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kawamura
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  An identification protocol for ESBL-producing Gram-negative bacteria bloodstream infections using a MinION nanopore sequencer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Med Microbiol
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  68
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1219
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  26
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BMed%2BMicrobiol%26rft.volume%253D68%26rft.spage%253D1219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.09.11.292896v3.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shin
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shin
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ko
                                  </span>
                                  <span class="cit-name-given-names">
                                   SR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Oh
                                  </span>
                                  <span class="cit-name-given-names">
                                   HS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ahn
                                  </span>
                                  <span class="cit-name-given-names">
                                   CY
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Elucidation of the bacterial communities associated with the harmful microalgae Alexandrium tamarense and Cochlodinium polykrikoides using nanopore sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci Rep
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  5323
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D8%26rft.spage%253D5323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.15" data-doi="10.1038/srep29681">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shin
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Go
                                  </span>
                                  <span class="cit-name-given-names">
                                   MJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                  <span class="cit-name-given-names">
                                   SY
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                  <span class="cit-name-given-names">
                                   SC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                  <span class="cit-name-given-names">
                                   CH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Analysis of the mouse gut microbiome using full-length 16S rRNA amplicon sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci Rep
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  29681
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D6%26rft.spage%253D29681%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep29681%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/srep29681&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F19%2F2020.09.11.292896.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.16" data-doi="10.1093/gigascience/giw015">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Benitez-Paez
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sanz
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Multi-locus and long amplicon sequencing approach to study microbial diversity at species level using the MinION portable nanopore sequencer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Gigascience
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  12
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgiw015%26rft_id%253Dinfo%253Apmid%252F29092041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/gigascience/giw015&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29092041&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F19%2F2020.09.11.292896.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.17" data-doi="10.1186/s40168-017-0336-9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kerkhof
                                  </span>
                                  <span class="cit-name-given-names">
                                   LJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dillon
                                  </span>
                                  <span class="cit-name-given-names">
                                   KP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Haggblom
                                  </span>
                                  <span class="cit-name-given-names">
                                   MM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McGuinness
                                  </span>
                                  <span class="cit-name-given-names">
                                   LR
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Profiling bacterial communities by MinION sequencing of ribosomal operons
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Microbiome
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  116
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMicrobiome%26rft.volume%253D5%26rft.spage%253D116%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs40168-017-0336-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s40168-017-0336-9&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.18" data-doi="10.1016/j.csbj.2021.02.020">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ciuffreda
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rodriguez-Perez
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Flores
                                  </span>
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Nanopore sequencing and its application to the study of microbial communities
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Comput Struct Biotechnol J
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1497
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  511
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DComput%2BStruct%2BBiotechnol%2BJ%26rft.volume%253D19%26rft.spage%253D1497%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.csbj.2021.02.020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.csbj.2021.02.020&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.09.11.292896v3.19">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kerkhof
                                  </span>
                                  <span class="cit-name-given-names">
                                   LJ
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Is Oxford Nanopore sequencing ready for analyzing complex microbiomes?
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  FEMS Microbiol Ecol
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  97
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.20">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pavlovic
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cavalieri
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mastromei
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pangallo
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Perito
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marvasi
                                  </span>
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  MinION technology for microbiome sequencing applications for the conservation of cultural heritage
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Microbiol Res
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  247
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  126727
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMicrobiol%2BRes%26rft.volume%253D247%26rft.spage%253D126727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.21" data-doi="10.3389/fmicb.2019.00620">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Peker
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Garcia-Croes
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dijkhuizen
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wiersma
                                  </span>
                                  <span class="cit-name-given-names">
                                   HH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   van Zanten
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wisselink
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  A Comparison of Three Different Bioinformatics Analyses of the 16S-23S rRNA Encoding Region for Bacterial Identification
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Front Microbiol
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  620
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFront%2BMicrobiol%26rft.volume%253D10%26rft.spage%253D620%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffmicb.2019.00620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.3389/fmicb.2019.00620&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Krehenwinkel
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pomerantz
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Henderson
                                  </span>
                                  <span class="cit-name-given-names">
                                   JB
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kennedy
                                  </span>
                                  <span class="cit-name-given-names">
                                   SR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lim
                                  </span>
                                  <span class="cit-name-given-names">
                                   JY
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Swamy
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Nanopore sequencing of long ribosomal DNA amplicons enables portable and simple biodiversity assessments with high phylogenetic resolution across broad taxonomic scale
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Gigascience
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brewer
                                  </span>
                                  <span class="cit-name-given-names">
                                   TE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Albertsen
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Edwards
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kirkegaard
                                  </span>
                                  <span class="cit-name-given-names">
                                   RH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rocha
                                  </span>
                                  <span class="cit-name-given-names">
                                   EPC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fierer
                                  </span>
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Unlinked rRNA genes are widespread among bacteria and archaea
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  ISME J
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  597
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  608
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DISME%2BJ%26rft.volume%253D14%26rft.spage%253D597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.24" data-doi="10.1038/s41598-020-65740-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dowden
                                  </span>
                                  <span class="cit-name-given-names">
                                   RA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McGuinness
                                  </span>
                                  <span class="cit-name-given-names">
                                   LR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wisniewski
                                  </span>
                                  <span class="cit-name-given-names">
                                   PJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Campbell
                                  </span>
                                  <span class="cit-name-given-names">
                                   SC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guers
                                  </span>
                                  <span class="cit-name-given-names">
                                   JJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Oydanich
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Host genotype and exercise exhibit species-level selection for members of the gut bacterial communities in the mouse digestive system
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci Rep
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  8984
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D10%26rft.spage%253D8984%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-020-65740-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41598-020-65740-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               25.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.25">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ibironke
                                  </span>
                                  <span class="cit-name-given-names">
                                   O
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McGuinness
                                  </span>
                                  <span class="cit-name-given-names">
                                   LR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lu
                                  </span>
                                  <span class="cit-name-given-names">
                                   SE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hussain
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weisel
                                  </span>
                                  <span class="cit-name-given-names">
                                   CP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Species-level evaluation of the human respiratory microbiome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Gigascience
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               26.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.26" data-doi="10.1371/journal.pcbi.1006102">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gibbons
                                  </span>
                                  <span class="cit-name-given-names">
                                   SM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Duvallet
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alm
                                  </span>
                                  <span class="cit-name-given-names">
                                   EJ
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Correcting for batch effects in case-control microbiome studies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Comput Biol
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  e1006102
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput%2BBiol%26rft.volume%253D14%26rft.spage%253De1006102%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1006102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1006102&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               27.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.27" data-doi="10.1371/journal.pone.0027310">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schloss
                                  </span>
                                  <span class="cit-name-given-names">
                                   PD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gevers
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Westcott
                                  </span>
                                  <span class="cit-name-given-names">
                                   SL
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Reducing the effects of PCR amplification and sequencing artifacts on 16S rRNA-based studies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS One
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  e27310
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSchloss%26rft.auinit1%253DP.%2BD.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253De27310%26rft.epage%253De27310%26rft.atitle%253DReducing%2Bthe%2Beffects%2Bof%2BPCR%2Bamplification%2Band%2Bsequencing%2Bartifacts%2Bon%2B16S%2BrRNA-based%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0027310%26rft_id%253Dinfo%253Apmid%252F22194782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0027310&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22194782&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F19%2F2020.09.11.292896.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               28.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.28">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Paramsothy
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nielsen
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kamm
                                  </span>
                                  <span class="cit-name-given-names">
                                   MA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Deshpande
                                  </span>
                                  <span class="cit-name-given-names">
                                   NP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Faith
                                  </span>
                                  <span class="cit-name-given-names">
                                   JJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Clemente
                                  </span>
                                  <span class="cit-name-given-names">
                                   JC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Specific Bacteria and Metabolites Associated With Response to Fecal Microbiota Transplantation in Patients With Ulcerative Colitis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Gastroenterology
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  156
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1440
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  54 e2
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGastroenterology%26rft.volume%253D156%26rft.spage%253D1440%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F19%2F2020.09.11.292896.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               29.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.29" data-doi="10.1093/cid/cix252">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bilinski
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grzesiowski
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sorensen
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Madry
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Muszynski
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Robak
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Fecal Microbiota Transplantation in Patients With Blood Disorders Inhibits Gut Colonization With Antibiotic-Resistant Bacteria: Results of a Prospective, Single-Center Study
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Clin Infect Dis
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  65
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  364
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  70
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DClin%2BInfect%2BDis%26rft.volume%253D65%26rft.spage%253D364%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcid%252Fcix252%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/cid/cix252&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F19%2F2020.09.11.292896.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               30.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Benitez-Paez
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hess
                                  </span>
                                  <span class="cit-name-given-names">
                                   AL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Krautbauer
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liebisch
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Christensen
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hjorth
                                  </span>
                                  <span class="cit-name-given-names">
                                   MF
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Sex, Food, and the Gut Microbiota: Disparate Response to Caloric Restriction Diet with Fiber Supplementation in Women and Men
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol Nutr Food Res
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  65
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  e2000996
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol%2BNutr%2BFood%2BRes%26rft.volume%253D65%26rft.spage%253De2000996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               31.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.31">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   El-Salhy
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Casen
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Valeur
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hausken
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hatlebakk
                                  </span>
                                  <span class="cit-name-given-names">
                                   JG
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Responses to faecal microbiota transplantation in female and male patients with irritable bowel syndrome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  World J Gastroenterol
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  2219
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  37
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DWorld%2BJ%2BGastroenterol%26rft.volume%253D27%26rft.spage%253D2219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               32.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.32" data-doi="10.1101/gr.113985.110">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kielbasa
                                  </span>
                                  <span class="cit-name-given-names">
                                   SM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wan
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sato
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Horton
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Frith
                                  </span>
                                  <span class="cit-name-given-names">
                                   MC
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Adaptive seeds tame genomic sequence comparison
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Res
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  21
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  487
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  93
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.113985.110%26rft_id%253Dinfo%253Apmid%252F21209072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzMvNDg3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDUvMTkvMjAyMC4wOS4xMS4yOTI4OTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               33.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.33" data-doi="10.1093/molbev/mst010">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Katoh
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Standley
                                  </span>
                                  <span class="cit-name-given-names">
                                   DM
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  MAFFT multiple sequence alignment software version 7: improvements in performance and usability
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol Biol Evol
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  772
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  80
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23329690&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F19%2F2020.09.11.292896.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000317002300004&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               34.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.34" data-doi="10.1371/journal.pcbi.1002195">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eddy
                                  </span>
                                  <span class="cit-name-given-names">
                                   SR
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Accelerated Profile HMM Searches
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Comput Biol
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  e1002195
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DEddy%26rft.auinit1%253DS.%2BR.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002195%26rft.epage%253De1002195%26rft.atitle%253DAccelerated%2BProfile%2BHMM%2BSearches.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002195%26rft_id%253Dinfo%253Apmid%252F22039361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002195&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22039361&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F19%2F2020.09.11.292896.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               35.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.35" data-doi="10.1093/nar/gks1219">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Quast
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pruesse
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yilmaz
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gerken
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schweer
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yarza
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The SILVA ribosomal RNA gene database project: improved data processing and web-based tools
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  41
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  D590
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1219%26rft_id%253Dinfo%253Apmid%252F23193283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gks1219&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23193283&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F19%2F2020.09.11.292896.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000312893300084&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               36.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.36" data-doi="10.1093/bioinformatics/bts252">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pruesse
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Peplies
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Glockner
                                  </span>
                                  <span class="cit-name-given-names">
                                   FO
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  SINA: accurate high-throughput multiple sequence alignment of ribosomal RNA genes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1823
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  9
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts252%26rft_id%253Dinfo%253Apmid%252F22556368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts252&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22556368&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F19%2F2020.09.11.292896.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000306136100004&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               37.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.11.292896v3.37" data-doi="10.1038/nmeth.f.303">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Caporaso
                                  </span>
                                  <span class="cit-name-given-names">
                                   JG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kuczynski
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stombaugh
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bittinger
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bushman
                                  </span>
                                  <span class="cit-name-given-names">
                                   FD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Costello
                                  </span>
                                  <span class="cit-name-given-names">
                                   EK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  QIIME allows analysis of high-throughput community sequencing data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Methods
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  335
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DCaporaso%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253D335%26rft.epage%253D336%26rft.atitle%253DQIIME%2Ballows%2Banalysis%2Bof%2Bhigh-throughput%2Bcommunity%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.f.303%26rft_id%253Dinfo%253Apmid%252F20383131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nmeth.f.303&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20383131&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F19%2F2020.09.11.292896.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000277175100003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.09.11.292896v3" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.12.17.473188v2" title="Stimulus-induced changes in 1/f-like background activity in EEG" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.05.18.492498v1" title="Precision motor timing via scalar input fluctuations" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted May 19, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.09.11.292896v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.09.11.292896v3.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.09.11.292896v3.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.09.11.292896v3" class="active" data-icon-position="" data-hide-link-title="0">
                                   Species- and strain-level assessment using rrn long-amplicons suggests donor’s influence on gut microbial transference via fecal transplants in metabolic syndrome subjects
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2552735"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-06TxCrAP5Rag8BjOjxakh_AB87fu67Sx-nxQUesOYQE"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484479194"/>
                                   <input type="hidden" name="captcha_token" value="1a030d5fd11202c6ea73f5d80b086e7d"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2552735--3" data-node-nid="2552735" id="node-2552735--4210699193" data-pisa="biorxiv;2020.09.11.292896v3" data-pisa-master="biorxiv;2020.09.11.292896" data-seqnum="2552735" data-apath="/biorxiv/early/2022/05/19/2020.09.11.292896.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Species- and strain-level assessment using rrn long-amplicons suggests donor’s influence on gut microbial transference via fecal transplants in metabolic syndrome subjects
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Alfonso
                                  </span>
                                  <span class="nlm-surname">
                                   Benítez-Páez
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Annick V.
                                  </span>
                                  <span class="nlm-surname">
                                   Hartstra
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Max
                                  </span>
                                  <span class="nlm-surname">
                                   Nieuwdorp
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Yolanda
                                  </span>
                                  <span class="nlm-surname">
                                   Sanz
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.09.11.292896;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.09.11.292896
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.09.11.292896v3" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.09.11.292896v3&title=Species-%20and%20strain-level%20assessment%20using%20rrn%20long-amplicons%20suggests%20donor%E2%80%99s%20influence%20on%20gut%20microbial%20transference%20via%20fecal%20transplants%20in%20metabolic%20syndrome%20subjects" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.11.292896v3&title=Species-%20and%20strain-level%20assessment%20using%20rrn%20long-amplicons%20suggests%20donor%E2%80%99s%20influence%20on%20gut%20microbial%20transference%20via%20fecal%20transplants%20in%20metabolic%20syndrome%20subjects" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.11.292896v3&text=Species-%20and%20strain-level%20assessment%20using%20rrn%20long-amplicons%20suggests%20donor%E2%80%99s%20influence%20on%20gut%20microbial%20transference%20via%20fecal%20transplants%20in%20metabolic%20syndrome%20subjects" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.09.11.292896v3&t=Species-%20and%20strain-level%20assessment%20using%20rrn%20long-amplicons%20suggests%20donor%E2%80%99s%20influence%20on%20gut%20microbial%20transference%20via%20fecal%20transplants%20in%20metabolic%20syndrome%20subjects" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.09.11.292896v3&title=Species-%20and%20strain-level%20assessment%20using%20rrn%20long-amplicons%20suggests%20donor%E2%80%99s%20influence%20on%20gut%20microbial%20transference%20via%20fecal%20transplants%20in%20metabolic%20syndrome%20subjects" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.09.11.292896v3&title=Species-%20and%20strain-level%20assessment%20using%20rrn%20long-amplicons%20suggests%20donor%E2%80%99s%20influence%20on%20gut%20microbial%20transference%20via%20fecal%20transplants%20in%20metabolic%20syndrome%20subjects&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.11.292896v3&title=Species-%20and%20strain-level%20assessment%20using%20rrn%20long-amplicons%20suggests%20donor%E2%80%99s%20influence%20on%20gut%20microbial%20transference%20via%20fecal%20transplants%20in%20metabolic%20syndrome%20subjects" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2552735--5" data-node-nid="2552735" id="citation-node-2552735--61382902292" data-pisa="biorxiv;2020.09.11.292896v3" data-pisa-master="biorxiv;2020.09.11.292896" data-seqnum="2552735" data-apath="/biorxiv/early/2022/05/19/2020.09.11.292896.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Species- and strain-level assessment using rrn long-amplicons suggests donor’s influence on gut microbial transference via fecal transplants in metabolic syndrome subjects
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Alfonso
                                      </span>
                                      <span class="nlm-surname">
                                       Benítez-Páez
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Annick V.
                                      </span>
                                      <span class="nlm-surname">
                                       Hartstra
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Max
                                      </span>
                                      <span class="nlm-surname">
                                       Nieuwdorp
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Yolanda
                                      </span>
                                      <span class="nlm-surname">
                                       Sanz
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.09.11.292896;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.09.11.292896
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Species-%20and%20strain-level%20assessment%20using%20rrn%20long-amplicons%20suggests%20donor%E2%80%99s%20influence%20on%20gut%20microbial%20transference%20via%20fecal%20transplants%20in%20metabolic%20syndrome%20subjects&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Ben%C3%ADtez-P%C3%A1ez%2C+Alfonso&rft.au=Hartstra%2C+Annick+V.&rft.au=Nieuwdorp%2C+Max&rft.au=Sanz%2C+Yolanda">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2552735/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2552735/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2552735/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2552735/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2552735/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2552735/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2552735/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2552735/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2552735/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2552735/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2552735/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2552735/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.11.292896v3&count=horizontal&via=&text=Species-%20and%20strain-level%20assessment%20using%20rrn%20long-amplicons%20suggests%20donor%E2%80%99s%20influence%20on%20gut%20microbial%20transference%20via%20fecal%20transplants%20in%20metabolic%20syndrome%20subjects&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.09.11.292896v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.09.11.292896v3&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.09.11.292896v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Microbiology
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23507)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11824)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11440)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15189)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47703)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
