<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.09.29.319285v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.09.29.319285v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.09.29.319285v2.ppt"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/02/18/2020.09.29.319285.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.09.29.319285v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Generated randomly and selected functionally? The nature of enterovirus recombination"/>
  <meta name="DC.Identifier" content="10.1101/2020.09.29.319285"/>
  <meta name="DC.Date" content="2022-02-18"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Genetic recombination in RNA viruses is an important evolutionary mechanism. It contributes to population diversity, host/tissue adaptation and compromises vaccine efficacy. Both the molecular mechanism and initial products of recombination are relatively poorly understood. We used an established poliovirus-based in vitro recombination assay to investigate the roles of sequence identity and RNA structure, both implicated or inferred from analysis of circulating recombinant viruses, in the process. In addition, we used next generation sequencing to investigate the early products of recombination after cellular co-infection with different poliovirus serotypes. In independent studies we find no evidence for a role for RNA identity or structure in determining recombination junctions location. Instead, genome function and fitness are of greater importance in determining the identity of recombinant progeny. These studies provide further insights into this important evolutionary mechanism and emphasise the critical nature of the selection process on a mixed virus population.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Fadi G. Alnaji"/>
  <meta name="DC.Contributor" content="Kirsten Bentley"/>
  <meta name="DC.Contributor" content="Ashley Pearson"/>
  <meta name="DC.Contributor" content="Andrew Woodman"/>
  <meta name="DC.Contributor" content="Jonathan Moore"/>
  <meta name="DC.Contributor" content="Helen Fox"/>
  <meta name="DC.Contributor" content="Andrew MacAdam"/>
  <meta name="DC.Contributor" content="David J. Evans"/>
  <meta name="article:published_time" content="2022-02-18"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Generated randomly and selected functionally? The nature of enterovirus recombination"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Genetic recombination in RNA viruses is an important evolutionary mechanism. It contributes to population diversity, host/tissue adaptation and compromises vaccine efficacy. Both the molecular mechanism and initial products of recombination are relatively poorly understood. We used an established poliovirus-based <i>in vitro</i> recombination assay to investigate the roles of sequence identity and RNA structure, both implicated or inferred from analysis of circulating recombinant viruses, in the process. In addition, we used next generation sequencing to investigate the early products of recombination after cellular co-infection with different poliovirus serotypes. In independent studies we find no evidence for a role for RNA identity or structure in determining recombination junctions location. Instead, genome function and fitness are of greater importance in determining the identity of recombinant progeny. These studies provide further insights into this important evolutionary mechanism and emphasise the critical nature of the selection process on a mixed virus population.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.09.29.319285v2"/>
  <meta name="citation_id" content="2020.09.29.319285v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.09.29.319285v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.09.29.319285v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.09.29.319285v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.09.29.319285"/>
  <meta name="citation_num_pages" content="23"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.09.29.319285"/>
  <meta name="citation_author" content="Fadi G. Alnaji"/>
  <meta name="citation_author_institution" content="Department of Microbiology, University of Illinois Champaign-Urbana"/>
  <meta name="citation_author_email" content="falnaji@illinois.edu"/>
  <meta name="citation_author_email" content="bentleyk@cardiff.ac.uk"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2893-9832"/>
  <meta name="citation_author" content="Kirsten Bentley"/>
  <meta name="citation_author_institution" content="Biomedical Sciences Research Complex and School of Biology, University of St Andrew"/>
  <meta name="citation_author_email" content="falnaji@illinois.edu"/>
  <meta name="citation_author_email" content="bentleyk@cardiff.ac.uk"/>
  <meta name="citation_author" content="Ashley Pearson"/>
  <meta name="citation_author_institution" content="Biomedical Sciences Research Complex and School of Biology, University of St Andrew"/>
  <meta name="citation_author" content="Andrew Woodman"/>
  <meta name="citation_author_institution" content="Department of Biochemistry & Molecular Biology, 201 Althouse Lab"/>
  <meta name="citation_author" content="Jonathan Moore"/>
  <meta name="citation_author_institution" content="John Innes Centre, Cell and Developmental Biology"/>
  <meta name="citation_author" content="Helen Fox"/>
  <meta name="citation_author_institution" content="Division of Virology, National Institute for Biological Standards and Control"/>
  <meta name="citation_author" content="Andrew MacAdam"/>
  <meta name="citation_author_institution" content="Division of Virology, National Institute for Biological Standards and Control"/>
  <meta name="citation_author" content="David J. Evans"/>
  <meta name="citation_author_institution" content="Biomedical Sciences Research Complex and School of Biology, University of St Andrew"/>
  <meta name="citation_reference" content="Hahn, C.S., et al., Western equine encephalitis virus is a recombinant virus. Proc Natl Acad Sci U S A, 1988. 85(16): p. 5997–6001."/>
  <meta name="citation_reference" content="Ge, X.Y., et al., Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor. Nature, 2013. 503(7477): p. 535–8."/>
  <meta name="citation_reference" content="Li, K.S., et al., Genesis of a highly pathogenic and potentially pandemic H5N1 influenza virus in eastern Asia. Nature, 2004. 430(6996): p. 209–13."/>
  <meta name="citation_reference" content="Smith, G.J., et al., Origins and evolutionary genomics of the 2009 swine-origin H1N1 influenza A epidemic. Nature, 2009. 459(7250): p. 1122–5."/>
  <meta name="citation_reference" content="Gmyl, A.P., et al., Nonreplicative RNA recombination in poliovirus. J Virol, 1999. 73(11): p. 8958–65."/>
  <meta name="citation_reference" content="Gmyl, A.P., et al., Nonreplicative homologous RNA recombination: promiscuous joining of RNA pieces? Rna, 2003. 9(10): p. 1221–31."/>
  <meta name="citation_reference" content="King, A.M., Preferred sites of recombination in poliovirus RNA: an analysis of 40 intertypic cross-over sequences. Nucleic Acids Res, 1988. 16(24): p. 11705–23."/>
  <meta name="citation_reference" content="Kirkegaard, K. and D. Baltimore, The mechanism of RNA recombination in poliovirus. Cell, 1986. 47(3): p. 433–43."/>
  <meta name="citation_reference" content="Woodman, A., et al., Biochemical and genetic analysis of the role of the viral polymerase in enterovirus recombination. Nucleic Acids Res, 2016. 44(14): p. 6883–95."/>
  <meta name="citation_reference" content="Nagy, P.D. and J.J. Bujarski, Efficient system of homologous RNA recombination in brome mosaic virus: sequence and structure requirements and accuracy of crossovers. J Virol, 1995. 69(1): p. 131–40."/>
  <meta name="citation_reference" content="Sergiescu, D., A. Aubert-Combiescu, and R. Crainic, Recombination between guanidine-resistant and dextran sulfate-resistant mutants of type 1 poliovirus. J Virol, 1969. 3(3): p. 326–30."/>
  <meta name="citation_reference" content="White, K.A. and T.J. Morris, RNA determinants of junction site selection in RNA virus recombinants and defective interfering RNAs. RNA, 1995. 1(10): p. 1029–40."/>
  <meta name="citation_reference" content="Nagy, P.D., C. Zhang, and A.E. Simon, Dissecting RNA recombination in vitro: role of RNA sequences and the viral replicase. EMBO J, 1998. 17(8): p. 2392–403."/>
  <meta name="citation_reference" content="Carpenter, C.D., et al., Involvement of a stem-loop structure in the location of junction sites in viral RNA recombination. J Mol Biol, 1995. 245(5): p. 608–22."/>
  <meta name="citation_reference" content="Bruyere, A., et al., Frequent homologous recombination events between molecules of one RNA component in a multipartite RNA virus. J Virol, 2000. 74(9): p. 4214–9."/>
  <meta name="citation_reference" content="Bujarski, J.J. and P. Kaesberg, Genetic recombination between RNA components of a multipartite plant virus. Nature, 1986. 321(6069): p. 528–31."/>
  <meta name="citation_reference" content="Rowe, C.L., et al., Generation of coronavirus spike deletion variants by high-frequency recombination at regions of predicted RNA secondary structure. J Virol, 1997. 71(8): p. 6183–90."/>
  <meta name="citation_reference" content="Lowry, K., et al., Recombination in enteroviruses is a biphasic replicative process involving the generation of greater-than genome length ‘imprecise’ intermediates. PLoS Pathog, 2014. 10(6): p. e1004191."/>
  <meta name="citation_reference" content="Egger, D. and K. Bienz, Intracellular location and translocation of silent and active poliovirus replication complexes. J Gen Virol, 2005. 86(Pt 3): p. 707–18."/>
  <meta name="citation_reference" content="Bowtie 2: Manual. 2016; Available from: http://bowtie-bio.sourceforge.net/bowtie2/manual.shtml#bowtie2-options-score-min."/>
  <meta name="citation_reference" content="Routh, A. and J.E. Johnson, Discovery of functional genomic motifs in viruses with ViReMa-a Virus Recombination Mapper-for analysis of next-generation sequencing data. Nucleic Acids Res, 2014. 42(2): p. e11."/>
  <meta name="citation_reference" content="Alnaji, F.G., et al., Sequencing Framework for the Sensitive Detection and Precise Mapping of Defective Interfering Particle-Associated Deletions across Influenza A and B Viruses. J Virol, 2019. 93(11)."/>
  <meta name="citation_reference" content="Figlerowicz, M., Role of RNA structure in non-homologous recombination between genomic molecules of brome mosaic virus. Nucleic Acids Res, 2000. 28(8): p. 1714–23."/>
  <meta name="citation_reference" content="Cascone, P.J., T.F. Haydar, and A.E. Simon, Sequences and structures required for recombination between virus-associated RNAs. Science, 1993. 260(5109): p. 801–5."/>
  <meta name="citation_reference" content="Simmonds, P., A. Tuplin, and D.J. Evans, Detection of genome-scale ordered RNA structure (GORS) in genomes of positive-stranded RNA viruses: Implications for virus evolution and host persistence. RNA, 2004. 10(9): p. 1337–51."/>
  <meta name="citation_reference" content="Bentley, K., et al., Imprecise recombinant viruses evolve via a fitness-driven, iterative process of polymerase template-switching events. PLoS Pathog, 2021. 17(8): p. e1009676."/>
  <meta name="citation_reference" content="Simon-Loriere, E. and E.C. Holmes, Why do RNA viruses recombine? Nat Rev Microbiol, 2011. 9(8): p. 617–26."/>
  <meta name="citation_reference" content="Worobey, M. and E.C. Holmes, Evolutionary aspects of recombination in RNA viruses. J Gen Virol, 1999. 80 (Pt 10): p. 2535–43."/>
  <meta name="citation_reference" content="Duffy, S., Why are RNA virus mutation rates so damn high? PLoS Biol, 2018. 16(8): p. e3000003."/>
  <meta name="citation_reference" content="Kempf, B.J., et al., Picornavirus RNA Recombination Counteracts Error Catastrophe. J Virol, 2019. 93(14)."/>
  <meta name="citation_reference" content="Crotty, S., C.E. Cameron, and R. Andino, RNA virus error catastrophe: direct molecular test by using ribavirin. Proc Natl Acad Sci U S A, 2001. 98(12): p. 6895–900."/>
  <meta name="citation_reference" content="Bentley, K. and D.J. Evans, Mechanisms and consequences of positive-strand RNA virus recombination. J Gen Virol, 2018. 99(10): p. 1345–1356."/>
  <meta name="citation_reference" content="Muslin, C., et al., Recombination in Enteroviruses, a Multi-Step Modular Evolutionary Process. Viruses, 2019. 11(9)."/>
  <meta name="citation_reference" content="Simmonds, P., SSE: a nucleotide and amino acid sequence analysis platform. BMC Res Notes, 2012. 5: p. 50."/>
  <meta name="twitter:title" content="Generated randomly and selected functionally? The nature of enterovirus recombination"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Genetic recombination in RNA viruses is an important evolutionary mechanism. It contributes to population diversity, host/tissue adaptation and compromises vaccine efficacy. Both the molecular mechanism and initial products of recombination are relatively poorly understood. We used an established poliovirus-based in vitro recombination assay to investigate the roles of sequence identity and RNA structure, both implicated or inferred from analysis of circulating recombinant viruses, in the process. In addition, we used next generation sequencing to investigate the early products of recombination after cellular co-infection with different poliovirus serotypes. In independent studies we find no evidence for a role for RNA identity or structure in determining recombination junctions location. Instead, genome function and fitness are of greater importance in determining the identity of recombinant progeny. These studies provide further insights into this important evolutionary mechanism and emphasise the critical nature of the selection process on a mixed virus population.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Generated randomly and selected functionally? The nature of enterovirus recombination"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.09.29.319285v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Genetic recombination in RNA viruses is an important evolutionary mechanism. It contributes to population diversity, host/tissue adaptation and compromises vaccine efficacy. Both the molecular mechanism and initial products of recombination are relatively poorly understood. We used an established poliovirus-based in vitro recombination assay to investigate the roles of sequence identity and RNA structure, both implicated or inferred from analysis of circulating recombinant viruses, in the process. In addition, we used next generation sequencing to investigate the early products of recombination after cellular co-infection with different poliovirus serotypes. In independent studies we find no evidence for a role for RNA identity or structure in determining recombination junctions location. Instead, genome function and fitness are of greater importance in determining the identity of recombinant progeny. These studies provide further insights into this important evolutionary mechanism and emphasise the critical nature of the selection process on a mixed virus population.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-02-18"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.09.29.319285v2"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2407740"/>
  <title>
   Generated randomly and selected functionally? The nature of enterovirus recombination | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"eghGbcK3XHpMdg__znm9VvaLnRA8zP4OApEsWLA0cVk"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2407740","apath":"\/biorxiv\/early\/2022\/02\/18\/2020.09.29.319285.atom","pisa":"biorxiv;2020.09.29.319285v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.09.29.319285v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.09.29.319285v2","title":"Generated randomly and selected functionally? The nature of enterovirus recombination","identifier":"node\/2407740"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FAtlb2aKv8cqfKx7guAk8OrDDEQDIdT2GDB0DVqeiy4z3RSp8Cwoal\\\\\\\/5PCxwU\\\\\\\/CZjWAovwz4XRZ8cgu95hUcVmkCS+HgGRwh0Yue4daa\\\\\\\/FYlqpnCFNQ90bTKXP4ZeI9MlfPYfkPXL5IlOJKW0PvCgaqaQCP48Z+s8EWn8LeRT2EJRRhqQym8x24LBC2lMHsgf25WkDAsKI0WiWNqNz6oyp5Oeroux4XIIpGq4IGG6SmwGBbEGFrqOiGx3f9W80nYNr2EgfI3jwpgLNb++3CoA+8nPLSwDczfM2ZRXKOWXKuj0+IeR9hBHvV3J4EWliq1th1jxYDDZQb2A95jcbWwgCsGiK6jSgjKmr4UwSBi23i8q5ZSnBOHhwjQjI+Hm\\\\\\\/z6G6kUa9RPNPD4dLswiRTlOe8vzXjtnnAsPgJ0J1QSj3SHkHSP3w7vVfjoFRN54FqnigdfajGQXpndvy6eZcXuyAWDgjlyldVOmeScFbj50Z9oiFHiIlfsfBMSW\\\\\\\/vOYyYIaQu4etGQieFAAPmbO\\\\\\\/t7Z+blvuZaf0kzg6d8lZK8WaaD0W50Ln3b6WF3Y+sIuWklXdLXR\\\\\\\/fwCbsys0tW4EBO3MT\\\\\\\/kHafsd4TAcQWt+jJBVxCOzr5Ck1QnBDFagWWg7RW0G4Tw+gfpxE0W+Y9Gg5CIUw42aDBO2FaZywQnmrt6cvO1WHr707ndxwtdX3wT\\\\\\\/QU6LBZKyI8ZB\\\\\\\/X1hVoQsCJxIU\\\\\\\/GdBaD0quKZlTeYiWFY81fYzCoRxSGxsXD7geQCI5zfHmovsLFSWf4exBerVL6+x9By8grMyUOl9a7ugBSNIBeouQ5169VhuPdvtgVMolnSav5FVddRSitC9npEnzUPPT7HJtyouYZJelY4NmN96dayGmaTRyOZqaQAyPZi5TucPllqLHXWtdS2EDLjqBV0ybKMFdunYWylWHI000GBinsDiC90ve7GAOPYQxpiUNf\\\\\\\/R8VfQCjeSqKWTPhlpE3Ay9psJa92lKnqA2YoVzqHyCf0ulwq9pW1AY8tDieKm4b2IOjux3Vi12r5xkvb8Lz7AezC6W6oc+aSkq2poVd+QQQCsLqrx7u4\\\\\\\/nKQR5G82lPBmbZPrjdA1WqlKHgEwt++alNKTR3Rfr0M8qAMa5a6pnqw2rbc3o3zvtj+e1eAutdQS1Wolr6Ng2V1HOC1T5ieAPraTCNoZ6\\\\\\\/f9UutWNtBymE\\\\\\\/a3SZTVDRwVgVnvwcmzIouApQZdjy8445dCG5stzy2e0d+lvGUPmqihRjz0cDVDcDZwbjnPipQsyN2GOCdcL8gqsQ9ZBBjdBcPYwxxETMj2jXs6r6gLjdd4Jt4CZH2HP9BJQQVky7WOayOGTvP4IIRb2jMOeYQ\\\\\\\/YQ+5kERFkQhkvDxqA\\\\\\\/ow9sKC52EhoDM=\\\u0022,\\\u0022iv\\\u0022:\\\u0022c+WZREZs33Q3+1Szk4cmDA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022cafa234166edbc22b7b78c7cc25d23a665a58f1941207a13089da61967c713c7\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.09.29.319285v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2407740 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.09.29.319285v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_2124230142">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_2124230142" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1207163871" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-6cowv8-PwQ6J__gC6dZd9jegA9gzL4RoHlUCqWeQKu8"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2407740" data-node-nid="2407740" id="node-2407740--2817508321" data-pisa="biorxiv;2020.09.29.319285v2" data-pisa-master="biorxiv;2020.09.29.319285" data-apath="/biorxiv/early/2022/02/18/2020.09.29.319285.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Generated randomly and selected functionally? The nature of enterovirus recombination
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0002-2893-9832" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Fadi G.
                      </span>
                      <span class="nlm-surname">
                       Alnaji
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Kirsten
                      </span>
                      <span class="nlm-surname">
                       Bentley
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Ashley
                      </span>
                      <span class="nlm-surname">
                       Pearson
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Andrew
                      </span>
                      <span class="nlm-surname">
                       Woodman
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="4">
                      <span class="nlm-given-names">
                       Jonathan
                      </span>
                      <span class="nlm-surname">
                       Moore
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="5">
                      <span class="nlm-given-names">
                       Helen
                      </span>
                      <span class="nlm-surname">
                       Fox
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="6">
                      <span class="nlm-given-names">
                       Andrew
                      </span>
                      <span class="nlm-surname">
                       MacAdam
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="7">
                      <span class="nlm-given-names">
                       David J.
                      </span>
                      <span class="nlm-surname">
                       Evans
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.09.29.319285
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2407740--2817508321" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Fadi G. Alnaji
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Microbiology, University of Illinois Champaign-Urbana
                       </span>
                       , Champaign-Urbana, 601 E. John St, Illinois, 61801,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Fadi%2BG.%2BAlnaji%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Alnaji%20FG&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AFadi%2BG.%2BAlnaji%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0002-2893-9832" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Fadi G. Alnaji
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:falnaji@illinois.edu" class="" data-icon-position="" data-hide-link-title="0">
                        falnaji@illinois.edu
                       </a>
                       <a href="mailto:bentleyk@cardiff.ac.uk" class="" data-icon-position="" data-hide-link-title="0">
                        bentleyk@cardiff.ac.uk
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Kirsten Bentley
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Biomedical Sciences Research Complex and School of Biology, University of St Andrew
                       </span>
                       , St. Andrews KY16 9ST
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Kirsten%2BBentley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Bentley%20K&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AKirsten%2BBentley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:falnaji@illinois.edu" class="" data-icon-position="" data-hide-link-title="0">
                        falnaji@illinois.edu
                       </a>
                       <a href="mailto:bentleyk@cardiff.ac.uk" class="" data-icon-position="" data-hide-link-title="0">
                        bentleyk@cardiff.ac.uk
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Ashley Pearson
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Biomedical Sciences Research Complex and School of Biology, University of St Andrew
                       </span>
                       , St. Andrews KY16 9ST
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Ashley%2BPearson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Pearson%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAshley%2BPearson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Andrew Woodman
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Biochemistry & Molecular Biology, 201 Althouse Lab
                       </span>
                       , University Park, PA 16802,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Andrew%2BWoodman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Woodman%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAndrew%2BWoodman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Jonathan Moore
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        John Innes Centre, Cell and Developmental Biology
                       </span>
                       , Norwich, NR4 7UH,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Jonathan%2BMoore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Moore%20J&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AJonathan%2BMoore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Helen Fox
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Division of Virology, National Institute for Biological Standards and Control
                       </span>
                       , Potters Bar, Hertfordshire, EN6 3QG,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Helen%2BFox%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Fox%20H&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AHelen%2BFox%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">
                     Andrew MacAdam
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Division of Virology, National Institute for Biological Standards and Control
                       </span>
                       , Potters Bar, Hertfordshire, EN6 3QG,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Andrew%2BMacAdam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=MacAdam%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAndrew%2BMacAdam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-7">
                    <div class="author-tooltip-name">
                     David J. Evans
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Biomedical Sciences Research Complex and School of Biology, University of St Andrew
                       </span>
                       , St. Andrews KY16 9ST
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=David%2BJ.%2BEvans%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Evans%20DJ&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ADavid%2BJ.%2BEvans%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.09.29.319285v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2407740" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2407740/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.09.29.319285v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2407740" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2407740/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.09.29.319285v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2407740" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2407740/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.09.29.319285v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2407740" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2407740/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.09.29.319285v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2407740" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2407740/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-3">
                             Genetic recombination in RNA viruses is an important evolutionary mechanism. It contributes to population diversity, host/tissue adaptation and compromises vaccine efficacy. Both the molecular mechanism and initial products of recombination are relatively poorly understood. We used an established poliovirus-based in vitro recombination assay to investigate the roles of sequence identity and RNA structure, both implicated or inferred from analysis of circulating recombinant viruses, in the process. In addition, we used next generation sequencing to investigate the early products of recombination after cellular co-infection with different poliovirus serotypes. In independent studies we find no evidence for a role for RNA identity or structure in determining recombination junctions location. Instead, genome function and fitness are of greater importance in determining the identity of recombinant progeny. These studies provide further insights into this important evolutionary mechanism and emphasise the critical nature of the selection process on a mixed virus population.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-5">
                             The rapid evolution of RNA viruses is attributable to both the error prone nature of viral RNA-dependent RNA polymerases (RdRp) and the extensive exchange of genetic information achieved through the processes of recombination and – in segmented viruses – reassortment. Together, these are important drivers of virus evolution, often linked to the emergence of novel pathogens and disease outbreaks [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             -
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ]. Recombination predominates in the single stranded positive-sense (mRNA-sense) RNA viruses and is proposed to occur via distinct replicative or non-replicative mechanisms. In the latter the formation of a full-length genome likely involves processing and ligation by cellular enzymes, and the detailed mechanism and biological significance remain unclear [
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ,
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ]. In contrast, replicative recombination between two compatible virus genomes co-infecting the same cell is increasingly well-studied [
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             -
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ].
                            </p>
                            <p id="p-6">
                             The importance of recombination in nature is typified by the frequent isolation of recombinant forms of members of the enterovirus genus of the family Picornaviridae. Poliovirus and other picornaviruses have been extensively used to study recombination and the underlying mechanisms involved [
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ,
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ]. These include the initial demonstration of the copy-choice strand-transfer event and numerous, but often conflicting studies interpreting the influence of RNA sequence and structure on the generation of viable recombinants [
                             <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ,
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             -
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             ]. To facilitate the analysis of early events in recombination we previously established an in vitro assay (the CRE-REP assay) that solely yields recombinant viruses [
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ].
                            </p>
                            <p id="p-7">
                             To directly address the potential influence of RNA structure and sequence identity on the strand-transfer event in recombination we have exploited the CRE-REP assay and modified the input genomes. By independently analysing genomes containing extensive regions of sequence identity, or templates modified to contain regions of high or low levels of RNA structure, we were unable to demonstrate any significant influence on the proportion of recombination events occurring in these modified regions. We extended these studies to investigate the early RNA products arising from a natural co-infection of cells with two serotypes of poliovirus using deep sequencing. In contrast to the generally clustered range of recombinants found using the CRE-REP assay [
                             <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ] these were distributed throughout the genomic region analysed. Based on our analyses we propose that neither sequence identity nor RNA structure are important primary determinants in recombination.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Results
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Influence of RNA structure and sequence on CRE-REP recombination
                             </h3>
                             <p id="p-8">
                              The CRE-REP assay [
                              <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                              ] utilises a poliovirus type 1 (PV1) luciferase-encoding subgenomic replicon as the polymerase donor and a poliovirus type 3 (PV3) genome bearing a lethal mutation in the cis-acting replication element (CRE) located within the 2C-coding region [
                              <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                               19
                              </a>
                              ,
                              <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              ] as the acceptor (
                              <strong>
                               <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                                Fig 1A
                               </a>
                              </strong>
                              ). Co-transfection of in vitro-generated RNA from both templates only yields viable recombinants if a suitable strand-transfer event 5′ to the functional CRE element in the donor and 3′ to the P1 capsid-coding region in the acceptor template occurs. We previously demonstrated that viable recombinants had junctions that predominantly clustered in one of two regions, spanning either the VP1/2A or 2A/2B polyprotein cleavage boundaries (Cluster 1 and Cluster 2 respectively, hereafter known as C1 and C2). We reasoned that by modifying the sequence identity or RNA structure within the C2 region of the donor or acceptor templates, we could investigate the local influences of sequence or RNA structure on recombination using the unmodified C1 region as an internal control. For example, a positive influence on recombination would yield a higher ratio of C2:C1 recombinants than observed in a control assay with unmodified templates.
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F1.large.jpg?width=800&height=600&carousel=1" title="Schematic representation of CRE-REP assays. (A) Intertypic CRE-REP assay showing the genomes of the PV3 acceptor and PV1 donor RNAs. Black arrows represent position of primers used to amplify recombinant genomes. The lower expanded image illustrates the recombination window and highlights examples of recombination events in Cluster 1 and Cluster 2. The position of a unique XbaI site used for screening is shown in the donor template. (B) Modified CRE-REP assays highlighting the region of modification within Cluster 2 and the resulting acceptor and donor templates with altered sequence identity or RNA secondary structure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2084576083" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Schematic representation of CRE-REP assays. <strong>(A)</strong> Intertypic CRE-REP assay showing the genomes of the PV3 acceptor and PV1 donor RNAs. Black arrows represent position of primers used to amplify recombinant genomes. The lower expanded image illustrates the recombination window and highlights examples of recombination events in Cluster 1 and Cluster 2. The position of a unique <em>Xba</em>I site used for screening is shown in the donor template. <strong>(B)</strong> Modified CRE-REP assays highlighting the region of modification within Cluster 2 and the resulting acceptor and donor templates with altered sequence identity or RNA secondary structure.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F1.medium.gif" width="440" height="391"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F1.medium.gif" width="440" height="391"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Fig 1.
                               </span>
                               <p id="p-9" class="first-child">
                                Schematic representation of CRE-REP assays.
                                <strong>
                                 (A)
                                </strong>
                                Intertypic CRE-REP assay showing the genomes of the PV3 acceptor and PV1 donor RNAs. Black arrows represent position of primers used to amplify recombinant genomes. The lower expanded image illustrates the recombination window and highlights examples of recombination events in Cluster 1 and Cluster 2. The position of a unique XbaI site used for screening is shown in the donor template.
                                <strong>
                                 (B)
                                </strong>
                                Modified CRE-REP assays highlighting the region of modification within Cluster 2 and the resulting acceptor and donor templates with altered sequence identity or RNA secondary structure.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-10">
                              We exploited the redundancy of the amino acid triplet code to design 450 nt sequences, centered on the 2A/2B cleavage site, with high or low levels of RNA structure (see Materials and Methods). All designed sequences exhibited the same level of sequence identity when compared to their parental sequence (85.5%) and retained the same level of sequence identity as present between the parental PV1 and PV3 CRE-REP partners (77.6% +/-1%;
                              <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
                               Table S1
                              </a>
                              ). In each case the encoded proteins were identical to the respective native PV1 donor or PV3 acceptor templates. The resulting modified RNA templates used in CRE-REP assays were: (1) the PV3-derived acceptor modified PV3
                              <sup>
                               L
                              </sup>
                              and PV3
                              <sup>
                               H
                              </sup>
                              (where L and H refer to low and high RNA structure respectively) and (2) the PV1-derived donor modified PV1
                              <sup>
                               L
                              </sup>
                              and PV1
                              <sup>
                               H
                              </sup>
                              (
                              <strong>
                               <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                                Fig 1B
                               </a>
                               and
                               <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                                S1
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-11">
                              Similarly, we engineered donor and acceptor templates such that the same 450 nt region was ∼98% identical. This was achieved by directly exchanging the PV1 sequence with that of PV3, or vice versa, with the exception of retention of the seven amino acids that differ between PV1 and PV3 in this region (see Materials and Methods) to avoid disruption of any in cis interactions involving these residues. The resulting modified RNA templates for use in CRE-REP assays were PV3
                              <sup>
                               1
                              </sup>
                              (a PV3-derived acceptor containing a stretch of PV1 sequence), and PV1
                              <sup>
                               3
                              </sup>
                              (a PV1-derived donor containing a stretch of PV3 sequence;
                              <strong>
                               <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                                Fig 1B
                               </a>
                               and
                               <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                                S1
                               </a>
                              </strong>
                              ). We confirmed that the sequences introduced to the CRE-REP donor and acceptor templates had no significant effect on virus replication (
                              <strong>
                               <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                                Fig S2
                               </a>
                              </strong>
                              ) and therefore should not influence the underlying replicative recombination process.
                             </p>
                             <p id="p-12">
                              To generate recombinant virus populations, we conducted CRE-REP assays in parallel. For each assay one unmodified template (either donor or acceptor) was paired with one modified donor or acceptor template. Assays are referred to by the name of the modified template of the RNA pair. Donor and acceptor RNAs were co-transfected into murine L929 cells (permissive but not susceptible to poliovirus) in equimolar amounts and the virus-containing supernatant harvested 30 hour post-transfection. In addition, we transfected each donor and acceptor RNA individually into L929 cells and confirmed that, as expected, none were alone able to generate infectious virus (data not shown). A total of three independent co-transfections were carried out for each assay and the yield of recombinant virus determined by plaque assay (
                              <strong>
                               <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                                Fig S3
                               </a>
                              </strong>
                              ). The unmodified CRE-REP control assay (WT) generated an average of 2.1 × 10
                              <sup>
                               3
                              </sup>
                              pfu/ml of recombinant viruses. The total yield of recombinants from CRE-REP assays with template modifications did not significantly differ from the WT control assay, demonstrating no obvious advantage or disadvantage in recombinant generation resulting from the modifications.
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Analysis of recombination junctions in the modified CRE-REP assay
                             </h3>
                             <p id="p-13">
                              The overall yield of recombinants (
                              <strong>
                               <a id="xref-fig-9-2" class="xref-fig" href="#F9">
                                Fig S3
                               </a>
                              </strong>
                              ) includes all those that map in C1 and C2 combined. To determine whether there were any changes in the ratio of recombination events occurring within the C1 and C2 regions as a result of the introduced modifications we sequenced a representative sample of recovered viruses. The supernatants from the triplicate transfections of each assay were pooled to generate a diverse virus population and minimise founder-effects. A total of 420 recombinants (60 from each CRE-REP assay) were isolated by limit dilution. Following viral RNA extraction and reverse transcription, the entire recombination region was amplified by PCR and sequenced to determine the location and features of each recombination junction (
                              <strong>
                               <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                                Fig 2A
                               </a>
                              </strong>
                              ).
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F2.large.jpg?width=800&height=600&carousel=1" title="Analysis of recombination junctions from CRE-REP assay. (A) The location of each recombination junction was mapped respective to each parental genome. Each line represents a unique recombinant within the population of precise (black) or imprecise-insertion (blue) recombinants (see Nomenclature, materials and method). (B) Junctions were defined as precise or imprecise and graphed according to location in Cluster 1 or Cluster 2. Unflattened data was compared to flattened data, using Fishers Exact Test and showed no significant difference in the distribution of recombinants. (C) Junctions were defined as for (B) and graphed by individual CRE-REP assay. Each modified assay was compared to wild type using Fishers Exact Test and showed no significant difference in the distribution of recombinants, either between assays or unflattened and flattened data sets." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2084576083" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Analysis of recombination junctions from CRE-REP assay. <strong>(A)</strong> The location of each recombination junction was mapped respective to each parental genome. Each line represents a unique recombinant within the population of precise (black) or imprecise-insertion (blue) recombinants (see Nomenclature, materials and method). <strong>(B)</strong> Junctions were defined as precise or imprecise and graphed according to location in Cluster 1 or Cluster 2. Unflattened data was compared to flattened data, using Fishers Exact Test and showed no significant difference in the distribution of recombinants. <strong>(C)</strong> Junctions were defined as for (B) and graphed by individual CRE-REP assay. Each modified assay was compared to wild type using Fishers Exact Test and showed no significant difference in the distribution of recombinants, either between assays or unflattened and flattened data sets.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F2.medium.gif" width="376" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F2.medium.gif" width="376" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig 2.
                               </span>
                               <p id="p-14" class="first-child">
                                Analysis of recombination junctions from CRE-REP assay.
                                <strong>
                                 (A)
                                </strong>
                                The location of each recombination junction was mapped respective to each parental genome. Each line represents a unique recombinant within the population of precise (black) or imprecise-insertion (blue) recombinants (see Nomenclature, materials and method).
                                <strong>
                                 (B)
                                </strong>
                                Junctions were defined as precise or imprecise and graphed according to location in Cluster 1 or Cluster 2. Unflattened data was compared to flattened data, using Fishers Exact Test and showed no significant difference in the distribution of recombinants.
                                <strong>
                                 (C)
                                </strong>
                                Junctions were defined as for (B) and graphed by individual CRE-REP assay. Each modified assay was compared to wild type using Fishers Exact Test and showed no significant difference in the distribution of recombinants, either between assays or unflattened and flattened data sets.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-15">
                              After sequencing, 119 samples were discarded due to ambiguities in the sequence that indicated the virus population was not clonal. Approximately one third of the remaining recombinants were present more than once in the dataset. These identical sequences could either arise independently due to unique recombination events or may reflect early recombination products that have undergone additional rounds of replication (founder-effects), so increasing their proportion in the mixed virus progeny. Since the experimental design could not discriminate between these, we considered both the data in its entirety (unflattened) and after discarding identical sequences (flattened).
                             </p>
                             <p id="p-16">
                              In the flattened data (n=205) 58% of all junctions mapped to the C1 region while 42% mapped to the C2 region. No statistical difference (Fishers Exact Test) was observed when compared to the unflattened data (n=301), where 53% mapped to C1 and 47% to C2 (
                              <strong>
                               <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                                Fig 2B
                               </a>
                              </strong>
                              ). Further analysis of the data demonstrated that there were also no statistically significant (Fishers Exact Test) favoured or disfavoured modified assay pairings that altered the distribution of junctions between the C1 and C2 regions when compared with the unmodified WT templates (
                              <strong>
                               <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                                Fig 2C
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-17">
                              These results suggested that neither local sequence identity, nor the gross level of RNA structure, play a major role in influencing the location of recombination junctions within viable recombinants that were selected, packaged and subsequently propagated.
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Isolation of recombinants following virus co-infection
                             </h3>
                             <p id="p-18">
                              If, as we suggest, RNA sequence and structure do not influence the process of recombination we would expect the junctions to have been functionally selected i.e. on viability or relative fitness, from a much more diverse spectrum of crossover events. Such a diverse population potentially includes both in-frame insertions and deletions and out-of-frame junctions. Of these, insertions have been previously reported [
                              <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                              ] but deletions or out-of-frame recombinants are highly unlikely to be present amongst viable recombinant progeny. To investigate recombinant genome diversity further we developed an assay based upon the next generation sequencing (NGS) analysis of a mixed virus population arising after co-infection with poliovirus serotypes 1 and 3.
                             </p>
                             <p id="p-19">
                              HeLa cells were co-infected at an MOI of 10 and intracellular viral RNA was harvested 5 hours post-infection (hpi), a duration previously shown to allow the production and detection of intracellular recombinants [
                              <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">
                               19
                              </a>
                              ], whilst minimising the selection and analysis of genomes that have been packaged or initiated further rounds of infection. Purified RNA was reverse transcribed, and PCR amplified across the region spanning Clusters 1 and 2, before analysis by gel electrophoresis (
                              <strong>
                               <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                                Fig 3A
                               </a>
                              </strong>
                              ). A distinct product of ∼1.3 kb, representing the expected size of a precise recombination product was observed, together with both larger (up to ∼3 kb) and smaller (∼0.45 kb) products. In addition, there was extensive diffuse background staining between these products suggestive of a range of different sized PCR products (
                              <strong>
                               <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                                Fig 3A
                               </a>
                               )
                              </strong>
                              . Preliminary analysis of 7 randomly-selected cloned PCR products by Sanger sequencing (
                              <strong>
                               <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">
                                Table S2
                               </a>
                              </strong>
                              ) showed a diversity of recombination junctions and encouraged us to study the population of molecules using NGS.
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F3.large.jpg?width=800&height=600&carousel=1" title="Isolation and characterization of recombinants generated from virus co-infection. (A) PCR amplification of the region between nucleotides 3235-4548 followed by agarose gel screening. The negative control is an equimolar mixture of in vitro transcribed full-length PV1 and PV3 RNA (B) The number of precise and imprecise (both insertion and deletion) recombinants was plotted against the number of NGS reads (left panel); each dot represents a unique recombinant. On the right panel, heatmaps show the distribution of sequence lengths of insertion (blue) and deletion (red) within the detected imprecise recombinants. Each cell represents a unique recombinant. (C) Parallel coordinates visualization of recombinants from co-infection studies. The location of each recombination junction was mapped respective to each parental genome. Each line represents a unique recombinant within the population of precise (black), imprecise-insertion (blue) and imprecise-deletion (red) recombinants." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2084576083" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Isolation and characterization of recombinants generated from virus co-infection. <strong>(A)</strong> PCR amplification of the region between nucleotides 3235-4548 followed by agarose gel screening. The negative control is an equimolar mixture of <em>in vitro</em> transcribed full-length PV1 and PV3 RNA <strong>(B)</strong> The number of precise and imprecise (both insertion and deletion) recombinants was plotted against the number of NGS reads (left panel); each dot represents a unique recombinant. On the right panel, heatmaps show the distribution of sequence lengths of insertion (blue) and deletion (red) within the detected imprecise recombinants. Each cell represents a unique recombinant. <strong>(C)</strong> Parallel coordinates visualization of recombinants from co-infection studies. The location of each recombination junction was mapped respective to each parental genome. Each line represents a unique recombinant within the population of precise (black), imprecise-insertion (blue) and imprecise-deletion (red) recombinants.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F3.medium.gif" width="440" height="236"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F3.medium.gif" width="440" height="236"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig 3.
                               </span>
                               <p id="p-20" class="first-child">
                                Isolation and characterization of recombinants generated from virus co-infection.
                                <strong>
                                 (A)
                                </strong>
                                PCR amplification of the region between nucleotides 3235-4548 followed by agarose gel screening. The negative control is an equimolar mixture of in vitro transcribed full-length PV1 and PV3 RNA
                                <strong>
                                 (B)
                                </strong>
                                The number of precise and imprecise (both insertion and deletion) recombinants was plotted against the number of NGS reads (left panel); each dot represents a unique recombinant. On the right panel, heatmaps show the distribution of sequence lengths of insertion (blue) and deletion (red) within the detected imprecise recombinants. Each cell represents a unique recombinant.
                                <strong>
                                 (C)
                                </strong>
                                Parallel coordinates visualization of recombinants from co-infection studies. The location of each recombination junction was mapped respective to each parental genome. Each line represents a unique recombinant within the population of precise (black), imprecise-insertion (blue) and imprecise-deletion (red) recombinants.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              The complexity of recombination events revealed by NGS sequence analysis
                             </h3>
                             <p id="p-21">
                              After the recombination-specific amplification of the viral RNA 5 hpi, the amplicons were fragmented before being sequenced on a MiSeq Illumina platform, generating ∼1.5 million reads of 250 nts length. We developed a pipeline based upon the aligner Bowtie2 [
                              <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              ] and the Viral Recombination Mapper (ViReMa) [
                              <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                               21
                              </a>
                              ] to analyse recombination junctions, and used it as previously described [
                              <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                              ]. Briefly, the entire dataset was aligned to WT PV3 and PV1 reference sequences using Bowtie2 and those that matched perfectly i.e. outwith the recombination junction, were eliminated. Remaining reads were analysed using ViReMa to identify the recombination junction (
                              <strong>
                               <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                                Table 1
                               </a>
                              </strong>
                              ).
                             </p>
                             <div id="T1" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2409617/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2409617/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2409617" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 1.
                               </span>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-22">
                              We identified ∼27 × 10
                              <sup>
                               3
                              </sup>
                              recombinant junction sequences (
                              <strong>
                               <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">
                                Table 1
                               </a>
                              </strong>
                              ) in the dataset, defining 323 unique junctions, between PV3 and PV1. There were no recombinant junctions in the control dataset. Identified recombinant junctions were initially grouped into those with no sequence duplication or deletion (so called precise junctions [
                              <a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                              ], n=203; see nomenclature in Materials and Methods), those with sequence insertions (imprecise insertions, n=35), or those with sequence deletions (imprecise deletions, n=85) (
                              <strong>
                               <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                                Fig 3B
                               </a>
                              </strong>
                              ). The two groups of imprecise junctions exhibited a range of different sized insertions (1 – 3150 nt) or deletions (1 – 896 nt;
                              <strong>
                               <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                                Fig 3B
                               </a>
                              </strong>
                              ). All 323 recombinants were mapped using parallel coordinate diagrams, which clearly showed that recombination junctions, whether precise or imprecise, were distributed throughout the analysis region (
                              <strong>
                               <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                                Fig 3C
                               </a>
                              </strong>
                              ), with no evidence for the clustering around the encoded polyprotein cleavage sequences seen in previous CRE-REP analysis [
                              <a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                              ].
                             </p>
                             <p id="p-23">
                              To validate these observations, we performed a technical replicate by independently extracting total cell RNA from the original co-infection sample (
                              <strong>
                               <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                                Fig S4
                               </a>
                               ,
                               <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">
                                Table 1
                               </a>
                              </strong>
                              ). Despite fewer NGS reads overall, and a concomitant reduction in ViReMa-detected junctions, we were still able to identify hundreds of recombinants between PV3 and PV1. Correlation analysis of the NGS reads for each junction in the two replicates, normalised to the total reads per junction, demonstrated the method produced statistically reproducible results (Spearman R=0.88). All subsequent analysis used the first sequenced population containing 323 unique junctions.
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Analysis of recombination junctions following NGS
                             </h3>
                             <p id="p-24">
                              The NGS analysis of junction sequences provides an independent approach to determine the randomness, or otherwise, of the recombination process. To formally test whether the observed crossover junctions between PV3 and PV1 were randomly distributed across the targeted region, we compared the precise junction locations with a modelled dataset containing randomly generated precise junctions. The comparison used a sliding cumulative score across the window of recombination to determine the occurrence of recombination events at each nucleotide position (
                              <strong>
                               <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                                Fig 4A
                               </a>
                              </strong>
                              ). We observed no significant difference between our NGS dataset and the randomly generated dataset (P=0.06; Mann–Whitney U test), supporting our contention that precise recombination junctions are randomly located throughout the targeted region.
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F4.large.jpg?width=800&height=600&carousel=1" title="Analysis of recombination junction sequences generated from virus co-infection. (A) The occurrence of recombination on the genome was given a score of 1 while no occurrence is 0. The number of recombination occurrences was calculated in a cumulative way i.e. the occurrence of recombination at each position is added to the occurrence at the previous position. This was applied to two populations; the precise recombinants from the virus co-infection sample and a random recombination model generated by Excel. The cumulative recombinant count (y-axis) was plotted against the locations on the PV1 (donor) amplicon (x-axis). (B) An Identical sequence length (IS) of 4-nt that hypothetically occurred twice in the genome was used as an example. The rectangles correspond to a part of the targeted region of PV1 and PV3, the lines represent recombinants and their lengths denote the NGS read. i.e. the longer the line the more NGS reads. The identical sequence is placed within a smaller rectangle and the red Ns represent a mismatch of any other nucleotides. Looking at a possible real situation in the left panel, the Precise Recombinants (PRs) occurred at different sites within the identical sequence with different reads. ViReMa will always report 1 PR per IS; something like the middle panel. In our linear model we convert ViReMa model into the right panel, where junction counts equal the number of sequence identity length + the mismatch, and NGS reads are equal for all junctions. (C) Individual counts of identical sequences (IS) of different lengths were summed within the amplified region and plotted along with the count of precise junctions that mapped to each different length of sequence identity. For each of the latter the average number of junctions per nucleotide was calculated assuming recombination was equally likely to occur at each position within identical donor and template sequences. The linear regression was calculated to compare the slopes between the fitted line and the perfect line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2084576083" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Analysis of recombination junction sequences generated from virus co-infection. <strong>(A)</strong> The occurrence of recombination on the genome was given a score of 1 while no occurrence is 0. The number of recombination occurrences was calculated in a cumulative way i.e. the occurrence of recombination at each position is added to the occurrence at the previous position. This was applied to two populations; the precise recombinants from the virus co-infection sample and a random recombination model generated by Excel. The cumulative recombinant count (y-axis) was plotted against the locations on the PV1 (donor) amplicon (x-axis). <strong>(B)</strong> An Identical sequence length (IS) of 4-nt that hypothetically occurred twice in the genome was used as an example. The rectangles correspond to a part of the targeted region of PV1 and PV3, the lines represent recombinants and their lengths denote the NGS read. i.e. the longer the line the more NGS reads. The identical sequence is placed within a smaller rectangle and the red Ns represent a mismatch of any other nucleotides. Looking at a possible real situation in the left panel, the Precise Recombinants (PRs) occurred at different sites within the identical sequence with different reads. ViReMa will always report 1 PR per IS; something like the middle panel. In our linear model we convert ViReMa model into the right panel, where junction counts equal the number of sequence identity length + the mismatch, and NGS reads are equal for all junctions. <strong>(C)</strong> Individual counts of identical sequences (IS) of different lengths were summed within the amplified region and plotted along with the count of precise junctions that mapped to each different length of sequence identity. For each of the latter the average number of junctions per nucleotide was calculated assuming recombination was equally likely to occur at each position within identical donor and template sequences. The linear regression was calculated to compare the slopes between the fitted line and the perfect line.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F4.medium.gif" width="440" height="353"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F4.medium.gif" width="440" height="353"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig 4.
                               </span>
                               <p id="p-25" class="first-child">
                                Analysis of recombination junction sequences generated from virus co-infection.
                                <strong>
                                 (A)
                                </strong>
                                The occurrence of recombination on the genome was given a score of 1 while no occurrence is 0. The number of recombination occurrences was calculated in a cumulative way i.e. the occurrence of recombination at each position is added to the occurrence at the previous position. This was applied to two populations; the precise recombinants from the virus co-infection sample and a random recombination model generated by Excel. The cumulative recombinant count (y-axis) was plotted against the locations on the PV1 (donor) amplicon (x-axis).
                                <strong>
                                 (B)
                                </strong>
                                An Identical sequence length (IS) of 4-nt that hypothetically occurred twice in the genome was used as an example. The rectangles correspond to a part of the targeted region of PV1 and PV3, the lines represent recombinants and their lengths denote the NGS read. i.e. the longer the line the more NGS reads. The identical sequence is placed within a smaller rectangle and the red Ns represent a mismatch of any other nucleotides. Looking at a possible real situation in the left panel, the Precise Recombinants (PRs) occurred at different sites within the identical sequence with different reads. ViReMa will always report 1 PR per IS; something like the middle panel. In our linear model we convert ViReMa model into the right panel, where junction counts equal the number of sequence identity length + the mismatch, and NGS reads are equal for all junctions.
                                <strong>
                                 (C)
                                </strong>
                                Individual counts of identical sequences (IS) of different lengths were summed within the amplified region and plotted along with the count of precise junctions that mapped to each different length of sequence identity. For each of the latter the average number of junctions per nucleotide was calculated assuming recombination was equally likely to occur at each position within identical donor and template sequences. The linear regression was calculated to compare the slopes between the fitted line and the perfect line.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-26">
                              As short regions of localized sequence identity have previously been suggested to influence the recombination process [
                              <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">
                               10
                              </a>
                              ,
                              <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                              ,
                              <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ,
                              <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                               24
                              </a>
                              ], we next investigated a role for such sequences in shaping our NGS dataset. Within the targeted region there are 219 short regions of sequence identity between 2 and 32 nt in length. ViReMa maps junctions to the first divergent nucleotide 3′ to a region of identity [
                              <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">
                               21
                              </a>
                              ], which makes it impossible to determine the exact nucleotide – within the identical sequence – at which strand transfer occurred. Because of this, we made the assumption that recombination was equally likely to occur between any nucleotide within a region of sequence identity and its adjacent 3′ nucleotide (
                              <strong>
                               <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                                Fig 4B
                               </a>
                              </strong>
                              ). We normalized the recombinants per nucleotide on this assumption; for example, if a precise junction with 100 NGS reads mapped 3′ to a 4 nt region of aligned PV3 and PV1 sequences, this equated to five precise junctions each represented by 20 reads (
                              <strong>
                               <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                                Fig 4B
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-27">
                              Notably there were no recombinants that mapped to the most extensive region (32 nt.) of sequence identity between the parental genomes. Of the remaining 218 regions (2 - 21 nt.), with reads normalised as described, there was no significant difference (P=0.93) in linear regression analysis of the fitted line for the observed data or the perfect line (
                              <strong>
                               <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                                Fig 4C
                               </a>
                              </strong>
                              ). This suggests, for precise junctions at least and at this time point of the co-infection of poliovirus serotypes 1 & 3, that recombination junctions are distributed independently of sequence identity.
                             </p>
                             <p id="p-28">
                              We additionally failed to detect enrichment of short regions of sequence identity at imprecise junctions (
                              <strong>
                               <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                                Fig 5A
                               </a>
                               )
                              </strong>
                              . Correlation analysis between the experimental data and an in silico library containing every possible imprecise recombinant within the targeted region, showed a significant positive correlation (R =0.8, p<0.01). This suggests that imprecise recombination tracks closely with the random model and indicates no role for sequence identity in their generation. Finally, we mapped all the imprecise recombinants to their genome positions and we could not detect any correlation between the NGS read frequency and the length of insertion/deletion sequences, reading frame, or location on the genome (
                              <strong>
                               <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                                Fig 5B
                               </a>
                              </strong>
                              ).
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F5.large.jpg?width=800&height=600&carousel=1" title="The influence of identical sequence length and reading frame on imprecise recombination. (A) The imprecise recombinants were counted at each identical length (x-axis) and the percentage proportion related to the total imprecise junctions was calculated (y-axis). The flattened data of the imprecise recombinants generated from the co-infection experiments (grey) was plotted against a random control (black), where every possible imprecise junction was created and classified using a custom perl script. (B) In the upper panel, the location of the detected imprecise recombinants on the PV1 genome was plotted against the NGS read count. In the lower panel, the size of either the insertions or deletions of the detected recombinants were plotted against NGS read count. Each dot represents a unique recombinant." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2084576083" data-figure-caption='<div class="highwire-markup">The influence of identical sequence length and reading frame on imprecise recombination. (A) The imprecise recombinants were counted at each identical length (x-axis) and the percentage proportion related to the total imprecise junctions was calculated (y-axis). The flattened data of the imprecise recombinants generated from the co-infection experiments (grey) was plotted against a random control (black), where every possible imprecise junction was created and classified using a custom perl script. (B) In the upper panel, the location of the detected imprecise recombinants on the PV1 genome was plotted against the NGS read count. In the lower panel, the size of either the insertions or deletions of the detected recombinants were plotted against NGS read count. Each dot represents a unique recombinant.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F5.medium.gif" width="440" height="149"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F5.medium.gif" width="440" height="149"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig 5.
                               </span>
                               <p id="p-29" class="first-child">
                                The influence of identical sequence length and reading frame on imprecise recombination. (A) The imprecise recombinants were counted at each identical length (x-axis) and the percentage proportion related to the total imprecise junctions was calculated (y-axis). The flattened data of the imprecise recombinants generated from the co-infection experiments (grey) was plotted against a random control (black), where every possible imprecise junction was created and classified using a custom perl script. (B) In the upper panel, the location of the detected imprecise recombinants on the PV1 genome was plotted against the NGS read count. In the lower panel, the size of either the insertions or deletions of the detected recombinants were plotted against NGS read count. Each dot represents a unique recombinant.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-30">
                              While the above analysis strongly suggested that recombination junctions were randomly distributed, it remained a possibility that localized regions of RNA structure may influence junction location. To investigate these possibilities, we applied mean folding energy difference (MFED) analysis [
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                              ] to all detected recombinants of all types. In comparing the MFED values from either the positive-or negative-strand to the number of recombination junctions occurring within the same sliding window, we were unable to detect any correlation between the presence, or absence, of predicted RNA structure and the location of recombination junctions (
                              <strong>
                               <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                                Fig 6
                               </a>
                              </strong>
                              ). This was in agreement with the CRE-REP assay analysis (
                              <strong>
                               <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                                Fig 2
                               </a>
                              </strong>
                              ) and further supports our view that RNA structure does not influence the location of recombination junctions.
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F6.large.jpg?width=800&height=600&carousel=1" title="The influence of RNA structure on recombination. The MFED was calculated for a 100 nt sliding window (at 30 nt increments) for both PV1 and PV3 positive and negative strands (dashed and solid lines respectively) and plotted against the number of total recombinants within the same sliding window. A Spearman correlation coefficient was calculated between the recombinant count and the MFED values on either strand." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2084576083" data-figure-caption='<div class="highwire-markup">The influence of RNA structure on recombination. The MFED was calculated for a 100 nt sliding window (at 30 nt increments) for both PV1 and PV3 positive and negative strands (dashed and solid lines respectively) and plotted against the number of total recombinants within the same sliding window. A Spearman correlation coefficient was calculated between the recombinant count and the MFED values on either strand.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F6.medium.gif" width="440" height="133"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 6." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F6.medium.gif" width="440" height="133"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 6." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig 6.
                               </span>
                               <p id="p-31" class="first-child">
                                The influence of RNA structure on recombination. The MFED was calculated for a 100 nt sliding window (at 30 nt increments) for both PV1 and PV3 positive and negative strands (dashed and solid lines respectively) and plotted against the number of total recombinants within the same sliding window. A Spearman correlation coefficient was calculated between the recombinant count and the MFED values on either strand.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-32">
                              Taken together, these studies suggest that recombination does not preferentially occur within regions of RNA structure or sequence identity between templates. We propose that recombination per se is a random process, but is followed by a rigorous selection for ‘functional’ genomes [
                              <a id="xref-ref-18-7" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                              ,
                              <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                               26
                              </a>
                              ]. It is the analysis of the latter that has produced the prevailing view of the importance of sequence identity or structure in their generation, whereas in this study - specifically examining early recombinant genomes - we cannot find evidence supporting their involvement.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-8">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-33">
                             Recombination is an important evolutionary process in the single-stranded positive-sense RNA viruses [
                             <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                              27
                             </a>
                             ,
                             <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                              28
                             </a>
                             ]. It is probably ubiquitous, and may be a necessary rescue mechanism to accommodate – and escape – the build-up of deleterious mutations that arise from the rapidly-processive [
                             <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                              29
                             </a>
                             ], error-prone polymerase [
                             <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                              30
                             </a>
                             ,
                             <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                              31
                             </a>
                             ]. Despite this, the molecular mechanisms underlying recombination remain relatively poorly understood. Several previous studies have implicated RNA secondary structure and particularly sequence similarity in determining the location of recombination junctions [12-14, 32, 33]. However, while it appears that this interpretation is widely accepted [
                             <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ,
                             <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ,
                             <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ,
                             <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">
                              24
                             </a>
                             ], the studies reported here suggest that the early products of recombination are not significantly influenced by RNA sequence identity or structure, necessitating a rethink of the defining features of this important process.
                            </p>
                            <p id="p-34">
                             By modifying our CRE-REP in vitro recombination assay [
                             <a id="xref-ref-18-8" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ], we investigated the influence of sequence identity or RNA structure in the parental genomes on the location of recombination junctions. We observed no significant variation in the distribution of recombinants occurring between the control region (Cluster 1) and the modified region (Cluster 2;
                             <strong>
                              <a id="xref-fig-2-5" class="xref-fig" href="#F2">
                               Fig 2
                              </a>
                             </strong>
                             ). Had either RNA sequence identity or structure been a significant influence on the mechanism of recombination we would have expected the distribution of recombinants to be skewed towards the Cluster region that favoured polymerase template-switching events. As all assays maintained a distribution similar to the wild type CRE-REP assay we inferred that neither sequence identity nor structure were a major influence on recombination.
                            </p>
                            <p id="p-35">
                             By design, the CRE-REP assay generates viable progeny viruses for analysis and, as such, does not allow the initial population of hybrid genomes, from which viable recombinants are selected, to be characterised. We therefore went on to analyse the RNA population from HeLa cells co-infected with wild type PV1 and PV3. Despite the analysis of several hundred recombinant junctions by NGS (
                             <strong>
                              <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                               Fig 3
                              </a>
                             </strong>
                             ) we could not detect any correlation between sequence identity (
                             <strong>
                              <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                               Figs 4
                              </a>
                              and
                              <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                               5
                              </a>
                             </strong>
                             ) or RNA structure (
                             <strong>
                              <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                               Fig 6
                              </a>
                             </strong>
                             ) and the enrichment of recombination junctions at any one particular site. Consequently, we propose that such regions undergo recombination at a rate no higher than would be expected by chance.
                            </p>
                            <p id="p-36">
                             We have recently shown that the resolution of imprecise recombinants to wild type length genomes occurs via repeated template-switching events and is highly dependent on virus fitness [
                             <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">
                              26
                             </a>
                             ]. This new understanding of resolution may help explain the discrepancy between the current prevailing view on recombination and the conclusions of this study if, as we suspect, the roles of sequence identity and structure are involved in genome function and fitness, rather than recombination per se. For example, studies that map crossover junctions to structured regions of recombinant genomes may conclude that they are causative, when the association may actually be correlative, and reflective of the genome modularity and relatively high fitness of a crossover in that specific region. Similarly, with genomes that exhibit variably extensive regions of sequence identity within an area in which recombinants are mapped, the functional selection may result in a clustering of crossovers in a region of sequence identity, not because of the identity per se, but because the resulting genome has enhanced fitness and is capable of competing better with others in the population.
                            </p>
                            <p id="p-37">
                             A number of interesting questions remain about this important evolutionary mechanism which will require further study. If recombinants are generated at random as we propose, what are the predominant functional criteria that determine their selection? At a more mechanistic level, how does the template-switching event occur? Although our analysis strongly suggests that the process is random, there remains the possibility that regions of micro-identity may influence the process. There remains the interesting possibility that precise and imprecise junctions could be generated by independent mechanisms. We consider this unlikely; a sequence- and RNA structure-independent process - coupled with functional selection - can account for all of the range of crossover junctions we identify, all that have been reported in the literature, and would fit with the laws of parsimony.
                            </p>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             MATERIAL AND METHODS
                            </h2>
                            <div id="sec-10" class="subsection">
                             <h3>
                              Virus and cell culture
                             </h3>
                             <p id="p-38">
                              HeLa cells and L929 mouse fibroblast cells were maintained in Dulbecco’s Modified Eagle Medium (DMEM) supplemented with 10% heat-inactivated FBS (FBS-DMEM). Poliovirus type 1 (Mahoney) and type 3 (Leon) were recovered following transfection of in vitro transcribed RNAs from full-length cDNAs. Additional virus stocks for replication competent PV3-2A/2B junction modified viruses FLC/PV3
                              <sup>
                               L
                              </sup>
                              , FLC/PV3
                              <sup>
                               H
                              </sup>
                              , and FLC/PV3
                              <sup>
                               1
                              </sup>
                              (see below) were similarly generated and all virus stocks quantified by plaque assay on HeLa cells. Growth kinetics of viruses were determined following synchronous infection of HeLa cells at a multiplicity of infection (MOI) of 5 in serum-free media (SF-DMEM). Unabsorbed virus was removed by washing with PBS and cells incubated in fresh FBS-DMEM at 37°C, 5% CO2. Supernatants containing virus were harvested at various time points post-infection and quantified by plaque assay. Recombinant viruses isolated from CRE-REP assays were biologically cloned by limit dilution in 96-well plates of confluent HeLa cells. Virus-containing supernatant was removed after 3 days incubation at 37°C, 5% CO2 and stored at -80°C, and the remaining cell monolayer stained with a 0.1% crystal violet solution. Virus supernatants from the highest dilutions causing complete cytopathic effect (CPE) were utilised in further analysis
                             </p>
                             <p id="p-39">
                              For co-infection studies, poliovirus type 1 and type 3 were used to co-infect confluent HeLa cells in four T175 flasks at an MOI of 10 of each virus. The cells were incubated for 30 minutes at 37°C before the virus was removed and the cells were washed with PBS. Fresh DMEM media was added and the flasks were incubated at 37°C for 5 h. After media removal, cells were washed with PBS, trypsinised and pooled in 2 ml DMEM media, giving a total of 7.6×10
                              <sup>
                               7
                              </sup>
                              cells. The cells was lysed by three freeze-thaw cycles to extract virus and the debris was discarded. The supernatant contained the viruses was filtered through 0.2 μm filters and the resulting filtrate used for RNA extraction, followed by RT-PCR and NGS sequencing
                             </p>
                            </div>
                            <div id="sec-11" class="subsection">
                             <h3>
                              Design of modified 2A/2B junction sequences
                             </h3>
                             <p id="p-40">
                              A 450 nt sequence, equivalent to nucleotides 3599 to 4045, of both PV1 and PV3 was re-designed using the software package SSE [
                              <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ] to generate sequences with the minimum and maximum amount of RNA structure possible by maintaining amino acid sequence and divergence between PV1 and PV3. The native PV1 and PV3 sequences were scrambled using the CDLR algorithm in SSE (
                              <strong>
                               <a id="xref-fig-11-1" class="xref-fig" href="#F11">
                                Fig S5
                               </a>
                              </strong>
                              ). This randomly scrambles the codon order, so disrupting any sequence-dependent underlying RNA structure, whilst maintaining the coding, codon usage and dinucleotide frequencies identical to those of the input sequence. CDLR randomisation was used to generate a large panel of variant sequences that retained these three characteristics. Each was processed using custom-written perl scripts to determine the sequence divergence from the input. Of those that were within 1% divergence (6230 PV1- and 6303 PV3-derived CDLR scrambles) the predicted RNA structure stability was calculated by determining the MFED (the minimal free energy difference [
                              <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                              ]. MFED is a measure of the intrinsic folding energy of an RNA sequence and is determined by comparing an input sequence with 99 sequence-order randomised variants. The sequence identity of the 10 highest and lowest energy sequences was analysed and representative sequences with similar sequence identity were selected for construction as cDNAs (
                              <strong>
                               <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">
                                Table S1
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-41">
                              The same 450 nt sequence was altered for sequence identity by replacing the PV1 sequence with that of PV3 and vice versa. The sequence exchange was not 100% as there are 7 amino acid differences between PV1 and PV3 in this region – 2A residues 87, 116, 123, and 2B residues 3, 22, 26 and 30. At these amino acid positions the sequence was not altered between PV1 and PV3. Synthesis of all modified 450 nt DNA fragments was carried out by GeneArt (Life Technologies) and provided as sequence verified, plasmid DNA.
                             </p>
                            </div>
                            <div id="sec-12" class="subsection">
                             <h3>
                              CRE-REP assay, plasmids, in vitro RNA transcription and transfection
                             </h3>
                             <p id="p-42">
                              The CRE-REP assay used to generate recombinant viruses has been described previously [
                              <a id="xref-ref-18-9" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                              ]. Wild type cDNAs of PV1 donor template, pRLucWT, and PV3 acceptor template, pT7/SL3, were used as a background for the generation of the 2A/2B modified cDNAs. Standard molecular biology techniques were used to replace the 450 nt sequence spanning the 2A/2B boundary with the synthesised DNAs described above to generate the CRE-REP cDNA variants referred to as PV1
                              <sup>
                               L
                              </sup>
                              , PV1
                              <sup>
                               H
                              </sup>
                              , PV1
                              <sup>
                               3
                              </sup>
                              , PV3
                              <sup>
                               L
                              </sup>
                              , PV3
                              <sup>
                               H
                              </sup>
                              and PV3
                              <sup>
                               1
                              </sup>
                              .
                             </p>
                             <p id="p-43">
                              In addition, the relevant PV3 sequences were placed into a background of the replication competent PV3 cDNA, pT7/FLC, for growth kinetics analysis. All cDNAs were confirmed by sequence analysis prior to use. Plasmids were linearized with ApaI (pRLucWT background) or SalI (pT7/SL3 or pT7/FLC background) and RNA transcribed using a HiScribe T7 High Yield RNA Synthesis kit (NEB) following the manufacturers’ protocol. RNA transcripts were DNaseI (NEB) treated to remove residual template DNA and column purified using a GeneJET RNA Purification Kit (ThermoFisher) prior to spectrophotometric quantification. For all CRE-REP variant assays equimolar amounts of both template RNAs (based on 250 ng of acceptor) were prepared with Lipofectamine 2000 (Life Technologies) in a 3:1 Lipofectamine 2000:RNA ratio as per manufacturers’ protocol, and transfected into 80% confluent L929 cell monolayers. Virus supernatant was recovered at 30 h post-transfection and virus quantified by plaque assay on HeLa cells. For virus stocks, 1 µg of RNA was transfected into 80% confluent HeLa cell monolayers using Lipofectamine 2000 as above, virus supernatant harvested at 12 h post-transfection and quantified by plaque assay on HeLa cells.
                             </p>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              RNA extraction and RT-PCR
                             </h3>
                             <p id="p-44">
                              Viral RNA was isolated from CRE-REP clarified cell culture supernatant samples using a GeneJET RNA Purification Kit (ThermoFisher) and reverse transcribed at 42°C using an oligo dT primer and SuperScript II reverse transcriptase (Life Technologies) as per manufacturers’ protocol. The region of recombination (VP1 to 2C) was amplified using primers PV3-F (5′-GCAAACATCTTCCAACCCGTCC-3′) and PV1-R (5′-TTGCTCTTGAACTGTATGTAGTTG-3′) and Taq polymerase (Life Technologies) with an initial denaturing at 94°C for 3 min, followed by 35 cycles of 94°C for 45 sec, 50°C for 30 sec and 72°C for 60 sec, and a final extension at 72°C for 10 min. Following poliovirus co-infection, total RNA was extracted from cells by QIAamp Viral RNA Mini Kit (Qiagen) and reverse transcribed as above. Subsequently 5 µl from the cDNA synthesis reaction was used in the PCR amplification. The region of recombination was amplified using primers PV3-F2 (5′-CTCCAAAGTCCGCATTTACA-3′) and PV1-R2 (5′-ATCAGGTTGGTTGCTACA-3′) and Taq polymerase (Promega) with an initial denaturing at 95°C for 2 min, followed by 35 cycles of 95°C for 1 min, 58.1°C for 30 sec and 72°C for 1.5 min, and a final extension at 72°C for 5 min.
                             </p>
                            </div>
                            <div id="sec-14" class="subsection">
                             <h3>
                              NGS library preparation
                             </h3>
                             <p id="p-45">
                              Illumina Nextera XT DNA kit was used to prepare the NGS samples for sequencing as per the manufacturer instructions. Briefly 1 ng from the amplicons was simultaneously fragmented and tagged with unique adapter sequences, followed by 10 cycles of PCR, before loading into the MiSeq instrument. The generated reads were uploaded to Illumina BaseSpace, a cloud-based genomics analysis and storage platform that directly integrates with all Illumina sequencers.
                             </p>
                            </div>
                            <div id="sec-15" class="subsection">
                             <h3>
                              Nomenclature
                             </h3>
                             <p id="p-46">
                              The terms precise and imprecise refer to whether the resulting recombinant genome is full length (precise) or contains insertions or deletions (imprecise) (
                              <strong>
                               <a id="xref-fig-12-1" class="xref-fig" href="#F12">
                                Fig S6
                               </a>
                              </strong>
                              ). We additionally defined the recombination junction as unambiguous, where the sequence between the donor and acceptor strands was dissimilar, or ambiguous where it was located within a short region of sequence identity in the parental genomes (
                              <strong>
                               <a id="xref-fig-12-2" class="xref-fig" href="#F12">
                                Fig S6
                               </a>
                              </strong>
                              ). We deliberately avoid use of the inexact term ‘homology’ which is variously used to indicate identity or relatedness. In two divergent recombining sequences, junctions can occur in regions of sequence identity or at positions where there is sufficient sequence divergence (whilst still being homologous) to unambiguously define the junction between the donor and acceptor sequences. Similarly, insertions or deletions can generate junctions where the origin of the sequences around the junction cannot be definitively assigned to the donor or recipient strand due to limited sequence identity around the junction.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-16">
                            <h2 class="">
                             DATA AVAILABILITY
                            </h2>
                            <p id="p-47">
                             The raw sequencing data is available in the NCBI sequence read archive (SRA) with accession number: PRJNA648771
                            </p>
                           </div>
                           <div class="section" id="sec-17">
                            <h2 class="">
                             FUNDING
                            </h2>
                            <p id="p-48">
                             FA was supported by a PhD studentship from the Ministry of Education, Government of Saudi Arabia and KB was supported by Biological Sciences Research Council award BB/M009343/1to D.J.E and an ISSF award from The Wellcome Trust to the BSRC, University of St Andrews.
                            </p>
                           </div>
                           <div class="section" id="sec-18">
                            <h2 class="">
                             SUPPLEMENTARY FIGURES
                            </h2>
                            <div id="F7" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F7.large.jpg?width=800&height=600&carousel=1" title="Sequence alignments of modified templates. The modified Cluster 2 sequences were aligned against parental PV1 and PV3 sequences using Clustal Omega." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2084576083" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Sequence alignments of modified templates.</span> The modified Cluster 2 sequences were aligned against parental PV1 and PV3 sequences using Clustal Omega.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Fig S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F7.medium.gif" width="440" height="226"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Fig S1." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F7.medium.gif" width="440" height="226"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig S1." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Fig S1.
                              </span>
                              <span class="caption-title">
                               Sequence alignments of modified templates.
                              </span>
                              <p id="p-49" class="first-child">
                               The modified Cluster 2 sequences were aligned against parental PV1 and PV3 sequences using Clustal Omega.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F8" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F8.large.jpg?width=800&height=600&carousel=1" title="CRE-REP modified genome showed comparable characteristics to WT CRE-REP geneoms. (A) Replication kinetics for donor RNAs. HeLa cells were transfected with 200 ng RNA/well in 24-well plates and luciferase activity measured over an 8 h time course. Error bars represent the standard deviation of three experiments with samples assayed in triplicate. (B) Replication kinetics for acceptor RNAs. HeLa cells were infected with virus stock at MOI of 5 and virus-containing supernatants harvested over 24 h. Virus titres were determined by plaque assay on HeLa cells. Error bars represent the standard deviation of three experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2084576083" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">CRE-REP modified genome showed comparable characteristics to WT CRE-REP geneoms.</span> (A) Replication kinetics for donor RNAs. HeLa cells were transfected with 200 ng RNA/well in 24-well plates and luciferase activity measured over an 8 h time course. Error bars represent the standard deviation of three experiments with samples assayed in triplicate. (B) Replication kinetics for acceptor RNAs. HeLa cells were infected with virus stock at MOI of 5 and virus-containing supernatants harvested over 24 h. Virus titres were determined by plaque assay on HeLa cells. Error bars represent the standard deviation of three experiments.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Fig S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F8.medium.gif" width="440" height="196"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Fig S2." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F8.medium.gif" width="440" height="196"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig S2." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Fig S2.
                              </span>
                              <span class="caption-title">
                               CRE-REP modified genome showed comparable characteristics to WT CRE-REP geneoms.
                              </span>
                              <p id="p-50" class="first-child">
                               (A) Replication kinetics for donor RNAs. HeLa cells were transfected with 200 ng RNA/well in 24-well plates and luciferase activity measured over an 8 h time course. Error bars represent the standard deviation of three experiments with samples assayed in triplicate. (B) Replication kinetics for acceptor RNAs. HeLa cells were infected with virus stock at MOI of 5 and virus-containing supernatants harvested over 24 h. Virus titres were determined by plaque assay on HeLa cells. Error bars represent the standard deviation of three experiments.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F9" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F9.large.jpg?width=800&height=600&carousel=1" title="Quantification and characterisation of donor and acceptor RNAs in modified CRE-REP assays. (A) Recombinant yield from modified CRE-REP assays. RNAs were co-transfected into L929 cells in equimolar amounts and virus-containing supernatants were harvested 30 h post-transfection. Recombinant virus yields were determined by plaque assay on HeLa cells and expressed as a percentage of the WT assay. Error bars represent standard error of the mean of three co-transfection experiments. All modified assays were not significantly different to WT, as determined by unpaired t-test. (B) Plaque morphology comparison between recombinant viruses obtained from each of the CRE-REP assays." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2084576083" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Quantification and characterisation of donor and acceptor RNAs in modified CRE-REP assays.</span> (A) Recombinant yield from modified CRE-REP assays. RNAs were co-transfected into L929 cells in equimolar amounts and virus-containing supernatants were harvested 30 h post-transfection. Recombinant virus yields were determined by plaque assay on HeLa cells and expressed as a percentage of the WT assay. Error bars represent standard error of the mean of three co-transfection experiments. All modified assays were not significantly different to WT, as determined by unpaired t-test. (B) Plaque morphology comparison between recombinant viruses obtained from each of the CRE-REP assays.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Fig S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F9.medium.gif" width="440" height="199"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Fig S3." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F9.medium.gif" width="440" height="199"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig S3." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Fig S3.
                              </span>
                              <span class="caption-title">
                               Quantification and characterisation of donor and acceptor RNAs in modified CRE-REP assays.
                              </span>
                              <p id="p-51" class="first-child">
                               (A) Recombinant yield from modified CRE-REP assays. RNAs were co-transfected into L929 cells in equimolar amounts and virus-containing supernatants were harvested 30 h post-transfection. Recombinant virus yields were determined by plaque assay on HeLa cells and expressed as a percentage of the WT assay. Error bars represent standard error of the mean of three co-transfection experiments. All modified assays were not significantly different to WT, as determined by unpaired t-test. (B) Plaque morphology comparison between recombinant viruses obtained from each of the CRE-REP assays.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F10" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F10.large.jpg?width=800&height=600&carousel=1" title="Sequencing analysis of the co-infection technical replicate sample. The number of precise and imprecise (insertion and deletion) recombinants was plotted against the number of NGS reads (left panel). Each dot represents a unique recombinant." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2084576083" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Sequencing analysis of the co-infection technical replicate sample.</span> The number of precise and imprecise (insertion and deletion) recombinants was plotted against the number of NGS reads (left panel). Each dot represents a unique recombinant.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Fig S4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F10.medium.gif" width="440" height="404"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Fig S4." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F10.medium.gif" width="440" height="404"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig S4." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Fig S4.
                              </span>
                              <span class="caption-title">
                               Sequencing analysis of the co-infection technical replicate sample.
                              </span>
                              <p id="p-52" class="first-child">
                               The number of precise and imprecise (insertion and deletion) recombinants was plotted against the number of NGS reads (left panel). Each dot represents a unique recombinant.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F11" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F11.large.jpg?width=800&height=600&carousel=1" title="Design flowchart for RNA sequences with high or low levels of RNA structure. The input sequences were scrambled using the CDLR algorithm within the SSE package (reference 25). This method scrambles codon order, but maintains the open reading frame, the encoded polyprotein, the dinucleotide and codon frequencies. Scrambled sequences within +/-1% sequence divergence from the input sequence were selected. Each was analysed and ordered for inherent RNA structure by calculating the mean folding energy difference (MFED) between the CDLR scrambled sequences and 99 sequence-order randomised variants (see reference 34). Representative high and low structured variants, both exhibiting similar sequence identity to the native input poliovirus sequence, were selected from the 10 sequences with the highest or lowest level of intrinsic structure respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2084576083" data-figure-caption='<div class="highwire-markup">Design flowchart for RNA sequences with high or low levels of RNA structure. The input sequences were scrambled using the CDLR algorithm within the SSE package (reference 25). This method scrambles codon order, but maintains the open reading frame, the encoded polyprotein, the dinucleotide and codon frequencies. Scrambled sequences within +/-1% sequence divergence from the input sequence were selected. Each was analysed and ordered for inherent RNA structure by calculating the mean folding energy difference (MFED) between the CDLR scrambled sequences and 99 sequence-order randomised variants (see reference 34). Representative high and low structured variants, both exhibiting similar sequence identity to the native input poliovirus sequence, were selected from the 10 sequences with the highest or lowest level of intrinsic structure respectively.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Fig S5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F11.medium.gif" width="273" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Fig S5." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F11.medium.gif" width="273" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig S5." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Fig S5.
                              </span>
                              <p id="p-53" class="first-child">
                               Design flowchart for RNA sequences with high or low levels of RNA structure. The input sequences were scrambled using the CDLR algorithm within the SSE package (reference 25). This method scrambles codon order, but maintains the open reading frame, the encoded polyprotein, the dinucleotide and codon frequencies. Scrambled sequences within +/-1% sequence divergence from the input sequence were selected. Each was analysed and ordered for inherent RNA structure by calculating the mean folding energy difference (MFED) between the CDLR scrambled sequences and 99 sequence-order randomised variants (see reference 34). Representative high and low structured variants, both exhibiting similar sequence identity to the native input poliovirus sequence, were selected from the 10 sequences with the highest or lowest level of intrinsic structure respectively.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F12" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F12.large.jpg?width=800&height=600&carousel=1" title="Recombination junction definition. Donor and recipient sequences are shown in red and blue respectively. Precise junctions have no insertions or deletions with respect to the length of the parental sequences. Imprecise junctions contain insertions or have sequences deleted. The position of unambiguous junctions can be defined by the position of the join in the donor and acceptor sequences. Ambiguous junctions, whether in precise or imprecise junctions, cannot be exactly defined due to limited sequence identity between donor and acceptor at the junction, and are highlighted in black underlined text. The two thin vertical lines refer to start and end of the precise recombinants, to visually aid reading the figure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2084576083" data-figure-caption='<div class="highwire-markup">Recombination junction definition. Donor and recipient sequences are shown in red and blue respectively. Precise junctions have no insertions or deletions with respect to the length of the parental sequences. Imprecise junctions contain insertions or have sequences deleted. The position of unambiguous junctions can be defined by the position of the join in the donor and acceptor sequences. Ambiguous junctions, whether in precise or imprecise junctions, cannot be exactly defined due to limited sequence identity between donor and acceptor at the junction, and are highlighted in black underlined text. The two thin vertical lines refer to start and end of the precise recombinants, to visually aid reading the figure.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Fig S6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F12.medium.gif" width="440" height="368"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Fig S6." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F12.medium.gif" width="440" height="368"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig S6." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/18/2020.09.29.319285/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Fig S6.
                              </span>
                              <p id="p-54" class="first-child">
                               Recombination junction definition. Donor and recipient sequences are shown in red and blue respectively. Precise junctions have no insertions or deletions with respect to the length of the parental sequences. Imprecise junctions contain insertions or have sequences deleted. The position of unambiguous junctions can be defined by the position of the join in the donor and acceptor sequences. Ambiguous junctions, whether in precise or imprecise junctions, cannot be exactly defined due to limited sequence identity between donor and acceptor at the junction, and are highlighted in
                               <span class="underline">
                                black underlined
                               </span>
                               text. The two thin vertical lines refer to start and end of the precise recombinants, to visually aid reading the figure.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="T2" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2409623/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup">
                                 <a href="/highwire/markup/2409623/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                                <li class="download-ppt last">
                                 <a href="/highwire/powerpoint/2409623" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                  Download powerpoint
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Table S1.
                              </span>
                              <span class="caption-title">
                               Percentage sequence identity of structure modified sequences
                              </span>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="T3" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2409618/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup">
                                 <a href="/highwire/markup/2409618/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                                <li class="download-ppt last">
                                 <a href="/highwire/powerpoint/2409618" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                  Download powerpoint
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Table S2.
                              </span>
                              <span class="caption-title">
                               Recombination junctions isolated by PCR cloning and sequenced by Sanger-sequencing. (Del = Deletion, Prec = Precise, Ins = Insertion)
                              </span>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             Reference
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.1" data-doi="10.1073/pnas.85.16.5997">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hahn
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Western equine encephalitis virus is a recombinant virus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc Natl Acad Sci U S A
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  1988
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  85
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  16
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  5997
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6001
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHahn%26rft.auinit1%253DC.%2BS.%26rft.volume%253D85%26rft.issue%253D16%26rft.spage%253D5997%26rft.epage%253D6001%26rft.atitle%253DWestern%2Bequine%2Bencephalitis%2Bvirus%2Bis%2Ba%2Brecombinant%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.85.16.5997%26rft_id%253Dinfo%253Apmid%252F3413072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODUvMTYvNTk5NyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAyLzE4LzIwMjAuMDkuMjkuMzE5Mjg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.09.29.319285v2.2" data-doi="10.1038/nature12711">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ge
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  503
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7477
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  535
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  8
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D503%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12711%26rft_id%253Dinfo%253Apmid%252F24172901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature12711&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24172901&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F18%2F2020.09.29.319285.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000327464200045&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.09.29.319285v2.3" data-doi="10.1038/nature02746">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Genesis of a highly pathogenic and potentially pandemic H5N1 influenza virus in eastern Asia
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  430
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6996
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  209
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  13
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DK.%2BS.%26rft.volume%253D430%26rft.issue%253D6996%26rft.spage%253D209%26rft.epage%253D213%26rft.atitle%253DGenesis%2Bof%2Ba%2Bhighly%2Bpathogenic%2Band%2Bpotentially%2Bpandemic%2BH5N1%2Binfluenza%2Bvirus%2Bin%2Beastern%2BAsia.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02746%26rft_id%253Dinfo%253Apmid%252F15241415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature02746&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15241415&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F18%2F2020.09.29.319285.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000222470600044&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.4" data-doi="10.1038/nature08182">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Origins and evolutionary genomics of the 2009 swine-origin H1N1 influenza A epidemic
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  459
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7250
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  1122
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  5
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSmith%26rft.auinit1%253DG.%2BJ.%26rft.volume%253D459%26rft.issue%253D7250%26rft.spage%253D1122%26rft.epage%253D1125%26rft.atitle%253DOrigins%2Band%2Bevolutionary%2Bgenomics%2Bof%2Bthe%2B2009%2Bswine-origin%2BH1N1%2Binfluenza%2BA%2Bepidemic.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08182%26rft_id%253Dinfo%253Apmid%252F19516283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature08182&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19516283&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F18%2F2020.09.29.319285.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000267636700042&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.5" data-doi="10.1128/JVI.73.11.8958-8965.1999">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gmyl
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Nonreplicative RNA recombination in poliovirus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Virol
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  1999
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  73
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  8958
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  65
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DGmyl%26rft.auinit1%253DA.%2BP.%26rft.volume%253D73%26rft.issue%253D11%26rft.spage%253D8958%26rft.epage%253D8965%26rft.atitle%253DNonreplicative%2BRNA%2BRecombination%2Bin%2BPoliovirus%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.73.11.8958-8965.1999%26rft_id%253Dinfo%253Apmid%252F10516001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3My8xMS84OTU4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDIvMTgvMjAyMC4wOS4yOS4zMTkyODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.6" data-doi="10.1261/rna.5111803">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gmyl
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Nonreplicative homologous RNA recombination: promiscuous joining of RNA pieces?
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Rna
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  1221
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  31
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DRNA%26rft.stitle%253DRNA%26rft.aulast%253DGMYL%26rft.auinit1%253DA.%2BP.%26rft.volume%253D9%26rft.issue%253D10%26rft.spage%253D1221%26rft.epage%253D1231%26rft.atitle%253DNonreplicative%2Bhomologous%2BRNA%2Brecombination%253A%2BPromiscuous%2Bjoining%2Bof%2BRNA%2Bpieces%253F%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.5111803%26rft_id%253Dinfo%253Apmid%252F13130136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjkvMTAvMTIyMSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAyLzE4LzIwMjAuMDkuMjkuMzE5Mjg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.7" data-doi="10.1093/nar/16.24.11705">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   King
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Preferred sites of recombination in poliovirus RNA: an analysis of 40 intertypic cross-over sequences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  1988
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  24
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  11705
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  23
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKing%26rft.auinit1%253DA.%2BM.Q.%26rft.volume%253D16%26rft.issue%253D24%26rft.spage%253D11705%26rft.epage%253D11723%26rft.atitle%253DPreferred%2Bsites%2Bof%2Brecombination%2Bin%2Bpoliovirus%2BRNA%253A%2Ban%2Banalysis%2Bof%2B40%2Bintertypic%2Bcross-over%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F16.24.11705%26rft_id%253Dinfo%253Apmid%252F2463516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/16.24.11705&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=2463516&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F18%2F2020.09.29.319285.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1988R615400022&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.8" data-doi="10.1016/0092-8674(86)90600-8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kirkegaard
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Baltimore
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  The mechanism of RNA recombination in poliovirus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  1986
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  47
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  433
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  43
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKirkegaard%26rft.auinit1%253DK.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DThe%2Bmechanism%2Bof%2BRNA%2Brecombination%2Bin%2Bpoliovirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252886%252990600-8%26rft_id%253Dinfo%253Apmid%252F3021340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/0092-8674(86)90600-8&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=3021340&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F18%2F2020.09.29.319285.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1986E805900013&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.09.29.319285v2.9" data-doi="10.1093/nar/gkw567">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Woodman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Biochemical and genetic analysis of the role of the viral polymerase in enterovirus recombination
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  44
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  14
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  6883
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  95
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw567%26rft_id%253Dinfo%253Apmid%252F27317698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkw567&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27317698&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F18%2F2020.09.29.319285.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nagy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.D.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bujarski
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Efficient system of homologous RNA recombination in brome mosaic virus: sequence and structure requirements and accuracy of crossovers
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Virol
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  1995
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  69
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  131
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  40
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DNagy%26rft.auinit1%253DP.%2BD.%26rft.volume%253D69%26rft.issue%253D1%26rft.spage%253D131%26rft.epage%253D140%26rft.atitle%253DEfficient%2Bsystem%2Bof%2Bhomologous%2BRNA%2Brecombination%2Bin%2Bbrome%2Bmosaic%2Bvirus%253A%2Bsequence%2Band%2Bstructure%2Brequirements%2Band%2Baccuracy%2Bof%2Bcrossovers%26rft_id%253Dinfo%253Apmid%252F7983703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjY5LzEvMTMxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDIvMTgvMjAyMC4wOS4yOS4zMTkyODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sergiescu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Aubert-Combiescu
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Crainic
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Recombination between guanidine-resistant and dextran sulfate-resistant mutants of type 1 poliovirus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Virol
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  1969
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  326
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  30
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DSergiescu%26rft.auinit1%253DD.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253D326%26rft.epage%253D330%26rft.atitle%253DRecombination%2BBetween%2BGuanidine-resistant%2Band%2BDextran%2BSulfate-resistant%2BMutants%2Bof%2BType%2B1%2BPoliovirus%26rft_id%253Dinfo%253Apmid%252F4305674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjc6IjMvMy8zMjYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMi8xOC8yMDIwLjA5LjI5LjMxOTI4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   White
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.A.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Morris
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  RNA determinants of junction site selection in RNA virus recombinants and defective interfering RNAs
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  RNA
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  1995
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  1
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  1029
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  40
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DRNA%26rft.stitle%253DRNA%26rft.aulast%253DWhite%26rft.auinit1%253DK.%2BA.%26rft.volume%253D1%26rft.issue%253D10%26rft.spage%253D1029%26rft.epage%253D1040%26rft.atitle%253DRNA%2Bdeterminants%2Bof%2Bjunction%2Bsite%2Bselection%2Bin%2BRNA%2Bvirus%2Brecombinants%2Band%2Bdefective%2Binterfering%2BRNAs%26rft_id%253Dinfo%253Apmid%252F8595558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjEvMTAvMTAyOSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAyLzE4LzIwMjAuMDkuMjkuMzE5Mjg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.09.29.319285v2.13" data-doi="10.1093/emboj/17.8.2392">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nagy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Simon
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Dissecting RNA recombination in vitro: role of RNA sequences and the viral replicase
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  EMBO J
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  1998
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  2392
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  403
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DNagy%26rft.auinit1%253DP.%2BD.%26rft.volume%253D17%26rft.issue%253D8%26rft.spage%253D2392%26rft.epage%253D2403%26rft.atitle%253DDissecting%2BRNA%2Brecombination%2Bin%2Bvitro%253A%2Brole%2Bof%2BRNA%2Bsequences%2Band%2Bthe%2Bviral%2Breplicase%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F17.8.2392%26rft_id%253Dinfo%253Apmid%252F9545250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIxNy84LzIzOTIiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMi8xOC8yMDIwLjA5LjI5LjMxOTI4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.09.29.319285v2.14" data-doi="10.1006/jmbi.1994.0050">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carpenter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Involvement of a stem-loop structure in the location of junction sites in viral RNA recombination
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Mol Biol
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  1995
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  245
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  608
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  22
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DCarpenter%26rft.auinit1%253DC.%2BD.%26rft.volume%253D245%26rft.issue%253D5%26rft.spage%253D608%26rft.epage%253D622%26rft.atitle%253DInvolvement%2Bof%2Ba%2Bstem-loop%2Bstructure%2Bin%2Bthe%2Blocation%2Bof%2Bjunction%2Bsites%2Bin%2Bviral%2BRNA%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1994.0050%26rft_id%253Dinfo%253Apmid%252F7844830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1006/jmbi.1994.0050&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=7844830&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F18%2F2020.09.29.319285.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1995QE94000014&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.09.29.319285v2.15" data-doi="10.1128/JVI.74.9.4214-4219.2000">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bruyere
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Frequent homologous recombination events between molecules of one RNA component in a multipartite RNA virus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Virol
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  74
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  4214
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  9
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DBruyere%26rft.auinit1%253DA.%26rft.volume%253D74%26rft.issue%253D9%26rft.spage%253D4214%26rft.epage%253D4219%26rft.atitle%253DFrequent%2BHomologous%2BRecombination%2BEvents%2Bbetween%2BMolecules%2Bof%2BOne%2BRNA%2BComponent%2Bin%2Ba%2BMultipartite%2BRNA%2BVirus%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.74.9.4214-4219.2000%26rft_id%253Dinfo%253Apmid%252F10756034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc0LzkvNDIxNCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAyLzE4LzIwMjAuMDkuMjkuMzE5Mjg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.09.29.319285v2.16" data-doi="10.1038/321528a0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bujarski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.J.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kaesberg
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Genetic recombination between RNA components of a multipartite plant virus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  1986
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  321
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6069
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  528
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  31
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBujarski%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D321%26rft.issue%253D6069%26rft.spage%253D528%26rft.epage%253D531%26rft.atitle%253DGenetic%2Brecombination%2Bbetween%2BRNA%2Bcomponents%2Bof%2Ba%2Bmultipartite%2Bplant%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F321528a0%26rft_id%253Dinfo%253Apmid%252F2423879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/321528a0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=2423879&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F18%2F2020.09.29.319285.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.17">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rowe
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Generation of coronavirus spike deletion variants by high-frequency recombination at regions of predicted RNA secondary structure
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Virol
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  1997
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  71
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  6183
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  90
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DRowe%26rft.auinit1%253DC.%2BL.%26rft.volume%253D71%26rft.issue%253D8%26rft.spage%253D6183%26rft.epage%253D6190%26rft.atitle%253DGeneration%2Bof%2Bcoronavirus%2Bspike%2Bdeletion%2Bvariants%2Bby%2Bhigh-frequency%2Brecombination%2Bat%2Bregions%2Bof%2Bpredicted%2BRNA%2Bsecondary%2Bstructure%26rft_id%253Dinfo%253Apmid%252F9223514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjcxLzgvNjE4MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAyLzE4LzIwMjAuMDkuMjkuMzE5Mjg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.18" data-doi="10.1371/journal.ppat.1004191">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lowry
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Recombination in enteroviruses is a biphasic replicative process involving the generation of greater-than genome length ‘imprecise’ intermediates
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Pathog
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  e1004191
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D10%26rft.spage%253De1004191%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1004191%26rft_id%253Dinfo%253Apmid%252F24945141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1004191&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24945141&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F18%2F2020.09.29.319285.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.19" data-doi="10.1099/vir.0.80442-0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Egger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bienz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Intracellular location and translocation of silent and active poliovirus replication complexes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Gen Virol
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  86
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  Pt 3
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  707
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  18
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DEgger%26rft.auinit1%253DD.%26rft.volume%253D86%26rft.issue%253D3%26rft.spage%253D707%26rft.epage%253D718%26rft.atitle%253DIntracellular%2Blocation%2Band%2Btranslocation%2Bof%2Bsilent%2Band%2Bactive%2Bpoliovirus%2Breplication%2Bcomplexes%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.80442-0%26rft_id%253Dinfo%253Apmid%252F15722531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1099/vir.0.80442-0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15722531&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F18%2F2020.09.29.319285.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000227367500021&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.09.29.319285v2.20">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-source">
                                  Bowtie 2: Manual
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ; Available from:
                                 <a href="http://bowtie-bio.sourceforge.net/bowtie2/manual.shtml#bowtie2-options-score-min">
                                  http://bowtie-bio.sourceforge.net/bowtie2/manual.shtml#bowtie2-options-score-min
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.21" data-doi="10.1093/nar/gkt916">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Routh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Johnson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Discovery of functional genomic motifs in viruses with ViReMa-a Virus Recombination Mapper-for analysis of next-generation sequencing data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  42
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  e11
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt916%26rft_id%253Dinfo%253Apmid%252F24137010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkt916&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24137010&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F18%2F2020.09.29.319285.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alnaji
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Sequencing Framework for the Sensitive Detection and Precise Mapping of Defective Interfering Particle-Associated Deletions across Influenza A and B Viruses
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Virol
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  93
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.23" data-doi="10.1093/nar/28.8.1714">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Figlerowicz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Role of RNA structure in non-homologous recombination between genomic molecules of brome mosaic virus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  1714
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  23
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DFiglerowicz%26rft.auinit1%253DM.%26rft.volume%253D28%26rft.issue%253D8%26rft.spage%253D1714%26rft.epage%253D1723%26rft.atitle%253DRole%2Bof%2BRNA%2Bstructure%2Bin%2Bnon-homologous%2Brecombination%2Bbetween%2Bgenomic%2Bmolecules%2Bof%2Bbrome%2Bmosaic%2Bvirus%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.8.1714%26rft_id%253Dinfo%253Apmid%252F10734190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/28.8.1714&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10734190&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F18%2F2020.09.29.319285.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000086699400007&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.24" data-doi="10.1126/science.8484119">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cascone
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Haydar
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Simon
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Sequences and structures required for recombination between virus-associated RNAs
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  1993
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  260
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5109
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  801
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  5
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCascone%26rft.auinit1%253DP.%26rft.volume%253D260%26rft.issue%253D5109%26rft.spage%253D801%26rft.epage%253D805%26rft.atitle%253DSequences%2Band%2Bstructures%2Brequired%2Bfor%2Brecombination%2Bbetween%2Bvirus-associated%2BRNAs%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.8484119%26rft_id%253Dinfo%253Apmid%252F8484119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNjAvNTEwOS84MDEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMi8xOC8yMDIwLjA5LjI5LjMxOTI4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               25.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.25" data-doi="10.1261/rna.7640104">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Simmonds
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tuplin
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Evans
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Detection of genome-scale ordered RNA structure (GORS) in genomes of positive-stranded RNA viruses: Implications for virus evolution and host persistence
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  RNA
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  1337
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  51
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.7640104%26rft_id%253Dinfo%253Apmid%252F15273323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjEwLzkvMTMzNyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAyLzE4LzIwMjAuMDkuMjkuMzE5Mjg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               26.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bentley
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Imprecise recombinant viruses evolve via a fitness-driven, iterative process of polymerase template-switching events
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Pathog
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  e1009676
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D17%26rft.spage%253De1009676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               27.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.27" data-doi="10.1038/nrmicro2614">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Simon-Loriere
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Holmes
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Why do RNA viruses recombine?
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Rev Microbiol
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  617
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  26
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DSimon-Loriere%26rft.auinit1%253DE.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D617%26rft.epage%253D626%26rft.atitle%253DWhy%2Bdo%2BRNA%2Bviruses%2Brecombine%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2614%26rft_id%253Dinfo%253Apmid%252F21725337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrmicro2614&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21725337&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F18%2F2020.09.29.319285.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               28.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.28" data-doi="10.1099/0022-1317-80-10-2535">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Worobey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Holmes
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Evolutionary aspects of recombination in RNA viruses
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Gen Virol
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  1999
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  80
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  Pt 10
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  2535
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  43
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DWorobey%26rft.auinit1%253DM.%26rft.volume%253D80%26rft.issue%253D10%26rft.spage%253D2535%26rft.epage%253D2543%26rft.atitle%253DEvolutionary%2Baspects%2Bof%2Brecombination%2Bin%2BRNA%2Bviruses%26rft_id%253Dinfo%253Adoi%252F10.1099%252F0022-1317-80-10-2535%26rft_id%253Dinfo%253Apmid%252F10573145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1099/0022-1317-80-10-2535&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10573145&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F18%2F2020.09.29.319285.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000082863400001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               29.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.29" data-doi="10.1371/journal.pbio.3000003">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Duffy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Why are RNA virus mutation rates so damn high?
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Biol
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  e3000003
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol%26rft.volume%253D16%26rft.spage%253De3000003%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.3000003%26rft_id%253Dinfo%253Apmid%252F30102691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pbio.3000003&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30102691&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F18%2F2020.09.29.319285.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               30.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kempf
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Picornavirus RNA Recombination Counteracts Error Catastrophe
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Virol
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  93
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  14
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               31.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.31" data-doi="10.1073/pnas.111085598">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Crotty
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cameron
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Andino
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  RNA virus error catastrophe: direct molecular test by using ribavirin
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc Natl Acad Sci U S A
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  98
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  6895
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  900
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.111085598%26rft_id%253Dinfo%253Apmid%252F11371613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTgvMTIvNjg5NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAyLzE4LzIwMjAuMDkuMjkuMzE5Mjg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               32.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.09.29.319285v2.32" data-doi="10.1099/jgv.0.001142">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bentley
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Evans
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Mechanisms and consequences of positive-strand RNA virus recombination
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Gen Virol
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  99
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  1345
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1356
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft.volume%253D99%26rft.spage%253D1345%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fjgv.0.001142%26rft_id%253Dinfo%253Apmid%252F30156526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1099/jgv.0.001142&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30156526&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F18%2F2020.09.29.319285.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               33.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.09.29.319285v2.33">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Muslin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Recombination in Enteroviruses, a Multi-Step Modular Evolutionary Process
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Viruses
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               34.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.319285v2.34" data-doi="10.1186/1756-0500-5-50">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Simmonds
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  SSE: a nucleotide and amino acid sequence analysis platform
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Res Notes
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 : p.
                                 <span class="cit-fpage">
                                  50
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DBMC%2BRes%2BNotes%26rft.aulast%253DSimmonds%26rft.auinit1%253DP.%26rft.volume%253D5%26rft.spage%253D50%26rft.epage%253D50%26rft.atitle%253DSSE%253A%2Ba%2Bnucleotide%2Band%2Bamino%2Bacid%2Bsequence%2Banalysis%2Bplatform.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-0500-5-50%26rft_id%253Dinfo%253Apmid%252F22264264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/1756-0500-5-50&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22264264&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F18%2F2020.09.29.319285.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.09.29.319285v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.05.11.443638v4" title="De novo identification of maximally deregulated subnetworks based on multi-omics data with DeRegNet" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2021.09.01.458560v2" title="A genetically linked pair of NLR immune receptors show contrasting patterns of evolution" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted February 18, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.09.29.319285v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.09.29.319285v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.09.29.319285v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   Generated randomly and selected functionally? The nature of enterovirus recombination
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2407740"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-n68g4qEEiDk-rr2FCA4c24I8jlAKI_j-fWowxPpvsWM"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484480684"/>
                                   <input type="hidden" name="captcha_token" value="2d2627f5fb5986100d7d7a3649817a93"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2407740--3" data-node-nid="2407740" id="node-2407740--41976948774" data-pisa="biorxiv;2020.09.29.319285v2" data-pisa-master="biorxiv;2020.09.29.319285" data-seqnum="2407740" data-apath="/biorxiv/early/2022/02/18/2020.09.29.319285.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Generated randomly and selected functionally? The nature of enterovirus recombination
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Fadi G.
                                  </span>
                                  <span class="nlm-surname">
                                   Alnaji
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Kirsten
                                  </span>
                                  <span class="nlm-surname">
                                   Bentley
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Ashley
                                  </span>
                                  <span class="nlm-surname">
                                   Pearson
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Andrew
                                  </span>
                                  <span class="nlm-surname">
                                   Woodman
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Jonathan
                                  </span>
                                  <span class="nlm-surname">
                                   Moore
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Helen
                                  </span>
                                  <span class="nlm-surname">
                                   Fox
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">
                                   Andrew
                                  </span>
                                  <span class="nlm-surname">
                                   MacAdam
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="7">
                                  <span class="nlm-given-names">
                                   David J.
                                  </span>
                                  <span class="nlm-surname">
                                   Evans
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.09.29.319285;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.09.29.319285
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.09.29.319285v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.319285v2&title=Generated%20randomly%20and%20selected%20functionally%3F%20The%20nature%20of%20enterovirus%20recombination" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.319285v2&title=Generated%20randomly%20and%20selected%20functionally%3F%20The%20nature%20of%20enterovirus%20recombination" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.319285v2&text=Generated%20randomly%20and%20selected%20functionally%3F%20The%20nature%20of%20enterovirus%20recombination" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.319285v2&t=Generated%20randomly%20and%20selected%20functionally%3F%20The%20nature%20of%20enterovirus%20recombination" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.319285v2&title=Generated%20randomly%20and%20selected%20functionally%3F%20The%20nature%20of%20enterovirus%20recombination" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.319285v2&title=Generated%20randomly%20and%20selected%20functionally%3F%20The%20nature%20of%20enterovirus%20recombination&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.319285v2&title=Generated%20randomly%20and%20selected%20functionally%3F%20The%20nature%20of%20enterovirus%20recombination" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2407740--5" data-node-nid="2407740" id="citation-node-2407740--61413796037" data-pisa="biorxiv;2020.09.29.319285v2" data-pisa-master="biorxiv;2020.09.29.319285" data-seqnum="2407740" data-apath="/biorxiv/early/2022/02/18/2020.09.29.319285.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Generated randomly and selected functionally? The nature of enterovirus recombination
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Fadi G.
                                      </span>
                                      <span class="nlm-surname">
                                       Alnaji
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Kirsten
                                      </span>
                                      <span class="nlm-surname">
                                       Bentley
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Ashley
                                      </span>
                                      <span class="nlm-surname">
                                       Pearson
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Andrew
                                      </span>
                                      <span class="nlm-surname">
                                       Woodman
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Jonathan
                                      </span>
                                      <span class="nlm-surname">
                                       Moore
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Helen
                                      </span>
                                      <span class="nlm-surname">
                                       Fox
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">
                                       Andrew
                                      </span>
                                      <span class="nlm-surname">
                                       MacAdam
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="7">
                                      <span class="nlm-given-names">
                                       David J.
                                      </span>
                                      <span class="nlm-surname">
                                       Evans
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.09.29.319285;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.09.29.319285
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Generated%20randomly%20and%20selected%20functionally%3F%20The%20nature%20of%20enterovirus%20recombination&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Alnaji%2C+Fadi+G.&rft.au=Bentley%2C+Kirsten&rft.au=Pearson%2C+Ashley&rft.au=Woodman%2C+Andrew&rft.au=Moore%2C+Jonathan&rft.au=Fox%2C+Helen&rft.au=MacAdam%2C+Andrew&rft.au=Evans%2C+David+J.">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2407740/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2407740/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2407740/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2407740/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2407740/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2407740/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2407740/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2407740/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2407740/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2407740/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2407740/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2407740/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.319285v2&count=horizontal&via=&text=Generated%20randomly%20and%20selected%20functionally%3F%20The%20nature%20of%20enterovirus%20recombination&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.319285v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.319285v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.09.29.319285v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Microbiology
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23507)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11824)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11440)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15189)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47703)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
