<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.02.07.938670v3.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.02.07.938670v3.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.02.07.938670v3.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/embed/graphic-2.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/04/04/2020.02.07.938670.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.07.938670v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Dynamic, adaptive sampling during nanopore sequencing using Bayesian experimental design"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.07.938670"/>
  <meta name="DC.Date" content="2022-04-04"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="One particularly promising feature of nanopore sequencing is the ability to reject reads, enabling real-time selection of molecules without complex sample preparation. This is based on the idea of deciding whether a molecule warrants full sequencing depending on reading a small initial part. Previously, such decisions have been based on a priori determination of which regions of the genome were considered of interest. Instead, here we consider more general and complex strategies that incorporate already-observed data in order to optimize the rejection strategy and maximise information gain from the sequencing process. For example, in the presence of coverage bias redistributing data from areas of high to areas of low coverage would be desirable.

We present BOSS-RUNS, a mathematical and algorithmic framework to calculate the expected benefit of new reads and generate dynamically updated decision strategies for nanopore sequencing. During sequencing, in real time, we quantify the current uncertainty at each site of one or multiple reference genomes, and for each novel DNA fragment being sequenced we decide whether the potential decrease in uncertainty at the sites it will most likely cover warrants reading it in its entirety. This dynamic, adaptive sampling allows real-time focus of sequencing efforts onto areas of highest benefit.

We demonstrate the effectiveness of BOSS-RUNS by mitigating coverage bias across and within the species of a microbial community. Additionally, we show that our approach leads to improved variant calling due to its ability to sample more data at the most relevant genomic positions.

### Competing Interest Statement

ML was a member of the ONT MinION access program and has received free flow cells and sequencing reagents in the past. ML has received reimbursement for travel, accommodation and conference fees to speak at events organized by ONT. EB is a paid consultant to ONT and a small-scale equity and options holder in ONT."/>
  <meta name="DC.Contributor" content="Lukas Weilguny"/>
  <meta name="DC.Contributor" content="Nicola De Maio"/>
  <meta name="DC.Contributor" content="Rory Munro"/>
  <meta name="DC.Contributor" content="Charlotte Manser"/>
  <meta name="DC.Contributor" content="Ewan Birney"/>
  <meta name="DC.Contributor" content="Matt Loose"/>
  <meta name="DC.Contributor" content="Nick Goldman"/>
  <meta name="article:published_time" content="2022-04-04"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Dynamic, adaptive sampling during nanopore sequencing using Bayesian experimental design"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>One particularly promising feature of nanopore sequencing is the ability to reject reads, enabling real-time selection of molecules without complex sample preparation. This is based on the idea of deciding whether a molecule warrants full sequencing depending on reading a small initial part. Previously, such decisions have been based on <i>a priori</i> determination of which regions of the genome were considered of interest. Instead, here we consider more general and complex strategies that incorporate already-observed data in order to optimize the rejection strategy and maximise information gain from the sequencing process. For example, in the presence of coverage bias redistributing data from areas of high to areas of low coverage would be desirable.</p><p>We present BOSS-RUNS, a mathematical and algorithmic framework to calculate the expected benefit of new reads and generate dynamically updated decision strategies for nanopore sequencing. During sequencing, in real time, we quantify the current uncertainty at each site of one or multiple reference genomes, and for each novel DNA fragment being sequenced we decide whether the potential decrease in uncertainty at the sites it will most likely cover warrants reading it in its entirety. This dynamic, adaptive sampling allows real-time focus of sequencing efforts onto areas of highest benefit.</p><p>We demonstrate the effectiveness of BOSS-RUNS by mitigating coverage bias across and within the species of a microbial community. Additionally, we show that our approach leads to improved variant calling due to its ability to sample more data at the most relevant genomic positions.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.07.938670v3"/>
  <meta name="citation_id" content="2020.02.07.938670v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.07.938670v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.07.938670v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.07.938670v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.07.938670"/>
  <meta name="citation_num_pages" content="39"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.07.938670"/>
  <meta name="citation_author" content="Lukas Weilguny"/>
  <meta name="citation_author_institution" content="European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Genome Campus"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6459-0431"/>
  <meta name="citation_author" content="Nicola De Maio"/>
  <meta name="citation_author_institution" content="European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Genome Campus"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1776-8564"/>
  <meta name="citation_author" content="Rory Munro"/>
  <meta name="citation_author_institution" content="DeepSeq, School of Life Sciences, Queen’s Medical Centre, University of Nottingham"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3158-0303"/>
  <meta name="citation_author" content="Charlotte Manser"/>
  <meta name="citation_author_institution" content="European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Genome Campus"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4125-8900"/>
  <meta name="citation_author" content="Ewan Birney"/>
  <meta name="citation_author_institution" content="European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Genome Campus"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8314-8497"/>
  <meta name="citation_author" content="Matt Loose"/>
  <meta name="citation_author_institution" content="DeepSeq, School of Life Sciences, Queen’s Medical Centre, University of Nottingham"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5264-0929"/>
  <meta name="citation_author" content="Nick Goldman"/>
  <meta name="citation_author_institution" content="European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Genome Campus"/>
  <meta name="citation_author_email" content="goldman@ebi.ac.uk"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8486-2211"/>
  <meta name="citation_reference" content="Barbitoff YA, Polev DE, Glotov AS, Serebryakova EA, Shcherbakova IV, Kiselev AM, Kostareva AA, Glotov OS, & Predeus AV. (2020). Systematic dissection of biases in whole-exome and whole-genome sequencing reveals major determinants of coding sequence coverage. Scientific Reports, 10(1), Article number: 2057. https://doi.org/10.1038/s41598-020-59026-y"/>
  <meta name="citation_reference" content="Bird AP. (1986). CpG-rich islands and the function of DNA methylation. Nature, 321 (6067), 209–213. https://doi.org/10.1038/321209a0"/>
  <meta name="citation_reference" content="Boykin L, Ghalab A, Marchi BRD, Savill A, Wainaina JM, Kinene T, Lamb S, Rodrigues M, Kehoe M, Ndunguru J, Tairo F, Sseruwagi P, Kayuki C, Mark D, Erasto J, Bachwenkizi H, Alicai T, Okao-Okuja G, Abridrabo P,… Kiarie S. (2018). Real time portable genome sequencing for global food security [version 1; peer review: 1 approved, 1 approved with reservations]. F1000Research, 7:1101. https://doi.org/10.12688/f1000research.15507.1"/>
  <meta name="citation_reference" content="Brinkerhoff H, Kang ASW, Liu J, Aksimentiev A, & Dekker C. (2021). Multiple rereads of single proteins at single–amino acid resolution using nanopores. Science, 374 (6574), 1509–1513. https://doi.org/10.1126/science.abl4381"/>
  <meta name="citation_reference" content="Broad Institute. (2019). Picard toolkit. https://broadinstitute.github.io/picard/"/>
  <meta name="citation_reference" content="Brockhurst MA, Harrison E, Hall JPJ, Richards T, McNally A, & MacLean C. (2019). The ecology and evolution of pangenomes. Current Biology, 29(20), R1094–R1103. https://doi.org/10.1016/j.cub.2019.08.012"/>
  <meta name="citation_reference" content="Chaloner K, & Verdinelli I. (1995). Bayesian experimental design: a review. Statistical Science, 10(3), 273–304. https://doi.org/10.1214/ss/1177009939"/>
  <meta name="citation_reference" content="Cleary JG, Braithwaite R, Gaastra K, Hilbush BS, Inglis S, Irvine SA, Jackson A, Littin R, Rathod M, Ware D, Zook JM, Trigg L, & De La Vega FM. (2015). Comparing variant call files for performance benchmarking of next-generation sequencing variant calling pipelines. bioRxiv. https://doi.org/10.1101/023754"/>
  <meta name="citation_reference" content="Colquhoun RM, Hall MB, Lima L, Roberts LW, Malone KM, Hunt M, Letcher B, Hawkey J, George S, Pankhurst L, & Iqbal Z. (2021). Pandora: nucleotide-resolution bacterial pan-genomics with reference graphs. Genome Biology, 22(1), Article number: 267. https://doi.org/10.1186/s13059-021-02473-1"/>
  <meta name="citation_reference" content="Danecek P, Bonfield JK, Liddle J, Marshall J, Ohan V, Pollard MO, Whitwham A, Keane T, McCarthy SA, Davies RM, & Li H. (2021). Twelve years of SAMtools and BCFtools. GigaScience, 10(2), giab008. https://doi.org/10.1093/gigascience/giab008"/>
  <meta name="citation_reference" content="Deamer D, Akeson M, & Branton D. (2016). Three decades of nanopore sequencing. Nature Biotechnology, 34 (5), 518–524. https://doi.org/10.1038/nbt.3423"/>
  <meta name="citation_reference" content="Djirackor L, Halldorsson S, Niehusmann P, Leske H, Capper D, Kuschel LP, Pahnke J, Due-Tønnessen BJ, Langmoen IA, Sandberg CJ, Euskirchen P, & Vik-Mo EO. (2021). Intraoperative DNA methylation classification of brain tumors impacts neurosurgical strategy. Neuro-Oncology Advances, 3(1), vdab149. https://doi.org/10.1093/noajnl/vdab149"/>
  <meta name="citation_reference" content="Ehrlich M, Gama-Sosa MA, Huang LH, Midgett RM, Kuo KC, McCune RA, & Gehrke C. (1982). Amount and distribution of 5-methylcytosine in human DNA from different types of tissues of cells. Nucleic Acids Research, 10(8), 2709–2721. https://doi.org/10.1093/nar/10.8.2709"/>
  <meta name="citation_reference" content="Garalde DR, Snell EA, Jachimowicz D, Sipos B, Lloyd JH, Bruce M, Pantic N, Admassu T, James P, Warland A, Jordan M, Ciccone J, Serra S, Keenan J, Martin S, McNeill L, Wallace EJ, Jayasinghe L, Wright C,… Turner DJ. (2018). Highly parallel direct RNA sequencing on an array of nanopores. Nature Methods, 15(3), 201–206. https://doi.org/10.1038/nmeth.4577"/>
  <meta name="citation_reference" content="Garrison E, Kronenberg ZN, Dawson ET, Pedersen BS, & Prins P. (2021). Vcflib and tools for processing the VCF variant call format. bioRxiv. https://doi.org/10.1101/2021.05.21.445151"/>
  <meta name="citation_reference" content="Garrison E, & Marth G. (2012). Haplotype-based variant detection from short-read sequencing. arXiv. https://doi.org/10.48550/ARXIV.1207.3907"/>
  <meta name="citation_reference" content="Gershow M, & Golovchenko JA. (2007). Recapturing and trapping single molecules with a solidstate nanopore. Nature Nanotechnology, 2(12), 775–779. https://doi.org/10.1038/nnano.2007.381"/>
  <meta name="citation_reference" content="Goodwin S, Gurtowski J, Ethe-Sayers S, Deshpande P, Schatz MC, & McCombie WR. (2015). Oxford nanopore sequencing, hybrid error correction, and de novo assembly of a eukaryotic genome. Genome Research, 25(11), 1750–1756. https://doi.org/10.1101/gr.191395.115"/>
  <meta name="citation_reference" content="Harris CR, Millman KJ, van der Walt SJ, Gommers R, Virtanen P, Cournapeau D, Wieser E, Taylor J, Berg S, Smith NJ, Kern R, Picus M, Hoyer S, van Kerkwijk MH, Brett M, Haldane A, del Río JF, Wiebe M, Peterson P,… Oliphant TE. (2020). Array programming with NumPy. Nature, 585(7825), 357–362. https://doi.org/10.1038/s41586-020-2649-2"/>
  <meta name="citation_reference" content="Jain M, Koren S, Miga KH, Quick J, Rand AC, Sasani TA, Tyson JR, Beggs AD, Dilthey AT, Fiddes IT, Malla S, Marriott H, Nieto T, O’Grady J, Olsen HE, Pedersen BS, Rhie A, Richardson H, Quinlan AR,… Loose M. (2018). Nanopore sequencing and assembly of a human genome with ultra-long reads. Nature Biotechnology, 36(4), 338–345. https://doi.org/10.1038/nbt.4060"/>
  <meta name="citation_reference" content="Jones N, & Pevzner P. (2004). An Introduction to Bioinformatics Algorithms. MIT Press, Cambridge, MA. https://mitpress.mit.edu/books/introduction-bioinformatics-algorithms"/>
  <meta name="citation_reference" content="Kovaka S, Fan Y, Ni B, Timp W, & Schatz MC. (2021). Targeted nanopore sequencing by real-time mapping of raw electrical signal with UNCALLED. Nature Biotechnology, 39 (4), 431–441. https://doi.org/10.1038/s41587-020-0731-9"/>
  <meta name="citation_reference" content="Krishnakumar R, Sinha A, Bird SW, Jayamohan H, Edwards HS, Schoeniger JS, Patel KD, Branda SS, & Bartsch MS. (2018). Systematic and stochastic influences on the performance of the minion nanopore sequencer across a range of nucleotide bias. Scientific Reports, 8(1), Article number: 3159. https://doi.org/10.1038/s41598-018-21484-w"/>
  <meta name="citation_reference" content="Kullback S, & Leibler RA. (1951). On information and sufficiency. The Annals of Mathematical Statistics, 22(1), 79–86. https://doi.org/10.1214/aoms/1177729694"/>
  <meta name="citation_reference" content="Lam SK, Pitrou A, & Seibert S. (2015). Numba: a LLVM-based Python JIT compiler. Proceedings of the Second Workshop on the LLVM Compiler Infrastructure in HPC, 7:1–6. https://doi.org/10.1145/2833157.2833162"/>
  <meta name="citation_reference" content="Lee I, Razaghi R, Gilpatrick T, Molnar M, Gershman A, Sadowski N, Sedlazeck FJ, Hansen KD, Simpson JT, & Timp W. (2020). Simultaneous profiling of chromatin accessibility and methylation on human cell lines with nanopore sequencing. Nature Methods, 17(12), 1191–1199. https://doi.org/10.1038/s41592-020-01000-7"/>
  <meta name="citation_reference" content="Leger A, Amaral PP, Pandolfini L, Capitanchik C, Capraro F, Miano V, Migliori V, Toolan-Kerr P, Sideri T, Enright AJ, Tzelepis K, van Werven FJ, Luscombe NM, Barbieri I, Ule J, Fitzgerald T, Birney E, Leonardi T, & Kouzarides T. (2021). RNA modifications detection by comparative Nanopore direct RNA sequencing. Nature Communications, 12(1), Article number: 7198. https://doi.org/10.1038/s41467-021-27393-3"/>
  <meta name="citation_reference" content="Li H. (2018). Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics, 34 (18), 3094–3100. https://doi.org/10.1093/bioinformatics/bty191"/>
  <meta name="citation_reference" content="Liu Y, Rosikiewicz W, Pan Z, Jillette N, Wang P, Taghbalout A, Foox J, Mason C, Carroll M, Cheng A, & Li S. (2021). DNA methylation-calling tools for Oxford Nanopore sequencing: a survey and human epigenome-wide evaluation. Genome Biology, 22(1), Article number: 295. https://doi.org/10.1186/s13059-021-02510-z"/>
  <meta name="citation_reference" content="Loose M, Malla S, & Stout M. (2016). Real-time selective sequencing using nanopore technology. Nature Methods, 13(9), 751. https://doi.org/10.1038/nmeth.3930"/>
  <meta name="citation_reference" content="Marquet M, Zöllkau J, Pastuschek J, Viehweger A, Schleußner E, Makarewicz O, Pletz MW, Ehricht R, & Brandt C. (2022). Evaluation of microbiome enrichment and host DNA de-pletion in human vaginal samples using oxford nanopore’s adaptive sequencing. Scientific Reports, 12(1), Article number: 4000. https://doi.org/10.1038/s41598-022-08003-8"/>
  <meta name="citation_reference" content="Martin S, Heavens D, Lan Y, Horsfield S, Clark MD, & Leggett RM. (2022). Nanopore adaptive sampling: a tool for enrichment of low abundance species in metagenomic samples. Genome Biology, 23(1), Article number: 11. https://doi.org/10.1186/s13059-021-02582-x"/>
  <meta name="citation_reference" content="McIntyre ABR, Alexander N, Grigorev K, Bezdan D, Sichtig H, Chiu CY, & Mason CE. (2019). Single-molecule sequencing detection of N6-methyladenine in microbial reference materials. Nature Communications, 10, Article number: 579. https://doi.org/10.1038/s41467-019-08289-9"/>
  <meta name="citation_reference" content="McKinney W. (2010). Data structures for statistical computing in Python. Python in Science Conference, 56–61. https://doi.org/10.25080/Majora-92bf1922-00a"/>
  <meta name="citation_reference" content="Medini D, Donati C, Tettelin H, Masignani V, & Rappuoli R. (2005). The microbial pan-genome. Current Opinion in Genetics & Development, 15(6), 589–594. https://doi.org/10.1016/j.gde.2005.09.006"/>
  <meta name="citation_reference" content="Miga KH, Koren S, Rhie A, Vollger MR, Gershman A, Bzikadze A, Brooks S, Howe E, Porubsky D, Logsdon GA, Schneider VA, Potapova T, Wood J, Chow W, Armstrong J, Fredrickson J, Pak E, Tigyi K, Kremitzki M,… Phillippy AM. (2020). Telomere-to-telomere assembly of a complete human X chromosome. Nature, 585 (7823), 79–84. https://doi.org/10.1038/s41586-020-2547-7"/>
  <meta name="citation_reference" content="Miller DE, Sulovari A, Wang T, Loucks H, Hoekzema K, Munson KM, Lewis AP, Fuerte EPA, Paschal CR, Thies J, Bennett JT, Glass I, Dipple KM, Patterson K, Bonkowski ES, Nelson Z, Squire A, Sikes M, Beckman E,… Eichler EE. (2020). Targeted long-read sequencing resolves complex structural variants and identifies missing disease-causing variants. bioRxiv. https://doi.org/10.1101/2020.11.03.365395"/>
  <meta name="citation_reference" content="Mokry M, Feitsma H, Nijman IJ, de Bruijn E, van der Zaag PJ, Guryev V, & Cuppen E. (2010). Accurate SNP and mutation detection by targeted custom microarray-based genomic enrichment of short-fragment sequencing libraries. Nucleic Acids Research, 38(10), e116. https://doi.org/10.1093/nar/gkq072"/>
  <meta name="citation_reference" content="Morton SM. (2021). natsort. https://github.com/SethMMorton/natsort"/>
  <meta name="citation_reference" content="Oxford Nanopore Technologies. (2020). Read Until-API. https://github.com/nanoporetech/read_until_api"/>
  <meta name="citation_reference" content="Oxford Nanopore Technologies. (2021). MinKNOW-API. https://github.com/nanoporetech/minknow_api"/>
  <meta name="citation_reference" content="Oxford Nanopore Technologies. (2022). medaka. https://github.com/nanoporetech/medaka"/>
  <meta name="citation_reference" content="Ozer EA, Allen JP, & Hauser AR. (2014). Characterization of the core and accessory genomes of Pseudomonas aeruginosa using bioinformatic tools Spine and AGEnt. BMC Genomics, 15(1), Article number: 737. https://doi.org/10.1186/1471-2164-15-737"/>
  <meta name="citation_reference" content="Patel A, Dogan H, Payne A, Sievers P, Schoebe N, Schrimpf D, Stichel D, Holmes N, Euskirchen P, Hench J, Frank S, Rosenstiel-Goidts V, Ratliff M, Etminan N, Unterberg A, Dieterich C, Herold-Mende C, Pfister SM, Wick W,… Sahm F. (2021). Rapid-CNS2: rapid comprehensive adaptive nanopore-sequencing of CNS tumors, a proof of concept study. medRxiv. https://doi.org/10.1101/2021.08.09.21261784"/>
  <meta name="citation_reference" content="Payne A, Holmes N, Clarke T, Munro R, Debebe BJ, & Loose M. (2021). Readfish enables targeted nanopore sequencing of gigabase-sized genomes. Nature Biotechnology, 39(4), 442–450. https://doi.org/10.1038/s41587-020-00746-x"/>
  <meta name="citation_reference" content="Payne A, Holmes N, Rakyan V, & Loose M. (2019). BulkVis: A graphical viewer for Oxford nanopore bulk FAST5 files. Bioinformatics, 35(13), 2193–2198. https://doi.org/10.1093/bioinformatics/bty841"/>
  <meta name="citation_reference" content="Pearson W. (2022). toml. https://github.com/uiri/toml"/>
  <meta name="citation_reference" content="Porubsky D, Ebert P, Audano PA, Vollger MR, Harvey WT, Marijon P, Ebler J, Munson KM, Sorensen M, Sulovari A, Haukness M, Ghareghani M, Lansdorp PM, Paten B, Devine SE, Sanders AD, Lee C, Chaisson MJP, Korbel JO,… Marschall T. (2021). Fully phased human genome assembly without parental data using single-cell strand sequencing and long reads. Nature Biotechnology, 39(3), 302–308. https://doi.org/10.1038/s41587-020-0719-5"/>
  <meta name="citation_reference" content="Quick J. (2019). The ‘Three Peaks’ faecal DNA extraction method for long-read sequencing. protocols.io. https://doi.org/10.17504/protocols.io.7rshm6e"/>
  <meta name="citation_reference" content="Quick J, Loman NJ, Duraffour S, Simpson JT, Severi E, Cowley L, Bore JA, Koundouno R, Dudas G, Mikhail A, Ouédraogo N, Afrough B, Bah A, Baum JHJ, Becker-Ziaja B, Boettcher JP, Cabeza-Cabrerizo M, Camino-Sánchez Á, Carter LL,… Carroll MW. (2016). Realtime, portable genome sequencing for Ebola surveillance. Nature, 530 (7589), 228–232. https://doi.org/10.1038/nature16996"/>
  <meta name="citation_reference" content="Richter M, Rosselló-Móra R, Glöckner FO, & Peplies J. (2016). JSpeciesWS: a web server for prokaryotic species circumscription based on pairwise genome comparison. Bioinformatics, 32(6), 929–931. https://doi.org/10.1093/bioinformatics/btv681"/>
  <meta name="citation_reference" content="Ross MG, Russ C, Costello M, Hollinger A, Lennon NJ, Hegarty R, Nusbaum C, & Jaffe DB. (2013). Characterizing and measuring bias in sequence data. Genome Biology, 14 (5), Article number: R51. https://doi.org/10.1186/gb-2013-14-5-r51"/>
  <meta name="citation_reference" content="Sereika M, Kirkegaard RH, Karst SM, Michaelsen TY, Sørensen EA, Wollenberg RD, & Albertsen M. (2021). Oxford nanopore R10.4 long-read sequencing enables near-perfect bacterial genomes from pure cultures and metagenomes without short-read or reference polishing. bioRxiv. https://doi.org/10.1101/2021.10.27.466057"/>
  <meta name="citation_reference" content="Shafin K, Pesout T, Chang PC, Nattestad M, Kolesnikov A, Goel S, Baid G, Kolmogorov M, Eizenga JM, Miga KH, Carnevali P, Jain M, Carroll A, & Paten B. (2021). Haplotype-aware variant calling with PEPPER-margin-DeepVariant enables high accuracy in nanopore long-reads. Nature Methods, 18(11), 1322–1332. https://doi.org/10.1038/s41592-021-01299-w"/>
  <meta name="citation_reference" content="Shannon CE. (1948). A mathematical theory of communication. Bell System Technical Journal, 27(3), 379–423. https://doi.org/10.1002/j.1538-7305.1948.tb01338.x"/>
  <meta name="citation_reference" content="Simpson JT, Workman RE, Zuzarte PC, David M, Dursi LJ, & Timp W. (2017). Detecting DNA cytosine methylation using nanopore sequencing. Nature Methods, 14 (4), 407–410. https://doi.org/10.1038/nmeth.4184"/>
  <meta name="citation_reference" content="Smit AFA, Hubley R, & Green P. (2015). RepeatMasker Open-4.0. http://www.repeatmasker.org"/>
  <meta name="citation_reference" content="Soifer l, Fong NL, Yi N, Ireland AT, Lam I, Sooknah M, Paw JS, Peluso P, Concepcion GT, Rank D, Hastie AR, Jojic V, Ruby JG, Botstein D, & Roy MA. (2020). Fully phased sequence of a diploid human genome determined de novo from the DNA of a single individual. G3 Genes/Genomes/Genetics, 10(9), 2911–2925. https://doi.org/10.1534/g3.119.400995"/>
  <meta name="citation_reference" content="Stevanovski I, Chintalaphani SR, Gamaarachchi H, Ferguson JM, Pineda SS, Scriba CK, Tchan M, Fung V, Ng K, Cortese A, Houlden H, Dobson-Stone C, Fitzpatrick L, Halliday G, Ravenscroft G, Davis MR, Laing NG, Fellner A, Kennerson M,… Deveson IW. (2022). Comprehensive genetic diagnosis of tandem repeat expansion disorders with programmable targeted nanopore sequencing. Science Advances, 8(9), eabm5386. https://doi.org/10.1126/sciadv.abm5386"/>
  <meta name="citation_reference" content="Tan G, Opitz L, Schlapbach R, & Rehrauer H. (2019). Long fragments achieve lower base quality in Illumina paired-end sequencing. Scientific Reports, 9(1), Article number: 2856. https://doi.org/10.1038/s41598-019-39076-7"/>
  <meta name="citation_reference" content="Ulrich JU, Lutfi A, Rutzen K, & Renard BY. (2022). ReadBouncer: precise and scalable adaptive sampling for nanopore sequencing. bioRxiv, 2022.02.01.478636. https://doi.org/10.1101/2022.02.01.478636"/>
  <meta name="citation_reference" content="Virtanen P, Gommers R, Oliphant TE, Haberland M, Reddy T, Cournapeau D, Burovski E, Peterson P, Weckesser W, Bright J, van der Walt SJ, Brett M, Wilson J, Millman KJ, Mayorov N, Nelson ARJ, Jones E, Kern R, Larson E,… SciPy 1.0 Contributors. (2020). SciPy 1.0: fundamental algorithms for scientific computing in Python. Nature Methods, 17, 261–272. https://doi.org/10.1038/s41592-019-0686-2"/>
  <meta name="citation_reference" content="Zhang J, Xie S, Xu J, Liu H, & Wan S. (2021). Cancer biomarkers discovery of methylation modification with direct high-throughput nanopore sequencing. Frontiers in Genetics, 12. https://doi.org/10.3389/fgene.2021.672804"/>
  <meta name="twitter:title" content="Dynamic, adaptive sampling during nanopore sequencing using Bayesian experimental design"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="One particularly promising feature of nanopore sequencing is the ability to reject reads, enabling real-time selection of molecules without complex sample preparation. This is based on the idea of deciding whether a molecule warrants full sequencing depending on reading a small initial part. Previously, such decisions have been based on a priori determination of which regions of the genome were considered of interest. Instead, here we consider more general and complex strategies that incorporate already-observed data in order to optimize the rejection strategy and maximise information gain from the sequencing process. For example, in the presence of coverage bias redistributing data from areas of high to areas of low coverage would be desirable.

We present BOSS-RUNS, a mathematical and algorithmic framework to calculate the expected benefit of new reads and generate dynamically updated decision strategies for nanopore sequencing. During sequencing, in real time, we quantify the current uncertainty at each site of one or multiple reference genomes, and for each novel DNA fragment being sequenced we decide whether the potential decrease in uncertainty at the sites it will most likely cover warrants reading it in its entirety. This dynamic, adaptive sampling allows real-time focus of sequencing efforts onto areas of highest benefit.

We demonstrate the effectiveness of BOSS-RUNS by mitigating coverage bias across and within the species of a microbial community. Additionally, we show that our approach leads to improved variant calling due to its ability to sample more data at the most relevant genomic positions.

### Competing Interest Statement

ML was a member of the ONT MinION access program and has received free flow cells and sequencing reagents in the past. ML has received reimbursement for travel, accommodation and conference fees to speak at events organized by ONT. EB is a paid consultant to ONT and a small-scale equity and options holder in ONT."/>
  <meta name="og-title" property="og:title" content="Dynamic, adaptive sampling during nanopore sequencing using Bayesian experimental design"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.07.938670v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="One particularly promising feature of nanopore sequencing is the ability to reject reads, enabling real-time selection of molecules without complex sample preparation. This is based on the idea of deciding whether a molecule warrants full sequencing depending on reading a small initial part. Previously, such decisions have been based on a priori determination of which regions of the genome were considered of interest. Instead, here we consider more general and complex strategies that incorporate already-observed data in order to optimize the rejection strategy and maximise information gain from the sequencing process. For example, in the presence of coverage bias redistributing data from areas of high to areas of low coverage would be desirable.

We present BOSS-RUNS, a mathematical and algorithmic framework to calculate the expected benefit of new reads and generate dynamically updated decision strategies for nanopore sequencing. During sequencing, in real time, we quantify the current uncertainty at each site of one or multiple reference genomes, and for each novel DNA fragment being sequenced we decide whether the potential decrease in uncertainty at the sites it will most likely cover warrants reading it in its entirety. This dynamic, adaptive sampling allows real-time focus of sequencing efforts onto areas of highest benefit.

We demonstrate the effectiveness of BOSS-RUNS by mitigating coverage bias across and within the species of a microbial community. Additionally, we show that our approach leads to improved variant calling due to its ability to sample more data at the most relevant genomic positions.

### Competing Interest Statement

ML was a member of the ONT MinION access program and has received free flow cells and sequencing reagents in the past. ML has received reimbursement for travel, accommodation and conference fees to speak at events organized by ONT. EB is a paid consultant to ONT and a small-scale equity and options holder in ONT."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-04-04"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.02.07.938670v3"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2480196"/>
  <title>
   Dynamic, adaptive sampling during nanopore sequencing using Bayesian experimental design | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"7LGjmlZEKBCS2T-evPdSwYWqJeFfDJ_6RGAnxF_qPe4"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2480196","apath":"\/biorxiv\/early\/2022\/04\/04\/2020.02.07.938670.atom","pisa":"biorxiv;2020.02.07.938670v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.02.07.938670v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.02.07.938670v3","title":"Dynamic, adaptive sampling during nanopore sequencing using Bayesian experimental design","identifier":"node\/2480196"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PRbf7rsCqqnQ0977h4g1hV6rMBrcgRQQ3FCvCqcFYodQC1\\\\\\\/6QtsmC5XdlbQ5hV6tj\\\\\\\/7OjRdS2gnoje6blT8Sy5amhYdeJKexTzAmEsflCIdK\\\\\\\/CsoRFMpS6ddu309cbIDBrmUhX8ZCnsT4kiXKcmSOB4JltcPF3pMS0cQfJYCqIB\\\\\\\/OL\\\\\\\/DzlBNTHKWM6\\\\\\\/qsejGuV0Bq8IFmG5wne0Bwjp65vzbrBhWJqRbrB94SGoVEwiukPX3jXzezh0nPsP9E6vg6Dlz5ukIIBGEPf0Z49igQEXEP7g9Kjc9i1XjbtVlhiVWD2bPeWWCQG96+aQxry8uWd9KzMnV\\\\\\\/A5aqtUYJqDEY\\\\\\\/w5ef4\\\\\\\/TLLo3Dr4FOEQH3IN4Ky1ljNJyjOJFP8e91jIzjOcX8MdsFSCpvLQa838JMfWN\\\\\\\/AmwedXEeCBFYVKjlqyhu9V2S4Qw4ExzQLhWJPkdzP3p7knKo7HN4VQ+CjPsUhUp\\\\\\\/qpnIhZ3SQls1TVOvaWwUgdnB1diUrU\\\\\\\/3ohQ+OQnlhCRTcBgy4bhoGGw0gSHziM\\\\\\\/xc3vcEiPS8XcDD21veUaAx6y2JziYlPpqA\\\\\\\/SZCKf512RHbSMNIV6dX7XawVaBX2fQaIpxhgd2CWbOTWWYU7YQLkhovfNDkGgtRTL5l4\\\\\\\/JokTa+uM1JH1RIvzA8W6dgTKRV1eBo3fHANz3sCgMwMksrzbI1pqewr+6IONiTNr8KsDB93rv0nmEc1kqspUfFZFdhwnsjGzpM7f+vx9sv3NM6ymNVbFaGvUHrKhfbKc5FccSQMmRK257sMECcOCGIhGLck+SOccLnpw2TI7G+DfxWenYsA58+ss9BIy51MZrWrJGEAqdaL8gHWcT3YZp0KY4\\\\\\\/9ncN0atJcu8cNk0tsCi81rrOVMi04nqqps88+dyAfZ+rphvcAiCFFeNzdbU4Sg\\\\\\\/ksIz7zNvmfi\\\\\\\/bgLKa2weDLO29BJ03EGv2bANLU0IHevhe6M3H\\\\\\\/oTi2tdmccYIF8T6Q2Jq0ljC2owmWoQcJGREoAg3UHTo1qJ+eR2oXmfzCODvZPm6gyGz0j1v0i3hRD3IBto9\\\\\\\/CA3FMZ0moA63\\\\\\\/d4ndtNJBTaIrgzmJh+gEZkRWlt0\\\\\\\/OMLIf76GssowP7ohIoKFEmt4HBdeEmCZctE4fMOEs69rKiLV16s7iV5uRYIEoaDZKYy\\\\\\\/nHcfHMYoeUh7SgUU53ZXHrMO5MAaSA4tSz9ihMT3iet4MptvQaih2H8aov2IrGZLo1pQyo4iInrl9T\\\\\\\/zrM7ukOLA4R5DKnY\\\\\\\/HSDFVV7kTJ3Wd2IbWW+iCjzxVrXE8rhhv7m37lGp5UEAO97HtJx3vFJ3Bt5sFBNmMmRORkgtG49mVewjqm+qpAmkajcyRyKJUbMA\\\\\\\/LAFTKGCqt7AnhcO2X3xcOfqK0=\\\u0022,\\\u0022iv\\\u0022:\\\u0022AWbkkBIlRXs4691oP\\\\\\\/gA9g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022a017259b14e7a2d064012cc7e645b826a18ff256d6cac363d4c7321bbdd85d00\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.02.07.938670v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2480196 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.02.07.938670v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_467344887">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_467344887" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_19146238" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-RZiySNwWLqkNEdQg7u5ocr8yrpA66VQzrvPWyeoI7iU"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2480196" data-node-nid="2480196" id="node-2480196--21674138567" data-pisa="biorxiv;2020.02.07.938670v3" data-pisa-master="biorxiv;2020.02.07.938670" data-apath="/biorxiv/early/2022/04/04/2020.02.07.938670.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Dynamic, adaptive sampling during nanopore sequencing using Bayesian experimental design
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0001-6459-0431" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Lukas
                      </span>
                      <span class="nlm-surname">
                       Weilguny
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0002-1776-8564" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Nicola
                      </span>
                      <span class="nlm-surname">
                       De Maio
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2">
                      <a href="http://orcid.org/0000-0003-3158-0303" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Rory
                      </span>
                      <span class="nlm-surname">
                       Munro
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3">
                      <a href="http://orcid.org/0000-0002-4125-8900" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Charlotte
                      </span>
                      <span class="nlm-surname">
                       Manser
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4">
                      <a href="http://orcid.org/0000-0001-8314-8497" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Ewan
                      </span>
                      <span class="nlm-surname">
                       Birney
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5">
                      <a href="http://orcid.org/0000-0002-5264-0929" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Matt
                      </span>
                      <span class="nlm-surname">
                       Loose
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6">
                      <a href="http://orcid.org/0000-0001-8486-2211" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Nick
                      </span>
                      <span class="nlm-surname">
                       Goldman
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.02.07.938670
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2480196--21674138567" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Lukas Weilguny
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Genome Campus
                       </span>
                       , Hinxton, Cambridgeshire, CB10 1SD,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Lukas%2BWeilguny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Weilguny%20L&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ALukas%2BWeilguny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-6459-0431" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Lukas Weilguny
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Nicola De Maio
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Genome Campus
                       </span>
                       , Hinxton, Cambridgeshire, CB10 1SD,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Nicola%2BDe%2BMaio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=De%20Maio%20N&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ANicola%2BDe%2BMaio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-1776-8564" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Nicola De Maio
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Rory Munro
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        DeepSeq, School of Life Sciences, Queen’s Medical Centre, University of Nottingham
                       </span>
                       , Nottingham NG7 2UH,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Rory%2BMunro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Munro%20R&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ARory%2BMunro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-3158-0303" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Rory Munro
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Charlotte Manser
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Genome Campus
                       </span>
                       , Hinxton, Cambridgeshire, CB10 1SD,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Charlotte%2BManser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Manser%20C&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ACharlotte%2BManser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-4125-8900" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Charlotte Manser
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Ewan Birney
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Genome Campus
                       </span>
                       , Hinxton, Cambridgeshire, CB10 1SD,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Ewan%2BBirney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Birney%20E&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AEwan%2BBirney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-8314-8497" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Ewan Birney
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Matt Loose
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        DeepSeq, School of Life Sciences, Queen’s Medical Centre, University of Nottingham
                       </span>
                       , Nottingham NG7 2UH,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Matt%2BLoose%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Loose%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AMatt%2BLoose%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-5264-0929" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Matt Loose
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">
                     Nick Goldman
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Genome Campus
                       </span>
                       , Hinxton, Cambridgeshire, CB10 1SD,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Nick%2BGoldman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Goldman%20N&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ANick%2BGoldman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0001-8486-2211" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Nick Goldman
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:goldman@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">
                        goldman@ebi.ac.uk
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.02.07.938670v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2480196" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2480196/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.07.938670v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2480196" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2480196/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.07.938670v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2480196" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2480196/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.07.938670v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2480196" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2480196/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.07.938670v3.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2480196" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2480196/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.02.07.938670v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2480196" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2480196/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-3">
                             One particularly promising feature of nanopore sequencing is the ability to reject reads, enabling real-time selection of molecules without complex sample preparation. This is based on the idea of deciding whether a molecule warrants full sequencing depending on reading a small initial part. Previously, such decisions have been based on a priori determination of which regions of the genome were considered of interest. Instead, here we consider more general and complex strategies that incorporate already-observed data in order to optimize the rejection strategy and maximise information gain from the sequencing process. For example, in the presence of coverage bias redistributing data from areas of high to areas of low coverage would be desirable.
                            </p>
                            <p id="p-4">
                             We present BOSS-RUNS, a mathematical and algorithmic framework to calculate the expected benefit of new reads and generate dynamically updated decision strategies for nanopore sequencing. During sequencing, in real time, we quantify the current uncertainty at each site of one or multiple reference genomes, and for each novel DNA fragment being sequenced we decide whether the potential decrease in uncertainty at the sites it will most likely cover warrants reading it in its entirety. This dynamic, adaptive sampling allows real-time focus of sequencing efforts onto areas of highest benefit.
                            </p>
                            <p id="p-5">
                             We demonstrate the effectiveness of BOSS-RUNS by mitigating coverage bias across and within the species of a microbial community. Additionally, we show that our approach leads to improved variant calling due to its ability to sample more data at the most relevant genomic positions.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             1 Introduction
                            </h2>
                            <p id="p-8">
                             Third-generation sequencing provides the unprecedented ability to generate reads consisting of multiple kilo- or even megabases (
                             <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                              Payne et al. 2019
                             </a>
                             ) compared to previous approaches which are limited to sequencing fragments of several hundred bases (after which the error rate increases significantly:
                             <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">
                              Tan et al. 2019
                             </a>
                             ). This technological advance has critical implications for many applications in genomics. For example, ultra-long reads are highly useful in increasing assembly contiguity and even allow the construction of telomere-to-telomere assemblies of entire chromosomes (
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              Jain et al. 2018
                             </a>
                             ;
                             <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                              Miga et al. 2020
                             </a>
                             ). Moreover, they can be used to interrogate variation in regions of a genome that are hard to decipher, such as repeats, centromeres or segmental duplications (
                             <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">
                              Shafin et al. 2021
                             </a>
                             ), or to generate fully-phased chromosome-level epigenetic maps (
                             <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                              Lee et al. 2020
                             </a>
                             ).
                            </p>
                            <p id="p-9">
                             One way of generating long sequencing reads is through the use of nanopores. This concept was first explored in the 1980s and commercialized in 2014 by Oxford Nanopore Technologies (ONT;
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              Deamer et al. 2016
                             </a>
                             ). It relies on the idea of using a protein nanopore as a biosensor that measures the fluctuations of an ionic current across the pore caused by the presence of nucleotides of a translocating DNA or RNA molecule.
                            </p>
                            <p id="p-10">
                             Besides providing long reads, ONT’s instruments offer several additional advantages. First, single-molecule real-time sequencing of molecules is possible without the need for prior amplification and can also be used to directly read RNA without reverse transcription (
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              Garalde et al. 2018
                             </a>
                             ). The generation of sequencing reads in real time, which in combination with fast library preparation immensely reduces the time needed to go from biological sample to data analysis, enables (e.g.) intraoperative decision making (
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              Djirackor et al. 2021
                             </a>
                             ), improved global food security by rapid identification of plant viruses (
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              Boykin et al. 2018
                             </a>
                             ) and portable genomic surveillance (
                             <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                              Quick et al. 2016
                             </a>
                             ).
                            </p>
                            <p id="p-11">
                             The most notable drawback of sequencing using nanopores is the elevated rate of sequencing errors compared to previous technologies. However, ONT’s nanopores, sequencing chemistry and basecallers, which are used to translate electrical signal into nucleobases, have been steadily improved and initial error rates of almost 40% (
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              Goodwin et al. 2015
                             </a>
                             ) have decreased to as low as ~1% (
                             <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">
                              Sereika et al. 2021
                             </a>
                             ) approaching the accuracy of >99% offered by short-read platforms.
                            </p>
                            <p id="p-12">
                             A particularly exciting and unique feature of sequencing using nanopores is the ability to reverse the voltage across the pores in order to reject fragments before reading them in their entirety, termed adaptive sampling or “Read Until” in its original implementation (
                             <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                              Loose et al. 2016
                             </a>
                             ;
                             <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                              Oxford Nanopore Technologies 2020
                             </a>
                             ). This enables real-time selection of molecules during sequencing based on a small initial part of a read without the need for complex sample preparation. Initially, identifying the genomic origin of these small fragments was achieved by matching the electrical signal (‘squiggles’:
                             <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">
                              Loose et al. 2016
                             </a>
                             ) directly to reference genomes translated into simulated current traces. Recent improvements, however, harness the computing power of GPUs for real-time basecalling, making it possible to utilize optimized bioinformatics tools for further processing, e.g. for alignment of reads to reference genomes in base-space (
                             <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                              Payne et al. 2021
                             </a>
                             ). This has led to much interest in experiments that can be aided by real-time selection of molecules for sequencing (e.g.
                             <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                              Miller et al. 2020
                             </a>
                             ;
                             <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                              Patel et al. 2021
                             </a>
                             ;
                             <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                              Marquet et al. 2022
                             </a>
                             ;
                             <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">
                              Stevanovski et al. 2022
                             </a>
                             ).
                            </p>
                            <p id="p-13">
                             In current implementations, underlying decisions about which reads to sequence completely are based on a priori choice, e.g. of regions of interest (ROIs) in a sequenced genome (
                             <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">
                              Loose et al. 2016
                             </a>
                             ;
                             <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">
                              Payne et al. 2021
                             </a>
                             ). This restricts their application to a narrow range of problems, where considerable background information is available in advance of sequencing a (potentially poorly characterised) sample. We propose that, in addition, such decisions could also incorporate the information obtained from already-sequenced fragments generated in the current sequencing run. This would allow for optimised, dynamic decisions that maximise the information gain during sequencing, leading to various potential advantages such as reduced time-to-answer, reduced cost, and increased confidence in variation calling.
                            </p>
                            <p id="p-14">
                             More specifically, during a sequencing experiment we might observe a distribution of coverage depth that does not correspond well to the aim of the experiment (observed vs. ideal coverage in
                             <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                              Fig. 1A, B
                             </a>
                             ). For example, for resequencing or variant calling most positions in a genome will match a known reference and their genotype can therefore be confirmed by observing only few reads. In contrast, some sites or regions will be different and their detection and identification might be the goal of such experiments. At these sites we ideally want to sample more sequencing data. Commonly at present, the overall coverage of the target genome would have to be increased in order to ensure sufficient sampling in biased regions. This leads to wasteful data acquisition in regions that are not of continued interest. In this work we address this issue by generating dynamic decision strategies that redistribute sequencing coverage to positions of greatest value at any point during an experiment.
                            </p>
                            <div id="F1" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F1.large.jpg?width=800&height=600&carousel=1" title="Methodological overview. A) When re-sequencing a genome, different sites might require different levels of coverage. For example, sites without variation are resolved by few reads (low ideal coverage). Additional accumulation of coverage at those positions is wasteful, whereas other sites with higher uncertainty (e.g. due to the presence of variation or lower-quality reference genome, marked by black triangles and rectangles) would benefit from more observations. B) Another reason for reduced efficiency of sequencing could be local fluctuations in the distribution of fragments’ origins, resulting in uneven coverage. C) We quantify the uncertainty about the genotype at each site by calculating posterior probabilities of all possible genotypes, based on prior probabilities and the sequence data observed so far. The expected shift in genotype uncertainty caused by observing a new read at that position is expressed as a “positional benefit score”. D) Next, we calculate the total expected benefit from a hypothetical read starting at each location, i.e. the sum of the positional scores that such a novel read might accumulate (depicted by the height of the expected benefit plots), weighted by the probability of reaching those positions (depicted by the color-gradients). As an example, the benefit of both forward (red) and reverse-oriented reads (blue) starting at two positions are shown. E) Finally, we derive a decision strategy for each position, shown as zeros and ones, that instructs the sequencer to either continue sequencing (1) or reject from the pore (0) a read that starts at that position. Stages C–E can be updated and iterated throughout the sequencing experiment. F) Overview of our model of the sequencing process. Since we parameterize time by the speed of bases translocating through pores we use number of bases and sequencing time interchangeably. At first (t = 0), a novel read is acquired by a nanopore and sequencing commences. After its initial μ bases are sequenced they are used to identify its starting position and orientation along the reference genome, which determines the fate of the nascent fragment according to the current decision strategy (E). Upon rejection (upper path), an additional time of ρ sequenced bases later the pore is freed, a new read is acquired after further time α, and the model iterates from the beginning. Conversely, upon acceptance (lower path), the molecule translocates through the pore until all l of its nucleotides (i.e. l – μ additional bases) have been read. A new read can then be acquired and the model iterates from the start." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-380954308" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Methodological overview.</span> A) When re-sequencing a genome, different sites might require different levels of coverage. For example, sites without variation are resolved by few reads (low ideal coverage). Additional accumulation of coverage at those positions is wasteful, whereas other sites with higher uncertainty (e.g. due to the presence of variation or lower-quality reference genome, marked by black triangles and rectangles) would benefit from more observations. B) Another reason for reduced efficiency of sequencing could be local fluctuations in the distribution of fragments’ origins, resulting in uneven coverage. C) We quantify the uncertainty about the genotype at each site by calculating posterior probabilities of all possible genotypes, based on prior probabilities and the sequence data observed so far. The expected shift in genotype uncertainty caused by observing a new read at that position is expressed as a “positional benefit score”. D) Next, we calculate the total expected benefit from a hypothetical read starting at each location, i.e. the sum of the positional scores that such a novel read might accumulate (depicted by the height of the expected benefit plots), weighted by the probability of reaching those positions (depicted by the color-gradients). As an example, the benefit of both forward (red) and reverse-oriented reads (blue) starting at two positions are shown. E) Finally, we derive a decision strategy for each position, shown as zeros and ones, that instructs the sequencer to either continue sequencing (1) or reject from the pore (0) a read that starts at that position. Stages C–E can be updated and iterated throughout the sequencing experiment. F) Overview of our model of the sequencing process. Since we parameterize time by the speed of bases translocating through pores we use number of bases and sequencing time interchangeably. At first (<em>t</em> = 0), a novel read is acquired by a nanopore and sequencing commences. After its initial <em>μ</em> bases are sequenced they are used to identify its starting position and orientation along the reference genome, which determines the fate of the nascent fragment according to the current decision strategy (E). Upon rejection (upper path), an additional time of <em>ρ</em> sequenced bases later the pore is freed, a new read is acquired after further time <em>α</em>, and the model iterates from the beginning. Conversely, upon acceptance (lower path), the molecule translocates through the pore until all <em>l</em> of its nucleotides (i.e. <em>l</em> – <em>μ</em> additional bases) have been read. A new read can then be acquired and the model iterates from the start.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F1.medium.gif" width="387" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F1.medium.gif" width="387" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                              <span class="fig-label">
                               Figure 1:
                              </span>
                              <span class="caption-title">
                               Methodological overview.
                              </span>
                              <p id="p-15" class="first-child">
                               A) When re-sequencing a genome, different sites might require different levels of coverage. For example, sites without variation are resolved by few reads (low ideal coverage). Additional accumulation of coverage at those positions is wasteful, whereas other sites with higher uncertainty (e.g. due to the presence of variation or lower-quality reference genome, marked by black triangles and rectangles) would benefit from more observations. B) Another reason for reduced efficiency of sequencing could be local fluctuations in the distribution of fragments’ origins, resulting in uneven coverage. C) We quantify the uncertainty about the genotype at each site by calculating posterior probabilities of all possible genotypes, based on prior probabilities and the sequence data observed so far. The expected shift in genotype uncertainty caused by observing a new read at that position is expressed as a “positional benefit score”. D) Next, we calculate the total expected benefit from a hypothetical read starting at each location, i.e. the sum of the positional scores that such a novel read might accumulate (depicted by the height of the expected benefit plots), weighted by the probability of reaching those positions (depicted by the color-gradients). As an example, the benefit of both forward (red) and reverse-oriented reads (blue) starting at two positions are shown. E) Finally, we derive a decision strategy for each position, shown as zeros and ones, that instructs the sequencer to either continue sequencing (1) or reject from the pore (0) a read that starts at that position. Stages C–E can be updated and iterated throughout the sequencing experiment. F) Overview of our model of the sequencing process. Since we parameterize time by the speed of bases translocating through pores we use number of bases and sequencing time interchangeably. At first (t = 0), a novel read is acquired by a nanopore and sequencing commences. After its initial μ bases are sequenced they are used to identify its starting position and orientation along the reference genome, which determines the fate of the nascent fragment according to the current decision strategy (E). Upon rejection (upper path), an additional time of ρ sequenced bases later the pore is freed, a new read is acquired after further time α, and the model iterates from the beginning. Conversely, upon acceptance (lower path), the molecule translocates through the pore until all l of its nucleotides (i.e. l – μ additional bases) have been read. A new read can then be acquired and the model iterates from the start.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-16">
                             We call our novel method BOSS-RUNS, for “Benefit-Optimising Short-term Strategies for Read Until Nanopore Sequencing”. In brief, we quantify uncertainty at each site during a resequencing experiment, calculate the expected benefit of new reads, and dynamically adapt the sequencing effort to focus on fragments from areas of highest uncertainty. Our implementation efficiently communicates with the sequencing device through the readfish toolkit (
                             <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">
                              Payne et al. 2021
                             </a>
                             ) and the Read Until API (
                             <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">
                              Oxford Nanopore Technologies 2020
                             </a>
                             ) to incorporate the streamed sequencing data and provide updated decision strategies in real-time.
                            </p>
                            <p id="p-17">
                             One promising application of our strategy is the possibility to compensate for the inherent tendency of some genomic regions to be sequenced at higher coverage than others, possibly due to GC content or other factors (
                             <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">
                              Ross et al. 2013
                             </a>
                             ;
                             <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                              Krishnakumar et al. 2018
                             </a>
                             ). In this scenario our method could lead to more homogeneous distribution of sequencing reads across a genome, with the benefit of increasing accuracy of genotype calling and reducing uncertainty in regions of low coverage. On the other hand, our method can also be used to focus the sequencing effort on ROIs or sites that show variation without the need for prior information about their location.
                            </p>
                            <p id="p-18">
                             We demonstrate our novel method by mitigating coverage bias in a microbial mock community leading to higher coverage depth of low-abundance species, an increased limit of detection, as well as improved variant calling. To summarize, our approach of dynamic, adaptive sampling allows us to change what is sampled during an ongoing sequencing experiment as a result of the data that has already been collected and enables redistribution of coverage to sites of highest biological interest without a priori knowledge of coverage bias or variant sites.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             2 Results
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              2.1 Quantifying information content of sequencing reads
                             </h3>
                             <p id="p-19">
                              In this work we present a framework that enables dynamic decisions during sequencing using nanopores. By calling it dynamic, adaptive sampling we describe our extension compared to current approaches, which are limited to a priori choice of target regions. In this section we give an overview of the decision framework, with further details and formal explanations provided in the Supplement (Suppl. Sect. 1.1).
                             </p>
                             <p id="p-20">
                              First, we capture the amount of information we have at each site of a genome under investigation by considering a probability distribution over all possible genotypes of that site. We then update this probability distribution as we collect data throughout the experiment, i.e. we calculate a posterior distribution that includes the observed nucleotides from reads that cover that position. Next, we calculate the remaining uncertainty and how much information we might gain from one further sequencing read covering that site. By combining these scores over adjacent sites we can quantify the expected information gain from a sequencing read solely by knowing its starting location and its orientation. Finally, by ranking the expected benefit of reads and taking into account the expected time of sequencing them we can calculate the optimal subset of sites to accepts reads from in order to increase the gain of benefit at that moment in the experiment. The following paragraphs give more details on the individual steps involved to arrive at such a dynamic decision strategy.
                             </p>
                             <div id="sec-4" class="subsection">
                              <h4>
                               Probability distribution of genotypes at each site of a genome
                              </h4>
                              <p id="p-21">
                               We start by defining a probability distribution of possible genotypes at each position of one or multiple genomes. Briefly, the genotype probability distribution takes both prior information about the genotype, e.g. from a reference genome, and already observed bases at a position into account. (Throughout, we use reference genome to describe any assembly used for a resequencing experiment and not necessarily a reference assembly that is representative of the investigated species.) Additionally, ploidy and probabilities of sequencing errors are considered.
                              </p>
                              <p id="p-22">
                               Given already observed read data D, containing n reads covering position i, we denote by d
                               <sub>
                                j,i
                               </sub>
                               ∈ B, with B = {A, C, G, T}, the nucleotide in read j that maps to i. For a haploid genome, the set of possible genotypes is G = B, whereas for diploid genomes G instead consists of unordered pairs g = {b
                               <sub>
                                1
                               </sub>
                               , b
                               <sub>
                                2
                               </sub>
                               }, with b
                               <sub>
                                1
                               </sub>
                               ,b
                               <sub>
                                2
                               </sub>
                               ∈ B. For simplicity we present the case of genetic diversity and sequencing errors only occurring as SNPs; an extension that includes deletions and is used in our applications is provided in the Supplement (Suppl. Sect. 1.2). We define prior probabilities for genotype g at position i as π
                               <sub>
                                i
                               </sub>
                               (g), and the probability of calling base d
                               <sub>
                                j,i
                               </sub>
                               assuming genotype g as ϕ(d
                               <sub>
                                j,i
                               </sub>
                               |g), which represents a matrix of observation probabilities given assumptions about ploidy and sequencing errors (details in Suppl. Sect. 1.1). The posterior probability of genotype g ∈ G at i, conditional on D, is then
                               <span class="disp-formula" id="disp-formula-1">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/graphic-2.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/graphic-2.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where Z
                               <sub>
                                i
                               </sub>
                               (D) represents a normalising constant, i.e. the likelihood of the data, that ensures the posterior probabilities sum to 1.
                              </p>
                              <p id="p-23">
                               This model allows us to quantify the uncertainty about the genotype at each site (
                               <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                                Fig. 1C
                               </a>
                               ) and in turn makes it possible to calculate the expected reduction in uncertainty resulting from observing a newly sequenced read. We call this expected reduction of uncertainty the positional benefit score of a site. This quantity summarizes the expected change in the genotype probability distribution given one additional observation at that position and is calculated as follows: given the current data (D) we imagine that we observe one additional nucleotide n at position i, i.e. d
                               <sub>
                                n+1,i
                               </sub>
                               , calling this augmented data D′. We then measure the difference between the distribution of genotype probabilities resulting from D and D′ by the Kullback-Leibler divergence (D
                               <sub>
                                KL
                               </sub>
                               ;
                               <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                                Kullback and Leibler 1951
                               </a>
                               ). Lastly, we sum over the different possible nucleotides d
                               <sub>
                                n+1,i
                               </sub>
                               , weighting their contributions by the estimated probability of observing them in the next read, to compute the expected reduction in uncertainty:
                               <span class="disp-formula" id="disp-formula-2">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/graphic-3.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/graphic-3.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where the estimated probability of observing nucleotide d
                               <sub>
                                n+1,i
                               </sub>
                               in the next read is given by
                               <span class="disp-formula" id="disp-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/graphic-4.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/graphic-4.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-24">
                               A practical way of calculating the positional benefit scores and some examples at different coverage patterns are given in the supplementary material (Suppl. Sect. 1.3, Suppl.
                               <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                                Fig. 1
                               </a>
                               ). Broadly speaking, positions that are already covered by many agreeing reads will receive a low score and, conversely, positions covered by few, or possibly contradictory reads will score highly as individual observations have higher potential of influencing the probability distribution. This technique of defining the information gain in terms of the Kullback-Leibler divergence of two distributions is used in Bayesian experimental design (
                               <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                                Chaloner and Verdinelli 1995
                               </a>
                               ) and is equivalent to evaluating the expected reduction in Shannon entropy (
                               <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">
                                Shannon 1948
                               </a>
                               ) brought by a new read.
                              </p>
                             </div>
                             <div id="sec-5" class="subsection">
                              <h4>
                               Estimating the expected benefit of sequencing reads
                              </h4>
                              <p id="p-25">
                               Since we want to quantify not only the remaining uncertainty at individual sites but also the potential information gain of future sequencing reads, we consider the fact that reads are derived from contiguous sections of a genome. So, as a next step, we combine the positional benefit scores across sites that a sequencing read might span, in order to evaluate the expected benefit of such a read (
                               <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                                Fig. 1D
                               </a>
                               ). We assume that a sequenced read will cover a number of consecutive sites of a reference genome equal to the molecule’s length l. The expected benefit is then calculated as the sum of consecutive positional scores, beginning from the read’s mapping starting position i, weighted by the distribution of previously observed read lengths, L(l). In other words, we form the sum
                               <span class="inline-formula" id="inline-formula-1">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-1.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-1.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               of consecutive positional benefit scores of a read of length l starting at position i with orientation o (o = 1 indicating a read in the forward direction relative to the reference genome, and 0 indicating the reverse direction); and then combine these, weighted by the probability that the read will reach that position (
                               <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                                Fig. 1D
                               </a>
                               ). For a forward-oriented read
                               <span class="inline-formula" id="inline-formula-2">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-2.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-2.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               will be:
                               <span class="disp-formula" id="disp-formula-4">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/graphic-5.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/graphic-5.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               (see Suppl. Sect. 1.4 for the reverse-oriented case), leading to the expected benefit:
                               <span class="disp-formula" id="disp-formula-5">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/graphic-6.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/graphic-6.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-26">
                               Here,
                               <span class="inline-formula" id="inline-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-3.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-3.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               represents the domain of L(l), i.e. all read lengths observed so far. In reality we use a truncated normal distribution as a prior for read lengths, which we continuously update with observed lengths of full-length sequencing reads throughout an experiment. More details about S
                               <sup>
                                l
                               </sup>
                               and L(l) are given in Suppl. Sect. 1.4.
                              </p>
                              <p id="p-27">
                               With this, we can quantify the expected information gain of a sequencing read solely on the basis of its genomic origin and orientation. Ultimately, a sequencing read that is expected to give a higher sum of scores, i.e. a greater reduction in the uncertainty of genotypes at the positions it covers, will be considered more useful than a read with a limited potential to alter the previously defined posterior probabilities. We provide an approximation to calculate this quantity efficiently based on a piece-wise approximation of the read length distribution in the Supplement (Suppl. Sect. 1.4).
                              </p>
                             </div>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              2.2 Decision framework to enable dynamic, adaptive sampling
                             </h3>
                             <p id="p-28">
                              Using the expected benefit of reads we can now define a framework for making decisions about which fragments to sequence fully and which to reject from nanopores. Note that in line with common usage we refer to fragments, i.e. DNA molecules, and reads, their translation into sequence space, interchangeably. Overall, our aim is to optimise the rate of accumulation of information, i.e. of expected benefit, across all pores and over time. As we collect data throughout the sequencing experiment the value of reads at different positions will change, and therefore the decision strategy will have to adapt to these changes dynamically in real-time. Such strategies are stored as Boolean vectors and indicate the decision to be made about a sequencing read starting at any genomic position (
                              <a id="xref-fig-1-6" class="xref-fig" href="#F1">
                               Fig. 1E
                              </a>
                              ).
                             </p>
                             <p id="p-29">
                              To define our decision strategies we parameterize the duration of individual steps in the sequencing process. As our time unit we use the amount of time it takes one base to translocate through a pore (
                              <a id="xref-fig-1-7" class="xref-fig" href="#F1">
                               Fig. 1F
                              </a>
                              ). Analogous to Read Until and readfish, we start sequencing a DNA fragment and use μ initial bases to determine its genomic origin and orientation. The value of μ is assumed constant in our model and can be adjusted to ensure mappings of sufficient quality, e.g. depending on the complexity or repeat content of the used reference genome. In reality μ depends on the size of individual data chunks used for real-time base-calling. The smallest useful setting is 0.4 seconds of input data, which corresponds to ~180 nucleotides assuming a translocation speed of 450nt/s. In our applications we used 0.8s of data and observed a mean length of 348nt for real-time base-called data chunks used to determine the origin of fragments. We further assume some constant time is needed to effect the rejection of a read (ρ) and to acquire a new read at a pore (α). In line with measurements from sequencing experiments our model assumes ρ = 300 and α = 300 by default. The effects of real-life conditions and constraints on these parameters are discussed in the Discussion. If a fragment is sequenced fully, time equal to its length l passes and benefit
                              <span class="inline-formula" id="inline-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is accrued (with expectations λ = E[L] and U
                              <sub>
                               i,o
                              </sub>
                              , respectively); by rejecting a read, time equal to l – μ – ρ can be saved and the expected gain of benefit is limited to the position scores of its initial fragment, i.e.
                              <span class="inline-formula" id="inline-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (see
                              <a id="xref-fig-1-8" class="xref-fig" href="#F1">
                               Fig. 1F
                              </a>
                              and Suppl. Fig. 2).
                             </p>
                             <div id="sec-7" class="subsection">
                              <h4>
                               Finding an optimal decision strategy to maximise information gain
                              </h4>
                              <p id="p-30">
                               With this parameterization of the sequencing process we determine an optimal sequencing strategy that maximises the expected benefit per unit of sequencing time given the currently available data. Such a strategy, denoted as
                               <span class="inline-formula" id="inline-formula-6">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-6.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-6.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , can also be seen as an indicator function that returns 0 or 1 for all combinations of genomic position and fragment orientation, e.g.
                               <span class="inline-formula" id="inline-formula-7">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-7.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-7.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               indicates the rejection of a forward-oriented read at position i and
                               <span class="inline-formula" id="inline-formula-8">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-8.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-8.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               the acceptance of a reverse-oriented read. Our aim is therefore to find an optimal strategy
                               <span class="inline-formula" id="inline-formula-9">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-9.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-9.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               that maximises the benefit per unit time
                               <span class="inline-formula" id="inline-formula-10">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-10.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-10.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               given the current data D:
                               <span class="disp-formula" id="disp-formula-6">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/graphic-7.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/graphic-7.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-31">
                               Here
                               <span class="inline-formula" id="inline-formula-11">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-11.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-11.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is the average expected benefit. Given a genome with a total length N and the average expected benefit of the initial parts of reads, i.e. the benefit accrued from the initial fragment used in the decision process, denoted
                               <span class="inline-formula" id="inline-formula-12">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-12.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-12.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , it takes the form:
                               <span class="disp-formula" id="disp-formula-7">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/graphic-8.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/graphic-8.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-32">
                               In other words, it is the sum of the average expected benefit from a read of μ bases and the average of a fully sequenced read, which adds further benefit of
                               <span class="inline-formula" id="inline-formula-13">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-13.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-13.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               if the indicator function for that position-orientation combination returns 1. Then,
                               <span class="inline-formula" id="inline-formula-14">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-14.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-14.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is the expected time needed to complete the processing (whether accepted or rejected) of a read:
                               <span class="disp-formula" id="disp-formula-8">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/graphic-9.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/graphic-9.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="inline-formula" id="inline-formula-15">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-15.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-15.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               denotes the size of the strategy, i.e. the number of position-orientation pairs for which the indicator function will return 1, and λ is the mean read length (E[L], as above).
                              </p>
                              <p id="p-33">
                               For simplicity, here we assume uniformity of the distribution of read origins; we present a generalisation used in our implementation in the Supplement (Suppl. Sects. 1.5 and 1.6).
                              </p>
                              <p id="p-34">
                               To compute the optimal strategy, we rank all of the position-orientation combinations (i, o) in decreasing order of the expected benefit gain from sequencing them in their entirety
                               <span class="inline-formula" id="inline-formula-16">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-16.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-16.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . Starting with an empty strategy (one that rejects all reads) we successively include the ranked sites and test after each one whether its contribution results in an improvement over the previous strategy, i.e. whether the current iteration achieves higher gain of benefit per time unit
                               <span class="inline-formula" id="inline-formula-17">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-17.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/04/2020.02.07.938670/embed/inline-graphic-17.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               than the preceding strategy that included one fewer site (position-orientation pair).
                              </p>
                              <p id="p-35">
                               We call our approach of finding an optimal strategy BOSS-RUNS: “Benefit-Optimising Shortterm Strategy for Read Until Nanopore Sequencing”. For further details, an overview of parameters and variables in the model and proof of optimality please see Suppl. Sects. 1.5 and 1.7 and Suppl. Table 1.
                              </p>
                             </div>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              2.3 Real-time implementation
                             </h3>
                             <p id="p-36">
                              Our novel method is implemented to interact with a version of readfish (
                              <a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">
                               Payne et al. 2021
                              </a>
                              ) modified to read updated target lists throughout the experiment (available from
                              <a href="https://github.com/LooseLab/readfish/tree/BossRuns/V0.0.1">
                               https://github.com/LooseLab/readfish/tree/BossRuns/V0.0.1
                              </a>
                              ). The sequencing device performs real-time basecalling and deposits fastq files containing batches of reads (4000 by default). BOSS-RUNS monitors the device’s output and periodically (by default every 30 seconds) includes all new data in an updated decision strategy. In this process new coverage counts are incorporated after mapping basecalled reads to one or more reference genomes using mappy, the python wrapper of min-imap2 (
                              <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                               Li 2018
                              </a>
                              ). If a read maps to more than one position, the best alignment is chosen based on mapping quality or the alignment score of the dynamic programming algorithm in case of a tie. Observed lengths of fully sequenced reads and their mapping positions are continuously used to update the empirical distributions of read lengths L(l) and read start locations and orientations F
                              <sub>
                               i,o
                              </sub>
                              . To prevent the strategy from getting too greedy updates are only applied when a region surpasses a threshold of average coverage (default: ≥5× in 20kb windows).
                             </p>
                             <p id="p-37">
                              In order to keep up with the real-time data stream and to ensure optimality of the strategy at any point in time, new results need to be calculated quickly. For this we use several optimisations including a fast algorithm to find approximate decision strategies, described in Suppl. Sect. 1.8. Our method can use either single or multiple reference chromosomes/genomes as input and optional masks to indicate initial ROIs, similarly to current approaches to adaptive sampling (
                              <a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">
                               Payne et al. 2021
                              </a>
                              ;
                              <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                               Martin et al. 2022
                              </a>
                              ). In that case the scope of the dynamically updated strategies is limited to the ROIs and flanking regions around them and reads originating outside will always be rejected. If multiple references are considered, the expected benefit of reads is calculated separately per reference and then used to derive a common decision strategy across all considered references. This is to ensure that we can account for differences in the distributions of read lengths and read starting positions between genomes, while also sequencing the most informative reads of a mixture instead of focusing on the most informative reads of each individual genome or chromosome.
                             </p>
                             <p id="p-38">
                              BOSS-RUNS is implemented in python and available at
                              <a href="https://github.com/goldman-gp-ebi/BOSS-RUNS">
                               https://github.com/goldman-gp-ebi/BOSS-RUNS
                              </a>
                              . We provide a conda environment for its dependencies: readfish (
                              <a id="xref-ref-45-6" class="xref-bibr" href="#ref-45">
                               Payne et al. 2021
                              </a>
                              ), ONT’s MinKnow API (4.2.4;
                              <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                               Oxford Nanopore Technologies 2021
                              </a>
                              ), numpy (1.21.1;
                              <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                               Harris et al. 2020
                              </a>
                              ), numba (0.53.1;
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               Lam et al. 2015
                              </a>
                              ), scipy (1.7.1;
                              <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">
                               Virtanen et al. 2020
                              </a>
                              ), mappy (2.22;
                              <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">
                               Li 2018
                              </a>
                              ), pandas (1.3.3;
                              <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                               McKinney 2010
                              </a>
                              ), toml (0.10.2;
                              <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                               Pearson 2022
                              </a>
                              ), and natsort (7.1.1;
                              <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                               Morton 2021
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              2.4 Dynamic enrichment of differentially abundant species
                             </h3>
                             <div id="sec-10" class="subsection">
                              <h4>
                               Experimental setup
                              </h4>
                              <p id="p-39">
                               Enrichment of ROIs by rejecting unwanted reads has been previously demonstrated (
                               <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">
                                Loose et al. 2016
                               </a>
                               ;
                               <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                                Kovaka et al. 2021
                               </a>
                               ;
                               <a id="xref-ref-45-7" class="xref-bibr" href="#ref-45">
                                Payne et al. 2021
                               </a>
                               ). BOSS-RUNS can be applied more generally, and makes use of targeted rejections even in the absence of specific ROIs. Here we consider a scenario of whole-genome resequencing where the entire genome is considered of interest, while we showcase a scenario with ROIs in the Supplement (Suppl. Sect. 2.1).
                              </p>
                              <p id="p-40">
                               One situation where the possibility of redistributing data is very effective is in the presence of coverage bias. This can be the case when sequencing either a single organism or multiple genomes. Our first experiment had two major goals. Firstly, we aimed to mitigate bias in coverage across multiple differentially-abundant genomes. Secondly, we demonstrate that using a dynamic approach to adaptive sampling can increase the sampling of sequencing reads from variant or difficult-to-resolve sites without prior knowledge of their location. We therefore sought to sequence eight bacterial species of the well-characterised zymoBIOMICS microbial mixture (ZymoBIOMICS DNA Standard II D6311, Zymo Research). The abundance of the organisms in this mixture is logarithmically distributed with the most abundant species, Listeria monocytogenes, comprising 90% of total DNA, the second most abundant P. aeruginosa circa 9%, the third (B. subtilis) 1%, etc. (
                               <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                                Fig. 2A
                               </a>
                               ). To measure the effects of BOSS-RUNS we divided the available pores on the flowcell and used our new method on one half while the remaining pores did not perform any read rejections and therefore acted as control.
                              </p>
                              <div id="F2" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F2.large.jpg?width=800&height=600&carousel=1" title="BOSS-RUNS can dynamically adapt to the differential abundance of bacteria in a mixture. A) We sequenced eight bacterial species of the zymoBIOMICS mixture with logarithmically distributed abundances covering seven orders of magnitude. E. coli and S. enterica are equally abundant at 0.1% of total DNA in this microbial community. Colors correspond to species as in (B–C) and (E–F). B) Starting with a strategy that accepts reads from any position in all considered genomes, we quickly observe rejections of reads from the most abundant bacteria L. monocytogenes, followed by P. aeruginosa and B. subtilis. The plot shows the proportion of position in each species’ genome for which a read starting there is accepted by the current BOSS-RUNS strategy, over the duration of the experiment. C) The proportion of the accepted fragments that derive from each of the different bacterial strains demonstrates the effect of the changing decision strategy in (B). The inset plot, focusing on the start of the experiment, shows how the strategy rejects almost all L. monocytogenes reads after the first 10min. D) The distribution of read lengths confirms that BOSS-RUNS rejects the majority of sequencing reads, with a clear peak that corresponds to the initial part of a sequencing read used in the decision process. E and F) The coverage distribution across all species using BOSS-RUNS (E) shows depletion of DNA from more abundant genomes in turn for enrichment of information from rare species, compared to the control section on the flowcell (F). Plots show the distribution of coverage depths over sites, with different layers within a plot indicating different time points within the experiment. Accumulation of coverage over time is shown by the distributions’ shift to the right within each panel; note the decreased, but still very high, coverage of L. monocytogenes using BOSS-RUNS, and the increased coverage for the other (less abundant) species. Results from the three least abundant bacterial species are omitted due to the differences not being obvious in this type of visualisation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-380954308" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">BOSS-RUNS can dynamically adapt to the differential abundance of bacteria in a mixture. A) We sequenced eight bacterial species of the zymoBIOMICS mixture with logarithmically distributed abundances covering seven orders of magnitude. <em>E. coli</em> and <em>S. enterica</em> are equally abundant at 0.1% of total DNA in this microbial community. Colors correspond to species as in (B–C) and (E–F). B) Starting with a strategy that accepts reads from any position in all considered genomes, we quickly observe rejections of reads from the most abundant bacteria <em>L. monocytogenes</em>, followed by <em>P. aeruginosa</em> and <em>B. subtilis</em>. The plot shows the proportion of position in each species’ genome for which a read starting there is accepted by the current BOSS-RUNS strategy, over the duration of the experiment. C) The proportion of the accepted fragments that derive from each of the different bacterial strains demonstrates the effect of the changing decision strategy in (B). The inset plot, focusing on the start of the experiment, shows how the strategy rejects almost all <em>L. monocytogenes</em> reads after the first 10min. D) The distribution of read lengths confirms that BOSS-RUNS rejects the majority of sequencing reads, with a clear peak that corresponds to the initial part of a sequencing read used in the decision process. E and F) The coverage distribution across all species using BOSS-RUNS (E) shows depletion of DNA from more abundant genomes in turn for enrichment of information from rare species, compared to the control section on the flowcell (F). Plots show the distribution of coverage depths over sites, with different layers within a plot indicating different time points within the experiment. Accumulation of coverage over time is shown by the distributions’ shift to the right within each panel; note the decreased, but still very high, coverage of <em>L. monocytogenes</em> using BOSS-RUNS, and the increased coverage for the other (less abundant) species. Results from the three least abundant bacterial species are omitted due to the differences not being obvious in this type of visualisation.</div></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F2.medium.gif" width="296" height="440"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F2.medium.gif" width="296" height="440"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 2:
                                </span>
                                <p id="p-41" class="first-child">
                                 BOSS-RUNS can dynamically adapt to the differential abundance of bacteria in a mixture. A) We sequenced eight bacterial species of the zymoBIOMICS mixture with logarithmically distributed abundances covering seven orders of magnitude. E. coli and S. enterica are equally abundant at 0.1% of total DNA in this microbial community. Colors correspond to species as in (B–C) and (E–F). B) Starting with a strategy that accepts reads from any position in all considered genomes, we quickly observe rejections of reads from the most abundant bacteria L. monocytogenes, followed by P. aeruginosa and B. subtilis. The plot shows the proportion of position in each species’ genome for which a read starting there is accepted by the current BOSS-RUNS strategy, over the duration of the experiment. C) The proportion of the accepted fragments that derive from each of the different bacterial strains demonstrates the effect of the changing decision strategy in (B). The inset plot, focusing on the start of the experiment, shows how the strategy rejects almost all L. monocytogenes reads after the first 10min. D) The distribution of read lengths confirms that BOSS-RUNS rejects the majority of sequencing reads, with a clear peak that corresponds to the initial part of a sequencing read used in the decision process. E and F) The coverage distribution across all species using BOSS-RUNS (E) shows depletion of DNA from more abundant genomes in turn for enrichment of information from rare species, compared to the control section on the flowcell (F). Plots show the distribution of coverage depths over sites, with different layers within a plot indicating different time points within the experiment. Accumulation of coverage over time is shown by the distributions’ shift to the right within each panel; note the decreased, but still very high, coverage of L. monocytogenes using BOSS-RUNS, and the increased coverage for the other (less abundant) species. Results from the three least abundant bacterial species are omitted due to the differences not being obvious in this type of visualisation.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-42">
                               To mimic a realistic sequencing experiment where we do not have prior knowledge about the exact bacterial strains contained in the mixture we used assemblies of closely-related strains instead of accurate assemblies generated from the strains known to be present in the community (see Sect. 4.2 for accessions). Additionally, this allowed us to evaluate the performance of our method in focusing on sites that differed between the reference genomes we used and the true genomes. Priors of genotype probabilities were also initialised using these references.
                              </p>
                             </div>
                             <div id="sec-11" class="subsection">
                              <h4>
                               BOSS-RUNS’ strategy
                              </h4>
                              <p id="p-43">
                               During the sequencing experiment we can observe how the decision strategy changes over time. As the genomes of individual bacteria are continuously resolved, i.e. we become more certain about the genotype at many sites, the proportion of positions at which we still require more information decreases. Due to the differential abundance of the considered species, L. monocytogenes is considered mostly resolved after only a few minutes followed later by P. aeruginosa and B. subtilis (
                               <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                                Fig. 2B
                               </a>
                               ). Accordingly, the proportion of accepted reads demonstrates that the focus switches from the most abundant bacteria towards rarer species (
                               <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                                Fig. 2C
                               </a>
                               ). Interestingly, the rate at which individual genomes are resolved is not equal across all bacteria in the mixture. For example, the proportion of fragment start sites from which reads would be accepted from L. monocytogenes or B. subtilis decreases to values close to 0. On the other hand, P. aeruginosa approaches a level of ~5.8% at the end of the experiment and does so at a slower rate. In other words, some sites in the genome of P. aeruginosa require more data to be confidently resolved and a portion of sites remains uncertain despite sampling data throughout the run. This is in part due to different levels of divergence between the strains in the zymo community and the reference genomes we used during the experiment, and in part due to differential effects of coverage bias also within each species’ genome.
                              </p>
                              <p id="p-44">
                               Given the large difference in abundance and the prompt resolution of L. monocytogenes, we expect the majority of sequencing reads to be rejected throughout the experiment. Indeed, the distribution of observed read lengths confirms that BOSS-RUNS ejects most molecules resulting in a peak at ~480bp (
                               <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                                Fig. 2D
                               </a>
                               ). The bimodality of the read length distribution after separating the sequencing data by target species also corresponds to our expectations given the proportion of rejected reads from each species (Suppl. Fig. 3). Interestingly, the presence of a peak at around 480bp in the read length distribution of rare species indicates that some reads from these species are also rejected. The majority of these false rejections (84%) was due to inability to determine the source species from the initial fragment. Notably, sequencing reads in this experiment were generally relatively short due to the necessity of preceding amplification.
                              </p>
                             </div>
                             <div id="sec-12" class="subsection">
                              <h4>
                               Improved sequencing of bacterial species
                              </h4>
                              <p id="p-45">
                               The effect of the changing decision strategy becomes evident when looking at the distribution of coverage depth over time. Sequencing coverage from the most abundant species is effectively redistributed to the other scarcer species compared to the control (
                               <a id="xref-fig-2-5" class="xref-fig" href="#F2">
                                Fig. 2E, F
                               </a>
                               ). For example, for E. coli and S. enterica, which comprise only 0.1% of the input DNA, we achieve 3.9 and 4 times the coverage compared to the control.
                              </p>
                              <p id="p-46">
                               Changes in mean coverage over time confirm these observations regarding the overall shape of the coverage distributions. Indeed, sacrificing data from heavily sampled organisms enables us to obtain more DNA from rare species (
                               <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                                Fig. 3A
                               </a>
                               ). For example, BOSS-RUNS achieves between 4.1 and 5.8 times more coverage of the scarce bacteria (E. coli, S. enterica, L. fermentum, E. faecalis). Focusing on low-coverage sites rather than mean coverage, the proportion of sites with coverage less than 5 × also highlights the advantage gained by using our dynamic decision strategies. This quantity decreases quicker, and reaches lower final levels, compared to the control for all but the most abundant genome (
                               <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                                Fig. 3B
                               </a>
                               ). The redistribution of sampled sequencing data from regions already well-covered to areas of low coverage is one of the main features of BOSSRUNS. In the case of B. subtilis, for example, this leads to less than 5% of sites with coverage less than 5× with BOSS-RUNS, against ~44% for the control. In rare species we checked whether the sites with a coverage of more than 5× were caused by reads mapping to repeats or other regions of low complexity. Analysing the overlap of these sites and the respective repeat-masked genomes revealed that this was not the case and the observed coverage at these sites was not artefactual but true enrichment (Suppl. Table 2).
                              </p>
                              <div id="F3" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F3.large.jpg?width=800&height=600&carousel=1" title="Various metrics demonstrate the improvements in sequencing the bacterial community using BOSS-RUNS (solid lines) and control (dashed lines) on the same flowcell in an experiment lasting 10h. A) The mean coverage depth across the genomes of all considered species. Sequencing coverage of more abundant species will generally be traded-off by BOSS-RUNS to collect more data from rare species. Upon (mostly) resolving individual genomes a change in the rate of data accumulation is visible, e.g. after ~180min for B. subtilis. B) The proportion of sites that remain at 0.99. Across all species BOSS-RUNS achieves lower numbers of unresolved sites throughout the experiment owing to both sampling more data from rarer species and redistributing data within each genome. D) Coverage evenness describes the homogeneity of observed data across a genome independent of absolute coverage. By focusing the sequencing effort on sites with low coverage our method leads to more even distribution of coverage. Note the different scales on the y-axes of all plots, to allow for the sampling statistics of species of widely varying abundances." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-380954308" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Various metrics demonstrate the improvements in sequencing the bacterial community using BOSS-RUNS (solid lines) and control (dashed lines) on the same flowcell in an experiment lasting 10h. A) The mean coverage depth across the genomes of all considered species. Sequencing coverage of more abundant species will generally be traded-off by BOSS-RUNS to collect more data from rare species. Upon (mostly) resolving individual genomes a change in the rate of data accumulation is visible, e.g. after ~180min for <em>B. subtilis</em>. B) The proportion of sites that remain at <5× depth reveals that data is efficiently redistributed by BOSS-RUNS to areas of low coverage. C) Using the genotype posterior probability distribution, we can classify sites as resolved if the probability of one genotype is >0.99. Across all species BOSS-RUNS achieves lower numbers of unresolved sites throughout the experiment owing to both sampling more data from rarer species and redistributing data within each genome. D) Coverage evenness describes the homogeneity of observed data across a genome independent of absolute coverage. By focusing the sequencing effort on sites with low coverage our method leads to more even distribution of coverage. Note the different scales on the <em>y</em>-axes of all plots, to allow for the sampling statistics of species of widely varying abundances.</div></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F3.medium.gif" width="280" height="440"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F3.medium.gif" width="280" height="440"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 3:
                                </span>
                                <p id="p-47" class="first-child">
                                 Various metrics demonstrate the improvements in sequencing the bacterial community using BOSS-RUNS (solid lines) and control (dashed lines) on the same flowcell in an experiment lasting 10h. A) The mean coverage depth across the genomes of all considered species. Sequencing coverage of more abundant species will generally be traded-off by BOSS-RUNS to collect more data from rare species. Upon (mostly) resolving individual genomes a change in the rate of data accumulation is visible, e.g. after ~180min for B. subtilis. B) The proportion of sites that remain at <5× depth reveals that data is efficiently redistributed by BOSS-RUNS to areas of low coverage. C) Using the genotype posterior probability distribution, we can classify sites as resolved if the probability of one genotype is >0.99. Across all species BOSS-RUNS achieves lower numbers of unresolved sites throughout the experiment owing to both sampling more data from rarer species and redistributing data within each genome. D) Coverage evenness describes the homogeneity of observed data across a genome independent of absolute coverage. By focusing the sequencing effort on sites with low coverage our method leads to more even distribution of coverage. Note the different scales on the y-axes of all plots, to allow for the sampling statistics of species of widely varying abundances.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-48">
                               We can also use the uncertainty about each sites’ genotype, as defined in our model, to classify sites into two groups, unresolved and resolved. When the posterior probability of one genotype at a site surpasses 0.99 we declare that site as resolved, and can count the sites which still require more data to reach that level of certainty. Again BOSS-RUNS shows higher performance by reaching lower numbers of unresolved sites in a shorter time (
                               <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                                Fig. 3C
                               </a>
                               ).
                              </p>
                              <p id="p-49">
                               Balancing coverage bias across genomes is not the only benefit of our new method though, as data is also redistributed within individual genomes. This effect is partly responsible for the gains described so far, but is overshadowed by the stark species abundance differences. By using a measure of evenness that describes the uniformity of coverage distribution and is relatively independent of the absolute coverage (
                               <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                                Mokry et al. 2010
                               </a>
                               ), we observe that BOSS-RUNS not only boosts the coverage of rare species, but also ensures that coverage is more uniform in all species, including those of higher abundance (e.g. P. aeruginosa and B. subtilis,
                               <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                                Fig. 3D
                               </a>
                               ). Even in cases where the total collected coverage of a bacterial strain might be lower it is possible that this more uniform distribution of coverage and focus on sites with higher uncertainty could achieve a more desirable outcome of the experiment with BOSS-RUNS, although in our experiment this effect is not readily visible in
                               <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                                Fig. 3
                               </a>
                               for L. monocytogenes, we note that the improved precision of SNP detection for this species (see below;
                               <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                                Fig. 5B
                               </a>
                               ) could be due to these effects.
                              </p>
                             </div>
                             <div id="sec-13" class="subsection">
                              <h4>
                               Redistributing coverage to under-sampled sites
                              </h4>
                              <p id="p-50">
                               Another way to explore the redistribution of data within genomes is to examine the already observed coverage at the sites that a read maps to when the decision about that read was made. If our method indeed successfully focuses on reads from areas with highest uncertainty we expect the mean coverage at sites spanned by accepted reads to be lower than sites spanned by rejected reads. An even bigger effect might be visible when looking at the minimum coverage within those spans, since the decision to accept reads might be driven by low coverage at individual sites as opposed to low average coverage in an area. To investigate this, we separated the reads by the decision made during the experiment: either to sequence them entirely or to reject them. We then record the mean and minimum coverage of all sites that a read maps to and pool those measurements at different timepoints.
                              </p>
                              <p id="p-51">
                               To demonstrate BOSS-RUNS’ effects, we focus on the results for a single species, B. subtilis, for which the decision strategy accepts all reads until ~80min into the sequencing experiment; then rejects an increasing proportion of reads; and is rejecting most, but not all, reads from ~200min onwards (see
                               <a id="xref-fig-2-6" class="xref-fig" href="#F2">
                                Fig. 2B
                               </a>
                               ). Looking at the background mean coverage at sites spanned by accepted and rejected reads, we see an overlap up until the size of the strategy starts to diminish (
                               <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                                Fig. 4
                               </a>
                               , left). Throughout the rest of the experiment, as expected, both the mean and minimum coverage of sites spanned by accepted reads are consistently lower than for rejected reads, albeit with larger variation due to the decreasing number of fragments sequenced in their entirety (
                               <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                                Fig. 4
                               </a>
                               ). In summary, this shows that despite this genome being mostly resolved early in the experiment, we continue to sample fragments covering positions of low average coverage or where we only have minimal information about individual sites. These results confirm that BOSS-RUNS focuses on these reads not solely due to the abundance difference but also due to the coverage variation within the data of B. subtilis. Notably, it might be surprising that we observe a few rejections of reads originating from this species right from the start of the experiment. This is due to reads failing to map when using only their initial parts during the decision process.
                              </p>
                              <div id="F4" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F4.large.jpg?width=800&height=600&carousel=1" title="Coverage is effectively redistributed by sampling more data from sites with low coverage. Focusing on B. subtilis, the mean observed coverage of reads’ mapping locations first overlaps for rejected and accepted reads (left). As more fragments are rejected from pores, the mean coverage of sites spanned by accepted reads becomes, and stays, consistently lower than for rejected ones. We observe an even larger difference in the minimum coverage of regions reads map to (right). In combination, these demonstrates that data is continually sampled from areas of low coverage even after most of this species’ genome has already been resolved." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-380954308" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Coverage is effectively redistributed by sampling more data from sites with low coverage. Focusing on <em>B. subtilis</em>, the mean observed coverage of reads’ mapping locations first overlaps for rejected and accepted reads (left). As more fragments are rejected from pores, the mean coverage of sites spanned by accepted reads becomes, and stays, consistently lower than for rejected ones. We observe an even larger difference in the minimum coverage of regions reads map to (right). In combination, these demonstrates that data is continually sampled from areas of low coverage even after most of this species’ genome has already been resolved.</div></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F4.medium.gif" width="440" height="200"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F4.medium.gif" width="440" height="200"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 4:
                                </span>
                                <p id="p-52" class="first-child">
                                 Coverage is effectively redistributed by sampling more data from sites with low coverage. Focusing on B. subtilis, the mean observed coverage of reads’ mapping locations first overlaps for rejected and accepted reads (left). As more fragments are rejected from pores, the mean coverage of sites spanned by accepted reads becomes, and stays, consistently lower than for rejected ones. We observe an even larger difference in the minimum coverage of regions reads map to (right). In combination, these demonstrates that data is continually sampled from areas of low coverage even after most of this species’ genome has already been resolved.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-14" class="subsection">
                             <h3>
                              2.5 Focused sequencing leads to improved variant calls
                             </h3>
                             <p id="p-53">
                              Next, we sought to perform variant calling for five of the eight bacterial species in the microbial mixture. (We excluded the three least abundant species from this analysis, as we did not collect enough data to make reliable calls.) With this analysis we tried to answer the questions of (a) whether we could successfully sample data from rare species in order to better identify the differences between reference assemblies and the true sample genomes, and (b) whether BOSS-RUNS can effectively focus on sites where we observe variation and therefore increased uncertainty.
                             </p>
                             <p id="p-54">
                              Our analysis is based on comparing inferred variants from data accumulated using BOSSRUNS (or the control) to a ground truth derived from deep, short read sequencing of the same bacterial strains (see
                              <a id="xref-sec-16-1" class="xref-sec" href="#sec-16">
                               Methods
                              </a>
                              , Sect. 4.3). By making comparisons at multiple timepoints throughout our experiment, we show how knowledge of variants accumulates over time (
                              <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                               Fig. 5
                              </a>
                              ); in future, such information could be used to shorten the duration of experiments needed to achieve particular levels of accuracy. For the most abundant species, L. monocytogenes, the decreased coverage with BOSS-RUNS leads to slightly lower sensitivity than for the control case for that species. Nevertheless, high sensitivity is achieved in a very short time, and the effective redistribution of coverage within this species’ genome leads to increased precision. In turn, however, for all the other species the increased and better-targeted coverage produced by BOSS-RUNS (see above) means more variants are discovered, with improved sensitivity and precision.
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F5.large.jpg?width=800&height=600&carousel=1" title="Dynamic, adaptive sampling leads to improved discovery of SNPs. We compared the variants called from data collected without adaptive sampling (control) and with BOSS-RUNS to a set of ground truth variants from deep, short read sequencing of the same bacterial strains. By performing variant discovery at different timepoints we can gain further insight into the advantages of our method. A) Whereas the sensitivity of BOSS-RUNS is slightly lower for the most abundant species L. monocytogenes, we observe a larger number of discovered true positives in all remaining genomes. To highlight differences, we set the y-axis ranges to 0–0.95 for the first three species, and 0–0.035 for the remaining two. B) The precision of variants called from data generated using BOSS-RUNS is at a similar level to the control or moderately higher." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-380954308" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Dynamic, adaptive sampling leads to improved discovery of SNPs.</span> We compared the variants called from data collected without adaptive sampling (control) and with BOSS-RUNS to a set of ground truth variants from deep, short read sequencing of the same bacterial strains. By performing variant discovery at different timepoints we can gain further insight into the advantages of our method. A) Whereas the sensitivity of BOSS-RUNS is slightly lower for the most abundant species <em>L. monocytogenes</em>, we observe a larger number of discovered true positives in all remaining genomes. To highlight differences, we set the y-axis ranges to 0–0.95 for the first three species, and 0–0.035 for the remaining two. B) The precision of variants called from data generated using BOSS-RUNS is at a similar level to the control or moderately higher.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F5.medium.gif" width="440" height="295"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F5.medium.gif" width="440" height="295"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/04/2020.02.07.938670/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5:
                               </span>
                               <span class="caption-title">
                                Dynamic, adaptive sampling leads to improved discovery of SNPs.
                               </span>
                               <p id="p-55" class="first-child">
                                We compared the variants called from data collected without adaptive sampling (control) and with BOSS-RUNS to a set of ground truth variants from deep, short read sequencing of the same bacterial strains. By performing variant discovery at different timepoints we can gain further insight into the advantages of our method. A) Whereas the sensitivity of BOSS-RUNS is slightly lower for the most abundant species L. monocytogenes, we observe a larger number of discovered true positives in all remaining genomes. To highlight differences, we set the y-axis ranges to 0–0.95 for the first three species, and 0–0.035 for the remaining two. B) The precision of variants called from data generated using BOSS-RUNS is at a similar level to the control or moderately higher.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-56">
                              Interestingly, even for the two bacteria P. aeruginosa and B. subtilis, which are considered mostly resolved by our method during the experiment leading to the majority of reads being rejected, we still see an increase in sensitivity at later stages of the run (
                              <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                               Fig. 5A
                              </a>
                              ). This is due to the ability of our method to sample more data specifically at positions where it is conducive to reducing uncertainty about the genotype. For example, after 10h of sequencing BOSS-RUNS finds 26,481 variant sites in P. aeruginosa (sensitivity 0.79), whereas we observe 23,541 SNPs from control data (sensitivity 0.68), despite the decision strategy rejecting fragments from >80% of the genome after the first 3h. At the same time the precision of variant calls on the data collected with BOSS-RUNS is either moderately higher or at a similar level to the control dataset (
                              <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                               Fig. 5B
                              </a>
                              ). In the rarer species the advantage of BOSS-RUNS simply collecting more data is evident as we are able to call SNPs at least in some regions (119 and 80 SNPs detected after 10h for E. coli and S. enterica, respectively), whereas the control data does not contain enough reads to produce any variant calls.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-15">
                            <h2 class="">
                             3 Discussion
                            </h2>
                            <p id="p-57">
                             In this work we have shown that our novel approach to dynamic, adaptive sampling for nanopore sequencing, implemented in BOSS-RUNS, provides a mathematical framework and fast algorithms to generate decisions strategies that optimise the rate of information gain during resequencing experiments. Compared to sequencing without adaptive sampling, this leads to an increase in the sequencing yield of on-target regions, specifically at positions of highest uncertainty. Additionally, it can effectively mitigate abundance bias or other sources of non-uniform coverage, for example from enrichment library preparation procedures, leading to smaller proportions of sites at low coverage depth and greater evenness of coverage. Furthermore, our methods can lead to improved discovery of variants by both sampling more data from negatively biased regions or species in the input material and by focusing the sequencing on sites where the underlying genotype is not clear from the data observed up to that point in time. Overall, our method can help to shorten the time-to-answer and reduce the cost of sequencing.
                            </p>
                            <p id="p-58">
                             Current methods of adaptive sampling base the rejection of sequencing reads solely on whether they cover some ROIs that were defined before starting the sequencing run. Instead, we can change our targets throughout an experiment in order to collect data where it is most useful to reduce the uncertainty about the underlying genotypes at each site.
                            </p>
                            <p id="p-59">
                             We demonstrated the effectiveness of our approach by sequencing a microbial mixture affected by coverage bias. We show that BOSS-RUNS adapts to the differential abundance of the targeted genomes without prior knowledge about the composition of the input library. It not only reduces the inter-species bias, but also redistributes coverage effectively within-species.
                            </p>
                            <p id="p-60">
                             In common with any resequencing experiment, the only piece of prior knowledge we do require is a reference genome that is related to the organism(s) we expect to observe in the sequenced material. This is used for two purposes. Firstly, we initialise the genotype prior probabilities according to the reference. These can either be informed by the nucleotides present in the chosen reference or they can be uniform for all possible genotypes, perhaps depending on how much divergence we expect in the input material. Priors signifying different amount of confidence in the chosen reference could also be used, but are not implemented due to the negligible effect of the priors in many cases (see Suppl. Fig. 1). Secondly, the reference genome is used to determine the origin and orientation of nascent sequencing reads. This is the basis for making decision: barring major structural rearrangements, moderate amounts of divergence will not have a substantial impact on the performance of BOSS-RUNS.
                            </p>
                            <p id="p-61">
                             Determining the mapping location of the initial part of a read using minimap2 does not achieve perfect sensitivity or precision, and indeed we see some reads rejected due to them not mapping with sufficient confidence (see Suppl. Fig. 3). Admittedly, using minimap2’s preset parameters for long reads is not ideal in this case and optimising parameters to better accommodate such rather short reads but with the error profile of long reads could reduce the number of falsely determined off-target reads. Alternatively, other methods for read classification could be used such as the recently described ReadBouncer, based on the DREAM index (
                             <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">
                              Ulrich et al. 2022
                             </a>
                             ).
                            </p>
                            <p id="p-62">
                             However, our combination with readfish, which uses GPU-accelerated high-accuracy basecalling, seems to be affected to a lesser degree than ONT’s built-in adaptive sampling, which has been reported to falsely reject more than one third of on-target reads in some cases (
                             <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">
                              Martin et al. 2022
                             </a>
                             ).
                            </p>
                            <p id="p-63">
                             In our experiment sequencing a mock bacterial community we use reference assemblies that are closely related to the actual strains contained in the mixture but not identical, to mimic a real setting where we do not know the true composition in advance. We measured their divergence in terms of the percentage of aligning nucleotides and ANI values using JSpecies (
                             <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                              Richter et al. 2016
                             </a>
                             ), which range from 86.07 to 99.7% and 98.82 to 99.92, respectively (Suppl. Fig. 4). We can see that the divergence between the chosen reference and the true experimental strain has potential implications on the performance of BOSS-RUNS. For example, the reference we used for P. aeruginosa harbours more differences from the true genome than there are between the reference and true genome of L. monocytogenes, leading to the visibly different respective plateaus of remaining unresolved sites approached during the experiment (
                             <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                              Fig. 3C
                             </a>
                             ). An important influence in the case of prokaryotes is the concept of pangenomes and the possibility of differential composition of accessory genes (
                             <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                              Medini et al. 2005
                             </a>
                             ;
                             <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                              Ozer et al. 2014
                             </a>
                             ). If a mixture contains more than one strain of a bacterial species we expect that sequences from all strains will be enriched, so long as they are contained in the chosen reference, as we will not be able to differentiate their origin with enough resolution. Indeed,
                             <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">
                              Martin et al. (2022)
                             </a>
                             have shown that targeting a single strain of E. coli also led to the enrichment of four other strains contained in the same gut microbiome standard. This phenomenon might also limit the advantage of BOSS-RUNS in extreme cases, since we will not be able to resolve regions of an assembly that are missing in the input library. We have implemented ways for BOSS-RUNS to ignore such missing areas, but still need to improve these techniques in order to better differentiate between absent parts of the genome and very low or spurious coverage, e.g. to prevent continuously waiting for reads from small parts of a genome in the hopes of resolving them, as visible in
                             <a id="xref-fig-2-7" class="xref-fig" href="#F2">
                              Fig. 2B
                             </a>
                             for P. aeruginosa. Alternatively we could also implement the use of pan-genome graphs to account for variable composition of strains instead of linear assemblies (
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              Colquhoun et al. 2021
                             </a>
                             ). For eukaryotes this limitation will have a much smaller impact due to the generally larger proportion of core vs. accessory gene content (
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              Brockhurst et al. 2019
                             </a>
                             ).
                            </p>
                            <p id="p-64">
                             Another possible limitation of our method comes from the fact that strategies generated by BOSS-RUNS are optimised to gain the most benefit in the short term. Early on during a sequencing run they may reject fragments that would be considered relatively more useful later on, which might seem counter-intuitive and can make the computed strategies greedy (in the sense described by
                             <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                              Jones and Pevzner 2004
                             </a>
                             ). As an example, we might reject reads from regions with average uncertainty at one point in time to focus instead on areas of high uncertainty; however, regions with average uncertainty could themselves turn into regions with relatively high uncertainty as the experiment continues: past rejection of reads from these areas might not have been advantageous in the long term. Further work will be needed to address this issue.
                            </p>
                            <p id="p-65">
                             As mentioned above, the read lengths in our applications (modelled by the distribution L with mean λ) are relatively short with a mean of 3.11kb, which was in part due to the need for amplification in order to achieve sufficiently high molecular weight DNA. But this might serve as a proxy for the challenging nature of extracting DNA from metagenomic samples, which often relies on harsh, multi-step procedures to ensure that cells from all contained species are lysed and genomic material is available for sequencing (
                             <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                              Quick 2019
                             </a>
                             ) and can lead to reads that are shorter than desired. For such short fragments the time needed for processing, rejection and acquisition of an alternative read (μ + ρ + α in our model) might approach the elapsed time for fully sequencing a typical read in its entirety. If reads were even shorter, using adaptive sampling might even be detrimental to the total amount of collected sequence.
                             <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">
                              Martin et al. (2022)
                             </a>
                             have recently modeled the influence of read length of the input library on the practically achievable enrichment (using Read Until but not dynamically updated strategies) of differentially abundant species in a mixture. They showed that average read length is a major determinant of the maximum level of enrichment and that the longer the reads, the larger the potential enrichment becomes (
                             <a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">
                              Martin et al. 2022
                             </a>
                             ). This may explain the moderate level of overall enrichment of the yield in our sequencing experiment.
                            </p>
                            <p id="p-66">
                             Other factors that could influence the advantages of our approach include the availability of DNA molecules for sequencing at the flowcell surface. Whereas we did not observe differences in the acquisition time of new fragments at pores, we could imagine that experimental conditions might influence this parameter, for example DNA concentration throughout a sequencing run. In addition, the effects of variation of the duration and magnitude of the voltage reversal for effecting rejections have not been well-characterised. Exploring different experimental conditions and parameters in the context of adaptive sampling poses interesting questions for future studies of this technique.
                            </p>
                            <p id="p-67">
                             A further consideration is any negative effect of repeatedly rejecting fragments on the status of nanopores on the flowcell. Pores might get blocked by DNA molecules that are not successfully ejected, which can reduce the number of active pores when using adaptive sampling (
                             <a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">
                              Loose et al. 2016
                             </a>
                             ). The use of nuclease flushed across the flowcell in order to free such clogged pores has been shown to mitigate this issue and recover blocked pores for sequencing (
                             <a id="xref-ref-45-8" class="xref-bibr" href="#ref-45">
                              Payne et al. 2021
                             </a>
                             ;
                             <a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">
                              Martin et al. 2022
                             </a>
                             ). In our experiment, however, we did not observe any indications that the performance of the section on the flowcell running BOSS-RUNS was degrading faster than the control section when comparing the number of data-transmitting channels and their idling times (Suppl. Fig. 5, Suppl. Fig. 6) and others have reported similarly small effects on pore health (
                             <a id="xref-ref-32-7" class="xref-bibr" href="#ref-32">
                              Martin et al. 2022
                             </a>
                             ). Depending on factors such as the proportion of rejected reads, read lengths and most importantly the total duration of the experiment, the impact on pore health might vary and especially for long sequencing experiments nuclease flushing could be important to retain a high number of active pores.
                            </p>
                            <p id="p-68">
                             In the future we would like to apply our method to a wider array of biological problems. For example, we could adapt our model to quantify the uncertainty about the presence of epigenetic modifications, such an methylation, which are now becoming analyzable in real time with nanopore sequencing (
                             <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">
                              Simpson et al. 2017
                             </a>
                             ;
                             <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                              Leger et al. 2021
                             </a>
                             ;
                             <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                              Liu et al. 2021
                             </a>
                             ;
                             <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">
                              Zhang et al. 2021
                             </a>
                             ). In that case we could also inform priors with sequence context, in order to take into account that some modifications such as 5-methylcytosine (5mC) occur most frequently at CpG dinucleotides (
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              Ehrlich et al. 1982
                             </a>
                             ;
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              Bird 1986
                             </a>
                             ). BOSS-RUNS could also be used to overcome biases that are inevitably introduced by some library preparation methods. In whole-exome sequencing, an example of one of the most widely used resequencing techniques, differential pull-down efficiencies of genes and also different exons within genes can lead to substantial coverage bias (
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              Barbitoff et al. 2020
                             </a>
                             ) which BOSS-RUNS could adapt to during an experiment and therefore help mitigate.
                            </p>
                            <p id="p-69">
                             Further, we plan to extend our model to collect additional information besides the observed nucleotides at each site. For example, we could record the distribution of read lengths in different areas of the genome to focus sequencing effort on covering an entire genome with sufficient number of very long reads in order to maximise the contiguity of assembled contigs. Creating an approximate assembly in real-time could also eliminate the need for a reference genome when making decisions with BOSS-RUNS. We could also use linkage information from observing adjacent variants in the same sequencing read in order to express information about which haplotype the observed variation comes from in diploid or polyploid samples. BOSS-RUNS could then enrich for fragments that have the potential of bridging across variants and therefore help construct haplotype-resolved assemblies, at least in genomic regions with enough diversity. This could aid in the ongoing effort of obtaining completely phased de novo genomes from single individuals, and might reduce the complexity of currently used sequencing protocols or combinations of methods (
                             <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">
                              Soifer et al. 2020
                             </a>
                             ;
                             <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                              Porubsky et al. 2021
                             </a>
                             ).
                            </p>
                            <p id="p-70">
                             Other innovations of nanopore sequencing in general might include the ability to repeatedly translocate the same native genomic fragment, which has been previously demonstrated for solid-state nanopores albeit without a nucleotide readout (
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              Gershow and Golovchenko 2007
                             </a>
                             ), and more recently also by using helicases to translocate a DNA-peptide construct through protein nanopores (
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              Brinkerhoff et al. 2021
                             </a>
                             ). If this was possible in a controlled fashion, the methods described here could be adapted to decide dynamically which reads to recapture and how many times to sequence them until their information content is exhausted and the sites they cover are resolved to some desired accuracy.
                            </p>
                            <p id="p-71">
                             To conclude, our method expands the applicability of adaptive sampling and can potentially improve the information gain in many standard scenarios by dynamically enriching areas of highest uncertainty, e.g. genomic regions with increased divergence, discrepancy to expectations or simply lower coverage depth than the average. This is achieved by utilizing information derived in real time throughout a sequencing experiment, and can be used without prior knowledge of variation of the sample from a reference sequence. Ensuring more homogeneous coverage and overall less uncertainty about genotypes by focusing on biologically interesting sites leads to improved efficiency of sequencing using nanopores in a wide range of applications. The resulting reduction in the time-to-answer might be critical in a clinical setting or in pathogen surveillance.
                            </p>
                           </div>
                           <div class="section" id="sec-16">
                            <h2 class="">
                             4 Methods
                            </h2>
                            <div id="sec-17" class="subsection">
                             <h3>
                              4.1 Configuration of sequencing experiments
                             </h3>
                             <p id="p-72">
                              Sequencing was conducted on an ONT GridION using R9.4 flowcells. Since the quality and number of active nanopores can vary between flowcells it would be difficult to compare experiments involving adaptive sampling performed on multiple flowcells. Therefore we separated a single flowcell by assigning 256 channels to each of two different conditions. One of these two regions used a decision strategy that continuously accepts any encountered read, i.e. a control sector not performing any adaptive sampling, whereas the other was acting according to the decision strategies provided by BOSS-RUNS. Readfish was configured to reject reads from this sector if they did not map or mapped to (one or more) off-target sites, i.e. sites not included in the current decision strategy, or if no sequence was obtained from a fragment. For all our experiments we used 0.8s of data to infer the genomic origin and orientation of fragments before making decisions, i.e. roughly 350bp (corresponding to μ in our model,
                              <a id="xref-fig-1-9" class="xref-fig" href="#F1">
                               Fig. 1F
                              </a>
                              ), which results in a mean read length of 482bp for rejected reads due also to the additional time (ρ) taken to process and effect decisions. BOSS-RUNS deposits new strategies as compressed Boolean numpy arrays for each genome or chromosome, which are subsequently reloaded by readfish upon file modification. Communicating rejection signals to the sequencing device is performed by readfish.
                             </p>
                            </div>
                            <div id="sec-18" class="subsection">
                             <h3>
                              4.2 Sequencing and analysis of the ZymoBIOMICS microbial reference
                             </h3>
                             <p id="p-73">
                              Input DNA from the ZymoBIOMICS Microbial Community DNA Standard II (Log Distribution D6311, Zymo Research) was prepared using SQK-LSK110 (ONT) and PCR amplified using the PCR expansion kit EXP-PCA001 (ONT). BOSS-RUNS and readfish depend on reference genomes to infer the origin of sequencing reads. In order to create a more realistic scenario where we do not know the exact bacterial strains, we decided not to use reference genomes from the strains contained in the microbial mixture, but instead used closely-related reference genomes identified by
                              <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                               McIntyre et al. (2019)
                              </a>
                              . The employed assemblies are available in the European Nucleotide Archive under accessions ASM14656v1, ASM584v2, ASM400627v1, ASM39716v1, ASM30761v1, ASM51030v1, ASM25313v1, ASM810v1. Basecalling was performed using Guppy (5.0.16), set to high-accuracy mode.
                             </p>
                             <p id="p-74">
                              To test whether increased coverage of rare species was due to repeats or low-complexity regions, we used RepeatMasker (4.1.2, default parameters;
                              <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">
                               Smit et al. 2015
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-19" class="subsection">
                             <h3>
                              4.3 Variant calling of bacterial species
                             </h3>
                             <p id="p-75">
                              To perform variant calling we used sequencing reads separated by their species of origin (using minimap2;
                              <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">
                               Li 2018
                              </a>
                              ) and further partitioned them to comprise the cumulative data from the beginning of the experiment up to and including 20 individual timepoints, each separated by approximately 30min of sequencing (using custom python scripts).
                             </p>
                             <p id="p-76">
                              To create a set of high-confidence variants we used publicly-available deep coverage short read sequencing of the zymoBIOMICS microbial community with evenly distributed abundances, which contains the same strains as the logarithmically distributed mixture (Zymo Research D6306). These data are available in the European Nucleotide Archive under the accession SRR13224035. Briefly, we mapped the separated reads to their respective assemblies (see previous section) using minimap2 (2.22;
                              <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">
                               Li 2018
                              </a>
                              ) and samtools (1.12;
                              <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                               Danecek et al. 2021
                              </a>
                              ), marked duplicates using picard (2.26.6, default parameters;
                              <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                               Broad Institute 2019
                              </a>
                              ), and called variants, i.e. the differences between the assemblies we used and the strains contained in the sequenced microbial community, with freebayes (1.3.5, default parameters;
                              <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                               Garrison and Marth 2012
                              </a>
                              ). Variants were filtered by minimum depth of coverage of 20 and quality score 20, transformed into their primitive constituents (vcflib, 1.0.2;
                              <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                               Garrison et al. 2021
                              </a>
                              ), and sorted using bcftools. Variant calling from nanopore data of the zymo microbial mixture was done using medaka (1.4.3, default parameters, model r941_prom_hac_variant_g507;
                              <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                               Oxford Nanopore Technologies 2022
                              </a>
                              ). For subsequent comparisons of vcf files we used vcfeval (rtg-tools, 3.12.1;
                              <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                               Cleary et al. 2015
                              </a>
                              ).
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-21">
                            <h2 class="">
                             Data access
                            </h2>
                            <p id="p-77">
                             The source code of BOSS-RUNS is available at
                             <a href="https://github.com/goldman-gp-ebi/BOSS-RUNS">
                              https://github.com/goldman-gp-ebi/BOSS-RUNS
                             </a>
                             . The sequencing data generated in this study have been submitted to the ENA database under accession number PRJEB51967.
                            </p>
                           </div>
                           <div class="section" id="sec-22">
                            <h2 class="">
                             Competing interest statement
                            </h2>
                            <p id="p-78">
                             ML was a member of the ONT MinION access program and has received free flow cells and sequencing reagents in the past. ML has received reimbursement for travel, accommodation and conference fees to speak at events organized by ONT. EB is a paid consultant to ONT and a small-scale equity and options holder in ONT.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-79">
                             We would like to thank Alex Payne for valuable insights and helpful discussions. This work was supported by the Biotechnology and Biological Sciences Research Council [grant number BB/N017099/1 to ML]. LW, NDM, CM, EB and NG were supported by the European Molecular Biology Laboratory. CM was also supported by Murray Edwards College, Cambridge, and by the Cambridge Mathematics Placements (CMP) programme.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-2">
                              <p id="p-7">
                               major update with new methods, algorithmic accelerations, software description plus availability, experimental results
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barbitoff
                                  </span>
                                  <span class="cit-name-given-names">
                                   YA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Polev
                                  </span>
                                  <span class="cit-name-given-names">
                                   DE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Glotov
                                  </span>
                                  <span class="cit-name-given-names">
                                   AS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Serebryakova
                                  </span>
                                  <span class="cit-name-given-names">
                                   EA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shcherbakova
                                  </span>
                                  <span class="cit-name-given-names">
                                   IV
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kiselev
                                  </span>
                                  <span class="cit-name-given-names">
                                   AM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kostareva
                                  </span>
                                  <span class="cit-name-given-names">
                                   AA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Glotov
                                  </span>
                                  <span class="cit-name-given-names">
                                   OS
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Predeus
                                  </span>
                                  <span class="cit-name-given-names">
                                   AV.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Systematic dissection of biases in whole-exome and whole-genome sequencing reveals major determinants of coding sequence coverage
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Scientific Reports
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ), Article number: 2057.
                                 <a href="https://doi.org/10.1038/s41598-020-59026-y">
                                  https://doi.org/10.1038/s41598-020-59026-y
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.2" data-doi="10.1038/321209a0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bird
                                  </span>
                                  <span class="cit-name-given-names">
                                   AP.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1986
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  CpG-rich islands and the function of DNA methylation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  321
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6067
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  209
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  213
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1038/321209a0">
                                  https://doi.org/10.1038/321209a0
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBird%26rft.auinit1%253DA.%2BP.%26rft.volume%253D321%26rft.issue%253D6067%26rft.spage%253D209%26rft.epage%253D213%26rft.atitle%253DCpG-rich%2Bislands%2Band%2Bthe%2Bfunction%2Bof%2BDNA%2Bmethylation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F321209a0%26rft_id%253Dinfo%253Apmid%252F2423876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/321209a0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=2423876&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F04%2F2020.02.07.938670.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1986C330700035&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Boykin
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ghalab
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marchi
                                  </span>
                                  <span class="cit-name-given-names">
                                   BRD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Savill
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wainaina
                                  </span>
                                  <span class="cit-name-given-names">
                                   JM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kinene
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lamb
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rodrigues
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kehoe
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ndunguru
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tairo
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sseruwagi
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kayuki
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mark
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Erasto
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bachwenkizi
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alicai
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Okao-Okuja
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Abridrabo
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,…
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kiarie
                                  </span>
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Real time portable genome sequencing for global food security [version 1; peer review: 1 approved, 1 approved with reservations]
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  F1000Research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1101
                                 </span>
                                 .
                                 <a href="https://doi.org/10.12688/f1000research.15507.1">
                                  https://doi.org/10.12688/f1000research.15507.1
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DF1000Research%26rft.volume%253D7%26rft.spage%253D1101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brinkerhoff
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kang
                                  </span>
                                  <span class="cit-name-given-names">
                                   ASW
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Aksimentiev
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dekker
                                  </span>
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Multiple rereads of single proteins at single–amino acid resolution using nanopores
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  374
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6574
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1509
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1513
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1126/science.abl4381">
                                  https://doi.org/10.1126/science.abl4381
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D374%26rft.spage%253D1509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.07.938670v3.5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  Broad Institute
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-source">
                                  Picard toolkit
                                 </span>
                                 .
                                 <a href="https://broadinstitute.github.io/picard/">
                                  https://broadinstitute.github.io/picard/
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.6" data-doi="10.1016/j.cub.2019.08.012">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brockhurst
                                  </span>
                                  <span class="cit-name-given-names">
                                   MA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harrison
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hall
                                  </span>
                                  <span class="cit-name-given-names">
                                   JPJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Richards
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McNally
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   MacLean
                                  </span>
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The ecology and evolution of pangenomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Current Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  29
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  20
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  R1094
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  R1103
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1016/j.cub.2019.08.012">
                                  https://doi.org/10.1016/j.cub.2019.08.012
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2BBiology%26rft.volume%253D29%26rft.spage%253DR1094%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2019.08.012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cub.2019.08.012&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.7" data-doi="10.1214/ss/1177009939">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chaloner
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Verdinelli
                                  </span>
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1995
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Bayesian experimental design: a review
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Statistical Science
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  273
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  304
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1214/ss/1177009939">
                                  https://doi.org/10.1214/ss/1177009939
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DStatistical%2BScience%26rft.volume%253D10%26rft.spage%253D273%26rft_id%253Dinfo%253Adoi%252F10.1214%252Fss%252F1177009939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1214/ss/1177009939&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1995UN93700003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.07.938670v3.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cleary
                                  </span>
                                  <span class="cit-name-given-names">
                                   JG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Braithwaite
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gaastra
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hilbush
                                  </span>
                                  <span class="cit-name-given-names">
                                   BS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Inglis
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Irvine
                                  </span>
                                  <span class="cit-name-given-names">
                                   SA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jackson
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Littin
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rathod
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ware
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zook
                                  </span>
                                  <span class="cit-name-given-names">
                                   JM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Trigg
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   De La Vega
                                  </span>
                                  <span class="cit-name-given-names">
                                   FM.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Comparing variant call files for performance benchmarking of next-generation sequencing variant calling pipelines
                                 </span>
                                 .
                                 <span class="cit-source">
                                  bioRxiv
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1101/023754">
                                  https://doi.org/10.1101/023754
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Colquhoun
                                  </span>
                                  <span class="cit-name-given-names">
                                   RM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hall
                                  </span>
                                  <span class="cit-name-given-names">
                                   MB
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lima
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Roberts
                                  </span>
                                  <span class="cit-name-given-names">
                                   LW
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Malone
                                  </span>
                                  <span class="cit-name-given-names">
                                   KM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hunt
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Letcher
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hawkey
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   George
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pankhurst
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Iqbal
                                  </span>
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Pandora: nucleotide-resolution bacterial pan-genomics with reference graphs
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ), Article number: 267.
                                 <a href="https://doi.org/10.1186/s13059-021-02473-1">
                                  https://doi.org/10.1186/s13059-021-02473-1
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.10" data-doi="10.1093/gigascience/giab008">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Danecek
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bonfield
                                  </span>
                                  <span class="cit-name-given-names">
                                   JK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liddle
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marshall
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ohan
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pollard
                                  </span>
                                  <span class="cit-name-given-names">
                                   MO
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Whitwham
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Keane
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McCarthy
                                  </span>
                                  <span class="cit-name-given-names">
                                   SA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Davies
                                  </span>
                                  <span class="cit-name-given-names">
                                   RM
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Twelve years of SAMtools and BCFtools
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  GigaScience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  giab008
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1093/gigascience/giab008">
                                  https://doi.org/10.1093/gigascience/giab008
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGigaScience%26rft.volume%253D10%26rft.spage%253Dgiab008%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgiab008%26rft_id%253Dinfo%253Apmid%252F33590861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/gigascience/giab008&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=33590861&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F04%2F2020.02.07.938670.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.11" data-doi="10.1038/nbt.3423">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Deamer
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Akeson
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Branton
                                  </span>
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Three decades of nanopore sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Biotechnology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  518
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  524
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1038/nbt.3423">
                                  https://doi.org/10.1038/nbt.3423
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BBiotechnology%26rft.volume%253D34%26rft.spage%253D518%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nbt.3423&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Djirackor
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Halldorsson
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Niehusmann
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leske
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Capper
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kuschel
                                  </span>
                                  <span class="cit-name-given-names">
                                   LP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pahnke
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Due-Tønnessen
                                  </span>
                                  <span class="cit-name-given-names">
                                   BJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Langmoen
                                  </span>
                                  <span class="cit-name-given-names">
                                   IA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sandberg
                                  </span>
                                  <span class="cit-name-given-names">
                                   CJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Euskirchen
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vik-Mo
                                  </span>
                                  <span class="cit-name-given-names">
                                   EO.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Intraoperative DNA methylation classification of brain tumors impacts neurosurgical strategy
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuro-Oncology Advances
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  vdab149
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1093/noajnl/vdab149">
                                  https://doi.org/10.1093/noajnl/vdab149
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuro-Oncology%2BAdvances%26rft.volume%253D3%26rft.spage%253D149vdab%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.13" data-doi="10.1093/nar/10.8.2709">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ehrlich
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gama-Sosa
                                  </span>
                                  <span class="cit-name-given-names">
                                   MA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Huang
                                  </span>
                                  <span class="cit-name-given-names">
                                   LH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Midgett
                                  </span>
                                  <span class="cit-name-given-names">
                                   RM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kuo
                                  </span>
                                  <span class="cit-name-given-names">
                                   KC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McCune
                                  </span>
                                  <span class="cit-name-given-names">
                                   RA
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gehrke
                                  </span>
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1982
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Amount and distribution of 5-methylcytosine in human DNA from different types of tissues of cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  2709
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2721
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1093/nar/10.8.2709">
                                  https://doi.org/10.1093/nar/10.8.2709
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEhrlich%26rft.auinit1%253DM.%26rft.volume%253D10%26rft.issue%253D8%26rft.spage%253D2709%26rft.epage%253D2721%26rft.atitle%253DAmount%2Band%2Bdistribution%2Bof%2B5-methylcytosine%2Bin%2Bhuman%2BDNA%2Bfrom%2Bdifferent%2Btypes%2Bof%2Btissues%2Bor%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F10.8.2709%26rft_id%253Dinfo%253Apmid%252F7079182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/10.8.2709&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=7079182&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F04%2F2020.02.07.938670.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1982NP13900017&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Garalde
                                  </span>
                                  <span class="cit-name-given-names">
                                   DR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Snell
                                  </span>
                                  <span class="cit-name-given-names">
                                   EA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jachimowicz
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sipos
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lloyd
                                  </span>
                                  <span class="cit-name-given-names">
                                   JH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bruce
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pantic
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Admassu
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   James
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Warland
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jordan
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ciccone
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Serra
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Keenan
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Martin
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McNeill
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wallace
                                  </span>
                                  <span class="cit-name-given-names">
                                   EJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jayasinghe
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wright
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,…
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Turner
                                  </span>
                                  <span class="cit-name-given-names">
                                   DJ.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Highly parallel direct RNA sequencing on an array of nanopores
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  201
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  206
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1038/nmeth.4577">
                                  https://doi.org/10.1038/nmeth.4577
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D15%26rft.spage%253D201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.07.938670v3.15">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Garrison
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kronenberg
                                  </span>
                                  <span class="cit-name-given-names">
                                   ZN
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dawson
                                  </span>
                                  <span class="cit-name-given-names">
                                   ET
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pedersen
                                  </span>
                                  <span class="cit-name-given-names">
                                   BS
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Prins
                                  </span>
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Vcflib and tools for processing the VCF variant call format
                                 </span>
                                 .
                                 <span class="cit-source">
                                  bioRxiv
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1101/2021.05.21.445151">
                                  https://doi.org/10.1101/2021.05.21.445151
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.07.938670v3.16">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Garrison
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marth
                                  </span>
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Haplotype-based variant detection from short-read sequencing
                                 </span>
                                 .
                                 <span class="cit-source">
                                  arXiv
                                 </span>
                                 .
                                 <a href="https://doi.org/10.48550/ARXIV.1207.3907">
                                  https://doi.org/10.48550/ARXIV.1207.3907
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.17">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gershow
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Golovchenko
                                  </span>
                                  <span class="cit-name-given-names">
                                   JA.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Recapturing and trapping single molecules with a solidstate nanopore
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Nanotechnology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  775
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  779
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1038/nnano.2007.381">
                                  https://doi.org/10.1038/nnano.2007.381
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BNanotechnology%26rft.volume%253D2%26rft.spage%253D775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.18" data-doi="10.1101/gr.191395.115">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goodwin
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gurtowski
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ethe-Sayers
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Deshpande
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schatz
                                  </span>
                                  <span class="cit-name-given-names">
                                   MC
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McCombie
                                  </span>
                                  <span class="cit-name-given-names">
                                   WR.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Oxford nanopore sequencing, hybrid error correction, and de novo assembly of a eukaryotic genome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1750
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1756
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1101/gr.191395.115">
                                  https://doi.org/10.1101/gr.191395.115
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.191395.115%26rft_id%253Dinfo%253Apmid%252F26447147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xMS8xNzUwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDQvMDQvMjAyMC4wMi4wNy45Mzg2NzAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.19" data-doi="10.1038/s41586-020-2649-2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harris
                                  </span>
                                  <span class="cit-name-given-names">
                                   CR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Millman
                                  </span>
                                  <span class="cit-name-given-names">
                                   KJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   van der Walt
                                  </span>
                                  <span class="cit-name-given-names">
                                   SJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gommers
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Virtanen
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cournapeau
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wieser
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Taylor
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Berg
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                  <span class="cit-name-given-names">
                                   NJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kern
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Picus
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hoyer
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   van Kerkwijk
                                  </span>
                                  <span class="cit-name-given-names">
                                   MH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brett
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Haldane
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   del Río
                                  </span>
                                  <span class="cit-name-given-names">
                                   JF
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wiebe
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Peterson
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,…
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Oliphant
                                  </span>
                                  <span class="cit-name-given-names">
                                   TE.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Array programming with NumPy
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  585
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7825
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  357
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  362
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1038/s41586-020-2649-2">
                                  https://doi.org/10.1038/s41586-020-2649-2
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D585%26rft.spage%253D357%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2649-2%26rft_id%253Dinfo%253Apmid%252F32939066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-020-2649-2&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=32939066&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F04%2F2020.02.07.938670.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.20" data-doi="10.1038/nbt.4060">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jain
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koren
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Miga
                                  </span>
                                  <span class="cit-name-given-names">
                                   KH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Quick
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rand
                                  </span>
                                  <span class="cit-name-given-names">
                                   AC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sasani
                                  </span>
                                  <span class="cit-name-given-names">
                                   TA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tyson
                                  </span>
                                  <span class="cit-name-given-names">
                                   JR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Beggs
                                  </span>
                                  <span class="cit-name-given-names">
                                   AD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dilthey
                                  </span>
                                  <span class="cit-name-given-names">
                                   AT
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fiddes
                                  </span>
                                  <span class="cit-name-given-names">
                                   IT
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Malla
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marriott
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nieto
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   O’Grady
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Olsen
                                  </span>
                                  <span class="cit-name-given-names">
                                   HE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pedersen
                                  </span>
                                  <span class="cit-name-given-names">
                                   BS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rhie
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Richardson
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Quinlan
                                  </span>
                                  <span class="cit-name-given-names">
                                   AR
                                  </span>
                                 </span>
                                 ,…
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Loose
                                  </span>
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Nanopore sequencing and assembly of a human genome with ultra-long reads
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Biotechnology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  36
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  338
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  345
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1038/nbt.4060">
                                  https://doi.org/10.1038/nbt.4060
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BBiotechnology%26rft.volume%253D36%26rft.spage%253D338%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.4060%26rft_id%253Dinfo%253Apmid%252F29431738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nbt.4060&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29431738&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F04%2F2020.02.07.938670.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.02.07.938670v3.21">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jones
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pevzner
                                  </span>
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ).
                                 <span class="cit-source">
                                  An Introduction to Bioinformatics Algorithms
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  MIT Press
                                 </span>
                                 ,
                                 <span class="cit-publ-loc">
                                  Cambridge, MA
                                 </span>
                                 .
                                 <a href="https://mitpress.mit.edu/books/introduction-bioinformatics-algorithms">
                                  https://mitpress.mit.edu/books/introduction-bioinformatics-algorithms
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kovaka
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fan
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ni
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Timp
                                  </span>
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schatz
                                  </span>
                                  <span class="cit-name-given-names">
                                   MC.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Targeted nanopore sequencing by real-time mapping of raw electrical signal with UNCALLED
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Biotechnology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  39
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  431
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  441
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1038/s41587-020-0731-9">
                                  https://doi.org/10.1038/s41587-020-0731-9
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BBiotechnology%26rft.volume%253D39%26rft.spage%253D431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Krishnakumar
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sinha
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bird
                                  </span>
                                  <span class="cit-name-given-names">
                                   SW
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jayamohan
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Edwards
                                  </span>
                                  <span class="cit-name-given-names">
                                   HS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schoeniger
                                  </span>
                                  <span class="cit-name-given-names">
                                   JS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Patel
                                  </span>
                                  <span class="cit-name-given-names">
                                   KD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Branda
                                  </span>
                                  <span class="cit-name-given-names">
                                   SS
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bartsch
                                  </span>
                                  <span class="cit-name-given-names">
                                   MS.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Systematic and stochastic influences on the performance of the minion nanopore sequencer across a range of nucleotide bias
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Scientific Reports
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ), Article number: 3159.
                                 <a href="https://doi.org/10.1038/s41598-018-21484-w">
                                  https://doi.org/10.1038/s41598-018-21484-w
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.24" data-doi="10.1016/j.transproceed.2017.12.043">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kullback
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leibler
                                  </span>
                                  <span class="cit-name-given-names">
                                   RA.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1951
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  On information and sufficiency
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Annals of Mathematical Statistics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  79
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  86
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1214/aoms/1177729694">
                                  https://doi.org/10.1214/aoms/1177729694
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BAnnals%2Bof%2BMathematical%2BStatistics%26rft.volume%253D22%26rft.spage%253D79%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.transproceed.2017.12.043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.transproceed.2017.12.043&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.25">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lam
                                  </span>
                                  <span class="cit-name-given-names">
                                   SK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pitrou
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Seibert
                                  </span>
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Numba: a LLVM-based Python JIT compiler
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the Second Workshop on the LLVM Compiler Infrastructure in HPC
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1145/2833157.2833162">
                                  https://doi.org/10.1145/2833157.2833162
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BSecond%2BWorkshop%2Bon%2Bthe%2BLLVM%2BCompiler%2BInfrastructure%2Bin%2BHPC%26rft.volume%253D7%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Razaghi
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gilpatrick
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Molnar
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gershman
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sadowski
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sedlazeck
                                  </span>
                                  <span class="cit-name-given-names">
                                   FJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hansen
                                  </span>
                                  <span class="cit-name-given-names">
                                   KD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Simpson
                                  </span>
                                  <span class="cit-name-given-names">
                                   JT
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Timp
                                  </span>
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Simultaneous profiling of chromatin accessibility and methylation on human cell lines with nanopore sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1191
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1199
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1038/s41592-020-01000-7">
                                  https://doi.org/10.1038/s41592-020-01000-7
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D17%26rft.spage%253D1191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.27">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leger
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Amaral
                                  </span>
                                  <span class="cit-name-given-names">
                                   PP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pandolfini
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Capitanchik
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Capraro
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Miano
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Migliori
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Toolan-Kerr
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sideri
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Enright
                                  </span>
                                  <span class="cit-name-given-names">
                                   AJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tzelepis
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   van Werven
                                  </span>
                                  <span class="cit-name-given-names">
                                   FJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Luscombe
                                  </span>
                                  <span class="cit-name-given-names">
                                   NM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barbieri
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ule
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fitzgerald
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Birney
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leonardi
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kouzarides
                                  </span>
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  RNA modifications detection by comparative Nanopore direct RNA sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Communications
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ), Article number: 7198.
                                 <a href="https://doi.org/10.1038/s41467-021-27393-3">
                                  https://doi.org/10.1038/s41467-021-27393-3
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.28" data-doi="10.1093/bioinformatics/bty191">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Minimap2: pairwise alignment for nucleotide sequences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  18
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  3094
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3100
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1093/bioinformatics/bty191">
                                  https://doi.org/10.1093/bioinformatics/bty191
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D3094%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty191%26rft_id%253Dinfo%253Apmid%252F29750242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty191&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29750242&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F04%2F2020.02.07.938670.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rosikiewicz
                                  </span>
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pan
                                  </span>
                                  <span class="cit-name-given-names">
                                   Z
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jillette
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Taghbalout
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Foox
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mason
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carroll
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cheng
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  DNA methylation-calling tools for Oxford Nanopore sequencing: a survey and human epigenome-wide evaluation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ), Article number: 295.
                                 <a href="https://doi.org/10.1186/s13059-021-02510-z">
                                  https://doi.org/10.1186/s13059-021-02510-z
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Loose
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Malla
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stout
                                  </span>
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Real-time selective sequencing using nanopore technology
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  751
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1038/nmeth.3930">
                                  https://doi.org/10.1038/nmeth.3930
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D13%26rft.spage%253D751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.31">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marquet
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zöllkau
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pastuschek
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Viehweger
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schleußner
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Makarewicz
                                  </span>
                                  <span class="cit-name-given-names">
                                   O
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pletz
                                  </span>
                                  <span class="cit-name-given-names">
                                   MW
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ehricht
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brandt
                                  </span>
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Evaluation of microbiome enrichment and host DNA de-pletion in human vaginal samples using oxford nanopore’s adaptive sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Scientific Reports
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ), Article number: 4000.
                                 <a href="https://doi.org/10.1038/s41598-022-08003-8">
                                  https://doi.org/10.1038/s41598-022-08003-8
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.32">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Martin
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Heavens
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lan
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Horsfield
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Clark
                                  </span>
                                  <span class="cit-name-given-names">
                                   MD
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leggett
                                  </span>
                                  <span class="cit-name-given-names">
                                   RM.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Nanopore adaptive sampling: a tool for enrichment of low abundance species in metagenomic samples
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  23
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ), Article number: 11.
                                 <a href="https://doi.org/10.1186/s13059-021-02582-x">
                                  https://doi.org/10.1186/s13059-021-02582-x
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.33">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McIntyre
                                  </span>
                                  <span class="cit-name-given-names">
                                   ABR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alexander
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grigorev
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bezdan
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sichtig
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chiu
                                  </span>
                                  <span class="cit-name-given-names">
                                   CY
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mason
                                  </span>
                                  <span class="cit-name-given-names">
                                   CE.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Single-molecule sequencing detection of N6-methyladenine in microbial reference materials
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Communications
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 , Article number: 579.
                                 <a href="https://doi.org/10.1038/s41467-019-08289-9">
                                  https://doi.org/10.1038/s41467-019-08289-9
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.07.938670v3.34">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McKinney
                                  </span>
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Data structures for statistical computing in Python
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Python in Science Conference
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  56
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  61
                                 </span>
                                 .
                                 <a href="https://doi.org/10.25080/Majora-92bf1922-00a">
                                  https://doi.org/10.25080/Majora-92bf1922-00a
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.35">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Medini
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Donati
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tettelin
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Masignani
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rappuoli
                                  </span>
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The microbial pan-genome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Current Opinion in Genetics & Development
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  589
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  594
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1016/j.gde.2005.09.006">
                                  https://doi.org/10.1016/j.gde.2005.09.006
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BGenetics%2B%2526%2BDevelopment%26rft.volume%253D15%26rft.spage%253D589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.36">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Miga
                                  </span>
                                  <span class="cit-name-given-names">
                                   KH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koren
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rhie
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vollger
                                  </span>
                                  <span class="cit-name-given-names">
                                   MR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gershman
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bzikadze
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brooks
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Howe
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Porubsky
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Logsdon
                                  </span>
                                  <span class="cit-name-given-names">
                                   GA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schneider
                                  </span>
                                  <span class="cit-name-given-names">
                                   VA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Potapova
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wood
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chow
                                  </span>
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Armstrong
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fredrickson
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pak
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tigyi
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kremitzki
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,…
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Phillippy
                                  </span>
                                  <span class="cit-name-given-names">
                                   AM.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Telomere-to-telomere assembly of a complete human X chromosome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  585
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7823
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  79
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  84
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1038/s41586-020-2547-7">
                                  https://doi.org/10.1038/s41586-020-2547-7
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D585%26rft.spage%253D79%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.07.938670v3.37">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Miller
                                  </span>
                                  <span class="cit-name-given-names">
                                   DE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sulovari
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Loucks
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hoekzema
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Munson
                                  </span>
                                  <span class="cit-name-given-names">
                                   KM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lewis
                                  </span>
                                  <span class="cit-name-given-names">
                                   AP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fuerte
                                  </span>
                                  <span class="cit-name-given-names">
                                   EPA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Paschal
                                  </span>
                                  <span class="cit-name-given-names">
                                   CR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thies
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bennett
                                  </span>
                                  <span class="cit-name-given-names">
                                   JT
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Glass
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dipple
                                  </span>
                                  <span class="cit-name-given-names">
                                   KM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Patterson
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bonkowski
                                  </span>
                                  <span class="cit-name-given-names">
                                   ES
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nelson
                                  </span>
                                  <span class="cit-name-given-names">
                                   Z
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Squire
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sikes
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Beckman
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,…
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eichler
                                  </span>
                                  <span class="cit-name-given-names">
                                   EE.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Targeted long-read sequencing resolves complex structural variants and identifies missing disease-causing variants
                                 </span>
                                 .
                                 <span class="cit-source">
                                  bioRxiv
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1101/2020.11.03.365395">
                                  https://doi.org/10.1101/2020.11.03.365395
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.38" data-doi="10.1093/nar/gkq072">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mokry
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Feitsma
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nijman
                                  </span>
                                  <span class="cit-name-given-names">
                                   IJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   de Bruijn
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   van der Zaag
                                  </span>
                                  <span class="cit-name-given-names">
                                   PJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guryev
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cuppen
                                  </span>
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Accurate SNP and mutation detection by targeted custom microarray-based genomic enrichment of short-fragment sequencing libraries
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  38
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  e116
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1093/nar/gkq072">
                                  https://doi.org/10.1093/nar/gkq072
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq072%26rft_id%253Dinfo%253Apmid%252F20164091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkq072&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20164091&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F04%2F2020.02.07.938670.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.07.938670v3.39">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Morton
                                  </span>
                                  <span class="cit-name-given-names">
                                   SM.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-source">
                                  natsort
                                 </span>
                                 .
                                 <a href="https://github.com/SethMMorton/natsort">
                                  https://github.com/SethMMorton/natsort
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.07.938670v3.40">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  Oxford Nanopore Technologies
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-source">
                                  Read Until-API
                                 </span>
                                 .
                                 <a href="https://github.com/nanoporetech/read_until_api">
                                  https://github.com/nanoporetech/read_until_api
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.07.938670v3.41">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  Oxford Nanopore Technologies
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-source">
                                  MinKNOW-API
                                 </span>
                                 .
                                 <a href="https://github.com/nanoporetech/minknow_api">
                                  https://github.com/nanoporetech/minknow_api
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.07.938670v3.42">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  Oxford Nanopore Technologies
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 ).
                                 <span class="cit-source">
                                  medaka
                                 </span>
                                 .
                                 <a href="https://github.com/nanoporetech/medaka">
                                  https://github.com/nanoporetech/medaka
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.43">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ozer
                                  </span>
                                  <span class="cit-name-given-names">
                                   EA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Allen
                                  </span>
                                  <span class="cit-name-given-names">
                                   JP
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hauser
                                  </span>
                                  <span class="cit-name-given-names">
                                   AR.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Characterization of the core and accessory genomes of Pseudomonas aeruginosa using bioinformatic tools Spine and AGEnt
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Genomics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ), Article number: 737.
                                 <a href="https://doi.org/10.1186/1471-2164-15-737">
                                  https://doi.org/10.1186/1471-2164-15-737
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.07.938670v3.44">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Patel
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dogan
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Payne
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sievers
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schoebe
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schrimpf
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stichel
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Holmes
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Euskirchen
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hench
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Frank
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rosenstiel-Goidts
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ratliff
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Etminan
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Unterberg
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dieterich
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Herold-Mende
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pfister
                                  </span>
                                  <span class="cit-name-given-names">
                                   SM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wick
                                  </span>
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 ,…
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sahm
                                  </span>
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Rapid-CNS2: rapid comprehensive adaptive nanopore-sequencing of CNS tumors, a proof of concept study
                                 </span>
                                 .
                                 <span class="cit-source">
                                  medRxiv
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1101/2021.08.09.21261784">
                                  https://doi.org/10.1101/2021.08.09.21261784
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.45">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Payne
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Holmes
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Clarke
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Munro
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Debebe
                                  </span>
                                  <span class="cit-name-given-names">
                                   BJ
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Loose
                                  </span>
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Readfish enables targeted nanopore sequencing of gigabase-sized genomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Biotechnology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  39
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  442
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  450
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1038/s41587-020-00746-x">
                                  https://doi.org/10.1038/s41587-020-00746-x
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BBiotechnology%26rft.volume%253D39%26rft.spage%253D442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.46">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Payne
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Holmes
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rakyan
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Loose
                                  </span>
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  BulkVis: A graphical viewer for Oxford nanopore bulk FAST5 files
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  35
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  13
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  2193
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2198
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1093/bioinformatics/bty841">
                                  https://doi.org/10.1093/bioinformatics/bty841
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D35%26rft.spage%253D2193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.07.938670v3.47">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pearson
                                  </span>
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 ).
                                 <span class="cit-source">
                                  toml
                                 </span>
                                 .
                                 <a href="https://github.com/uiri/toml">
                                  https://github.com/uiri/toml
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.48">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Porubsky
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ebert
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Audano
                                  </span>
                                  <span class="cit-name-given-names">
                                   PA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vollger
                                  </span>
                                  <span class="cit-name-given-names">
                                   MR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harvey
                                  </span>
                                  <span class="cit-name-given-names">
                                   WT
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marijon
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ebler
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Munson
                                  </span>
                                  <span class="cit-name-given-names">
                                   KM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sorensen
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sulovari
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Haukness
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ghareghani
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lansdorp
                                  </span>
                                  <span class="cit-name-given-names">
                                   PM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Paten
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Devine
                                  </span>
                                  <span class="cit-name-given-names">
                                   SE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sanders
                                  </span>
                                  <span class="cit-name-given-names">
                                   AD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chaisson
                                  </span>
                                  <span class="cit-name-given-names">
                                   MJP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Korbel
                                  </span>
                                  <span class="cit-name-given-names">
                                   JO
                                  </span>
                                 </span>
                                 ,…
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marschall
                                  </span>
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Fully phased human genome assembly without parental data using single-cell strand sequencing and long reads
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Biotechnology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  39
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  302
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  308
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1038/s41587-020-0719-5">
                                  https://doi.org/10.1038/s41587-020-0719-5
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BBiotechnology%26rft.volume%253D39%26rft.spage%253D302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.07.938670v3.49">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Quick
                                  </span>
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The ‘Three Peaks’ faecal DNA extraction method for long-read sequencing
                                 </span>
                                 .
                                 <span class="cit-source">
                                  protocols.io
                                 </span>
                                 .
                                 <a href="https://doi.org/10.17504/protocols.io.7rshm6e">
                                  https://doi.org/10.17504/protocols.io.7rshm6e
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.50" data-doi="10.1038/nature16996">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Quick
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Loman
                                  </span>
                                  <span class="cit-name-given-names">
                                   NJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Duraffour
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Simpson
                                  </span>
                                  <span class="cit-name-given-names">
                                   JT
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Severi
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cowley
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bore
                                  </span>
                                  <span class="cit-name-given-names">
                                   JA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koundouno
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dudas
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mikhail
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ouédraogo
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Afrough
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bah
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Baum
                                  </span>
                                  <span class="cit-name-given-names">
                                   JHJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Becker-Ziaja
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Boettcher
                                  </span>
                                  <span class="cit-name-given-names">
                                   JP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cabeza-Cabrerizo
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Camino-Sánchez
                                  </span>
                                  <span class="cit-name-given-names">
                                   Á
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carter
                                  </span>
                                  <span class="cit-name-given-names">
                                   LL
                                  </span>
                                 </span>
                                 ,…
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carroll
                                  </span>
                                  <span class="cit-name-given-names">
                                   MW.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Realtime, portable genome sequencing for Ebola surveillance
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  530
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7589
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  228
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  232
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1038/nature16996">
                                  https://doi.org/10.1038/nature16996
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D530%26rft.spage%253D228%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature16996%26rft_id%253Dinfo%253Apmid%252F26840485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature16996&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26840485&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F04%2F2020.02.07.938670.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.51" data-doi="10.1093/bioinformatics/btv681">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Richter
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rosselló-Móra
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Glöckner
                                  </span>
                                  <span class="cit-name-given-names">
                                   FO
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Peplies
                                  </span>
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  JSpeciesWS: a web server for prokaryotic species circumscription based on pairwise genome comparison
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  32
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  929
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  931
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1093/bioinformatics/btv681">
                                  https://doi.org/10.1093/bioinformatics/btv681
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv681%26rft_id%253Dinfo%253Apmid%252F26576653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv681&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26576653&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F04%2F2020.02.07.938670.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.52">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ross
                                  </span>
                                  <span class="cit-name-given-names">
                                   MG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Russ
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Costello
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hollinger
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lennon
                                  </span>
                                  <span class="cit-name-given-names">
                                   NJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hegarty
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nusbaum
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jaffe
                                  </span>
                                  <span class="cit-name-given-names">
                                   DB.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Characterizing and measuring bias in sequence data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ), Article number: R51.
                                 <a href="https://doi.org/10.1186/gb-2013-14-5-r51">
                                  https://doi.org/10.1186/gb-2013-14-5-r51
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.07.938670v3.53">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sereika
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kirkegaard
                                  </span>
                                  <span class="cit-name-given-names">
                                   RH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Karst
                                  </span>
                                  <span class="cit-name-given-names">
                                   SM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Michaelsen
                                  </span>
                                  <span class="cit-name-given-names">
                                   TY
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sørensen
                                  </span>
                                  <span class="cit-name-given-names">
                                   EA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wollenberg
                                  </span>
                                  <span class="cit-name-given-names">
                                   RD
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Albertsen
                                  </span>
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Oxford nanopore R10.4 long-read sequencing enables near-perfect bacterial genomes from pure cultures and metagenomes without short-read or reference polishing
                                 </span>
                                 .
                                 <span class="cit-source">
                                  bioRxiv
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1101/2021.10.27.466057">
                                  https://doi.org/10.1101/2021.10.27.466057
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.54" data-doi="10.1038/s41592-021-01299-w">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shafin
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pesout
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chang
                                  </span>
                                  <span class="cit-name-given-names">
                                   PC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nattestad
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kolesnikov
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goel
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Baid
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kolmogorov
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eizenga
                                  </span>
                                  <span class="cit-name-given-names">
                                   JM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Miga
                                  </span>
                                  <span class="cit-name-given-names">
                                   KH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carnevali
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jain
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carroll
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Paten
                                  </span>
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Haplotype-aware variant calling with PEPPER-margin-DeepVariant enables high accuracy in nanopore long-reads
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1322
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1332
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1038/s41592-021-01299-w">
                                  https://doi.org/10.1038/s41592-021-01299-w
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D18%26rft.spage%253D1322%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41592-021-01299-w%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41592-021-01299-w&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.55">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shannon
                                  </span>
                                  <span class="cit-name-given-names">
                                   CE.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1948
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A mathematical theory of communication
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bell System Technical Journal
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  379
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  423
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1002/j.1538-7305.1948.tb01338.x">
                                  https://doi.org/10.1002/j.1538-7305.1948.tb01338.x
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBell%2BSystem%2BTechnical%2BJournal%26rft.volume%253D27%26rft.spage%253D379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.56">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Simpson
                                  </span>
                                  <span class="cit-name-given-names">
                                   JT
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Workman
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zuzarte
                                  </span>
                                  <span class="cit-name-given-names">
                                   PC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   David
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dursi
                                  </span>
                                  <span class="cit-name-given-names">
                                   LJ
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Timp
                                  </span>
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Detecting DNA cytosine methylation using nanopore sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  407
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  410
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1038/nmeth.4184">
                                  https://doi.org/10.1038/nmeth.4184
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D14%26rft.spage%253D407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.07.938670v3.57">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smit
                                  </span>
                                  <span class="cit-name-given-names">
                                   AFA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hubley
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Green
                                  </span>
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-source">
                                  RepeatMasker Open-4.0
                                 </span>
                                 .
                                 <a href="http://www.repeatmasker.org">
                                  http://www.repeatmasker.org
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.58">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Soifer
                                  </span>
                                  <span class="cit-name-given-names">
                                   l
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fong
                                  </span>
                                  <span class="cit-name-given-names">
                                   NL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yi
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ireland
                                  </span>
                                  <span class="cit-name-given-names">
                                   AT
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lam
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sooknah
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Paw
                                  </span>
                                  <span class="cit-name-given-names">
                                   JS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Peluso
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Concepcion
                                  </span>
                                  <span class="cit-name-given-names">
                                   GT
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rank
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hastie
                                  </span>
                                  <span class="cit-name-given-names">
                                   AR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jojic
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ruby
                                  </span>
                                  <span class="cit-name-given-names">
                                   JG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Botstein
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Roy
                                  </span>
                                  <span class="cit-name-given-names">
                                   MA.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Fully phased sequence of a diploid human genome determined de novo from the DNA of a single individual
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  G3 Genes/Genomes/Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  2911
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2925
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1534/g3.119.400995">
                                  https://doi.org/10.1534/g3.119.400995
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DG3%2BGenes%252FGenomes%252FGenetics%26rft.volume%253D10%26rft.spage%253D2911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.59">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stevanovski
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chintalaphani
                                  </span>
                                  <span class="cit-name-given-names">
                                   SR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gamaarachchi
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ferguson
                                  </span>
                                  <span class="cit-name-given-names">
                                   JM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pineda
                                  </span>
                                  <span class="cit-name-given-names">
                                   SS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Scriba
                                  </span>
                                  <span class="cit-name-given-names">
                                   CK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tchan
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fung
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ng
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cortese
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Houlden
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dobson-Stone
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fitzpatrick
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Halliday
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ravenscroft
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Davis
                                  </span>
                                  <span class="cit-name-given-names">
                                   MR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Laing
                                  </span>
                                  <span class="cit-name-given-names">
                                   NG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fellner
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kennerson
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,…
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Deveson
                                  </span>
                                  <span class="cit-name-given-names">
                                   IW.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Comprehensive genetic diagnosis of tandem repeat expansion disorders with programmable targeted nanopore sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science Advances
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  eabm5386
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1126/sciadv.abm5386">
                                  https://doi.org/10.1126/sciadv.abm5386
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.volume%253D8%26rft.spage%253Deabm5386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.60">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tan
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Opitz
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schlapbach
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rehrauer
                                  </span>
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Long fragments achieve lower base quality in Illumina paired-end sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Scientific Reports
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ), Article number: 2856.
                                 <a href="https://doi.org/10.1038/s41598-019-39076-7">
                                  https://doi.org/10.1038/s41598-019-39076-7
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.07.938670v3.61">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ulrich
                                  </span>
                                  <span class="cit-name-given-names">
                                   JU
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lutfi
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rutzen
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Renard
                                  </span>
                                  <span class="cit-name-given-names">
                                   BY.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  ReadBouncer: precise and scalable adaptive sampling for nanopore sequencing
                                 </span>
                                 .
                                 <span class="cit-source">
                                  bioRxiv
                                 </span>
                                 , 2022.02.01.478636.
                                 <a href="https://doi.org/10.1101/2022.02.01.478636">
                                  https://doi.org/10.1101/2022.02.01.478636
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.62">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Virtanen
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gommers
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Oliphant
                                  </span>
                                  <span class="cit-name-given-names">
                                   TE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Haberland
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reddy
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cournapeau
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burovski
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Peterson
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weckesser
                                  </span>
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bright
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   van der Walt
                                  </span>
                                  <span class="cit-name-given-names">
                                   SJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brett
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wilson
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Millman
                                  </span>
                                  <span class="cit-name-given-names">
                                   KJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mayorov
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nelson
                                  </span>
                                  <span class="cit-name-given-names">
                                   ARJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jones
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kern
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Larson
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,…
                                 <span class="cit-auth cit-collab">
                                  SciPy 1.0 Contributors
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  SciPy 1.0: fundamental algorithms for scientific computing in Python
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  261
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  272
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1038/s41592-019-0686-2">
                                  https://doi.org/10.1038/s41592-019-0686-2
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D17%26rft.spage%253D261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938670v3.63">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xie
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wan
                                  </span>
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Cancer biomarkers discovery of methylation modification with direct high-throughput nanopore sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Frontiers in Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 .
                                 <a href="https://doi.org/10.3389/fgene.2021.672804">
                                  https://doi.org/10.3389/fgene.2021.672804
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.02.07.938670v3" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.03.30.486499v2" title="Brequinar and Dipyridamole in Combination Exhibits Synergistic Antiviral Activity Against SARS-CoV-2 in vitro: Rationale for a host-acting antiviral treatment strategy for COVID-19" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2021.09.01.458486v2" title="Quantifying the impact of ecological memory on the dynamics of interacting communities" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted April 04, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.02.07.938670v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.02.07.938670v3.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.02.07.938670v3.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.02.07.938670v3" class="active" data-icon-position="" data-hide-link-title="0">
                                   Dynamic, adaptive sampling during nanopore sequencing using Bayesian experimental design
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2480196"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-qQI9l1ThhionZilIIDpL4QK_GQeDl58vi8JkTdtM26U"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484466977"/>
                                   <input type="hidden" name="captcha_token" value="899c397a53f1114298f3266a58684b30"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2480196--3" data-node-nid="2480196" id="node-2480196--4169626634" data-pisa="biorxiv;2020.02.07.938670v3" data-pisa-master="biorxiv;2020.02.07.938670" data-seqnum="2480196" data-apath="/biorxiv/early/2022/04/04/2020.02.07.938670.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Dynamic, adaptive sampling during nanopore sequencing using Bayesian experimental design
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Lukas
                                  </span>
                                  <span class="nlm-surname">
                                   Weilguny
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Nicola
                                  </span>
                                  <span class="nlm-surname">
                                   De Maio
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Rory
                                  </span>
                                  <span class="nlm-surname">
                                   Munro
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Charlotte
                                  </span>
                                  <span class="nlm-surname">
                                   Manser
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Ewan
                                  </span>
                                  <span class="nlm-surname">
                                   Birney
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Matt
                                  </span>
                                  <span class="nlm-surname">
                                   Loose
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">
                                   Nick
                                  </span>
                                  <span class="nlm-surname">
                                   Goldman
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.02.07.938670;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.02.07.938670
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.02.07.938670v3" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.02.07.938670v3&title=Dynamic%2C%20adaptive%20sampling%20during%20nanopore%20sequencing%20using%20Bayesian%20experimental%20design" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.07.938670v3&title=Dynamic%2C%20adaptive%20sampling%20during%20nanopore%20sequencing%20using%20Bayesian%20experimental%20design" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.07.938670v3&text=Dynamic%2C%20adaptive%20sampling%20during%20nanopore%20sequencing%20using%20Bayesian%20experimental%20design" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.02.07.938670v3&t=Dynamic%2C%20adaptive%20sampling%20during%20nanopore%20sequencing%20using%20Bayesian%20experimental%20design" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.02.07.938670v3&title=Dynamic%2C%20adaptive%20sampling%20during%20nanopore%20sequencing%20using%20Bayesian%20experimental%20design" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.02.07.938670v3&title=Dynamic%2C%20adaptive%20sampling%20during%20nanopore%20sequencing%20using%20Bayesian%20experimental%20design&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.07.938670v3&title=Dynamic%2C%20adaptive%20sampling%20during%20nanopore%20sequencing%20using%20Bayesian%20experimental%20design" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2480196--5" data-node-nid="2480196" id="citation-node-2480196--6279182670" data-pisa="biorxiv;2020.02.07.938670v3" data-pisa-master="biorxiv;2020.02.07.938670" data-seqnum="2480196" data-apath="/biorxiv/early/2022/04/04/2020.02.07.938670.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Dynamic, adaptive sampling during nanopore sequencing using Bayesian experimental design
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Lukas
                                      </span>
                                      <span class="nlm-surname">
                                       Weilguny
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Nicola
                                      </span>
                                      <span class="nlm-surname">
                                       De Maio
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Rory
                                      </span>
                                      <span class="nlm-surname">
                                       Munro
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Charlotte
                                      </span>
                                      <span class="nlm-surname">
                                       Manser
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Ewan
                                      </span>
                                      <span class="nlm-surname">
                                       Birney
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Matt
                                      </span>
                                      <span class="nlm-surname">
                                       Loose
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">
                                       Nick
                                      </span>
                                      <span class="nlm-surname">
                                       Goldman
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.02.07.938670;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.02.07.938670
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Dynamic%2C%20adaptive%20sampling%20during%20nanopore%20sequencing%20using%20Bayesian%20experimental%20design&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Weilguny%2C+Lukas&rft.au=De+Maio%2C+Nicola&rft.au=Munro%2C+Rory&rft.au=Manser%2C+Charlotte&rft.au=Birney%2C+Ewan&rft.au=Loose%2C+Matt&rft.au=Goldman%2C+Nick">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2480196/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2480196/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2480196/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2480196/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2480196/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2480196/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2480196/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2480196/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2480196/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2480196/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2480196/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2480196/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.07.938670v3&count=horizontal&via=&text=Dynamic%2C%20adaptive%20sampling%20during%20nanopore%20sequencing%20using%20Bayesian%20experimental%20design&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.02.07.938670v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.02.07.938670v3&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.02.07.938670v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Genomics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15186)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9196)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
