<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.05.962001v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.05.962001v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.05.962001v2.ppt"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/05/11/2020.03.05.962001.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.05.962001v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Extensive single-cell genomics reveals bacterial diversity and diverse phage host ranges in the area in and around the Red Sea"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.05.962001"/>
  <meta name="DC.Date" content="2022-05-11"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="To improve our understanding of the environmental microbiome, we developed a single-cell genome sequencing platform, named SAG-gel, which utilizes gel beads for single-cell isolation, cell lysis, and whole genome amplification (WGA) for sequencing. SAG-gel enables serial, parallel and independent reactions of > 100,000 single cells in a single tube, delivering high-quality genome recovery with storable randomized single-cell genome libraries. From soil and marine environmental sources, we acquired 734 partial genomes that are recapitulated in 231 species, 35% of which were assigned as high-to-medium qualities. We found that each genome to be almost unique and 98.7% of them were newly identified, implying the complex genetic diversities across 44 phyla. The various metabolic capabilities including virulence factors and biosynthetic gene clusters were found across the lineages at single-cell resolution. This technology will accelerate the accumulation of reference genomes of uncharacterized environmental microbes and provide us new insights for their roles.

### Competing Interest Statement

M.H. and H.T. are shareholders in bitBiome, to which the patents 849pertaining to SAG-gel workflow were transferred."/>
  <meta name="DC.Contributor" content="Yohei Nishikawa"/>
  <meta name="DC.Contributor" content="Masato Kogawa"/>
  <meta name="DC.Contributor" content="Masahito Hosokawa"/>
  <meta name="DC.Contributor" content="Ryota Wagatsuma"/>
  <meta name="DC.Contributor" content="Katsuhiko Mineta"/>
  <meta name="DC.Contributor" content="Kai Takahashi"/>
  <meta name="DC.Contributor" content="Keigo Ide"/>
  <meta name="DC.Contributor" content="Kei Yura"/>
  <meta name="DC.Contributor" content="Hayedeh Behzad"/>
  <meta name="DC.Contributor" content="Takashi Gojobori"/>
  <meta name="DC.Contributor" content="Haruko Takeyama"/>
  <meta name="article:published_time" content="2022-05-11"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Extensive single-cell genomics reveals bacterial diversity and diverse phage host ranges in the area in and around the Red Sea"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>To improve our understanding of the environmental microbiome, we developed a single-cell genome sequencing platform, named SAG-gel, which utilizes gel beads for single-cell isolation, cell lysis, and whole genome amplification (WGA) for sequencing. SAG-gel enables serial, parallel and independent reactions of &gt; 100,000 single cells in a single tube, delivering high-quality genome recovery with storable randomized single-cell genome libraries. From soil and marine environmental sources, we acquired 734 partial genomes that are recapitulated in 231 species, 35% of which were assigned as high-to-medium qualities. We found that each genome to be almost unique and 98.7% of them were newly identified, implying the complex genetic diversities across 44 phyla. The various metabolic capabilities including virulence factors and biosynthetic gene clusters were found across the lineages at single-cell resolution. This technology will accelerate the accumulation of reference genomes of uncharacterized environmental microbes and provide us new insights for their roles.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.05.962001v2"/>
  <meta name="citation_id" content="2020.03.05.962001v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.05.962001v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.05.962001v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.05.962001v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.05.962001"/>
  <meta name="citation_num_pages" content="32"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.05.962001"/>
  <meta name="citation_author" content="Yohei Nishikawa"/>
  <meta name="citation_author_institution" content="Research Organization for Nano & Life Innovation, Waseda University"/>
  <meta name="citation_author_institution" content="Computational Bio Big-Data Open Innovation Laboratory, AIST-Waseda University"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5443-7230"/>
  <meta name="citation_author" content="Masato Kogawa"/>
  <meta name="citation_author_institution" content="Research Organization for Nano & Life Innovation, Waseda University"/>
  <meta name="citation_author" content="Masahito Hosokawa"/>
  <meta name="citation_author_institution" content="Research Organization for Nano & Life Innovation, Waseda University"/>
  <meta name="citation_author_institution" content="Computational Bio Big-Data Open Innovation Laboratory, AIST-Waseda University"/>
  <meta name="citation_author_institution" content="Department of Life Science and Medical Bioscience, Waseda University"/>
  <meta name="citation_author_institution" content="Institute for Advanced Research of Biosystem Dynamics, Waseda Research Institute for Science and Engineering, Graduate School of Advanced Science and Engineering, Waseda University"/>
  <meta name="citation_author_email" content="masahosokawa@aoni.waseda.jp"/>
  <meta name="citation_author_email" content="haruko-takeyama@waseda.jp"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7070-4690"/>
  <meta name="citation_author" content="Ryota Wagatsuma"/>
  <meta name="citation_author_institution" content="Computational Bio Big-Data Open Innovation Laboratory, AIST-Waseda University"/>
  <meta name="citation_author_institution" content="Department of Life Science and Medical Bioscience, Waseda University"/>
  <meta name="citation_author" content="Katsuhiko Mineta"/>
  <meta name="citation_author_institution" content="Research Organization for Nano & Life Innovation, Waseda University"/>
  <meta name="citation_author_institution" content="King Abdullah University of Science and Technology (KAUST), Computer, Electrical, and Mathematical Sciences and Engineering (CEMSE) Division"/>
  <meta name="citation_author_institution" content="King Abdullah University of Science and Technology (KAUST), Computational Bioscience Research Center (CBRC)"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4727-045X"/>
  <meta name="citation_author" content="Kai Takahashi"/>
  <meta name="citation_author_institution" content="Department of Life Science and Medical Bioscience, Waseda University"/>
  <meta name="citation_author" content="Keigo Ide"/>
  <meta name="citation_author_institution" content="Computational Bio Big-Data Open Innovation Laboratory, AIST-Waseda University"/>
  <meta name="citation_author_institution" content="Department of Life Science and Medical Bioscience, Waseda University"/>
  <meta name="citation_author" content="Kei Yura"/>
  <meta name="citation_author_institution" content="Research Organization for Nano & Life Innovation, Waseda University"/>
  <meta name="citation_author_institution" content="Department of Life Science and Medical Bioscience, Waseda University"/>
  <meta name="citation_author_institution" content="Graduate School of Humanities and Sciences, Ochanomizu University"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0197-5478"/>
  <meta name="citation_author" content="Hayedeh Behzad"/>
  <meta name="citation_author_institution" content="King Abdullah University of Science and Technology (KAUST), Biological and Environmental Sciences and Engineering (BESE) Division"/>
  <meta name="citation_author" content="Takashi Gojobori"/>
  <meta name="citation_author_institution" content="King Abdullah University of Science and Technology (KAUST), Computational Bioscience Research Center (CBRC)"/>
  <meta name="citation_author_institution" content="King Abdullah University of Science and Technology (KAUST), Biological and Environmental Sciences and Engineering (BESE) Division"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7850-1743"/>
  <meta name="citation_author" content="Haruko Takeyama"/>
  <meta name="citation_author_institution" content="Research Organization for Nano & Life Innovation, Waseda University"/>
  <meta name="citation_author_institution" content="Computational Bio Big-Data Open Innovation Laboratory, AIST-Waseda University"/>
  <meta name="citation_author_institution" content="Department of Life Science and Medical Bioscience, Waseda University"/>
  <meta name="citation_author_institution" content="Institute for Advanced Research of Biosystem Dynamics, Waseda Research Institute for Science and Engineering, Graduate School of Advanced Science and Engineering, Waseda University"/>
  <meta name="citation_author_email" content="masahosokawa@aoni.waseda.jp"/>
  <meta name="citation_author_email" content="haruko-takeyama@waseda.jp"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2058-8185"/>
  <meta name="citation_reference" content="Nayfach, S., Shi, Z.J., Seshadri, R., Pollard, K.S. & Kyrpides, N.C. New insights from uncultivated genomes of the global human gut microbiome. Nature 568, 505–510 (2019)."/>
  <meta name="citation_reference" content="Rinke, C. et al. Insights into the phylogeny and coding potential of microbial dark matter. Nature 499, 431–437 (2013)."/>
  <meta name="citation_reference" content="Alneberg, J. et al. Binning metagenomic contigs by coverage and composition. Nat Methods 11, 1144–1146 (2014)."/>
  <meta name="citation_reference" content="Stewart, R.D. et al. Assembly of 913 microbial genomes from metagenomic sequencing of the cow rumen. Nat Commun 9, 870 (2018)."/>
  <meta name="citation_reference" content="Parks, D.H. et al. Recovery of nearly 8,000 metagenome-assembled genomes substantially expands the tree of life. Nat Microbiol 2, 1533–1542 (2017)."/>
  <meta name="citation_reference" content="Stalder, T., Press, M.O., Sullivan, S., Liachko, I. & Top, E.M. Linking the resistome and plasmidome to the microbiome. ISME J 13, 2437–2446 (2019)."/>
  <meta name="citation_reference" content="Xu, Y. & Zhao, F. Single-cell metagenomics: challenges and applications. Protein Cell 9, 501–510 (2018)."/>
  <meta name="citation_reference" content="Blainey, P.C. The future is now: single-cell genomics of bacteria and archaea. FEMS Microbiol Rev 37, 407–427 (2013)."/>
  <meta name="citation_reference" content="Marcy, Y. et al. Nanoliter reactors improve multiple displacement amplification of genomes from single cells. PLoS Genet 3, 1702–1708 (2007)."/>
  <meta name="citation_reference" content="Gole, J. et al. Massively parallel polymerase cloning and genome sequencing of single cells using nanoliter microwells. Nat Biotechnol 31, 1126–1132 (2013)."/>
  <meta name="citation_reference" content="Nishikawa, Y. et al. Monodisperse Picoliter Droplets for Low-Bias and Contamination-Free Reactions in Single-Cell Whole Genome Amplification. PLoS One 10, e0138733 (2015)."/>
  <meta name="citation_reference" content="Hosokawa, M., Nishikawa, Y., Kogawa, M. & Takeyama, H. Massively parallel whole genome amplification for single-cell sequencing using droplet microfluidics. Sci Rep 7, 5199 (2017)."/>
  <meta name="citation_reference" content="Lan, F., Demaree, B., Ahmed, N. & Abate, A.R. Single-cell genome sequencing at ultra-high-throughput with microfluidic droplet barcoding. Nat Biotechnol 35, 640–646 (2017)."/>
  <meta name="citation_reference" content="Pasolli, E. et al. Extensive Unexplored Human Microbiome Diversity Revealed by Over 150,000 Genomes from Metagenomes Spanning Age, Geography, and Lifestyle. Cell 176, 649–662 e620 (2019)."/>
  <meta name="citation_reference" content="Martinez-Garcia, M. et al. High-throughput single-cell sequencing identifies photoheterotrophs and chemoautotrophs in freshwater bacterioplankton. ISME J 6, 113–123 (2012)."/>
  <meta name="citation_reference" content="Xu, L., Brito, I.L., Alm, E.J. & Blainey, P.C. Virtual microfluidics for digital quantification and single-cell sequencing. Nat Methods 13, 759–762 (2016)."/>
  <meta name="citation_reference" content="Parks, D.H. et al. A standardized bacterial taxonomy based on genome phylogeny substantially revises the tree of life. Nat Biotechnol 36, 996–1004 (2018)."/>
  <meta name="citation_reference" content="Ondov, B.D. et al. Mash: fast genome and metagenome distance estimation using MinHash. Genome Biol 17, 132 (2016)."/>
  <meta name="citation_reference" content="Sutcliffe, I.C. Cell envelope architecture in the Chloroflexi: a shifting frontline in a phylogenetic turf war. Environ Microbiol 13, 279–282 (2011)."/>
  <meta name="citation_reference" content="Al-Amoudi, S. et al. Metagenomics as a preliminary screen for antimicrobial bioprospecting. Gene 594, 248–258 (2016)."/>
  <meta name="citation_reference" content="Kennedy, K., Hall, M.W., Lynch, M.D., Moreno-Hagelsieb, G. & Neufeld, J.D. Evaluating bias of illumina-based bacterial 16S rRNA gene profiles. Appl Environ Microbiol 80, 5717–5722 (2014)."/>
  <meta name="citation_reference" content="Emms, D.M. & Kelly, S. OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy. Genome Biol 16, 157 (2015)."/>
  <meta name="citation_reference" content="Thompson, L.R. et al. Metagenomic covariation along densely sampled environmental gradients in the Red Sea. ISME J 11, 138–151 (2017)."/>
  <meta name="citation_reference" content="Alzubaidy, H. et al. Rhizosphere microbiome metagenomics of gray mangroves (Avicennia marina) in the Red Sea. Gene 576, 626–636 (2016)."/>
  <meta name="citation_reference" content="Imam, S., Noguera, D.R. & Donohue, T.J. Global insights into energetic and metabolic networks in Rhodobacter sphaeroides. Bmc Systems Biology 7 (2013)."/>
  <meta name="citation_reference" content="Kogawa, M., Hosokawa, M., Nishikawa, Y., Mori, K. & Takeyama, H. Obtaining high-quality draft genomes from uncultured microbes by cleaning and co-assembly of single-cell amplified genomes. Sci Rep 8, 2059 (2018)."/>
  <meta name="citation_reference" content="Haroon, M.F., Thompson, L.R., Parks, D.H., Hugenholtz, P. & Stingl, U. A catalogue of 136 microbial draft genomes from Red Sea metagenomes. Sci Data 3, 160050 (2016)."/>
  <meta name="citation_reference" content="Durham, B.P. et al. Draft genome sequence of marine alphaproteobacterial strain HIMB11, the first cultivated representative of a unique lineage within the Roseobacter clade possessing an unusually small genome. Stand Genomic Sci 9, 632–645 (2014)."/>
  <meta name="citation_reference" content="Arai, W. et al. MAPLE 2.3.0: an improved system for evaluating the functionomes of genomes and metagenomes. Biosci Biotechnol Biochem 82, 1515–1517 (2018)."/>
  <meta name="citation_reference" content="Chevrette, M.G. et al. Taxonomic and Metabolic Incongruence in the Ancient Genus Streptomyces. Front Microbiol 10, 2170 (2019)."/>
  <meta name="citation_reference" content="Shaiber, A. & Eren, A.M. Composite Metagenome-Assembled Genomes Reduce the Quality of Public Genome Repositories. Mbio 10 (2019)."/>
  <meta name="citation_reference" content="Lasken, R.S. & Stockwell, T.B. Mechanism of chimera formation during the Multiple Displacement Amplification reaction. BMC Biotechnol 7, 19 (2007)."/>
  <meta name="citation_reference" content="Chijiiwa, R. et al. Single-cell genomics of uncultured bacteria reveals dietary fiber responders in the mouse gut microbiota. Microbiome 8, 5 (2020)."/>
  <meta name="citation_reference" content="Sanders, J.G. et al. Optimizing sequencing protocols for leaderboard metagenomics by combining long and short reads. Genome Biol 20, 226 (2019)."/>
  <meta name="citation_reference" content="Hosokawa, M. et al. Droplet-based microfluidics for high-throughput screening of a metagenomic library for isolation of microbial enzymes. Biosens Bioelectron 67, 379–385 (2015)."/>
  <meta name="citation_reference" content="Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics 25, 1754–1760 (2009)."/>
  <meta name="citation_reference" content="Bankevich, A. et al. SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol 19, 455–477 (2012)."/>
  <meta name="citation_reference" content="Gurevich, A., Saveliev, V., Vyahhi, N. & Tesler, G. QUAST: quality assessment tool for genome assemblies. Bioinformatics 29, 1072–1075 (2013)."/>
  <meta name="citation_reference" content="Li, H. et al. The Sequence Alignment/Map format and SAMtools. Bioinformatics 25, 2078–2079 (2009)."/>
  <meta name="citation_reference" content="Edgar, R.C. Search and clustering orders of magnitude faster than BLAST. Bioinformatics 26, 2460–2461 (2010)."/>
  <meta name="citation_reference" content="Parks, D.H., Imelfort, M., Skennerton, C.T., Hugenholtz, P. & Tyson, G.W. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. Genome Res 25, 1043–1055 (2015)."/>
  <meta name="citation_reference" content="Seemann, T. Prokka: rapid prokaryotic genome annotation. Bioinformatics 30, 2068–2069 (2014)."/>
  <meta name="citation_reference" content="Bowers, R.M. et al. Minimum information about a single amplified genome (MISAG) and a metagenome-assembled genome (MIMAG) of bacteria and archaea. Nat Biotechnol 35, 725–731 (2017)."/>
  <meta name="citation_reference" content="Letunic, I. & Bork, P. Interactive tree of life (iTOL) v3: an online tool for the display and annotation of phylogenetic and other trees. Nucleic Acids Res 44, W242–245 (2016)."/>
  <meta name="citation_reference" content="Jain, C., Rodriguez, R.L., Phillippy, A.M., Konstantinidis, K.T. & Aluru, S. High throughput ANI analysis of 90K prokaryotic genomes reveals clear species boundaries. Nat Commun 9, 5114 (2018)."/>
  <meta name="citation_reference" content="Roux, S., Enault, F., Hurwitz, B.L. & Sullivan, M.B. VirSorter: mining viral signal from microbial genomic data. PeerJ 3, e985 (2015)."/>
  <meta name="citation_reference" content="Blin, K. et al. antiSMASH 5.0: updates to the secondary metabolite genome mining pipeline. Nucleic Acids Res 47, W81–W87 (2019)."/>
  <meta name="citation_reference" content="Carattoli, A. et al. In silico detection and typing of plasmids using PlasmidFinder and plasmid multilocus sequence typing. Antimicrob Agents Chemother 58, 3895–3903 (2014)."/>
  <meta name="citation_reference" content="Eren, A.M. et al. Anvi’o: an advanced analysis and visualization platform for ‘omics data. PeerJ 3, e1319 (2015)."/>
  <meta name="citation_reference" content="Ankenbrand, M.J., Hohlfeld, S., Hackl, T. & Förster, F. AliTV—interactive visualization of whole genome comparisons. PeerJ Computer Science 3 (2017)."/>
  <meta name="twitter:title" content="Extensive single-cell genomics reveals bacterial diversity and diverse phage host ranges in the area in and around the Red Sea"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="To improve our understanding of the environmental microbiome, we developed a single-cell genome sequencing platform, named SAG-gel, which utilizes gel beads for single-cell isolation, cell lysis, and whole genome amplification (WGA) for sequencing. SAG-gel enables serial, parallel and independent reactions of > 100,000 single cells in a single tube, delivering high-quality genome recovery with storable randomized single-cell genome libraries. From soil and marine environmental sources, we acquired 734 partial genomes that are recapitulated in 231 species, 35% of which were assigned as high-to-medium qualities. We found that each genome to be almost unique and 98.7% of them were newly identified, implying the complex genetic diversities across 44 phyla. The various metabolic capabilities including virulence factors and biosynthetic gene clusters were found across the lineages at single-cell resolution. This technology will accelerate the accumulation of reference genomes of uncharacterized environmental microbes and provide us new insights for their roles.

### Competing Interest Statement

M.H. and H.T. are shareholders in bitBiome, to which the patents 849pertaining to SAG-gel workflow were transferred."/>
  <meta name="og-title" property="og:title" content="Extensive single-cell genomics reveals bacterial diversity and diverse phage host ranges in the area in and around the Red Sea"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.05.962001v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="To improve our understanding of the environmental microbiome, we developed a single-cell genome sequencing platform, named SAG-gel, which utilizes gel beads for single-cell isolation, cell lysis, and whole genome amplification (WGA) for sequencing. SAG-gel enables serial, parallel and independent reactions of > 100,000 single cells in a single tube, delivering high-quality genome recovery with storable randomized single-cell genome libraries. From soil and marine environmental sources, we acquired 734 partial genomes that are recapitulated in 231 species, 35% of which were assigned as high-to-medium qualities. We found that each genome to be almost unique and 98.7% of them were newly identified, implying the complex genetic diversities across 44 phyla. The various metabolic capabilities including virulence factors and biosynthetic gene clusters were found across the lineages at single-cell resolution. This technology will accelerate the accumulation of reference genomes of uncharacterized environmental microbes and provide us new insights for their roles.

### Competing Interest Statement

M.H. and H.T. are shareholders in bitBiome, to which the patents 849pertaining to SAG-gel workflow were transferred."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-05-11"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.05.962001v2"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2539361"/>
  <title>
   Extensive single-cell genomics reveals bacterial diversity and diverse phage host ranges in the area in and around the Red Sea | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"kt8rIL9DiwdAHivj8CkYgNH8d2jeZ8A5rI8bLiEfNiA"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.03.05.962001v2"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.03.05.962001v2"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.05.962001v2"}],"nid":"2539361","apath":"\/biorxiv\/early\/2022\/05\/11\/2020.03.05.962001.atom","pisa":"biorxiv;2020.03.05.962001v2","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.05.962001v2","title":"Extensive single-cell genomics reveals bacterial diversity and diverse phage host ranges in the area in and around the Red Sea","identifier":"node\/2539361"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BhK3azoiPnWyv6wIP3ojkO6TI7wRcn+R5+OG\\\\\\\/qIis\\\\\\\/thTv5aPTrGYf6L+rDnjNsfLdwJZ5ko1Yz5rNgco0\\\\\\\/IUJOao+sul94Y5cpt49wOe+VMP+42Mx8Pmgq6hvlA5scCRg6C88Y4J6avBHrJ8o8LikEOGyl3XxFkJR3hzpkarvazyZQH3p5aesGWGKOuFgZqtoLOZX4Gg9PSQMHj2tHBfMUWbqJI8QOneM7IpbB1Ig86rIXUdgg7g2sv0kMo9YgywSXs+wf\\\\\\\/2mSEdANzieozNlinSYY2co4KpGVg8fRhT8F15nbIcA1wW44hVggcgBiNmDUsogiTWXZQRZl+i2PEQpTLvQB\\\\\\\/r93QO9neW+PKTMnipCGJ+9A+1G07Xp\\\\\\\/M85hCUrSQ61TlcvsDGRozG9q7WoSOxgean4vH6R6MlqTDnQE1JEcrTotdhKid17lfLe4ZDEqvF6RPXZ037MLfPXP4zt8xDWf2OkmC+utqcuXCf45XI20q8gYviqo0SJMfrLWh\\\\\\\/UvWuDtjRHs\\\\\\\/dsQufu5dCiKVGfof2cCyzkx18LgeZHXKjSkAHYBJt2UUXuVBMlslJrStCIVYGC\\\\\\\/GRPaimxN6WZf6IuRGLm1xTb5Ea1Hb4jhd2sr\\\\\\\/y5YrhX94eGgtlka2gX6N5MBB5RnSAAcyVsq+Hp1VWzXu4vlKDOSnbiUCtt0wETUAGZPmy0jMs5WL58ptzYNiVZwq7sPOcAlEScPVWybfsmouFLtqZrOTxvgr2XBZQw8\\\\\\\/KPdOcO3NBmwLxKARebNyZIlWEFxZLS8A9wPoCtlh\\\\\\\/BKsceJxm+rDzK\\\\\\\/InVzXYII\\\\\\\/7RIAI5feLs8agXh21zCObh2wIMTlzdvyvq1tIdrIeEJJQoLy5k21aW9lxZ\\\\\\\/wkBdhkTLA1DDGG5MUM\\\\\\\/vQ1\\\\\\\/VMRJZjUl9y+nxAhL1TPwBZRbEtLG41BdqlEoVme7Lcaq9MmodaWCxFpg5HeyVYHWVPIErmF41i45K5pUoU2Q5ZL1cqWIbC\\\\\\\/eUhntPfkj60QpU09qvsfZ7mbvZvXcXxONhY5e\\\\\\\/pPCb9JnDrLGfHIywob0mxEuTpQos\\\\\\\/PRiMVWu8ihSAwv1Z3nVBhBgS+8YFcve9kVz78x4Mw6zNXGQ1bGiNkbVYLLLONrpElbtY7HjjttLO4lmy8erpG5+rkttyq0erbsZ9k9pxK+9RdMzF4zNFOmTBqYDwmxHMwpoO\\\\\\\/om3MSBW7dEvh2HXK3kroYmdJmf2c1YtX43ru3WuUHtVdmpUXSeyjLEddNT0TQMFTKhzeJTVOwJDTlVuI4SxFg\\\\\\\/HIpZItfcCdYopzhFerWBSoWdMbTXanSclnpSuFTCWD+PZjm8Krd3u\\\\\\\/HfNpqTCTZyjb3NiLMJJWtxKZLbRh1a\\\\\\\/9k2jh79It42n+8ZaCoF96Pk=\\\u0022,\\\u0022iv\\\u0022:\\\u0022kaoFJ7cBqYPh8iztweia1g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00226e987c2f488968bd9e8326b74f90645bc65a8998ee35574df9b85cb3bbb80f51\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.05.962001v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2539361 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.05.962001v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_1929633950">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_1929633950" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_421280221" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-lUar-Cy9ngm3Q6qaFKiNmqHt0gtZW9jIGt8Yv4CBT3w"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2539361" data-node-nid="2539361" id="node-2539361--2314597809" data-pisa="biorxiv;2020.03.05.962001v2" data-pisa-master="biorxiv;2020.03.05.962001" data-apath="/biorxiv/early/2022/05/11/2020.03.05.962001.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Extensive single-cell genomics reveals bacterial diversity and diverse phage host ranges in the area in and around the Red Sea
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0001-5443-7230" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Yohei
                      </span>
                      <span class="nlm-surname">
                       Nishikawa
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Masato
                      </span>
                      <span class="nlm-surname">
                       Kogawa
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2">
                      <a href="http://orcid.org/0000-0002-7070-4690" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Masahito
                      </span>
                      <span class="nlm-surname">
                       Hosokawa
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Ryota
                      </span>
                      <span class="nlm-surname">
                       Wagatsuma
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4">
                      <a href="http://orcid.org/0000-0002-4727-045X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Katsuhiko
                      </span>
                      <span class="nlm-surname">
                       Mineta
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="5">
                      <span class="nlm-given-names">
                       Kai
                      </span>
                      <span class="nlm-surname">
                       Takahashi
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="6">
                      <span class="nlm-given-names">
                       Keigo
                      </span>
                      <span class="nlm-surname">
                       Ide
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7">
                      <a href="http://orcid.org/0000-0003-0197-5478" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Kei
                      </span>
                      <span class="nlm-surname">
                       Yura
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="8">
                      <span class="nlm-given-names">
                       Hayedeh
                      </span>
                      <span class="nlm-surname">
                       Behzad
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9">
                      <a href="http://orcid.org/0000-0001-7850-1743" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Takashi
                      </span>
                      <span class="nlm-surname">
                       Gojobori
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10">
                      <a href="http://orcid.org/0000-0002-2058-8185" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Haruko
                      </span>
                      <span class="nlm-surname">
                       Takeyama
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.03.05.962001
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2539361--2314597809" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Yohei Nishikawa
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Research Organization for Nano & Life Innovation, Waseda University
                       </span>
                       , 513 Waseda tsurumaki-cho, Shinjuku-ku, Tokyo, 162–0041,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Computational Bio Big-Data Open Innovation Laboratory, AIST-Waseda University
                       </span>
                       , 3-4-1 Okubo, Shinjuku-ku, Tokyo, 169–0072,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Yohei%2BNishikawa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Nishikawa%20Y&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AYohei%2BNishikawa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-5443-7230" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Yohei Nishikawa
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Masato Kogawa
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Research Organization for Nano & Life Innovation, Waseda University
                       </span>
                       , 513 Waseda tsurumaki-cho, Shinjuku-ku, Tokyo, 162–0041,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Masato%2BKogawa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Kogawa%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AMasato%2BKogawa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Masahito Hosokawa
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Research Organization for Nano & Life Innovation, Waseda University
                       </span>
                       , 513 Waseda tsurumaki-cho, Shinjuku-ku, Tokyo, 162–0041,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Computational Bio Big-Data Open Innovation Laboratory, AIST-Waseda University
                       </span>
                       , 3-4-1 Okubo, Shinjuku-ku, Tokyo, 169–0072,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Life Science and Medical Bioscience, Waseda University
                       </span>
                       , 2-2 Wakamatsu-cho, Shinjuku-ku, Tokyo, 162–8480,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Institute for Advanced Research of Biosystem Dynamics, Waseda Research Institute for Science and Engineering, Graduate School of Advanced Science and Engineering, Waseda University
                       </span>
                       , 3-4-1 Okubo, Shinjuku-ku, Tokyo 169-8555,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Masahito%2BHosokawa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Hosokawa%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AMasahito%2BHosokawa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0002-7070-4690" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Masahito Hosokawa
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:masahosokawa@aoni.waseda.jp" class="" data-icon-position="" data-hide-link-title="0">
                        masahosokawa@aoni.waseda.jp
                       </a>
                       <a href="mailto:haruko-takeyama@waseda.jp" class="" data-icon-position="" data-hide-link-title="0">
                        haruko-takeyama@waseda.jp
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Ryota Wagatsuma
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Computational Bio Big-Data Open Innovation Laboratory, AIST-Waseda University
                       </span>
                       , 3-4-1 Okubo, Shinjuku-ku, Tokyo, 169–0072,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Life Science and Medical Bioscience, Waseda University
                       </span>
                       , 2-2 Wakamatsu-cho, Shinjuku-ku, Tokyo, 162–8480,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Ryota%2BWagatsuma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Wagatsuma%20R&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ARyota%2BWagatsuma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Katsuhiko Mineta
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Research Organization for Nano & Life Innovation, Waseda University
                       </span>
                       , 513 Waseda tsurumaki-cho, Shinjuku-ku, Tokyo, 162–0041,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        King Abdullah University of Science and Technology (KAUST), Computer, Electrical, and Mathematical Sciences and Engineering (CEMSE) Division
                       </span>
                       , Thuwal 23955-6900,
                       <span class="nlm-country">
                        Saudi Arabia
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        King Abdullah University of Science and Technology (KAUST), Computational Bioscience Research Center (CBRC)
                       </span>
                       , Thuwal 23955-6900,
                       <span class="nlm-country">
                        Saudi Arabia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Katsuhiko%2BMineta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Mineta%20K&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AKatsuhiko%2BMineta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-4727-045X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Katsuhiko Mineta
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Kai Takahashi
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Life Science and Medical Bioscience, Waseda University
                       </span>
                       , 2-2 Wakamatsu-cho, Shinjuku-ku, Tokyo, 162–8480,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Kai%2BTakahashi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Takahashi%20K&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AKai%2BTakahashi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">
                     Keigo Ide
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Computational Bio Big-Data Open Innovation Laboratory, AIST-Waseda University
                       </span>
                       , 3-4-1 Okubo, Shinjuku-ku, Tokyo, 169–0072,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Life Science and Medical Bioscience, Waseda University
                       </span>
                       , 2-2 Wakamatsu-cho, Shinjuku-ku, Tokyo, 162–8480,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Keigo%2BIde%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Ide%20K&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AKeigo%2BIde%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-7">
                    <div class="author-tooltip-name">
                     Kei Yura
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Research Organization for Nano & Life Innovation, Waseda University
                       </span>
                       , 513 Waseda tsurumaki-cho, Shinjuku-ku, Tokyo, 162–0041,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Life Science and Medical Bioscience, Waseda University
                       </span>
                       , 2-2 Wakamatsu-cho, Shinjuku-ku, Tokyo, 162–8480,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        7
                       </span>
                       <span class="nlm-institution">
                        Graduate School of Humanities and Sciences, Ochanomizu University
                       </span>
                       , 2-1-1 Otsuka, Bunkyo-ku, Tokyo 112-8610,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Kei%2BYura%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Yura%20K&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AKei%2BYura%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-0197-5478" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Kei Yura
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-8">
                    <div class="author-tooltip-name">
                     Hayedeh Behzad
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        8
                       </span>
                       <span class="nlm-institution">
                        King Abdullah University of Science and Technology (KAUST), Biological and Environmental Sciences and Engineering (BESE) Division
                       </span>
                       , Thuwal 23955-6900,
                       <span class="nlm-country">
                        Saudi Arabia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Hayedeh%2BBehzad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Behzad%20H&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AHayedeh%2BBehzad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-9">
                    <div class="author-tooltip-name">
                     Takashi Gojobori
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        King Abdullah University of Science and Technology (KAUST), Computational Bioscience Research Center (CBRC)
                       </span>
                       , Thuwal 23955-6900,
                       <span class="nlm-country">
                        Saudi Arabia
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        8
                       </span>
                       <span class="nlm-institution">
                        King Abdullah University of Science and Technology (KAUST), Biological and Environmental Sciences and Engineering (BESE) Division
                       </span>
                       , Thuwal 23955-6900,
                       <span class="nlm-country">
                        Saudi Arabia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Takashi%2BGojobori%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Gojobori%20T&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ATakashi%2BGojobori%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-7850-1743" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Takashi Gojobori
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-10">
                    <div class="author-tooltip-name">
                     Haruko Takeyama
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Research Organization for Nano & Life Innovation, Waseda University
                       </span>
                       , 513 Waseda tsurumaki-cho, Shinjuku-ku, Tokyo, 162–0041,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Computational Bio Big-Data Open Innovation Laboratory, AIST-Waseda University
                       </span>
                       , 3-4-1 Okubo, Shinjuku-ku, Tokyo, 169–0072,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Life Science and Medical Bioscience, Waseda University
                       </span>
                       , 2-2 Wakamatsu-cho, Shinjuku-ku, Tokyo, 162–8480,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Institute for Advanced Research of Biosystem Dynamics, Waseda Research Institute for Science and Engineering, Graduate School of Advanced Science and Engineering, Waseda University
                       </span>
                       , 3-4-1 Okubo, Shinjuku-ku, Tokyo 169-8555,
                       <span class="nlm-country">
                        Japan
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Haruko%2BTakeyama%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Takeyama%20H&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AHaruko%2BTakeyama%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0002-2058-8185" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Haruko Takeyama
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:masahosokawa@aoni.waseda.jp" class="" data-icon-position="" data-hide-link-title="0">
                        masahosokawa@aoni.waseda.jp
                       </a>
                       <a href="mailto:haruko-takeyama@waseda.jp" class="" data-icon-position="" data-hide-link-title="0">
                        haruko-takeyama@waseda.jp
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.03.05.962001v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2539361" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2539361/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.03.05.962001v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2539361" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2539361/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.03.05.962001v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2539361" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2539361/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.03.05.962001v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2539361" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2539361/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.03.05.962001v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2539361" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2539361/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.03.05.962001v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2539361" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2539361/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-4">
                             To improve our understanding of the environmental microbiome, we developed a single-cell genome sequencing platform, named SAG-gel, which utilizes gel beads for single-cell isolation, cell lysis, and whole genome amplification (WGA) for sequencing. SAG-gel enables serial, parallel and independent reactions of > 100,000 single cells in a single tube, delivering high-quality genome recovery with storable randomized single-cell genome libraries. From soil and marine environmental sources, we acquired 734 partial genomes that are recapitulated in 231 species, 35% of which were assigned as high-to-medium qualities. We found that each genome to be almost unique and 98.7% of them were newly identified, implying the complex genetic diversities across 44 phyla. The various metabolic capabilities including virulence factors and biosynthetic gene clusters were found across the lineages at single-cell resolution. This technology will accelerate the accumulation of reference genomes of uncharacterized environmental microbes and provide us new insights for their roles.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-6">
                             Technological innovations in genome sequencing have improved our understanding of the genomic characteristics, genetic diversity, and metabolic capabilities of various biological samples. In particular, metagenomics and singlecell genomics exhibit great potential to enhance our understanding of the genomic structure and dynamics of complex microbial communities
                             <sup>
                              <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                               1
                              </a>
                              ,
                              <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                               2
                              </a>
                             </sup>
                             . The recent accumulation of metagenomic sequencing data and advances in computational techniques have enabled the integration of sequencing data, generating metagenome-assembled genomes (MAGs)
                             <sup>
                              <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                              ,
                              <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                             </sup>
                             . Although MAGs are expected to provide an expedient path for exploring microbial dark matter
                             <sup>
                              <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                             </sup>
                             , several problems remain including the inefficiency of genome reconstruction, 16S rRNA gene recovery
                             <sup>
                              <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                             </sup>
                             , and identification of microbial plasmids
                             <sup>
                              <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                               6
                              </a>
                             </sup>
                             .
                            </p>
                            <p id="p-7">
                             To overcome these problems and confirm the reliability of sequencing data, single-cell genome sequencing plays an important role
                             <sup>
                              <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                               7
                              </a>
                              ,
                              <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                             </sup>
                             . Recently, microfluidic-based whole genome amplification (WGA) has been identified as a replacement for conventional fluorescence-activated cell sorting (FACS)-associated multi-well platforms
                             <sup>
                              <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ,
                              <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                               10
                              </a>
                             </sup>
                             , minimizing operating costs as well as the risk of DNA contamination, leading to increased reaction efficiency and throughput. In particular, droplet microfluidics is a noteworthy technique considered suitable for handling single cells
                             <sup>
                              <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                              –
                              <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                             </sup>
                             . However, the immiscible oil phase makes it difficult to conduct multistep reactions in droplets, causing an obstacle to lysis gram-positive and -negative bacteria and to exclude the inhibitors for WGA.
                            </p>
                            <p id="p-8">
                             Herein we describe a method for multiplexed single-cell genome sequencing with massively produced single-cell amplified genomes (SAGs) in the gel that we named SAG-gel sequencing. We obtained high-coverage single-cell genome sequences of various environmental samples including soil, sediment, and seawater collected from the area surrounding the Red Sea in Saudi Arabia. This paper demonstrates that massively parallel single-cell genomics accumulates genome sequencing data of unknown species, unveils the obscured diversity present in uncultured environmental microbiome, and provides insight regarding their function in ecosystems.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Results
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Workflow of SAG-gel platform
                             </h3>
                             <p id="p-9">
                              The strategy of SAG-gel is to perform a series of reactions on encapsulated single cells in a massively parallel manner (
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Fig. 1a
                              </a>
                              ). Single cells are massively captured in gel beads and lysed by enzyme cocktails designed for both gram-positive and - negative bacteria. Through the whole process, SAG-gel converts tiny genomes of various microbial cells into the amplified genomes in uniform-shaped beads floated in a single-tube. The gel matrix facilitates maintaining the compartmented genomes during cell lysis, washing, and WGA processes and preventing crosscontamination, and protecting the amplified DNA for long-term storage.
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F1.large.jpg?width=800&height=600&carousel=1" title="Single-cell amplified genome in the gel (SAG-gel) approach for massively parallel single-cell genome sequencing. a: Bacterial suspensions are encapsulated into 40 μm of microfluidic droplets at the single-cell level (0.1 cell/droplet) with ultra-low melting temperature agarose solutions. Collected droplets are solidified by cooling and single-cells are captured in gel matrix. Gel-beads are transitioned in aqueous phase, which enables repeating “reaction and wash” steps. Through these steps, single-cells were lysed and whole genome was amplified. Gel-beads after WGA can be isolated into multi-well plate with FACS, which can be applied to subsequent analysis including the next generation sequencing. b: Microscopic image of microfluidic water in oil (W/O) droplets and gel-beads. Amplified DNA can be visualized by adding DNA intercalating dye (SYBR Green). The scale bar is 50 μm." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-322709305" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Single-cell amplified genome in the gel (SAG-gel) approach for massively parallel single-cell genome sequencing.</span> a: Bacterial suspensions are encapsulated into 40 μm of microfluidic droplets at the single-cell level (0.1 cell/droplet) with ultra-low melting temperature agarose solutions. Collected droplets are solidified by cooling and single-cells are captured in gel matrix. Gel-beads are transitioned in aqueous phase, which enables repeating “reaction and wash” steps. Through these steps, single-cells were lysed and whole genome was amplified. Gel-beads after WGA can be isolated into multi-well plate with FACS, which can be applied to subsequent analysis including the next generation sequencing. b: Microscopic image of microfluidic water in oil (W/O) droplets and gel-beads. Amplified DNA can be visualized by adding DNA intercalating dye (SYBR Green). The scale bar is 50 μm.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F1.medium.gif" width="440" height="335"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F1.medium.gif" width="440" height="335"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1.
                               </span>
                               <span class="caption-title">
                                Single-cell amplified genome in the gel (SAG-gel) approach for massively parallel single-cell genome sequencing.
                               </span>
                               <p id="p-10" class="first-child">
                                a: Bacterial suspensions are encapsulated into 40 μm of microfluidic droplets at the single-cell level (0.1 cell/droplet) with ultra-low melting temperature agarose solutions. Collected droplets are solidified by cooling and single-cells are captured in gel matrix. Gel-beads are transitioned in aqueous phase, which enables repeating “reaction and wash” steps. Through these steps, single-cells were lysed and whole genome was amplified. Gel-beads after WGA can be isolated into multi-well plate with FACS, which can be applied to subsequent analysis including the next generation sequencing.
                               </p>
                               <p id="p-11">
                                b: Microscopic image of microfluidic water in oil (W/O) droplets and gel-beads. Amplified DNA can be visualized by adding DNA intercalating dye (SYBR Green). The scale bar is 50 μm.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-12">
                              Based on this concept, we developed a gel-beads based single-cell processing method by using gram-positive and -negative model bacteria (Escherichia coli and Bacillus subtilis). After cell encapsulation, gel beads were dispersed into the aqueous phase, which enables small molecules to penetrate into gel beads. By repeating a series of “reaction and wash” steps, practical step-by-step reactions including cell lysis, DNA purification, and WGA can be achieved (
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Fig. 1b
                              </a>
                              ). The gel beads containing SAGs were then specifically isolated into multi-well plates for re-amplifying as storage SAG libraries (Supplementary Fig. 1a). The use of enzyme cocktails enhances the genome cover rate by 9.1-25% and improves genome amplification biases compared to conventional alkaline lysis treatment (Supplementary Fig. 1b and c). We confirmed that SAG-gel excludes the risk of cross-contamination between gel beads, showing all of the sequenced SAGs had > 99.6% of their reads mapping to either E. coli or B. subtilis (Supplementary Fig. 1d). The de novo assembled contigs showed a lower number of misassembled and unaligned contigs in SAG-gel (Supplementary Fig. 1e and Supplementary Table 1). In addition, SAG-gel showed a larger length of N50 and lower number of mismatches and indels, suggesting that accurate genome information can be acquired compared to conventional in-tube WGA reactions.
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Single-cell genome sequencing from a variety of environmental bacteria
                             </h3>
                             <p id="p-13">
                              We applied SAG-gel for single-cell genome sequencing of 8 environmental samples including 6 soil samples (beach soil: S1; desert soil: S2; mangrove soil: S3; fresh sea sediment: S4; frozen sea sediment: S5; and seashore soil: S6) and 2 seawater samples (harbor seawater: W1 and open ocean seawater: W2). At first, we individually generated 1,970 SAG reactions from isolated gel beads and 1,031 out of them were identified as positive prokaryotic fractions according to PCR and 16S rRNA gene sequencing. Marine microbiome (S5 and S6) showed higher positive reaction rates (> 61%) than soil microbiome (29-56%). After the sequencing of 929 positive prokaryotic fractions and subsequent data curation, we recovered 16 high-quality, 244 medium-quality, and 474 low-quality draft genomes, ranging from 0.65 to 5.8 Mbp in total length and 18 to 1,347 contigs (
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Fig. 2 a, b
                              </a>
                              , Supplementary Table 2). High-quality SAGs yielded an average N50 of 83.2 kb, while medium- and low-quality ones yielded 27.5 kb and 12.3 kb, respectively. Regarding the presence of tRNAs from high-to-low SAG quality, the average numbers were 19.8, 16.5, and 10.6, respectively. The SAG-containing beads isolation with FACS effectively prevented the contaminated SAG prevalence (7.62% (56/735)) compared to the bead isolation with manual picking (38.7% (75/194)) (Supplementary Fig. 2). The average completeness for each sampling site was 32.9% in S1, 37.1% in S2, 42.3% in S3, 36.2% in S4, 54.8% in S5, 39.4% in S6, 42.2% in W1, and 53.6% in W2 (
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig. 2 c
                              </a>
                              ), while the completeness was broadly distributed from 1.37% to 99.9%.
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F2.large.jpg?width=800&height=600&carousel=1" title="734 SAGs recovered from 8 environmental samples a: Genome completeness and contamination of each SAG is plotted. The SAGs were classified as high, medium, low, and contamination according to the score of completeness and contamination, and the presence of tRNA sequence detected defined by the Genomic Standards Consortium. b: Distributions of the number of contigs, the number of tRNA, N50, and total length classified by the quality of draft genomes. c: Distributions of completeness in each sampling site. Solid line is the median and dotted line is the quartile." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-322709305" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">734 SAGs recovered from 8 environmental samples</span> a: Genome completeness and contamination of each SAG is plotted. The SAGs were classified as high, medium, low, and contamination according to the score of completeness and contamination, and the presence of tRNA sequence detected defined by the Genomic Standards Consortium. b: Distributions of the number of contigs, the number of tRNA, N50, and total length classified by the quality of draft genomes. c: Distributions of completeness in each sampling site. Solid line is the median and dotted line is the quartile.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F2.medium.gif" width="440" height="320"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F2.medium.gif" width="440" height="320"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2.
                               </span>
                               <span class="caption-title">
                                734 SAGs recovered from 8 environmental samples
                               </span>
                               <p id="p-14" class="first-child">
                                a: Genome completeness and contamination of each SAG is plotted. The SAGs were classified as high, medium, low, and contamination according to the score of completeness and contamination, and the presence of tRNA sequence detected defined by the Genomic Standards Consortium.
                               </p>
                               <p id="p-15">
                                b: Distributions of the number of contigs, the number of tRNA, N50, and total length classified by the quality of draft genomes.
                               </p>
                               <p id="p-16">
                                c: Distributions of completeness in each sampling site. Solid line is the median and dotted line is the quartile.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Taxonomic distribution of SAGs
                             </h3>
                             <p id="p-17">
                              Out of 734 SAGs, 76.0% (558/734) recovered > 500 bp 16S rRNA gene sequence and 58.7% (431/734) were > 1,400 bp in length (Supplementary Table 3). SAGgel platform yielded longer sequences for genome assemblies and notably higher recovery rates of 16S rRNA gene compared to 7–17% in metagenome-based analysis
                              <sup>
                               <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">
                                5
                               </a>
                               ,
                               <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                                14
                               </a>
                              </sup>
                              and 23–27% in other single-cell genome analyses
                              <sup>
                               <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                                15
                               </a>
                               ,
                               <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                                16
                               </a>
                              </sup>
                              . The 16S rRNA gene-based phylogenetic analysis revealed that 117 (102 specific) sequences > 1,400 bp and 204 (173 specific) sequences > 500 bp have no obvious reference sequences (
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Fig. 3a
                              </a>
                              ). To annotate all SAGs including those without 16S rRNA gene sequence, we utilized GTDB-Tk
                              <sup>
                               <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                                17
                               </a>
                              </sup>
                              for taxonomic classification, resulting in 650 SAGs classified as 11 archaea and 639 bacteria, consisting of 44 phyla, 75 classes, 117 orders, and 127 families (Supplementary Table 3). Proteobacteria was dominant [277 (43.3%)] and Desulfobacterota [70 (10.9%)] and Cyanobacteriota [44 (6.9%)] followed. We found a total of 231 species from 650 SAGs (Supplementary Table 4), 98.7% (228 species) of which were newly identified with > 0.05 mash distance to reference genomes registered in Refseq
                              <sup>
                               <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                                18
                               </a>
                              </sup>
                              , suggesting that SAG-gel enables the large-scale accumulation of undescribed genomes from complex microbial communities (Supplementary Fig. 3).
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F3.large.jpg?width=800&height=600&carousel=1" title="Taxonomical classifications of SAGs a: Taxonomic annotation of 639 bacterial SAGs by phylum level. Inner circle: Top hit phylum; outer circle: sampling site; inner dot: 16S rRNA gene detected; outer dot: < 97% identity with top hit result. Any phyla which was not in the top ten phyla was clustered as “others.” b: Comparison of bacterial composition in 16S rRNA gene sequencing and SAGs (Phylum level). Taxonomic identification of SAGs was conducted by GTDB-Tk. In S1, the bacterial composition of 16S rRNA gene sequencing was undetermined because of the shortage of extract DNA. Any phyla which shared < 5% was clustered as “others”." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-322709305" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Taxonomical classifications of SAGs</span> a: Taxonomic annotation of 639 bacterial SAGs by phylum level. Inner circle: Top hit phylum; outer circle: sampling site; inner dot: 16S rRNA gene detected; outer dot: < 97% identity with top hit result. Any phyla which was not in the top ten phyla was clustered as “others.” b: Comparison of bacterial composition in 16S rRNA gene sequencing and SAGs (Phylum level). Taxonomic identification of SAGs was conducted by GTDB-Tk. In S1, the bacterial composition of 16S rRNA gene sequencing was undetermined because of the shortage of extract DNA. Any phyla which shared < 5% was clustered as “others”.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F3.medium.gif" width="372" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F3.medium.gif" width="372" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3.
                               </span>
                               <span class="caption-title">
                                Taxonomical classifications of SAGs
                               </span>
                               <p id="p-18" class="first-child">
                                a: Taxonomic annotation of 639 bacterial SAGs by phylum level. Inner circle: Top hit phylum; outer circle: sampling site; inner dot: 16S rRNA gene detected; outer dot: < 97% identity with top hit result. Any phyla which was not in the top ten phyla was clustered as “others.”
                               </p>
                               <p id="p-19">
                                b: Comparison of bacterial composition in 16S rRNA gene sequencing and SAGs (Phylum level). Taxonomic identification of SAGs was conducted by GTDB-Tk. In S1, the bacterial composition of 16S rRNA gene sequencing was undetermined because of the shortage of extract DNA. Any phyla which shared < 5% was clustered as “others”.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-20">
                              We compared the phylum-level taxonomic compositions assessed from SAGs and metagenomic 16S rRNA gene amplicon sequencing (
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Fig. 3b
                              </a>
                              ). While the number of major phyla (> 5% in relative abundance) detected in 16S rRNA gene sequencing was 11, SAGs recovered 10 phyla—all except for Chloroflexi, which is reported to be a tough bacterium for DNA extraction
                              <sup>
                               <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                                19
                               </a>
                              </sup>
                              . The bacterial composition of SAGs was not consistent with the composition of 16S rRNA gene sequencing except for mangrove (S3), where the result also corresponded to the previously reported data
                              <sup>
                               <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                                20
                               </a>
                              </sup>
                              (Supplementary Fig. 4). In general, because SAGs reflect the exact cell numbers whereas 16S rRNA gene sequencing reflects the copy number of 16S rRNA genes and is also affected by amplification bias in the library preparation step
                              <sup>
                               <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                                21
                               </a>
                              </sup>
                              , these two methods may exhibit different bacterial composition. From the randomized sampling, SAG-gel can also recover SAGs of very rare phyla (< 0.1% in 16S rRNA gene sequencing), including Omnitrophota bacteria in sea sediment (S4) and Elusimicrobiota bacteria in seashore (S6).
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Determining gene distributions at single-cell resolution
                             </h3>
                             <p id="p-21">
                              From each sample set consists of 734 SAGs, we compared sampling site-specific enriched genes. Out of 3,221 orthologous gene groups (OGs)
                              <sup>
                               <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                                22
                               </a>
                              </sup>
                              consisting of > 100 genes, 2,621 OGs (81%) were enriched (p < 0.01) in at least one sampling site (
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Fig. 4a
                              </a>
                              ) (Supplementary Table 5). For example, OGs related to phosphate acquisition (OG0000970, 0002021) were enriched in seawaters (W1 and W2) as previously reported
                              <sup>
                               <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                                23
                               </a>
                              </sup>
                              . In addition, phycobilisome protein (OG0001535) and phycobilisome linker polypeptide (OG0001628) were enriched in W2, while cytochrome c (OG0000546) and cytochrome P450 (OG0000546) were enriched in W1. In mangrove (S3) where heavy metals are accumulated
                              <sup>
                               <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                                24
                               </a>
                              </sup>
                              , OGs related to heavy metal resistance (OG0001092) exhibited higher abundance. In sea sediment (S4), sulfatase (OG0000005) and the radical SAM superfamily (OG0000056), which are mainly derived from Omnitrophota and Desulfobacterota bacteria, were enriched, while these features were not detected in frozen sea sediment (S5). In desert (S2), OGs related to spore germination (OG0002927 and OG0003158) were enriched. In seashore (S6), OGs related to aldehyde ferredoxin oxidoreductase (OG0000150) mainly derived from archaea were enriched. These results reflect the characteristics of each sampling site or sample storage conditions. The extraction of genomic features from a set of multiple draft genomes is useful for overviewing the whole microbial communities of the target environment like a metagenomic analysis manner. Furthermore, we can reveal which bacteria are responsible for the environment-specific genetic characteristics at single-cell resolution by in-depth sequence assignments.
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F4.large.jpg?width=800&height=600&carousel=1" title="SAG-gel reveals distributions of viral signals and secondary metabolite biosynthetic gene clusters (BGCs) in target environment a: Heatmap of sampling site specific orthologous gene groups (OGs) (p < 0.01) based on the Z score. Out of 3221 orthologous gene groups (OGs) which consist of > 100 genes, 2621 OGs were enriched (p < 0.01) in at least one sampling site and the smallest 50 OGs were extracted for figure drawing. b: Detected gene number of viral signals and BGCs classified by their category. In VirSorter, phages matching the prophage database (VirSorter categories 1 and 2: viruses with ‘indeterminate integration status’; 4 and 5: integrated prophage) were included. In antiSMASH, BGCs of detected numbers " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-322709305" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">SAG-gel reveals distributions of viral signals and secondary metabolite biosynthetic gene clusters (BGCs) in target environment</span> a: Heatmap of sampling site specific orthologous gene groups (OGs) (p < 0.01) based on the Z score. Out of 3221 orthologous gene groups (OGs) which consist of > 100 genes, 2621 OGs were enriched (p < 0.01) in at least one sampling site and the smallest 50 OGs were extracted for figure drawing. b: Detected gene number of viral signals and BGCs classified by their category. In VirSorter, phages matching the prophage database (VirSorter categories 1 and 2: viruses with ‘indeterminate integration status’; 4 and 5: integrated prophage) were included. In antiSMASH, BGCs of detected numbers <10 were classified as “others.” c: Distributions of viral signals and BGCs by SAGs colored by sampling sites. d: Completeness, taxonomical classification, and the number of viral signals and BGCs were summarized by SAGs in S3: mangrove soil. High- and mediumquality SAGs were included. In the result of BGCs, clusters assigned as “cf_putative” were not counted.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F4.medium.gif" width="341" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F4.medium.gif" width="341" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4.
                               </span>
                               <span class="caption-title">
                                SAG-gel reveals distributions of viral signals and secondary metabolite biosynthetic gene clusters (BGCs) in target environment
                               </span>
                               <p id="p-22" class="first-child">
                                a: Heatmap of sampling site specific orthologous gene groups (OGs) (p < 0.01) based on the Z score. Out of 3221 orthologous gene groups (OGs) which consist of > 100 genes, 2621 OGs were enriched (p < 0.01) in at least one sampling site and the smallest 50 OGs were extracted for figure drawing.
                               </p>
                               <p id="p-23">
                                b: Detected gene number of viral signals and BGCs classified by their category. In VirSorter, phages matching the prophage database (VirSorter categories 1 and 2: viruses with ‘indeterminate integration status’; 4 and 5: integrated prophage) were included. In antiSMASH, BGCs of detected numbers <10 were classified as “others.”
                               </p>
                               <p id="p-24">
                                c: Distributions of viral signals and BGCs by SAGs colored by sampling sites.
                               </p>
                               <p id="p-25">
                                d: Completeness, taxonomical classification, and the number of viral signals and BGCs were summarized by SAGs in S3: mangrove soil. High- and mediumquality SAGs were included. In the result of BGCs, clusters assigned as “cf_putative” were not counted.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-26">
                              Viral signals and biosynthetic gene clusters (BGCs) were detected from various kinds of bacteria in each sampling site, suggesting that these genes are widely spread in the area surrounding the Red Sea (
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Fig. 4b, c
                              </a>
                              ) (Supplementary Table 6, 7). In contrast, most plasmid sequences were distributed in Enterobacteriaceae (Supplementary Table 8), which may be due to the bias of the existing database
                              <sup>
                               <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">
                                6
                               </a>
                              </sup>
                              . In terms of viral sequences, 385 sequences were detected in total, with about 34.3% (252/734) of SAGs carrying at least one viral sequence. In soil samples, 77% (17/22) of SAGs carried at least one viral sequence in beach (S1), followed by desert (S2) (53%) and mangrove (S3) (44%). In contrast, seawaters (W1 and W2) exhibited lower rates (20 and 13%, respectively). When we focused on viral signals per single cell, a SAG from mangrove (S3) assigned as Cyanobacteria bacterium UBA9579 carried 8 viral sequences while the median number of viral signals in single cells was 1.0 (
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               Fig. 4c
                              </a>
                              ). On the other hand, 3,676 BGCs were detected in total, with about 94.6% (694/734) of SAGs carrying at least one BGC. Saccharide was dominant (41.5%), followed by bacteriocin-related BGCs and fatty acid-related BGCs (
                              <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                               Fig. 4b
                              </a>
                              ). While the median number of BGCs in single cells was 5.0, two SAGs assigned as Acidobacteriota bacterium UBA890 and Oligoflexales bacterium from sea sediment (S4) had 21 BGCs (
                              <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                               Fig. 4c
                              </a>
                              ). In Acidobacteriota bacterium UBA890, 8 bacteriocins were detected. In Oligoflexales bacterium, 7 saccharides were detected (Supplementary Fig. 5). SAG-gel links the taxonomy and metabolic functions at single-cell resolution, thus enabling to search for the target genes or target hosts from complex microbiome on demand, which is difficult in metagenomic binning approaches (
                              <a id="xref-fig-4-6" class="xref-fig" href="#F4">
                               Fig. 4d
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Whole-genome comparative analysis of bacteria within species identical in 16S rRNA gene
                             </h3>
                             <p id="p-27">
                              From seawater (W1), we found 28 SAGs shared 99.9% identities over 1,457 bp of 16S rRNA gene and assessed the average nucleotide identity (ANI) and ortholog presence/absence within them. They were assigned as Rhodobacter spp., which are a commonly found genus in freshwater or marine environments and are known to have a wide range of metabolic capabilities
                              <sup>
                               <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                                25
                               </a>
                              </sup>
                              . ANI suggested that 28 SAGs fall into several clusters, with 2 large clusters (Rhodobacter spp. RS1 and RS2) (Supplementary Fig. 6a). The intra-cluster ANI was 95.3 (RS1) and 96.3 (RS2) while the inter-cluster ANI was 91.1. We then combined each SAG
                              <sup>
                               <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                                26
                               </a>
                              </sup>
                              and two draft genomes of RS1 and RS2 were acquired that had not been described in the previously reported catalogue of microbial draft genomes from the Red Sea
                              <sup>
                               <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                                27
                               </a>
                              </sup>
                              . The completeness and contamination of RS1 and RS2 were 80.16% and 2.19%, and 91.39% and 4.33%, respectively. Both are > 2.8 Mb in genome size with > 2,900 coding DNA sequences (CDSs). The sequence identity of 5S rRNA (109 bp) and 23S rRNA (2,558 bp) was 100% and 99.8%, respectively. While 143 draft genomes of 20 species including uncultured Rhodobacter have been registered in the NCBI taxonomy database, RS1 and RS2 most closely related to Rhodobacteraceae bacterium HIMB11 isolated from coastal seawater in Kaneohe Bay
                              <sup>
                               <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                                28
                               </a>
                              </sup>
                              , exhibiting > 99.6% identity with 5S, 16S, and 23S rRNA sequences. Although the genome sizes and CDS (3.1 Mbp and 3,183 in HIMB11) are comparable to each other, ANI over the whole genome was 91.7 (RS1-RS2), 91.4 (RS1-HIMB11), and 92.4 (RS2-HIMB11), respectively. When we compared the average amino acid identity (AAI) of the translated CDSs, the score was 91.4% (RS1-RS2).
                             </p>
                             <p id="p-28">
                              Rhodobacter spp. RS1 and RS2 diverged genetically and showed different alignments in the entire genomic sequences while retaining the high sequence identity of the 16S rRNA gene (Supplementary Fig. 6a, b). We compared the sequences of 13 draft genomes including 6 from RS1, 6 from RS2, and HIMB11, which suggested that each strain shared the unique gene sets (
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Fig. 5
                              </a>
                              ). In total, 34,423 genes classified into 6,274 clusters were defined and the genes belonging to only one SAG constituted 6.9% (2,372/34,423), 92.8% (2,202/2,372) of which were unannotated. KEGG pathway analysis
                              <sup>
                               <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                                29
                               </a>
                              </sup>
                              with HIMB11 revealed that the module completion ratio (MCR) of glucose/mannose transport system and fructose transport system was 25% in RS1 and RS2, while the MCR of phosphonate transport system, iron complex transport system, zinc transport system, putative zinc/manganese transport system, and lipopolysaccharide export system was 100% only in RS1 and RS2 (Supplementary Table 9). SAG-gel clarified the incongruence of the 16S rRNA gene and ANI within some Rhodobacter spp. and identified the different functional modules. This single-cell resolution genome analysis can shed light on these taxonomic and metabolic incongruences obscured in uncultured bacteria and provide highly-resolved microbial diversities
                              <sup>
                               <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                                30
                               </a>
                              </sup>
                              .
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F5.large.jpg?width=800&height=600&carousel=1" title="Comparative genome analysis of Rhodobacter spp. Comparative genome analysis of 6 RS1 SAGs and 6 RS2 SAGs. HIMB11, the reference genome, is the genome of known Rhodobacter sp. containing > 99% identical 16S rRNA gene. The figure shows the presence-absence of 6,274 gene clusters in the pangenome of 12 SAGs and the reference genome. Inter-strain average nucleotide identity showed < 94%, and RS1-specific or RS2-specific gene clusters were detected." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-322709305" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Comparative genome analysis of <em>Rhodobacter</em> spp.</span> Comparative genome analysis of 6 RS1 SAGs and 6 RS2 SAGs. HIMB11, the reference genome, is the genome of known <em>Rhodobacter</em> sp. containing > 99% identical 16S rRNA gene. The figure shows the presence-absence of 6,274 gene clusters in the pangenome of 12 SAGs and the reference genome. Inter-strain average nucleotide identity showed < 94%, and RS1-specific or RS2-specific gene clusters were detected.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F5.medium.gif" width="440" height="429"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F5.medium.gif" width="440" height="429"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/11/2020.03.05.962001/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5.
                               </span>
                               <span class="caption-title">
                                Comparative genome analysis of Rhodobacter spp.
                               </span>
                               <p id="p-29" class="first-child">
                                Comparative genome analysis of 6 RS1 SAGs and 6 RS2 SAGs. HIMB11, the reference genome, is the genome of known Rhodobacter sp. containing > 99% identical 16S rRNA gene. The figure shows the presence-absence of 6,274 gene clusters in the pangenome of 12 SAGs and the reference genome. Inter-strain average nucleotide identity showed < 94%, and RS1-specific or RS2-specific gene clusters were detected.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-8">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-30">
                             Unlike metagenomic binning or the conventional single-cell genomics, SAG-gel associated single-cell genomics efficiently provides draft genomes containing fulllength 16S rRNA genes from the complex microbial communities. In SAG-gel platform, the SAG-containing beads can be stored for months at 4°C and the platebased SAG library can be frozen for longer storage, expanding the possibility to conduct scaled-up sequencing in various sequencing machine on demand. Although the randomized single-cell collection manner is feasible to accumulate the genetic information according to the real microbial abundance in the environment, we would be able to enrich specific microbial fraction with the sample preprocessing such as cell size fraction and the specific cell enrichment based on phylogenetic or functional defining combining with fluorescent markers with FACS. In addition, prior to deep sequencing, the SAG library can be screened for specific genes or taxa based on marker genes at single-cell resolution, which will be useful for identifying pathogenic bacteria and metabolic producers by detecting viral signals, BGCs, and plasmids even when the target cell is underrepresented.
                            </p>
                            <p id="p-31">
                             We demonstrated SAG-gel can be applied to diverse environmental samples including soil and seawater without any modification of protocols because of its versatility. From the first screening of prokaryotic amplicons from SAG reactions, about half of SAGs were successfully obtained while showing the presence of the 16S rRNA gene. The yield ratio for uncontaminated SAGs was 79% (734/929) which includes rare bacteria in 16S rRNA gene sequencing. By using FACS for fluorescence-positive beads isolation, 84% of SAGs were assigned as draft genomes with > 0% completeness and < 10% contamination. FACS-based beads isolation is feasible for the selection of SAGs from the pool of amplified DNA containing beads in high throughput and contamination-less manner (Supplementary Table 2). The combination of the randomized SAG-gel pool and FACS-based beads isolation enables us to obtain bias-free genomic population data from individual cells and provide an overview of the environment-specific genetic potential like metagenomic population analysis (
                             <a id="xref-fig-4-7" class="xref-fig" href="#F4">
                              Fig. 4
                             </a>
                             ). It also revealed that there are a variety of microbial species in the environment that have not been described yet (
                             <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                              Fig. 3
                             </a>
                             ). The newly obtained SAGs were sufficiently detailed to be described and be compared their gene characteristics at the single-cell resolution (Supplementary Fig. 5).
                            </p>
                            <p id="p-32">
                             The current metagenomic approach is powerful to estimate the environmentspecific gene contents; furthermore, the metagenomic binning tools enable the production of draft genomes based on nucleotide composition and coverage depth. However, when multiple species or strains belonging to the same genera are present at similar abundances and have similar nucleotide compositions, the metagenomic binning approach often fails to separate contigs from these taxa into the correct genome bins
                             <sup>
                              <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                             </sup>
                             . In contrast, our approach generates individual SAGs for each species, which can be clustered by pairwise determination of ANI or taxonomic marker identities. As shown in the comparative genome analysis of uncultured Rhodobacter spp., massive SAG production reveals the intra-species diversity in individual environments. We found that the co-existence of multiple Rhodobacter strains in the same microbial community and they presumably have different functional modules while retaining highly conserved 16S rRNA gene sequences (Fig. 6 and Supplementary Fig. 6). We should pay close attention to the existence of these taxonomically identical but functionally different microbes, especially in the sequencing of geographically separated environmental samples and symbiotic or commensal bacteria from different hosts.
                            </p>
                            <p id="p-33">
                             Our results reveal that the single-cell draft genomes work as de novo-assembled sequences as well as the genomes of isolates. However, unlike the genomes of isolates, challenges in genome assembly are not fully resolved using SAG-gel platform alone, as WGA process generates numerous chimeric reads
                             <sup>
                              <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                               32
                              </a>
                             </sup>
                             , hampering the assembly of long contiguous sequences and randomly biased amplified genomes
                             <sup>
                              <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                             </sup>
                             , and causing incomplete genome coverage. By combining our methods with other tools including cleaning and co-assembly of SAGs from the same species, the number of chimeric contigs and genome incompleteness could be significantly reduced
                             <sup>
                              <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">
                               26
                              </a>
                              ,
                              <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                               33
                              </a>
                             </sup>
                             . Draft genomes acquired by SAG-gel can also be combined with other sequence data sets including shotgun metagenome sequencing and long-read sequencing, all of which will promote genome finishing
                             <sup>
                              <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                             </sup>
                             . Moreover, we recognize the shortage of sequenced SAGs to cover most microbial species in complex microbial communities. In our estimation, we could pool samples as 384- to 768-plex SAG libraries for sequencing in a single lane on the Illumina HiSeq platform while acquiring enough sequencing depth for de novo assembly.
                            </p>
                            <p id="p-34">
                             The ability of SAG-gel to generate qualified draft genomes in a high-throughput manner will contribute to increases the comprehensiveness of reference genomes without the need for laborious isolation and cultivation. While we focused on the environmental microbiome, SAG-gel platform is also applicable to other types of cells including eukaryotic cells. We expect that SAG-gel platform will contribute to disclose the diversity and heterogeneity of biological communities.
                            </p>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             Methods
                            </h2>
                            <div id="sec-10" class="subsection">
                             <h3>
                              Fabrication of microfluidic device
                             </h3>
                             <p id="p-35">
                              A microfluidic device for generating monodispersed picoliter-sized droplets was fabricated according to a previously-reported method
                              <sup>
                               <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                                35
                               </a>
                              </sup>
                              . In short, a flow-focusing microfluidic device was designed using AutoCAD (AutoDesk, Sausalito, CA, USA) and fabricated using conventional soft-lithography techniques. A photomask pattern was transferred to a layer of negative photoresist (SU8-3050; Microchem, Newton, MA, USA) coating a glass wafer (40 mm × 47 mm). All microchannels were 50 μm tall and 100 μm wide, except at the cross-junction area. The cross-junction was designed to be 17 μm wide for the aqueous phase and 10 μm wide for the continuous oil phase. Polydimethylsiloxane (PDMS; Sylgard 184; Dow Corning Corp., Midland, MI) and its cross-linker were mixed thoroughly at a ratio of 10:1 (w/w) then degassed. The PDMS mixture was poured over the master mold and cured for at least 1 h at 60 °C. After curing, the slabs were punched with a 0.75-mm biopsy punch (World Precision Instruments, Sarasota, FL, USA) and PDMS slabs and glass slides were bonded by plasma treatment (Plasma Cleaner PDG-32G; Harrick Scientific, Ossining, NY, USA) followed by baking for at least 30 min at 60 °C. Finally, the microchannel was filled with Aquapel solution (PPG Industries, Pittsburgh, PA, USA) to produce a hydrophobic surface coating; thereafter, excess Aquapel was blown off with air.
                             </p>
                            </div>
                            <div id="sec-11" class="subsection">
                             <h3>
                              Preparation of model bacteria suspensions
                             </h3>
                             <p id="p-36">
                              For genome sequencing analysis, E. coli K-12 strain (ATCC 10798; genome size: 4.6 Mbp) and B. subtilis (ATCC 6633; genome size: 4.0 Mbp) were used as model bacteria. E. coli K-12 cells were pre-cultured in Luria-Bertani (LB) medium (1.0% Bacto tryptone [BD Biosciences, Franklin Lakes, NJ, USA], 0.5% yeast extract [BD Biosciences], 1.0% NaCl [Sigma-Aldrich, Hamburg, Germany], pH 7.0) for 16 h. B. subtilis cells were pre-cultured in brain heart infusion broth (ATCC medium 44; Thermo Fisher Scientific, Waltham, MA, USA.) for 16 h. For cell collection, 1 mL of cultured medium was dispensed into a 1.5 mL tube and centrifuged at 8000 × g for 5 min. After removing the supernatant, the collected cells were resuspended in UV-treated Dulbecco’s phosphate-buffered saline (-) (DPBS, Thermo Fisher Scientific) and washed three times with DPBS. Finally, the cells were resuspended in 500 μL DPBS and cell concentration was calculated with a bacteria counter. To evaluate cross-contamination, E. coli and B. subtilis cells were mixed at a ratio of 1:1. For single-cell encapsulation, the cell concentrations of E. coli and B. subtilis were adjusted to 3.0 × 10
                              <sup>
                               3
                              </sup>
                              cells/μL at a concentration of 0.1 cell/droplet. All preparations for the cell suspension and further processes were performed under an open-interior clean bench (KOACH T 500-F; KOKEN LTD., Tokyo, Japan) except for droplet generation and isolation with FACS.
                             </p>
                            </div>
                            <div id="sec-12" class="subsection">
                             <h3>
                              Environmental sample preparation
                             </h3>
                             <p id="p-37">
                              Eight samples from seven different sampling sites were collected for this study. Environmental samples consisted of six soil and sediment samples [beach soil: S1(22°17’35.6”N 39°05’26.0”E); desert soil: S2(22°19’03.0”N 39°08’36.7”E); mangrove soil: S3(22°18’53.3”N 39°05’29.2”E); fresh sea sediment: S4(22° 17.988’N, 39° 03.427 ’E); frozen sea sediment: S5(22° 17.988’N, 39° 03.427 ’E); and seashore soil: S6(22°17’17.5”N 39°05’42.3”E)] and two seawater samples [harbor seawater: W1(22°18’16.9”N 39°06’12.3”E) and open ocean seawater: W2(22° 17.988’N, 39° 03.427 ’E)]. For soil samples, 20–30 g of soil was collected 10 cm beneath the top layer and maintained in 50 mL tubes on ice until preparation of the cell suspension. For sediment samples, 20–30 g of soil was collected from the seafloor at the depth of 25 m with van Veen Grab sampler and kept into 50 mL tubes. For the S5 sample, sea sediment was collected on 30th April 2017, frozen by dry ice, and directly kept on −80°C for 14 months. It was thawed on ice immediately before the preparation of cell suspensions. S4 was also kept on ice until the preparation of cell suspensions. W1 and W2 was collected from the surface. From each sampling site, 30 L of seawater was collected into plastic tanks. Cell suspensions were prepared as soon as possible after arrival at the laboratory. All samples except for S5 were collected from the 24th to the 27th of June, 2018 and proceeded to WGA.
                             </p>
                             <p id="p-38">
                              Ten grams each of samples S1 to S6 was dispensed into three 50 mL tubes (Iwaki Science Products Department, Iwaki Glass Co. Ltd., Chiba, Japan). Then DPBS was added to each tube up to the 40-mL volume marker and the contents were mixed thoroughly. The suspended solution was kept on ice for 5 min and the supernatant was collected into another 50 mL tube. The supernatant was filtered with a 5-ψm MF-Millipore membrane filter (Merck Millipore, Milan, Italy). The flow-through was centrifuged at 10,000 ×g for 5 min with a benchtop centrifuge (Thermo Fisher Scientific). After removing the supernatant, the pellet was resuspended in 10 mL of DPBS and dispensed into a 1.5 mL tube (Axygen Biosciences, Hangzhou, China). Each suspension was centrifuged at 10,000 ×g for 5 min with a tabletop ultracentrifuge and washed three times with DPBS.
                             </p>
                             <p id="p-39">
                              For the seawater samples, 4 L of seawater was filtered with a 5-ψm MF-Millipore membrane filter. The flow-through was collected and filtered with a 0.22-μm filter (Merck Millipore). Then the 0.22-μm filter was suspended in 10 mL of DPBS and vortexed thoroughly to suspend trapped bacterial fractions; 10 mL of bacterial suspensions was dispensed into 1.5 mL tubes, centrifuged at 10,000 ×g for 5 min with a tabletop ultracentrifuge, and washed three times with DPBS.
                             </p>
                             <p id="p-40">
                              We prepared two tubes of cell suspensions for each sample. One tube was used for single-cell genome sequencing and the other for 16S rRNA gene sequencing. For single-cell genome sequencing, the cell concentration was adjusted to 3.0 × 10
                              <sup>
                               3
                              </sup>
                              cells/μL for encapsulating single cells at a concentration of 0.1 cell/droplet (40 μm diameter). For 16S rRNA gene sequencing, total metagenomic DNA was extracted with PowerLyzer Soil DNA Extraction kit (QIAGEN) from the cell suspensions. Then 16S rRNA gene sequencing libraries were prepared according to Illumina’s protocol and run on an Illumina MiSeq for 300 cycles of paired-end sequencing using a MiSeq v3 600-cycle reagent kit (Illumina, San Diego, CA, USA). For S1, the amount of DNA extracted was insufficient to prepare the sequencing library.
                             </p>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              Single-cell encapsulation into agarose gel beads
                             </h3>
                             <p id="p-41">
                              Ultra-low gelling temperature agarose A5030 (Sigma-Aldrich) was mixed into DPBS and incubated at 85 °C for 30 min. By mixing the cell suspensions with agarose solutions, 1.5% agarose cell suspensions with 3.0 × 10
                              <sup>
                               3
                              </sup>
                              cells/μL were prepared. Agarose cell suspensions were loaded into PTFE tubing (AWG 24) connected to a Mitos P-pump (Dolomite, Charleston, MA, USA). By controlling the air pressure, agarose cell suspensions and 2% Pico-Surf™ 1 in Novec™ 7500 (Dolomite) as carrier oil were introduced into a manufacturer-fabricated microfluidic device. Microfluidic droplets with a diameter of 40 μm (volume: 34 pL) were generated for encapsulation of > 100,000 single cells within 30 minutes (35,000 droplets/min) at a concentration of 0.1 cell/droplet.
                             </p>
                             <p id="p-42">
                              Droplets were collected in 1.5 mL tubes via PTFE tubing from the outlet and incubated on ice for 15 min. After droplet solidification, the collected droplets were broken by 1H,1H,2H,2H-perfluoro-1-octanol (Sigma-Aldrich). Next, 500 μL of acetone (Sigma-Aldrich) was added to the 1.5 mL tube and vortexed thoroughly. Gel beads descended to the bottom of the 1.5 mL tube because of their greater specific gravity, which enables collection of gel beads by centrifugation. After collecting the gel beads with a tabletop centrifuge, the supernatant was removed. Then 500 μL of isopropanol (Sigma-Aldrich) was added to the 1.5 mL tube and vortexed thoroughly. After gel-bead collection with a tabletop centrifuge, the supernatant was removed. Finally, 500 μL of DPBS was added to the 1.5 mL tube and vortexed thoroughly. The gel beads were washed with DPBS three times. During these steps, the gel beads were transitioned from an oil phase to an aqueous phase.
                             </p>
                            </div>
                            <div id="sec-14" class="subsection">
                             <h3>
                              Cell lysis and WGA in agarose gel beads
                             </h3>
                             <p id="p-43">
                              After the gel beads were collected with a tabletop centrifuge and supernatant was removed, they were treated with two different lysis protocols: Alkaline treatment and Enzyme cocktail treatment.
                             </p>
                             <div id="sec-15" class="subsection">
                              <h4>
                               1) Alkaline treatment and WGA
                              </h4>
                              <p id="p-44">
                               The following steps were conducted in 0.2 mL tubes (Axygen Biosciences): 3 μL of D2 buffer from a REPLI-g Single Cell Kit (Thermo Fisher Scientific) was added to 4 μL of droplet suspension. After incubation at 40 °C for 10 min, 40 μL of WGAm mixture (3 μL of Stop Solution, 9 μL of H
                               <sub>
                                2
                               </sub>
                               O, 29 μL of reaction buffer, and 2 μL of DNA polymerase) was added and incubated at 30 °C for 180 min.
                              </p>
                             </div>
                             <div id="sec-16" class="subsection">
                              <h4>
                               2) Enzyme cocktail treatment and WGA
                              </h4>
                              <p id="p-45">
                               Two hundred microliters of lysozyme solution (50 U/μL Ready-lyse lysozyme [Lucigen, WI, USA], 2 U/mL zymolyase [Zymo Research, Orange, CA, USA], 22 U/mL lysostaphin [Sigma-Aldrich], and 250 U/mL mutanolysin [Sigma-Aldrich] in DPBS) was added and incubated at 37 °C overnight. Gel beads were washed with DPBS three times and 200 μL of achromopeptidase solution (0.5 mg/mL achromopeptidase [Wako, Tokyo, Japan] in DPBS) was added and incubated at 37 °C for 8 h. Then the gel beads were washed with DPBS three times and 200 μL of proteinase K solution (1 mg/mL proteinase K [Promega, Madison, WI] and 0.5% SDS [Wako, Tokyo, Japan] in DPBS) was added and incubated at 40 °C overnight. After washing the gel beads with DPBS five times, the supernatant was removed. Then the gel beads proceeded to the alkaline treatment and WGA described above.
                              </p>
                              <p id="p-46">
                               For SAG-gel with model bacteria, both two lysis protocols were performed for evaluation, while enzyme cocktail treatment was performed for environmental samples.
                              </p>
                             </div>
                            </div>
                            <div id="sec-17" class="subsection">
                             <h3>
                              Confirmation of DNA amplification
                             </h3>
                             <p id="p-47">
                              After WGA, the gel beads were collected with a tabletop centrifuge and the supernatant was removed. The gel beads were resuspended in DPBS and washed three times. Then the gel beads were stained with 1× SYBR Green (Thermo Fisher Scientific) in DPBS and transferred onto a glass slide for microscopic observation. After confirmation of DNA amplification, fluorescence-positive gel beads were isolated by FACS or manual picking. For the model bacteria experiment, gel-bead isolation was conducted by FACS. For the environmental sample experiment, gel beads from S1, S2, S5, S6, and W1 were subjected to manual picking within 2 days after WGA. The remaining samples were maintained in DPBS and isolated by FACS within 1 week. To compare the isolation methods, both manual picking and FACS-based isolation were conducted for S6 and W1. From 8 sampling sites, 48–480 fluorescence-positive gel beads were isolated and subjected to 2nd-round WGA.
                             </p>
                             <div id="sec-18" class="subsection">
                              <h4>
                               1) Single-droplet sorting with manual picking
                              </h4>
                              <p id="p-48">
                               SYBR-stained gel beads were transferred onto a glass slide. Fluorescencepositive gel beads were manually picked using a micropipette (Drummond, Camlab, Cambridge, UK) under the open clean system (KOKEN LTD.). Then each bead was dispensed into a 96-well plate with 0.8 μL of DPBS and proceeded to a 2nd-round of WGA or maintained at −30 °C for longer storage.
                              </p>
                             </div>
                             <div id="sec-19" class="subsection">
                              <h4>
                               2) Single-droplet sorting with FACS
                              </h4>
                              <p id="p-49">
                               Flow cytometric analysis and sorting were performed with a BD FACSMelody™ Cell Sorter (BD Biosciences) equipped with a 488 nm excitation laser. The nozzle diameter was 100 μm. The sample flow rate was adjusted to approximately 30 events per second. The number of sorted beads is summarized in Supplementary Table 1a, b; 0.8 μL of DPBS was dispensed into each well prior to gel-bead sorting. After sorting, the 96-well plate proceeded to a 2nd-round of WGA or maintained at −30 °C for longer storage.
                              </p>
                             </div>
                            </div>
                            <div id="sec-20" class="subsection">
                             <h3>
                              2nd-round WGA
                             </h3>
                             <p id="p-50">
                              2nd-round WGA was performed with the REPLI-g Single Cell Kit (Thermo Fisher Scientific); 0.6 μL of D2 buffer was added to each well and incubated at 65 °C for 10 min. Then 8.6 μL of WGA mixture (0.6 μL of Stop Solution, 1.8 μL of H
                              <sub>
                               2
                              </sub>
                              O, 5.8 μL of reaction buffer, and 0.4 μL of DNA polymerase) was added and incubated at 30 °C for 120 min. The WGA reaction was terminated by heating at 65 °C for 3 min. After the 2nd-round of amplification, master library plates of single amplified genomes (SAGs) were prepared.
                             </p>
                            </div>
                            <div id="sec-21" class="subsection">
                             <h3>
                              DNA quantification and 16S rRNA gene sequence analysis
                             </h3>
                             <p id="p-51">
                              The amplicon yields of the 2nd-round WGA were quantified by a Qubit dsDNA HS assay kit (Thermo Fisher Scientific). For model bacteria, 87 samples out of 92 (94.5%) showed sufficient DNA amplification for library preparation. For environmental samples, in order to confirm amplification from a single bacterial cell, 16S rRNA gene sequencing was also performed against the 2nd-round WGA products. Primer pair sequences for the V3V4 region were used according to Illumina’s MiSeq system protocols (Forward: 5’-TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGCCTACGGGNGGCWG CAG-3’, Reverse: 5’-GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAGGACTACHVGGGTA TCTAATCC-3’). PCR amplification was confirmed with agarose electrophoresis (100 V, 15 min) and the amplicon sequences were obtained via Sanger sequencing (Fasmac, Kanagawa, JAPAN).
                             </p>
                            </div>
                            <div id="sec-22" class="subsection">
                             <h3>
                              Construction of single-cell genome libraries and whole genome sequencing
                             </h3>
                             <p id="p-52">
                              For the sequencing analysis, Illumina libraries were prepared using amplicons from the 2nd-round WGA products. We used a Nextera XT DNA sample prep kit (Illumina) according to the manufacturer’s instructions. Libraries from model bacterial samples were sequenced on an Illumina MiSeq for 75 cycles of paired- end sequencing, generating a total of 50.0 M paired-end reads and 7.34 Gbp. Libraries from environmental samples were sequenced on an Illumina HiSeq for 150 cycles of paired-end sequencing (GENEWIZ, South Plainfield, NJ, USA), generating a total of 1.19 G paired-end reads and 356 Gbp.
                             </p>
                            </div>
                            <div id="sec-23" class="subsection">
                             <h3>
                              Sequencing analysis for model bacteria
                             </h3>
                             <p id="p-53">
                              The sequence raw reads were mapped to the NCBI reference genome (NIH, Bethesda, MD, USA) with BWA
                              <sup>
                               <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                                36
                               </a>
                              </sup>
                              for evaluation of cross contamination. NC_00913 (E. coli substrain MG1655) and CP011496 (Escherichia coli strain NCM3722 plasmid F) were used as a reference for the E. coli K-12 strain. NCBI reference genome NC_014479 was used for B. subtilis subsp. spizizenii str. W23. Then the acquired reads were down-sampled to 10× mean mapping depth (46 Mbp for E. coli and 40 Mbp for B. subtilis). Sequence reads were de novo assembled with SPAdes 3.5.0
                              <sup>
                               <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                              </sup>
                              and qualified by QUAST 2.3.
                              <sup>
                               <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                                38
                               </a>
                              </sup>
                              Contigs (> 500 bp) from each sample were mapped to the reference genome with BWA. Genome coverage was calculated using SAMtools
                              <sup>
                               <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                                39
                               </a>
                              </sup>
                              .
                             </p>
                            </div>
                            <div id="sec-24" class="subsection">
                             <h3>
                              16S rRNA gene amplicon sequencing analysis of environmental samples
                             </h3>
                             <p id="p-54">
                              Meta 16S short-read information of the V3V4 region was acquired from each environmental sample by Illumina MiSeq 300 cycle paired-end sequencing. OTU clustering was conducted by USEARCH
                              <sup>
                               <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                                40
                               </a>
                              </sup>
                              from integrated pair-end reads longer than 400 bp and each OTU was annotated taxonomically by BLAST search against the Silva 132 database. OTUs were clustered by phylum level and the bar chart was drawn for the phyla accounting for > 5% (
                              <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                               Fig. 3b
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-25" class="subsection">
                             <h3>
                              Sequencing analysis for environmental samples
                             </h3>
                             <p id="p-55">
                              Sequence reads were de novo assembled with SPAdes
                              <sup>
                               <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                              </sup>
                              and contigs (> 1,000 bp) underwent the following analysis. Completeness and contamination were calculated with CheckM
                              <sup>
                               <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                                41
                               </a>
                              </sup>
                              . The number of contigs, N50, and total length was evaluated with QUAST
                              <sup>
                               <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">
                                38
                               </a>
                              </sup>
                              . The number of tRNA was examined with Prokka
                              <sup>
                               <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                                42
                               </a>
                              </sup>
                              . The draft genome quality was evaluated by the standards developed by GSC
                              <sup>
                               <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                                43
                               </a>
                              </sup>
                              . After removing samples which were classified as contamination or exhibited 0% completeness, the remaining SAGs underwent the following analysis. 16S rRNA gene sequence was extracted with Prokka and assigned to the 16S rRNA gene sequence database (Silva). The 16S rRNA genes which exhibited < 97% identity to the top hit result to the database were assigned to have no obvious reference sequences.
                             </p>
                             <p id="p-56">
                              SAGs were taxonomically classified with GTDB-Tk
                              <sup>
                               <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">
                                17
                               </a>
                              </sup>
                              and the results were visualized with iTOl
                              <sup>
                               <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                                44
                               </a>
                              </sup>
                              . Then the GTDB-Tk results were summarized at the phylum level to compare bacterial composition of SAGs and 16S rRNA gene sequencing. We evaluated the number of species using the mash distance 0.05 as the cutoff for species delineation, which equates to an ANI of ≥ 95 %
                              <sup>
                               <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                                45
                               </a>
                              </sup>
                              .
                             </p>
                             <p id="p-57">
                              In order to characterize gene distributions at each sampling site, orthologous gene groups (OGs) were detected by OrthoFinder
                              <sup>
                               <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">
                                22
                               </a>
                              </sup>
                              . The detection number of each OG was summarized for each sampling site. Based on the number of OGs, the OGs with p-values < 0.01 were extracted. For the heatmap, 25 OGs exhibiting the lowest p-values were extracted.
                             </p>
                             <p id="p-58">
                              SAGs were referenced against VirSorter
                              <sup>
                               <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                                46
                               </a>
                              </sup>
                              , antiSMASH
                              <sup>
                               <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                                47
                               </a>
                              </sup>
                              (search option: -- transatpks_da --clusterblast --subclusterblast --knownclusterblast --smcogs --inclusive --borderpredict --full-hmmer) and Plasmidfinder
                              <sup>
                               <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                                48
                               </a>
                              </sup>
                              for detecting viral signals, BGCs, and plasmids. For VirSorter, viral signals matching the VirSorter categories of 1, 2, 4, and 5 were considered and the viral signals matching the VirSorter categories of 3 and 6 were ignored. For antiSMASH, BGCs of detected numbers < 10 were classified as “others.”
                             </p>
                             <p id="p-59">
                              SAGs that exhibited >95% ANI were co-assembled with ccSAG
                              <sup>
                               <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">
                                26
                               </a>
                              </sup>
                              and the composite draft genomes and the draft genome of Rhodobacteraceae bacterium HIMB11 were evaluated with QUAST
                              <sup>
                               <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">
                                38
                               </a>
                              </sup>
                              , CheckM
                              <sup>
                               <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">
                                41
                               </a>
                              </sup>
                              , Prokka
                              <sup>
                               <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">
                                42
                               </a>
                              </sup>
                              , and FastANI
                              <sup>
                               <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">
                                45
                               </a>
                              </sup>
                              (
                              <a href="https://github.com/ParBLiSS/FastANI">
                               https://github.com/ParBLiSS/FastANI
                              </a>
                              ). Anvi’o
                              <sup>
                               <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                                49
                               </a>
                              </sup>
                              was used to compare the sequences from 13 draft genomes (6 from RS1, 6 from RS2, and HIMB11) which were chosen in order based on the value of completeness. To visualize the alignments over the whole sequence, draft genomes of RS1 and RS2 were aligned to the shotgun sequencing data of HIMB11 with AliTV
                              <sup>
                               <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                                50
                               </a>
                              </sup>
                              . Filter links by their identity were 65–100% and filter links by their length were ≧ 2,500 bp. For detailed analysis, each draft genome was searched against Genomaple-2.3.2
                              <sup>
                               <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">
                                29
                               </a>
                              </sup>
                              and module completion ratio (MCR) in each functional module was evaluated. Unless otherwise noted, the analysis was performed with default settings for each tool.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-27">
                            <h2 class="">
                             Accession number
                            </h2>
                            <p id="p-60">
                             All sequencing data generated in this study will be available in the DDBJ Sequenced Read Archive under the accession numbers PSUB004727 upon final publication.
                            </p>
                           </div>
                           <div class="section" id="sec-28">
                            <h2 class="">
                             Author Contributions
                            </h2>
                            <p id="p-62">
                             YN, MK and MH contributed equally to this work.
                            </p>
                            <p id="p-63">
                             YN, MK, MH, TG, and HT conceived and designed the experiments. YN, MK, MH, KT, and KM collected samples. HB conducted the DNA extraction. YN, MK, and KT conducted the other experiments and analyzed the results. YN, MH, MK, and HT wrote the manuscript.
                            </p>
                           </div>
                           <div class="section" id="sec-29">
                            <h2 class="">
                             Competing Interests statement
                            </h2>
                            <p id="p-64">
                             M.H. and H.T. are shareholders in bitBiome, to which the patents pertaining to SAG-gel workflow were transferred.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements (optional)
                            </h2>
                            <p id="p-61">
                             This work was supported by JST-PRESTO grant number JPMJPR15FA, MEXT KAKENHI grant numbers 18H01801 and 17H06158, and the funding from King Abdullah University of Science and Technology (KAUST), under award numbers URF/1/1976/03/01, URF/1/1976/17/01, URF/1/1976/20/01 and FCS/1/3326/01/01. The super-computing resource was provided by the Human Genome Center (University of Tokyo).
                            </p>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.1" data-doi="10.1038/s41586-019-1058-x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nayfach
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Seshadri
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pollard
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kyrpides
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.C.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  New insights from uncultivated genomes of the global human gut microbiome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  568
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  505
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  510
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D568%26rft.spage%253D505%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-019-1058-x%26rft_id%253Dinfo%253Apmid%252F30867587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-019-1058-x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30867587&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.2" data-doi="10.1038/nature12352">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rinke
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Insights into the phylogeny and coding potential of microbial dark matter
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  499
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  431
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  437
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D499%26rft.spage%253D431%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12352%26rft_id%253Dinfo%253Apmid%252F23851394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature12352&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23851394&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000322157900031&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.3" data-doi="10.1038/nmeth.3103">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alneberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Binning metagenomic contigs by coverage and composition
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Methods
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1144
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1146
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D11%26rft.spage%253D1144%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3103%26rft_id%253Dinfo%253Apmid%252F25218180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nmeth.3103&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25218180&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000344580600021&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.4" data-doi="10.1038/s41467-018-03317-6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stewart
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.D.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Assembly of 913 microbial genomes from metagenomic sequencing of the cow rumen
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Commun
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  870
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D9%26rft.spage%253D870%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-03317-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41467-018-03317-6&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Parks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.H.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Recovery of nearly 8,000 metagenome-assembled genomes substantially expands the tree of life
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Microbiol
                                 </abbr>
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1533
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1542
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BMicrobiol%26rft.volume%253D2%26rft.spage%253D1533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.6" data-doi="10.1038/s41396-019-0446-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stalder
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Press
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.O.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sullivan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liachko
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Top
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Linking the resistome and plasmidome to the microbiome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  ISME J
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2437
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2446
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DISME%2BJ%26rft.volume%253D13%26rft.spage%253D2437%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41396-019-0446-4%26rft_id%253Dinfo%253Apmid%252F31147603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41396-019-0446-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=31147603&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.7" data-doi="10.1007/s13238-018-0544-5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Single-cell metagenomics: challenges and applications
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Protein Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  501
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  510
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProtein%2BCell%26rft.volume%253D9%26rft.spage%253D501%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13238-018-0544-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s13238-018-0544-5&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.8" data-doi="10.1111/1574-6976.12015">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blainey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.C.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The future is now: single-cell genomics of bacteria and archaea
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  FEMS Microbiol Rev
                                 </abbr>
                                 <span class="cit-vol">
                                  37
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  407
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  427
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol%2BRev%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1574-6976.12015%26rft_id%253Dinfo%253Apmid%252F23298390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/1574-6976.12015&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23298390&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000317364700007&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.9" data-doi="10.1371/journal.pgen.0030155">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marcy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Nanoliter reactors improve multiple displacement amplification of genomes from single cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Genet
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1702
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1708
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMarcy%26rft.auinit1%253DY.%26rft.volume%253D3%26rft.issue%253D9%26rft.spage%253D1702%26rft.epage%253D1708%26rft.atitle%253DNanoliter%2Breactors%2Bimprove%2Bmultiple%2Bdisplacement%2Bamplification%2Bof%2Bgenomes%2Bfrom%2Bsingle%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030155%26rft_id%253Dinfo%253Apmid%252F17892324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030155&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17892324&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000249767800013&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.10" data-doi="10.1038/nbt.2720">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gole
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Massively parallel polymerase cloning and genome sequencing of single cells using nanoliter microwells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Biotechnol
                                 </abbr>
                                 <span class="cit-vol">
                                  31
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1126
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1132
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BBiotechnol%26rft.volume%253D31%26rft.spage%253D1126%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2720%26rft_id%253Dinfo%253Apmid%252F24213699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nbt.2720&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24213699&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.11" data-doi="10.1371/journal.pone.0138733">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nishikawa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Monodisperse Picoliter Droplets for Low-Bias and Contamination-Free Reactions in Single-Cell Whole Genome Amplification
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS One
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e0138733
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0138733%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0138733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0138733&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.05.962001v2.12" data-doi="10.1038/s41598-017-05436-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hosokawa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nishikawa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kogawa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Takeyama
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Massively parallel whole genome amplification for single-cell sequencing using droplet microfluidics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci Rep
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  5199
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D7%26rft.spage%253D5199%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-05436-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41598-017-05436-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.13" data-doi="10.1038/nbt.3880">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Demaree
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ahmed
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Abate
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Single-cell genome sequencing at ultra-high-throughput with microfluidic droplet barcoding
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Biotechnol
                                 </abbr>
                                 <span class="cit-vol">
                                  35
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  640
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  646
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BBiotechnol%26rft.volume%253D35%26rft.spage%253D640%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nbt.3880&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pasolli
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Extensive Unexplored Human Microbiome Diversity Revealed by Over 150,000 Genomes from Metagenomes Spanning Age, Geography, and Lifestyle
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  176
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  649
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  662
                                 </span>
                                 <span class="cit-page-range">
                                  e620
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D176%26rft.spage%253D649%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.15" data-doi="10.1038/ismej.2011.84">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Martinez-Garcia
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  High-throughput single-cell sequencing identifies photoheterotrophs and chemoautotrophs in freshwater bacterioplankton
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  ISME J
                                 </abbr>
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  113
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  123
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DISME%2BJ%26rft.aulast%253DMartinez-Garcia%26rft.auinit1%253DM.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D113%26rft.epage%253D123%26rft.atitle%253DHigh-throughput%2Bsingle-cell%2Bsequencing%2Bidentifies%2Bphotoheterotrophs%2Band%2Bchemoautotrophs%2Bin%2Bfreshwater%2Bbacterioplankton.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fismej.2011.84%26rft_id%253Dinfo%253Apmid%252F21716306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ismej.2011.84&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21716306&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000299052500011&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.16" data-doi="10.1038/nmeth.3955">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brito
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alm
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blainey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.C.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Virtual microfluidics for digital quantification and single-cell sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Methods
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  759
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  762
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D13%26rft.spage%253D759%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3955%26rft_id%253Dinfo%253Apmid%252F27479330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nmeth.3955&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27479330&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.17" data-doi="10.1038/nbt.4229">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Parks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.H.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  A standardized bacterial taxonomy based on genome phylogeny substantially revises the tree of life
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Biotechnol
                                 </abbr>
                                 <span class="cit-vol">
                                  36
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  996
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1004
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BBiotechnol%26rft.volume%253D36%26rft.spage%253D996%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.4229%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nbt.4229&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.18" data-doi="10.1186/s13059-016-0997-x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ondov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.D.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Mash: fast genome and metagenome distance estimation using MinHash
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  132
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D17%26rft.spage%253D132%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-0997-x%26rft_id%253Dinfo%253Apmid%252F27323842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-016-0997-x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27323842&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.19" data-doi="10.1111/j.1462-2920.2010.02339.x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sutcliffe
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.C.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Cell envelope architecture in the Chloroflexi: a shifting frontline in a phylogenetic turf war
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Environ Microbiol
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  279
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  282
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.stitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.aulast%253DSutcliffe%26rft.auinit1%253DI.%2BC.%26rft.volume%253D13%26rft.issue%253D2%26rft.spage%253D279%26rft.epage%253D282%26rft.atitle%253DCell%2Benvelope%2Barchitecture%2Bin%2Bthe%2BChloroflexi%253A%2Ba%2Bshifting%2Bfrontline%2Bin%2Ba%2Bphylogenetic%2Bturf%2Bwar.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1462-2920.2010.02339.x%26rft_id%253Dinfo%253Apmid%252F20860732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/j.1462-2920.2010.02339.x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20860732&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.20" data-doi="10.1016/j.gene.2016.09.021">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Al-Amoudi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Metagenomics as a preliminary screen for antimicrobial bioprospecting
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Gene
                                 </abbr>
                                 <span class="cit-vol">
                                  594
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  248
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  258
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D594%26rft.spage%253D248%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2016.09.021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.gene.2016.09.021&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.21" data-doi="10.1128/AEM.01451-14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kennedy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hall
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lynch
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moreno-Hagelsieb
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Neufeld
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.D.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Evaluating bias of illumina-based bacterial 16S rRNA gene profiles
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Appl Environ Microbiol
                                 </abbr>
                                 <span class="cit-vol">
                                  80
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  5717
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  5722
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAppl%2BEnviron%2BMicrobiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.01451-14%26rft_id%253Dinfo%253Apmid%252F25002428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI4MC8xOC81NzE3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDUvMTEvMjAyMC4wMy4wNS45NjIwMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.22" data-doi="10.1186/s13059-015-0721-2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Emms
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kelly
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  157
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D16%26rft.spage%253D157%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0721-2%26rft_id%253Dinfo%253Apmid%252F26243257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-015-0721-2&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26243257&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.23" data-doi="10.1038/ismej.2016.99">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thompson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.R.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Metagenomic covariation along densely sampled environmental gradients in the Red Sea
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  ISME J
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  138
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  151
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DISME%2BJ%26rft.volume%253D11%26rft.spage%253D138%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fismej.2016.99%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ismej.2016.99&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.24" data-doi="10.1016/j.gene.2015.10.032">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alzubaidy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Rhizosphere microbiome metagenomics of gray mangroves (Avicennia marina) in the Red Sea
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Gene
                                 </abbr>
                                 <span class="cit-vol">
                                  576
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  626
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  636
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D576%26rft.spage%253D626%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2015.10.032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.gene.2015.10.032&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               25.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.25">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Imam
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Noguera
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.R.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Donohue
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Global insights into energetic and metabolic networks in Rhodobacter sphaeroides
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bmc Systems Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               26.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kogawa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hosokawa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nishikawa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mori
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Takeyama
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Obtaining high-quality draft genomes from uncultured microbes by cleaning and co-assembly of single-cell amplified genomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci Rep
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2059
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D8%26rft.spage%253D2059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               27.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.27">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Haroon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thompson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Parks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hugenholtz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stingl
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   U.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  A catalogue of 136 microbial draft genomes from Red Sea metagenomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci Data
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  160050
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSci%2BData%26rft.volume%253D3%26rft.spage%253D160050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               28.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.28" data-doi="10.4056/sigs.4998989">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Durham
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.P.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Draft genome sequence of marine alphaproteobacterial strain HIMB11, the first cultivated representative of a unique lineage within the Roseobacter clade possessing an unusually small genome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Stand Genomic Sci
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  632
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  645
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DStand%2BGenomic%2BSci%26rft.volume%253D9%26rft.spage%253D632%26rft_id%253Dinfo%253Adoi%252F10.4056%252Fsigs.4998989%26rft_id%253Dinfo%253Apmid%252F25197450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.4056/sigs.4998989&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25197450&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               29.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.29" data-doi="10.1080/09168451.2018.1476122">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Arai
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  MAPLE 2.3.0: an improved system for evaluating the functionomes of genomes and metagenomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biosci Biotechnol Biochem
                                 </abbr>
                                 <span class="cit-vol">
                                  82
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1515
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1517
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiosci%2BBiotechnol%2BBiochem%26rft.volume%253D82%26rft.spage%253D1515%26rft_id%253Dinfo%253Adoi%252F10.1080%252F09168451.2018.1476122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1080/09168451.2018.1476122&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               30.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chevrette
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.G.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Taxonomic and Metabolic Incongruence in the Ancient Genus Streptomyces
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Front Microbiol
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2170
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFront%2BMicrobiol%26rft.volume%253D10%26rft.spage%253D2170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               31.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.31">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shaiber
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eren
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Composite Metagenome-Assembled Genomes Reduce the Quality of Public Genome Repositories
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mbio
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               32.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.32" data-doi="10.1186/1472-6750-7-19">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lasken
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stockwell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.B.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Mechanism of chimera formation during the Multiple Displacement Amplification reaction
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Biotechnol
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  19
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2Bbiotechnology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBiotechnol%26rft.aulast%253DLasken%26rft.auinit1%253DR.%2BS.%26rft.volume%253D7%26rft.spage%253D19%26rft.epage%253D19%26rft.atitle%253DMechanism%2Bof%2Bchimera%2Bformation%2Bduring%2Bthe%2BMultiple%2BDisplacement%2BAmplification%2Breaction.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1472-6750-7-19%26rft_id%253Dinfo%253Apmid%252F17430586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/1472-6750-7-19&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17430586&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               33.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.33" data-doi="10.1186/s40168-019-0779-2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chijiiwa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Single-cell genomics of uncultured bacteria reveals dietary fiber responders in the mouse gut microbiota
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Microbiome
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  5
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMicrobiome%26rft.volume%253D8%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs40168-019-0779-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s40168-019-0779-2&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               34.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.34" data-doi="10.1186/s13059-019-1834-9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sanders
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.G.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Optimizing sequencing protocols for leaderboard metagenomics by combining long and short reads
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  226
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D20%26rft.spage%253D226%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-019-1834-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-019-1834-9&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               35.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.35">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hosokawa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Droplet-based microfluidics for high-throughput screening of a metagenomic library for isolation of microbial enzymes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biosens Bioelectron
                                 </abbr>
                                 <span class="cit-vol">
                                  67
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  379
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  385
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiosens%2BBioelectron%26rft.volume%253D67%26rft.spage%253D379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               36.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.36" data-doi="10.1093/bioinformatics/btp324">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Durbin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Fast and accurate short read alignment with Burrows-Wheeler transform
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1754
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1760
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19451168&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000267665900006&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               37.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.37" data-doi="10.1089/cmb.2012.0021">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bankevich
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Comput Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  455
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  477
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DBankevich%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D455%26rft.epage%253D477%26rft.atitle%253DSPAdes%253A%2Ba%2Bnew%2Bgenome%2Bassembly%2Balgorithm%2Band%2Bits%2Bapplications%2Bto%2Bsingle-cell%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2012.0021%26rft_id%253Dinfo%253Apmid%252F22506599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1089/cmb.2012.0021&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22506599&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               38.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.38" data-doi="10.1093/bioinformatics/btt086">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gurevich
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Saveliev
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vyahhi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tesler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  QUAST: quality assessment tool for genome assemblies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  29
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1072
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1075
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt086%26rft_id%253Dinfo%253Apmid%252F23422339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt086&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23422339&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000318109300015&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               39.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.39" data-doi="10.1093/bioinformatics/btp352">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The Sequence Alignment/Map format and SAMtools
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2078
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2079
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19505943&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000268808600014&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               40.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.40" data-doi="10.1093/bioinformatics/btq461">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Edgar
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.C.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Search and clustering orders of magnitude faster than BLAST
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2460
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2461
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq461%26rft_id%253Dinfo%253Apmid%252F20709691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq461&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20709691&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000282170000016&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               41.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.41" data-doi="10.1101/gr.186072.114">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Parks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Imelfort
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Skennerton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hugenholtz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tyson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.W.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Res
                                 </abbr>
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1043
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1055
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.186072.114%26rft_id%253Dinfo%253Apmid%252F25977477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI1LzcvMTA0MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA1LzExLzIwMjAuMDMuMDUuOTYyMDAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               42.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.42" data-doi="10.1093/bioinformatics/btu153">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Seemann
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Prokka: rapid prokaryotic genome annotation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2068
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2069
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu153%26rft_id%253Dinfo%253Apmid%252F24642063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu153&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24642063&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000339814300017&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               43.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.43" data-doi="10.1038/nbt.3893">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bowers
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Minimum information about a single amplified genome (MISAG) and a metagenome-assembled genome (MIMAG) of bacteria and archaea
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Biotechnol
                                 </abbr>
                                 <span class="cit-vol">
                                  35
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  725
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  731
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BBiotechnol%26rft.volume%253D35%26rft.spage%253D725%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3893%26rft_id%253Dinfo%253Apmid%252F28787424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nbt.3893&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28787424&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               44.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.44" data-doi="10.1093/nar/gkw290">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Letunic
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bork
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Interactive tree of life (iTOL) v3: an online tool for the display and annotation of phylogenetic and other trees
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 <span class="cit-vol">
                                  44
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  W242
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  245
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw290%26rft_id%253Dinfo%253Apmid%252F27095192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkw290&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27095192&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               45.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.45" data-doi="10.1038/s41467-018-07641-9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jain
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rodriguez
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Phillippy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Konstantinidis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.T.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Aluru
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  High throughput ANI analysis of 90K prokaryotic genomes reveals clear species boundaries
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Commun
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  5114
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D9%26rft.spage%253D5114%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-07641-9%26rft_id%253Dinfo%253Apmid%252F30504855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41467-018-07641-9&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30504855&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               46.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.46" data-doi="10.7717/peerj.985">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Roux
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Enault
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hurwitz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.L.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sullivan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.B.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  VirSorter: mining viral signal from microbial genomic data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PeerJ
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e985
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D3%26rft.spage%253D985e%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.985%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.7717/peerj.985&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               47.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.47" data-doi="10.1093/nar/gkz310">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  antiSMASH 5.0: updates to the secondary metabolite genome mining pipeline
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 <span class="cit-vol">
                                  47
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  W81
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  W87
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D47%26rft.spage%253DW81%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkz310%26rft_id%253Dinfo%253Apmid%252F31032519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkz310&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=31032519&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               48.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.48" data-doi="10.1128/AAC.02412-14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carattoli
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  In silico detection and typing of plasmids using PlasmidFinder and plasmid multilocus sequence typing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Antimicrob Agents Chemother
                                 </abbr>
                                 <span class="cit-vol">
                                  58
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3895
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3903
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAntimicrob%2BAgents%2BChemother%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAAC.02412-14%26rft_id%253Dinfo%253Apmid%252F24777092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjk6IjU4LzcvMzg5NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA1LzExLzIwMjAuMDMuMDUuOTYyMDAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               49.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.49" data-doi="10.7717/peerj.1319">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eren
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Anvi’o: an advanced analysis and visualization platform for ‘omics data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PeerJ
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e1319
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D3%26rft.spage%253De1319%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.1319%26rft_id%253Dinfo%253Apmid%252F26500826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.7717/peerj.1319&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26500826&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F11%2F2020.03.05.962001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               50.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.03.05.962001v2.50">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ankenbrand
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hohlfeld
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hackl
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Förster
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  AliTV—interactive visualization of whole genome comparisons
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PeerJ Computer Science
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.05.962001v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.03.09.483683v2" title="NLGC: Network Localized Granger Causality with Application to MEG Directional Functional Connectivity Analysis" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.05.11.491498v1" title="Context-dependent transcriptional regulation by Drosophila Polycomb Response Elements" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted May 11, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.03.05.962001v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.03.05.962001v2.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.03.05.962001v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.03.05.962001v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   Extensive single-cell genomics reveals bacterial diversity and diverse phage host ranges in the area in and around the Red Sea
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2539361"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-PHvviTDQ9U6p8zKq0CzCPB6hfgD9HwD5_NNqnuMJQl0"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484467708"/>
                                   <input type="hidden" name="captcha_token" value="62dc63009dba7bc98116f8955ce4beca"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2539361--3" data-node-nid="2539361" id="node-2539361--41788991024" data-pisa="biorxiv;2020.03.05.962001v2" data-pisa-master="biorxiv;2020.03.05.962001" data-seqnum="2539361" data-apath="/biorxiv/early/2022/05/11/2020.03.05.962001.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Extensive single-cell genomics reveals bacterial diversity and diverse phage host ranges in the area in and around the Red Sea
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Yohei
                                  </span>
                                  <span class="nlm-surname">
                                   Nishikawa
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Masato
                                  </span>
                                  <span class="nlm-surname">
                                   Kogawa
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Masahito
                                  </span>
                                  <span class="nlm-surname">
                                   Hosokawa
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Ryota
                                  </span>
                                  <span class="nlm-surname">
                                   Wagatsuma
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Katsuhiko
                                  </span>
                                  <span class="nlm-surname">
                                   Mineta
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Kai
                                  </span>
                                  <span class="nlm-surname">
                                   Takahashi
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">
                                   Keigo
                                  </span>
                                  <span class="nlm-surname">
                                   Ide
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="7">
                                  <span class="nlm-given-names">
                                   Kei
                                  </span>
                                  <span class="nlm-surname">
                                   Yura
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="8">
                                  <span class="nlm-given-names">
                                   Hayedeh
                                  </span>
                                  <span class="nlm-surname">
                                   Behzad
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="9">
                                  <span class="nlm-given-names">
                                   Takashi
                                  </span>
                                  <span class="nlm-surname">
                                   Gojobori
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="10">
                                  <span class="nlm-given-names">
                                   Haruko
                                  </span>
                                  <span class="nlm-surname">
                                   Takeyama
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.03.05.962001;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.03.05.962001
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.05.962001v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.962001v2&title=Extensive%20single-cell%20genomics%20reveals%20bacterial%20diversity%20and%20diverse%20phage%20host%20ranges%20in%20the%20area%20in%20and%20around%20the%20Red%20Sea" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.962001v2&title=Extensive%20single-cell%20genomics%20reveals%20bacterial%20diversity%20and%20diverse%20phage%20host%20ranges%20in%20the%20area%20in%20and%20around%20the%20Red%20Sea" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.962001v2&text=Extensive%20single-cell%20genomics%20reveals%20bacterial%20diversity%20and%20diverse%20phage%20host%20ranges%20in%20the%20area%20in%20and%20around%20the%20Red%20Sea" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.962001v2&t=Extensive%20single-cell%20genomics%20reveals%20bacterial%20diversity%20and%20diverse%20phage%20host%20ranges%20in%20the%20area%20in%20and%20around%20the%20Red%20Sea" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.962001v2&title=Extensive%20single-cell%20genomics%20reveals%20bacterial%20diversity%20and%20diverse%20phage%20host%20ranges%20in%20the%20area%20in%20and%20around%20the%20Red%20Sea" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.962001v2&title=Extensive%20single-cell%20genomics%20reveals%20bacterial%20diversity%20and%20diverse%20phage%20host%20ranges%20in%20the%20area%20in%20and%20around%20the%20Red%20Sea&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.962001v2&title=Extensive%20single-cell%20genomics%20reveals%20bacterial%20diversity%20and%20diverse%20phage%20host%20ranges%20in%20the%20area%20in%20and%20around%20the%20Red%20Sea" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2539361--5" data-node-nid="2539361" id="citation-node-2539361--6118021940" data-pisa="biorxiv;2020.03.05.962001v2" data-pisa-master="biorxiv;2020.03.05.962001" data-seqnum="2539361" data-apath="/biorxiv/early/2022/05/11/2020.03.05.962001.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Extensive single-cell genomics reveals bacterial diversity and diverse phage host ranges in the area in and around the Red Sea
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Yohei
                                      </span>
                                      <span class="nlm-surname">
                                       Nishikawa
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Masato
                                      </span>
                                      <span class="nlm-surname">
                                       Kogawa
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Masahito
                                      </span>
                                      <span class="nlm-surname">
                                       Hosokawa
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Ryota
                                      </span>
                                      <span class="nlm-surname">
                                       Wagatsuma
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Katsuhiko
                                      </span>
                                      <span class="nlm-surname">
                                       Mineta
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Kai
                                      </span>
                                      <span class="nlm-surname">
                                       Takahashi
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">
                                       Keigo
                                      </span>
                                      <span class="nlm-surname">
                                       Ide
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="7">
                                      <span class="nlm-given-names">
                                       Kei
                                      </span>
                                      <span class="nlm-surname">
                                       Yura
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="8">
                                      <span class="nlm-given-names">
                                       Hayedeh
                                      </span>
                                      <span class="nlm-surname">
                                       Behzad
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="9">
                                      <span class="nlm-given-names">
                                       Takashi
                                      </span>
                                      <span class="nlm-surname">
                                       Gojobori
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="10">
                                      <span class="nlm-given-names">
                                       Haruko
                                      </span>
                                      <span class="nlm-surname">
                                       Takeyama
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.03.05.962001;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.03.05.962001
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Extensive%20single-cell%20genomics%20reveals%20bacterial%20diversity%20and%20diverse%20phage%20host%20ranges%20in%20the%20area%20in%20and%20around%20the%20Red%20Sea&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Nishikawa%2C+Yohei&rft.au=Kogawa%2C+Masato&rft.au=Hosokawa%2C+Masahito&rft.au=Wagatsuma%2C+Ryota&rft.au=Mineta%2C+Katsuhiko&rft.au=Takahashi%2C+Kai&rft.au=Ide%2C+Keigo&rft.au=Yura%2C+Kei&rft.au=Behzad%2C+Hayedeh&rft.au=Gojobori%2C+Takashi&rft.au=Takeyama%2C+Haruko">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2539361/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2539361/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2539361/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2539361/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2539361/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2539361/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2539361/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2539361/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2539361/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2539361/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2539361/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2539361/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.962001v2&count=horizontal&via=&text=Extensive%20single-cell%20genomics%20reveals%20bacterial%20diversity%20and%20diverse%20phage%20host%20ranges%20in%20the%20area%20in%20and%20around%20the%20Red%20Sea&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.962001v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.962001v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.03.05.962001v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Genomics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15187)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
