<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.05.06.081372v3.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.05.06.081372v3.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.05.06.081372v3.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-10.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/06/15/2020.05.06.081372.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.05.06.081372v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Contrasting action and posture coding with hierarchical deep neural network models of proprioception"/>
  <meta name="DC.Identifier" content="10.1101/2020.05.06.081372"/>
  <meta name="DC.Date" content="2022-06-15"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Biological motor control is versatile and efficient. Muscles are flexible and undergo continuous changes, requiring distributed adaptive control mechanisms. How proprioception solves this problem in the brain is unknown. The canonical role of proprioception is representing the body state, yet we hypothesize that the proprioceptive system can decode high-level, multi-feature actions. To test this theory, we pursue a task-driven modeling approach.We generated a large synthetic dataset of human arm trajectories tracing the alphabet in 3D space and use a musculoskeletal model plus modeled muscle spindle inputs to extract muscle activity. We then contrast two tasks, one character trajectory-decoding and another action recognition task that allows training of hierarchical models to decode position, or classify the character identity from the spindle firing patterns. Artificial neural networks could robustly solve these tasks, and the networks’ units show tuning properties akin to neurons in the primate somatosensory cortex and the brainstem. Remarkably, only the action-recognition trained, and not the trajectory decoding trained, models possess directional selective units (which are also uniformly distributed), as in the primate brain. Taken together, our model is the first to link tuning properties in the proprioceptive system at multiple levels to the behavioral level. We find that action-recognition, rather than the canonical trajectory-decoding hypothesis, better explains what is known about the proprioceptive system.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Kai J. Sandbrink"/>
  <meta name="DC.Contributor" content="Pranav Mamidanna"/>
  <meta name="DC.Contributor" content="Claudio Michaelis"/>
  <meta name="DC.Contributor" content="Matthias Bethge"/>
  <meta name="DC.Contributor" content="Mackenzie Weygandt Mathis"/>
  <meta name="DC.Contributor" content="Alexander Mathis"/>
  <meta name="article:published_time" content="2022-06-15"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Contrasting action and posture coding with hierarchical deep neural network models of proprioception"/>
  <meta name="citation_abstract" lang="en" content="<p>Biological motor control is versatile and efficient. Muscles are flexible and undergo continuous changes, requiring distributed adaptive control mechanisms. How proprioception solves this problem in the brain is unknown. The canonical role of proprioception is representing the body state, yet we hypothesize that the proprioceptive system can decode high-level, multi-feature actions. To test this theory, we pursue a task-driven modeling approach.We generated a large synthetic dataset of human arm trajectories tracing the alphabet in 3D space and use a musculoskeletal model plus modeled muscle spindle inputs to extract muscle activity. We then contrast two tasks, one character trajectory-decoding and another action recognition task that allows training of hierarchical models to decode position, or classify the character identity from the spindle firing patterns. Artificial neural networks could robustly solve these tasks, and the networks’ units show tuning properties akin to neurons in the primate somatosensory cortex and the brainstem. Remarkably, only the action-recognition trained, and not the trajectory decoding trained, models possess directional selective units (which are also uniformly distributed), as in the primate brain. Taken together, our model is the first to link tuning properties in the proprioceptive system at multiple levels to the behavioral level. We find that action-recognition, rather than the canonical trajectory-decoding hypothesis, better explains what is known about the proprioceptive system.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.05.06.081372v3"/>
  <meta name="citation_id" content="2020.05.06.081372v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.05.06.081372v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.05.06.081372v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.05.06.081372v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.05.06.081372"/>
  <meta name="citation_num_pages" content="27"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.05.06.081372"/>
  <meta name="citation_author" content="Kai J. Sandbrink"/>
  <meta name="citation_author_institution" content="The Rowland Institute at Harvard, Harvard University"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0360-075X"/>
  <meta name="citation_author" content="Pranav Mamidanna"/>
  <meta name="citation_author_institution" content="Institute for Theoretical Physics, Eberhard Karls Universityät Tübingen & Tübingen AI Center"/>
  <meta name="citation_author_institution" content="Department of Health Science and Technology, Aalborg University"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2095-3314"/>
  <meta name="citation_author" content="Claudio Michaelis"/>
  <meta name="citation_author_institution" content="Institute for Theoretical Physics, Eberhard Karls Universityät Tübingen & Tübingen AI Center"/>
  <meta name="citation_author" content="Matthias Bethge"/>
  <meta name="citation_author_institution" content="Institute for Theoretical Physics, Eberhard Karls Universityät Tübingen & Tübingen AI Center"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6417-7812"/>
  <meta name="citation_author" content="Mackenzie Weygandt Mathis"/>
  <meta name="citation_author_institution" content="The Rowland Institute at Harvard, Harvard University"/>
  <meta name="citation_author_institution" content="Brain Mind Institute, School of Life Sciences, École Polytechnique Fédérale de Lausanne"/>
  <meta name="citation_author" content="Alexander Mathis"/>
  <meta name="citation_author_institution" content="The Rowland Institute at Harvard, Harvard University"/>
  <meta name="citation_author_institution" content="Brain Mind Institute, School of Life Sciences, École Polytechnique Fédérale de Lausanne"/>
  <meta name="citation_author_email" content="alexander.mathis@epfl.ch"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3777-2202"/>
  <meta name="citation_reference" content="R Chris Miall, Nick M Kitchen, Se-Ho Nam, Hannah Lefumat, Alix G Renault, Kristin Ørstavik, Jonathan D Cole, and Fabrice R Sarlegna. Proprioceptive loss and the perception, control and learning of arm movements in humans: evidence from sensory neuronopathy. Experimental brain research, pages 1–19, 2018."/>
  <meta name="citation_reference" content="Uwe Proske and Simon C Gandevia. The Proprioceptive Senses: Their Roles in Signaling Body Shape, Body Position and Movement, and Muscle Force. Physiological Reviews, 92(4):1651–1697, 2012. ISSN 0031-9333. doi: 10.1152/physrev.00048.2011."/>
  <meta name="citation_reference" content="Benoit P Delhaye, Katie H Long, and Sliman J Bensmaia. Neural basis of touch and proprioception in primate cortex. Comprehensive Physiology, 8(4):1575, 2018."/>
  <meta name="citation_reference" content="Francis J Clark, Curt R Burgess, James W Chapin, and WT Lipscomb. Role of intramuscular receptors in the awareness of limb position. Journal of Neurophysiology, 54(6):1529–1540, 1985."/>
  <meta name="citation_reference" content="Peter BC Matthews. The response of de-efferented muscle spindle receptors to stretching at different velocities. The Journal of Physiology, 168(3):660–678, 1963."/>
  <meta name="citation_reference" content="Peter BC Matthews. Muscle spindles: their messages and their fusimotor supply. Handbook of physiology: I. The nervous system. American Physiological Society.[AGF], 1981."/>
  <meta name="citation_reference" content="Nikolai A Bernstein. The co-ordination and regulation of movements, volume 1. Oxford, New York, Pergamon Press, 1967."/>
  <meta name="citation_reference" content="Gianfranco Bosco, A. Rankin, and Richard E Poppele. Representation of passive hindlimb postures in cat spinocerebellar activity. J Neurophysiol, 76(2):715–26, 1996. ISSN 0022-3077."/>
  <meta name="citation_reference" content="John C. Tuthill and Eiman Azim. Proprioception. Current Biology, 28(5):R193–R194, 2018. ISSN 09609822. doi: 10.1016/j.cub.2018.01.064."/>
  <meta name="citation_reference" content="Joseph T Francis, Shaohua Xu, and John K Chapin. Proprioceptive and cutaneous representations in the rat ventral posterolateral thalamus. Journal of neurophysiology, 99(5):2291–2304, 2008."/>
  <meta name="citation_reference" content="James M Goodman, Gregg A Tabot, Alex S Lee, Aneesha K Suresh, Alexander T Rajan, Nicholas G Hatsopoulos, and Sliman Bensmaia. Postural representations of the hand in the primate sensorimotor cortex. Neuron, 104(5):1000–1009, 2019."/>
  <meta name="citation_reference" content="Raeed H Chowdhury, Joshua I Glaser, and Lee E Miller. Area 2 of primary somatosensory cortex encodes kinematics of the whole arm. Elife, 9, 2020."/>
  <meta name="citation_reference" content="Christoph Fromm and Edward V Evarts. Pyramidal tract neurons in somatosensory cortex: central and peripheral inputs during voluntary movement. Brain research, 238(1):186–191, 1982."/>
  <meta name="citation_reference" content="Michel JL Prud’Homme and John F Kalaska. Proprioceptive activity in primate primary somatosensory cortex during active arm reaching movements. Journal of neurophysiology, 72(5):2280–2301, 1994."/>
  <meta name="citation_reference" content="Mackenzie W Mathis, Alexander Mathis, and Naoshige Uchida. Somatosensory cortex plays an essential role in forelimb motor adaptation in mice. Neuron, 93:p1493–1503.e6, 2017."/>
  <meta name="citation_reference" content="Neeraj Kumar, Timothy F Manning, and David J Ostry. Somatosensory cortex participates in the consolidation of human motor memory. PLoS biology, 17(10), 2019."/>
  <meta name="citation_reference" content="Uwe Proske and Simon C Gandevia. The proprioceptive senses: their roles in signaling body shape, body position and movement, and muscle force. Physiological reviews, 92(4):1651–1697, 2012."/>
  <meta name="citation_reference" content="Michael SA Graziano. Ethological action maps: a paradigm shift for the motor cortex. Trends in cognitive sciences, 20(2):121–132, 2016."/>
  <meta name="citation_reference" content="Olga Russakovsky, Jia Deng, Hao Su, Jonathan Krause, Sanjeev Satheesh, Sean Ma, Zhiheng Huang, Andrej Karpathy, Aditya Khosla, Michael Bernstein, et al. Imagenet large scale visual recognition challenge. International Journal of Computer Vision, 115(3): 211–252, 2015."/>
  <meta name="citation_reference" content="Seyed-Mahdi Khaligh-Razavi and Nikolaus Kriegeskorte. Deep supervised, but not unsupervised, models may explain it cortical representation. PLoS computational biology, 10(11):e1003915, 2014."/>
  <meta name="citation_reference" content="Daniel LK Yamins, Ha Hong, Charles F Cadieu, Ethan A Solomon, Darren Seibert, and James J DiCarlo. Performance-optimized hierarchical models predict neural responses in higher visual cortex. Proceedings of the National Academy of Sciences, 111(23):8619–8624, 2014."/>
  <meta name="citation_reference" content="Radoslaw Martin Cichy, Aditya Khosla, Dimitrios Pantazis, Antonio Torralba, and Aude Oliva. Comparison of deep neural networks to spatio-temporal cortical dynamics of human visual object recognition reveals hierarchical correspondence. Scientific reports, 6: 27755, 2016."/>
  <meta name="citation_reference" content="Daniel LK Yamins and James J DiCarlo. Using goal-driven deep learning models to understand sensory cortex. Nature neuroscience, 19(3):356, 2016."/>
  <meta name="citation_reference" content="Martin Schrimpf, Jonas Kubilius, Ha Hong, Najib J Majaj, Rishi Rajalingham, Elias B Issa, Kohitij Kar, Pouya Bashivan, Jonathan Prescott-Roy, Kailyn Schmidt, et al. Brain-score: Which artificial neural network for object recognition is most brain-like? BioRxiv, page 407007, 2018."/>
  <meta name="citation_reference" content="Santiago A Cadena, George H Denfield, Edgar Y Walker, Leon A Gatys, Andreas S Tolias, Matthias Bethge, and Alexander S Ecker. Deep convolutional models improve predictions of macaque v1 responses to natural images. PLoS computational biology, 15(4): e1006897, 2019."/>
  <meta name="citation_reference" content="Katherine R Storrs, Tim C Kietzmann, Alexander Walther, Johannes Mehrer, and Nikolaus Kriegeskorte. Diverse deep neural networks all predict human inferior temporal cortex well, after training and fitting. Journal of Cognitive Neuroscience, 33(10):2044–2064, 2021."/>
  <meta name="citation_reference" content="Blake A Richards, Timothy P Lillicrap, Philippe Beaudoin, Yoshua Bengio, Rafal Bogacz, Amelia Christensen, Claudia Clopath, Rui Ponte Costa, Archy de Berker, Surya Ganguli, et al. A deep learning framework for neuroscience. Nature Neuroscience, 22(11): 1761–1770, 2019."/>
  <meta name="citation_reference" content="Chengxu Zhuang, Jonas Kubilius, Mitra JZ Hartmann, and Daniel L Yamins. Toward goal-driven neural network models for the rodent whisker-trigeminal system. In Advances in Neural Information Processing Systems, pages 2555–2565, 2017."/>
  <meta name="citation_reference" content="Subramanian Sundaram, Petr Kellnhofer, Yunzhu Li, Jun-Yan Zhu, Antonio Torralba, and Wojciech Matusik. Learning the signatures of the human grasp using a scalable tactile glove. Nature, 569(7758): 698–702, 2019."/>
  <meta name="citation_reference" content="Martin Haesemeyer, Alexander F Schier, and Florian Engert. Convergent temperature representations in artificial and biological neural networks. Neuron, 103(6):1123–1134, 2019."/>
  <meta name="citation_reference" content="Alexander JE Kell, Daniel LK Yamins, Erica N Shook, Sam V Norman-Haignere, and Josh H McDermott. A task-optimized neural network replicates human auditory behavior, predicts brain responses, and reveals a cortical processing hierarchy. Neuron, 98 (3):630–644, 2018."/>
  <meta name="citation_reference" content="Ben H Williams, Marc Toussaint, and Amos J Storkey. Extracting motion primitives from natural handwriting data. In International Conference on Artificial Neural Networks, pages 634–643. Springer, 2006."/>
  <meta name="citation_reference" content="Katherine R Saul, Xiao Hu, Craig M Goehler, Meghan E Vidt, Melissa Daly, Anca Velisar, and Wendy M Murray. Benchmarking of dynamic simulation predictions in two software platforms using an upper limb musculoskeletal model. Computer methods in biomechanics and biomedical engineering, 18(13):1445–1458, 2015."/>
  <meta name="citation_reference" content="Michael Dimitriou and Benoni B Edin. Discharges in human muscle receptor afferents during block grasping. Journal of Neuroscience, 28(48):12632–12642, 2008."/>
  <meta name="citation_reference" content="Michael Dimitriou and Benoni B Edin. Discharges in human muscle spindle afferents during a key-pressing task. The Journal of physiology, 586(22):5455–5470, 2008."/>
  <meta name="citation_reference" content="Dua Dheeru and Efi Karra Taniskidou. UCI machine learning repository, 2017."/>
  <meta name="citation_reference" content="Laurens van der Maaten and Geoffrey Hinton. Visualizing data using t-sne. Journal of machine learning research, 9(Nov):2579–2605, 2008."/>
  <meta name="citation_reference" content="Chih-Wei Hsu and Chih-Jen Lin. A comparison of methods for multiclass support vector machines. IEEE transactions on Neural Networks, 13(2):415–425, 2002."/>
  <meta name="citation_reference" content="Sébastien B Hausmann, Alessandro Marin Vargas, Alexander Mathis, and Mackenzie W Mathis. Measuring and modeling the motor system with machine learning. Current Opinion in Neurobiology, 70:11–23, 2021."/>
  <meta name="citation_reference" content="David E Rumelhart, Geoffrey E Hinton, James L McClelland, et al. A general framework for parallel distributed processing. Parallel distributed processing: Explorations in the microstructure of cognition, 1(45-76):26, 1986."/>
  <meta name="citation_reference" content="Yann LeCun, Léon Bottou, Yoshua Bengio, Patrick Haffner, et al. Gradient-based learning applied to document recognition. Proceedings of the IEEE, 86(11):2278–2324, 1998."/>
  <meta name="citation_reference" content="Sepp Hochreiter and Jürgen Schmidhuber. Long short-term memory. Neural computation, 9(8):1735–1780, 1997."/>
  <meta name="citation_reference" content="Edith Ribot-Ciscar, Mikael Bergenheim, Frédéric Albert, and Jean-Pierre Roll. Proprioceptive population coding of limb position in humans. Experimental brain research, 149(4):512–519, 2003."/>
  <meta name="citation_reference" content="P. Kibleur, S. R. Tata, N. Greiner, S. Conti, B. Barra, K. Zhuang, M. Kaeser, A. Ijspeert, and M. Capogrosso. Spatiotemporal maps of proprioceptive inputs to the cervical spinal cord during three-di-mensional reaching and grasping. IEEE Transactions on Neural Systems and Rehabilitation Engineering, pages 1–1, 2020."/>
  <meta name="citation_reference" content="Bernd Illing, Wulfram Gerstner, and Johanni Brea. Biologically plausible deep learning—but how far can we go with shallow networks? Neural Networks, 118:90–101, 2019."/>
  <meta name="citation_reference" content="Frederic Albert, Edith Ribot-Ciscar, Michel Fiocchi, Mikael Bergenheim, and Jean-Pierre Roll. Proprioceptive feedback in humans expresses motor invariants during writing. Experimental brain research, 164(2):242–249, 2005."/>
  <meta name="citation_reference" content="Kai J Sandbrink, Pranav Mamidanna, Claudio Michaelis, Mackenzie Weygandt Mathis, Matthias Bethge, and Alexander Mathis. Task-driven hierarchical deep neural network models of the proprioceptive pathway. bioRxiv, 2020."/>
  <meta name="citation_reference" content="Arthur Prochazka and Monica Gorassini. Ensemble firing of muscle afferents recorded during normal locomotion in cats. The Journal of physiology, 507(1):293–304, 1998."/>
  <meta name="citation_reference" content="Christopher Versteeg, Joshua M Rosenow, Sliman J Bensmaia, and Lee E Miller. Encoding of limb state by single neurons in the cuneate nucleus of awake monkeys. Journal of Neurophysiology, 2021."/>
  <meta name="citation_reference" content="Thomas Serre. Deep learning: the good, the bad, and the ugly. Annual Review of Vision Science, 5:399–426, 2019."/>
  <meta name="citation_reference" content="Andrew Jackson, Jaideep Mavoori, and Eberhard E Fetz. Correlations between the same motor cortex cells and arm muscles during a trained task, free behavior, and natural sleep in the macaque monkey. Journal of neurophysiology, 97(1):360–374, 2007."/>
  <meta name="citation_reference" content="Simon C Gandevia, Kathyrn M Refshauge, and David F Collins. Proprioception: peripheral inputs and perceptual interactions. In Sen-sorimotor control of movement and posture, pages 61–68. Springer, 2002."/>
  <meta name="citation_reference" content="Milana P Mileusnic, Ian E Brown, Ning Lan, and Gerald E Loeb. Mathematical models of proprioceptors. i. control and transduction in the muscle spindle. Journal of neurophysiology, 96(4):1772–1788, 2006."/>
  <meta name="citation_reference" content="Jean-Marc Aimonetti, Valérie Hospod, Jean-Pierre Roll, and Edith Ribot-Ciscar. Cutaneous afferents provide a neuronal population vector that encodes the orientation of human ankle movements. The Journal of physiology, 580(2):649–658, 2007."/>
  <meta name="citation_reference" content="Kyle P Blum, Boris Lamotte D’Incamps, Daniel Zytnicki, and Lena H Ting. Force encoding in muscle spindles during stretch of passive muscle. PLoS computational biology, 13(9):e1005767, 2017."/>
  <meta name="citation_reference" content="Rochelle Ackerley, Marie Chancel, Jean-Marc Aimonetti, Edith Ribot-Ciscar, and Anne Kavounoudias. Seeing your foot move changes muscle proprioceptive feedback. Eneuro, 6(2), 2019."/>
  <meta name="citation_reference" content="Katherine RS Holzbaur, Wendy M Murray, and Scott L Delp. A model of the upper extremity for simulating musculoskeletal surgery and analyzing neuromuscular control. Annals of biomedical engineering, 33(6):829–840, 2005."/>
  <meta name="citation_reference" content="Aaron D’Souza, Sethu Vijayakumar, and Stefan Schaal. Learning inverse kinematics. In Proceedings 2001 IEEE/RSJ International Conference on Intelligent Robots and Systems. Expanding the Societal Role of Robotics in the the Next Millennium (Cat. No. 01CH37180), volume 1, pages 298–303. IEEE, 2001."/>
  <meta name="citation_reference" content="Scott L Delp, Frank C Anderson, Allison S Arnold, Peter Loan, Ayman Habib, Chand T John, Eran Guendelman, and Darryl G Thelen. Opensim: open-source software to create and analyze dynamic simulations of movement. IEEE transactions on biomedical engineering, 54(11):1940–1950, 2007."/>
  <meta name="citation_reference" content="Ajay Seth, Michael Sherman, Jeffrey A Reinbolt, and Scott L Delp. Opensim: a musculoskeletal modeling and simulation framework for in silico investigations and exchange. Procedia Iutam, 2:212–232, 2011."/>
  <meta name="citation_reference" content="Vaughan G Macefield and Thomas P Knellwolf. Functional properties of human muscle spindles. Journal of neurophysiology, 2018."/>
  <meta name="citation_reference" content="Arthur Prochazka and Monica Gorassini. Models of ensemble firing of muscle spindle afferents recorded during normal locomotion in cats. The Journal of physiology, 507(1):277–291, 1998."/>
  <meta name="citation_reference" content="Fabian Pedregosa, Gaël Varoquaux, Alexandre Gramfort, Vincent Michel, Bertrand Thirion, Olivier Grisel, Mathieu Blondel, Peter Prettenhofer, Ron Weiss, Vincent Dubourg, et al. Scikit-learn: Machine learning in python. the Journal of machine Learning research, 12: 2825–2830, 2011."/>
  <meta name="citation_reference" content="Nikolaus Kriegeskorte, Marieke Mur, and Peter A Bandettini. Repre-sentational similarity analysis-connecting the branches of systems neuroscience. Frontiers in systems neuroscience, 2:4, 2008."/>
  <meta name="citation_reference" content="Apostolos P Georgopoulos, John F Kalaska, Roberto Caminiti, and Joe T Massey. On the relations between the direction of two-dimensional arm movements and cell discharge in primate motor cortex. The Journal of neuroscience : the official journal of the Society for Neuroscience, 2 11:1527–37, 1982."/>
  <meta name="citation_reference" content="Pauli Virtanen, Ralf Gommers, Travis E Oliphant, Matt Haberland, Tyler Reddy, David Cournapeau, Evgeni Burovski, Pearu Peterson, Warren Weckesser, Jonathan Bright, et al. Scipy 1.0: fundamental algorithms for scientific computing in python. Nature methods, 17 (3):261–272, 2020."/>
  <meta name="citation_reference" content="Martín Abadi, Paul Barham, Jianmin Chen, Zhifeng Chen, Andy Davis, Jeffrey Dean, Matthieu Devin, Sanjay Ghemawat, Geoffrey Irving, Michael Isard, et al. Tensorflow: A system for large-scale machine learning. In 12th {USENIX} Symposium on Operating Systems Design and Implementation ({OSDI} 16), pages 265–283, 2016."/>
  <meta name="twitter:title" content="Contrasting action and posture coding with hierarchical deep neural network models of proprioception"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Biological motor control is versatile and efficient. Muscles are flexible and undergo continuous changes, requiring distributed adaptive control mechanisms. How proprioception solves this problem in the brain is unknown. The canonical role of proprioception is representing the body state, yet we hypothesize that the proprioceptive system can decode high-level, multi-feature actions. To test this theory, we pursue a task-driven modeling approach.We generated a large synthetic dataset of human arm trajectories tracing the alphabet in 3D space and use a musculoskeletal model plus modeled muscle spindle inputs to extract muscle activity. We then contrast two tasks, one character trajectory-decoding and another action recognition task that allows training of hierarchical models to decode position, or classify the character identity from the spindle firing patterns. Artificial neural networks could robustly solve these tasks, and the networks’ units show tuning properties akin to neurons in the primate somatosensory cortex and the brainstem. Remarkably, only the action-recognition trained, and not the trajectory decoding trained, models possess directional selective units (which are also uniformly distributed), as in the primate brain. Taken together, our model is the first to link tuning properties in the proprioceptive system at multiple levels to the behavioral level. We find that action-recognition, rather than the canonical trajectory-decoding hypothesis, better explains what is known about the proprioceptive system.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Contrasting action and posture coding with hierarchical deep neural network models of proprioception"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.05.06.081372v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Biological motor control is versatile and efficient. Muscles are flexible and undergo continuous changes, requiring distributed adaptive control mechanisms. How proprioception solves this problem in the brain is unknown. The canonical role of proprioception is representing the body state, yet we hypothesize that the proprioceptive system can decode high-level, multi-feature actions. To test this theory, we pursue a task-driven modeling approach.We generated a large synthetic dataset of human arm trajectories tracing the alphabet in 3D space and use a musculoskeletal model plus modeled muscle spindle inputs to extract muscle activity. We then contrast two tasks, one character trajectory-decoding and another action recognition task that allows training of hierarchical models to decode position, or classify the character identity from the spindle firing patterns. Artificial neural networks could robustly solve these tasks, and the networks’ units show tuning properties akin to neurons in the primate somatosensory cortex and the brainstem. Remarkably, only the action-recognition trained, and not the trajectory decoding trained, models possess directional selective units (which are also uniformly distributed), as in the primate brain. Taken together, our model is the first to link tuning properties in the proprioceptive system at multiple levels to the behavioral level. We find that action-recognition, rather than the canonical trajectory-decoding hypothesis, better explains what is known about the proprioceptive system.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-06-15"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.05.06.081372v3"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2596403"/>
  <title>
   Contrasting action and posture coding with hierarchical deep neural network models of proprioception | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"lwEOJfkpAzZ_T6gv-rrEpUY_Lg6CPdxizkLiRXYCtzo"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2596403","apath":"\/biorxiv\/early\/2022\/06\/15\/2020.05.06.081372.atom","pisa":"biorxiv;2020.05.06.081372v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.05.06.081372v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.05.06.081372v3","title":"Contrasting action and posture coding with hierarchical deep neural network models of proprioception","identifier":"node\/2596403"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jLigw1fShYJLXLaZFoXCa8HgMTlT1hIyIdUk4wbeR5B9uxenwqqZZ3lYwKnuibwwa7cZJRkWTSeZPlMCBkNvpOhqK4rAC+cnSYEkvTDdQqk\\\\\\\/ssoPu791ej61Fz4Gd54KGmVh\\\\\\\/kr1ZbI4OHAEv\\\\\\\/yHB8IIrL6ufAuX76OmVVpLOCY\\\\\\\/yo301wQbWDeRqY\\\\\\\/sA2r2FNZYMSokmNklnXKyr6G431E9fRmYDMU2Pr\\\\\\\/AKp4REiiP6GtDEaSsvT56idKNl8BZdWhOOW5iMMiMAFBmc1mhv4YcUgbvZ3ov8fOnu1pOVryrAMGlgE9Ckam3mZXGmnNIaG9p\\\\\\\/rTcRPUxfyyHxq0NhN5Y\\\\\\\/pBmTOaWKgfaOhlLEap8gLTOq\\\\\\\/Sm7yOQH1CEdetotXc7pqpIWi6mt+vjnzoqq5hJnjNhmAdPiWUnh\\\\\\\/eldYq0VmYNZDTnLavd8BULylo\\\\\\\/yfKg6Jto5CsAgRyMlg2Nz0WlNuIYUyVRPoHDF5PdjZgpLBeFatq4EXdRQtrSJlKfQssM18nHzyNcI9otm\\\\\\\/JMML7QAZMNvFP2WlkQVxOXMhBK8g69qmADoZCKLvPBsavQWlq5DmBmOLaFL0VDSqOjw+7KThHRBoiL0YElOr1N8bQbLjgjGx97Pm3Cqr\\\\\\\/9HzMvEfQW5M1gkDRK5UflIiDmsNhiP5+dAveMPRds5+WXeumRoon3VbY6k0rxq+rfgv6v9KOrHVWuDWasLQ7g+niYWUAn0r+dEoUKC9m1W6ProWRQPLL6As\\\\\\\/mP69W9sBmlp2XGHG4NLnOSHvUetlum1ytLenufIeyiMhR0mMpY5V4PWHBRw0ajq\\\\\\\/ogkRTsOV6Dhcvi44Xmwcnl9wLsQLHvYxYP7tQlTACiadaCOva54a3BSTRTR5GxSjmiQfJCG42UbOPhyao3OYePui\\\\\\\/almswsEvW6G+aiIe\\\\\\\/Ya5XAGYAPn2o7dKbNRHZ6snMn\\\\\\\/m6LIFpPEUCbFHTmiiJnBUmgRU0hAj7aICb2l+kU7m37OIbLJR3\\\\\\\/01sb\\\\\\\/Td5P6dsJ+gzzqiSmFlQ0rJpus7MHx9ZW6iNcsAKX1NI7ZNTvdOl8QZjMlRwBlRJZ2\\\\\\\/buZ8chNfkXRGQtTIOJk4Z1i9zNYNCjRMjrAit44YIiyDJlP5yj\\\\\\\/f17dF7IzYBMYldMBJD5seTQIZ2aTOB0xI8LOdXu0G7wwdyxo30Myi19o9isvIbA29WBkPac7dtFs1daQML\\\\\\\/mYlsxlqMS+Cab9iMqMScv4wwdY5CNBhGwA032iBlgP0hwTaL0LTU1\\\\\\\/VDkD9iauiacZNi3Crdx6IDgmUoX+G59sSDJpTiFWU8d4ypf43sfDL6NTxeYdTkXGkekmLj\\\\\\\/eQN60K2hyEl6KcuF58bEr2QNaZJOxpWJkmeFeg0AWGKqyt6tP2FdPfI=\\\u0022,\\\u0022iv\\\u0022:\\\u0022E6l9vfi8AWzS0TQAMUNgAA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022ac1a8ce9de8e58cac7abd9a1be9e8ed0a1a4509c43150cbb6bfc4eed3e517af2\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.05.06.081372v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2596403 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.05.06.081372v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_577957715">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_577957715" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_981921002" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-UemgbiF4mMLXd_ibYdz_tQ--mbwzQHjj1VqZgpRiBjA"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2596403" data-node-nid="2596403" id="node-2596403--2139456670" data-pisa="biorxiv;2020.05.06.081372v3" data-pisa-master="biorxiv;2020.05.06.081372" data-apath="/biorxiv/early/2022/06/15/2020.05.06.081372.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Contrasting action and posture coding with hierarchical deep neural network models of proprioception
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0003-0360-075X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Kai J.
                      </span>
                      <span class="nlm-surname">
                       Sandbrink
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0002-2095-3314" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Pranav
                      </span>
                      <span class="nlm-surname">
                       Mamidanna
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Claudio
                      </span>
                      <span class="nlm-surname">
                       Michaelis
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3">
                      <a href="http://orcid.org/0000-0002-6417-7812" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Matthias
                      </span>
                      <span class="nlm-surname">
                       Bethge
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="4">
                      <span class="nlm-given-names">
                       Mackenzie Weygandt
                      </span>
                      <span class="nlm-surname">
                       Mathis
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5">
                      <a href="http://orcid.org/0000-0002-3777-2202" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Alexander
                      </span>
                      <span class="nlm-surname">
                       Mathis
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.05.06.081372
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2596403--2139456670" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Kai J. Sandbrink
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        The Rowland Institute at Harvard, Harvard University
                       </span>
                       , Cambridge, MA
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Kai%2BJ.%2BSandbrink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Sandbrink%20KJ&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AKai%2BJ.%2BSandbrink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-0360-075X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Kai J. Sandbrink
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Pranav Mamidanna
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Institute for Theoretical Physics, Eberhard Karls Universityät Tübingen & Tübingen AI Center
                       </span>
                       , Tübingen,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Health Science and Technology, Aalborg University
                       </span>
                       , Aalborg,
                       <span class="nlm-country">
                        Denmark
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Pranav%2BMamidanna%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Mamidanna%20P&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3APranav%2BMamidanna%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-2095-3314" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Pranav Mamidanna
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Claudio Michaelis
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Institute for Theoretical Physics, Eberhard Karls Universityät Tübingen & Tübingen AI Center
                       </span>
                       , Tübingen,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Claudio%2BMichaelis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Michaelis%20C&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AClaudio%2BMichaelis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Matthias Bethge
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Institute for Theoretical Physics, Eberhard Karls Universityät Tübingen & Tübingen AI Center
                       </span>
                       , Tübingen,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Matthias%2BBethge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Bethge%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AMatthias%2BBethge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-6417-7812" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Matthias Bethge
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Mackenzie Weygandt Mathis
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        The Rowland Institute at Harvard, Harvard University
                       </span>
                       , Cambridge, MA
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Brain Mind Institute, School of Life Sciences, École Polytechnique Fédérale de Lausanne
                       </span>
                       , Lausanne,
                       <span class="nlm-country">
                        Switzerland
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Mackenzie%2BWeygandt%2BMathis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Mathis%20MW&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AMackenzie%2BWeygandt%2BMathis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Alexander Mathis
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        The Rowland Institute at Harvard, Harvard University
                       </span>
                       , Cambridge, MA
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Brain Mind Institute, School of Life Sciences, École Polytechnique Fédérale de Lausanne
                       </span>
                       , Lausanne,
                       <span class="nlm-country">
                        Switzerland
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Alexander%2BMathis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Mathis%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAlexander%2BMathis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0002-3777-2202" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Alexander Mathis
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:alexander.mathis@epfl.ch" class="" data-icon-position="" data-hide-link-title="0">
                        alexander.mathis@epfl.ch
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.05.06.081372v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2596403" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2596403/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.06.081372v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2596403" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2596403/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.06.081372v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2596403" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2596403/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.06.081372v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2596403" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2596403/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.06.081372v3.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2596403" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2596403/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.05.06.081372v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2596403" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2596403/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2>
                             Abstract
                            </h2>
                            <p id="p-4">
                             Biological motor control is versatile and efficient. Muscles are flexible and undergo continuous changes, requiring distributed adaptive control mechanisms. How proprioception solves this problem in the brain is unknown. The canonical role of proprioception is representing the body state, yet we hypothesize that the proprioceptive system can decode high-level, multi-feature actions. To test this theory, we pursue a task-driven modeling approach.We generated a large synthetic dataset of human arm trajectories tracing the alphabet in 3D space and use a musculoskeletal model plus modeled muscle spindle inputs to extract muscle activity. We then contrast two tasks, one character trajectory-decoding and another action recognition task that allows training of hierarchical models to decode position, or classify the character identity from the spindle firing patterns. Artificial neural networks could robustly solve these tasks, and the networks’ units show tuning properties akin to neurons in the primate somatosensory cortex and the brainstem. Remarkably, only the action-recognition trained, and not the trajectory decoding trained, models possess directional selective units (which are also uniformly distributed), as in the primate brain. Taken together, our model is the first to link tuning properties in the proprioceptive system at multiple levels to the behavioral level. We find that action-recognition, rather than the canonical trajectory-decoding hypothesis, better explains what is known about the proprioceptive system.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-7">
                             Proprioception is a critical component of our ability to perform complex movements, localize our body in space, and adapt to environmental changes (
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             –
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ). Our movements are generated by a large number of muscles and are sensed via a diverse set of receptors, most importantly muscle spindles, which carry highly multiplexed information (
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ,
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ). For instance, arm movements are sensed via distributed and individually ambiguous activity patterns of muscle spindles, which depend on relative joint configurations rather than the absolute hand position (
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ,
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ). Interpreting this high dimensional input (around 50 muscles for a human arm) of distributed information at the relevant behavioral level poses a challenging decoding problem for the central nervous system (
                             <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ,
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             ). Proprioceptive information from the receptors undergoes several processing steps before reaching somatosensory cortex (
                             <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ,
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ,
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ) - from the spindles that synapse in Clarke’s nucleus, to the brainstem, thalamus (
                             <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ,
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ), and finally to somatosensory cortex (S1). In cortex, a number of tuning properties have been observed, such as responsiveness to varied combinations of joints and muscle lengths (
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ,
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ), sensitivity to different loads and angles (
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ), and broad and uni-modal tuning for movement direction during arm movements (
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ). The proprioceptive information in S1 is then hypothesized to serve as the basis of a wide variety of tasks, via its connections to motor cortex and higher somatosensory processing regions (1–3, 15, 16).
                            </p>
                            <p id="p-8">
                             One key role of proprioception is to sense the state of the body—i.e., posture. This information subserves many other functions, from balance to motor learning. Thus, to gain insights into the computations of the proprioceptive system, we quantitatively compare two different goals in a task-driven fashion: a trajectory-decoding task, and an action recognition task (
                             <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                              Figure 1
                             </a>
                             ). The trajectory-decoding task represents the canonical view of proprioception (
                             <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ,
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             ). Alternatively, the role of the proprioceptive system could be to infer actions (i.e., complex sequences of postures). Our hypothesis is motivated by the observation that action segmentation would be an efficient way to represent complex behavior, and it could directly drive the “action map” in motor cortex (
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ).
                            </p>
                            <div id="F1" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F1.large.jpg?width=800&height=600&carousel=1" title="Contrasting spindle-based tasks to study proprioception: Proprioceptive inputs that correspond to the tracing of individual letters were simulated using a musculoskeletal model of a human arm. This scalable, large-scale dataset can then be used to train deep neural networks models of the proprioceptive pathway to either classify the character (Action Recognition Task, ART) or to localize the hand (Trajectory Decoding Task, TDT) based on the input muscle spindle firing rates. These models can be analyzed, compared and contrasted to what is known about the proprioceptive system in primates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1989804583" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Contrasting spindle-based tasks to study proprioception:</span> Proprioceptive inputs that correspond to the tracing of individual letters were simulated using a musculoskeletal model of a human arm. This scalable, large-scale dataset can then be used to train deep neural networks models of the proprioceptive pathway to either classify the character <strong>(Action Recognition Task, ART)</strong> or to localize the hand <strong>(Trajectory Decoding Task, TDT)</strong> based on the input muscle spindle firing rates. These models can be analyzed, compared and contrasted to what is known about the proprioceptive system in primates.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F1.medium.gif" width="440" height="140"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F1.medium.gif" width="440" height="140"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                              <span class="fig-label">
                               Figure 1.
                              </span>
                              <span class="caption-title">
                               Contrasting spindle-based tasks to study proprioception:
                              </span>
                              <p id="p-9" class="first-child">
                               Proprioceptive inputs that correspond to the tracing of individual letters were simulated using a musculoskeletal model of a human arm. This scalable, large-scale dataset can then be used to train deep neural networks models of the proprioceptive pathway to either classify the character
                               <strong>
                                (Action Recognition Task, ART)
                               </strong>
                               or to localize the hand
                               <strong>
                                (Trajectory Decoding Task, TDT)
                               </strong>
                               based on the input muscle spindle firing rates. These models can be analyzed, compared and contrasted to what is known about the proprioceptive system in primates.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-10">
                             Large-scale datasets like ImageNet (
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              19
                             </a>
                             ) that present a challenging visual object-recognition task, have allowed the training of deep neural networks whose representations closely resemble tuning properties of single neurons in the ventral pathway of primates (
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              20
                             </a>
                             –
                             <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                              26
                             </a>
                             ). This goal-driven modelling approach (
                             <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ,
                             <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                              27
                             </a>
                             ,
                             <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                              28
                             </a>
                             ) has since successfully been applied to other sensory modalities such as touch (
                             <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                              29
                             </a>
                             ,
                             <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                              30
                             </a>
                             ), thermosensation (
                             <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                              31
                             </a>
                             ) and audition (
                             <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                              32
                             </a>
                             ). However, unlike for vision and audition, where large annotated datasets of raw images or sound are readily available, data of relevant proprioceptive stimuli (as well as task goals) are not.
                            </p>
                            <p id="p-11">
                             To create a large-scale passive movement dataset, we started from human motion data for drawing different Latin characters (
                             <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                              33
                             </a>
                             ). Next, a musculoskeletal model of the human upper limb (
                             <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                              34
                             </a>
                             ) is employed to generate muscle length configurations corresponding to drawing the pen-tip trajectories in multiple horizontal and vertical planes. These are ultimately converted into proprioceptive inputs using models of spindle Ia and II models (
                             <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                              35
                             </a>
                             ,
                             <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                              36
                             </a>
                             ). We then used the tasks to train families of neural networks to either decode the full trajectory of the hand-written characters, or classify the characters, both purely from the estimated spindle firing rates. Through an extensive hyper-parameter search we found neural networks for various architectures that optimally solve the task. We then analyzed those models and found that models trained on the action recognition, but not the trajectory decoding task more closely resemble what is known about tuning properties in the proprioceptive pathway. Collectively, we present a framework for studying the proprioceptive pathway using goal-driven modeling by synthesizing datasets of muscle (spindle) activities in order to test new theories of coding.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Results
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Spindle-based biomechanical character recognition task
                             </h3>
                             <p id="p-12">
                              To model the proprioceptive system we designed two real-world proprioceptive tasks. The objective is to either classify or reconstruct Latin alphabet characters based on the proprioceptive inputs that arise when the arm is passively moved (
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Figure 1
                              </a>
                              ). In this way, we effectively computationally isolate proprioception from active movement—a challenge in experimental work. To create this we used a dataset of pen-tip trajectories for the 20 characters that can be handwritten in a single stroke (thus excluding f, i, j, k, t and x, which are multi-stroke) (
                              <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">
                               33
                              </a>
                              ,
                              <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                               37
                              </a>
                              ). Then we generated one million end-effector (hand) trajectories by scaling, rotating, shearing, translating and varying the speed of each original trajectory (
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Figure 2A-C
                              </a>
                              ;
                              <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              ).
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F2.large.jpg?width=800&height=600&carousel=1" title="The proprioceptive character recognition dataset generation. (A) Multiple example pen-tip trajectories for five of the 20 letters are shown. (B) Same trajectories as in A, plotted as time courses of Cartesian coordinates. (C) Creating (hand) end-effector trajectories from pen-tip trajectories. (left) An example trajectory of character ‘a’ resized to fit in a 10cm x 10cm grid, linearly interpolated from the true trajectory while maintaining true velocity profile. (right) This trajectory is further transformed by scaling, rotating and varying its speed. (D) Candidate starting points to write the character in space. (left) A 2-link 4 degree of freedom (DoF) model human arm is used to randomly select several candidate starting points in the workspace of the arm (right), such that written characters are all strictly reachable by the arm. (E) (left to right and down) Given a sample trajectory in C and a starting point in the arm’s work-space, the trajectory is then drawn on either a vertical or horizontal plane that passes through the starting point. We then apply inverse kinematics to solve for the joint angles required to produce the traced trajectory. (F) (left to right): The joint angles obtained in E are used to drive a musculoskeletal model of the human arm in OpenSim, to obtain equilibrium muscle fiber-length trajectories of 25 relevant upper arm muscles. These muscle fiber-lengths and their instantaneous velocities together form the proprioceptive inputs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1989804583" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">The proprioceptive character recognition dataset generation.</span> (<strong>A</strong>) Multiple example pen-tip trajectories for five of the 20 letters are shown. (<strong>B</strong>) Same trajectories as in <strong>A</strong>, plotted as time courses of Cartesian coordinates. (<strong>C</strong>) Creating (hand) end-effector trajectories from pen-tip trajectories. (<em>left</em>) An example trajectory of character ‘a’ resized to fit in a 10cm x 10cm grid, linearly interpolated from the true trajectory while maintaining true velocity profile. (<em>right</em>) This trajectory is further transformed by scaling, rotating and varying its speed. (<strong>D</strong>) Candidate starting points to write the character in space. (<em>left</em>) A 2-link 4 degree of freedom (DoF) model human arm is used to randomly select several candidate starting points in the workspace of the arm (<em>right</em>), such that written characters are all strictly reachable by the arm. (<strong>E</strong>) (<em>left to right and down</em>) Given a sample trajectory in <strong>C</strong> and a starting point in the arm’s work-space, the trajectory is then drawn on either a vertical or horizontal plane that passes through the starting point. We then apply inverse kinematics to solve for the joint angles required to produce the traced trajectory. (<strong>F</strong>) (<em>left to right</em>): The joint angles obtained in <strong>E</strong> are used to drive a musculoskeletal model of the human arm in OpenSim, to obtain equilibrium muscle fiber-length trajectories of 25 relevant upper arm muscles. These muscle fiber-lengths and their instantaneous velocities together form the proprioceptive inputs.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F2.medium.gif" width="440" height="283"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F2.medium.gif" width="440" height="283"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2.
                               </span>
                               <span class="caption-title">
                                The proprioceptive character recognition dataset generation.
                               </span>
                               <p id="p-13" class="first-child">
                                (
                                <strong>
                                 A
                                </strong>
                                ) Multiple example pen-tip trajectories for five of the 20 letters are shown. (
                                <strong>
                                 B
                                </strong>
                                ) Same trajectories as in
                                <strong>
                                 A
                                </strong>
                                , plotted as time courses of Cartesian coordinates. (
                                <strong>
                                 C
                                </strong>
                                ) Creating (hand) end-effector trajectories from pen-tip trajectories. (left) An example trajectory of character ‘a’ resized to fit in a 10cm x 10cm grid, linearly interpolated from the true trajectory while maintaining true velocity profile. (right) This trajectory is further transformed by scaling, rotating and varying its speed. (
                                <strong>
                                 D
                                </strong>
                                ) Candidate starting points to write the character in space. (left) A 2-link 4 degree of freedom (DoF) model human arm is used to randomly select several candidate starting points in the workspace of the arm (right), such that written characters are all strictly reachable by the arm. (
                                <strong>
                                 E
                                </strong>
                                ) (left to right and down) Given a sample trajectory in
                                <strong>
                                 C
                                </strong>
                                and a starting point in the arm’s work-space, the trajectory is then drawn on either a vertical or horizontal plane that passes through the starting point. We then apply inverse kinematics to solve for the joint angles required to produce the traced trajectory. (
                                <strong>
                                 F
                                </strong>
                                ) (left to right): The joint angles obtained in
                                <strong>
                                 E
                                </strong>
                                are used to drive a musculoskeletal model of the human arm in OpenSim, to obtain equilibrium muscle fiber-length trajectories of 25 relevant upper arm muscles. These muscle fiber-lengths and their instantaneous velocities together form the proprioceptive inputs.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="T1" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2598813/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2598813/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2598813" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 1.
                               </span>
                               <p id="p-14" class="first-child">
                                Variable range for the utilized data augmentation applied to the original pen-tip trajectory dataset. Furthermore, the character trajectories are translated to start at various starting points throughout the arm’s workspace. Overall yielding movements in 26 horizontal and 18 vertical planes.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-15">
                              To translate end-effector trajectories into 3D arm movements we computed the joint-angle trajectories through inverse kinematics using a constrained optimization approach (
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Figure 2D-E
                              </a>
                              and methods). We iteratively constrain the solution space by choosing joint angles in vicinity of the previous configuration in order to eliminate redundancy. To cover a large 3D workspace we placed the characters in multiple horizontal (
                              <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">
                               26
                              </a>
                              ) and vertical (
                              <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                              ) planes and calculated corresponding joint-angle trajectories (starting points are illustrated in
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Figure 2D
                              </a>
                              ). A human upper-limb model in OpenSim (
                              <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ) was then used to compute equilibrium muscle lengths for 25 muscles in the upper arm that lead to the corresponding joint angle trajectory (
                              <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                               Figure 2F
                              </a>
                              , Suppl. Video). We did not include hand muscles for simplicity, therefore the location of the end-effector is taken to be the hand location.
                             </p>
                             <p id="p-16">
                              Based on these simulations, we generated proprioceptive inputs as muscle length and muscle velocity, which approximate receptor inputs during passive movement (see Methods). From this set, we selected a subset of two hundred thousand examples with smooth, non-jerky joint angle and muscle length changes, while making sure that the set is balanced in terms of the number of examples per class (see Methods). Since not all characters take the same amount of time to write, we padded the movements with static postures corresponding to the starting and ending postures of the movement and randomized the initiation of the movement in order to maintain ambiguity about when the writing begins. At the end of this process each sample consists of simulated proprioceptive inputs from each of the 25 muscles over a period of 4.8 seconds, simulated at 66.7 Hz. The dataset was split into a training, validation, and test set with a 72-8-20 ratio.
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Recognizing characters from muscle activity is challenging
                             </h3>
                             <p id="p-17">
                              We reasoned that several factors complicate the recognition of a specific character. Firstly, the end-effector position is only present as a distributed pattern of muscle activity. Secondly, the same character will give rise to widely different proprioceptive inputs depending on different arm configurations.
                             </p>
                             <p id="p-18">
                              To test these hypotheses, we first visualized the data at the level of proprioceptive inputs by using t-distributed stochastic neighbor embedding (t-SNE, 38). This illustrated that character identity was indeed entangled (
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Figure 3A
                              </a>
                              ). Then, we trained pairwise support vector machine (SVM) classifiers as baseline models for character recognition. Here, the influence of the specific geometry of each character is notable. On average the pairwise accuracy is 86.6 ± 12.5 (mean ± S.D., N = 190 pairs,
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Figure 3B
                              </a>
                              ). As expected, similar looking characters were harder to distinguish at the level of the proprioceptive input—i.e. “e” and “y” were easily distinguishable but “m” and “w” were not (
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Figure 3B
                              </a>
                              ).
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F3.large.jpg?width=800&height=600&carousel=1" title="Quantifying action recognition and decoding task performance. (A) t-SNE embedding of the end-effector coordinates (left) and proprioceptive inputs (right). (B) Classification performance for all pairs of characters with binary SVMs trained on proprioceptive inputs. Chance level accuracy=50%. The pairwise accuracy is 86.6 ± 12.5% (mean ± S.D., N = 190 pairs). Subset of data is also illustrated as circular graph, edge color denotes the classification accuracy. For clarity, only pairs with performance less than 70% are shown, which corresponds to the bottom 12% of all pairs. (C) Performance of baseline models: Multi-class SVM performance computed using a one-vs-one strategy for different types of input/kinematic representations on the action recognition task (left). Performance of ordinary least-squares linear regression on the trajectory decoding task (right). Note there is no point for end-effector coordinates as this is trivial. (D) Neural networks are trained on two tasks - proprioceptive action recognition and trajectory decoding based on proprioceptive inputs. We tested three neural network architecture families. Each model is comprised of one or more processing layers as shown here. Processing of spatial and temporal information takes place through a series of 1-D or 2-D convolutional layers or a recurrent LSTM layer. (E) Performance of neural network models on the tasks: test performance of the 50 networks of each type is plotted against the number of layers of processing in the networks for the action recognition (left) and trajectory decoding (center) tasks separately, and against each other (right). Note we jittered the number of layers for visibility, but per model it is discrete." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1989804583" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Quantifying action recognition and decoding task performance.</span> (<strong>A</strong>) t-SNE embedding of the end-effector coordinates (left) and proprioceptive inputs (right). (<strong>B</strong>) Classification performance for all pairs of characters with binary SVMs trained on proprioceptive inputs. Chance level accuracy=50%. The pairwise accuracy is 86.6 <em>±</em> 12.5% (mean <em>±</em> S.D., <em>N</em> = 190 pairs). Subset of data is also illustrated as circular graph, edge color denotes the classification accuracy. For clarity, only pairs with performance less than 70% are shown, which corresponds to the bottom 12% of all pairs. (<strong>C</strong>) Performance of baseline models: Multi-class SVM performance computed using a one-vs-one strategy for different types of input/kinematic representations on the action recognition task (left). Performance of ordinary least-squares linear regression on the trajectory decoding task (right). Note there is no point for end-effector coordinates as this is trivial. (<strong>D</strong>) Neural networks are trained on two tasks - proprioceptive action recognition and trajectory decoding based on proprioceptive inputs. We tested three neural network architecture families. Each model is comprised of one or more processing layers as shown here. Processing of spatial and temporal information takes place through a series of 1-D or 2-D convolutional layers or a recurrent LSTM layer. (<strong>E</strong>) Performance of neural network models on the tasks: test performance of the 50 networks of each type is plotted against the number of layers of processing in the networks for the action recognition (left) and trajectory decoding (center) tasks separately, and against each other (right). Note we jittered the number of layers for visibility, but per model it is discrete.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F3.medium.gif" width="404" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F3.medium.gif" width="404" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3.
                               </span>
                               <span class="caption-title">
                                Quantifying action recognition and decoding task performance.
                               </span>
                               <p id="p-19" class="first-child">
                                (
                                <strong>
                                 A
                                </strong>
                                ) t-SNE embedding of the end-effector coordinates (left) and proprioceptive inputs (right). (
                                <strong>
                                 B
                                </strong>
                                ) Classification performance for all pairs of characters with binary SVMs trained on proprioceptive inputs. Chance level accuracy=50%. The pairwise accuracy is 86.6 ± 12.5% (mean ± S.D., N = 190 pairs). Subset of data is also illustrated as circular graph, edge color denotes the classification accuracy. For clarity, only pairs with performance less than 70% are shown, which corresponds to the bottom 12% of all pairs. (
                                <strong>
                                 C
                                </strong>
                                ) Performance of baseline models: Multi-class SVM performance computed using a one-vs-one strategy for different types of input/kinematic representations on the action recognition task (left). Performance of ordinary least-squares linear regression on the trajectory decoding task (right). Note there is no point for end-effector coordinates as this is trivial. (
                                <strong>
                                 D
                                </strong>
                                ) Neural networks are trained on two tasks - proprioceptive action recognition and trajectory decoding based on proprioceptive inputs. We tested three neural network architecture families. Each model is comprised of one or more processing layers as shown here. Processing of spatial and temporal information takes place through a series of 1-D or 2-D convolutional layers or a recurrent LSTM layer. (
                                <strong>
                                 E
                                </strong>
                                ) Performance of neural network models on the tasks: test performance of the 50 networks of each type is plotted against the number of layers of processing in the networks for the action recognition (left) and trajectory decoding (center) tasks separately, and against each other (right). Note we jittered the number of layers for visibility, but per model it is discrete.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-20">
                              To quantify the separability between all characters we used a one-against-one strategy with the trained pairwise classifiers (
                              <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                               39
                              </a>
                              ). The performance of this multi-class decoder was poor regardless of whether the input was end-effector coordinates, joint angles, normalized muscle lengths, or proprioceptive inputs (
                              <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                               Figure 3C
                              </a>
                              ). Taken together, these analyses highlight that it is difficult to extract the character class from those representations. Collectively, we demonstrated that the action recognition task is challenging as illustrated by t-SNE embedding (
                              <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                               Figure 3A
                              </a>
                              ) and quantified by SVMs (
                              <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                               Figure 3B, C
                              </a>
                              ). In contrast, as expected, accurately decoding the end-effector position (by linear regression) from the proprioceptive input is much simpler, with an average decoding error of 1.72cm, in a 3D workspace approximately 90×90×120cm
                              <sup>
                               3
                              </sup>
                              (
                              <a id="xref-fig-3-7" class="xref-fig" href="#F3">
                               Figure 3C
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Neural networks models of proprioception
                             </h3>
                             <p id="p-21">
                              We explore the ability of three types of artificial neural network models (ANNs) to solve the proprioceptive character recognition and decoding tasks. ANNs are powerful models for both their performance and for elucidating neural representations and computations (
                              <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ,
                              <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                               40
                              </a>
                              ). An ANN consists of layers of simplified units (“neurons”) whose connectivity patterns mimic the hierarchical, integrative properties of biological neurons and anatomical pathways (
                              <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ,
                              <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">
                               27
                              </a>
                              ,
                              <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                               41
                              </a>
                              ). As candidate models we parameterized a (spatio)temporal convolutional neural network, a spatial-temporal convolutional network (both TCNs, 42), and a recurrent neural network (a long short-term memory (LSTM) model, 43), which impose different inductive priors on the computations. We refer to these three types as spatial-temporal, spatiotemporal and LSTM networks (
                              <a id="xref-fig-3-8" class="xref-fig" href="#F3">
                               Figure 3D
                              </a>
                              ).
                             </p>
                             <p id="p-22">
                              Importantly, the different models differ in the way they integrate spatial and temporal information along the hierarchy. These two types of information can be processed either sequentially, as is the case for the spatial-temporal network type that contains layers with one-dimensional filters that first integrate information across the different muscles, followed by an equal number of layers that integrate only in the temporal direction or simultaneously, using two-dimensional kernels, as they are in the spatiotemporal network. In the LSTM networks, spatial information was integrated similarly to the spatial-temporal networks, before entering the LSTM layer.
                             </p>
                             <p id="p-23">
                              Candidate models for each class can be created by varying hyper-parameters such as the number of layers, number and size of spatial and temporal filters, type of regularization and response normalization, among others (see
                              <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              , Methods). As a first step to restrict the number of models, we performed a hyper-parameter architecture search by selecting models according to their performance on the proprioceptive tasks. We should emphasize that our ANNs are simultaneously integrating proprioceptive inputs and time, unlike standard feed-forward CNN-models of the visual pathway that just operate on images (
                              <a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ). TCNs have been shown to be excellent for time-series modeling (
                              <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                               44
                              </a>
                              ), and therefore naturally describe neurons along a sensory pathway that integrate spatio-temporal inputs.
                             </p>
                             <div id="T2" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2598802/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2598802/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2598802" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 2.
                               </span>
                               <span class="caption-title">
                                Hyper-parameters for neural network architecture search.
                               </span>
                               <p id="p-24" class="first-child">
                                To form candidate networks, first a number of layers (per type) is chosen, ranging from 2-8 (in multiples of 2) for spatial-temporal models and 1-4 for the spatiotemporal and LSTM ones. Next, a spatial and temporal kernel size per Layer is picked where relevant, which remains unchanged throughout the network. For the spatiotemporal model, the kernel size is equal in both the spatial and temporal directions in each layer. Then, for each layer, an associated number of kernels/feature maps is chosen such that it never decreases along the hierarchy. Finally, a spatial and temporal stride is chosen. For the LSTM networks, the number of recurrent units is also chosen. All parameters are randomized independently and 50 models are sampled per network type.
                                <strong>
                                 Columns 2-4
                                </strong>
                                Hyper-parameter values for the top-performing models in the ART. The values given under the “spatial” rows count for both the spatial and temporal directions for the spatiotemporal model.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Architecture search & representational changes
                             </h3>
                             <p id="p-25">
                              To find models that could solve the proprioceptive tasks, we performed an architecture search and trained 150 models (50 models per type). Notably, we trained the same model (as specified by architectural parameters) on both tasks by modifying the output and the loss function used to train the model. All networks were trained using the Adam optimizer (
                              <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                               45
                              </a>
                              ) until performance on the validation set saturated. After training, all models were evaluated on a unseen test set (
                              <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                               Figure S1A
                              </a>
                              ).
                             </p>
                             <div id="F8" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F8.large.jpg?width=800&height=600&carousel=1" title="Network Performance. (A) Training vs. test performance for all networks. Shallower networks tend to overfit more. (B) Performance of networks is plotted against the all parameters of the networks. Note: parameters of the final (fully connected) layer are not counted." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1989804583" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Network Performance.</span> (<strong>A</strong>) Training vs. test performance for all networks. Shallower networks tend to overfit more. (<strong>B</strong>) Performance of networks is plotted against the all parameters of the networks. Note: parameters of the final (fully connected) layer are not counted.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F8.medium.gif" width="440" height="137"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S1." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F8.medium.gif" width="440" height="137"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S1.
                               </span>
                               <span class="caption-title">
                                Network Performance.
                               </span>
                               <p id="p-119" class="first-child">
                                (
                                <strong>
                                 A
                                </strong>
                                ) Training vs. test performance for all networks. Shallower networks tend to overfit more. (
                                <strong>
                                 B
                                </strong>
                                ) Performance of networks is plotted against the all parameters of the networks. Note: parameters of the final (fully connected) layer are not counted.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-26">
                              Models of all three types achieved excellent performance on the action recognition task (ART) (
                              <a id="xref-fig-3-9" class="xref-fig" href="#F3">
                               Figure 3E
                              </a>
                              ; multi-class accuracy of 98.86% ± 0.04, Mean ± SEM for the best spatial-temporal model, 97.93% ± 0.03 for the best spatiotemporal model, and 99.56% ± 0.04 for the best LSTM model, N = 5 randomly initialized models). The parameters of the best performing architectures are displayed in
                              <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              . The same models could also accurately solve the trajectory decoding task (TDT) (
                              <a id="xref-fig-3-10" class="xref-fig" href="#F3">
                               Figure 3E
                              </a>
                              ; with decoding errors of only 0.22 cm ± 0.005, Mean ± SEM for the best spatial-temporal model, .13 cm ± 0.003 for the best spatiotemporal model, and 0.05 cm ± 0.01 for the best LSTM model, N = 5 randomly initialized models). Of the hyper-parameters considered, the depth of the networks influenced performance the most (
                              <a id="xref-fig-3-11" class="xref-fig" href="#F3">
                               Figures 3E
                              </a>
                              ,
                              <a id="xref-fig-8-2" class="xref-fig" href="#F8">
                               S1B
                              </a>
                              ). Further, the performance on the two tasks were related: models performing well on one task tend to perform well on the other 3E.
                             </p>
                             <p id="p-27">
                              Having found models that robustly solve the ART and TDT, we sought to analyze their properties. We created five pretraining (control) and post-training (trained) pairs of models for the best-performing model architecture for further analysis. We will refer to those as “instantiations”. As expected, the randomly initialized models performed at chance level (5%) on the ART.
                             </p>
                             <p id="p-28">
                              How did the population activity change across the layers after learning the tasks? Initially, we focus on the best spatial-temporal model and then show that our analysis extends to the other model types. We compared the representations across different layers for each trained model to its random initialization by linear Centered Kernel Alignment (CKA, see Methods). This analysis revealed that for all instantiations the representations remained similar between the trained and control models for the first few layers and then deviate in the middle to final layers of the network (
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Figure 4A
                              </a>
                              ). Furthermore, trained models not only differed from the random initialization, but also across tasks, and the divergence appeared earlier (
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Figure 4A
                              </a>
                              ). Therefore, we found that both learning and the task substantially change the representations. Next, we aimed to understand how the tasks are solved, i.e., how the different stimuli are transformed across the hierarchy.
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F4.large.jpg?width=800&height=600&carousel=1" title="Low-dimensional embedding of network layers reveals structure. (A) Similarity in representations (CKA) between the trained and the control models for each of the five instantiations of the best performing spatial-temporal models (left and center). CKA between models trained on recognition vs decoding (right). (B) t-distributed stochastic neighbor embedding (t-SNE) for each layer of one instantiation of the best performing spatial-temporal model trained on both tasks. Each data point is a random stimulus sample (N = 2, 000, 50 per stimulus). (C) Representational Dissimilarity Matrices (RDM). Character level representation are calculated through percentile representational dissimilarity matrices for proprioceptive inputs (left) and final layer features (right) of one instantiation of the best performing spatio-temporal model trained on recognition task. (D) Similarity in stimulus representations between representational dissimilarity matrices of an Oracle (ideal observer) and each layer for the five instantiations of the ART-trained models and their controls. (E) Decoding error (in cm) along the hierarchy for each model type trained on the decoding task. (F) CKA between models trained on recognition vs decoding for the five instantiations of all network types (right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1989804583" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Low-dimensional embedding of network layers reveals structure.</span> (<strong>A</strong>) Similarity in representations (CKA) between the trained and the control models for each of the five instantiations of the best performing spatial-temporal models (left and center). CKA between models trained on recognition vs decoding (right). (<strong>B</strong>) t-distributed stochastic neighbor embedding (t-SNE) for each layer of one instantiation of the best performing spatial-temporal model trained on both tasks. Each data point is a random stimulus sample (<em>N</em> = 2, 000, 50 per stimulus). (<strong>C</strong>) Representational Dissimilarity Matrices (RDM). Character level representation are calculated through percentile representational dissimilarity matrices for proprioceptive inputs (left) and final layer features (right) of one instantiation of the best performing spatio-temporal model trained on recognition task. (<strong>D</strong>) Similarity in stimulus representations between representational dissimilarity matrices of an Oracle (ideal observer) and each layer for the five instantiations of the ART-trained models and their controls. (<strong>E</strong>) Decoding error (in cm) along the hierarchy for each model type trained on the decoding task. (<strong>F</strong>) CKA between models trained on recognition vs decoding for the five instantiations of all network types (right).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F4.medium.gif" width="440" height="409"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F4.medium.gif" width="440" height="409"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4.
                               </span>
                               <span class="caption-title">
                                Low-dimensional embedding of network layers reveals structure.
                               </span>
                               <p id="p-29" class="first-child">
                                (
                                <strong>
                                 A
                                </strong>
                                ) Similarity in representations (CKA) between the trained and the control models for each of the five instantiations of the best performing spatial-temporal models (left and center). CKA between models trained on recognition vs decoding (right). (
                                <strong>
                                 B
                                </strong>
                                ) t-distributed stochastic neighbor embedding (t-SNE) for each layer of one instantiation of the best performing spatial-temporal model trained on both tasks. Each data point is a random stimulus sample (N = 2, 000, 50 per stimulus). (
                                <strong>
                                 C
                                </strong>
                                ) Representational Dissimilarity Matrices (RDM). Character level representation are calculated through percentile representational dissimilarity matrices for proprioceptive inputs (left) and final layer features (right) of one instantiation of the best performing spatio-temporal model trained on recognition task. (
                                <strong>
                                 D
                                </strong>
                                ) Similarity in stimulus representations between representational dissimilarity matrices of an Oracle (ideal observer) and each layer for the five instantiations of the ART-trained models and their controls. (
                                <strong>
                                 E
                                </strong>
                                ) Decoding error (in cm) along the hierarchy for each model type trained on the decoding task. (
                                <strong>
                                 F
                                </strong>
                                ) CKA between models trained on recognition vs decoding for the five instantiations of all network types (right).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-30">
                              To illustrate the geometry of the ANN representations and how the different characters are disentangled across the hierarchy we used t-SNE to visualize the structure. For the ART, the different characters remain entangled in the representations throughout most of the processing hierarchy before separating in the final layers (spatial-temporal model:
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               Figure 4B
                              </a>
                              and for the other model classes S2A). To quantify this, we computed representational dissimilarity matrices (RDM; see Methods). We found that different instances for the same characters were not represented similarly at the level of proprioceptive inputs, but at at the level of the last convolutional layer for the trained models (
                              <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                               Figure 4C
                              </a>
                              and for other model classes S2B). To quantify how the characters are represented across the hierarchy, we computed the similarity to an Oracle’s RDM, where an Oracle (or ideal observer) would have a block structure, with dissimilarity 0 for all stimuli of the same class and 1 (100th percentile) otherwise (
                              <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                               Figure S2B
                              </a>
                              ). We found for all model instantiations similarity only increased towards the last layers (
                              <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                               Figure 4D
                              </a>
                              ). This finding corroborates the visual impression gained via t-SNE that different characters are disentangled near the end of the processing hierarchy (
                              <a id="xref-fig-4-6" class="xref-fig" href="#F4">
                               Figures 4B
                              </a>
                              ,
                              <a id="xref-fig-9-2" class="xref-fig" href="#F9">
                               S2A, C
                              </a>
                              ).
                             </p>
                             <div id="F9" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F9.large.jpg?width=800&height=600&carousel=1" title="Extended analysis of network models. (A) t-distributed stochastic neighbor embedding (t-SNE) embedding for each layer of the best spatiotemporal and LSTM model. Each data point is a random stimulus sample (N=4,000, 200 per character). (B) Representational Dissimilarity Matrices (RDM) of an ideal observer “Oracle”, which by definition has low dissimilarity for different samples of the same character and high dissimilarity for different samples of different characters. Character level representation are calculated through percentile representational dissimilarity matrices for proprioceptive inputs and final layer features of one instantiation of the best performing spatiotemporal and LSTM model trained on recognition task.(C) CKA between models trained on recognition vs decoding for all network types (N = 50 per network type)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1989804583" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Extended analysis of network models.</span> (<strong>A</strong>) t-distributed stochastic neighbor embedding (t-SNE) embedding for each layer of the best spatiotemporal and LSTM model. Each data point is a random stimulus sample (N=4,000, 200 per character). (<strong>B</strong>) Representational Dissimilarity Matrices (RDM) of an ideal observer “Oracle”, which by definition has low dissimilarity for different samples of the same character and high dissimilarity for different samples of different characters. Character level representation are calculated through percentile representational dissimilarity matrices for proprioceptive inputs and final layer features of one instantiation of the best performing spatiotemporal and LSTM model trained on recognition task.(<strong>C</strong>) CKA between models trained on recognition vs decoding for all network types (<em>N</em> = 50 per network type).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F9.medium.gif" width="440" height="297"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S2." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F9.medium.gif" width="440" height="297"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S2.
                               </span>
                               <span class="caption-title">
                                Extended analysis of network models.
                               </span>
                               <p id="p-120" class="first-child">
                                (
                                <strong>
                                 A
                                </strong>
                                ) t-distributed stochastic neighbor embedding (t-SNE) embedding for each layer of the best spatiotemporal and LSTM model. Each data point is a random stimulus sample (N=4,000, 200 per character). (
                                <strong>
                                 B
                                </strong>
                                ) Representational Dissimilarity Matrices (RDM) of an ideal observer “Oracle”, which by definition has low dissimilarity for different samples of the same character and high dissimilarity for different samples of different characters. Character level representation are calculated through percentile representational dissimilarity matrices for proprioceptive inputs and final layer features of one instantiation of the best performing spatiotemporal and LSTM model trained on recognition task.(
                                <strong>
                                 C
                                </strong>
                                ) CKA between models trained on recognition vs decoding for all network types (N = 50 per network type).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-31">
                              How is the TDT solved across the hierarchy? In contrast to the ART trained models, as expected, representations of characters remained entangled throughout (
                              <a id="xref-fig-4-7" class="xref-fig" href="#F4">
                               Figure 4B
                              </a>
                              ,
                              <a id="xref-fig-9-3" class="xref-fig" href="#F9">
                               S2A
                              </a>
                              ). We found that the end-effector position can be decoded across the hierarchy (
                              <a id="xref-fig-4-8" class="xref-fig" href="#F4">
                               Figure 4E
                              </a>
                              ). This result is expected, as even from the proprioceptive input a linear readout achieves good performance (
                              <a id="xref-fig-3-12" class="xref-fig" href="#F3">
                               Figure 3C
                              </a>
                              ). Finally, we quantified CKA scores across the different architecture classes and found that with increasing depth the representations diverge between the two tasks (
                              <a id="xref-fig-4-9" class="xref-fig" href="#F4">
                               Figures 4F
                              </a>
                              ,
                              <a id="xref-fig-9-4" class="xref-fig" href="#F9">
                               S2C
                              </a>
                              ). Collectively, this suggests that characters are not immediately separable in ART-models, but the end-effector can be well decoded in TDT-models throughout the architecture.
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Single unit tuning properties
                             </h3>
                             <p id="p-32">
                              To gain insight into why ART- and TDT-trained models differ in their representations, we examined single unit tuning properties. In the primate these have been well described (
                              <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                              ,
                              <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                              ), thus present as an ideal comparison point. Specifically, we analyzed the units for end-effector position, speed, direction, velocity, and acceleration tuning. We performed these analyses by relating variables (such as movement direction) to the activity of single units during the continuous movement (see Methods). Units with a test-R
                              <sup>
                               2
                              </sup>
                              > 0.2 were considered “tuned” to that feature (this is a conservative value in comparison to experimental studies, e.g., 0.07 for (
                              <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                              )).
                             </p>
                             <p id="p-33">
                              Given the precedence in the literature, we focused on direction tuning in all horizontal planes. We fit directional tuning curves to the units with respect to the instantaneous movement direction. As illustrated in examples, the ART spatial-temporal model (as well as proprioceptive inputs), directional tuning can be observed for the typical units shown (
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Figure 5A, B
                              </a>
                              ). Spindle afferents are known to be tuned to motion, i.e. velocity and direction (
                              <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                               46
                              </a>
                              ). We verified the tuning of the spindles and found that the spindle component, tuned for muscle length, is primarily tuned for position (median R
                              <sup>
                               2
                              </sup>
                              = 0.40, N = 25) rather than kinematics (median direction R
                              <sup>
                               2
                              </sup>
                              = 0.0031, median velocity R
                              <sup>
                               2
                              </sup>
                              = 0.0026, N = 25), whereas the spindle component tuned for changes in muscle length were primarily tuned for kinematics (median direction R
                              <sup>
                               2
                              </sup>
                              = 0.55, velocity R
                              <sup>
                               2
                              </sup>
                              = 0.81, N = 25), and only poorly tuned for position (median R
                              <sup>
                               2
                              </sup>
                              = 0.34, N = 25). For the ART model, direction selectivity was prominent in middle layers 1-6 before decreasing by layer 8, and a fractions of units exhibited tuning to other kinematic variables with R
                              <sup>
                               2
                              </sup>
                              > 0.2 (
                              <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                               Figures 5C
                              </a>
                              ,
                              <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                               S3A,B
                              </a>
                              ).
                             </p>
                             <div id="F10" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F10.large.jpg?width=800&height=600&carousel=1" title="Kinematic tuning of single neurons. (A) For an example spatial-temporal model instantiation, the distribution of test R2 scores for both the ART- and TDT-trained models are shown, for direction, speed, velocity, acceleration, and labels. (8/3890 scores excluded over all layers for ART-trained, 284/3890 for TDT-trained; see Methods). (B) The individual traces (faint) as well as the means (dark) of 90%-quantiles over all five model instantiations of models trained on action recognition and trajectory decoding are shown for direction tuning (solid line) and acceleration tuning (dashed line). (C, D) Same as A,B but for the spatiotemporal model. (0/2330 scores excluded for ART-trained, 132/2330 for TDT; see Methods). (E, F) Same as A,B but for the LSTM model.(4/6530 scores excluded for ART-trained, 1024/6530 for TDT-trained; see Methods)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1989804583" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Kinematic tuning of single neurons.</span> (<strong>A</strong>) For an example spatial-temporal model instantiation, the distribution of test <em>R</em><sup>2</sup> scores for both the ART- and TDT-trained models are shown, for direction, speed, velocity, acceleration, and labels. (8/3890 scores excluded over all layers for ART-trained, 284/3890 for TDT-trained; see Methods). (<strong>B</strong>) The individual traces (faint) as well as the means (dark) of 90%-quantiles over all five model instantiations of models trained on action recognition and trajectory decoding are shown for direction tuning (solid line) and acceleration tuning (dashed line). (<strong>C</strong>, D) Same as A,B but for the spatiotemporal model. (0/2330 scores excluded for ART-trained, 132/2330 for TDT; see Methods). (<strong>E</strong>, F) Same as A,B but for the LSTM model.(4/6530 scores excluded for ART-trained, 1024/6530 for TDT-trained; see Methods).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F10.medium.gif" width="440" height="324"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S3." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F10.medium.gif" width="440" height="324"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S3.
                               </span>
                               <span class="caption-title">
                                Kinematic tuning of single neurons.
                               </span>
                               <p id="p-121" class="first-child">
                                (
                                <strong>
                                 A
                                </strong>
                                ) For an example spatial-temporal model instantiation, the distribution of test R
                                <sup>
                                 2
                                </sup>
                                scores for both the ART- and TDT-trained models are shown, for direction, speed, velocity, acceleration, and labels. (8/3890 scores excluded over all layers for ART-trained, 284/3890 for TDT-trained; see Methods). (
                                <strong>
                                 B
                                </strong>
                                ) The individual traces (faint) as well as the means (dark) of 90%-quantiles over all five model instantiations of models trained on action recognition and trajectory decoding are shown for direction tuning (solid line) and acceleration tuning (dashed line). (
                                <strong>
                                 C
                                </strong>
                                , D) Same as A,B but for the spatiotemporal model. (0/2330 scores excluded for ART-trained, 132/2330 for TDT; see Methods). (
                                <strong>
                                 E
                                </strong>
                                , F) Same as A,B but for the LSTM model.(4/6530 scores excluded for ART-trained, 1024/6530 for TDT-trained; see Methods).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F5.large.jpg?width=800&height=600&carousel=1" title="Analysis of single unit tuning properties for spatial-temporal models. (A) Polar scatter plots showing the activation of units (radius r) as a function of end-effector direction as represented by the angle θ for directionally tuned units in different layers of the top-performing spatial temporal model trained on the action recognition task. Directions correspond to that of the end-effector while tracing characters in the model workspace. The activation strengths of one (velocity-dependent) muscle spindle, one unit in layer 3, 5 and 8 of the are shown. (B) Similar to A, except that now radius describes velocity, and color represents activation strength. The contours are determined following linear interpolation, with gaps filled in by neighbor interpolation, and results smoothed using a Gaussian filter. Examples of one muscle spindle, one unit in layer 3, 5 and 8 are shown. (C) For each layer of one trained instantiation, the units are classified into types based on their tuning. A unit was classified as belonging to a particular type if its tuning had a test R2 > 0.2. Tested features were direction tuning, speed tuning, velocity tuning, Cartesian and polar position tuning, acceleration tuning, and label tuning (14/3890 scores excluded for ART-trained, 291/3890 for TDT-trained; see Methods). (D) The same plot but for the spatial-temporal model of the same architecture but trained on the trajectory decoding task. (E) For an example instantiation, the distribution of test R2 scores for both the ART and TDT trained models are shown, for five kinds of kinematic tuning for each layer: direction tuning, speed tuning, Cartesian position tuning, polar position tuning, and label-specificity indicated by different shades and arranged left-right for each layer including spindles. The solid line connects the 90%-quantiles of two of the tuning curve types, direction tuning (dark) and position tuning (light). Tuning scores were excluded if they were equal to 1, indicating a constant neuron, or less than − 0.1, indicating an improper fit (8 scores excluded; see Methods). (F) The means of 90%-quantiles over all five model instantiations of models trained on ART and TDT are shown for direction tuning (dark) and position tuning (light). 95%-confidence intervals are shown over instantiations (N = 5)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1989804583" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Analysis of single unit tuning properties for spatial-temporal models.</span> (<strong>A</strong>) Polar scatter plots showing the activation of units (radius <em>r</em>) as a function of end-effector direction as represented by the angle <em>θ</em> for directionally tuned units in different layers of the top-performing spatial temporal model trained on the action recognition task. Directions correspond to that of the end-effector while tracing characters in the model workspace. The activation strengths of one (velocity-dependent) muscle spindle, one unit in layer 3, 5 and 8 of the are shown. (<strong>B</strong>) Similar to A, except that now radius describes velocity, and color represents activation strength. The contours are determined following linear interpolation, with gaps filled in by neighbor interpolation, and results smoothed using a Gaussian filter. Examples of one muscle spindle, one unit in layer 3, 5 and 8 are shown. (<strong>C</strong>) For each layer of one trained instantiation, the units are classified into types based on their tuning. A unit was classified as belonging to a particular type if its tuning had a test <em>R</em><sup>2</sup> <em>></em> 0.2. Tested features were direction tuning, speed tuning, velocity tuning, Cartesian and polar position tuning, acceleration tuning, and label tuning (14/3890 scores excluded for ART-trained, 291/3890 for TDT-trained; see Methods). (<strong>D</strong>) The same plot but for the spatial-temporal model of the same architecture but trained on the trajectory decoding task. (<strong>E</strong>) For an example instantiation, the distribution of test <em>R</em><sup>2</sup> scores for both the ART and TDT trained models are shown, for five kinds of kinematic tuning for each layer: direction tuning, speed tuning, Cartesian position tuning, polar position tuning, and label-specificity indicated by different shades and arranged left-right for each layer including spindles. The solid line connects the 90%-quantiles of two of the tuning curve types, direction tuning (dark) and position tuning (light). Tuning scores were excluded if they were equal to 1, indicating a constant neuron, or less than − 0.1, indicating an improper fit (8 scores excluded; see Methods). (<strong>F</strong>) The means of 90%-quantiles over all five model instantiations of models trained on ART and TDT are shown for direction tuning (dark) and position tuning (light). 95%-confidence intervals are shown over instantiations (<em>N</em> = 5).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F5.medium.gif" width="440" height="335"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F5.medium.gif" width="440" height="335"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5.
                               </span>
                               <span class="caption-title">
                                Analysis of single unit tuning properties for spatial-temporal models.
                               </span>
                               <p id="p-34" class="first-child">
                                (
                                <strong>
                                 A
                                </strong>
                                ) Polar scatter plots showing the activation of units (radius r) as a function of end-effector direction as represented by the angle θ for directionally tuned units in different layers of the top-performing spatial temporal model trained on the action recognition task. Directions correspond to that of the end-effector while tracing characters in the model workspace. The activation strengths of one (velocity-dependent) muscle spindle, one unit in layer 3, 5 and 8 of the are shown. (
                                <strong>
                                 B
                                </strong>
                                ) Similar to A, except that now radius describes velocity, and color represents activation strength. The contours are determined following linear interpolation, with gaps filled in by neighbor interpolation, and results smoothed using a Gaussian filter. Examples of one muscle spindle, one unit in layer 3, 5 and 8 are shown. (
                                <strong>
                                 C
                                </strong>
                                ) For each layer of one trained instantiation, the units are classified into types based on their tuning. A unit was classified as belonging to a particular type if its tuning had a test R
                                <sup>
                                 2
                                </sup>
                                > 0.2. Tested features were direction tuning, speed tuning, velocity tuning, Cartesian and polar position tuning, acceleration tuning, and label tuning (14/3890 scores excluded for ART-trained, 291/3890 for TDT-trained; see Methods). (
                                <strong>
                                 D
                                </strong>
                                ) The same plot but for the spatial-temporal model of the same architecture but trained on the trajectory decoding task. (
                                <strong>
                                 E
                                </strong>
                                ) For an example instantiation, the distribution of test R
                                <sup>
                                 2
                                </sup>
                                scores for both the ART and TDT trained models are shown, for five kinds of kinematic tuning for each layer: direction tuning, speed tuning, Cartesian position tuning, polar position tuning, and label-specificity indicated by different shades and arranged left-right for each layer including spindles. The solid line connects the 90%-quantiles of two of the tuning curve types, direction tuning (dark) and position tuning (light). Tuning scores were excluded if they were equal to 1, indicating a constant neuron, or less than − 0.1, indicating an improper fit (8 scores excluded; see Methods). (
                                <strong>
                                 F
                                </strong>
                                ) The means of 90%-quantiles over all five model instantiations of models trained on ART and TDT are shown for direction tuning (dark) and position tuning (light). 95%-confidence intervals are shown over instantiations (N = 5).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-35">
                              In contrast, for the TDT model, no directional tuning was observed, but positional tuning (
                              <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                               Figure 5D
                              </a>
                              ). These observations are further corroborated when comparing the distributions of tuning properties (
                              <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                               Figure 5E
                              </a>
                              ) and 90%-quantiles for all the instantiations (
                              <a id="xref-fig-5-5" class="xref-fig" href="#F5">
                               Figure 5F
                              </a>
                              ). The difference in median tuning score between the two differently trained groups of models across the five model instantiations becomes significant starting in the first layer for both direction (layer 1 t(4) = 13.07, p=0.0002; layer 2 t(4) = 15.79, p=0.0001; layer 3 t(4) = 39.74, p=2.65e-06; layer 4 t(4) = 5.24, p=5.24e-06; layer 5 t(4) = 35.76, p=3.65e-06; layer 6 t(4) = 12.97, p=0.0002; layer 7 t(4) = 3.605, p=0.017; t(4) = 19.35, p=4.21e-05) and position (layer 1 t(4) = −11.21, p=0.0004; layer 2 t(4) = −28.59, p=8.91e-06; layer 3 t(4) = −21.27, p=2.89e-05; layer 4 t(4) = −13.91, p=0.0002; layer 5 t(4) = −18.55, p=4.97e-05; layer 6 t(4) = −61.59, p=4.16e-07; layer 7 t(4) = −24.08, p=1.77e-05; t(4) = −25.80, p=1.34e-05).
                             </p>
                             <p id="p-36">
                              Given that the ART models are trained to recognize characters, we asked if single units are well tuned for specific characters. To test this we trained an SVM to classify characters from the single unit activations. Even in the final layer (before the readout) of the spatial-temporal model, the median classification performance over the five model instantiations as measured by the normalized area under the ROC curve-based selectivity index for single units was 0.212 ± 0.005 (mean ± SEM, N = 5 instantiations), and was never higher than 0.41 for any individual unit across all model instantiations (see Methods). Thus, even in the final layer there are effectively no single-character specific units. Of course, combining the different units of the final fully connected layer gives a high fidelity readout of the character and allows the model to achieve high classification accuracy. Thus, character identity is represented in a distributed way. In contrast, and as expected, character identity is poorly encoded in single cells for the TDT model (
                              <a id="xref-fig-5-6" class="xref-fig" href="#F5">
                               Figure 5D,F
                              </a>
                              ).
                             </p>
                             <p id="p-37">
                              These main results also hold for the other architecture classes. In spatiotemporal models, in which both spatial and temporal processing occurs between all layers, we observe a monotonic decrease in the directional tuning across the four layers for the ART task and a quick decay for the TDT task (
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Figure 6 A, B
                              </a>
                              ). Speed and acceleration tuning are present in the ART, but not in the TDT models (
                              <a id="xref-fig-10-2" class="xref-fig" href="#F10">
                               Figure S3B, C
                              </a>
                              ). Conversely, we find that positional coding is stable for TDT models and not the ART models. The same results hold true for LSTM model (
                              <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                               Figures 6 C, D
                              </a>
                              ,
                              <a id="xref-fig-10-3" class="xref-fig" href="#F10">
                               S3E, F
                              </a>
                              ). The differences in directional and Cartesian positional tuning were statistically significant for all layers according to a paired t-test with 4 degrees of freedom for both model types. Thus, for all architecture classes we find that strong direction selective tuning is present in early layers of models trained with the ART task, but not the TDT task.
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F6.large.jpg?width=800&height=600&carousel=1" title="Analysis of single unit tuning properties for spatiotemporal and LSTM models. (A) For an example instantiation of the top-performing spatiotemporal model, the distribution of test R2 scores for both the trained and control model are shown, for five kinds of kinematic tuning for each layer: direction tuning, speed tuning, Cartesian position tuning, polar position tuning, and label-specificity. The solid line connects the 90%-quantiles of two of the tuning curve types, direction tuning (dark) and position tuning (light). (0/2330 scores excluded summed over all layers for ART-trained, 129/2330 for TDT; see Methods). (B) The means of 90%-quantiles over all five model instantiations of models trained on action recognition and trajectory decoding are shown for direction tuning (dark) and position tuning (light). 95%-confidence intervals are shown over instantiations (N = 5). (C) The same plot as in (A) but for the top-performing LSTM model (0/6530 scores excluded for ART-trained, 1024/6530 for TDT; see Methods). (D) The same plot as B, for the LSTM model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1989804583" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Analysis of single unit tuning properties for spatiotemporal and LSTM models.</span> (<strong>A</strong>) For an example instantiation of the top-performing spatiotemporal model, the distribution of test <em>R</em><sup>2</sup> scores for both the trained and control model are shown, for five kinds of kinematic tuning for each layer: direction tuning, speed tuning, Cartesian position tuning, polar position tuning, and label-specificity. The solid line connects the 90%-quantiles of two of the tuning curve types, direction tuning (dark) and position tuning (light). (0/2330 scores excluded summed over all layers for ART-trained, 129/2330 for TDT; see Methods). (<strong>B</strong>) The means of 90%-quantiles over all five model instantiations of models trained on action recognition and trajectory decoding are shown for direction tuning (dark) and position tuning (light). 95%-confidence intervals are shown over instantiations (<em>N</em> = 5). (<strong>C</strong>) The same plot as in (A) but for the top-performing LSTM model (0/6530 scores excluded for ART-trained, 1024/6530 for TDT; see Methods). (<strong>D</strong>) The same plot as B, for the LSTM model.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F6.medium.gif" width="440" height="223"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F6.medium.gif" width="440" height="223"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6.
                               </span>
                               <span class="caption-title">
                                Analysis of single unit tuning properties for spatiotemporal and LSTM models.
                               </span>
                               <p id="p-38" class="first-child">
                                (
                                <strong>
                                 A
                                </strong>
                                ) For an example instantiation of the top-performing spatiotemporal model, the distribution of test R
                                <sup>
                                 2
                                </sup>
                                scores for both the trained and control model are shown, for five kinds of kinematic tuning for each layer: direction tuning, speed tuning, Cartesian position tuning, polar position tuning, and label-specificity. The solid line connects the 90%-quantiles of two of the tuning curve types, direction tuning (dark) and position tuning (light). (0/2330 scores excluded summed over all layers for ART-trained, 129/2330 for TDT; see Methods). (
                                <strong>
                                 B
                                </strong>
                                ) The means of 90%-quantiles over all five model instantiations of models trained on action recognition and trajectory decoding are shown for direction tuning (dark) and position tuning (light). 95%-confidence intervals are shown over instantiations (N = 5). (
                                <strong>
                                 C
                                </strong>
                                ) The same plot as in (A) but for the top-performing LSTM model (0/6530 scores excluded for ART-trained, 1024/6530 for TDT; see Methods). (
                                <strong>
                                 D
                                </strong>
                                ) The same plot as B, for the LSTM model.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-39">
                              Our results so far suggests that the primate proprioceptive pathway is more consistent with the action recognition hypothesis, but to corroborate this, we also assessed decoding performance, which measures representational information. For all architecture types movement direction and speed can be better decoded from ART than from TDT trained models (
                              <a id="xref-fig-11-1" class="xref-fig" href="#F11">
                               Figure S4A,C,E
                              </a>
                              ). In contrast, for all architectures, position can be better decoded for TDT than for ART trained models (
                              <a id="xref-fig-11-2" class="xref-fig" href="#F11">
                               Figure S4B,D,F
                              </a>
                              ). These results are consistent with the single-cell encoding results, and again lend support for the proprioceptive systems involvement in action representation.
                             </p>
                             <div id="F11" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F11.large.jpg?width=800&height=600&carousel=1" title="Population decoding analysis of ART vs. TDT models. (A) Population decoding of speed (light) and direction (dark) for spatial-temporal models for the ART- and TDT-trained models. The faint line shows the R2 score for an individual model; the dark one the mean over all instantiations (N = 5). (B) Population decoding of endeffector position (X and Y coordinates) for spatial temporal models. The faint line shows the R2 score for an individual model; the dark one the mean over all instantiations (N = 5). (C) Same as A but for spatiotemporal models. (D) Same as B but for spatiotemporal models. (E) Same as A but for LSTM models. (F) Same as B but for LSTM models." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1989804583" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Population decoding analysis of ART vs. TDT models.</span> (<strong>A</strong>) Population decoding of speed (<em>light</em>) and direction (<em>dark</em>) for spatial-temporal models for the ART- and TDT-trained models. The faint line shows the <em>R</em><sup>2</sup> score for an individual model; the dark one the mean over all instantiations (<em>N</em> = 5). (<strong>B</strong>) Population decoding of endeffector position (X and Y coordinates) for spatial temporal models. The faint line shows the <em>R</em><sup>2</sup> score for an individual model; the dark one the mean over all instantiations (<em>N</em> = 5). (<strong>C</strong>) Same as A but for spatiotemporal models. (<strong>D</strong>) Same as B but for spatiotemporal models. (<strong>E</strong>) Same as A but for LSTM models. (<strong>F</strong>) Same as B but for LSTM models.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F11.medium.gif" width="440" height="366"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S4." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F11.medium.gif" width="440" height="366"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S4.
                               </span>
                               <span class="caption-title">
                                Population decoding analysis of ART vs. TDT models.
                               </span>
                               <p id="p-122" class="first-child">
                                (
                                <strong>
                                 A
                                </strong>
                                ) Population decoding of speed (light) and direction (dark) for spatial-temporal models for the ART- and TDT-trained models. The faint line shows the R
                                <sup>
                                 2
                                </sup>
                                score for an individual model; the dark one the mean over all instantiations (N = 5). (
                                <strong>
                                 B
                                </strong>
                                ) Population decoding of endeffector position (X and Y coordinates) for spatial temporal models. The faint line shows the R
                                <sup>
                                 2
                                </sup>
                                score for an individual model; the dark one the mean over all instantiations (N = 5). (
                                <strong>
                                 C
                                </strong>
                                ) Same as A but for spatiotemporal models. (
                                <strong>
                                 D
                                </strong>
                                ) Same as B but for spatiotemporal models. (
                                <strong>
                                 E
                                </strong>
                                ) Same as A but for LSTM models. (
                                <strong>
                                 F
                                </strong>
                                ) Same as B but for LSTM models.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-40">
                              So far, we have directly compared TDT and ART models. This does not address task-training as such. Namely, we found directional selective units in ART-models and positional-selective units in TDT-models, but how do those models compare to randomly initialized models (controls)? Remarkably, directional selectivity is similar for ART and control models (
                              <a id="xref-fig-12-1" class="xref-fig" href="#F12">
                               Figure S5
                              </a>
                              ). In contrast to controls, ART-trained models harbor less positional tuned units. The situation is reversed for TDT-trained models – those models gain positionally tuned units and lose directionally selective units during task training (
                              <a id="xref-fig-13-1" class="xref-fig" href="#F13">
                               Figure S6
                              </a>
                              ). Consistent with those encoding results, position can be worse and direction and speed similarly decoded from ART-models than from controls, respectively (
                              <a id="xref-fig-14-1" class="xref-fig" href="#F14">
                               Figure S7
                              </a>
                              ). Conversely, direction and speed can be worse and position better decoded from TDT-models than controls (
                              <a id="xref-fig-14-2" class="xref-fig" href="#F14">
                               Figure S7
                              </a>
                              ).
                             </p>
                             <div id="F12" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F12.large.jpg?width=800&height=600&carousel=1" title="Analysis of single unit tuning properties for ART-trained models and controls (A) For an example instantiation of the top-performing spatial-temporal model, the distribution of test R2 scores for both the trained and control model are shown, for five kinds of kinematic tuning for each layer: direction tuning, speed tuning, Cartesian position tuning, polar position tuning, and label-specificity. The solid line connects the 90%-quantiles of two of the tuning curve types, direction tuning (dark) and position tuning (light). (8/3890 scores excluded summed over all layers for ART-trained, 294/3890 for controls; see Methods). (B) The means of 90%-quantiles over all five model instantiations of models trained on action recognition and trajectory decoding are shown for direction tuning (dark) and position tuning (light). 95%-confidence intervals are shown over instantiations (N = 5). The same plot as in (A) but for the top-performing spatiotemporal model (133/2330 scores excluded for ART-trained, 60/2330 for the control; see Methods). (D) The same plot as B, for the spatiotemporal model. (E) The same plot as in (A) but for the top-performing LSTM model (4/6530 scores excluded for ART-trained, 328/6530 for the control; see Methods). (F) The same plot as B, for the LSTM model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1989804583" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Analysis of single unit tuning properties for ART-trained models and controls</span> (<strong>A</strong>) For an example instantiation of the top-performing spatial-temporal model, the distribution of test <em>R</em><sup>2</sup> scores for both the trained and control model are shown, for five kinds of kinematic tuning for each layer: direction tuning, speed tuning, Cartesian position tuning, polar position tuning, and label-specificity. The solid line connects the 90%-quantiles of two of the tuning curve types, direction tuning (dark) and position tuning (light). (8/3890 scores excluded summed over all layers for ART-trained, 294/3890 for controls; see Methods). (<strong>B</strong>) The means of 90%-quantiles over all five model instantiations of models trained on action recognition and trajectory decoding are shown for direction tuning (dark) and position tuning (light). 95%-confidence intervals are shown over instantiations (<em>N</em> = 5). The same plot as in (A) but for the top-performing spatiotemporal model (133/2330 scores excluded for ART-trained, 60/2330 for the control; see Methods). (<strong>D</strong>) The same plot as B, for the spatiotemporal model. (<strong>E</strong>) The same plot as in (A) but for the top-performing LSTM model (4/6530 scores excluded for ART-trained, 328/6530 for the control; see Methods). (<strong>F</strong>) The same plot as B, for the LSTM model.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F12.medium.gif" width="440" height="323"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S5." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F12.medium.gif" width="440" height="323"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S5.
                               </span>
                               <span class="caption-title">
                                Analysis of single unit tuning properties for ART-trained models and controls
                               </span>
                               <p id="p-123" class="first-child">
                                (
                                <strong>
                                 A
                                </strong>
                                ) For an example instantiation of the top-performing spatial-temporal model, the distribution of test R
                                <sup>
                                 2
                                </sup>
                                scores for both the trained and control model are shown, for five kinds of kinematic tuning for each layer: direction tuning, speed tuning, Cartesian position tuning, polar position tuning, and label-specificity. The solid line connects the 90%-quantiles of two of the tuning curve types, direction tuning (dark) and position tuning (light). (8/3890 scores excluded summed over all layers for ART-trained, 294/3890 for controls; see Methods). (
                                <strong>
                                 B
                                </strong>
                                ) The means of 90%-quantiles over all five model instantiations of models trained on action recognition and trajectory decoding are shown for direction tuning (dark) and position tuning (light). 95%-confidence intervals are shown over instantiations (N = 5). The same plot as in (A) but for the top-performing spatiotemporal model (133/2330 scores excluded for ART-trained, 60/2330 for the control; see Methods). (
                                <strong>
                                 D
                                </strong>
                                ) The same plot as B, for the spatiotemporal model. (
                                <strong>
                                 E
                                </strong>
                                ) The same plot as in (A) but for the top-performing LSTM model (4/6530 scores excluded for ART-trained, 328/6530 for the control; see Methods). (
                                <strong>
                                 F
                                </strong>
                                ) The same plot as B, for the LSTM model.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F13" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F13.large.jpg?width=800&height=600&carousel=1" title="Analysis of single unit tuning properties for TDT-trained models and controls (A) For an example instantiation of the top-performing spatial-temporal model, the distribution of test R2 scores for both the trained and control model are shown, for five kinds of kinematic tuning for each layer: direction tuning, speed tuning, Cartesian position tuning, polar position tuning, and label-specificity. The solid line connects the 90%-quantiles of two of the tuning curve types, direction tuning (dark) and position tuning (light). (287/3890 scores excluded summed over all layers for TDT-trained, 369/3890 for controls; see Methods). (B) The means of 90%-quantiles over all five model instantiations of models trained on action recognition and trajectory decoding are shown for direction tuning (dark) and position tuning (light). 95%-confidence intervals are shown over instantiations (N = 5). (C) The same plot as in (A) but for the top-performing spatiotemporal model (133/2330 scores excluded for TDT-trained, 60/2330 for the control; see Methods). (D) The same plot as B, for the spatiotemporal model. (E) The same plot as in (A) but for the top-performing LSTM model (1024/6530 scores excluded for TDT-trained, 328/6530 for the control; see Methods). (F) The same plot as B, for the LSTM model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1989804583" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Analysis of single unit tuning properties for TDT-trained models and controls</span> (<strong>A</strong>) For an example instantiation of the top-performing spatial-temporal model, the distribution of test <em>R</em><sup>2</sup> scores for both the trained and control model are shown, for five kinds of kinematic tuning for each layer: direction tuning, speed tuning, Cartesian position tuning, polar position tuning, and label-specificity. The solid line connects the 90%-quantiles of two of the tuning curve types, direction tuning (dark) and position tuning (light). (287/3890 scores excluded summed over all layers for TDT-trained, 369/3890 for controls; see Methods). (<strong>B</strong>) The means of 90%-quantiles over all five model instantiations of models trained on action recognition and trajectory decoding are shown for direction tuning (dark) and position tuning (light). 95%-confidence intervals are shown over instantiations (<em>N</em> = 5). (<strong>C</strong>) The same plot as in (A) but for the top-performing spatiotemporal model (133/2330 scores excluded for TDT-trained, 60/2330 for the control; see Methods). (<strong>D</strong>) The same plot as B, for the spatiotemporal model. (<strong>E</strong>) The same plot as in (A) but for the top-performing LSTM model (1024/6530 scores excluded for TDT-trained, 328/6530 for the control; see Methods). (<strong>F</strong>) The same plot as B, for the LSTM model.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F13.medium.gif" width="440" height="320"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S6." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F13.medium.gif" width="440" height="320"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S6." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S6.
                               </span>
                               <span class="caption-title">
                                Analysis of single unit tuning properties for TDT-trained models and controls
                               </span>
                               <p id="p-124" class="first-child">
                                (
                                <strong>
                                 A
                                </strong>
                                ) For an example instantiation of the top-performing spatial-temporal model, the distribution of test R
                                <sup>
                                 2
                                </sup>
                                scores for both the trained and control model are shown, for five kinds of kinematic tuning for each layer: direction tuning, speed tuning, Cartesian position tuning, polar position tuning, and label-specificity. The solid line connects the 90%-quantiles of two of the tuning curve types, direction tuning (dark) and position tuning (light). (287/3890 scores excluded summed over all layers for TDT-trained, 369/3890 for controls; see Methods). (
                                <strong>
                                 B
                                </strong>
                                ) The means of 90%-quantiles over all five model instantiations of models trained on action recognition and trajectory decoding are shown for direction tuning (dark) and position tuning (light). 95%-confidence intervals are shown over instantiations (N = 5). (
                                <strong>
                                 C
                                </strong>
                                ) The same plot as in (A) but for the top-performing spatiotemporal model (133/2330 scores excluded for TDT-trained, 60/2330 for the control; see Methods). (
                                <strong>
                                 D
                                </strong>
                                ) The same plot as B, for the spatiotemporal model. (
                                <strong>
                                 E
                                </strong>
                                ) The same plot as in (A) but for the top-performing LSTM model (1024/6530 scores excluded for TDT-trained, 328/6530 for the control; see Methods). (
                                <strong>
                                 F
                                </strong>
                                ) The same plot as B, for the LSTM model.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F14" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F14.large.jpg?width=800&height=600&carousel=1" title="Analysis of population decoding for ART-trained and control models. (A) Population decoding of speed (light) and direction (dark) for the ART-trained and control for spatial-temporal models (left), spatiotemporal (middle) and LSTM (right) models. The faint line shows the R2 score for an individual model; the dark one the mean over all instantiations (N = 5). (B) Population decoding of endeffector position (X and Y coordinates) for spatial temporal models. The faint line shows the R2 score for an individual model; the dark one the mean over all instantiations (N = 5)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1989804583" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Analysis of population decoding for ART-trained and control models.</span> (<strong>A</strong>) Population decoding of speed (<em>light</em>) and direction (<em>dark</em>) for the ART-trained and control for spatial-temporal models (<em>left</em>), spatiotemporal (<em>middle</em>) and LSTM (<em>right</em>) models. The faint line shows the <em>R</em><sup>2</sup> score for an individual model; the dark one the mean over all instantiations (<em>N</em> = 5). (<strong>B</strong>) Population decoding of endeffector position (X and Y coordinates) for spatial temporal models. The faint line shows the <em>R</em><sup>2</sup> score for an individual model; the dark one the mean over all instantiations (<em>N</em> = 5).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F14.medium.gif" width="440" height="215"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S7." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F14.medium.gif" width="440" height="215"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S7." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S7.
                               </span>
                               <span class="caption-title">
                                Analysis of population decoding for ART-trained and control models.
                               </span>
                               <p id="p-125" class="first-child">
                                (
                                <strong>
                                 A
                                </strong>
                                ) Population decoding of speed (light) and direction (dark) for the ART-trained and control for spatial-temporal models (left), spatiotemporal (middle) and LSTM (right) models. The faint line shows the R
                                <sup>
                                 2
                                </sup>
                                score for an individual model; the dark one the mean over all instantiations (N = 5). (
                                <strong>
                                 B
                                </strong>
                                ) Population decoding of endeffector position (X and Y coordinates) for spatial temporal models. The faint line shows the R
                                <sup>
                                 2
                                </sup>
                                score for an individual model; the dark one the mean over all instantiations (N = 5).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              Uniformity and coding invariance
                             </h3>
                             <p id="p-41">
                              We compared population coding properties to further elucidate the similarity to S1. We measured the distributions of preferred directions and whether coding properties are invariant across different workspaces (reaching planes). Prud’homme and Kalaska found a relatively uniform distribution of preferred directions in primate S1 during a center out reaching 2D manipulandum-based task (
                              <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                               Figure 7A
                              </a>
                              from (
                              <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                              )). In contrast, most velocity tuned spindle afferents have preferred directions located along one major axis pointing frontally and slightly away from the body (
                              <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                               Figure 7B
                              </a>
                              ). Qualitatively, it appears that the ART trained model had more uniformly distributed preferred directions in the middle layers compared to control models with randomly initialized weights (
                              <a id="xref-fig-7-3" class="xref-fig" href="#F7">
                               Figure 7C
                              </a>
                              ).
                             </p>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F7.large.jpg?width=800&height=600&carousel=1" title="Distribution of preferred directions and invariance of representation across workspaces. (A) Adopted from (14); distribution of preferred directions in primate S1. (B) Distribution of preferred directions for spindle input. (C) Distribution of preferred directions for one spatial-temporal model instantiation (all units with R2 > 0.2 are included). Bottom: the corresponding control. For visibility all histograms are scaled to the same size and the colors indicate the number of tuned neurons. (D) For quantifying uniformity, we calculated the total absolute deviation from the corresponding uniform distribution over the bins in the histogram (red line in inset) for the spatial-temporal model (left), the spatiotemporal model (middle), and the LSTM model (right). Normalized absolute deviation from uniform distribution for preferred directions per instantiation are shown (N = 5, faint lines) for trained and control models as well as mean and 95%-confidence intervals over instantiations (solid line; N = 5). Note that there is no data for layers 7 and 8 of the trained spatial-temporal model, layer 8 of the untrained spatial-temporal model, and layer 4 of the spatiotemporal model as they have no direction selective units (R2 > 0.2). (E) For quantifying invariance we calculated mean absolute deviation in preferred orientation for units from the central plane to each other vertical plane (for units with R2 > 0.2). Results are shown for each instantiation (N = 5, faint lines) for trained and control models plus mean (solid) and 95%-confidence intervals over instantiations (N = 5). Note that there is no data for layer 4 of the trained spatiotemporal model, as it have no direction selective units (R2 > 0.2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1989804583" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Distribution of preferred directions and invariance of representation across workspaces.</span> (<strong>A</strong>) Adopted from (14); distribution of preferred directions in primate S1. (<strong>B</strong>) Distribution of preferred directions for spindle input. (<strong>C</strong>) Distribution of preferred directions for one spatial-temporal model instantiation (all units with <em>R</em><sup>2</sup> <em>></em> 0.2 are included). Bottom: the corresponding control. For visibility all histograms are scaled to the same size and the colors indicate the number of tuned neurons. (<strong>D</strong>) For quantifying uniformity, we calculated the total absolute deviation from the corresponding uniform distribution over the bins in the histogram (red line in inset) for the spatial-temporal model (<em>left</em>), the spatiotemporal model (<em>middle</em>), and the LSTM model (<em>right</em>). Normalized absolute deviation from uniform distribution for preferred directions per instantiation are shown (<em>N</em> = 5, faint lines) for trained and control models as well as mean and 95%-confidence intervals over instantiations (solid line; <em>N</em> = 5). Note that there is no data for layers 7 and 8 of the trained spatial-temporal model, layer 8 of the untrained spatial-temporal model, and layer 4 of the spatiotemporal model as they have no direction selective units (<em>R</em><sup>2</sup> <em>></em> 0.2). (<strong>E</strong>) For quantifying invariance we calculated mean absolute deviation in preferred orientation for units from the central plane to each other vertical plane (for units with <em>R</em><sup>2</sup> <em>></em> 0.2). Results are shown for each instantiation (<em>N</em> = 5, faint lines) for trained and control models plus mean (solid) and 95%-confidence intervals over instantiations (<em>N</em> = 5). Note that there is no data for layer 4 of the trained spatiotemporal model, as it have no direction selective units (<em>R</em><sup>2</sup> <em>></em> 0.2).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F7.medium.gif" width="440" height="328"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F7.medium.gif" width="440" height="328"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 7.
                               </span>
                               <span class="caption-title">
                                Distribution of preferred directions and invariance of representation across workspaces.
                               </span>
                               <p id="p-42" class="first-child">
                                (
                                <strong>
                                 A
                                </strong>
                                ) Adopted from (
                                <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">
                                 14
                                </a>
                                ); distribution of preferred directions in primate S1. (
                                <strong>
                                 B
                                </strong>
                                ) Distribution of preferred directions for spindle input. (
                                <strong>
                                 C
                                </strong>
                                ) Distribution of preferred directions for one spatial-temporal model instantiation (all units with R
                                <sup>
                                 2
                                </sup>
                                > 0.2 are included). Bottom: the corresponding control. For visibility all histograms are scaled to the same size and the colors indicate the number of tuned neurons. (
                                <strong>
                                 D
                                </strong>
                                ) For quantifying uniformity, we calculated the total absolute deviation from the corresponding uniform distribution over the bins in the histogram (red line in inset) for the spatial-temporal model (left), the spatiotemporal model (middle), and the LSTM model (right). Normalized absolute deviation from uniform distribution for preferred directions per instantiation are shown (N = 5, faint lines) for trained and control models as well as mean and 95%-confidence intervals over instantiations (solid line; N = 5). Note that there is no data for layers 7 and 8 of the trained spatial-temporal model, layer 8 of the untrained spatial-temporal model, and layer 4 of the spatiotemporal model as they have no direction selective units (R
                                <sup>
                                 2
                                </sup>
                                > 0.2). (
                                <strong>
                                 E
                                </strong>
                                ) For quantifying invariance we calculated mean absolute deviation in preferred orientation for units from the central plane to each other vertical plane (for units with R
                                <sup>
                                 2
                                </sup>
                                > 0.2). Results are shown for each instantiation (N = 5, faint lines) for trained and control models plus mean (solid) and 95%-confidence intervals over instantiations (N = 5). Note that there is no data for layer 4 of the trained spatiotemporal model, as it have no direction selective units (R
                                <sup>
                                 2
                                </sup>
                                > 0.2).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-43">
                              To quantify uniformity we calculated the total absolute deviation (TAD) from uniformity in the distribution of preferred directions. The results indicate that the distribution of preferred directions becomes more uniform in middle layers for all instantiations of the different model architectures (
                              <a id="xref-fig-7-4" class="xref-fig" href="#F7">
                               Figure 7D
                              </a>
                              ), and that this difference is statistically significant for the spatial-temporal model beginning in layer 3 (layer 3 t(4) = − 3.41, p=0.027; layer 4 t(4) = − 8.12, p=0.001; layer 5 t(4) = − 4.08, p=0.015; layer 6 t(4) = − 5.55, p=0.005). This analysis revealed that while randomly initialized models also have directionally selective units, those units are less uniformly distributed than in models trained with the ART task. Importantly, TDT trained modules have almost no directionally selective units, further corroborating that ART models are more consistent with Prud’homme and Kalaska’s findings (
                              <a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                              ).
                             </p>
                             <p id="p-44">
                              Lastly, we could directly test if preferred tuning directions (of tuned units) were maintained across different planes due to the fact that we created trajectories in multiple vertical and horizontal planes. We hypothesized that for the trained networks preferred orientations would be more preserved across planes compared to controls. In order to examine how an individual unit’s preferred direction changed across different planes, directional tuning curve models were fit in each horizontal/vertical plane separately (examples in
                              <a id="xref-fig-16-1" class="xref-fig" href="#F16">
                               Figure S9A, B
                              </a>
                              ). To measure the representational invariance, we took the mean absolute deviation (MAD) of the preferred tuning direction for directionally tuned units (R
                              <sup>
                               2
                              </sup>
                              > 0.2) across planes (see Methods) and averaged over all planes (
                              <a id="xref-fig-7-5" class="xref-fig" href="#F7">
                               Figures 7E
                              </a>
                              ,
                              <a id="xref-fig-16-2" class="xref-fig" href="#F16">
                               S9A
                              </a>
                              ). For the spatial-temporal model across vertical workspaces, layers 3-6 were indeed more invariant in their preferred directions (layer 3: t(4)= -11.38, p=0.0003; layer 4: t(4) = -10.23, p=0.0005; layer 5: t(4) = -12.17, p=0.002; layer 6: t(4) = - 13.18, p=0.0002;
                              <a id="xref-fig-7-6" class="xref-fig" href="#F7">
                               Figure 7E
                              </a>
                              ; variation in preferred direction illustrated for layer 5 in
                              <a id="xref-fig-16-3" class="xref-fig" href="#F16">
                               Figure S9D
                              </a>
                              for trained model and in
                              <a id="xref-fig-16-4" class="xref-fig" href="#F16">
                               Figure S9E
                              </a>
                              for controls). The difference in invariance for the horizontal planes was likewise statistically significant for layers 4-6 (
                              <a id="xref-fig-16-5" class="xref-fig" href="#F16">
                               Figure S9A
                              </a>
                              ). The difference in invariance here might only become statistically significant one layer later because the spindles are already more invariant in the horizontal planes (MAD: 0.225 ± 2.78e − 17, mean ± SEM, N = 25;
                              <a id="xref-fig-16-6" class="xref-fig" href="#F16">
                               Figure S9B
                              </a>
                              ) than the vertical workspaces (MAD: 0.439 ± 5e − 17, mean ± SEM, N = 16;
                              <a id="xref-fig-16-7" class="xref-fig" href="#F16">
                               Figure S9C
                              </a>
                              ), meaning that it takes a greater amount of invariance in the trained networks for differences with the control networks to become statistically apparent. For the spatiotemporal and LSTM models, the relatively slower increase in invariance in the horizontal direction is exaggerated even more. For the LSTM model, the neuron tuning does not become stronger for the horizontal planes until the recurrent laye. The spatiotemporal trained models even begin by being less invariant in the horizontal planes before the difference evens out in layer 3r (FigureS9A).
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             Discussion
                            </h2>
                            <div id="sec-10" class="subsection">
                             <h3>
                              Task-Driven Modeling of Proprioception
                             </h3>
                             <p id="p-45">
                              For various anatomical and experimental reasons, recording proprioceptive activity during natural movements is technically challenging (
                              <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                              ,
                              <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                               47
                              </a>
                              ). Furthermore, “presenting” particular proprioceptive-only stimuli is difficult, which poses substantial challenges for systems identification approaches. This highlights the importance of developing accurate, normative models that can explain neural representations across the proprioceptive pathway, as has been successfully done in the visual system (
                              <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              –
                              <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">
                               26
                              </a>
                              ). To tackle this, we combined human movement data, biomechanical modeling, as well as deep learning to provide a blueprint for studying the proprioceptive pathway.
                             </p>
                             <p id="p-46">
                              We presented a task-driven approach to study the proprioceptive system based on our hypothesis that proprioception can be understood normatively as having to solve action-recognition from receptor inputs. We created a passive character recognition task for a simulated human biomechanical arm paired with a muscle spindle model and found that deep neural networks can be trained to accurately solve the action recognition task. Inferring the character from passive arm traces was chosen as it is a type of task that humans can easily perform and because it covers a wide range of natural movements of the arm. The perception is also likely fast, so that feed-forward processing is a good approximation (while we also find similar results with recurrent models). Additionally, character recognition is an influential task for studying ANNs, for instance MNIST (
                              <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                               42
                              </a>
                              ,
                              <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                               48
                              </a>
                              ). Moreover, when writing movements were imposed onto the ankle with a fixed knee joint, the movement trajectory could be decoded from a few spindles using a population vector model, suggesting that spindle information is accurate enough for decoding (
                              <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                               49
                              </a>
                              ). Lastly, while the underlying movements are natural and of ethological importance for humans, the task itself is only a small subset of human upper-limb function. Thus, it posed an interesting question whether such a task would be sufficient to induce representations similar to biological neurons.
                             </p>
                             <p id="p-47">
                              We put forth a normative model of the proprioceptive system, which is experimentally testable. This builds on our earlier preprint (
                              <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                               50
                              </a>
                              ) of this work, which only considered action recognition and used a different receptor model (
                              <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                               51
                              </a>
                              ) that used changes in the length of the muscles. Here, we also include positional sensing and an additional tasks to directly test our hypothesis of action coding vs. the canonical view of proprioception. We confirm that in ART-trained models, but not in randomly initialized models, the intermediate representations contain directionally selective neurons that are uniformly distributed (
                              <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">
                               50
                              </a>
                              ). Furthermore, we had predicted that earlier layers and in particular muscle spindles, have a biased, bidirectionally tuned distribution. This distribution was later found experimentally for single units in the cuneate nucleus (
                              <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">
                               52
                              </a>
                              ). Here we still, robustly find this result but with different spindle models (
                              <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">
                               35
                              </a>
                              ,
                              <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">
                               36
                              </a>
                              ). However, these PD distribution result does not hold when the identical architectures are trained with the trajectory decoding task. In those models directionally tuned neurons do not emerge, in fact they are “unlearned” in comparison to random initializations for TDT (
                              <a id="xref-fig-13-2" class="xref-fig" href="#F13">
                               Figure S6
                              </a>
                              ).
                             </p>
                             <p id="p-48">
                              The distribution of preferred directions becomes more uniform over the course of the processing hierarchy (
                              <a id="xref-fig-6-3" class="xref-fig" href="#F6">
                               Figure 6
                              </a>
                              and
                              <a id="xref-fig-7-7" class="xref-fig" href="#F7">
                               7
                              </a>
                              ), similar to the distribution of preferred tuning in somatosensory cortex (
                              <a id="xref-ref-14-7" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                              ). This does not occur in the randomized controls, which instead maintained an input distribution centered on the primary axis of preferred directions of the muscular tuning curves. Furthermore, the task-trained models make a prediction about the distribution of preferred directions along the proprioceptive pathway. For instance, we predict that in the brainstem - i.e. cuneate nucleus - preferred directions are aligned along major axes inherited from muscle spindles that correspond to biomechanical constraints (consistant with Versteeg et al. (
                              <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">
                               52
                              </a>
                              )). A key element of robust object recognition is invariance to task-irrelevant variables (
                              <a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ,
                              <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">
                               53
                              </a>
                              ). In our computational study, we could probe many different workspaces (26 horizontal and 18 vertical) to reveal that training on the character recognition task makes directional tuning more invariant (
                              <a id="xref-fig-7-8" class="xref-fig" href="#F7">
                               Figure 7E
                              </a>
                              ). This together with our observation that directional tuning is simply inherited from muscle spindles, highlights the importance of sampling the movement space well, as also emphasized by pioneering experimental studies (
                              <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">
                               54
                              </a>
                              ). We also note that the predictions depend on the musculoskelatal model and the movement statistics. In fact, we predict that e.g., distributions of tuning and invariances might be different in mice, a species that has a different body orientation from primates. This will be explored in future studies.
                             </p>
                            </div>
                            <div id="sec-11" class="subsection">
                             <h3>
                              Limitations and future directions
                             </h3>
                             <p id="p-49">
                              Our model only encompasses proprioception and trained in a supervised fashion. However, it is quite natural to interpret the supervised feedback stemming from other senses. For instance, the visual system could naturally provide information of the hand localization or about the type of character.
                             </p>
                             <p id="p-50">
                              Here we used different types of temporal convolutional and recurrent network architectures. In future work it will be important to investigate emerging, perhaps more biologically-relevant architectures to better understand how muscle spindles are integrated in upstream circuits. While we used spindle Ia and II models, it is known that multiple receptors, namely cutaneous, joint, and muscle receptors play a role for limb localization and kinesthesia (
                              <a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                              ,
                              <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">
                               55
                              </a>
                              –
                              <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">
                               58
                              </a>
                              ). For instance, a recent simulation study by Kibleur et al. highlighted the complex spatio-temporal structure of proprioceptive information at the level of the cervical spinal cord (
                              <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">
                               47
                              </a>
                              ). Furthermore, due to fusimotor drive receptor activity can be modulated by other modalities, e.g., vision (
                              <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">
                               59
                              </a>
                              ). In the future, models for other afferents, golgi tendon organ incl. the fusimotor drive as well as cutaneous receptors can be added, to study their role in the context of various tasks (
                              <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">
                               40
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-12" class="subsection">
                             <h3>
                              Conclusions
                             </h3>
                             <p id="p-51">
                              We proposed action recognition from peripheral inputs as an objective to study proprioception. We developed task-driven models of the proprioceptive system and showed that diverse preferred tuning directions and invariance across 3D space emerges in neural networks emerges in such systems – this emergence was not found for models trained on representing the state of the body. Due to their hierarchical nature, the network models provide not only a description of neurons previously found in physiology studies, they make predictions about coding properties, such as the biased distribution of direction tuning in subcortical areas.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-13">
                            <h2 class="">
                             Methods
                            </h2>
                            <div id="sec-14" class="subsection">
                             <h3>
                              Proprioceptive Character Trajectories: Dataset and Tasks
                             </h3>
                             <div id="sec-15" class="subsection">
                              <h4>
                               The character trajectories dataset
                              </h4>
                              <p id="p-52">
                               The movement data for our task was obtained from the UCI Machine Learning Repository character trajectories dataset (
                               <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">
                                33
                               </a>
                               ,
                               <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                               ). In brief, the dataset contains 2,858 pen-tip trajectories for 20 single-stroke characters (excluding f, i, j, k, t and x, which were multi-stroke in this dataset) in the Latin alphabet, written by a single person on an Intuos 3 Wacom digitization tablet providing pen-tip position and pressure information at 200 Hz. The size of the characters was such that they all approximately fit within a 1 × 1 cm grid. Since we aimed to study the proprioception of the whole arm, we first interpolated the trajectories to lie within a 10 × 10 cm grid and discarded the pen-tip pressure information. Trajectories were interpolated linearly while maintaining the velocity profiles of the original trajectories. Empirically, we found that on average it takes 3 times longer to write a character in the 10 × 10 cm grid than in the small 1 × 1 one. Therefore, the time interval between samples was increased from 5ms (200 Hz) to 15ms (66.7 Hz) when interpolating trajectories. The resulting 2,858 character trajectories served as the basis for our end-effector trajectories.
                              </p>
                             </div>
                             <div id="sec-16" class="subsection">
                              <h4>
                               Computing joint angles and muscle length trajectories
                              </h4>
                              <p id="p-53">
                               Using these end-effector trajectories, we sought to generate realistic proprioceptive inputs while passively executing such movements. For this purpose, we used an open-source musculoskeletal model of the human upper limb, the upper extremity dynamic model by Saul et al. (
                               <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">
                                34
                               </a>
                               ,
                               <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">
                                60
                               </a>
                               ). The model includes 50 Hill-type muscle-tendon actuators crossing the shoulder, elbow, forearm and wrist. While the kinematic foundations of the model enable it with 15 degrees of freedom (DoF), 8 DoF were eliminated by enforcing the hand to form a grip posture. We further eliminated 3 DoF by disabling the model to have elbow rotation, wrist flexion and rotation. The four remaining DoF are elbow flexion (θ
                               <sub>
                                ef
                               </sub>
                               ), shoulder rotation (θ
                               <sub>
                                sr
                               </sub>
                               ), shoulder elevation i.e, thoracohumeral angle (θ
                               <sub>
                                se
                               </sub>
                               ) and elevation plane of the shoulder (θ
                               <sub>
                                sep
                               </sub>
                               ).
                              </p>
                              <p id="p-54">
                               The first step in extracting the spindle activations involved computing the joint angles for the 4 DoF from the endeffector trajectories using constrained inverse kinematics. We built a 2-link 4 DoF arm with arm-lengths corresponding to those of the upper extremity dynamic model (
                               <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">
                                60
                               </a>
                               ). To determine the joint-angle trajectories, we first define the forward kinematics equations that convert a given joint-angle configuration of the arm to its end-effector position. For a given joint-angle configuration of the arm
                               <strong>
                                q
                               </strong>
                               = [θ
                               <sub>
                                ef
                               </sub>
                               , θ
                               <sub>
                                sr
                               </sub>
                               , θ
                               <sub>
                                se
                               </sub>
                               , θ
                               <sub>
                                sep
                               </sub>
                               ]
                               <sup>
                                T
                               </sup>
                               , the end-effector position
                               <strong>
                                e
                               </strong>
                               ∈ ℝ
                               <sup>
                                3
                               </sup>
                               in an absolute frame of reference {S} centered on the shoulder is given by
                               <span class="disp-formula" id="disp-formula-1">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-10.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-10.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               with position of the end-effector (hand)
                               <strong>
                                e
                               </strong>
                               <sub>
                                0
                               </sub>
                               and elbow
                               <strong>
                                l
                               </strong>
                               <sub>
                                <strong>
                                 0
                                </strong>
                               </sub>
                               when the arm is at rest and rotation matrices
                               <span class="disp-formula" id="disp-formula-2">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-11.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-11.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-12.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-12.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-55">
                               Thereby, R
                               <sub>
                                S
                               </sub>
                               is the rotation matrix at the shoulder joint, R
                               <sub>
                                L
                               </sub>
                               is the rotation matrix at the elbow obtained by combinations of intrinsic rotations around the X, Y and Z axes which are defined according to the upper extremity dynamic model (
                               <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">
                                60
                               </a>
                               ), treating the joint angles as Euler angles and R
                               <sub>
                                X
                               </sub>
                               , R
                               <sub>
                                Y
                               </sub>
                               , R
                               <sub>
                                Z
                               </sub>
                               - the three basic rotation matrices.
                              </p>
                              <p id="p-56">
                               Given the forward kinematics equations, the joint-angles
                               <strong>
                                q
                               </strong>
                               for an end-effector position
                               <strong>
                                e
                               </strong>
                               can be obtained by iteratively solving a constrained inverse kinematics problem for all times t = 0 … T :
                               <span class="disp-formula" id="disp-formula-4">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-13.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-13.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-57">
                               Where
                               <strong>
                                q
                               </strong>
                               (− 1) is a natural pose in the center of the workspace (see
                               <a id="xref-fig-2-5" class="xref-fig" href="#F2">
                                Figure 2D
                               </a>
                               ) and each
                               <strong>
                                q
                               </strong>
                               (t) is a posture pointing to
                               <strong>
                                e
                               </strong>
                               (t), while being close to the previous posture
                               <strong>
                                q
                               </strong>
                               (t − 1). Thereby, {θ
                               <sub>
                                min
                               </sub>
                               , θ
                               <sub>
                                max
                               </sub>
                               } define the limits for each joint-angle. For a given end-effector trajectory
                               <strong>
                                e
                               </strong>
                               (t), joint-angle trajectories are thus computed from the previous time point in order to generate smooth movements in joint space. This approach is inspired by D’Souza et al. (
                               <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">
                                61
                               </a>
                               ).
                              </p>
                              <p id="p-58">
                               Finally, for a given joint trajectory
                               <strong>
                                q
                               </strong>
                               (t) we passively moved the arm through the joint-angle trajectories in the OpenSim 3.3 simulation environment (
                               <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">
                                62
                               </a>
                               ,
                               <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">
                                63
                               </a>
                               ), computing at each time point the equilibrium muscle lengths
                               <strong>
                                m
                               </strong>
                               (t) ∈ ℝ
                               <sup>
                                25
                               </sup>
                               , since the actuation of the 4 DoFs is achieved by 25 muscles. For simplicity, we computed equilibrium muscle configurations given joint angles as an approximation to passive movement.
                              </p>
                             </div>
                             <div id="sec-17" class="subsection">
                              <h4>
                               Proprioceptive inputs
                              </h4>
                              <p id="p-59">
                               While several mechanoreceptors provide proprioceptive information, including joint receptors, Golgi tendon organs and skin stretch receptors, the muscle spindles are regarded as the most important for conveying position and movement related information (
                               <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">
                                2
                               </a>
                               ,
                               <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">
                                51
                               </a>
                               ,
                               <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">
                                64
                               </a>
                               ,
                               <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">
                                65
                               </a>
                               ). Here we are inspired by Dimitriou and Edin’s recordings from human spindles (
                               <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">
                                35
                               </a>
                               ,
                               <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">
                                36
                               </a>
                               ). They found that both Ia and II units are well predicted by combinations (for parameters k
                               <sub>
                                1
                               </sub>
                               … k
                               <sub>
                                5
                               </sub>
                               ) of muscle length l, muscle velocity l′, acceleration l″ and EMG:
                               <span class="disp-formula" id="disp-formula-5">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-14.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-14.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-60">
                               As we model passive movement, the associated EMG activity is negligible. To simplify the aggregate information flowing from one muscle (via multiple Ia and II spindles), we consider a more generic/functional representation of proprioceptive information as consisting of muscle length and velocity signals, which are approximately conveyed by muscle spindles during passive movements. Therefore, in addition to the equilibrium muscle lengths
                               <strong>
                                m
                               </strong>
                               (t), we input muscle velocity
                               <strong>
                                v
                               </strong>
                               (t) signal obtained by taking the first derivative. Taken together, {
                               <strong>
                                m
                               </strong>
                               (t),
                               <strong>
                                v
                               </strong>
                               (t)} form the proprioceptive inputs to train models of the proprioceptive system.
                              </p>
                             </div>
                             <div id="sec-18" class="subsection">
                              <h4>
                               A scalable proprioceptive character trajectories dataset
                              </h4>
                              <p id="p-61">
                               We move our arms in various configurations and write at varying speeds. Thus, several axes of variation were added to each (original) trajectory by (
                               <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">
                                1
                               </a>
                               ) applying affine transformations such as scaling, rotation and shear, (
                               <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">
                                2
                               </a>
                               ) modifying the speed at which the character is written, (
                               <a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">
                                3
                               </a>
                               ) writing the character at several different locations (chosen from a grid of candidate starting points) in the 3D workspace of the arm, and (
                               <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
                                4
                               </a>
                               ) writing the characters on either transverse (horizontal) or frontal (vertical) planes of which there were 26 and 18 respectively, placed at a spatial distance of 3cm from each other (see
                               <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">
                                Table 1
                               </a>
                               for parameter ranges). We first generated a dataset of end-effector trajectories of 1 million samples by generating variants of each original trajectory, by scaling, rotating, shearing, translating and varying its speed. For each end-effector trajectory, we compute the joint-angle trajectory by performing inverse kinematics. Subsequently, we simulate the muscle length and muscle velocity trajectories. Since different characters take different amount of time to be written, we pad the movements with static postures corresponding to the starting and ending postures of the movement, and jitter the beginning of the writing to maintain ambiguity about when the writing begins.
                              </p>
                              <p id="p-62">
                               From this dataset of trajectories, we selected a subset of trajectories such that the integral of joint-space jerk (third derivative of movement) was less than 1 rad/s
                               <sup>
                                3
                               </sup>
                               so as to ensure that the arm movement is sufficiently smooth. Among these, we picked the trajectories for which the integral of muscle-space jerk was minimal, while making sure that the dataset is balanced in terms of the number of examples per class, resulting in 200,000 samples. The final dataset consists of muscle length and velocity trajectories from each of the 25 muscles over a period of 320 time points, simulated at 66.7 Hz (i.e, 4.8 seconds). In other words, the dimensionality of the proprioceptive inputs in our tasks is 25×320×2. The dataset was then split into a training, validation and test set with a 72 − 8 − 20 ratio.
                              </p>
                             </div>
                             <div id="sec-19" class="subsection">
                              <h4>
                               Action recognition and trajectory decoding tasks
                              </h4>
                              <p id="p-63">
                               Having simulated a large scale dataset of proprioceptive character trajectories, we designed two tasks - (
                               <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">
                                1
                               </a>
                               ) the action recognition task (ART) to classify the identity of the character based on the proprioceptive inputs, and (
                               <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">
                                2
                               </a>
                               ) the trajectory decoding task (TDT) to decode the endeffector coordinates (at each time step) from proprioceptive inputs. Baseline models (SVMs for the ART and linear regression for TDT) were first trained to investigate the difficulty of the task, followed by a suite of deep neural networks that aim to model the proprioceptive pathway.
                              </p>
                             </div>
                             <div id="sec-20" class="subsection">
                              <h4>
                               Low dimensional embedding of population activity
                              </h4>
                              <p id="p-64">
                               −
                              </p>
                              <p id="p-65">
                               To visualize population activity (of kinematic or network representations), we created low-dimensional embeddings of the proprioceptive inputs (
                               <a id="xref-fig-3-13" class="xref-fig" href="#F3">
                                Figure 3A
                               </a>
                               ) as well as the the layers of the neural network models, along time, and space/muscles dimensions (
                               <a id="xref-fig-4-10" class="xref-fig" href="#F4">
                                Figure 4B
                               </a>
                               and
                               <a id="xref-fig-9-5" class="xref-fig" href="#F9">
                                Figure S2A
                               </a>
                               ). To this end, we first used Principal Components Analysis (PCA) to reduce the space to 50 dimensions, typically retaining around 75−80% of the variance. We then used t-distributed stochastic neighbor embedding (t-SNE, 38) using sklearn (
                               <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">
                                66
                               </a>
                               ) with a perplexity of 40 for 300 iterations, to reduce these 50 dimensions down to two for visualization.
                              </p>
                             </div>
                             <div id="sec-21" class="subsection">
                              <h4>
                               Support vector machine (SVM) analysis for action recognition
                              </h4>
                              <p id="p-66">
                               To establish a baseline performance for multi-class recognition we used pairwise SVMs with the one-against-one method (
                               <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">
                                39
                               </a>
                               ). That is, we train
                               <span class="inline-formula" id="inline-formula-1">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-1.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-1.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               pairwise (linear) SVM classifiers (
                               <a id="xref-fig-3-14" class="xref-fig" href="#F3">
                                Figure 3C
                               </a>
                               ) and at test time implement a voting strategy based on the confidences of each classifier to determine the class identity. We trained SVMs for each input modality (end-effector trajectories, joint angle trajectories, muscle fiber-length trajectories and proprioceptive inputs) to determine how the format affects performance. All pairwise classifiers were trained using a hinge loss, and crossvalidation was performed with 9 regularization constants logarithmically spaced between 10
                               <sup>
                                −4
                               </sup>
                               and 10
                               <sup>
                                4
                               </sup>
                               .
                              </p>
                             </div>
                             <div id="sec-22" class="subsection">
                              <h4>
                               Baseline linear regression model for trajectory decoding
                              </h4>
                              <p id="p-67">
                               To establish how well one could decode endeffector coordinates from the joint, muscle and proprioceptive inputs, we trained linear regressors with ordinary least-squares loss using stochastic gradient descent until the validation loss saturated (with a tolerance of 10
                               <sup>
                                −3
                               </sup>
                               ). Inputs and outputs to the model were first transformed using a standard scaler to center (remove the mean) and scale to unit variance over each feature in order to train faster. At test time, the same scalers were reused. Decoding error was determined as the squared error (L-2 norm) of the predicted and true endeffector coordinates in 3D.
                              </p>
                             </div>
                            </div>
                            <div id="sec-23" class="subsection">
                             <h3>
                              Models of the proprioceptive system
                             </h3>
                             <p id="p-68">
                              We trained two types of convolutional networks and one type of recurrent network on the two tasks. Each model is characterized by the layers used – convolutional and/or recurrent – which specify how the spatial and temporal information in the proprioceptive inputs is processed and integrated.
                             </p>
                             <p id="p-69">
                              Each convolutional layer contains a set of convolutional filters of a given kernel size and stride, along with response normalization and a point-wise non-linearity. The convolutional filters can either be 1-dimensional, processing only spatial or temporal information or 2-dimensional, processing both types of information simultaneously. For response normalization we use layer normalization (
                              <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">
                               67
                              </a>
                              ), a commonly used normalization scheme to train deep neural networks, where the response of a neuron is normalized by the response of all neurons of that layer. As point-wise non-linearity we use rectified linear units. Each recurrent layer contains a single LSTM (long short-term memory) cell with a given number of units that process the input one time step at a time.
                             </p>
                             <p id="p-70">
                              Depending on what type of convolutional layers are used and how they are arranged, we classify convolutional models into two subtypes (
                              <a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">
                               1
                              </a>
                              ) spatial-temporal and (
                              <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">
                               2
                              </a>
                              ) spatiotemporal networks. Spatial-temporal networks are formed by combining multiple 1-dimensional spatial and temporal convolutional layers. That is, the proprioceptive inputs from different muscles are first combined to attain a condensed representation of the ‘spatial’ information in the inputs, through a hierarchy of spatial convolutional layers. This hierarchical arrangement of the layers leads to increasingly larger receptive fields in spatial (or temporal) dimension that typically (for most parameters) gives rise to a representation of the whole arm at some point in the hierarchy. The temporal information is then integrated using temporal convolutional layers. In the spatiotemporal networks, multiple 2-d convolutional layers where convolutional filters are applied simultaneously across spatial and temporal dimensions are stacked together. The LSTM models on the other hand are formed by combining multiple 1-dimensional spatial convolutional layers and a single LSTM layer at the end of a stack of spatial filters that recurrently processes the temporal information. For each network, the features at the final layer are mapped by a single fully connected layer either onto a 20 dimensional (logits) or a 3 dimensional output (end-effector coordinates).
                             </p>
                             <p id="p-71">
                              For each specific network type the following hyper-parameters were used: number of layers, number and size of spatial and temporal filters and their corresponding stride (see
                              <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              ). Using this set of architectural parameters, 50 models of each type were randomly generated. Notably, we trained the same model (as specified by the architecture) on both tasks.
                             </p>
                             <div id="sec-24" class="subsection">
                              <h4>
                               Network training and evaluation procedure
                              </h4>
                              <p id="p-72">
                               The action-recognition trained models were trained by minimizing the softmax cross entropy loss using the Adam Optimizer (
                               <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">
                                45
                               </a>
                               ) with an initial learning rate of 0.0005, batch size of 256 and decay parameters (β
                               <sub>
                                1
                               </sub>
                               and β
                               <sub>
                                2
                               </sub>
                               ) of 0.9 and 0.999. During training the performance was monitored on the left-out validation set. When the validation error did not improve for 5 consecutive epochs, we decreased the learning rate by a factor of 4. After the second time the validation error saturated, we ended the training and evaluated accuracy of the networks on the test set. Overall we observe that the trained networks generalized well to the test data, even though the shallower networks tended to overfit S1A.
                              </p>
                              <p id="p-73">
                               The TDT trained models on the other hand were trained to minimize the mean squared error between predicted and true trajectories. Hyperparameter settings for the optimizer, batch size and early stopping procedure used during training remained same across both tasks. Here, we observe that train and test decoding errors were highly correlated, and thereby achieve excellent generalization to test data S1A.
                              </p>
                             </div>
                             <div id="sec-25" class="subsection">
                              <h4>
                               Comparison with controls
                              </h4>
                              <p id="p-74">
                               For each of the three types of models, the architecture belonging to the best performing model on the ART (as identified via the hyper-parameter search) was chosen as the basis of the analysis (
                               <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">
                                Table 2
                               </a>
                               ). The resulting sizes of each layer’s representation across the hierarchy are given in
                               <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">
                                Table 3
                               </a>
                               . For each different model type, five sets of random weights were initialized and saved. Then, each instantiation was trained on both ART and TDT using the same training procedure as described in the previous section, and the weights were saved again after training. This gives a before and after structure for each run that allows us to isolate the effect of task-training.
                              </p>
                              <div id="T3" class="table pos-float">
                               <div class="table-inline table-callout-links">
                                <div class="callout">
                                 <span>
                                  View this table:
                                 </span>
                                 <ul class="callout-links">
                                  <li class="view-inline first">
                                   <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2598807/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                    View inline
                                   </a>
                                  </li>
                                  <li class="view-popup">
                                   <a href="/highwire/markup/2598807/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                    View popup
                                   </a>
                                  </li>
                                  <li class="download-ppt last">
                                   <a href="/highwire/powerpoint/2598807" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                    Download powerpoint
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                               </div>
                               <div class="table-caption">
                                <span class="table-label">
                                 Table 3.
                                </span>
                                <p id="p-75" class="first-child">
                                 Size of representation at each layer for best performing architecture of each network type (spatial x temporal x filter dimensions).
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-26" class="subsection">
                             <h3>
                              Population comparisons
                             </h3>
                             <div id="sec-27" class="subsection">
                              <h4>
                               Centered Kernel Analysis
                              </h4>
                              <p id="p-76">
                               In order to provide a population-level comparison between the trained and control models (
                               <a id="xref-fig-4-11" class="xref-fig" href="#F4">
                                Figure 4A
                               </a>
                               ), we used Linear Centered Kernel Analysis (CKA) for a high-level comparison of each layers’ activation patterns (
                               <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">
                                68
                               </a>
                               ). CKA is an alternative that extends Canonical Correlation Analysis (CCA) by weighting activation patterns by the eigenvalues of the corresponding eigenvectors (
                               <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">
                                68
                               </a>
                               ). As such, it maintains CCA’s invariance to orthogonal transformations and isotropic scaling, yet retains a greater sensitivity to similarities. Using this analysis, we quantified the similarity of the activation of each layer of the trained models with those of the respective controls in response to identical stimuli comprising 50% of the test set for each of the five model instantiations.
                              </p>
                             </div>
                             <div id="sec-28" class="subsection">
                              <h4>
                               Representational similarity analysis
                              </h4>
                              <p id="p-77">
                               Representational Similarity Analysis (RSA) is a tool to investigate population level representations among competing models (
                               <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">
                                69
                               </a>
                               ). The basic building block of RSA is a representational dissimilarity matrix (RDM). Given stimuli {s
                               <sub>
                                1
                               </sub>
                               , s
                               <sub>
                                2
                               </sub>
                               , …, s
                               <sub>
                                n
                               </sub>
                               } and vectors of population responses {r
                               <sub>
                                1
                               </sub>
                               , r
                               <sub>
                                2
                               </sub>
                               , …, r
                               <sub>
                                n
                               </sub>
                               }, the RDM is defined as:
                               <span class="disp-formula" id="disp-formula-6">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-16.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-16.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-78">
                               One of the main advantages of RDMs is that it characterizes the geometry of stimulus representation in a way that is independent of the dimensionality of the feature representations, so we can easily compare between arbitrary representations of a given stimulus set. Example RDMs for proprioceptive inputs, as well as the final layer before the readout for the best models of each type are shown in
                               <a id="xref-fig-4-12" class="xref-fig" href="#F4">
                                Figure 4C
                               </a>
                               ,
                               <a id="xref-fig-9-6" class="xref-fig" href="#F9">
                                S2B
                               </a>
                               . Each RDM is computed for a random sample of 4,000 character trajectories (200 from each class) by using the correlation distance between corresponding feature representations. To compactly summarize how well a network disentangles the stimuli we compare the RDM of each layer to the RDM of the ideal observer, which has a RDM with perfect block structure (with dissimilarity values 0 for all stimuli of the same class and 1 (100 percentile) otherwise; see
                               <a id="xref-fig-9-7" class="xref-fig" href="#F9">
                                Figure S2B
                               </a>
                               ).
                              </p>
                             </div>
                            </div>
                            <div id="sec-29" class="subsection">
                             <h3>
                              Single unit analysis
                             </h3>
                             <div id="sec-30" class="subsection">
                              <h4>
                               Comparing the tuning curves
                              </h4>
                              <p id="p-79">
                               To elucidate the emerging coding properties of single units, we determined label specificity and fit tuning curves. Specifically, we focused on kinematic properties such as direction, velocity, acceleration and position of the end-effector for movements (
                               <a id="xref-fig-5-7" class="xref-fig" href="#F5">
                                Figure 5
                               </a>
                               ,
                               <a id="xref-fig-7-9" class="xref-fig" href="#F7">
                                7
                               </a>
                               ). For computational tractability, 20, 000 of the original trajectories were randomly selected for the ART-trained S and ST models, and 4000 for TDT-trained ones as well as for both kinds of LSTM models. Each time point was treated as its own independent sample. In convolutional layers in which the hidden layers had a reduced temporal dimensionality than the input, the input trace was downsampled. Only those time points were kept that correpond to the center of the receptive fields of the units in the hidden layers.
                              </p>
                              <p id="p-80">
                               A train-test split of 80 − 20 was used. The tuning curves were fit and tested jointly on all movements in planes with a common orientation, vertical or horizontal. The analysis was repeated for each of the five trained and control models. For each of the five different types of tuning curves (the four biological ones and label specificity) and for each model instantiation, distributions of test scores were computed (
                               <a id="xref-fig-5-8" class="xref-fig" href="#F5">
                                Figure 5
                               </a>
                               ,
                               <a id="xref-fig-7-10" class="xref-fig" href="#F7">
                                7
                               </a>
                               ).
                              </p>
                              <p id="p-81">
                               When plotting comparisons between different types of models (ART, TDT, and controls), the confidence interval for the mean (CLM) using an α = 5% significance level based on the t-statistic was displayed.
                              </p>
                             </div>
                             <div id="sec-31" class="subsection">
                              <h4>
                               Label tuning (selectivity index)
                              </h4>
                              <p id="p-82">
                               The networks’ ability to solve the proprioceptive task poses the question if individual neurons serve as character detectors. To this end, SVMs were fit with linear kernels using a one vs. rest strategy for multi-class classification based on the firing rate of each node, resulting in linear decision boundaries for each letter. Each individual SVM serves as a binary classifier for the trajectory belonging to a certain character or not, based on that neuron’s firing rates. For each SVM, auROC was calculated, giving a measure of how well the label can be determined based on the firing rate of an individual node alone. The label specificity of that node was then determined by taking the maximum over all characters. Finally, the auROC score was normalized into a selectivity index: 2 ((auROC) − 0.5)).
                              </p>
                             </div>
                             <div id="sec-32" class="subsection">
                              <h4>
                               Position, direction, velocity, & acceleration
                              </h4>
                              <p id="p-83">
                               For the kinematic tuning curves the coefficient of determination R
                               <sup>
                                2
                               </sup>
                               on the test set was used as the primary metric of evaluation. These tuning curves were fitted using ordinary least squares linear regression, with regularization proving unnecessary due to the high number of data points and the low number of parameters (
                               <a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">
                                2
                               </a>
                               -
                               <a id="xref-ref-3-8" class="xref-bibr" href="#ref-3">
                                3
                               </a>
                               ) in the models.
                              </p>
                             </div>
                             <div id="sec-33" class="subsection">
                              <h4>
                               Position tuning
                              </h4>
                              <p id="p-84">
                               Position
                               <span class="inline-formula" id="inline-formula-2">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-2.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-2.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is initially defined with respect to the center of the workspace. For trajectories in a horizontal plane (workspace), a position vector was defined with respect to the starting position
                               <span class="inline-formula" id="inline-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-3.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-3.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               of each trace,
                               <span class="inline-formula" id="inline-formula-4">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-4.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-4.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . This was also represented in polar coordinates
                               <span class="inline-formula" id="inline-formula-5">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-5.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-5.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , where φ
                               <sub>
                                t
                               </sub>
                               ∈ (− π, π] is the angle measured with the counterclockwise direction defined as positive between the position vector and the vector
                               <span class="inline-formula" id="inline-formula-6">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-6.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-6.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , i.e. the vector extending away from the body, and
                               <span class="inline-formula" id="inline-formula-7">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-7.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-7.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . Positional tuning of the neural activity N of node ν was evaluated by fitting models both using Cartesian coordinates,
                               <span class="disp-formula" id="disp-formula-7">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-17.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-17.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               as well as polar ones,
                               <span class="disp-formula" id="disp-formula-8">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-18.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-18.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where φ
                               <sub>
                                PD
                               </sub>
                               is a parameter representing a neuron’s preferred direction for position. For trajectories in the vertical plane, all definitions are equivalent, but with coordinates (y, z)
                               <sup>
                                T
                               </sup>
                               .
                              </p>
                             </div>
                             <div id="sec-34" class="subsection">
                              <h4>
                               Direction
                              </h4>
                              <p id="p-85">
                               In order to examine the strength of kinematic tuning, tuning curves relating direction, velocity, and acceleration to neural activity were fitted. Since all trajectories take place either in a horizontal or vertical plane, the instantaneous velocity vector at time t can be described in two components as
                               <span class="inline-formula" id="inline-formula-8">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-8.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-8.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               or (y, z)
                               <sup>
                                T
                               </sup>
                               for trajectories in a vertical plane, or alternately in polar coordinates,
                               <span class="inline-formula" id="inline-formula-9">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-9.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-9.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , with θ
                               <sub>
                                t
                               </sub>
                               ∈ (−π, π] representing the angle between the velocity vector and the x-axis, and
                               <span class="inline-formula" id="inline-formula-10">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-10.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-10.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               representing the speed.
                              </p>
                              <p id="p-86">
                               First, a tuning curve was fit that excludes the magnitude of velocity but focuses on the instantaneous direction, putting the angle of the polar representation of velocity θ
                               <sub>
                                t
                               </sub>
                               in relation to each neuron’s preferred direction θ
                               <sub>
                                PD
                               </sub>
                               .
                               <span class="disp-formula" id="disp-formula-9">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-19.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-19.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-87">
                               To fit this model, 9 was re-expressed as a simple linear sum using the cosine sum and difference formula cos(α + β) = cos α cos β − sin α sin β, a reformulation that eases the computational burden of the analysis significantly (
                               <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">
                                70
                               </a>
                               ). In this formulation, the equation for directional tuning becomes:
                               <span class="disp-formula" id="disp-formula-10">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-20.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-20.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-88">
                               The preferred direction θ
                               <sub>
                                PD
                               </sub>
                               is now contained in the in the coefficients α
                               <sub>
                                1
                               </sub>
                               = α cos θ
                               <sub>
                                PD
                               </sub>
                               and α
                               <sub>
                                2
                               </sub>
                               = α sin θ
                               <sub>
                                PD
                               </sub>
                               .
                              </p>
                              <p id="p-89">
                               The quality of fit of this type of tuning curve was visualized using polar scatter plots in which the angle of the data point corresponds to the angle θ in the polar representation of velocity and the radius corresponds to the node’s activation. In the figures the direction of movement was defined so that 0
                               <sup>
                                °
                               </sup>
                               (Y) corresponds to movement to the right of the body and progressing counterclockwise, a movement straight (“forward”) away from the body corresponds to 90
                               <sup>
                                °
                               </sup>
                               (X) (
                               <a id="xref-fig-5-9" class="xref-fig" href="#F5">
                                Figures 5A, B
                               </a>
                               ;
                               <a id="xref-fig-7-11" class="xref-fig" href="#F7">
                                7A, B, C
                               </a>
                               ).
                              </p>
                             </div>
                             <div id="sec-35" class="subsection">
                              <h4>
                               Speed
                              </h4>
                              <p id="p-90">
                               Two linear models for activity N at a node ν for velocity were fit.
                              </p>
                              <p id="p-91">
                               The first is based on its magnitude, speed,
                               <span class="disp-formula" id="disp-formula-11">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-21.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-21.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <div id="sec-36" class="subsection">
                              <h4>
                               Velocity
                              </h4>
                              <p id="p-92">
                               The second velocity-based tuning curve factors in both directional and speed components:
                               <span class="disp-formula" id="disp-formula-12">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-22.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-22.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-93">
                               The quality of fit of this type of tuning curve was visualized using polar filled contour plots in which the angle of the data point corresponds to the angle θ in the polar representation of velocity, the radius corresponds to the speed, and the node’s activation is represented by the height. For the visualizations (
                               <a id="xref-fig-5-10" class="xref-fig" href="#F5">
                                Figure 5B
                               </a>
                               ), to cover the whole range of angle and radius given a finite number of samples, the activation was first linearly interpolated. Then, missing regions were filled in using nearest neighbor interpolation. Finally, the contour was smoothed using a Gaussian filter.
                              </p>
                             </div>
                             <div id="sec-37" class="subsection">
                              <h4>
                               Acceleration
                              </h4>
                              <p id="p-94">
                               Acceleration is defined analogously to velocity by
                               <span class="inline-formula" id="inline-formula-11">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-11.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-11.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-12">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-12.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/inline-graphic-12.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . A simple linear relationship with acceleration magnitude was tested:
                               <span class="disp-formula" id="disp-formula-13">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-23.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/15/2020.05.06.081372/embed/graphic-23.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-95">
                               In subsequent analyses, scores were excluded if they were equal to 1 (indicating a dead neuron whose output was constant) or if they were less than -0.1 (indicating a fit that did not converge).
                              </p>
                             </div>
                             <div id="sec-38" class="subsection">
                              <h4>
                               Classification of neurons into different types
                              </h4>
                              <p id="p-96">
                               The neurons were classified as belonging to a certain type if the corresponding kinematic model yielded a test-R
                               <sup>
                                2
                               </sup>
                               > 0.2. Seven different model types were evaluated:
                              </p>
                              <ol class="list-ord" id="list-1">
                               <li id="list-item-1">
                                <p id="p-97">
                                 Direction tuning
                                </p>
                               </li>
                               <li id="list-item-2">
                                <p id="p-98">
                                 Velocity tuning
                                </p>
                               </li>
                               <li id="list-item-3">
                                <p id="p-99">
                                 Direction & velocity tuning
                                </p>
                               </li>
                               <li id="list-item-4">
                                <p id="p-100">
                                 Position (Cartesian)
                                </p>
                               </li>
                               <li id="list-item-5">
                                <p id="p-101">
                                 Position (Polar)
                                </p>
                               </li>
                               <li id="list-item-6">
                                <p id="p-102">
                                 Acceleration tuning
                                </p>
                               </li>
                               <li id="list-item-7">
                                <p id="p-103">
                                 Label specificity
                                </p>
                               </li>
                              </ol>
                              <p id="p-104">
                               These were treated as distinct classes for the purposes of classification (
                               <a id="xref-fig-5-11" class="xref-fig" href="#F5">
                                Figure 5C
                               </a>
                               ).
                              </p>
                             </div>
                            </div>
                            <div id="sec-39" class="subsection">
                             <h3>
                              Population Decoding Analysis
                             </h3>
                             <p id="p-105">
                              We also performed population-level decoding analysis for the kinematic tuning curve types (
                              <a id="xref-fig-11-3" class="xref-fig" href="#F11">
                               Figure S4
                              </a>
                              ,
                              <a id="xref-fig-15-1" class="xref-fig" href="#F15">
                               S8
                              </a>
                              ,
                              <a id="xref-fig-14-3" class="xref-fig" href="#F14">
                               S7
                              </a>
                              ). The same data sets were used as for the single cell encoding analysis, except with switched predictors and targets. The firing rates of all neurons in a hidden layer at a single time point were jointly used as predictors for the kinematic variable at the center of the receptive field at the corresponding input layer.
                             </p>
                             <div id="F15" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F15.large.jpg?width=800&height=600&carousel=1" title="Analysis of population decoding for TDT-trained and control models. (A) Population decoding of speed (light) and direction (dark) for the TDT-trained and control for spatial-temporal models (left), spatiotemporal (middle) and LSTM (right) models. The faint line shows the R2 score for an individual model; the dark one the mean over all instantiations (N = 5). (B) Population decoding of endeffector position (X and Y coordinates) for spatial-temporal models (left), spatiotemporal (middle) and LSTM (right) models. The faint line shows the R2 score for an individual model; the dark one the mean over all instantiations (N = 5)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1989804583" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Analysis of population decoding for TDT-trained and control models.</span> (<strong>A</strong>) Population decoding of speed (<em>light</em>) and direction (<em>dark</em>) for the TDT-trained and control for spatial-temporal models (<em>left</em>), spatiotemporal (<em>middle</em>) and LSTM (<em>right</em>) models. The faint line shows the <em>R</em><sup>2</sup> score for an individual model; the dark one the mean over all instantiations (<em>N</em> = 5). (<strong>B</strong>) Population decoding of endeffector position (X and Y coordinates) for spatial-temporal models (<em>left</em>), spatiotemporal (<em>middle</em>) and LSTM (<em>right</em>) models. The faint line shows the <em>R</em><sup>2</sup> score for an individual model; the dark one the mean over all instantiations (<em>N</em> = 5).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F15.medium.gif" width="440" height="217"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S8." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F15.medium.gif" width="440" height="217"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S8." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S8.
                               </span>
                               <span class="caption-title">
                                Analysis of population decoding for TDT-trained and control models.
                               </span>
                               <p id="p-126" class="first-child">
                                (
                                <strong>
                                 A
                                </strong>
                                ) Population decoding of speed (light) and direction (dark) for the TDT-trained and control for spatial-temporal models (left), spatiotemporal (middle) and LSTM (right) models. The faint line shows the R
                                <sup>
                                 2
                                </sup>
                                score for an individual model; the dark one the mean over all instantiations (N = 5). (
                                <strong>
                                 B
                                </strong>
                                ) Population decoding of endeffector position (X and Y coordinates) for spatial-temporal models (left), spatiotemporal (middle) and LSTM (right) models. The faint line shows the R
                                <sup>
                                 2
                                </sup>
                                score for an individual model; the dark one the mean over all instantiations (N = 5).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-106">
                              This analysis was repeated for each of the following kinematic variables:
                             </p>
                             <ol class="list-ord" id="list-2">
                              <li id="list-item-8">
                               <p id="p-107">
                                Direction
                               </p>
                              </li>
                              <li id="list-item-9">
                               <p id="p-108">
                                Speed
                               </p>
                              </li>
                              <li id="list-item-10">
                               <p id="p-109">
                                X Position (Cartesian)
                               </p>
                              </li>
                              <li id="list-item-11">
                               <p id="p-110">
                                Y Position (Cartesian)
                               </p>
                              </li>
                             </ol>
                             <p id="p-111">
                              For each of these, the accuracy was evaluate using r2 score. The encoding strength for X and Y position in Cartesian coordinates was additionally jointly evaluated by calculating the average distance between true and predicted points of the trajectory. To prevent overfitting, ridge regularization was used with a regularization strength of α = 1.
                             </p>
                            </div>
                            <div id="sec-40" class="subsection">
                             <h3>
                              Distribution of Preferred Directions
                             </h3>
                             <p id="p-112">
                              Higher order features of the models were also evaluated and compared between the trained models and their controls. The first property was the distribution of preferred directions fit for all horizontal planes in each layer. If a neuron’s directiononly tuning yields a test-R
                              <sup>
                               2
                              </sup>
                              > 0.2, its preferred direction was included in the distribution. Within a layer, the preferred direction of all neurons was binned into 18 equidistant intervals (
                              <a id="xref-fig-7-12" class="xref-fig" href="#F7">
                               Figure 7B,C
                              </a>
                              ) in order to enable a direct comparison with the findings by Prud’homme and Kalaska (
                              <a id="xref-ref-14-8" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                              ). They found that the preferred direction of tuning curves was relatively evenly spread in S1 (
                              <a id="xref-fig-7-13" class="xref-fig" href="#F7">
                               Figure 7A
                              </a>
                              ); our analysis showed that this was not the case for muscle spindles (
                              <a id="xref-fig-7-14" class="xref-fig" href="#F7">
                               Figure 7B
                              </a>
                              ). Thus, we formed the hypothesis that the preferred directions in the trained networks was more uniform in the trained networks than in the random ones. For quantification, absolute deviation from uniformity was used as a metric. To calculate this metric, the deviation from the mean height of a bin in the circular histograms was calculated for each angular bin. Then, the absolute value of this deviation was summed over all bins. We then normalize the result by the number of significantly directionally tuned neurons in a layer, and compare the result for the trained and control networks (
                              <a id="xref-fig-7-15" class="xref-fig" href="#F7">
                               Figure 7D
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-41" class="subsection">
                             <h3>
                              Preferred direction invariance
                             </h3>
                             <p id="p-113">
                              We also hypothesized that the representation in the trained network would be more invariant across different horizontal and vertical planes, respectively. To test this, directional tuning curves were fit for each individual plane. A central plane was chosen as a basis of comparison (plane at z = 0 for the horizontal planes and at x = 30 for vertical). Changes in preferred direction of neurons are shown for spindles (
                              <a id="xref-fig-16-8" class="xref-fig" href="#F16">
                               Figure S9A
                              </a>
                              ), as well as for neurons of layer 5 of one instantiation of the trained and control spatial-temporal model (
                              <a id="xref-fig-16-9" class="xref-fig" href="#F16">
                               Figure S9B
                              </a>
                              ). Generalization was then evaluated as follows: for neurons with R
                              <sup>
                               2
                              </sup>
                              > 0.2, the average deviation of the neurons’ preferred directions over all different planes from those in the central plane was summed up and normalized by the number of planes and neurons, yielding a total measure for the neurons’ consistency in preferred direction in any given layer (vertical:
                              <a id="xref-fig-7-16" class="xref-fig" href="#F7">
                               Figure 7F
                              </a>
                              ; horizontal:
                              <a id="xref-fig-7-17" class="xref-fig" href="#F7">
                               Figure 7E
                              </a>
                              ). If a plane had fewer than three directionally tuned neurons, its results were excluded.
                             </p>
                             <div id="F16" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F16.large.jpg?width=800&height=600&carousel=1" title="Invariance of preferred orientations. (A) For quantifying invariance we calculated mean absolute deviation in preferred orientation for units from a central plane at z = 0 to each other horizontal plane (for units with R2 > 0.2). Results are shown for each instantiation (N = 5, faint lines) for trained and control models plus mean (solid) and 95%-confidence intervals over instantiations (N = 5) for the spatial-temporal (left), spatiotemporal (right), and LSTM (right) networks. Note that there is no data for layer 4 of the trained spatiotemporal model, as it has no direction selective units (R2 > 0.2). (B) Deviation in preferred direction for individual spindles (N=25). The preferred directions are fit for each plane and displayed in relation to a central horizontal (left) and vertical plane (right). Individual gray lines are for all units (spindles) with R2 > 0.2, the thick red line marks the mean. (C) Same as B, but for direction tuning in vertical planes for units in layer 5 of one instantiation of the best spatial-temporal model for the trained (left) and control model (right). Individual gray lines are for units with R2 > 0.2, and the red line is the plane-wise mean. (D) Same as in B but for layer 5 of the trained spatial-temporal network. (E) Same as in D but for layer 5 of the corresponding control network." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1989804583" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Invariance of preferred orientations.</span> (<strong>A</strong>) For quantifying invariance we calculated mean absolute deviation in preferred orientation for units from a central plane at <em>z</em> = 0 to each other horizontal plane (for units with <em>R</em><sup>2</sup> <em>></em> 0.2). Results are shown for each instantiation (<em>N</em> = 5, faint lines) for trained and control models plus mean (solid) and 95%-confidence intervals over instantiations (<em>N</em> = 5) for the spatial-temporal <em>(left)</em>, spatiotemporal <em>(right)</em>, and LSTM <em>(right)</em> networks. Note that there is no data for layer 4 of the trained spatiotemporal model, as it has no direction selective units (<em>R</em><sup>2</sup> <em>></em> 0.2). (<strong>B</strong>) Deviation in preferred direction for individual spindles (N=25). The preferred directions are fit for each plane and displayed in relation to a central horizontal (<em>left</em>) and vertical plane (<em>right</em>). Individual gray lines are for all units (spindles) with <em>R</em><sup>2</sup> <em>></em> 0.2, the thick red line marks the mean. (<strong>C</strong>) Same as B, but for direction tuning in vertical planes for units in layer 5 of one instantiation of the best spatial-temporal model for the trained (<em>left</em>) and control model (<em>right</em>). Individual gray lines are for units with <em>R</em><sup>2</sup> <em>></em> 0.2, and the red line is the plane-wise mean. (<strong>D</strong>) Same as in B but for layer 5 of the trained spatial-temporal network. (<strong>E</strong>) Same as in D but for layer 5 of the corresponding control network.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F16.medium.gif" width="440" height="409"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S9." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F16.medium.gif" width="440" height="409"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S9." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S9.
                               </span>
                               <span class="caption-title">
                                Invariance of preferred orientations.
                               </span>
                               <p id="p-127" class="first-child">
                                (
                                <strong>
                                 A
                                </strong>
                                ) For quantifying invariance we calculated mean absolute deviation in preferred orientation for units from a central plane at z = 0 to each other horizontal plane (for units with R
                                <sup>
                                 2
                                </sup>
                                > 0.2). Results are shown for each instantiation (N = 5, faint lines) for trained and control models plus mean (solid) and 95%-confidence intervals over instantiations (N = 5) for the spatial-temporal (left), spatiotemporal (right), and LSTM (right) networks. Note that there is no data for layer 4 of the trained spatiotemporal model, as it has no direction selective units (R
                                <sup>
                                 2
                                </sup>
                                > 0.2). (
                                <strong>
                                 B
                                </strong>
                                ) Deviation in preferred direction for individual spindles (N=25). The preferred directions are fit for each plane and displayed in relation to a central horizontal (left) and vertical plane (right). Individual gray lines are for all units (spindles) with R
                                <sup>
                                 2
                                </sup>
                                > 0.2, the thick red line marks the mean. (
                                <strong>
                                 C
                                </strong>
                                ) Same as B, but for direction tuning in vertical planes for units in layer 5 of one instantiation of the best spatial-temporal model for the trained (left) and control model (right). Individual gray lines are for units with R
                                <sup>
                                 2
                                </sup>
                                > 0.2, and the red line is the plane-wise mean. (
                                <strong>
                                 D
                                </strong>
                                ) Same as in B but for layer 5 of the trained spatial-temporal network. (
                                <strong>
                                 E
                                </strong>
                                ) Same as in D but for layer 5 of the corresponding control network.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F17" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F17.large.jpg?width=800&height=600&carousel=1" title="Supplementary Video Video depicts the OpenSim model being passively moved to match the human-drawn character “a” for three different variants; drawn vertically (left, right) and horizontally (middle)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1989804583" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Supplementary Video</span> Video depicts the OpenSim model being passively moved to match the human-drawn character “a” for three different variants; drawn vertically (left, right) and horizontally (middle).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S10." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F17.medium.gif" width="440" height="145"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S10." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F17.medium.gif" width="440" height="145"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S10." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/15/2020.05.06.081372/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S10.
                               </span>
                               <span class="caption-title">
                                Supplementary Video
                               </span>
                               <p id="p-128" class="first-child">
                                Video depicts the OpenSim model being passively moved to match the human-drawn character “a” for three different variants; drawn vertically (left, right) and horizontally (middle).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-42" class="subsection">
                             <h3>
                              Statistical Testing
                             </h3>
                             <p id="p-114">
                              To test whether differences were statistically significant between trained and control models paired t-tests were used with a pre-set significance level of α = 0.05.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-44">
                            <h2 class="">
                             Software
                            </h2>
                            <p id="p-115">
                             We used the scientific Python stack (
                             <a href="http://python.org">
                              python.org
                             </a>
                             ): Numpy, Pandas, Matplotlib, SciPy (
                             <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">
                              71
                             </a>
                             ) and scikit-learn (
                             <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">
                              66
                             </a>
                             ). Open-Sim (
                             <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">
                              34
                             </a>
                             ,
                             <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">
                              62
                             </a>
                             ,
                             <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">
                              63
                             </a>
                             ) was used for biomechanics simulations and Tensorflow was used for constructing and training the neural network models (
                             <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">
                              72
                             </a>
                             ).
                            </p>
                           </div>
                           <div class="section" id="sec-45">
                            <h2 class="">
                             Code and data
                            </h2>
                            <p id="p-116">
                             Code and data will be shared upon publication.
                            </p>
                           </div>
                           <div class="section" id="sec-46">
                            <h2 class="">
                             Funding
                            </h2>
                            <p id="p-118">
                             K.J.S.: Werner Siemens Fellowship of the Swiss Study Foundation; P. M.: Smart Start I, Bernstein Center for Computational Neuroscience; M.W.M: the Rowland Fellowship from the Rowland Institute at Harvard. A.M. and M.W.M. funding from EPFL.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgments
                            </h2>
                            <p id="p-117">
                             We are grateful to Alessandro Marin Vargas, Alberto Chiappa, Andy Bennetto and Bartek Borzyszkowski for comments on an earlier version of the manuscript. K.J.S. Current Address: Department of Experimental Psychology, University of Oxford, United Kingdom.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-others" id="fn-1">
                              <p id="p-1">
                               <a class="rev-xref" href="#xref-fn-1-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                *
                               </span>
                               co-first authors;
                              </p>
                             </li>
                             <li class="fn-others" id="fn-2">
                              <p id="p-2">
                               <a class="rev-xref" href="#xref-fn-2-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                †
                               </span>
                               co-senior authors;
                              </p>
                             </li>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-3">
                              <p id="p-6">
                               Introduced a second task, recurrent models and many other updates.
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                  <span class="cit-name-surname">
                                   Chris Miall
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nick M
                                  </span>
                                  <span class="cit-name-surname">
                                   Kitchen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Se-Ho
                                  </span>
                                  <span class="cit-name-surname">
                                   Nam
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Hannah
                                  </span>
                                  <span class="cit-name-surname">
                                   Lefumat
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alix G
                                  </span>
                                  <span class="cit-name-surname">
                                   Renault
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kristin
                                  </span>
                                  <span class="cit-name-surname">
                                   Ørstavik
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jonathan D
                                  </span>
                                  <span class="cit-name-surname">
                                   Cole
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Fabrice R
                                  </span>
                                  <span class="cit-name-surname">
                                   Sarlegna
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Proprioceptive loss and the perception, control and learning of arm movements in humans: evidence from sensory neuronopathy
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Experimental brain research
                                 </abbr>
                                 , pages
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  19
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.2" data-doi="10.1152/physrev.00048.2011">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Uwe
                                  </span>
                                  <span class="cit-name-surname">
                                   Proske
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Simon C
                                  </span>
                                  <span class="cit-name-surname">
                                   Gandevia
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The Proprioceptive Senses: Their Roles in Signaling Body Shape, Body Position and Movement, and Muscle Force
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Physiological Reviews
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  92
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1651
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1697
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 . ISSN
                                 <span class="cit-issn">
                                  0031-9333
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1152/physrev.00048.2011
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhysiological%2BReviews%26rft.stitle%253DPhysiol.%2BRev.%26rft.aulast%253DProske%26rft.auinit1%253DU.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D1651%26rft.epage%253D1697%26rft.atitle%253DThe%2BProprioceptive%2BSenses%253A%2BTheir%2BRoles%2Bin%2BSignaling%2BBody%2BShape%252C%2BBody%2BPosition%2Band%2BMovement%252C%2Band%2BMuscle%2BForce%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysrev.00048.2011%26rft_id%253Dinfo%253Apmid%252F23073629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1152/physrev.00048.2011&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23073629&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Benoit P
                                  </span>
                                  <span class="cit-name-surname">
                                   Delhaye
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Katie H
                                  </span>
                                  <span class="cit-name-surname">
                                   Long
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sliman J
                                  </span>
                                  <span class="cit-name-surname">
                                   Bensmaia
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Neural basis of touch and proprioception in primate cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Comprehensive Physiology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1575
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DComprehensive%2BPhysiology%26rft.volume%253D8%26rft.spage%253D1575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Francis J
                                  </span>
                                  <span class="cit-name-surname">
                                   Clark
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Curt R
                                  </span>
                                  <span class="cit-name-surname">
                                   Burgess
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   James W
                                  </span>
                                  <span class="cit-name-surname">
                                   Chapin
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   WT
                                  </span>
                                  <span class="cit-name-surname">
                                   Lipscomb
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Role of intramuscular receptors in the awareness of limb position
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neurophysiology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  54
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1529
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1540
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1985
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeurophysiology%26rft.stitle%253DJ.%2BNeurophysiol.%26rft.aulast%253DClark%26rft.auinit1%253DF.%2BJ.%26rft.volume%253D54%26rft.issue%253D6%26rft.spage%253D1529%26rft.epage%253D1540%26rft.atitle%253DRole%2Bof%2Bintramuscular%2Breceptors%2Bin%2Bthe%2Bawareness%2Bof%2Blimb%2Bposition%26rft_id%253Dinfo%253Apmid%252F4087047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=4087047&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1985AXT2000012&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.5" data-doi="10.1113/jphysiol.1963.sp007214">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Peter BC
                                  </span>
                                  <span class="cit-name-surname">
                                   Matthews
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The response of de-efferented muscle spindle receptors to stretching at different velocities
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Journal of Physiology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  168
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  660
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  678
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1963
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BPhysiology%26rft.stitle%253DJ.%2BPhysiol.%26rft.aulast%253DMatthews%26rft.auinit1%253DP.%2BB.%2BC.%26rft.volume%253D168%26rft.issue%253D3%26rft.spage%253D660%26rft.epage%253D678%26rft.atitle%253DThe%2Bresponse%2Bof%2Bde-efferented%2Bmuscle%2Bspindle%2Breceptors%2Bto%2Bstretching%2Bat%2Bdifferent%2Bvelocities%26rft_id%253Dinfo%253Adoi%252F10.1113%252Fjphysiol.1963.sp007214%26rft_id%253Dinfo%253Apmid%252F14067950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1113/jphysiol.1963.sp007214&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=14067950&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A19636755B00008&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.05.06.081372v3.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Peter BC
                                  </span>
                                  <span class="cit-name-surname">
                                   Matthews
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-chapter-title">
                                  Muscle spindles: their messages and their fusimotor supply
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Handbook of physiology: I. The nervous system
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  American Physiological Society.[AGF]
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1981
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.05.06.081372v3.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nikolai A
                                  </span>
                                  <span class="cit-name-surname">
                                   Bernstein
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  The co-ordination and regulation of movements
                                 </span>
                                 , volume
                                 <span class="cit-vol">
                                  1
                                 </span>
                                 .
                                 <span class="cit-publ-loc">
                                  Oxford, New York
                                 </span>
                                 ,
                                 <span class="cit-publ-name">
                                  Pergamon Press
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1967
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Gianfranco
                                  </span>
                                  <span class="cit-name-surname">
                                   Bosco
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Rankin
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Richard E
                                  </span>
                                  <span class="cit-name-surname">
                                   Poppele
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Representation of passive hindlimb postures in cat spinocerebellar activity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Neurophysiol
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  76
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  715
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  26
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 . ISSN
                                 <span class="cit-issn">
                                  0022-3077
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeurophysiology%26rft.stitle%253DJ.%2BNeurophysiol.%26rft.aulast%253DBosco%26rft.auinit1%253DG.%26rft.volume%253D76%26rft.issue%253D2%26rft.spage%253D715%26rft.epage%253D726%26rft.atitle%253DRepresentation%2Bof%2Bpassive%2Bhindlimb%2Bpostures%2Bin%2Bcat%2Bspinocerebellar%2Bactivity%26rft_id%253Dinfo%253Apmid%252F8871193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=8871193&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1996VD97200005&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.9" data-doi="10.1016/j.cub.2017.11.070">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tuthill
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Eiman
                                  </span>
                                  <span class="cit-name-surname">
                                   Azim
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Proprioception
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Current Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  R193
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  R194
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 . ISSN
                                 <span class="cit-issn">
                                  09609822
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.cub.2018.01.064
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2BBiology%26rft.volume%253D28%26rft.spage%253DR193%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2017.11.070%26rft_id%253Dinfo%253Apmid%252F29510102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cub.2017.11.070&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29510102&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.10" data-doi="10.1152/jn.01206.2007">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joseph T
                                  </span>
                                  <span class="cit-name-surname">
                                   Francis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Shaohua
                                  </span>
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John K
                                  </span>
                                  <span class="cit-name-surname">
                                   Chapin
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Proprioceptive and cutaneous representations in the rat ventral posterolateral thalamus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of neurophysiology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  99
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2291
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2304
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bneurophysiology%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fjn.01206.2007%26rft_id%253Dinfo%253Apmid%252F18287546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1152/jn.01206.2007&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18287546&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000255811500022&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   James M
                                  </span>
                                  <span class="cit-name-surname">
                                   Goodman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Gregg A
                                  </span>
                                  <span class="cit-name-surname">
                                   Tabot
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alex S
                                  </span>
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Aneesha K
                                  </span>
                                  <span class="cit-name-surname">
                                   Suresh
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alexander T
                                  </span>
                                  <span class="cit-name-surname">
                                   Rajan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nicholas G
                                  </span>
                                  <span class="cit-name-surname">
                                   Hatsopoulos
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sliman
                                  </span>
                                  <span class="cit-name-surname">
                                   Bensmaia
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Postural representations of the hand in the primate sensorimotor cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  104
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1000
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1009
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D104%26rft.spage%253D1000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Raeed H
                                  </span>
                                  <span class="cit-name-surname">
                                   Chowdhury
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joshua I
                                  </span>
                                  <span class="cit-name-surname">
                                   Glaser
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Lee E
                                  </span>
                                  <span class="cit-name-surname">
                                   Miller
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Area 2 of primary somatosensory cortex encodes kinematics of the whole arm
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Elife
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.13" data-doi="10.1016/0006-8993(82)90781-8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christoph
                                  </span>
                                  <span class="cit-name-surname">
                                   Fromm
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Edward V
                                  </span>
                                  <span class="cit-name-surname">
                                   Evarts
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Pyramidal tract neurons in somatosensory cortex: central and peripheral inputs during voluntary movement
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Brain research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  238
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  186
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  191
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1982
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBrain%2Bresearch%26rft.stitle%253DBrain%2BRes%26rft.aulast%253DFromm%26rft.auinit1%253DC.%26rft.volume%253D238%26rft.issue%253D1%26rft.spage%253D186%26rft.epage%253D191%26rft.atitle%253DPyramidal%2Btract%2Bneurons%2Bin%2Bsomatosensory%2Bcortex%253A%2Bcentral%2Band%2Bperipheral%2Binputs%2Bduring%2Bvoluntary%2Bmovement.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0006-8993%252882%252990781-8%26rft_id%253Dinfo%253Apmid%252F6805854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/0006-8993(82)90781-8&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=6805854&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1982NN92400016&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Michel
                                  </span>
                                  <span class="cit-name-given-names">
                                   JL Prud’Homme
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John F
                                  </span>
                                  <span class="cit-name-surname">
                                   Kalaska
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Proprioceptive activity in primate primary somatosensory cortex during active arm reaching movements
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of neurophysiology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  72
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2280
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2301
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1994
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeurophysiology%26rft.stitle%253DJ.%2BNeurophysiol.%26rft.aulast%253DPrud%2527homme%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D72%26rft.issue%253D5%26rft.spage%253D2280%26rft.epage%253D2301%26rft.atitle%253DProprioceptive%2Bactivity%2Bin%2Bprimate%2Bprimary%2Bsomatosensory%2Bcortex%2Bduring%2Bactive%2Barm%2Breaching%2Bmovements%26rft_id%253Dinfo%253Apmid%252F7884459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=7884459&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1994PT20200018&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.06.081372v3.15" data-doi="10.1016/j.neuron.2017.02.049">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mackenzie W
                                  </span>
                                  <span class="cit-name-surname">
                                   Mathis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alexander
                                  </span>
                                  <span class="cit-name-surname">
                                   Mathis
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Naoshige
                                  </span>
                                  <span class="cit-name-surname">
                                   Uchida
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Somatosensory cortex plays an essential role in forelimb motor adaptation in mice
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  93
                                 </span>
                                 :p
                                 <span class="cit-fpage">
                                  1493
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1503
                                 </span>
                                 .e6,
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D93%26rft.spage%253D1493%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2017.02.049%26rft_id%253Dinfo%253Apmid%252F28334611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuron.2017.02.049&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28334611&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.06.081372v3.16">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Neeraj
                                  </span>
                                  <span class="cit-name-surname">
                                   Kumar
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Timothy F
                                  </span>
                                  <span class="cit-name-surname">
                                   Manning
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   David J
                                  </span>
                                  <span class="cit-name-surname">
                                   Ostry
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Somatosensory cortex participates in the consolidation of human motor memory
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ),
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.17" data-doi="10.1152/physrev.00048.2011">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Uwe
                                  </span>
                                  <span class="cit-name-surname">
                                   Proske
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Simon C
                                  </span>
                                  <span class="cit-name-surname">
                                   Gandevia
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The proprioceptive senses: their roles in signaling body shape, body position and movement, and muscle force
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Physiological reviews
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  92
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1651
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1697
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhysiological%2BReviews%26rft.stitle%253DPhysiol.%2BRev.%26rft.aulast%253DProske%26rft.auinit1%253DU.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D1651%26rft.epage%253D1697%26rft.atitle%253DThe%2BProprioceptive%2BSenses%253A%2BTheir%2BRoles%2Bin%2BSignaling%2BBody%2BShape%252C%2BBody%2BPosition%2Band%2BMovement%252C%2Band%2BMuscle%2BForce%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysrev.00048.2011%26rft_id%253Dinfo%253Apmid%252F23073629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1152/physrev.00048.2011&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23073629&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.18" data-doi="10.1016/j.tics.2015.10.008">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael SA
                                  </span>
                                  <span class="cit-name-surname">
                                   Graziano
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Ethological action maps: a paradigm shift for the motor cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Trends in cognitive sciences
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  121
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  132
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bcognitive%2Bsciences%26rft.volume%253D20%26rft.spage%253D121%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tics.2015.10.008%26rft_id%253Dinfo%253Apmid%252F26628112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.tics.2015.10.008&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26628112&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.19">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Olga
                                  </span>
                                  <span class="cit-name-surname">
                                   Russakovsky
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jia
                                  </span>
                                  <span class="cit-name-surname">
                                   Deng
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Hao
                                  </span>
                                  <span class="cit-name-surname">
                                   Su
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jonathan
                                  </span>
                                  <span class="cit-name-surname">
                                   Krause
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sanjeev
                                  </span>
                                  <span class="cit-name-surname">
                                   Satheesh
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sean
                                  </span>
                                  <span class="cit-name-surname">
                                   Ma
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Zhiheng
                                  </span>
                                  <span class="cit-name-surname">
                                   Huang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Andrej
                                  </span>
                                  <span class="cit-name-surname">
                                   Karpathy
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Aditya
                                  </span>
                                  <span class="cit-name-surname">
                                   Khosla
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael
                                  </span>
                                  <span class="cit-name-surname">
                                   Bernstein
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Imagenet large scale visual recognition challenge
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  International Journal of Computer Vision
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  115
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  211
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  252
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BComputer%2BVision%26rft.volume%253D115%26rft.spage%253D211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.20">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Seyed-Mahdi
                                  </span>
                                  <span class="cit-name-surname">
                                   Khaligh-Razavi
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nikolaus
                                  </span>
                                  <span class="cit-name-surname">
                                   Kriegeskorte
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Deep supervised, but not unsupervised, models may explain it cortical representation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS computational biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e1003915
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bcomputational%2Bbiology%26rft.volume%253D10%26rft.spage%253D1003915e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.06.081372v3.21" data-doi="10.1073/pnas.1403112111">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Daniel LK
                                  </span>
                                  <span class="cit-name-surname">
                                   Yamins
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ha
                                  </span>
                                  <span class="cit-name-surname">
                                   Hong
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Charles F
                                  </span>
                                  <span class="cit-name-surname">
                                   Cadieu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ethan A
                                  </span>
                                  <span class="cit-name-surname">
                                   Solomon
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Darren
                                  </span>
                                  <span class="cit-name-surname">
                                   Seibert
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   James J
                                  </span>
                                  <span class="cit-name-surname">
                                   DiCarlo
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Performance-optimized hierarchical models predict neural responses in higher visual cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  111
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  23
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  8619
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  8624
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1403112111%26rft_id%253Dinfo%253Apmid%252F24812127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzIzLzg2MTkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNi8xNS8yMDIwLjA1LjA2LjA4MTM3Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.06.081372v3.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Radoslaw Martin
                                  </span>
                                  <span class="cit-name-surname">
                                   Cichy
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Aditya
                                  </span>
                                  <span class="cit-name-surname">
                                   Khosla
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Dimitrios
                                  </span>
                                  <span class="cit-name-surname">
                                   Pantazis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Antonio
                                  </span>
                                  <span class="cit-name-surname">
                                   Torralba
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Aude
                                  </span>
                                  <span class="cit-name-surname">
                                   Oliva
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Comparison of deep neural networks to spatio-temporal cortical dynamics of human visual object recognition reveals hierarchical correspondence
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Scientific reports
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  27755
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScientific%2Breports%26rft.volume%253D6%26rft.spage%253D27755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.23" data-doi="10.1038/nn.4244">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Daniel LK
                                  </span>
                                  <span class="cit-name-surname">
                                   Yamins
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   James J
                                  </span>
                                  <span class="cit-name-surname">
                                   DiCarlo
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Using goal-driven deep learning models to understand sensory cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  356
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.volume%253D19%26rft.spage%253D356%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn.4244%26rft_id%253Dinfo%253Apmid%252F26906502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nn.4244&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26906502&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.06.081372v3.24">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Martin
                                  </span>
                                  <span class="cit-name-surname">
                                   Schrimpf
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jonas
                                  </span>
                                  <span class="cit-name-surname">
                                   Kubilius
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ha
                                  </span>
                                  <span class="cit-name-surname">
                                   Hong
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Najib J
                                  </span>
                                  <span class="cit-name-surname">
                                   Majaj
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Rishi
                                  </span>
                                  <span class="cit-name-surname">
                                   Rajalingham
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Elias B
                                  </span>
                                  <span class="cit-name-surname">
                                   Issa
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kohitij
                                  </span>
                                  <span class="cit-name-surname">
                                   Kar
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Pouya
                                  </span>
                                  <span class="cit-name-surname">
                                   Bashivan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jonathan
                                  </span>
                                  <span class="cit-name-surname">
                                   Prescott-Roy
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kailyn
                                  </span>
                                  <span class="cit-name-surname">
                                   Schmidt
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Brain-score: Which artificial neural network for object recognition is most brain-like?
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  BioRxiv
                                 </abbr>
                                 , page
                                 <span class="cit-fpage">
                                  407007
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               25.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.06.081372v3.25">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Santiago A
                                  </span>
                                  <span class="cit-name-surname">
                                   Cadena
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   George H
                                  </span>
                                  <span class="cit-name-surname">
                                   Denfield
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Edgar Y
                                  </span>
                                  <span class="cit-name-surname">
                                   Walker
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Leon A
                                  </span>
                                  <span class="cit-name-surname">
                                   Gatys
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Andreas S
                                  </span>
                                  <span class="cit-name-surname">
                                   Tolias
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Matthias
                                  </span>
                                  <span class="cit-name-surname">
                                   Bethge
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alexander S
                                  </span>
                                  <span class="cit-name-surname">
                                   Ecker
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Deep convolutional models improve predictions of macaque v1 responses to natural images
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS computational biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e1006897
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bcomputational%2Bbiology%26rft.volume%253D15%26rft.spage%253De1006897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               26.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.26" data-doi="10.1162/jocn\_a\_01755">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Katherine R
                                  </span>
                                  <span class="cit-name-surname">
                                   Storrs
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tim C
                                  </span>
                                  <span class="cit-name-surname">
                                   Kietzmann
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alexander
                                  </span>
                                  <span class="cit-name-surname">
                                   Walther
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Johannes
                                  </span>
                                  <span class="cit-name-surname">
                                   Mehrer
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nikolaus
                                  </span>
                                  <span class="cit-name-surname">
                                   Kriegeskorte
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Diverse deep neural networks all predict human inferior temporal cortex well, after training and fitting
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Cognitive Neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  33
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2044
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2064
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BCognitive%2BNeuroscience%26rft.volume%253D33%26rft.spage%253D2044%26rft_id%253Dinfo%253Adoi%252F10.1162%252Fjocn%255C_a%255C_01755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1162/jocn\_a\_01755&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               27.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.27" data-doi="10.1038/s41593-019-0520-2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Blake A
                                  </span>
                                  <span class="cit-name-surname">
                                   Richards
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Timothy P
                                  </span>
                                  <span class="cit-name-surname">
                                   Lillicrap
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Philippe
                                  </span>
                                  <span class="cit-name-surname">
                                   Beaudoin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yoshua
                                  </span>
                                  <span class="cit-name-surname">
                                   Bengio
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Rafal
                                  </span>
                                  <span class="cit-name-surname">
                                   Bogacz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Amelia
                                  </span>
                                  <span class="cit-name-surname">
                                   Christensen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Claudia
                                  </span>
                                  <span class="cit-name-surname">
                                   Clopath
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Rui Ponte
                                  </span>
                                  <span class="cit-name-surname">
                                   Costa
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Archy
                                  </span>
                                  <span class="cit-name-surname">
                                   de Berker
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Surya
                                  </span>
                                  <span class="cit-name-surname">
                                   Ganguli
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  A deep learning framework for neuroscience
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1761
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1770
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BNeuroscience%26rft.volume%253D22%26rft.spage%253D1761%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41593-019-0520-2%26rft_id%253Dinfo%253Apmid%252F31659335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41593-019-0520-2&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=31659335&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               28.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.28">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Andrew
                                  </span>
                                  <span class="cit-name-surname">
                                   Saxe
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Stephanie
                                  </span>
                                  <span class="cit-name-surname">
                                   Nelli
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christopher
                                  </span>
                                  <span class="cit-name-surname">
                                   Summerfield
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  If deep learning is the answer, then what is the question?
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  arXiv preprint
                                 </abbr>
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  <span class="cit-pub-id-scheme-arxiv">
                                   arxiv:
                                  </span>
                                  2004.07580
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               29.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Chengxu
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhuang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jonas
                                  </span>
                                  <span class="cit-name-surname">
                                   Kubilius
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mitra JZ
                                  </span>
                                  <span class="cit-name-surname">
                                   Hartmann
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Daniel L
                                  </span>
                                  <span class="cit-name-surname">
                                   Yamins
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Toward goal-driven neural network models for the rodent whisker-trigeminal system
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  In Advances in Neural Information Processing Systems
                                 </abbr>
                                 , pages
                                 <span class="cit-fpage">
                                  2555
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2565
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               30.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.30" data-doi="10.1038/s41586-019-1234-z">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Subramanian
                                  </span>
                                  <span class="cit-name-surname">
                                   Sundaram
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Petr
                                  </span>
                                  <span class="cit-name-surname">
                                   Kellnhofer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yunzhu
                                  </span>
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jun-Yan
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Antonio
                                  </span>
                                  <span class="cit-name-surname">
                                   Torralba
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Wojciech
                                  </span>
                                  <span class="cit-name-surname">
                                   Matusik
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Learning the signatures of the human grasp using a scalable tactile glove
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  569
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7758
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  698
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  702
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D569%26rft.spage%253D698%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-019-1234-z%26rft_id%253Dinfo%253Apmid%252F31142856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-019-1234-z&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=31142856&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               31.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.31">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Martin
                                  </span>
                                  <span class="cit-name-surname">
                                   Haesemeyer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alexander F
                                  </span>
                                  <span class="cit-name-surname">
                                   Schier
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Florian
                                  </span>
                                  <span class="cit-name-surname">
                                   Engert
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Convergent temperature representations in artificial and biological neural networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  103
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1123
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1134
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D103%26rft.spage%253D1123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               32.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.32">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alexander JE
                                  </span>
                                  <span class="cit-name-surname">
                                   Kell
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Daniel LK
                                  </span>
                                  <span class="cit-name-surname">
                                   Yamins
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Erica N
                                  </span>
                                  <span class="cit-name-surname">
                                   Shook
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sam V
                                  </span>
                                  <span class="cit-name-surname">
                                   Norman-Haignere
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Josh H
                                  </span>
                                  <span class="cit-name-surname">
                                   McDermott
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A task-optimized neural network replicates human auditory behavior, predicts brain responses, and reveals a cortical processing hierarchy
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  98
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  630
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  644
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D98%26rft.spage%253D630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               33.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.05.06.081372v3.33">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ben H
                                  </span>
                                  <span class="cit-name-surname">
                                   Williams
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Marc
                                  </span>
                                  <span class="cit-name-surname">
                                   Toussaint
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Amos J
                                  </span>
                                  <span class="cit-name-surname">
                                   Storkey
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-chapter-title">
                                  Extracting motion primitives from natural handwriting data
                                 </span>
                                 .
                                 <span class="cit-source">
                                  In International Conference on Artificial Neural Networks
                                 </span>
                                 , pages
                                 <span class="cit-fpage">
                                  634
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  643
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               34.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.34" data-doi="doi:10.1080/10255842.2014.916698">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Katherine R
                                  </span>
                                  <span class="cit-name-surname">
                                   Saul
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Xiao
                                  </span>
                                  <span class="cit-name-surname">
                                   Hu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Craig M
                                  </span>
                                  <span class="cit-name-surname">
                                   Goehler
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Meghan E
                                  </span>
                                  <span class="cit-name-surname">
                                   Vidt
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Melissa
                                  </span>
                                  <span class="cit-name-surname">
                                   Daly
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Anca
                                  </span>
                                  <span class="cit-name-surname">
                                   Velisar
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Wendy M
                                  </span>
                                  <span class="cit-name-surname">
                                   Murray
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Benchmarking of dynamic simulation predictions in two software platforms using an upper limb musculoskeletal model
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Computer methods in biomechanics and biomedical engineering
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  13
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1445
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1458
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DComputer%2Bmethods%2Bin%2Bbiomechanics%2Band%2Bbiomedical%2Bengineering%26rft.volume%253D18%26rft.spage%253D1445%26rft_id%253Dinfo%253Adoi%252Fdoi%253A10.1080%252F10255842.2014.916698%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=doi:10.1080/10255842.2014.916698&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               35.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.35" data-doi="10.1523/JNEUROSCI.3357-08.2008">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael
                                  </span>
                                  <span class="cit-name-surname">
                                   Dimitriou
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Benoni B
                                  </span>
                                  <span class="cit-name-surname">
                                   Edin
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Discharges in human muscle receptor afferents during block grasping
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  48
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  12632
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  12642
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DDimitriou%26rft.auinit1%253DM.%26rft.volume%253D28%26rft.issue%253D48%26rft.spage%253D12632%26rft.epage%253D12642%26rft.atitle%253DDischarges%2Bin%2BHuman%2BMuscle%2BReceptor%2BAfferents%2Bduring%2BBlock%2BGrasping%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.3357-08.2008%26rft_id%253Dinfo%253Apmid%252F19036957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjExOiIyOC80OC8xMjYzMiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA2LzE1LzIwMjAuMDUuMDYuMDgxMzcyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               36.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.36" data-doi="10.1113/jphysiol.2008.160036">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael
                                  </span>
                                  <span class="cit-name-surname">
                                   Dimitriou
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Benoni B
                                  </span>
                                  <span class="cit-name-surname">
                                   Edin
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Discharges in human muscle spindle afferents during a key-pressing task
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Journal of physiology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  586
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  22
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  5455
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  5470
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bphysiology%26rft_id%253Dinfo%253Adoi%252F10.1113%252Fjphysiol.2008.160036%26rft_id%253Dinfo%253Apmid%252F18801840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1113/jphysiol.2008.160036&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18801840&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               37.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.37">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Dua
                                  </span>
                                  <span class="cit-name-surname">
                                   Dheeru
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Efi Karra
                                  </span>
                                  <span class="cit-name-surname">
                                   Taniskidou
                                  </span>
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  UCI machine learning repository
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               38.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.06.081372v3.38">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Laurens
                                  </span>
                                  <span class="cit-name-surname">
                                   van der Maaten
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Geoffrey
                                  </span>
                                  <span class="cit-name-surname">
                                   Hinton
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Visualizing data using t-sne
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of machine learning research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 (
                                 <span class="cit-month">
                                  Nov
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2579
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2605
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmachine%2Blearning%2Bresearch%26rft.volume%253D9%26rft.spage%253D2579%26rft_id%253Dinfo%253Apmid%252FWOS%253A00026263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=WOS:00026263&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               39.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.39" data-doi="10.1109/72.991427">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Chih-Wei
                                  </span>
                                  <span class="cit-name-surname">
                                   Hsu
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Chih-Jen
                                  </span>
                                  <span class="cit-name-surname">
                                   Lin
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A comparison of methods for multiclass support vector machines
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  IEEE transactions on Neural Networks
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  415
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  425
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIEEE%2Btransactions%2Bon%2Bneural%2Bnetworks%2B%252F%2Ba%2Bpublication%2Bof%2Bthe%2BIEEE%2BNeural%2BNetworks%2BCouncil%26rft.stitle%253DIEEE%2BTrans%2BNeural%2BNetw%26rft.aulast%253DHsu%26rft.auinit1%253DC.%2BW.%26rft.volume%253D13%26rft.issue%253D2%26rft.spage%253D415%26rft.epage%253D425%26rft.atitle%253DA%2Bcomparison%2Bof%2Bmethods%2Bfor%2Bmulticlass%2Bsupport%2Bvector%2Bmachines.%26rft_id%253Dinfo%253Adoi%252F10.1109%252F72.991427%26rft_id%253Dinfo%253Apmid%252F18244442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1109/72.991427&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18244442&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000174519400014&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               40.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.40" data-doi="10.1016/j.conb.2021.04.004">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sébastien B
                                  </span>
                                  <span class="cit-name-surname">
                                   Hausmann
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alessandro Marin
                                  </span>
                                  <span class="cit-name-surname">
                                   Vargas
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alexander
                                  </span>
                                  <span class="cit-name-surname">
                                   Mathis
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mackenzie W
                                  </span>
                                  <span class="cit-name-surname">
                                   Mathis
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Measuring and modeling the motor system with machine learning
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Current Opinion in Neurobiology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  70
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  11
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  23
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BNeurobiology%26rft.volume%253D70%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.conb.2021.04.004%26rft_id%253Dinfo%253Apmid%252F34116423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.conb.2021.04.004&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=34116423&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               41.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.41">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   David E
                                  </span>
                                  <span class="cit-name-surname">
                                   Rumelhart
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Geoffrey E
                                  </span>
                                  <span class="cit-name-surname">
                                   Hinton
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   James L
                                  </span>
                                  <span class="cit-name-surname">
                                   McClelland
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  A general framework for parallel distributed processing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Parallel distributed processing: Explorations in the microstructure of cognition
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  1
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  45-76
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  26
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1986
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DParallel%2Bdistributed%2Bprocessing%253A%2BExplorations%2Bin%2Bthe%2Bmicrostructure%2Bof%2Bcognition%26rft.volume%253D1%26rft.spage%253D26%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               42.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.42" data-doi="10.1109/5.726791">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yann
                                  </span>
                                  <span class="cit-name-surname">
                                   LeCun
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Léon
                                  </span>
                                  <span class="cit-name-surname">
                                   Bottou
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yoshua
                                  </span>
                                  <span class="cit-name-surname">
                                   Bengio
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Patrick
                                  </span>
                                  <span class="cit-name-surname">
                                   Haffner
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Gradient-based learning applied to document recognition
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the IEEE
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  86
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2278
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2324
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1998
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BIEEE%26rft.volume%253D86%26rft.spage%253D2278%26rft_id%253Dinfo%253Adoi%252F10.1109%252F5.726791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1109/5.726791&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000076557300010&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               43.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.06.081372v3.43" data-doi="10.1162/neco.1997.9.8.1735">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sepp
                                  </span>
                                  <span class="cit-name-surname">
                                   Hochreiter
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jürgen
                                  </span>
                                  <span class="cit-name-surname">
                                   Schmidhuber
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Long short-term memory
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neural computation
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1735
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1780
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1997
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeural%2BComputation%26rft.stitle%253DNeural%2BComput.%26rft.aulast%253DHochreiter%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D1735%26rft.epage%253D1780%26rft.atitle%253DLong%2BShort-Term%2BMemory%26rft_id%253Dinfo%253Adoi%252F10.1162%252Fneco.1997.9.8.1735%26rft_id%253Dinfo%253Apmid%252F9377276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1162/neco.1997.9.8.1735&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=9377276&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1997YA04500007&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               44.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.44">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Shaojie
                                  </span>
                                  <span class="cit-name-surname">
                                   Bai
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                  <span class="cit-name-surname">
                                   Zico Kolter
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Vladlen
                                  </span>
                                  <span class="cit-name-surname">
                                   Koltun
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  An empirical evaluation of generic convolutional and recurrent networks for sequence modeling
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  arXiv preprint
                                 </abbr>
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  <span class="cit-pub-id-scheme-arxiv">
                                   arxiv:
                                  </span>
                                  1803.01271
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               45.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.45">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Diederik P
                                  </span>
                                  <span class="cit-name-surname">
                                   Kingma
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jimmy
                                  </span>
                                  <span class="cit-name-surname">
                                   Ba
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Adam: A method for stochastic optimization
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  arXiv preprint
                                 </abbr>
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  <span class="cit-pub-id-scheme-arxiv">
                                   arxiv:
                                  </span>
                                  1412.6980
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               46.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.46" data-doi="10.1007/s00221-003-1384-x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Edith
                                  </span>
                                  <span class="cit-name-surname">
                                   Ribot-Ciscar
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mikael
                                  </span>
                                  <span class="cit-name-surname">
                                   Bergenheim
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Frédéric
                                  </span>
                                  <span class="cit-name-surname">
                                   Albert
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jean-Pierre
                                  </span>
                                  <span class="cit-name-surname">
                                   Roll
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Proprioceptive population coding of limb position in humans
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Experimental brain research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  149
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  512
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  519
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DExperimental%2Bbrain%2Bresearch.%2B%2BExperimentelle%2BHirnforschung.%2B%2BExp%25C3%25A9rimentation%2Bc%25C3%25A9r%25C3%25A9brale%26rft.stitle%253DExp%2BBrain%2BRes%26rft.aulast%253DRibot-Ciscar%26rft.auinit1%253DE.%26rft.volume%253D149%26rft.issue%253D4%26rft.spage%253D512%26rft.epage%253D519%26rft.atitle%253DProprioceptive%2Bpopulation%2Bcoding%2Bof%2Blimb%2Bposition%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00221-003-1384-x%26rft_id%253Dinfo%253Apmid%252F12677332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s00221-003-1384-x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=12677332&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000182462300013&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               47.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.47">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kibleur
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tata
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Greiner
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Conti
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Barra
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhuang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kaeser
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ijspeert
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Capogrosso
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Spatiotemporal maps of proprioceptive inputs to the cervical spinal cord during three-di-mensional reaching and grasping
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  IEEE Transactions on Neural Systems and Rehabilitation Engineering
                                 </abbr>
                                 , pages
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               48.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.48">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Bernd
                                  </span>
                                  <span class="cit-name-surname">
                                   Illing
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Wulfram
                                  </span>
                                  <span class="cit-name-surname">
                                   Gerstner
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Johanni
                                  </span>
                                  <span class="cit-name-surname">
                                   Brea
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Biologically plausible deep learning—but how far can we go with shallow networks?
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Neural Networks
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  118
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  90
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  101
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeural%2BNetworks%26rft.volume%253D118%26rft.spage%253D90%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               49.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.49" data-doi="10.1007/s00221-005-2246-5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Frederic
                                  </span>
                                  <span class="cit-name-surname">
                                   Albert
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Edith
                                  </span>
                                  <span class="cit-name-surname">
                                   Ribot-Ciscar
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michel
                                  </span>
                                  <span class="cit-name-surname">
                                   Fiocchi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mikael
                                  </span>
                                  <span class="cit-name-surname">
                                   Bergenheim
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jean-Pierre
                                  </span>
                                  <span class="cit-name-surname">
                                   Roll
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Proprioceptive feedback in humans expresses motor invariants during writing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Experimental brain research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  164
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  242
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  249
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DExperimental%2Bbrain%2Bresearch.%2B%2BExperimentelle%2BHirnforschung.%2B%2BExp%25C3%25A9rimentation%2Bc%25C3%25A9r%25C3%25A9brale%26rft.stitle%253DExp%2BBrain%2BRes%26rft.aulast%253DAlbert%26rft.auinit1%253DF.%26rft.volume%253D164%26rft.issue%253D2%26rft.spage%253D242%26rft.epage%253D249%26rft.atitle%253DProprioceptive%2Bfeedback%2Bin%2Bhumans%2Bexpresses%2Bmotor%2Binvariants%2Bduring%2Bwriting.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00221-005-2246-5%26rft_id%253Dinfo%253Apmid%252F15856208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s00221-005-2246-5&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15856208&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000230625000011&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               50.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.50">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kai J
                                  </span>
                                  <span class="cit-name-surname">
                                   Sandbrink
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Pranav
                                  </span>
                                  <span class="cit-name-surname">
                                   Mamidanna
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Claudio
                                  </span>
                                  <span class="cit-name-surname">
                                   Michaelis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mackenzie Weygandt
                                  </span>
                                  <span class="cit-name-surname">
                                   Mathis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Matthias
                                  </span>
                                  <span class="cit-name-surname">
                                   Bethge
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alexander
                                  </span>
                                  <span class="cit-name-surname">
                                   Mathis
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Task-driven hierarchical deep neural network models of the proprioceptive pathway
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               51.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.51" data-doi="10.1111/j.1469-7793.1998.293bu.x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Arthur
                                  </span>
                                  <span class="cit-name-surname">
                                   Prochazka
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Monica
                                  </span>
                                  <span class="cit-name-surname">
                                   Gorassini
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Ensemble firing of muscle afferents recorded during normal locomotion in cats
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Journal of physiology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  507
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  293
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  304
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1998
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BPhysiology%26rft.stitle%253DJ.%2BPhysiol.%26rft.aulast%253DProchazka%26rft.auinit1%253DA.%26rft.volume%253D507%26rft.issue%253D1%26rft.spage%253D293%26rft.epage%253D304%26rft.atitle%253DEnsemble%2Bfiring%2Bof%2Bmuscle%2Bafferents%2Brecorded%2Bduring%2Bnormal%2Blocomotion%2Bin%2Bcats%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-7793.1998.293bu.x%26rft_id%253Dinfo%253Apmid%252F9490855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/j.1469-7793.1998.293bu.x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=9490855&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000072401700023&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               52.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.52">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christopher
                                  </span>
                                  <span class="cit-name-surname">
                                   Versteeg
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joshua M
                                  </span>
                                  <span class="cit-name-surname">
                                   Rosenow
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sliman J
                                  </span>
                                  <span class="cit-name-surname">
                                   Bensmaia
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Lee E
                                  </span>
                                  <span class="cit-name-surname">
                                   Miller
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Encoding of limb state by single neurons in the cuneate nucleus of awake monkeys
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neurophysiology
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               53.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.53">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Thomas
                                  </span>
                                  <span class="cit-name-surname">
                                   Serre
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Deep learning: the good, the bad, and the ugly
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Annual Review of Vision Science
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  399
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  426
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BVision%2BScience%26rft.volume%253D5%26rft.spage%253D399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               54.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.54" data-doi="10.1152/jn.00710.2006">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Andrew
                                  </span>
                                  <span class="cit-name-surname">
                                   Jackson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jaideep
                                  </span>
                                  <span class="cit-name-surname">
                                   Mavoori
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Eberhard E
                                  </span>
                                  <span class="cit-name-surname">
                                   Fetz
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Correlations between the same motor cortex cells and arm muscles during a trained task, free behavior, and natural sleep in the macaque monkey
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of neurophysiology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  97
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  360
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  374
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bneurophysiology%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fjn.00710.2006%26rft_id%253Dinfo%253Apmid%252F17021028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1152/jn.00710.2006&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17021028&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000243532900034&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               55.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.55">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Simon C
                                  </span>
                                  <span class="cit-name-surname">
                                   Gandevia
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kathyrn M
                                  </span>
                                  <span class="cit-name-surname">
                                   Refshauge
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   David F
                                  </span>
                                  <span class="cit-name-surname">
                                   Collins
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Proprioception: peripheral inputs and perceptual interactions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  In Sen-sorimotor control of movement and posture
                                 </abbr>
                                 , pages
                                 <span class="cit-fpage">
                                  61
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  68
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               56.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.06.081372v3.56" data-doi="10.1152/jn.00868.2005">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Milana P
                                  </span>
                                  <span class="cit-name-surname">
                                   Mileusnic
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ian E
                                  </span>
                                  <span class="cit-name-surname">
                                   Brown
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ning
                                  </span>
                                  <span class="cit-name-surname">
                                   Lan
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Gerald E
                                  </span>
                                  <span class="cit-name-surname">
                                   Loeb
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Mathematical models of proprioceptors. i. control and transduction in the muscle spindle
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of neurophysiology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  96
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1772
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1788
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bneurophysiology%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fjn.00868.2005%26rft_id%253Dinfo%253Apmid%252F16672301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1152/jn.00868.2005&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16672301&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               57.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.06.081372v3.57" data-doi="10.1113/jphysiol.2006.123075">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jean-Marc
                                  </span>
                                  <span class="cit-name-surname">
                                   Aimonetti
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Valérie
                                  </span>
                                  <span class="cit-name-surname">
                                   Hospod
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jean-Pierre
                                  </span>
                                  <span class="cit-name-surname">
                                   Roll
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Edith
                                  </span>
                                  <span class="cit-name-surname">
                                   Ribot-Ciscar
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Cutaneous afferents provide a neuronal population vector that encodes the orientation of human ankle movements
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Journal of physiology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  580
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  649
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  658
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bphysiology%26rft_id%253Dinfo%253Adoi%252F10.1113%252Fjphysiol.2006.123075%26rft_id%253Dinfo%253Apmid%252F17255169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1113/jphysiol.2006.123075&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17255169&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000245646900027&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               58.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.58">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kyle P
                                  </span>
                                  <span class="cit-name-surname">
                                   Blum
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Boris
                                  </span>
                                  <span class="cit-name-surname">
                                   Lamotte D’Incamps
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Daniel
                                  </span>
                                  <span class="cit-name-surname">
                                   Zytnicki
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Lena H
                                  </span>
                                  <span class="cit-name-surname">
                                   Ting
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Force encoding in muscle spindles during stretch of passive muscle
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS computational biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e1005767
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bcomputational%2Bbiology%26rft.volume%253D13%26rft.spage%253De1005767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               59.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.59">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Rochelle
                                  </span>
                                  <span class="cit-name-surname">
                                   Ackerley
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Marie
                                  </span>
                                  <span class="cit-name-surname">
                                   Chancel
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jean-Marc
                                  </span>
                                  <span class="cit-name-surname">
                                   Aimonetti
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Edith
                                  </span>
                                  <span class="cit-name-surname">
                                   Ribot-Ciscar
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Anne
                                  </span>
                                  <span class="cit-name-surname">
                                   Kavounoudias
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Seeing your foot move changes muscle proprioceptive feedback
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Eneuro
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ),
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               60.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.60" data-doi="10.1007/s10439-005-3320-7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Katherine RS
                                  </span>
                                  <span class="cit-name-surname">
                                   Holzbaur
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Wendy M
                                  </span>
                                  <span class="cit-name-surname">
                                   Murray
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Scott L
                                  </span>
                                  <span class="cit-name-surname">
                                   Delp
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A model of the upper extremity for simulating musculoskeletal surgery and analyzing neuromuscular control
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Annals of biomedical engineering
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  33
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  829
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  840
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bbiomedical%2Bengineering%26rft.stitle%253DAnn%2BBiomed%2BEng%26rft.aulast%253DHolzbaur%26rft.auinit1%253DK.%2BR.%26rft.volume%253D33%26rft.issue%253D6%26rft.spage%253D829%26rft.epage%253D840%26rft.atitle%253DA%2Bmodel%2Bof%2Bthe%2Bupper%2Bextremity%2Bfor%2Bsimulating%2Bmusculoskeletal%2Bsurgery%2Band%2Banalyzing%2Bneuromuscular%2Bcontrol.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10439-005-3320-7%26rft_id%253Dinfo%253Apmid%252F16078622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s10439-005-3320-7&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16078622&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000230075100011&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               61.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.05.06.081372v3.61">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Aaron
                                  </span>
                                  <span class="cit-name-surname">
                                   D’Souza
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sethu
                                  </span>
                                  <span class="cit-name-surname">
                                   Vijayakumar
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Stefan
                                  </span>
                                  <span class="cit-name-surname">
                                   Schaal
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-chapter-title">
                                  Learning inverse kinematics
                                 </span>
                                 .
                                 <span class="cit-source">
                                  In Proceedings 2001 IEEE/RSJ International Conference on Intelligent Robots and Systems. Expanding the Societal Role of Robotics in the the Next Millennium (Cat. No. 01CH37180)
                                 </span>
                                 , volume
                                 <span class="cit-vol">
                                  1
                                 </span>
                                 , pages
                                 <span class="cit-fpage">
                                  298
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  303
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  IEEE
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%2BProceedings%2B2001%2BIEEE%252FRSJ%2BInternational%2BConference%2Bon%2BIntelligent%2BRobots%2Band%2BSystems.%2BExpanding%2Bthe%2BSocietal%2BRole%2Bof%2BRobotics%2Bin%2Bthe%2Bthe%2BNext%2BMillennium%2B%2528Cat.%2BNo.%2B01CH37180%2529%26rft.volume%253D1%26rft.spage%253D298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               62.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.62" data-doi="10.1109/TBME.2007.901024">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Scott L
                                  </span>
                                  <span class="cit-name-surname">
                                   Delp
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Frank C
                                  </span>
                                  <span class="cit-name-surname">
                                   Anderson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Allison S
                                  </span>
                                  <span class="cit-name-surname">
                                   Arnold
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Peter
                                  </span>
                                  <span class="cit-name-surname">
                                   Loan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ayman
                                  </span>
                                  <span class="cit-name-surname">
                                   Habib
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Chand T
                                  </span>
                                  <span class="cit-name-surname">
                                   John
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Eran
                                  </span>
                                  <span class="cit-name-surname">
                                   Guendelman
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Darryl G
                                  </span>
                                  <span class="cit-name-surname">
                                   Thelen
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Opensim: open-source software to create and analyze dynamic simulations of movement
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  IEEE transactions on biomedical engineering
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  54
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1940
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1950
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIEEE%2Btransactions%2Bon%2Bbio-medical%2Bengineering%26rft.stitle%253DIEEE%2BTrans%2BBiomed%2BEng%26rft.aulast%253DDelp%26rft.auinit1%253DS.%2BL.%26rft.volume%253D54%26rft.issue%253D11%26rft.spage%253D1940%26rft.epage%253D1950%26rft.atitle%253DOpenSim%253A%2Bopen-source%2Bsoftware%2Bto%2Bcreate%2Band%2Banalyze%2Bdynamic%2Bsimulations%2Bof%2Bmovement.%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTBME.2007.901024%26rft_id%253Dinfo%253Apmid%252F18018689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1109/TBME.2007.901024&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18018689&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000250449200004&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               63.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.63" data-doi="10.1016/j.piutam.2011.04.021">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ajay
                                  </span>
                                  <span class="cit-name-surname">
                                   Seth
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael
                                  </span>
                                  <span class="cit-name-surname">
                                   Sherman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jeffrey A
                                  </span>
                                  <span class="cit-name-surname">
                                   Reinbolt
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Scott L
                                  </span>
                                  <span class="cit-name-surname">
                                   Delp
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Opensim: a musculoskeletal modeling and simulation framework for in silico investigations and exchange
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Procedia Iutam
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  212
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  232
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProcedia%2BIutam%26rft.volume%253D2%26rft.spage%253D212%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.piutam.2011.04.021%26rft_id%253Dinfo%253Apmid%252F25893160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.piutam.2011.04.021&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25893160&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000314462800020&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               64.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.64">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Vaughan G
                                  </span>
                                  <span class="cit-name-surname">
                                   Macefield
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Thomas P
                                  </span>
                                  <span class="cit-name-surname">
                                   Knellwolf
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Functional properties of human muscle spindles
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of neurophysiology
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               65.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.65" data-doi="10.1111/j.1469-7793.1998.277bu.x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Arthur
                                  </span>
                                  <span class="cit-name-surname">
                                   Prochazka
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Monica
                                  </span>
                                  <span class="cit-name-surname">
                                   Gorassini
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Models of ensemble firing of muscle spindle afferents recorded during normal locomotion in cats
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Journal of physiology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  507
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  277
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  291
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1998
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BPhysiology%26rft.stitle%253DJ.%2BPhysiol.%26rft.aulast%253DProchazka%26rft.auinit1%253DA.%26rft.volume%253D507%26rft.issue%253D1%26rft.spage%253D277%26rft.epage%253D291%26rft.atitle%253DModels%2Bof%2Bensemble%2Bfiring%2Bof%2Bmuscle%2Bspindle%2Bafferents%2Brecorded%2Bduring%2Bnormal%2Blocomotion%2Bin%2Bcats%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-7793.1998.277bu.x%26rft_id%253Dinfo%253Apmid%252F9490851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/j.1469-7793.1998.277bu.x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=9490851&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F15%2F2020.05.06.081372.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000072401700022&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               66.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.66">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Fabian
                                  </span>
                                  <span class="cit-name-surname">
                                   Pedregosa
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Gaël
                                  </span>
                                  <span class="cit-name-surname">
                                   Varoquaux
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alexandre
                                  </span>
                                  <span class="cit-name-surname">
                                   Gramfort
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Vincent
                                  </span>
                                  <span class="cit-name-surname">
                                   Michel
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Bertrand
                                  </span>
                                  <span class="cit-name-surname">
                                   Thirion
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Olivier
                                  </span>
                                  <span class="cit-name-surname">
                                   Grisel
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mathieu
                                  </span>
                                  <span class="cit-name-surname">
                                   Blondel
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Peter
                                  </span>
                                  <span class="cit-name-surname">
                                   Prettenhofer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ron
                                  </span>
                                  <span class="cit-name-surname">
                                   Weiss
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Vincent
                                  </span>
                                  <span class="cit-name-surname">
                                   Dubourg
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Scikit-learn: Machine learning in python
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  the Journal of machine Learning research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  2825
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2830
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253Dthe%2BJournal%2Bof%2Bmachine%2BLearning%2Bresearch%26rft.volume%253D12%26rft.spage%253D2825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000298103200003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               67.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.67">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jimmy Lei
                                  </span>
                                  <span class="cit-name-surname">
                                   Ba
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jamie Ryan
                                  </span>
                                  <span class="cit-name-surname">
                                   Kiros
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Geoffrey E
                                  </span>
                                  <span class="cit-name-surname">
                                   Hinton
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Layer normalization
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  arXiv preprint
                                 </abbr>
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  <span class="cit-pub-id-scheme-arxiv">
                                   arxiv:
                                  </span>
                                  1607.06450
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               68.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.68">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Simon
                                  </span>
                                  <span class="cit-name-surname">
                                   Kornblith
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mohammad
                                  </span>
                                  <span class="cit-name-surname">
                                   Norouzi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Honglak
                                  </span>
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Geoffrey
                                  </span>
                                  <span class="cit-name-surname">
                                   Hinton
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Similarity of neural network representations revisited
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  arXiv preprint
                                 </abbr>
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  <span class="cit-pub-id-scheme-arxiv">
                                   arxiv:
                                  </span>
                                  1905.00414
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               69.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.69">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nikolaus
                                  </span>
                                  <span class="cit-name-surname">
                                   Kriegeskorte
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Marieke
                                  </span>
                                  <span class="cit-name-surname">
                                   Mur
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Peter A
                                  </span>
                                  <span class="cit-name-surname">
                                   Bandettini
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Repre-sentational similarity analysis-connecting the branches of systems neuroscience
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Frontiers in systems neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  4
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2Bsystems%2Bneuroscience%26rft.volume%253D2%26rft.spage%253D4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               70.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.70" data-doi="10.1523/JNEUROSCI.02-11-01527.1982">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Apostolos P
                                  </span>
                                  <span class="cit-name-surname">
                                   Georgopoulos
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John F
                                  </span>
                                  <span class="cit-name-surname">
                                   Kalaska
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Roberto
                                  </span>
                                  <span class="cit-name-surname">
                                   Caminiti
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joe T
                                  </span>
                                  <span class="cit-name-surname">
                                   Massey
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  On the relations between the direction of two-dimensional arm movements and cell discharge in primate motor cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Journal of neuroscience : the official journal of the Society for Neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 11:
                                 <span class="cit-fpage">
                                  1527
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  37
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1982
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DGeorgopoulos%26rft.auinit1%253DA.%26rft.volume%253D2%26rft.issue%253D11%26rft.spage%253D1527%26rft.epage%253D1537%26rft.atitle%253DOn%2Bthe%2Brelations%2Bbetween%2Bthe%2Bdirection%2Bof%2Btwo-dimensional%2Barm%2Bmovements%2Band%2Bcell%2Bdischarge%2Bin%2Bprimate%2Bmotor%2Bcortex%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.02-11-01527.1982%26rft_id%253Dinfo%253Apmid%252F7143039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjk6IjIvMTEvMTUyNyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA2LzE1LzIwMjAuMDUuMDYuMDgxMzcyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               71.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.71">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Pauli
                                  </span>
                                  <span class="cit-name-surname">
                                   Virtanen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ralf
                                  </span>
                                  <span class="cit-name-surname">
                                   Gommers
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Travis E
                                  </span>
                                  <span class="cit-name-surname">
                                   Oliphant
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Matt
                                  </span>
                                  <span class="cit-name-surname">
                                   Haberland
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tyler
                                  </span>
                                  <span class="cit-name-surname">
                                   Reddy
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   David
                                  </span>
                                  <span class="cit-name-surname">
                                   Cournapeau
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Evgeni
                                  </span>
                                  <span class="cit-name-surname">
                                   Burovski
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Pearu
                                  </span>
                                  <span class="cit-name-surname">
                                   Peterson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Warren
                                  </span>
                                  <span class="cit-name-surname">
                                   Weckesser
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jonathan
                                  </span>
                                  <span class="cit-name-surname">
                                   Bright
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Scipy 1.0: fundamental algorithms for scientific computing in python
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  261
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  272
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D17%26rft.spage%253D261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               72.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.081372v3.72">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Martín
                                  </span>
                                  <span class="cit-name-surname">
                                   Abadi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Paul
                                  </span>
                                  <span class="cit-name-surname">
                                   Barham
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jianmin
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Zhifeng
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Andy
                                  </span>
                                  <span class="cit-name-surname">
                                   Davis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jeffrey
                                  </span>
                                  <span class="cit-name-surname">
                                   Dean
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Matthieu
                                  </span>
                                  <span class="cit-name-surname">
                                   Devin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sanjay
                                  </span>
                                  <span class="cit-name-surname">
                                   Ghemawat
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Geoffrey
                                  </span>
                                  <span class="cit-name-surname">
                                   Irving
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael
                                  </span>
                                  <span class="cit-name-surname">
                                   Isard
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Tensorflow: A system for large-scale machine learning
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  In 12th {USENIX} Symposium on Operating Systems Design and Implementation ({OSDI} 16)
                                 </abbr>
                                 , pages
                                 <span class="cit-fpage">
                                  265
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  283
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.05.06.081372v3" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.12.17.473166v3" title="Antibiotic Resistance Genes and Taxa Analysis from Mat and Planktonic Microbiomes of Antarctic Perennial Ice-covered Lake Fryxell and Lake Bonney" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.05.28.493835v2" title="A Diabetic Mice Model For Studying Skin Wound Healing" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted June 15, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.05.06.081372v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.05.06.081372v3.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.05.06.081372v3.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.05.06.081372v3" class="active" data-icon-position="" data-hide-link-title="0">
                                   Contrasting action and posture coding with hierarchical deep neural network models of proprioception
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2596403"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-ufHqja0JYihkUPrKvTYtTddqUpoSlrpx9IzX0vQRVow"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484470390"/>
                                   <input type="hidden" name="captcha_token" value="08427e77b07ebbcdf08ed61a430f10dd"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2596403--3" data-node-nid="2596403" id="node-2596403--41280776270" data-pisa="biorxiv;2020.05.06.081372v3" data-pisa-master="biorxiv;2020.05.06.081372" data-seqnum="2596403" data-apath="/biorxiv/early/2022/06/15/2020.05.06.081372.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Contrasting action and posture coding with hierarchical deep neural network models of proprioception
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Kai J.
                                  </span>
                                  <span class="nlm-surname">
                                   Sandbrink
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Pranav
                                  </span>
                                  <span class="nlm-surname">
                                   Mamidanna
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Claudio
                                  </span>
                                  <span class="nlm-surname">
                                   Michaelis
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Matthias
                                  </span>
                                  <span class="nlm-surname">
                                   Bethge
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Mackenzie Weygandt
                                  </span>
                                  <span class="nlm-surname">
                                   Mathis
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Alexander
                                  </span>
                                  <span class="nlm-surname">
                                   Mathis
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.05.06.081372;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.05.06.081372
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.05.06.081372v3" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.081372v3&title=Contrasting%20action%20and%20posture%20coding%20with%20hierarchical%20deep%20neural%20network%20models%20of%20proprioception" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.081372v3&title=Contrasting%20action%20and%20posture%20coding%20with%20hierarchical%20deep%20neural%20network%20models%20of%20proprioception" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.081372v3&text=Contrasting%20action%20and%20posture%20coding%20with%20hierarchical%20deep%20neural%20network%20models%20of%20proprioception" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.081372v3&t=Contrasting%20action%20and%20posture%20coding%20with%20hierarchical%20deep%20neural%20network%20models%20of%20proprioception" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.081372v3&title=Contrasting%20action%20and%20posture%20coding%20with%20hierarchical%20deep%20neural%20network%20models%20of%20proprioception" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.081372v3&title=Contrasting%20action%20and%20posture%20coding%20with%20hierarchical%20deep%20neural%20network%20models%20of%20proprioception&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.081372v3&title=Contrasting%20action%20and%20posture%20coding%20with%20hierarchical%20deep%20neural%20network%20models%20of%20proprioception" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2596403--5" data-node-nid="2596403" id="citation-node-2596403--6877245814" data-pisa="biorxiv;2020.05.06.081372v3" data-pisa-master="biorxiv;2020.05.06.081372" data-seqnum="2596403" data-apath="/biorxiv/early/2022/06/15/2020.05.06.081372.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Contrasting action and posture coding with hierarchical deep neural network models of proprioception
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Kai J.
                                      </span>
                                      <span class="nlm-surname">
                                       Sandbrink
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Pranav
                                      </span>
                                      <span class="nlm-surname">
                                       Mamidanna
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Claudio
                                      </span>
                                      <span class="nlm-surname">
                                       Michaelis
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Matthias
                                      </span>
                                      <span class="nlm-surname">
                                       Bethge
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Mackenzie Weygandt
                                      </span>
                                      <span class="nlm-surname">
                                       Mathis
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Alexander
                                      </span>
                                      <span class="nlm-surname">
                                       Mathis
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.05.06.081372;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.05.06.081372
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Contrasting%20action%20and%20posture%20coding%20with%20hierarchical%20deep%20neural%20network%20models%20of%20proprioception&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Sandbrink%2C+Kai+J.&rft.au=Mamidanna%2C+Pranav&rft.au=Michaelis%2C+Claudio&rft.au=Bethge%2C+Matthias&rft.au=Mathis%2C+Mackenzie+Weygandt&rft.au=Mathis%2C+Alexander">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2596403/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2596403/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2596403/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2596403/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2596403/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2596403/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2596403/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2596403/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2596403/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2596403/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2596403/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2596403/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.081372v3&count=horizontal&via=&text=Contrasting%20action%20and%20posture%20coding%20with%20hierarchical%20deep%20neural%20network%20models%20of%20proprioception&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.081372v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.081372v3&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.05.06.081372v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/neuroscience" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Neuroscience
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15187)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
