<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.11.18.387837v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.11.18.387837v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.11.18.387837v2.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-2.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/01/09/2020.11.18.387837.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.11.18.387837v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Inferring parameters of cancer evolution from sequencing and clinical data"/>
  <meta name="DC.Identifier" content="10.1101/2020.11.18.387837"/>
  <meta name="DC.Date" content="2022-01-09"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and after a driver event occurred, and growth rates of both initiated cancer cells and subsequently appearing subclones. We demonstrate that in order to obtain accurate estimates of mutation rate and timing of events, observed mutation counts should be corrected to account for clonal mutations that occurred after the founding of the tumor, as well as sequencing coverage. We apply our methodology to reconstruct the individual evolutionary histories of chronic lymphocytic leukemia patients, finding that the parental leukemic clone typically appears within the first fifteen years of life.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Nathan Lee"/>
  <meta name="DC.Contributor" content="Ivana Bozic"/>
  <meta name="article:published_time" content="2022-01-09"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Inferring parameters of cancer evolution from sequencing and clinical data"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and after a driver event occurred, and growth rates of both initiated cancer cells and subsequently appearing subclones. We demonstrate that in order to obtain accurate estimates of mutation rate and timing of events, observed mutation counts should be corrected to account for clonal mutations that occurred after the founding of the tumor, as well as sequencing coverage. We apply our methodology to reconstruct the individual evolutionary histories of chronic lymphocytic leukemia patients, finding that the parental leukemic clone typically appears within the first fifteen years of life.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.11.18.387837v2"/>
  <meta name="citation_id" content="2020.11.18.387837v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.11.18.387837v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.11.18.387837v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.11.18.387837v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.11.18.387837"/>
  <meta name="citation_num_pages" content="34"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.11.18.387837"/>
  <meta name="citation_author" content="Nathan Lee"/>
  <meta name="citation_author_institution" content="Department of Applied Mathematics, University of Washington"/>
  <meta name="citation_author" content="Ivana Bozic"/>
  <meta name="citation_author_institution" content="Department of Applied Mathematics, University of Washington"/>
  <meta name="citation_author_institution" content="Herbold Computational Biology Program, Fred Hutchinson Cancer Research Center"/>
  <meta name="citation_author_email" content="ibozic@uw.edu"/>
  <meta name="citation_reference" content="Nowell, P. C. The Clonal Evolution of Tumor Cell Populations. Science 194, 23–28 (1976). URL http://www.jstor.org/stable/1742535."/>
  <meta name="citation_reference" content="Stratton, M. R., Campbell, P. J. & Futreal, P. A. The cancer genome. Nature 458, 719–724 (2009). URL https://www.nature.com/articles/nature07943."/>
  <meta name="citation_reference" content="Hanahan, D. & Weinberg, R. A. Hallmarks of Cancer: The Next Generation. Cell 144, 646–674 (2011). URL https://www.cell.com/fulltext/S0092-8674(11)00127-9."/>
  <meta name="citation_reference" content="Bailey, M. H. et al. Comprehensive Characterization of Cancer Driver Genes and Mutations. Cell 173, 371–385.e18 (2018). URL https://www.cell.com/cell/abstract/S0092-8674(18)30237-X."/>
  <meta name="citation_reference" content="Williams, M. J., Werner, B., Barnes, C. P., Graham, T. A. & Sottoriva, A. Identification of neutral tumor evolution across cancer types. Nature Genetics 48, 238–244 (2016). URL https://www.nature.com/articles/ng.3489."/>
  <meta name="citation_reference" content="Kimura, M. Evolutionary Rate at the Molecular Level. Nature 217, 624–626 (1968). URL https://www.nature.com/articles/217624a0."/>
  <meta name="citation_reference" content="Kimura, M. Genetic variability maintained in a finite population due to mutational production of neutral and nearly neutral isoalleles*. Genetics Research 11, 247–270 (1968). URL http://www.cambridge.org/core/journals/genetics-research/article/genetic-variability-maintained-in-a-finite-population-due-to-mutational-production-of-neutral-and-A74BD3A5D72ED2C52444FD99DFE483EF."/>
  <meta name="citation_reference" content="Smith, J. M. & Haigh, J. The hitch-hiking effect of a favourable gene. Genetics Research 23, 23–35 (1974). URL http://www.cambridge.org/core/journals/genetics-research/article/hitchhiking-effect-of-a-favourable-gene/918291A3B62BD50E1AE5C1F22165EF1B."/>
  <meta name="citation_reference" content="Turajlic, S., Sottoriva, A., Graham, T. & Swanton, C. Resolving genetic heterogeneity in cancer. Nature Reviews Genetics 20, 404–416 (2019). URL https://www.nature.com/articles/s41576-019-0114-6."/>
  <meta name="citation_reference" content="Vogelstein, B. et al. Cancer Genome Landscapes. Science 339, 1546–1558 (2013). URL https://science.sciencemag.org/content/339/6127/1546."/>
  <meta name="citation_reference" content="Lawrence, M. S. et al. Mutational heterogeneity in cancer and the search for new cancer-associated genes. Nature 499, 214–218 (2013). URL https://www.nature.com/articles/nature12213."/>
  <meta name="citation_reference" content="Merlo, L. M. F., Pepper, J. W., Reid, B. J. & Maley, C. C. Cancer as an evolutionary and ecological process. Nature Reviews Cancer 6, 924–935 (2006). URL https://www.nature.com/articles/nrc2013."/>
  <meta name="citation_reference" content="Pepper, J. W., Findlay, C. S., Kassen, R., Spencer, S. L. & Maley, C. C. SYNTHESIS: Cancer research meets evolutionary biology. Evolutionary Applications 2, 62–70 (2009). URL https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1752-4571.2008.00063.x."/>
  <meta name="citation_reference" content="Tsao, J.-L. et al. Genetic reconstruction of individual colorectal tumor histories. Proceedings of the National Academy of Sciences 97, 1236–1241 (2000). URL https://www.pnas.org/content/97/3/1236."/>
  <meta name="citation_reference" content="Jones, S. et al. Comparative lesion sequencing provides insights into tumor evolution. Proceedings of the National Academy of Sciences of the United States of America 105, 4283– 4288 (2008). URL https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2393770/."/>
  <meta name="citation_reference" content="Yachida, S. et al. Distant metastasis occurs late during the genetic evolution of pancreatic cancer. Nature 467, 1114–1117 (2010). URL https://www.nature.com/articles/nature09515."/>
  <meta name="citation_reference" content="Naxerova, K. et al. Hypermutable DNA chronicles the evolution of human colon cancer. Proceedings of the National Academy of Sciences 111, E1889–E1898 (2014). URL http://www.pnas.org/cgi/doi/10.1073/pnas.1400179111."/>
  <meta name="citation_reference" content="McGranahan, N. et al. Clonal status of actionable driver events and the timing of mutational processes in cancer evolution. Science Translational Medicine 7, 283ra54–283ra54 (2015). URL https://stm.sciencemag.org/content/7/283/283ra54."/>
  <meta name="citation_reference" content="Mitchell, T. J. et al. Timing the Landmark Events in the Evolution of Clear Cell Renal Cell Cancer: TRACERx Renal. Cell 173, 611–623.e17 (2018). URL http://www.sciencedirect.com/science/article/pii/S0092867418301648."/>
  <meta name="citation_reference" content="PCAWG Evolution & Heterogeneity Working Group et al. The evolutionary history of 2,658 cancers. Nature 578, 122–128 (2020). URL http://www.nature.com/articles/s41586-019-1907-7."/>
  <meta name="citation_reference" content="Sundermann, L. K., Wintersinger, J., Rätsch, G., Stoye, J. & Morris, Q. Reconstructing tumor evolutionary histories and clone trees in polynomial-time with SubMARine. PLOS Computational Biology 17, e1008400 (2021). URL https://dx.plos.org/10.1371/journal.pcbi.1008400."/>
  <meta name="citation_reference" content="PCAWG Evolution and Heterogeneity Working Group et al. Reconstructing evolutionary trajectories of mutation signature activities in cancer using TrackSig. Nature Communications 11, 731 (2020). URL http://www.nature.com/articles/s41467-020-14352-7."/>
  <meta name="citation_reference" content="Tomasetti, C. & Bozic, I. The (not so) immortal strand hypothesis. Stem Cell Research 14, 238–241 (2015)."/>
  <meta name="citation_reference" content="Werner, B. et al. Measuring single cell divisions in human tissues from multi-region sequencing data. Nature Communications 11, 1–9 (2020). URL https://www.nature.com/articles/s41467-020-14844-6. Number: 1 Publisher: Nature Publishing Group."/>
  <meta name="citation_reference" content="Bozic, I. et al. Accumulation of driver and passenger mutations during tumor progression. Proceedings of the National Academy of Sciences of the United States of America 107, 18545–18550 (2010). URL https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2972991/."/>
  <meta name="citation_reference" content="Sun, R. et al. Between-region genetic divergence reflects the mode and tempo of tumor evolution. Nature Genetics 49, 1015–1024 (2017). URL https://www.nature.com/articles/ng.3891."/>
  <meta name="citation_reference" content="Gruber, M. et al. Growth dynamics in naturally progressing chronic lymphocytic leukaemia. Nature 570, 474–479 (2019). URL https://www.nature.com/articles/s41586-019-1252-x."/>
  <meta name="citation_reference" content="Salichos, L., Meyerson, W., Warrell, J. & Gerstein, M. Estimating growth patterns and driver effects in tumor evolution from individual samples. Nature Communications 11, 1–14 (2020). URL https://www.nature.com/articles/s41467-020-14407-9."/>
  <meta name="citation_reference" content="Noble, R. et al. Spatial structure governs the mode of tumour evolution. Nature Ecology & Evolution (2021). URL https://www.nature.com/articles/s41559-021-01615-9."/>
  <meta name="citation_reference" content="Chkhaidze, K. et al. Spatially constrained tumour growth affects the patterns of clonal selection and neutral drift in cancer genomic data. PLOS Computational Biology 15, e1007243 (2019). URL https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007243."/>
  <meta name="citation_reference" content="Fu, X. et al. Spatial patterns of tumour growth impact clonal diversification in a computational model and the TRACERx Renal study. Nature Ecology & Evolution (2021). URL https://www.nature.com/articles/s41559-021-01586-x."/>
  <meta name="citation_reference" content="Williams, M. J. et al. Quantification of subclonal selection in cancer from bulk sequencing data. Nature Genetics 50, 895 (2018). URL https://www.nature.com/articles/s41588-018-0128-6."/>
  <meta name="citation_reference" content="Avanzini, S. et al. A mathematical model of ctDNA shedding predicts tumor detection size. Science Advances 6, eabc4308 (2020). URL https://www.science.org/doi/10.1126/sciadv.abc4308."/>
  <meta name="citation_reference" content="Bozic, I. et al. Evolutionary dynamics of cancer in response to targeted combination therapy. eLife 2, e00747 (2013). URL https://elifesciences.org/articles/00747."/>
  <meta name="citation_reference" content="Dinh, K. N., Jaksik, R., Kimmel, M., Lambert, A. & Tavaré, S. Statistical Inference for the Evolutionary History of Cancer Genomes. Statistical Science 35, 129–144 (2020). URL https://projecteuclid.org/euclid.ss/1583226033."/>
  <meta name="citation_reference" content="Lahouel, K. et al. Revisiting the tumorigenesis timeline with a data-driven generative model. Proceedings of the National Academy of Sciences 117, 857–864 (2020). URL http://www.pnas.org/lookup/doi/10.1073/pnas.1914589117."/>
  <meta name="citation_reference" content="Bozic, I. & Wu, C. J. Delineating the evolutionary dynamics of cancer from theory to reality. Nature Cancer 1, 580–588 (2020). URL http://www.nature.com/articles/s43018-020-0079-6."/>
  <meta name="citation_reference" content="citation_title=Branching Process Models of Cancer, Mathematical Biosciences Institute Lecture Series;citation_pages=1-63;citation_year=2015"/>
  <meta name="citation_reference" content="Tavaré, S. The linear birth–death process: an inferential retrospective. Advances in Applied Probability 50, 253–269 (2018). URL https://www.cambridge.org/core/product/identifier/S0001867818000848/type/journal_article."/>
  <meta name="citation_reference" content="Heyde, A., Reiter, J. G., Naxerova, K. & Nowak, M. A. Consecutive seeding and transfer of genetic diversity in metastasis. Proceedings of the National Academy of Sciences 116, 14129–14137 (2019). URL https://www.pnas.org/content/116/28/14129."/>
  <meta name="citation_reference" content="Griffith, M. et al. Optimizing Cancer Genome Sequencing and Analysis. Cell Systems 1, 210–223 (2015). URL http://www.sciencedirect.com/science/article/pii/S2405471215001131."/>
  <meta name="citation_reference" content="Haber, D. A. & Velculescu, V. E. Blood-Based Analyses of Cancer: Circulating Tumor Cells and Circulating Tumor DNA. Cancer Discovery 4, 650–661 (2014). URL https://cancerdiscovery.aacrjournals.org/content/4/6/650."/>
  <meta name="citation_reference" content="Leshchiner, I. et al. Comprehensive analysis of tumour initiation, spatial and temporal progression under multiple lines of treatment. preprint, Bioinformatics (2018). URL http://biorxiv.org/lookup/doi/10.1101/508127."/>
  <meta name="citation_reference" content="Myers, M. A., Satas, G. & Raphael, B. J. CALDER: Inferring Phylogenetic Trees from Longitudinal Tumor Samples. Cell Systems 8, 514–522.e5 (2019). URL http://www.sciencedirect.com/science/article/pii/S2405471219301917."/>
  <meta name="citation_reference" content="Carlsson, G., Gullberg, B. & Hafström, L. Estimation of liver tumor volume using different formulas?An experimental study in rats. Journal of Cancer Research and Clinical Oncology 105, 20–23 (1983). URL http://link.springer.com/10.1007/BF00391826."/>
  <meta name="citation_reference" content="Bozic, I., Gerold, J. M. & Nowak, M. A. Quantifying Clonal and Subclonal Passenger Mutations in Cancer Evolution. PLOS Computational Biology 12, e1004731 (2016). URL http://dx.plos.org/10.1371/journal.pcbi.1004731."/>
  <meta name="citation_reference" content="Blokzijl, F. et al. Tissue-specific mutation accumulation in human adult stem cells during life. Nature 538, 260–264 (2016). URL http://www.nature.com/articles/nature19768."/>
  <meta name="citation_reference" content="Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. preprint, Genomics (2021). URL http://biorxiv.org/lookup/doi/10.1101/2021.08.16.456475."/>
  <meta name="citation_reference" content="Auslander, N., Wolf, Y. I. & Koonin, E. V. In silico learning of tumor evolution through mutational time series. Proceedings of the National Academy of Sciences 116, 9501–9510 (2019). URL https://www.pnas.org/content/116/19/9501."/>
  <meta name="citation_reference" content="Petljak, M. et al. Characterizing Mutational Signatures in Human Cancer Cell Lines Reveals Episodic APOBEC Mutagenesis. Cell 176, 1282–1294.e20 (2019). URL https://linkinghub.elsevier.com/retrieve/pii/S0092867419301618."/>
  <meta name="citation_reference" content="Gillespie, D. T. Exact stochastic simulation of coupled chemical reactions. The Journal of Physical Chemistry 81, 2340–2361 (1977). URL https://doi.org/10.1021/j100540a008."/>
  <meta name="citation_reference" content="Bozic, I., Paterson, C. & Waclaw, B. On measuring selection in cancer from subclonal mutation frequencies. PLOS Computational Biology 15, e1007368 (2019). URL https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007368."/>
  <meta name="citation_reference" content="Fabre, M. A. et al. The longitudinal dynamics and natural history of clonal haematopoiesis. preprint, Cancer Biology (2021). URL http://biorxiv.org/lookup/doi/10.1101/2021.08.12.455048."/>
  <meta name="twitter:title" content="Inferring parameters of cancer evolution from sequencing and clinical data"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and after a driver event occurred, and growth rates of both initiated cancer cells and subsequently appearing subclones. We demonstrate that in order to obtain accurate estimates of mutation rate and timing of events, observed mutation counts should be corrected to account for clonal mutations that occurred after the founding of the tumor, as well as sequencing coverage. We apply our methodology to reconstruct the individual evolutionary histories of chronic lymphocytic leukemia patients, finding that the parental leukemic clone typically appears within the first fifteen years of life.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Inferring parameters of cancer evolution from sequencing and clinical data"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.11.18.387837v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and after a driver event occurred, and growth rates of both initiated cancer cells and subsequently appearing subclones. We demonstrate that in order to obtain accurate estimates of mutation rate and timing of events, observed mutation counts should be corrected to account for clonal mutations that occurred after the founding of the tumor, as well as sequencing coverage. We apply our methodology to reconstruct the individual evolutionary histories of chronic lymphocytic leukemia patients, finding that the parental leukemic clone typically appears within the first fifteen years of life.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-01-09"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.11.18.387837v2"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2342328"/>
  <title>
   Inferring parameters of cancer evolution from sequencing and clinical data | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GW-q1yS4llZsZXYilALi0SfHZdla2dBZSfgKsyPoQv0__Mq2l818MUdgw7DQFZ42QKliXne5RR1HOY89XjtvuppE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Sjm8qRC6pDZnJVeNrkx67-ZH1Mte_Ys5jCfEFFU1muc__r6-n363p7S972yCeOZaxDruAzBGzgr4Ie4Ci5dXnv0k__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__-O5AqrChkaQBJNz3KVCNgPD7B2WnlmFaBWhKn3OLnSk__HPUPKsZe95KwbHSnKRhhlLxkHEchbpL1f2QRQnZQX38__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__49Ok9ZaeGdzHaYY0HeAxSZkJaw1DaCwfnGvLBVZ_w1Q__Tcognn9uta6IhrOugHP9-zOAzZitowtsZynB_751i8Q__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__6hIesE0QCBXoXlEIn01f-78o39DmDNEoInQ35EHxoXw__6eCrf7S317r9k23utIBFky8AdpcfocvNZW1IPTr3SUE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Vx66iYj7KBcuF71tzBm8Y4UzUqT2dtUwq3cVuu4mh_8"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2342328","apath":"\/biorxiv\/early\/2022\/01\/09\/2020.11.18.387837.atom","pisa":"biorxiv;2020.11.18.387837v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.11.18.387837v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.11.18.387837v2","title":"Inferring parameters of cancer evolution from sequencing and clinical data","identifier":"node\/2342328"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022w8k4BcJFXA2gB9nku3k54EgrLhs4Opn\\\\\\\/g8sGF9bubzfQGVQe2+FQWGjkY2fuBoicCxMEbUuWC\\\\\\\/+nYO67WckvXNRC7nb\\\\\\\/4Qn1qO7MvHGMwCyymw+FVCXc57yOEmtHJyCZh8kHKIv4iQwdXgY0maqhw\\\\\\\/3uz0rgg1C7BRLMBbxWTlCLXrjLGVAepBcT6TzAPLWB+JMu\\\\\\\/KqTpZwnK21Z7uxTU96l1C12lDaSW0wiJfouA8O54RT7ZJ\\\\\\\/ktevgR2NvS0zgq0W3xzIHiyC6BVrsIJ2viz+WFgUEoyyWb7UBrGbE29DzKu9aa37mg3h+B0aa00endGPhsSLMfR4MbE6r\\\\\\\/8Jvb\\\\\\\/MSFgDgFSumvyub1Z94ioZK4sQ2IQWwENcZbJ9NiOdDM9kgAYdxfm0KFhsfLFDAdm3Sw89uLNfgLbeLZ3b7CmVqyhNVLqhXpmtjqL+GHrA73m5b66P5LklF9F53epbjmfrdeJrOh3gFlqxv2CMfgsLuRATjHACOF4ZoDVaMp1+8jMobfdkhnLMcepcwKTZh6XaeX2sCQdWfvoTpxeaywKA+uWi0PheDFJXOE07P5Rsbzp30MLteuk7ayzOSUlxE1cDNwoZzg\\\\\\\/kop5ueWMaDqFm3rJ6W530k2hWNbL5jkLDZeypWIQVpeA5NGLs0nxSgneixrqM+b9wuJiqTRMB2pUVi9zv6aXnbvIwWihUQMDU7B+6fwT0l4Ql6FUXt+ISux5hRLKRaWhghf+Va8BxhWWTjmUP4mjbqxe\\\\\\\/J1mE07K0chTQ12TUbhnr38wGbqoQ0\\\\\\\/w\\\\\\\/KzVwOzd709qq4b13nYgi8ikk\\\\\\\/5oypkAC1YubMIqNUSWMrYZ\\\\\\\/UkXudgxZ\\\\\\\/QMiVxP4TdABy9agRxaOyJDd7l0YAMFWryiRAX0ga9vNuVGi4vw\\\\\\\/pob\\\\\\\/cVGuTj0CqmPe\\\\\\\/M8VR8y\\\\\\\/KWE3GCMicXI06wGPonsjh7MaCQ7MmR3eV4goWaWB2Uq\\\\\\\/1ENNKBxjaJ4lK0lsXqdaXZyXRgbM1Vuz0L89Al\\\\\\\/bYNC0CvnDJkiGTmkZch4jyB9IOTeQla8MuQNFSquslIWRjG6QkF0MQgN3U6G2d2kkBbIYi\\\\\\\/+qmOxdPE6SrcdAHBWf9tKIc3AbbDCGJHRu+46E905PkwXOAGgk2pwXYBodU46LwnV1Y8iH0TRZvPUfAHQ8mFWhuimeyrtTXZMTTDWqY6ShA4MpO3H\\\\\\\/PYv8ptvqsN1TR2F1JXnmT9IvnYKOqBVPdJAQpi5aEqZreKCFWe\\\\\\\/yp3xcIKGdeNSL+AWYuLbcyZStfemN6mLdaZNljuq8x7xF7lgvTKUkvLbpxkUP9Vkk\\\\\\\/2Sk6XqX6xK+\\\\\\\/lvC25i\\\\\\\/VgF3dEtfDrgdbLgihd3xntrWoNLMzMQ+0i9AEwWCWJXZ6t+mJBTBi3S5MYZc=\\\u0022,\\\u0022iv\\\u0022:\\\u0022CIpYPoILq8BUjhYnf73Y5w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00223e9886e0d8b1d6a5e1aee0d54ed833762b180dda53d9b293678cb4c544067484\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.11.18.387837v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2342328 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.11.18.387837v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_2044057782">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_2044057782" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1331155458" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-KyiG_vzUPekW7_Ve8q9QLwJtONIcMIl173Q-loRGbXo"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2342328" data-node-nid="2342328" id="node-2342328--2716685385" data-pisa="biorxiv;2020.11.18.387837v2" data-pisa-master="biorxiv;2020.11.18.387837" data-apath="/biorxiv/early/2022/01/09/2020.11.18.387837.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Inferring parameters of cancer evolution from sequencing and clinical data
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first" data-delta="0">
                      <span class="nlm-given-names">
                       Nathan
                      </span>
                      <span class="nlm-surname">
                       Lee
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Ivana
                      </span>
                      <span class="nlm-surname">
                       Bozic
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.11.18.387837
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2342328--2716685385" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Nathan Lee
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Applied Mathematics, University of Washington
                       </span>
                       , Seattle, WA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Nathan%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Lee%20N&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ANathan%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Ivana Bozic
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Applied Mathematics, University of Washington
                       </span>
                       , Seattle, WA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Herbold Computational Biology Program, Fred Hutchinson Cancer Research Center
                       </span>
                       , Seattle, WA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Ivana%2BBozic%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Bozic%20I&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AIvana%2BBozic%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:ibozic@uw.edu" class="" data-icon-position="" data-hide-link-title="0">
                        ibozic@uw.edu
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.11.18.387837v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2342328" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2342328/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.18.387837v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2342328" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2342328/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.18.387837v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2342328" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2342328/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.18.387837v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2342328" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2342328/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.18.387837v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2342328" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2342328/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.11.18.387837v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2342328" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2342328/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-2">
                             As a cancer develops, its cells accrue new mutations, resulting in a heterogeneous, complex genomic profile. We make use of this heterogeneity to derive simple, analytic estimates of parameters driving carcinogenesis and reconstruct the timeline of selective events following initiation of an individual cancer. Using stochastic computer simulations of cancer growth, we show that we can accurately estimate mutation rate, time before and after a driver event occurred, and growth rates of both initiated cancer cells and subsequently appearing subclones. We demonstrate that in order to obtain accurate estimates of mutation rate and timing of events, observed mutation counts should be corrected to account for clonal mutations that occurred after the founding of the tumor, as well as sequencing coverage. We apply our methodology to reconstruct the individual evolutionary histories of chronic lymphocytic leukemia patients, finding that the parental leukemic clone typically appears within the first fifteen years of life.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-4">
                             When a cell accrues a sequence of driver mutations – genetic alterations that provide a proliferative advantage relative to surrounding cells – it can begin to divide uncontrollably and eventually develop the complex features of a cancer [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             –
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ]. Thousands of specific driver mutations have been implicated in carcinogenesis, with individual tumors harboring from few to dozens of drivers, depending on the cancer type [
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ]. Mutations that don’t have a significant effect on cellular fitness also arise, both before and after tumor initiation [
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ]. These neutral mutations, or “passengers”, can reach detectable frequencies by random genetic drift or the positive selection of a driver mutation in the same cell [
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             –
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ]. Mutational burden detectable by bulk sequencing reveals tens to thousands of passengers per tumor [
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ,
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ].
                            </p>
                            <p id="p-5">
                             Genome sequencing technologies have revealed the heterogeneous, informative genetic profiles produced by the evolutionary process driving carcinogenesis [
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ,
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ]. These genetic profiles have been used to obtain insight into specific features of the carcinogenic process operating in individual patients. For example, the molecular clock feature of passenger mutations has been employed to measure timing of early events in tumor formation, as well as identify stages of tumorigenesis and metastasis [
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             –
                             <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                              22
                             </a>
                             ]. Other studies have estimated mutation rates [
                             <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ,
                             <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ,
                             <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                              24
                             </a>
                             ], selective growth advantages of cancer subclones [
                             <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                              25
                             </a>
                             –
                             <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                              28
                             </a>
                             ], and the effect of spatial structure on cancer evolution [
                             <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                              29
                             </a>
                             –
                             <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                              31
                             </a>
                             ]. We note that previous approaches typically only estimate one or a few parameters of cancer evolution. In addition, many state of the art methods make use of computationally expensive approaches [
                             <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">
                              24
                             </a>
                             ,
                             <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                              30
                             </a>
                             ,
                             <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                              32
                             </a>
                             ] or simplifying assumptions, such as approximating tumor expansion as deterministic or ignoring cell death [
                             <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                              27
                             </a>
                             ,
                             <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">
                              32
                             </a>
                             ].
                            </p>
                            <p id="p-6">
                             Mathematical models of cancer progression, especially when used in conjunction with experimental and clinical data, can provide important insights into the evolutionary history of cancer [
                             <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ,
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              19
                             </a>
                             ,
                             <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                              33
                             </a>
                             –
                             <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                              37
                             </a>
                             ]. Branching processes – a type of a stochastic process – can be used to model how different populations of dividing, dying, and mutating cells in a tumor evolve over time [
                             <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                              38
                             </a>
                             ]. Their theory and applications have been well developed to model the multistage nature of cancer development [
                             <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">
                              25
                             </a>
                             ,
                             <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">
                              29
                             </a>
                             ,
                             <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                              35
                             </a>
                             ,
                             <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">
                              38
                             </a>
                             –
                             <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                              40
                             </a>
                             ]. Here we use a branching process model of carcinogenesis to derive a comprehensive reconstruction of an individual tumor’s evolution.
                            </p>
                            <p id="p-7">
                             Tumors can grow for many years, even decades, before they reach detectable size [
                             <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             ]. Typically, tumor samples used for sequencing would be obtained at the end of the tumor’s natural, untreated progression. More recently, longitudinal sequencing, where a tumor is sequenced at multiple times during its development, has provided better resolution of tumor growth dynamics and evolution in various cancer types [
                             <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">
                              27
                             </a>
                             ,
                             <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                              41
                             </a>
                             –
                             <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                              44
                             </a>
                             ]. We establish that two longitudinal bulk sequencing and tumor size measurements are sufficient to reconstruct virtually all parameters (mutation rate, growth rates, times of appearance of driver mutations, and time since the driver mutation) of cancer evolution in individual patients. Our analytic approach yields simple formulas for the parameters; thus estimation of the parameters governing cancer growth is not computationally intensive, regardless of tumor size. Our framework makes possible a personalized, high-resolution reconstruction of a tumor’s timeline of selective events and quantitative characterization of the evolutionary dynamics of the subclones making up the tumor.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Results
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Model
                             </h3>
                             <p id="p-8">
                              We consider a multi-type branching process of tumor expansion (
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Fig. 1a
                              </a>
                              ). Tumor growth is started with a single initiated cell at time 0. Initiated tumor cells divide with rate b and die with rate d. These cells already have the driver mutations necessary for expansion, so we assume b > d. The population of initiated cells can go extinct due to stochastic fluctuations, or survive stochastic drift and start growing (on average) exponentially with net growth rate r = b−d. We will focus only on those populations that survived stochastic drift.
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F1.large.jpg?width=800&height=600&carousel=1" title="Stochastic branching process model of tumor evolution. (a) Stochastic branching process model for tumor expansion. Initiated tumor cells (blue) divide with birth rate b, die with death rate d, and accrue passenger mutations with mutation rate u. Type-1 cells, which carry the driver mutation, divide with birth rate b1, die with death rate d1, and accrue passenger mutations with mutation rate u. (b) The initiated tumor, or type-0, (blue) population growth is initiated from a single cell. A driver mutation occurs in a single type-0 cell at time t1, starting the type-1 population (red). The tumor is bulk sequenced at times t1 + t and t1 + t + δ. (c) By the time the tumor is observed, it has a high level of genetic heterogeneity due to the mutations that have accrued in both type-0 (blue) and type-1 populations (red). Each yellow star represents a different passenger mutation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-500246560" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Stochastic branching process model of tumor evolution.</span> (a) Stochastic branching process model for tumor expansion. Initiated tumor cells (blue) divide with birth rate <em>b</em>, die with death rate <em>d</em>, and accrue passenger mutations with mutation rate <em>u</em>. Type-1 cells, which carry the driver mutation, divide with birth rate <em>b</em><sub>1</sub>, die with death rate <em>d</em><sub>1</sub>, and accrue passenger mutations with mutation rate <em>u</em>. (b) The initiated tumor, or type-0, (blue) population growth is initiated from a single cell. A driver mutation occurs in a single type-0 cell at time <em>t</em><sub>1</sub>, starting the type-1 population (red). The tumor is bulk sequenced at times <em>t</em><sub>1</sub> + <em>t</em> and <em>t</em><sub>1</sub> + <em>t</em> + <em>δ</em>. (c) By the time the tumor is observed, it has a high level of genetic heterogeneity due to the mutations that have accrued in both type-0 (blue) and type-1 populations (red). Each yellow star represents a different passenger mutation.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F1.medium.gif" width="440" height="332"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F1.medium.gif" width="440" height="332"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1:
                               </span>
                               <span class="caption-title">
                                Stochastic branching process model of tumor evolution.
                               </span>
                               <p id="p-9" class="first-child">
                                (a) Stochastic branching process model for tumor expansion. Initiated tumor cells (blue) divide with birth rate b, die with death rate d, and accrue passenger mutations with mutation rate u. Type-1 cells, which carry the driver mutation, divide with birth rate b
                                <sub>
                                 1
                                </sub>
                                , die with death rate d
                                <sub>
                                 1
                                </sub>
                                , and accrue passenger mutations with mutation rate u. (b) The initiated tumor, or type-0, (blue) population growth is initiated from a single cell. A driver mutation occurs in a single type-0 cell at time t
                                <sub>
                                 1
                                </sub>
                                , starting the type-1 population (red). The tumor is bulk sequenced at times t
                                <sub>
                                 1
                                </sub>
                                + t and t
                                <sub>
                                 1
                                </sub>
                                + t + δ. (c) By the time the tumor is observed, it has a high level of genetic heterogeneity due to the mutations that have accrued in both type-0 (blue) and type-1 populations (red). Each yellow star represents a different passenger mutation.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-10">
                              At some time t
                              <sub>
                               1
                              </sub>
                              > 0 a new driver mutation occurs in a single initiated tumor cell, starting a new independent birth-death process, with birth rate b
                              <sub>
                               1
                              </sub>
                              and death rate d
                              <sub>
                               1
                              </sub>
                              (
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Fig. 1b
                              </a>
                              ). Net growth rate of cells with the new driver is r
                              <sub>
                               1
                              </sub>
                              = b
                              <sub>
                               1
                              </sub>
                              − d
                              <sub>
                               1
                              </sub>
                              . The new driver increases the rate of growth, i.e., r
                              <sub>
                               1
                              </sub>
                              > r. We define the driver’s selective growth advantage by g = (r
                              <sub>
                               1
                              </sub>
                              /r − 1). In addition, both populations of cells (with and without the driver) accrue passenger mutations with rate u (
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Fig. 1c
                              </a>
                              ).
                             </p>
                             <p id="p-11">
                              After the driver mutation occurs, an additional time t passes before the tumor is observed. Cells containing i new driver mutations, where i is either 0 or 1, will be referred to as type-i cells or simply, clone i. In Materials and Methods we also analyze the more general case of two nested or sibling driver mutations, as well as the fully generalized case of any clonal structure that might arise during tumor expansion.
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Parameter estimates from two longitudinal measurements
                             </h3>
                             <p id="p-12">
                              We demonstrate that with two longitudinal bulk sequencing measurements, it is possible to accurately estimate net growth rates, time of appearance of a driver mutation, time between a driver mutation and observation, and mutation rate in the tumor. The tumor is first sequenced at time of observation, t
                              <sub>
                               1
                              </sub>
                              + t, where both time of driver mutation, t
                              <sub>
                               1
                              </sub>
                              , and time from driver mutation to observation, t, are yet unknown (
                              <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                               Fig. 1b
                              </a>
                              ). A second bulk sequencing is performed at t
                              <sub>
                               1
                              </sub>
                              + t + δ, a known δ time units after the tumor is first observed (
                              <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                               Fig. 1b
                              </a>
                              ). From the bulk sequencing data, the fraction of cells carrying the driver mutation, α
                              <sub>
                               1
                              </sub>
                              and α
                              <sub>
                               2
                              </sub>
                              , can be measured at the timepoints t
                              <sub>
                               1
                              </sub>
                              + t and t
                              <sub>
                               1
                              </sub>
                              + t + δ, respectively. We denote total number of cells in the tumor at the two bulk sequencing timepoints as M
                              <sub>
                               1
                              </sub>
                              and M
                              <sub>
                               2
                              </sub>
                              . Number of cells in the tumor can be estimated from measurements of tumor volume [
                              <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                               45
                              </a>
                              ].
                             </p>
                             <p id="p-13">
                              Equating expected values of the sizes of type-0 and type-1 population at the two bulk sequencing time points with the measured numbers of cells present in clones 0 and 1, we obtain estimates of the net growth rates of the two subclones:
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              From the growth rate estimates and subclone sizes, we can approximate the expected value of the time a population in a branching process takes to reach an observed size [
                              <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">
                               38
                              </a>
                              ]. This yields an estimate of the time t from the appearance of driver mutation until observation:
                              <span class="disp-formula" id="disp-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Using the bulk sequencing data from the second timepoint, γ, the number of subclonal passengers between the specified frequencies f
                              <sub>
                               1
                              </sub>
                              and f
                              <sub>
                               2
                              </sub>
                              , can be measured. Using results from previous work [
                              <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                               46
                              </a>
                              ], we derive the expected value of γ (Materials and Methods), which can be used to estimate the mutation rate u:
                              <span class="disp-formula" id="disp-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              The m passenger mutations that were present in the original type-1 cell when the driver mutation occurred (
                              <a id="xref-fig-1-6" class="xref-fig" href="#F1">
                               Fig. 1c
                              </a>
                              ) are present in all type-1 cells. m can be estimated from bulk sequencing data, and used to estimate time of appearance of the driver. We maximize the likelihood function P (m|t
                              <sub>
                               1
                              </sub>
                              ) with respect to time of appearance of the driver, t
                              <sub>
                               1
                              </sub>
                              , (see Materials and Methods) to obtain the maximum likelihood estimate
                              <span class="disp-formula" id="disp-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-6.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-6.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Using formulas (
                              <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">
                               4
                              </a>
                              ) and (
                              <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">
                               5
                              </a>
                              ), we can now estimate t
                              <sub>
                               1
                              </sub>
                              .
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Estimates verified in simulated tumors
                             </h3>
                             <p id="p-14">
                              To assess the accuracy of the parameter estimates for several modes of tumor evolution, we simulate tumor growth by performing a Monte Carlo simulation, which simulates the birth, death, and accumulation of mutations in the individual cells that make up a tumor. This simulation generates the mutation frequency and tumor size data used by the estimates (see Methods section for details of simulation). We simulate three different types of tumors (slow growing, fast growing, and no cell death), with a high and a low mutation rate for each.
                             </p>
                             <p id="p-15">
                              In a simulation of a fast growing tumor with a single subclonal driver mutation that confers a strong selective growth advantage of 100%, we can accurately estimate growth rates, mutation rate, time of driver event, and time since driver event (
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              ). Growth rates of both initiated tumor and driver subclones can be estimated with a high degree of accuracy, achieving mean percentage error (MPE) of -0.07% and 0.03% for the lower mutation rate (u = 1) scenario. The mutation rate u and estimates for time of driver appearance, t
                              <sub>
                               1
                              </sub>
                              , and time since driver, t, can also be estimated accurately, with MPEs of -0.9%, 3.8%, and -0.4%, respectively. Estimates for u, t
                              <sub>
                               1
                              </sub>
                              , and t have a somewhat greater degree of variation compared to the growth rate estimates, due to the inherent randomness of the number of mutations and time to reach the observed size that occur in each realization of the stochastic process.
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F2.large.jpg?width=800&height=600&carousel=1" title="Accuracy of parameter inferences from simulated data. We simulated tumor growth by performing a Monte Carlo simulation, which simulates the birth, death, and accumulation of mutations in the individual cells that make up a tumor, and generates the mutation frequency and tumor size data used by the estimates. Mean percent errors (MPEs) of estimates are shown in black above the plots, and mean absolute percent errors (MAPEs) are shown in gray. Boxes contain 25th-75th quartiles, with median indicated by thick horizontal black line. Whiskers of boxplots indicate 2.5 and 97.5 percentiles. Violins are smoothed density estimates of the percent error data points. Ground truth parameter set: b = b1 = 0.25, d = 0.18, d1 = 0.11, t1 = 70, t = 50, δ = 20, f1 = 1%, and f2 = 20%. Mutation rate (a) u = 1, (b) u = 3. At least 100 Monte Carlo simulation runs with a surviving tumor performed for each parameter combination." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-500246560" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Accuracy of parameter inferences from simulated data.</span> We simulated tumor growth by performing a Monte Carlo simulation, which simulates the birth, death, and accumulation of mutations in the individual cells that make up a tumor, and generates the mutation frequency and tumor size data used by the estimates. Mean percent errors (MPEs) of estimates are shown in black above the plots, and mean absolute percent errors (MAPEs) are shown in gray. Boxes contain 25th-75th quartiles, with median indicated by thick horizontal black line. Whiskers of boxplots indicate 2.5 and 97.5 percentiles. Violins are smoothed density estimates of the percent error data points. Ground truth parameter set: <em>b</em> = <em>b</em><sub>1</sub> = 0.25, <em>d</em> = 0.18, <em>d</em><sub>1</sub> = 0.11, <em>t</em><sub>1</sub> = 70, <em>t</em> = 50, <em>δ</em> = 20, <em>f</em><sub>1</sub> = 1%, and <em>f</em><sub>2</sub> = 20%. Mutation rate (a) <em>u</em> = 1, (b) <em>u</em> = 3. At least 100 Monte Carlo simulation runs with a surviving tumor performed for each parameter combination.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F2.medium.gif" width="440" height="159"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F2.medium.gif" width="440" height="159"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2:
                               </span>
                               <span class="caption-title">
                                Accuracy of parameter inferences from simulated data.
                               </span>
                               <p id="p-16" class="first-child">
                                We simulated tumor growth by performing a Monte Carlo simulation, which simulates the birth, death, and accumulation of mutations in the individual cells that make up a tumor, and generates the mutation frequency and tumor size data used by the estimates. Mean percent errors (MPEs) of estimates are shown in black above the plots, and mean absolute percent errors (MAPEs) are shown in gray. Boxes contain 25th-75th quartiles, with median indicated by thick horizontal black line. Whiskers of boxplots indicate 2.5 and 97.5 percentiles. Violins are smoothed density estimates of the percent error data points. Ground truth parameter set: b = b
                                <sub>
                                 1
                                </sub>
                                = 0.25, d = 0.18, d
                                <sub>
                                 1
                                </sub>
                                = 0.11, t
                                <sub>
                                 1
                                </sub>
                                = 70, t = 50, δ = 20, f
                                <sub>
                                 1
                                </sub>
                                = 1%, and f
                                <sub>
                                 2
                                </sub>
                                = 20%. Mutation rate (a) u = 1, (b) u = 3. At least 100 Monte Carlo simulation runs with a surviving tumor performed for each parameter combination.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-17">
                              For the parameter regime with no cell death and the regime for a slow-growing tumor, we again achieve high accuracies for the net growth rates (Fig. S1, Fig. S2). In the lower mutation rate (u = 1) scenario, parameter estimates for the mutation rate u and time of driver appearace t
                              <sub>
                               1
                              </sub>
                              can be accurately estimated for both regimes, with MPEs of -1.3% and 4.9% for the no cell death case, and MPEs of -3% and 3.7% for the slow-growing tumor. The t, time since driver event, estimates have somewhat higher errors, with MPE of -6.3% for the no cell death case, and MPE of 30.3% for the slow-growing tumor.
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Correcting mutation counts observed from genome sequencing data
                             </h3>
                             <p id="p-18">
                              We note that in our estimate for the time of appearance of the driver, t
                              <sub>
                               1
                              </sub>
                              (see formula (
                              <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">
                               5
                              </a>
                              )), used for comparison to simulated data, we employed a correction to m, the number of mutations that were present in the founder type-1 cell at t
                              <sub>
                               1
                              </sub>
                              . From sequencing data, these m mutations are indistinguishable (
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Fig. 3a
                              </a>
                              ) from mutations that occurred after t
                              <sub>
                               1
                              </sub>
                              in type-1 cells, and reached fixation in the type-1 population [
                              <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">
                               46
                              </a>
                              ]. Thus, the value of m observed from sequencing data, m
                              <sub>
                               obs
                              </sub>
                              , will overestimate the true m. In Materials and Methods we show that the expected value of the number of passengers that occurred after t
                              <sub>
                               1
                              </sub>
                              and reached fixation in the type-1 population is u/r
                              <sub>
                               1
                              </sub>
                              . We subtract this correction factor from m
                              <sub>
                               obs
                              </sub>
                              :
                              <span class="disp-formula" id="disp-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F3.large.jpg?width=800&height=600&carousel=1" title="Corrections for observed mutation counts. (a) If passenger mutations (circles with stars) that occur after the driver reach fixation in the driver population (red), then they are indistinguishable from the passengers that were present in the first cell with the driver, which accrued in the type-0 population (blue). The estimate of when the driver occurred needs to account for these mutations (circled). In (b), we compare percent errors of parameter estimates for time from tumor initiaton until appearance of a driver subclone, t1, with and without this correction (Eq. (6)). Errors for estimate with correction (Eq. (12)) are shown in blue, and for estimate without correction (Eq. (5)) in orange. Errors are plotted as a kernel density estimate for Monte Carlo simulations of slow growing tumor with mutation rate u = 5. Mean percent errors (MPEs) and mean absolute percent errors (MAPEs) are listed. (c) Mutations present on two or fewer variant reads (red) are filtered out in post-processing. Mutations with more than two variant reads (black) are included. The number of subclonal mutations between frequencies f1 and f2, γ, which is used in the mutation rate estimate, must be corrected for mutations that are filtered out. In (d), the percent errors for the observed (orange) and corrected (blue) γ (Eq. (7)) are plotted as kernel density estimates. Observed mutations are those that passed post-processing, i.e. those that have more than L = 2 mutant reads. True mutation frequencies were generated from 135 surviving runs of a Monte Carlo simulation of a fast growing tumor with mutation rate u = 1, from which sequencing reads were simulated with 200x average coverage (see Materials and Methods). Percent errors are calculated relative to the true γ measured from the true mutation frequencies." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-500246560" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Corrections for observed mutation counts.</span> (a) If passenger mutations (circles with stars) that occur after the driver reach fixation in the driver population (red), then they are indistinguishable from the passengers that were present in the first cell with the driver, which accrued in the type-0 population (blue). The estimate of when the driver occurred needs to account for these mutations (circled). In (b), we compare percent errors of parameter estimates for time from tumor initiaton until appearance of a driver subclone, <em>t</em><sub>1</sub>, with and without this correction (Eq. (6)). Errors for estimate with correction (Eq. (12)) are shown in blue, and for estimate without correction (Eq. (5)) in orange. Errors are plotted as a kernel density estimate for Monte Carlo simulations of slow growing tumor with mutation rate u = 5. Mean percent errors (MPEs) and mean absolute percent errors (MAPEs) are listed. (c) Mutations present on two or fewer variant reads (red) are filtered out in post-processing. Mutations with more than two variant reads (black) are included. The number of subclonal mutations between frequencies <em>f</em><sub>1</sub> and <em>f</em><sub>2</sub>, <em>γ</em>, which is used in the mutation rate estimate, must be corrected for mutations that are filtered out. In (d), the percent errors for the observed (orange) and corrected (blue) <em>γ</em> (Eq. (7)) are plotted as kernel density estimates. Observed mutations are those that passed post-processing, i.e. those that have more than <em>L</em> = 2 mutant reads. True mutation frequencies were generated from 135 surviving runs of a Monte Carlo simulation of a fast growing tumor with mutation rate <em>u</em> = 1, from which sequencing reads were simulated with 200x average coverage (see Materials and Methods). Percent errors are calculated relative to the true <em>γ</em> measured from the true mutation frequencies.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F3.medium.gif" width="440" height="290"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F3.medium.gif" width="440" height="290"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3:
                               </span>
                               <span class="caption-title">
                                Corrections for observed mutation counts.
                               </span>
                               <p id="p-19" class="first-child">
                                (a) If passenger mutations (circles with stars) that occur after the driver reach fixation in the driver population (red), then they are indistinguishable from the passengers that were present in the first cell with the driver, which accrued in the type-0 population (blue). The estimate of when the driver occurred needs to account for these mutations (circled). In (b), we compare percent errors of parameter estimates for time from tumor initiaton until appearance of a driver subclone, t
                                <sub>
                                 1
                                </sub>
                                , with and without this correction (
                                <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">
                                 Eq. (6)
                                </a>
                                ). Errors for estimate with correction (
                                <a id="xref-disp-formula-12-1" class="xref-disp-formula" href="#disp-formula-12">
                                 Eq. (12)
                                </a>
                                ) are shown in blue, and for estimate without correction (
                                <a id="xref-disp-formula-5-3" class="xref-disp-formula" href="#disp-formula-5">
                                 Eq. (5)
                                </a>
                                ) in orange. Errors are plotted as a kernel density estimate for Monte Carlo simulations of slow growing tumor with mutation rate u = 5. Mean percent errors (MPEs) and mean absolute percent errors (MAPEs) are listed. (c) Mutations present on two or fewer variant reads (red) are filtered out in post-processing. Mutations with more than two variant reads (black) are included. The number of subclonal mutations between frequencies f
                                <sub>
                                 1
                                </sub>
                                and f
                                <sub>
                                 2
                                </sub>
                                , γ, which is used in the mutation rate estimate, must be corrected for mutations that are filtered out. In (d), the percent errors for the observed (orange) and corrected (blue) γ (
                                <a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">
                                 Eq. (7)
                                </a>
                                ) are plotted as kernel density estimates. Observed mutations are those that passed post-processing, i.e. those that have more than L = 2 mutant reads. True mutation frequencies were generated from 135 surviving runs of a Monte Carlo simulation of a fast growing tumor with mutation rate u = 1, from which sequencing reads were simulated with 200x average coverage (see Materials and Methods). Percent errors are calculated relative to the true γ measured from the true mutation frequencies.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-20">
                              The correction for the m mutations present in the original type-1 cell (
                              <a id="xref-disp-formula-6-2" class="xref-disp-formula" href="#disp-formula-6">
                               6
                              </a>
                              ) at time t
                              <sub>
                               1
                              </sub>
                              improves the accuracy of the estimate for time of appearance of driver mutation t
                              <sub>
                               1
                              </sub>
                              . For the fast growing tumor with mutation rate u = 1 (Fig. S3a), the correction lowers the mean percent error (MPE) of the t
                              <sub>
                               1
                              </sub>
                              estimate from 14.0% to 3.8%. For the slow growing tumor with mutation rate u = 5 (
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Fig. 3b
                              </a>
                              ), the correction lowers the MPE of the t
                              <sub>
                               1
                              </sub>
                              estimate from 22.0% to 5.7% (
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Fig. 3b
                              </a>
                              ).
                             </p>
                             <p id="p-21">
                              Another issue arises from obtaining mutation count γ, number of mutations with frequency between f
                              <sub>
                               1
                              </sub>
                              and f
                              <sub>
                               2
                              </sub>
                              , from genome sequencing data. When sequencing data is post-processed by filtering out mutations with L or fewer variant reads, low-frequency mutations will be difficult to detect [
                              <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">
                               35
                              </a>
                              ] (
                              <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                               Fig. 3c
                              </a>
                              ). For a sample with average sequencing coverage of R and tumor purity p, mutations with mutant allele frequency below L/(pR) will typically not be observable. As a result, since mutations with frequencies between f
                              <sub>
                               1
                              </sub>
                              and f
                              <sub>
                               2
                              </sub>
                              count towards γ, if f
                              <sub>
                               1
                              </sub>
                              ≤ 2L/(pR), the observed number of subclonal mutations between frequencies f
                              <sub>
                               1
                              </sub>
                              and f
                              <sub>
                               2
                              </sub>
                              , γ
                              <sub>
                               obs
                              </sub>
                              , will underestimate the true value, γ. In the Materials and Methods, we derive a correction for γ, based on the expected value of the number of subclonal mutations present at cancer cell frequencies (CCFs) between f
                              <sub>
                               1
                              </sub>
                              and 2L/(pR):
                              <span class="disp-formula" id="disp-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-10.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-10.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Before applying our methodology to patient sequencing data, we estimated the validity of the above correction applied to observed simulated mutation counts. When we simulate sequencing reads from simulated mutation frequencies (see Materials and Methods) and post-process by removing mutations with L = 2 or fewer variant reads, the adjustment we derived for mutation count γ (
                              <a id="xref-disp-formula-7-2" class="xref-disp-formula" href="#disp-formula-7">
                               7
                              </a>
                              ) is critical, even for average sequencing coverage of 200x (
                              <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                               Fig. 3d
                              </a>
                              ). Without any correction, the observed γ has MPE of -53.3% compared to true γ, but with the correction, the computed γ has MPE of -1.4%. When average coverage is 100x, this correction becomes even more important, as many of the low-frequency mutations are discarded (Fig. S3b). Without any correction, the observed γ has MPE of -79.7%. With the correction the computed γ has MPE of -3.4%. The accuracy of the γ measurement affects our estimate of the mutation rate (
                              <a id="xref-disp-formula-4-2" class="xref-disp-formula" href="#disp-formula-4">
                               4
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Estimating parameters for individual patients with CLL
                             </h3>
                             <p id="p-22">
                              We use our formulas to infer the patient-specific parameters of cancer evolution for four patients with chronic lymphocytic leukemia (CLL) whose growth patterns and clonal dynamics were analyzed in [
                              <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">
                               27
                              </a>
                              ]. These CLLs had peripheral white blood cell (WBC) counts measured and whole exome sequencing (WES) performed at least twice before treatment. We consider patients whose WBC counts were classified as having an exponential-like growth pattern, with average γ
                              <sub>
                               obs
                              </sub>
                              > 2 and 3 or fewer macsoscopic subclones (i.e. subclones with cancer cell fractions of 20% or greater for at least one pre-treatment time point). As in Ref. [
                              <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">
                               27
                              </a>
                              ], we perform subclonal reconstruction for each patient using PhylogicNDT [
                              <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                               43
                              </a>
                              ]. To obtain confidence intervals for our parameter estimates, we utilize a sampling procedure to account for model and measurement uncertainties, including uncertainties in subclone frequencies, fitted growth curves, and the Poisson process for mutation accumulation (see Materials and Methods). For each patient’s tumor, we compute estimates of the growth rate of each clone, exome mutation rate, the times that each subclone arose, and how long each subclone expanded before the tumor was detected (Table S1). We reconstruct these histories for tumors with various clonal structures.
                             </p>
                             <p id="p-23">
                              Patients 3 and 21 are examples of a CLL with a single subclone (
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Fig. 4
                              </a>
                              ). For Patient 3, Clone 0, the most recent common ancestor (MRCA) of this patient’s CLL, was initiated when the patient was 14.6 [1.4, 26.8] years old (median and [95% confidence interval] of estimate). Clone 0 grew with a net growth rate of 0.51 [0.20, 0.85] per year. 18.9 years later, Clone 1 was initiated when the patient was 33.5 [24.1, 39.2] years old. Clone 1 expanded with a growth rate of 0.85 [0.65, 1.04] per year (corresponding to a selective growth advantage of 68.7% over Clone 0), and the patient was diagnosed 29.5 [23.8, 38.9] years later at age 63. We find that the CLL exome mutation rate was 0.48 [0.39, 0.59] mutations per year in this patient.
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F4.large.jpg?width=800&height=600&carousel=1" title="Reconstructing the timeline of CLL evolution in patients. We applied our methodology to estimate subclonal growth rates, mutation rates and evolutionary timelines in CLL tumors from Ref. [27]. Vertical height of a clone represents its log10-scaled size. Phylogenetic trees, colored by clone number, show annotated driver mutations along the trees’ edges. For each patient, we show estimates for patient age at CLL initiation and times of appearance of CLL subclones. Dashed white line indicates when the patient was diagnosed. Solid black arrows indicate times of bulk sequencing measurements." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-500246560" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Reconstructing the timeline of CLL evolution in patients.</span> We applied our methodology to estimate subclonal growth rates, mutation rates and evolutionary timelines in CLL tumors from Ref. [27]. Vertical height of a clone represents its log<sub>10</sub>-scaled size. Phylogenetic trees, colored by clone number, show annotated driver mutations along the trees’ edges. For each patient, we show estimates for patient age at CLL initiation and times of appearance of CLL subclones. Dashed white line indicates when the patient was diagnosed. Solid black arrows indicate times of bulk sequencing measurements.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F4.medium.gif" width="305" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F4.medium.gif" width="305" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/09/2020.11.18.387837/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4:
                               </span>
                               <span class="caption-title">
                                Reconstructing the timeline of CLL evolution in patients.
                               </span>
                               <p id="p-24" class="first-child">
                                We applied our methodology to estimate subclonal growth rates, mutation rates and evolutionary timelines in CLL tumors from Ref. [
                                <a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">
                                 27
                                </a>
                                ]. Vertical height of a clone represents its log
                                <sub>
                                 10
                                </sub>
                                -scaled size. Phylogenetic trees, colored by clone number, show annotated driver mutations along the trees’ edges. For each patient, we show estimates for patient age at CLL initiation and times of appearance of CLL subclones. Dashed white line indicates when the patient was diagnosed. Solid black arrows indicate times of bulk sequencing measurements.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-25">
                              For patient 21, we estimate that the parental clone (MRCA, Clone 0) of this patient’s CLL was initiated when the patient was 6.4 [0.3, 16.7] years old, and grew with a net growth rate of 0.79 [0.30, 1.14] per year. Clone 1 appeared when the patient was 19.6 [10.8, 24.0] years old, and grew more quickly than Clone 0, with a growth rate of 1.52 [1.01, 2.04] per year (corresponding to selective growth advantage of 91.4% over Clone 0). Clone 1 contained a FGFR1 mutation, which might have been acting as a driver of the increased net proliferation. Clone 1 then grew for 15.4 [11.0, 24.2] years before the patient was diagnosed at age 35. We estimate that this patient’s CLL had an exome mutation rate of 0.20 [0.19, 0.23] mutations per year.
                             </p>
                             <p id="p-26">
                              Patients 6 and 9 present more complex clonal structures. CLL of Patient 9 contains two sibling subclones, Clones 1 and 2, in addition to the parental population, Clone 0. Clone 2 contains a nested subclone (Clone 3). Clone 0 arose when the patient was 4.9 [1.2, 10.8] years old, and had a growth rate of 0.28 [0.17, 0.42] per year. Clone 1 arose when the patient was 18.8 [8.8, 35.1] years old. At the time of sequencing, Clone 1 had a negative growth rate of -0.40 [-0.45, -0.19] (/year). Clone 2, containing a KRAS mutation, had the largest net growth rate of the three clones (0.67 [0.49, 0.94] per year), corresponding to a selective growth advantage of 140.9% over the parental clone. Clone 2 arose when the patient was 21.3 [7.7, 31.7] years old. Clone 3 was initiated from within Clone 2 when the patient was 24.8 [10.3, 37.6] years old. We estimate that the CLL exome mutation rate of Patient 9 is 0.36 [0.35, 0.37] mutations/year.
                             </p>
                             <p id="p-27">
                              CLL of Patient 6 contains two sibling subclones (Clones 1 and 2) descendant from the leukemic MRCA Clone 0. Clone 1 has a nested subclone (Clone 3). We estimate that the CLL was initiated when the patient was 2.8 [0.1, 13.2] years old. Clone 0 then grew at a rate of 0.68 [0.15, 1.30] per year. Approximately 33 years after the appearance of Clone 0, when the patient was 35.4 [21.7, 46.1] years old, the first subclone, Clone 1 appeared. Clone 1 had a net growth rate of 0.41 [0.08, 0.73] per year. Clone 3 arose from within Clone 1 when the patient was 45.9 [31.3, 54.6] years old. This clone had net growth rate 1.09 [0.65, 1.78] per year. Clone 3 harbored a driver mutation in ASXL1 and had selective growth advantage of 60.8% over Clone 0. Clone 2, nested in parental clone (Clone 0), was initiated when the patient was 46.7 [25.6, 57.5] years old and had growth rate 0.46 [0.08, 0.85] per year. The patient was then diagnosed at age 58, eventually needing treatment 12.0 years after diagnosis. In Patient 6, we estimate a CLL exome mutation rate of 0.15 [0.12, 0.19] mutations per year.
                             </p>
                             <p id="p-28">
                              The average mutation rate in the four CLL patients we analyze is 0.30 mutations/year. This rate is over the exome, which accounts for ∼ 1% of the human genome. Our average estimated mutation rate in CLL exomes is similar to the measured rate of accumulation of mutations in human tissues of 40 mutations per year over the entire genome [
                              <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                               47
                              </a>
                              ]. Other recent work has estimated a mutation rate of 17 mutations per year in human haematopoietic stem cell/multipotent progenitors [
                              <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                               48
                              </a>
                              ]. Our estimated mutation rates during CLL progression are on par or higher than the recent estimates in healthy hematopoietic cells [
                              <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">
                               48
                              </a>
                              ], in line with the expectation that mutation rates may be increased in cancer. The estimated times of appearance of CLL subclones are very long, on the order of 10 years or more. This finding is agreement with results from Gruber et al. [
                              <a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">
                               27
                              </a>
                              ], who find few new CLL subclones over years to a decade of evolution. We observe that CLL initiation occurred early in most patients, within the first fifteen years of their lives, consistent with recent work in other cancer types [
                              <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">
                               19
                              </a>
                              ,
                              <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                               36
                              </a>
                              ].
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-8">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-29">
                             We use a stochastic branching process model to reconstruct the timing of driver events and quantify the evolutionary dynamics of different subclonal populations of cancer cells. We can accurately estimate the growth rates of tumor subclones, selective growth advantage of individual driver mutations, mutation rate in the tumor, time between tumor initiation and appearance of a subclonal driver mutation, and time between driver mutation and tumor observation. Together, this allows us to estimate the age of the patient at tumor initiation, as well as the age at appearance of a subclonal driver.
                            </p>
                            <p id="p-30">
                             Previous work has computed relative order of driver events [
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ,
                             <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                              21
                             </a>
                             ,
                             <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                              49
                             </a>
                             ], while other studies have given estimates for scaled mutation rates and time of events [
                             <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">
                              24
                             </a>
                             ,
                             <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">
                              32
                             </a>
                             ]. However, we present estimates for absolute, unscaled mutation rates and times, which are easily interpretable and don’t implicitly depend on unknown parameters. We assume that mutations accrue with time, and not only at cell divisions, which simplifies derivations and is supported by recent experimental data [
                             <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">
                              47
                             </a>
                             ].
                            </p>
                            <p id="p-31">
                             For individual CLLs that underwent bulk sequencing at two time points [
                             <a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">
                              27
                             </a>
                             ], we infer growth rates of individual subclones, mutation rate in the tumor, the times when cancer subclones began growing, and the time between driver mutations and the patient’s diagnosis. Our inferences are limited by the relatively low number of mutations present in CLL, as well as sequencing coverage [
                             <a id="xref-ref-27-8" class="xref-bibr" href="#ref-27">
                              27
                             </a>
                             ]. The accuracy of estimates presented here is expected to be even higher in cancer types with more mutations, with whole genome sequencing available, or with higher sequencing coverage. Our methodology is in principle applicable to any cancer type, not only CLL or liquid cancers. We note, however, that in the case of solid tumors, multiple biopsies would potentially be needed to fully account for the existing heterogeneity.
                            </p>
                            <p id="p-32">
                             Our model and derivations assume a fixed mutation rate u after transformation and fixed growth rates of cancer subclones, similar to previous approaches [
                             <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">
                              24
                             </a>
                             ,
                             <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">
                              30
                             </a>
                             ,
                             <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">
                              35
                             </a>
                             ]. Using an exponential model of cancer growth with constant mutation and growth rate to estimate parameters of cancer evolution has its weaknesses: some cancer subclones (such as Clone 1 from Pt. 9) not only do not grow exponentially, they actually decline in absolute cell numbers. Sudden genomic instability events, or a change in cancer mutation and/or growth rate over time could also introduce errors into our parameter inferences. Recent sequencing data points to mutational processes that change over time during cancer evolution [
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              20
                             </a>
                             ,
                             <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                              50
                             </a>
                             ]; incorporating possible changes in the mutation and/or growth rate into the model would require much higher density of sequencing and clinical data [
                             <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">
                              37
                             </a>
                             ].
                            </p>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             Materials and Methods
                            </h2>
                            <div id="sec-10" class="subsection">
                             <h3>
                              Branching process model of tumor evolution
                             </h3>
                             <p id="p-33">
                              We employ a continuous, multi-type branching process model of cancer evolution. Tumor expansion is initiated by a single type-0, or initiated tumor cell. Type-0 cells divide with rate b and die with rate d, yielding a net growth rate of r = b − d. At time t
                              <sub>
                               1
                              </sub>
                              , a single driver mutation is introduced into a randomly selected cell in the type-0 population, founding a new type-1 population of cells. This type-1 population undergoes its own independent branching process. They divide with rate b
                              <sub>
                               1
                              </sub>
                              , die with rate d
                              <sub>
                               1
                              </sub>
                              , and have net growth rate r
                              <sub>
                               1
                              </sub>
                              = b
                              <sub>
                               1
                              </sub>
                              − d
                              <sub>
                               1
                              </sub>
                              . If the driver mutation gives type-1 cells a selective growth advantage over the type-0 population, then r
                              <sub>
                               1
                              </sub>
                              > r. With the ratios of the growth rates denoted as s = r
                              <sub>
                               1
                              </sub>
                              /r, the growth advantage can be quantified as g = (s−1)·100%. In the case of neutral evolution, g = 0. If there is a selective advantage, g > 0. Neutral mutations, or passengers, have no effect on the cell’s fitness, and accrue according to a Poisson process with rate u. We assume an infinite alleles model such that there is no back mutation and an infinite sites model such that every new passenger mutation is unique. Only surviving populations are considered. All derivations below will condition on survival. The type-0 and type-1 populations at time t will be denoted as X
                              <sub>
                               0
                              </sub>
                              (t) and X
                              <sub>
                               1
                              </sub>
                              (t), respectively.
                             </p>
                            </div>
                            <div id="sec-11" class="subsection">
                             <h3>
                              Measurements sufficient to determine evolutionary history
                             </h3>
                             <p id="p-34">
                              We derive estimates for parameters describing the carcinogenic process using measurements taken from two timepoints late in the tumor’s development. We require sequencing of the tumor at the two timepoints, when the tumor is first observed at the unknown time t
                              <sub>
                               1
                              </sub>
                              + t and a specified δ later, at t
                              <sub>
                               1
                              </sub>
                              + t + δ. From these two bulk sequencing measurements, we obtain measurements of α
                              <sub>
                               1
                              </sub>
                              and α
                              <sub>
                               2
                              </sub>
                              , the fraction of cells carrying the driver mutation at t
                              <sub>
                               1
                              </sub>
                              + t and t
                              <sub>
                               1
                              </sub>
                              + t + δ, respectively. In addition, from the bulk sequencing at t
                              <sub>
                               1
                              </sub>
                              + t + δ, we obtain measurements of m, the number of mutations present in the founder type-1 cell, as well as γ, the number of mutations with frequency between the specified f
                              <sub>
                               1
                              </sub>
                              and f
                              <sub>
                               2
                              </sub>
                              . The total population size at these times, M
                              <sub>
                               1
                              </sub>
                              and M
                              <sub>
                               2
                              </sub>
                              , is also measured.
                             </p>
                            </div>
                            <div id="sec-12" class="subsection">
                             <h3>
                              Expected value of γ, number subclonal mutations
                             </h3>
                             <p id="p-35">
                              For a population consisting of a single clone with birth and death rates b and d, the expected number of subclonal mutations present at a frequency larger than f is shown to be [
                              <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">
                               46
                              </a>
                              ]
                              <span class="disp-formula" id="disp-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-12.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-12.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where δ = d/b and ū is the probability that a daughter cell gains a new passenger mutation at cell division. In this paper, we allow mutations to occur at any point in time and consider the absolute mutation rate per cell, u, which is equal to ūb. Then the expected number of subclonal mutations between f
                              <sub>
                               1
                              </sub>
                              and f
                              <sub>
                               2
                              </sub>
                              , 𝔼γ, is
                              <span class="disp-formula" id="disp-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-13.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-13.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-14.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-14.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where r = b − d > 0.
                             </p>
                             <p id="p-36">
                              Now we derive 𝔼γ in the case of clones 0 through k, each clone with growth rate r
                              <sub>
                               i
                              </sub>
                              > 0 and fraction
                              <span class="inline-formula" id="inline-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Each clone i has
                              <span class="inline-formula" id="inline-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              expected subclonal passengers between frequencies f
                              <sub>
                               1
                              </sub>
                              and f
                              <sub>
                               2
                              </sub>
                              . Thus, the total expected number of passengers with frequencies between f
                              <sub>
                               1
                              </sub>
                              and f
                              <sub>
                               2
                              </sub>
                              is
                              <span class="disp-formula" id="disp-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-15.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-15.gif"/>
                                </noscript>
                               </span>
                              </span>
                              For the simplest case we consider, a tumor with a single driver mutation occurring in the initiated tumor population, there is a type-0 population with growth rate r and a type-1 population with growth rate r
                              <sub>
                               1
                              </sub>
                              .
                              <a id="xref-disp-formula-11-1" class="xref-disp-formula" href="#disp-formula-11">
                               Equation (11)
                              </a>
                              reduces to
                              <span class="disp-formula" id="disp-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-16.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-16.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where α is the fraction of cells having the driver mutation.
                             </p>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              Derivation of estimates of evolutionary parameters
                             </h3>
                             <p id="p-37">
                              With the two bulk sequencings at t
                              <sub>
                               1
                              </sub>
                              + t and t
                              <sub>
                               1
                              </sub>
                              + t + δ, we are able to derive estimates for t
                              <sub>
                               1
                              </sub>
                              , t, r, r
                              <sub>
                               1
                              </sub>
                              , and u. First we solve for r and r
                              <sub>
                               1
                              </sub>
                              , based on the estimated cell counts at t
                              <sub>
                               1
                              </sub>
                              + t and t
                              <sub>
                               1
                              </sub>
                              + t + δ. The observed type-i cell count is equated to the expected value of the type-i population size, conditioned on survival. For the type-0 population,
                              <span class="disp-formula" id="disp-formula-13">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-17.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-17.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-14">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-18.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-18.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Proceeding similarly for the type-1 population, we obtain
                              <span class="disp-formula" id="disp-formula-15">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-19.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-19.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-16">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-20.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-20.gif"/>
                                </noscript>
                               </span>
                              </span>
                              The expected value of the first time a population of type-1 cells in a branching process reaches the observed size α
                              <sub>
                               1
                              </sub>
                              M
                              <sub>
                               1
                              </sub>
                              is [
                              <a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">
                               38
                              </a>
                              ]
                              <span class="disp-formula" id="disp-formula-17">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-21.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-21.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-18">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-22.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-22.gif"/>
                                </noscript>
                               </span>
                              </span>
                              which we approximate as
                              <span class="disp-formula" id="disp-formula-19">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-23.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-23.gif"/>
                                </noscript>
                               </span>
                              </span>
                              We make use of two approximations to arrive at (
                              <a id="xref-disp-formula-19-1" class="xref-disp-formula" href="#disp-formula-19">
                               19
                              </a>
                              ). First, we neglect the second term in (
                              <a id="xref-disp-formula-18-1" class="xref-disp-formula" href="#disp-formula-18">
                               18
                              </a>
                              ), which serves as a small correction to the first term. This term will be dominated by the first term as it increases with logarithm of the cancer size. For r
                              <sub>
                               1
                              </sub>
                              = 0.5, α
                              <sub>
                               1
                              </sub>
                              M
                              <sub>
                               1
                              </sub>
                              ∼ 10
                              <sup>
                               11
                              </sup>
                              , and r
                              <sub>
                               1
                              </sub>
                              ≈ b
                              <sub>
                               1
                              </sub>
                              , the second term (1.2) will be only 2.3% of the first term (50.7). For any growth rate, the second term will be 2.3% of the first term. Second, we assume r
                              <sub>
                               1
                              </sub>
                              is similar in magnitude to b
                              <sub>
                               1
                              </sub>
                              .
                             </p>
                             <p id="p-38">
                              With the measurement of γ, the number of subclonal passengers with frequency between f
                              <sub>
                               1
                              </sub>
                              and f
                              <sub>
                               2
                              </sub>
                              , we can estimate the mutation rate u. In the previous section we derive the expected value of γ as
                              <span class="disp-formula" id="disp-formula-20">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-24.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-24.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Using the estimates of r and r
                              <sub>
                               1
                              </sub>
                              from (
                              <a id="xref-disp-formula-15-1" class="xref-disp-formula" href="#disp-formula-15">
                               15
                              </a>
                              ) and (
                              <a id="xref-disp-formula-16-1" class="xref-disp-formula" href="#disp-formula-16">
                               16
                              </a>
                              ), and the measured value of γ from the second bulk sequencing,
                              <a id="xref-disp-formula-20-1" class="xref-disp-formula" href="#disp-formula-20">
                               equation (20)
                              </a>
                              can be solved for the mutation rate u,
                              <span class="disp-formula" id="disp-formula-21">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-25.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-25.gif"/>
                                </noscript>
                               </span>
                              </span>
                              When estimating mutation rate for the CLL patients from Ref. [
                              <a id="xref-ref-27-9" class="xref-bibr" href="#ref-27">
                               27
                              </a>
                              ], for which there is bulk sequencing at two or more timepoints, we average the mutation rate calculated at each of these timepoints. (
                              <a id="xref-disp-formula-21-1" class="xref-disp-formula" href="#disp-formula-21">
                               21
                              </a>
                              ) is applied for each timepoint with the respective CCFs and observed γ values for each timepoint.
                             </p>
                             <p id="p-39">
                              To derive the maximum likelihood estimates of t
                              <sub>
                               1
                              </sub>
                              , we consider the likelihood function P (m|t
                              <sub>
                               1
                              </sub>
                              ). The number of passenger mutations present in the founder type-1 cell that appeared at time t
                              <sub>
                               1
                              </sub>
                              is a Poisson process with rate u. Thus,
                              <span class="disp-formula" id="disp-formula-22">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-26.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-26.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Maximizing the logarithm of the likelihood function with respect to t
                              <sub>
                               1
                              </sub>
                              yields a MLE for t
                              <sub>
                               1
                              </sub>
                              in terms of estimated or measured quantities:
                              <span class="disp-formula" id="disp-formula-23">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-27.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-27.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                            </div>
                            <div id="sec-14" class="subsection">
                             <h3>
                              Estimating number of unobserved subclonal mutations from sequencing data
                             </h3>
                             <p id="p-40">
                              When sequencing data is post-processed by filtering out any mutations with L or fewer variant reads, the number of mutations between f
                              <sub>
                               1
                              </sub>
                              and f
                              <sub>
                               2
                              </sub>
                              will likely be underestimated if 2L/(Rp) > f
                              <sub>
                               1
                              </sub>
                              , where R is average sequencing coverage and p is tumor purity. Define γ
                              <sub>
                               obs
                              </sub>
                              as the observed number of mutations between frequencies f
                              <sub>
                               1
                              </sub>
                              and f
                              <sub>
                               2
                              </sub>
                              , after post-processing has been performed that filtered out any mutations with L or fewer variant reads. The expected number of subclonal mutations between frequencies f
                              <sub>
                               1
                              </sub>
                              and x is given by
                              <span class="disp-formula" id="disp-formula-24">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-28.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-28.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where c is a constant that will vary depending on the patient and sample. It can be fit on the sequencing data by noting
                              <span class="disp-formula" id="disp-formula-25">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-29.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-29.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-26">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-30.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-30.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Therefore, c can estimated from the sequencing data as
                              <span class="disp-formula" id="disp-formula-27">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-31.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-31.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Then, we can estimate γ as
                              <span class="disp-formula" id="disp-formula-28">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-32.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-32.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                            </div>
                            <div id="sec-15" class="subsection">
                             <h3>
                              Number of passengers reaching fixation after t
                              <sub>
                               1
                              </sub>
                             </h3>
                             <p id="p-41">
                              We estimate the number of passengers that occurred after t
                              <sub>
                               1
                              </sub>
                              and reached fixation in the type-1 population in order to adjust the m
                              <sub>
                               obs
                              </sub>
                              mutation count. From [
                              <a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">
                               46
                              </a>
                              ], when mutations occur at cell division, the expected number of clonal passengers is δū/(1 − δ). ū is the probability that a daughter cell gains a new passenger mutation at cell division, so the mutation rate is u = ūb
                              <sub>
                               1
                              </sub>
                              . For the type-1 population, δ = d
                              <sub>
                               1
                              </sub>
                              /b
                              <sub>
                               1
                              </sub>
                              < 1. When mutations accrue over time, and not only at divisions, the expected number of clonal passengers is thus
                              <span class="disp-formula" id="disp-formula-29">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-33.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-33.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Similarly, for a clone i, the expected number of passengers that occur after time t
                              <sub>
                               i
                              </sub>
                              and reach fixation is
                              <span class="disp-formula" id="disp-formula-30">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-34.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-34.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where r
                              <sub>
                               i
                              </sub>
                              = b
                              <sub>
                               i
                              </sub>
                              − d
                              <sub>
                               i
                              </sub>
                              > 0.
                             </p>
                            </div>
                            <div id="sec-16" class="subsection">
                             <h3>
                              Simulation of tumor evolution and sequencing data
                             </h3>
                             <p id="p-42">
                              To assess the accuracy of the analytic results, we perform a continuous time Monte Carlo simulation to model tumor evolution and collection of sequencing data with an implementation of the Gillespie algorithm [
                              <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                               51
                              </a>
                              ]. Simulations are written in C/C++.
                             </p>
                             <p id="p-43">
                              The type-j population has division rate b
                              <sub>
                               j
                              </sub>
                              , death rate d
                              <sub>
                               j
                              </sub>
                              , and mutation rate u. Mutations can occur at any point of the cell cycle, not just during division. z
                              <sub>
                               n
                              </sub>
                              is the number of type-j cells with passenger n as their most recent passenger mutation. The type-0 population is initiated with a single cell at time 0, and the type-1 population is initiated with a single cell at time t
                              <sub>
                               1
                              </sub>
                              . Let a be the vector recording the ancestor of new mutations. Element a
                              <sub>
                               i
                              </sub>
                              is the subclonal ancestor of the ith passenger mutation. Repeat 1-4 while time is less than t
                              <sub>
                               1
                              </sub>
                              + t + δ.
                             </p>
                             <ol class="list-ord" id="list-1">
                              <li id="list-item-1">
                               <p id="p-44">
                                Set Γ = N
                                <sub>
                                 j
                                </sub>
                                (b
                                <sub>
                                 j
                                </sub>
                                + d
                                <sub>
                                 j
                                </sub>
                                + u). Time increment to next event time is randomly sampled from Exp[Γ].
                               </p>
                               <ul class="list-unord" id="list-2">
                                <li id="list-item-2">
                                 <p id="p-45">
                                  For type-0, if time is greater than or equal to t
                                  <sub>
                                   1
                                  </sub>
                                  for first time, randomly select type-0 subclone i to have driver mutation, remove one cell from type-0 population count, and set N
                                  <sub>
                                   1
                                  </sub>
                                  = 1. Record the true value of m, the number of passenger mutations present in the founder type-1 cell.
                                 </p>
                                </li>
                               </ul>
                              </li>
                              <li id="list-item-3">
                               <p id="p-46">
                                Randomly select cell, with most recent passenger mutation i, to have the event.
                               </p>
                              </li>
                              <li id="list-item-4">
                               <p id="p-47">
                                Determine which type of event and update population and mutation frequencies. Sample Y from Uniform[0, Γ] to determine event type:
                               </p>
                               <ol class="list-romanlower" id="list-3">
                                <li id="list-item-5">
                                 <p id="p-48">
                                  y ∈ (0, b
                                  <sub>
                                   j
                                  </sub>
                                  ) → birth. N
                                  <sub>
                                   j
                                  </sub>
                                  += 1, z
                                  <sub>
                                   i
                                  </sub>
                                  += 1.
                                 </p>
                                </li>
                                <li id="list-item-6">
                                 <p id="p-49">
                                  y ∈ (b
                                  <sub>
                                   j
                                  </sub>
                                  , b
                                  <sub>
                                   j
                                  </sub>
                                  + d
                                  <sub>
                                   j
                                  </sub>
                                  ) → death. N
                                  <sub>
                                   j
                                  </sub>
                                  −= 1, z
                                  <sub>
                                   i
                                  </sub>
                                  −= 1.
                                 </p>
                                </li>
                                <li id="list-item-7">
                                 <p id="p-50">
                                  y ∈ (b
                                  <sub>
                                   j
                                  </sub>
                                  + d
                                  <sub>
                                   j
                                  </sub>
                                  , b
                                  <sub>
                                   j
                                  </sub>
                                  + d
                                  <sub>
                                   j
                                  </sub>
                                  + u) → passenger mutation. Suppose it’s the kth passenger, z
                                  <sub>
                                   i
                                  </sub>
                                  −= 1, z
                                  <sub>
                                   k
                                  </sub>
                                  = 1. Update ancestor: a
                                  <sub>
                                   k
                                  </sub>
                                  = i.
                                 </p>
                                </li>
                               </ol>
                              </li>
                              <li id="list-item-8">
                               <p id="p-51">
                                For type-0, if time is less than t
                                <sub>
                                 1
                                </sub>
                                and population goes extinct, restart simulation. For type-1, if time is greater than t
                                <sub>
                                 1
                                </sub>
                                and population goes extinct, restart type-1 simulation at t
                                <sub>
                                 1
                                </sub>
                                with a single cell.
                               </p>
                              </li>
                              <li id="list-item-9">
                               <p id="p-52">
                                Reindex to remove extinct passenger mutations, and traverse back through ancestor vector
                                <strong>
                                 a
                                </strong>
                                to sum total number of cells with each passenger.
                               </p>
                              </li>
                             </ol>
                             <p id="p-53">
                              Measurements are taken at bulk sequencing times t
                              <sub>
                               1
                              </sub>
                              + t and t
                              <sub>
                               1
                              </sub>
                              + t + δ. If time is greater than or equal to t
                              <sub>
                               1
                              </sub>
                              + t, we measure M
                              <sub>
                               1
                              </sub>
                              = N
                              <sub>
                               0
                              </sub>
                              + N
                              <sub>
                               1
                              </sub>
                              and α
                              <sub>
                               1
                              </sub>
                              = N
                              <sub>
                               1
                              </sub>
                              /(N
                              <sub>
                               0
                              </sub>
                              + N
                              <sub>
                               1
                              </sub>
                              ). Then an additional bulk sequencing measurement is taken at the final time t
                              <sub>
                               1
                              </sub>
                              + t + δ, where we measure M
                              <sub>
                               2
                              </sub>
                              = N
                              <sub>
                               0
                              </sub>
                              + N
                              <sub>
                               1
                              </sub>
                              and α
                              <sub>
                               2
                              </sub>
                              = N
                              <sub>
                               1
                              </sub>
                              /(N
                              <sub>
                               0
                              </sub>
                              + N
                              <sub>
                               1
                              </sub>
                              ). At t
                              <sub>
                               1
                              </sub>
                              + t + δ, we measure γ, the number of mutations with frequency between f
                              <sub>
                               1
                              </sub>
                              and f
                              <sub>
                               2
                              </sub>
                              .
                             </p>
                             <p id="p-54">
                              To measure m
                              <sub>
                               obs
                              </sub>
                              , the observed number of passengers in the founder type-1 cell, we count the number of passengers present in all type-1 cells. We also save the true value of m.
                             </p>
                             <p id="p-55">
                              For when we calculate a percent error of corrected and observed γ values in
                              <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                               Figure 3d
                              </a>
                              and Supplementary Figure 3b, we simulate sequencing data by sampling from the mutation frequencies obtained in the Monte Carlo simulation, outlined above, using the approach of [
                              <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">
                               35
                              </a>
                              ]. Define average sequencing coverage as R, number of cells at time of sequencing as M, Z
                              <sub>
                               i
                              </sub>
                              as the number of cells with mutation i, R
                              <sub>
                               i
                              </sub>
                              as read coverage, and χ
                              <sub>
                               i
                              </sub>
                              as the true mutation frequency from Monte Carlo simulation. For each saved Monte Carlo simulation run, repeat the following 100 times:
                             </p>
                             <ol class="list-ord" id="list-4">
                              <li id="list-item-10">
                               <p id="p-56">
                                Generate read coverage: R
                                <sub>
                                 i
                                </sub>
                                ∼ Binomial[M, R/M ]
                               </p>
                              </li>
                              <li id="list-item-11">
                               <p id="p-57">
                                Generate number of cells carrying mutation i: Z
                                <sub>
                                 i
                                </sub>
                                ∼ Binomial[R
                                <sub>
                                 i
                                </sub>
                                , χ
                                <sub>
                                 i
                                </sub>
                                /2]
                               </p>
                              </li>
                              <li id="list-item-12">
                               <p id="p-58">
                                Post-processing. If there are L = 2 or fewer variant reads, discard mutation.
                               </p>
                              </li>
                              <li id="list-item-13">
                               <p id="p-59">
                                Measure γ
                                <sub>
                                 obs
                                </sub>
                                , the observed number of subclonal mutations between frequencies f
                                <sub>
                                 1
                                </sub>
                                and f
                                <sub>
                                 2
                                </sub>
                                : γ
                                <sub>
                                 obs
                                </sub>
                                = Σ
                                <sub>
                                 i
                                </sub>
                                I(f
                                <sub>
                                 1
                                </sub>
                                ≤ 2Z
                                <sub>
                                 i
                                </sub>
                                /R ≤ f
                                <sub>
                                 2
                                </sub>
                                , Z
                                <sub>
                                 i
                                </sub>
                                > L)
                               </p>
                              </li>
                              <li id="list-item-14">
                               <p id="p-60">
                                Calculate the truth, γ
                                <sub>
                                 true
                                </sub>
                                , from the true mutation frequencies: γ
                                <sub>
                                 true
                                </sub>
                                = Σ
                                <sub>
                                 i
                                </sub>
                                I(f
                                <sub>
                                 1
                                </sub>
                                ≤ χ
                                <sub>
                                 i
                                </sub>
                                ≤ f
                                <sub>
                                 2
                                </sub>
                                )
                               </p>
                              </li>
                             </ol>
                            </div>
                            <div id="sec-17" class="subsection">
                             <h3>
                              Parameter values for simulations
                             </h3>
                             <p id="p-61">
                              For the simulation we consider three parameter sets corresponding to three modes of tumor evolution: a fast growing tumor, slow growing tumor, and tumor with no cell death. For each parameter regime we have a low and high mutation rate. Mutation rate parameter values lie within observed genome wide point mutation rates per day [
                              <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">
                               52
                              </a>
                              ]. For the fast growing tumor b = b
                              <sub>
                               1
                              </sub>
                              = 0.25, d = 0.18, d
                              <sub>
                               1
                              </sub>
                              = 0.11, t
                              <sub>
                               1
                              </sub>
                              = 70, t = 50, δ = 20, and u = 1, 3. For the slow growing tumor b = 0.25, b
                              <sub>
                               1
                              </sub>
                              = 0.25, d = 0.225, d
                              <sub>
                               1
                              </sub>
                              = 0.2125, t
                              <sub>
                               1
                              </sub>
                              = 180, t = 135, δ = 45, and u = 1, 5. For the parameter regime with no cell death b = 0.25, b
                              <sub>
                               1
                              </sub>
                              = 0.375, d = d
                              <sub>
                               1
                              </sub>
                              = 0.0, t
                              <sub>
                               1
                              </sub>
                              = 23, t = 17, δ = 6, and u = 1, 10. The fast growing tumor dynamics are from [
                              <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ]. The slower growing tumor parameter regime has a reduced net growth of r = 0.025, compared to the fast growing tumor’s net growth rate of r = 0.07.
                             </p>
                            </div>
                            <div id="sec-18" class="subsection">
                             <h3>
                              Subclonal reconstruction of CLL sequencing data
                             </h3>
                             <p id="p-62">
                              The sequencing data from all CLLs analyzed is from Ref. [
                              <a id="xref-ref-27-10" class="xref-bibr" href="#ref-27">
                               27
                              </a>
                              ], Supplementary Tables 2-4. As in that publication, we use PhylogicNDT [
                              <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">
                               43
                              </a>
                              ] to perform subclonal reconstruction. We run the Cluster and BuildTree modules of PhylogicNDT on the longitudinal mutation data from Supplementary Table 3 of [
                              <a id="xref-ref-27-11" class="xref-bibr" href="#ref-27">
                               27
                              </a>
                              ], using mutation alternate/reference counts, copy number, and tumor purity at all pre-treatment time points. Then for each patient, PhylogicNDT outputs a clonal reconstruction, which includes a phylogenetic tree of the subclones and posterior distributions of subclone CCFs. Additionally, it clusters mutations and assigns them to clones. We directly use subclone assignments and posteriors generated from PhylogicNDT. In our analysis we focus on estimating timing and growth rates of macroscopic subclones whose CCFs are greater than 20% for at least one pre-treatment timepoint.
                             </p>
                            </div>
                            <div id="sec-19" class="subsection">
                             <h3>
                              Accounting for uncertainties in subclone frequencies and growth rates
                             </h3>
                             <p id="p-63">
                              Our estimates for parameters of cancer evolution require as input the information on the number of subclonal populations in the tumor, their CCFs and their phylogenetic relationships. In order to obtain this information, we use PhylogicNDT [
                              <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">
                               43
                              </a>
                              ], which performs subclonal reconstruction of longitudinal cancer sequencing data. The uncertainty in subclone CCFs reported by PhylogicNDT affects our estimates for subclone growth rates, which in turn affect the estimates of mutation rate and and time t between driver(s) and diagnosis. We account for this uncertainty by drawing from the CCF posterior distributions that are output by PhylogicNDT. Using these sampled CCF values, we then calculate growth rates, mutation rate u, and time t between driver(s) and diagnosis, thereby generating confidence intervals for these parameters due to CCF uncertainty.
                             </p>
                             <p id="p-64">
                              To estimate subclonal growth rates, we fit an exponential growth curve to subclonal sizes measured at two or more time points. This regression yields fitted values for each clone’s growth rate and age. To account for uncertainty in the curve fit (in the case of more than two longitudinal samples), we sample the growth rates and age of clone from a bivariate normal distribution with mean equal to the fitted parameters and variance equal to the covariance matrix of the fitted parameters. In line with recent findings [
                              <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">
                               53
                              </a>
                              ], we found that sometimes the estimated growth rate is smaller than minimal possible growth rate necessary to reach the observed clone size. In that case, for calculating mutation rate, time of the driver(s), and time between driver(s) and diagnosis we use the minimal growth rate.
                             </p>
                            </div>
                            <div id="sec-20" class="subsection">
                             <h3>
                              Accounting for model uncertainty
                             </h3>
                             <p id="p-65">
                              The largest source of model uncertainty is the Poisson process for how mutations accumulate, which is used to estimate the time t
                              <sub>
                               1
                              </sub>
                              of the driver mutation. In the simulation experiments, the time t
                              <sub>
                               1
                              </sub>
                              had the largest error and variation (
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              ). The estimate for t
                              <sub>
                               1
                              </sub>
                              depends on the m mutations present in all cells in the driver subclone. The observed m is a single random sample from a Poisson distribution. To account for the uncertainty in t
                              <sub>
                               1
                              </sub>
                              arising from m in the CLLs analyzed, we sample t
                              <sub>
                               i
                              </sub>
                              from the posterior distributions P (t
                              <sub>
                               1
                              </sub>
                              |m). This source of model uncertainty due to the Poisson process will be most significant for cancers like CLL with a smaller number of mutations.
                             </p>
                             <p id="p-66">
                              The time t between driver mutation and diagnosis (t) is a random variable due to the stochasticity of cancer cell growth, and will naturally have a certain amount of variation. Time between driver event and diagnosis in a branching process follows a Gumbel distribution [
                              <a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">
                               38
                              </a>
                              ], and will have a constant variance. The mean, however, will increase with the logarithm of the cancer cell counts, which for the CLLs analyzed are ∼ 10
                              <sup>
                               11
                              </sup>
                              . The simulations of cancer evolution grow to smaller tumor sizes (∼ 10
                              <sup>
                               5
                              </sup>
                              ) and, as a result, the estimate for t has a significant amount of uncertainty (
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              ). However, for time scales necessary to generate a tumor, the estimate for t will be quite accurate. For commonly observed tumor sizes, the stochastic fluctuations in the time for the cancer to reach that size will be smaller relative to the magnitude of the time. For a cancer with cell count ∼ 10
                              <sup>
                               11
                              </sup>
                              , the standard deviation of the time t will be less than 5% of its expected value.
                             </p>
                            </div>
                            <div id="sec-21" class="subsection">
                             <h3>
                              Tumor with two nested driver subclones
                             </h3>
                             <p id="p-67">
                              Here we consider the case where there are two nested driver subclones (Fig. S4a). “Nested” means that all cells carrying the second driver mutation also carry the first. Type-0, or initiated tumor, cells have birth rate b
                              <sub>
                               0
                              </sub>
                              , death rate d
                              <sub>
                               0
                              </sub>
                              , and net growth rate r
                              <sub>
                               0
                              </sub>
                              = b
                              <sub>
                               0
                              </sub>
                              − d
                              <sub>
                               0
                              </sub>
                              . Type 1 cells, which only have the first driver, have birth rate b
                              <sub>
                               1
                              </sub>
                              , death rate d
                              <sub>
                               1
                              </sub>
                              , and net growth rate r
                              <sub>
                               1
                              </sub>
                              = b
                              <sub>
                               1
                              </sub>
                              − d
                              <sub>
                               1
                              </sub>
                              . Type-2 cells, which carry both drivers, have birth rate b
                              <sub>
                               2
                              </sub>
                              , death rate d
                              <sub>
                               2
                              </sub>
                              , and net growth rate r
                              <sub>
                               2
                              </sub>
                              = b
                              <sub>
                               2
                              </sub>
                              − d
                              <sub>
                               2
                              </sub>
                              . The first driver occurred in a type-0 cell at time t
                              <sub>
                               1
                              </sub>
                              . The second driver occurred in a type-1 cell at
                              <span class="inline-formula" id="inline-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . The mutation rate u is the same for all subclones.
                             </p>
                             <p id="p-68">
                              At times
                              <span class="inline-formula" id="inline-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , the tumor is bulk sequenced. The bulk sequencing allows the measurement of the fraction of cells with driver 1 at time
                              <span class="inline-formula" id="inline-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-6.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-6.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , α
                              <sub>
                               1
                              </sub>
                              ; the fraction of cells with driver 2 at
                              <span class="inline-formula" id="inline-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ; fraction of cells with driver 1 at time
                              <span class="inline-formula" id="inline-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ; the fraction of cells with driver 2 at
                              <span class="inline-formula" id="inline-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ; and the observed number of subclonal passenger mutations between frequencies f
                              <sub>
                               1
                              </sub>
                              and f
                              <sub>
                               2
                              </sub>
                              , γ
                              <sub>
                               obs
                              </sub>
                              . Note that the fraction of the population that is a type-1 cell at the two times is α
                              <sub>
                               1
                              </sub>
                              − α
                              <sub>
                               2
                              </sub>
                              and β
                              <sub>
                               1
                              </sub>
                              − β
                              <sub>
                               2
                              </sub>
                              . The fraction of type-0 cells at the two bulk sequencing timepoints are 1 − α
                              <sub>
                               1
                              </sub>
                              and 1 − β
                              <sub>
                               1
                              </sub>
                              . The number of total cells at bulk sequencing timepoints are M
                              <sub>
                               1
                              </sub>
                              and M
                              <sub>
                               2
                              </sub>
                              . Equating the estimated cell counts to the expected value of the type-i population size X
                              <sub>
                               i
                              </sub>
                              , conditioned on survival,
                              <span class="disp-formula" id="disp-formula-31">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-35.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-35.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-32">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-36.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-36.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-33">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-37.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-37.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-34">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-38.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-38.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Solving the above equations for r
                              <sub>
                               i
                              </sub>
                              , we obtain the growth rate estimates:
                              <span class="disp-formula" id="disp-formula-35">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-39.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-39.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-36">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-40.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-40.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-37">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-41.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-41.gif"/>
                                </noscript>
                               </span>
                              </span>
                              The expected value of the first time a population of type-2 cells in a branching process reaches the observed size α
                              <sub>
                               2
                              </sub>
                              M
                              <sub>
                               1
                              </sub>
                              [
                              <a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">
                               38
                              </a>
                              ],
                              <span class="disp-formula" id="disp-formula-38">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-42.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-42.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-39">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-43.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-43.gif"/>
                                </noscript>
                               </span>
                              </span>
                              can be approximated as
                              <span class="disp-formula" id="disp-formula-40">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-44.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-44.gif"/>
                                </noscript>
                               </span>
                              </span>
                              We make use of two approximations to arrive at (
                              <a id="xref-disp-formula-40-1" class="xref-disp-formula" href="#disp-formula-40">
                               40
                              </a>
                              ). First, we neglect the second term in (
                              <a id="xref-disp-formula-39-1" class="xref-disp-formula" href="#disp-formula-39">
                               39
                              </a>
                              ), which serves as a small correction to the first term. Second, we assume r
                              <sub>
                               2
                              </sub>
                              is similar in magnitude to b
                              <sub>
                               2
                              </sub>
                              .
                             </p>
                             <p id="p-69">
                              By (
                              <a id="xref-disp-formula-11-2" class="xref-disp-formula" href="#disp-formula-11">
                               11
                              </a>
                              ),
                              <span class="disp-formula" id="disp-formula-41">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-45.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-45.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Using the estimates for r
                              <sub>
                               0
                              </sub>
                              , r
                              <sub>
                               1
                              </sub>
                              , and r
                              <sub>
                               2
                              </sub>
                              from (
                              <a id="xref-disp-formula-35-1" class="xref-disp-formula" href="#disp-formula-35">
                               35
                              </a>
                              )-(
                              <a id="xref-disp-formula-37-1" class="xref-disp-formula" href="#disp-formula-37">
                               37
                              </a>
                              ), and setting (
                              <a id="xref-disp-formula-41-1" class="xref-disp-formula" href="#disp-formula-41">
                               41
                              </a>
                              ) equal to the value of γ obtained from (
                              <a id="xref-disp-formula-28-1" class="xref-disp-formula" href="#disp-formula-28">
                               28
                              </a>
                              ) and the second bulk sequencing, u can be estimated:
                              <span class="disp-formula" id="disp-formula-42">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-46.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-46.gif"/>
                                </noscript>
                               </span>
                              </span>
                              When estimating mutation rate for the CLL patients from Ref. [
                              <a id="xref-ref-27-12" class="xref-bibr" href="#ref-27">
                               27
                              </a>
                              ], for which there is bulk sequencing at two or more timepoints, we average the mutation rate calculated at each of these timepoints. (
                              <a id="xref-disp-formula-42-1" class="xref-disp-formula" href="#disp-formula-42">
                               42
                              </a>
                              ) is applied for each timepoint with the respective CCFs and observed γ values for each timepoint.
                             </p>
                             <p id="p-70">
                              Every type-1 cell carries the m
                              <sub>
                               1
                              </sub>
                              passenger mutations that were present in the original type-1 cell when the first driver mutation mutation occurred at t
                              <sub>
                               1
                              </sub>
                              . Similarly, every type-2 cell carries the m
                              <sub>
                               2
                              </sub>
                              passengers that were present in the founder type-2 cell when the second driver mutation occurred at t
                              <sub>
                               2
                              </sub>
                              . Note, none of the m
                              <sub>
                               1
                              </sub>
                              mutations are counted towards m
                              <sub>
                               2
                              </sub>
                              . Now we consider the likelihood function
                              <span class="disp-formula" id="disp-formula-43">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-47.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-47.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-44">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-48.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-48.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-45">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-49.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-49.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Now, maximizing the logarithm of (
                              <a id="xref-disp-formula-45-1" class="xref-disp-formula" href="#disp-formula-45">
                               45
                              </a>
                              ) with respect to t
                              <sub>
                               1
                              </sub>
                              and
                              <span class="inline-formula" id="inline-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-10.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-10.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ,
                              <span class="disp-formula" id="disp-formula-46">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-50.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-50.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-47">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-51.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-51.gif"/>
                                </noscript>
                               </span>
                              </span>
                              The number of passengers present in the founder type-i cell cannot be directly observed, but we can measure m
                              <sub>
                               i obs
                              </sub>
                              , the number of passengers present in all type-i cells. An expected u/r
                              <sub>
                               1
                              </sub>
                              passengers occurring after t
                              <sub>
                               1
                              </sub>
                              in type-1 cells and reaching fixation in the type-1 subclone will be incorrectly included in m
                              <sub>
                               1 obs
                              </sub>
                              , rather than in m
                              <sub>
                               2 obs
                              </sub>
                              (see Methods). Similarly, an expected u/r
                              <sub>
                               2
                              </sub>
                              passengers occurring after t
                              <sub>
                               2
                              </sub>
                              in type-2 cells and reaching fixation in the type-2 subclone will be incorrectly included in m
                              <sub>
                               2 obs
                              </sub>
                              . Thus,
                              <span class="disp-formula" id="disp-formula-48">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-52.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-52.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-49">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-53.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-53.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                            </div>
                            <div id="sec-22" class="subsection">
                             <h3>
                              Tumor with two sibling driver subclones
                             </h3>
                             <p id="p-71">
                              Here we consider a tumor with two “sibling” driver mutations (Fig. S4b). Sibling driver mutations are drivers that occur in separate subclones. In this case, cells are either initiated tumor cell (type-0), carry driver 1 (type-1), or carry driver 2 (type-2). No cells contain both drivers. Driver 1 occurred in a type-0 cell at time t
                              <sub>
                               1
                              </sub>
                              . Driver 2 occurred in a type-0 cell at t
                              <sub>
                               2
                              </sub>
                              . Type-0 cells have birth rate b
                              <sub>
                               0
                              </sub>
                              , death rate d
                              <sub>
                               0
                              </sub>
                              , and net growth rate r
                              <sub>
                               0
                              </sub>
                              = b
                              <sub>
                               0
                              </sub>
                              − d
                              <sub>
                               0
                              </sub>
                              . Type-1 cells, which carry driver 1, have birth rate b
                              <sub>
                               1
                              </sub>
                              , death rate d
                              <sub>
                               1
                              </sub>
                              , and net growth rate r
                              <sub>
                               1
                              </sub>
                              = b
                              <sub>
                               1
                              </sub>
                              − d
                              <sub>
                               1
                              </sub>
                              . Type-2 cells, which carry driver 2, have birth rate b
                              <sub>
                               2
                              </sub>
                              , death rate d
                              <sub>
                               2
                              </sub>
                              , and net growth rate r
                              <sub>
                               2
                              </sub>
                              = b
                              <sub>
                               2
                              </sub>
                              − d
                              <sub>
                               2
                              </sub>
                              . The mutation rate u is the same for all subclones.
                             </p>
                             <p id="p-72">
                              Suppose time τ
                              <sub>
                               i
                              </sub>
                              elapses between driver mutation i and tumor observation. Bulk sequencing of the tumor is performed at t
                              <sub>
                               1
                              </sub>
                              + τ
                              <sub>
                               1
                              </sub>
                              (or equivalently t
                              <sub>
                               2
                              </sub>
                              + τ
                              <sub>
                               2
                              </sub>
                              ), and a known δ later. Sequencing the tumor allows the measurement of the fraction of cells with driver 1 at the first sequencing, α
                              <sub>
                               1
                              </sub>
                              ; the fraction of cells with driver 2 at the first sequencing, α
                              <sub>
                               2
                              </sub>
                              ; fraction of cells with driver 1 at the second sequencing, β
                              <sub>
                               1
                              </sub>
                              ; the fraction of cells with driver 2 at the second sequencing, β
                              <sub>
                               2
                              </sub>
                              ; and the number of subclonal passenger mutations between frequencies f
                              <sub>
                               1
                              </sub>
                              and f
                              <sub>
                               2
                              </sub>
                              , γ. The fraction of type-0 cells at the two bulk sequencing timepoints are 1 − α
                              <sub>
                               1
                              </sub>
                              − α
                              <sub>
                               2
                              </sub>
                              and 1 − β
                              <sub>
                               1
                              </sub>
                              − β
                              <sub>
                               2
                              </sub>
                              . The number of total cells at the two sequencing timepoints are M
                              <sub>
                               1
                              </sub>
                              and M
                              <sub>
                               2
                              </sub>
                              .
                             </p>
                             <p id="p-73">
                              Equating the estimated cell counts to the expected value of the type-i population size X
                              <sub>
                               i
                              </sub>
                              , conditioned on survival,
                              <span class="disp-formula" id="disp-formula-50">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-54.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-54.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-51">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-55.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-55.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-52">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-56.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-56.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-53">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-57.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-57.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Solving the above equations for r
                              <sub>
                               i
                              </sub>
                              , we obtain
                              <span class="disp-formula" id="disp-formula-54">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-58.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-58.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-55">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-59.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-59.gif"/>
                                </noscript>
                               </span>
                              </span>
                              The expected value of the first time a population of type-i cells in a branching process reaches the observed size α
                              <sub>
                               i
                              </sub>
                              M
                              <sub>
                               1
                              </sub>
                              is [
                              <a id="xref-ref-38-7" class="xref-bibr" href="#ref-38">
                               38
                              </a>
                              ]
                              <span class="disp-formula" id="disp-formula-56">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-60.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-60.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-57">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-61.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-61.gif"/>
                                </noscript>
                               </span>
                              </span>
                              which we approximate as
                              <span class="disp-formula" id="disp-formula-58">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-62.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-62.gif"/>
                                </noscript>
                               </span>
                              </span>
                              We use two approximations to arrive at (
                              <a id="xref-disp-formula-58-1" class="xref-disp-formula" href="#disp-formula-58">
                               58
                              </a>
                              ). We neglect the second term in (
                              <a id="xref-disp-formula-57-1" class="xref-disp-formula" href="#disp-formula-57">
                               57
                              </a>
                              ), which serves as a small correction to the first term. Second, we assume r
                              <sub>
                               i
                              </sub>
                              is similar in magnitude to b
                              <sub>
                               i
                              </sub>
                              .
                             </p>
                             <p id="p-74">
                              By (
                              <a id="xref-disp-formula-11-3" class="xref-disp-formula" href="#disp-formula-11">
                               11
                              </a>
                              ),
                              <span class="disp-formula" id="disp-formula-59">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-63.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-63.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Using the estimates for r
                              <sub>
                               0
                              </sub>
                              , r
                              <sub>
                               1
                              </sub>
                              , and r
                              <sub>
                               2
                              </sub>
                              from (
                              <a id="xref-disp-formula-54-1" class="xref-disp-formula" href="#disp-formula-54">
                               54
                              </a>
                              ) and (
                              <a id="xref-disp-formula-55-1" class="xref-disp-formula" href="#disp-formula-55">
                               55
                              </a>
                              ), and setting (
                              <a id="xref-disp-formula-59-1" class="xref-disp-formula" href="#disp-formula-59">
                               59
                              </a>
                              ) equal to the value of γ obtained from (
                              <a id="xref-disp-formula-28-2" class="xref-disp-formula" href="#disp-formula-28">
                               28
                              </a>
                              ) and the second bulk sequencing, u can be estimated:
                              <span class="disp-formula" id="disp-formula-60">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-64.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-64.gif"/>
                                </noscript>
                               </span>
                              </span>
                              When estimating mutation rate for the CLL patients from Ref. [
                              <a id="xref-ref-27-13" class="xref-bibr" href="#ref-27">
                               27
                              </a>
                              ], for which there is bulk sequencing at two or more timepoints, we average the mutation rate calculated at each of these timepoints. (
                              <a id="xref-disp-formula-60-1" class="xref-disp-formula" href="#disp-formula-60">
                               60
                              </a>
                              ) is applied for each timepoint with the respective CCFs and observed γ values for each timepoint.
                             </p>
                             <p id="p-75">
                              Every type-1 cell carries the m
                              <sub>
                               1
                              </sub>
                              passenger mutations that were present in the original type-1 cell when the first driver mutation mutation occurred at t
                              <sub>
                               1
                              </sub>
                              . Similarly, every type-2 cell carries the m
                              <sub>
                               2
                              </sub>
                              passengers that were present in the founder type-2 cell when the second driver mutation occurred at t
                              <sub>
                               2
                              </sub>
                              . We assume that m
                              <sub>
                               1
                              </sub>
                              and m
                              <sub>
                               2
                              </sub>
                              don’t contain any shared mutations. In the CLL dataset we use, this is true. We consider the likelihood function P (m
                              <sub>
                               1
                              </sub>
                              , m
                              <sub>
                               2
                              </sub>
                              |t
                              <sub>
                               1
                              </sub>
                              , t
                              <sub>
                               2
                              </sub>
                              )
                              <span class="disp-formula" id="disp-formula-61">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-65.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-65.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-62">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-66.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-66.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Maximizing the logarithm of (
                              <a id="xref-disp-formula-62-1" class="xref-disp-formula" href="#disp-formula-62">
                               62
                              </a>
                              ) with respect to t
                              <sub>
                               1
                              </sub>
                              and t
                              <sub>
                               2
                              </sub>
                              yields the maximum likelihood estimates:
                              <span class="disp-formula" id="disp-formula-63">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-67.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-67.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-64">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-68.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-68.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Using the same approach as in the case of a single driver, we obtain the corrections for the observed number of mutations present in all cells of each subclone:
                              <span class="disp-formula" id="disp-formula-65">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-69.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-69.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-66">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-70.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-70.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                            </div>
                            <div id="sec-23" class="subsection">
                             <h3>
                              Fully generalized estimates for any phylogeny of k drivers
                             </h3>
                             <p id="p-76">
                              Here we derive estimates for a completely general tumor phylogeny. Suppose a tumor has k driver mutations. In this general case, define a type-i cell as a cell where its most recent driver mutation was driver i. Note that a type-i cell can have between 0 and k − 1 other driver mutations. A phylogenetic reconstruction of the k driver mutations is necessary for the completely general case. From this phylogenetic tree, the ancestor of each subclone can be obtained. Define the function a(i) as the ancestor of the type-i population. That is, if all driver mutations contained in the type-i population are ordered, a(i) gives the driver mutation that occurred prior to i. Define t
                              <sub>
                               i
                              </sub>
                              as the time between when driver i occurred and when the type-i cells’ previous driver mutation occurred. At time of observation, assume the type-i population has κ
                              <sub>
                               i
                              </sub>
                              total driver mutations, where 1 ≤ κ
                              <sub>
                               i
                              </sub>
                              ≤ k for all 1 ≤ i ≤ k. Denote the time between the type-i’s κ
                              <sub>
                               i
                              </sub>
                              , or last, driver mutation and when the tumor is observed as τ
                              <sub>
                               i
                              </sub>
                              . This is the time between the founder type-i cell’s birth and tumor observation. Then the tumor is first observed and bulk sequenced at
                              <span class="inline-formula" id="inline-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-11.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-11.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (equivalently τ
                              <sub>
                               0
                              </sub>
                              for i = 0), where we denote a
                              <sup>
                               j
                              </sup>
                              as the jth iterate of the function a:
                              <span class="disp-formula" id="disp-formula-67">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-71.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-71.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-68">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-72.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-72.gif"/>
                                </noscript>
                               </span>
                              </span>
                              The tumor is also bulk sequenced at
                              <span class="inline-formula" id="inline-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-12.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-12.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (equivalently τ
                              <sub>
                               0
                              </sub>
                              + δ for i = 0). These assumptions allow for any subclone phylogeny, including combinations of the previously discussed sibling and nested subclone types.
                             </p>
                             <p id="p-77">
                              The bulk sequencing allows the measurement of the fraction of cells with driver i at T
                              <sub>
                               1
                              </sub>
                              , α
                              <sub>
                               i
                              </sub>
                              ; the fraction of cells with driver i at time T
                              <sub>
                               2
                              </sub>
                              , β
                              <sub>
                               i
                              </sub>
                              ; and the number of subclonal passenger mutations between frequencies f
                              <sub>
                               1
                              </sub>
                              and f
                              <sub>
                               2
                              </sub>
                              , γ. Again, the number of total cells at measurement times T
                              <sub>
                               1
                              </sub>
                              and T
                              <sub>
                               2
                              </sub>
                              are M
                              <sub>
                               1
                              </sub>
                              and M
                              <sub>
                               2
                              </sub>
                              . To write the type-i frequencies,
                              <span class="inline-formula" id="inline-formula-13">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-13.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-13.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-14">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-14.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-14.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , in terms of the driver frequencies, we subtract the fraction of cells descending from type-i cells but gaining additional driver mutation(s) after i, from the fraction of cells containing driver i:
                              <span class="disp-formula" id="disp-formula-69">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-73.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-73.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-70">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-74.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-74.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where δ
                              <sub>
                               i,a(j)
                              </sub>
                              is the Kronecker delta, defined as
                              <span class="disp-formula" id="disp-formula-71">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-75.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-75.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Equating the estimated cell counts at the first bulk sequencing timepoint to the expected value of the type-i population size X
                              <sub>
                               i
                              </sub>
                              , conditioned on survival,
                              <span class="disp-formula" id="disp-formula-72">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-76.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-76.gif"/>
                                </noscript>
                               </span>
                              </span>
                              And similarly, at the second bulk sequencing timepoint,
                              <span class="disp-formula" id="disp-formula-73">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-77.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-77.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-74">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-78.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-78.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Solving the above equations for r
                              <sub>
                               i
                              </sub>
                              , we obtain
                              <span class="disp-formula" id="disp-formula-75">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-79.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-79.gif"/>
                                </noscript>
                               </span>
                              </span>
                              By (
                              <a id="xref-disp-formula-11-4" class="xref-disp-formula" href="#disp-formula-11">
                               11
                              </a>
                              )
                              <span class="disp-formula" id="disp-formula-76">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-80.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-80.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Now, using the growth rate estimates r
                              <sub>
                               i
                              </sub>
                              and the subclone sizes, we can estimate each τ
                              <sub>
                               i
                              </sub>
                              . The expected value of the first time a population of type-i cells in a branching process reaches the observed size
                              <span class="inline-formula" id="inline-formula-15">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-15.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/inline-graphic-15.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is [
                              <a id="xref-ref-38-8" class="xref-bibr" href="#ref-38">
                               38
                              </a>
                              ]
                              <span class="disp-formula" id="disp-formula-77">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-81.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-81.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-78">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-82.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-82.gif"/>
                                </noscript>
                               </span>
                              </span>
                              which we approximate as
                              <span class="disp-formula" id="disp-formula-79">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-83.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-83.gif"/>
                                </noscript>
                               </span>
                              </span>
                              We make use of two approximations to arrive at (
                              <a id="xref-disp-formula-79-1" class="xref-disp-formula" href="#disp-formula-79">
                               78
                              </a>
                              ). First, we neglect the second term in (
                              <a id="xref-disp-formula-78-1" class="xref-disp-formula" href="#disp-formula-78">
                               77
                              </a>
                              ), which serves as a small correction to the first term. Second, we assume r
                              <sub>
                               i
                              </sub>
                              is similar in magnitude to b
                              <sub>
                               i
                              </sub>
                              .
                             </p>
                             <p id="p-78">
                              Using the (k + 1) r
                              <sub>
                               i
                              </sub>
                              estimates from (
                              <a id="xref-disp-formula-75-1" class="xref-disp-formula" href="#disp-formula-75">
                               74
                              </a>
                              ), and setting (
                              <a id="xref-disp-formula-76-1" class="xref-disp-formula" href="#disp-formula-76">
                               75
                              </a>
                              ) equal to the value of γ obtained at the second bulk sequencing from (
                              <a id="xref-disp-formula-28-3" class="xref-disp-formula" href="#disp-formula-28">
                               28
                              </a>
                              ), u can be estimated:
                              <span class="disp-formula" id="disp-formula-80">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-84.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-84.gif"/>
                                </noscript>
                               </span>
                              </span>
                              When estimating mutation rate for the CLL patients from Ref. [
                              <a id="xref-ref-27-14" class="xref-bibr" href="#ref-27">
                               27
                              </a>
                              ], for which there is bulk sequencing at two or more timepoints, we average the mutation rate calculated at each of these timepoints. (
                              <a id="xref-disp-formula-80-1" class="xref-disp-formula" href="#disp-formula-80">
                               79
                              </a>
                              ) is applied for each timepoint with the respective CCFs and observed γ values for each timepoint.
                             </p>
                             <p id="p-79">
                              The number of passengers present in the original type i founder cell cannot be directly observed, but we can measure m
                              <sub>
                               i
                              </sub>
                              , the number of clonal passengers present in the type i population, only including passengers not present in other clones. We will assume that the m
                              <sub>
                               i
                              </sub>
                              don’t contain any shared mutations, which is true for the CLL dataset we consider. The likelihood function P (m
                              <sub>
                               1
                              </sub>
                              , …, m
                              <sub>
                               k
                              </sub>
                              |t
                              <sub>
                               1
                              </sub>
                              , …, t
                              <sub>
                               k
                              </sub>
                              ) is proportional to
                              <span class="disp-formula" id="disp-formula-81">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-85.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-85.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Then, maximizing the logarithm of (
                              <a id="xref-disp-formula-81-1" class="xref-disp-formula" href="#disp-formula-81">
                               80
                              </a>
                              ) with respect to t
                              <sub>
                               1
                              </sub>
                              , t
                              <sub>
                               2
                              </sub>
                              , …, t
                              <sub>
                               k
                              </sub>
                              ,
                              <span class="disp-formula" id="disp-formula-82">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-86.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-86.gif"/>
                                </noscript>
                               </span>
                              </span>
                              The observed clonal passengers in the founder type-i cell will incorrectly include passengers that reached fixation in the type-i population after driver mutation i occurred, instead of correctly being counted toward the descendant of clone i. As a result, we again correct for the expected number of these passengers, u/r
                              <sub>
                               i
                              </sub>
                              . That is,
                              <span class="disp-formula" id="disp-formula-83">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-87.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/09/2020.11.18.387837/embed/graphic-87.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-25">
                            <h2 class="">
                             Availability of data and materials
                            </h2>
                            <p id="p-80">
                             All simulated data generated during this study are included in this published article and its supplementary information files. CLL data analyzed is publicly available in Supplementary Tables from Ref. [
                             <a id="xref-ref-27-15" class="xref-bibr" href="#ref-27">
                              27
                             </a>
                             ]. Code can be found at
                             <a href="https://github.com/nathanlee543/Cancer_Inf_Sims">
                              https://github.com/nathanlee543/Cancer_Inf_Sims
                             </a>
                            </p>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               [1].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.1" data-doi="10.1126/science.959840">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nowell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. C.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The Clonal Evolution of Tumor Cell Populations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  194
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  23
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  28
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1976
                                 </span>
                                 ). URL
                                 <a href="http://www.jstor.org/stable/1742535">
                                  http://www.jstor.org/stable/1742535
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DNowell%26rft.auinit1%253DP.%26rft.volume%253D194%26rft.issue%253D4260%26rft.spage%253D23%26rft.epage%253D28%26rft.atitle%253DThe%2Bclonal%2Bevolution%2Bof%2Btumor%2Bcell%2Bpopulations%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.959840%26rft_id%253Dinfo%253Apmid%252F959840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIxOTQvNDI2MC8yMyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAxLzA5LzIwMjAuMTEuMTguMzg3ODM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [2].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.11.18.387837v2.2" data-doi="10.1038/nature07943">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stratton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Campbell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Futreal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The cancer genome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  458
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  719
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  724
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ). URL
                                 <a href="https://www.nature.com/articles/nature07943">
                                  https://www.nature.com/articles/nature07943
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DStratton%26rft.auinit1%253DM.%2BR.%26rft.volume%253D458%26rft.issue%253D7239%26rft.spage%253D719%26rft.epage%253D724%26rft.atitle%253DThe%2Bcancer%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07943%26rft_id%253Dinfo%253Apmid%252F19360079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature07943&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19360079&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F09%2F2020.11.18.387837.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000265193600031&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [3].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.3" data-doi="10.1016/j.cell.2011.02.013">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hanahan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weinberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Hallmarks of Cancer: The Next Generation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  144
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  646
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  674
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ). URL
                                 <a href="https://www.cell.com/fulltext/S0092-8674(11)00127-9">
                                  https://www.cell.com/fulltext/S0092-8674(11)00127-9
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHanahan%26rft.auinit1%253DD.%26rft.volume%253D144%26rft.issue%253D5%26rft.spage%253D646%26rft.epage%253D674%26rft.atitle%253DHallmarks%2Bof%2Bcancer%253A%2Bthe%2Bnext%2Bgeneration.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.02.013%26rft_id%253Dinfo%253Apmid%252F21376230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.02.013&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21376230&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F09%2F2020.11.18.387837.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000288007100007&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [4].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.4" data-doi="10.1016/j.cell.2018.02.060">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bailey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. H.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Comprehensive Characterization of Cancer Driver Genes and Mutations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  173
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  371
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  385
                                 </span>
                                 .e18 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ). URL
                                 <a href="https://www.cell.com/cell/abstract/S0092-8674(18)30237-X">
                                  https://www.cell.com/cell/abstract/S0092-8674(18)30237-X
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D173%26rft.spage%253D371%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.02.060%26rft_id%253Dinfo%253Apmid%252F29625053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.02.060&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29625053&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F09%2F2020.11.18.387837.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [5].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.5" data-doi="10.1038/ng.3489">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Williams
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Werner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barnes
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Graham
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sottoriva
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Identification of neutral tumor evolution across cancer types
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Genetics
                                 </abbr>
                                 <span class="cit-vol">
                                  48
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  238
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  244
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ). URL
                                 <a href="https://www.nature.com/articles/ng.3489">
                                  https://www.nature.com/articles/ng.3489
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BGenetics%26rft.volume%253D48%26rft.spage%253D238%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3489%26rft_id%253Dinfo%253Apmid%252F26780609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ng.3489&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26780609&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F09%2F2020.11.18.387837.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [6].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.6" data-doi="10.1038/217624a0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kimura
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Evolutionary Rate at the Molecular Level
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  217
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  624
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  626
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1968
                                 </span>
                                 ). URL
                                 <a href="https://www.nature.com/articles/217624a0">
                                  https://www.nature.com/articles/217624a0
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKimura%26rft.auinit1%253DM.%26rft.volume%253D217%26rft.issue%253D5129%26rft.spage%253D624%26rft.epage%253D626%26rft.atitle%253DEvolutionary%2Brate%2Bat%2Bthe%2Bmolecular%2Blevel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F217624a0%26rft_id%253Dinfo%253Apmid%252F5637732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/217624a0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=5637732&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F09%2F2020.11.18.387837.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1968A646300008&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [7].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.11.18.387837v2.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kimura
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Genetic variability maintained in a finite population due to mutational production of neutral and nearly neutral isoalleles*
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics Research
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  247
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  270
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1968
                                 </span>
                                 ). URL
                                 <a href="http://www.cambridge.org/core/journals/genetics-research/article/genetic-variability-maintained-in-a-finite-population-due-to-mutational-production-of-neutral-and-A74BD3A5D72ED2C52444FD99DFE483EF">
                                  http://www.cambridge.org/core/journals/genetics-research/article/genetic-variability-maintained-in-a-finite-population-due-to-mutational-production-of-neutral-and-A74BD3A5D72ED2C52444FD99DFE483EF
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%2BResearch%26rft.volume%253D11%26rft.spage%253D247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [8].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.11.18.387837v2.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Haigh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The hitch-hiking effect of a favourable gene
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics Research
                                 </abbr>
                                 <span class="cit-vol">
                                  23
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  23
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  35
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1974
                                 </span>
                                 ). URL
                                 <a href="http://www.cambridge.org/core/journals/genetics-research/article/hitchhiking-effect-of-a-favourable-gene/918291A3B62BD50E1AE5C1F22165EF1B">
                                  http://www.cambridge.org/core/journals/genetics-research/article/hitchhiking-effect-of-a-favourable-gene/918291A3B62BD50E1AE5C1F22165EF1B
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%2BResearch%26rft.volume%253D23%26rft.spage%253D23%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [9].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Turajlic
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sottoriva
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Graham
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Swanton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Resolving genetic heterogeneity in cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Reviews Genetics
                                 </abbr>
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  404
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  416
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ). URL
                                 <a href="https://www.nature.com/articles/s41576-019-0114-6">
                                  https://www.nature.com/articles/s41576-019-0114-6
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BGenetics%26rft.volume%253D20%26rft.spage%253D404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [10].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.10" data-doi="10.1126/science.1235122">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vogelstein
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Cancer Genome Landscapes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  339
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1546
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1558
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ). URL
                                 <a href="https://science.sciencemag.org/content/339/6127/1546">
                                  https://science.sciencemag.org/content/339/6127/1546
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DVogelstein%26rft.auinit1%253DB.%26rft.volume%253D339%26rft.issue%253D6127%26rft.spage%253D1546%26rft.epage%253D1558%26rft.atitle%253DCancer%2BGenome%2BLandscapes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1235122%26rft_id%253Dinfo%253Apmid%252F23539594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzkvNjEyNy8xNTQ2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMDkvMjAyMC4xMS4xOC4zODc4MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [11].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.11" data-doi="10.1038/nature12213">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lawrence
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Mutational heterogeneity in cancer and the search for new cancer-associated genes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  499
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  214
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  218
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ). URL
                                 <a href="https://www.nature.com/articles/nature12213">
                                  https://www.nature.com/articles/nature12213
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D499%26rft.spage%253D214%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12213%26rft_id%253Dinfo%253Apmid%252F23770567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature12213&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23770567&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F09%2F2020.11.18.387837.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000321557600063&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [12].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.12" data-doi="10.1038/nrc2013">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Merlo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L. M. F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pepper
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reid
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maley
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. C.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Cancer as an evolutionary and ecological process
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Reviews Cancer
                                 </abbr>
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  924
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  935
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ). URL
                                 <a href="https://www.nature.com/articles/nrc2013">
                                  https://www.nature.com/articles/nrc2013
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BCancer%26rft.stitle%253DNat%2BRev%2BCancer%26rft.aulast%253DPetricoin%26rft.auinit1%253DE.%2BF.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253D924%26rft.epage%253D935%26rft.atitle%253DCancer%2Bas%2Ban%2Bevolutionary%2Band%2Becological%2Bprocess.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrc2013%26rft_id%253Dinfo%253Apmid%252F17109012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrc2013&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17109012&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F09%2F2020.11.18.387837.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000242244400013&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [13].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pepper
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Findlay
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kassen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Spencer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. L.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maley
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. C.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  SYNTHESIS: Cancer research meets evolutionary biology
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Evolutionary Applications
                                 </abbr>
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  62
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  70
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ). URL
                                 <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1752-4571.2008.00063.x">
                                  https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1752-4571.2008.00063.x
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D2%26rft.spage%253D62%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [14].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.14" data-doi="10.1073/pnas.97.3.1236">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tsao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.-L.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Genetic reconstruction of individual colorectal tumor histories
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  97
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1236
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1241
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 ). URL
                                 <a href="https://www.pnas.org/content/97/3/1236">
                                  https://www.pnas.org/content/97/3/1236
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DTsao%26rft.auinit1%253DJ.-L.%26rft.volume%253D97%26rft.issue%253D3%26rft.spage%253D1236%26rft.epage%253D1241%26rft.atitle%253DGenetic%2Breconstruction%2Bof%2Bindividual%2Bcolorectal%2Btumor%2Bhistories%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.97.3.1236%26rft_id%253Dinfo%253Apmid%252F10655514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5Ny8zLzEyMzYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMS8wOS8yMDIwLjExLjE4LjM4NzgzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [15].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.11.18.387837v2.15" data-doi="10.1073/pnas.0712345105">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jones
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Comparative lesion sequencing provides insights into tumor evolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences of the United States of America
                                 </abbr>
                                 <span class="cit-vol">
                                  105
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  4283
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4288
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ). URL
                                 <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2393770/">
                                  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2393770/
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0712345105%26rft_id%253Dinfo%253Apmid%252F18337506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzExLzQyODMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMS8wOS8yMDIwLjExLjE4LjM4NzgzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [16].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.16" data-doi="10.1038/nature09515">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yachida
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Distant metastasis occurs late during the genetic evolution of pancreatic cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  467
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1114
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1117
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ). URL
                                 <a href="https://www.nature.com/articles/nature09515">
                                  https://www.nature.com/articles/nature09515
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DYachida%26rft.auinit1%253DS.%26rft.volume%253D467%26rft.issue%253D7319%26rft.spage%253D1114%26rft.epage%253D1117%26rft.atitle%253DDistant%2Bmetastasis%2Boccurs%2Blate%2Bduring%2Bthe%2Bgenetic%2Bevolution%2Bof%2Bpancreatic%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09515%26rft_id%253Dinfo%253Apmid%252F20981102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature09515&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20981102&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F09%2F2020.11.18.387837.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000283548600049&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [17].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.11.18.387837v2.17" data-doi="10.1073/pnas.1400179111">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Naxerova
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Hypermutable DNA chronicles the evolution of human colon cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  111
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  E1889
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  E1898
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ). URL
                                 <a href="http://www.pnas.org/cgi/doi/10.1073/pnas.1400179111">
                                  http://www.pnas.org/cgi/doi/10.1073/pnas.1400179111
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1400179111%26rft_id%253Dinfo%253Apmid%252F24753616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzE4L0UxODg5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMDkvMjAyMC4xMS4xOC4zODc4MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [18].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.18" data-doi="10.1126/scitranslmed.aaa1408">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McGranahan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Clonal status of actionable driver events and the timing of mutational processes in cancer evolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science Translational Medicine
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  283ra54
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  283ra54
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ). URL
                                 <a href="https://stm.sciencemag.org/content/7/283/283ra54">
                                  https://stm.sciencemag.org/content/7/283/283ra54
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%2BTranslational%2BMedicine%26rft.stitle%253DSci%2BTransl%2BMed%26rft.aulast%253DMcGranahan%26rft.auinit1%253DN.%26rft.volume%253D7%26rft.issue%253D283%26rft.spage%253D283ra54%26rft.epage%253D283ra54%26rft.atitle%253DClonal%2Bstatus%2Bof%2Bactionable%2Bdriver%2Bevents%2Band%2Bthe%2Btiming%2Bof%2Bmutational%2Bprocesses%2Bin%2Bcancer%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscitranslmed.aaa1408%26rft_id%253Dinfo%253Apmid%252F25877892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6InNjaXRyYW5zbWVkIjtzOjU6InJlc2lkIjtzOjEzOiI3LzI4My8yODNyYTU0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMDkvMjAyMC4xMS4xOC4zODc4MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [19].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.19" data-doi="10.1016/j.cell.2018.02.020">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mitchell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Timing the Landmark Events in the Evolution of Clear Cell Renal Cell Cancer: TRACERx Renal
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  173
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  611
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  623
                                 </span>
                                 .e17 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ). URL
                                 <a href="http://www.sciencedirect.com/science/article/pii/S0092867418301648">
                                  http://www.sciencedirect.com/science/article/pii/S0092867418301648
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D173%26rft.spage%253D611%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.02.020%26rft_id%253Dinfo%253Apmid%252F29656891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.02.020&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29656891&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F09%2F2020.11.18.387837.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [20].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.20" data-doi="10.1038/s41586-019-1907-7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  PCAWG Evolution & Heterogeneity Working Group et al
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The evolutionary history of 2,658 cancers
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  578
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  122
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  128
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ). URL
                                 <a href="http://www.nature.com/articles/s41586-019-1907-7">
                                  http://www.nature.com/articles/s41586-019-1907-7
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D578%26rft.spage%253D122%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-019-1907-7%26rft_id%253Dinfo%253Apmid%252F32025013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-019-1907-7&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=32025013&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F09%2F2020.11.18.387837.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [21].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.21">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sundermann
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L. K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wintersinger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rätsch
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stoye
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Morris
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Q.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Reconstructing tumor evolutionary histories and clone trees in polynomial-time with SubMARine
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLOS Computational Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e1008400
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ). URL
                                 <a href="https://dx.plos.org/10.1371/journal.pcbi.1008400">
                                  https://dx.plos.org/10.1371/journal.pcbi.1008400
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLOS%2BComputational%2BBiology%26rft.volume%253D17%26rft.spage%253D1008400e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [22].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  PCAWG Evolution and Heterogeneity Working Group et al
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Reconstructing evolutionary trajectories of mutation signature activities in cancer using TrackSig
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Communications
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  731
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ). URL
                                 <a href="http://www.nature.com/articles/s41467-020-14352-7">
                                  http://www.nature.com/articles/s41467-020-14352-7
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D11%26rft.spage%253D731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [23].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tomasetti
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bozic
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The (not so) immortal strand hypothesis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Stem Cell Research
                                 </abbr>
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  238
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  241
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DStem%2BCell%2BResearch%26rft.volume%253D14%26rft.spage%253D238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [24].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.24">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Werner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Measuring single cell divisions in human tissues from multi-region sequencing data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Communications
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  9
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ). URL
                                 <a href="https://www.nature.com/articles/s41467-020-14844-6">
                                  https://www.nature.com/articles/s41467-020-14844-6
                                 </a>
                                 . Number: 1 Publisher: Nature Publishing Group.
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D11%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [25].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.25" data-doi="10.1073/pnas.1010978107">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bozic
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Accumulation of driver and passenger mutations during tumor progression
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences of the United States of America
                                 </abbr>
                                 <span class="cit-vol">
                                  107
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  18545
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  18550
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ). URL
                                 <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2972991/">
                                  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2972991/
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1010978107%26rft_id%253Dinfo%253Apmid%252F20876136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzQzLzE4NTQ1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMDkvMjAyMC4xMS4xOC4zODc4MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [26].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.11.18.387837v2.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sun
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Between-region genetic divergence reflects the mode and tempo of tumor evolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Genetics
                                 </abbr>
                                 <span class="cit-vol">
                                  49
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1015
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1024
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ). URL
                                 <a href="https://www.nature.com/articles/ng.3891">
                                  https://www.nature.com/articles/ng.3891
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BGenetics%26rft.volume%253D49%26rft.spage%253D1015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [27].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference [27] in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.27">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gruber
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Growth dynamics in naturally progressing chronic lymphocytic leukaemia
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  570
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  474
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  479
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ). URL
                                 <a href="https://www.nature.com/articles/s41586-019-1252-x">
                                  https://www.nature.com/articles/s41586-019-1252-x
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D570%26rft.spage%253D474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [28].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.28">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Salichos
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Meyerson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Warrell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gerstein
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Estimating growth patterns and driver effects in tumor evolution from individual samples
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Communications
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  14
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ). URL
                                 <a href="https://www.nature.com/articles/s41467-020-14407-9">
                                  https://www.nature.com/articles/s41467-020-14407-9
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D11%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [29].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.11.18.387837v2.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Noble
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Spatial structure governs the mode of tumour evolution
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Nature Ecology & Evolution
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ). URL
                                 <a href="https://www.nature.com/articles/s41559-021-01615-9">
                                  https://www.nature.com/articles/s41559-021-01615-9
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [30].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chkhaidze
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Spatially constrained tumour growth affects the patterns of clonal selection and neutral drift in cancer genomic data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLOS Computational Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e1007243
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ). URL
                                 <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007243">
                                  https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007243
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLOS%2BComputational%2BBiology%26rft.volume%253D15%26rft.spage%253De1007243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [31].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.11.18.387837v2.31">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Spatial patterns of tumour growth impact clonal diversification in a computational model and the TRACERx Renal study
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Nature Ecology & Evolution
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ). URL
                                 <a href="https://www.nature.com/articles/s41559-021-01586-x">
                                  https://www.nature.com/articles/s41559-021-01586-x
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [32].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference [32] in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.32" data-doi="10.1038/s41588-018-0128-6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Williams
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Quantification of subclonal selection in cancer from bulk sequencing data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Genetics
                                 </abbr>
                                 <span class="cit-vol">
                                  50
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  895
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ). URL
                                 <a href="https://www.nature.com/articles/s41588-018-0128-6">
                                  https://www.nature.com/articles/s41588-018-0128-6
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BGenetics%26rft.volume%253D50%26rft.spage%253D895%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-018-0128-6%26rft_id%253Dinfo%253Apmid%252F29808029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41588-018-0128-6&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29808029&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F09%2F2020.11.18.387837.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [33].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference [33] in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.33" data-doi="10.1126/sciadv.abc4308">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Avanzini
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  A mathematical model of ctDNA shedding predicts tumor detection size
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science Advances
                                 </abbr>
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  eabc4308
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ). URL
                                 <a href="https://www.science.org/doi/10.1126/sciadv.abc4308">
                                  https://www.science.org/doi/10.1126/sciadv.abc4308
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DAvanzini%26rft.auinit1%253DS.%26rft.volume%253D6%26rft.issue%253D50%26rft.spage%253Deabc4308%26rft.epage%253Deabc4308%26rft.atitle%253DA%2Bmathematical%2Bmodel%2Bof%2BctDNA%2Bshedding%2Bpredicts%2Btumor%2Bdetection%2Bsize%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.abc4308%26rft_id%253Dinfo%253Apmid%252F33310847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMzoiNi81MC9lYWJjNDMwOCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAxLzA5LzIwMjAuMTEuMTguMzg3ODM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [34].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [34] in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.34" data-doi="10.7554/eLife.00747">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bozic
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Evolutionary dynamics of cancer in response to targeted combination therapy
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  eLife
                                 </abbr>
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e00747
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ). URL
                                 <a href="https://elifesciences.org/articles/00747">
                                  https://elifesciences.org/articles/00747
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.00747%26rft_id%253Dinfo%253Apmid%252F23805382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.7554/eLife.00747&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23805382&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F09%2F2020.11.18.387837.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [35].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [35] in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.35" data-doi="10.1214/19-sts7561">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dinh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jaksik
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kimmel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lambert
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tavaré
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Statistical Inference for the Evolutionary History of Cancer Genomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Statistical Science
                                 </abbr>
                                 <span class="cit-vol">
                                  35
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  129
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  144
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ). URL
                                 <a href="https://projecteuclid.org/euclid.ss/1583226033">
                                  https://projecteuclid.org/euclid.ss/1583226033
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DStatistical%2BScience%26rft.volume%253D35%26rft.spage%253D129%26rft_id%253Dinfo%253Adoi%252F10.1214%252F19-sts7561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1214/19-sts7561&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [36].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference [36] in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.36" data-doi="10.1073/pnas.1914589117">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lahouel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Revisiting the tumorigenesis timeline with a data-driven generative model
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  117
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  857
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  864
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ). URL
                                 <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.1914589117">
                                  http://www.pnas.org/lookup/doi/10.1073/pnas.1914589117
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1914589117%26rft_id%253Dinfo%253Apmid%252F31882448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTcvMi84NTciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMS8wOS8yMDIwLjExLjE4LjM4NzgzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [37].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference [37] in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.37">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bozic
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Delineating the evolutionary dynamics of cancer from theory to reality
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Cancer
                                 </abbr>
                                 <span class="cit-vol">
                                  1
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  580
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  588
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ). URL
                                 <a href="http://www.nature.com/articles/s43018-020-0079-6">
                                  http://www.nature.com/articles/s43018-020-0079-6
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BCancer%26rft.volume%253D1%26rft.spage%253D580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [38].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference [38] in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.11.18.387837v2.38">
                               <div class="cit-metadata">
                                <ol class="duplicate">
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-surname">
                                    Durrett
                                   </span>
                                   ,
                                   <span class="cit-name-given-names">
                                    R.
                                   </span>
                                  </span>
                                 </li>
                                </ol>
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Durrett
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-chapter-title">
                                  Branching Process Models of Cancer
                                 </span>
                                 . In
                                 <span class="cit-ed">
                                  <span class="cit-name-surname">
                                   Durrett
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 (ed.)
                                 <span class="cit-source">
                                  Branching Process Models of Cancer, Mathematical Biosciences Institute Lecture Series
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  63
                                 </span>
                                 (
                                 <span class="cit-publ-name">
                                  Springer International Publishing
                                 </span>
                                 ,
                                 <span class="cit-publ-loc">
                                  Cham
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ). URL
                                 <a href="https://doi.org/10.1007/978-3-319-16065-8_1">
                                  https://doi.org/10.1007/978-3-319-16065-8_1
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [39].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.11.18.387837v2.39">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tavaré
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The linear birth–death process: an inferential retrospective
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Advances in Applied Probability
                                 </abbr>
                                 <span class="cit-vol">
                                  50
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  253
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  269
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ). URL
                                 <a href="https://www.cambridge.org/core/product/identifier/S0001867818000848/type/journal_article">
                                  https://www.cambridge.org/core/product/identifier/S0001867818000848/type/journal_article
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2BApplied%2BProbability%26rft.volume%253D50%26rft.spage%253D253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [40].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference [40] in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.40" data-doi="10.1073/pnas.1819408116">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Heyde
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reiter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Naxerova
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nowak
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Consecutive seeding and transfer of genetic diversity in metastasis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  116
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  14129
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  14137
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ). URL
                                 <a href="https://www.pnas.org/content/116/28/14129">
                                  https://www.pnas.org/content/116/28/14129
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1819408116%26rft_id%253Dinfo%253Apmid%252F31239334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE2LzI4LzE0MTI5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMDkvMjAyMC4xMS4xOC4zODc4MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [41].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [41] in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.41">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Griffith
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Optimizing Cancer Genome Sequencing and Analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell Systems
                                 </abbr>
                                 <span class="cit-vol">
                                  1
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  210
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  223
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ). URL
                                 <a href="http://www.sciencedirect.com/science/article/pii/S2405471215001131">
                                  http://www.sciencedirect.com/science/article/pii/S2405471215001131
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%2BSystems%26rft.volume%253D1%26rft.spage%253D210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [42].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.11.18.387837v2.42" data-doi="10.1158/2159-8290.CD-13-1014">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Haber
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Velculescu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V. E.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Blood-Based Analyses of Cancer: Circulating Tumor Cells and Circulating Tumor DNA
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cancer Discovery
                                 </abbr>
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  650
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  661
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ). URL
                                 <a href="https://cancerdiscovery.aacrjournals.org/content/4/6/650">
                                  https://cancerdiscovery.aacrjournals.org/content/4/6/650
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCancer%2BDiscovery%26rft_id%253Dinfo%253Adoi%252F10.1158%252F2159-8290.CD-13-1014%26rft_id%253Dinfo%253Apmid%252F24801577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiY2FuZGlzYyI7czo1OiJyZXNpZCI7czo3OiI0LzYvNjUwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMDkvMjAyMC4xMS4xOC4zODc4MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [43].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference [43] in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.11.18.387837v2.43">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leshchiner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Comprehensive analysis of tumour initiation, spatial and temporal progression under multiple lines of treatment. preprint
                                 </span>
                                 ,
                                 <span class="cit-source">
                                  Bioinformatics
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ). URL
                                 <a href="http://biorxiv.org/lookup/doi/10.1101/508127">
                                  http://biorxiv.org/lookup/doi/10.1101/508127
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [44].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference [44] in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.44">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Myers
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Satas
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Raphael
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  CALDER: Inferring Phylogenetic Trees from Longitudinal Tumor Samples
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell Systems
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  514
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  522
                                 </span>
                                 .e5 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ). URL
                                 <a href="http://www.sciencedirect.com/science/article/pii/S2405471219301917">
                                  http://www.sciencedirect.com/science/article/pii/S2405471219301917
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%2BSystems%26rft.volume%253D8%26rft.spage%253D514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [45].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference [45] in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.45" data-doi="10.1007/BF00391826">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carlsson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gullberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hafström
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Estimation of liver tumor volume using different formulas?An experimental study in rats
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Cancer Research and Clinical Oncology
                                 </abbr>
                                 <span class="cit-vol">
                                  105
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  20
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  23
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1983
                                 </span>
                                 ). URL
                                 <a href="http://link.springer.com/10.1007/BF00391826">
                                  http://link.springer.com/10.1007/BF00391826
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcancer%2Bresearch%2Band%2Bclinical%2Boncology%26rft.stitle%253DJ%2BCancer%2BRes%2BClin%2BOncol%26rft.aulast%253DCarlsson%26rft.auinit1%253DG.%26rft.volume%253D105%26rft.issue%253D1%26rft.spage%253D20%26rft.epage%253D23%26rft.atitle%253DEstimation%2Bof%2Bliver%2Btumor%2Bvolume%2Busing%2Bdifferent%2Bformulas%2B-%2Ban%2Bexperimental%2Bstudy%2Bin%2Brats.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00391826%26rft_id%253Dinfo%253Apmid%252F6833336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/BF00391826&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=6833336&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F09%2F2020.11.18.387837.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1983QE27000003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [46].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference [46] in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.46">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bozic
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gerold
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nowak
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Quantifying Clonal and Subclonal Passenger Mutations in Cancer Evolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLOS Computational Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e1004731
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ). URL
                                 <a href="http://dx.plos.org/10.1371/journal.pcbi.1004731">
                                  http://dx.plos.org/10.1371/journal.pcbi.1004731
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLOS%2BComputational%2BBiology%26rft.volume%253D12%26rft.spage%253De1004731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [47].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference [47] in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.47" data-doi="10.1038/nature19768">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blokzijl
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Tissue-specific mutation accumulation in human adult stem cells during life
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  538
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  260
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  264
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ). URL
                                 <a href="http://www.nature.com/articles/nature19768">
                                  http://www.nature.com/articles/nature19768
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D538%26rft.spage%253D260%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature19768%26rft_id%253Dinfo%253Apmid%252F27698416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature19768&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27698416&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F09%2F2020.11.18.387837.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [48].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference [48] in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.11.18.387837v2.48">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mitchell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Clonal dynamics of haematopoiesis across the human lifespan. preprint
                                 </span>
                                 ,
                                 <span class="cit-source">
                                  Genomics
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ). URL
                                 <a href="http://biorxiv.org/lookup/doi/10.1101/2021.08.16.456475">
                                  http://biorxiv.org/lookup/doi/10.1101/2021.08.16.456475
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [49].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference [49] in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.49" data-doi="10.1073/pnas.1901695116">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Auslander
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wolf
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y. I.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koonin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. V.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  In silico learning of tumor evolution through mutational time series
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  116
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  9501
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  9510
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ). URL
                                 <a href="https://www.pnas.org/content/116/19/9501">
                                  https://www.pnas.org/content/116/19/9501
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1901695116%26rft_id%253Dinfo%253Apmid%252F31015295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE2LzE5Lzk1MDEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMS8wOS8yMDIwLjExLjE4LjM4NzgzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [50].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference [50] in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.50" data-doi="10.1016/j.cell.2019.02.012">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Petljak
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Characterizing Mutational Signatures in Human Cancer Cell Lines Reveals Episodic APOBEC Mutagenesis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  176
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1282
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1294
                                 </span>
                                 .e20 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ). URL
                                 <a href="https://linkinghub.elsevier.com/retrieve/pii/S0092867419301618">
                                  https://linkinghub.elsevier.com/retrieve/pii/S0092867419301618
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D176%26rft.spage%253D1282%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2019.02.012%26rft_id%253Dinfo%253Apmid%252F30849372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2019.02.012&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30849372&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F09%2F2020.11.18.387837.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [51].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference [51] in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.51" data-doi="10.1021/j100540a008">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gillespie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. T.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Exact stochastic simulation of coupled chemical reactions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Journal of Physical Chemistry
                                 </abbr>
                                 <span class="cit-vol">
                                  81
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2340
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2361
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1977
                                 </span>
                                 ). URL
                                 <a href="https://doi.org/10.1021/j100540a008">
                                  https://doi.org/10.1021/j100540a008
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BPhysical%2BChemistry%26rft.volume%253D81%26rft.spage%253D2340%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fj100540a008%26rft_id%253Dinfo%253Apmid%252FA1977EE49800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1021/j100540a008&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1977EE49800&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F09%2F2020.11.18.387837.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1977EE49800008&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [52].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference [52] in text" id="ref-52">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.18.387837v2.52">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bozic
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Paterson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Waclaw
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  On measuring selection in cancer from subclonal mutation frequencies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLOS Computational Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e1007368
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ). URL
                                 <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007368">
                                  https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007368
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLOS%2BComputational%2BBiology%26rft.volume%253D15%26rft.spage%253De1007368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [53].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference [53] in text" id="ref-53">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.11.18.387837v2.53">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fabre
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. A.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The longitudinal dynamics and natural history of clonal haematopoiesis. preprint
                                 </span>
                                 ,
                                 <span class="cit-source">
                                  Cancer Biology
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ). URL
                                 <a href="http://biorxiv.org/lookup/doi/10.1101/2021.08.12.455048">
                                  http://biorxiv.org/lookup/doi/10.1101/2021.08.12.455048
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.11.18.387837v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/483263v3" title="A learned embedding for efficient joint analysis of millions of mass spectra" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2021.02.07.430172v2" title="Consistent population activity on the scale of minutes in the mouse hippocampus" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted January 09, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.11.18.387837v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.11.18.387837v2.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.11.18.387837v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.11.18.387837v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   Inferring parameters of cancer evolution from sequencing and clinical data
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2342328"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-Zt9Ub8H3hgCbYU0bCi6FiXXkIqT-btdbtvbPe5xNxkM"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484486556"/>
                                   <input type="hidden" name="captcha_token" value="81de3d6630735f9794cc06be498e53de"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2342328--3" data-node-nid="2342328" id="node-2342328--42079211817" data-pisa="biorxiv;2020.11.18.387837v2" data-pisa-master="biorxiv;2020.11.18.387837" data-seqnum="2342328" data-apath="/biorxiv/early/2022/01/09/2020.11.18.387837.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Inferring parameters of cancer evolution from sequencing and clinical data
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Nathan
                                  </span>
                                  <span class="nlm-surname">
                                   Lee
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Ivana
                                  </span>
                                  <span class="nlm-surname">
                                   Bozic
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.11.18.387837;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.11.18.387837
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.11.18.387837v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.11.18.387837v2&title=Inferring%20parameters%20of%20cancer%20evolution%20from%20sequencing%20and%20clinical%20data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.18.387837v2&title=Inferring%20parameters%20of%20cancer%20evolution%20from%20sequencing%20and%20clinical%20data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.18.387837v2&text=Inferring%20parameters%20of%20cancer%20evolution%20from%20sequencing%20and%20clinical%20data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.11.18.387837v2&t=Inferring%20parameters%20of%20cancer%20evolution%20from%20sequencing%20and%20clinical%20data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.11.18.387837v2&title=Inferring%20parameters%20of%20cancer%20evolution%20from%20sequencing%20and%20clinical%20data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.11.18.387837v2&title=Inferring%20parameters%20of%20cancer%20evolution%20from%20sequencing%20and%20clinical%20data&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.18.387837v2&title=Inferring%20parameters%20of%20cancer%20evolution%20from%20sequencing%20and%20clinical%20data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2342328--5" data-node-nid="2342328" id="citation-node-2342328--61763169397" data-pisa="biorxiv;2020.11.18.387837v2" data-pisa-master="biorxiv;2020.11.18.387837" data-seqnum="2342328" data-apath="/biorxiv/early/2022/01/09/2020.11.18.387837.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Inferring parameters of cancer evolution from sequencing and clinical data
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Nathan
                                      </span>
                                      <span class="nlm-surname">
                                       Lee
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Ivana
                                      </span>
                                      <span class="nlm-surname">
                                       Bozic
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.11.18.387837;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.11.18.387837
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Inferring%20parameters%20of%20cancer%20evolution%20from%20sequencing%20and%20clinical%20data&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Lee%2C+Nathan&rft.au=Bozic%2C+Ivana">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2342328/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2342328/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2342328/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2342328/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2342328/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2342328/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2342328/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2342328/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2342328/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2342328/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2342328/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2342328/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.18.387837v2&count=horizontal&via=&text=Inferring%20parameters%20of%20cancer%20evolution%20from%20sequencing%20and%20clinical%20data&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.11.18.387837v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.11.18.387837v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.11.18.387837v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/cancer-biology" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Cancer Biology
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23507)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11824)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11440)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15189)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47703)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
