<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.09.29.315374v4.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.09.29.315374v4.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.09.29.315374v4.ppt"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/03/04/2020.09.29.315374.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.09.29.315374v4"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="AutoCoEv – a high-throughput in silico pipeline for predicting inter-protein co-evolution"/>
  <meta name="DC.Identifier" content="10.1101/2020.09.29.315374"/>
  <meta name="DC.Date" content="2022-03-04"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Protein-protein communications govern cellular processes via complex regulatory networks, that are still far from being understood. Thus, identifying novel interactions between proteins can significantly facilitate our comprehension of the mechanistic principles of protein functions. Co-evolution between proteins is a sign of functional communication and, as such, provides a powerful approach to search for novel direct or indirect molecular partners. However, evolutionary analysis of large arrays of proteins, in silico, is a highly time-consuming effort, which has limited the usage of this method to protein pairs or small protein groups. Here, we developed AutoCoEv, a user-friendly computational pipeline for the search of co-evolution between a large number of proteins. By driving 15 individual programs, culminating in CAPS2 as the software for detecting co-evolution, AutoCoEv achieves seamless automation and parallelization of the workflow. Importantly, we provide a patch to CAPS2 source code to strengthen its statistical output, allowing for multiple comparisons correction and enhanced analysis of the results. We apply the pipeline to inspect co-evolution among 324 proteins identified to locate at the vicinity of the lipid rafts of B lymphocytes. We successfully detected multiple strong coevolutionary relations between the proteins, predicting many novel partners and previously unidentified clusters of functionally related molecules. We conclude that AutoCoEv, available at <https://github.com/mattilalab/autocoev>, can be used to predict functional interactions from large datasets in a time and cost-efficient manner.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Petar B. Petrov"/>
  <meta name="DC.Contributor" content="Luqman O. Awoniyi"/>
  <meta name="DC.Contributor" content="Vid Šuštar"/>
  <meta name="DC.Contributor" content="M. Özge Balcı"/>
  <meta name="DC.Contributor" content="Pieta K. Mattila"/>
  <meta name="article:published_time" content="2022-03-04"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="AutoCoEv – a high-throughput in silico pipeline for predicting inter-protein co-evolution"/>
  <meta name="citation_abstract" lang="en" content="<p>Protein-protein communications govern cellular processes via complex regulatory networks, that are still far from being understood. Thus, identifying novel interactions between proteins can significantly facilitate our comprehension of the mechanistic principles of protein functions. Co-evolution between proteins is a sign of functional communication and, as such, provides a powerful approach to search for novel direct or indirect molecular partners. However, evolutionary analysis of large arrays of proteins, in silico, is a highly time-consuming effort, which has limited the usage of this method to protein pairs or small protein groups. Here, we developed AutoCoEv, a user-friendly computational pipeline for the search of co-evolution between a large number of proteins. By driving 15 individual programs, culminating in CAPS2 as the software for detecting co-evolution, AutoCoEv achieves seamless automation and parallelization of the workflow. Importantly, we provide a patch to CAPS2 source code to strengthen its statistical output, allowing for multiple comparisons correction and enhanced analysis of the results. We apply the pipeline to inspect co-evolution among 324 proteins identified to locate at the vicinity of the lipid rafts of B lymphocytes. We successfully detected multiple strong coevolutionary relations between the proteins, predicting many novel partners and previously unidentified clusters of functionally related molecules. We conclude that AutoCoEv, available at https://github.com/mattilalab/autocoev, can be used to predict functional interactions from large datasets in a time and cost-efficient manner.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.09.29.315374v4"/>
  <meta name="citation_id" content="2020.09.29.315374v4"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.09.29.315374v4"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.09.29.315374v4.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.09.29.315374v4.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.09.29.315374"/>
  <meta name="citation_num_pages" content="14"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.09.29.315374"/>
  <meta name="citation_author" content="Petar B. Petrov"/>
  <meta name="citation_author_institution" content="Institute of Biomedicine and MediCity Research Laboratories, University of Turku"/>
  <meta name="citation_author_institution" content="Turku Bioscience, University of Turku and Åbo Akademi University"/>
  <meta name="citation_author_email" content="petar.petrov@utu.fi"/>
  <meta name="citation_author_email" content="pieta.mattila@utu.fi"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5551-8032"/>
  <meta name="citation_author" content="Luqman O. Awoniyi"/>
  <meta name="citation_author_institution" content="Institute of Biomedicine and MediCity Research Laboratories, University of Turku"/>
  <meta name="citation_author_institution" content="Turku Bioscience, University of Turku and Åbo Akademi University"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7809-3268"/>
  <meta name="citation_author" content="Vid Šuštar"/>
  <meta name="citation_author_institution" content="Turku Bioscience, University of Turku and Åbo Akademi University"/>
  <meta name="citation_author" content="M. Özge Balcı"/>
  <meta name="citation_author_institution" content="Institute of Biomedicine and MediCity Research Laboratories, University of Turku"/>
  <meta name="citation_author_institution" content="Turku Bioscience, University of Turku and Åbo Akademi University"/>
  <meta name="citation_author" content="Pieta K. Mattila"/>
  <meta name="citation_author_institution" content="Institute of Biomedicine and MediCity Research Laboratories, University of Turku"/>
  <meta name="citation_author_institution" content="Turku Bioscience, University of Turku and Åbo Akademi University"/>
  <meta name="citation_author_email" content="petar.petrov@utu.fi"/>
  <meta name="citation_author_email" content="pieta.mattila@utu.fi"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2805-0686"/>
  <meta name="citation_reference" content="Baussand J, Carbone A. A Combinatorial Approach to Detect Coevolved Amino Acid Networks in Protein Families of Variable Divergence. PLoS Comput Biol 2009; 5:"/>
  <meta name="citation_reference" content="Kuriyan J. Allostery and coupled sequence variation in nuclear hormone receptors. Cell 2004; 116:354–356"/>
  <meta name="citation_reference" content="Fares MA, McNally D. CAPS: coevolution analysis using protein sequences. Bioinformatics 2006; 22:2821–2822"/>
  <meta name="citation_reference" content="Oteri F, Nadalin F, Champeimont R, et al. BIS2Analyzer: a server for co-evolution analysis of conserved protein families. Nucleic Acids Research 2017; 45:W307–W314"/>
  <meta name="citation_reference" content="Wang S, Sun S, Li Z, et al. Accurate De Novo Prediction of Protein Contact Map by Ultra-Deep Learning Model. PLOS Computational Biology 2017; 13:e1005324"/>
  <meta name="citation_reference" content="Morcos F, Pagnani A, Lunt B, et al. Direct-coupling analysis of residue coevolution captures native contacts across many protein families. Proc. Natl. Acad. Sci. U.S.A. 2011; 108:E1293–1301"/>
  <meta name="citation_reference" content="Hopf TA, Green AG, Schubert B, et al. The EVcouplings Python framework for coevolutionary sequence analysis. Bioinformatics 2019; 35:1582–1584"/>
  <meta name="citation_reference" content="Ovchinnikov S, Kamisetty H, Baker D. Robust and accurate prediction of residue–residue interactions across protein interfaces using evolutionary information. eLife 2014; 3:e02030"/>
  <meta name="citation_reference" content="Simonetti FL, Teppa E, Chernomoretz A, et al. MISTIC: Mutual information server to infer coevolution. Nucleic Acids Res. 2013; 41:W8–14"/>
  <meta name="citation_reference" content="Wang Y, Correa Marrero M, Medema MH, et al. Coevolution-based prediction of protein–protein interactions in polyketide biosynthetic assembly lines. Bioinformatics 2020; 36:4846–4853"/>
  <meta name="citation_reference" content="Fares MA, Travers SAA. A Novel Method for Detecting Intramolecular Coevolution: Adding a Further Dimension to Selective Constraints Analyses. Genetics 2006; 173:9–23"/>
  <meta name="citation_reference" content="Cong Q, Anishchenko I, Ovchinnikov S, et al. Protein interaction networks revealed by proteome coevolution. Science 2019; 365:185–189"/>
  <meta name="citation_reference" content="Travers SAA, Fares MA. Functional Coevolutionary Networks of the Hsp70–Hop–Hsp90 System Revealed through Computational Analyses. Molecular Biology and Evolution 2007; 24:1032–1044"/>
  <meta name="citation_reference" content="Huang Y, Temperley ND, Ren L, et al. Molecular evolution of the vertebrate TLR1 gene family - a complex history of gene duplication, gene conversion, positive selection and co-evolution. BMC Evol Biol 2011; 11:149"/>
  <meta name="citation_reference" content="Ruiz-González MX, Fares MA. Coevolution analyses illuminate the dependencies between amino acid sites in the chaperonin system GroES-L. BMC Evol Biol 2013; 13:156"/>
  <meta name="citation_reference" content="Petrov P, Syrjänen R, Smith J, et al. Characterization of the Avian Trojan Gene Family Reveals Contrasting Evolutionary Constraints. PLoS ONE 2015; 10:e0121672"/>
  <meta name="citation_reference" content="Champeimont R, Laine E, Hu S-W, et al. Coevolution analysis of Hepatitis C virus genome to identify the structural and functional dependency network of viral proteins. Sci Rep 2016; 6:26401"/>
  <meta name="citation_reference" content="Awoniyi LO, Šuštar V, Hernández-Pérez S, et al. APEX2 proximity biotinylation reveals protein dynamics triggered by B cell receptor activation. bioRxiv 2020; 2020.09.29.318766"/>
  <meta name="citation_reference" content="Consortium TU. UniProt: a worldwide hub of protein knowledge. Nucleic Acids Res 2019; 47:D506–D515"/>
  <meta name="citation_reference" content="Kumar S, Stecher G, Suleski M, et al. TimeTree: A Resource for Timelines, Timetrees, and Divergence Times. Mol. Biol. Evol. 2017; 34:1812–1819"/>
  <meta name="citation_reference" content="Kriventseva EV, Kuznetsov D, Tegenfeldt F, et al. OrthoDB v10: sampling the diversity of animal, plant, fungal, protist, bacterial and viral genomes for evolutionary and functional annotations of orthologs. Nucleic Acids Res 2019; 47:D807–D811"/>
  <meta name="citation_reference" content="Camacho C, Coulouris G, Avagyan V, et al. BLAST+: architecture and applications. BMC Bioinformatics 2009; 10:421"/>
  <meta name="citation_reference" content="Sela I, Ashkenazy H, Katoh K, et al. GUIDANCE2: accurate detection of unreliable alignment regions accounting for the uncertainty of multiple parameters. Nucleic Acids Res 2015; 43:W7–W14"/>
  <meta name="citation_reference" content="Wong KM, Suchard MA, Huelsenbeck JP. Alignment uncertainty and genomic analysis. Science 2008; 319:473–476"/>
  <meta name="citation_reference" content="Katoh K, Standley DM. MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability. Mol Biol Evol 2013; 30:772–780"/>
  <meta name="citation_reference" content="Edgar RC. MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res. 2004; 32:1792–1797"/>
  <meta name="citation_reference" content="Löytynoja A. Phylogeny-aware alignment with PRANK. Methods Mol. Biol. 2014; 1079:155–170"/>
  <meta name="citation_reference" content="Castresana J. Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol. Biol. Evol. 2000; 17:540–552"/>
  <meta name="citation_reference" content="Guindon S, Dufayard J-F, Lefort V, et al. New Algorithms and Methods to Estimate Maximum-Likelihood Phylogenies: Assessing the Performance of PhyML 3.0. Syst Biol 2010; 59:307–321"/>
  <meta name="citation_reference" content="Vilella AJ, Severin J, Ureta-Vidal A, et al. EnsemblCompara GeneTrees: Complete, duplication-aware phylogenetic trees in vertebrates. Genome Res 2009; 19:327–335"/>
  <meta name="citation_reference" content="Tange O. GNU Parallel 20201222 (‘Vaccine’). 2020;"/>
  <meta name="citation_reference" content="Shen W, Le S, Li Y, et al. SeqKit: A Cross-Platform and Ultrafast Toolkit for FASTA/Q File Manipulation. PLOS ONE 2016; 11:e0163962"/>
  <meta name="citation_reference" content="R: The R Project for Statistical Computing."/>
  <meta name="citation_reference" content="Shannon P, Markiel A, Ozier O, et al. Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks. Genome Res 2003; 13:2498–2504"/>
  <meta name="citation_reference" content="Blohm P, Frishman G, Smialowski P, et al. Negatome 2.0: a database of non-interacting proteins derived by literature mining, manual annotation and protein structure analysis. Nucleic Acids Res 2014; 42:D396–400"/>
  <meta name="citation_reference" content="Giurgiu M, Reinhard J, Brauner B, et al. CORUM: the comprehensive resource of mammalian protein complexes-2019. Nucleic Acids Res 2019; 47:D559–D563"/>
  <meta name="citation_reference" content="Lassmann T, Sonnhammer ELL. Kalign, Kalignvu and Mumsa: web servers for multiple sequence alignment. Nucleic Acids Research 2006; 34:W596–W599"/>
  <meta name="citation_reference" content="Consortium TU. UniProt: a worldwide hub of protein knowledge. Nucleic Acids Res 2019; 47:D506–D515"/>
  <meta name="citation_reference" content="Moreno-Morcillo M, Grande-García A, Ruiz-Ramos A, et al. Structural Insight into the Core of CAD, the Multifunctional Protein Leading De Novo Pyrimidine Biosynthesis. Structure 2017; 25:912–923.e5"/>
  <meta name="citation_reference" content="Thompson JD, Linard B, Lecompte O, et al. A Comprehensive Benchmark Study of Multiple Sequence Alignment Methods: Current Challenges and Future Perspectives. PLOS ONE 2011; 6:e18093"/>
  <meta name="citation_reference" content="Notredame C, Higgins DG, Heringa J. T-Coffee: A novel method for fast and accurate multiple sequence alignment. J Mol Biol 2000; 302:205–217"/>
  <meta name="citation_reference" content="Sievers F, Higgins DG. Clustal Omega for making accurate alignments of many protein sequences. Protein Science 2018; 27:135–145"/>
  <meta name="citation_reference" content="Stamatakis A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics 2014; 30:1312–1313"/>
  <meta name="citation_reference" content="Ronquist F, Teslenko M, van der Mark P, et al. MrBayes 3.2: efficient Bayesian phylogenetic inference and model choice across a large model space. Syst. Biol. 2012; 61:539–542"/>
  <meta name="citation_reference" content="Nguyen L-T, Schmidt HA, von Haeseler A, et al. IQ-TREE: A Fast and Effective Stochastic Algorithm for Estimating Maximum-Likelihood Phylogenies. Molecular Biology and Evolution 2015; 32:268–274"/>
  <meta name="citation_reference" content="Anand S, Foot N, Ang C-S, et al. Arrestin-Domain Containing Protein 1 (Arrdc1) Regulates the Protein Cargo and Release of Extracellular Vesicles. PROTEOMICS 2018; 18:1800266"/>
  <meta name="citation_reference" content="Carrey EA, Dietz C, Glubb DM, et al. Detection and location of the enzymes of de novo pyrimidine biosynthesis in mammalian spermatozoa. Reproduction 2002; 123:757–768"/>
  <meta name="citation_reference" content="Puusepp S, Kovacs-Nagy R, Alhaddad B, et al. Compound heterozygous SPATA5 variants in four families and functional studies of SPATA5 deficiency. Eur J Hum Genet 2018; 26:407–419"/>
  <meta name="citation_reference" content="Amstalden van Hove ER, Smith DF, Heeren RMA. A concise review of mass spectrometry imaging. J Chromatogr A 2010; 1217:3946–3954"/>
  <meta name="citation_reference" content="Letunic I, Bork P. 20 years of the SMART protein domain annotation resource. Nucleic Acids Res 2018; 46:D493–D496"/>
  <meta name="citation_reference" content="Mistry J, Chuguransky S, Williams L, et al. Pfam: The protein families database in 2021. Nucleic Acids Research 2021; 49:D412–D419"/>
  <meta name="citation_reference" content="Erdős G, Pajkos M, Dosztányi Z. IUPred3: prediction of protein disorder enhanced with unambiguous experimental annotation and visualization of evolutionary conservation. Nucleic Acids Res 2021; 49:W297–W303"/>
  <meta name="citation_reference" content="Yu G, Wang L-G, Han Y, et al. clusterProfiler: an R Package for Comparing Biological Themes Among Gene Clusters. OMICS: A Journal of Integrative Biology 2012; 16:284–287"/>
  <meta name="citation_reference" content="Hulsen T, de Vlieg J, Alkema W. BioVenn – a web application for the comparison and visualization of biological lists using area-proportional Venn diagrams. BMC Genomics 2008; 9:488"/>
  <meta name="citation_reference" content="Li M, Li D, Tang Y, et al. CytoCluster: A Cytoscape Plugin for Cluster Analysis and Visualization of Biological Networks. International Journal of Molecular Sciences 2017; 18:1880"/>
  <meta name="twitter:title" content="AutoCoEv – a high-throughput in silico pipeline for predicting inter-protein co-evolution"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Protein-protein communications govern cellular processes via complex regulatory networks, that are still far from being understood. Thus, identifying novel interactions between proteins can significantly facilitate our comprehension of the mechanistic principles of protein functions. Co-evolution between proteins is a sign of functional communication and, as such, provides a powerful approach to search for novel direct or indirect molecular partners. However, evolutionary analysis of large arrays of proteins, in silico, is a highly time-consuming effort, which has limited the usage of this method to protein pairs or small protein groups. Here, we developed AutoCoEv, a user-friendly computational pipeline for the search of co-evolution between a large number of proteins. By driving 15 individual programs, culminating in CAPS2 as the software for detecting co-evolution, AutoCoEv achieves seamless automation and parallelization of the workflow. Importantly, we provide a patch to CAPS2 source code to strengthen its statistical output, allowing for multiple comparisons correction and enhanced analysis of the results. We apply the pipeline to inspect co-evolution among 324 proteins identified to locate at the vicinity of the lipid rafts of B lymphocytes. We successfully detected multiple strong coevolutionary relations between the proteins, predicting many novel partners and previously unidentified clusters of functionally related molecules. We conclude that AutoCoEv, available at <https://github.com/mattilalab/autocoev>, can be used to predict functional interactions from large datasets in a time and cost-efficient manner.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="AutoCoEv – a high-throughput in silico pipeline for predicting inter-protein co-evolution"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.09.29.315374v4"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Protein-protein communications govern cellular processes via complex regulatory networks, that are still far from being understood. Thus, identifying novel interactions between proteins can significantly facilitate our comprehension of the mechanistic principles of protein functions. Co-evolution between proteins is a sign of functional communication and, as such, provides a powerful approach to search for novel direct or indirect molecular partners. However, evolutionary analysis of large arrays of proteins, in silico, is a highly time-consuming effort, which has limited the usage of this method to protein pairs or small protein groups. Here, we developed AutoCoEv, a user-friendly computational pipeline for the search of co-evolution between a large number of proteins. By driving 15 individual programs, culminating in CAPS2 as the software for detecting co-evolution, AutoCoEv achieves seamless automation and parallelization of the workflow. Importantly, we provide a patch to CAPS2 source code to strengthen its statistical output, allowing for multiple comparisons correction and enhanced analysis of the results. We apply the pipeline to inspect co-evolution among 324 proteins identified to locate at the vicinity of the lipid rafts of B lymphocytes. We successfully detected multiple strong coevolutionary relations between the proteins, predicting many novel partners and previously unidentified clusters of functionally related molecules. We conclude that AutoCoEv, available at <https://github.com/mattilalab/autocoev>, can be used to predict functional interactions from large datasets in a time and cost-efficient manner.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-03-04"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.09.29.315374v4"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2431660"/>
  <title>
   AutoCoEv – a high-throughput in silico pipeline for predicting inter-protein co-evolution | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"oaIV8q-JmhsVhkH0EfbTdvb8aMd_nJXJcp28CFgh7CE"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2431660","apath":"\/biorxiv\/early\/2022\/03\/04\/2020.09.29.315374.atom","pisa":"biorxiv;2020.09.29.315374v4","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.09.29.315374v4"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.09.29.315374v4","title":"AutoCoEv \u2013 a high-throughput in silico pipeline for predicting inter-protein co-evolution","identifier":"node\/2431660"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rAia1F3xUb1JVoGIuxTJtVk0MpW2Zgu3LPGsd1dRIes1xdvSrXuOj9bNRS0tcBODFli\\\\\\\/hjxCpQ6Tt2tbhNmURMAumrXKKAni4lovU22D6hIYaSB1+vRb5cMmkouEdO+4xlCdTENngEx12ejD0587H10EM4WPsMY2imjPzWpMwZHDFl8r7hoBjMZt54COg+u7TmyyPuJUSwIsHqWVgOFxOdTdzzeqDmpfckxMbHDF+6NqWCBh46HbcqCnxPL0NUJFHRQgBaW6l0CIS3zrqORwAaFrCHYcUYk9AL5pPLQivi8LqA8ODaVgkRfeH65H0uYB7A5H2C7EPu8T0ChAF3scFE191nv\\\\\\\/LjpYUQ2Vs+3zawMSD30ylaR+brWslrNOL6hcfxbffq3W86UvSCAjSX6eJ+5IQ9doGM7rnN\\\\\\\/uLFZEfO19xmUgefX1rYuDAal7VBJtNjnIrsQkXqt6EEfL0igS\\\\\\\/+KJkg+UWofMNJ+imux+EwLqaMvznLivGLyGnK0eVCXlWwWDCM8ATOKK+Rl7yn4GmVE2ysquoYkbX9m8S4K5BGpQdjrIhvQBCTDUkbgEGoFCrQSDJtCGOb58iwwtLD+tQjcMvBLWVL40l9qbc5gElKSthhm2DkEcJxrAjJNbuIzbqoQNagmPShiF1G3hNUnyiY71rTYFVAqeBwsmR4+ZtUXJHWMlegtHfDg\\\\\\\/Amdio8\\\\\\\/beb6pUP8kSGjNT7KMnIccN91iI59mVlcltuhsZMIgxYMrA7xchrHBKXHmgrT+N7qaCRtfVB52lixK2f2m\\\\\\\/EzvCEuwp5Ozwc\\\\\\\/SU11BvKAo0EEnnceFpmFYdT+AXoBP36HAl9TL3gnE5TbFREZNXhigOIUf8OPz\\\\\\\/tphvJU9EMSa1DHUckheRmoJyBDGlysn8CZb+HZITwW6MiHACs5ZJ\\\\\\\/PC31KNZ+vgA9rUp8H+8slPrOjFBmJHcbQrlm+bSTQmvMreqeokpK5u+3LXpDzVSeh2smzZPa58DKKD7J2M239TLey1PRUh15S597Oc\\\\\\\/hg33+1q3lXQkmCbR+3BTQEBWjSTs78I8HHlOKVEiBTxT9hharjoQ9KXskguLInmd6YfY\\\\\\\/aosiXSmim11usGYVzZj1rmztCK8crrhUOV23pF28mwJQlP3u57\\\\\\\/528s7IxGtn4ymZRZVDsG8oEznOBZ\\\\\\\/hQZyZSAu0JlKd53MgXRrbhoAjs0Nobgzj1hziH+jSnWcneYeFvW+Ucy+C7FZ9hNrVlfz9w7+RVL7Ho5EgCZXujbnrRVM6VGQWSQ38umg15YgrFETlaAYiKg55L0RIrHVti2x8L5ZmkjgH4jPeT2NKzIcpo5Nd2lvzjr2JvHaN98\\\\\\\/Si+2niGVMgzF4lzeWIBoNntKZrVL3GPwHSQFgkZMhmTURbu5Y=\\\u0022,\\\u0022iv\\\u0022:\\\u0022UZjf12cvGQqms3s7FzVhSA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00226794594462c0918f743dee13bd72a0faba05b1701854e5d9be88430eac0f2f98\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.09.29.315374v4.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2431660 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.09.29.315374v4.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_414660478">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_414660478" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_75729790" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-VWwgXMmf98ui3InZr0rHf2dwt30_KTpRi31F-2qTYWw"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2431660" data-node-nid="2431660" id="node-2431660--2332902980" data-pisa="biorxiv;2020.09.29.315374v4" data-pisa-master="biorxiv;2020.09.29.315374" data-apath="/biorxiv/early/2022/03/04/2020.09.29.315374.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    AutoCoEv – a high-throughput in silico pipeline for predicting inter-protein co-evolution
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0001-5551-8032" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Petar B.
                      </span>
                      <span class="nlm-surname">
                       Petrov
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0001-7809-3268" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Luqman O.
                      </span>
                      <span class="nlm-surname">
                       Awoniyi
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Vid
                      </span>
                      <span class="nlm-surname">
                       Šuštar
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       M. Özge
                      </span>
                      <span class="nlm-surname">
                       Balcı
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4">
                      <a href="http://orcid.org/0000-0003-2805-0686" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Pieta K.
                      </span>
                      <span class="nlm-surname">
                       Mattila
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.09.29.315374
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2431660--2332902980" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Petar B. Petrov
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Institute of Biomedicine and MediCity Research Laboratories, University of Turku
                       </span>
                       , Turku,
                       <span class="nlm-country">
                        Finland
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Turku Bioscience, University of Turku and Åbo Akademi University
                       </span>
                       , Turku,
                       <span class="nlm-country">
                        Finland
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Petar%2BB.%2BPetrov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Petrov%20PB&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3APetar%2BB.%2BPetrov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0001-5551-8032" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Petar B. Petrov
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:petar.petrov@utu.fi" class="" data-icon-position="" data-hide-link-title="0">
                        petar.petrov@utu.fi
                       </a>
                       <a href="mailto:pieta.mattila@utu.fi" class="" data-icon-position="" data-hide-link-title="0">
                        pieta.mattila@utu.fi
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Luqman O. Awoniyi
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Institute of Biomedicine and MediCity Research Laboratories, University of Turku
                       </span>
                       , Turku,
                       <span class="nlm-country">
                        Finland
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Turku Bioscience, University of Turku and Åbo Akademi University
                       </span>
                       , Turku,
                       <span class="nlm-country">
                        Finland
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Luqman%2BO.%2BAwoniyi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Awoniyi%20LO&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ALuqman%2BO.%2BAwoniyi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-7809-3268" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Luqman O. Awoniyi
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Vid Šuštar
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Turku Bioscience, University of Turku and Åbo Akademi University
                       </span>
                       , Turku,
                       <span class="nlm-country">
                        Finland
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Vid%2B%C5%A0u%C5%A1tar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=%C5%A0u%C5%A1tar%20V&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AVid%2B%25C5%25A0u%25C5%25A1tar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     M. Özge Balcı
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Institute of Biomedicine and MediCity Research Laboratories, University of Turku
                       </span>
                       , Turku,
                       <span class="nlm-country">
                        Finland
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Turku Bioscience, University of Turku and Åbo Akademi University
                       </span>
                       , Turku,
                       <span class="nlm-country">
                        Finland
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=M.%2B%C3%96zge%2BBalc%C4%B1%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Balc%C4%B1%20MZ&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AM.%2B%25C3%2596zge%2BBalc%25C4%25B1%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Pieta K. Mattila
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Institute of Biomedicine and MediCity Research Laboratories, University of Turku
                       </span>
                       , Turku,
                       <span class="nlm-country">
                        Finland
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Turku Bioscience, University of Turku and Åbo Akademi University
                       </span>
                       , Turku,
                       <span class="nlm-country">
                        Finland
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Pieta%2BK.%2BMattila%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Mattila%20PK&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3APieta%2BK.%2BMattila%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0003-2805-0686" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Pieta K. Mattila
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:petar.petrov@utu.fi" class="" data-icon-position="" data-hide-link-title="0">
                        petar.petrov@utu.fi
                       </a>
                       <a href="mailto:pieta.mattila@utu.fi" class="" data-icon-position="" data-hide-link-title="0">
                        pieta.mattila@utu.fi
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.09.29.315374v4" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2431660" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2431660/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.09.29.315374v4.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2431660" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2431660/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.09.29.315374v4.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2431660" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2431660/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.09.29.315374v4.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2431660" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2431660/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.09.29.315374v4.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2431660" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2431660/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.09.29.315374v4.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:2431660" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:2431660/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.09.29.315374v4.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2431660" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2431660/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2>
                             Abstract
                            </h2>
                            <p id="p-2">
                             Protein-protein communications govern cellular processes via complex regulatory networks, that are still far from being understood. Thus, identifying novel interactions between proteins can significantly facilitate our comprehension of the mechanistic principles of protein functions. Co-evolution between proteins is a sign of functional communication and, as such, provides a powerful approach to search for novel direct or indirect molecular partners. However, evolutionary analysis of large arrays of proteins, in silico, is a highly time-consuming effort, which has limited the usage of this method to protein pairs or small protein groups. Here, we developed AutoCoEv, a user-friendly computational pipeline for the search of co-evolution between a large number of proteins. By driving 15 individual programs, culminating in CAPS2 as the software for detecting co-evolution, AutoCoEv achieves seamless automation and parallelization of the workflow. Importantly, we provide a patch to CAPS2 source code to strengthen its statistical output, allowing for multiple comparisons correction and enhanced analysis of the results. We apply the pipeline to inspect co-evolution among 324 proteins identified to locate at the vicinity of the lipid rafts of B lymphocytes. We successfully detected multiple strong coevolutionary relations between the proteins, predicting many novel partners and previously unidentified clusters of functionally related molecules. We conclude that AutoCoEv, available at
                             <a href="https://github.com/mattilalab/autocoev">
                              https://github.com/mattilalab/autocoev
                             </a>
                             , can be used to predict functional interactions from large datasets in a time and cost-efficient manner.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             1. Introduction
                            </h2>
                            <p id="p-6">
                             The biological function of proteins is carried out through association and communication with various molecules, the majority of which are other proteins. Thus, screening for novel interactions, either direct or indirect, is of high importance for deciphering the complexity of protein networks. It has been shown that relations between proteins can be extrapolated from the evolutionary history of their genes, via in silico analysis of co-evolution [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ,
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ].
                            </p>
                            <p id="p-7">
                             The evolution of proteins is influenced by structural and functional constraints between amino acids, enforcing their adaptation in a concerted manner. Detecting intra- or inter-molecular co-evolution is regarded as a sign of functional co-dependence between residues within the same protein, or between sites belonging to different partners, respectively [
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ]. Various computational approaches for prediction have been described, among which are BIS2 [
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ], ContactMap [
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ], DCA [
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ], Evcouplings [
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             ], GREMLIN [
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ], MISTIC [
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ], PKSpop [
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ] and CAPS2 [
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ]. Notably, a comprehensive large-scale study was shown recently by Cong et al for bacterial proteome [
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ]. However, many of the searches for inter-protein co-evolution have been confined to a relatively small number of partners, where an existing correlation has been initially anticipated [
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             –
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             ]. If applied to large datasets, such computational approaches would demand a high degree of automation, an issue that we successfully address in this work.
                            </p>
                            <p id="p-8">
                             Here, we developed an automated computational pipeline called AutoCoEv, for the large-scale screening for protein interactions, that is user-friendly and ready to use by the broader public. In the center of the workflow is CAPS2 (Coevolution Analysis using Protein Sequences 2) software, that compares the evolutionary rates between sites in the form of their correlated variance [
                             <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ]. By driving 15 programs, AutoCoEv achieves a high level of automation and flexibility, as well as, processes parallelization, enabling the analysis of hundreds of proteins on a regular computer. We demonstrate the performance of the pipeline by analyzing 324 lymphocyte lipid raft resident proteins [
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ], identified in a proximity biotinylation screen, for their potential functional relation.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             2. Implementation
                            </h2>
                            <p id="p-9">
                             The preparation pipeline for most coevolutionary analyses has a relatively simple concept. Typically, for each protein of interest, a multiple sequence alignment (MSA) is produced from its orthologues in different species, optionally combined with a phylogenetic tree. However, this process requires the correct identification of orthologues, their sequences retrieval and high-quality alignment, all tied together by various filtrations and file format conversions. Automating these steps also necessitates a robust quality-check during and after the process, all of which can present significant challenges if done manually. While developing AutoCoEv we paid significant attention to incorporate ways to evaluate the quality of the key preparatory steps and, finally, to evaluate the results for their robustness.
                            </p>
                            <div id="sec-3" class="subsection">
                             <h3>
                              2.1 Command line interface, configuration and input
                             </h3>
                             <p id="p-10">
                              AutoCoEv is written in BASH and offers a simple menu-driven command line interface (CLI), in which the individual steps are enumerated (
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Figure 1a
                              </a>
                              ). Options for the programs that AutoCoEv drives, as well as filtering parameters, are configured in a single file (settings.conf), described in detail in the manual distributed with the script. Once configuration has been set, simply going through the steps consecutively will conduct the work-flow in an automated manner.
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F1.large.jpg?width=800&height=600&carousel=1" title="AutoCoEv pipeline. A) Menu overview. Left: Steps 1-7 download, check and extract databases; Steps 8-10 index and process the databases; Right: Steps 1-3 deal with homologous sequences retrieval; steps 4-5 carry out the identification of most appropriate orthologues; step 6 calls Guidance to exclude sequences that are too divergent; steps 7-9 create the MSA, phylogenetic trees and protein pairwise combinations; steps 10-11 run parallelized CAPS for each unique protein pair combination “bidirectionally”; step 12 processes results and does statistical analyses. B) Pipeline overview. Yellow. Reading the user-provided lists of proteins of interest and species to be searched, the script communicates between databases to extract genes (ODB) and orthologous groups (OG) identifiers (ID). Homologous sequences are then blasted against the UniProt sequences from the reference organism (e.g. mouse or human) in order to prepare a FASTA list of most appropriate orthologues. Before MSA, orthologues are assessed by Guidance and too divergent ones are removed; Orange. Orthologues are aligned by selected method (MAFFT, MUSCLE or PRANK) and scanned by Gblocks, to report regions of low quality. PhyML calculates trees from the MSA generated in the previous step, optionally using an external tree as a guide; Green. Create all unique protein pairs in folders, each folder having two sub-folders for MSA and (if prepared by PhyML) trees. Blue: CAPS2 is run for each protein pair folder in a parallelized fashion via GNU/Parallel If co-evolution is detected, CAPS2 is run again, this time “reversing” the protein load order (e.g. A vs B followed by B vs A). Purple. The output in each pairs folder is inspected and processed, followed by FDR correction of p-values and Chi squared test. Finally, the results are prepared as a table ready for the network analysis by Cytoscape." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2055454938" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">AutoCoEv pipeline.</span> <strong>A)</strong> Menu overview. <strong>Left</strong>: Steps 1-7 download, check and extract databases; Steps 8-10 index and process the databases; <strong>Right</strong>: Steps 1-3 deal with homologous sequences retrieval; steps 4-5 carry out the identification of most appropriate orthologues; step 6 calls Guidance to exclude sequences that are too divergent; steps 7-9 create the MSA, phylogenetic trees and protein pairwise combinations; steps 10-11 run parallelized CAPS for each unique protein pair combination “bidirectionally”; step 12 processes results and does statistical analyses. <strong>B)</strong> Pipeline overview. <strong>Yellow</strong>. Reading the user-provided lists of proteins of interest and species to be searched, the script communicates between databases to extract genes (ODB) and orthologous groups (OG) identifiers (ID). Homologous sequences are then blasted against the UniProt sequences from the reference organism (e.g. mouse or human) in order to prepare a FASTA list of most appropriate orthologues. Before MSA, orthologues are assessed by Guidance and too divergent ones are removed; <strong>Orange</strong>. Orthologues are aligned by selected method (MAFFT, MUSCLE or PRANK) and scanned by Gblocks, to report regions of low quality. PhyML calculates trees from the MSA generated in the previous step, optionally using an external tree as a guide; <strong>Green</strong>. Create all unique protein pairs in folders, each folder having two sub-folders for MSA and (if prepared by PhyML) trees. <strong>Blue</strong>: CAPS2 is run for each protein pair folder in a parallelized fashion via GNU/Parallel If co-evolution is detected, CAPS2 is run again, this time “reversing” the protein load order (e.g. A vs B followed by B vs A). <strong>Purple</strong>. The output in each pairs folder is inspected and processed, followed by FDR correction of p-values and <em>Chi</em> squared test. Finally, the results are prepared as a table ready for the network analysis by Cytoscape.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F1.medium.gif" width="440" height="372"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F1.medium.gif" width="440" height="372"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1.
                               </span>
                               <span class="caption-title">
                                AutoCoEv pipeline.
                               </span>
                               <p id="p-11" class="first-child">
                                <strong>
                                 A)
                                </strong>
                                Menu overview.
                                <strong>
                                 Left
                                </strong>
                                : Steps 1-7 download, check and extract databases; Steps 8-10 index and process the databases;
                                <strong>
                                 Right
                                </strong>
                                : Steps 1-3 deal with homologous sequences retrieval; steps 4-5 carry out the identification of most appropriate orthologues; step 6 calls Guidance to exclude sequences that are too divergent; steps 7-9 create the MSA, phylogenetic trees and protein pairwise combinations; steps 10-11 run parallelized CAPS for each unique protein pair combination “bidirectionally”; step 12 processes results and does statistical analyses.
                                <strong>
                                 B)
                                </strong>
                                Pipeline overview.
                                <strong>
                                 Yellow
                                </strong>
                                . Reading the user-provided lists of proteins of interest and species to be searched, the script communicates between databases to extract genes (ODB) and orthologous groups (OG) identifiers (ID). Homologous sequences are then blasted against the UniProt sequences from the reference organism (e.g. mouse or human) in order to prepare a FASTA list of most appropriate orthologues. Before MSA, orthologues are assessed by Guidance and too divergent ones are removed;
                                <strong>
                                 Orange
                                </strong>
                                . Orthologues are aligned by selected method (MAFFT, MUSCLE or PRANK) and scanned by Gblocks, to report regions of low quality. PhyML calculates trees from the MSA generated in the previous step, optionally using an external tree as a guide;
                                <strong>
                                 Green
                                </strong>
                                . Create all unique protein pairs in folders, each folder having two sub-folders for MSA and (if prepared by PhyML) trees.
                                <strong>
                                 Blue
                                </strong>
                                : CAPS2 is run for each protein pair folder in a parallelized fashion via GNU/Parallel If co-evolution is detected, CAPS2 is run again, this time “reversing” the protein load order (e.g. A vs B followed by B vs A).
                                <strong>
                                 Purple
                                </strong>
                                . The output in each pairs folder is inspected and processed, followed by FDR correction of p-values and Chi squared test. Finally, the results are prepared as a table ready for the network analysis by Cytoscape.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-12">
                              As an input, AutoCoEv requires a list of proteins with their UniProt identifiers [
                              <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                               19
                              </a>
                              ] and a list of species, for which orthologues will be searched. Optionally, a phylogenetic tree may be provided from an external source, such as TimeTree [
                              <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              ], to be used as a guide when trees are calculated from MSA (see later, Multiple sequence alignments and trees). Upon start, AutoCoEv offers to download the required databases from OrthoDB [
                              <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                               21
                              </a>
                              ] and to run initial preparations, such as FASTA database indexing (
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Figure 1a
                              </a>
                              , left). Once databases are in place and input files are loaded, the pipeline proceeds to the main menu that carries out the work-flow (
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Figure 1a
                              </a>
                              , right).
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              2.2 Identification of orthologues
                             </h3>
                             <p id="p-13">
                              For each protein of in the user-provided list, AutoCoEv consults with OrthoDB, searching for homologues from the species of interest (
                              <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                               Figure 1b
                              </a>
                              , yellow panel). The script matches the UniProt ID of each protein to its OrthoDB ID, then extracts its unique orthologues group (OG) ID at a given level of organisms (e.g. Eukaryota, Metazoa, Vertebrata, Tetrapoda, Mammalia). This level, or node, is specified by the user and depends on the species for which orthologues are searched. The script will report proteins with missing OG IDs at OrthoDB, as well as, species for which no orthologue was found.
                             </p>
                             <p id="p-14">
                              AutoCoEv prepares a list of homologues for each protein, however, there may be more than one per species, for example due to alternative splicing or gene duplication. Therefore, the homologues from each species are compared to the UniProt sequence of the user-provided protein, by pBLAST [
                              <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                              ]. After this reciprocal BLAST against the “reference” organism, AutoCoEv selects the best hit per species. Importantly, users have the option to omit even the best hits if they do not pass certain criteria, such as identity to the reference sequence and alignment gaps. With this filtering step, the script avoids the inclusion of erroneous or not complete sequences that can skew the MSA in the next step. As a result, each protein holds a collection of automatically curated orthologous sequences, one per species. Before the MSA step (next), AutoCoEv additionally consults with program Guidance [
                              <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ] to assess whether some orthologues are too divergent from the rest. The presence of such sequences can, again, affect the robustness of the alignment in the next step and it may be desirable to omit them.
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              2.3 Multiple sequence alignment and trees
                             </h3>
                             <p id="p-15">
                              CAPS2 detects co-evolution between two proteins by extrapolating from their MSAs, hence the quality of the alignments is of crucial importance [
                              <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                               24
                              </a>
                              ]. For MSA creation (
                              <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                               Figure 1b
                              </a>
                              , orange panel), AutoCoEv offers a choice of three widely-used and accurate programs: MAFFT (Multiple Alignment using Fast Fourier Transform) [
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                              ], MUSCLE (MUltiple Sequence Comparison by Log-Expectation) [
                              <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                               26
                              </a>
                              ] and PRANK (Probabilistic Alignment Kit) [
                              <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                               27
                              </a>
                              ] (
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Figure 2B
                              </a>
                              ). Different MAFFT aliases are supported (e.g L-INS-i, E-INS-i, G-INS-i), while for PRANK an external phylogenetic tree (e.g. obtained from TimeTree) can be specified as a guide. After MSAs are generated, the script inspects them by program Gblocks [
                              <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                               28
                              </a>
                              ], to assess the quality of the alignments regions. This information is reported in the final output, allowing the user to filter out co-evolving amino acids that belong to poorly aligned columns.
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F2.large.jpg?width=800&height=600&carousel=1" title="Determining AutoCoEv strategy. A) Strategies. Combinations of MSA method and phylogenetic trees calculation methods are referred as “strategies”. B) Negatome database. Percentage of protein pairs for which co-evolution was detected and the total number of co-evolving sites. C) CORUM database. Co-evolution was detected in 5 out of the first 10 complexes. Shown are the two largest. S: number of complex subunits (found in OrthoDB), P: protein pairs detected, E: number of edges (total number of co-evolving amino acids)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2055454938" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Determining AutoCoEv strategy. <strong>A)</strong> Strategies. Combinations of MSA method and phylogenetic trees calculation methods are referred as “strategies”. <strong>B)</strong> Negatome database. Percentage of protein pairs for which co-evolution was detected and the total number of co-evolving sites. C) CORUM database. Co-evolution was detected in 5 out of the first 10 complexes. Shown are the two largest. S: number of complex subunits (found in OrthoDB), P: protein pairs detected, E: number of edges (total number of co-evolving amino acids).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F2.medium.gif" width="440" height="285"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F2.medium.gif" width="440" height="285"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2.
                               </span>
                               <p id="p-16" class="first-child">
                                Determining AutoCoEv strategy.
                                <strong>
                                 A)
                                </strong>
                                Strategies. Combinations of MSA method and phylogenetic trees calculation methods are referred as “strategies”.
                                <strong>
                                 B)
                                </strong>
                                Negatome database. Percentage of protein pairs for which co-evolution was detected and the total number of co-evolving sites. C) CORUM database. Co-evolution was detected in 5 out of the first 10 complexes. Shown are the two largest. S: number of complex subunits (found in OrthoDB), P: protein pairs detected, E: number of edges (total number of co-evolving amino acids).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-17">
                              By default, CAPS2 generates its own BioNJ (neighbor-joining) distance-based phylogenetic trees from the proteins MSAs at runtime. The trees are not made available to the user, therefore we patched the program to print the tree to the output, allowing for inspection. Alternatively, trees calculated by another program can be used, which may improve the sensitivity of CAPS2. If this is preferred, AutoCoEv calls PhyML (Phylogenetic estimation using Maximum Likelihood) [
                              <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                              ] (
                              <a id="xref-fig-1-6" class="xref-fig" href="#F1">
                               Figure 1b
                              </a>
                              , orange panel). An external tree (e.g. from TimeTree) can be specified as a guide, while the generated trees can be rooted by TreeBeST [
                              <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ], by minimizing height.
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              2.4 Detection of inter-protein co-evolution by CAPS2
                             </h3>
                             <p id="p-18">
                              The computational time required for the co-evolution detection presents a major bottleneck, as CAPS2 lacks CPU multi-threading. To overcome this limitation, AutoCoEv runs CAPS2 on individual protein pairs via GNU/Parallel [
                              <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              ], as described below.
                             </p>
                             <p id="p-19">
                              First, AutoCoEv produces all unique pairwise combinations between the proteins from the user-provided list (
                              <a id="xref-fig-1-7" class="xref-fig" href="#F1">
                               Figure 1b
                              </a>
                              , green panel). The script creates an individual folder dedicated to each pair and determines the species where an orthologous sequence was found for both proteins. Species that are not shared by the two proteins have their sequences removed from the MSAs by SeqKit [
                              <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                               32
                              </a>
                              ], and are trimmed from the trees (if PhyML is used) by TreeBeST. This is important, since the presence of too many not shared species seems to deteriorate the stability of CAPS2. On the other hand, too little species result in poor reliability of the coevolution detection [
                              <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                              ], therefore users can specify a minimum threshold of shared species for a protein pair (e.g. 20).
                             </p>
                             <p id="p-20">
                              During the AutoCoEv development, we noticed that the order in which CAPS2 loads its input files, seems to have an effect on the inter-molecular analyses. Therefore, to improve the specificity and reliability of the analysis, we designed our script to run CAPS2 twice, so that those proteins pairs (e.g. A vs B) where co-evolution was detected in the first run get selected for asecond run, this time reversing the order (e.g. B vs A), by slightly renaming the files (
                              <a id="xref-fig-1-8" class="xref-fig" href="#F1">
                               Figure 1b
                              </a>
                              , blue panel). Since CAPS2 loads input files randomly, we additionally patched the program to always load files in alphabetical order. Upon completion of the second run, AutoCoEv extracts the amino acid pairs predicted as co-evolving in both runs.
                             </p>
                             <p id="p-21">
                              AutoCoEv does this for all protein pairs, using GNU/Parallel to spawn multiple instances of CAPS2, each operating in a single proteins pair folder. As a result, the script dramatically speeds up the time of computation.
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              2.5 Post-run processing of the results
                             </h3>
                             <p id="p-22">
                              At run time, CAPS2 uses an α-value threshold (e.g. α = 0.01) for the probability of error in rejecting the null hypothesis (type I error), when significant co-evolving sites are detected. Amino acid pairs that pass the threshold are reported in the results of CAPS2, however the actual p-values of their correlations are not. This poses limitations to statistical analyses, such as control of the false discovery rate (FDR), critical for large datasets. Therefore, we patched CAPS2 to calculate and output p-values when inter-protein co-evolution is searched (see Supplementary information, “P-values of the results”).
                             </p>
                             <p id="p-23">
                              After CAPS2 runs are completed in all protein pair folders, AutoCoEv processes the results in several steps of filtering, sorting and assessment (
                              <a id="xref-fig-1-9" class="xref-fig" href="#F1">
                               Figure 1b
                              </a>
                              , violet panel). Following initial clean-ups, the script calls R [
                              <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                               33
                              </a>
                              ] to produce adjusted p-values of the co-evolving sites from each protein pair. By default, CAPS2 applies a chi squared (χ2) test when more than two proteins are analyzed, based on the number of detected co-evolving amino acids between them. In our pipeline, CAPS2 always runs for just 2 proteins at a time, therefore AutoCoEv replicates the χ
                              <sup>
                               2
                              </sup>
                              -test, when the analyses of all protein pairs are completed (see Supplementary information, “Chi squared test”).
                             </p>
                             <p id="p-24">
                              Results are saved in two spreadsheet files: one containing all individual co-evolving amino acids, while the other summarizes the results per protein pair. Both spreadsheets are ready to import to Cytoscape [
                              <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ] for network visualization and further analysis, such as filtering and cluster analysis.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-8">
                            <h2 class="">
                             3. Application
                            </h2>
                            <p id="p-25">
                             We searched for orthologues from 50 placental mammals (Table S1) in the analyses described below. The choice of methods for generation of the MSAs and phylogenetic trees is critical for evolutionary studies, therefore, we first tested 6 different combinations (
                             <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                              Figure 2a
                             </a>
                             ). We used the Negatome database [
                             <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                              35
                             </a>
                             ] of non-interacting proteins, for which we expect to detect little or no co-evolution.
                            </p>
                            <p id="p-26">
                             We tested 211 protein pairs from mouse (Table S2) and observed that while all strategies predicted very low percentage of protein pairs with co-evolution, the strategies 2 and 3 (MSAs by MAFFT L-INS-i and PRANK with phylogenetic trees automatically generated by CAPS2), yielded the lowest numbers of co-evolving residues (
                             <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                              Figure 2b
                             </a>
                             , right). Using PhyML-generated phylogenetic trees in strategies 4-6 seemed to significantly increase the numbers of detected co-evolving amino acids. As a very low amount of coevolution was expected in this dataset and with the aim to minimize false positive hits, we decided to continue with the more conservative strategies with CAPS2-generated phylogenetic trees. In addition, the strategies 3 and 6 using PRANK showed the smallest fraction of protein pairs for which coevolution was detected in both CAPS2 runs of different directionalities but not on the same amino acids (therefore omitted in the final results), suggesting a better reliability with this MSA method (Supplementary figure S1a). To summarize, we considered that the best specificity was offered by strategy 3: combination of PRANK-generated MSAs with phylogenetic trees automatically calculated by CAPS2 at run-time.
                            </p>
                            <p id="p-27">
                             We then used the CORUM database of known protein complexes [
                             <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                              36
                             </a>
                             ], in order to test the selected strategy 3 on proteins for which co-evolution is expected. We analyzed the 10 largest complexes from mouse (number of subunits > 10), from which we detected co-evolution in 5 (Table S3,
                             <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                              Figure 2c
                             </a>
                             , Supplementary figure S1b). The highest number of co-evolving proteins were detected within the two largest complexes: the Parvulin-associated pre-rRNP complex and Respiratory chain complex. Although it is expected that such multiprotein complexes have gone through remarkable co-evolution, our analysis only assessed the situation in mammals and, thus, is likely to miss the interactions within highly conserved protein domains. Together the analysis of the negatome and the protein complexes pointed towards capabilities of AutoCoEv to predict protein-protein interactions and allowed us to proceed for large-scale data analysis.
                            </p>
                            <div id="sec-9" class="subsection">
                             <h3>
                              3.1 Lipid rafts dataset
                             </h3>
                             <p id="p-28">
                              We used AutoCoEv to predict novel partners in a set of 324 proteins from mouse (Table S4), located at the lipid-raft membrane domains of B cells. The proteins were identified in a preceding study from our research group, by an APEX2 proximity biotinylation-based proteomics analysis [
                              <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                              ].
                             </p>
                             <p id="p-29">
                              Although our Negatome AutoCoEv runs inclined us to pick PRANK for our dataset, we again compared, in this large dataset, the MSAs produced by the other two programs, too. Mumsa [
                              <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                               37
                              </a>
                              ], indicated that all three programs had produced high quality alignments, however the highest scores were clearly assigned to PRANK (Supplementary figure S2a). After screening for too divergent sequences and shared species, we had 46775 unique protein pairs to be tested for co-evolution. Following the CAPS2 double run step with PRANK alignments and automatic trees (strategy 3), we obtained a network of 61 nodes and 282 protein pairs (
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Figure 3a
                              </a>
                              ). The number of predicted co-evolving pairs was significantly lower than that when MUSCLE or MAFFT L-INS-i alignments were used. However, the results obtained with PRANK MSAs had best overlap with the results obtained by the other two methods (Supplementary figure S2b). The MSA quality scores and the best concordance of the co-evolving pairs with those detected by the other strategies (1 and 2), favored PRANK as the alignment method for subsequent analyses.
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F3.large.jpg?width=800&height=600&carousel=1" title="Proteins predicted to co-evolve in the dataset of 324 proteins from the vicinity of the B cell membrane rafts. A) Network analysis of the co-evolving pairs. Number of nodes (N), pairs (P) and edges (E) are indicated. Nodes size and colour reflect betweenness and closeness centrality, respectively. Edges colour corresponds to the co-evolution p-value of each amino acid pair. B) Gene ontology analysis of the network proteins. The cellular processes in which proteins are involved are indicated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2055454938" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Proteins predicted to co-evolve in the dataset of 324 proteins from the vicinity of the B cell membrane rafts. <strong>A)</strong> Network analysis of the co-evolving pairs. Number of nodes (N), pairs (P) and edges (E) are indicated. Nodes size and colour reflect betweenness and closeness centrality, respectively. Edges colour corresponds to the co-evolution p-value of each amino acid pair. <strong>B)</strong> Gene ontology analysis of the network proteins. The cellular processes in which proteins are involved are indicated.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F3.medium.gif" width="285" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F3.medium.gif" width="285" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3.
                               </span>
                               <p id="p-30" class="first-child">
                                Proteins predicted to co-evolve in the dataset of 324 proteins from the vicinity of the B cell membrane rafts.
                                <strong>
                                 A)
                                </strong>
                                Network analysis of the co-evolving pairs. Number of nodes (N), pairs (P) and edges (E) are indicated. Nodes size and colour reflect betweenness and closeness centrality, respectively. Edges colour corresponds to the co-evolution p-value of each amino acid pair.
                                <strong>
                                 B)
                                </strong>
                                Gene ontology analysis of the network proteins. The cellular processes in which proteins are involved are indicated.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-31">
                              The network obtained from the predicted coevolving proteins had several major node hubs (
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Figure 3a
                              </a>
                              ), as defined by their closeness and betweenness centrality, namely Gars (Glycine-tRNA ligase), Cad (Carbamoyl-phosphate synthetase 2, Aspartate trans-carbamylase, and Dihydroorotase), Hnrnpab (Heterogeneous nuclear ribonucleoprotein A/B) and Eif3b (Eukaryotic translation initiation factor 3 subunit B). As reported by UniProt [
                              <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                               38
                              </a>
                              ], all of them are multi-domain proteins, involved in translation, metabolism and transcription regulation. Performing gene ontology (GO) analyses on all 61 proteins indicated that they play a role in a wide range of processes and pathways (
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Figure 3b
                              </a>
                              ), such as in cell division, protein synthesis, cellular response, metabolism, membrane transport and more.
                             </p>
                             <p id="p-32">
                              We sought to single out proteins that are tightly interlinked, in order to suggest potential candidates for further investigation in the wet-lab. We filtered the results by p-value (p < 0.005), alignment region quality (determined by Gblocks) and MSA column gaps (less than 20%), obtaining a smaller network, with an overall organization very similar to the parent (
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Figure 4a
                              </a>
                              ). About 25% of the protein pairs were also found in the STRING database (combined confidence score > 0.15), both before and after the network filtering (
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Figure 4b
                              </a>
                              ). Then, we preformed clustering analysis by CytoCluster (ClusterONE, number of nodes < 20) and distinguished a relatively compact cluster of 18 nodes (
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               Figure 4c
                              </a>
                              ). The cluster incorporated total of 41 proteins pairs, 19 of which were found in STRING (
                              <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                               Figure 4c
                              </a>
                              ).
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F4.large.jpg?width=800&height=600&carousel=1" title="Cluster analysis of the co-evolution network. A) Cluster analysis of the co-evolution network. Edges were filtered by p-value (p" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2055454938" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Cluster analysis of the co-evolution network. <strong>A)</strong> Cluster analysis of the co-evolution network. Edges were filtered by p-value (p<0.005), Gblocks score (good) and MSA column gaps (less than 20%). Nodes that are found in the cluster identified by ClusterONE (C, arrow) are outlined in orange. See Figure 3 legend for nodes size, nodes colour and edge colour. <strong>B)</strong> Protein pairs found in the STRING database. Overlapping circles represent the whole network (N), filtered network (FN, Figure 3a) and cluster (CL, Figure 4c). <strong>C)</strong> The identified cluster. Protein pairs supported also by STRING are shown in green.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F4.medium.gif" width="440" height="304"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F4.medium.gif" width="440" height="304"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4.
                               </span>
                               <p id="p-33" class="first-child">
                                Cluster analysis of the co-evolution network.
                                <strong>
                                 A)
                                </strong>
                                Cluster analysis of the co-evolution network. Edges were filtered by p-value (p<0.005), Gblocks score (good) and MSA column gaps (less than 20%). Nodes that are found in the cluster identified by ClusterONE (C, arrow) are outlined in orange. See
                                <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                                 Figure 3
                                </a>
                                legend for nodes size, nodes colour and edge colour.
                                <strong>
                                 B)
                                </strong>
                                Protein pairs found in the STRING database. Overlapping circles represent the whole network (N), filtered network (FN,
                                <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                                 Figure 3a
                                </a>
                                ) and cluster (CL,
                                <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                                 Figure 4c
                                </a>
                                ).
                                <strong>
                                 C)
                                </strong>
                                The identified cluster. Protein pairs supported also by STRING are shown in green.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-34">
                              The major hub node in the cluster is Cad, a large (243 kDa) protein with multi-catalytic activity, involved in de novo synthesis of pyrimidine [
                              <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                               39
                              </a>
                              ]. Cad was predicted to co-evolve with 10 proteins (Supplementary figure S3a) via 7 amino acids: 20A, 1728L, 1887G, 1892A, 2108S, 2114S and 2160A. The residues were found in its GATase (glutamine amidotransferase), DHOase (dihydroorotase), DRBS (disordered region binding sites) and ATCase (Aspartate transcarbamylase) regions (
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Figure 5a
                              </a>
                              ).
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F5.large.jpg?width=800&height=600&carousel=1" title="Suggested interactions between Cad and the proteins predicted to co-evolve with it. A) Co-evolving sites from Cad. The co-evolving amino acid residues are indicated by dashed lines and plotted onto the protein topology. Proteins co-evolving with each site are shown as mini-clusters. Protein regions are indicated in bold next to the scheme: GATase (glutamine amidotransferase), DHOase (dihydroorotase), DRBS (disordered region binding sites) and ATCase (Aspartate transcarbamylase); domains are indicated within scheme: CPSase (Carbamoyl-phosphate synthase small chain), DDL (D-alanine-D-alanine ligase), MGS (Methylglyoxal synthase), AMH (Amidohydrolase family domain), OTCase (Ornithine carbamoyltransferase, carbamoyl-P binding domain); active sites, phosphorylation sites (Ser, Thr) and metal-binding sites (Mn, Zn) are indicated as lollipops. B) The amino acid residues from Spata5 predicted to co-evolve with Cad. Amino acids are plotted onto the protein topology and paired sites from Cad are indicated. Domains: CDC48 (Cell division protein 48), AAA (ATPases associated with a variety of cellular activities), lid (AAA+ lid domain)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2055454938" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Suggested interactions between Cad and the proteins predicted to co-evolve with it. <strong>A)</strong> Co-evolving sites from Cad. The co-evolving amino acid residues are indicated by dashed lines and plotted onto the protein topology. Proteins co-evolving with each site are shown as mini-clusters. Protein regions are indicated in bold next to the scheme: GATase (glutamine amidotransferase), DHOase (dihydroorotase), DRBS (disordered region binding sites) and ATCase (Aspartate transcarbamylase); domains are indicated within scheme: CPSase (Carbamoyl-phosphate synthase small chain), DDL (D-alanine-D-alanine ligase), MGS (Methylglyoxal synthase), AMH (Amidohydrolase family domain), OTCase (Ornithine carbamoyltransferase, carbamoyl-P binding domain); active sites, phosphorylation sites (Ser, Thr) and metal-binding sites (Mn, Zn) are indicated as lollipops. <strong>B)</strong> The amino acid residues from Spata5 predicted to co-evolve with Cad. Amino acids are plotted onto the protein topology and paired sites from Cad are indicated. Domains: CDC48 (Cell division protein 48), AAA (ATPases associated with a variety of cellular activities), lid (AAA+ lid domain).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F5.medium.gif" width="440" height="215"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F5.medium.gif" width="440" height="215"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/04/2020.09.29.315374/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5.
                               </span>
                               <p id="p-35" class="first-child">
                                Suggested interactions between Cad and the proteins predicted to co-evolve with it.
                                <strong>
                                 A)
                                </strong>
                                Co-evolving sites from Cad. The co-evolving amino acid residues are indicated by dashed lines and plotted onto the protein topology. Proteins co-evolving with each site are shown as mini-clusters. Protein regions are indicated in bold next to the scheme: GATase (glutamine amidotransferase), DHOase (dihydroorotase), DRBS (disordered region binding sites) and ATCase (Aspartate transcarbamylase); domains are indicated within scheme: CPSase (Carbamoyl-phosphate synthase small chain), DDL (D-alanine-D-alanine ligase), MGS (Methylglyoxal synthase), AMH (Amidohydrolase family domain), OTCase (Ornithine carbamoyltransferase, carbamoyl-P binding domain); active sites, phosphorylation sites (Ser, Thr) and metal-binding sites (Mn, Zn) are indicated as lollipops.
                                <strong>
                                 B)
                                </strong>
                                The amino acid residues from Spata5 predicted to co-evolve with Cad. Amino acids are plotted onto the protein topology and paired sites from Cad are indicated. Domains: CDC48 (Cell division protein 48), AAA (ATPases associated with a variety of cellular activities), lid (AAA+ lid domain).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-36">
                              By GO analysis, Cad and its 10 co-evolving proteins were suggested to play roles in angiogenesis, ER, endothelium, membrane transport and translation (Supplementary figure S4). The proteins are predominantly cytoplasmic, nucleus-associated and membrane-associated (Table S5). The ATP-dependent chaperone Spata5 showed highest number of non-overlapping co-evolving residue pairs with Cad: 4 sites co-evolving with 3 sites from Cad (Supplementary figure S3b). They all resided within its first AAA–lid3 ATPase tandem domains, co-evolving with the CPSase domain, DRBS and the OTCase domain from Cad (
                              <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                               Figure 5b
                              </a>
                              ).
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-10">
                            <h2 class="">
                             4. Discussion
                            </h2>
                            <p id="p-37">
                             In this work, we present the AutoCoEv pipeline: an interactive script for the large-scale prediction of inter-protein co-evolution by CAPS2. Searching for signs of co-evolution in silico is a powerful means to predict novel functional interactions between proteins. By default, inter-molecular analysis in CAPS2 was designed for a handful of proteins, or a single pair, as illustrated by its web interface (
                             <a href="http://caps.tcd.ie/caps/">
                              http://caps.tcd.ie/caps/
                             </a>
                             ). However, the availability of CAPS2 for offline use, grants a great deal of flexibility achievable via scripting. Requiring only a list of proteins and a list of species, AutoCoEv automatically performs database searching and identification of orthologous sequences with a best hit. The pipeline offers further automation, parallelization and quality assessment on the subsequent steps and seamlessly achieves the batch processing of hundreds of input proteins. AutoCoEv also contains post-analysis tools enabling efficient analysis and ranking of the results. We propose that automated prediction of co-evolution provides a powerful and affordable tool to facilitate selection of candidates from large protein data sets for further analysis.
                            </p>
                            <p id="p-38">
                             Prediction of co-evolution requires, at first, a collection of protein ortholoques from various species and generation of the MSAs. Already this is a sizable data mining and organizing task data, but is followed by extremely heavy residue-to-residue comparisons of MSAs. The challenging nature of the analysis is illustrated by the waiting times when using co-evolution analysis software that are available on servers. For instance, we used the BIS2, MISTIC, MISTIC2, ContactMap, GREMLIN and DCA servers aiming to analyze a single protein pair: Cad – Spata5. To detect inter-molecular co-evolution, BIS2 requires that the MSAs of the two proteins are concatenated and we did the same for MISTIC(2). The queue/runtime was ~22 hours for MISTIC, while MISTIC2, DCA and BIS2 crashed. GREMLIN and ContactMap did not accept MSA over 1000 and 1100 amino acids, while GREMLIN had the additional warning that 85 jobs were currently running and that our submission “may take forever to complete”. Thus, it is clear that large-scale analyses cannot be practically performed using the server-based tools accessible online. On contrary, AutoCoEv runs locally on Linux, thus avoiding queue waiting times and other limitations that arise when using a public server. The only results we obtained, from MISTIC, were very challenging to interpret, since the program (naturally) detected numerous inter-molecular co-evolving sites, making it virtually impossible to distinguish the inter-molecular co-evolution. For a comparison, running the CAPS2 bidirectional step via AutoCoev took ~48h on an i7-9700KF CPU (8 cores @3.6GHz) with 64GB RAM for 46775 protein pairs. In addition, CAPS2 does not have a MSA-length limitation (e.g. 1000 amino acids), allowing the processing of large proteins, such as Cad. The post-run processing by AutoCoEv provides users with a comprehensive table, that can also be directly imported in Cytoskape for further network analysis.
                            </p>
                            <p id="p-39">
                             Since the choice of alignment software largely depends on the sequences being aligned, our script already drives three of the most widely used MSA programs [
                             <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                              40
                             </a>
                             ], and we consider incorporation of additional methods in the future, such as T-coffee [
                             <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                              41
                             </a>
                             ] and ClustalΩ [
                             <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                              42
                             </a>
                             ]. For our dataset, PRANK appeared to be the most suitable MSA method, however users can choose also from MUSCLE or MAFFT. Using PhyML-calculated trees seemed to increase the sensitivity of CAPS2, something that was undesired in our analyses, as we aimed to minimize false-positives. However, if greater sensitivity is required or if co-evolution is initially expected, for example for known protein complexes, PhyML offers a reliable means for trees calculation outside of CAPS2. In addition, we are also planning to implement AutoCoEv wrappers around RAxML [
                             <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                              43
                             </a>
                             ], MrBayes [
                             <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                              44
                             </a>
                             ] and IQ-TREE [
                             <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                              45
                             </a>
                             ] in the future.
                            </p>
                            <p id="p-40">
                             While developing AutoCoEv, we applied several improvements to CAPS2 which, being an open source program, allows for feature implementation. Our patch making the program report p-values, greatly augments the verbosity of the results and allows for additional statistical tests. Moreover, allowing the user to inspect the phylogenetic trees produced by CAPS2 helps in the assessment of the results. Our observation that the order in which the two input files are loaded seems to matter for the outcome of the results was unexpected. Thus, to increase the confidence of the results, we opted to run the program twice, “bidirectionally”, then extract the residue pairs for which both runs agree. To ensure the repeatability of the process between different computers, we further patched CAPS2 to always sort input by alphabetical/numerical order. We believe this simple workaround greatly improved the specificity and the reliability of the program. Importantly, it should be noted that AutoCoEv is open-source and well amendable for inclusion of other programs, too. We welcome developers of co-evolution analysis programs to consider utilizing the AutoCoEv pipeline and testing their own program in the place of CAPS2, for high-throughput analysis.
                            </p>
                            <p id="p-41">
                             Here, we analyzed hundreds of proteins on a regular computer, while the script is designed to work with even thousands of proteins, provided a high computing power is available. As an example, we focused on Cad, the central node within a compact cluster identified from our network. Cad, together with Eif3b, Fasn, Pfkp and Rnp, have been reported to localize in extracellular vesicles [
                             <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                              46
                             </a>
                             ], supporting the predicted functional association between them. A novel relationship between Cad and Spata5 was suggested by their high number of co-evolving sites. Cad has been shown to locate towards mitochondria of mammalian spermatozoa [
                             <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                              47
                             </a>
                             ], while Spata5, being an ATPase is essential in mitochondrial morphogenesis during early spermatogenesis [
                             <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                              48
                             </a>
                             ]. Therefore, we hypothesis that Cad may provide a pyrimidine nucleotide pool that could stimulate Spata5’s ATPase during spermatogenesis. The proteins discussed here represent only a nominal part of the full myriad of possible candidates for further detailed analyses.
                            </p>
                            <p id="p-42">
                             We trust that AutoCoEv, as an affordable and unbiased in silico analysis, could benefit various large scale protein interaction studies, like imaging mass spectrometry [
                             <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                              49
                             </a>
                             ], by providing another viewpoint to the connections between the proteins and, thus, help in identifying interesting proteins or pathways for further studies.
                            </p>
                           </div>
                           <div class="section" id="sec-11">
                            <h2 class="">
                             4. Materials and methods
                            </h2>
                            <div id="sec-12" class="subsection">
                             <h3>
                              4.1 Availability and required software
                             </h3>
                             <p id="p-43">
                              AutoCoEv is written in BASH and is under MIT license, freely available from the GitHub repository of our group (
                              <a href="https://github.com/mattilalab/autocoev">
                               https://github.com/mattilalab/autocoev
                              </a>
                              ). Development was done on Slackware (
                              <a href="http://www.slackware.com/">
                               http://www.slackware.com/
                              </a>
                              ) and CRUX (
                              <a href="https://crux.nu/">
                               https://crux.nu/
                              </a>
                              ) distributions of GNU/Linux.
                             </p>
                             <p id="p-44">
                              Software tools that AutoCoEv drives and their versions used in our analyses are: CAPS (2.0 patched, see Supplementary Information), Datamash (1.7), Exonerate (2.4.0), Gblocks (0.91b), Guidance (2.02), MAFFT (7.471), MUSCLE (3.8.1551), NCBI BLAST+ (2.12.0), PRANK (170427), Parallel (20211122), PhyML (3.3.20200621), R (4.1.2), SeqKit (0.16.1), squizz (0.99d) and TreeBeST (git:347fa82, Ensembl modifications).
                             </p>
                             <p id="p-45">
                              See AutoCoEv on GitHub for details, manual, as well as for instructions for setting up the pipeline on Ubuntu (
                              <a href="https://ubuntu.com/">
                               https://ubuntu.com/
                              </a>
                              ) or Debian (
                              <a href="https://www.debian.org/">
                               https://www.debian.org/
                              </a>
                              ). We provide a pre-compiled, static binary of CAPS2 with our patches applied, and a virtual machine image with all requirements pre-installed.
                             </p>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              4.2 Databases
                             </h3>
                             <p id="p-46">
                              AutoCoEv uses the following databases from OrthoDB (
                              <a href="https://www.orthodb.org/">
                               https://www.orthodb.org/
                              </a>
                              ): odb10v1_all_fasta, odb10v1_gene_xrefs, odb10v1_OG2genes. The script also communicates with UniProt (
                              <a href="https://www.uniprot.org/">
                               https://www.uniprot.org/
                              </a>
                              ) to download the latest sequence of each protein of interest.
                             </p>
                             <p id="p-47">
                              Non-interacting protein pairs from mouse (Table S2) were obtained from Negatome (
                              <a href="http://mips.helmholtz-muenchen.de/proj/ppi/negatome/">
                               http://mips.helmholtz-muenchen.de/proj/ppi/negatome/
                              </a>
                              ). Conversely, protein complexes were obtained from CORUM (
                              <a href="http://mips.helmholtz-muenchen.de/corum/">
                               http://mips.helmholtz-muenchen.de/corum/
                              </a>
                              ) and sorted by size. The first 10 largest (subunits > 10) from mouse were retrieved (Ids: 3047, 382, 39, 6938, 538, 2750, 496, 572, 582, 1001), whereas co-evolution was found within 5 complexes (Table S3).
                             </p>
                             <p id="p-48">
                              Protein networks were compared against STRING (
                              <a href="https://string-db.org/">
                               https://string-db.org/
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-14" class="subsection">
                             <h3>
                              4.3 Proximity biotinylation
                             </h3>
                             <p id="p-49">
                              For details, see Awoniyi et al bioRxiv [
                              <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                              ], a preceding study from our group, published in parallel with this work. Briefly, lysates of B cells stimulated with 10μg/mL antibody against BCR were collected after 0 min, 5 min, 10 min and 15 min time points. The APEX2 system was used to induce biotinylation of proteins within 20 nm range in close proximity to the BCR. Samples were subjected to streptavidin affinity purification followed by mass spectrometry analysis. MaxQuant (1.6.17.0) was used for database search and after differential enrichment analysis with NormalyzerDE (1.6.0), a list of 346 proteins, proposed as raft-resident, was prepared. From these proteins, 324 were found in OrthoDB and used with here with AutoCoEv.
                             </p>
                            </div>
                            <div id="sec-15" class="subsection">
                             <h3>
                              4.4 AutoCoEv run-time parameters
                             </h3>
                             <p id="p-50">
                              In our analyses, we used 50 mammalian species (Table S1) and configured Mus musculus, as a reference organism (taxid: 10090) and Mammalia as OrthoDB node level (taxid: 40674) in settings.conf. For the reciprocal BLAST, we set the minimum allowed alignment identity to 35% and the maximum allowed gaps to 25%. Guidance was used with MUSCLE and had a cutoff of 0.95. Three MSA methods were used in independent runs: MUSCLE, MAFFT alias L-INS-I and PRANK, while Gblocks allowed gaps were set to half (-b5=h). When PhyML trees were used, PhyML was run with default settings and the produced trees were rooted by TreeBeST. At the protein pairing step, the minimum required common species between each two proteins was set to 20. CAPS2 was run with bootstrap threshold 0.6 and convergence option (-c).
                             </p>
                            </div>
                            <div id="sec-16" class="subsection">
                             <h3>
                              4.5 Protein Characterization
                             </h3>
                             <p id="p-51">
                              Protein sequence functional information was retrieved using UniProt. Domain organizations was searched at SMART [
                              <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                               50
                              </a>
                              ] (Simple Modular Architecture Research Tool,
                              <a href="http://smart.embl-heidelberg.de/">
                               http://smart.embl-heidelberg.de/
                              </a>
                              ) and Pfam [
                              <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                               51
                              </a>
                              ] (
                              <a href="http://pfam.xfam.org/">
                               http://pfam.xfam.org/
                              </a>
                              ). Disordered region binding sites were predicted by Anchor2/IUPred3 [
                              <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">
                               52
                              </a>
                              ] (
                              <a href="https://iupred3.elte.hu/">
                               https://iupred3.elte.hu/
                              </a>
                              ). Graphical representation of proteins was rendered using Pfam cusrtom domain generator (
                              <a href="http://pfam.xfam.org/generate_graphic/">
                               http://pfam.xfam.org/generate_graphic/
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-17" class="subsection">
                             <h3>
                              4.6 Data analyses
                             </h3>
                             <p id="p-52">
                              Post-run analyses were done by R (
                              <a href="https://www.r-project.org/">
                               https://www.r-project.org/
                              </a>
                              ) and Gnumeric spreadsheet (
                              <a href="http://www.gnumeric.org/">
                               http://www.gnumeric.org/
                              </a>
                              ). Gene Ontology was done by clusterProfiler (4.2.2) package for R [
                              <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">
                               53
                              </a>
                              ]. Venn diagrams were generated by DeepVenn [
                              <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">
                               54
                              </a>
                              ]. Networks were visualized in Cytoscape (3.8.2) and cluster analyses were performed by CytoCluster/ClusterONE [
                              <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">
                               55
                              </a>
                              ]. All figures were assembled in Inkscape (
                              <a href="https://inkscape.org/">
                               https://inkscape.org/
                              </a>
                              ) while icons artwork in
                              <a id="xref-fig-1-10" class="xref-fig" href="#F1">
                               Figure 1
                              </a>
                              is from the Tango icon project (
                              <a href="http://tango.freedesktop.org/">
                               http://tango.freedesktop.org/
                              </a>
                              ).
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-19">
                            <h2 class="">
                             Supplementary materials
                            </h2>
                            <p id="p-53">
                             The following supporting information is included in the end of this document: Supplementary figures S1-S4; Supplementary tables S1-S5; Supplementary information.
                            </p>
                           </div>
                           <div class="section" id="sec-20">
                            <h2 class="">
                             Author contributions
                            </h2>
                            <p id="p-54">
                             PBP developed AutoCoEv, analyzed the data, designed the figures and wrote the manuscript. LOA developed the statistical evaluation of the results by R scripting and performed GO analyses. VŠ conceived the original idea of the automated pipeline. MÖB performed proteins characterization and analyses. PKM contributed to the study design, trouble-shooting, and manuscript preparation.
                            </p>
                           </div>
                           <div class="section" id="sec-21">
                            <h2 class="">
                             Funding
                            </h2>
                            <p id="p-55">
                             This work was supported by the Academy of Finland (grant ID: 25700, 296684, 307313, and 327378 to PM; 286712 to VŠ), as well as Sigrid Juselius and Jane and Aatos Erkko, and Magnus Ehrnrooth foundations.
                            </p>
                           </div>
                           <div class="section" id="sec-22">
                            <h2 class="">
                             Institutional Review Board Statement
                            </h2>
                            <p id="p-56">
                             Not applicable.
                            </p>
                           </div>
                           <div class="section" id="sec-23">
                            <h2 class="">
                             Informed Consent Statement
                            </h2>
                            <p id="p-57">
                             Not applicable.
                            </p>
                           </div>
                           <div class="section" id="sec-24">
                            <h2 class="">
                             Data Availability Statement
                            </h2>
                            <p id="p-58">
                             AutoCoEv is available at
                             <a href="https://github.com/mattilalab/autocoev">
                              https://github.com/mattilalab/autocoev
                             </a>
                            </p>
                           </div>
                           <div class="section" id="sec-25">
                            <h2 class="">
                             Conflicts of interest
                            </h2>
                            <p id="p-60">
                             The authors declare no competing interests
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-59">
                             We thank the members of the Lymphocyte Cytoskeleton lab for their critical comments on our manuscript. The authors would like to thank Martti Tolvanen for discussions on the theoretical basis and editing the final manuscript, Akseli Mantila for help with the C++ code of CAPS2 and Dian Dimitrov for advising on mathematics and statistics.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-4">
                               The script has been significantly improved with a dual CAPS2 step (bidirectional).
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-2">
                              <p id="p-5">
                               <a href="https://github.com/mattilalab/autocoev">
                                https://github.com/mattilalab/autocoev
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Baussand
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carbone
                                  </span>
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  A Combinatorial Approach to Detect Coevolved Amino Acid Networks in Protein Families of Variable Divergence
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Comput Biol
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 :
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kuriyan
                                  </span>
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Allostery and coupled sequence variation in nuclear hormone receptors
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  116
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  354
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  356
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKuriyan%26rft.auinit1%253DJ.%26rft.volume%253D116%26rft.issue%253D3%26rft.spage%253D354%26rft.epage%253D356%26rft.atitle%253DAllostery%2Band%2Bcoupled%2Bsequence%2Bvariation%2Bin%2Bnuclear%2Bhormone%2Breceptors.%26rft_id%253Dinfo%253Apmid%252F15016369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15016369&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.3" data-doi="10.1093/bioinformatics/btl493">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fares
                                  </span>
                                  <span class="cit-name-given-names">
                                   MA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McNally
                                  </span>
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  CAPS: coevolution analysis using protein sequences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  2821
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2822
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl493%26rft_id%253Dinfo%253Apmid%252F17005535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl493&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17005535&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000241958000018&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.4" data-doi="10.1093/nar/gkx336">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Oteri
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nadalin
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Champeimont
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  BIS2Analyzer: a server for co-evolution analysis of conserved protein families
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Research
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  45
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  W307
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  W314
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D45%26rft.spage%253DW307%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx336%26rft_id%253Dinfo%253Apmid%252F28472458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkx336&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28472458&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sun
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   Z
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Accurate De Novo Prediction of Protein Contact Map by Ultra-Deep Learning Model
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLOS Computational Biology
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  e1005324
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLOS%2BComputational%2BBiology%26rft.volume%253D13%26rft.spage%253D1005324e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.6" data-doi="10.1073/pnas.1111471108">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Morcos
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pagnani
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lunt
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Direct-coupling analysis of residue coevolution captures native contacts across many protein families
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci. U.S.A
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  108
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  E1293
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1301
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1111471108%26rft_id%253Dinfo%253Apmid%252F22106262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzQ5L0UxMjkzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMDQvMjAyMC4wOS4yOS4zMTUzNzQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.7" data-doi="10.1093/bioinformatics/bty862">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hopf
                                  </span>
                                  <span class="cit-name-given-names">
                                   TA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Green
                                  </span>
                                  <span class="cit-name-given-names">
                                   AG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schubert
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The EVcouplings Python framework for coevolutionary sequence analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  35
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1582
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1584
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D35%26rft.spage%253D1582%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty862%26rft_id%253Dinfo%253Apmid%252F30304492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty862&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30304492&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.8" data-doi="10.7554/eLife.02030">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ovchinnikov
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kamisetty
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Baker
                                  </span>
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Robust and accurate prediction of residue–residue interactions across protein interfaces using evolutionary information
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  eLife
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  e02030
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DeLife%26rft.stitle%253Delife%26rft.volume%253D3%26rft.issue%253D0%26rft.spage%253De02030%26rft.epage%253De02030%26rft.atitle%253DRobust%2Band%2Baccurate%2Bprediction%2Bof%2Bresidue-residue%2Binteractions%2Bacross%2Bprotein%2Binterfaces%2Busing%2Bevolutionary%2Binformation%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.02030%26rft_id%253Dinfo%253Apmid%252F24842992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.7554/eLife.02030&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24842992&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.9" data-doi="10.1093/nar/gkt427">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Simonetti
                                  </span>
                                  <span class="cit-name-given-names">
                                   FL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Teppa
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chernomoretz
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  MISTIC: Mutual information server to infer coevolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  41
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  W8
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  14
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt427%26rft_id%253Dinfo%253Apmid%252F23716641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkt427&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23716641&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000323603200003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Correa Marrero
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Medema
                                  </span>
                                  <span class="cit-name-given-names">
                                   MH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Coevolution-based prediction of protein–protein interactions in polyketide biosynthetic assembly lines
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  36
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  4846
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4853
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D36%26rft.spage%253D4846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.11" data-doi="10.1534/genetics.105.053249">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fares
                                  </span>
                                  <span class="cit-name-given-names">
                                   MA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Travers
                                  </span>
                                  <span class="cit-name-given-names">
                                   SAA.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  A Novel Method for Detecting Intramolecular Coevolution: Adding a Further Dimension to Selective Constraints Analyses
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  173
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  9
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  23
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.053249%26rft_id%253Dinfo%253Apmid%252F16547113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiMTczLzEvOSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAzLzA0LzIwMjAuMDkuMjkuMzE1Mzc0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.12" data-doi="10.1126/science.aaw6718">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cong
                                  </span>
                                  <span class="cit-name-given-names">
                                   Q
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Anishchenko
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ovchinnikov
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Protein interaction networks revealed by proteome coevolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  365
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  185
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  189
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCong%26rft.auinit1%253DQ.%26rft.volume%253D365%26rft.issue%253D6449%26rft.spage%253D185%26rft.epage%253D189%26rft.atitle%253DProtein%2Binteraction%2Bnetworks%2Brevealed%2Bby%2Bproteome%2Bcoevolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaw6718%26rft_id%253Dinfo%253Apmid%252F31296772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjUvNjQ0OS8xODUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8wNC8yMDIwLjA5LjI5LjMxNTM3NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.13" data-doi="10.1093/molbev/msm022">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Travers
                                  </span>
                                  <span class="cit-name-given-names">
                                   SAA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fares
                                  </span>
                                  <span class="cit-name-given-names">
                                   MA.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Functional Coevolutionary Networks of the Hsp70–Hop–Hsp90 System Revealed through Computational Analyses
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  24
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1032
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1044
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm022%26rft_id%253Dinfo%253Apmid%252F17267421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/msm022&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17267421&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000245353200017&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.09.29.315374v4.14" data-doi="10.1186/1471-2148-11-149">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Huang
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Temperley
                                  </span>
                                  <span class="cit-name-given-names">
                                   ND
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ren
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Molecular evolution of the vertebrate TLR1 gene family - a complex history of gene duplication, gene conversion, positive selection and co-evolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Evol Biol
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  149
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DHuang%26rft.auinit1%253DY.%26rft.volume%253D11%26rft.spage%253D149%26rft.epage%253D149%26rft.atitle%253DMolecular%2Bevolution%2Bof%2Bthe%2Bvertebrate%2BTLR1%2Bgene%2Bfamily--a%2Bcomplex%2Bhistory%2Bof%2Bgene%2Bduplication%252C%2Bgene%2Bconversion%252C%2Bpositive%2Bselection%2Band%2Bco-evolution.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-11-149%26rft_id%253Dinfo%253Apmid%252F21619680%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/1471-2148-11-149&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21619680&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.09.29.315374v4.15" data-doi="10.1186/1471-2148-13-156">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ruiz-González
                                  </span>
                                  <span class="cit-name-given-names">
                                   MX
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fares
                                  </span>
                                  <span class="cit-name-given-names">
                                   MA.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Coevolution analyses illuminate the dependencies between amino acid sites in the chaperonin system GroES-L
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Evol Biol
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  156
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol%2BBiol%26rft.volume%253D13%26rft.spage%253D156%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-13-156%26rft_id%253Dinfo%253Apmid%252F23875653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/1471-2148-13-156&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23875653&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.09.29.315374v4.16">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Petrov
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Syrjänen
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Characterization of the Avian Trojan Gene Family Reveals Contrasting Evolutionary Constraints
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS ONE
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  e0121672
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D10%26rft.spage%253De0121672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.17">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Champeimont
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Laine
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hu
                                  </span>
                                  <span class="cit-name-given-names">
                                   S-W
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Coevolution analysis of Hepatitis C virus genome to identify the structural and functional dependency network of viral proteins
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci Rep
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  26401
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D6%26rft.spage%253D26401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.18">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Awoniyi
                                  </span>
                                  <span class="cit-name-given-names">
                                   LO
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Šuštar
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hernández-Pérez
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  APEX2 proximity biotinylation reveals protein dynamics triggered by B cell receptor activation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ; 2020.09.29.318766
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.19" data-doi="10.1093/nar/gky1049">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Consortium
                                  </span>
                                  <span class="cit-name-given-names">
                                   TU.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  UniProt: a worldwide hub of protein knowledge
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  47
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  D506
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D515
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D47%26rft.spage%253DD506%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky1049%26rft_id%253Dinfo%253Apmid%252F30395287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gky1049&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30395287&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.20" data-doi="doi:10.1093/molbev/msx116">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kumar
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stecher
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Suleski
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  TimeTree: A Resource for Timelines, Timetrees, and Divergence Times
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol. Biol. Evol
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1812
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1819
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft.volume%253D34%26rft.spage%253D1812%26rft_id%253Dinfo%253Adoi%252Fdoi%253A10.1093%252Fmolbev%252Fmsx116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=doi:10.1093/molbev/msx116&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.21" data-doi="10.1093/nar/gky1053">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kriventseva
                                  </span>
                                  <span class="cit-name-given-names">
                                   EV
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kuznetsov
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tegenfeldt
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  OrthoDB v10: sampling the diversity of animal, plant, fungal, protist, bacterial and viral genomes for evolutionary and functional annotations of orthologs
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  47
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  D807
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D811
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D47%26rft.spage%253DD807%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky1053%26rft_id%253Dinfo%253Apmid%252F30395283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gky1053&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30395283&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.22" data-doi="10.1186/1471-2105-10-421">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Camacho
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Coulouris
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Avagyan
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  BLAST+: architecture and applications
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Bioinformatics
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  421
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DCamacho%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.spage%253D421%26rft.epage%253D421%26rft.atitle%253DBLAST%252B%253A%2Barchitecture%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-421%26rft_id%253Dinfo%253Apmid%252F20003500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-421&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20003500&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.23" data-doi="10.1093/nar/gkv318">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sela
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ashkenazy
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Katoh
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  GUIDANCE2: accurate detection of unreliable alignment regions accounting for the uncertainty of multiple parameters
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  43
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  W7
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  W14
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv318%26rft_id%253Dinfo%253Apmid%252F25883146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkv318&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25883146&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.24" data-doi="10.1126/science.1151532">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wong
                                  </span>
                                  <span class="cit-name-given-names">
                                   KM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Suchard
                                  </span>
                                  <span class="cit-name-given-names">
                                   MA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Huelsenbeck
                                  </span>
                                  <span class="cit-name-given-names">
                                   JP.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Alignment uncertainty and genomic analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  319
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  473
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  476
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWong%26rft.auinit1%253DK.%2BM.%26rft.volume%253D319%26rft.issue%253D5862%26rft.spage%253D473%26rft.epage%253D476%26rft.atitle%253DAlignment%2BUncertainty%2Band%2BGenomic%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1151532%26rft_id%253Dinfo%253Apmid%252F18218900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTkvNTg2Mi80NzMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8wNC8yMDIwLjA5LjI5LjMxNTM3NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               25.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.25" data-doi="10.1093/molbev/mst010">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Katoh
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Standley
                                  </span>
                                  <span class="cit-name-given-names">
                                   DM.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol Biol Evol
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  772
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  780
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23329690&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000317002300004&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               26.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.26" data-doi="10.1093/nar/gkh340">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Edgar
                                  </span>
                                  <span class="cit-name-given-names">
                                   RC.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  MUSCLE: multiple sequence alignment with high accuracy and high throughput
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  32
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1792
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1797
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15034147&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000220487200025&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               27.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.27" data-doi="10.1007/978-1-62703-646-7_10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Löytynoja
                                  </span>
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Phylogeny-aware alignment with PRANK
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Methods Mol. Biol
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  1079
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  155
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  170
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol%26rft.volume%253D1079%26rft.spage%253D155%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-62703-646-7_10%26rft_id%253Dinfo%253Apmid%252F24170401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/978-1-62703-646-7_10&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24170401&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000328376500011&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               28.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.28" data-doi="10.1093/oxfordjournals.molbev.a026334">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Castresana
                                  </span>
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol. Biol. Evol
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  540
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  552
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DCastresana%26rft.auinit1%253DJ.%26rft.volume%253D17%26rft.issue%253D4%26rft.spage%253D540%26rft.epage%253D552%26rft.atitle%253DSelection%2Bof%2BConserved%2BBlocks%2Bfrom%2BMultiple%2BAlignments%2Bfor%2BTheir%2BUse%2Bin%2BPhylogenetic%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026334%26rft_id%253Dinfo%253Apmid%252F10742046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a026334&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10742046&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000086337700008&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               29.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.29" data-doi="10.1093/sysbio/syq010">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guindon
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dufayard
                                  </span>
                                  <span class="cit-name-given-names">
                                   J-F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lefort
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  New Algorithms and Methods to Estimate Maximum-Likelihood Phylogenies: Assessing the Performance of PhyML 3.0
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Syst Biol
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  59
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  307
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  321
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSyst%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyq010%26rft_id%253Dinfo%253Apmid%252F20525638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/sysbio/syq010&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20525638&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000276528300006&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               30.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.30" data-doi="10.1101/gr.073585.107">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vilella
                                  </span>
                                  <span class="cit-name-given-names">
                                   AJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Severin
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ureta-Vidal
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  EnsemblCompara GeneTrees: Complete, duplication-aware phylogenetic trees in vertebrates
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Res
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  327
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  335
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.073585.107%26rft_id%253Dinfo%253Apmid%252F19029536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzIvMzI3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMDQvMjAyMC4wOS4yOS4zMTUzNzQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               31.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.31">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tange
                                  </span>
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  GNU Parallel 20201222 (‘Vaccine’)
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               32.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.32" data-doi="10.1371/journal.pone.0163962">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shen
                                  </span>
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Le
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  SeqKit: A Cross-Platform and Ultrafast Toolkit for FASTA/Q File Manipulation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLOS ONE
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  e0163962
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D11%26rft.spage%253De0163962%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0163962%26rft_id%253Dinfo%253Apmid%252F27706213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0163962&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27706213&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               33.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.33">
                               <div class="cit-metadata">
                                <cite>
                                 <abbr class="cit-jnl-abbrev">
                                  R: The R Project for Statistical Computing
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               34.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.34" data-doi="10.1101/gr.1239303">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shannon
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Markiel
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ozier
                                  </span>
                                  <span class="cit-name-given-names">
                                   O
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Res
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  2498
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2504
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DShannon%26rft.auinit1%253DP.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D2498%26rft.epage%253D2504%26rft.atitle%253DCytoscape%253A%2BA%2BSoftware%2BEnvironment%2Bfor%2BIntegrated%2BModels%2Bof%2BBiomolecular%2BInteraction%2BNetworks%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1239303%26rft_id%253Dinfo%253Apmid%252F14597658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMS8yNDk4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMDQvMjAyMC4wOS4yOS4zMTUzNzQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               35.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.35" data-doi="10.1093/nar/gkt1079">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blohm
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Frishman
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smialowski
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Negatome 2.0: a database of non-interacting proteins derived by literature mining, manual annotation and protein structure analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  42
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  D396
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  400
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1079%26rft_id%253Dinfo%253Apmid%252F24214996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkt1079&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24214996&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000331139800059&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               36.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.36" data-doi="10.1093/nar/gky973">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Giurgiu
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reinhard
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brauner
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  CORUM: the comprehensive resource of mammalian protein complexes-2019
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  47
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  D559
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D563
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D47%26rft.spage%253DD559%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky973%26rft_id%253Dinfo%253Apmid%252F30357367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gky973&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30357367&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               37.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.37" data-doi="10.1093/nar/gkl191">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lassmann
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sonnhammer
                                  </span>
                                  <span class="cit-name-given-names">
                                   ELL.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Kalign, Kalignvu and Mumsa: web servers for multiple sequence alignment
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Research
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  W596
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  W599
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DLassmann%26rft.auinit1%253DT.%26rft.volume%253D34%26rft.issue%253DWeb%2BServer%2Bissue%26rft.spage%253DW596%26rft.epage%253DW599%26rft.atitle%253DKalign%252C%2BKalignvu%2Band%2BMumsa%253A%2Bweb%2Bservers%2Bfor%2Bmultiple%2Bsequence%2Balignment.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl191%26rft_id%253Dinfo%253Apmid%252F16845078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkl191&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16845078&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000245650200119&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               38.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.38" data-doi="10.1093/nar/gky1049">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Consortium
                                  </span>
                                  <span class="cit-name-given-names">
                                   TU.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  UniProt: a worldwide hub of protein knowledge
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  47
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  D506
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D515
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D47%26rft.spage%253DD506%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky1049%26rft_id%253Dinfo%253Apmid%252F30395287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gky1049&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30395287&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               39.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.39" data-doi="10.1016/j.str.2017.04.012">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moreno-Morcillo
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grande-García
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ruiz-Ramos
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Structural Insight into the Core of CAD, the Multifunctional Protein Leading De Novo Pyrimidine Biosynthesis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Structure
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  912
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  923
                                 </span>
                                 .
                                 <span class="cit-page-range">
                                  e5
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DStructure%26rft.volume%253D25%26rft.spage%253D912%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.str.2017.04.012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.str.2017.04.012&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               40.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.40" data-doi="10.1371/journal.pone.0018093">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thompson
                                  </span>
                                  <span class="cit-name-given-names">
                                   JD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Linard
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lecompte
                                  </span>
                                  <span class="cit-name-given-names">
                                   O
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  A Comprehensive Benchmark Study of Multiple Sequence Alignment Methods: Current Challenges and Future Perspectives
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLOS ONE
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  e18093
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DThompson%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253De18093%26rft.epage%253De18093%26rft.atitle%253DA%2Bcomprehensive%2Bbenchmark%2Bstudy%2Bof%2Bmultiple%2Bsequence%2Balignment%2Bmethods%253A%2Bcurrent%2Bchallenges%2Band%2Bfuture%2Bperspectives.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0018093%26rft_id%253Dinfo%253Apmid%252F21483869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0018093&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21483869&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               41.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.41" data-doi="10.1006/jmbi.2000.4042">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Notredame
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Higgins
                                  </span>
                                  <span class="cit-name-given-names">
                                   DG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Heringa
                                  </span>
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  T-Coffee: A novel method for fast and accurate multiple sequence alignment
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Mol Biol
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  302
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  205
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  217
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DNotredame%26rft.auinit1%253DC.%26rft.volume%253D302%26rft.issue%253D1%26rft.spage%253D205%26rft.epage%253D217%26rft.atitle%253DT-Coffee%253A%2BA%2Bnovel%2Bmethod%2Bfor%2Bfast%2Band%2Baccurate%2Bmultiple%2Bsequence%2Balignment.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.2000.4042%26rft_id%253Dinfo%253Apmid%252F10964570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1006/jmbi.2000.4042&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10964570&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000089352000015&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               42.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.42" data-doi="10.1002/PRO.3290">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sievers
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Higgins
                                  </span>
                                  <span class="cit-name-given-names">
                                   DG.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Clustal Omega for making accurate alignments of many protein sequences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Protein Science
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  135
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  145
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProtein%2BScience%26rft.volume%253D27%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.1002%252FPRO.3290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/PRO.3290&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               43.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.43" data-doi="10.1093/bioinformatics/btu033">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stamatakis
                                  </span>
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1312
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1313
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24451623&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000336095100024&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               44.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.44" data-doi="10.1093/sysbio/sys029">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ronquist
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Teslenko
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   van der Mark
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  MrBayes 3.2: efficient Bayesian phylogenetic inference and model choice across a large model space
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Syst. Biol
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  61
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  539
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  542
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsys029%26rft_id%253Dinfo%253Apmid%252F22357727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/sysbio/sys029&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22357727&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               45.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.45" data-doi="10.1093/molbev/msu300">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nguyen
                                  </span>
                                  <span class="cit-name-given-names">
                                   L-T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schmidt
                                  </span>
                                  <span class="cit-name-given-names">
                                   HA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   von Haeseler
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  IQ-TREE: A Fast and Effective Stochastic Algorithm for Estimating Maximum-Likelihood Phylogenies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  32
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  268
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  274
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu300%26rft_id%253Dinfo%253Apmid%252F25371430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/msu300&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25371430&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               46.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.46">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Anand
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Foot
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ang
                                  </span>
                                  <span class="cit-name-given-names">
                                   C-S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Arrestin-Domain Containing Protein 1 (Arrdc1) Regulates the Protein Cargo and Release of Extracellular Vesicles
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PROTEOMICS
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1800266
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPROTEOMICS%26rft.volume%253D18%26rft.spage%253D1800266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               47.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.47" data-doi="10.1530/rep.0.1230757">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carrey
                                  </span>
                                  <span class="cit-name-given-names">
                                   EA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dietz
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Glubb
                                  </span>
                                  <span class="cit-name-given-names">
                                   DM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Detection and location of the enzymes of de novo pyrimidine biosynthesis in mammalian spermatozoa
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Reproduction
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  123
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  757
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  768
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DReproduction%26rft.stitle%253DReproduction%26rft.aulast%253DCarrey%26rft.auinit1%253DE.%26rft.volume%253D123%26rft.issue%253D6%26rft.spage%253D757%26rft.epage%253D768%26rft.atitle%253DDetection%2Band%2Blocation%2Bof%2Bthe%2Benzymes%2Bof%2Bde%2Bnovo%2Bpyrimidine%2Bbiosynthesis%2Bin%2Bmammalian%2Bspermatozoa%26rft_id%253Dinfo%253Adoi%252F10.1530%252Frep.0.1230757%26rft_id%253Dinfo%253Apmid%252F12052230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoicmVwcm9kIjtzOjU6InJlc2lkIjtzOjk6IjEyMy82Lzc1NyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAzLzA0LzIwMjAuMDkuMjkuMzE1Mzc0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               48.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.48">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Puusepp
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kovacs-Nagy
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alhaddad
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Compound heterozygous SPATA5 variants in four families and functional studies of SPATA5 deficiency
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Eur J Hum Genet
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  407
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  419
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DEur%2BJ%2BHum%2BGenet%26rft.volume%253D26%26rft.spage%253D407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               49.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.49" data-doi="10.1016/j.chroma.2010.01.033">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Amstalden van Hove
                                  </span>
                                  <span class="cit-name-given-names">
                                   ER
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                  <span class="cit-name-given-names">
                                   DF
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Heeren
                                  </span>
                                  <span class="cit-name-given-names">
                                   RMA.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  A concise review of mass spectrometry imaging
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Chromatogr A
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  1217
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  3946
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3954
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bchromatography.%2BA%26rft.stitle%253DJ%2BChromatogr%2BA%26rft.aulast%253DAmstalden%2Bvan%2BHove%26rft.auinit1%253DE.%2BR.%26rft.volume%253D1217%26rft.issue%253D25%26rft.spage%253D3946%26rft.epage%253D3954%26rft.atitle%253DA%2Bconcise%2Breview%2Bof%2Bmass%2Bspectrometry%2Bimaging.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chroma.2010.01.033%26rft_id%253Dinfo%253Apmid%252F20223463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.chroma.2010.01.033&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20223463&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000278779000006&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               50.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.50" data-doi="10.1093/nar/gkx922">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Letunic
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bork
                                  </span>
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  20 years of the SMART protein domain annotation resource
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  46
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  D493
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D496
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DD493%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx922%26rft_id%253Dinfo%253Apmid%252F29040681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkx922&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29040681&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               51.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.51" data-doi="10.1093/NAR/GKAA913">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mistry
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chuguransky
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Williams
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Pfam: The protein families database in 2021
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Research
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  49
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  D412
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D419
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D49%26rft.spage%253DD412%26rft_id%253Dinfo%253Adoi%252F10.1093%252FNAR%252FGKAA913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/NAR/GKAA913&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               52.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.52" data-doi="10.1093/nar/gkab408">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Erdős
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pajkos
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dosztányi
                                  </span>
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  IUPred3: prediction of protein disorder enhanced with unambiguous experimental annotation and visualization of evolutionary conservation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  49
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  W297
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  W303
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D49%26rft.spage%253DW297%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkab408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkab408&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               53.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.53" data-doi="10.1089/omi.2011.0118">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yu
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  <span class="cit-name-given-names">
                                   L-G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Han
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  clusterProfiler: an R Package for Comparing Biological Themes Among Gene Clusters
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  OMICS: A Journal of Integrative Biology
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  284
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  287
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DOMICS%253A%2BA%2BJournal%2Bof%2BIntegrative%2BBiology%26rft.volume%253D16%26rft.spage%253D284%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fomi.2011.0118%26rft_id%253Dinfo%253Apmid%252FPMC3339379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1089/omi.2011.0118&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=PMC3339379&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               54.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.54" data-doi="10.1186/1471-2164-9-488">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hulsen
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   de Vlieg
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alkema
                                  </span>
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  BioVenn – a web application for the comparison and visualization of biological lists using area-proportional Venn diagrams
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Genomics
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  488
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHulsen%26rft.auinit1%253DT.%26rft.volume%253D9%26rft.spage%253D488%26rft.epage%253D488%26rft.atitle%253DBioVenn%2B-%2Ba%2Bweb%2Bapplication%2Bfor%2Bthe%2Bcomparison%2Band%2Bvisualization%2Bof%2Bbiological%2Blists%2Busing%2Barea-proportional%2BVenn%2Bdiagrams.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-488%26rft_id%253Dinfo%253Apmid%252F18925949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-488&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18925949&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F04%2F2020.09.29.315374.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               55.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.29.315374v4.55">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tang
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  CytoCluster: A Cytoscape Plugin for Cluster Analysis and Visualization of Biological Networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  International Journal of Molecular Sciences
                                 </abbr>
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1880
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BMolecular%2BSciences%26rft.volume%253D18%26rft.spage%253D1880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.09.29.315374v4" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.03.03.482887v2" title="Inferring cell-specific gene regulatory networks from single cell gene expression data" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2021.07.19.452956v3" title="The Tabula Sapiens: a multiple organ single cell transcriptomic atlas of humans" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted March 04, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.09.29.315374v4.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.09.29.315374v4.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/03/04/2020.09.29.315374.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.09.29.315374v4.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.09.29.315374v4" class="active" data-icon-position="" data-hide-link-title="0">
                                   AutoCoEv – a high-throughput in silico pipeline for predicting inter-protein co-evolution
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2431660"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-VzegzZQzZ4cUjL4eugTbi5jvKf0iFFgwOnjmWxpt1lc"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484480844"/>
                                   <input type="hidden" name="captcha_token" value="6e314db40ed0cd7bad7b9f9cfa14a097"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2431660--3" data-node-nid="2431660" id="node-2431660--4228353558" data-pisa="biorxiv;2020.09.29.315374v4" data-pisa-master="biorxiv;2020.09.29.315374" data-seqnum="2431660" data-apath="/biorxiv/early/2022/03/04/2020.09.29.315374.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 AutoCoEv – a high-throughput in silico pipeline for predicting inter-protein co-evolution
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Petar B.
                                  </span>
                                  <span class="nlm-surname">
                                   Petrov
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Luqman O.
                                  </span>
                                  <span class="nlm-surname">
                                   Awoniyi
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Vid
                                  </span>
                                  <span class="nlm-surname">
                                   Šuštar
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   M. Özge
                                  </span>
                                  <span class="nlm-surname">
                                   Balcı
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Pieta K.
                                  </span>
                                  <span class="nlm-surname">
                                   Mattila
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.09.29.315374;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.09.29.315374
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.09.29.315374v4" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.315374v4&title=AutoCoEv%20%E2%80%93%20a%20high-throughput%20in%20silico%20pipeline%20for%20predicting%20inter-protein%20co-evolution" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.315374v4&title=AutoCoEv%20%E2%80%93%20a%20high-throughput%20in%20silico%20pipeline%20for%20predicting%20inter-protein%20co-evolution" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.315374v4&text=AutoCoEv%20%E2%80%93%20a%20high-throughput%20in%20silico%20pipeline%20for%20predicting%20inter-protein%20co-evolution" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.315374v4&t=AutoCoEv%20%E2%80%93%20a%20high-throughput%20in%20silico%20pipeline%20for%20predicting%20inter-protein%20co-evolution" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.315374v4&title=AutoCoEv%20%E2%80%93%20a%20high-throughput%20in%20silico%20pipeline%20for%20predicting%20inter-protein%20co-evolution" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.315374v4&title=AutoCoEv%20%E2%80%93%20a%20high-throughput%20in%20silico%20pipeline%20for%20predicting%20inter-protein%20co-evolution&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.315374v4&title=AutoCoEv%20%E2%80%93%20a%20high-throughput%20in%20silico%20pipeline%20for%20predicting%20inter-protein%20co-evolution" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2431660--5" data-node-nid="2431660" id="citation-node-2431660--61044456172" data-pisa="biorxiv;2020.09.29.315374v4" data-pisa-master="biorxiv;2020.09.29.315374" data-seqnum="2431660" data-apath="/biorxiv/early/2022/03/04/2020.09.29.315374.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     AutoCoEv – a high-throughput in silico pipeline for predicting inter-protein co-evolution
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Petar B.
                                      </span>
                                      <span class="nlm-surname">
                                       Petrov
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Luqman O.
                                      </span>
                                      <span class="nlm-surname">
                                       Awoniyi
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Vid
                                      </span>
                                      <span class="nlm-surname">
                                       Šuštar
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       M. Özge
                                      </span>
                                      <span class="nlm-surname">
                                       Balcı
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Pieta K.
                                      </span>
                                      <span class="nlm-surname">
                                       Mattila
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.09.29.315374;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.09.29.315374
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=AutoCoEv%20%E2%80%93%20a%20high-throughput%20in%20silico%20pipeline%20for%20predicting%20inter-protein%20co-evolution&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Petrov%2C+Petar+B.&rft.au=Awoniyi%2C+Luqman+O.&rft.au=%C5%A0u%C5%A1tar%2C+Vid&rft.au=Balc%C4%B1%2C+M.+%C3%96zge&rft.au=Mattila%2C+Pieta+K.">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2431660/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2431660/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2431660/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2431660/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2431660/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2431660/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2431660/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2431660/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2431660/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2431660/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2431660/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2431660/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.315374v4&count=horizontal&via=&text=AutoCoEv%20%E2%80%93%20a%20high-throughput%20in%20silico%20pipeline%20for%20predicting%20inter-protein%20co-evolution&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.315374v4" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.09.29.315374v4&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.09.29.315374v4" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Bioinformatics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23507)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11824)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11440)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15189)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47703)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
