<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.01.11.902890v3.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.01.11.902890v3.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.01.11.902890v3.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/embed/graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/02/16/2020.01.11.902890.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.01.11.902890v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="A population-level statistic for assessing Mendelian behavior of genotyping-by-sequencing data from highly duplicated genomes"/>
  <meta name="DC.Identifier" content="10.1101/2020.01.11.902890"/>
  <meta name="DC.Date" content="2022-02-16"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Background Given the economic and environmental importance of allopolyploids and other species with highly duplicated genomes, there is a need for methods to distinguish paralogs, i.e. duplicate sequences within a genome, from Mendelian loci, i.e. single copy sequences that pair at meiosis. The ratio of observed to expected heterozygosity is an effective tool for filtering loci but requires genotyping to be performed first at a high computational cost, whereas counting the number of sequence tags detected per genotype is computationally quick but very ineffective in inbred or polyploid populations. Therefore, new methods are needed for filtering paralogs.

Results We introduce a novel statistic, H  ind / H  E , that uses the probability that two reads sampled from a genotype will belong to different alleles, instead of observed heterozygosity. The expected value of H  ind / H  E  is the same across all loci in a dataset, regardless of read depth or allele frequency. In contrast to methods based on observed heterozygosity, it can be estimated and used for filtering loci prior to genotype calling. In addition to filtering paralogs, it can be used to filter loci with null alleles or high overdispersion, and identify individuals with unexpected ploidy and hybrid status. We demonstrate that the statistic is useful at read depths as low as five to 10, well below the depth needed for accurate genotype calling in polyploid and outcrossing species.

Conclusions Our methodology for estimating H  ind / H  E  across loci and individuals, as well as determining reasonable thresholds for filtering loci, is implemented in polyRAD v1.6, available at <https://github.com/lvclark/polyRAD>. In large sequencing datasets, we anticipate that the ability to filter markers and identify problematic individuals prior to genotype calling will save researchers considerable computational time.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Lindsay V. Clark"/>
  <meta name="DC.Contributor" content="Wittney Mays"/>
  <meta name="DC.Contributor" content="Alexander E. Lipka"/>
  <meta name="DC.Contributor" content="Erik J. Sacks"/>
  <meta name="article:published_time" content="2022-02-16"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="A population-level statistic for assessing Mendelian behavior of genotyping-by-sequencing data from highly duplicated genomes"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<h3>Background</h3>
<p>Given the economic and environmental importance of allopolyploids and other species with highly duplicated genomes, there is a need for methods to distinguish paralogs, i.e. duplicate sequences within a genome, from Mendelian loci, i.e. single copy sequences that pair at meiosis. The ratio of observed to expected heterozygosity is an effective tool for filtering loci but requires genotyping to be performed first at a high computational cost, whereas counting the number of sequence tags detected per genotype is computationally quick but very ineffective in inbred or polyploid populations. Therefore, new methods are needed for filtering paralogs.</p><h3>Results</h3>
<p>We introduce a novel statistic, <i>H</i><sub><i>ind</i></sub>/<i>H</i><sub><i>E</i></sub>, that uses the probability that two reads sampled from a genotype will belong to different alleles, instead of observed heterozygosity. The expected value of <i>H</i><sub><i>ind</i></sub>/<i>H</i><sub><i>E</i></sub> is the same across all loci in a dataset, regardless of read depth or allele frequency. In contrast to methods based on observed heterozygosity, it can be estimated and used for filtering loci prior to genotype calling. In addition to filtering paralogs, it can be used to filter loci with null alleles or high overdispersion, and identify individuals with unexpected ploidy and hybrid status. We demonstrate that the statistic is useful at read depths as low as five to 10, well below the depth needed for accurate genotype calling in polyploid and outcrossing species.</p><h3>Conclusions</h3>
<p>Our methodology for estimating <i>H</i><sub><i>ind</i></sub>/<i>H</i><sub><i>E</i></sub> across loci and individuals, as well as determining reasonable thresholds for filtering loci, is implemented in polyRAD v1.6, available at https://github.com/lvclark/polyRAD. In large sequencing datasets, we anticipate that the ability to filter markers and identify problematic individuals prior to genotype calling will save researchers considerable computational time.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.01.11.902890v3"/>
  <meta name="citation_id" content="2020.01.11.902890v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.01.11.902890v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.11.902890v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.11.902890v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.01.11.902890"/>
  <meta name="citation_num_pages" content="36"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.01.11.902890"/>
  <meta name="citation_author" content="Lindsay V. Clark"/>
  <meta name="citation_author_institution" content="Roy J. Carver Biotechnology Center, University of Illinois at Urbana-Champaign"/>
  <meta name="citation_author_email" content="lvclark@illinois.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3881-9252"/>
  <meta name="citation_author" content="Wittney Mays"/>
  <meta name="citation_author_institution" content="Department of Crop Sciences, University of Illinois at Urbana-Champaign"/>
  <meta name="citation_author_institution" content="Sandia National Laboratories"/>
  <meta name="citation_author" content="Alexander E. Lipka"/>
  <meta name="citation_author_institution" content="Department of Crop Sciences, University of Illinois at Urbana-Champaign"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1571-8528"/>
  <meta name="citation_author" content="Erik J. Sacks"/>
  <meta name="citation_author_institution" content="Department of Crop Sciences, University of Illinois at Urbana-Champaign"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7537-4558"/>
  <meta name="citation_reference" content="Renny-Byfield S, Wendel JF. Doubling down on genomes: Polyploidy and crop plants. Am J Bot. 2014;101:1711–25. doi:10.3732/ajb.1400119."/>
  <meta name="citation_reference" content="citation_title=The Evolution of the Genome;citation_pages=427-517;citation_year=2005"/>
  <meta name="citation_reference" content="Session AM, Uno Y, Kwon T, Chapman JA, Toyoda A, Takahashi S, et al. Genome evolution in the allotetraploid frog Xenopus laevis. Nature. 2016;538:336–43. doi:10.1038/nature19840."/>
  <meta name="citation_reference" content="Dufresne F, Stift M, Vergilino R, Mable BK. Recent progress and challenges in population genetics of polyploid organisms: an overview of current state-of-the-art molecular and statistical tools. Mol Ecol. 2014;23:40–69. doi:10.1111/mec.12581."/>
  <meta name="citation_reference" content="Kaur S, Francki MG, Forster JW. Identification, characterization and interpretation of single-nucleotide sequence variation in allopolyploid crop species. Plant Biotechnol J. 2012;10:125–38. doi:10.1111/j.1467-7652.2011.00644.x."/>
  <meta name="citation_reference" content="Nadukkalam Ravindran P, Bentzen P, Bradbury IR, Beiko RG. PMERGE: Computational filtering of paralogous sequences from RAD-seq data. Ecol Evol. 2018;8:7002–13. doi:10.1002/ece3.4219."/>
  <meta name="citation_reference" content="Verdu CF, Guichoux E, Quevauvillers S, De Thier O, Laizet Y, Delcamp A, et al. Dealing with paralogy in RADseq data: in silico detection and single nucleotide polymorphism validation in Robinia pseudoacacia L. Ecol E vol. 2016;6:7323–33. doi:10.1002/ece3.2466."/>
  <meta name="citation_reference" content="Meirmans PG, Van Tienderen PH. The effects of inheritance in tetraploids on genetic diversity and population divergence. Heredity (Edinb). 2013;110:131–7. doi:10.1038/hdy.2012.80."/>
  <meta name="citation_reference" content="Hohenlohe PA, Amish SJ, Catchen JM, Allendorf FW, Luikart G. Next-generation RAD sequencing identifies thousands of SNPs for assessing hybridization between rainbow and westslope cutthroat trout. Mol Ecol Resour. 2011;11 Suppl 1:117–22. doi:10.1111/j.1755-0998.2010.02967.x."/>
  <meta name="citation_reference" content="Arruda MP, Brown P, Brown-Guedira G, Krill AM, Thurber C, Merrill KR, et al. Genome-wide association mapping of Fusarium head blight resistance in wheat using genotyping-by-sequencing. Plant Genome. 2016;9. doi:10.3835/plantgenome2015.04.0028."/>
  <meta name="citation_reference" content="Tinker NA, Bekele WA, Hattori J. Haplotag: Software for Haplotype-Based Genotyping-by-Sequencing Analysis. G3. 2016;6:857–63. doi:10.1534/g3.115.024596."/>
  <meta name="citation_reference" content="Clark LV, Lipka AE, Sacks EJ. polyRAD: Genotype Calling with Uncertainty from Sequencing Data in Polyploids and Diploids. G3. 2019;9:663–73. doi:10.1534/g3.118.200913."/>
  <meta name="citation_reference" content="Gerard D, Ferrão LFV, Garcia AAF, Stephens M. Genotyping Polyploids from Messy Sequencing Data. Genetics. 2018;210 November:789–807. doi:10.1534/genetics.118.301468."/>
  <meta name="citation_reference" content="Blischak PD, Kubatko LS, Wolfe AD. SNP genotyping and parameter estimation in polyploids using low-coverage sequencing data. Bioinformatics. 2018;34:407–15. doi:10.1093/bioinformatics/btx587."/>
  <meta name="citation_reference" content="Catchen JM, Amores A, Hohenlohe P, Cresko W, Postlethwait JH. Stacks: building and genotyping loci de novo from short-read sequences. G3. 2011;1:171–82. doi:10.1534/g3.111.000240."/>
  <meta name="citation_reference" content="Peterson BK, Weber JN, Kay EH, Fisher HS, Hoekstra HE. Double Digest RADseq: An Inexpensive Method for De Novo SNP Discovery and Genotyping in Model and Non-Model Species. PLoS One. 2012;7:e37135. doi:10.1371/journal.pone.0037135."/>
  <meta name="citation_reference" content="Willis SC, Hollenbeck CM, Puritz JB, Gold JR, Portnoy DS. Haplotyping RAD loci: an efficient method to filter paralogs and account for physical linkage. Mol Ecol Resour. 2017;17:955–65. doi:10.1111/1755-0998.12647."/>
  <meta name="citation_reference" content="Gayral P, Melo-Ferreira J, Glémin S, Bierne N, Carneiro M, Nabholz B, et al. Reference-Free Population Genomics from Next-Generation Transcriptome Data and the Vertebrate-Invertebrate Gap. PLoS Genet. 2013;9. doi:10.1371/journal.pgen.1003457."/>
  <meta name="citation_reference" content="McKinney GJ, Waples RK, Seeb LW, Seeb JE. Paralogs are revealed by proportion of heterozygotes and deviations in read ratios in genotyping-by-sequencing data from natural populations. Mol Ecol Resour. 2017;17:656–69. doi:10.1111/1755-0998.12613."/>
  <meta name="citation_reference" content="Lu F, Lipka AE, Glaubitz J, Elshire R, Cherney JH, Casler MD, et al. Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol. PLoS Genet. 2013;9:e1003215. doi:10.1371/journal.pgen.1003215."/>
  <meta name="citation_reference" content="Simpson EH. Measurement of diversity. Nature. 1949;163:688. doi:10.1038/163688a0."/>
  <meta name="citation_reference" content="Clark LV, Jin X, Petersen KK, Anzoua KG, Bagmet L, Chebukin P, et al. Population structure of Miscanthus sacchariflorus reveals two major polyploidization events, tetraploid-mediated unidirectional introgression from diploid M. sinensis, and diversity centred around the Yellow Sea. Ann Bot. 2019;124:731–48. doi:10.1093/aob/mcy161."/>
  <meta name="citation_reference" content="Clevenger JP, Ozias-Akins P. SWEEP: A Tool for Filtering High-Quality SNPs in Polyploid Crops. G3. 2015;5:1797–803. doi:10.1534/g3.115.019703."/>
  <meta name="citation_reference" content="Mollinari M, Olukolu BA, Da Pereira GS, Khan A, Gemenet D, Craig Yencho G, et al. Unraveling the hexaploid sweetpotato inheritance using ultra-dense multilocus mapping. G3 Genes, Genomes, Genet. 2020;10:281–92. doi:10.1534/g3.119.400620."/>
  <meta name="citation_reference" content="Kyriakidou M, Tai HH, Anglin NL, Ellis D, Strömvik M V. Current strategies of polyploid plant genome sequence assembly. Front Plant Sci. 2018;9:1660. doi:10.3389/fpls.2018.01660."/>
  <meta name="citation_reference" content="Ma X-F, Jensen E, Alexandrov N, Troukhan M, Zhang L, Thomas-Jones S, et al. High resolution genetic mapping by genome sequencing reveals genome duplication and tetraploid genetic structure of the diploid Miscanthus sinensis. PLoS One. 2012;7:e33821. doi:10.1371/journal.pone.0033821."/>
  <meta name="citation_reference" content="Swaminathan K, Chae WB, Mitros T, Varala K, Xie L, Barling A, et al. A framework genetic map for Miscanthus sinensis from RNAseq-based markers shows recent tetraploidy. BMC Genomics. 2012;13:142. doi:10.1186/1471-2164-13-142."/>
  <meta name="citation_reference" content="Kim C, Zhang D, Auckland SA, Rainville LK, Jakob K, Kronmiller B, et al. SSR-based genetic maps of Miscanthus sinensis and M. sacchariflorus, and their comparison to sorghum. Theor Appl Genet. 2012;124:1325–38. doi:10.1007/s00122-012-1790-1."/>
  <meta name="citation_reference" content="Clark LV, Stewart JR, Nishiwaki A, Toma Y, Kjeldsen JB, Jørgensen U, et al. Genetic structure of Miscanthus sinensis and Miscanthus sacchariflorus in Japan indicates a gradient of bidirectional but asymmetric introgression. J Exp Bot. 2015;66:4213–25. doi:10.1093/jxb/eru511."/>
  <meta name="citation_reference" content="Hirayoshi I, Nishikawa K, Kato R. Cytogenetical Studies on forage plants. (IV) Self-incompatibility in Miscanthus. Japanese J Plant Breed. 1955;5:167–70. doi:10.1270/jsbbs1951.5.167."/>
  <meta name="citation_reference" content="Bradbury PJ, Zhang Z, Kroon DE, Casstevens TM, Ramdoss Y, Buckler ES. TASSEL: software for association mapping of complex traits in diverse samples. Bioinformatics. 2007;23:2633–5. doi:10.1093/bioinformatics/btm308."/>
  <meta name="citation_reference" content="Mitros T, Session AM, James BT, Wu GA, Belaffif MB, Clark L V., et al. Genome biology of the paleotetraploid perennial biomass crop Miscanthus. Nat Commun. 2020;11:5442. doi:10.1038/s41467-020-18923-6."/>
  <meta name="citation_reference" content="McCormick RF, Truong SK, Sreedasyam A, Jenkins J, Shu S, Sims D, et al. The Sorghum bicolor reference genome: improved assembly, gene annotations, a transcriptome atlas, and signatures of genome organization. Plant J. 2018;93:338–54. doi:10.1111/tpj.13781."/>
  <meta name="citation_reference" content="Langmead B, Salzberg SL. Fast gapped-read alignment with Bowtie 2. Nat Methods. 2012;9:357–9. doi:10.1038/nmeth.1923."/>
  <meta name="citation_reference" content="Clark LV, Sacks EJ. TagDigger: user-friendly extraction of read counts from GBS and RAD-seq data. Source Code Biol Med. 2016;11:11. doi:10.1186/s13029-016-0057-7."/>
  <meta name="twitter:title" content="A population-level statistic for assessing Mendelian behavior of genotyping-by-sequencing data from highly duplicated genomes"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Background Given the economic and environmental importance of allopolyploids and other species with highly duplicated genomes, there is a need for methods to distinguish paralogs, i.e. duplicate sequences within a genome, from Mendelian loci, i.e. single copy sequences that pair at meiosis. The ratio of observed to expected heterozygosity is an effective tool for filtering loci but requires genotyping to be performed first at a high computational cost, whereas counting the number of sequence tags detected per genotype is computationally quick but very ineffective in inbred or polyploid populations. Therefore, new methods are needed for filtering paralogs.

Results We introduce a novel statistic, H  ind / H  E , that uses the probability that two reads sampled from a genotype will belong to different alleles, instead of observed heterozygosity. The expected value of H  ind / H  E  is the same across all loci in a dataset, regardless of read depth or allele frequency. In contrast to methods based on observed heterozygosity, it can be estimated and used for filtering loci prior to genotype calling. In addition to filtering paralogs, it can be used to filter loci with null alleles or high overdispersion, and identify individuals with unexpected ploidy and hybrid status. We demonstrate that the statistic is useful at read depths as low as five to 10, well below the depth needed for accurate genotype calling in polyploid and outcrossing species.

Conclusions Our methodology for estimating H  ind / H  E  across loci and individuals, as well as determining reasonable thresholds for filtering loci, is implemented in polyRAD v1.6, available at <https://github.com/lvclark/polyRAD>. In large sequencing datasets, we anticipate that the ability to filter markers and identify problematic individuals prior to genotype calling will save researchers considerable computational time.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="A population-level statistic for assessing Mendelian behavior of genotyping-by-sequencing data from highly duplicated genomes"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.01.11.902890v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Background Given the economic and environmental importance of allopolyploids and other species with highly duplicated genomes, there is a need for methods to distinguish paralogs, i.e. duplicate sequences within a genome, from Mendelian loci, i.e. single copy sequences that pair at meiosis. The ratio of observed to expected heterozygosity is an effective tool for filtering loci but requires genotyping to be performed first at a high computational cost, whereas counting the number of sequence tags detected per genotype is computationally quick but very ineffective in inbred or polyploid populations. Therefore, new methods are needed for filtering paralogs.

Results We introduce a novel statistic, H  ind / H  E , that uses the probability that two reads sampled from a genotype will belong to different alleles, instead of observed heterozygosity. The expected value of H  ind / H  E  is the same across all loci in a dataset, regardless of read depth or allele frequency. In contrast to methods based on observed heterozygosity, it can be estimated and used for filtering loci prior to genotype calling. In addition to filtering paralogs, it can be used to filter loci with null alleles or high overdispersion, and identify individuals with unexpected ploidy and hybrid status. We demonstrate that the statistic is useful at read depths as low as five to 10, well below the depth needed for accurate genotype calling in polyploid and outcrossing species.

Conclusions Our methodology for estimating H  ind / H  E  across loci and individuals, as well as determining reasonable thresholds for filtering loci, is implemented in polyRAD v1.6, available at <https://github.com/lvclark/polyRAD>. In large sequencing datasets, we anticipate that the ability to filter markers and identify problematic individuals prior to genotype calling will save researchers considerable computational time.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-02-16"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.01.11.902890v3"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2405243"/>
  <title>
   A population-level statistic for assessing Mendelian behavior of genotyping-by-sequencing data from highly duplicated genomes | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"s3XPE9NengGEmwK1mSYDl_-80blszTD2EZR8CE3Wxg0"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2405243","apath":"\/biorxiv\/early\/2022\/02\/16\/2020.01.11.902890.atom","pisa":"biorxiv;2020.01.11.902890v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.01.11.902890v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.01.11.902890v3","title":"A population-level statistic for assessing Mendelian behavior of genotyping-by-sequencing data from highly duplicated genomes","identifier":"node\/2405243"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AvEApFvHAFSVNAX6UIO0Jkt+98oo3gCzQlVv2wZl07SzY5jMvK\\\\\\\/mntxd83VQT8lkTtGpZhip4d8teC5fk31Waft83MHkZJI4\\\\\\\/DY3e8Ux5NFjOebQZtOYp3lNa\\\\\\\/nhauxxEvx78OKJ7Rntqfx1Kdc7lethF8Yc6M1xHejZPXiktl5g0W79rcHLMmJ0rtxW48yFuX+K80FgQdY+tfPZOHAyt5rrJ50TTQ38I8YbBjgpVx0hIb+EKgDze87eQzFpTrxQO2Ho+dQx7gNh9wzEj4ol3DhZnPKFB3Hr5xxK7vuwpiHse\\\\\\\/gp6o7shftGtNEETuywwJwV3ElWKDR1JkSooxt9b++0lDVujgmuxIaOvf5DbeRWoZvQ2I96eU1F\\\\\\\/DJ11qFHy96lQe8ftaqkhbwQ04bjedL61N02IDrDoNNXnxJGtWrztvdsZBBYB42GKOYtLJzF6RHOrkEyDZvJbIGcLGnUbKVPYkEXSyccKwBGDPFNKxswsET+++m6dJTMNeQXE4bqnbpbLPaJTNyiTOtrxq+3VrvDHC2j6WrTfukL9L1EmJVRHf0guuVD\\\\\\\/qw+9SYDxXKOBN\\\\\\\/zkF0BtDFwRJseJgX2BpQP2jH7ltQRsj\\\\\\\/ZOxV2av8d49Hnm3R+4ewTBc+2klUlsNiKXfNaFJhjY9KqfFw4MLJj5+YJFAelSLxO7Ly4qVJnszNBUAUGg2LeKf\\\\\\\/8PupZ0NqoiRj\\\\\\\/DWYp8xYk5f87c2I\\\\\\\/qaHzjVtZJ1xiAu2lqjYbogWHv8o8kp3brvzRC63NbPVABy97qHsQPc0FnIKBdTJGAsE\\\\\\\/Yj5r6akKPTiqhaM+xDLqq8aRbiw3UgTJ8f1Paemx9F+lM5NC0\\\\\\\/mqDF\\\\\\\/QbbigUxbkls4eIxyZZs4tSxYzS78ldnxfVhyXBnECPkniqSr3H5gIwKUeVk5GpMy6t2WDLu6gGqAyCmALvxiq6NR\\\\\\\/ErJqdKrRMM9awXkTSUMzrAFwE40oqZNf5RQQuL3B2WHOC4eyMMrRPsZZaSopIi6F6jiC6ye8JlqkSpKUccNi3K1MUhd8oSPlrbhexaWnGNHKS\\\\\\\/48t9fLPh33IfPEoWAtURfn5Ww58jfYaqLWxaeQ3P5t2qXmL2TJticlNVcUVHQrPwIq6v\\\\\\\/EgL6wpELPPPi4bH2N8o4rkNlj3Ar76L0z0pDT4dH\\\\\\\/Dxh9zAOpdmT1YV97DLONxudGzSklZvPVUrxQld1Xi5tOcJjmOSpjHh28MHtRVJjwxFd9tf6nonei0ypnEU9wiSX9FL2GZrCaxoXy0r4GLnN\\\\\\\/CZu9JoO4yAutWvB\\\\\\\/F5dawnzjqbQlO23gfdfpIhq3JUJ38PwtUb7QgHet42rsJZvfqLVgfPaslxIDKqmj5ZKQWgtESA31z+80RqVJuiM7\\\\\\\/euAel+S6x5nC\\\\\\\/Y=\\\u0022,\\\u0022iv\\\u0022:\\\u0022l4kjCfFD5zZtjebF92OQ9A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002283d77ff322df2691ff78fd25adae6ad6a0388bb0150962957fc3e1f73408bd29\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.01.11.902890v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2405243 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.01.11.902890v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_688905468">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_688905468" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1662079059" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-PFfJSeqJFq-Hngg_ZmXpPozSr4yrBs3wQ87FYy-LcRg"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2405243" data-node-nid="2405243" id="node-2405243--282751013" data-pisa="biorxiv;2020.01.11.902890v3" data-pisa-master="biorxiv;2020.01.11.902890" data-apath="/biorxiv/early/2022/02/16/2020.01.11.902890.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    A population-level statistic for assessing Mendelian behavior of genotyping-by-sequencing data from highly duplicated genomes
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0002-3881-9252" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Lindsay V.
                      </span>
                      <span class="nlm-surname">
                       Clark
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Wittney
                      </span>
                      <span class="nlm-surname">
                       Mays
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2">
                      <a href="http://orcid.org/0000-0003-1571-8528" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Alexander E.
                      </span>
                      <span class="nlm-surname">
                       Lipka
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3">
                      <a href="http://orcid.org/0000-0001-7537-4558" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Erik J.
                      </span>
                      <span class="nlm-surname">
                       Sacks
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.01.11.902890
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2405243--282751013" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Lindsay V. Clark
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Roy J. Carver Biotechnology Center, University of Illinois at Urbana-Champaign
                       </span>
                       , Urbana, IL 61801,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Lindsay%2BV.%2BClark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Clark%20LV&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ALindsay%2BV.%2BClark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0002-3881-9252" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Lindsay V. Clark
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:lvclark@illinois.edu" class="" data-icon-position="" data-hide-link-title="0">
                        lvclark@illinois.edu
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Wittney Mays
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Crop Sciences, University of Illinois at Urbana-Champaign
                       </span>
                       , Urbana, IL 61801,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Sandia National Laboratories
                       </span>
                       , Livermore, CA 94551,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Wittney%2BMays%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Mays%20W&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AWittney%2BMays%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Alexander E. Lipka
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Crop Sciences, University of Illinois at Urbana-Champaign
                       </span>
                       , Urbana, IL 61801,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Alexander%2BE.%2BLipka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Lipka%20AE&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAlexander%2BE.%2BLipka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-1571-8528" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Alexander E. Lipka
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Erik J. Sacks
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Crop Sciences, University of Illinois at Urbana-Champaign
                       </span>
                       , Urbana, IL 61801,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Erik%2BJ.%2BSacks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Sacks%20EJ&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AErik%2BJ.%2BSacks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-7537-4558" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Erik J. Sacks
                      </a>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.01.11.902890v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2405243" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2405243/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.01.11.902890v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2405243" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2405243/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.01.11.902890v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2405243" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2405243/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.01.11.902890v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2405243" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2405243/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.01.11.902890v3.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2405243" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2405243/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.01.11.902890v3.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:2405243" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:2405243/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.01.11.902890v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2405243" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2405243/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <div id="sec-1" class="subsection">
                             <p id="p-2">
                              <strong>
                               Background
                              </strong>
                              Given the economic and environmental importance of allopolyploids and other species with highly duplicated genomes, there is a need for methods to distinguish paralogs, i.e. duplicate sequences within a genome, from Mendelian loci, i.e. single copy sequences that pair at meiosis. The ratio of observed to expected heterozygosity is an effective tool for filtering loci but requires genotyping to be performed first at a high computational cost, whereas counting the number of sequence tags detected per genotype is computationally quick but very ineffective in inbred or polyploid populations. Therefore, new methods are needed for filtering paralogs.
                             </p>
                            </div>
                            <div id="sec-2" class="subsection">
                             <p id="p-3">
                              <strong>
                               Results
                              </strong>
                              We introduce a novel statistic, H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              , that uses the probability that two reads sampled from a genotype will belong to different alleles, instead of observed heterozygosity. The expected value of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              is the same across all loci in a dataset, regardless of read depth or allele frequency. In contrast to methods based on observed heterozygosity, it can be estimated and used for filtering loci prior to genotype calling. In addition to filtering paralogs, it can be used to filter loci with null alleles or high overdispersion, and identify individuals with unexpected ploidy and hybrid status. We demonstrate that the statistic is useful at read depths as low as five to 10, well below the depth needed for accurate genotype calling in polyploid and outcrossing species.
                             </p>
                            </div>
                            <div id="sec-3" class="subsection">
                             <p id="p-4">
                              <strong>
                               Conclusions
                              </strong>
                              Our methodology for estimating H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              across loci and individuals, as well as determining reasonable thresholds for filtering loci, is implemented in polyRAD v1.6, available at
                              <a href="https://github.com/lvclark/polyRAD">
                               https://github.com/lvclark/polyRAD
                              </a>
                              . In large sequencing datasets, we anticipate that the ability to filter markers and identify problematic individuals prior to genotype calling will save researchers considerable computational time.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-4">
                            <h2 class="">
                             Background
                            </h2>
                            <p id="p-10">
                             Highly duplicated genome sequences are common throughout the plant kingdom. These include recent allopolyploids such as wheat, cotton, canola, strawberry, and coffee, as well as species with evidence of ancient whole genome duplication such as maize and legumes [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ]. This phenomenon is also present in the animal kingdom, for example allopolyploidy in the model frog Xenopus, as well as an ancient tetraploidization event followed by diploidization in salmonid fishes [
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ,
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ]. For species in which paralogous sequences no longer pair at meiosis, accurate separation of paralogs in DNA and RNA sequence analysis, including reference genome assembly, remains challenging [
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ]. This separation of paralogs is especially important in variant calling, because SNPs and indels will not behave in a Mendelian fashion if the reads originate from more than one locus yet are erroneously attributed to a single locus [
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ]. Accurate variant calling therefore impacts all downstream analysis that assumes Mendelian inheritance, including linkage and QTL mapping, association studies, genomic selection, population genetics, and parentage analysis. For example, failure to remove paralogs from downstream analysis has been demonstrated to bias estimates of allele frequency and inbreeding as well as population structure [
                             <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ,
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             –
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ].
                            </p>
                            <p id="p-11">
                             Due in part to the difficulty of assembling highly duplicated reference genomes, several methods have been published for filtering collapsed paralogous loci from genotyping-by-sequencing (GBS, including restriction-site associated DNA sequencing (RAD) approaches) datasets without the need for a reference genome. The most straightforward approach is to call genotypes and then determine if observed heterozygosity exceeds expected heterozygosity [
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             –
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ]. However, sampling error at low read depth can confound this filtering step by causing heterozygotes to be miscalled as homozygotes, lowering the observed heterozygosity. Moreover, estimating observed heterozygosity becomes complicated when polysomic inheritance is expected, due to the challenge of estimating allele copy number. Bayesian genotype calling methods mitigate the underestimation of observed heterozygosity, but at substantial computational cost [
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             –
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ]. Another approach is to filter loci that have read depth above an arbitrary threshold [
                             <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ], although due to differences in amplification efficiency based on fragment size and GC content, this method could fail to filter some paralogs while filtering other non-paralogous loci. Peterson et al. [
                             <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             ] developed a method, extended by Willis et al. [
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             ], that involved counting the number of unique haplotypes per individual for a putative locus, with the idea that in a collapsed paralog, the number of haplotypes would exceed the ploidy. However, this method can be confounded by sequencing error and cross-contamination among samples, and its sensitivity depends on allele frequencies, inbreeding, and ploidy. Other approaches have examined read depth ratios within individual genotypes [
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ] as well as read depth ratios in combination with observed heterozygosity [
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              19
                             </a>
                             ]. Lastly, multiple methods identify putative paralogs based on networks of similarity among sequence tags [
                             <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ,
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              20
                             </a>
                             ].
                            </p>
                            <p id="p-12">
                             We present a novel statistic, H
                             <sub>
                              ind
                             </sub>
                             /H
                             <sub>
                              E
                             </sub>
                             , for evaluating marker quality, in particular for assessing whether a marker represents one Mendelian locus or multiple collapsed paralogous loci, based upon read depth distribution in a population. For a Mendelian locus, the statistic has the same expected value regardless of number of alleles, allele frequency, and total read depth. As a result, the distribution of the statistic can be visualized across loci in order to identify threshold values for filtering. The expected value can be calculated from ploidy (assuming disomic or polysomic inheritance) and the inbreeding coefficient, or the mode value of the statistic in a population can be used to estimate ploidy or inbreeding. Notably, because genotype calls are not needed in order to estimate this statistic, it can be used for filtering loci before any genotype calling is performed, saving computation time. Technical parameters such as sequencing error and overdispersion can influence estimates, but are explored here using simulated data so that they can be accounted for. We extend our Bayesian genotype calling software polyRAD [
                             <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ] to implement the novel statistic and determine appropriate cutoffs.
                            </p>
                           </div>
                           <div class="section" id="sec-5">
                            <h2 class="">
                             Results
                            </h2>
                            <div id="sec-6" class="subsection">
                             <h3>
                              The H
                              <sub>
                               ind
                              </sub>
                              statistic
                             </h3>
                             <p id="p-13">
                              Here we describe a novel statistic, H
                              <sub>
                               ind
                              </sub>
                              , that is based on sequence read depth across all alleles at a given locus and sample, and is agnostic of genotype calls, inheritance mode, and ploidy. It is related to observed heterozygosity, H
                              <sub>
                               O
                              </sub>
                              , which in a diploid can be thought of as a matrix of ones and zeros indicating whether the genotype at each sample*locus is heterozygous. H
                              <sub>
                               ind
                              </sub>
                              is instead a number ranging from zero to one, indicating the probability that if two sequencing reads were sampled without replacement at that sample*locus, they would represent different alleles. The abbreviation “ind” stands for “individual”, as it is calculated for each individual before averaging across a population. It can be calculated for SNP loci or for multiallelic haplotype- or tag-based loci, as long as allelic read depth is available.
                             </p>
                             <p id="p-14">
                              The expected value for H
                              <sub>
                               ind
                              </sub>
                              in a natural population of diploids or polysomic polyploids is:
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where k is the ploidy, H
                              <sub>
                               E
                              </sub>
                              is the expected heterozygosity at the same locus, and F is the inbreeding coefficient. H
                              <sub>
                               E
                              </sub>
                              is the probability that two alleles drawn at random from the population will be different, (1 – F) is the probability that two alleles randomly drawn from an individual will not be identical by descent, and (k – 1)/k is the probability that two sequencing reads originated from different chromosome copies. Multiplied together, these three terms yield the probability that two sequence reads from one sample at one locus will be different from each other.
                             </p>
                             <p id="p-15">
                              If we divide H
                              <sub>
                               ind
                              </sub>
                              by H
                              <sub>
                               E
                              </sub>
                              :
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              we now have a statistic that is only dependent on ploidy and inbreeding, two parameters that we will assume to be consistent across samples and loci.
                             </p>
                             <p id="p-16">
                              In a mapping population, the term H
                              <sub>
                               E
                              </sub>
                              * (1 – F) must be replaced by the probability, for a given locus, that two locus copies in a progeny will be different alleles. This requires knowledge of the ploidy, parental genotypes, and population design including number of generations of backcrossing and self-fertilization. This probability, which we will call H
                              <sub>
                               E.map
                              </sub>
                              , can be estimated by simulation of the cross. The expectation is then:
                              <span class="disp-formula" id="disp-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Common factors that influence H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              are listed in
                              <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              , and explored in subsequent sections.
                             </p>
                             <div id="T1" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2408508/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2408508/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2408508" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 1.
                               </span>
                               <span class="caption-title">
                                Biological and technical parameters that influence the expected value and variance of H
                                <sub>
                                 ind
                                </sub>
                                /H
                                <sub>
                                 E
                                </sub>
                                .
                               </span>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Empirical estimation of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                             </h3>
                             <p id="p-17">
                              Say that we have sequence read depths, {d
                              <sub>
                               1m
                              </sub>
                              … d
                              <sub>
                               jm
                              </sub>
                              }, across a set of j alleles at a single locus in an individual m. Total read depth in one individual is
                              <span class="disp-formula" id="disp-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              As long as there are two or more reads, we can estimate H
                              <sub>
                               ind
                              </sub>
                              within that individual using the Gini-Simpson index [
                              <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                               21
                              </a>
                              ]:
                              <span class="disp-formula" id="disp-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/graphic-6.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/graphic-6.gif"/>
                                </noscript>
                               </span>
                              </span>
                              For a population of n individuals with sequencing reads, allele frequencies are estimated from average within-individual read depth ratios:
                              <span class="disp-formula" id="disp-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                              And expected heterozygosity is estimated as
                              <span class="disp-formula" id="disp-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Averaged across n individuals with two or more reads at a given locus in a natural population, the expectation is then:
                              <span class="disp-formula" id="disp-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                              In a mapping population,
                              <span class="inline-formula" id="inline-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/inline-graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/inline-graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is estimated in the same way. H
                              <sub>
                               E.map
                              </sub>
                              is estimated from parental genotypes and population design, and the expected average ratio within a locus is given in
                              <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">
                               Eqn. 3
                              </a>
                              .
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              Utility of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              for detecting collapsed paralogs in a diversity panel
                             </h3>
                             <p id="p-18">
                              To compare the distribution of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              values for Mendelian loci versus collapsed paralogs, we aligned M. sacchariflorus tag sequences to the M. sinensis reference genome, in which they should align to the correct paralog most of the time, and to the S. bicolor reference genome, in which two paralogs from Miscanthus correspond to one alignment location. We found that loci with a mean read depth less than five had very low estimates of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              , likely due to restriction site polymorphisms or other technical issues (
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Fig. 1
                              </a>
                              and Additional File 1: Fig. S1). As mean read depth increased above 100 in our dataset, however, loci tended to have H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              values above the expectation for a Mendelian locus, suggesting that most loci at this depth and higher were in fact collapsed paralogs (
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Fig. 1
                              </a>
                              , and Additional File 1: Figs. S1 and S2).
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F1.large.jpg?width=800&height=600&carousel=1" title="Relationship between Hind/HE statistic and mean sequence read depth per locus. Loci were called across 356 diploid and 268 tetraploid Miscanthus sacchariflorus based on alignments to the M. sinensis reference genome. The number of loci in each depth category is indicated. Fig. S1 in Additional File 1 provides justification for the depth thresholds for categories. The expected value for a Mendelian locus in Hardy-Weinberg equilibrium is shown with a dashed line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1344923505" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Relationship between <em>H</em><sub><em>ind</em></sub>/<em>H</em><sub><em>E</em></sub> statistic and mean sequence read depth per locus.</span> Loci were called across 356 diploid and 268 tetraploid <em>Miscanthus sacchariflorus</em> based on alignments to the <em>M. sinensis</em> reference genome. The number of loci in each depth category is indicated. Fig. S1 in Additional File 1 provides justification for the depth thresholds for categories. The expected value for a Mendelian locus in Hardy-Weinberg equilibrium is shown with a dashed line.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F1.medium.gif" width="284" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F1.medium.gif" width="284" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1.
                               </span>
                               <span class="caption-title">
                                Relationship between H
                                <sub>
                                 ind
                                </sub>
                                /H
                                <sub>
                                 E
                                </sub>
                                statistic and mean sequence read depth per locus.
                               </span>
                               <p id="p-19" class="first-child">
                                Loci were called across 356 diploid and 268 tetraploid Miscanthus sacchariflorus based on alignments to the M. sinensis reference genome. The number of loci in each depth category is indicated. Fig. S1 in Additional File 1 provides justification for the depth thresholds for categories. The expected value for a Mendelian locus in Hardy-Weinberg equilibrium is shown with a dashed line.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-20">
                              When a mean depth of five was used as a cutoff and the M. sinensis genome was used as a reference, the peak H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              value was slightly below the expected values of 0.5 for diploids and 0.75 for tetraploids (
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              ), indicating some inbreeding, likely due to population structure [
                              <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                              ]. A second peak was observed at a higher value of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              (
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              ), likely representing sets of tags that belonged to different Mendelian loci despite aligning to the same location (i.e. misalignments). When S. bicolor was used as the reference genome, the opposite trend was observed, where most loci had a H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              above the expected value, indicating collapsed paralogs, but a second peak was observed closer to the expected value, indicating regions in the S. bicolor genome that may only have synteny with one region of the M. sinensis genome (
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              ).
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F2.large.jpg?width=800&height=600&carousel=1" title="Effect of reference genome and ploidy on Hind/HE per locus in Miscanthus sacchariflorus. Loci with a mean read depth below five were omitted, leaving 11,516 loci aligned to the M. sinensis reference and 8,820 loci aligned to the Sorghum bicolor reference. Expected values for Mendelian loci under Hardy-Weinberg equilibrium are shown with dashed lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1344923505" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Effect of reference genome and ploidy on <em>H</em><sub><em>ind</em></sub>/<em>H</em><sub><em>E</em></sub> per locus in <em>Miscanthus sacchariflorus</em>.</span> Loci with a mean read depth below five were omitted, leaving 11,516 loci aligned to the <em>M. sinensis</em> reference and 8,820 loci aligned to the <em>Sorghum bicolor</em> reference. Expected values for Mendelian loci under Hardy-Weinberg equilibrium are shown with dashed lines.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F2.medium.gif" width="440" height="167"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F2.medium.gif" width="440" height="167"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2.
                               </span>
                               <span class="caption-title">
                                Effect of reference genome and ploidy on H
                                <sub>
                                 ind
                                </sub>
                                /H
                                <sub>
                                 E
                                </sub>
                                per locus in Miscanthus sacchariflorus.
                               </span>
                               <p id="p-21" class="first-child">
                                Loci with a mean read depth below five were omitted, leaving 11,516 loci aligned to the M. sinensis reference and 8,820 loci aligned to the Sorghum bicolor reference. Expected values for Mendelian loci under Hardy-Weinberg equilibrium are shown with dashed lines.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-22">
                              Although the peaks overlapped somewhat, they were distinct enough that a reasonable threshold for identifying putative collapsed paralogs could be visually determined (
                              <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              ). Moreover, although the diploid and tetraploid datasets were processed separately, they were largely in agreement about which loci were Mendelian and which were collapsed paralogs (Additional File 1: Fig. S2), suggesting that the filtering performed in one population can be applied to another population, which could be especially useful for populations that are too small for accurate estimation of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              .
                             </p>
                             <p id="p-23">
                              In both the diploid and tetraploid datasets, the distribution and peak values of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              were similar regardless of whether biallelic SNPs or multiallelic, haplotype-based markers were used (Additional File 1: Fig. S3). However, the variance of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              was approximately 20% higher when SNPs were used, suggesting that the higher information content of multiallelic markers improves the precision of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              estimates.
                             </p>
                             <p id="p-24">
                              The ExpectedHindHe function in polyRAD was used to set thresholds for filtering the diploid and tetraploid datasets. Based on results from the TestOverdispersion function, the overdispersion parameter was set to 11 for diploids and 10 for tetraploids. Based on the observed distribution of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              in the dataset, the inbreeding coefficient was set to 0.35 for diploids and 0.25 for tetraploids. Based on these parameters, as well as read depth and allele frequencies in the datasets, the ranges for retaining 95% of Mendelian loci were 0.175 to 0.584 in diploids and 0.356 to 0.716 in tetraploids as estimated by ExpectedHindHe, resulting in 40.2% and 42.3% of loci being filtered, respectively (
                              <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              ). Markers within genes were underrepresented among markers that were filtered for having H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              below the lower threshold, and overrepresented among markers that were filtered for having H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              above the upper threshold, significant in Fisher’s Exact Test at P < 0.0005 (
                              <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              ). Markers that were filtered having H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              above the upper threshold tended to have minor allele frequencies that were very low, consistent with the markers representing sequencing error rather than true alleles, or very high, consistent with the markers representing collapsed paralogs (
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Fig. 3
                              </a>
                              ).
                             </p>
                             <div id="T2" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2408513/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2408513/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2408513" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 2.
                               </span>
                               <span class="caption-title">
                                Contingency tables of number of markers retained and filtered for being above or below H
                                <sub>
                                 ind
                                </sub>
                                /H
                                <sub>
                                 E
                                </sub>
                                thresholds in Miscanthus sacchariflorus, by whether or not the marker was within a gene.
                               </span>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F3.large.jpg?width=800&height=600&carousel=1" title="Filtering by Hind/HE vs. minor allele frequency in Miscanthus sacchariflorus. A dataset of 10,458 SNP loci was tested across 356 diploid and 268 tetraploid individuals. Blue dashed lines indicate filtering thresholds to retain 95% of Mendelian loci based on simulated distributions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1344923505" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Filtering by <em>H</em><sub><em>ind</em></sub>/<em>H</em><sub><em>E</em></sub> vs. minor allele frequency in <em>Miscanthus sacchariflorus</em>.</span> A dataset of 10,458 SNP loci was tested across 356 diploid and 268 tetraploid individuals. Blue dashed lines indicate filtering thresholds to retain 95% of Mendelian loci based on simulated distributions.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F3.medium.gif" width="243" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F3.medium.gif" width="243" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3.
                               </span>
                               <span class="caption-title">
                                Filtering by H
                                <sub>
                                 ind
                                </sub>
                                /H
                                <sub>
                                 E
                                </sub>
                                vs. minor allele frequency in Miscanthus sacchariflorus.
                               </span>
                               <p id="p-25" class="first-child">
                                A dataset of 10,458 SNP loci was tested across 356 diploid and 268 tetraploid individuals. Blue dashed lines indicate filtering thresholds to retain 95% of Mendelian loci based on simulated distributions.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              By individual, H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              reflects ploidy and hybrid status
                             </h3>
                             <p id="p-26">
                              In addition to evaluating the mean H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              within loci, we also obtained the mean statistic within individuals in order to assess the utility of the statistic for determining ploidy. We found that H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              increased with ploidy, largely independent of read depth (
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Fig. 4
                              </a>
                              ). Although the distributions overlapped too much for H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              to be a conclusive indicator of ploidy, it could still potentially be used to identify outlier individuals whose ploidy should be confirmed by other means (e.g. flow cytometry). Additionally, because our empirical dataset included many natural interspecific (M. sacchariflorus × M. sinensis) F1 hybrid and backcross individuals, we were also able to observe that H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              values were considerably higher in hybrids than in non-hybrids, reflecting higher heterozygosity (
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Fig. 4
                              </a>
                              ).
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F4.large.jpg?width=800&height=600&carousel=1" title="Relationship between ploidy, sequence read depth, hybrid ancestry, and Hind/HE among 620 M. sacchariflorus individuals. Ploidy and proportion of ancestry from M. sinensis (hybrid ancestry) were determined previously [22]. Read depth and Hind/HE were averaged across 10,000 loci. The expected value for Hind/HE under Hardy-Weinberg equilibrium is shown with the dashed line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1344923505" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Relationship between ploidy, sequence read depth, hybrid ancestry, and <em>H</em><sub><em>ind</em></sub>/<em>H</em><sub><em>E</em></sub> among 620 <em>M. sacchariflorus</em> individuals.</span> Ploidy and proportion of ancestry from <em>M. sinensis</em> (hybrid ancestry) were determined previously [22]. Read depth and <em>H</em><sub><em>ind</em></sub>/<em>H</em><sub><em>E</em></sub> were averaged across 10,000 loci. The expected value for <em>H</em><sub><em>ind</em></sub>/<em>H</em><sub><em>E</em></sub> under Hardy-Weinberg equilibrium is shown with the dashed line.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F4.medium.gif" width="440" height="264"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F4.medium.gif" width="440" height="264"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4.
                               </span>
                               <span class="caption-title">
                                Relationship between ploidy, sequence read depth, hybrid ancestry, and H
                                <sub>
                                 ind
                                </sub>
                                /H
                                <sub>
                                 E
                                </sub>
                                among 620 M. sacchariflorus individuals.
                               </span>
                               <p id="p-27" class="first-child">
                                Ploidy and proportion of ancestry from M. sinensis (hybrid ancestry) were determined previously [
                                <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">
                                 22
                                </a>
                                ]. Read depth and H
                                <sub>
                                 ind
                                </sub>
                                /H
                                <sub>
                                 E
                                </sub>
                                were averaged across 10,000 loci. The expected value for H
                                <sub>
                                 ind
                                </sub>
                                /H
                                <sub>
                                 E
                                </sub>
                                under Hardy-Weinberg equilibrium is shown with the dashed line.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-10" class="subsection">
                             <h3>
                              Variance and bias in the H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              statistic using simulated data
                             </h3>
                             <p id="p-28">
                              Using simulated data resembling a diversity panel or natural population, the mean H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              estimate decreased as inbreeding increased, with diploid and tetraploid loci being indistinguishable at an inbreeding coefficient of 0.8 or higher (
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Fig. 5
                              </a>
                              ). Sequencing error had little effect on the estimate at a minor allele frequency of 0.05, but caused an inflated estimate at a minor allele frequency of 0.01, particularly as inbreeding increased (
                              <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                               Fig. 5
                              </a>
                              ). Variance and bias in the statistic were minimized if there were at least 500 samples, minor allele frequency was 0.05 or higher, and read depth was at least 5 (
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Fig. 6
                              </a>
                              ). Ploidy had negligible impact on variance and bias (
                              <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                               Fig. 6
                              </a>
                              ). Read depth and minor allele frequency influenced the estimates for collapsed paralogs, but not enough to interfere with distinguishing them from Mendelian markers (
                              <a id="xref-fig-6-3" class="xref-fig" href="#F6">
                               Fig. 6
                              </a>
                              ). As expected, overdispersion (deviation of read depth ratios from allelic dosage ratios) reduced the mean H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              estimate, with the effect of overdispersion being greater at higher minor allele frequencies (Additional File 1: Fig. S4). The H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              estimate also decreased linearly as null allele frequency increased (Additional File 1: Fig. S5).
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F5.large.jpg?width=800&height=600&carousel=1" title="Combined effects of inbreeding, ploidy, minor allele frequency (MAF), and sequencing error on mean estimates of Hind/HE using simulated data. At each combination of parameters, 20,000 biallelic loci were simulated with a read depth of 20 and overdispersion parameter of 20. The x-axis indicates the inbreeding coefficient (the probability that two alleles in an individual are identical by descent) while the y-axis indicates the Hind/HE estimate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1344923505" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Combined effects of inbreeding, ploidy, minor allele frequency (MAF), and sequencing error on mean estimates of <em>H</em><sub><em>ind</em></sub>/<em>H</em><sub><em>E</em></sub> using simulated data.</span> At each combination of parameters, 20,000 biallelic loci were simulated with a read depth of 20 and overdispersion parameter of 20. The x-axis indicates the inbreeding coefficient (the probability that two alleles in an individual are identical by descent) while the y-axis indicates the <em>H</em><sub><em>ind</em></sub>/<em>H</em><sub><em>E</em></sub> estimate.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F5.medium.gif" width="440" height="231"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F5.medium.gif" width="440" height="231"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5.
                               </span>
                               <span class="caption-title">
                                Combined effects of inbreeding, ploidy, minor allele frequency (MAF), and sequencing error on mean estimates of H
                                <sub>
                                 ind
                                </sub>
                                /H
                                <sub>
                                 E
                                </sub>
                                using simulated data.
                               </span>
                               <p id="p-29" class="first-child">
                                At each combination of parameters, 20,000 biallelic loci were simulated with a read depth of 20 and overdispersion parameter of 20. The x-axis indicates the inbreeding coefficient (the probability that two alleles in an individual are identical by descent) while the y-axis indicates the H
                                <sub>
                                 ind
                                </sub>
                                /H
                                <sub>
                                 E
                                </sub>
                                estimate.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F6.large.jpg?width=800&height=600&carousel=1" title="Effect of sample size, read depth, and minor allele frequency on variance and bias of estimates of Hind/HE. For each combination of ploidy, sample size (N), read depth, and minor allele frequency (MAF), 5000 biallelic Mendelian loci were simulated under Hardy-Weinberg Equilibrium with an overdispersion parameter of 20 and sequencing error rate of 0.001. Additionally, 5000 collapsed paralogs, each consisting of two Mendelian loci, were simulated under each set of the same parameters. (A) Standard deviation of Hind/HE estimates. (B) Mean Hind/HE estimates. Expected values are 0.5 for diploids and 0.75 for tetraploids; deviations from these values indicate bias in estimation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1344923505" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Effect of sample size, read depth, and minor allele frequency on variance and bias of estimates of <em>H</em><sub><em>ind</em></sub>/<em>H</em><sub><em>E</em></sub>.</span> For each combination of ploidy, sample size (N), read depth, and minor allele frequency (MAF), 5000 biallelic Mendelian loci were simulated under Hardy-Weinberg Equilibrium with an overdispersion parameter of 20 and sequencing error rate of 0.001. Additionally, 5000 collapsed paralogs, each consisting of two Mendelian loci, were simulated under each set of the same parameters. (A) Standard deviation of <em>H</em><sub><em>ind</em></sub>/<em>H</em><sub><em>E</em></sub> estimates. (B) Mean <em>H</em><sub><em>ind</em></sub>/<em>H</em><sub><em>E</em></sub> estimates. Expected values are 0.5 for diploids and 0.75 for tetraploids; deviations from these values indicate bias in estimation.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F6.medium.gif" width="440" height="401"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F6.medium.gif" width="440" height="401"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6.
                               </span>
                               <span class="caption-title">
                                Effect of sample size, read depth, and minor allele frequency on variance and bias of estimates of H
                                <sub>
                                 ind
                                </sub>
                                /H
                                <sub>
                                 E
                                </sub>
                                .
                               </span>
                               <p id="p-30" class="first-child">
                                For each combination of ploidy, sample size (N), read depth, and minor allele frequency (MAF), 5000 biallelic Mendelian loci were simulated under Hardy-Weinberg Equilibrium with an overdispersion parameter of 20 and sequencing error rate of 0.001. Additionally, 5000 collapsed paralogs, each consisting of two Mendelian loci, were simulated under each set of the same parameters. (A) Standard deviation of H
                                <sub>
                                 ind
                                </sub>
                                /H
                                <sub>
                                 E
                                </sub>
                                estimates. (B) Mean H
                                <sub>
                                 ind
                                </sub>
                                /H
                                <sub>
                                 E
                                </sub>
                                estimates. Expected values are 0.5 for diploids and 0.75 for tetraploids; deviations from these values indicate bias in estimation.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-31">
                              In simulated F1 mapping populations, the standard deviation of the H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              ranged from 0.012 to 0.076 depending on the marker type (
                              <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                               Fig. 7
                              </a>
                              ). In tetraploids, marker types with high expected heterozygosity in the progeny, such as triplex x nulliplex and triplex x simplex, had lower variance in the estimate than marker types with lower expected heterozygosity in the progeny, such as simplex x nulliplex and simplex x simplex (
                              <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                               Fig. 7
                              </a>
                              ). A few rare markers had H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              estimates that deviated very far from the expected value, indicating that the parents were incorrectly genotyped (
                              <a id="xref-fig-7-3" class="xref-fig" href="#F7">
                               Fig. 7
                              </a>
                              ).
                             </p>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F7.large.jpg?width=800&height=600&carousel=1" title="Distribution of Hind/HE estimates in simulated F1 mapping populations. For each cross type, 5000 biallelic loci with a read depth of 20, overdispersion parameter of 20, and sequencing error rate of 0.001 were simulated across 500 individuals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1344923505" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Distribution of <em>H</em><sub><em>ind</em></sub>/<em>H</em><sub><em>E</em></sub> estimates in simulated F1 mapping populations.</span> For each cross type, 5000 biallelic loci with a read depth of 20, overdispersion parameter of 20, and sequencing error rate of 0.001 were simulated across 500 individuals.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F7.medium.gif" width="440" height="228"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F7.medium.gif" width="440" height="228"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/16/2020.01.11.902890/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 7.
                               </span>
                               <span class="caption-title">
                                Distribution of H
                                <sub>
                                 ind
                                </sub>
                                /H
                                <sub>
                                 E
                                </sub>
                                estimates in simulated F1 mapping populations.
                               </span>
                               <p id="p-32" class="first-child">
                                For each cross type, 5000 biallelic loci with a read depth of 20, overdispersion parameter of 20, and sequencing error rate of 0.001 were simulated across 500 individuals.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-11" class="subsection">
                             <h3>
                              Comparison with other approaches
                             </h3>
                             <p id="p-33">
                              To compare effectiveness at filtering paralogs between H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              and other approaches, 1000 Mendelian loci and 1000 collapsed paralogs were simulated in 200 diploid and 200 tetraploid individuals at three levels of inbreeding. The median allele frequency was 0.026 and median read depth per Mendelian locus was 21. For each statistic, the 95
                              <sup>
                               th
                              </sup>
                              percentile for Mendelian loci was determined, and the proportion of collapsed paralogs that would be filtered at that threshold was estimated. The H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              approach and observed over expected heterozygosity (H
                              <sub>
                               O
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              ) performed best, with H
                              <sub>
                               O
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              having the disadvantage that genotyping must be performed before it can be estimated, thus increasing processing time two orders of magnitude over H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              (
                              <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">
                               Table 3
                              </a>
                              ). The H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              thresholds used for filtering were 0.58, 0.41, and 0.17 in diploids and 0.76, 0.48, and 0.17 in tetraploids at inbreeding levels of 0.1, 0.5, and 0.9, respectively. The haplotype counting approach [
                              <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">
                               17
                              </a>
                              ] and allelic depth ratio Z-score approach [
                              <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">
                               19
                              </a>
                              ] both performed reasonably well in diploids but were much less effective in tetraploids, with haplotype counting being useless in tetraploids at high inbreeding, while the Z-score approach additionally suffered in terms of computational time due to the need for genotyping. However, haplotype counting used 2- to 3-fold less computational time than H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              , and thus could be advantageous in diploids when millions of loci are being processed. Lastly, filtering on read depth alone was not very effective given the variation in read depth among loci.
                             </p>
                             <div id="T3" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2408517/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2408517/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2408517" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 3.
                               </span>
                               <span class="caption-title">
                                Effectiveness of various statistics for identifying paralogs, using simulated data across three levels of inbreeding. Standard error is shown for proportion paralogs filtered.
                               </span>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-12">
                            <h2 class="">
                             Discussion
                            </h2>
                            <div id="sec-13" class="subsection">
                             <h3>
                              Properties of the H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              statistic
                             </h3>
                             <p id="p-34">
                              While H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              can be used, in combination with other metrics, to assess locus quality, this should be performed with an understanding of what biological and technical phenomena can cause it to deviate from the expected value. Inbreeding from any source will lower the expected value below (k – 1)/k, where k is the ploidy; this includes not only self-fertilization and preferential mating with relatives, but also population structure, which is why we observed values below (k – 1)/k even in self-incompatible, wind-pollinated M. sacchariflorus (
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Figs. 1
                              </a>
                              -
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               4
                              </a>
                              ). A benefit of this, however, is that as long as ploidy is known and overdispersion can be reasonably estimated (e.g. with the TestOverdispersion function in polyRAD), H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              can be used to estimate inbreeding, either at the population or individual level, directly from sequence read depth. Given that we observed H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              to be inflated at low minor allele frequencies, we recommend using the mode H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              at markers with minor allele frequency of at least 0.05 for estimating inbreeding. Additionally, individuals that are hybrids between species or between highly diverged populations, as well as DNA samples that are an accidental mix of two or more individuals, may have H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              above the expected value (
                              <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                               Fig. 4
                              </a>
                              ). Strong selection for homozygotes or heterozygotes at particular loci would be expected to lower and raise H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              , respectively.
                             </p>
                             <p id="p-35">
                              At the locus level, a H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              that exceeds the expected value can be an indication that alleles are derived from paralogous loci rather than a true Mendelian locus. More broadly, if all alleles truly belong to a single locus, then the expected value is (1 – F)(k – 1)/k. However, if a set of random, independent alleles were assigned to one putative locus, the expected value of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              would be one, because the probability of sampling reads from two different alleles within one individual would be the same as the probability of sampling reads from two different alleles in the general population. In the M. sacchariflorus dataset, markers within genes were overrepresented among markers that were filtered for having H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              above the expected value, likely due to high sequence conservation between paralogs (
                              <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              ). A H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              of zero could indicate a cytoplasmic marker, because while there may be variation in the population, each individual would only be expected to possess reads from one allele. Loci with highly overdispersed read depth distributions due to technical issues such as differential fragment size or variation in library preparation would also be expected to have H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              below expectations; it may be advantageous to filter these from the dataset as they will tend to yield poor-quality genotype calls. Lastly, loci with common null alleles have lower than expected H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              (Additional File 1: Fig. S3), resulting in a tendency to filter loci that are not within genes as these regions are less conserved (
                              <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              ). Null alleles can be the result of restriction cut site polymorphism in RAD-based techniques, primer binding site mutations in amplicon sequencing, or deletion mutations using any genotyping method. Because they are a common problem, H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              can be used to identify and filter loci with null alleles.
                             </p>
                             <p id="p-36">
                              The expected value of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              is independent of read depth, number of individuals sampled, and the allele frequency. However, all of these factors influence the variance of the estimate, and low minor allele frequency especially can bias it upwards (
                              <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                               Fig. 5
                              </a>
                              -
                              <a id="xref-fig-6-4" class="xref-fig" href="#F6">
                               6
                              </a>
                              ). As there is no generalized formula to estimate the variance of a ratio, the variance of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              cannot be estimated mathematically. Moreover, sequencing error inflates the estimate at low minor allele frequency (
                              <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                               Fig. 5
                              </a>
                              ), and polyRAD cannot account for sequence quality scores or alignment quality scores since it only imports allelic read depth. We therefore recommend simulating data for Mendelian loci given the ploidy, inbreeding, sample size, sequencing error rate, and distribution of read depth and allele frequency observed in the dataset of interest. The distribution of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              across simulated loci then can be used to determine cutoff values for filtering loci in the empirical dataset. The ExpectedHindHe and ExpecteHindHeMapping functions are available in polyRAD for this purpose, and suggest cutoffs for filtering loci in order to retain 95% of Mendelian loci. Depending on the downstream application, we recommend considering the number of markers needed versus the importance of marker quality when determining thresholds for read depth, allele frequency, and H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              .
                             </p>
                             <p id="p-37">
                              H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              is more useful for detecting paralogs when haplotypes are treated as alleles (i.e. loci can be multiallelic), as opposed to when all loci are treated as biallelic SNPs, simply due to the fact that multiallelic markers are more information-rich than biallelic markers for the same distribution of minor allele frequencies. We observed that, for the same set of SNPs in M. sacchariflorus, the median value of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              per locus was very similar regardless of whether they were phased into haplotypes within the span of a single RAD tag, but the variance in H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              was about 20% higher for SNPs vs. haplotypes (Additional File 1: Fig. S3). This improved power and information content is why polyRAD generally imports multiallelic, haplotype-based genotypes rather than SNPs as the default. Other methods for marker calling in highly duplicated genomes have also benefitted from the use of haplotype information [
                              <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                              ,
                              <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ], and multiallelic markers have been found to be advantageous over biallelic SNPs for linkage mapping in polyploids [
                              <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                               24
                              </a>
                              ]. It should be noted that in this study we only phased SNPs that were certain to have originated from the same sequencing reads based on physical linkage and read depth. The H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              statistic cannot be estimated using haplotypes spanning longer distances, given that read depth will vary from locus to locus within haplotype.
                             </p>
                            </div>
                            <div id="sec-14" class="subsection">
                             <h3>
                              Uses of the H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              statistic
                             </h3>
                             <p id="p-38">
                              We anticipate locus-filtering to be the most common application of the H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              statistic, with major advantages being that it is not biased by read depth or allele frequency and can be estimated prior to genotype calling. We demonstrate that it is similar to H
                              <sub>
                               O
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              in effectiveness for filtering paralogs, with substantial savings on computational time (
                              <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              ). We should note that our H
                              <sub>
                               O
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              estimates used Bayesian genotype calls from polyRAD, which mitigate the underestimation of observed heterozygosity as compared to naïve genotype calls [
                              <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                              ]. Stringency of filtering should depend on the genotype quality needed for downstream analysis; for example, parentage analysis and QTL mapping are sensitive to genotyping errors, whereas genome-wide association studies and estimations of population structure from principal components analysis are less sensitive. Missing data rate, median read depth, and minor allele frequency are common criteria that should be used in combination with H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              to determine which loci to retain for downstream analysis. In our empirical dataset, we found the loci ranging in depth from five to 100 had the best distribution of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              (
                              <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                               Fig. 1
                              </a>
                              ), but a higher minimum depth may be required for applications that require accurate genotype calling, and the optimal maximum depth used in filtering depends on the overall depth of the dataset. The use of observed heterozygosity, read depth ratios within genotypes, and number of haplotypes per individual are redundant with H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              and unnecessary if it has already been used for filtering. In addition to its use for detecting paralogs in highly duplicated genomes, H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              can be used for marker filtering in less duplicated genomes where occasional paralogs are still an issue.
                             </p>
                             <p id="p-39">
                              Additionally, in any species, markers with low values of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              (e.g. below the 95% confidence interval generated by simulated data) are likely to have null alleles, high overdispersion, or other technical issues and should generally be removed from the dataset. We found that using H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              to filter our M. sacchariflorus dataset impacted minor allele frequency and proportion of markers in genes in ways consistent with the removal of markers with null alleles, collapsed paralogs, or false alleles due to sequencing error (
                              <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              and
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Fig. 3
                              </a>
                              ).
                             </p>
                             <p id="p-40">
                              Although less accurate for determining ploidy than techniques such as flow cytometry, when averaged within individuals, H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              can be used to identify individuals whose ploidy might deviate from expectations and should be confirmed. If flow cytometry is not an option, several other tools exist for the estimation of ploidy directly from next-generation sequencing data [
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                              ]. Lastly, H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              could be potentially useful for improving reference genome assemblies, increasing the value of complementing a de novo assembly with a resequencing or genotyping-by-sequencing effort in a large population or diversity panel. Regions of the reference genome that contain collapsed paralogs are expected to have inflated H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              values, which could be visualized in a smoothed plot of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              vs. alignment position.
                             </p>
                             <p id="p-41">
                              At a minor allele frequency of 0.05, a read depth of five or higher is sufficient to estimate H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              with minimal variance (
                              <a id="xref-fig-6-5" class="xref-fig" href="#F6">
                               Fig. 6
                              </a>
                              ). It is notable that a read depth of five is too low to call genotypes with confidence, to some extent in diploids but especially in polyploids. However, using the H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              statistic, such low depth data are useful for a variety of applications such as identification of outlier individuals in terms of ploidy and hybridity, estimation of inbreeding, identification of loci with technical issues, and assessment of reference genome quality. This in turn can enable researchers to reduce sequencing costs by generating preliminary, low-depth datasets to evaluate these issues before (or instead of) sequencing more deeply.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-15">
                            <h2 class="">
                             Conclusions
                            </h2>
                            <p id="p-42">
                             Here we introduce the H
                             <sub>
                              ind
                             </sub>
                             /H
                             <sub>
                              E
                             </sub>
                             statistic, which can be used for evaluating marker and sample quality in genotyping-by-sequencing datasets for a variety of downstream applications. We demonstrate that reads from paralogous loci cause the statistic to be above the expected value, whereas technical issues such as overdispersion and null alleles cause the statistic to be below the expected value. In typical datasets (hundreds of individuals, read depth above five) the statistic has sufficiently low variance to be useful for filtering loci. The polyRAD R package can estimate H
                             <sub>
                              ind
                             </sub>
                             /H
                             <sub>
                              E
                             </sub>
                             , suggest filtering cutoffs based on simulated data, and perform genotyping after filtering.
                            </p>
                           </div>
                           <div class="section" id="sec-16">
                            <h2 class="">
                             Materials and Methods
                            </h2>
                            <div id="sec-17" class="subsection">
                             <h3>
                              Implementation in polyRAD
                             </h3>
                             <p id="p-43">
                              Functions for estimating H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              and H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E.map
                              </sub>
                              are available in polyRAD v1.2 and later, and are named HindHe and HindHeMapping, respectively. Both utilize an internal Rcpp function for fast calculation, take a RADdata object as input, and return a matrix of values, with samples in rows and loci in columns. The mean value across rows can then be used to get a per-sample estimate, for identifying individuals that are interspecies hybrids or unexpected ploidies. The mean value across columns can be used to get a per-locus estimate for filtering loci. Additionally, polyRAD v1.5 and later includes the ExpectedHindHe and ExpectedHindHeMapping functions, which simulate data to emulate the sample size, allele frequency distribution or parental genotypes, and read depth distribution of an empirical dataset, and return the distribution of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              as if all loci were Mendelian, giving the user reasonable thresholds to use for filtering loci.
                             </p>
                             <p id="p-44">
                              PolyRAD v1.6 is currently available on CRAN, and can be installed using install.packages(“polyRAD”).
                             </p>
                            </div>
                            <div id="sec-18" class="subsection">
                             <h3>
                              Datasets for testing
                             </h3>
                             <p id="p-45">
                              Two types of datasets were used to test H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              : (1) empirical data from a diversity panel of Miscanthus sacchariflorus, and (2) simulated datasets of diversity panels and of biparental F1 mapping populations. Previously published RAD-seq data for an M. sacchariflorus diversity panel [
                              <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                              ] were used for the empirical tests. All species in the Miscanthus genus share an ancient genome duplication, increasing the chromosome number to 19 from the base of 10 in the Andropogoneae tribe [
                              <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                               26
                              </a>
                              –
                              <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                               28
                              </a>
                              ]. Moreover, some populations of M. sacchariflorus display autotetraploidy in addition to this genome duplication (4x = 76) [
                              <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                              ,
                              <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                              ], allowing us to test our algorithm in situations where tetrasomic inheritance is expected, in addition to the more typical disomic inheritance. Miscanthus is also highly heterozygous due to being wind-pollinated and self-incompatible [
                              <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ], thus heterozygosity cannot be used to identify paralogs as easily as it could in an inbred crop species. Together, these factors make M. sacchariflorus an ideal test case.
                             </p>
                             <p id="p-46">
                              To compare values of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              in putatively Mendelian markers versus collapsed paralogs, markers were called from the same dataset using either Miscanthus sinensis or Sorghum bicolor as a reference because M. sinensis has a whole genome duplication with respect to S. bicolor. Raw sequence reads from M. sacchariflorus were processed by the TASSEL-GBSv2 pipeline [
                              <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              ] to identify unique tag sequences and their depths in all individuals. Tag sequences were then aligned to the Miscanthus sinensis v7.1 reference genome [
                              <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                               32
                              </a>
                              ] and the Sorghum bicolor v3.1.1 reference genome [
                              <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                               33
                              </a>
                              ] using Bowtie 2 [
                              <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ]. The tag manager feature of TagDigger [
                              <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                               35
                              </a>
                              ] was used to process the SAM files, recording the alignment location of each tag in both reference genomes. Tag alignment locations within the S. bicolor reference were retained for further analysis if they corresponded to two alignment locations in the M. sinensis reference matching the known synteny between chromosomes. Under this filtering, 239,501 tags were retained at 18,402 S. bicolor alignment locations corresponding to 36,804 M. sinensis alignment locations, in a set of 356 diploid and 268 tetraploid individuals. H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              was then estimated per-locus in polyRAD for both the M. sinensis and S. bicolor alignments.
                             </p>
                             <p id="p-47">
                              To compare the variance of H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              when biallelic SNPs were used versus multiallelic, haplotype-based markers, the TASSEL-GBSv2 pipeline was used to call SNP variants from M. sacchariflorus and export them to VCF. Markers from chromosome 1 were imported to polyRAD using VCF2RADdata, with and without the option to phase SNPs into haplotypes, yielding 3710 and 10,458 loci, respectively. The phasing performed by VCF2RADdata only phases SNPs that are certain to have originated from the same reads based on allelic read depth and physical distance. H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              was then estimated by locus in polyRAD separately for diploids and tetraploids.
                             </p>
                             <p id="p-48">
                              Simulated diversity panel datasets were generated in order to assess the effect of minor allele frequency, sample size, read depth, sequencing error, overdispersion, inbreeding, ploidy, and null alleles on variance and bias of the H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              statistic, using the SimGenotypes and SimAlleleDepth functions in polyRAD v1.6. See Clark et al. [
                              <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                              ] (
                              <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">
                               Eqn. 2
                              </a>
                              ) for a definition of the overdispersion parameter; lower values result in allelic read depths that deviate further from the ratios expected based on allelic dosage. Three sets of data were simulated. (1) Minor allele frequencies of 0.01, 0.05, and 0.1; sample sizes of 100, 500, and 1000; and genotype read depths of 2, 5, 10, 20, 50, and 100 were simulated in all combinations under diploidy and tetraploidy, with no inbreeding, a sequencing error rate of 0.001, and an overdispersion parameter of 20. For each combination, 5000 biallelic loci were simulated, as well as 5000 collapsed paralogs that each consisted of two Mendelian loci combined. (2) Minor allele frequencies of 0.01 and 0.05, overdispersion spanning all integers from 5 to 20, sequencing error rates of 0 and 0.001, and inbreeding (F; the probability that two locus copies in an individual are identical by descent) spanning all intervals of 0.1 from 0 to 1 were simulated in all combinations under diploidy and tetraploidy, with a sample size of 500 and a read depth of 20. For each combination, 20,000 biallelic loci were simulated. (3) Minor non-null allele frequencies of 0.01 and 0.05 and null allele frequencies of 0.01, 0.05, 0.1, and 0.2 were simulated in all combinations under diploidy and tetraploidy, with a sample size of 500, a read depth of 20, a sequencing error rate of 0.001, overdispersion of 20, and no inbreeding. For each combination, 5000 triallelic (with one allele being null, i.e. having all of its reads discarded) loci were simulated.
                             </p>
                             <p id="p-49">
                              Simulated F1 mapping population datasets were generated in order to assess the effect of ploidy and marker type on variance of the H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              statistic. For diploids, testcross (homozygote x heterozygote) and F2 (heterozygote x heterozygote) markers were evaluated. For tetraploids, simplex x nulliplex (AAAB x AAAA), duplex x nulliplex (AABB x AAAA), triplex x nulliplex (ABBB x AAAA), simplex x simplex (AAAB x AAAB), simplex x duplex (AAAB x AABB), simplex x triplex (AAAB x ABBB), and duplex x duplex (AABB x AABB) markers were evaluated. For each marker type, 5000 biallelic markers were simulated in a population with 500 offspring, with a read depth of 20, a sequencing error rate of 0.001, and overdispersion parameter of 20.
                             </p>
                             <p id="p-50">
                              To evaluate effectiveness of various approaches for filtering paralogs, 1000 Mendelian loci and 1000 collapsed paralogs were simulated in 200 diploid and 200 tetraploid individuals each at three levels of inbreeding. Number of alleles was evenly distributed from two to eight in Mendelian loci. Allele frequency was sampled from a gamma distribution with shape of 0.3 and scale of 1, divided by 10 and added to 0.01 to ensure a minimum minor allele frequency, given that allele frequency filtering is typically performed during variant calling and/or data import. One allele frequency at each locus was generated as one minus the sum of all other allele frequencies, to emulate the typical situation of one common allele and one or more rare alleles. Genotypes were simulated from the allele frequencies assuming an inbreeding coefficient (F) of 0.1, 0.5, or 0.9. Mean read depth per locus was drawn from a gamma distribution with a shape of 3.2 and scale of 8. Read depth at individual genotypes was then drawn from a gamma distribution with the locus depth / 10 as the shape, and a scale of 10. Allelic read depth was simulated assuming an overdispersion parameter of 20 and a sequencing error rate of 0.001. Collapsed paralogs were simulated in the same way, but with number of alleles per locus ranging from one to eight, and two random loci being combined to form a collapsed paralog.
                             </p>
                            </div>
                            <div id="sec-19" class="subsection">
                             <h3>
                              Comparison with other approaches
                             </h3>
                             <p id="p-51">
                              To call genotypes for the H
                              <sub>
                               O
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              and Z-score [
                              <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">
                               19
                              </a>
                              ] approaches, the IterateHWE function in polyRAD was used with default parameters to obtain genotype probabilities, and then GetProbableGenotypes was used to get discrete genotypes, with genotypes set to missing if allele copy numbers did not add up to the ploidy. To extend its use to polyploids, H
                              <sub>
                               O
                              </sub>
                              was estimated as the probability that two alleles sampled from a genotype without replacement would be different from each other, averaged across individuals within a locus. The Z-score approach [
                              <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">
                               19
                              </a>
                              ] was originally only defined for biallelic markers in diploids. To extend it for multiallelic markers and polyploid species, for each marker genotypes with ploidy – 1 copies of the most common allele (i.e. the heterozygous genotype class expected to be most common) were identified, and allelic read depth summed across those samples. Deviation of read depth of the most common allele from the expected ratio was then estimated as a Z-score:
                              <span class="disp-formula" id="disp-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/graphic-19.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/16/2020.01.11.902890/embed/graphic-19.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Where N is the total read depth across all samples in the given heterozygous genotype class, and N
                              <sub>
                               A
                              </sub>
                              is the read depth of the common allele summed across those same samples. The number of haplotypes per genotype was counted as the number of haplotypes with read depth of three or higher, following Willis et al. [
                              <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">
                               17
                              </a>
                              ].
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-21">
                            <h2 class="">
                             Declarations
                            </h2>
                            <div class="section" id="sec-22">
                             <h3>
                              Ethics approval and consent to participate
                             </h3>
                             <p id="p-52">
                              Not applicable
                             </p>
                            </div>
                            <div class="section" id="sec-23">
                             <h3>
                              Consent for publication
                             </h3>
                             <p id="p-53">
                              Not applicable
                             </p>
                            </div>
                            <div class="section" id="sec-24">
                             <h3>
                              Availability of data and materials
                             </h3>
                             <p id="p-54">
                              Raw sequence reads for M. sacchariflorus are available on the NCBI Sequence Read Archive under accessions SRP026347, SRP048207, SRP063572, and SRP087645. Genotype calls and read depths for M. sacchariflorus are available on the Illinois Data Bank at
                              <a href="https://doi.org/10.13012/B2IDB-8170405_V1">
                               https://doi.org/10.13012/B2IDB-8170405_V1
                              </a>
                              . Tag alignments and counts for M. sacchariflorus using the M. sinensis and S. bicolor reference genomes are available on the Illinois Data Bank at
                              <a href="https://doi.org/10.13012/B2IDB-4814898_V1">
                               https://doi.org/10.13012/B2IDB-4814898_V1
                              </a>
                              . All scripts for testing the H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              statistic are available on GitHub at
                              <a href="https://github.com/lvclark/paralog_id">
                               https://github.com/lvclark/paralog_id
                              </a>
                              , archived on Zenodo at
                              <a href="https://doi.org/10.5281/zenodo.5425343">
                               https://doi.org/10.5281/zenodo.5425343
                              </a>
                              .
                             </p>
                             <p id="p-55">
                              Project name: polyRAD
                             </p>
                             <p id="p-56">
                              Project home page:
                              <a href="https://github.com/lvclark/polyRAD">
                               https://github.com/lvclark/polyRAD
                              </a>
                             </p>
                             <p id="p-57">
                              Archived version:
                              <a href="https://doi.org/10.5281/zenodo.1143744">
                               https://doi.org/10.5281/zenodo.1143744
                              </a>
                             </p>
                             <p id="p-58">
                              Operating system: Platform independent
                             </p>
                             <p id="p-59">
                              Programming language: R, C++ via Rcpp
                             </p>
                             <p id="p-60">
                              Other requirements: R ≥ 3.5.0; CRAN packages fastmatch, Rccp, and stringi; Bioconductor package pcaMethods
                             </p>
                             <p id="p-61">
                              License: GNU GPL (≥2)
                             </p>
                             <p id="p-62">
                              Any restrictions to use by non-academics: None
                             </p>
                            </div>
                            <div class="section" id="sec-25">
                             <h3>
                              Competing interests
                             </h3>
                             <p id="p-63">
                              The authors declare they have no competing interests.
                             </p>
                            </div>
                            <div class="section" id="sec-26">
                             <h3>
                              Funding
                             </h3>
                             <p id="p-64">
                              This material is based upon work supported by the National Science Foundation under Grant No. 1661490. The funding body was not involved in the design, analysis, or interpretation of the study.
                             </p>
                            </div>
                            <div class="section" id="sec-27">
                             <h3>
                              Authors’ contributions
                             </h3>
                             <p id="p-65">
                              LVC designed the H
                              <sub>
                               ind
                              </sub>
                              /H
                              <sub>
                               E
                              </sub>
                              statistic, wrote the polyRAD software, performed the analysis, and wrote the manuscript. WM performed literature review and tested the software and statistic. AEL gave statistical advice. EJS provided the M. sacchariflorus datasets. All authors read and approved the final manuscript.
                             </p>
                            </div>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-66">
                             We thank Jiale He for testing various methods to detect paralogous loci in Miscanthus.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-6">
                               Analyses were added in response to feedback from peer review, in particular to test the effect of sequencing error on the Hind/He statistic, compare the statistic to others that are used to filter paralogs, and examine how filtering using the statistic impacts the distribution of minor allele frequency and the proportion of markers in genes.
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-2">
                              <p id="p-7">
                               <a href="https://doi.org/10.5281/zenodo.5425343">
                                https://doi.org/10.5281/zenodo.5425343
                               </a>
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-3">
                              <p id="p-8">
                               <a href="https://doi.org/10.13012/B2IDB-8170405_V1">
                                https://doi.org/10.13012/B2IDB-8170405_V1
                               </a>
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-4">
                              <p id="p-9">
                               <a href="https://doi.org/10.13012/B2IDB-4814898_V1">
                                https://doi.org/10.13012/B2IDB-4814898_V1
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.1" data-doi="10.3732/ajb.1400119">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Renny-Byfield
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wendel
                                  </span>
                                  <span class="cit-name-given-names">
                                   JF
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Doubling down on genomes: Polyploidy and crop plants
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Am J Bot
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  101
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1711
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  25
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.3732/ajb.1400119
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAm%2BJ%2BBot%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1400119%26rft_id%253Dinfo%253Apmid%252F25090999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjExOiIxMDEvMTAvMTcxMSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAyLzE2LzIwMjAuMDEuMTEuOTAyODkwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.01.11.902890v3.2">
                               <div class="cit-metadata">
                                <ol class="duplicate">
                                 <li>
                                  In:
                                 </li>
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-surname">
                                    Gregory
                                   </span>
                                   <span class="cit-name-given-names">
                                    TR
                                   </span>
                                  </span>
                                  , editor
                                 </li>
                                </ol>
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gregory
                                  </span>
                                  <span class="cit-name-given-names">
                                   TR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mable
                                  </span>
                                  <span class="cit-name-given-names">
                                   BK
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-chapter-title">
                                  Polyploidy in Animals
                                 </span>
                                 . In:
                                 <span class="cit-ed">
                                  <span class="cit-name-surname">
                                   Gregory
                                  </span>
                                  <span class="cit-name-given-names">
                                   TR
                                  </span>
                                 </span>
                                 , editor.
                                 <span class="cit-source">
                                  The Evolution of the Genome
                                 </span>
                                 .
                                 <span class="cit-publ-loc">
                                  San Diego
                                 </span>
                                 :
                                 <span class="cit-publ-name">
                                  Elsevier
                                 </span>
                                 ;
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 . p.
                                 <span class="cit-fpage">
                                  427
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  517
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.3" data-doi="10.1038/nature19840">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Session
                                  </span>
                                  <span class="cit-name-given-names">
                                   AM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Uno
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kwon
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chapman
                                  </span>
                                  <span class="cit-name-given-names">
                                   JA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Toyoda
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Takahashi
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Genome evolution in the allotetraploid frog Xenopus laevis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  538
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  336
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  43
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/nature19840
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D538%26rft.spage%253D336%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature19840%26rft_id%253Dinfo%253Apmid%252F27762356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature19840&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27762356&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F16%2F2020.01.11.902890.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.4" data-doi="10.1111/mec.12581">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dufresne
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stift
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vergilino
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mable
                                  </span>
                                  <span class="cit-name-given-names">
                                   BK
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Recent progress and challenges in population genetics of polyploid organisms: an overview of current state-of-the-art molecular and statistical tools
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol Ecol
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  23
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  40
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  69
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1111/mec.12581
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D23%26rft.spage%253D40%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/mec.12581&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000330950900007&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.5" data-doi="10.1111/j.1467-7652.2011.00644.x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kaur
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Francki
                                  </span>
                                  <span class="cit-name-given-names">
                                   MG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Forster
                                  </span>
                                  <span class="cit-name-given-names">
                                   JW
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Identification, characterization and interpretation of single-nucleotide sequence variation in allopolyploid crop species
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Plant Biotechnol J
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  125
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  38
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1111/j.1467-7652.2011.00644.x
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DKaur%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D125%26rft.epage%253D138%26rft.atitle%253DIdentification%252C%2Bcharacterization%2Band%2Binterpretation%2Bof%2Bsingle-nucleotide%2Bsequence%2Bvariation%2Bin%2Ballopolyploid%2Bcrop%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2011.00644.x%26rft_id%253Dinfo%253Apmid%252F21831136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2011.00644.x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21831136&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F16%2F2020.01.11.902890.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000298781400001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.6" data-doi="10.1002/ece3.4219">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nadukkalam Ravindran
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bentzen
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bradbury
                                  </span>
                                  <span class="cit-name-given-names">
                                   IR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Beiko
                                  </span>
                                  <span class="cit-name-given-names">
                                   RG
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  PMERGE: Computational filtering of paralogous sequences from RAD-seq data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Ecol E
                                 </abbr>
                                 vol.
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  7002
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  13
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1002/ece3.4219
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DEcol%2BE%26rft.volume%253D8%26rft.spage%253D7002%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fece3.4219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/ece3.4219&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.11.902890v3.7" data-doi="10.1002/ece3.2466">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Verdu
                                  </span>
                                  <span class="cit-name-given-names">
                                   CF
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guichoux
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Quevauvillers
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   De Thier
                                  </span>
                                  <span class="cit-name-given-names">
                                   O
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Laizet
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Delcamp
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Dealing with paralogy in RADseq data: in silico detection and single nucleotide polymorphism validation in Robinia pseudoacacia L
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Ecol E vol
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  7323
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  33
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1002/ece3.2466
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DEcol%2BE%2Bvol%26rft.volume%253D6%26rft.spage%253D7323%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fece3.2466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/ece3.2466&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.8" data-doi="10.1038/hdy.2012.80">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Meirmans
                                  </span>
                                  <span class="cit-name-given-names">
                                   PG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Van Tienderen
                                  </span>
                                  <span class="cit-name-given-names">
                                   PH
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The effects of inheritance in tetraploids on genetic diversity and population divergence
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Heredity (Edinb)
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  110
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  131
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  7
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/hdy.2012.80
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D110%26rft.spage%253D131%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2012.80%26rft_id%253Dinfo%253Apmid%252F23211786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/hdy.2012.80&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23211786&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F16%2F2020.01.11.902890.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000313836500007&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.9" data-doi="10.1111/j.1755-0998.2010.02967.x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hohenlohe
                                  </span>
                                  <span class="cit-name-given-names">
                                   PA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Amish
                                  </span>
                                  <span class="cit-name-given-names">
                                   SJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Catchen
                                  </span>
                                  <span class="cit-name-given-names">
                                   JM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Allendorf
                                  </span>
                                  <span class="cit-name-given-names">
                                   FW
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Luikart
                                  </span>
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Next-generation RAD sequencing identifies thousands of SNPs for assessing hybridization between rainbow and westslope cutthroat trout
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol Ecol Resour
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 <span class="cit-issue">
                                  Suppl 1
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  117
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  22
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1111/j.1755-0998.2010.02967.x
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%2BResour%26rft.volume%253D11%26rft.spage%253D117%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2010.02967.x%26rft_id%253Dinfo%253Apmid%252F21429168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2010.02967.x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21429168&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F16%2F2020.01.11.902890.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000287485100011&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.11.902890v3.10" data-doi="10.3835/plantgenome2015.04.0028">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Arruda
                                  </span>
                                  <span class="cit-name-given-names">
                                   MP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brown
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brown-Guedira
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Krill
                                  </span>
                                  <span class="cit-name-given-names">
                                   AM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thurber
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Merrill
                                  </span>
                                  <span class="cit-name-given-names">
                                   KR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Genome-wide association mapping of Fusarium head blight resistance in wheat using genotyping-by-sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Plant Genome
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.3835/plantgenome2015.04.0028
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2015.04.0028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.3835/plantgenome2015.04.0028&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.11" data-doi="10.1534/g3.115.024596">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tinker
                                  </span>
                                  <span class="cit-name-given-names">
                                   NA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bekele
                                  </span>
                                  <span class="cit-name-given-names">
                                   WA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hattori
                                  </span>
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Haplotag: Software for Haplotype-Based Genotyping-by-Sequencing Analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  G3
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  857
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  63
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1534/g3.115.024596
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DG3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.024596%26rft_id%253Dinfo%253Apmid%252F26818073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjYvNC84NTciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMi8xNi8yMDIwLjAxLjExLjkwMjg5MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.12" data-doi="10.1534/g3.118.200913">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Clark
                                  </span>
                                  <span class="cit-name-given-names">
                                   LV
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lipka
                                  </span>
                                  <span class="cit-name-given-names">
                                   AE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sacks
                                  </span>
                                  <span class="cit-name-given-names">
                                   EJ
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  polyRAD: Genotype Calling with Uncertainty from Sequencing Data in Polyploids and Diploids
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  G3
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  663
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  73
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1534/g3.118.200913
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DG3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200913%26rft_id%253Dinfo%253Apmid%252F30655271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjkvMy82NjMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMi8xNi8yMDIwLjAxLjExLjkwMjg5MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.11.902890v3.13" data-doi="10.1534/genetics.118.301468">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gerard
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ferrão
                                  </span>
                                  <span class="cit-name-given-names">
                                   LFV
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Garcia
                                  </span>
                                  <span class="cit-name-given-names">
                                   AAF
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stephens
                                  </span>
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Genotyping Polyploids from Messy Sequencing Data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  210
                                 </span>
                                 <span class="cit-month">
                                  November
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  789
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  807
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1534/genetics.118.301468
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301468%26rft_id%253Dinfo%253Apmid%252F30185430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjEwLzMvNzg5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDIvMTYvMjAyMC4wMS4xMS45MDI4OTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.14" data-doi="10.1093/bioinformatics/btx587">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blischak
                                  </span>
                                  <span class="cit-name-given-names">
                                   PD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kubatko
                                  </span>
                                  <span class="cit-name-given-names">
                                   LS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wolfe
                                  </span>
                                  <span class="cit-name-given-names">
                                   AD
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  SNP genotyping and parameter estimation in polyploids using low-coverage sequencing data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  407
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  15
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/bioinformatics/btx587
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D407%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx587&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.15" data-doi="10.1534/g3.111.000240">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Catchen
                                  </span>
                                  <span class="cit-name-given-names">
                                   JM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Amores
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hohenlohe
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cresko
                                  </span>
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Postlethwait
                                  </span>
                                  <span class="cit-name-given-names">
                                   JH
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Stacks: building and genotyping loci de novo from short-read sequences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  G3
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  1
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  171
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  82
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1534/g3.111.000240
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D1%26rft.spage%253D171%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000240%26rft_id%253Dinfo%253Apmid%252F22384329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1534/g3.111.000240&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22384329&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F16%2F2020.01.11.902890.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.16" data-doi="10.1371/journal.pone.0037135">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Peterson
                                  </span>
                                  <span class="cit-name-given-names">
                                   BK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weber
                                  </span>
                                  <span class="cit-name-given-names">
                                   JN
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kay
                                  </span>
                                  <span class="cit-name-given-names">
                                   EH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fisher
                                  </span>
                                  <span class="cit-name-given-names">
                                   HS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hoekstra
                                  </span>
                                  <span class="cit-name-given-names">
                                   HE
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Double Digest RADseq: An Inexpensive Method for De Novo SNP Discovery and Genotyping in Model and Non-Model Species
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS One
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  e37135
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1371/journal.pone.0037135
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeterson%26rft.auinit1%253DB.%2BK.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37135%26rft.epage%253De37135%26rft.atitle%253DDouble%2Bdigest%2BRADseq%253A%2Ban%2Binexpensive%2Bmethod%2Bfor%2Bde%2Bnovo%2BSNP%2Bdiscovery%2Band%2Bgenotyping%2Bin%2Bmodel%2Band%2Bnon-model%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037135%26rft_id%253Dinfo%253Apmid%252F22675423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037135&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22675423&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F16%2F2020.01.11.902890.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.17" data-doi="10.1111/1755-0998.12647">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Willis
                                  </span>
                                  <span class="cit-name-given-names">
                                   SC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hollenbeck
                                  </span>
                                  <span class="cit-name-given-names">
                                   CM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Puritz
                                  </span>
                                  <span class="cit-name-given-names">
                                   JB
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gold
                                  </span>
                                  <span class="cit-name-given-names">
                                   JR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Portnoy
                                  </span>
                                  <span class="cit-name-given-names">
                                   DS
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Haplotyping RAD loci: an efficient method to filter paralogs and account for physical linkage
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol Ecol Resour
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  955
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  65
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1111/1755-0998.12647
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%2BResour%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/1755-0998.12647&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.18" data-doi="10.1371/journal.pgen.1003457">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gayral
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Melo-Ferreira
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Glémin
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bierne
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carneiro
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nabholz
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Reference-Free Population Genomics from Next-Generation Transcriptome Data and the Vertebrate-Invertebrate Gap
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Genet
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1371/journal.pgen.1003457
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D9%26rft.spage%253De1003457%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003457%26rft_id%253Dinfo%253Apmid%252F23593039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003457&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23593039&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F16%2F2020.01.11.902890.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.19" data-doi="10.1111/1755-0998.12613">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McKinney
                                  </span>
                                  <span class="cit-name-given-names">
                                   GJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Waples
                                  </span>
                                  <span class="cit-name-given-names">
                                   RK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Seeb
                                  </span>
                                  <span class="cit-name-given-names">
                                   LW
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Seeb
                                  </span>
                                  <span class="cit-name-given-names">
                                   JE
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Paralogs are revealed by proportion of heterozygotes and deviations in read ratios in genotyping-by-sequencing data from natural populations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol Ecol Resour
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  656
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  69
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1111/1755-0998.12613
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%2BResour%26rft.volume%253D17%26rft.spage%253D656%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/1755-0998.12613&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.20" data-doi="10.1371/journal.pgen.1003215">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lu
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lipka
                                  </span>
                                  <span class="cit-name-given-names">
                                   AE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Glaubitz
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Elshire
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cherney
                                  </span>
                                  <span class="cit-name-given-names">
                                   JH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Casler
                                  </span>
                                  <span class="cit-name-given-names">
                                   MD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Genet
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  e1003215
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1371/journal.pgen.1003215
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D9%26rft.spage%253De1003215%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003215%26rft_id%253Dinfo%253Apmid%252F23349638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003215&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23349638&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F16%2F2020.01.11.902890.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.21" data-doi="10.1038/163688a0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Simpson
                                  </span>
                                  <span class="cit-name-given-names">
                                   EH
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Measurement of diversity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1949
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  163
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  688
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/163688a0
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D163%26rft.spage%253D688%26rft_id%253Dinfo%253Adoi%252F10.1038%252F163688a0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/163688a0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.22" data-doi="10.1093/aob/mcy161">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Clark
                                  </span>
                                  <span class="cit-name-given-names">
                                   LV
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jin
                                  </span>
                                  <span class="cit-name-given-names">
                                   X
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Petersen
                                  </span>
                                  <span class="cit-name-given-names">
                                   KK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Anzoua
                                  </span>
                                  <span class="cit-name-given-names">
                                   KG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bagmet
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chebukin
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Population structure of Miscanthus sacchariflorus reveals two major polyploidization events, tetraploid-mediated unidirectional introgression from diploid M. sinensis, and diversity centred around the Yellow Sea
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Ann Bot
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  124
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  731
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  48
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/aob/mcy161
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnn%2BBot%26rft.volume%253D124%26rft.spage%253D731%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcy161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/aob/mcy161&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.23" data-doi="10.1534/g3.115.019703">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Clevenger
                                  </span>
                                  <span class="cit-name-given-names">
                                   JP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ozias-Akins
                                  </span>
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  SWEEP: A Tool for Filtering High-Quality SNPs in Polyploid Crops
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  G3
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1797
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  803
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1534/g3.115.019703
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DG3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.019703%26rft_id%253Dinfo%253Apmid%252F26153076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjUvOS8xNzk3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDIvMTYvMjAyMC4wMS4xMS45MDI4OTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.24" data-doi="10.1534/g3.119.400620">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mollinari
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Olukolu
                                  </span>
                                  <span class="cit-name-given-names">
                                   BA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Da Pereira
                                  </span>
                                  <span class="cit-name-given-names">
                                   GS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Khan
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gemenet
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Craig Yencho
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Unraveling the hexaploid sweetpotato inheritance using ultra-dense multilocus mapping
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  G3 Genes, Genomes, Genet
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  281
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  92
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1534/g3.119.400620
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DG3%2BGenes%252C%2BGenomes%252C%2BGenet%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.119.400620%26rft_id%253Dinfo%253Apmid%252F31732504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjEwLzEvMjgxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDIvMTYvMjAyMC4wMS4xMS45MDI4OTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               25.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.25" data-doi="10.3389/fpls.2018.01660">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kyriakidou
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tai
                                  </span>
                                  <span class="cit-name-given-names">
                                   HH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Anglin
                                  </span>
                                  <span class="cit-name-given-names">
                                   NL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ellis
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Strömvik
                                  </span>
                                  <span class="cit-name-given-names">
                                   M V.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Current strategies of polyploid plant genome sequence assembly
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Front Plant Sci
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1660
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.3389/fpls.2018.01660
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFront%2BPlant%2BSci%26rft.volume%253D9%26rft.spage%253D1660%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2018.01660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.3389/fpls.2018.01660&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               26.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.26" data-doi="10.1371/journal.pone.0033821">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ma
                                  </span>
                                  <span class="cit-name-given-names">
                                   X-F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jensen
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alexandrov
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Troukhan
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thomas-Jones
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  High resolution genetic mapping by genome sequencing reveals genome duplication and tetraploid genetic structure of the diploid Miscanthus sinensis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS One
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  e33821
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1371/journal.pone.0033821
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMa%26rft.auinit1%253DX.%2BF.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De33821%26rft.epage%253De33821%26rft.atitle%253DHigh%2Bresolution%2Bgenetic%2Bmapping%2Bby%2Bgenome%2Bsequencing%2Breveals%2Bgenome%2Bduplication%2Band%2Btetraploid%2Bgenetic%2Bstructure%2Bof%2Bthe%2Bdiploid%2BMiscanthus%2Bsinensis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0033821%26rft_id%253Dinfo%253Apmid%252F22439001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0033821&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22439001&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F16%2F2020.01.11.902890.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               27.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.11.902890v3.27" data-doi="10.1186/1471-2164-13-142">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Swaminathan
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chae
                                  </span>
                                  <span class="cit-name-given-names">
                                   WB
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mitros
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Varala
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xie
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barling
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  A framework genetic map for Miscanthus sinensis from RNAseq-based markers shows recent tetraploidy
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Genomics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  142
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1186/1471-2164-13-142
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DSwaminathan%26rft.auinit1%253DK.%26rft.volume%253D13%26rft.spage%253D142%26rft.epage%253D142%26rft.atitle%253DA%2Bframework%2Bgenetic%2Bmap%2Bfor%2BMiscanthus%2Bsinensis%2Bfrom%2BRNAseq-based%2Bmarkers%2Bshows%2Brecent%2Btetraploidy.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-142%26rft_id%253Dinfo%253Apmid%252F22524439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-142&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22524439&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F16%2F2020.01.11.902890.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               28.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.28" data-doi="10.1007/s00122-012-1790-1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Auckland
                                  </span>
                                  <span class="cit-name-given-names">
                                   SA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rainville
                                  </span>
                                  <span class="cit-name-given-names">
                                   LK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jakob
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kronmiller
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  SSR-based genetic maps of Miscanthus sinensis and M
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  sacchariflorus, and their comparison to sorghum. Theor Appl Genet
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  124
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1325
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  38
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1007/s00122-012-1790-1
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DKim%26rft.auinit1%253DC.%26rft.volume%253D124%26rft.issue%253D7%26rft.spage%253D1325%26rft.epage%253D1338%26rft.atitle%253DSSR-based%2Bgenetic%2Bmaps%2Bof%2BMiscanthus%2Bsinensis%2Band%2BM.%2Bsacchariflorus%252C%2Band%2Btheir%2Bcomparison%2Bto%2Bsorghum.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1790-1%26rft_id%253Dinfo%253Apmid%252F22274765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s00122-012-1790-1&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22274765&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F16%2F2020.01.11.902890.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               29.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.29" data-doi="10.1093/jxb/eru511">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Clark
                                  </span>
                                  <span class="cit-name-given-names">
                                   LV
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stewart
                                  </span>
                                  <span class="cit-name-given-names">
                                   JR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nishiwaki
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Toma
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kjeldsen
                                  </span>
                                  <span class="cit-name-given-names">
                                   JB
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jørgensen
                                  </span>
                                  <span class="cit-name-given-names">
                                   U
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Genetic structure of Miscanthus sinensis and Miscanthus sacchariflorus in Japan indicates a gradient of bidirectional but asymmetric introgression
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Exp Bot
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  66
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  4213
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  25
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/jxb/eru511
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BExp%2BBot%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Feru511%26rft_id%253Dinfo%253Apmid%252F25618143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/jxb/eru511&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25618143&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F16%2F2020.01.11.902890.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               30.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.30" data-doi="10.1270/jsbbs1951.5.167">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hirayoshi
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nishikawa
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kato
                                  </span>
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Cytogenetical Studies on forage plants. (IV) Self-incompatibility in Miscanthus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Japanese J Plant Breed
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1955
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  167
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  70
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1270/jsbbs1951.5.167
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJapanese%2BJ%2BPlant%2BBreed%26rft.volume%253D5%26rft.spage%253D167%26rft_id%253Dinfo%253Adoi%252F10.1270%252Fjsbbs1951.5.167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1270/jsbbs1951.5.167&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               31.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.31" data-doi="10.1093/bioinformatics/btm308">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bradbury
                                  </span>
                                  <span class="cit-name-given-names">
                                   PJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  <span class="cit-name-given-names">
                                   Z
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kroon
                                  </span>
                                  <span class="cit-name-given-names">
                                   DE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Casstevens
                                  </span>
                                  <span class="cit-name-given-names">
                                   TM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ramdoss
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Buckler
                                  </span>
                                  <span class="cit-name-given-names">
                                   ES
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  TASSEL: software for association mapping of complex traits in diverse samples
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  23
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  2633
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  5
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/bioinformatics/btm308
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm308&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17586829&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F16%2F2020.01.11.902890.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000250673800021&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               32.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.32" data-doi="10.1038/s41467-020-18923-6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mitros
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Session
                                  </span>
                                  <span class="cit-name-given-names">
                                   AM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   James
                                  </span>
                                  <span class="cit-name-given-names">
                                   BT
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  <span class="cit-name-given-names">
                                   GA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Belaffif
                                  </span>
                                  <span class="cit-name-given-names">
                                   MB
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Clark
                                  </span>
                                  <span class="cit-name-given-names">
                                   L V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Genome biology of the paleotetraploid perennial biomass crop Miscanthus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Commun
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  5442
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/s41467-020-18923-6
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D11%26rft.spage%253D5442%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-020-18923-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41467-020-18923-6&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               33.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.33" data-doi="10.1111/tpj.13781">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McCormick
                                  </span>
                                  <span class="cit-name-given-names">
                                   RF
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Truong
                                  </span>
                                  <span class="cit-name-given-names">
                                   SK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sreedasyam
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jenkins
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shu
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sims
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The Sorghum bicolor reference genome: improved assembly, gene annotations, a transcriptome atlas, and signatures of genome organization
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Plant J
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  93
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  338
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  54
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1111/tpj.13781
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPlant%2BJ%26rft.volume%253D93%26rft.spage%253D338%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13781%26rft_id%253Dinfo%253Apmid%252F29161754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/tpj.13781&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29161754&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F16%2F2020.01.11.902890.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               34.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.34" data-doi="10.1038/nmeth.1923">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Langmead
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Salzberg
                                  </span>
                                  <span class="cit-name-given-names">
                                   SL
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Fast gapped-read alignment with Bowtie 2
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Methods
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  357
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  9
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/nmeth.1923
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22388286&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F16%2F2020.01.11.902890.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000302218500017&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               35.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.11.902890v3.35" data-doi="10.1186/s13029-016-0057-7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Clark
                                  </span>
                                  <span class="cit-name-given-names">
                                   LV
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sacks
                                  </span>
                                  <span class="cit-name-given-names">
                                   EJ
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  TagDigger: user-friendly extraction of read counts from GBS and RAD-seq data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Source Code Biol Med
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  11
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1186/s13029-016-0057-7
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSource%2BCode%2BBiol%2BMed%26rft.volume%253D11%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13029-016-0057-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13029-016-0057-7&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.01.11.902890v3" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.11.03.467178v3" title="MTG16 (CBFA2T3) regulates colonic epithelial differentiation, colitis, and tumorigenesis by repressing E protein transcription factors" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.02.16.479736v1" title="TraDIS-validate: a method for curating ordered gene-replacement libraries" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted February 16, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.01.11.902890v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.01.11.902890v3.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/02/16/2020.01.11.902890.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.01.11.902890v3.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.01.11.902890v3" class="active" data-icon-position="" data-hide-link-title="0">
                                   A population-level statistic for assessing Mendelian behavior of genotyping-by-sequencing data from highly duplicated genomes
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2405243"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-92W-NqchHGdCJgz04nj-FzcJBcHM_YNmWdc3HptmtMI"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484466471"/>
                                   <input type="hidden" name="captcha_token" value="3f86e8dcac37959005c00c86d5d9b905"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2405243--3" data-node-nid="2405243" id="node-2405243--4691341865" data-pisa="biorxiv;2020.01.11.902890v3" data-pisa-master="biorxiv;2020.01.11.902890" data-seqnum="2405243" data-apath="/biorxiv/early/2022/02/16/2020.01.11.902890.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 A population-level statistic for assessing Mendelian behavior of genotyping-by-sequencing data from highly duplicated genomes
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Lindsay V.
                                  </span>
                                  <span class="nlm-surname">
                                   Clark
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Wittney
                                  </span>
                                  <span class="nlm-surname">
                                   Mays
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Alexander E.
                                  </span>
                                  <span class="nlm-surname">
                                   Lipka
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Erik J.
                                  </span>
                                  <span class="nlm-surname">
                                   Sacks
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.01.11.902890;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.01.11.902890
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.01.11.902890v3" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.01.11.902890v3&title=A%20population-level%20statistic%20for%20assessing%20Mendelian%20behavior%20of%20genotyping-by-sequencing%20data%20from%20highly%20duplicated%20genomes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.11.902890v3&title=A%20population-level%20statistic%20for%20assessing%20Mendelian%20behavior%20of%20genotyping-by-sequencing%20data%20from%20highly%20duplicated%20genomes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.11.902890v3&text=A%20population-level%20statistic%20for%20assessing%20Mendelian%20behavior%20of%20genotyping-by-sequencing%20data%20from%20highly%20duplicated%20genomes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.01.11.902890v3&t=A%20population-level%20statistic%20for%20assessing%20Mendelian%20behavior%20of%20genotyping-by-sequencing%20data%20from%20highly%20duplicated%20genomes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.01.11.902890v3&title=A%20population-level%20statistic%20for%20assessing%20Mendelian%20behavior%20of%20genotyping-by-sequencing%20data%20from%20highly%20duplicated%20genomes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.01.11.902890v3&title=A%20population-level%20statistic%20for%20assessing%20Mendelian%20behavior%20of%20genotyping-by-sequencing%20data%20from%20highly%20duplicated%20genomes&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.11.902890v3&title=A%20population-level%20statistic%20for%20assessing%20Mendelian%20behavior%20of%20genotyping-by-sequencing%20data%20from%20highly%20duplicated%20genomes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2405243--5" data-node-nid="2405243" id="citation-node-2405243--61360639315" data-pisa="biorxiv;2020.01.11.902890v3" data-pisa-master="biorxiv;2020.01.11.902890" data-seqnum="2405243" data-apath="/biorxiv/early/2022/02/16/2020.01.11.902890.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     A population-level statistic for assessing Mendelian behavior of genotyping-by-sequencing data from highly duplicated genomes
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Lindsay V.
                                      </span>
                                      <span class="nlm-surname">
                                       Clark
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Wittney
                                      </span>
                                      <span class="nlm-surname">
                                       Mays
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Alexander E.
                                      </span>
                                      <span class="nlm-surname">
                                       Lipka
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Erik J.
                                      </span>
                                      <span class="nlm-surname">
                                       Sacks
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.01.11.902890;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.01.11.902890
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=A%20population-level%20statistic%20for%20assessing%20Mendelian%20behavior%20of%20genotyping-by-sequencing%20data%20from%20highly%20duplicated%20genomes&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Clark%2C+Lindsay+V.&rft.au=Mays%2C+Wittney&rft.au=Lipka%2C+Alexander+E.&rft.au=Sacks%2C+Erik+J.">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2405243/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2405243/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2405243/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2405243/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2405243/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2405243/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2405243/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2405243/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2405243/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2405243/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2405243/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2405243/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.11.902890v3&count=horizontal&via=&text=A%20population-level%20statistic%20for%20assessing%20Mendelian%20behavior%20of%20genotyping-by-sequencing%20data%20from%20highly%20duplicated%20genomes&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.01.11.902890v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.01.11.902890v3&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.01.11.902890v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Bioinformatics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15186)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9196)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
