<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.medrxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.10.13.20212035v4.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.10.13.20212035v4.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.10.13.20212035v4.ppt"/>
  <meta name="article_thumbnail" content="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/embed/graphic-6.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/medrxiv/early/2022/08/16/2020.10.13.20212035.atom"/>
  <meta name="HW.pisa" content="medrxiv;2020.10.13.20212035v4"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Virus detection and identification in minutes using single-particle imaging and deep learning"/>
  <meta name="DC.Identifier" content="10.1101/2020.10.13.20212035"/>
  <meta name="DC.Date" content="2022-08-16"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory Press"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The increasing frequency and magnitude of viral outbreaks in recent decades, epitomized by the current COVID-19 pandemic, has resulted in an urgent need for rapid and sensitive diagnostic methods. Here, we present a methodology for virus detection and identification that uses a convolutional neural network to distinguish between microscopy images of single intact particles of different viruses. Our assay achieves labeling, imaging and virus identification in less than five minutes and does not require any lysis, purification or amplification steps. The trained neural network was able to differentiate SARS-CoV-2 from negative clinical samples, as well as from other common respiratory pathogens such as influenza and seasonal human coronaviruses. Additionally, we were able to differentiate closely related strains of influenza, as well as SARS-CoV-2 variants. Single-particle imaging combined with deep learning therefore offers a promising alternative to traditional viral diagnostic and genomic sequencing methods, and has the potential for significant impact.

### Competing Interest Statement

This study was supported by the National Institute for Health Research (NIHR) Health Protection Research Unit in Healthcare Associated Infections and Antimicrobial Resistance (NIHR200915), a partnership between the UK Health Security Agency (UKHSA) and the University of Oxford, and the NIHR Oxford Biomedical Research Centre (BRC). The report presents independent research. The views expressed are those of the author(s) and not necessarily those of the NIHR, UKHSA or the Department of Health and Social Care. The work was carried out using a wide-field microscope from Oxford Nanoimaging, a company in which A.N.K. is a co-founder and shareholder, and is being commercialised by OxDx Ltd., a company in which N.C.R. and N.S. are co-founders. Patent applications relating to the work have been submitted by N.C.R., A.N.K. and N.S. (PCT/GB2019/053073 and PCT/GB2021/050990). The authors declare no other competing interests. 

### Funding Statement

We are grateful to Micron Oxford, funded by Wellcome Strategic Awards (091911 and 107457; PI Ilan Davis), for their loan of their microscope and to Nadia Halidi for her help with the instrument. This research was supported by a Royal Society Dorothy Hodgkin Research Fellowship DKR00620 and Research Grant for Research Fellows RGF\R1\180054 (N.C.R.), the University of Oxford COVID-19 Research Response Fund (N.C.R and A.N.K.), a BBSRC-funded studentship (N.S.), and Wellcome Trust grant 110164/Z/15/Z (A.N.K.). All data will be available upon request.

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

North West-Greater Manchester South Research Ethics Committee [REC], REC Ref:19/NW/0730

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

Data available upon request."/>
  <meta name="DC.Contributor" content="Nicolas Shiaelis"/>
  <meta name="DC.Contributor" content="Alexander Tometzki"/>
  <meta name="DC.Contributor" content="Leon Peto"/>
  <meta name="DC.Contributor" content="Andrew McMahon"/>
  <meta name="DC.Contributor" content="Christof Hepp"/>
  <meta name="DC.Contributor" content="Erica Bickerton"/>
  <meta name="DC.Contributor" content="Cyril Favard"/>
  <meta name="DC.Contributor" content="Delphine Muriaux"/>
  <meta name="DC.Contributor" content="Monique Andersson"/>
  <meta name="DC.Contributor" content="Sarah Oakley"/>
  <meta name="DC.Contributor" content="Alison Vaughan"/>
  <meta name="DC.Contributor" content="Philippa C. Matthews"/>
  <meta name="DC.Contributor" content="Nicole Stoesser"/>
  <meta name="DC.Contributor" content="Derrick Crook"/>
  <meta name="DC.Contributor" content="Achillefs N. Kapanidis"/>
  <meta name="DC.Contributor" content="Nicole C. Robb"/>
  <meta name="article:published_time" content="2022-08-16"/>
  <meta name="citation_title" content="Virus detection and identification in minutes using single-particle imaging and deep learning"/>
  <meta name="citation_abstract" lang="en" content="<h3>ABSTRACT</h3>
<p>The increasing frequency and magnitude of viral outbreaks in recent decades, epitomized by the current COVID-19 pandemic, has resulted in an urgent need for rapid and sensitive diagnostic methods. Here, we present a methodology for virus detection and identification that uses a convolutional neural network to distinguish between microscopy images of single intact particles of different viruses. Our assay achieves labeling, imaging and virus identification in less than five minutes and does not require any lysis, purification or amplification steps. The trained neural network was able to differentiate SARS-CoV-2 from negative clinical samples, as well as from other common respiratory pathogens such as influenza and seasonal human coronaviruses. Additionally, we were able to differentiate closely related strains of influenza, as well as SARS-CoV-2 variants. Single-particle imaging combined with deep learning therefore offers a promising alternative to traditional viral diagnostic and genomic sequencing methods, and has the potential for significant impact.</p>"/>
  <meta name="citation_journal_title" content="medRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory Press"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="medrxiv;2020.10.13.20212035v4"/>
  <meta name="citation_id" content="2020.10.13.20212035v4"/>
  <meta name="citation_public_url" content="https://www.medrxiv.org/content/10.1101/2020.10.13.20212035v4"/>
  <meta name="citation_abstract_html_url" content="https://www.medrxiv.org/content/10.1101/2020.10.13.20212035v4.abstract"/>
  <meta name="citation_full_html_url" content="https://www.medrxiv.org/content/10.1101/2020.10.13.20212035v4.full"/>
  <meta name="citation_pdf_url" content="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.10.13.20212035"/>
  <meta name="citation_num_pages" content="24"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_firstpage" content="2020.10.13.20212035"/>
  <meta name="citation_author" content="Nicolas Shiaelis"/>
  <meta name="citation_author_institution" content="Biological Physics Research Group, Clarendon Laboratory, Department of Physics, University of Oxford"/>
  <meta name="citation_author_email" content="Nicole.Robb@warwick.ac.uk"/>
  <meta name="citation_author_email" content="kapanidis@physics.ox.ac.uk"/>
  <meta name="citation_author_email" content="nicolas.shiaelis@st-annes.ox.ac.uk"/>
  <meta name="citation_author" content="Alexander Tometzki"/>
  <meta name="citation_author_institution" content="Biological Physics Research Group, Clarendon Laboratory, Department of Physics, University of Oxford"/>
  <meta name="citation_author" content="Leon Peto"/>
  <meta name="citation_author_institution" content="Nuffield Department of Medicine, University of Oxford"/>
  <meta name="citation_author_institution" content="Department of Microbiology, Oxford University Hospitals NHS Foundation Trust"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4100-2163"/>
  <meta name="citation_author" content="Andrew McMahon"/>
  <meta name="citation_author_institution" content="Biological Physics Research Group, Clarendon Laboratory, Department of Physics, University of Oxford"/>
  <meta name="citation_author" content="Christof Hepp"/>
  <meta name="citation_author_institution" content="Biological Physics Research Group, Clarendon Laboratory, Department of Physics, University of Oxford"/>
  <meta name="citation_author" content="Erica Bickerton"/>
  <meta name="citation_author_institution" content="The Pirbright Institute"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4012-1283"/>
  <meta name="citation_author" content="Cyril Favard"/>
  <meta name="citation_author_institution" content="Membrane Domains and Viral Assembly, IRIM, UMR 9004 CNRS & University of Montpellier"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8304-2980"/>
  <meta name="citation_author" content="Delphine Muriaux"/>
  <meta name="citation_author_institution" content="Membrane Domains and Viral Assembly, IRIM, UMR 9004 CNRS & University of Montpellier"/>
  <meta name="citation_author_institution" content="CEMIPAI, UMS 3725 CNRS & University of Montpellier"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8517-9342"/>
  <meta name="citation_author" content="Monique Andersson"/>
  <meta name="citation_author_institution" content="Department of Microbiology, Oxford University Hospitals NHS Foundation Trust"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0619-1074"/>
  <meta name="citation_author" content="Sarah Oakley"/>
  <meta name="citation_author_institution" content="Department of Microbiology, Oxford University Hospitals NHS Foundation Trust"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2484-8247"/>
  <meta name="citation_author" content="Alison Vaughan"/>
  <meta name="citation_author_institution" content="Nuffield Department of Medicine, University of Oxford"/>
  <meta name="citation_author_institution" content="NIHR Oxford Biomedical Research Centre, University of Oxford"/>
  <meta name="citation_author" content="Philippa C. Matthews"/>
  <meta name="citation_author_institution" content="Nuffield Department of Medicine, University of Oxford"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4036-4269"/>
  <meta name="citation_author" content="Nicole Stoesser"/>
  <meta name="citation_author_institution" content="Nuffield Department of Medicine, University of Oxford"/>
  <meta name="citation_author_institution" content="NIHR Health Protection Research Unit in Healthcare Associated Infections and Antimicrobial Resistance at the University of Oxford in partnership with Public Health England, University of Oxford"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4508-7969"/>
  <meta name="citation_author" content="Derrick Crook"/>
  <meta name="citation_author_institution" content="Nuffield Department of Medicine, University of Oxford"/>
  <meta name="citation_author_institution" content="NIHR Oxford Biomedical Research Centre, University of Oxford"/>
  <meta name="citation_author_institution" content="NIHR Health Protection Research Unit in Healthcare Associated Infections and Antimicrobial Resistance at the University of Oxford in partnership with Public Health England, University of Oxford"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0590-2850"/>
  <meta name="citation_author" content="Achillefs N. Kapanidis"/>
  <meta name="citation_author_institution" content="Biological Physics Research Group, Clarendon Laboratory, Department of Physics, University of Oxford"/>
  <meta name="citation_author_institution" content="The Kavli Institute for Nanoscience Discovery, New Biochemistry Building, University of Oxford"/>
  <meta name="citation_author_email" content="Nicole.Robb@warwick.ac.uk"/>
  <meta name="citation_author_email" content="kapanidis@physics.ox.ac.uk"/>
  <meta name="citation_author_email" content="nicolas.shiaelis@st-annes.ox.ac.uk"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6699-136X"/>
  <meta name="citation_author" content="Nicole C. Robb"/>
  <meta name="citation_author_institution" content="Biological Physics Research Group, Clarendon Laboratory, Department of Physics, University of Oxford"/>
  <meta name="citation_author_institution" content="Warwick Medical School, University of Warwick"/>
  <meta name="citation_author_email" content="Nicole.Robb@warwick.ac.uk"/>
  <meta name="citation_author_email" content="kapanidis@physics.ox.ac.uk"/>
  <meta name="citation_author_email" content="nicolas.shiaelis@st-annes.ox.ac.uk"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7740-6629"/>
  <meta name="citation_reference" content="B. Udugama et al., Diagnosing COVID-19: The Disease and Tools for Detection. ACS Nano 14, 3822–3835 (2020)."/>
  <meta name="citation_reference" content="W. E. Huang et al., RT-LAMP for rapid diagnosis of coronavirus SARS-CoV-2. Microb Biotechnol 13, 950–961 (2020)."/>
  <meta name="citation_reference" content="R. Lu et al., Development of a Novel Reverse Transcription Loop-Mediated Isothermal Amplification Method for Rapid Detection of SARS-CoV-2. Virol Sin, (2020)."/>
  <meta name="citation_reference" content="L. Lamb, Bartolone, SN, Elijah, W, Chancellor, MB, Rapid Detection of Novel Coronavirus (COVID-19) by Reverse Transcription-Loop-Mediated Isothermal Amplification. medRxiv, (2020)."/>
  <meta name="citation_reference" content="Y. Zhang, Odiwuor, N., Xiong, J., Sun, L., Nyaruaba, RO, Wei, H, Tanner, NA, Rapid Molecular Detection of SARS-CoV-2 (COVID-19) Virus RNA Using Colorimetric LAMP. medRxiv, (2020)."/>
  <meta name="citation_reference" content="M. El-Tholoth, E. Anis, H. H. Bau, Two stage, nested isothermal amplification in a single tube. Analyst 146, 1311–1319 (2021)."/>
  <meta name="citation_reference" content="J. Rodriguez-Manzano et al., Handheld Point-of-Care System for Rapid Detection of SARS-CoV-2 Extracted RNA in under 20 min. ACS Cent Sci 7, 307–317 (2021)."/>
  <meta name="citation_reference" content="J. Dinnes et al., Rapid, point-of-care antigen and molecular-based tests for diagnosis of SARS-CoV-2 infection. Cochrane Database Syst Rev 3, CD013705 (2021)."/>
  <meta name="citation_reference" content="N. C. Robb et al., Rapid functionalisation and detection of viruses via a novel Ca(2+)-mediated virus-DNA interaction. Sci Rep 9, 16219 (2019)."/>
  <meta name="citation_reference" content="W. Rawat, Z. Wang, Deep Convolutional Neural Networks for Image Classification: A Comprehensive Review. Neural Comput 29, 2352–2449 (2017)."/>
  <meta name="citation_reference" content="Y. LeCun, Y. Bengio, G. Hinton, Deep learning. Nature 521, 436–444 (2015)."/>
  <meta name="citation_reference" content="R. F. Laine et al., Structured illumination microscopy combined with machine learning enables the high throughput analysis and classification of virus structure. Elife 7, (2018)."/>
  <meta name="citation_reference" content="E. Ito, T. Sato, D. Sano, E. Utagawa, T. Kato, Virus Particle Detection by Convolutional Neural Network in Transmission Electron Microscopy Images. Food Environ Virol 10, 201–208 (2018)."/>
  <meta name="citation_reference" content="R. Arnaout et al., SARS-CoV2 Testing: The Limit of Detection Matters. bioRxiv, (2020)."/>
  <meta name="citation_reference" content="Y. Pan, D. Zhang, P. Yang, L. L. M. Poon, Q. Wang, Viral load of SARS-CoV-2 in clinical samples. Lancet Infect Dis 20, 411–412 (2020)."/>
  <meta name="citation_reference" content="A. Awadasseid, Y. Wu, Y. Tanaka, W. Zhang, Initial success in the identification and management of the coronavirus disease 2019 (COVID-19) indicates human-to-human transmission in Wuhan, China. Int J Biol Sci 16, 1846–1860 (2020)."/>
  <meta name="citation_reference" content="B. Michen, T. Graule, Isoelectric points of viruses. J Appl Microbiol 109, 388–397 (2010)."/>
  <meta name="citation_reference" content="C. Scheller et al., Physicochemical properties of SARS-CoV-2 for drug targeting, virus inactivation and attenuation, vaccine formulation and quality control. Electrophoresis 41, 1137–1151 (2020)."/>
  <meta name="citation_reference" content="E. C. Sadasiv, T. T. Yeh, P. W. Chang, Protein pI alteration related to strain variation of infectious bronchitis virus, an avian Coronavirus. J Virol Methods 33, 115–125 (1991)."/>
  <meta name="citation_reference" content="DeepDreaming with TensorFlow. (2022)."/>
  <meta name="citation_reference" content="S. Lyonnais et al., Atomic force microscopy analysis of native infectious and inactivated SARS-CoV-2 virions. Sci Rep 11, 11885 (2021)."/>
  <meta name="citation_reference" content="R. Casais, V. Thiel, S. G. Siddell, D. Cavanagh, P. Britton, Reverse genetics system for the avian coronavirus infectious bronchitis virus. J Virol 75, 12359–12369 (2001)."/>
  <meta name="citation_reference" content="A. Bal et al., Two-step strategy for the identification of SARS-CoV-2 variant of concern 202012/01 and other variants with spike deletion H69-V70, France, August to December 2020. Euro Surveill 26, (2021)."/>
  <meta name="citation_reference" content="C. M. Bishop, Pattern Recognition and Machine Learning. (Springer, New York, NY, 2006), pp. 738."/>
  <meta name="citation_reference" content="Mathworks. (2019)."/>
  <meta name="citation_reference" content="D. G. Altman, J. M. Bland, Diagnostic tests. 1: Sensitivity and specificity. BMJ 308, 1552 (1994)."/>
  <meta name="twitter:title" content="Virus detection and identification in minutes using single-particle imaging and deep learning"/>
  <meta name="twitter:site" content="@medrxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.medrxiv.org/sites/default/files/images/medrxiv_logo_homepage7-5-small-test-up.png"/>
  <meta name="twitter:description" content="The increasing frequency and magnitude of viral outbreaks in recent decades, epitomized by the current COVID-19 pandemic, has resulted in an urgent need for rapid and sensitive diagnostic methods. Here, we present a methodology for virus detection and identification that uses a convolutional neural network to distinguish between microscopy images of single intact particles of different viruses. Our assay achieves labeling, imaging and virus identification in less than five minutes and does not require any lysis, purification or amplification steps. The trained neural network was able to differentiate SARS-CoV-2 from negative clinical samples, as well as from other common respiratory pathogens such as influenza and seasonal human coronaviruses. Additionally, we were able to differentiate closely related strains of influenza, as well as SARS-CoV-2 variants. Single-particle imaging combined with deep learning therefore offers a promising alternative to traditional viral diagnostic and genomic sequencing methods, and has the potential for significant impact.

### Competing Interest Statement

This study was supported by the National Institute for Health Research (NIHR) Health Protection Research Unit in Healthcare Associated Infections and Antimicrobial Resistance (NIHR200915), a partnership between the UK Health Security Agency (UKHSA) and the University of Oxford, and the NIHR Oxford Biomedical Research Centre (BRC). The report presents independent research. The views expressed are those of the author(s) and not necessarily those of the NIHR, UKHSA or the Department of Health and Social Care. The work was carried out using a wide-field microscope from Oxford Nanoimaging, a company in which A.N.K. is a co-founder and shareholder, and is being commercialised by OxDx Ltd., a company in which N.C.R. and N.S. are co-founders. Patent applications relating to the work have been submitted by N.C.R., A.N.K. and N.S. (PCT/GB2019/053073 and PCT/GB2021/050990). The authors declare no other competing interests. 

### Funding Statement

We are grateful to Micron Oxford, funded by Wellcome Strategic Awards (091911 and 107457; PI Ilan Davis), for their loan of their microscope and to Nadia Halidi for her help with the instrument. This research was supported by a Royal Society Dorothy Hodgkin Research Fellowship DKR00620 and Research Grant for Research Fellows RGF\R1\180054 (N.C.R.), the University of Oxford COVID-19 Research Response Fund (N.C.R and A.N.K.), a BBSRC-funded studentship (N.S.), and Wellcome Trust grant 110164/Z/15/Z (A.N.K.). All data will be available upon request.

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

North West-Greater Manchester South Research Ethics Committee [REC], REC Ref:19/NW/0730

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

Data available upon request."/>
  <meta name="og-title" property="og:title" content="Virus detection and identification in minutes using single-particle imaging and deep learning"/>
  <meta name="og-url" property="og:url" content="https://www.medrxiv.org/content/10.1101/2020.10.13.20212035v4"/>
  <meta name="og-site-name" property="og:site_name" content="medRxiv"/>
  <meta name="og-description" property="og:description" content="The increasing frequency and magnitude of viral outbreaks in recent decades, epitomized by the current COVID-19 pandemic, has resulted in an urgent need for rapid and sensitive diagnostic methods. Here, we present a methodology for virus detection and identification that uses a convolutional neural network to distinguish between microscopy images of single intact particles of different viruses. Our assay achieves labeling, imaging and virus identification in less than five minutes and does not require any lysis, purification or amplification steps. The trained neural network was able to differentiate SARS-CoV-2 from negative clinical samples, as well as from other common respiratory pathogens such as influenza and seasonal human coronaviruses. Additionally, we were able to differentiate closely related strains of influenza, as well as SARS-CoV-2 variants. Single-particle imaging combined with deep learning therefore offers a promising alternative to traditional viral diagnostic and genomic sequencing methods, and has the potential for significant impact.

### Competing Interest Statement

This study was supported by the National Institute for Health Research (NIHR) Health Protection Research Unit in Healthcare Associated Infections and Antimicrobial Resistance (NIHR200915), a partnership between the UK Health Security Agency (UKHSA) and the University of Oxford, and the NIHR Oxford Biomedical Research Centre (BRC). The report presents independent research. The views expressed are those of the author(s) and not necessarily those of the NIHR, UKHSA or the Department of Health and Social Care. The work was carried out using a wide-field microscope from Oxford Nanoimaging, a company in which A.N.K. is a co-founder and shareholder, and is being commercialised by OxDx Ltd., a company in which N.C.R. and N.S. are co-founders. Patent applications relating to the work have been submitted by N.C.R., A.N.K. and N.S. (PCT/GB2019/053073 and PCT/GB2021/050990). The authors declare no other competing interests. 

### Funding Statement

We are grateful to Micron Oxford, funded by Wellcome Strategic Awards (091911 and 107457; PI Ilan Davis), for their loan of their microscope and to Nadia Halidi for her help with the instrument. This research was supported by a Royal Society Dorothy Hodgkin Research Fellowship DKR00620 and Research Grant for Research Fellows RGF\R1\180054 (N.C.R.), the University of Oxford COVID-19 Research Response Fund (N.C.R and A.N.K.), a BBSRC-funded studentship (N.S.), and Wellcome Trust grant 110164/Z/15/Z (A.N.K.). All data will be available upon request.

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

North West-Greater Manchester South Research Ethics Committee [REC], REC Ref:19/NW/0730

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

Data available upon request."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.medrxiv.org/sites/default/files/images/medrxiv_logo_homepage7-5-small-test-up.png"/>
  <meta name="citation_date" content="2022-08-16"/>
  <meta name="description" content="medRxiv - The Preprint Server for Health Sciences"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.medrxiv.org/content/10.1101/2020.10.13.20212035v4"/>
  <link rel="shortlink" href="https://www.medrxiv.org/node/549662"/>
  <title>
   Virus detection and identification in minutes using single-particle imaging and deep learning | medRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__MF4P_i8lKWP7DQR7GmjDAVTRBKLnzxvs1Dvton-Awi0__yi5OflfckG_G8-Li4dDQQ3wO9cGIcdPx_YQ7ZZDuj5w__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__5mxrBK4DuyvC2MR5irx_TzVn9c8vr4vIjmBXjunHfxE__rtQnXv8GOc-AsynbhzSlQBD_2lVxAkB6XEFUCK72qXc__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom_medrxiv.css" media="all"/>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-141606356-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"lmjfrLyuUMR2kBBaEuqqluCZfwJbEkCseDEm1cbL7jY"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"549662","apath":"\/medrxiv\/early\/2022\/08\/16\/2020.10.13.20212035.atom","pisa":"medrxiv;2020.10.13.20212035v4","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"medrxiv;2020.10.13.20212035v4"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"medRxiv","url":"https:\/\/www.medrxiv.org\/content\/10.1101\/2020.10.13.20212035v4","title":"Virus detection and identification in minutes using single-particle imaging and deep learning","identifier":"node\/549662"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022h4AFLXd5e7c6NF1U8A5ggHgI4quLIvLA\\\\\\\/EQCXNdv17pXysshEdbNNJyvC31IpjoL1EV3Tk7cwzTbklql79wu4KpcadKKktY\\\\\\\/mh2npAut59M21Td47whnR4do4brB5l7xhArB26E495T7nDX4AXo7gCQXBI8EiN9yZYWMEtix0nBuv5MGkGdLrYubbRLWbxq4i9duGXvabsXSj05xrgtpOzsxznkBaVCiqjlU5SKkQyZ4SBlv3LWDCUIUERvw5uTJ\\\\\\\/Exyln881JGtTcmSNJDaqXzL+5fGK+5pomYAeuDnXorch+ZaMlB80t86xJS1yaC99mit6v2ZZY+8RaB7OTmHan6SzPeEwBrbxSiXVcRbbjY7rSAL5riXBq5HM8QiJSmQO7ORbOpyWW8GKP5oA8tQb6CSvM7DroXWQnRHKGqf2DkcAmGrGG8xtwkAL3w4I9rrJcJ1jNf\\\\\\\/TGMk3YZEr\\\\\\\/Dg4qmoS1o9UlN5cB7wDu6KqlCQmGW4inNnYY2yBWCEB70YwjQTXFgnr9X+B0Lrq7PM5VSJWczJdKAUjpvnbwtfK+P5NTxYkjxRDnPrC7Bg246+A+QiSEKJn8j3xT0l+Pvmj0hNfCCffVKf+WUN3VIw581CRp\\\\\\\/jySR\\\\\\\/DJ9g3Hm1hll0IfW2s4NpcpDyhLM5MR2o9qPBbIXymbRf4hx0R+sd34vPx14ERrNwtAH4BkkxchfWDu+5j0OxmmnqsRnTvQ+SHpHshazD26vpONjZCRPX55JdxYGk3fiom46jAx24GhuIEU+zWuAYw4yvqY3c9AJlKj7BTj54XHD30Lhqa+H+dxwf2sra57WYcjwAEev32wW+mn51f9UJsxSKAjPnCarydsobvxoxwfPbmpJJsmBsjXAU9uIoY9HnMp6HJ1\\\\\\\/yTaAr8g0vzhbH386LwtU+I7cCnvba3a43GClWznMaIlmXjj94Q7h9VJw\\\\\\\/kGaKICZLT15G+Cdk9XGyCVAXnVIZsDG6BjBV4GxqkFNYOZgKwSjgw2yuQqcfBgWpKCaTWzIF2oT9o5qZbbhldYnQRk++RLoB6aW0Qn2E8hsk0LdjCbvYo8Ym4Na6lThl2Z1YzCSLC\\\\\\\/FAehLOa9PYLnBtJc+tAd+2IiKv0IVfAO5S4j6Xu2A5ZVtLh5LU5g07CmDmlYnAiCmOHM693be01kxy8ISAFnfuabUtdDdy6nKNYcOIzTMKNmg7dYpdhpTRE0+k\\\\\\\/Yr+c2FHvRbiyhHpO4+DnOUAtsZSO2tRen4jJXAl+BSe0DDZLrnjzcLvRjcxQb1for\\\\\\\/jaeLpR+P4oJq5t+eQghDfcTRpf8T9ZT1yh8pHQq0Pd\\\\\\\/DDN6xOkDrKQ6\\\\\\\/ghNrILGkzCKxEg40+XvBex4W+qRNygdOUdWaUbln8GNYPTtrUNLkmlaE=\\\u0022,\\\u0022iv\\\u0022:\\\u0022zLa+AzpbmWfRbwzKeBqRlQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00222371893caf512a4df3cab548f3face7594d1df38bc7a385fc0f5320add2d18e5\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.10.13.20212035v4.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.medrxiv.org\/sites\/default\/files\/medrxiv_internal_logo.png"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"medrxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-549662 node-type-highwire-article context-content hw-default-jcode-medrxiv hw-article-type-article">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-P4HH5NV" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-P4HH5NV');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="medRxiv" src="https://www.medrxiv.org/sites/default/files/medrxiv_internal_logo.png"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/content/about-medrxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.10.13.20212035v4.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_873337789">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_873337789" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_2146532365" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-c4CZCqg5k5nF_16crdkPAbcd9Tmt4ZD3WgXpRUuc4w0"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node549662" data-node-nid="549662" id="node-549662--21330360157" data-pisa="medrxiv;2020.10.13.20212035v4" data-pisa-master="medrxiv;2020.10.13.20212035" data-apath="/medrxiv/early/2022/08/16/2020.10.13.20212035.atom" data-hw-author-tooltip-instance="">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-medrxiv-article-top clearfix has-author-tooltip">
                   <h1 class="highwire-cite-title" id="page-title">
                    Virus detection and identification in minutes using single-particle imaging and deep learning
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first" data-delta="0">
                      <span class="nlm-given-names">
                       Nicolas
                      </span>
                      <span class="nlm-surname">
                       Shiaelis
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Alexander
                      </span>
                      <span class="nlm-surname">
                       Tometzki
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2">
                      <a href="http://orcid.org/0000-0002-4100-2163" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Leon
                      </span>
                      <span class="nlm-surname">
                       Peto
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Andrew
                      </span>
                      <span class="nlm-surname">
                       McMahon
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="4">
                      <span class="nlm-given-names">
                       Christof
                      </span>
                      <span class="nlm-surname">
                       Hepp
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5">
                      <a href="http://orcid.org/0000-0002-4012-1283" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Erica
                      </span>
                      <span class="nlm-surname">
                       Bickerton
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6">
                      <a href="http://orcid.org/0000-0002-8304-2980" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Cyril
                      </span>
                      <span class="nlm-surname">
                       Favard
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7">
                      <a href="http://orcid.org/0000-0001-8517-9342" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Delphine
                      </span>
                      <span class="nlm-surname">
                       Muriaux
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8">
                      <a href="http://orcid.org/0000-0003-0619-1074" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Monique
                      </span>
                      <span class="nlm-surname">
                       Andersson
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9">
                      <a href="http://orcid.org/0000-0002-2484-8247" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Sarah
                      </span>
                      <span class="nlm-surname">
                       Oakley
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="10">
                      <span class="nlm-given-names">
                       Alison
                      </span>
                      <span class="nlm-surname">
                       Vaughan
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11">
                      <a href="http://orcid.org/0000-0002-4036-4269" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Philippa C.
                      </span>
                      <span class="nlm-surname">
                       Matthews
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12">
                      <a href="http://orcid.org/0000-0002-4508-7969" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Nicole
                      </span>
                      <span class="nlm-surname">
                       Stoesser
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13">
                      <a href="http://orcid.org/0000-0002-0590-2850" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Derrick
                      </span>
                      <span class="nlm-surname">
                       Crook
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="14">
                      <a href="http://orcid.org/0000-0001-6699-136X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Achillefs N.
                      </span>
                      <span class="nlm-surname">
                       Kapanidis
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="15">
                      <a href="http://orcid.org/0000-0002-7740-6629" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Nicole C.
                      </span>
                      <span class="nlm-surname">
                       Robb
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.10.13.20212035
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-549662--21330360157" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Nicolas Shiaelis
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Biological Physics Research Group, Clarendon Laboratory, Department of Physics, University of Oxford
                       </span>
                       , Oxford, OX1 3PU,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Nicolas%2BShiaelis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Shiaelis%20N&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ANicolas%2BShiaelis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:Nicole.Robb@warwick.ac.uk" class="" data-icon-position="" data-hide-link-title="0">
                        Nicole.Robb@warwick.ac.uk
                       </a>
                       <a href="mailto:kapanidis@physics.ox.ac.uk" class="" data-icon-position="" data-hide-link-title="0">
                        kapanidis@physics.ox.ac.uk
                       </a>
                       <a href="mailto:nicolas.shiaelis@st-annes.ox.ac.uk" class="" data-icon-position="" data-hide-link-title="0">
                        nicolas.shiaelis@st-annes.ox.ac.uk
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Alexander Tometzki
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Biological Physics Research Group, Clarendon Laboratory, Department of Physics, University of Oxford
                       </span>
                       , Oxford, OX1 3PU,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Alexander%2BTometzki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Tometzki%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAlexander%2BTometzki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Leon Peto
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Nuffield Department of Medicine, University of Oxford
                       </span>
                       , Oxford, OX3 9DU,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Microbiology, Oxford University Hospitals NHS Foundation Trust
                       </span>
                       , Oxford, OX3 9DU,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Leon%2BPeto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Peto%20L&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ALeon%2BPeto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-4100-2163" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Leon Peto
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Andrew McMahon
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Biological Physics Research Group, Clarendon Laboratory, Department of Physics, University of Oxford
                       </span>
                       , Oxford, OX1 3PU,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Andrew%2BMcMahon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=McMahon%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAndrew%2BMcMahon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Christof Hepp
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Biological Physics Research Group, Clarendon Laboratory, Department of Physics, University of Oxford
                       </span>
                       , Oxford, OX1 3PU,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Christof%2BHepp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Hepp%20C&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AChristof%2BHepp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Erica Bickerton
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        The Pirbright Institute
                       </span>
                       , Ash Road, Pirbright, Woking, Surrey, GU24 0NF,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Erica%2BBickerton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Bickerton%20E&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AErica%2BBickerton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-4012-1283" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Erica Bickerton
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">
                     Cyril Favard
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Membrane Domains and Viral Assembly, IRIM, UMR 9004 CNRS & University of Montpellier
                       </span>
                       , 1919, route de Mende, 34293 Montpellier,
                       <span class="nlm-country">
                        France
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Cyril%2BFavard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Favard%20C&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ACyril%2BFavard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-8304-2980" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Cyril Favard
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-7">
                    <div class="author-tooltip-name">
                     Delphine Muriaux
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Membrane Domains and Viral Assembly, IRIM, UMR 9004 CNRS & University of Montpellier
                       </span>
                       , 1919, route de Mende, 34293 Montpellier,
                       <span class="nlm-country">
                        France
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        CEMIPAI, UMS 3725 CNRS & University of Montpellier
                       </span>
                       , 1919, route de Mende, 34293 Montpellier,
                       <span class="nlm-country">
                        France
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Delphine%2BMuriaux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Muriaux%20D&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ADelphine%2BMuriaux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-8517-9342" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Delphine Muriaux
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-8">
                    <div class="author-tooltip-name">
                     Monique Andersson
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Microbiology, Oxford University Hospitals NHS Foundation Trust
                       </span>
                       , Oxford, OX3 9DU,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Monique%2BAndersson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Andersson%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AMonique%2BAndersson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-0619-1074" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Monique Andersson
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-9">
                    <div class="author-tooltip-name">
                     Sarah Oakley
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Microbiology, Oxford University Hospitals NHS Foundation Trust
                       </span>
                       , Oxford, OX3 9DU,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Sarah%2BOakley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Oakley%20S&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ASarah%2BOakley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-2484-8247" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Sarah Oakley
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-10">
                    <div class="author-tooltip-name">
                     Alison Vaughan
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Nuffield Department of Medicine, University of Oxford
                       </span>
                       , Oxford, OX3 9DU,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        7
                       </span>
                       <span class="nlm-institution">
                        NIHR Oxford Biomedical Research Centre, University of Oxford
                       </span>
                       , Oxford,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Alison%2BVaughan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Vaughan%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAlison%2BVaughan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-11">
                    <div class="author-tooltip-name">
                     Philippa C. Matthews
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Nuffield Department of Medicine, University of Oxford
                       </span>
                       , Oxford, OX3 9DU,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Philippa%2BC.%2BMatthews%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Matthews%20PC&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3APhilippa%2BC.%2BMatthews%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-4036-4269" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Philippa C. Matthews
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-12">
                    <div class="author-tooltip-name">
                     Nicole Stoesser
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Nuffield Department of Medicine, University of Oxford
                       </span>
                       , Oxford, OX3 9DU,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        8
                       </span>
                       <span class="nlm-institution">
                        NIHR Health Protection Research Unit in Healthcare Associated Infections and Antimicrobial Resistance at the University of Oxford in partnership with Public Health England, University of Oxford
                       </span>
                       , Oxford,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Nicole%2BStoesser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Stoesser%20N&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ANicole%2BStoesser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-4508-7969" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Nicole Stoesser
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-13">
                    <div class="author-tooltip-name">
                     Derrick Crook
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Nuffield Department of Medicine, University of Oxford
                       </span>
                       , Oxford, OX3 9DU,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        7
                       </span>
                       <span class="nlm-institution">
                        NIHR Oxford Biomedical Research Centre, University of Oxford
                       </span>
                       , Oxford,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        8
                       </span>
                       <span class="nlm-institution">
                        NIHR Health Protection Research Unit in Healthcare Associated Infections and Antimicrobial Resistance at the University of Oxford in partnership with Public Health England, University of Oxford
                       </span>
                       , Oxford,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Derrick%2BCrook%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Crook%20D&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ADerrick%2BCrook%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-0590-2850" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Derrick Crook
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-14">
                    <div class="author-tooltip-name">
                     Achillefs N. Kapanidis
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Biological Physics Research Group, Clarendon Laboratory, Department of Physics, University of Oxford
                       </span>
                       , Oxford, OX1 3PU,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        10
                       </span>
                       <span class="nlm-institution">
                        The Kavli Institute for Nanoscience Discovery, New Biochemistry Building, University of Oxford
                       </span>
                       , South Parks Road, Oxford, OX1 3QU
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Achillefs%2BN.%2BKapanidis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Kapanidis%20AN&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAchillefs%2BN.%2BKapanidis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0001-6699-136X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Achillefs N. Kapanidis
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:Nicole.Robb@warwick.ac.uk" class="" data-icon-position="" data-hide-link-title="0">
                        Nicole.Robb@warwick.ac.uk
                       </a>
                       <a href="mailto:kapanidis@physics.ox.ac.uk" class="" data-icon-position="" data-hide-link-title="0">
                        kapanidis@physics.ox.ac.uk
                       </a>
                       <a href="mailto:nicolas.shiaelis@st-annes.ox.ac.uk" class="" data-icon-position="" data-hide-link-title="0">
                        nicolas.shiaelis@st-annes.ox.ac.uk
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-15">
                    <div class="author-tooltip-name">
                     Nicole C. Robb
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Biological Physics Research Group, Clarendon Laboratory, Department of Physics, University of Oxford
                       </span>
                       , Oxford, OX1 3PU,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        9
                       </span>
                       <span class="nlm-institution">
                        Warwick Medical School, University of Warwick
                       </span>
                       , Coventry, CV4 7AL,
                       <span class="nlm-country">
                        United Kingdom
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Nicole%2BC.%2BRobb%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Robb%20NC&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ANicole%2BC.%2BRobb%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0002-7740-6629" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Nicole C. Robb
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:Nicole.Robb@warwick.ac.uk" class="" data-icon-position="" data-hide-link-title="0">
                        Nicole.Robb@warwick.ac.uk
                       </a>
                       <a href="mailto:kapanidis@physics.ox.ac.uk" class="" data-icon-position="" data-hide-link-title="0">
                        kapanidis@physics.ox.ac.uk
                       </a>
                       <a href="mailto:nicolas.shiaelis@st-annes.ox.ac.uk" class="" data-icon-position="" data-hide-link-title="0">
                        nicolas.shiaelis@st-annes.ox.ac.uk
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.10.13.20212035v4" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:549662" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:549662/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.13.20212035v4.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:549662" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:549662/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.13.20212035v4.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:549662" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:549662/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.13.20212035v4.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:549662" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:549662/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.13.20212035v4.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:549662" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:549662/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.13.20212035v4.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:549662" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:549662/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.10.13.20212035v4.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:549662" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:549662/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.medrxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             ABSTRACT
                            </h2>
                            <p id="p-2">
                             The increasing frequency and magnitude of viral outbreaks in recent decades, epitomized by the current COVID-19 pandemic, has resulted in an urgent need for rapid and sensitive diagnostic methods. Here, we present a methodology for virus detection and identification that uses a convolutional neural network to distinguish between microscopy images of single intact particles of different viruses. Our assay achieves labeling, imaging and virus identification in less than five minutes and does not require any lysis, purification or amplification steps. The trained neural network was able to differentiate SARS-CoV-2 from negative clinical samples, as well as from other common respiratory pathogens such as influenza and seasonal human coronaviruses. Additionally, we were able to differentiate closely related strains of influenza, as well as SARS-CoV-2 variants. Single-particle imaging combined with deep learning therefore offers a promising alternative to traditional viral diagnostic and genomic sequencing methods, and has the potential for significant impact.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             INTRODUCTION
                            </h2>
                            <p id="p-16">
                             The SARS-CoV-2 betacoronavirus has infected hundreds of millions of people since its emergence, resulting in numerous deaths, and causing worldwide social and economic disruption. The emergence of a number of variants of concern (VOCs) that pose an increased risk to global public health by affecting transmission, associated disease severity or vaccine efficacy, have further complicated response efforts.
                            </p>
                            <p id="p-17">
                             Current SARS-CoV-2 diagnostic methods include nucleic acid amplification tests, antigen detection, and serology tests (
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ). Reverse transcriptase polymerase chain reaction (RT-PCR) is considered the gold standard for diagnosis; however, RT-PCR takes several hours to provide a result, is restricted to specialized laboratories (as it requires viral lysis and RNA extraction), and can be limited by supply chain issues. Isothermal nucleic acid amplification methods, such as loop-mediated isothermal amplification (RT-LAMP), offer a promising alternative that does not require thermal cycling and can provide results within an hour (
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             -
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             ); however, these methods are still subject to similar supply chain issues as RT-PCR. Lateral-flow immunochromatographic assays using gold nanoparticles and a colorimetric label to detect SARS-CoV-2-specific antigens provide a rapid platform for point-of-contact virus detection, but can have lower sensitivities (
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ). Viral strain, or variant, identification largely relies on sequencing of the viral genome. There is thus an urgent need for new viral detection approaches, particularly ones that can be deployed in non-laboratory settings.
                            </p>
                            <p id="p-18">
                             In previous published work we described a robust method to rapidly label enveloped virus particles using a solution of a divalent cation (such as Ca
                             <sup>
                              2+
                             </sup>
                             ), short DNAs of non-specific sequence and a particle with a negatively charged lipid bilayer, and suggested that the cations facilitate an interaction between the negatively charged polar heads of the viral lipid membrane and the negatively charged phosphates of the nucleic acid (
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ). By including a fluorophore on the DNAs, we have been able to easily generate bright fluorescent particles for any enveloped virus tested to date (multiple strains of Influenza A, Influenza B, baculovirus, respiratory syncytial virus (RSV), Infectious Bronchitis Virus (IBV), human coronaviruses OC43, HKU1 and NS63, and SARS-CoV-2). We have previously characterised the labelling method to show that virus-specific signals were only observed when the cation, virus and fluorescent DNA were all present (with signals being absent in controls where any one of these major components were excluded), that we could co-stain cation-labelled virus particles with virus-specific antibodies, and that the size of the labelled particles correlated perfectly with that observed in electron microscopy images of the virus (
                             <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ), providing us with confidence that we can specifically label viruses with this method.
                            </p>
                            <p id="p-19">
                             To address the need for new viral detection approaches, we have used this labelling method to develop a novel diagnostic test that relies on the detection of intact virus particles using wide-field fluorescence imaging. Our method starts with the near-instantaneous fluorescence labeling of viruses in a sample; we subsequently surface-immobilise labelled particles, collect diffraction-limited images containing thousands of labelled particles, and finally use image analysis and machine-learning to identify different viruses in biological and clinical samples (
                             <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                              Fig.1A
                             </a>
                             ). Our approach exploits the fact that distinct virus types and strains have differences in surface chemistry, size, and shape, which in turn affects the fluorophore distribution and density over the surface of different viruses. Such differences can be captured by convolutional neural networks (CNNs) (
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ,
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ), which have been used previously to classify super-resolved microscopy images of heterogeneous virus populations into particle classes with distinct structural features (
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ), and to detect virus particles in transmission electron microscopy images (
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ). We have shown that we can use this methodology to differentiate a range of viruses in clinical samples, with high overall sample accuracies of 98.0% (using 51 clinical samples on multiple different versions of the trained network) and 97.1% (using 104 clinical samples on a single trained network). We therefore see an opportunity for our testing platform to make an impact not only during pandemics, but also in the future as a general-use method for diagnostics and a surveillance platform for new emerging pathogens.
                            </p>
                            <div id="F1" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F1.large.jpg?width=800&height=600&carousel=1" title="A fluorescent labelling and imaging strategy to detect viruses. A) Overview. i) Viruses were labelled and imaged. Individual signals were isolated and a convolutional neural network (CNN) was trained to exploit differences in the features of different viruses to identify them. ii) Signals from unknown samples can then be fed into the trained CNN to allow iii) virus classification. B) Representative fields of view (FOVs) of infectious bronchitis virus (CoV (IBV)). 1×104 PFU/mL virus was labelled with 0.23 M SrCl2, 1 nM Cy3 (green) DNA and 1 nM Atto647N (red) DNA before being imaged. Green DNA was observed in the green channel (top panels) and red DNA in the red channel (middle panels); merged red and green localisations are shown in the lower panels. Scale bar 10µm. A negative control where DNA was replaced with water is included. C&D) Zoomed-in images from B), white boxes represent examples of colocalised particles. Scale bar, 5 µm. E) Segmentation process. (i) Cropped FOV from the red channel. (ii) Intensity filtering applied to i) to produce a binary image. (iii) Area filtering applied to ii) to include only 10-100 pixel objects. (iv) Location image associated with i). (v) Colocalised signals in the location image. (vi) Bounding boxes (BBXs) found from iii) drawn onto v). Non-colocalised objects (cyan) are rejected. (vii) Colocalised objects (red) are drawn over i). Scale bar 10µm. F) Plot of mean number of BBXs per FOV for labelled CoV (IBV) and the negative controls. Error bars represent the standard deviation of 81 FOVs from a single slide. Statistical significance was determined by one-way ANOVA, *P=6.01×10−22." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-774246475" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">A fluorescent labelling and imaging strategy to detect viruses.</span> A) Overview. i) Viruses were labelled and imaged. Individual signals were isolated and a convolutional neural network (CNN) was trained to exploit differences in the features of different viruses to identify them. ii) Signals from unknown samples can then be fed into the trained CNN to allow iii) virus classification. B) Representative fields of view (FOVs) of infectious bronchitis virus (CoV (IBV)). 1×10<sup>4</sup> PFU/mL virus was labelled with 0.23 M SrCl<sub>2</sub>, 1 nM Cy3 (green) DNA and 1 nM Atto647N (red) DNA before being imaged. Green DNA was observed in the green channel (top panels) and red DNA in the red channel (middle panels); merged red and green localisations are shown in the lower panels. Scale bar 10µm. A negative control where DNA was replaced with water is included. C&D) Zoomed-in images from B), white boxes represent examples of colocalised particles. Scale bar, 5 µm. E) Segmentation process. (i) Cropped FOV from the red channel. (ii) Intensity filtering applied to i) to produce a binary image. (iii) Area filtering applied to ii) to include only 10-100 pixel objects. (iv) Location image associated with i). (v) Colocalised signals in the location image. (vi) Bounding boxes (BBXs) found from iii) drawn onto v). Non-colocalised objects (cyan) are rejected. (vii) Colocalised objects (red) are drawn over i). Scale bar 10µm. F) Plot of mean number of BBXs per FOV for labelled CoV (IBV) and the negative controls. Error bars represent the standard deviation of 81 FOVs from a single slide. Statistical significance was determined by one-way ANOVA, *P=6.01×10<sup>−22</sup>.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F1.medium.gif" width="357" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F1.medium.gif" width="357" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                              <span class="fig-label">
                               Figure 1.
                              </span>
                              <span class="caption-title">
                               A fluorescent labelling and imaging strategy to detect viruses.
                              </span>
                              <p id="p-20" class="first-child">
                               A) Overview. i) Viruses were labelled and imaged. Individual signals were isolated and a convolutional neural network (CNN) was trained to exploit differences in the features of different viruses to identify them. ii) Signals from unknown samples can then be fed into the trained CNN to allow iii) virus classification.
                              </p>
                              <p id="p-21">
                               B) Representative fields of view (FOVs) of infectious bronchitis virus (CoV (IBV)). 1×10
                               <sup>
                                4
                               </sup>
                               PFU/mL virus was labelled with 0.23 M SrCl
                               <sub>
                                2
                               </sub>
                               , 1 nM Cy3 (green) DNA and 1 nM Atto647N (red) DNA before being imaged. Green DNA was observed in the green channel (top panels) and red DNA in the red channel (middle panels); merged red and green localisations are shown in the lower panels. Scale bar 10µm. A negative control where DNA was replaced with water is included.
                              </p>
                              <p id="p-22">
                               C&D) Zoomed-in images from B), white boxes represent examples of colocalised particles. Scale bar, 5 µm.
                              </p>
                              <p id="p-23">
                               E) Segmentation process. (i) Cropped FOV from the red channel. (ii) Intensity filtering applied to i) to produce a binary image. (iii) Area filtering applied to ii) to include only 10-100 pixel objects. (iv) Location image associated with i). (v) Colocalised signals in the location image. (vi) Bounding boxes (BBXs) found from iii) drawn onto v). Non-colocalised objects (cyan) are rejected. (vii) Colocalised objects (red) are drawn over i). Scale bar 10µm.
                              </p>
                              <p id="p-24">
                               F) Plot of mean number of BBXs per FOV for labelled CoV (IBV) and the negative controls. Error bars represent the standard deviation of 81 FOVs from a single slide. Statistical significance was determined by one-way ANOVA, *P=6.01×10
                               <sup>
                                −22
                               </sup>
                               .
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             RESULTS
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Labelled virus particles can be efficiently detected with TIRF microscopy
                             </h3>
                             <p id="p-25">
                              To demonstrate our ability to label, immobilize, and image virus particles, we initially used infectious bronchitis virus (IBV), an avian coronavirus (CoV). We labelled IBV using a divalent cation (here, Sr
                              <sup>
                               2+
                              </sup>
                              , which performs very similarly to Ca
                              <sup>
                               2+
                              </sup>
                              ; see below) and a mixture of green and red fluorescent DNAs (labelled with Cy3 or Atto647N fluorophores, respectively); immobilized particles on a chitosan-coated glass slide; and imaged particles using total-internal-reflection fluorescence microscopy (TIRF) (Sup.Fig.1A). Fluorescent labelling was achieved within seconds via a single-step addition of labelling mixture (see Methods), after which the viruses were immediately immobilized. The resulting images contained particles with either single green or red fluorescence signals (shown as green and red particles), as well as colocalised green and red fluorescence signals (shown as yellow particles) (
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Fig.1B-D
                              </a>
                              ). Efficient virus labelling was achieved using either CaCl
                              <sub>
                               2
                              </sub>
                              (Sup.Fig.1B&C) or SrCl
                              <sub>
                               2
                              </sub>
                              (
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Fig. 1B-D
                              </a>
                              ), although both solutions resulted in a number of colocalized signals in the virus-negative controls, likely due to random coincidence or cation-mediated clustering of DNAs on the surface. Omission of DNAs resulted in complete loss of the fluorescent signal (
                              <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                               Fig.1B
                              </a>
                              , right panels).
                             </p>
                             <p id="p-26">
                              Prior to use for machine learning, the virus images were pre-processed to isolate individual image signals into bounding boxes (BBXs) using segmentation of the (FOV) through adaptive filtering (
                              <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                               Fig.1E
                              </a>
                              ). The BBX-based approach is preferred over the use of full field of views (FOVs) for classification, since the former is immune to features such as virus concentration or variability in the background and illumination pattern. The raw FOVs from the red channel (
                              <a id="xref-fig-1-6" class="xref-fig" href="#F1">
                               Fig.1Ei
                              </a>
                              ) were converted into a binary format (
                              <a id="xref-fig-1-7" class="xref-fig" href="#F1">
                               Fig.1Eii
                              </a>
                              ) and area filtering used to disregard objects with a total area (i.e. width x length) smaller than 10 pixels (single fluorophores) or larger than 100 pixels (aggregates, cells or cell fragments) (
                              <a id="xref-fig-1-8" class="xref-fig" href="#F1">
                               Fig.1Eiii
                              </a>
                              ). At the same time, to enrich our sampling for viruses and exclude signals not arising from virus particles, the location image (showing the green, red and yellow signals from both channels;
                              <a id="xref-fig-1-9" class="xref-fig" href="#F1">
                               Fig.1Eiv
                              </a>
                              ) was used to identify colocalised signals (
                              <a id="xref-fig-1-10" class="xref-fig" href="#F1">
                               Fig.1Ev
                              </a>
                              ). This information was then combined with the signals identified in the filtered binary image (
                              <a id="xref-fig-1-11" class="xref-fig" href="#F1">
                               Fig.1Eiii
                              </a>
                              ) to reject signals not meeting the colocalisation condition (
                              <a id="xref-fig-1-12" class="xref-fig" href="#F1">
                               Fig.1Evi
                              </a>
                              ; cyan boxes) and retain signals meeting the colocalisation condition (
                              <a id="xref-fig-1-13" class="xref-fig" href="#F1">
                               Fig.1Evi-vii
                              </a>
                              ; red boxes). The segmentation was fully automated, allowing each FOV to be processed in ∼2 seconds. In this experiment, the mean number of colocalised BBXs per FOV obtained when IBV was present was ∼6-fold higher than when the virus was absent (
                              <a id="xref-fig-1-14" class="xref-fig" href="#F1">
                               Fig.1F
                              </a>
                              ), confirming that virus-specific images are being captured in our pre-processing step.
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Positive and negative virus images can be distinguished using deep learning
                             </h3>
                             <p id="p-27">
                              Having shown that we could efficiently image virus samples and isolate the resulting signals into BBXs, we hypothesized that we could use a custom-built CNN to differentiate between signals observed in virus-positive and virus-negative samples, as well as between images of different viruses. To explore this, we fluorescently labelled and imaged IBV, three laboratory-grown influenza A strains: H3N2 A/Udorn/72 (Udorn), H3N2 A/Aichi/68 (X31), and H1N1 A/PR8/8/34 (PR8), and a virus-negative control consisting of virus-free cell culture media (
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Fig.2A
                              </a>
                              ). The viruses are similar in size and shape, and cannot be distinguished by eye in diffraction-limited microscope images of fluorescently labelled particles (Sup.Fig.2). After image segmentation and examination of the properties of the resulting BBXs however, we observed that the four viruses exhibited small, yet statistically significant, differences in maximum pixel intensity, area, and semi-major-to-semi-minor-axis-ratio within the BBXs (
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig.2B-D
                              </a>
                              ); e.g., IBV appears brighter than influenza, whereas Udorn occupies a larger area than the other viruses.
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F2.large.jpg?width=800&height=600&carousel=1" title="Design of a convolutional neural network to classify imaged viruses. A) Representative FOVs of fluorescently labelled coronavirus (CoV (IBV)), two strains of H3N2 influenza (A/Udorn/72 (Udorn) and A/Aichi/68 (X31)), an H1N1 influenza strain (A/PR8/8/34 (PR8)) and a negative control where virus was substituted with allantoic fluid. The samples were immobilized and labelled with 0.23M SrCl2, 1nM Cy3 (green) DNA and 1nM Atto647N (red) DNA before being imaged. Merged red and green localisations are shown, examples of colocalisations are highlighted with white boxes. Scale bar 10µm. B-D) Normalised frequency plots of the maximum pixel intensity, area and semi-major-to-semi-minor-axis-ratio within the BBXs of the four different viruses. Values taken from 81 FOVs from a single slide for each virus. Statistical significance was determined by one-way ANOVA, P values depicted above graphs. E) Illustration of the 15-layer shallow convolutional neural network. Following the input layer (inputs comprising BBXs from the segmentation process), the network consists of three convolutions (stages 1-3). Stages 1 and 2 each contain a ReLU layer to introduce non-linearity, a batch normalisation layer (not shown) and a max pooling layer, while stage 3 lacks a max-pooling layer. The classification stage has a fully-connected layer and a softmax layer to convert the output of the previous layer to a normalised probability distribution, allowing the initial input to be classified." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-774246475" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Design of a convolutional neural network to classify imaged viruses.</span> A) Representative FOVs of fluorescently labelled coronavirus (CoV (IBV)), two strains of H3N2 influenza (A/Udorn/72 (Udorn) and A/Aichi/68 (X31)), an H1N1 influenza strain (A/PR8/8/34 (PR8)) and a negative control where virus was substituted with allantoic fluid. The samples were immobilized and labelled with 0.23M SrCl<sub>2</sub>, 1nM Cy3 (green) DNA and 1nM Atto647N (red) DNA before being imaged. Merged red and green localisations are shown, examples of colocalisations are highlighted with white boxes. Scale bar 10µm. B-D) Normalised frequency plots of the maximum pixel intensity, area and semi-major-to-semi-minor-axis-ratio within the BBXs of the four different viruses. Values taken from 81 FOVs from a single slide for each virus. Statistical significance was determined by one-way ANOVA, P values depicted above graphs. E) Illustration of the 15-layer shallow convolutional neural network. Following the input layer (inputs comprising BBXs from the segmentation process), the network consists of three convolutions (stages 1-3). Stages 1 and 2 each contain a ReLU layer to introduce non-linearity, a batch normalisation layer (not shown) and a max pooling layer, while stage 3 lacks a max-pooling layer. The classification stage has a fully-connected layer and a softmax layer to convert the output of the previous layer to a normalised probability distribution, allowing the initial input to be classified.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F2.medium.gif" width="440" height="362"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F2.medium.gif" width="440" height="362"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2.
                               </span>
                               <span class="caption-title">
                                Design of a convolutional neural network to classify imaged viruses.
                               </span>
                               <p id="p-28" class="first-child">
                                A) Representative FOVs of fluorescently labelled coronavirus (CoV (IBV)), two strains of H3N2 influenza (A/Udorn/72 (Udorn) and A/Aichi/68 (X31)), an H1N1 influenza strain (A/PR8/8/34 (PR8)) and a negative control where virus was substituted with allantoic fluid. The samples were immobilized and labelled with 0.23M SrCl
                                <sub>
                                 2
                                </sub>
                                , 1nM Cy3 (green) DNA and 1nM Atto647N (red) DNA before being imaged. Merged red and green localisations are shown, examples of colocalisations are highlighted with white boxes. Scale bar 10µm.
                               </p>
                               <p id="p-29">
                                B-D) Normalised frequency plots of the maximum pixel intensity, area and semi-major-to-semi-minor-axis-ratio within the BBXs of the four different viruses. Values taken from 81 FOVs from a single slide for each virus. Statistical significance was determined by one-way ANOVA, P values depicted above graphs.
                               </p>
                               <p id="p-30">
                                E) Illustration of the 15-layer shallow convolutional neural network. Following the input layer (inputs comprising BBXs from the segmentation process), the network consists of three convolutions (stages 1-3). Stages 1 and 2 each contain a ReLU layer to introduce non-linearity, a batch normalisation layer (not shown) and a max pooling layer, while stage 3 lacks a max-pooling layer. The classification stage has a fully-connected layer and a softmax layer to convert the output of the previous layer to a normalised probability distribution, allowing the initial input to be classified.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-31">
                              This was further supported by super-resolution imaging of cation-labelled virus particles. Fluorescence-based super-resolution microscopy allowed us to take both diffraction limited and high resolution images of the same fields of view, providing a direct comparison between the signals isolated into BBXs for the machine learning and their super-resolved versions. We immobilised biotinylated viruses on pegylated coverslips before labelling them with CaCl
                              <sub>
                               2
                              </sub>
                              and a DNA conjugated to a photoswitchable Alexa647 dye. When imaged, the fluorescent signals from the Alexa647 DNAs on the virus particles were recorded and each resulting localization was precisely fitted to reconstruct a super-resolved image. Cluster analysis of the super-resolved localisations revealed that the fluorescent signals observed in the diffraction limited images of labelled samples correspond to particles of the correct size and shape of virions, and that different virus classes appear to have subtle differences in their labelling density, area and shape (Sup.Fig.3). These small differences, as well as more abstract image features such as pixel correlations, can be exploited by deep learning algorithms to classify the viruses.
                             </p>
                             <p id="p-32">
                              To classify the different samples, we constructed a 15-layer CNN (
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Fig.2E
                              </a>
                              , see legend for detail). We started by imaging IBV and a virus-negative control consisting of only SrCl
                              <sub>
                               2
                              </sub>
                              and DNA. The two samples were independently imaged four times each over a three-day period. Imaging over three days allowed any potential heterogeneity in the image procurement process (such as small differences in temperature on different days) to be captured, in order to enhance the ability of the trained models to classify data from future datasets. The resulting BBXs obtained for each sample were then randomly divided into a training dataset (70%) and a validation dataset (30%). The training dataset was used to train the CNN to differentiate IBV from negative signals, using ∼3000 BBXs per sample.
                             </p>
                             <p id="p-33">
                              The trained network was validated using the remaining 30% of the data (that the network had never seen before). The first data point in the network validation session was at 50% accuracy (as expected for a completely random classification of objects into two categories), followed by an initial rapid increase in validation accuracy as the network detected the most obvious parameters, followed by a slower increase as the number of iterations increased (Sup.Fig.4A). This was accompanied by a similar decrease in the Loss Function (Sup.Fig.4B). The entire training and validation process took 12 minutes to complete (Sup.Fig.4C).
                             </p>
                             <p id="p-34">
                              Results of the network validation are shown as a confusion matrix, commonly used to visualize performance measures for classification problems (
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Fig.3A
                              </a>
                              ). The rows correspond to the predicted class (Output Class), the columns to the true class (Target Class), and the far-right, bottom cell represents the overall validation accuracy (hereafter, accuracy) of the model for each classified particle. For IBV vs. negative, the trained network was able to differentiate positive and negative samples with high accuracy (91.4%), sensitivity (91.9%) and specificity (90.9%) (
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Fig.3B
                              </a>
                              ). Of note, these probabilities refer to the identification of single virus particles in the sample and not the whole sample; the probability of correctly identifying a sample with hundreds or thousands of virus particles will therefore increase (see later).
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F3.large.jpg?width=800&height=600&carousel=1" title="Network validation results for laboratory-grown virus strains. A) Network validation results shown as a confusion matrix. Rows - predicted class (output class); columns - true class (target class); right column - positive and negative predictive values (percentages of BBXs that are correctly and incorrectly predicted); bottom row - sensitivity and specificity. B) Confusion matrix of CoV (IBV) positive and negative samples. C&D) Confusion matrices of CoV (IBV) vs. influenza Udorn or PR8. E) Confusion matrix of influenza PR8 vs. influenza WSN. F) Confusion matrix of CoV (IBV) vs. a pooled dataset of the virus-negative control and three influenza A strains. G) A trained network is robust over significant time. The network was trained on data from images of the virus IBV and allantoic fluid as a negative control. Each data point (orange for sensitivity; green for specificity) corresponds to the classification result for signals detected at different dates over a period of 135 days. Error bars represent standard deviation. H) Defining the limit of detection for accurate machine learning classification. Increasing concentrations of IBV were labelled and imaged, the resulting images were fed into the trained network. The number of normalised positive particles (positive particles/all particles) increased linearly with increasing virus concentration. Error bars represent standard deviation. The limit of detection (LOD) was defined as 6×104 PFU/mL, with 99.85% certainty." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-774246475" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Network validation results for laboratory-grown virus strains.</span> A) Network validation results shown as a confusion matrix. Rows - predicted class (output class); columns - true class (target class); right column - positive and negative predictive values (percentages of BBXs that are correctly and incorrectly predicted); bottom row - sensitivity and specificity. B) Confusion matrix of CoV (IBV) positive and negative samples. C&D) Confusion matrices of CoV (IBV) vs. influenza Udorn or PR8. E) Confusion matrix of influenza PR8 vs. influenza WSN. F) Confusion matrix of CoV (IBV) vs. a pooled dataset of the virus-negative control and three influenza A strains. G) A trained network is robust over significant time. The network was trained on data from images of the virus IBV and allantoic fluid as a negative control. Each data point (orange for sensitivity; green for specificity) corresponds to the classification result for signals detected at different dates over a period of 135 days. Error bars represent standard deviation. H) Defining the limit of detection for accurate machine learning classification. Increasing concentrations of IBV were labelled and imaged, the resulting images were fed into the trained network. The number of normalised positive particles (positive particles/all particles) increased linearly with increasing virus concentration. Error bars represent standard deviation. The limit of detection (LOD) was defined as 6×10<sup>4</sup> PFU/mL, with 99.85% certainty.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F3.medium.gif" width="420" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F3.medium.gif" width="420" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3.
                               </span>
                               <span class="caption-title">
                                Network validation results for laboratory-grown virus strains.
                               </span>
                               <p id="p-35" class="first-child">
                                A) Network validation results shown as a confusion matrix. Rows - predicted class (output class); columns - true class (target class); right column - positive and negative predictive values (percentages of BBXs that are correctly and incorrectly predicted); bottom row - sensitivity and specificity.
                               </p>
                               <p id="p-36">
                                B) Confusion matrix of CoV (IBV) positive and negative samples.
                               </p>
                               <p id="p-37">
                                C&D) Confusion matrices of CoV (IBV) vs. influenza Udorn or PR8.
                               </p>
                               <p id="p-38">
                                E) Confusion matrix of influenza PR8 vs. influenza WSN.
                               </p>
                               <p id="p-39">
                                F) Confusion matrix of CoV (IBV) vs. a pooled dataset of the virus-negative control and three influenza A strains.
                               </p>
                               <p id="p-40">
                                G) A trained network is robust over significant time. The network was trained on data from images of the virus IBV and allantoic fluid as a negative control. Each data point (orange for sensitivity; green for specificity) corresponds to the classification result for signals detected at different dates over a period of 135 days. Error bars represent standard deviation.
                               </p>
                               <p id="p-41">
                                H) Defining the limit of detection for accurate machine learning classification. Increasing concentrations of IBV were labelled and imaged, the resulting images were fed into the trained network. The number of normalised positive particles (positive particles/all particles) increased linearly with increasing virus concentration. Error bars represent standard deviation. The limit of detection (LOD) was defined as 6×10
                                <sup>
                                 4
                                </sup>
                                PFU/mL, with 99.85% certainty.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Efficient classification of different virus strains across optical systems using deep learning
                             </h3>
                             <p id="p-42">
                              Next, we tested the network’s ability to distinguish between different virus types and strains by training the network on BBXs obtained (as described above) from images of IBV and influenza Udorn, X31, PR8 and H1N1 A/WSN/33 (WSN) strains. The network easily distinguished between the coronavirus and influenza, with a validation accuracy of 95.5% for IBV vs. Udorn (
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Fig.3C
                              </a>
                              ) and 94.3% for IBV vs. PR8 (
                              <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                               Fig.3D
                              </a>
                              ). The network was also able to differentiate between closely related strains of influenza (WSN vs. PR8), albeit with a slightly lower accuracy of 79.6% (
                              <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                               Fig.3E
                              </a>
                              ), perhaps reflecting the greater homogeneity between H1N1 strains of the same virus. The ability to distinguish between different influenza viruses that were grown in the same cell line (i.e. WSN and PR8 were both grown in MDCK cells) established that our classification is not host-cell dependent. The network was also able to distinguish between IBV and a pooled dataset consisting of the virus-negative control and three influenza strains (92.2%) (
                              <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                               Fig.3F
                              </a>
                              ), and importantly, was able to distinguish three viruses from each other in a multi-classifier experiment (the coronavirus IBV and two influenza strains, PR8 and WSN; 81.9%) (Sup.Fig.5).
                             </p>
                             <p id="p-43">
                              To demonstrate the general applicability of our approach, we performed similar experiments using a second optical system (a Zeiss Elyra 7 with a 63x objective rather than an ONI Nanoimager with a 100x objective). We compared two strains of influenza (WSN and Udorn) with negative samples lacking virus, and with each other; and found that we were able to distinguish the samples with accuracies ranging from ∼74-78% (Sup.Fig.6), thus establishing that virus classification is independent of the imaging conditions (e.g. exposure, illumination) and microscope type (e.g. magnification, numerical aperture). We also showed that a neural network returned robust results over significant time without requiring re-training, with no decrease in sensitivity or specificity over a period of 135 days (
                              <a id="xref-fig-3-7" class="xref-fig" href="#F3">
                               Fig.3G
                              </a>
                              ).
                             </p>
                             <p id="p-44">
                              We estimated the limit of detection (LOD) of our assay by testing the ability of the network to accurately detect decreasing IBV, WSN and SARS-CoV-2 concentrations (
                              <a id="xref-fig-3-8" class="xref-fig" href="#F3">
                               Fig.3H
                              </a>
                              and Sup.Fig.7). Training was performed on laboratory propagated virus samples of known titre, followed by normalisation of the number of BBXs in each class by the total number of BBXs in the sample (to counter variations between samples). Images were analysed by the trained network, and the number of particles classified as positive was fitted with increasing virus concentration, giving estimated LODs of 6×10
                              <sup>
                               4
                              </sup>
                              , 4.6×10
                              <sup>
                               4
                              </sup>
                              and 5.4×10
                              <sup>
                               4
                              </sup>
                              PFU/mL for the three virus strains tested. This sensitivity, as expected, was lower than that of amplification-based methods like RT-PCR (∼10
                              <sup>
                               2
                              </sup>
                              PFU/mL (
                              <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                              )), however is still within a clinically useful range; SARS-CoV-2 viral loads have been demonstrated to be between 10
                              <sup>
                               4
                              </sup>
                              to 10
                              <sup>
                               7
                              </sup>
                              copies per mL in throat swab and sputum samples (
                              <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Classification of clinical samples with high accuracy
                             </h3>
                             <p id="p-45">
                              Having demonstrated our assay on laboratory-grown viruses, we next assessed clinical samples (workflow in
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Fig.4A
                              </a>
                              ). Throat swabs from 33 patients negative for virus (as determined by RT-PCR), or positive for SARS-CoV-2, seasonal hCoVs (OC43, HKU1 or NL63) or human influenza A (as determined by RT-PCR) were inactivated with formaldehyde before being labelled and immobilised (see Methods). Robust and reproducible labelling of viruses in clinical samples was achieved (Sup.Fig.8). Images of the samples captured over three different days were used to train and validate the network to answer a variety of paired questions (e.g. SARS-CoV-2 vs. negative, or SARS-CoV-2 vs. hCoVs; details of clinical samples used for network training and validation are described in Table 1), and similarly to above, the results of the network validation were depicted as confusion matrices.
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F4.large.jpg?width=800&height=600&carousel=1" title="A deep learning network can differentiate viruses in clinical samples. A) Workflow for training and validation of clinical samples. Samples were collected from 33 patients, labelled and imaged on a microscope over three different days. The images were processed to isolate the individual signals into BBXs. 70% of the BBXs were used to train a convolutional neural network (CNN), resulting in a trained model. The remaining 30% of the BBXs were used to validate the trained model, providing the result in a confusion matrix. B) Confusion matrix showing that a trained network could differentiate positive (Alpha variant) SARS-CoV-2 and negative clinical samples. C) Confusion matrix showing that a trained network could differentiate influenza A (Flu A) positive clinical samples from negative samples. D) Confusion matrix showing that a trained network could differentiate SARS-CoV-2 samples (original Wuhan variant) from seasonal human coronavirus (hCoV) samples." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-774246475" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">A deep learning network can differentiate viruses in clinical samples.</span> A) Workflow for training and validation of clinical samples. Samples were collected from 33 patients, labelled and imaged on a microscope over three different days. The images were processed to isolate the individual signals into BBXs. 70% of the BBXs were used to train a convolutional neural network (CNN), resulting in a trained model. The remaining 30% of the BBXs were used to validate the trained model, providing the result in a confusion matrix. B) Confusion matrix showing that a trained network could differentiate positive (Alpha variant) SARS-CoV-2 and negative clinical samples. C) Confusion matrix showing that a trained network could differentiate influenza A (Flu A) positive clinical samples from negative samples. D) Confusion matrix showing that a trained network could differentiate SARS-CoV-2 samples (original Wuhan variant) from seasonal human coronavirus (hCoV) samples.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F4.medium.gif" width="357" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F4.medium.gif" width="357" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4.
                               </span>
                               <span class="caption-title">
                                A deep learning network can differentiate viruses in clinical samples.
                               </span>
                               <p id="p-46" class="first-child">
                                A) Workflow for training and validation of clinical samples. Samples were collected from 33 patients, labelled and imaged on a microscope over three different days. The images were processed to isolate the individual signals into BBXs. 70% of the BBXs were used to train a convolutional neural network (CNN), resulting in a trained model. The remaining 30% of the BBXs were used to validate the trained model, providing the result in a confusion matrix.
                               </p>
                               <p id="p-47">
                                B) Confusion matrix showing that a trained network could differentiate positive (Alpha variant) SARS-CoV-2 and negative clinical samples.
                               </p>
                               <p id="p-48">
                                C) Confusion matrix showing that a trained network could differentiate influenza A (Flu A) positive clinical samples from negative samples.
                               </p>
                               <p id="p-49">
                                D) Confusion matrix showing that a trained network could differentiate SARS-CoV-2 samples (original Wuhan variant) from seasonal human coronavirus (hCoV) samples.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-50">
                              Our initial results with SARS-CoV-2 clinical samples showed a lower validation accuracy than that achieved with laboratory-grown virus strains (∼70% at the BBX level, Sup.Fig.9A). However, the accuracy was substantially improved by performing labelling at a higher pH (pH 8), likely due to the higher isoelectric point (pI; the pH at which the net charge of the particle is zero) of SARS-CoV-2 relative to influenza (pI of ∼9 compared to ∼6) (
                              <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              -
                              <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                              ). As the virions are more negatively charged at higher pH, they are more efficiently labelled using the cationic solution and more efficiently captured by the charged chitosan surface on the glass slide, leading to more efficient SARS-CoV-2 detection and improved detection accuracy. Using the optimized protocol, the trained network was able to distinguish between virus-positive and virus-negative clinical samples with excellent accuracy, distinguishing between SARS-CoV-2-positive and negative BBXs with a validation accuracy of ∼93% (
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Fig.4B
                              </a>
                              ).
                             </p>
                             <p id="p-51">
                              We could also distinguish between Flu A and negative BBXs with a validation accuracy of ∼84% at the BBX level (
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               Fig.4C
                              </a>
                              ), and between seasonal hCoV and negative samples with an accuracy of ∼78% (Sup.Fig.9B). The network could also distinguish SARS-CoV-2 from seasonal hCoVs with a validation accuracy of ∼73% (
                              <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                               Fig.4D
                              </a>
                              ) and SARS-CoV-2 from Flu A with a validation accuracy of ∼70% (Sup.Fig.9C), potentially useful in diagnosing co-circulating infections. Lastly, the network was able to distinguish between negative samples and combined data from two variants of SARS-CoV-2, the original Wuhan strain (SARS-CoV-2) and the Alpha variant, with an accuracy of ∼75% (Sup.Fig.9D), and between the two variants with an accuracy of∼70% (Sup.Fig.9E).
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Testing of trained networks on independent clinical samples
                             </h3>
                             <p id="p-52">
                              Next, we tested the trained network’s ability to diagnose independent clinical samples never seen before for either network training or validation. A total of 51 samples (from a different set of patients to those used for network training/validation), comprised of negative samples, or samples positive for SARS-CoV-2, Flu A or seasonal hCoVs, were imaged on a fourth day and assessed by the trained networks described in the section above within a few seconds. By comparing the results to RT-PCR and carrying out chi-squared tests where necessary (
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Fig.5A
                              </a>
                              and Sup.Fig.10, Steps 1&2), we showed that 50 out of 51 clinical samples tested were classified correctly, giving an excellent overall sample accuracy of 98.0% (
                              <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                               Fig.5B
                              </a>
                              and Table 2). For the negative samples, 11 out of 11 were classified correctly, giving a perfect sample specificity of 100%, and for the positive samples, 39 out of 40 were classified correctly, giving a very high sample sensitivity of 97.5%. We observed that the single incorrectly classified sample provided a much lower number of BBXs than the other samples (Table 2), suggesting that the viral load in this sample may have been close our limit of detection and thus explaining the misclassification.
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F5.large.jpg?width=800&height=600&carousel=1" title="Independent testing of the trained network with clinical samples. A) Schematic of workflow of independent testing. Previously unseen samples are imaged, the images are processed into BBXs which are fed through a trained network. When necessary, a chi-squared statistical test is performed to test the null hypothesis that the sample is negative. If the p-value is smaller than a pre-set confidence threshold the null hypothesis is rejected and the sample is classified as positive. If the p-value is greater than the threshold the sample is classified as negative. B) Summary of independent testing results using multiple trained models. 51 patient samples that were not used for network training or validation were run through different trained versions of the network, detailed on the bottom of the plot. Some samples were tested in multiple versions of the network, for further details see Table 2. Chi-squared tests were carried out to classify the samples (see Sup.Fig.8 and Table 2; samples with a p-value smaller than a pre-set confidence threshold were classified as positive) and the results compared to RT-PCR. 50 out of 51 samples were classified correctly (incorrect classification shown in black), giving an overall sample accuracy of 98.03%. Results were obtained using two different microscopes (see methods). C) Summary of independent testing results using a single trained network. 104 patient samples were analysed as in B), but tested in a single trained network (SARS-CoV-2 vs. negative). D) Variant classification of clinical samples identified as positive in C). The BBXs from images of the clinical samples classified as positive by the first network were passed through a second trained model (Wuhan + Alpha SARS-CoV-2 vs. Delta SARS-CoV-2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-774246475" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Independent testing of the trained network with clinical samples.</span> A) Schematic of workflow of independent testing. Previously unseen samples are imaged, the images are processed into BBXs which are fed through a trained network. When necessary, a chi-squared statistical test is performed to test the null hypothesis that the sample is negative. If the p-value is smaller than a pre-set confidence threshold the null hypothesis is rejected and the sample is classified as positive. If the p-value is greater than the threshold the sample is classified as negative. B) Summary of independent testing results using multiple trained models. 51 patient samples that were not used for network training or validation were run through different trained versions of the network, detailed on the bottom of the plot. Some samples were tested in multiple versions of the network, for further details see Table 2. Chi-squared tests were carried out to classify the samples (see Sup.Fig.8 and Table 2; samples with a p-value smaller than a pre-set confidence threshold were classified as positive) and the results compared to RT-PCR. 50 out of 51 samples were classified correctly (incorrect classification shown in black), giving an overall sample accuracy of 98.03%. Results were obtained using two different microscopes (see methods). C) Summary of independent testing results using a single trained network. 104 patient samples were analysed as in B), but tested in a single trained network (SARS-CoV-2 vs. negative). D) Variant classification of clinical samples identified as positive in C). The BBXs from images of the clinical samples classified as positive by the first network were passed through a second trained model (Wuhan + Alpha SARS-CoV-2 vs. Delta SARS-CoV-2).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F5.medium.gif" width="299" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F5.medium.gif" width="299" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/08/16/2020.10.13.20212035/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5.
                               </span>
                               <span class="caption-title">
                                Independent testing of the trained network with clinical samples.
                               </span>
                               <p id="p-53" class="first-child">
                                A) Schematic of workflow of independent testing. Previously unseen samples are imaged, the images are processed into BBXs which are fed through a trained network. When necessary, a chi-squared statistical test is performed to test the null hypothesis that the sample is negative. If the p-value is smaller than a pre-set confidence threshold the null hypothesis is rejected and the sample is classified as positive. If the p-value is greater than the threshold the sample is classified as negative.
                               </p>
                               <p id="p-54">
                                B) Summary of independent testing results using multiple trained models. 51 patient samples that were not used for network training or validation were run through different trained versions of the network, detailed on the bottom of the plot. Some samples were tested in multiple versions of the network, for further details see Table 2. Chi-squared tests were carried out to classify the samples (see Sup.Fig.8 and Table 2; samples with a p-value smaller than a pre-set confidence threshold were classified as positive) and the results compared to RT-PCR. 50 out of 51 samples were classified correctly (incorrect classification shown in black), giving an overall sample accuracy of 98.03%. Results were obtained using two different microscopes (see methods).
                               </p>
                               <p id="p-55">
                                C) Summary of independent testing results using a single trained network. 104 patient samples were analysed as in B), but tested in a single trained network (SARS-CoV-2 vs. negative).
                               </p>
                               <p id="p-56">
                                D) Variant classification of clinical samples identified as positive in C). The BBXs from images of the clinical samples classified as positive by the first network were passed through a second trained model (Wuhan + Alpha SARS-CoV-2 vs. Delta SARS-CoV-2).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-57">
                              Within this analysis, we also tested whether we could differentiate different SARS-CoV-2 variants. The BBXs from images of seven clinical samples that had been classified as positive by a SARS-CoV-2 vs. negative network were passed through a second trained network to test whether it was the original Wuhan variant or the Alpha variant that first arose in the UK in 2020 (Sup.Fig.10, Steps 3&4). All seven variant samples were classified correctly (
                              <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                               Fig.5B
                              </a>
                              ).
                             </p>
                             <p id="p-58">
                              Given the potential clinical relevance of a test that can diagnose different SARS-CoV-2 variants without the need for sequencing, we decided to explore this further using an additional 104 clinical samples, 68 of which were determined to be virus-negative and 36 of which were determined to be SARS-CoV-2 positive by RT-PCR. The samples were taken from patients from November 2020 to July 2021, and of the positive samples, 14 were the original Wuhan variant, 12 were the Alpha variant (as suggested by a spike gene target failure in RT-PCR) and 10 were the Delta variant. Samples were labelled, immobilised and imaged as normal, followed by processing of the images into BBXs. In the first step of the analysis the 104 samples were classified as either SARS-CoV-2-positive or negative (results of the network validation in Sup.Fig.11A). Two of the negative samples were inconclusive due to a low number of BBXs (less than the 5 needed for the chi-squared test). Of the remaining 102 samples, all but three of the samples were classified correctly (
                              <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                               Fig.5C
                              </a>
                              ). Of the three samples that were misclassified one was a negative sample and 2 were positive samples (
                              <a id="xref-fig-5-5" class="xref-fig" href="#F5">
                               Fig.5C
                              </a>
                              ), providing us with an overall sample accuracy of 97.1%, a sensitivity of 94.4% and a specificity of 98.5%.
                             </p>
                             <p id="p-59">
                              For the subsequent variant classification step, the BBXs from images of the clinical samples classified as positive by the first network (35 samples in total) were passed through a second trained model (Wuhan + Alpha SARS-CoV-2 vs. Delta SARS-CoV-2) (Sup.Fig.11B). Two samples gave inconclusive results as the resulting p-values were closer than 3 orders of magnitude (Sup.Fig.10, Steps 3&4). The single negative sample that had been incorrectly classified as positive by the first trained model was classified as Delta, while six remaining positive samples were misclassified (two Delta, two Alpha and 2 Wuhan variant), giving an overall sample accuracy of 77.1%, inconclusive rate of 5.7% and misclassification rate of 17.1% (
                              <a id="xref-fig-5-6" class="xref-fig" href="#F5">
                               Fig.5D
                              </a>
                              and Table 4). Whilst our variant classification accuracy is lower than our overall positive vs. negative test accuracy, we believe that this may still be of use in the context of a rapid variant screening assay in the absence of sequencing facilities.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-8">
                            <h2 class="">
                             DISCUSSION
                            </h2>
                            <p id="p-60">
                             In summary, we have shown a proof of principle for the use of single-particle fluorescence microscopy and deep learning to rapidly detect and classify viruses, including coronaviruses. We have carried out two clinical tests of our method using 155 patient samples in total, which provided high overall sample accuracies of 98.0% and 97.1%. We note that these results were obtained with samples collected over a significant time-period, using a range of different collection kits containing different volumes of viral transport media, and stored at different temperatures for varying periods of time. Given these sampling inconsistencies, which could potentially impact the number of intact virus particles in each sample, the results of these small-scale clinical trials are extremely encouraging, demonstrating the potential of our method as a viable diagnostic test.
                            </p>
                            <p id="p-61">
                             Our initial proof of principle experiments, carried out using viruses grown in cell culture, demonstrated that the CNN can distinguish not only between samples with and without virus, but also between the avian coronavirus IBV and various strains of influenza with high accuracies of >90% per particle (e.g. IBV vs. Udorn, IBV vs. PR8). We confirmed our approach using multiple stocks of cell grown viruses, and different microscopes. A truly independent validation of the trained network using samples that have not been used in either training or validation wasn’t possible using cell grown viruses however, as all stocks are essentially the same virus, grown in the same cell line and similar conditions, even when grown at different times. Importantly, this was possible using clinical samples though, as novel samples could be obtained from new patients that were entirely independent of those used for training/validation. The accurate classification of completely independent clinical samples served as clear proof that our network could use image information to accurately classify samples never seen before.
                            </p>
                            <p id="p-62">
                             We also accounted for any other potential confounding factors such as sample preparation (by only comparing samples prepared and inactivated in the same way), as well as differences in virus concentration and differences in imaging quality, by only classifying the isolated signals from individual viruses and not full fields-of-view. This approach renders the network completely agnostic to virus concentration, signal density or small day-to-day imaging differences such as uneven illumination, and means that virus classification is solely dependent on the fluorescent images of the virus. We have even shown that related H3N2 virus strains, prepared in the same way in the same cells, could be reliably distinguished from each other, establishing that classification is independent of the host cell that the virus samples were grown in. All of this, together with our finding that clinical samples that produced different CT values in RT-PCR were correctly classified independently of their concentration, provides several lines of strong evidence that our network can truly differentiate between different viruses.
                            </p>
                            <p id="p-63">
                             The power of our method comes from the ability to rapidly and universally label enveloped viruses in a sample, and swiftly image them using diffraction limited microscopy. We have shown that even with the limited information present in the low-resolution images, a trained CNN can very effectively differentiate between virus strains. This is based on our findings that different virus families, and even different virus strains, exhibit small differences in their distributions of size, shape and labelling efficiency when labelled using the cation mediated method (
                             <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                              Fig.2
                             </a>
                             and Sup.Fig.3). The labelling is an electrostatic interaction between the phosphate backbone of the DNA and the lipid membrane of the virus (
                             <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ). Different viruses will therefore exhibit differences in labelling efficiency and coverage due to their different isoelectric points (the pH at which a virus has a neutral surface charge), e.g. WSN: 4.7, PR8: 5.3, IBV: 7.2 and SC2: 8.5 (
                             <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             -
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              19
                             </a>
                             ). The surface charge of virus strains is also likely to be affected by mutations in the surface glycoproteins, thus explaining our ability to effectively differentiate variants. We explored the patterns ‘seen’ by two different networks using the DeepDreamImage function in Matlab (
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              20
                             </a>
                             ), which suggests that a network trained to differentiate between negative and positive SARS-CoV-2 samples may see positive signals as rounded shapes with high intensity in the centre of the BBX, whereas the pixels have much lower values in the centre of the BBX for a negative sample, supporting our theory that virus signals are brighter than the signals observed in a negative sample (Sup.Fig.12A&B). Comparison of the patterns observed by a network trained to differentiate between two variants of SARS-CoV-2 are harder to interpret, however there are higher intensity pixels in both than in the negative sample (Sup.Fig.12C&D).
                            </p>
                            <p id="p-64">
                             During our second clinical validation that initially tested for SARS-CoV-2-positive or negative, followed by variant classification, two samples did not provide enough BBXs for us to accurately conclude a result using the chi-squared test. Interestingly, both of the inconclusive samples were negatives (validated by RT-PCR); further testing of more samples may establish that this can happen only for negative samples in which case it may be a useful indicator for a negative diagnosis. The small number of positive samples that were misclassified may have been due to low viral load, which can be improved through further optimisation of the protocol, such as by improving virus immobilisation, sample concentration, or limiting the time the sample is stored for before being imaged. It is possible that the single negative sample that was misclassified may have contained another virus that was not SARS-CoV-2; in further iterations of the test we will use multi-classifier networks to recognise all the major families of circulating respiratory viruses (e.g., Influenza, HCoVs, SARS-CoV-2 and RSV), providing a multi-pathogen testing platform.
                            </p>
                            <p id="p-65">
                             The current gold standard for viral diagnostics is RT-PCR, which requires a time consuming (∼30 mins) RNA extraction step, followed by the main assay (which can take several hours). In this manuscript, we describe a laboratory-based proof-of-principle assay that involves i) instantaneous sample labelling, ii) 10 sec for sample mounting, iii) 40 sec for focusing, iv) 2 min for image acquisition (81 FOVs) and v) 20 sec for analysis, thus easily providing a result within just 5 minutes. Our assay requires no RNA extraction but in order to easily work with clinical samples in a containment level 2 laboratory, we initially inactivated samples with a low concentration of formaldehyde (4%) for 30 minutes prior to sample preparation, which has been shown to maintain virus particle shape while rendering them non-infectious (
                             <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                              21
                             </a>
                             ). In later experiments presented here, we moved to working with samples inactivated with 1% formaldehyde in just 5 minutes (having used plaque assays to show that this lower concentration and time was still sufficient to fully inactivate samples); rendering the entire test complete from start to finish within just 10 minutes. This makes our test significantly faster than RT-PCR, which may render the test invaluable in point-of-care situations where time is critical. Our envisaged commercial version of the test will not require inactivation at all (making it even faster) and will use a cost-effective and simplified microscope, making the assay accessible to many.
                            </p>
                            <p id="p-66">
                             Our approach also avoids the need for viral lysis or amplification and the associated cost, tedium and supply-chain issues. A key advantage of our test over enzyme-based methods such as RT-PCR or RT-LAMP is that the reagents are very affordable. The most expensive component is the fluorescent DNA, which we typically use at 1 nM concentration in a very small sample volume (20 μL); even in the small volumes that we order DNA, the cost amounts to ∼2.7 pence to run 10,000 labelling reactions, demonstrating the scalability of our test. The non-specific detection of intact viral particles (rather than genome fragments) can report directly on infectivity (although this was not evaluated here), and has the advantages of speed, the ability to detect multiple virus types in a single labelling step, and robustness against potential mutations in the viral genome. Our algorithms are versatile and can be trained to differentiate between many different viruses, independently of how they are labelled, immobilized and imaged.
                            </p>
                            <p id="p-67">
                             Here, we have tested the ability of our models to differentiate between the original genotype of SARS-CoV-2, as well as two variants of concern, Alpha and Delta. The ease with which the network can be retrained suggests that it can be rapidly repurposed to detect new and emerging SARS-CoV-2 variants such as Omicron. Given its simplicity and rapid nature, our technology could also be used outside of specialized laboratories, such as in airports, workplaces and care homes. These unique capabilities should enable extremely rapid, mobile, and real-time analysis of patient and community samples during pandemic situations.
                            </p>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             METHODS
                            </h2>
                            <div id="sec-10" class="subsection">
                             <h3>
                              Laboratory grown virus strains and DNAs
                             </h3>
                             <p id="p-68">
                              The influenza strains (H1N1 A/Puerto Rico/8/1934 (PR8), H3N2 A/Udorn/72 (Udorn), H1N1 A/WSN/33 (WSN) and H3N2 A/Aichi/68 (X31)) used in this study have been described previously (
                              <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ). Briefly, WSN, PR8 and Udorn were grown in Madin-Darby bovine kidney (MDBK) or Madin-Darby canine kidney (MDCK) cells and X31 was grown in embryonated chicken eggs. The cell culture supernatant or allantoic fluid was collected and the viruses were titred by plaque assay. Titres of PR8, Udorn, WSN and X31 were 1×10
                              <sup>
                               8
                              </sup>
                              plaque forming units (PFU)/mL, 1×10
                              <sup>
                               7
                              </sup>
                              PFU/ mL, 2×10
                              <sup>
                               6
                              </sup>
                              PFU/mL and 4.5×10
                              <sup>
                               8
                              </sup>
                              PFU/mL respectively. The coronavirus IBV (Beau-R strain) (
                              <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                              ) was grown in embryonated chicken eggs and titred by plaque assay (1×10
                              <sup>
                               6
                              </sup>
                              PFU/mL). Influenza and IBV were inactivated by the addition of 2% formaldehyde before use. SARS-CoV-2 was grown in Vero E6 cells and titred by plaque assay (1.05×10
                              <sup>
                               6
                              </sup>
                              PFU/mL). The virus was inactivated by addition of 4% formaldehyde before use.
                             </p>
                             <p id="p-69">
                              Single-stranded oligonucleotides labelled with either red or green dyes were purchased from IBA (Germany). We have previously characterised a range of DNAs of different length and sequence and shown that as long as the DNA is longer than 20 bases robust labelling occurs, regardless of sequence(
                              <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ). The ‘red’ DNA used in this manuscript was modified at the 5’ end with ATTO647N (5’ ACAGCACCACAGACCACCCGCGGATGCCGGTCCCTACGCGTCGCTGTCACGCTGGCTGTTTGTCTTCCTGCC 3’) and the ‘green’ DNA was modified at the 3’ end with Cy3 (5’ GGGTTTGGGTTGGGTTGGGTTTTTGGGTTTGGGTTGGGTTGGGAAAAA 3’). The DNA used for super-resolution imaging was modified at the 5’ end with Alexa647 (5’ TCCGCTCTCACAATTCCACACATTATACGAGCCGAAGCATAAAGTGTCAAGCCT 3’).
                             </p>
                            </div>
                            <div id="sec-11" class="subsection">
                             <h3>
                              Clinical samples
                             </h3>
                             <p id="p-70">
                              Ethical approval was obtained for the use of anonymised oro- or nasopharyngeal specimens from patients for the diagnosis of influenza and other respiratory pathogens, including SARS-CoV-2 (North West-Greater Manchester South Research Ethics Committee [REC], REC Ref:19/NW/0730). Specimens were maintained in Copan Universal Transport Medium (UTM) before being inactivated in 4% final concentration of formaldehyde (Pierce) for 30 minutes at room temperature, or 1% formaldehyde for 5 minutes at room temperature for the 104 samples used in the second clinical trial (
                              <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">
                               21
                              </a>
                              ). Samples were confirmed as SARS-CoV-2-positive or negative using either the Public Health England 2019-nCoV real-time RT-PCR RdRp gene assay or RealStar SARS-CoV-2 RT-PCR Kit (Altona diagnostics). Testing for other respiratory pathogens and sub-typing of seasonal human coronavirus (hCoV) samples as OC43, HKU1 or NL63 strains was conducted using the BioFire FilmArray Respiratory Panel (Biomerieux, Marcy-L’Etoile, France) and Cepheid Xpert Xpress Flu/RSV (Cepheid, Sunnyvale, CA, USA).
                             </p>
                             <p id="p-71">
                              We used 213 clinical samples in total, taken from patients from November 2020 to July 2021. In order to train the network, we imaged samples from different patients over three days (different sample prep from the same patient samples on each day). We used 70% of the BBXs isolated from all the images taken over the three days to train the network, leaving the remaining 30% of the BBXs for network validation, the results of which are shown in the confusion matrices. Each confusion matrix corresponds to an individually trained model. In total, 58 clinical samples were used for training and validation of the network.
                             </p>
                             <p id="p-72">
                              We then carried out two independent tests of the trained networks using clinical samples not used for either training or validation. The first test used 51 patient samples comprised of negative samples, or samples positive for SARS-CoV-2, Flu A or seasonal hCoVs. The second test used 104 patient samples comprised of negative samples, or samples positive for SARS-CoV-2. Of the positives, 14 were the original Wuhan variant, 12 were the Alpha variant (indicated by a spike gene target failure in RT-PCR [TaqPath Covi-19 combo kit, ThermoFisher]) (
                              <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ) and 10 were the Delta variant (confirmed through RT-PCR [Taqman SARS-CoV-2 mutation panel [ThermoFisher]).
                             </p>
                            </div>
                            <div id="sec-12" class="subsection">
                             <h3>
                              Sample preparation
                             </h3>
                             <p id="p-73">
                              Both positive and negative samples prepared in the same way (e.g. inactivated in the same concentration of formaldehyde or labelled in the same buffer), and only samples similarly prepared were compared with each other. Glass slides were treated with 0.015 mg/mL chitosan (a linear polysaccharide) in 0.1 M acetic acid for 30 min before being washed thrice with MilliQ water, or with 0.01% poly-L-lysine (Sigma) for 15 min (
                              <a id="xref-fig-3-9" class="xref-fig" href="#F3">
                               Fig.3A
                              </a>
                              , Sup.Fig.6, Sup.Fig.10C). Unless otherwise stated, virus stocks (typically 10 µL) were diluted in 0.23 M CaCl
                              <sub>
                               2
                              </sub>
                              or SrCl
                              <sub>
                               2
                              </sub>
                              (as described in the figure legends) and 1 nM of each fluorescently-labelled DNA in a final volume of 20 µL, before being added to the slide surface. For SARS-CoV-2 imaging, the cationic labelling solution was buffered with 20 mM Tris, pH 8. Virus labelling with CaCl
                              <sub>
                               2
                              </sub>
                              has been described previously (
                              <a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ); SrCl
                              <sub>
                               2
                              </sub>
                              provides similar results (
                              <a id="xref-fig-1-15" class="xref-fig" href="#F1">
                               Fig.1
                              </a>
                              ). For laboratory grown virus stocks, negatives were taken using virus-free Minimal Essential Media (Gibco) or allantoic fluid from uninfected eggs in place of the virus.
                             </p>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              Imaging
                             </h3>
                             <p id="p-74">
                              Images were captured using three wide-field Nanoimager microscopes (
                              <a id="xref-ref-9-7" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ). ‘Microscope 1’ was equipped with a Hamamatsu Flash 4 LT.1 sCMOS camera and ‘Microscopes 2 and 3’ were equipped with a Hamamatsu Flash4 V3 sCMOS camera; in all other respects, the systems were identical. The sample was imaged using total internal reflection fluorescence (TIRF) microscopy and a 100x oil-immersion objective. The laser illumination was focused at a typical angle of 53° with respect to the normal. Movies of 5 frames per field of view (FOV) (measuring 75 × 49 µm) were taken at a frequency of 33 Hz and exposure time of 30 ms, with laser intensities kept constant at 0.78 kW/cm
                              <sup>
                               2
                              </sup>
                              for the red (640 nm) and 1.09 kW/cm
                              <sup>
                               2
                              </sup>
                              for the green (532 nm) laser. To automate the task and ensure no bias in the selection of FOVs, the whole sample was scanned using the multiple acquisition capability of the microscope; 81 FOVs were imaged in 2 minutes. Defocusing was carefully controlled using an automated autofocus mechanism to bring the sample to a pre-defined axial position before each field of view was exposed to the excitation lasers. This was achieved by imaging the reflection of a near-IR laser off the glass/sample medium interface and matching the image to a pre-recorded reference image.
                             </p>
                             <p id="p-75">
                              Data in Sup. Fig. 6 was acquired using a Zeiss Elyra 7 microscope equipped with two pco.edge sCMOS (version 4.2 CL HS) cameras. TIRF images were acquired using the alpha Plan-Apochromat 63x/1.46 oil objective. A laser intensity of 10% for the 641nm laser was used for imaging Atto647N. Laser intensities of 6% for the 561nm and 3% for the 488nm laser were used for imaging Cy3. The exposure time was 50ms.
                             </p>
                            </div>
                            <div id="sec-14" class="subsection">
                             <h3>
                              Super resolution
                             </h3>
                             <p id="p-76">
                              For super resolution imaging, passivated microscope slides were prepared by washing in acetone and Vectabond solution (Vector Laboratories) before being incubated with NHS-PEG:Biotin-NHS-PEG in an 80:1 ratio. 0.5 mg/mL neutravidin was incubated for 10 minutes at room temperature on the slide shortly before virus was added. Viruses were biotinylated by incubation in a 1 mg/mL Sulfo-NHS-LC-Biotin (ThermoFisher) for 3 hours at 37°C before being labelled with 0.23 M CaCl
                              <sub>
                               2
                              </sub>
                              and 1 nM Alexa647-labelled DNA in a final volume of 20 µL, before being added to the slide surface. After incubation for 30 minutes at room temperature the slide was washed thrice in 1x PBS before imaging in 50 mM MEA and an enzymatic oxygen scavenging system consisting of 1 mg/mL glucose oxidase, 40 μg/mL catalase, and 1.0% (wt/vol) glucose. Super-resolution localisations were extracted using the inbuilt Nanoimager software and analysed further in Matlab. Localisations were clustered with DBScan using a minimum cluster size of 50 and an epsilon of 30nm, followed by computing the convex hull to find the area of the clustered points.
                             </p>
                            </div>
                            <div id="sec-15" class="subsection">
                             <h3>
                              Data Segmentation
                             </h3>
                             <p id="p-77">
                              Each FOV in the red channel was turned into a binary image using MATLAB’s built-in imbinarize function with adaptive filtering sensitivity set to 0.5. Adaptive filtering uses statistics about the neighbourhood of each pixel it operates on to determine whether the pixel is foreground or background. The filter sensitivity is a variable which, when increased, makes it easier to pass the foreground threshold. The bwpropfilt function was used to exclude objects with an area outside the range 10-100 pixels (1 pixel = 117nm), aiming to disregard free ssDNA and aggregates. We imaged single fluorophores and found that that they did not exit 10 pixels in area, giving us a lower limit, and arbitrarily chose 100 pixels as the upper limit to exclude very large aggregates or cellular debris. The regionprops function was employed to extract properties of each found object: area, semi-major to semi-minor axis ratio (or simply, axis ratio), coordinates of the object’s centre, bounding box (BBX) encasing the object, and maximum pixel intensity within the BBX.
                             </p>
                             <p id="p-78">
                              Accompanying each FOV is a location image (LI) summarising the locations of signals received from each channel (red and green); colocalised signals in the LI image were shown in yellow. Objects found in the red FOV were compared with their corresponding signal in the associated LI. Objects that did not arise from colocalised signals were rejected. The qualifying BBXs were then drawn onto the raw FOV and images of the encased individual viruses were saved.
                             </p>
                            </div>
                            <div id="sec-16" class="subsection">
                             <h3>
                              Machine Learning
                             </h3>
                             <p id="p-79">
                              The CNN used only the red channel as input, as analysis using both channels was not found to improve the overall accuracy. No normalisation of the images was carried out, however the bounding boxes (BBXs) from the data segmentation had variable sizes (never larger than 17 pixels in any direction due to the size filtering). Thus, all the BBX were resized such that they had a final size of 17×17 pixels by means of padding (adding extra pixels with 0 grey-value until they reach the required size).
                             </p>
                             <p id="p-80">
                              The resized images were used as the input for the 15-layer CNN. The network was built using Matlab 2020b and trained using the computer’s GPU (specifications: NVIDIA 2080Ti, 32 GB RAM, i7 processor). The network had 3 convolutional layers in total, with kernels of 2×2 for the first two convolutions and 3×3 for the last one. The learning rate was set to 0.01 and the learning schedule rate remained constant throughout the training. The hyperparameters remained the same throughout the training process for all models; the mini batch size was set to 1000, the maximum number of epochs to 100 and the validation frequency to 20 iterations.
                             </p>
                             <p id="p-81">
                              In the classification layer, trainNetwork took the values from the softmax function and assigned each input to one of K mutually exclusive classes using the cross entropy function for a 1-of-K coding scheme (
                              <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                               24
                              </a>
                              ),
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/08/16/2020.10.13.20212035/embed/graphic-6.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/08/16/2020.10.13.20212035/embed/graphic-6.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where N is the number of samples, K is the number of classes, t
                              <sub>
                               ij
                              </sub>
                              is the indicator that the i
                              <sup>
                               th
                              </sup>
                              sample belongs to the j
                              <sup>
                               th
                              </sup>
                              class, and y
                              <sub>
                               ij
                              </sub>
                              is the output for sample i for class j, which in this case, is the value from the softmax function. That is, it is the probability that the network associates the i
                              <sup>
                               th
                              </sup>
                              input with class j (
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                              ). A stochastic gradient descent with momentum set to 0.9 was used as the optimizer.
                             </p>
                            </div>
                            <div id="sec-17" class="subsection">
                             <h3>
                              Statistical Analysis
                             </h3>
                             <div id="sec-18" class="subsection">
                              <h4>
                               Confusion matrices
                              </h4>
                              <p id="p-82">
                               The results of each network validation are shown as a confusion matrix, which make used of the following terms:
                              </p>
                              <ul class="list-unord" id="list-1">
                               <li id="list-item-1">
                                <p id="p-83">
                                 True positive (TP): BBXs correctly identified as positive,
                                </p>
                               </li>
                               <li id="list-item-2">
                                <p id="p-84">
                                 False Positive (FP): BBXs incorrectly identified as positive,
                                </p>
                               </li>
                               <li id="list-item-3">
                                <p id="p-85">
                                 True negative (TN): BBXs correctly identified as negative, and
                                </p>
                               </li>
                               <li id="list-item-4">
                                <p id="p-86">
                                 False negative (FN): BBXs incorrectly identified as negative.
                                </p>
                               </li>
                              </ul>
                              <p id="p-87">
                               Sensitivity refers to the ability of the test to correctly identify positive BBXs. It can be calculated by dividing the number of true positives over the total number of positives(
                               <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                                26
                               </a>
                               ).
                               <span class="disp-formula" id="disp-formula-2">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/08/16/2020.10.13.20212035/embed/graphic-7.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/08/16/2020.10.13.20212035/embed/graphic-7.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Specificity refers to the ability of the test to correctly identify negative BBXs. It can be calculated by dividing the number of true negatives over the total number of negatives(
                               <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">
                                26
                               </a>
                               ).
                               <span class="disp-formula" id="disp-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/08/16/2020.10.13.20212035/embed/graphic-8.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/08/16/2020.10.13.20212035/embed/graphic-8.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               The percentages of BBXs that are correctly and incorrectly predicted by the trained model are known as the positive predictive value (PPV) and negative predictive value (NPV) respectively.
                               <span class="disp-formula" id="disp-formula-4">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/08/16/2020.10.13.20212035/embed/graphic-9.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/08/16/2020.10.13.20212035/embed/graphic-9.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-5">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/08/16/2020.10.13.20212035/embed/graphic-10.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/08/16/2020.10.13.20212035/embed/graphic-10.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               The overall balanced validation accuracy of the model is given by:
                               <span class="disp-formula" id="disp-formula-6">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/08/16/2020.10.13.20212035/embed/graphic-11.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/08/16/2020.10.13.20212035/embed/graphic-11.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <div id="sec-19" class="subsection">
                              <h4>
                               Limit of Detection
                              </h4>
                              <p id="p-88">
                               In order to calculate the limit of detection (LOD), increasing concentrations of the CoV IBV (dilutions in allantoic fluid) were labelled and imaged, the resulting images were pre-processed, and the individual signals were fed into the trained network. The normalised average of TP (TP/TP+FP) and standard deviation (STD) were calculated and plotted against the corresponding concentrations as a scatter plot. The plot was fitted as a linear regression, as given by:
                               <span class="disp-formula" id="disp-formula-7">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/08/16/2020.10.13.20212035/embed/graphic-12.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/08/16/2020.10.13.20212035/embed/graphic-12.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Where the virus concentration was treated as the independent variable and a represents the LOD. For the final value of the LOD a+(3STD)=6*10^4 PFU/mL was used, which corresponds to a 99.85% confidence interval assuming a normal distribution. Experiments with the influenza strain AWSN/33 were carried out in a similar way.
                              </p>
                              <p id="p-89">
                               In order to calculate the LOD of SARS-CoV-2, experiments were carried out in a similar way, however the plot was fitted as a sigmoid, as given by:
                               <span class="disp-formula" id="disp-formula-8">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/08/16/2020.10.13.20212035/embed/graphic-13.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/08/16/2020.10.13.20212035/embed/graphic-13.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Where L is the maximum normalised positive value (considered as the sensitivity of the model), b adds bias to the output and changes its range from [0,L] to [b,L+b], k scales the input, and X
                               <sub>
                                0
                               </sub>
                               is the point at which the sigmoid should output the value L/2.
                              </p>
                             </div>
                             <div id="sec-20" class="subsection">
                              <h4>
                               Chi-Squared Test
                              </h4>
                              <p id="p-90">
                               In order to go from single BBX classification to calling the result of a clinical sample as a whole the Chi-squared test was used, which takes into consideration the total number of bounding boxes, the number of BBXs that were classified as positive or negative, and the specificity of the trained model (i.e. the probability of classifying a negative signal as such). By taking into account the specificity of each trained model and the total number of signals in a sample we account for the variability in the number of detected signals between samples. The test also considers that statistically a number of the bounding boxes will be misclassified. The Chi-squared test is a statistical hypothesis test that assumes (the null hypothesis) that the observed frequencies for a categorical variable match the expected frequencies for the categorical variable and can be calculated from the equation below:
                               <span class="disp-formula" id="disp-formula-9">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/08/16/2020.10.13.20212035/embed/graphic-14.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/08/16/2020.10.13.20212035/embed/graphic-14.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where χ
                               <sup>
                                2
                               </sup>
                               = chi squared, O
                               <sub>
                                i
                               </sub>
                               =observed value, E
                               <sub>
                                i
                               </sub>
                               =expected value and n is the number of labels. The threshold p-value for a test can vary depending on the trained model but in general is smaller than p-value=0.01 which corresponds to a confidence of greater than 99%. In this manuscript the Null hypothesis is that the sample is negative and it is only rejected when the p-value is below the threshold in which case the sample is classified as positive.
                              </p>
                             </div>
                            </div>
                           </div>
                           <div class="section data-availability" id="sec-22">
                            <h2 class="">
                             Data Availability
                            </h2>
                            <p id="p-91">
                             Data available upon request.
                            </p>
                           </div>
                           <div class="section" id="sec-23">
                            <h2 class="">
                             Author Contributions
                            </h2>
                            <p id="p-93">
                             N.S., A.T., A.M., C.H., A.N.K. and N.C.R designed and carried out experiments, analysed data and interpreted results. N.S. wrote analysis software. E.B. provided reagents. C.F. and D.M. validated the virus inactivation method. L.P., M.A., S.O., A.V., P.C.M., N.S. and D.C. collected, diagnosed and typed clinical samples that were considered to be appropriate for the purposes of this study. N.C.R. wrote the manuscript. All authors reviewed and approved the final manuscript.
                            </p>
                           </div>
                           <div class="section" id="sec-24">
                            <h2 class="">
                             Competing Interests statement
                            </h2>
                            <p id="p-94">
                             This study was supported by the National Institute for Health Research (NIHR) Health Protection Research Unit in Healthcare Associated Infections and Antimicrobial Resistance (NIHR200915), a partnership between the UK Health Security Agency (UKHSA) and the University of Oxford, and the NIHR Oxford Biomedical Research Centre (BRC). The report presents independent research. The views expressed are those of the author(s) and not necessarily those of the NIHR, UKHSA or the Department of Health and Social Care.
                            </p>
                            <p id="p-95">
                             The work was carried out using a wide-field microscope from Oxford Nanoimaging, a company in which A.N.K. is a co-founder and shareholder, and is being commercialised by OxDx Ltd., a company in which N.C.R. and N.S. are co-founders. Patent applications relating to the work have been submitted by N.C.R., A.N.K. and N.S. (PCT/GB2019/053073 and PCT/GB2021/050990). The authors declare no other competing interests.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-92">
                             We are grateful to Micron Oxford, funded by Wellcome Strategic Awards (091911 and 107457; PI Ilan Davis), for their loan of their microscope and to Nadia Halidi for her help with the instrument. Special thanks to Dr Rebecca Moore and Prof William James (Sir William Dunn School of Pathology, Oxford, UK) for providing laboratory grown SARS-CoV-2. We thank the microbiology laboratory staff at the John Radcliffe Hospital, Oxford. This research was supported by a Royal Society Dorothy Hodgkin Research Fellowship DKR00620 and Research Grant for Research Fellows RGF\R1\180054 (N.C.R.), the University of Oxford COVID-19 Research Response Fund (N.C.R and A.N.K.), a BBSRC-funded studentship (N.S.), and Wellcome Trust grant 110164/Z/15/Z (A.N.K.). All data needed to evaluate the conclusions in the paper are present in the paper and/or the Supplementary Materials.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-15">
                               New figures added with additional clinical data (Figs 4 and 5), Supplemental files updated.
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.1" data-doi="10.1021/acsnano.0c02624">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Udugama
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Diagnosing COVID-19: The Disease and Tools for Detection
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  ACS Nano
                                 </abbr>
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3822
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3835
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DACS%2BNano%26rft.volume%253D14%26rft.spage%253D3822%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facsnano.0c02624%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1021/acsnano.0c02624&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F08%2F16%2F2020.10.13.20212035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W. E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Huang
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  RT-LAMP for rapid diagnosis of coronavirus SARS-CoV-2
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Microb Biotechnol
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  950
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  961
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMicrob%2BBiotechnol%26rft.volume%253D13%26rft.spage%253D950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.13.20212035v4.3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lu
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Development of a Novel Reverse Transcription Loop-Mediated Isothermal Amplification Method for Rapid Detection of SARS-CoV-2
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Virol Sin
                                 </abbr>
                                 , (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.13.20212035v4.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lamb
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bartolone
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   SN
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Elijah
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chancellor
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   MB
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Rapid Detection of Novel Coronavirus (COVID-19) by Reverse Transcription-Loop-Mediated Isothermal Amplification
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  medRxiv
                                 </abbr>
                                 , (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.13.20212035v4.5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Odiwuor
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xiong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sun
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nyaruaba
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   RO
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wei
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tanner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   NA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Rapid Molecular Detection of SARS-CoV-2 (COVID-19) Virus RNA Using Colorimetric LAMP
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  medRxiv
                                 </abbr>
                                 , (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.13.20212035v4.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   El-Tholoth
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Anis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H. H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bau
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Two stage, nested isothermal amplification in a single tube
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Analyst
                                 </abbr>
                                 <span class="cit-vol">
                                  146
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1311
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1319
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnalyst%26rft.volume%253D146%26rft.spage%253D1311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Rodriguez-Manzano
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Handheld Point-of-Care System for Rapid Detection of SARS-CoV-2 Extracted RNA in under 20 min
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  ACS Cent Sci
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  307
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  317
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DACS%2BCent%2BSci%26rft.volume%253D7%26rft.spage%253D307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.8" data-doi="10.1002/14651858.CD013705.pub2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Dinnes
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Rapid, point-of-care antigen and molecular-based tests for diagnosis of SARS-CoV-2 infection
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cochrane Database Syst Rev
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  CD013705
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCochrane%2BDatabase%2BSyst%2BRev%26rft.volume%253D3%26rft.spage%253DCD013705%26rft_id%253Dinfo%253Adoi%252F10.1002%252F14651858.CD013705.pub2%26rft_id%253Dinfo%253Apmid%252F33760236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/14651858.CD013705.pub2&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=33760236&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F08%2F16%2F2020.10.13.20212035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.9" data-doi="10.1038/s41598-019-52759-5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N. C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Robb
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Rapid functionalisation and detection of viruses via a novel Ca(2+)-mediated virus-DNA interaction
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci Rep
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  16219
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D9%26rft.spage%253D16219%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-019-52759-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41598-019-52759-5&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.10" data-doi="10.1162/NECO_a_00990">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Rawat
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Deep Convolutional Neural Networks for Image Classification: A Comprehensive Review
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neural Comput
                                 </abbr>
                                 <span class="cit-vol">
                                  29
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2352
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2449
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeural%2BComput%26rft.volume%253D29%26rft.spage%253D2352%26rft_id%253Dinfo%253Adoi%252F10.1162%252FNECO_a_00990%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1162/NECO_a_00990&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F08%2F16%2F2020.10.13.20212035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.11" data-doi="10.1038/nature14539">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   LeCun
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bengio
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hinton
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Deep learning
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  521
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  436
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  444
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D521%26rft.spage%253D436%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14539%26rft_id%253Dinfo%253Apmid%252F26017442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature14539&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26017442&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F08%2F16%2F2020.10.13.20212035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Laine
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Structured illumination microscopy combined with machine learning enables the high throughput analysis and classification of virus structure
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Elife
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 , (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.13" data-doi="10.1007/s12560-018-9335-7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ito
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sato
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sano
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Utagawa
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kato
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Virus Particle Detection by Convolutional Neural Network in Transmission Electron Microscopy Images
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Food Environ Virol
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  201
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  208
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFood%2BEnviron%2BVirol%26rft.volume%253D10%26rft.spage%253D201%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12560-018-9335-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s12560-018-9335-7&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Arnaout
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  SARS-CoV2 Testing: The Limit of Detection Matters
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 , (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.15" data-doi="10.1016/S1473-3099(20)30113-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L. L. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Poon
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Q.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Viral load of SARS-CoV-2 in clinical samples
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Lancet Infect Dis
                                 </abbr>
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  411
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  412
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DLancet%2BInfect%2BDis%26rft.volume%253D20%26rft.spage%253D411%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1473-3099%252820%252930113-4%26rft_id%253Dinfo%253Apmid%252F32105638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/S1473-3099(20)30113-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=32105638&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F08%2F16%2F2020.10.13.20212035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.16">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Awadasseid
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tanaka
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Initial success in the identification and management of the coronavirus disease 2019 (COVID-19) indicates human-to-human transmission in Wuhan, China
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Int J Biol Sci
                                 </abbr>
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1846
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1860
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DInt%2BJ%2BBiol%2BSci%26rft.volume%253D16%26rft.spage%253D1846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.13.20212035v4.17" data-doi="10.1111/j.1365-2672.2010.04663.x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Michen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Graule
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Isoelectric points of viruses
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Appl Microbiol
                                 </abbr>
                                 <span class="cit-vol">
                                  109
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  388
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  397
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BMicrobiology%2B%2528Print%2529%26rft.stitle%253DJournal%2Bof%2BApplied%2BMicrobiology%2B%2528Print%2529%26rft.aulast%253DMichen%26rft.auinit1%253DB.%26rft.volume%253D109%26rft.issue%253D2%26rft.spage%253D388%26rft.epage%253D397%26rft.atitle%253DIsoelectric%2Bpoints%2Bof%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2672.2010.04663.x%26rft_id%253Dinfo%253Apmid%252F20102425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/j.1365-2672.2010.04663.x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20102425&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F08%2F16%2F2020.10.13.20212035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.18">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Scheller
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Physicochemical properties of SARS-CoV-2 for drug targeting, virus inactivation and attenuation, vaccine formulation and quality control
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Electrophoresis
                                 </abbr>
                                 <span class="cit-vol">
                                  41
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1137
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1151
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DElectrophoresis%26rft.volume%253D41%26rft.spage%253D1137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.19">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E. C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sadasiv
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yeh
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P. W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Protein pI alteration related to strain variation of infectious bronchitis virus, an avian Coronavirus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Virol Methods
                                 </abbr>
                                 <span class="cit-vol">
                                  33
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  115
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  125
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1991
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirological%2Bmethods%26rft.stitle%253DJ%2BVirol%2BMethods%26rft.aulast%253DSadasiv%26rft.auinit1%253DE.%2BC.%26rft.volume%253D33%26rft.issue%253D1-2%26rft.spage%253D115%26rft.epage%253D125%26rft.atitle%253DProtein%2BpI%2Balteration%2Brelated%2Bto%2Bstrain%2Bvariation%2Bof%2Binfectious%2Bbronchitis%2Bvirus%252C%2Ban%2Bavian%2BCoronavirus.%26rft_id%253Dinfo%253Apmid%252F1658026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=1658026&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F08%2F16%2F2020.10.13.20212035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.20">
                               <div class="cit-metadata">
                                <cite>
                                 <abbr class="cit-jnl-abbrev">
                                  DeepDreaming with TensorFlow
                                 </abbr>
                                 . (
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.21" data-doi="10.1038/s41598-021-91371-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lyonnais
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Atomic force microscopy analysis of native infectious and inactivated SARS-CoV-2 virions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci Rep
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  11885
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D11%26rft.spage%253D11885%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-021-91371-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41598-021-91371-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.22" data-doi="10.1128/JVI.75.24.12359-12369.2001">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Casais
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                  <span class="cit-name-surname">
                                   Thiel
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S. G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Siddell
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cavanagh
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Britton
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Reverse genetics system for the avian coronavirus infectious bronchitis virus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Virol
                                 </abbr>
                                 <span class="cit-vol">
                                  75
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  12359
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  12369
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DCasais%26rft.auinit1%253DR.%26rft.volume%253D75%26rft.issue%253D24%26rft.spage%253D12359%26rft.epage%253D12369%26rft.atitle%253DReverse%2BGenetics%2BSystem%2Bfor%2Bthe%2BAvian%2BCoronavirus%2BInfectious%2BBronchitis%2BVirus%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.75.24.12359-12369.2001%26rft_id%253Dinfo%253Apmid%252F11711626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI3NS8yNC8xMjM1OSI7czo0OiJhdG9tIjtzOjUwOiIvbWVkcnhpdi9lYXJseS8yMDIyLzA4LzE2LzIwMjAuMTAuMTMuMjAyMTIwMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bal
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Two-step strategy for the identification of SARS-CoV-2 variant of concern 202012/01 and other variants with spike deletion H69-V70, France, August to December 2020
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Euro Surveill
                                 </abbr>
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 , (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.13.20212035v4.24">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bishop
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-source">
                                  Pattern Recognition and Machine Learning
                                 </span>
                                 . (
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 ,
                                 <span class="cit-publ-loc">
                                  New York, NY
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  738
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               25.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.25">
                               <div class="cit-metadata">
                                <cite>
                                 <abbr class="cit-jnl-abbrev">
                                  Mathworks
                                 </abbr>
                                 . (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               26.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.13.20212035v4.26" data-doi="10.1136/bmj.308.6943.1552">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D. G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Altman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bland
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Diagnostic tests. 1: Sensitivity and specificity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMJ
                                 </abbr>
                                 <span class="cit-vol">
                                  308
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1552
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1994
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMJ%26rft.stitle%253DBMJ%26rft.aulast%253DAltman%26rft.auinit1%253DD%2BG%26rft.volume%253D308%26rft.issue%253D6943%26rft.spage%253D1552%26rft.epage%253D1552%26rft.atitle%253DStatistics%2BNotes%253A%2BDiagnostic%2Btests%2B1%253A%2Bsensitivity%2Band%2Bspecificity%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fbmj.308.6943.1552%26rft_id%253Dinfo%253Apmid%252F8019315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYm1qIjtzOjU6InJlc2lkIjtzOjEzOiIzMDgvNjk0My8xNTUyIjtzOjQ6ImF0b20iO3M6NTA6Ii9tZWRyeGl2L2Vhcmx5LzIwMjIvMDgvMTYvMjAyMC4xMC4xMy4yMDIxMjAzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://medRxiv.disqus.com/?url=https%3A%2F%2Fwww.medrxiv.org%2Fcontent%2F10.1101%2F2020.10.13.20212035v4" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.03.17.21253760v2" title="COVID-19 risk perception and vaccine acceptance in individuals with chronic disease" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.03.30.22273193v2" title="Effectiveness of an Inactivated Covid-19 Vaccine with Homologous and Heterologous Boosters against Omicron in Brazil" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted August 16, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.10.13.20212035v4.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.10.13.20212035v4.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/08/16/2020.10.13.20212035.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.10.13.20212035v4.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about medRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.10.13.20212035v4" class="active" data-icon-position="" data-hide-link-title="0">
                                   Virus detection and identification in minutes using single-particle imaging and deep learning
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from medRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the medRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/549662"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-HzifKKmzsXrSgYQ7SGKA9z9ixfJQAh1bhqZO4WPed0g"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="277155937"/>
                                   <input type="hidden" name="captcha_token" value="13338722404eaef25c82b9b10faca7fb"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node549662--3" data-node-nid="549662" id="node-549662--4984607103" data-pisa="medrxiv;2020.10.13.20212035v4" data-pisa-master="medrxiv;2020.10.13.20212035" data-seqnum="549662" data-apath="/medrxiv/early/2022/08/16/2020.10.13.20212035.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Virus detection and identification in minutes using single-particle imaging and deep learning
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Nicolas
                                  </span>
                                  <span class="nlm-surname">
                                   Shiaelis
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Alexander
                                  </span>
                                  <span class="nlm-surname">
                                   Tometzki
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Leon
                                  </span>
                                  <span class="nlm-surname">
                                   Peto
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Andrew
                                  </span>
                                  <span class="nlm-surname">
                                   McMahon
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Christof
                                  </span>
                                  <span class="nlm-surname">
                                   Hepp
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Erica
                                  </span>
                                  <span class="nlm-surname">
                                   Bickerton
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">
                                   Cyril
                                  </span>
                                  <span class="nlm-surname">
                                   Favard
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="7">
                                  <span class="nlm-given-names">
                                   Delphine
                                  </span>
                                  <span class="nlm-surname">
                                   Muriaux
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="8">
                                  <span class="nlm-given-names">
                                   Monique
                                  </span>
                                  <span class="nlm-surname">
                                   Andersson
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="9">
                                  <span class="nlm-given-names">
                                   Sarah
                                  </span>
                                  <span class="nlm-surname">
                                   Oakley
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="10">
                                  <span class="nlm-given-names">
                                   Alison
                                  </span>
                                  <span class="nlm-surname">
                                   Vaughan
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="11">
                                  <span class="nlm-given-names">
                                   Philippa C.
                                  </span>
                                  <span class="nlm-surname">
                                   Matthews
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="12">
                                  <span class="nlm-given-names">
                                   Nicole
                                  </span>
                                  <span class="nlm-surname">
                                   Stoesser
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="13">
                                  <span class="nlm-given-names">
                                   Derrick
                                  </span>
                                  <span class="nlm-surname">
                                   Crook
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="14">
                                  <span class="nlm-given-names">
                                   Achillefs N.
                                  </span>
                                  <span class="nlm-surname">
                                   Kapanidis
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="15">
                                  <span class="nlm-given-names">
                                   Nicole C.
                                  </span>
                                  <span class="nlm-surname">
                                   Robb
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 medRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.10.13.20212035;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.10.13.20212035
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.medrxiv.org/content/10.1101/2020.10.13.20212035v4" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.medrxiv.org/content/10.1101/2020.10.13.20212035v4&title=Virus%20detection%20and%20identification%20in%20minutes%20using%20single-particle%20imaging%20and%20deep%20learning" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.medrxiv.org/content/10.1101/2020.10.13.20212035v4&title=Virus%20detection%20and%20identification%20in%20minutes%20using%20single-particle%20imaging%20and%20deep%20learning" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.medrxiv.org/content/10.1101/2020.10.13.20212035v4&text=Virus%20detection%20and%20identification%20in%20minutes%20using%20single-particle%20imaging%20and%20deep%20learning" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.medrxiv.org/content/10.1101/2020.10.13.20212035v4&t=Virus%20detection%20and%20identification%20in%20minutes%20using%20single-particle%20imaging%20and%20deep%20learning" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.medrxiv.org/content/10.1101/2020.10.13.20212035v4&title=Virus%20detection%20and%20identification%20in%20minutes%20using%20single-particle%20imaging%20and%20deep%20learning" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.medrxiv.org/content/10.1101/2020.10.13.20212035v4&title=Virus%20detection%20and%20identification%20in%20minutes%20using%20single-particle%20imaging%20and%20deep%20learning&summary=&source=medRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.medrxiv.org/content/10.1101/2020.10.13.20212035v4&title=Virus%20detection%20and%20identification%20in%20minutes%20using%20single-particle%20imaging%20and%20deep%20learning" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node549662--5" data-node-nid="549662" id="citation-node-549662--6175075517" data-pisa="medrxiv;2020.10.13.20212035v4" data-pisa-master="medrxiv;2020.10.13.20212035" data-seqnum="549662" data-apath="/medrxiv/early/2022/08/16/2020.10.13.20212035.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Virus detection and identification in minutes using single-particle imaging and deep learning
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Nicolas
                                      </span>
                                      <span class="nlm-surname">
                                       Shiaelis
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Alexander
                                      </span>
                                      <span class="nlm-surname">
                                       Tometzki
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Leon
                                      </span>
                                      <span class="nlm-surname">
                                       Peto
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Andrew
                                      </span>
                                      <span class="nlm-surname">
                                       McMahon
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Christof
                                      </span>
                                      <span class="nlm-surname">
                                       Hepp
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Erica
                                      </span>
                                      <span class="nlm-surname">
                                       Bickerton
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">
                                       Cyril
                                      </span>
                                      <span class="nlm-surname">
                                       Favard
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="7">
                                      <span class="nlm-given-names">
                                       Delphine
                                      </span>
                                      <span class="nlm-surname">
                                       Muriaux
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="8">
                                      <span class="nlm-given-names">
                                       Monique
                                      </span>
                                      <span class="nlm-surname">
                                       Andersson
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="9">
                                      <span class="nlm-given-names">
                                       Sarah
                                      </span>
                                      <span class="nlm-surname">
                                       Oakley
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="10">
                                      <span class="nlm-given-names">
                                       Alison
                                      </span>
                                      <span class="nlm-surname">
                                       Vaughan
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="11">
                                      <span class="nlm-given-names">
                                       Philippa C.
                                      </span>
                                      <span class="nlm-surname">
                                       Matthews
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="12">
                                      <span class="nlm-given-names">
                                       Nicole
                                      </span>
                                      <span class="nlm-surname">
                                       Stoesser
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="13">
                                      <span class="nlm-given-names">
                                       Derrick
                                      </span>
                                      <span class="nlm-surname">
                                       Crook
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="14">
                                      <span class="nlm-given-names">
                                       Achillefs N.
                                      </span>
                                      <span class="nlm-surname">
                                       Kapanidis
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="15">
                                      <span class="nlm-given-names">
                                       Nicole C.
                                      </span>
                                      <span class="nlm-surname">
                                       Robb
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     medRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.10.13.20212035;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.10.13.20212035
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Virus%20detection%20and%20identification%20in%20minutes%20using%20single-particle%20imaging%20and%20deep%20learning&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=medRxiv&rft.au=Shiaelis%2C+Nicolas&rft.au=Tometzki%2C+Alexander&rft.au=Peto%2C+Leon&rft.au=McMahon%2C+Andrew&rft.au=Hepp%2C+Christof&rft.au=Bickerton%2C+Erica&rft.au=Favard%2C+Cyril&rft.au=Muriaux%2C+Delphine&rft.au=Andersson%2C+Monique&rft.au=Oakley%2C+Sarah&rft.au=Vaughan%2C+Alison&rft.au=Matthews%2C+Philippa+C.&rft.au=Stoesser%2C+Nicole&rft.au=Crook%2C+Derrick&rft.au=Kapanidis%2C+Achillefs+N.&rft.au=Robb%2C+Nicole+C.">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/549662/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/549662/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/549662/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/549662/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/549662/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/549662/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/549662/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/549662/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/549662/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/549662/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/549662/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/549662/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.medrxiv.org/content/10.1101/2020.10.13.20212035v4&count=horizontal&via=&text=Virus%20detection%20and%20identification%20in%20minutes%20using%20single-particle%20imaging%20and%20deep%20learning&counturl=https%3A//www.medrxiv.org/content/10.1101/2020.10.13.20212035v4" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.medrxiv.org/content/10.1101/2020.10.13.20212035v4&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.medrxiv.org/content/10.1101/2020.10.13.20212035v4" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/infectious_diseases" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Infectious Diseases (except HIV/AIDS)
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/addiction-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Addiction Medicine
                          </a>
                          <span class="article-count">
                           (218)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/allergy-and-immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Allergy and Immunology
                          </a>
                          <span class="article-count">
                           (496)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/anesthesia" class="" data-icon-position="" data-hide-link-title="0">
                           Anesthesia
                          </a>
                          <span class="article-count">
                           (106)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cardiovascular-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Cardiovascular Medicine
                          </a>
                          <span class="article-count">
                           (1121)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/dentistry-and-oral-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Dentistry and Oral Medicine
                          </a>
                          <span class="article-count">
                           (198)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/dermatology" class="" data-icon-position="" data-hide-link-title="0">
                           Dermatology
                          </a>
                          <span class="article-count">
                           (141)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/emergency-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Emergency Medicine
                          </a>
                          <span class="article-count">
                           (275)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/endocrinology-including-diabetes-mellitus-and-metabolic-disease" class="" data-icon-position="" data-hide-link-title="0">
                           Endocrinology (including Diabetes Mellitus and Metabolic Disease)
                          </a>
                          <span class="article-count">
                           (510)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (9829)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/forensic-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Forensic Medicine
                          </a>
                          <span class="article-count">
                           (5)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/gastroenterology" class="" data-icon-position="" data-hide-link-title="0">
                           Gastroenterology
                          </a>
                          <span class="article-count">
                           (484)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetic-and-genomic-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Genetic and Genomic Medicine
                          </a>
                          <span class="article-count">
                           (2344)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/geriatric-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Geriatric Medicine
                          </a>
                          <span class="article-count">
                           (225)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-economics" class="" data-icon-position="" data-hide-link-title="0">
                           Health Economics
                          </a>
                          <span class="article-count">
                           (465)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-informatics" class="" data-icon-position="" data-hide-link-title="0">
                           Health Informatics
                          </a>
                          <span class="article-count">
                           (1574)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-policy" class="" data-icon-position="" data-hide-link-title="0">
                           Health Policy
                          </a>
                          <span class="article-count">
                           (738)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-systems-and-quality-improvement" class="" data-icon-position="" data-hide-link-title="0">
                           Health Systems and Quality Improvement
                          </a>
                          <span class="article-count">
                           (609)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/hematology" class="" data-icon-position="" data-hide-link-title="0">
                           Hematology
                          </a>
                          <span class="article-count">
                           (239)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/hiv-aids" class="" data-icon-position="" data-hide-link-title="0">
                           HIV/AIDS
                          </a>
                          <span class="article-count">
                           (510)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/infectious_diseases" class="" data-icon-position="" data-hide-link-title="0">
                           Infectious Diseases (except HIV/AIDS)
                          </a>
                          <span class="article-count">
                           (11686)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/intensive-care-and-critical-care-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Intensive Care and Critical Care Medicine
                          </a>
                          <span class="article-count">
                           (617)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/medical-education" class="" data-icon-position="" data-hide-link-title="0">
                           Medical Education
                          </a>
                          <span class="article-count">
                           (242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/medical-ethics" class="" data-icon-position="" data-hide-link-title="0">
                           Medical Ethics
                          </a>
                          <span class="article-count">
                           (67)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nephrology" class="" data-icon-position="" data-hide-link-title="0">
                           Nephrology
                          </a>
                          <span class="article-count">
                           (259)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neurology" class="" data-icon-position="" data-hide-link-title="0">
                           Neurology
                          </a>
                          <span class="article-count">
                           (2170)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nursing" class="" data-icon-position="" data-hide-link-title="0">
                           Nursing
                          </a>
                          <span class="article-count">
                           (134)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nutrition" class="" data-icon-position="" data-hide-link-title="0">
                           Nutrition
                          </a>
                          <span class="article-count">
                           (340)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/obstetrics-and-gynecology" class="" data-icon-position="" data-hide-link-title="0">
                           Obstetrics and Gynecology
                          </a>
                          <span class="article-count">
                           (427)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/occupational-and-environmental-health" class="" data-icon-position="" data-hide-link-title="0">
                           Occupational and Environmental Health
                          </a>
                          <span class="article-count">
                           (521)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/oncology" class="" data-icon-position="" data-hide-link-title="0">
                           Oncology
                          </a>
                          <span class="article-count">
                           (1188)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ophthalmology" class="" data-icon-position="" data-hide-link-title="0">
                           Ophthalmology
                          </a>
                          <span class="article-count">
                           (368)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/orthopedics" class="" data-icon-position="" data-hide-link-title="0">
                           Orthopedics
                          </a>
                          <span class="article-count">
                           (129)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/otolaryngology" class="" data-icon-position="" data-hide-link-title="0">
                           Otolaryngology
                          </a>
                          <span class="article-count">
                           (221)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pain-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Pain Medicine
                          </a>
                          <span class="article-count">
                           (148)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/palliative-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Palliative Medicine
                          </a>
                          <span class="article-count">
                           (50)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (315)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pediatrics" class="" data-icon-position="" data-hide-link-title="0">
                           Pediatrics
                          </a>
                          <span class="article-count">
                           (702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-therapeutics" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Therapeutics
                          </a>
                          <span class="article-count">
                           (304)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/primary-care-research" class="" data-icon-position="" data-hide-link-title="0">
                           Primary Care Research
                          </a>
                          <span class="article-count">
                           (269)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/psychiatry-and-clinical-psychology" class="" data-icon-position="" data-hide-link-title="0">
                           Psychiatry and Clinical Psychology
                          </a>
                          <span class="article-count">
                           (2201)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/public-and-global-health" class="" data-icon-position="" data-hide-link-title="0">
                           Public and Global Health
                          </a>
                          <span class="article-count">
                           (4705)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/radiology-and-imaging" class="" data-icon-position="" data-hide-link-title="0">
                           Radiology and Imaging
                          </a>
                          <span class="article-count">
                           (786)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/rehabilitation-medicine-and-physical-therapy" class="" data-icon-position="" data-hide-link-title="0">
                           Rehabilitation Medicine and Physical Therapy
                          </a>
                          <span class="article-count">
                           (462)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/respiratory-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Respiratory Medicine
                          </a>
                          <span class="article-count">
                           (626)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/rheumatology" class="" data-icon-position="" data-hide-link-title="0">
                           Rheumatology
                          </a>
                          <span class="article-count">
                           (276)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/sexual-and-reproductive-health" class="" data-icon-position="" data-hide-link-title="0">
                           Sexual and Reproductive Health
                          </a>
                          <span class="article-count">
                           (227)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/sports-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Sports Medicine
                          </a>
                          <span class="article-count">
                           (214)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/surgery" class="" data-icon-position="" data-hide-link-title="0">
                           Surgery
                          </a>
                          <span class="article-count">
                           (254)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Toxicology
                          </a>
                          <span class="article-count">
                           (43)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/transplantation" class="" data-icon-position="" data-hide-link-title="0">
                           Transplantation
                          </a>
                          <span class="article-count">
                           (121)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/urology" class="" data-icon-position="" data-hide-link-title="0">
                           Urology
                          </a>
                          <span class="article-count">
                           (94)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom_medrxiv.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
