<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.10.06.327700v3.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.10.06.327700v3.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.10.06.327700v3.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/07/05/2020.10.06.327700.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.10.06.327700v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="The price of a bit: energetic costs and the evolution of cellular signaling"/>
  <meta name="DC.Identifier" content="10.1101/2020.10.06.327700"/>
  <meta name="DC.Date" content="2022-07-05"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Recent experiments have uncovered a fundamental information scale for cellular signaling networks: the correlation between input and output concentrations of molecules in a signaling pathway corresponds to at most 1-3 bits of mutual information. Our understanding of the physical constraints and evolutionary pressures that determine this scale remains incomplete. By focusing on a basic element of signaling pathways, the kinase-phosphatase enzymatic push-pull loop, we highlight the pivotal role played by energy resources available for signaling and their expenditure: the chemical potential energy of ATP hydrolysis, and the rate of ATP consumption. Scanning a broad range of reaction parameters based on enzymatic databases, we find that ATP chemical potentials in modern organisms are just above the threshold necessary to achieve empirical mutual information values. We also derive an analytical relation for the minimum ATP consumption required to maintain a certain signal fidelity across a range of input frequencies, where we quantify fidelity either through instantaneous or time-delayed mutual information. Attempting to increase signal fidelity beyond a few bits lowers the bandwidth, the maximum characteristic signal frequency that the network can handle at a given energy cost. The observed information scale thus represents a balancing act between fidelity and the ability to process fast-changing environmental signals. Our analytical relation defines a performance limit for kinase-phosphatase networks, and we find evidence that a component of the yeast osmotic shock pathway may be close to the optimality line. By quantifying the evolutionary pressures that operate on these networks, we argue that this is not a coincidence: natural selection on energy expenditures is capable of pushing signaling systems toward optimality, particularly in unicellular organisms. Our theoretical framework is directly verifiable using existing experimental techniques, and predicts that more examples of such optimality should exist in nature.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Teng-Long Wang"/>
  <meta name="DC.Contributor" content="Benjamin Kuznets-Speck"/>
  <meta name="DC.Contributor" content="Joseph Broderick"/>
  <meta name="DC.Contributor" content="Michael Hinczewski"/>
  <meta name="article:published_time" content="2022-07-05"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="The price of a bit: energetic costs and the evolution of cellular signaling"/>
  <meta name="citation_abstract" lang="en" content="<p>Recent experiments have uncovered a fundamental information scale for cellular signaling networks: the correlation between input and output concentrations of molecules in a signaling pathway corresponds to at most 1-3 bits of mutual information. Our understanding of the physical constraints and evolutionary pressures that determine this scale remains incomplete. By focusing on a basic element of signaling pathways, the kinase-phosphatase enzymatic push-pull loop, we highlight the pivotal role played by energy resources available for signaling and their expenditure: the chemical potential energy of ATP hydrolysis, and the rate of ATP consumption. Scanning a broad range of reaction parameters based on enzymatic databases, we find that ATP chemical potentials in modern organisms are just above the threshold necessary to achieve empirical mutual information values. We also derive an analytical relation for the minimum ATP consumption required to maintain a certain signal fidelity across a range of input frequencies, where we quantify fidelity either through instantaneous or time-delayed mutual information. Attempting to increase signal fidelity beyond a few bits lowers the bandwidth, the maximum characteristic signal frequency that the network can handle at a given energy cost. The observed information scale thus represents a balancing act between fidelity and the ability to process fast-changing environmental signals. Our analytical relation defines a performance limit for kinase-phosphatase networks, and we find evidence that a component of the yeast osmotic shock pathway may be close to the optimality line. By quantifying the evolutionary pressures that operate on these networks, we argue that this is not a coincidence: natural selection on energy expenditures is capable of pushing signaling systems toward optimality, particularly in unicellular organisms. Our theoretical framework is directly verifiable using existing experimental techniques, and predicts that more examples of such optimality should exist in nature.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.10.06.327700v3"/>
  <meta name="citation_id" content="2020.10.06.327700v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.10.06.327700v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.10.06.327700v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.10.06.327700v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.10.06.327700"/>
  <meta name="citation_num_pages" content="44"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.10.06.327700"/>
  <meta name="citation_author" content="Teng-Long Wang"/>
  <meta name="citation_author_institution" content="Department of Physics, Case Western Reserve University"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3012-4659"/>
  <meta name="citation_author" content="Benjamin Kuznets-Speck"/>
  <meta name="citation_author_institution" content="Department of Physics, Case Western Reserve University"/>
  <meta name="citation_author_institution" content="Biophysics Graduate Group, University of California"/>
  <meta name="citation_author" content="Joseph Broderick"/>
  <meta name="citation_author_institution" content="Department of Physics, Case Western Reserve University"/>
  <meta name="citation_author" content="Michael Hinczewski"/>
  <meta name="citation_author_institution" content="Department of Physics, Case Western Reserve University"/>
  <meta name="citation_author_email" content="mxh605@case.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2837-7697"/>
  <meta name="citation_reference" content="Balázsi, G., van Oudenaarden, A. & Collins, J. J. Cellular decision making and biological noise: from microbes to mammals. Cell 144, 910–925 (2011)."/>
  <meta name="citation_reference" content="Govern, C. C. & ten Wolde, P. R. Optimal resource allocation in cellular sensing systems. Proc. Natl. Acad. Sci. 111, 17486–17491 (2014)."/>
  <meta name="citation_reference" content="ten Wolde, P. R., Becker, N. B., Ouldridge, T. E. & Mugler, A. Fundamental limits to cellular sensing. J. Stat. Phys. 162, 1395–1424 (2016)."/>
  <meta name="citation_reference" content="Lan, G. & Tu, Y. Information processing in bacteria: memory, computation, and statistical physics: a key issues review. Rep. Prog. Phys. 79, 052601 (2016)."/>
  <meta name="citation_reference" content="Logan, C. Y. & Nusse, R. The Wnt signaling pathway in development and disease. Annu. Rev. Cell Dev. Biol. 20, 781–810 (2004)."/>
  <meta name="citation_reference" content="Parsons, D. W. et al. Colorectal cancer: mutations in a signalling pathway. Nature 436, 792 (2005)."/>
  <meta name="citation_reference" content="Sarkar, F. H., Li, Y., Wang, Z. & Kong, D. Cellular signaling perturbation by natural products. Cell. Sign. 21, 1541–1547 (2009)."/>
  <meta name="citation_reference" content="Riera, C. E., Merkwirth, C., De Magalhaes Filho, C. D. & Dillin, A. Signaling networks determining life span. Annu. Rev. Biochem. 85, 35–64 (2016)."/>
  <meta name="citation_reference" content="Uda, S. Application of information theory in systems biology. Biophys. Rev. 1–8 (2020)."/>
  <meta name="citation_reference" content="Tkačik, G., Callan, C. G. & Bialek, W. Information flow and optimization in transcriptional regulation. Proc. Natl. Acad. Sci. 105, 12265–12270 (2008)."/>
  <meta name="citation_reference" content="Cheong, R., Rhee, A., Wang, C. J., Nemenman, I. & Levchenko, A. Information transduction capacity of noisy biochemical signaling networks. Science 334, 354–358 (2011)."/>
  <meta name="citation_reference" content="Uda, S. et al. Robustness and compensation of information transmission of signaling pathways. Science 341, 558–561 (2013)."/>
  <meta name="citation_reference" content="Voliotis, M., Perrett, R. M., McWilliams, C., McArdle, C. A. & Bowsher, C. G. Information transfer by leaky, heterogeneous, protein kinase signaling systems. Proc. Natl. Acad. Sci. 111, E326–E333 (2014)."/>
  <meta name="citation_reference" content="Selimkhanov, J. et al. Accurate information transmission through dynamic biochemical signaling networks. Science 346, 1370–1373 (2014)."/>
  <meta name="citation_reference" content="Potter, G. D., Byrd, T. A., Mugler, A. & Sun, B. Dynamic sampling and information encoding in biochemical networks. Biophys. J. 112, 795–804 (2017)."/>
  <meta name="citation_reference" content="Suderman, R., Bachman, J. A., Smith, A., Sorger, P. K. & Deeds, E. J. Fundamental trade-offs between information flow in single cells and cellular populations. Proc. Natl. Acad. Sci. 114, 5755–5760 (2017)."/>
  <meta name="citation_reference" content="Keshelava, A. et al. High capacity in g protein-coupled receptor signaling. Nat. Commun. 9, 1–8 (2018)."/>
  <meta name="citation_reference" content="Shannon, C. E. A mathematical theory of communication. Bell Systems Tech. J. 27, 379–423 (1948)."/>
  <meta name="citation_reference" content="Cover, T. M. & Thomas, J. A. Elements of information theory (John Wiley & Sons, 2012)."/>
  <meta name="citation_reference" content="Cao, Y., Wang, H., Ouyang, Q. & Tu, Y. The free-energy cost of accurate biochemical oscillations. Nat. Phys. 11, 772–778 (2015)."/>
  <meta name="citation_reference" content="Hasegawa, Y. Optimal temporal patterns for dynamical cellular signaling. New J. Phys. 18, 113031 (2016)."/>
  <meta name="citation_reference" content="Mehta, P., Lang, A. H. & Schwab, D. J. Landauer in the age of synthetic biology: energy consumption and information processing in biochemical networks. J. Stat. Phys. 162, 1153–1166 (2016)."/>
  <meta name="citation_reference" content="Ouldridge, T. E., Govern, C. C. & ten Wolde, P. R. Thermodynamics of computational copying in biochemical systems. Phys. Rev. X 7, 021004 (2017)."/>
  <meta name="citation_reference" content="Milo, R. & Phillips, R. Cell biology by the numbers (Garland Science, 2015)."/>
  <meta name="citation_reference" content="Lane, N. & Martin, W. F. The origin of membrane bioenergetics. Cell 151, 1406–1416 (2012)."/>
  <meta name="citation_reference" content="Stadtman, E. & Chock, P. Superiority of interconvertible enzyme cascades in metabolic regulation: analysis of monocyclic systems. Proc. Natl. Acad. Sci. 74, 2761–2765 (1977)."/>
  <meta name="citation_reference" content="Goldbeter, A. & Koshland, D. E. An amplified sensitivity arising from covalent modification in biological systems. Proc. Natl. Acad. Sci. 78, 6840–6844 (1981)."/>
  <meta name="citation_reference" content="Detwiler, P. B., Ramanathan, S., Sengupta, A. & Shraiman, B. I. Engineering aspects of enzymatic signal transduction: photoreceptors in the retina. Biophys. J. 79, 2801–2817 (2000)."/>
  <meta name="citation_reference" content="Heinrich, R., Neel, B. G. & Rapoport, T. A. Mathematical models of protein kinase signal transduction. Molecular cell 9, 957–970 (2002)."/>
  <meta name="citation_reference" content="Hinczewski, M. & Thirumalai, D. Cellular signaling networks function as generalized wiener-kolmogorov filters to suppress noise. Phys. Rev. X 4, 041017 (2014)."/>
  <meta name="citation_reference" content="Hinczewski, M. & Thirumalai, D. Noise control in gene regulatory networks with negative feedback. J. Phys. Chem. B 120, 6166–6177 (2016)."/>
  <meta name="citation_reference" content="Lynch, M. & Marinov, G. K. The bioenergetic costs of a gene. Proc. Natl. Acad. Sci. 112, 15690–15695 (2015)."/>
  <meta name="citation_reference" content="Ilker, E. & Hinczewski, M. Modeling the growth of organisms validates a general relation between metabolic costs and natural selection. Phys. Rev. Lett. 122, 238101 (2019)."/>
  <meta name="citation_reference" content="Sturm, O. E. et al. The mammalian MAPK/ERK pathway exhibits properties of a negative feedback amplifier. Sci. Signal. 3, ra90–ra90 (2010)."/>
  <meta name="citation_reference" content="Saxena, M., Williams, S., Taskén, K. & Mustelin, T. Crosstalk between camp-dependent kinase and map kinase through a protein tyrosine phosphatase. Nat. Cell Biol. 1, 305 (1999)."/>
  <meta name="citation_reference" content="Salazar, C. & Höfer, T. Multisite protein phosphorylation–from molecular mechanisms to kinetic models. FEBS J. 276, 3177–3198 (2009)."/>
  <meta name="citation_reference" content="Fersht, A. Structure and mechanism in protein science: a guide to enzyme catalysis and protein folding (Macmillan, 1999)."/>
  <meta name="citation_reference" content="Qian, H. Cooperativity and specificity in enzyme kinetics: a single-molecule time-based per-spective. Biophys. J. 95, 10–17 (2008)."/>
  <meta name="citation_reference" content="Gillespie, D. T. The chemical Langevin equation. J. Chem. Phys. 113, 297–306 (2000)."/>
  <meta name="citation_reference" content="Gillespie, D. T. Exact stochastic simulation of coupled chemical reactions. J. Phys. Chem. 81, 2340–2361 (1977)."/>
  <meta name="citation_reference" content="Bar-Even, A. et al. The moderately efficient enzyme: evolutionary and physicochemical trends shaping enzyme parameters. Biochemistry 50, 4402–4410 (2011)."/>
  <meta name="citation_reference" content="Liebermeister, W. & Klipp, E. Biochemical networks with uncertain parameters. IEE Proc.-Syst. Biol. 152, 97–107 (2005)."/>
  <meta name="citation_reference" content="Wang, M., Herrmann, C. J., Simonovic, M., Szklarczyk, D. & von Mering, C. Version 4.0 of PaxDb: protein abundance data, integrated across model organisms, tissues, and cell-lines. Proteomics 15, 3163–3168 (2015)."/>
  <meta name="citation_reference" content="UniProt Consortium. UniProt: a worldwide hub of protein knowledge. Nucleic Acids Res. 47, D506–D515 (2018)."/>
  <meta name="citation_reference" content="Wittig, U. et al. SABIO-RK—database for biochemical reaction kinetics. Nucleic Acids Res. 40, D790–D796 (2011)."/>
  <meta name="citation_reference" content="Zi, Z., Liebermeister, W. & Klipp, E. A quantitative study of the Hog1 MAPK response to fluctuating osmotic stress in Saccharomyces cerevisiae. PLOS ONE 5, e9522 (2010)."/>
  <meta name="citation_reference" content="Hersen, P., McClean, M. N., Mahadevan, L. & Ramanathan, S. Signal processing by the hog map kinase pathway. Proc. Natl. Acad. Sci. 105, 7165–7170 (2008)."/>
  <meta name="citation_reference" content="Wiener, N. Extrapolation, Interpolation and Smoothing of Stationary Times Series (Wiley, New York, 1949)."/>
  <meta name="citation_reference" content="Kolmogorov, A. N. Interpolation and extrapolation of stationary random sequences. Izv. Akad. Nauk SSSR., Ser. Mat. 5, 3–14 (1941)."/>
  <meta name="citation_reference" content="Bode, H. W. & Shannon, C. E. A simplified derivation of linear least square smoothing and prediction theory. Proc. Inst. Radio. Engin. 38, 417–425 (1950)."/>
  <meta name="citation_reference" content="Becker, N. B., Mugler, A. & ten Wolde, P. R. Optimal prediction by cellular signaling networks. Phys. Rev. Lett. 115 (258103, 2015)."/>
  <meta name="citation_reference" content="Zechner, C., Seelig, G., Rullan, M. & Khammash, M. Molecular circuits for dynamic noise filtering. Proc. Natl. Acad. Sci. 113, 4729–4734 (2016)."/>
  <meta name="citation_reference" content="Samanta, H. S., Hinczewski, M. & Thirumalai, D. Optimal information transfer in enzymatic networks: A field theoretic formulation. Phys. Rev. E 96, 012406 (2017)."/>
  <meta name="citation_reference" content="Hathcock, D., Sheehy, J., Weisenberger, C., Ilker, E. & Hinczewski, M. Noise filtering and prediction in biological signaling networks. IEEE Trans. Mol. Biol. Multi-Scale Commun. 2, 16–30 (2016)."/>
  <meta name="citation_reference" content="Nemenman, I. Gain control in molecular information processing: lessons from neuroscience. Phys. Biol. 9, 026003 (2012)."/>
  <meta name="citation_reference" content="Mancini, F., Wiggins, C. H., Marsili, M. & Walczak, A. M. Time-dependent information transmission in a model regulatory circuit. Phys. Rev. E 88, 022708 (2013)."/>
  <meta name="citation_reference" content="Mettetal, J. T., Muzzey, D., Gómez-Uribe, C. & van Oudenaarden, A. The frequency dependence of osmo-adaptation in Saccharomyces cerevisiae. Science 319, 482–484 (2008)."/>
  <meta name="citation_reference" content="Charlesworth, B. Effective population size and patterns of molecular evolution and variation. Nature Rev. Genet. 10, 195 (2009)."/>
  <meta name="citation_reference" content="Gillespie, J. H. Population genetics: a concise guide (JHU Press, 2010)."/>
  <meta name="citation_reference" content="Kimura, M. On the probability of fixation of mutant genes in a population. Genetics 47, 713–719 (1962)."/>
  <meta name="citation_reference" content="Orgel, L. E. & Crick, F. H. Selfish DNA: the ultimate parasite. Nature 284, 604 (1980)."/>
  <meta name="citation_reference" content="Wagner, A. Energy constraints on the evolution of gene expression. Mol. Biol. Evol. 22, 1365–1374 (2005)."/>
  <meta name="citation_reference" content="Mahmoudabadi, G., Milo, R. & Phillips, R. Energetic cost of building a virus. Proc. Natl. Acad. Sci. 114, E4324–E4333 (2017)."/>
  <meta name="citation_reference" content="Tsai, I. J., Bensasson, D., Burt, A. & Koufopanou, V. Population genomics of the wild yeast Saccharomyces paradoxus: quantifying the life cycle. Proc. Natl. Acad. Sci. 105, 4957–4962 (2008)."/>
  <meta name="citation_reference" content="Qian, H. Cooperativity and specificity in enzyme kinetics: a single-molecule time-based perspective. Biophys. J. 95, 10–17 (2008)."/>
  <meta name="citation_reference" content="Hinczewski, M. & Thirumalai, D. Cellular signaling networks function as generalized Wiener-Kolmogorov filters to suppress noise. Phys. Rev. X 4, 041017 (2014)."/>
  <meta name="citation_reference" content="Gillespie, D. T. The chemical Langevin equation. J. Chem. Phys. 113, 297–306 (2000)."/>
  <meta name="citation_reference" content="Gillespie, D. T. Exact stochastic simulation of coupled chemical reactions. J. Phys. Chem. 81, 2340–2361 (1977)."/>
  <meta name="citation_reference" content="Hathcock, D., Sheehy, J., Weisenberger, C., Ilker, E. & Hinczewski, M. Noise filtering and prediction in biological signaling networks. IEEE Trans. Mol. Biol. Multi-Scale Commun. 2, 16–30 (2016)."/>
  <meta name="citation_reference" content="Bar-Even, A. et al. The moderately efficient enzyme: evolutionary and physicochemical trends shaping enzyme parameters. Biochemistry 50, 4402–4410 (2011)."/>
  <meta name="citation_reference" content="Liebermeister, W. & Klipp, E. Biochemical networks with uncertain parameters. IEE Proc.-Syst. Biol. 152, 97–107 (2005)."/>
  <meta name="citation_reference" content="Wang, M., Herrmann, C. J., Simonovic, M., Szklarczyk, D. & von Mering, C. Version 4.0 of PaxDb: protein abundance data, integrated across model organisms, tissues, and cell-lines. Proteomics 15, 3163–3168 (2015)."/>
  <meta name="citation_reference" content="Wittig, U. et al. SABIO-RK—database for biochemical reaction kinetics. Nucleic Acids Res. 40, D790–D796 (2011)."/>
  <meta name="citation_reference" content="UniProt Consortium. UniProt: a worldwide hub of protein knowledge. Nucleic Acids Res. 47, D506–D515 (2018)."/>
  <meta name="citation_reference" content="Milo, R. What is the total number of protein molecules per cell volume? a call to rethink some published values. Bioessays 35, 1050–1055 (2013)."/>
  <meta name="citation_reference" content="Schoeberl, B., Eichler-Jonsson, C., Gilles, E. D. & Müller, G. Computational modeling of the dynamics of the map kinase cascade activated by surface and internalized EGF receptors. Nat. Biotech. 20, 370–375 (2002)."/>
  <meta name="citation_reference" content="Zi, Z., Liebermeister, W. & Klipp, E. A quantitative study of the Hog1 MAPK response to fluctuating osmotic stress in Saccharomyces cerevisiae. PLOS ONE 5, e9522 (2010)."/>
  <meta name="citation_reference" content="Mettetal, J. T., Muzzey, D., Gómez-Uribe, C. & van Oudenaarden, A. The frequency dependence of osmo-adaptation in saccharomyces cerevisiae. Science 319, 482–484 (2008)."/>
  <meta name="citation_reference" content="Hersen, P., McClean, M. N., Mahadevan, L. & Ramanathan, S. Signal processing by the HOG MAP kinase pathway. Proc. Natl. Acad. Sci. 105, 7165–7170 (2008)."/>
  <meta name="citation_reference" content="Mattison, C. P. & Ota, I. M. Two protein tyrosine phosphatases, Ptp2 and Ptp3, modulate the subcellular localization of the Hog1 MAP kinase in yeast. Genes Dev. 14, 1229–1235 (2000)."/>
  <meta name="citation_reference" content="Fersht, A. Structure and mechanism in protein science: a guide to enzyme catalysis and protein folding (Macmillan, 1999)."/>
  <meta name="citation_reference" content="Ghaemmaghami, S. et al. Global analysis of protein expression in yeast. Nature 425, 737–741 (2003)."/>
  <meta name="citation_reference" content="Milo, R. & Phillips, R. Cell biology by the numbers (Garland Science, 2015)."/>
  <meta name="citation_reference" content="Ilker, E. & Hinczewski, M. Modeling the growth of organisms validates a general relation between metabolic costs and natural selection. Phys. Rev. Lett. 122, 238101 (2019)."/>
  <meta name="citation_reference" content="Lynch, M. & Marinov, G. K. The bioenergetic costs of a gene. Proc. Natl. Acad. Sci. 112, 15690–15695 (2015)."/>
  <meta name="twitter:title" content="The price of a bit: energetic costs and the evolution of cellular signaling"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Recent experiments have uncovered a fundamental information scale for cellular signaling networks: the correlation between input and output concentrations of molecules in a signaling pathway corresponds to at most 1-3 bits of mutual information. Our understanding of the physical constraints and evolutionary pressures that determine this scale remains incomplete. By focusing on a basic element of signaling pathways, the kinase-phosphatase enzymatic push-pull loop, we highlight the pivotal role played by energy resources available for signaling and their expenditure: the chemical potential energy of ATP hydrolysis, and the rate of ATP consumption. Scanning a broad range of reaction parameters based on enzymatic databases, we find that ATP chemical potentials in modern organisms are just above the threshold necessary to achieve empirical mutual information values. We also derive an analytical relation for the minimum ATP consumption required to maintain a certain signal fidelity across a range of input frequencies, where we quantify fidelity either through instantaneous or time-delayed mutual information. Attempting to increase signal fidelity beyond a few bits lowers the bandwidth, the maximum characteristic signal frequency that the network can handle at a given energy cost. The observed information scale thus represents a balancing act between fidelity and the ability to process fast-changing environmental signals. Our analytical relation defines a performance limit for kinase-phosphatase networks, and we find evidence that a component of the yeast osmotic shock pathway may be close to the optimality line. By quantifying the evolutionary pressures that operate on these networks, we argue that this is not a coincidence: natural selection on energy expenditures is capable of pushing signaling systems toward optimality, particularly in unicellular organisms. Our theoretical framework is directly verifiable using existing experimental techniques, and predicts that more examples of such optimality should exist in nature.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="The price of a bit: energetic costs and the evolution of cellular signaling"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.10.06.327700v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Recent experiments have uncovered a fundamental information scale for cellular signaling networks: the correlation between input and output concentrations of molecules in a signaling pathway corresponds to at most 1-3 bits of mutual information. Our understanding of the physical constraints and evolutionary pressures that determine this scale remains incomplete. By focusing on a basic element of signaling pathways, the kinase-phosphatase enzymatic push-pull loop, we highlight the pivotal role played by energy resources available for signaling and their expenditure: the chemical potential energy of ATP hydrolysis, and the rate of ATP consumption. Scanning a broad range of reaction parameters based on enzymatic databases, we find that ATP chemical potentials in modern organisms are just above the threshold necessary to achieve empirical mutual information values. We also derive an analytical relation for the minimum ATP consumption required to maintain a certain signal fidelity across a range of input frequencies, where we quantify fidelity either through instantaneous or time-delayed mutual information. Attempting to increase signal fidelity beyond a few bits lowers the bandwidth, the maximum characteristic signal frequency that the network can handle at a given energy cost. The observed information scale thus represents a balancing act between fidelity and the ability to process fast-changing environmental signals. Our analytical relation defines a performance limit for kinase-phosphatase networks, and we find evidence that a component of the yeast osmotic shock pathway may be close to the optimality line. By quantifying the evolutionary pressures that operate on these networks, we argue that this is not a coincidence: natural selection on energy expenditures is capable of pushing signaling systems toward optimality, particularly in unicellular organisms. Our theoretical framework is directly verifiable using existing experimental techniques, and predicts that more examples of such optimality should exist in nature.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-07-05"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.10.06.327700v3"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2629145"/>
  <title>
   The price of a bit: energetic costs and the evolution of cellular signaling | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"o_TDEluPsw2McY4vTI6GjHi7qiNVkgUvvA82xe4EBpU"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2629145","apath":"\/biorxiv\/early\/2022\/07\/05\/2020.10.06.327700.atom","pisa":"biorxiv;2020.10.06.327700v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.10.06.327700v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.10.06.327700v3","title":"The price of a bit: energetic costs and the evolution of cellular signaling","identifier":"node\/2629145"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AlnkT8JslB+zFM0cvdTGyhWW26qCRoEZjXt8S+NdMEwp7Cg2uwIqlj+9UVPgmpRAN2GgIs\\\\\\\/srGJ97TVNIW9AUWsVKBlcvvN8MCNh4TtQp5ytIC9\\\\\\\/KR\\\\\\\/dcT6A7tYB61RVvC7Wbpz8OKVFvXOfGBdW3+\\\\\\\/4P26sFLmMLh+aCRJ0dIz80h0+eXJBitUWZ+rjfqUF9EQ4TGKxGDcJlRYGGoimcWht7Egi3jXaOWz9IaKyMT62peuAUmUYWZpBB+3JA42pKGhXStM+9ZbR3DycptzcJOKvDo\\\\\\\/z1WXZDa3SevqCJEH7eXBlie6uza80t9gmH8hTKIut\\\\\\\/UUkRiSbsf6Fqdoqrjbtm2m8a+69Qe2YYs1+sodW\\\\\\\/kV2XyXOC81eVV3eW8KerIbt3szGoVZzJTwritlc2Qxy4F1nmTRSssJsRjDw\\\\\\\/Dh9s8ofsBSekiRcMaIgr+7EbyWnzixC\\\\\\\/WA9YcLbYINe5EwHP\\\\\\\/yOIvqWn8nwLQIx5uknH+gnWxiQ4CA1PfyH11\\\\\\\/R+U88\\\\\\\/MMCttqsXaRmVrSYXehUtzpRxtXzlDg8M36lDwfXi3oWR5CSYgtYqSSJbgvfSZI\\\\\\\/Luk+++Q97vOJMsQaB\\\\\\\/G+Uk6ndHYRmYVCeUqQyPB2nRnu\\\\\\\/tk7f7T2XIObTKbBTUox3GD9OBaJ65VG5n2RGmvjb0CHUwsX6plpQhSYW7+e3aEC25JMR0GSq901awxHjszEsEuEh4GldX5Cj342pFd1kyVIgzYtM8rOKmQrcGp+kBAjSscUxk78FQFPZSyNQof2jjcN0oED2VuRSv\\\\\\\/pwFY10K1rQDQMq7zT6TUsV80E+zKj4E\\\\\\\/OTmw94YT2abLXFFr4ejU6DeCDn3aOgtY5YFaVW0aZ0Z9yLk3cyVpouKhhCCt6s5wRFWKeLv1RH91pSffGAxuq6koeGOLdZUzBmxGyc7T\\\\\\\/zMsxJhR8tc8wiTQX7NJZ0phKihStE\\\\\\\/4XWLqC1nJuChtVzSPUwQnbb9btq1dtlhdAvQTXEGHDmKkir7flvmtuYpD1ra3IJKlT4jHgEEqHwCevQOr7PEBQ9+qx8JWhBeWeWf1SjEtSYV\\\\\\\/WGcFwgIh2N4NbOA1tUa\\\\\\\/Ot9QknxKJQhUG6myCMg0pdokqaRNu8N\\\\\\\/kCas3o1cESB21CfSPEeRFSuZ\\\\\\\/KxFZ8Dd03B2n5YyyT6C796NG6dszJqANIknqsQrGflvyhh7XYiA3owoYLUimpCydZVF+rIEvCu6b88eG+PRWkFgxKNUOHw6+SUXtraX2MKFGnep25R9wKYL6DppWkIuLpb7VTrGN1siBagdzoQ3T8mGHwslm3QhviZWp1+FcecyF9a08EJWG0y2HAdCRNsscCdESbzlwucWZ4jL8wgZTXzBsJRWyJ1DnfjBhc3ychA0YuhxfQnYnY3c=\\\u0022,\\\u0022iv\\\u0022:\\\u0022sCqE4ttTFkwVH9Ij8VI93g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00226e4d9234c7822f426397547385ee7dd308d645255ef95fef52bc5b945a309dee\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.10.06.327700v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2629145 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.10.06.327700v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_978799444">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_978799444" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_394943425" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-Gz331GOmFgZuVBI93dfd5TzF_Eob2ZSJO_U2F7Bo-98"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2629145" data-node-nid="2629145" id="node-2629145--21216004123" data-pisa="biorxiv;2020.10.06.327700v3" data-pisa-master="biorxiv;2020.10.06.327700" data-apath="/biorxiv/early/2022/07/05/2020.10.06.327700.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    The price of a bit: energetic costs and the evolution of cellular signaling
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0002-3012-4659" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Teng-Long
                      </span>
                      <span class="nlm-surname">
                       Wang
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Benjamin
                      </span>
                      <span class="nlm-surname">
                       Kuznets-Speck
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Joseph
                      </span>
                      <span class="nlm-surname">
                       Broderick
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3">
                      <a href="http://orcid.org/0000-0003-2837-7697" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Michael
                      </span>
                      <span class="nlm-surname">
                       Hinczewski
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.10.06.327700
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2629145--21216004123" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Teng-Long Wang
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Physics, Case Western Reserve University
                       </span>
                       , Cleveland, OH 44106
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Teng-Long%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Wang%20T&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ATeng-Long%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-3012-4659" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Teng-Long Wang
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Benjamin Kuznets-Speck
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Physics, Case Western Reserve University
                       </span>
                       , Cleveland, OH 44106
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Biophysics Graduate Group, University of California
                       </span>
                       , Berkeley, CA 94720,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Benjamin%2BKuznets-Speck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Kuznets-Speck%20B&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ABenjamin%2BKuznets-Speck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Joseph Broderick
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Physics, Case Western Reserve University
                       </span>
                       , Cleveland, OH 44106
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Joseph%2BBroderick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Broderick%20J&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AJoseph%2BBroderick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Michael Hinczewski
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Physics, Case Western Reserve University
                       </span>
                       , Cleveland, OH 44106
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Michael%2BHinczewski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Hinczewski%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AMichael%2BHinczewski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0003-2837-7697" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Michael Hinczewski
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:mxh605@case.edu" class="" data-icon-position="" data-hide-link-title="0">
                        mxh605@case.edu
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.10.06.327700v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2629145" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2629145/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.06.327700v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2629145" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2629145/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.06.327700v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2629145" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2629145/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.06.327700v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2629145" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2629145/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.06.327700v3.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:2629145" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:2629145/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.10.06.327700v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2629145" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2629145/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2>
                             Abstract
                            </h2>
                            <p id="p-2">
                             Recent experiments have uncovered a fundamental information scale for cellular signaling networks: the correlation between input and output concentrations of molecules in a signaling pathway corresponds to at most 1-3 bits of mutual information. Our understanding of the physical constraints and evolutionary pressures that determine this scale remains incomplete. By focusing on a basic element of signaling pathways, the kinase-phosphatase enzymatic push-pull loop, we highlight the pivotal role played by energy resources available for signaling and their expenditure: the chemical potential energy of ATP hydrolysis, and the rate of ATP consumption. Scanning a broad range of reaction parameters based on enzymatic databases, we find that ATP chemical potentials in modern organisms are just above the threshold necessary to achieve empirical mutual information values. We also derive an analytical relation for the minimum ATP consumption required to maintain a certain signal fidelity across a range of input frequencies, where we quantify fidelity either through instantaneous or time-delayed mutual information. Attempting to increase signal fidelity beyond a few bits lowers the bandwidth, the maximum characteristic signal frequency that the network can handle at a given energy cost. The observed information scale thus represents a balancing act between fidelity and the ability to process fast-changing environmental signals. Our analytical relation defines a performance limit for kinase-phosphatase networks, and we find evidence that a component of the yeast osmotic shock pathway may be close to the optimality line. By quantifying the evolutionary pressures that operate on these networks, we argue that this is not a coincidence: natural selection on energy expenditures is capable of pushing signaling systems toward optimality, particularly in unicellular organisms. Our theoretical framework is directly verifiable using existing experimental techniques, and predicts that more examples of such optimality should exist in nature.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             I. INTRODUCTION
                            </h2>
                            <p id="p-6">
                             Survival for living cells depends in part on accurate and responsive signaling: the ability to collect enough information about the micro-environment to make decisions in response to external stimuli such as nutrients, hormones, and toxic agents [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ]. This capacity to react to extracellular cues developed early in evolutionary history, and is now seen at all levels of biological organization, from chemotaxis in unicellular organisms [
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             –
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ] to the pathways that regulate cell differentiation and disease in multicellular life [
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             –
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ]. Despite the resulting diversity of biochemical networks that implement this signaling, information theory provides a powerful universal framework to quantify the amount of information transferred through a network, allowing comparisons between different systems [
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ].
                            </p>
                            <p id="p-7">
                             Over the last decade a remarkable experimental consensus has emerged from such comparisons: studies of both prokaryotic and eukaryotic signaling pathways have found they can transmit at most ∼ 1 to 3 bits of information [
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             –
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             ]. These values refer to mutual infor-mation (MI) between pathway input (concentrations of a molecule representing the signal) and output (concentrations of a downstream molecule produced by the network, sampled either at a single or multiple time points). MI is a measure of signal fidelity, representing the degree of correlation between input and output. Experiments have typically focused on a closely related quantity known as the channel capacity [
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ,
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              19
                             </a>
                             ]: the maximum MI achievable among all input distributions.
                            </p>
                            <p id="p-8">
                             The consistently small channel capacities observed in cellular signaling pathways seem to indicate that cells operate with a fairly coarse representation of their surroundings: n bits of MI corresponds to being able to reliably distinguish between 2
                             <sup>
                              n
                             </sup>
                             levels of the input, so a 1 bit pathway can only discriminate between “high” versus “low” signal concentrations. Though 1 bit is typical for MI measured at single time points, one can achieve higher MIs by focusing on output responses collected over several time points [
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ,
                             <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ], or by designing the experiment to isolate single-cell responses (as opposed to estimating MI from the responses of a population of cells) [
                             <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             ]. But these enhancements, which can push values to the 2-3 bit range, do not change the fundamental order of magnitude of the MI.
                            </p>
                            <p id="p-9">
                             The central question we explore in this work is to what extent this fundamental information scale is shaped by the energy requirements of the underlying biochemical signaling networks. In order to transmit information, these networks necessarily need to operate out of equilibrium, fueled by processes like ATP hydrolysis that consume energetic resources. Recent research highlights these costs as an essential factor in understanding constraints on signaling [
                             <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             –
                             <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ,
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              20
                             </a>
                             –
                             <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ], often focusing on the ATP hydrolysis chemical potential difference
                             <span class="inline-formula" id="inline-formula-1">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-1.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-1.gif"/>
                               </noscript>
                              </span>
                             </span>
                             between the reactant (ATP) and products (ADP and inorganic phosphate, P
                             <sub>
                              i
                             </sub>
                             ), quantifying the free energy available to drive the system per ATP. Crossing a certain minimum threshold of Δμ is a prerequisite for a variety of signaling functions: accurate read-out of ligand-bound receptors [
                             <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ,
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ,
                             <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ], maintaining the phase coherence of oscillations in circadian clocks [
                             <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">
                              20
                             </a>
                             ], or preserving the integrity of methylation-based “mem-ory” to facilitate adaptation in chemotaxis [
                             <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ]. This threshold is typically a few times larger (i.e. by a factor of ∼ 3 − 4 [
                             <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ,
                             <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ]) than the energy scale of thermal fluctuations, k
                             <sub>
                              B
                             </sub>
                             T, where k
                             <sub>
                              B
                             </sub>
                             is the Boltzmann constant and T the temperature. And indeed cells across the various domains of life maintain a sufficiently high Δμ ≈ 21 − 29 k
                             <sub>
                              B
                             </sub>
                             T [
                             <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                              24
                             </a>
                             ] to enable such functions. The large value and remarkably narrow range of Δμ observed in modern organisms opens up additional questions. The metabolic cycles that sustain Δμ, constantly replenishing ATP as it is hydrolyzed, must almost necessarily have been far more inefficient and wasteful in the earliest stages of evolutionary history [
                             <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                              25
                             </a>
                             ]. To what degree could organisms operating with smaller Δμ still process information about their environment? What kinds of evolutionary pressures might have driven Δμ to its modern range? And if the costs of individual signaling systems are non-trivial [
                             <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ,
                             <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ], could natural selection have driven these networks toward optimized, energy-efficient solutions?
                            </p>
                            <p id="p-10">
                             To investigate these issues, we focus on one of the canonical signaling circuits in biology, the kinase-phosphatase “push-pull loop”, which often forms a basic unit of more complicated signaling cascades [
                             <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                              26
                             </a>
                             –
                             <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                              29
                             </a>
                             ]. An active kinase enzyme instigates the “push”, chemically modifying a substrate protein via phosphorylation (consuming ATP in the process), while a phosphatase enzyme provides the “pull”, dephosphorylating the modified substrate, reverting it to its original state. We derive the relationships between three facets of the system: i) the MI between the input (active kinase) and output (phosphorylated substrate) molecular populations; ii) the timescales over which the input signal varies; and iii) the energy requirements, expressed in terms of Δμ and the rate of ATP consumption. Exploring the entire spectrum of kinase/phosphatase enzymatic parameters from bioinformatic databases, we find that physiological Δμ values are just large enough to enable a MI of 1-2 bits for the widest possible parameter range. However to achieve this MI for signals that vary rapidly in time becomes more challenging, requiring both precise fine-tuning of parameters and a certain minimum rate of ATP consumption. In fact, taking advantage of results from optimal noise filter theory [
                             <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                              30
                             </a>
                             ,
                             <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                              31
                             </a>
                             ], we derive a remarkably simple analytical relationship that describes the tradeoffs between minimum ATP rate, the MI, and the maximum characteristic signal frequency (the so-called bandwidth) which the push-pull network can handle. Verified via extensive numerical simulations across the whole gamut of enzymatic parameters, this relation is a novel theoretical prediction that can be directly tested in future experiments. We also demonstrate that analogous constraints hold even in the general case of time-delayed MI, where we look at correlation between the output and input signals separated by a time offset. The analytical relation rationalizes the observed range of MI by showing that values much higher than 1-2 bits would require sacrificing the ability to process fast-changing signals. Finally we explore the question of whether there exist evolutionary pressures that would push such a system to be energy efficient, optimizing the ATP consumption for a given target MI and bandwidth. Using a recently developed formalism relating metabolic costs to the strength of natural selection [
                             <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                              32
                             </a>
                             ,
                             <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                              33
                             </a>
                             ], we show that these pressures can indeed be significant, particularly for single-celled organisms. We highlight a kinase-phosphatase loop in the yeast Hog1 signaling pathway as a system that may have been optimized by such pressures.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             II. THEORY
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              A. Modeling an enzymatic push-pull loop
                             </h3>
                             <p id="p-11">
                              This push-pull network consists of two opposing reactions: a kinase enzyme instigates the “push”, chemically modifying a substrate protein via phosphorylation, while a phosphatase enzyme provides the “pull”, dephosphorylating the modified substrate, reverting it to its original state [
                              <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">
                               26
                              </a>
                              –
                              <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                              ]. Since a single kinase can catalyze the phosphorylation of many substrate proteins, this loop can effectively act like an amplifier [
                              <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                               28
                              </a>
                              ], translating a weaker signal (a small cellular population of an active kinase) into a stronger one (a large population of a phosphorylated substrate). Often the substrate itself is a kinase that can exist in catalyti-cally inactive and active states, with activation triggered by phosphorylation. In this case one can have multi-tiered signaling cascades enhancing the amplification (as shown schematically in
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Fig. 1A
                              </a>
                              ) with the active substrate produced by one loop serving as the kinase for a downstream loop [
                              <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ]. More complex signaling networks are also possible, with multiple cascades connected by crosstalk through shared components [
                              <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                               35
                              </a>
                              ], feedback from downstream to upstream populations [
                              <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ], or activation requiring multisite phosphorylation [
                              <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                               36
                              </a>
                              ]. However, the starting point for understanding any of these more complex signaling topologies is the behavior of a single loop, with a substrate activated / deactivated through a single phosphorylation site.
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F1.large.jpg?width=800&height=600&carousel=1" title="(A) A schematic signaling pathway involving cascades of kinase phosphorylation, initiated by a receptor embedded in the cell membrane that responds to extracellular ligands. The system we focus on will be one stage of the pathway, a kinase-phosphatase push-pull loop, highlighted in the dashed box. (B) The molecular species and reaction parameters of the push-pull loop. The kinase (K) binds to the substrate (S), forming the complex (SK) that catalyzes the production of phosphorylated substrate (S*). Phosphatase (P) binds to S*, forming a complex  that catalyzes the dephosporylation of the substrate. Forward reaction / binding rates are labeled in black, while reverse reaction / unbinding rates are in red. (C) The loop serves to transduce an input signal, defined as the total population of kinase (bound or unbound), X(t) = K(t) + SK(t), into an output, defined as the total population of phosphorylated substrate, . The input signal has a characteristic autocorrelation time ." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383868837" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>(A)</strong> A schematic signaling pathway involving cascades of kinase phosphorylation, initiated by a receptor embedded in the cell membrane that responds to extracellular ligands. The system we focus on will be one stage of the pathway, a kinase-phosphatase push-pull loop, highlighted in the dashed box. <strong>(B)</strong> The molecular species and reaction parameters of the push-pull loop. The kinase (<em>K</em>) binds to the substrate (<em>S</em>), forming the complex (<em>SK</em>) that catalyzes the production of phosphorylated substrate (<em>S*</em>). Phosphatase (<em>P</em>) binds to <em>S*</em>, forming a complex  that catalyzes the dephosporylation of the substrate. Forward reaction / binding rates are labeled in black, while reverse reaction / unbinding rates are in red. <strong>(C)</strong> The loop serves to transduce an input signal, defined as the total population of kinase (bound or unbound), <em>X</em>(<em>t</em>) = <em>K</em>(<em>t</em>) + <em>SK</em>(<em>t</em>), into an output, defined as the total population of phosphorylated substrate, . The input signal has a characteristic autocorrelation time .</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="FIG. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F1.medium.gif" width="440" height="224"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="FIG. 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F1.medium.gif" width="440" height="224"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                FIG. 1.
                               </span>
                               <p id="p-12" class="first-child">
                                <strong>
                                 (A)
                                </strong>
                                A schematic signaling pathway involving cascades of kinase phosphorylation, initiated by a receptor embedded in the cell membrane that responds to extracellular ligands. The system we focus on will be one stage of the pathway, a kinase-phosphatase push-pull loop, highlighted in the dashed box.
                                <strong>
                                 (B)
                                </strong>
                                The molecular species and reaction parameters of the push-pull loop. The kinase (K) binds to the substrate (S), forming the complex (SK) that catalyzes the production of phosphorylated substrate (S*). Phosphatase (P) binds to S*, forming a complex
                                <span class="inline-formula" id="inline-formula-2">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F1/embed/inline-graphic-2.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F1/embed/inline-graphic-2.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                that catalyzes the dephosporylation of the substrate. Forward reaction / binding rates are labeled in black, while reverse reaction / unbinding rates are in red.
                                <strong>
                                 (C)
                                </strong>
                                The loop serves to transduce an input signal, defined as the total population of kinase (bound or unbound), X(t) = K(t) + SK(t), into an output, defined as the total population of phosphorylated substrate,
                                <span class="inline-formula" id="inline-formula-3">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F1/embed/inline-graphic-3.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F1/embed/inline-graphic-3.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                . The input signal has a characteristic autocorrelation time
                                <span class="inline-formula" id="inline-formula-4">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F1/embed/inline-graphic-4.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F1/embed/inline-graphic-4.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                .
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-13">
                              The reaction scheme of a single push-pull loop is shown in
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Fig. 1B
                              </a>
                              . Binding of free kinase (population K(t) at time t) to substrate (population S(t)) occurs with rate constant κ
                              <sub>
                               b
                              </sub>
                              , forming a kinase-substrate complex (population S
                              <sub>
                               K
                              </sub>
                              (t)). Phosphorylation of the substrate and its subsequent release constitutes the catalytic step, with rate κ
                              <sub>
                               r
                              </sub>
                              , yielding free phosphorylated substrates (population S
                              <sup>
                               *
                              </sup>
                              (t)). A phosphatase can subsequently bind, with rate ρ
                              <sub>
                               b
                              </sub>
                              , forming a phosphatase-substrate complex (population
                              <span class="inline-formula" id="inline-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ), and catalyzing the dephosphorylation / release of the substrate with rate ρ
                              <sub>
                               r
                              </sub>
                              . These reactions also can occur in reverse: kinase-substrate unbinding (rate κ
                              <sub>
                               u
                              </sub>
                              ), reverse kinase catalysis (rate κ
                              <sub>
                               −r
                              </sub>
                              ), phosphatase-substrate unbinding (rate ρ
                              <sub>
                               u
                              </sub>
                              ) and reverse phosphatase catalysis (rate ρ
                              <sub>
                               −r
                              </sub>
                              ). Under physiological conditions some of these reverse rates may be negligible compared to their forward counterparts, but accounting for them is crucial to enforce thermodynamic consistency. In fact the product of the ratios of the reverse rates relative to the forward ones must satisfy a key thermodynamic relation arising from the principle of local detailed balance (closely related to the Haldane relation for enzymes) [
                              <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                               37
                              </a>
                              ,
                              <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                               38
                              </a>
                              ],
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-14">
                              This relation is derived in the Supplementary Information (SI), and reflects the fact that for every complete traversal of the loop along the forward direction (clockwise along the black arrows in
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Fig. 1B
                              </a>
                              ) a single ATP molecule is removed from the environment, hydrolyzed, and the products ADP and inorganic phosphate P
                              <sub>
                               i
                              </sub>
                              released back into the surroundings. Δμ depends on the concentrations [ATP], [ADP], and [P
                              <sub>
                               i
                              </sub>
                              ] through
                              <span class="inline-formula" id="inline-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-6.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-6.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , where
                              <span class="inline-formula" id="inline-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the standard free energy of ATP hydrolysis (
                              <span class="inline-formula" id="inline-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                              at room temperature [
                              <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">
                               24
                              </a>
                              ]). Living systems expend energetic resources to maintain an imbalance of [ATP] relative to [ADP] and [P
                              <sub>
                               i
                              </sub>
                              ], making Δμ in physiological conditions larger than
                              <span class="inline-formula" id="inline-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Despite the wide variety of metabolic pathways used to achieve this, measured Δμ values in organisms from E. coli to humans lie within a relatively narrow range, Δμ ≈ 21 − 29 k
                              <sub>
                               B
                              </sub>
                              T [
                              <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">
                               24
                              </a>
                              ]. This means reverse rates are sufficiently slow that the numerator in
                              <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">
                               Eq. (1)
                              </a>
                              is 9-12 orders of magnitude smaller than the denominator. One of the questions we tackle below is the significance of this disparity for transmitting information through the loop.
                             </p>
                             <p id="p-15">
                              To quantitatively measure this information transfer, it is useful to explicitly describe the network behavior in terms of transducing an input signal into an amplified output, with degradation of the signal due to the stochastic nature of the reactions that mediate this process. We take the time-dependent input X(t) = K(t) + S
                              <sub>
                               K
                              </sub>
                              (t) to be the population of active kinases (both free and substrate-bound), and the corresponding output signal
                              <span class="inline-formula" id="inline-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-10.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-10.gif"/>
                                </noscript>
                               </span>
                              </span>
                              as the population of phosphorylated substrates (free and phosphatase-bound). For any specific system, the input kinases would be activated through a particular upstream signaling network. Here, however, we are interested in a more general problem: what is the effectiveness of this loop in processing a variety of possible input signals, spanning different amplitudes and timescales. The simplest mechanism that allows us to tune the dynamical characteristics of the input is to imagine the kinases activated at a constant rate F and deactivated at a constant rate γ
                              <sub>
                               K
                              </sub>
                              . We focus on the long-time limit where a stationary state has been achieved, and so F allows us to regulate the amplitude of the input signal while γ
                              <sub>
                               K
                              </sub>
                              controls the autocorrelation time of the input fluctuations. While the analysis below could be done for other, system-specific models of the input, our choice allows us to explore a broad range of possible inputs to establish general bounds on information processing through the loop. With this input model, the reaction network model is fully specified. For a given set of parameters (drawn from distributions based on kinase/phosphatase biochemical information collected in enzymatic databases, as described below) we can derive analytical results for dynamical quantities using the linearized chemical Langevin approximation [
                              <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                               39
                              </a>
                              ]. As shown in the SI, this provides excellent agreement with the exact kinetic Monte Carlo [
                              <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                               40
                              </a>
                              ] simulation results in the parameter ranges of interest.
                             </p>
                             <p id="p-16">
                              In focusing on how X(t) is transduced to Y (t), we frame our analysis in terms of three properties of the system. The first is the autocorrelation time of the input,
                              <span class="inline-formula" id="inline-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-11.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-11.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , defined through
                              <span class="inline-formula" id="inline-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-12.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-12.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , where the bar denotes an average over an ensemble of trajectories in the stationary state and
                              <span class="inline-formula" id="inline-formula-13">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-13.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-13.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Note that instantaneous averages like
                              <span class="inline-formula" id="inline-formula-14">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-14.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-14.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-15">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-15.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-15.gif"/>
                                </noscript>
                               </span>
                              </span>
                              are independent of t in the stationary state.
                              <span class="inline-formula" id="inline-formula-16">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-16.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-16.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the characteristic timescale of the input fluctuations, and we will denote its inverse, γ
                              <sub>
                               x
                              </sub>
                              , as the effective “frequency” of the input. The second property is related to the mean rate at which phosphorylated substrates are produced through the catalytic reaction step,
                              <span class="inline-formula" id="inline-formula-17">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-17.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-17.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , relative to the mean total number of activated kinases
                              <span class="inline-formula" id="inline-formula-18">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-18.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-18.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . We define the gain parameter
                              <span class="inline-formula" id="inline-formula-19">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-19.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-19.gif"/>
                                </noscript>
                               </span>
                              </span>
                              as a measure of the production of output for a given input level. Both γ
                              <sub>
                               x
                              </sub>
                              and R
                              <sub>
                               0
                              </sub>
                              can be expressed, to a good approximation, in terms of the reaction rates as follows (see SI for derivation):
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <span class="inline-formula" id="inline-formula-20">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-20.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-20.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Here κ
                              <sub>
                               −
                              </sub>
                              ≡ κ
                              <sub>
                               u
                              </sub>
                              + κ
                              <sub>
                               r
                              </sub>
                              , ρ
                              <sub>
                               −
                              </sub>
                              ≡ρ
                              <sub>
                               u
                              </sub>
                              + ρ
                              <sub>
                               r
                              </sub>
                              . Note the dependence on mean unmodified substrate
                              <span class="inline-formula" id="inline-formula-21">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-21.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-21.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and free phosphatase
                              <span class="inline-formula" id="inline-formula-22">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-22.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-22.gif"/>
                                </noscript>
                               </span>
                              </span>
                              populations: these two numbers are free parameters that (along with the reaction rates) determine the network dynamics.
                             </p>
                             <p id="p-17">
                              The final property of interest is the MI between input and output. Unless otherwise noted, we focus on the instantaneous stationary MI between X(t) and Y (t) at the same time point t, which we denote as ℐ. However in Sec. III C we also consider the MI between X(t) and Y (t+α), which we will denote with a subscript, ℐ
                              <sub>
                               α
                              </sub>
                              , where α is the time offset. This allows us to investigate whether including a time delay α > 0 (compensating for the finite propagation time of the signal) substantially changes the observed input-output correlation. For the instantaneous (α = 0) case, the MI is defined in terms of the joint probability P (X, Y) of observing input value X and output value Y at the same moment of time, and the corresponding marginal probabilities P (X) and P (Y),
                              <span class="disp-formula" id="disp-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-18">
                              The value of ℐ is non-negative in all cases, and is measured in bits, with larger values translating to a greater degree of correlation between input and output. For our parameter ranges, P (X, Y) can be approximated as a bivariate Gaussian, and so we use an expression for ℐ valid in this limit that is more convenient to evaluate [
                              <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">
                               19
                              </a>
                              ]:
                              <span class="disp-formula" id="disp-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-19">
                              Here E = 1 − ρ
                              <sup>
                               2
                              </sup>
                              , where ρ is the Pearson correlation coefficient, and hence lies in the range 0 ≤ E ≤ 1. For E = 0 (or equivalently ℐ = ∞) we have perfect correlation between the input and output signal, while E = 1 (ℐ = 0) corresponds to an output that is completely independent of the input. For the α ≠ 0 case, the expressions for ℐ
                              <sub>
                               α
                              </sub>
                              would be analogous, except we use P
                              <sub>
                               α
                              </sub>
                              (X, Y), the joint probability of observing X at time t and Y at time t + α. All averages, for example the cross-correlation
                              <span class="inline-formula" id="inline-formula-23">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-23.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-23.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , are then calculated with respect to P
                              <sub>
                               α
                              </sub>
                              (X,Y).
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              B. Determining the enzymatic parameter range
                             </h3>
                             <p id="p-20">
                              Once the input signal is specified through F and γ
                              <sub>
                               K
                              </sub>
                              , there are ten parameters related to the kinase, phosphatase, and substrate that determine the observables of interest γ
                              <sub>
                               x
                              </sub>
                              , R
                              <sub>
                               0
                              </sub>
                              , and I discussed above. These parameters are:
                              <span class="inline-formula" id="inline-formula-24">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-24.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-24.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . We know from surveys of enzymatic parameters that each of these quantities can span several orders of magnitude among different systems, often with an approximately log-normal distribution [
                              <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                               41
                              </a>
                              ,
                              <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                               42
                              </a>
                              ]. To understand the performance limits of enzymatic loops in general, it makes sense to explore the entire range of biologically realistic parameters, rather than focus on a single choice of parameters. Existing online databases are excellent resources for this purpose, and
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              shows the resulting histograms of kinase / phosphatase parameters (full extraction details are available in the SI). For the substrate protein (which we take as a kinase) and the phosphatase, the concentrations [S] and [P] in
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig. 2A
                              </a>
                              are derived from the PaxDb protein abundance database [
                              <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                               43
                              </a>
                              ], using UnitProt gene ontology associations to identify kinases and phosphatases [
                              <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                               44
                              </a>
                              ]. Enzymatic reaction parameters are available in the Sabio-RK database [
                              <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                               45
                              </a>
                              ]. The reaction rates κ
                              <sub>
                               r
                              </sub>
                              and ρ
                              <sub>
                               r
                              </sub>
                              (
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Fig. 2D
                              </a>
                              ) are typically listed directly, but the others are most often in specific combinations: the Michaelis constants
                              <span class="inline-formula" id="inline-formula-25">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-25.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-25.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for kinase/phosphatase respectively (
                              <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                               Fig. 2B
                              </a>
                              ) and the specificity ratios
                              <span class="inline-formula" id="inline-formula-26">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-26.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-26.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (
                              <a id="xref-fig-2-5" class="xref-fig" href="#F2">
                               Fig. 2C
                              </a>
                              ). For all of these parameters there is a paucity of data on phosphatases relative to kinases, but the phosphatase ranges seem to largely overlap with those of kinases. Thus for simplicity we take kinase and phosphatase parameters to have the same distributions (log-normal) and use a numerical fitting procedure to find an overall log-normal joint probability distribution for the eight underlying model parameters represented in the data:
                              <span class="inline-formula" id="inline-formula-27">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-27.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-27.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (see SI). Note that data in concentration units (like [S] and [P] in molars) is converted to mean abundance (
                              <span class="inline-formula" id="inline-formula-28">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-28.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-28.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-29">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-29.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-29.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ) by assuming a volume of 30 fL (comparable to the cytoplasmic volume of yeast [
                              <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">
                               24
                              </a>
                              ,
                              <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                               46
                              </a>
                              ]). This procedure is designed so that the resulting joint distribution yields marginal probability densities (solid curves in
                              <a id="xref-fig-2-6" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              ) that exhibit good agreement with the histogram data for any of the measured parameter combinations. Despite this agreement, we note that the joint distribution likely spans a portion of the parameter space larger than the true distribution of biological values: this is because it cannot fully capture correlations between different parameters. (Such correlations are difficult to reconstruct since many database entries are incomplete, containing some but not all of the enzymatic parameters.) For our purposes, having a distribution that effectively acts like a superset of the biological distribution is fine: whatever performance bounds we infer from the whole distribution will then also apply to the subset of the distribution that corresponds to current real-world systems. Moreover this also allows us to explore a larger enzymatic design space, which may have been accessible at earlier points in evolutionary history.
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F2.large.jpg?width=800&height=600&carousel=1" title="Enzymatic parameter ranges for kinases/phosphatases based on the PaxDb [43] and Sabio-RK [45] databases. Because of the relative lack of phosphatase data (orange histograms) relative to kinases (blue histograms), we fit an overall log-normal joint probability to the total data set including both kinases and phosphatases. The marginal distributions from that global fit are plotted as purple curves. The parameters are as follows: (A) kinase substrate [S] and phosphatase [P] concentrations; (B) kinase/phosphatase Michaelis constants; (C) the corresponding specificity ratios ; (D) kinase/phosphatase catalytic rates ?r and ?r." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383868837" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Enzymatic parameter ranges for kinases/phosphatases based on the PaxDb [43] and Sabio-RK [45] databases. Because of the relative lack of phosphatase data (orange histograms) relative to kinases (blue histograms), we fit an overall log-normal joint probability to the total data set including both kinases and phosphatases. The marginal distributions from that global fit are plotted as purple curves. The parameters are as follows: <strong>(A)</strong> kinase substrate [<em>S</em>] and phosphatase [<em>P</em>] concentrations; <strong>(B)</strong> kinase/phosphatase Michaelis constants; <strong>(C)</strong> the corresponding specificity ratios ; <strong>(D)</strong> kinase/phosphatase catalytic rates <em>?</em><sub><em>r</em></sub> and <em>?</em><sub><em>r</em></sub>.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="FIG. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F2.medium.gif" width="440" height="342"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="FIG. 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F2.medium.gif" width="440" height="342"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                FIG. 2.
                               </span>
                               <p id="p-21" class="first-child">
                                Enzymatic parameter ranges for kinases/phosphatases based on the PaxDb [
                                <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">
                                 43
                                </a>
                                ] and Sabio-RK [
                                <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">
                                 45
                                </a>
                                ] databases. Because of the relative lack of phosphatase data (orange histograms) relative to kinases (blue histograms), we fit an overall log-normal joint probability to the total data set including both kinases and phosphatases. The marginal distributions from that global fit are plotted as purple curves. The parameters are as follows:
                                <strong>
                                 (A)
                                </strong>
                                kinase substrate [S] and phosphatase [P] concentrations;
                                <strong>
                                 (B)
                                </strong>
                                kinase/phosphatase Michaelis constants
                                <span class="inline-formula" id="inline-formula-30">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F2/embed/inline-graphic-30.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F2/embed/inline-graphic-30.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                ;
                                <strong>
                                 (C)
                                </strong>
                                the corresponding specificity ratios
                                <span class="inline-formula" id="inline-formula-31">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F2/embed/inline-graphic-31.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F2/embed/inline-graphic-31.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                ;
                                <strong>
                                 (D)
                                </strong>
                                kinase/phosphatase catalytic rates ?
                                <sub>
                                 r
                                </sub>
                                and ?
                                <sub>
                                 r
                                </sub>
                                .
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-22">
                              Two of the model parameters are still unaccounted for: the reverse reaction rates κ
                              <sub>
                               −r
                              </sub>
                              and ρ
                              <sub>
                               −r
                              </sub>
                              . Though usually small in magnitude and typically not measured in enzyme kinetic assays, we also know that they are crucially related to Δμ through the local detailed balance relation of
                              <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">
                               Eq. (1)
                              </a>
                              . Thus, as explained in the next section, these become important free parameters that we can vary to explore signaling efficiency and its dependence on Δμ.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-5">
                            <h2 class="">
                             III. RESULTS
                            </h2>
                            <div id="sec-6" class="subsection">
                             <h3>
                              A. Minimum cost of transmitting information
                             </h3>
                             <p id="p-23">
                              Given the model described above, with a parameter set drawn at random from the empirical joint distribution, we can ask a basic first question: what is the minimum chemical potential difference Δμ required to achieve a certain mutual information ℐ? The answer will depend on the nature of the input signal X(t), and thus we would like to test different effective input frequencies γ
                              <sub>
                               x
                              </sub>
                              . To do this we will fix the mean free kinase concentration at the level of a low amplitude input, [K] = 5 nM, and vary γ
                              <sub>
                               K
                              </sub>
                              , which varies γ
                              <sub>
                               x
                              </sub>
                              according to
                              <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">
                               Eq. (2)
                              </a>
                              with
                              <span class="inline-formula" id="inline-formula-32">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-32.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-32.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for fixed
                              <span class="inline-formula" id="inline-formula-33">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-33.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-33.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . In the SI we also show the same analysis for [K] = 0.5 and 50 nM, with results qualitatively similar to those described below. After drawing enzyme parameters from the joint distribution and specifying γ
                              <sub>
                               x
                              </sub>
                              at a given [K], the only two free parameters are the reverse reaction rates κ
                              <sub>
                               −r
                              </sub>
                              and ρ
                              <sub>
                               −r
                              </sub>
                              .
                             </p>
                             <p id="p-24">
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Fig. 3A
                              </a>
                              shows a contour diagram of ℐ as a function of κ
                              <sub>
                               −r
                              </sub>
                              and ρ
                              <sub>
                               −r
                              </sub>
                              for a sample enzyme parameter set and value of γ
                              <sub>
                               x
                              </sub>
                              . Superimposed are dotted lines of constant Δμ from
                              <a id="xref-disp-formula-1-3" class="xref-disp-formula" href="#disp-formula-1">
                               Eq. (1)
                              </a>
                              . If one were interested in achieving a particular ℐ value, for example ℐ = 1 bit, one can then numerically determine the κ
                              <sub>
                               −r
                              </sub>
                              and ρ
                              <sub>
                               −r
                              </sub>
                              point along the ℐ = 1 bit contour where Δμ is smallest. For this specific enzyme parameter set and γ
                              <sub>
                               x
                              </sub>
                              , the value turns out to be Δμ = 6.72 k
                              <sub>
                               B
                              </sub>
                              T, which would then be recorded as the minimum necessary Δμ to achieve 1 bit of MI. Note that it is not guaranteed that a minimum Δμ solution exists for every parameter set sampled from the joint distribution. If the ℐ contours plateau at a maximum less than 1 bit, no possible Δμ will allow that particular system to achieve the desired MI target. We will return to this important point below.
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F3.large.jpg?width=800&height=600&carousel=1" title="(A) A representative contour diagram of ℐ (solid curves) as a function of κ−r and ρ−r for a parameter set drawn randomly from the joint distribution. Dotted lines denote contours of constant Δμ. In this case Δμ = 6.72 kBT is the smallest value at which the system can achieve ℐ = 1 bit. (B) For a sample parameter set, the minimum Δμ needed to achieve ℐ = 1, 2 bits as a function of input frequency γx. For the 1 bit case, the dashed line represents , the maximum γx compatible with ℐ = 1 bit for this parameter set. As described in the text, we highlight two points along the curve: one at a frequency  at roughly 95% of the bandwidth, and the other at frequency  at roughly 1% of the bandwidth. The points will be plotted for a many random draws of the enzyme parameters from the joint distribution in the lower panels of the figue. (C) For each target value of ℐ= 1, 1.5, 2 bits, the percentage probability of randomly drawing a parameter set that has a  higher than a given frequency. (D-F) The distribution of  (blue) and  (green) for many random parameter draws, keeping only those that can achieve ℐ = 1 bit (D), 1.5 bits (E), or 2 bits (F). The probabilities of successfully drawing such a set are shown in red in each panel. The blue and green circles denote the median of each distribution respectively. (G-I) The same  distributions as in panels (D-F), except plotted in terms of gain R0 on the vertical axis. The solid line is the analytical maximum bandwidth bound  of Eq. (5). The purple circle in panel G shows the estimated result for the near-optimal yeast Pbs2/Hog1 system." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383868837" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>(A)</strong> A representative contour diagram of <em>ℐ</em> (solid curves) as a function of <em>κ</em><sub>−<em>r</em></sub> and <em>ρ</em><sub>−<em>r</em></sub> for a parameter set drawn randomly from the joint distribution. Dotted lines denote contours of constant Δ<em>μ</em>. In this case Δ<em>μ</em> = 6.72 <em>k</em><sub><em>B</em></sub><em>T</em> is the smallest value at which the system can achieve <em>ℐ</em> = 1 bit. <strong>(B)</strong> For a sample parameter set, the minimum Δ<em>μ</em> needed to achieve <em>ℐ</em> = 1, 2 bits as a function of input frequency <em>γ</em><sub><em>x</em></sub>. For the 1 bit case, the dashed line represents , the maximum <em>γ</em><sub><em>x</em></sub> compatible with ℐ = 1 bit for this parameter set. As described in the text, we highlight two points along the curve: one at a frequency  at roughly 95% of the bandwidth, and the other at frequency  at roughly 1% of the bandwidth. The points will be plotted for a many random draws of the enzyme parameters from the joint distribution in the lower panels of the figue. <strong>(C)</strong> For each target value of ℐ= 1, 1.5, 2 bits, the percentage probability of randomly drawing a parameter set that has a  higher than a given frequency. <strong>(D-F)</strong> The distribution of  (blue) and  (green) for many random parameter draws, keeping only those that can achieve <em>ℐ</em> = 1 bit (D), 1.5 bits (E), or 2 bits (F). The probabilities of successfully drawing such a set are shown in red in each panel. The blue and green circles denote the median of each distribution respectively. <strong>(G-I)</strong> The same  distributions as in panels (D-F), except plotted in terms of gain <em>R</em><sub>0</sub> on the vertical axis. The solid line is the analytical maximum bandwidth bound  of Eq. (5). The purple circle in panel G shows the estimated result for the near-optimal yeast Pbs2/Hog1 system.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="FIG. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F3.medium.gif" width="440" height="377"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="FIG. 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F3.medium.gif" width="440" height="377"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                FIG. 3.
                               </span>
                               <p id="p-25" class="first-child">
                                <strong>
                                 (A)
                                </strong>
                                A representative contour diagram of ℐ (solid curves) as a function of κ
                                <sub>
                                 −r
                                </sub>
                                and ρ
                                <sub>
                                 −r
                                </sub>
                                for a parameter set drawn randomly from the joint distribution. Dotted lines denote contours of constant Δμ. In this case Δμ = 6.72 k
                                <sub>
                                 B
                                </sub>
                                T is the smallest value at which the system can achieve ℐ = 1 bit.
                                <strong>
                                 (B)
                                </strong>
                                For a sample parameter set, the minimum Δμ needed to achieve ℐ = 1, 2 bits as a function of input frequency γ
                                <sub>
                                 x
                                </sub>
                                . For the 1 bit case, the dashed line represents
                                <span class="inline-formula" id="inline-formula-34">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F3/embed/inline-graphic-34.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F3/embed/inline-graphic-34.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , the maximum γ
                                <sub>
                                 x
                                </sub>
                                compatible with ℐ = 1 bit for this parameter set. As described in the text, we highlight two points along the curve: one at a frequency
                                <span class="inline-formula" id="inline-formula-35">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F3/embed/inline-graphic-35.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F3/embed/inline-graphic-35.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                at roughly 95% of the bandwidth, and the other at frequency
                                <span class="inline-formula" id="inline-formula-36">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F3/embed/inline-graphic-36.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F3/embed/inline-graphic-36.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                at roughly 1% of the bandwidth. The points will be plotted for a many random draws of the enzyme parameters from the joint distribution in the lower panels of the figue.
                                <strong>
                                 (C)
                                </strong>
                                For each target value of ℐ= 1, 1.5, 2 bits, the percentage probability of randomly drawing a parameter set that has a
                                <span class="inline-formula" id="inline-formula-37">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F3/embed/inline-graphic-37.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F3/embed/inline-graphic-37.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                higher than a given frequency.
                                <strong>
                                 (D-F)
                                </strong>
                                The distribution of
                                <span class="inline-formula" id="inline-formula-38">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F3/embed/inline-graphic-38.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F3/embed/inline-graphic-38.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                (blue) and
                                <span class="inline-formula" id="inline-formula-39">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F3/embed/inline-graphic-39.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F3/embed/inline-graphic-39.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                (green) for many random parameter draws, keeping only those that can achieve ℐ = 1 bit (D), 1.5 bits (E), or 2 bits (F). The probabilities of successfully drawing such a set are shown in red in each panel. The blue and green circles denote the median of each distribution respectively.
                                <strong>
                                 (G-I)
                                </strong>
                                The same
                                <span class="inline-formula" id="inline-formula-40">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F3/embed/inline-graphic-40.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F3/embed/inline-graphic-40.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                distributions as in panels (D-F), except plotted in terms of gain R
                                <sub>
                                 0
                                </sub>
                                on the vertical axis. The solid line is the analytical maximum bandwidth bound
                                <span class="inline-formula" id="inline-formula-41">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F3/embed/inline-graphic-41.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F3/embed/inline-graphic-41.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                of
                                <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">
                                 Eq. (5)
                                </a>
                                . The purple circle in panel G shows the estimated result for the near-optimal yeast Pbs2/Hog1 system.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-26">
                              If one keeps the enzyme parameters (other than κ
                              <sub>
                               −r
                              </sub>
                              and ρ
                              <sub>
                               −r
                              </sub>
                              ) fixed, and just varies γ
                              <sub>
                               x
                              </sub>
                              , an interesting trend appears in the minimum Δμ results.
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Fig. 3B
                              </a>
                              shows two examples of minimum Δμ curves, for target ℐ values of 1 and 2 bits respectively. For a given ℐ target, the minimum Δμ is nearly constant at low input frequencies, but then increases rapidly and diverges at a maximum frequency which we will dub the “bandwidth” of the system. This intuitively makes sense: the higher the input frequency, the more rapid the catalytic reaction rates needed to accurately transmit the signal through the system, increasing the required Δμ threshold. However, there is an inherent limit, given finite enzyme catalysis rates. Above the bandwidth, whose value depends on the enzyme parameters, the system can no longer achieve the target ℐ. The higher the informational burden (i.e. increasing the target ℐ from 1 to 2 bits) the lower the bandwidth: if one desires higher fidelity transmission, the range of transmissible signal frequencies will suffer.
                             </p>
                             <p id="p-27">
                              To make more sense of these results, it is useful to look at a broad sample of enzyme parameters rather than a single set. To visualize global behaviors, we will calculate two numerical results for each set drawn from our joint distribution. The procedure is as follows: i) Sample an enzyme parameter set from the distribution; ii) Determine if it can achieve our target ℐ for any input frequency; iii) If the answer is yes, vary γ
                              <sub>
                               K
                              </sub>
                              until one finds the maximum possible value
                              <span class="inline-formula" id="inline-formula-42">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-42.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-42.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where one can still achieve the ℐ target. iv) Calculate the minimum Δμ for an input signal very near the bandwidth frequency, where
                              <span class="inline-formula" id="inline-formula-43">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-43.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-43.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . We will call this result Δμ
                              <sup>
                               high
                              </sup>
                              . The corresponding input frequency is
                              <span class="inline-formula" id="inline-formula-44">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-44.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-44.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . v) Analogously, calculate the minimum Δμ for an input signal with a frequency much lower than the bandwidth, where
                              <span class="inline-formula" id="inline-formula-45">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-45.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-45.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . This set of results we denote as Δμ
                              <sup>
                               low
                              </sup>
                              and
                              <span class="inline-formula" id="inline-formula-46">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-46.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-46.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Fig. 3B
                              </a>
                              shows the two points
                              <span class="inline-formula" id="inline-formula-47">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-47.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-47.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-48">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-48.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-48.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ) as blue and green dots respectively for that particular parameter set at ℐ = 1 bit. These two points encapsulate several key features of the minimum Δμ versus γ
                              <sub>
                               x
                              </sub>
                              curve: Δμ
                              <sup>
                               low
                              </sup>
                              roughly corresponds to an “entry level” price, the minimum ATP hydrolysis chemical potential necessary to transmit the signal at any frequency, while the difference Δμ
                              <sup>
                               high
                              </sup>
                              − Δμ
                              <sup>
                               low
                              </sup>
                              is the premium one has to pay to transmit signals near the highest possible frequencies. The value
                              <span class="inline-formula" id="inline-formula-49">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-49.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-49.gif"/>
                                </noscript>
                               </span>
                              </span>
                              approximately corresponds to the bandwidth.
                             </p>
                             <p id="p-28">
                              If one were to make numerous draws from the parameter distribution, and plot
                              <span class="inline-formula" id="inline-formula-50">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-50.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-50.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-51">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-51.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-51.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ) for each draw, one would get a cloud of blue and green dots. These are shown in
                              <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                               Fig. 3D-F
                              </a>
                              for target ℐ of 1, 1.5, and 2 bits respectively. As mentioned above, not every draw will lead to a parameter set that can achieve the target, and the plots are labeled by the fraction of draws that are capable of reaching that particular value of ℐ. That fraction decreases with ℐ, from 13% for ℐ = 1 bit down to only 2% for ℐ = 2 bits. As ℐ increases not only does it become progressively more difficult to find enzymatic parameters compatible with higher fidelity, but the accessible frequency range becomes more restricted.
                              <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                               Fig. 3C
                              </a>
                              shows the percentage of the parameter space that can achieve bandwidths higher than a given frequency for different ℐ. For example let us consider the frequency 1.22 × 10
                              <sup>
                               −3
                              </sup>
                              s
                              <sup>
                               −1
                              </sup>
                              , which is the ℐ = 1 bit bandwidth for the yeast Pbs2/Hog1 system described in detail in Sec. IIIC. (This system is part of the Hog1 osmotic stress response pathway, whose overall bandwidth has been experimentally estimated to be of a similar scale [
                              <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                               47
                              </a>
                              ]). From
                              <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                               Fig. 3C
                              </a>
                              it is evident that only about 0.41% of the draws from the parameter distribution have
                              <span class="inline-formula" id="inline-formula-52">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-52.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-52.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for a target ℐ = 1 bit. If one were to attempt to transmit signals at such high frequencies for ℐ = 2 bits, the fraction of compatible parameter space shrinks to a miniscule 9 × 10
                              <sup>
                               −3
                              </sup>
                              %. This reflects the exquisite fine-tuning required to put together a set of enzymatic loops capable of responding to quick, life-or-death variations of the external environment on time scales of a couple of minutes. Going much beyond ℐ = 1 bit and maintaining fast response times for a single push-pull loop is extremely difficult, and hence it makes sense that biology settles for ℐ in the vicinity of 1 bit in many circumstances.
                             </p>
                             <p id="p-29">
                              Going much below 1 bit poses another set of difficulties, since such systems would not even be able to reliably transmit the difference between high and low values of input signal. For signaling that can occur over longer timescales (hours instead of minutes) it becomes much easier to find compatible parameter sets, with the median of the distribution of
                              <span class="inline-formula" id="inline-formula-53">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-53.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-53.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for ℐ = 1 bit around ∼ 6 × 10
                              <sup>
                               −5
                              </sup>
                              s
                              <sup>
                               −1
                              </sup>
                              .
                             </p>
                             <p id="p-30">
                              From the perspective of costs, the bulk of the distribution of entry level prices Δμ
                              <sup>
                               low
                              </sup>
                              for ℐ = 1 bit is ≳1 k
                              <sub>
                               B
                              </sub>
                              T. Any system much below this would be too close to equilibrium (reverse rates comparable to forward rates) for effective information transfer to occur. The median of the Δμ
                              <sup>
                               low
                              </sup>
                              distribution in
                              <a id="xref-fig-3-7" class="xref-fig" href="#F3">
                               Fig. 3C
                              </a>
                              is 4 k
                              <sub>
                               B
                              </sub>
                              T, increasing to about 6 k
                              <sub>
                               B
                              </sub>
                              T for ℐ = 2 bits in
                              <a id="xref-fig-3-8" class="xref-fig" href="#F3">
                               Fig. 3E
                              </a>
                              . These values are on the same scale as estimates of minimum Δμ ∼ 4 k
                              <sub>
                               B
                              </sub>
                              T ln 2 required for 99% accurate readout of a ligand-bound receptor via the activation of a downstream molecule, assuming an arbitrarily slow readout process [
                              <a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ]. In that system (as in ours), processing information at faster time scales requires large Δμ. Indeed we find that the median values for Δμ
                              <sup>
                               high
                              </sup>
                              range between 8 − 10 k
                              <sub>
                               B
                              </sub>
                              T for ℐ = 1 − 2 bits. The minimum Δμ near the bandwidth is typically shifted up by about 4 k
                              <sub>
                               B
                              </sub>
                              T, reflecting the premium necessary to transmit near the frequency limit. Paying this premium is worthwhile: frequencies
                              <span class="inline-formula" id="inline-formula-54">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-54.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-54.gif"/>
                                </noscript>
                               </span>
                              </span>
                              accessible at Δμ
                              <sup>
                               low
                              </sup>
                              prices are likely far too low to have biological relevance, with the distributions of
                              <span class="inline-formula" id="inline-formula-55">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-55.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-55.gif"/>
                                </noscript>
                               </span>
                              </span>
                              largely below 10
                              <sup>
                               −5
                              </sup>
                              s
                              <sup>
                               −1
                              </sup>
                              . To get the ability to respond to signals at more biologically reasonable time scales thus means being capable of transmitting closer to the bandwidth, making Δμ
                              <sup>
                               high
                              </sup>
                              a more useful measure of minimum biological costs.
                             </p>
                             <p id="p-31">
                              The Δμ
                              <sup>
                               high
                              </sup>
                              distributions show that it is possible to have signaling systems that transmit at least 1 bit of MI and operate at Δμ lower than the current physiological range (Δμ ≈ 21−29 k
                              <sub>
                               B
                              </sub>
                              T [
                              <a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">
                               24
                              </a>
                              ], indicated in pink in
                              <a id="xref-fig-3-9" class="xref-fig" href="#F3">
                               Fig. 3D-F
                              </a>
                              ). This is true even for systems with the fastest responses (large
                              <span class="inline-formula" id="inline-formula-56">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-56.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-56.gif"/>
                                </noscript>
                               </span>
                              </span>
                              near the right edges of the distribution). This means that one can imagine enzymatic signaling systems in the earliest stages of evolutionary history that can reliably distinguish high and low inputs even before ATP metabolism (maintaining high ATP concentrations relative to ADP and P
                              <sub>
                               i
                              </sub>
                              ) reached its modern levels of efficiency.
                             </p>
                             <p id="p-32">
                              In fact a fascinating universal feature of the distributions is that the physiological Δμ range lies just above the top edge of the distributions. Naively it would seem as if the physiological values are just high enough to allow these signaling loops to transmit ℐ = 1 – 2 bits across the broadest possible parameter subset. This gives evolution the largest possible space in which to tweak tradeoffs between fidelity and response times without running into chemical potential limitations. Of course Δμ influences not just signaling networks but the entire range of cellular functions, so it is impossible to say with certainty what factors played the largest role in determining the values of Δμ we see in present-day organisms. But at least from the perspective of signaling at the level of a push-pull loop, it is clear that Δμ ≈ 21 − 29 k
                              <sub>
                               B
                              </sub>
                              T is more than good enough for basic information transfer needs, and there would be no benefit in having a system with substantially higher Δμ. To maintain Δμ = 40 or 50 k
                              <sub>
                               B
                              </sub>
                              T for example, would require significant additional metabolic resources, with little payoff in terms of either ℐ or bandwidth.
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              B. Analytical bound describes tradeoff between bandwidth and information
                             </h3>
                             <p id="p-33">
                              The results above already illustrated the tradeoff between bandwidth and MI, with pa-rameter sets that achieve very large
                              <span class="inline-formula" id="inline-formula-57">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-57.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-57.gif"/>
                                </noscript>
                               </span>
                              </span>
                              becoming progressively harder to find as the target ℐ increases. Can we understand this relationship in more detail? For this purpose we take advantage of optimal noise filter theories, originally developed in the context of signal processing [
                              <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                               48
                              </a>
                              –
                              <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                               50
                              </a>
                              ], and in recent years applied to a variety of biological signaling networks [
                              <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ,
                              <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              ,
                              <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                               51
                              </a>
                              –
                              <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">
                               54
                              </a>
                              ]. The original motivation involved designing a filter for a signal corrupted by noise, such that the output matched the uncorrupted input signal as closely as possible. In the biological context, this same framework allows us to put bounds on the maximum MI achievable between input and output signals for given input and enzymatic parameters. As shown in the SI, our enzymatic push-pull loop can be approximately mapped onto an effective two-species input-output system, which is then amenable to analytical treatment using the Wiener-Kolmogorov optimal filter theory [
                              <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ,
                              <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">
                               48
                              </a>
                              –
                              <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">
                               50
                              </a>
                              ].
                             </p>
                             <p id="p-34">
                              The end result is a remarkably simple analytical relation between the maximum possible bandwidth
                              <span class="inline-formula" id="inline-formula-58">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-58.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-58.gif"/>
                                </noscript>
                               </span>
                              </span>
                              achievable given a target value of ℐ,
                              <span class="disp-formula" id="disp-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-35">
                              The only other enzymatic parameter that appears in the relation is the gain R
                              <sub>
                               0
                              </sub>
                              , a measure of output production relative to the input.
                              <a id="xref-fig-3-10" class="xref-fig" href="#F3">
                               Fig. 3G-I
                              </a>
                              shows the same parameter set distribution as the
                              <span class="inline-formula" id="inline-formula-59">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-59.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-59.gif"/>
                                </noscript>
                               </span>
                              </span>
                              points in
                              <a id="xref-fig-3-11" class="xref-fig" href="#F3">
                               Fig. 3D-F
                              </a>
                              , except replotted in terms of
                              <span class="inline-formula" id="inline-formula-60">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-60.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-60.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , where R
                              <sub>
                               0
                              </sub>
                              is the gain for each parameter set. The solid line is the bound of
                              <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">
                               Eq. (5)
                              </a>
                              . Even though this bound is based on an approximation of the full enzymatic system, and hence is not guaranteed to be exact, it still provides an excellent cutoff for the distribution of
                              <span class="inline-formula" id="inline-formula-61">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-61.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-61.gif"/>
                                </noscript>
                               </span>
                              </span>
                              points. For systems at a certain R
                              <sub>
                               0
                              </sub>
                              , we see that as ℐ is increased and the denominator in
                              <a id="xref-disp-formula-5-3" class="xref-disp-formula" href="#disp-formula-5">
                               Eq. (5)
                              </a>
                              gets larger, the maximum bandwidth
                              <span class="inline-formula" id="inline-formula-62">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-62.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-62.gif"/>
                                </noscript>
                               </span>
                              </span>
                              shifts to lower values. If we are interested in a fast response time, increasing ℐ systematically reduces the compatible parameter space, since we are forced to rely on cases with larger and larger R
                              <sub>
                               0
                              </sub>
                              . Thus
                              <a id="xref-disp-formula-5-4" class="xref-disp-formula" href="#disp-formula-5">
                               Eq. (5)
                              </a>
                              rationalizes the earlier observation of limited options for networks that can simultaneously respond to signals fluctuating on minute time scales and achieve ℐ significantly larger than 1 bit.
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              C. Time-delayed mutual information
                             </h3>
                             <p id="p-36">
                              Up to now we have focused on the instantaneous MI ℐ between input and output, which is constrained not just by noise in signal transduction, but also by the fact that changes in the input take a finite time to propagate into the output. We can compensate for the finite propagation speed by using the time-delayed MI ℐ
                              <sub>
                               α
                              </sub>
                              , which describes the correlation between the input X(t) and the output Y (t+ α) at a later time [
                              <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">
                               55
                              </a>
                              ,
                              <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">
                               56
                              </a>
                              ]. Indeed some of the gains in MI observed in experiments collecting outputs over multiple time points [
                              <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                              ,
                              <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                              ] reflect the fact that later output time points can have higher correlations with the current input.
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Fig. 4A
                              </a>
                              shows the typical behavior of I
                              <sub>
                               α
                              </sub>
                              versus α, for a parameter set that achieves instantaneous MI ℐ
                              <sub>
                               0
                              </sub>
                              = 1 bit at α = 0, with
                              <span class="inline-formula" id="inline-formula-63">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-63.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-63.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . The MI reaches a maximum value
                              <span class="inline-formula" id="inline-formula-64">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-64.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-64.gif"/>
                                </noscript>
                               </span>
                              </span>
                              bits at time delay α
                              <sub>
                               m
                              </sub>
                              > 0, which is usually a fraction of the characteristic time scale
                              <span class="inline-formula" id="inline-formula-65">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-65.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-65.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . The results for the chemical Langevin approach (solid curve) are consistent with kinetic Monte Carlo simulations (circles). For completeness we also show the behavior of ℐ
                              <sub>
                               α
                              </sub>
                              for α < 0, which describes the extent to which the output can anticipate future input signals. As expected, ℐ
                              <sub>
                               α
                              </sub>
                              < ℐ
                              <sub>
                               0
                              </sub>
                              for α < 0, and ℐ
                              <sub>
                               α
                              </sub>
                              decreases as the time offset | α | becomes larger, reflecting the difficulty of forecasting the input as one looks further into the future.
                             </p>
                             <p id="p-37">
                              The approximate two species Wiener-Kolmogorov optimal filter theory described in the previous section easily generalizes to incorporate time delay [
                              <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">
                               54
                              </a>
                              ] (details in the SI). It yields an analytical bound I
                              <sup>
                               max
                              </sup>
                              (Λ) on the MI regardless of time offset α:
                              <span class="disp-formula" id="disp-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F4.large.jpg?width=800&height=600&carousel=1" title="(A) ℐα, the MI between input and output with a time offset α, for a sample parameter set where the system achieves instantaneous MI ℐ0 = 1 bit. The maximum  is achieved at a finite time delay αm > 0. Results from the chemical Langevin approximation (CL) agree with kinetic Monte Carlo simulations (KMC). The analytical bound ℐmax(Λ) from Eq. (6) is shown as a horizontal line, with Λ = 117 for this parameter set. (B) The three sets of points correspond to the parameter sets distributions used in Fig. 3G-I, with ℐ0 = 1 bit (blue), 1.5 bits (green), and 2 bits (red). For each parameter set, the maximum time-delayed  is plotted versus . The bound ℐmax (Λ), based on the approximate two species optimal filter theory, is shown as a black curve." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383868837" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>(A)</strong> <em>ℐ</em><sub><em>α</em></sub>, the MI between input and output with a time offset <em>α</em>, for a sample parameter set where the system achieves instantaneous MI <em>ℐ</em><sub>0</sub> = 1 bit. The maximum  is achieved at a finite time delay <em>α</em><sub><em>m</em></sub> <em>></em> 0. Results from the chemical Langevin approximation (CL) agree with kinetic Monte Carlo simulations (KMC). The analytical bound <em>ℐ</em><sup>max</sup>(Λ) from Eq. (6) is shown as a horizontal line, with Λ = 117 for this parameter set. <strong>(B)</strong> The three sets of points correspond to the parameter sets distributions used in Fig. 3G-I, with <em>ℐ</em><sub>0</sub> = 1 bit (blue), 1.5 bits (green), and 2 bits (red). For each parameter set, the maximum time-delayed  is plotted versus . The bound <em>ℐ</em><sup>max</sup> (Λ), based on the approximate two species optimal filter theory, is shown as a black curve.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="FIG. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F4.medium.gif" width="440" height="197"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="FIG. 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F4.medium.gif" width="440" height="197"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                FIG. 4.
                               </span>
                               <p id="p-38" class="first-child">
                                <strong>
                                 (A)
                                </strong>
                                ℐ
                                <sub>
                                 α
                                </sub>
                                , the MI between input and output with a time offset α, for a sample parameter set where the system achieves instantaneous MI ℐ
                                <sub>
                                 0
                                </sub>
                                = 1 bit. The maximum
                                <span class="inline-formula" id="inline-formula-66">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F4/embed/inline-graphic-66.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F4/embed/inline-graphic-66.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                is achieved at a finite time delay α
                                <sub>
                                 m
                                </sub>
                                > 0. Results from the chemical Langevin approximation (CL) agree with kinetic Monte Carlo simulations (KMC). The analytical bound ℐ
                                <sup>
                                 max
                                </sup>
                                (Λ) from
                                <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">
                                 Eq. (6)
                                </a>
                                is shown as a horizontal line, with Λ = 117 for this parameter set.
                                <strong>
                                 (B)
                                </strong>
                                The three sets of points correspond to the parameter sets distributions used in
                                <a id="xref-fig-3-12" class="xref-fig" href="#F3">
                                 Fig. 3G-I
                                </a>
                                , with ℐ
                                <sub>
                                 0
                                </sub>
                                = 1 bit (blue), 1.5 bits (green), and 2 bits (red). For each parameter set, the maximum time-delayed
                                <span class="inline-formula" id="inline-formula-67">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F4/embed/inline-graphic-67.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F4/embed/inline-graphic-67.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                is plotted versus
                                <span class="inline-formula" id="inline-formula-68">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F4/embed/inline-graphic-68.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F4/embed/inline-graphic-68.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                . The bound ℐ
                                <sup>
                                 max
                                </sup>
                                (Λ), based on the approximate two species optimal filter theory, is shown as a black curve.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-39">
                              The value I
                              <sup>
                               max
                              </sup>
                              (Λ) = 1.53 for Λ = 117 is shown as a horizontal line in
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Fig. 4A
                              </a>
                              , so for this parameter set the actual maximum
                              <span class="inline-formula" id="inline-formula-69">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-69.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-69.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is not near the bound. However there are systems that can achieve I
                              <sup>
                               max
                              </sup>
                              (Λ).
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               Fig. 4B
                              </a>
                              shows parameter sets from
                              <a id="xref-fig-3-13" class="xref-fig" href="#F3">
                               Fig. 3G
                              </a>
                              (blue), 3H (green), and 3I (red), corresponding to systems with instantaneous MI values of ℐ
                              <sub>
                               0
                              </sub>
                              = 1, 1.5, or 2 bits respectively. For each parameter set, the maximum time-delayed
                              <span class="inline-formula" id="inline-formula-70">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-70.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-70.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is plotted versus
                              <span class="inline-formula" id="inline-formula-71">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-71.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-71.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . The bound I
                              <sup>
                               max
                              </sup>
                              (Λ) is plotted as a solid curve, and there is a subset of systems with performance near the bound. As we noted earlier, the two species optimal filter theory is an approximation for the full enzymatic system, and hence the bound does not have to be exact. But it still provides a close estimate of the MI limits for the full system, with the largest violations of the bound only ∼ 4%. The benefits of time delay are particularly evident for parameter sets with lower ℐ
                              <sub>
                               0
                              </sub>
                              : some cases with ℐ
                              <sub>
                               0
                              </sub>
                              = 1 bit exhibit
                              <span class="inline-formula" id="inline-formula-72">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-72.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-72.gif"/>
                                </noscript>
                               </span>
                              </span>
                              approaching 1.5 bits.
                             </p>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              D. Optimality and the yeast Pbs2/Hog1 push-pull loop
                             </h3>
                             <p id="p-40">
                              Returning to the discussion of instantaneous MI, there is an alternative way of thinking about the R
                              <sub>
                               0
                              </sub>
                              versus
                              <span class="inline-formula" id="inline-formula-73">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-73.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-73.gif"/>
                                </noscript>
                               </span>
                              </span>
                              results in
                              <a id="xref-fig-3-14" class="xref-fig" href="#F3">
                               Fig. 3G-I
                              </a>
                              . Imagine a system working at
                              <span class="inline-formula" id="inline-formula-74">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-74.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-74.gif"/>
                                </noscript>
                               </span>
                              </span>
                              with a certain gain parameter R
                              <sub>
                               0
                              </sub>
                              and achieving a target value ℐ. Comparing other parameter sets with the same bandwidth
                              <span class="inline-formula" id="inline-formula-75">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-75.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-75.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and target ℐ (taking a vertical slice of one of the panels in
                              <a id="xref-fig-3-15" class="xref-fig" href="#F3">
                               Fig. 3G-I
                              </a>
                              ), they will have a variety of different R
                              <sub>
                               0
                              </sub>
                              values, but all of these will be bounded from below by the minimum value
                              <span class="disp-formula" id="disp-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-11.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-11.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-41">
                              When
                              <span class="inline-formula" id="inline-formula-76">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-76.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-76.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , the system sits on the optimality line of
                              <a id="xref-disp-formula-5-5" class="xref-disp-formula" href="#disp-formula-5">
                               Eq. (5)
                              </a>
                              , with
                              <span class="inline-formula" id="inline-formula-77">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-77.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-77.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <p id="p-42">
                              The discrepancy between R
                              <sub>
                               0
                              </sub>
                              and
                              <span class="inline-formula" id="inline-formula-78">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-78.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-78.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for a given system allows us to see how close the signaling behavior is to optimality. Let us take a concrete biological example: the Pbs2/Hog1 enzymatic push-pull loop from yeast, part of the Hog1 signaling pathway that allows the organism to respond to osmotic stress. As described in the SI, key parameters for this system can be estimated based on an earlier model [
                              <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">
                               46
                              </a>
                              ] fit to microfluidic experimental data where yeast was exposed to periodic salt shocks [
                              <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">
                               57
                              </a>
                              ]. The results for the bandwidth and gain for ℐ = 1 bit are:
                              <span class="inline-formula" id="inline-formula-79">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-79.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-79.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and R
                              <sub>
                               0
                              </sub>
                              = 0.0621 ± 0.0001 s
                              <sup>
                               −1
                              </sup>
                              , with the error bars reflecting uncertainties due to unknown parameters (where we used priors based on the log-normal distributions in
                              <a id="xref-fig-2-7" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              .) The scale of the predicted bandwidth
                              <span class="inline-formula" id="inline-formula-80">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-80.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-80.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is consistent with microfluidic estimates. Ref. [
                              <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">
                               47
                              </a>
                              ] found a steep dropoff in the mean amplitude of the Hog1 response to periodic step-like changes in external osmolyte concentrations when the frequencies of the changes increased from 10
                              <sup>
                               −3
                              </sup>
                              s
                              <sup>
                               −1
                              </sup>
                              to 10
                              <sup>
                               −2
                              </sup>
                              s
                              <sup>
                               −1
                              </sup>
                              . At frequencies beyond the dropoff the Hog1 output can no longer reproduce the osmolyte input at high fidelity. Though the form of the input in this case is different than in our model, and the experiment probes the entire pathway rather than just the Pbs2/Hog1 component, the similarity in scales to our
                              <span class="inline-formula" id="inline-formula-81">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-81.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-81.gif"/>
                                </noscript>
                               </span>
                              </span>
                              value suggests that the Pbs2/Hog1 system may play a major role in determining the bandwidth of the whole pathway (since the bandwidth of the whole is constrained by the bandwidths of the components).
                             </p>
                             <p id="p-43">
                              Intriguingly, the estimated gain R
                              <sub>
                               0
                              </sub>
                              is very close to the minimum possible value
                              <span class="inline-formula" id="inline-formula-82">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-82.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-82.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for signaling at the bandwidth
                              <span class="inline-formula" id="inline-formula-83">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-83.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-83.gif"/>
                                </noscript>
                               </span>
                              </span>
                              with ℐ = 1 bit, as seen in
                              <a id="xref-fig-3-16" class="xref-fig" href="#F3">
                               Fig. 3G
                              </a>
                              . Using
                              <a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">
                               Eq. (7)
                              </a>
                              , we find
                              <span class="inline-formula" id="inline-formula-84">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-84.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-84.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . This naturally leads to the question: is the fact that this system lies so close to optimality a coincidence, or are there reasons why natural selection might favor minimizing R
                              <sub>
                               0
                              </sub>
                              in this case? To answer this question, we first have to consider the relationship between gain and ATP consumption.
                             </p>
                            </div>
                            <div id="sec-10" class="subsection">
                             <h3>
                              E. Minimum ATP consumption to achieve a certain signaling fidelity and bandwidth
                             </h3>
                             <p id="p-44">
                              This bound on the gain parameter in
                              <a id="xref-disp-formula-7-2" class="xref-disp-formula" href="#disp-formula-7">
                               Eq. (7)
                              </a>
                              is directly related to the metabolic cost of signaling, since higher production of the output per given input level will generally require a higher rate of phosphorylation events. We can roughly quantify the average rate of phosphorylation: in the stationary state this is just the mean rate of the kinase-catalyzed reaction step,
                              <span class="inline-formula" id="inline-formula-85">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-85.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-85.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Assuming one ATP hydrolyzed per reaction, A is the mean rate at which ATP is consumed by the system, and is related to R
                              <sub>
                               0
                              </sub>
                              through
                              <span class="inline-formula" id="inline-formula-86">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-86.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-86.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , as shown in the SI. In the enzymatic parameter ranges we consider, κ
                              <sub>
                               r
                              </sub>
                              is typically much larger than R
                              <sub>
                               0
                              </sub>
                              , so we can approximate this relation as
                              <span class="inline-formula" id="inline-formula-87">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-87.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-87.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Using
                              <a id="xref-disp-formula-7-3" class="xref-disp-formula" href="#disp-formula-7">
                               Eq. (7)
                              </a>
                              we can then estimate the minimum possible ATP consumption rate given a target ℐ and bandwidth
                              <span class="inline-formula" id="inline-formula-88">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-88.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-88.gif"/>
                                </noscript>
                               </span>
                              </span>
                              :
                              <span class="disp-formula" id="disp-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-12.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-12.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-45">
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Fig. 5A
                              </a>
                              shows the same parameter set values as the
                              <span class="inline-formula" id="inline-formula-89">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-89.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-89.gif"/>
                                </noscript>
                               </span>
                              </span>
                              points in
                              <a id="xref-fig-3-17" class="xref-fig" href="#F3">
                               Fig. 3D
                              </a>
                              for ℐ = 1 bit, except plotted in terms of
                              <span class="inline-formula" id="inline-formula-90">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-90.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-90.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . The A values are exact, but the approximate relation of
                              <a id="xref-disp-formula-8-1" class="xref-disp-formula" href="#disp-formula-8">
                               Eq. (8)
                              </a>
                              provides an excellent lower bound on the distribution. Qualitatively, the individual elements of
                              <a id="xref-disp-formula-8-2" class="xref-disp-formula" href="#disp-formula-8">
                               Eq. (8)
                              </a>
                              all make intuitive sense. An increase in any of the con-stituent factors (the mean free input kinase population
                              <span class="inline-formula" id="inline-formula-91">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-91.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-91.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , the target information ℐ, the bandwidth
                              <span class="inline-formula" id="inline-formula-92">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-92.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-92.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ) puts greater demands on the signaling system, requiring more catalytic activity and hence faster ATP consumption. Note that the above results are easily generalized if the reaction step consumes more than one ATP: for example the effective model for yeast Pbs2/Hog1 discussed above involves phosphorylation at two sites, which would lead to expressions for A and A
                              <sub>
                               min
                              </sub>
                              getting a prefactor of two.
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F5.large.jpg?width=800&height=600&carousel=1" title="(A) The same  point distribution as in Fig. 3C for ℐ = 1 bit, except plotted in terms of ATP consumption rate A on the vertical axis. The solid line is the approximate lower bound Amin on ATP consumption given by Eq. (8). (B) This distribution replotted with selection coefficient |s| on the vertical axis. |s| quantifies the fitness cost associated with a system that achieves the target ℐ = 1 bit but is sub-optimal in ATP consumption, relative to an optimal variant where A = Amin. The value of |s| becomes evolutionarily significant when it is higher than a “drift threshold” , where Ne is the effective population of the organism (a measure of genetic diversity). The ranges of  for different classes of organisms are shown on the right [32, 58]. The vertical dotted line corresponds to the estimated  for the yeast Pbs2/Hog1 system." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383868837" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>(A)</strong> The same  point distribution as in Fig. 3C for <em>ℐ</em> = 1 bit, except plotted in terms of ATP consumption rate <em>A</em> on the vertical axis. The solid line is the approximate lower bound <em>A</em><sub>min</sub> on ATP consumption given by Eq. (8). <strong>(B)</strong> This distribution replotted with selection coefficient |<em>s</em>| on the vertical axis. |<em>s</em>| quantifies the fitness cost associated with a system that achieves the target <em>ℐ</em> = 1 bit but is sub-optimal in ATP consumption, relative to an optimal variant where <em>A</em> = <em>A</em><sub>min</sub>. The value of |<em>s</em>| becomes evolutionarily significant when it is higher than a “drift threshold” , where <em>N</em><sub><em>e</em></sub> is the effective population of the organism (a measure of genetic diversity). The ranges of  for different classes of organisms are shown on the right [32, 58]. The vertical dotted line corresponds to the estimated  for the yeast Pbs2/Hog1 system.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="FIG. 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F5.medium.gif" width="440" height="169"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="FIG. 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F5.medium.gif" width="440" height="169"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                FIG. 5.
                               </span>
                               <p id="p-46" class="first-child">
                                <strong>
                                 (A)
                                </strong>
                                The same
                                <span class="inline-formula" id="inline-formula-93">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F5/embed/inline-graphic-93.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F5/embed/inline-graphic-93.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                point distribution as in
                                <a id="xref-fig-3-18" class="xref-fig" href="#F3">
                                 Fig. 3C
                                </a>
                                for ℐ = 1 bit, except plotted in terms of ATP consumption rate A on the vertical axis. The solid line is the approximate lower bound A
                                <sub>
                                 min
                                </sub>
                                on ATP consumption given by
                                <a id="xref-disp-formula-8-3" class="xref-disp-formula" href="#disp-formula-8">
                                 Eq. (8)
                                </a>
                                .
                                <strong>
                                 (B)
                                </strong>
                                This distribution replotted with selection coefficient |s| on the vertical axis. |s| quantifies the fitness cost associated with a system that achieves the target ℐ = 1 bit but is sub-optimal in ATP consumption, relative to an optimal variant where A = A
                                <sub>
                                 min
                                </sub>
                                . The value of |s| becomes evolutionarily significant when it is higher than a “drift threshold”
                                <span class="inline-formula" id="inline-formula-94">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F5/embed/inline-graphic-94.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F5/embed/inline-graphic-94.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , where N
                                <sub>
                                 e
                                </sub>
                                is the effective population of the organism (a measure of genetic diversity). The ranges of
                                <span class="inline-formula" id="inline-formula-95">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F5/embed/inline-graphic-95.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F5/embed/inline-graphic-95.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                for different classes of organisms are shown on the right [
                                <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">
                                 32
                                </a>
                                ,
                                <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">
                                 58
                                </a>
                                ]. The vertical dotted line corresponds to the estimated
                                <span class="inline-formula" id="inline-formula-96">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F5/embed/inline-graphic-96.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F5/embed/inline-graphic-96.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                for the yeast Pbs2/Hog1 system.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-11" class="subsection">
                             <h3>
                              F. Evolutionary pressure on the metabolic costs of signaling
                             </h3>
                             <p id="p-47">
                              It is clear from
                              <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                               Fig. 5A
                              </a>
                              that for many parameter set choices the ATP consumption rate A is significantly larger than for a system near optimality (A ≈ A
                              <sub>
                               min
                              </sub>
                              ) given the same ℐ and
                              <span class="inline-formula" id="inline-formula-97">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-97.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-97.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Let us consider a specific scenario where the bandwidth
                              <span class="inline-formula" id="inline-formula-98">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-98.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-98.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and the target ℐ are sufficient for the biological function of the signaling i.e. there are rapidly diminishing fitness returns in going to higher bandwidth and signal fidelity. In this scenario a system with A > A
                              <sub>
                               min
                              </sub>
                              has no significant adaptive advantage over one with A ≈ A
                              <sub>
                               min
                              </sub>
                              , but instead incurs a fitness penalty because of the superfluous ATP consumption. Would there be evolutionary pressure on this sub-optimal system to move toward optimality?
                             </p>
                             <p id="p-48">
                              The answer to this question has practical ramifications, because it will allow us to predict whether we should expect to see natural enzymatic push-pull loops cluster around the optimality line (as we saw in the yeast Pbs2-Hog1 example). The alternative, in the absence of strong evolutionary pressure to optimize, is a wider dispersion, more similar to
                              <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                               Fig. 5A
                              </a>
                              where the points are drawn at random from the enzymatic parameter distribution. Note that this is a question that is directly amenable to future kinetic experiments: for systems where we can fully characterize the enzymatic parameters of the push-pull loop (for both the kinase and phosphatase), all the relevant quantities like
                              <span class="inline-formula" id="inline-formula-99">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-99.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-99.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , A, and I can be calculated.
                             </p>
                             <p id="p-49">
                              Naively one might expect evolution to always drive systems to optimality due to natural selection, but genetic drift can play a significant competing role, allowing sub-optimal variants to flourish and even fix in a population [
                              <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">
                               59
                              </a>
                              ]. To be specific, let us consider a unicellular organism that reproduces via binary fission, and two genetic variants of that organism that differ in the enzymatic parameters of a push-pull signaling loop: both variants achieve the same
                              <span class="inline-formula" id="inline-formula-100">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-100.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-100.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and ℐ, but one has A > A
                              <sub>
                               min
                              </sub>
                              and one has A = A
                              <sub>
                               min
                              </sub>
                              . Let us denote the relative fitness of the sub-optimal versus the optimal type as 1 + s, defining a selection coefficient s. In other words the sub-optimal variant will have on average 1 + s offspring relative to the optimal one during the generation time of the optimal type. In the scenario described above, where the extra production does not confer any adaptive advantage and only imposes a metabolic cost, we will have s < 0, because the superfluous ATP consumption will lead to slower growth.
                             </p>
                             <p id="p-50">
                              The magnitude of s determines the degree of selective pressure on the sub-optimal variant. The key quantity that sets the relevant scale for s is the effective population N
                              <sub>
                               e
                              </sub>
                              of the organism, the size of an idealized population that exhibits the same changes in genetic diversity per generation due to drift as the actual population [
                              <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">
                               58
                              </a>
                              ]. When s < 0 and |s| ≫
                              <span class="inline-formula" id="inline-formula-101">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-101.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-101.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , natural selection dominates drift, exponentially suppressing the probability of a sub-optimal mutant fixing in a population of optimal organisms. On the other hand if |s| ≪
                              <span class="inline-formula" id="inline-formula-102">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-102.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-102.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , drift dominates, and the fixation probability of sub-optimal mutants is roughly the same as for a neutral (s = 0) mutation [
                              <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">
                               60
                              </a>
                              ]. In this case it would be difficult to maintain optimality in a population over the long term. N
                              <sub>
                               e
                              </sub>
                              for organisms is typically smaller than their actual population in the wild, and varies by several orders of magnitude among different classes: for unicellular species it can be as high as ∼ 10
                              <sup>
                               9
                              </sup>
                              − 10
                              <sup>
                               10
                              </sup>
                              in bacteria down to ∼ 10
                              <sup>
                               6
                              </sup>
                              − 10
                              <sup>
                               8
                              </sup>
                              in single-celled eukaryotes [
                              <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">
                               32
                              </a>
                              ,
                              <a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">
                               58
                              </a>
                              ]. (It becomes even smaller among higher eukaryotes, going down to ∼ 10
                              <sup>
                               4
                              </sup>
                              in vertebrates.) The corresponding ranges for the “drift threshold”
                              <span class="inline-formula" id="inline-formula-103">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-103.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-103.gif"/>
                                </noscript>
                               </span>
                              </span>
                              [
                              <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">
                               32
                              </a>
                              ] are shown on the right in
                              <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                               Fig. 5B
                              </a>
                              .
                             </p>
                             <p id="p-51">
                              The question then becomes: how do we estimate s and how does it compare to the relevant
                              <span class="inline-formula" id="inline-formula-104">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-104.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-104.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for the class of interest? For the case where a variant imposes metabolic costs but no adaptive advantage, there is a very useful relation that posits s ∼ −δC
                              <sub>
                               T
                              </sub>
                              /C
                              <sub>
                               T
                              </sub>
                              [
                              <a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">
                               32
                              </a>
                              ,
                              <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">
                               61
                              </a>
                              ,
                              <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">
                               62
                              </a>
                              ]. Here C
                              <sub>
                               T
                              </sub>
                              is the total resting metabolic expenditure of an organism during a generation time, measured for example in units of P, where 1 P = one phosphate bond hydrolyzed (ATP or ATP equivalent consumed). δC
                              <sub>
                               T
                              </sub>
                              is the extra expenditure incurred by the more costly mutant. This relation has already been used to explore selective pressures in yeast [
                              <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">
                               62
                              </a>
                              ], unicellular prokaryotes and eukaryotes [
                              <a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">
                               32
                              </a>
                              ], and viral infections [
                              <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">
                               63
                              </a>
                              ]. It was recently derived from first principles through a general bioenergetic growth model [
                              <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">
                               33
                              </a>
                              ], where the relation was refined with a more accurate prefactor: s ≈ − ln(R
                              <sub>
                               b
                              </sub>
                              )δC
                              <sub>
                               T
                              </sub>
                              /C
                              <sub>
                               T
                              </sub>
                              . Here R
                              <sub>
                               b
                              </sub>
                              is the mean number of offspring per individual (i.e. R
                              <sub>
                               b
                              </sub>
                              = 2 for binary fission).
                             </p>
                             <p id="p-52">
                              The value of C
                              <sub>
                               T
                              </sub>
                              can be readily estimated for single-celled organisms, where it scales roughly with cell volume [
                              <a id="xref-ref-32-7" class="xref-bibr" href="#ref-32">
                               32
                              </a>
                              ,
                              <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">
                               33
                              </a>
                              ]. Given the 30 fL cell volume used in our calculations, and assuming a generation time (cell division time) t
                              <sub>
                               r
                              </sub>
                              = 1 hr, we find C
                              <sub>
                               T
                              </sub>
                              ≈ 7×10
                              <sup>
                               11
                              </sup>
                              P (see details in the SI), comparable in magnitude to experimental estimates for yeast [
                              <a id="xref-ref-32-8" class="xref-bibr" href="#ref-32">
                               32
                              </a>
                              ]. Since δC
                              <sub>
                               T
                              </sub>
                              reflects the extra ATP consumed by the costly mutant (with consumption rate A) versus the optimal variant (rate A
                              <sub>
                               min
                              </sub>
                              ) over one generation time, we can write δC
                              <sub>
                               T
                              </sub>
                              = (A − A
                              <sub>
                               min
                              </sub>
                              )t
                              <sub>
                               r
                              </sub>
                              . We can thus calculate s for all the near-bandwidth ℐ = 1 bit parameter sets represented in
                              <a id="xref-fig-5-5" class="xref-fig" href="#F5">
                               Fig. 5A
                              </a>
                              . The results for |s| versus
                              <span class="inline-formula" id="inline-formula-105">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-105.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-105.gif"/>
                                </noscript>
                               </span>
                              </span>
                              are plotted in
                              <a id="xref-fig-5-6" class="xref-fig" href="#F5">
                               Fig. 5B
                              </a>
                              . Because increased ATP consumption is required to achieve larger bandwidths (as seen in
                              <a id="xref-disp-formula-8-4" class="xref-disp-formula" href="#disp-formula-8">
                               Eq. (8)
                              </a>
                              ), the distribution of selective penalties |s| for being sub-optimal is pushed to larger values with greater
                              <span class="inline-formula" id="inline-formula-106">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-106.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-106.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . In other words, higher bandwidths make the energetic stakes more significant.
                             </p>
                             <p id="p-53">
                              We can now rationalize why the yeast Pbs2/Hog1 loop might be close to optimality. The bandwidth for that system (indicated by a vertical dashed line in
                              <a id="xref-fig-5-7" class="xref-fig" href="#F5">
                               Fig. 5B
                              </a>
                              ) is near the higher end of the spectrum. Suboptimal parameter values that achieve approximately the same bandwidth at ℐ = 1 bit span a range of |s| values between 10
                              <sup>
                               −8
                              </sup>
                              and 10
                              <sup>
                               −4
                              </sup>
                              . Given N
                              <sub>
                               e
                              </sub>
                              = 10
                              <sup>
                               6
                              </sup>
                              − 10
                              <sup>
                               8
                              </sup>
                              for single-celled eukaryotes [
                              <a id="xref-ref-32-9" class="xref-bibr" href="#ref-32">
                               32
                              </a>
                              ,
                              <a id="xref-ref-58-4" class="xref-bibr" href="#ref-58">
                               58
                              </a>
                              ], and estimates of N
                              <sub>
                               e
                              </sub>
                              ≈ 10
                              <sup>
                               7
                              </sup>
                              for wild yeast populations [
                              <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">
                               64
                              </a>
                              ], these suboptimal systems likely have |s| near or above the drift threshold
                              <span class="inline-formula" id="inline-formula-107">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-107.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-107.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Thus we would expect yeast to be under evolutionary pressure to optimize the energy expenditures associated with the enzymatic loop.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-12">
                            <h2 class="">
                             IV. DISCUSSION AND CONCLUSIONS
                            </h2>
                            <p id="p-54">
                             The kinase-phosphatase push-pull signaling network, which maintains a certain value of mutual information ℐ between input and output, incurs energetic costs in the form of ATP consumption. These costs have two related facets: (i) the free energy expenditure Δμ for each hydrolysis reaction, and (ii) the number of such reactions A per unit time. Achieving empirical values like ℐ = 1 − 2 bits requires satisfying both aspects of the cost. There is a minimal price in terms of Δμ to achieve any given ℐ, and this price increases if one demands either greater fidelity (larger ℐ) or the ability to process faster signals (larger γ
                             <sub>
                              x
                             </sub>
                             ). Modern cells are more than willing to pay this part of the price, with Δμ sufficiently high to meet the minimal requirements for any enzymatic parameter set that hits a target ℐ on the order of 1 bit. However, as the distributions in
                             <a id="xref-fig-3-19" class="xref-fig" href="#F3">
                              Fig. 3D-F
                             </a>
                             illustrate, there are certainly options for signaling systems that work at similar fidelities under conditions of smaller Δμ, the presumptive scenario earlier in evolutionary history. In all cases we require some degree of fine-tuning of enzymatic parameters: the higher the fidelity or frequency demands, the smaller the fraction of parameter space that satisfies them. This leaves vanishingly small room to achieve networks that operate at ℐ significantly larger than the known empirical range.
                            </p>
                            <p id="p-55">
                             For particular parameter combinations the system is optimal, exhibiting the maximum possible bandwidth (
                             <span class="inline-formula" id="inline-formula-108">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-108.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-108.gif"/>
                               </noscript>
                              </span>
                             </span>
                             of
                             <a id="xref-disp-formula-5-6" class="xref-disp-formula" href="#disp-formula-5">
                              Eq. (5)
                             </a>
                             ) with the minimal ATP consumption (A
                             <sub>
                              min
                             </sub>
                             of
                             <a id="xref-disp-formula-8-5" class="xref-disp-formula" href="#disp-formula-8">
                              Eq. (8)
                             </a>
                             ). These theoretical bounds can also be generalized to the case of time-delayed MI, giving the simple expression for the overall maximum MI shown in
                             <a id="xref-disp-formula-6-2" class="xref-disp-formula" href="#disp-formula-6">
                              Eq. (6)
                             </a>
                             . Is such optimality widely realized in nature? Analyzing the selective pressures due to superfluous ATP expenditures indicates that this is a worthwhile question to pursue. We have already identified one near-optimal candidate in the yeast Hog1 signaling pathway. Based on the results of the previous section, we predict that the best place to look for others is among signaling pathways with high bandwidths, for example ∼ 10
                             <sup>
                              −3
                             </sup>
                             − 10
                             <sup>
                              −2
                             </sup>
                             s
                             <sup>
                              −1
                             </sup>
                             at the extremes of the current biological distribution. Here the metabolic costs of being suboptimal would be significant for single-celled organisms.
                            </p>
                            <p id="p-56">
                             More broadly, strong selective pressure on the costs of running signaling networks in single-celled organisms is likely to be a widespread phenomenon. To give another example, the expenditure of running the chemotaxis machinery in E. coli has been estimated to be about ∼ 10
                             <sup>
                              7
                             </sup>
                             P per ∼ 1 hr cell cycle [
                             <a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ,
                             <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ]. Compared to a value of C
                             <sub>
                              T
                             </sub>
                             ≈ 2 × 10
                             <sup>
                              10
                             </sup>
                             P for E. coli [
                             <a id="xref-ref-32-10" class="xref-bibr" href="#ref-32">
                              32
                             </a>
                             ,
                             <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">
                              33
                             </a>
                             ], we get an |s| ∼ 10
                             <sup>
                              −4
                             </sup>
                             , which is definitely significant for a bacterial population. We have barely begun to understand the kinds of optimization that such selective pressure has induced. Our approach readily generalizes beyond the kinase-phosphatase system, setting the stage for exploring these issues in a much wider array of biochemical networks.
                            </p>
                           </div>
                           <div class="section" id="sec-13">
                            <h2 class="">
                             Data and code availability
                            </h2>
                            <p id="p-57">
                             The code for our analysis, along with the data used to generate the figures, is available at:
                             <a href="https://github.com/hincz-lab/cell-signaling">
                              https://github.com/hincz-lab/cell-signaling
                             </a>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-14">
                            <h2 class="">
                             Supplementary Information
                            </h2>
                            <div class="section" id="sec-15">
                             <h3>
                              I. DERIVATION OF THE LOCAL DETAILED BALANCE RELATION
                             </h3>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F6.large.jpg?width=800&height=600&carousel=1" title="The enzymatic push-pull loop from the perspective of an individual substrate molecule. The protein can exist in one of four states: unmodified substrate (σ), bound to kinase (σK), phos-phorylated (σ∗), and bound to phosphatase while phosphorylated . The forward (clockwise) transition rates between these states are indicated in black, while the reverse (counterclockwise) rates are in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383868837" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">The enzymatic push-pull loop from the perspective of an individual substrate molecule. The protein can exist in one of four states: unmodified substrate (<em>σ</em>), bound to kinase (<em>σ</em><sub><em>K</em></sub>), phos-phorylated (<em>σ</em><sup><em>∗</em></sup>), and bound to phosphatase while phosphorylated . The forward (clockwise) transition rates between these states are indicated in black, while the reverse (counterclockwise) rates are in red.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="FIG. S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F6.medium.gif" width="440" height="335"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="FIG. S1." src="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F6.medium.gif" width="440" height="335"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. S1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                FIG. S1.
                               </span>
                               <p id="p-59" class="first-child">
                                The enzymatic push-pull loop from the perspective of an individual substrate molecule. The protein can exist in one of four states: unmodified substrate (σ), bound to kinase (σ
                                <sub>
                                 K
                                </sub>
                                ), phos-phorylated (σ
                                <sup>
                                 ∗
                                </sup>
                                ), and bound to phosphatase while phosphorylated
                                <span class="inline-formula" id="inline-formula-109">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F6/embed/inline-graphic-109.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F6/embed/inline-graphic-109.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                . The forward (clockwise) transition rates between these states are indicated in black, while the reverse (counterclockwise) rates are in red.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-60">
                              To derive the local detailed balance relation of main text
                              <a id="xref-disp-formula-1-4" class="xref-disp-formula" href="#disp-formula-1">
                               Eq. (1)
                              </a>
                              , it is convenient to focus on the reactions from the perspective of an individual substrate molecule [
                              <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">
                               1
                              </a>
                              ]. A given molecule in our model can be in one of four states, indicated in
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Fig. S1
                              </a>
                              with corresponding forward and reverse transition rates. For example if the molecule is an unmodified sub-strate (state σ) it can transition to a kinase-bound substrate (state σ
                              <sub>
                               K
                              </sub>
                              ) with rate κ
                              <sub>
                               b
                              </sub>
                              [K], proportional to the surrounding concentration [K] of kinase molecules. It can revert from σ
                              <sub>
                               K
                              </sub>
                              to σ with rate κ
                              <sub>
                               u
                              </sub>
                              . The other transitions in
                              <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                               Fig. S1
                              </a>
                              are defined analogously, with forward rates colored black and reverse rates in red. Local detailed balance entails that product of reverse rates divided by the product of forward rates is equal to exp(βΔG), where ΔG is the free energy change of the system associated with a single forward traversal of the loop and β = (k
                              <sub>
                               B
                              </sub>
                              T)
                              <sup>
                               −1
                              </sup>
                              [
                              <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">
                               1
                              </a>
                              ]. Since after one loop from σ to σ the substrate is back in the same state (as well as the kinase and phosphatase), there is no contribution to ΔG from these molecules. However a single loop leads to the hydrolysis of a single molecule of ATP, so ΔG =−Δμ, as defined in the main text. Putting everything together, the local detailed balance relation reads
                              <span class="disp-formula" id="disp-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-15.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-15.gif"/>
                                </noscript>
                               </span>
                              </span>
                              yielding main text
                              <a id="xref-disp-formula-1-5" class="xref-disp-formula" href="#disp-formula-1">
                               Eq. (1)
                              </a>
                              .
                             </p>
                            </div>
                            <div class="section" id="sec-16">
                             <h3>
                              II. CHEMICAL LANGEVIN APPROACH FOR THE KINASE-PHOSPHATASE PUSH-PULL LOOP
                             </h3>
                             <p id="p-61">
                              In this section we derive the stationary state properties of the kinase-phosphatase push-pull loop via the chemical Langevin approximation. The derivation will follow analogously to Ref. [
                              <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">
                               2
                              </a>
                              ], except here the system is more complicated due to the inclusion of reverse enzymatic reactions. The end goal will be a method to estimate the instantaneous mutual information ℐ, given by main text
                              <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">
                               Eq. (4)
                              </a>
                              ,
                              <span class="disp-formula" id="disp-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-16.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-16.gif"/>
                                </noscript>
                               </span>
                              </span>
                              which requires evaluating the variances of the input and output, var
                              <span class="inline-formula" id="inline-formula-110">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-110.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-110.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , as well as the covariance cov
                              <span class="inline-formula" id="inline-formula-111">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-111.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-111.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . The quantity E here will be referred to as the “error” in signal propagation between input and output, and can be equivalently expressed as E = 1− ρ
                              <sup>
                               2
                              </sup>
                              , where ρ is the Pearson correlation coefficient between X and Y.
                             </p>
                             <div class="section" id="sec-17">
                              <h4>
                               A. Dynamical equations
                              </h4>
                              <p id="p-62">
                               Our starting point is the full system of reactions for the enzymatic push-pull loop,
                               <span class="disp-formula" id="disp-formula-11">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-17.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-17.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where ∅ represents the void (upstream deactivated kinase which does not enter into our model). The corresponding chemical Langevin equations [
                               <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">
                                3
                               </a>
                               ] are given by:
                               <span class="disp-formula" id="disp-formula-12">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-18.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-18.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where the last line ensures that the total populations of free or bound phosphatase
                               <span class="inline-formula" id="inline-formula-112">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-112.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-112.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and free or bound substrate in all its forms
                               <span class="inline-formula" id="inline-formula-113">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-113.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-113.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               remain constant. The noise terms
                               <span class="inline-formula" id="inline-formula-114">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-114.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-114.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where η
                               <sub>
                                i
                               </sub>
                               (t) are Gaussian noise functions with zero mean and correlations
                               <span class="inline-formula" id="inline-formula-115">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-115.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-115.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . The five noise terms are associated with reactions in the system, and the corresponding prefactors represent the sum of the mean production (forward) and deactivation/unbinding (backward) contributions to each reaction:
                               <span class="disp-formula" id="disp-formula-13">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-19.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-19.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-63">
                               As described in the next section, we will be linearizing
                               <a id="xref-disp-formula-12-1" class="xref-disp-formula" href="#disp-formula-12">
                                Eq. (S4)
                               </a>
                               , keeping terms up to first order in deviations from the stationary state values. In this linearized approach, the stationary state populations are given by:
                               <span class="disp-formula" id="disp-formula-14">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-20.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-20.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               with the following definitions:
                               <span class="disp-formula" id="disp-formula-15">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-21.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-21.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-64">
                               The input (total kinase) is X = K + S
                               <sub>
                                K
                               </sub>
                               and the output (total activated substrate) is
                               <span class="inline-formula" id="inline-formula-116">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-116.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-116.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , and hence
                               <a id="xref-disp-formula-14-1" class="xref-disp-formula" href="#disp-formula-14">
                                Eq. (S6)
                               </a>
                               can be used to calculate the stationary values
                               <span class="inline-formula" id="inline-formula-117">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-117.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-117.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-118">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-118.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-118.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               .
                              </p>
                             </div>
                             <div class="section" id="sec-18">
                              <h4>
                               B. Second moments
                              </h4>
                              <p id="p-65">
                               In order to calculate the variance and covariance of the input and output, we also need to know
                               <span class="inline-formula" id="inline-formula-119">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-119.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-119.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . To estimate these quantities, the first step is to switch variables in
                               <a id="xref-disp-formula-12-2" class="xref-disp-formula" href="#disp-formula-12">
                                Eq. (S4)
                               </a>
                               to focus on deviations from the stationary state values:
                               <span class="inline-formula" id="inline-formula-120">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-120.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-120.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . We can in turn rewrite these four variables in terms of the input and output deviations
                               <span class="inline-formula" id="inline-formula-121">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-121.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-121.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-122">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-122.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-122.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               :
                               <span class="disp-formula" id="disp-formula-16">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-22.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-22.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where we have introduced two additional auxiliary variables δX
                               <sub>
                                q
                               </sub>
                               and δY
                               <sub>
                                q
                               </sub>
                               . Plugging
                               <a id="xref-disp-formula-16-1" class="xref-disp-formula" href="#disp-formula-16">
                                Eq. (S8)
                               </a>
                               into
                               <a id="xref-disp-formula-12-3" class="xref-disp-formula" href="#disp-formula-12">
                                Eq. (S4)
                               </a>
                               , we simplify the system through linearization, ignoring any terms of second order or higher in the deviations. As demonstrated below in comparisons with kinetic Monte Carlo (KMC) simulations [
                               <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">
                                4
                               </a>
                               ] of the original system, this linearized chemical Langevin approximation works well for our parameter ranges. Finally, we Fourier transform the linearized
                               <a id="xref-disp-formula-12-4" class="xref-disp-formula" href="#disp-formula-12">
                                Eq. (S4)
                               </a>
                               , and the resulting system of equations takes the form
                               <span class="disp-formula" id="disp-formula-17">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-23.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-23.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="inline-formula" id="inline-formula-123">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-123.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-123.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               denotes the Fourier transform of quantity Q(t). The matrix M is given by:
                               <span class="disp-formula" id="disp-formula-18">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-24.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-24.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-66">
                               The Fourier-space system of equations
                               <a id="xref-disp-formula-17-1" class="xref-disp-formula" href="#disp-formula-17">
                                Eq. (S9)
                               </a>
                               -
                               <a id="xref-disp-formula-18-1" class="xref-disp-formula" href="#disp-formula-18">
                                (S10)
                               </a>
                               can be solved for
                               <span class="inline-formula" id="inline-formula-124">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-124.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-124.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-125">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-125.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-125.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . The expressions are complicated, but take the form of a linear combination of Fourier-space noise terms:
                               <span class="disp-formula" id="disp-formula-19">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-25.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-25.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="inline-formula" id="inline-formula-126">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-126.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-126.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-127">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-127.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-127.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               are some prefactors which can be expressed as rational functions of ω. The prefactors have the property
                               <span class="inline-formula" id="inline-formula-128">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-128.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-128.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . In Fourier space the correlations among the noise terms take the form
                               <span class="inline-formula" id="inline-formula-129">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-129.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-129.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . Hence we can calculate the input power spectral density (PSD) P
                               <sub>
                                X
                               </sub>
                               (ω), the output PSD P
                               <sub>
                                Y
                               </sub>
                               (ω) and the cross PSD P
                               <sub>
                                XY
                               </sub>
                               (ω), defined via
                               <span class="disp-formula" id="disp-formula-20">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-26.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-26.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-67">
                               Plugging
                               <a id="xref-disp-formula-19-1" class="xref-disp-formula" href="#disp-formula-19">
                                Eq. (S11)
                               </a>
                               into
                               <a id="xref-disp-formula-20-1" class="xref-disp-formula" href="#disp-formula-20">
                                Eq. (S12)
                               </a>
                               , we find expressions for the PSDs in terms of the prefactor functions:
                               <span class="disp-formula" id="disp-formula-21">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-27.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-27.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-68">
                               The final step is to calculate the second moments from integrals of the PSDs, using the inverse Fourier transform of
                               <a id="xref-disp-formula-21-1" class="xref-disp-formula" href="#disp-formula-21">
                                Eq. (S13)
                               </a>
                               evaluated at t = t
                               <sup>
                                i
                               </sup>
                               :
                               <span class="disp-formula" id="disp-formula-22">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-28.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-28.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-69">
                               Given the explicit expressions for the prefactor functions in
                               <a id="xref-disp-formula-21-2" class="xref-disp-formula" href="#disp-formula-21">
                                Eq. (S13)
                               </a>
                               (which are available as part of the Mathematica notebooks in the Github repository associated with the manuscript), one can numerically evaluate the integrals in
                               <a id="xref-disp-formula-22-1" class="xref-disp-formula" href="#disp-formula-22">
                                Eq. (S14)
                               </a>
                               to get the moments.
                              </p>
                             </div>
                             <div class="section" id="sec-19">
                              <h4>
                               C. Comparison to kinetic Monte Carlo simulations for mutual information
                              </h4>
                              <p id="p-70">
                               The chemical Langevin calculation of the second moments allows us to use
                               <a id="xref-disp-formula-10-1" class="xref-disp-formula" href="#disp-formula-10">
                                Eq. (S2)
                               </a>
                               to estimate the mutual information ℐ. We can then check whether this estimate is consistent with the results we would get from KMC simulations of the full system.
                               <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                                Fig. S2
                               </a>
                               shows this comparison for two sample parameter sets drawn from the enzymatic parameter distribution described in Sec. IV. Since we are interested in exploring the full range of chemical potentials Δμ, in each case we calculate ℐ varying the reverse-to-forward rate ratio κ
                               <sub>
                                −r
                               </sub>
                               /κ
                               <sub>
                                r
                               </sub>
                               , keeping all other parameters constant. Through main text
                               <a id="xref-disp-formula-1-6" class="xref-disp-formula" href="#disp-formula-1">
                                Eq. (1)
                               </a>
                               , increasing κ
                               <sub>
                                −r
                               </sub>
                               /κ
                               <sub>
                                r
                               </sub>
                               corresponds to decreasing the magnitude of Δμ. At very large Δμ (small κ
                               <sub>
                                −r
                               </sub>
                               /κ
                               <sub>
                                r
                               </sub>
                               ) the ℐ curves saturate at the maximum possible mutual information for that parameter set, while at small Δμ (large κ
                               <sub>
                                −r
                               </sub>
                               /κ
                               <sub>
                                r
                               </sub>
                               ) the mutual information approaches zero, the equilibrium limit. Across the whole range we see that the chemical Langevin theoretical prediction is in close agreement with the KMC results.
                              </p>
                             </div>
                            </div>
                            <div class="section" id="sec-20">
                             <h3>
                              III. CHARACTERISTIC FREQUENCY γ
                              <sub>
                               x
                              </sub>
                              , GAIN R
                              <sub>
                               0
                              </sub>
                              , AND THE CONDITIONS FOR WIENER-KOLMOGOROV NOISE FILTER OPTIMALITY
                             </h3>
                             <div class="section" id="sec-21">
                              <h4>
                               A. Deriving the γ
                               <sub>
                                x
                               </sub>
                               and R
                               <sub>
                                0
                               </sub>
                               expressions in main text
                               <a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">
                                Eq. (2)
                               </a>
                              </h4>
                              <p id="p-71">
                               Since the effective frequency γ
                               <sub>
                                x
                               </sub>
                               of the input and the gain R
                               <sub>
                                0
                               </sub>
                               play central roles in the analysis, having simple closed form approximations for them [main text
                               <a id="xref-disp-formula-2-3" class="xref-disp-formula" href="#disp-formula-2">
                                Eq. (2)
                               </a>
                               ] is useful. The original definitions of these two variables, as described in the main text, are as follows: (i) γ
                               <sub>
                                x
                               </sub>
                               is related to the autocorrelation of input fluctuations,
                               <span class="inline-formula" id="inline-formula-130">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-130.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-130.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               ; (ii)
                               <span class="inline-formula" id="inline-formula-131">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-131.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-131.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               measures output production for a given level of input. As demonstrated in the next section, both of these can be calculated from KMC simulations (at significant computational expense for each different set of parameters). Alternatively, the chemical Langevin approximation of SI Sec. II can be used to derive somewhat cumbersome analytical expressions.
                              </p>
                              <p id="p-72">
                               However the most convenient option is to take advantage of the meaning of γ
                               <sub>
                                x
                               </sub>
                               and R
                               <sub>
                                0
                               </sub>
                               in an effective, two-species description of the kinase-phosphatase reaction network. Imagine a system with an input species population X(t), output Y (t), and a simplified chemistry with only four reactions: production of input at rate F, deactivation of input at rate γ
                               <sub>
                                x
                               </sub>
                               X(t), production of output at rate R
                               <sub>
                                0
                               </sub>
                               X(t), and deactivation of output at rate γ
                               <sub>
                                y
                               </sub>
                               Y (t). In this two-species system the inverse input autocorrelation time is given by the deactivation rate parameter γ
                               <sub>
                                x
                               </sub>
                               , and the coefficient R
                               <sub>
                                0
                               </sub>
                               in the output production rate is also the gain parameter. To relate this simplified model to the full reaction network of SI Sec. II, we compare analogous quantities in the simplified and full schemes. For example, let us take the mean input population
                               <span class="inline-formula" id="inline-formula-132">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-132.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-132.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . In the simplified scheme this is given by
                               <span class="disp-formula" id="disp-formula-23">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-29.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-29.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-73">
                               In the full network
                               <span class="inline-formula" id="inline-formula-133">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-133.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-133.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               can be calculated from
                               <a id="xref-disp-formula-14-2" class="xref-disp-formula" href="#disp-formula-14">
                                Eq. (S6)
                               </a>
                               as
                               <span class="disp-formula" id="disp-formula-24">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-30.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-30.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where the C
                               <sub>
                                i
                               </sub>
                               are expressed in terms of full network parameters in
                               <a id="xref-disp-formula-15-1" class="xref-disp-formula" href="#disp-formula-15">
                                Eq. (S7)
                               </a>
                               . Comparing
                               <a id="xref-disp-formula-23-1" class="xref-disp-formula" href="#disp-formula-23">
                                Eqs. (S15)
                               </a>
                               and
                               <a id="xref-disp-formula-24-1" class="xref-disp-formula" href="#disp-formula-24">
                                (S16)
                               </a>
                               we see that γ
                               <sub>
                                x
                               </sub>
                               should be given by
                               <span class="disp-formula" id="disp-formula-25">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-31.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-31.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               which is the first expression in main text
                               <a id="xref-disp-formula-2-4" class="xref-disp-formula" href="#disp-formula-2">
                                Eq. (2)
                               </a>
                               . Similarly the mean production rate of the output in the simplified scheme is
                               <span class="inline-formula" id="inline-formula-134">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-134.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-134.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . In the full system the mean output production is the average rate at which new phosphorylated substrate is produced via catalysis by the kinase-substrate complex,
                               <span class="disp-formula" id="disp-formula-26">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-32.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-32.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where we have again used
                               <a id="xref-disp-formula-14-3" class="xref-disp-formula" href="#disp-formula-14">
                                Eqs. (S6)
                               </a>
                               -
                               <a id="xref-disp-formula-15-2" class="xref-disp-formula" href="#disp-formula-15">
                                (S7)
                               </a>
                               . Comparing
                               <a id="xref-disp-formula-26-1" class="xref-disp-formula" href="#disp-formula-26">
                                Eq. (S18)
                               </a>
                               to
                               <span class="inline-formula" id="inline-formula-135">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-135.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-135.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , we see that R
                               <sub>
                                0
                               </sub>
                               should correspond to
                               <span class="disp-formula" id="disp-formula-27">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-33.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-33.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               which is the second expression in main text
                               <a id="xref-disp-formula-2-5" class="xref-disp-formula" href="#disp-formula-2">
                                Eq. (2)
                               </a>
                               .
                              </p>
                              <div id="F7" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F7.large.jpg?width=800&height=600&carousel=1" title="The mutual information ℐ for the enzymatic push-pull loop as a function of the reverse-forward rate ratio . The predictions from the chemical Langevin approach (dashed line) are compared against the corresponding KMC simulation results (circles). The parameters sets are as follows (all units are s−1 except for the mean populations; molar units have been converted to populations by assuming a cell volume of 30 fL): (top) κb = 2.94 × 10−6, ρb = 3.68 × 10−7, κu = 1.58 × 10−2, ρu = 4.42 × 10−4, κr = 12.8, ρr = 1.34, ρ−r = 2.50 × 10−5, F = 2.49 × 10−3, γk = 2.68 × 10−5,  and  ; (bottom) κb = 2.32 × 10−5, ρb = 1.46 × 10−4, κu = 6.94 × 10−2, ρu = 5.48, κr = 0.994, ρr = 5.05 × 10−2, ρ−r = 2.06 × 10−8, F = 2.46 × 10−2, γk = 2.65 × 10−4,  and ." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383868837" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">The mutual information ℐ for the enzymatic push-pull loop as a function of the reverse-forward rate ratio . The predictions from the chemical Langevin approach (dashed line) are compared against the corresponding KMC simulation results (circles). The parameters sets are as follows (all units are s<sup>−1</sup> except for the mean populations; molar units have been converted to populations by assuming a cell volume of 30 fL): (top) <em>κ</em><sub><em>b</em></sub> = 2.94 <em>×</em> 10<sup>−6</sup>, <em>ρ</em><sub><em>b</em></sub> = 3.68 <em>×</em> 10<sup>−7</sup>, <em>κ</em><sub><em>u</em></sub> = 1.58 <em>×</em> 10<sup>−2</sup>, <em>ρ</em><sub><em>u</em></sub> = 4.42 <em>×</em> 10<sup>−4</sup>, <em>κ</em><sub><em>r</em></sub> = 12.8, <em>ρ</em><sub><em>r</em></sub> = 1.34, <em>ρ</em><sub>−<em>r</em></sub> = 2.50 <em>×</em> 10<sup>−5</sup>, <em>F</em> = 2.49 <em>×</em> 10<sup>−3</sup>, <em>γ</em><sub><em>k</em></sub> = 2.68 <em>×</em> 10<sup>−5</sup>,  and  ; (bottom) <em>κ</em><sub><em>b</em></sub> = 2.32 <em>×</em> 10<sup>−5</sup>, <em>ρ</em><sub><em>b</em></sub> = 1.46 <em>×</em> 10<sup>−4</sup>, <em>κ</em><sub><em>u</em></sub> = 6.94 <em>×</em> 10<sup>−2</sup>, <em>ρ</em><sub><em>u</em></sub> = 5.48, <em>κ</em><sub><em>r</em></sub> = 0.994, <em>ρ</em><sub><em>r</em></sub> = 5.05 <em>×</em> 10<sup>−2</sup>, <em>ρ</em><sub>−<em>r</em></sub> = 2.06 <em>×</em> 10<sup>−8</sup>, <em>F</em> = 2.46 <em>×</em> 10<sup>−2</sup>, <em>γ</em><sub><em>k</em></sub> = 2.65 <em>×</em> 10<sup>−4</sup>,  and .</div></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="FIG. S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F7.medium.gif" width="388" height="440"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="FIG. S2." src="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F7.medium.gif" width="388" height="440"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. S2." data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 FIG. S2.
                                </span>
                                <p id="p-74" class="first-child">
                                 The mutual information ℐ for the enzymatic push-pull loop as a function of the reverse-forward rate ratio
                                 <span class="inline-formula" id="inline-formula-136">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F7/embed/inline-graphic-136.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F7/embed/inline-graphic-136.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 . The predictions from the chemical Langevin approach (dashed line) are compared against the corresponding KMC simulation results (circles). The parameters sets are as follows (all units are s
                                 <sup>
                                  −1
                                 </sup>
                                 except for the mean populations; molar units have been converted to populations by assuming a cell volume of 30 fL): (top) κ
                                 <sub>
                                  b
                                 </sub>
                                 = 2.94 × 10
                                 <sup>
                                  −6
                                 </sup>
                                 , ρ
                                 <sub>
                                  b
                                 </sub>
                                 = 3.68 × 10
                                 <sup>
                                  −7
                                 </sup>
                                 , κ
                                 <sub>
                                  u
                                 </sub>
                                 = 1.58 × 10
                                 <sup>
                                  −2
                                 </sup>
                                 , ρ
                                 <sub>
                                  u
                                 </sub>
                                 = 4.42 × 10
                                 <sup>
                                  −4
                                 </sup>
                                 , κ
                                 <sub>
                                  r
                                 </sub>
                                 = 12.8, ρ
                                 <sub>
                                  r
                                 </sub>
                                 = 1.34, ρ
                                 <sub>
                                  −r
                                 </sub>
                                 = 2.50 × 10
                                 <sup>
                                  −5
                                 </sup>
                                 , F = 2.49 × 10
                                 <sup>
                                  −3
                                 </sup>
                                 , γ
                                 <sub>
                                  k
                                 </sub>
                                 = 2.68 × 10
                                 <sup>
                                  −5
                                 </sup>
                                 ,
                                 <span class="inline-formula" id="inline-formula-137">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F7/embed/inline-graphic-137.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F7/embed/inline-graphic-137.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 and
                                 <span class="inline-formula" id="inline-formula-138">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F7/embed/inline-graphic-138.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F7/embed/inline-graphic-138.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 ; (bottom) κ
                                 <sub>
                                  b
                                 </sub>
                                 = 2.32 × 10
                                 <sup>
                                  −5
                                 </sup>
                                 , ρ
                                 <sub>
                                  b
                                 </sub>
                                 = 1.46 × 10
                                 <sup>
                                  −4
                                 </sup>
                                 , κ
                                 <sub>
                                  u
                                 </sub>
                                 = 6.94 × 10
                                 <sup>
                                  −2
                                 </sup>
                                 , ρ
                                 <sub>
                                  u
                                 </sub>
                                 = 5.48, κ
                                 <sub>
                                  r
                                 </sub>
                                 = 0.994, ρ
                                 <sub>
                                  r
                                 </sub>
                                 = 5.05 × 10
                                 <sup>
                                  −2
                                 </sup>
                                 , ρ
                                 <sub>
                                  −r
                                 </sub>
                                 = 2.06 × 10
                                 <sup>
                                  −8
                                 </sup>
                                 , F = 2.46 × 10
                                 <sup>
                                  −2
                                 </sup>
                                 , γ
                                 <sub>
                                  k
                                 </sub>
                                 = 2.65 × 10
                                 <sup>
                                  −4
                                 </sup>
                                 ,
                                 <span class="inline-formula" id="inline-formula-139">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F7/embed/inline-graphic-139.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F7/embed/inline-graphic-139.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 and
                                 <span class="inline-formula" id="inline-formula-140">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F7/embed/inline-graphic-140.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/F7/embed/inline-graphic-140.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 .
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                             </div>
                             <div class="section" id="sec-22">
                              <h4>
                               B. Validation through kinetic Monte Carlo simulations
                              </h4>
                              <p id="p-75">
                               To verify that the expressions for γ
                               <sub>
                                x
                               </sub>
                               and R
                               <sub>
                                0
                               </sub>
                               derived above are good approximations, we ran KMC simulations for various parameter sets drawn at random from the enzymatic parameter distribution detailed in the SI Sec. IV. For each parameter set the simulation was run long enough after reaching the stationary state to collect sufficient statistics for both the mean population values and the input autocorrelation function. As described above, these allow us to calculate γ
                               <sub>
                                x
                               </sub>
                               and R
                               <sub>
                                0
                               </sub>
                               . The simulation results are compared against the approximation from
                               <a id="xref-disp-formula-25-1" class="xref-disp-formula" href="#disp-formula-25">
                                Eqs. (S17)
                               </a>
                               and
                               <a id="xref-disp-formula-27-1" class="xref-disp-formula" href="#disp-formula-27">
                                (S19)
                               </a>
                               in
                               <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                                Fig. S3
                               </a>
                               . The agreement is excellent for both quantities, across the entire range of γ
                               <sub>
                                x
                               </sub>
                               and R
                               <sub>
                                0
                               </sub>
                               values. Thus we can confidently use the simple analytical expressions of
                               <a id="xref-disp-formula-25-2" class="xref-disp-formula" href="#disp-formula-25">
                                Eqs. (S17)
                               </a>
                               and
                               <a id="xref-disp-formula-27-2" class="xref-disp-formula" href="#disp-formula-27">
                                (S19)
                               </a>
                               to predict γ
                               <sub>
                                x
                               </sub>
                               and R
                               <sub>
                                0
                               </sub>
                               for any given parameter set.
                              </p>
                             </div>
                             <div class="section" id="sec-23">
                              <h4>
                               C. Relating maximum bandwidth, minimum ATP consumption rate, and mutual information via Wiener-Kolmogorov optimal noise filter theory
                              </h4>
                              <p id="p-76">
                               One of the benefits of the approximate relation between the full system and the two-species model described in Sec. IIIA is that it allows us to use results from the two-species case to make predictions for the behavior of the kinase-phosphatase push-pull loop. The two-species model has been analyzed in detail in Refs. [
                               <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">
                                2
                               </a>
                               ,
                               <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">
                                5
                               </a>
                               ], where it was shown to be able to map onto a Wiener-Kolmogorov optimal noise filter. The error E from
                               <a id="xref-disp-formula-10-2" class="xref-disp-formula" href="#disp-formula-10">
                                Eq. (S2)
                               </a>
                               for the two-species case can be evaluated in closed form as [
                               <a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">
                                2
                               </a>
                               ]:
                               <span class="disp-formula" id="disp-formula-28">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-35.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-35.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-77">
                               It achieves its minimum value (hence maximizing the mutual information ℐ) when the following condition is fulfilled:
                               <span class="disp-formula" id="disp-formula-29">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-36.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-36.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where Λ = R
                               <sub>
                                0
                               </sub>
                               /γ
                               <sub>
                                x
                               </sub>
                               . The corresponding minimum E, where the system behaves like an optimal Wiener-Kolmogorov (WK) noise filter is given by:
                               <span class="disp-formula" id="disp-formula-30">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-37.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-37.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-78">
                               Interestingly, this remains the bound even if we generalize the output production term R
                               <sub>
                                0
                               </sub>
                               X(t) to be nonlinear in X(t) [
                               <a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">
                                2
                               </a>
                               ]. Using the relation between E and ℐ in
                               <a id="xref-disp-formula-10-3" class="xref-disp-formula" href="#disp-formula-10">
                                Eq. (S2)
                               </a>
                               , we can translate the bound E ≥ E
                               <sub>
                                WK
                               </sub>
                               into an equivalent statement that
                               <span class="inline-formula" id="inline-formula-141">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-141.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-141.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               at a given value of mutual information ℐ. The value of
                               <span class="inline-formula" id="inline-formula-142">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-142.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-142.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is shown in main text
                               <a id="xref-disp-formula-5-7" class="xref-disp-formula" href="#disp-formula-5">
                                Eq. (5)
                               </a>
                               :
                               <span class="disp-formula" id="disp-formula-31">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-38.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-38.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <div id="F8" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F8.large.jpg?width=800&height=600&carousel=1" title="Comparison of the simple analytical approximations for R0 from Eq. (S19) (top) and γx from Eq. (S17) (bottom) versus KMC simulation results. Each point corresponds to a parameter set drawn randomly from the enzymatic parameter distribution described in SI Sec. IV. The red dashed line corresponds to perfect agreement. Error bars for R0 are smaller than the symbol size, and hence not indicated in the figure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383868837" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Comparison of the simple analytical approximations for <em>R</em><sub>0</sub> from Eq. (S19) (top) and <em>γ</em><sub><em>x</em></sub> from Eq. (S17) (bottom) versus KMC simulation results. Each point corresponds to a parameter set drawn randomly from the enzymatic parameter distribution described in SI Sec. IV. The red dashed line corresponds to perfect agreement. Error bars for <em>R</em><sub>0</sub> are smaller than the symbol size, and hence not indicated in the figure.</div></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="FIG. S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F8.medium.gif" width="440" height="426"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="FIG. S3." src="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F8.medium.gif" width="440" height="426"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. S3." data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 FIG. S3.
                                </span>
                                <p id="p-79" class="first-child">
                                 Comparison of the simple analytical approximations for R
                                 <sub>
                                  0
                                 </sub>
                                 from
                                 <a id="xref-disp-formula-27-3" class="xref-disp-formula" href="#disp-formula-27">
                                  Eq. (S19)
                                 </a>
                                 (top) and γ
                                 <sub>
                                  x
                                 </sub>
                                 from
                                 <a id="xref-disp-formula-25-3" class="xref-disp-formula" href="#disp-formula-25">
                                  Eq. (S17)
                                 </a>
                                 (bottom) versus KMC simulation results. Each point corresponds to a parameter set drawn randomly from the enzymatic parameter distribution described in SI Sec. IV. The red dashed line corresponds to perfect agreement. Error bars for R
                                 <sub>
                                  0
                                 </sub>
                                 are smaller than the symbol size, and hence not indicated in the figure.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-80">
                               As shown in main text
                               <a id="xref-fig-3-20" class="xref-fig" href="#F3">
                                Figs. 3G-I
                               </a>
                               , the above
                               <span class="inline-formula" id="inline-formula-143">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-143.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-143.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               expression provides an excellent approximate upper bound on the
                               <span class="inline-formula" id="inline-formula-144">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-144.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-144.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               values calculated for the full enzymatic system. Even though the effective two-species model lacks reverse rates, it provides a useful tool for deriving this bound, since the maximum bandwidth is achieved when the reverse rates are negligible (large
                              </p>
                              <p id="p-81">
                               As mentioned in the discussion around main text
                               <a id="xref-disp-formula-7-4" class="xref-disp-formula" href="#disp-formula-7">
                                Eq. (7)
                               </a>
                               , the expression for
                               <span class="inline-formula" id="inline-formula-145">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-145.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-145.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               in
                               <a id="xref-disp-formula-31-1" class="xref-disp-formula" href="#disp-formula-31">
                                Eq. (S23)
                               </a>
                               also has an alternative interpretation. This gives the minimum production rate
                               <span class="inline-formula" id="inline-formula-146">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-146.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-146.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               necessary to achieve mutual information ℐ at a certain bandwidth
                               <span class="inline-formula" id="inline-formula-147">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-147.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-147.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               :
                               <span class="disp-formula" id="disp-formula-32">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-40.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-40.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-82">
                               By relating R
                               <sub>
                                0
                               </sub>
                               in turn to the ATP consumption rate
                               <span class="inline-formula" id="inline-formula-148">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-148.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-148.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , we can convert
                               <a id="xref-disp-formula-32-1" class="xref-disp-formula" href="#disp-formula-32">
                                Eq. (S24)
                               </a>
                               into an expression for the minimum necessary ATP consumption rate A
                               <sub>
                                min
                               </sub>
                               . To accomplish this, note that A can be rewritten as:
                               <span class="disp-formula" id="disp-formula-33">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-41.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-41.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where we have used
                               <a id="xref-disp-formula-14-4" class="xref-disp-formula" href="#disp-formula-14">
                                Eqs. (S6)
                               </a>
                               and
                               <a id="xref-disp-formula-27-4" class="xref-disp-formula" href="#disp-formula-27">
                                (S19)
                               </a>
                               . Finally, taking advantage of the fact that typically κ
                               <sub>
                                r
                               </sub>
                               ≫ R
                               <sub>
                                0
                               </sub>
                               for the parameter distributions of interest, we make the approximation
                               <span class="inline-formula" id="inline-formula-149">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-149.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-149.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . This allows us to derive main text
                               <a id="xref-disp-formula-8-6" class="xref-disp-formula" href="#disp-formula-8">
                                Eq. (8)
                               </a>
                               :
                               <span class="disp-formula" id="disp-formula-34">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-42.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-42.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <div class="section" id="sec-24">
                              <h4>
                               D. Optimality with respect to time-offset mutual information
                              </h4>
                              <p id="p-83">
                               The two-species optimality calculations of the previous section can be generalized to investigate mutual information ℐ
                               <sub>
                                α
                               </sub>
                               between input X(t) and output Y (t + α) with a time offset α. Following the formalism of Ref. [
                               <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">
                                5
                               </a>
                               ], the Wiener-Kolmogorov minimal error E
                               <sub>
                                WK,α
                               </sub>
                               for arbitrary α is
                               <span class="disp-formula" id="disp-formula-35">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-43.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-43.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-84">
                               For α = 0 this expression reduces to E
                               <sub>
                                WK
                               </sub>
                               from
                               <a id="xref-disp-formula-30-1" class="xref-disp-formula" href="#disp-formula-30">
                                Eq. (S22)
                               </a>
                               . The associated mutual information ℐ
                               <sub>
                                WK
                               </sub>
                               (α) = (1/2) ln
                               <sub>
                                2
                               </sub>
                               E
                               <sub>
                                WK
                               </sub>
                               (α) varies with α in a manner qualitatively similar to main text
                               <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                                Fig. 4A
                               </a>
                               . It peaks at a value
                               <span class="inline-formula" id="inline-formula-150">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-150.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-150.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               given by:
                               <span class="disp-formula" id="disp-formula-36">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-44.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-44.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-85">
                               The mutual information
                               <span class="inline-formula" id="inline-formula-151">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-151.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-151.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               at the peak can be expressed as
                               <span class="disp-formula" id="disp-formula-37">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-45.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-45.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-86">
                               Since the expression in
                               <a id="xref-disp-formula-37-1" class="xref-disp-formula" href="#disp-formula-37">
                                Eq. (S29)
                               </a>
                               is somewhat unwieldy, we can simplify it using a close upper bound,
                               <span class="disp-formula" id="disp-formula-38">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-46.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-46.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <div id="T1" class="table pos-float">
                               <div class="table-inline table-callout-links">
                                <div class="callout">
                                 <span>
                                  View this table:
                                 </span>
                                 <ul class="callout-links">
                                  <li class="view-inline first">
                                   <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2633239/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                    View inline
                                   </a>
                                  </li>
                                  <li class="view-popup">
                                   <a href="/highwire/markup/2633239/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                    View popup
                                   </a>
                                  </li>
                                  <li class="download-ppt last">
                                   <a href="/highwire/powerpoint/2633239" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                    Download powerpoint
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                               </div>
                               <div class="table-caption">
                                <span class="table-label">
                                 TABLE S1.
                                </span>
                                <p id="p-87" class="first-child">
                                 Results of log-normal fits to various kinase/phosphatase enzymatic parameters. For each fit the mean log
                                 <sub>
                                  10
                                 </sub>
                                 <span class="inline-formula" id="inline-formula-152">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/T1/embed/inline-graphic-152.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/T1/embed/inline-graphic-152.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 and standard deviation σ
                                 <sub>
                                  x
                                 </sub>
                                 are listed. The top rows of the table correspond to individual fits to parameters collected from the PaxDb and Sabio-RK databases. The bottom rows show the results of a joint fit, described in the text of SI Sec. IV
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-88">
                               The bound ℐ
                               <sup>
                                max
                               </sup>
                               (Λ) is never more than ≈ 0.04 bits above
                               <span class="inline-formula" id="inline-formula-153">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-153.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-153.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               for any Λ, and converges to
                               <span class="inline-formula" id="inline-formula-154">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-154.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-154.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               in the limit Λ → ∞. Hence for all practical purposes we can use
                               <a id="xref-disp-formula-38-1" class="xref-disp-formula" href="#disp-formula-38">
                                Eq. (S30)
                               </a>
                               in place of
                               <a id="xref-disp-formula-37-2" class="xref-disp-formula" href="#disp-formula-37">
                                Eq. (S29)
                               </a>
                               to estimate the bound on the mutual information, as we do in main text
                               <a id="xref-disp-formula-6-3" class="xref-disp-formula" href="#disp-formula-6">
                                Eq. (6)
                               </a>
                               .
                              </p>
                             </div>
                            </div>
                            <div class="section" id="sec-25">
                             <h3>
                              IV. ENZYMATIC PARAMETER DISTRIBUTION
                             </h3>
                             <p id="p-89">
                              Earlier surveys of enzymatic kinetic parameters in Refs. [
                              <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">
                               6
                              </a>
                              ,
                              <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">
                               7
                              </a>
                              ], over broader classes than just kinases and phosphatases, showed that their distributions could be approximately described by log-normal distributions. For a given parameter x, we will denote this as
                              <span class="inline-formula" id="inline-formula-155">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-155.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-155.gif"/>
                                </noscript>
                               </span>
                              </span>
                              or in other words that the base-10 logarithm of x is distributed according to a normal distribution with mean
                              <span class="inline-formula" id="inline-formula-156">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-156.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-156.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and standard deviation σ
                              <sub>
                               x
                              </sub>
                              . The value
                              <span class="inline-formula" id="inline-formula-157">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-157.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-157.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the median of the resulting log-normal distribution for x.
                             </p>
                             <p id="p-90">
                              For our work the focus is on kinases and phosphatases, and we are interested in looking at the push-pull loop signaling behavior over the entire distribution of biologically plausible parameters. The parameter data we collected, summarized in the histograms of main text
                              <a id="xref-fig-2-8" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              , had far more representation of kinases than phosphatases, which is a well known limitation of the existing experimental literature. Despite this sampling issue, the orders of magnitude spanned by phosphatase parameters were comparable to those of the kinases. For each parameter type, we thus decided to fit both types of enzyme with a single overall distribution, based on pooling of all the available kinase and phosphatase data together. The data available from the databases took the forms listed below (all raw data and the files used to process it are included in the Github repository associated with the manuscript). The mean
                              <span class="inline-formula" id="inline-formula-158">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-158.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-158.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and standard deviation σ
                              <sub>
                               x
                              </sub>
                              values from the log-normal fits for the different parameter classes are listed in the first four rows of
                              <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                               Table S1
                              </a>
                              .
                             </p>
                             <p id="p-91">
                              Enzymatic data:
                             </p>
                             <ul class="list-unord" id="list-1">
                              <li id="list-item-1">
                               <p id="p-92">
                                Mean substrate [S] and phosphatase [P] concentrations, where the substrate is taken to be a kinase [main text
                                <a id="xref-fig-2-9" class="xref-fig" href="#F2">
                                 Fig. 2A
                                </a>
                                ]. These numbers were derived from the PaxDb protein abundance database [
                                <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">
                                 8
                                </a>
                                ], taking advantage of UnitProt gene ontology associations to focus on just kinases and phosphatases in signal transduction pathways [
                                <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">
                                 10
                                </a>
                                ]. Each PaxDb data entry is in terms of ppm (parts per million) of abundance, relative to the total number of proteins in the cell. To convert from ppm to molar concentrations, we looked at data from human cells (which had the best representation in the database), and used the estimated total concentration of 2.7×10
                                <sup>
                                 6
                                </sup>
                                proteins per μm
                                <sup>
                                 3
                                </sup>
                                for human cells [
                                <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">
                                 11
                                </a>
                                ]. The latter concentration corresponds to 4.48×10
                                <sup>
                                 −3
                                </sup>
                                M. If y is the abundance in ppm units, then 4.48(y/10
                                <sup>
                                 6
                                </sup>
                                ) ×10
                                <sup>
                                 −3
                                </sup>
                                M is the corresponding molar concentration. Note that total concentrations are very similar across many different types of species [
                                <a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">
                                 11
                                </a>
                                ], so there should not be a strong species-dependence in the analysis. For example the same analysis in mouse cells rather than human ones yields quantitatively similar results: a mean kinase/phosphatase concentration 10
                                <sup>
                                 −8.31
                                </sup>
                                M (versus 10
                                <sup>
                                 −7.93
                                </sup>
                                M in human cells), and a log-normal standard deviation of 1.03 (versus 0.84 in human cells).
                               </p>
                              </li>
                              <li id="list-item-2">
                               <p id="p-93">
                                Reaction parameters [main text
                                <a id="xref-fig-2-10" class="xref-fig" href="#F2">
                                 Fig. 2B-D
                                </a>
                                ]. These values were taken from the Sabio-RK database [
                                <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">
                                 9
                                </a>
                                ], where they were most often available in the following forms: Michaelis constants
                                <span class="inline-formula" id="inline-formula-159">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-159.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-159.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                for the kinase/phosphatase (main text
                                <a id="xref-fig-2-11" class="xref-fig" href="#F2">
                                 Fig. 2B
                                </a>
                                ), the corresponding specificity ratios
                                <span class="inline-formula" id="inline-formula-160">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-160.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-160.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                (main text
                                <a id="xref-fig-2-12" class="xref-fig" href="#F2">
                                 Fig. 2C
                                </a>
                                ), and the reaction rates κ
                                <sub>
                                 r
                                </sub>
                                and ρ
                                <sub>
                                 r
                                </sub>
                                (main text
                                <a id="xref-fig-2-13" class="xref-fig" href="#F2">
                                 Fig. 2D
                                </a>
                                ). The resulting distributions were entirely consistent (though slightly narrower) with the distributions for the same parameter types analyzed in Ref. [
                                <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">
                                 6
                                </a>
                                ], which considered all enzymes (not just kinases and phosphatases).
                               </p>
                              </li>
                             </ul>
                             <p id="p-94">
                              Note that the six reaction parameter types that were collected from the Sabio-RK database
                              <span class="inline-formula" id="inline-formula-161">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-161.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-161.gif"/>
                                </noscript>
                               </span>
                              </span>
                              are not directly in the form that we need to calculate push-pull loop signaling properties. For the latter we would like to know (κ
                              <sub>
                               b
                              </sub>
                              , ρ
                              <sub>
                               b
                              </sub>
                              , κ
                              <sub>
                               u
                              </sub>
                              , ρ
                              <sub>
                               u
                              </sub>
                              , κ
                              <sub>
                               r
                              </sub>
                              , ρ
                              <sub>
                               r
                              </sub>
                              ), or equivalently
                              <span class="inline-formula" id="inline-formula-162">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-162.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-162.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Here the dissociation constants are defined as
                              <span class="inline-formula" id="inline-formula-163">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-163.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-163.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-164">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-164.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-164.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Let us denote the parameter vector
                              <span class="inline-formula" id="inline-formula-165">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-165.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-165.gif"/>
                                </noscript>
                               </span>
                              </span>
                              as
                              <strong>
                               v
                              </strong>
                              , with components v
                              <sub>
                               α
                              </sub>
                              , α = 1, …, 6. We would like to find a joint distribution for
                              <strong>
                               v
                              </strong>
                              that is self-consistent with the individual log-normal distributions for the alternative parameter types fitted directly from the database values (first 4 rows of
                              <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">
                               Table S1
                              </a>
                              ). We will assume the simplest form for the joint distribution Φ: a product of individual log-normal distributions for each parameter v
                              <sub>
                               α
                              </sub>
                              , with median values
                              <span class="inline-formula" id="inline-formula-166">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-166.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-166.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and standard deviations σ
                              <sub>
                               α
                              </sub>
                              :
                              <span class="disp-formula" id="disp-formula-39">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-48.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-48.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-95">
                              Note that the v
                              <sub>
                               α
                              </sub>
                              ln(10) term in the denominator of the prefactor comes from the Jacobian due to the variable change between log
                              <sub>
                               10
                              </sub>
                              v
                              <sub>
                               α
                              </sub>
                              and v
                              <sub>
                               α
                              </sub>
                              . This ensures that the probability is properly normalized:
                              <span class="inline-formula" id="inline-formula-167">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-167.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-167.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . As explained above, kinase and phosphatase parameters are assumed to be drawn from the same distributions, so we enforce that
                              <span class="inline-formula" id="inline-formula-168">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-168.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-168.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , and analogously for the standard deviations σ
                              <sub>
                               α
                              </sub>
                              . This leaves six distinct values that determine the distribution:
                              <span class="inline-formula" id="inline-formula-169">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-169.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-169.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <p id="p-96">
                              To estimate these six distribution parameters, we use the following iterative numerical fitting procedure. We start with a guess for
                              <span class="inline-formula" id="inline-formula-170">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-170.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-170.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and then draw 10
                              <sup>
                               4
                              </sup>
                              parameter sets
                              <strong>
                               v
                              </strong>
                              from the resulting distribution Φ(
                              <strong>
                               v
                              </strong>
                              ). For each parameter set we can calculate the alternative parameter types
                              <span class="inline-formula" id="inline-formula-171">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-171.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-171.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . We then fit the resulting 10
                              <sup>
                               4
                              </sup>
                              values for these alternative types to individual log-normal distributions, and compare the means and standard deviations to the empirical results in the top half of
                              <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">
                               Table S1
                              </a>
                              . The sum of the relative absolute errors between the new joint fit values and the empirical results for the means / standard deviations is our overall goodness-of-fit measure. We perturb our guess for
                              <span class="inline-formula" id="inline-formula-172">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-172.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-172.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and accept the perturbation if it improves the goodness-of-fit. This procedure is iterated until convergence. The results of this joint fit are shown in the bottom half of
                              <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">
                               Table S1
                              </a>
                              . The joint fit predictions for the binding rate (κ
                              <sub>
                               b
                              </sub>
                              , ρ
                              <sub>
                               b
                              </sub>
                              ) and dissociation constant
                              <span class="inline-formula" id="inline-formula-173">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-173.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-173.gif"/>
                                </noscript>
                               </span>
                              </span>
                              distributions are consistent with earlier estimates of these parameters in specific kinase/phosphatase systems [
                              <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">
                               12
                              </a>
                              ]. As another consistency check, the joint fit distribution for the reaction rates (κ
                              <sub>
                               r
                              </sub>
                              , ρ
                              <sub>
                               r
                              </sub>
                              ) is nearly identical to the individual empirical fit based on the Sabio-RK database values.
                             </p>
                             <p id="p-97">
                              Finally we note that the simple joint distribution Φ(
                              <strong>
                               v
                              </strong>
                              ) in
                              <a id="xref-disp-formula-39-1" class="xref-disp-formula" href="#disp-formula-39">
                               Eq. (S31)
                              </a>
                              is by construction too broad: it may produce the correct marginal distributions for quantities collected from the Sabio-RK database, but it ignores any correlations between those individual parameters that may be present in natural systems. Estimating these correlations from the existing database entries is quite challenging, because relatively few entries have a complete list of all the parameters of interest. Hence, as explained in the main text, we take Φ(
                              <strong>
                               v
                              </strong>
                              ) to be effectively a superset: it should contain the true, presumably narrower, biological distribution plus parameter sets that are less likely to be observed in nature. A convenient aspect of this interpretation is that any collective conclusion we draw from the entire distribution Φ(
                              <strong>
                               v
                              </strong>
                              ) should also be true for the subset of biological parameters. Moreover we can thus explore a larger design space (potentially available for evolution) than what we currently observe in modern biological systems.
                             </p>
                            </div>
                            <div class="section" id="sec-26">
                             <h3>
                              V. RESULTS FOR ALTERNATIVE INPUT KINASE CONCENTRATIONS
                             </h3>
                             <p id="p-98">
                              The results in main text
                              <a id="xref-fig-3-21" class="xref-fig" href="#F3">
                               Fig. 3D-F
                              </a>
                              were for a mean input kinase concentration [K] = 5 nM. In
                              <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                               Fig. S4
                              </a>
                              we show the analogous results for two different choices: [K] = 0.5 nM (left column) and [K] = 50 nM (right column). The main conclusions remain unchanged: the physiological Δμ range (highlighted in pink) is always just above the upper edge of the
                              <span class="inline-formula" id="inline-formula-174">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-174.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-174.gif"/>
                                </noscript>
                               </span>
                              </span>
                              cloud, and the number of available parameter sets decreases rapidly as the mutual information ℐ is increased.
                             </p>
                            </div>
                            <div class="section" id="sec-27">
                             <h3>
                              VI. ANALYSIS OF THE PBS2-HOG1 PUSH-PULL LOOP IN YEAST
                             </h3>
                             <p id="p-99">
                              To illustrate our theoretical framework in a concrete biological example, let us consider a kinase-phosphatase loop from one of the most extensively studied signaling pathways: the Hog1 mitogen-activated protein kinase (MAPK) pathway that allows yeast to adapt to extracellular osmotic changes [
                              <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">
                               13
                              </a>
                              –
                              <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">
                               15
                              </a>
                              ]. We will focus in particular on the final portion of the pathway, where the active (phosphorylated) kinase Pbs2pp catalyzes the conversion of inactive Hog1 into phosphorylated Hog1pp. The latter protein is interchanged quickly between cytoplasm and nucleus, where it regulates a variety of responses to osmotic stress. Hog1pp is dephosphorylated by a combination of phosphatases Ptp2 (mainly in the nucleus) and Ptp3 in the cytoplasm [
                              <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">
                               16
                              </a>
                              ]. Thus Pbs2pp will play the role of K in our model, Hog1 will be S, Hog1pp will be S
                              <sup>
                               ∗
                              </sup>
                              , and Ptp2/Ptp3 will be P. To parameterize our model, we start with a more detailed theoretical description of the entire pathway developed by Zi et al. [
                              <a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">
                               13
                              </a>
                              ]. A key appeal of this work is that its parameters were carefully fit to extensive experimental data from yeast cells exposed to different time series of external salt shocks in microfluidic experiments [
                              <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">
                               14
                              </a>
                              ]. However since the parameters of Zi et al. are not expressed in the same form as the enzymatic reaction rates of our model, we do have to convert from their framework to ours, as described below.
                             </p>
                             <div id="F9" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F9.large.jpg?width=800&height=600&carousel=1" title="Analogous to main Figure 3D-F, except for input kinase concentration [K] = 5 nM (left column) and 50 nM (right column). The rows correspond to mutual information ℐ = 1, 1.5, and 2 bits respectively. The probabilities of successfully drawing a parameter set that achieves the specified ℐ value are shown in red in panel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383868837" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Analogous to main Figure 3D-F, except for input kinase concentration [<em>K</em>] = 5 nM (left column) and 50 nM (right column). The rows correspond to mutual information <em>ℐ</em> = 1, 1.5, and 2 bits respectively. The probabilities of successfully drawing a parameter set that achieves the specified <em>ℐ</em> value are shown in red in panel.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="FIG. S4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F9.medium.gif" width="440" height="410"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="FIG. S4." src="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F9.medium.gif" width="440" height="410"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. S4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/05/2020.10.06.327700/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                FIG. S4.
                               </span>
                               <p id="p-100" class="first-child">
                                Analogous to main
                                <a id="xref-fig-3-22" class="xref-fig" href="#F3">
                                 Figure 3D-F
                                </a>
                                , except for input kinase concentration [K] = 5 nM (left column) and 50 nM (right column). The rows correspond to mutual information ℐ = 1, 1.5, and 2 bits respectively. The probabilities of successfully drawing a parameter set that achieves the specified ℐ value are shown in red in panel.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div class="section" id="sec-28">
                              <h4>
                               A. Parameter estimation based on earlier literature
                              </h4>
                              <p id="p-101">
                               Ref. [
                               <a id="xref-ref-77-3" class="xref-bibr" href="#ref-77">
                                13
                               </a>
                               ] explicitly distinguishes between the concentration of Hog1 and Hog1pp in the cytoplasm and nucleus, denoted with c and n superscripts respectively: [Hog1
                               <sup>
                                c
                               </sup>
                               ], [Hog1
                               <sup>
                                n
                               </sup>
                               ], [Hog1pp
                               <sup>
                                c
                               </sup>
                               ], [Hog1pp
                               <sup>
                                n
                               </sup>
                               ]. If we are interested in the average concentrations overall, we can denote these as:
                               <span class="disp-formula" id="disp-formula-40">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-50.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-50.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where V
                               <sub>
                                c
                               </sub>
                               and V
                               <sub>
                                n
                               </sub>
                               are the volumes of the cytoplasm and nucleus respectively, taken to have a ratio of V
                               <sub>
                                n
                               </sub>
                               /V
                               <sub>
                                c
                               </sub>
                               = 0.14 [
                               <a id="xref-ref-77-4" class="xref-bibr" href="#ref-77">
                                13
                               </a>
                               ].
                               <a id="xref-disp-formula-40-1" class="xref-disp-formula" href="#disp-formula-40">
                                Eq. (S32)
                               </a>
                               also implies:
                               <span class="disp-formula" id="disp-formula-41">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-51.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-51.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where f = V
                               <sub>
                                c
                               </sub>
                               /(V
                               <sub>
                                c
                               </sub>
                               + V
                               <sub>
                                n
                               </sub>
                               ) = 0.88. As a simplification of
                               <a id="xref-disp-formula-40-2" class="xref-disp-formula" href="#disp-formula-40">
                                Eq. (S32)
                               </a>
                               , we note in Ref. [
                               <a id="xref-ref-77-5" class="xref-bibr" href="#ref-77">
                                13
                               </a>
                               ] import and export of the Hog1 proteins is fast relative to other reactions, and for a given input level the system rapidly reaches a stationary state with [Hog1
                               <sup>
                                n
                               </sup>
                               ]≈[Hog1
                               <sup>
                                c
                               </sup>
                               ]≈ [S], [Hog1pp
                               <sup>
                                n
                               </sup>
                               ]≈[Hog1pp
                               <sup>
                                c
                               </sup>
                               ]≈ [S
                               <sup>
                                ∗
                               </sup>
                               ].
                              </p>
                              <p id="p-102">
                               We can now look at individual reactions that contribute to the time derivatives on the right-hand sides of
                               <a id="xref-disp-formula-41-1" class="xref-disp-formula" href="#disp-formula-41">
                                Eq. (S33)
                               </a>
                               and find their analogues in our model. For example the phosphorylation step that converts Hog1
                               <sup>
                                c
                               </sup>
                               to Hog1pp
                               <sup>
                                cs
                               </sup>
                               is expressed in Ref. [
                               <a id="xref-ref-77-6" class="xref-bibr" href="#ref-77">
                                13
                               </a>
                               ] as an effective second order reaction of the form
                               <span class="inline-formula" id="inline-formula-175">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-175.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-175.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , with rate constant
                               <span class="inline-formula" id="inline-formula-176">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-176.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-176.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . This contributes positively to d[Hog1pp
                               <sup>
                                c
                               </sup>
                               ]/dt and with a minus sign to d[Hog1
                               <sup>
                                c
                               </sup>
                               ]/dt, and so leads to contributions magnitude
                               <span class="inline-formula" id="inline-formula-177">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-177.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-177.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               to the righthand sides of
                               <a id="xref-disp-formula-41-2" class="xref-disp-formula" href="#disp-formula-41">
                                Eq. (S33)
                               </a>
                               . Note that even though activation of Hog1 is actually a double phosphorylation (of a threonine and tyrosine residue), the entire process in this case can be well approximated through a single rate constant.
                              </p>
                              <p id="p-103">
                               In our model the conversion of S to S
                               <sup>
                                ∗
                               </sup>
                               occurs through the intermediate state S
                               <sub>
                                K
                               </sub>
                               . However if we want to compare to the phosphorylation step of Ref. [
                               <a id="xref-ref-77-7" class="xref-bibr" href="#ref-77">
                                13
                               </a>
                               ] in order to match parameters, we can look at the deterministic contribution to the dynamics (ignoring fluctuations) in the Michaelis-Menten approximation for enzyme kinetics [
                               <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">
                                17
                               </a>
                               ]. In this picture the phosphorylation reaction contributes to d[S]/dt and d[S
                               <sup>
                                ∗
                               </sup>
                               ]/dt through a term of magnitude
                               <span class="inline-formula" id="inline-formula-178">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-178.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-178.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , where the last simplification is valid when
                               <span class="inline-formula" id="inline-formula-179">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-179.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-179.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . If we compare
                               <span class="inline-formula" id="inline-formula-180">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-180.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-180.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               to
                               <span class="inline-formula" id="inline-formula-181">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-181.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-181.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , noting that [K] = [Pbs2pp] and [S] ≈ [Hog1
                               <sup>
                                c
                               </sup>
                               ], we can make the following identification:
                               <span class="disp-formula" id="disp-formula-42">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-52.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-52.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-104">
                               The dephosphorylation steps in Ref. [
                               <a id="xref-ref-77-8" class="xref-bibr" href="#ref-77">
                                13
                               </a>
                               ] are modeled as two pseudo-first-order reactions: conversion of Hog1pp
                               <sup>
                                c
                               </sup>
                               to Hog1
                               <sup>
                                c
                               </sup>
                               with rate
                               <span class="inline-formula" id="inline-formula-182">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-182.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-182.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , and the conversion of Hog1pp
                               <sup>
                                n
                               </sup>
                               to Hog1
                               <sup>
                                n
                               </sup>
                               with rate
                               <span class="inline-formula" id="inline-formula-183">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-183.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-183.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . The pseudo-first-order rate constants are given by:
                               <span class="inline-formula" id="inline-formula-184">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-184.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-184.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-185">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-185.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-185.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . These reactions will lead to contributions of magnitude
                               <span class="inline-formula" id="inline-formula-186">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-186.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-186.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               to the right-hand sides of
                               <a id="xref-disp-formula-41-3" class="xref-disp-formula" href="#disp-formula-41">
                                Eq. (S33)
                               </a>
                               . In our model (using a similar Michaelis-Menten approximation to the one described above, with
                               <span class="inline-formula" id="inline-formula-187">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-187.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-187.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               ), the analogous expression for dephosphorylation is effectively a second-order reaction with rate
                               <span class="inline-formula" id="inline-formula-188">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-188.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-188.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . Comparison of the two expressions, using the approximation [Hog1pp
                               <sup>
                                n
                               </sup>
                               ]≈[Hog1pp
                               <sup>
                                c
                               </sup>
                               ]≈ [S
                               <sup>
                                ∗
                               </sup>
                               ], leads to the identification:
                               <span class="disp-formula" id="disp-formula-43">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-53.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-53.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <div id="T2" class="table pos-float">
                               <div class="table-inline table-callout-links">
                                <div class="callout">
                                 <span>
                                  View this table:
                                 </span>
                                 <ul class="callout-links">
                                  <li class="view-inline first">
                                   <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2633244/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                    View inline
                                   </a>
                                  </li>
                                  <li class="view-popup">
                                   <a href="/highwire/markup/2633244/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                    View popup
                                   </a>
                                  </li>
                                  <li class="download-ppt last">
                                   <a href="/highwire/powerpoint/2633244" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                    Download powerpoint
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                               </div>
                               <div class="table-caption">
                                <span class="table-label">
                                 TABLE S2.
                                </span>
                                <p id="p-105" class="first-child">
                                 Summary of parameters for the yeast Pbs2/Hog1 system estimated from earlier literature.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-106">
                               Here we set [P] = 0.058 μM as an average measure of phosphatase concentrations, to facilitate the conversion from pseudo-first-order to second-order rate constants. The value of [P] is based on estimates of the concentrations of the two phosphatases in yeast from Ref. [
                               <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">
                                18
                               </a>
                               ]: 0.049 μM for Ptp3 in the cytoplasm, and 0.067 μM for Ptp2 in the nucleus, where we have used V
                               <sub>
                                c
                               </sub>
                               = f (V
                               <sub>
                                c
                               </sub>
                               + V
                               <sub>
                                n
                               </sub>
                               ), V
                               <sub>
                                n
                               </sub>
                               = (1 − f)(V
                               <sub>
                                c
                               </sub>
                               + V
                               <sub>
                                n
                               </sub>
                               ) and V
                               <sub>
                                c
                               </sub>
                               + V
                               <sub>
                                n
                               </sub>
                               ≈ 30 fL [
                               <a id="xref-ref-77-9" class="xref-bibr" href="#ref-77">
                                13
                               </a>
                               ,
                               <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">
                                19
                               </a>
                               ] to convert from populations to concentrations. Since the concentrations were of similar scale, we let [P] be the mean of the two values.
                              </p>
                              <p id="p-107">
                               As a consistency check to make sure the final estimates of the specificity ratios
                               <span class="inline-formula" id="inline-formula-189">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-189.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-189.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-190">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-190.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-190.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               in
                               <a id="xref-disp-formula-42-1" class="xref-disp-formula" href="#disp-formula-42">
                                Eqs. (S34)
                               </a>
                               -
                               <a id="xref-disp-formula-43-1" class="xref-disp-formula" href="#disp-formula-43">
                                (S35)
                               </a>
                               are biologically plausible, we can compare them with the distribution of these ratios among kinases/phosphatases from the Sabio-RK database in main text
                               <a id="xref-fig-2-14" class="xref-fig" href="#F2">
                                Fig. 2C
                               </a>
                               . The values for the Hog1/Pbs2 system are not unusual, and lie near the higher end of the range, at about the 0.87 quantile. The final parameter value we can estimate from the literature is the mean Hog1 concentration [S] = 0.38 μM, based on the abundance reported in Ref. [
                               <a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">
                                18
                               </a>
                               ].
                              </p>
                             </div>
                             <div class="section" id="sec-29">
                              <h4>
                               B. Estimation of remaining parameters
                              </h4>
                              <p id="p-108">
                               Based on the above analysis, we have estimates for four quantities in the Pbs2/Hog1 system drawn from the earlier literature:
                               <span class="inline-formula" id="inline-formula-191">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-191.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-191.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . These are summarized in
                               <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
                                Table S2
                               </a>
                               . The relationship of the enzymatic reaction/binding/unbinding rate parameters to the estimated values then takes the form:
                               <span class="disp-formula" id="disp-formula-44">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-55.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-55.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-109">
                               The above parameters depend on the values of
                               <span class="inline-formula" id="inline-formula-192">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-192.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-192.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . While we do not know what these are for the Pbs2/Hog1 system, we can draw their values from the corresponding empirical log-normal distributions described in
                               <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">
                                Table S1
                               </a>
                               . By repeating the draw many times, we can check how our final optimality analysis (see below) depends on the precise values of the unknown parameters. As it turns out the dependence of R
                               <sub>
                                0
                               </sub>
                               ,
                               <span class="inline-formula" id="inline-formula-193">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-193.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-193.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-194">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-194.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-194.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               on the unknown values is quite weak, and we will be able to make robust estimates for these quantities. In the cases of
                               <span class="inline-formula" id="inline-formula-195">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-195.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-195.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-196">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-196.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-196.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , we constrain the random draw from their log-normal distributions to enforce
                               <span class="inline-formula" id="inline-formula-197">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-197.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-197.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-198">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-198.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-198.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . This ensures self-consistency with the assumptions
                               <span class="inline-formula" id="inline-formula-199">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-199.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-199.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-200">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-200.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-200.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , which were used in the previous subsection to match the form of the phosphorylation / dephosphorylation reactions between Ref. [
                               <a id="xref-ref-77-10" class="xref-bibr" href="#ref-77">
                                13
                               </a>
                               ] and our model. The final two parameters are the reverse reaction rates κ
                               <sub>
                                −r
                               </sub>
                               and ρ
                               <sub>
                                −r
                               </sub>
                               . Since we do not have any experimental estimates of these for the Pbs2/Hog1 system, we assume that the physiological value of Δμ in yeast (around 21 k
                               <sub>
                                B
                               </sub>
                               T [
                               <a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">
                                19
                               </a>
                               ]) is sufficiently high that κ
                               <sub>
                                −r
                               </sub>
                               and ρ
                               <sub>
                                −r
                               </sub>
                               are negligible under normal conditions.
                              </p>
                             </div>
                             <div class="section" id="sec-30">
                              <h4>
                               C. Bandwidth and gain
                              </h4>
                              <p id="p-110">
                               Given the parameter estimation procedure described above, we can calculate
                               <span class="inline-formula" id="inline-formula-201">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-201.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-201.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               for each draw of the unknown parameters. The results remain within a narrow distribution, relatively insensitive to the values of the unknown parameters. The mean and standard deviations for 50 draws are:
                               <span class="inline-formula" id="inline-formula-202">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-202.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/inline-graphic-202.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                            </div>
                            <div class="section" id="sec-31">
                             <h3>
                              VII. ESTIMATION OF TOTAL RESTING METABOLIC EXPENDITURE
                             </h3>
                             <p id="p-111">
                              For single-celled organisms, the total resting metabolic expenditure C
                              <sub>
                               T
                              </sub>
                              can be estimated by the approach outlined in Ref. [
                              <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">
                               20
                              </a>
                              ]. C
                              <sub>
                               T
                              </sub>
                              has two contributions: C
                              <sub>
                               T
                              </sub>
                              = C
                              <sub>
                               G
                              </sub>
                              + t
                              <sub>
                               r
                              </sub>
                              C
                              <sub>
                               M
                              </sub>
                              . Here C
                              <sub>
                               G
                              </sub>
                              is the expenditure involved in growth during one generation time t
                              <sub>
                               r
                              </sub>
                              , and C
                              <sub>
                               M
                              </sub>
                              is the maintenance cost per unit time. Using a large collection of metabolic data from Ref. [
                              <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">
                               21
                              </a>
                              ], covering both prokaryotes and single-celled eukaryotes, one can observe that both C
                              <sub>
                               M
                              </sub>
                              and C
                              <sub>
                               T
                              </sub>
                              scale approximately linearly with cell volume V, agreeing with the prediction of the bioenergetic growth model of Ref. [
                              <a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">
                               20
                              </a>
                              ]. The expression for C
                              <sub>
                               T
                              </sub>
                              based on the results of these linear fits is [
                              <a id="xref-ref-84-3" class="xref-bibr" href="#ref-84">
                               20
                              </a>
                              ]:
                              <span class="disp-formula" id="disp-formula-45">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-56.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/05/2020.10.06.327700/embed/graphic-56.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where the unit P corresponds to the hydrolysis of a phosphate bond (i.e. the consumption of one ATP or ATP equivalent). Using the main text values of V = 30 fL and t
                              <sub>
                               r
                              </sub>
                              = 3600 s, we get C
                              <sub>
                               T
                              </sub>
                              = 7.0 × 10
                              <sup>
                               11
                              </sup>
                              P.
                             </p>
                            </div>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgments
                            </h2>
                            <p id="p-58">
                             We thank Shishir Adhikari for useful discussions. Parts of the numerical analysis were carried out using the High Performance Computing Resource in the Core Facility for Advanced Research Computing at Case Western Reserve University.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-4">
                               The theory in the manuscript has been extended to include both time-delayed and instantaneous mutual information. In particular, there is now a general analytical bound that covers both scenarios.
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-2">
                              <p id="p-5">
                               <a href="https://github.com/hincz-lab/cell-signaling">
                                https://github.com/hincz-lab/cell-signaling
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2>
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               [1].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.1" data-doi="10.1016/j.cell.2011.01.030">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Balázsi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   van Oudenaarden
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Collins
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Cellular decision making and biological noise: from microbes to mammals
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  144
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  910
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  925
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBalazsi%26rft.auinit1%253DG.%26rft.volume%253D144%26rft.issue%253D6%26rft.spage%253D910%26rft.epage%253D925%26rft.atitle%253DCellular%2Bdecision%2Bmaking%2Band%2Bbiological%2Bnoise%253A%2Bfrom%2Bmicrobes%2Bto%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.01.030%26rft_id%253Dinfo%253Apmid%252F21414483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.01.030&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21414483&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000288543500010&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [2].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.2" data-doi="10.1073/pnas.1411524111">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Govern
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   ten Wolde
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Optimal resource allocation in cellular sensing systems
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  111
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  17486
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  17491
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1411524111%26rft_id%253Dinfo%253Apmid%252F25422473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzQ5LzE3NDg2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDcvMDUvMjAyMC4xMC4wNi4zMjc3MDAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [3].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.3" data-doi="10.1007/s10955-015-1440-5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   ten Wolde
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Becker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N. B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ouldridge
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. E.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mugler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Fundamental limits to cellular sensing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J. Stat. Phys
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  162
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1395
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1424
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BPhys%26rft.volume%253D162%26rft.spage%253D1395%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10955-015-1440-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s10955-015-1440-5&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [4].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.4" data-doi="10.1088/0034-4885/79/5/052601">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Information processing in bacteria: memory, computation, and statistical physics: a key issues review
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Rep. Prog. Phys
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  79
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  052601
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DRep.%2BProg.%2BPhys%26rft.volume%253D79%26rft.spage%253D052601%26rft_id%253Dinfo%253Adoi%252F10.1088%252F0034-4885%252F79%252F5%252F052601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1088/0034-4885/79/5/052601&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [5].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.5" data-doi="10.1146/annurev.cellbio.20.010403.113126">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Logan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. Y.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nusse
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The Wnt signaling pathway in development and disease
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Annu. Rev. Cell Dev. Biol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  781
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  810
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bcell%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BCell%2BDev%2BBiol%26rft.aulast%253DLogan%26rft.auinit1%253DC.%2BY.%26rft.volume%253D20%26rft.spage%253D781%26rft.epage%253D810%26rft.atitle%253DThe%2BWnt%2Bsignaling%2Bpathway%2Bin%2Bdevelopment%2Band%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.cellbio.20.010403.113126%26rft_id%253Dinfo%253Apmid%252F15473860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1146/annurev.cellbio.20.010403.113126&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15473860&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000225318200029&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [6].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.06.327700v3.6" data-doi="10.1038/436792a">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Parsons
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. W.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Colorectal cancer: mutations in a signalling pathway
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  436
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  792
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DParsons%26rft.auinit1%253DD.%2BW.%26rft.volume%253D436%26rft.issue%253D7052%26rft.spage%253D792%26rft.epage%253D792%26rft.atitle%253DColorectal%2Bcancer%253A%2Bmutations%2Bin%2Ba%2Bsignalling%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F436792a%26rft_id%253Dinfo%253Apmid%252F16094359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/436792a&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16094359&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000231116500033&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [7].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.06.327700v3.7" data-doi="10.1016/j.cellsig.2009.03.009">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sarkar
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F. H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Cellular signaling perturbation by natural products
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell. Sign
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  21
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1541
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1547
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCellular%2Bsignalling%26rft.stitle%253DCell%2BSignal%26rft.aulast%253DSarkar%26rft.auinit1%253DF.%2BH.%26rft.volume%253D21%26rft.issue%253D11%26rft.spage%253D1541%26rft.epage%253D1547%26rft.atitle%253DCellular%2Bsignaling%2Bperturbation%2Bby%2Bnatural%2Bproducts.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cellsig.2009.03.009%26rft_id%253Dinfo%253Apmid%252F19298854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cellsig.2009.03.009&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19298854&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [8].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Riera
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Merkwirth
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   De Magalhaes Filho
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. D.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dillin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Signaling networks determining life span
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Annu. Rev. Biochem
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  85
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  35
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  64
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BBiochem%26rft.volume%253D85%26rft.spage%253D35%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [9].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Uda
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Application of information theory in systems biology
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biophys. Rev
                                 </abbr>
                                 .
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  8
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [10].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.10" data-doi="10.1073/pnas.0806077105">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tkačik
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Callan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. G.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bialek
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Information flow and optimization in transcriptional regulation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  105
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  12265
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  12270
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0806077105%26rft_id%253Dinfo%253Apmid%252F18719112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzM0LzEyMjY1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDcvMDUvMjAyMC4xMC4wNi4zMjc3MDAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [11].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.06.327700v3.11" data-doi="10.1126/science.1204553">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cheong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rhee
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nemenman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Levchenko
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Information transduction capacity of noisy biochemical signaling networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  334
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  354
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  358
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1204553%26rft_id%253Dinfo%253Apmid%252F21921160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzQvNjA1NC8zNTQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNy8wNS8yMDIwLjEwLjA2LjMyNzcwMC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [12].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.06.327700v3.12" data-doi="10.1126/science.1234511">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Uda
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Robustness and compensation of information transmission of signaling pathways
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  341
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  558
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  561
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DUda%26rft.auinit1%253DS.%26rft.volume%253D341%26rft.issue%253D6145%26rft.spage%253D558%26rft.epage%253D561%26rft.atitle%253DRobustness%2Band%2BCompensation%2Bof%2BInformation%2BTransmission%2Bof%2BSignaling%2BPathways%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1234511%26rft_id%253Dinfo%253Apmid%252F23908238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDEvNjE0NS81NTgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNy8wNS8yMDIwLjEwLjA2LjMyNzcwMC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [13].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.06.327700v3.13" data-doi="10.1073/pnas.1314446111">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Voliotis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Perrett
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McWilliams
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McArdle
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bowsher
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. G.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Information transfer by leaky, heterogeneous, protein kinase signaling systems
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  111
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  E326
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  E333
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1314446111%26rft_id%253Dinfo%253Apmid%252F24395805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTExLzMvRTMyNiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA3LzA1LzIwMjAuMTAuMDYuMzI3NzAwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [14].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.14" data-doi="10.1126/science.1254933">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Selimkhanov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Accurate information transmission through dynamic biochemical signaling networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  346
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1370
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1373
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSelimkhanov%26rft.auinit1%253DJ.%26rft.volume%253D346%26rft.issue%253D6215%26rft.spage%253D1370%26rft.epage%253D1373%26rft.atitle%253DAccurate%2Binformation%2Btransmission%2Bthrough%2Bdynamic%2Bbiochemical%2Bsignaling%2Bnetworks%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1254933%26rft_id%253Dinfo%253Apmid%252F25504722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDYvNjIxNS8xMzcwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDcvMDUvMjAyMC4xMC4wNi4zMjc3MDAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [15].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.15" data-doi="10.1016/j.bpj.2016.12.045">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Potter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Byrd
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mugler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sun
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Dynamic sampling and information encoding in biochemical networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biophys. J
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  112
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  795
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  804
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiophys.%2BJ%26rft.volume%253D112%26rft.spage%253D795%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bpj.2016.12.045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.bpj.2016.12.045&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [16].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.06.327700v3.16" data-doi="10.1073/pnas.1615660114">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Suderman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bachman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sorger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. K.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Deeds
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Fundamental trade-offs between information flow in single cells and cellular populations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  114
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  5755
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  5760
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1615660114%26rft_id%253Dinfo%253Apmid%252F28500273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE0LzIyLzU3NTUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNy8wNS8yMDIwLjEwLjA2LjMyNzcwMC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [17].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.17" data-doi="10.1038/s41467-018-02974-x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Keshelava
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  High capacity in g protein-coupled receptor signaling
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Commun
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  8
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-02974-x%26rft_id%253Dinfo%253Apmid%252F29317637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41467-018-02974-x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29317637&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [18].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.18">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shannon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. E.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  A mathematical theory of communication
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bell Systems Tech. J
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  379
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  423
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1948
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBell%2BSystems%2BTech.%2BJ%26rft.volume%253D27%26rft.spage%253D379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [19].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.06.327700v3.19">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cover
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thomas
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. A.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  Elements of information theory
                                 </span>
                                 (
                                 <span class="cit-publ-name">
                                  John Wiley & Sons
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [20].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.20">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ouyang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Q.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The free-energy cost of accurate biochemical oscillations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Phys
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  772
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  778
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BPhys%26rft.volume%253D11%26rft.spage%253D772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [21].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.06.327700v3.21">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hasegawa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Optimal temporal patterns for dynamical cellular signaling
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  New J. Phys
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  113031
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNew%2BJ.%2BPhys%26rft.volume%253D18%26rft.spage%253D113031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [22].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.06.327700v3.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mehta
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. H.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schwab
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Landauer in the age of synthetic biology: energy consumption and information processing in biochemical networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J. Stat. Phys
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  162
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1153
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1166
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BPhys%26rft.volume%253D162%26rft.spage%253D1153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [23].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ouldridge
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Govern
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   ten Wolde
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Thermodynamics of computational copying in biochemical systems
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Phys. Rev. X
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  021004
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhys.%2BRev.%2BX%26rft.volume%253D7%26rft.spage%253D021004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [24].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.06.327700v3.24">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Milo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Phillips
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  Cell biology by the numbers
                                 </span>
                                 (
                                 <span class="cit-publ-name">
                                  Garland Science
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [25].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.25" data-doi="10.1016/j.cell.2012.11.050">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lane
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Martin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W. F.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The origin of membrane bioenergetics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  151
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1406
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1416
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D151%26rft.spage%253D1406%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.11.050%26rft_id%253Dinfo%253Apmid%252F23260134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.11.050&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23260134&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000312890300008&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [26].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference [26] in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.26" data-doi="10.1073/pnas.74.7.2761">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stadtman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chock
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Superiority of interconvertible enzyme cascades in metabolic regulation: analysis of monocyclic systems
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  74
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2761
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2765
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1977
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DStadtman%26rft.auinit1%253DE.%2BR.%26rft.volume%253D74%26rft.issue%253D7%26rft.spage%253D2761%26rft.epage%253D2765%26rft.atitle%253DSuperiority%2Bof%2Binterconvertible%2Benzyme%2Bcascades%2Bin%2Bmetabolic%2Bregulation%253A%2Banalysis%2Bof%2Bmonocyclic%2Bsystems.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.74.7.2761%26rft_id%253Dinfo%253Apmid%252F268625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3NC83LzI3NjEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNy8wNS8yMDIwLjEwLjA2LjMyNzcwMC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [27].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.06.327700v3.27" data-doi="10.1073/pnas.78.11.6840">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goldbeter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koshland
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. E.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  An amplified sensitivity arising from covalent modification in biological systems
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  78
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  6840
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6844
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1981
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DGoldbeter%26rft.auinit1%253DA.%26rft.volume%253D78%26rft.issue%253D11%26rft.spage%253D6840%26rft.epage%253D6844%26rft.atitle%253DAn%2Bamplified%2Bsensitivity%2Barising%2Bfrom%2Bcovalent%2Bmodification%2Bin%2Bbiological%2Bsystems.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.78.11.6840%26rft_id%253Dinfo%253Apmid%252F6947258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNzgvMTEvNjg0MCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA3LzA1LzIwMjAuMTAuMDYuMzI3NzAwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [28].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.28" data-doi="10.1016/S0006-3495(00)76519-2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Detwiler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ramanathan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sengupta
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shraiman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. I.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Engineering aspects of enzymatic signal transduction: photoreceptors in the retina
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biophys. J
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  79
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2801
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2817
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiophysical%2BJournal%26rft.stitle%253DBiophys.%2BJ%26rft.aulast%253DDetwiler%26rft.auinit1%253DP.%2BB.%26rft.volume%253D79%26rft.issue%253D6%26rft.spage%253D2801%26rft.epage%253D2817%26rft.atitle%253DEngineering%2BAspects%2Bof%2BEnzymatic%2BSignal%2BTransduction%253A%2BPhotoreceptors%2Bin%2Bthe%2BRetina%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0006-3495%252800%252976519-2%26rft_id%253Dinfo%253Apmid%252F11106590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/S0006-3495(00)76519-2&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=11106590&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000165749000003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [29].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.29" data-doi="10.1016/S1097-2765(02)00528-2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Heinrich
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Neel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. G.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rapoport
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Mathematical models of protein kinase signal transduction
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular cell
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  957
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  970
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DHeinrich%26rft.auinit1%253DR.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D957%26rft.epage%253D970%26rft.atitle%253DMathematical%2Bmodels%2Bof%2Bprotein%2Bkinase%2Bsignal%2Btransduction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252802%252900528-2%26rft_id%253Dinfo%253Apmid%252F12049733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/S1097-2765(02)00528-2&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=12049733&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000175967100008&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [30].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hinczewski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thirumalai
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Cellular signaling networks function as generalized wiener-kolmogorov filters to suppress noise
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Phys. Rev. X
                                 </abbr>
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  041017
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhys.%2BRev.%2BX%26rft.volume%253D4%26rft.spage%253D041017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [31].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.31">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hinczewski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thirumalai
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Noise control in gene regulatory networks with negative feedback
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J. Phys. Chem. B
                                 </abbr>
                                 <span class="cit-vol">
                                  120
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  6166
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6177
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ.%2BPhys.%2BChem.%2BB%26rft.volume%253D120%26rft.spage%253D6166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [32].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference [32] in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.32" data-doi="10.1073/pnas.1514974112">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lynch
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marinov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. K.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The bioenergetic costs of a gene
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  112
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  15690
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  15695
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1514974112%26rft_id%253Dinfo%253Apmid%252F26575626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzUxLzE1NjkwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDcvMDUvMjAyMC4xMC4wNi4zMjc3MDAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [33].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference [33] in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.33">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ilker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hinczewski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Modeling the growth of organisms validates a general relation between metabolic costs and natural selection
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Phys. Rev. Lett
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  122
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  238101
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhys.%2BRev.%2BLett%26rft.volume%253D122%26rft.spage%253D238101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [34].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [34] in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.34" data-doi="10.1126/scisignal.2001212">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sturm
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O. E.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The mammalian MAPK/ERK pathway exhibits properties of a negative feedback amplifier
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci. Signal
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  ra90
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  ra90
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%2BSignaling%26rft.stitle%253DSci%2BSignal%26rft.aulast%253DSturm%26rft.auinit1%253DO.%2BE.%26rft.volume%253D3%26rft.issue%253D153%26rft.spage%253Dra90%26rft.epage%253Dra90%26rft.atitle%253DThe%2BMammalian%2BMAPK%252FERK%2BPathway%2BExhibits%2BProperties%2Bof%2Ba%2BNegative%2BFeedback%2BAmplifier%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscisignal.2001212%26rft_id%253Dinfo%253Apmid%252F21177493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoic2lndHJhbnMiO3M6NToicmVzaWQiO3M6MTA6IjMvMTUzL3JhOTAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNy8wNS8yMDIwLjEwLjA2LjMyNzcwMC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [35].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [35] in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.35" data-doi="10.1038/13024">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Saxena
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Williams
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Taskén
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mustelin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Crosstalk between camp-dependent kinase and map kinase through a protein tyrosine phosphatase
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Cell Biol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  1
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  305
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1999
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DSaxena%26rft.auinit1%253DM.%26rft.volume%253D1%26rft.issue%253D5%26rft.spage%253D305%26rft.epage%253D311%26rft.atitle%253DCrosstalk%2Bbetween%2BcAMP-dependent%2Bkinase%2Band%2BMAP%2Bkinase%2Bthrough%2Ba%2Bprotein%2Btyrosine%2Bphosphatase.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F13024%26rft_id%253Dinfo%253Apmid%252F10559944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/13024&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10559944&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000083169600020&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [36].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference [36] in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.36" data-doi="10.1111/j.1742-4658.2009.07027.x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Salazar
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Höfer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Multisite protein phosphorylation–from molecular mechanisms to kinetic models
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  FEBS J
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  276
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3177
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3198
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFEBS%2BJournal%26rft.stitle%253DFEBS%2BJ%26rft.aulast%253DSalazar%26rft.auinit1%253DC.%26rft.volume%253D276%26rft.issue%253D12%26rft.spage%253D3177%26rft.epage%253D3198%26rft.atitle%253DMultisite%2Bprotein%2Bphosphorylation--from%2Bmolecular%2Bmechanisms%2Bto%2Bkinetic%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1742-4658.2009.07027.x%26rft_id%253Dinfo%253Apmid%252F19438722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/j.1742-4658.2009.07027.x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19438722&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [37].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference [37] in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.06.327700v3.37">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fersht
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  Structure and mechanism in protein science: a guide to enzyme catalysis and protein folding
                                 </span>
                                 (
                                 <span class="cit-publ-name">
                                  Macmillan
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1999
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [38].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference [38] in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.38" data-doi="10.1529/biophysj.108.131771">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Qian
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Cooperativity and specificity in enzyme kinetics: a single-molecule time-based per-spective
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biophys. J
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  95
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  10
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  17
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiophysical%2BJournal%26rft.stitle%253DBiophys.%2BJ%26rft.volume%253D95%26rft.issue%253D1%26rft.spage%253D10%26rft.epage%253D17%26rft.atitle%253DCooperativity%2Band%2Bspecificity%2Bin%2Benzyme%2Bkinetics%253A%2Ba%2Bsingle-molecule%2Btime-based%2Bperspective.%26rft_id%253Dinfo%253Adoi%252F10.1529%252Fbiophysj.108.131771%26rft_id%253Dinfo%253Apmid%252F18441030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1529/biophysj.108.131771&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18441030&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000256668200006&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [39].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference [39] in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.39" data-doi="10.1063/1.481811">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gillespie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. T.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The chemical Langevin equation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J. Chem. Phys
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  113
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  297
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  306
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ.%2BChem.%2BPhys%26rft.volume%253D113%26rft.spage%253D297%26rft_id%253Dinfo%253Adoi%252F10.1063%252F1.481811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1063/1.481811&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000087750800036&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [40].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference [40] in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.40" data-doi="10.1021/j100540a008">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gillespie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. T.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Exact stochastic simulation of coupled chemical reactions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J. Phys. Chem
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  81
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2340
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2361
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1977
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ.%2BPhys.%2BChem%26rft.volume%253D81%26rft.spage%253D2340%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fj100540a008%26rft_id%253Dinfo%253Apmid%252FA1977EE49800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1021/j100540a008&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1977EE49800&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1977EE49800008&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [41].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [41] in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.41" data-doi="10.1021/bi2002289">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bar-Even
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The moderately efficient enzyme: evolutionary and physicochemical trends shaping enzyme parameters
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biochemistry
                                 </abbr>
                                 <span class="cit-vol">
                                  50
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  4402
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4410
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DBiochemistry%26rft.aulast%253DBar-Even%26rft.auinit1%253DA.%26rft.volume%253D50%26rft.issue%253D21%26rft.spage%253D4402%26rft.epage%253D4410%26rft.atitle%253DThe%2Bmoderately%2Befficient%2Benzyme%253A%2Bevolutionary%2Band%2Bphysicochemical%2Btrends%2Bshaping%2Benzyme%2Bparameters.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi2002289%26rft_id%253Dinfo%253Apmid%252F21506553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1021/bi2002289&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21506553&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000290837400004&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [42].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference [42] in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.42">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liebermeister
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Klipp
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Biochemical networks with uncertain parameters
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  IEE Proc.-Syst. Biol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  152
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  97
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  107
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIEE%2BProc.-Syst.%2BBiol%26rft.volume%253D152%26rft.spage%253D97%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [43].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference [43] in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.43" data-doi="10.1002/pmic.201400441">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Herrmann
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Simonovic
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Szklarczyk
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   von Mering
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Version 4.0 of PaxDb: protein abundance data, integrated across model organisms, tissues, and cell-lines
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proteomics
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3163
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3168
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProteomics%26rft.volume%253D15%26rft.spage%253D3163%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpmic.201400441%26rft_id%253Dinfo%253Apmid%252F25656970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/pmic.201400441&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25656970&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [44].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference [44] in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.44" data-doi="10.1093/nar/gky1049">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  UniProt Consortium
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  UniProt: a worldwide hub of protein knowledge
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  47
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  D506
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D515
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D47%26rft.spage%253DD506%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky1049%26rft_id%253Dinfo%253Apmid%252F30395287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gky1049&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30395287&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [45].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference [45] in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.45">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wittig
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   U.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  SABIO-RK—database for biochemical reaction kinetics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  40
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  D790
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D796
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D40%26rft.spage%253DD790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [46].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference [46] in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.46" data-doi="10.1371/journal.pone.0009522">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liebermeister
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Klipp
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  A quantitative study of the Hog1 MAPK response to fluctuating osmotic stress in Saccharomyces cerevisiae
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLOS ONE
                                 </abbr>
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e9522
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DZi%26rft.auinit1%253DZ.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De9522%26rft.epage%253De9522%26rft.atitle%253DA%2Bquantitative%2Bstudy%2Bof%2Bthe%2BHog1%2BMAPK%2Bresponse%2Bto%2Bfluctuating%2Bosmotic%2Bstress%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0009522%26rft_id%253Dinfo%253Apmid%252F20209100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0009522&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20209100&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [47].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference [47] in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.47" data-doi="10.1073/pnas.0710770105">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hersen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McClean
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mahadevan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ramanathan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Signal processing by the hog map kinase pathway
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  105
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  7165
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  7170
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0710770105%26rft_id%253Dinfo%253Apmid%252F18480263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzIwLzcxNjUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNy8wNS8yMDIwLjEwLjA2LjMyNzcwMC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [48].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference [48] in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.06.327700v3.48">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wiener
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  Extrapolation, Interpolation and Smoothing of Stationary Times Series
                                 </span>
                                 (
                                 <span class="cit-publ-name">
                                  Wiley
                                 </span>
                                 ,
                                 <span class="cit-publ-loc">
                                  New York
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1949
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [49].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.06.327700v3.49">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kolmogorov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. N.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Interpolation and extrapolation of stationary random sequences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Izv. Akad. Nauk SSSR., Ser. Mat
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  14
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1941
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIzv.%2BAkad.%2BNauk%2BSSSR.%252C%2BSer.%2BMat%26rft.volume%253D5%26rft.spage%253D3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [50].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference [50] in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.50">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bode
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H. W.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shannon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. E.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  A simplified derivation of linear least square smoothing and prediction theory
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Inst. Radio. Engin
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  38
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  417
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  425
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1950
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BInst.%2BRadio.%2BEngin%26rft.volume%253D38%26rft.spage%253D417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [51].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference [51] in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.51">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Becker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N. B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mugler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   ten Wolde
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Optimal prediction by cellular signaling networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Phys. Rev. Lett
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  115
                                 </span>
                                 (
                                 <span class="cit-fpage">
                                  258103
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhys.%2BRev.%2BLett%26rft.volume%253D115%26rft.spage%253D258103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [52].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.06.327700v3.52" data-doi="10.1073/pnas.1517109113">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zechner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Seelig
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rullan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Khammash
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Molecular circuits for dynamic noise filtering
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  113
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  4729
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4734
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1517109113%26rft_id%253Dinfo%253Apmid%252F27078094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzE3LzQ3MjkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNy8wNS8yMDIwLjEwLjA2LjMyNzcwMC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [53].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.06.327700v3.53">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Samanta
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H. S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hinczewski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thirumalai
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Optimal information transfer in enzymatic networks: A field theoretic formulation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Phys. Rev. E
                                 </abbr>
                                 <span class="cit-vol">
                                  96
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  012406
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhys.%2BRev.%2BE%26rft.volume%253D96%26rft.spage%253D012406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [54].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference [54] in text" id="ref-54">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.54">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hathcock
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sheehy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weisenberger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ilker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hinczewski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Noise filtering and prediction in biological signaling networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  IEEE Trans. Mol. Biol. Multi-Scale Commun
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  16
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  30
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BMol.%2BBiol.%2BMulti-Scale%2BCommun%26rft.volume%253D2%26rft.spage%253D16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [55].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference [55] in text" id="ref-55">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.55">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nemenman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Gain control in molecular information processing: lessons from neuroscience
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Phys. Biol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  026003
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DPhys%2BBiol%26rft.aulast%253DNemenman%26rft.auinit1%253DI.%26rft.volume%253D9%26rft.issue%253D2%26rft.spage%253D026003%26rft.epage%253D026003%26rft.atitle%253DGain%2Bcontrol%2Bin%2Bmolecular%2Binformation%2Bprocessing%253A%2Blessons%2Bfrom%2Bneuroscience.%26rft_id%253Dinfo%253Apmid%252F22475952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22475952&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [56].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference [56] in text" id="ref-56">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.56">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mancini
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wiggins
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marsili
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Walczak
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Time-dependent information transmission in a model regulatory circuit
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Phys. Rev. E
                                 </abbr>
                                 <span class="cit-vol">
                                  88
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  022708
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhys.%2BRev.%2BE%26rft.volume%253D88%26rft.spage%253D022708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [57].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference [57] in text" id="ref-57">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.57" data-doi="10.1126/science.1151582">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mettetal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Muzzey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gómez-Uribe
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   van Oudenaarden
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The frequency dependence of osmo-adaptation in Saccharomyces cerevisiae
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  319
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  482
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  484
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMettetal%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D319%26rft.issue%253D5862%26rft.spage%253D482%26rft.epage%253D484%26rft.atitle%253DThe%2BFrequency%2BDependence%2Bof%2BOsmo-Adaptation%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1151582%26rft_id%253Dinfo%253Apmid%252F18218902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTkvNTg2Mi80ODIiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNy8wNS8yMDIwLjEwLjA2LjMyNzcwMC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [58].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference [58] in text" id="ref-58">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.58" data-doi="10.1038/nrg2526">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Charlesworth
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Effective population size and patterns of molecular evolution and variation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Rev. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  195
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D195%26rft.epage%253D205%26rft.atitle%253DFundamental%2Bconcepts%2Bin%2Bgenetics%253A%2Beffective%2Bpopulation%2Bsize%2Band%2Bpatterns%2Bof%2Bmolecular%2Bevolution%2Band%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2526%26rft_id%253Dinfo%253Apmid%252F19204717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrg2526&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19204717&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000263457000012&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [59].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference [59] in text" id="ref-59">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.06.327700v3.59">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gillespie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. H.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  Population genetics: a concise guide
                                 </span>
                                 (
                                 <span class="cit-publ-name">
                                  JHU Press
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [60].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference [60] in text" id="ref-60">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.60">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kimura
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  On the probability of fixation of mutant genes in a population
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics
                                 </abbr>
                                 <span class="cit-vol">
                                  47
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  713
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  719
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1962
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKimura%26rft.auinit1%253DM.%26rft.volume%253D47%26rft.issue%253D6%26rft.spage%253D713%26rft.epage%253D719%26rft.atitle%253DON%2BTHE%2BPROBABILITY%2BOF%2BFIXATION%2BOF%2BMUTANT%2BGENES%2BIN%2BA%2BPOPULATION%26rft_id%253Dinfo%253Apmid%252F14456043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI0Ny82LzcxMyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA3LzA1LzIwMjAuMTAuMDYuMzI3NzAwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [61].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference [61] in text" id="ref-61">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.61" data-doi="10.1038/284604a0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Orgel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L. E.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Crick
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F. H.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Selfish DNA: the ultimate parasite
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  284
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  604
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1980
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DOrgel%26rft.auinit1%253DL.%2BE.%26rft.volume%253D284%26rft.issue%253D5757%26rft.spage%253D604%26rft.epage%253D607%26rft.atitle%253DSelfish%2BDNA%253A%2Bthe%2Bultimate%2Bparasite.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F284604a0%26rft_id%253Dinfo%253Apmid%252F7366731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/284604a0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=7366731&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1980JN44200031&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [62].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference [62] in text" id="ref-62">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.62" data-doi="10.1093/molbev/msi126">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wagner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Energy constraints on the evolution of gene expression
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol. Biol. Evol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1365
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1374
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi126%26rft_id%253Dinfo%253Apmid%252F15758206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/msi126&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15758206&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000229279100001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [63].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference [63] in text" id="ref-63">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.63" data-doi="10.1073/pnas.1701670114">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mahmoudabadi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Milo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Phillips
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Energetic cost of building a virus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  114
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  E4324
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  E4333
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1701670114%26rft_id%253Dinfo%253Apmid%252F28512219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzIyL0U0MzI0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDcvMDUvMjAyMC4xMC4wNi4zMjc3MDAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [64].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference [64] in text" id="ref-64">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.64" data-doi="10.1073/pnas.0707314105">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tsai
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bensasson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koufopanou
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Population genomics of the wild yeast Saccharomyces paradoxus: quantifying the life cycle
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  105
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  4957
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4962
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0707314105%26rft_id%253Dinfo%253Apmid%252F18344325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzEyLzQ5NTciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNy8wNS8yMDIwLjEwLjA2LjMyNzcwMC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <div class="section ref-list" id="ref-list-2">
                            <h2>
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               [1].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference [1] in text" id="ref-65">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.65" data-doi="10.1529/biophysj.108.131771">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Qian
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Cooperativity and specificity in enzyme kinetics: a single-molecule time-based perspective
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biophys. J
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  95
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  10
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  17
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiophysical%2BJournal%26rft.stitle%253DBiophys.%2BJ%26rft.volume%253D95%26rft.issue%253D1%26rft.spage%253D10%26rft.epage%253D17%26rft.atitle%253DCooperativity%2Band%2Bspecificity%2Bin%2Benzyme%2Bkinetics%253A%2Ba%2Bsingle-molecule%2Btime-based%2Bperspective.%26rft_id%253Dinfo%253Adoi%252F10.1529%252Fbiophysj.108.131771%26rft_id%253Dinfo%253Apmid%252F18441030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1529/biophysj.108.131771&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18441030&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000256668200006&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [2].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference [2] in text" id="ref-66">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.66">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hinczewski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thirumalai
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Cellular signaling networks function as generalized Wiener-Kolmogorov filters to suppress noise
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Phys. Rev. X
                                 </abbr>
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  041017
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhys.%2BRev.%2BX%26rft.volume%253D4%26rft.spage%253D041017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [3].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference [3] in text" id="ref-67">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.67" data-doi="10.1063/1.481811">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gillespie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. T.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The chemical Langevin equation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J. Chem. Phys
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  113
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  297
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  306
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ.%2BChem.%2BPhys%26rft.volume%253D113%26rft.spage%253D297%26rft_id%253Dinfo%253Adoi%252F10.1063%252F1.481811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1063/1.481811&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000087750800036&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [4].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference [4] in text" id="ref-68">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.68" data-doi="10.1021/j100540a008">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gillespie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. T.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Exact stochastic simulation of coupled chemical reactions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J. Phys. Chem
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  81
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2340
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2361
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1977
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ.%2BPhys.%2BChem%26rft.volume%253D81%26rft.spage%253D2340%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fj100540a008%26rft_id%253Dinfo%253Apmid%252FA1977EE49800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1021/j100540a008&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1977EE49800&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1977EE49800008&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [5].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference [5] in text" id="ref-69">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.69">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hathcock
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sheehy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weisenberger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ilker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hinczewski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Noise filtering and prediction in biological signaling networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  IEEE Trans. Mol. Biol. Multi-Scale Commun
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  16
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  30
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BMol.%2BBiol.%2BMulti-Scale%2BCommun%26rft.volume%253D2%26rft.spage%253D16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [6].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference [6] in text" id="ref-70">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.70" data-doi="10.1021/bi2002289">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bar-Even
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The moderately efficient enzyme: evolutionary and physicochemical trends shaping enzyme parameters
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biochemistry
                                 </abbr>
                                 <span class="cit-vol">
                                  50
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  4402
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4410
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DBiochemistry%26rft.aulast%253DBar-Even%26rft.auinit1%253DA.%26rft.volume%253D50%26rft.issue%253D21%26rft.spage%253D4402%26rft.epage%253D4410%26rft.atitle%253DThe%2Bmoderately%2Befficient%2Benzyme%253A%2Bevolutionary%2Band%2Bphysicochemical%2Btrends%2Bshaping%2Benzyme%2Bparameters.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi2002289%26rft_id%253Dinfo%253Apmid%252F21506553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1021/bi2002289&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21506553&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000290837400004&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [7].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference [7] in text" id="ref-71">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.71">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liebermeister
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Klipp
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Biochemical networks with uncertain parameters
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  IEE Proc.-Syst. Biol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  152
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  97
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  107
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIEE%2BProc.-Syst.%2BBiol%26rft.volume%253D152%26rft.spage%253D97%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [8].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference [8] in text" id="ref-72">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.72" data-doi="10.1002/pmic.201400441">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Herrmann
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Simonovic
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Szklarczyk
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   von Mering
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Version 4.0 of PaxDb: protein abundance data, integrated across model organisms, tissues, and cell-lines
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proteomics
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3163
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3168
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProteomics%26rft.volume%253D15%26rft.spage%253D3163%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpmic.201400441%26rft_id%253Dinfo%253Apmid%252F25656970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/pmic.201400441&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25656970&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [9].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference [9] in text" id="ref-73">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.73">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wittig
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   U.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  SABIO-RK—database for biochemical reaction kinetics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  40
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  D790
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D796
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D40%26rft.spage%253DD790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [10].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference [10] in text" id="ref-74">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.74" data-doi="10.1093/nar/gky1049">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  UniProt Consortium
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  UniProt: a worldwide hub of protein knowledge
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  47
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  D506
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D515
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D47%26rft.spage%253DD506%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky1049%26rft_id%253Dinfo%253Apmid%252F30395287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gky1049&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30395287&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [11].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference [11] in text" id="ref-75">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.75" data-doi="10.1002/bies.201300066">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Milo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  What is the total number of protein molecules per cell volume? a call to rethink some published values
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioessays
                                 </abbr>
                                 <span class="cit-vol">
                                  35
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1050
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1055
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioessays%26rft.volume%253D35%26rft.spage%253D1050%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.201300066%26rft_id%253Dinfo%253Apmid%252F24114984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/bies.201300066&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24114984&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [12].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference [12] in text" id="ref-76">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.76" data-doi="10.1038/nbt0402-370">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schoeberl
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eichler-Jonsson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gilles
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. D.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Müller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Computational modeling of the dynamics of the map kinase cascade activated by surface and internalized EGF receptors
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Biotech
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  370
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  375
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DSchoeberl%26rft.auinit1%253DB.%26rft.volume%253D20%26rft.issue%253D4%26rft.spage%253D370%26rft.epage%253D375%26rft.atitle%253DComputational%2Bmodeling%2Bof%2Bthe%2Bdynamics%2Bof%2Bthe%2BMAP%2Bkinase%2Bcascade%2Bactivated%2Bby%2Bsurface%2Band%2Binternalized%2BEGF%2Breceptors.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt0402-370%26rft_id%253Dinfo%253Apmid%252F11923843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nbt0402-370&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=11923843&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000174681900024&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [13].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference [13] in text" id="ref-77">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.77" data-doi="10.1371/journal.pone.0009522">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liebermeister
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Klipp
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  A quantitative study of the Hog1 MAPK response to fluctuating osmotic stress in Saccharomyces cerevisiae
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLOS ONE
                                 </abbr>
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e9522
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DZi%26rft.auinit1%253DZ.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De9522%26rft.epage%253De9522%26rft.atitle%253DA%2Bquantitative%2Bstudy%2Bof%2Bthe%2BHog1%2BMAPK%2Bresponse%2Bto%2Bfluctuating%2Bosmotic%2Bstress%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0009522%26rft_id%253Dinfo%253Apmid%252F20209100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0009522&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20209100&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [14].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference [14] in text" id="ref-78">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.78" data-doi="10.1126/science.1151582">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mettetal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Muzzey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gómez-Uribe
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   van Oudenaarden
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The frequency dependence of osmo-adaptation in saccharomyces cerevisiae
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  319
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  482
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  484
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMettetal%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D319%26rft.issue%253D5862%26rft.spage%253D482%26rft.epage%253D484%26rft.atitle%253DThe%2BFrequency%2BDependence%2Bof%2BOsmo-Adaptation%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1151582%26rft_id%253Dinfo%253Apmid%252F18218902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTkvNTg2Mi80ODIiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNy8wNS8yMDIwLjEwLjA2LjMyNzcwMC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [15].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference [15] in text" id="ref-79">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.79" data-doi="10.1073/pnas.0710770105">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hersen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McClean
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mahadevan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ramanathan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Signal processing by the HOG MAP kinase pathway
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  105
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  7165
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  7170
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0710770105%26rft_id%253Dinfo%253Apmid%252F18480263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzIwLzcxNjUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNy8wNS8yMDIwLjEwLjA2LjMyNzcwMC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [16].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference [16] in text" id="ref-80">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.80" data-doi="10.1101/gad.14.10.1229">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mattison
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. P.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ota
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I. M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Two protein tyrosine phosphatases, Ptp2 and Ptp3, modulate the subcellular localization of the Hog1 MAP kinase in yeast
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genes Dev
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1229
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1235
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DMattison%26rft.auinit1%253DC.%2BP.%26rft.volume%253D14%26rft.issue%253D10%26rft.spage%253D1229%26rft.epage%253D1235%26rft.atitle%253DTwo%2Bprotein%2Btyrosine%2Bphosphatases%252C%2BPtp2%2Band%2BPtp3%252C%2Bmodulate%2Bthe%2Bsubcellular%2Blocalization%2Bof%2Bthe%2BHog1%2BMAP%2Bkinase%2Bin%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.14.10.1229%26rft_id%253Dinfo%253Apmid%252F10817757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE0LzEwLzEyMjkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNy8wNS8yMDIwLjEwLjA2LjMyNzcwMC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [17].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference [17] in text" id="ref-81">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.06.327700v3.81">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fersht
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  Structure and mechanism in protein science: a guide to enzyme catalysis and protein folding
                                 </span>
                                 (
                                 <span class="cit-publ-name">
                                  Macmillan
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1999
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [18].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference [18] in text" id="ref-82">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.82" data-doi="10.1038/nature02046">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ghaemmaghami
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Global analysis of protein expression in yeast
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  425
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  737
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  741
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGhaemmaghami%26rft.auinit1%253DS.%26rft.volume%253D425%26rft.issue%253D6959%26rft.spage%253D737%26rft.epage%253D741%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bprotein%2Bexpression%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02046%26rft_id%253Dinfo%253Apmid%252F14562106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature02046&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=14562106&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F05%2F2020.10.06.327700.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000185924500046&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [19].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference [19] in text" id="ref-83">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.06.327700v3.83">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Milo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Phillips
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  Cell biology by the numbers
                                 </span>
                                 (
                                 <span class="cit-publ-name">
                                  Garland Science
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [20].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference [20] in text" id="ref-84">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.06.327700v3.84">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ilker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hinczewski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-chapter-title">
                                  Modeling the growth of organisms validates a general relation between metabolic costs and natural selection
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Phys. Rev. Lett
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  122
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  238101
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhys.%2BRev.%2BLett%26rft.volume%253D122%26rft.spage%253D238101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [21].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference [21] in text" id="ref-85">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.06.327700v3.85" data-doi="10.1073/pnas.1514974112">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lynch
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marinov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. K.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The bioenergetic costs of a gene
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  112
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  15690
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  15695
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1514974112%26rft_id%253Dinfo%253Apmid%252F26575626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzUxLzE1NjkwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDcvMDUvMjAyMC4xMC4wNi4zMjc3MDAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.10.06.327700v3" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.06.29.498117v1" title="COVID-19 Neuropathology: evidence for SARS-CoV-2 invasion of Human Brainstem Nuclei" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.03.10.483885v1" title="Cancer related subarachnoid hemorrhage: a multicenter retrospective study using propensity score matching analysis" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted July 05, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.10.06.327700v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/07/05/2020.10.06.327700.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.10.06.327700v3.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.10.06.327700v3" class="active" data-icon-position="" data-hide-link-title="0">
                                   The price of a bit: energetic costs and the evolution of cellular signaling
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2629145"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-syvneV-k2Fxt4yx612H-uii6e8JayDa34bBDBN2hlU0"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484481606"/>
                                   <input type="hidden" name="captcha_token" value="9c10dd894f4dec9a3978068afbee43a0"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2629145--3" data-node-nid="2629145" id="node-2629145--456005570" data-pisa="biorxiv;2020.10.06.327700v3" data-pisa-master="biorxiv;2020.10.06.327700" data-seqnum="2629145" data-apath="/biorxiv/early/2022/07/05/2020.10.06.327700.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 The price of a bit: energetic costs and the evolution of cellular signaling
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Teng-Long
                                  </span>
                                  <span class="nlm-surname">
                                   Wang
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Benjamin
                                  </span>
                                  <span class="nlm-surname">
                                   Kuznets-Speck
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Joseph
                                  </span>
                                  <span class="nlm-surname">
                                   Broderick
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Michael
                                  </span>
                                  <span class="nlm-surname">
                                   Hinczewski
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.10.06.327700;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.10.06.327700
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.10.06.327700v3" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.10.06.327700v3&title=The%20price%20of%20a%20bit%3A%20energetic%20costs%20and%20the%20evolution%20of%20cellular%20signaling" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.10.06.327700v3&title=The%20price%20of%20a%20bit%3A%20energetic%20costs%20and%20the%20evolution%20of%20cellular%20signaling" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.10.06.327700v3&text=The%20price%20of%20a%20bit%3A%20energetic%20costs%20and%20the%20evolution%20of%20cellular%20signaling" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.10.06.327700v3&t=The%20price%20of%20a%20bit%3A%20energetic%20costs%20and%20the%20evolution%20of%20cellular%20signaling" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.10.06.327700v3&title=The%20price%20of%20a%20bit%3A%20energetic%20costs%20and%20the%20evolution%20of%20cellular%20signaling" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.10.06.327700v3&title=The%20price%20of%20a%20bit%3A%20energetic%20costs%20and%20the%20evolution%20of%20cellular%20signaling&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.10.06.327700v3&title=The%20price%20of%20a%20bit%3A%20energetic%20costs%20and%20the%20evolution%20of%20cellular%20signaling" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2629145--5" data-node-nid="2629145" id="citation-node-2629145--61568538315" data-pisa="biorxiv;2020.10.06.327700v3" data-pisa-master="biorxiv;2020.10.06.327700" data-seqnum="2629145" data-apath="/biorxiv/early/2022/07/05/2020.10.06.327700.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     The price of a bit: energetic costs and the evolution of cellular signaling
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Teng-Long
                                      </span>
                                      <span class="nlm-surname">
                                       Wang
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Benjamin
                                      </span>
                                      <span class="nlm-surname">
                                       Kuznets-Speck
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Joseph
                                      </span>
                                      <span class="nlm-surname">
                                       Broderick
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Michael
                                      </span>
                                      <span class="nlm-surname">
                                       Hinczewski
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.10.06.327700;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.10.06.327700
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=The%20price%20of%20a%20bit%3A%20energetic%20costs%20and%20the%20evolution%20of%20cellular%20signaling&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Wang%2C+Teng-Long&rft.au=Kuznets-Speck%2C+Benjamin&rft.au=Broderick%2C+Joseph&rft.au=Hinczewski%2C+Michael">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2629145/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2629145/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2629145/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2629145/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2629145/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2629145/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2629145/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2629145/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2629145/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2629145/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2629145/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2629145/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.10.06.327700v3&count=horizontal&via=&text=The%20price%20of%20a%20bit%3A%20energetic%20costs%20and%20the%20evolution%20of%20cellular%20signaling&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.10.06.327700v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.10.06.327700v3&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.10.06.327700v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/systems-biology" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Systems Biology
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23507)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11824)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11440)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15189)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47703)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
