<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.11.03.366252v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.11.03.366252v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.11.03.366252v2.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/04/24/2020.11.03.366252.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.11.03.366252v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Copy number-aware deconvolution of tumor-normal DNA methylation profiles"/>
  <meta name="DC.Identifier" content="10.1101/2020.11.03.366252"/>
  <meta name="DC.Date" content="2022-04-24"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Aberrant methylation is a hallmark of cancer, but bulk tumor data is confounded by admixed normal cells and copy number changes. Here, we introduce Copy number-Aware Methylation Deconvolution Analysis of Cancers (CAMDAC; <https://github.com/VanLoo-lab/CAMDAC>), which outputs tumor purity, allele-specific copy number and deconvolved methylation estimates. We apply CAMDAC to 122 multi-region samples from 38 TRACERx non-small cell lung cancers profiled by reduced representation bisulfite sequencing. CAMDAC copy number profiles parallel those derived from genome sequencing and highlight widespread chromosomal instability. Deconvolved polymorphism-independent methylation rates enable unbiased tumor-normal and tumor-tumor differential methylation calling. Read-phasing validates CAMDAC methylation rates and directly links genotype and epitype. We show increased epigenetic instability in adenocarcinoma vs. squamous cell carcinoma, frequent hypermethylation at sites carrying somatic mutations, and parallel copy number losses and methylation changes at imprinted loci. Unlike bulk methylomes, CAMDAC profiles recapitulate tumor phylogenies and evidence distinct patterns of epigenetic heterogeneity in lung cancer.

### Competing Interest Statement

E.L.C. and G.A.W. are currently working for and hold shares of Achilles Therapeutics. C.S. acknowledges grant support from Pfizer, AstraZeneca, Bristol Myers Squibb, Roche-Ventana, Boehringer-Ingelheim, Archer Dx Inc (collaboration in minimal residual disease sequencing technologies) and Ono Pharmaceutical, is an AstraZeneca Advisory Board member and Chief Investigator for the MeRmaiD1 clinical trial, has consulted for Pfizer, Novartis, GlaxoSmithKline, MSD, Bristol Myers Squibb, Celgene, AstraZeneca, Illumina, Genentech, Roche-Ventana, GRAIL, Medicxi, Bicycle Therapeutics, and the Sarah Cannon Research Institute, has stock options in Apogen Biotechnologies, Epic Bioscience, GRAIL, and is co-founder and has shares of Achilles Therapeutics. M.J.-H. has consulted, and is a member of the Scientific Advisory Board and Steering Committee, for Achilles Therapeutics, has received speaker honoraria from Astex Pharmaceuticals, Oslo Cancer Cluster, and holds a patent PCT/US2017/028013 relating to methods for lung cancer detection. All other authors declare no competing interests."/>
  <meta name="DC.Contributor" content="Elizabeth Larose Cadieux"/>
  <meta name="DC.Contributor" content="Nana E. Mensah"/>
  <meta name="DC.Contributor" content="Carla Castignani"/>
  <meta name="DC.Contributor" content="Miljana Tanić"/>
  <meta name="DC.Contributor" content="Gareth A. Wilson"/>
  <meta name="DC.Contributor" content="Michelle Dietzen"/>
  <meta name="DC.Contributor" content="Pawan Dhami"/>
  <meta name="DC.Contributor" content="Heli Vaikkinen"/>
  <meta name="DC.Contributor" content="Annelien Verfaillie"/>
  <meta name="DC.Contributor" content="Cristina Cotobal Martin"/>
  <meta name="DC.Contributor" content="Toby Baker"/>
  <meta name="DC.Contributor" content="Thomas B. K. Watkins"/>
  <meta name="DC.Contributor" content="Selvaraju Veeriah"/>
  <meta name="DC.Contributor" content="Mariam Jamal-Hanjani"/>
  <meta name="DC.Contributor" content="Nnennaya Kanu"/>
  <meta name="DC.Contributor" content="Nicholas McGranahan"/>
  <meta name="DC.Contributor" content="Andrew Feber"/>
  <meta name="DC.Contributor" content="TRACERx Consortium"/>
  <meta name="DC.Contributor" content="Charles Swanton"/>
  <meta name="DC.Contributor" content="Stephan Beck"/>
  <meta name="DC.Contributor" content="Jonas Demeulemeester"/>
  <meta name="DC.Contributor" content="Peter Van Loo"/>
  <meta name="article:published_time" content="2022-04-24"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Copy number-aware deconvolution of tumor-normal DNA methylation profiles"/>
  <meta name="citation_abstract" lang="en" content="<h3>SUMMARY</h3>
<p>Aberrant methylation is a hallmark of cancer, but bulk tumor data is confounded by admixed normal cells and copy number changes. Here, we introduce Copy number-Aware Methylation Deconvolution Analysis of Cancers (CAMDAC; https://github.com/VanLoo-lab/CAMDAC), which outputs tumor purity, allele-specific copy number and deconvolved methylation estimates. We apply CAMDAC to 122 multi-region samples from 38 TRACERx non-small cell lung cancers profiled by reduced representation bisulfite sequencing. CAMDAC copy number profiles parallel those derived from genome sequencing and highlight widespread chromosomal instability. Deconvolved polymorphism-independent methylation rates enable unbiased tumor-normal and tumor-tumor differential methylation calling. Read-phasing validates CAMDAC methylation rates and directly links genotype and epitype. We show increased epigenetic instability in adenocarcinoma <i>vs.</i> squamous cell carcinoma, frequent hypermethylation at sites carrying somatic mutations, and parallel copy number losses and methylation changes at imprinted loci. Unlike bulk methylomes, CAMDAC profiles recapitulate tumor phylogenies and evidence distinct patterns of epigenetic heterogeneity in lung cancer.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.11.03.366252v2"/>
  <meta name="citation_id" content="2020.11.03.366252v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.11.03.366252v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.11.03.366252v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.11.03.366252v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.11.03.366252"/>
  <meta name="citation_num_pages" content="99"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.11.03.366252"/>
  <meta name="citation_author" content="Elizabeth Larose Cadieux"/>
  <meta name="citation_author_institution" content="The Francis Crick Institute"/>
  <meta name="citation_author_institution" content="University College London Cancer Institute"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9320-8040"/>
  <meta name="citation_author" content="Nana E. Mensah"/>
  <meta name="citation_author_institution" content="The Francis Crick Institute"/>
  <meta name="citation_author_institution" content="University College London Cancer Institute"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0832-2816"/>
  <meta name="citation_author" content="Carla Castignani"/>
  <meta name="citation_author_institution" content="The Francis Crick Institute"/>
  <meta name="citation_author_institution" content="University College London Cancer Institute"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8689-9659"/>
  <meta name="citation_author" content="Miljana Tanić"/>
  <meta name="citation_author_institution" content="University College London Cancer Institute"/>
  <meta name="citation_author_institution" content="Institute for Oncology and Radiology of Serbia"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3102-776X"/>
  <meta name="citation_author" content="Gareth A. Wilson"/>
  <meta name="citation_author_institution" content="The Francis Crick Institute"/>
  <meta name="citation_author_institution" content="University College London Cancer Institute"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5703-9710"/>
  <meta name="citation_author" content="Michelle Dietzen"/>
  <meta name="citation_author_institution" content="The Francis Crick Institute"/>
  <meta name="citation_author_institution" content="University College London Cancer Institute"/>
  <meta name="citation_author_institution" content="Cancer Research UK Lung Cancer Centre of Excellence, University College London Cancer Institute"/>
  <meta name="citation_author" content="Pawan Dhami"/>
  <meta name="citation_author_institution" content="University College London Cancer Institute"/>
  <meta name="citation_author_institution" content="NIHR Biomedical Research Centre Guy’s and St Thomas’ NHS Foundation Trust"/>
  <meta name="citation_author" content="Heli Vaikkinen"/>
  <meta name="citation_author_institution" content="University College London Cancer Institute"/>
  <meta name="citation_author_institution" content="NIHR Biomedical Research Centre Guy’s and St Thomas’ NHS Foundation Trust"/>
  <meta name="citation_author" content="Annelien Verfaillie"/>
  <meta name="citation_author_institution" content="The Francis Crick Institute"/>
  <meta name="citation_author" content="Cristina Cotobal Martin"/>
  <meta name="citation_author_institution" content="The Francis Crick Institute"/>
  <meta name="citation_author" content="Toby Baker"/>
  <meta name="citation_author_institution" content="The Francis Crick Institute"/>
  <meta name="citation_author" content="Thomas B. K. Watkins"/>
  <meta name="citation_author_institution" content="The Francis Crick Institute"/>
  <meta name="citation_author" content="Selvaraju Veeriah"/>
  <meta name="citation_author_institution" content="University College London Cancer Institute"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1217-0387"/>
  <meta name="citation_author" content="Mariam Jamal-Hanjani"/>
  <meta name="citation_author_institution" content="Cancer Research UK Lung Cancer Centre of Excellence, University College London Cancer Institute"/>
  <meta name="citation_author_institution" content="Cancer Metastasis Laboratory, University College London Cancer Institute"/>
  <meta name="citation_author_institution" content="Department of Medical Oncology, University College London Hospitals"/>
  <meta name="citation_author" content="Nnennaya Kanu"/>
  <meta name="citation_author_institution" content="University College London Cancer Institute"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7232-1952"/>
  <meta name="citation_author" content="Nicholas McGranahan"/>
  <meta name="citation_author_institution" content="University College London Cancer Institute"/>
  <meta name="citation_author_institution" content="Cancer Research UK Lung Cancer Centre of Excellence, University College London Cancer Institute"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9537-4045"/>
  <meta name="citation_author" content="Andrew Feber"/>
  <meta name="citation_author_institution" content="Translational Epigenetics, Division of Molecular Pathology, Institute for Cancer Research"/>
  <meta name="citation_author_institution" content="Translational Research Laboratory, Centre for Molecular Pathology, Royal Marsden NHS Trust"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5282-0498"/>
  <meta name="citation_author" content="TRACERx Consortium"/>
  <meta name="citation_author" content="Charles Swanton"/>
  <meta name="citation_author_institution" content="The Francis Crick Institute"/>
  <meta name="citation_author_institution" content="Cancer Research UK Lung Cancer Centre of Excellence, University College London Cancer Institute"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4299-3018"/>
  <meta name="citation_author" content="Stephan Beck"/>
  <meta name="citation_author_institution" content="University College London Cancer Institute"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5290-2151"/>
  <meta name="citation_author" content="Jonas Demeulemeester"/>
  <meta name="citation_author_institution" content="The Francis Crick Institute"/>
  <meta name="citation_author_institution" content="Department of Human Genetics, University of Leuven"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2660-2478"/>
  <meta name="citation_author" content="Peter Van Loo"/>
  <meta name="citation_author_institution" content="The Francis Crick Institute"/>
  <meta name="citation_author_institution" content="Department of Genetics, The University of Texas MD Anderson Cancer Center"/>
  <meta name="citation_author_institution" content="Department of Genomic Medicine, The University of Texas MD Anderson Cancer Center"/>
  <meta name="citation_author_email" content="Peter.VanLoo@crick.ac.uk"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0292-1949"/>
  <meta name="citation_reference" content="Alexandrov, L.B., Nik-Zainal, S., Wedge, D.C., Aparicio, S. a J.R., Behjati, S., Biankin, A. V, Bignell, G.R., Bolli, N., Borg, A., Børresen-Dale, A.-L., et al. (2013). Signatures of mutational processes in human cancer. Nature 500, 415–421."/>
  <meta name="citation_reference" content="Alves, J.M., Prieto, T., and Posada, D. (2017). Multiregional Tumor Trees Are Not Phylogenies. Trends in Cancer 3, 546–550."/>
  <meta name="citation_reference" content="Auton, A., Abecasis, G.R., Altshuler, D.M., Durbin, R.M., Bentley, D.R., Chakravarti, A., Clark, A.G., Donnelly, P., Eichler, E.E., Flicek, P., et al. (2015). A global reference for human genetic variation. Nature 526, 68–74."/>
  <meta name="citation_reference" content="Barrett, J.E., Feber, A., Herrero, J., Tanic, M., Wilson, G.A., Swanton, C., and Beck, S. (2017). Quantification of tumour evolution and heterogeneity via Bayesian epiallele detection. BMC Bioinformatics 18, 354."/>
  <meta name="citation_reference" content="Baylin, S.B., and Jones, P.A. (2016). Epigenetic Determinants of Cancer. Cold Spring Harb. Perspect. Biol. 8, a019505."/>
  <meta name="citation_reference" content="Benjamini, Y., and Speed, T.P. (2012). Summarizing and correcting the GC content bias in high-throughput sequencing. Nucleic Acids Res. 40, e72–e72."/>
  <meta name="citation_reference" content="Bird, A. (2002). DNA methylation patterns and epigenetic memory. Genes Dev. 16, 6–21."/>
  <meta name="citation_reference" content="Brocks, D., Assenov, Y., Minner, S., Bogatyrova, O., Simon, R., Koop, C., Oakes, C., Zucknick, M., Lipka, D.B., Weischenfeldt, J., et al. (2014). Intratumor DNA Methylation Heterogeneity Reflects Clonal Evolution in Aggressive Prostate Cancer. Cell Rep. 8, 798–806."/>
  <meta name="citation_reference" content="Capper, D., Jones, D.T.W., Sill, M., Hovestadt, V., Schrimpf, D., Sturm, D., Koelsche, C., Sahm, F., Chavez, L., Reuss, D.E., et al. (2018). DNA methylation-based classification of central nervous system tumours. Nature 555, 469–474."/>
  <meta name="citation_reference" content="Carter, S.L., Cibulskis, K., Helman, E., McKenna, A., Shen, H., Zack, T., Laird, P.W., Onofrio, R.C., Winckler, W., Weir, B.A., et al. (2012). Absolute quantification of somatic DNA alterations in human cancer. Nat. Biotechnol. 30, 413–421."/>
  <meta name="citation_reference" content="Chakravarthy, A., Furness, A., Joshi, K., Ghorani, E., Ford, K., Ward, M.J., King, E. V., Lechner, M., Marafioti, T., Quezada, S.A., et al. (2018). Pan-cancer deconvolution of tumour composition using DNA methylation. Nat. Commun. 9, 3220."/>
  <meta name="citation_reference" content="Chen, X., Ashoor, H., Musich, R., Wang, J., Zhang, M., Zhang, C., Lu, M., and Li, S. (2021). epihet for intra-tumoral epigenetic heterogeneity analysis and visualization. Sci. Rep. 11, 376."/>
  <meta name="citation_reference" content="Costello, J.F., Shchors, K., Fiore, C., Zhao, Y., Bilenky, M., Hirst, M., Maunakea, A.K., Ballinger, T.J., Jones, S.J.M., Nagarajan, R.P., et al. (2010). Conserved role of intragenic DNA methylation in regulating alternative promoters. Nature 466, 253–257."/>
  <meta name="citation_reference" content="Coulondre, C., Miller, J.H., Farabaugh, P.J., and Gilbert, W. (1978). Molecular basis of base substitution hotspots in Escherichia coli. Nature 274, 775–780."/>
  <meta name="citation_reference" content="Cui, H., Onyango, P., Brandenburg, S., Wu, Y., Hsieh, C.L., and Feinberg, A.P. (2002). Loss of imprinting in colorectal cancer linked to hypomethylation of H19 and IGF2. Cancer Res. 62, 6442–6446."/>
  <meta name="citation_reference" content="Deaton, A.M., and Bird, A. (2011). CGIs and the regulation of transcription. Genes Dev 25, 1010–1022."/>
  <meta name="citation_reference" content="Deniz, Ö., Frost, J.M., and Branco, M.R. (2019). Regulation of transposable elements by DNA modifications. Nat. Rev. Genet. 20, 417–431."/>
  <meta name="citation_reference" content="Dentro, S.C., Wedge, D.C., and Van Loo, P. (2017). Principles of Reconstructing the Subclonal Architecture of Cancers. Cold Spring Harb. Perspect. Med. 7, a026625."/>
  <meta name="citation_reference" content="Dentro, S.C., Leshchiner, I., Haase, K., Tarabichi, M., Wintersinger, J., Deshwar, A.G., Yu, K., Rubanova, Y., Macintyre, G., Demeulemeester, J., et al. (2021). Characterizing genetic intra-tumor heterogeneity across 2,658 human cancer genomes. Cell 184, 2239–2254.e39."/>
  <meta name="citation_reference" content="Dolzhenko, E., and Smith, A.D. (2014). Using beta-binomial regression for high-precision differential methylation analysis in multifactor whole-genome bisulfite sequencing experiments. BMC Bioinformatics 15, 215."/>
  <meta name="citation_reference" content="Eckhardt, F., Lewin, J., Cortese, R., Rakyan, V.K., Attwood, J., Burger, M., Burton, J., Cox, T. V, Davies, R., Down, T.A., et al. (2006). DNA methylation profiling of human chromosomes 6, 20 and 22. Nat. Genet. 38, 1378–1385."/>
  <meta name="citation_reference" content="Feber, A., Dhami, P., Dong, L., de Winter, P., Tan, W.S., Martínez-Fernández, M., Paul, D.S., Hynes-Allen, A., Rezaee, S., Gurung, P., et al. (2017). UroMark—a urinary biomarker assay for the detection of bladder cancer. Clin. Epigenetics 9, 8."/>
  <meta name="citation_reference" content="Feinberg, A.P., Ohlsson, R., and Henikoff, S. (2006). The epigenetic progenitor origin of human cancer. Nat. Rev. Genet. 7, 21–33."/>
  <meta name="citation_reference" content="Feinberg, A.P., Koldobskiy, M.A., and Göndör, A. (2016). Epigenetic modulators, modifiers and mediators in cancer aetiology and progression. Nat. Rev. Genet. 17, 284–299."/>
  <meta name="citation_reference" content="Feingold, E.A., Good, P.J., Guyer, M.S., Kamholz, S., Liefer, L., Wetterstrand, K., Collins, F.S., Gingeras, T.R., Kampa, D., Sekinger, E.A., et al. (2004). The ENCODE (ENCyclopedia of DNA Elements) Project. Science 306, 636–640."/>
  <meta name="citation_reference" content="Fernández, J.M., de la Torre, V., Richardson, D., Royo, R., Puiggròs, M., Moncunill, V., Fragkogianni, S., Clarke, L., Flicek, P., Rico, D., et al. (2016). The BLUEPRINT Data Analysis Portal. Cell Syst."/>
  <meta name="citation_reference" content="Flavahan, W.A., Gaskell, E., and Bernstein, B.E. (2017). Epigenetic plasticity and the hallmarks of cancer. Science 357, eaal2380."/>
  <meta name="citation_reference" content="Frevel, M.A.E., Sowerby, S.J., Petersen, G.B., and Reeve, A.E. (1999). Methylation Sequencing Analysis Refines the Region of H19 Epimutation in Wilms Tumor. J. Biol. Chem. 274, 29331–29340."/>
  <meta name="citation_reference" content="Gaiti, F., Chaligne, R., Gu, H., Brand, R.M., Kothen-Hill, S., Schulman, R.C., Grigorev, K., Risso, D., Kim, K.-T., Pastore, A., et al. (2019). Epigenetic evolution and lineage histories of chronic lymphocytic leukaemia. Nature 569, 576–580."/>
  <meta name="citation_reference" content="Goyal, R. (2006). Accuracy of DNA methylation pattern preservation by the Dnmt1 methyltransferase. Nucleic Acids Res. 34, 1182–1188."/>
  <meta name="citation_reference" content="Greger, V., Passarge, E., Höpping, W., Messmer, E., and Horsthemke, B. (1989). Epigenetic changes may contribute to the formation and spontaneous regression of retinoblastoma. Hum. Genet. 83, 155–158."/>
  <meta name="citation_reference" content="Guo, S., Diep, D., Plongthongkum, N., Fung, H.-L., Zhang, K., and Zhang, K. (2017). Identification of methylation haplotype blocks aids in deconvolution of heterogeneous tissue samples and tumor tissue-of-origin mapping from plasma DNA. Nat. Genet. 49, 635–642."/>
  <meta name="citation_reference" content="Hanahan, D., and Weinberg, R.A. (2011). Hallmarks of Cancer: The Next Generation. Cell 144, 646–674."/>
  <meta name="citation_reference" content="Hannum, G., Guinney, J., Zhao, L., Zhang, L., Hughes, G., Sadda, S., Klotzle, B., Bibikova, M., Fan, J.-B., Gao, Y., et al. (2013). Genome-wide Methylation Profiles Reveal Quantitative Views of Human Aging Rates. Mol. Cell 49, 359–367."/>
  <meta name="citation_reference" content="Hansen, K.D., Timp, W., Bravo, H.C., Sabunciyan, S., Langmead, B., McDonald, O.G., Wen, B., Wu, H., Liu, Y., Diep, D., et al. (2011). Increased methylation variation in epigenetic domains across cancer types. Nat. Genet. 43, 768–775."/>
  <meta name="citation_reference" content="Hansen, K.D., Langmead, B., and Irizarry, R.A. (2012). BSmooth: from whole genome bisulfite sequencing reads to differentially methylated regions. Genome Biol. 13, R83."/>
  <meta name="citation_reference" content="Hao, J.-J., Lin, D.-C., Dinh, H.Q., Mayakonda, A., Jiang, Y.-Y., Chang, C., Jiang, Y., Lu, C.-C., Shi, Z.-Z., Xu, X., et al. (2016). Spatial intratumoral heterogeneity and temporal clonal evolution in esophageal squamous cell carcinoma. Nat. Genet. 48, 1500–1507."/>
  <meta name="citation_reference" content="Heyn, H., and Esteller, M. (2012). DNA methylation profiling in the clinic: applications and challenges. Nat. Rev. Genet. 13, 679–692."/>
  <meta name="citation_reference" content="Horvath, S. (2013). DNA methylation age of human tissues and cell types. Genome Biol. 14, R115."/>
  <meta name="citation_reference" content="Hu, X., Estecio, M.R., Chen, R., Reuben, A., Wang, L., Fujimoto, J., Carrot-Zhang, J., McGranahan, N., Ying, L., Fukuoka, J., et al. (2021). Evolution of DNA methylome from precancerous lesions to invasive lung adenocarcinomas. Nat. Commun. 12, 687."/>
  <meta name="citation_reference" content="Hua, X., Zhao, W., Pesatori, A.C., Consonni, D., Caporaso, N.E., Zhang, T., Zhu, B., Wang, M., Jones, K., Hicks, B., et al. (2020). Genetic and epigenetic intratumor heterogeneity impacts prognosis of lung adenocarcinoma. Nat. Commun. 11, 2459."/>
  <meta name="citation_reference" content="Huang, F.W., Hodis, E., Xu, M.J., Kryukov, G. V., Chin, L., and Garraway, L.A. (2013). Highly Recurrent TERT Promoter Mutations in Human Melanoma. Science 339, 957–959."/>
  <meta name="citation_reference" content="Hutter, C., and Zenklusen, J.C. (2018). The Cancer Genome Atlas: Creating Lasting Value beyond Its Data. Cell 173, 283–285."/>
  <meta name="citation_reference" content="ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium (2020). Pan-cancer analysis of whole genomes. Nature 578, 82–93."/>
  <meta name="citation_reference" content="Jamal-Hanjani, M., Wilson, G.A., McGranahan, N., Birkbak, N.J., Watkins, T.B.K., Veeriah, S., Shafi, S., Johnson, D.H., Mitter, R., Rosenthal, R., et al. (2017). Tracking the Evolution of Non–Small-Cell Lung Cancer. N. Engl. J. Med. 376, 2109–2121."/>
  <meta name="citation_reference" content="Kerkel, K., Spadola, A., Yuan, E., Kosek, J., Jiang, L., Hod, E., Li, K., Murty, V. V., Schupf, N., Vilain, E., et al. (2008). Genomic surveys by methylation-sensitive SNP analysis identify sequence-dependent allele-specific DNA methylation. Nat. Genet. 40, 904–908."/>
  <meta name="citation_reference" content="Klein, H.U., and Hebestreit, K. (2016). An evaluation of methods to test predefined genomic regions for differential methylation in bisulfite sequencing data. Brief. Bioinform. 17, 796–807."/>
  <meta name="citation_reference" content="Klughammer, J., Kiesel, B., Roetzer, T., Fortelny, N., Nemc, A., Nenning, K.-H., Furtner, J., Sheffield, N.C., Datlinger, P., Peter, N., et al. (2018). The DNA methylation landscape of glioblastoma disease progression shows extensive heterogeneity in time and space. Nat. Med. 24, 1611–1624."/>
  <meta name="citation_reference" content="Koch, A., Joosten, S.C., Feng, Z., de Ruijter, T.C., Draht, M.X., Melotte, V., Smits, K.M., Veeck, J., Herman, J.G., Van Neste, L., et al. (2018). Analysis of DNA methylation in cancer: location revisited. Nat. Rev. Clin. Oncol. 15, 459–466."/>
  <meta name="citation_reference" content="Kohli, R.M., and Zhang, Y. (2013). TET enzymes, TDG and the dynamics of DNA demethylation. Nature 502, 472–479."/>
  <meta name="citation_reference" content="Krueger, F., and Andrews, S.R. (2011). Bismark: A flexible aligner and methylation caller for Bisulfite-Seq applications. Bioinformatics 27, 1571–1572."/>
  <meta name="citation_reference" content="Kulis, M., Heath, S., Bibikova, M., Queirós, A.C., Navarro, A., Clot, G., Martínez-Trillos, A., Castellano, G., Brun-Heath, I., Pinyol, M., et al. (2012). Epigenomic analysis detects widespread gene-body DNA hypomethylation in chronic lymphocytic leukemia. Nat. Genet. 44, 1236–1242."/>
  <meta name="citation_reference" content="Landan, G., Cohen, N.M., Mukamel, Z., Bar, A., Molchadsky, A., Brosh, R., Horn-Saban, S., Zalcenstein, D.A., Goldfinger, N., Zundelevich, A., et al. (2012). Epigenetic polymorphism and the stochastic formation of differentially methylated regions in normal and cancerous tissues. Nat. Genet. 44, 1207–1214."/>
  <meta name="citation_reference" content="Landau, D.A., Clement, K., Ziller, M.J., Boyle, P., Fan, J., Gu, H., Stevenson, K., Sougnez, C., Wang, L., Li, S., et al. (2014). Locally Disordered Methylation Forms the Basis of Intratumor Methylome Variation in Chronic Lymphocytic Leukemia. Cancer Cell 26, 813–825."/>
  <meta name="citation_reference" content="Li, S., Garrett-Bakelman, F.E., Chung, S.S., Sanders, M.A., Hricik, T., Rapaport, F., Patel, J., Dillon, R., Vijay, P., Brown, A.L., et al. (2016). Distinct evolution and dynamics of epigenetic and genetic heterogeneity in acute myeloid leukemia. Nat. Med. 22, 792–799."/>
  <meta name="citation_reference" content="Lister, R., Pelizzola, M., Dowen, R.H., Hawkins, R.D., Hon, G., Tonti-Filippini, J., Nery, J.R., Lee, L., Ye, Z., Ngo, Q.-M., et al. (2009). Human DNA methylomes at base resolution show widespread epigenomic differences. Nature 462, 315–322."/>
  <meta name="citation_reference" content="Liu, Y., Siegmund, K.D., Laird, P.W., and Berman, B.P. (2012). Bis-SNP: Combined DNA methylation and SNP calling for Bisulfite-seq data. Genome Biol. 13, R61."/>
  <meta name="citation_reference" content="Lyon, M.F. (1962). Sex chromatin and gene action in the mammalian X-chromosome. Am. J. Hum. Genet. 14, 135–148."/>
  <meta name="citation_reference" content="Mansour, M.R., Abraham, B.J., Anders, L., Berezovskaya, A., Gutierrez, A., Durbin, A.D., Etchin, J., Lawton, L., Sallan, S.E., Silverman, L.B., et al. (2014). An oncogenic super-enhancer formed through somatic mutation of a noncoding intergenic element. Science 346, 1373–1377."/>
  <meta name="citation_reference" content="Martelotto, L.G. (2020). ‘Frankenstein’ protocol for nuclei isolation from fresh and frozen tissue for snRNAseq."/>
  <meta name="citation_reference" content="Martin-Trujillo, A., Vidal, E., Monteagudo-Sánchez, A., Sanchez-Delgado, M., Moran, S., Hernandez Mora, J.R., Heyn, H., Guitart, M., Esteller, M., and Monk, D. (2017). Copy number rather than epigenetic alterations are the major dictator of imprinted methylation in tumors. Nat. Commun. 8, 467."/>
  <meta name="citation_reference" content="Martin, M. (2011). Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.Journal 17, 10."/>
  <meta name="citation_reference" content="Mazor, T., Pankov, A., Johnson, B.E., Hong, C., Hamilton, E.G., Bell, R.J.A., Smirnov, I. V, Reis, G.F., Phillips, J.J., Barnes, M.J., et al. (2015). DNA Methylation and Somatic Mutations Converge on the Cell Cycle and Define Similar Evolutionary Histories in Brain Tumors. Cancer Cell 28, 307–317."/>
  <meta name="citation_reference" content="Meissner, A. (2005). Reduced representation bisulfite sequencing for comparative high-resolution DNA methylation analysis. Nucleic Acids Res. 33, 5868–5877."/>
  <meta name="citation_reference" content="Morova, T., McNeill, D.R., Lallous, N., Gönen, M., Dalal, K., Wilson, D.M., Gürsoy, A., Keskin, Ö., and Lack, N.A. (2020). Androgen receptor-binding sites are highly mutated in prostate cancer. Nat. Commun. 11, 832."/>
  <meta name="citation_reference" content="Neininger, K., Marschall, T., and Helms, V. (2019). SNP and indel frequencies at transcription start sites and at canonical and alternative translation initiation sites in the human genome. PLoS One 14, e0214816."/>
  <meta name="citation_reference" content="Nik-Zainal, S., Van Loo, P., Wedge, D.C., Alexandrov, L.B., Greenman, C.D., Lau, K.W., Raine, K., Jones, D., Marshall, J., Ramakrishna, M., et al. (2012). The life history of 21 breast cancers. Cell 149, 994–1007."/>
  <meta name="citation_reference" content="Nordlund, J., Bäcklin, C.L., Wahlberg, P., Busche, S., Berglund, E.C., Eloranta, M.-L., Flaegstad, T., Forestier, E., Frost, B.-M., Harila-Saari, A., et al. (2013). Genome-wide signatures of differential DNA methylation in pediatric acute lymphoblastic leukemia. Genome Biol. 14, r105."/>
  <meta name="citation_reference" content="Oakes, C.C., Seifert, M., Assenov, Y., Gu, L., Przekopowitz, M., Ruppert, A.S., Wang, Q., Imbusch, C.D., Serva, A., Koser, S.D., et al. (2016). DNA methylation dynamics during B cell maturation underlie a continuum of disease phenotypes in chronic lymphocytic leukemia. Nat. Genet. 48, 253–264."/>
  <meta name="citation_reference" content="Ogawa, O., Eccles, M.R., Szeto, J., McNoe, L.A., Yun, K., Maw, M.A., Smith, P.J., and Reeve, A.E. (1993). Relaxation of insulin-like growth factor II gene imprinting implicated in Wilms’ tumour. Nature 362, 749–751."/>
  <meta name="citation_reference" content="Paradis, E., Claude, J., and Strimmer, K. (2004). APE: Analyses of Phylogenetics and Evolution in R language. Bioinformatics 20, 289–290."/>
  <meta name="citation_reference" content="Pfister, S.M., Eils, J., Taylor, M.D., Yaspo, M.-L., Pietsch, T., Bunt, J., Stachurski, K., Korshunov, A., Warnatz, H.-J., Koster, J., et al. (2014). Decoding the regulatory landscape of medulloblastoma using DNA methylation sequencing. Nature 510, 537–541."/>
  <meta name="citation_reference" content="Quail, D.F., and Joyce, J.A. (2013). Microenvironmental regulation of tumor progression and metastasis. Nat. Med. 19, 1423–1437."/>
  <meta name="citation_reference" content="Raineri, E., Dabad, M., and Heath, S. (2014). A Note on Exact Differences between Beta Distributions in Genomic (Methylation) Studies. PLoS One 9, e97349."/>
  <meta name="citation_reference" content="Reik, W., and Walter, J. (2001). Genomic imprinting: parental influence on the genome. Nat. Rev. Genet. 2, 21–32."/>
  <meta name="citation_reference" content="Robinson, D. (2017). Introduction to Empirical Bayes: Examples from Baseball Statistics."/>
  <meta name="citation_reference" content="Robinson, M.D., Kahraman, A., Law, C.W., Lindsay, H., Nowicka, M., Weber, L.M., and Zhou, X. (2014). Statistical methods for detecting differentially methylated loci and regions. Front. Genet. 5, 1–7."/>
  <meta name="citation_reference" content="Rosenthal, R., McGranahan, N., Herrero, J., Taylor, B.S., and Swanton, C. (2016). deconstructSigs: delineating mutational processes in single tumors distinguishes DNA repair deficiencies and patterns of carcinoma evolution. Genome Biol. 17, 31."/>
  <meta name="citation_reference" content="Rosenthal, R., Larose Cadieux, E., Salgado, R., Bakir, M. Al, Moore, D.A., Hiley, C.T., Lund, T., Tanić, M., Reading, J.L., Joshi, K., et al. (2019). Neoantigen-directed immune escape in lung cancer evolution. Nature 567, 479–485."/>
  <meta name="citation_reference" content="Ryba, T., Hiratani, I., Lu, J., Itoh, M., Kulik, M., Zhang, J., Schulz, T.C., Robins, A.J., Dalton, S., and Gilbert, D.M. (2010). Evolutionarily conserved replication timing profiles predict long-range chromatin interactions and distinguish closely related cell types. Genome Res. 20, 761–770."/>
  <meta name="citation_reference" content="Schultz, M.D., He, Y., Whitaker, J.W., Hariharan, M., Mukamel, E.A., Leung, D., Rajagopal, N., Nery, J.R., Urich, M.A., Chen, H., et al. (2015). Human body epigenome maps reveal noncanonical DNA methylation variation. Nature 523, 212–216."/>
  <meta name="citation_reference" content="Sheffield, N.C., Pierron, G., Klughammer, J., Datlinger, P., Schönegger, A., Schuster, M., Hadler, J., Surdez, D., Guillemot, D., Lapouble, E., et al. (2017). DNA methylation heterogeneity defines a disease spectrum in Ewing sarcoma. Nat. Med. 23, 386–395."/>
  <meta name="citation_reference" content="Shen, H., and Laird, P.W. (2013). Interplay between the Cancer Genome and Epigenome. Cell 153, 38–55."/>
  <meta name="citation_reference" content="Smith, Z.D., and Meissner, A. (2013). DNA methylation: Roles in mammalian development. Nat. Rev. Genet. 14, 204–220."/>
  <meta name="citation_reference" content="Stunnenberg, H.G., Hirst, M., Abrignani, S., Adams, D., de Almeida, M., Altucci, L., Amin, V., Amit, I., Antonarakis, S.E., Aparicio, S., et al. (2016). The International Human Epigenome Consortium: A Blueprint for Scientific Collaboration and Discovery. Cell 167, 1145–1149."/>
  <meta name="citation_reference" content="Sun, Z., Cunningham, J., Slager, S., and Kocher, J.-P. (2015). Base resolution methylome profiling: considerations in platform selection, data preprocessing and analysis. Epigenomics 7, 813–828."/>
  <meta name="citation_reference" content="Sutherland, K.D., and Berns, A. (2010). Cell of origin of lung cancer. Mol. Oncol. 4, 397–403."/>
  <meta name="citation_reference" content="Tarabichi, M., Salcedo, A., Deshwar, A.G., Ni Leathlobhair, M., Wintersinger, J., Wedge, D.C., Van Loo, P., Morris, Q.D., and Boutros, P.C. (2021). A practical guide to cancer subclonal reconstruction from DNA sequencing. Nat. Methods 18, 144–155."/>
  <meta name="citation_reference" content="Teschendorff, A.E., Breeze, C.E., Zheng, S.C., and Beck, S. (2017). A comparison of reference-based algorithms for correcting cell-type heterogeneity in Epigenome-Wide Association Studies. BMC Bioinformatics 18, 105."/>
  <meta name="citation_reference" content="Van Loo, P., Nordgard, S.H., Lingjaerde, O.C., Russnes, H.G., Rye, I.H., Sun, W., Weigman, V.J., Marynen, P., Zetterberg, A., Naume, B., et al. (2010). Allele-specific copy number analysis of tumors. Proc. Natl. Acad. Sci. 107, 16910–16915."/>
  <meta name="citation_reference" content="Wen, B., Montano, C., Wu, Z., Gabo, K., Potash, J.B., Ji, H., Sabunciyan, S., Rongione, M., Webster, M., Irizarry, R.A., et al. (2009). The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores. Nat. Genet. 41, 178–186."/>
  <meta name="citation_reference" content="Wu, H., Xu, T., Feng, H., Chen, L., Li, B., Yao, B., Qin, Z., Jin, P., and Conneely, K.N. (2015). Detection of differentially methylated regions from whole-genome bisulfite sequencing data without replicates. Nucleic Acids Res. 43, 141."/>
  <meta name="citation_reference" content="Zhao, J., Li, D., Seo, J., Allen, A.S., and Gordân, R. (2017). Quantifying the Impact of Non-coding Variants on Transcription Factor-DNA Binding. pp. 336–352."/>
  <meta name="citation_reference" content="Zhao, S.G., Chen, W.S., Li, H., Foye, A., Zhang, M., Sjöström, M., Aggarwal, R., Playdle, D., Liao, A., Alumkal, J.J., et al. (2020). The DNA methylation landscape of advanced prostate cancer. Nat. Genet. 52, 778–789."/>
  <meta name="citation_reference" content="Zheng, X., Zhao, Q., Wu, H.-J., Li, W., Wang, H., Meyer, C.A., Qin, Q.A., Xu, H., Zang, C., Jiang, P., et al. (2014). MethylPurify: tumor purity deconvolution and differential methylation detection from single tumor DNA methylomes. Genome Biol. 15, 419."/>
  <meta name="citation_reference" content="Zheng, X., Zhang, N., Wu, H.-J., and Wu, H. (2017). Estimating and accounting for tumor purity in the analysis of DNA methylation data from cancer studies. Genome Biol. 18, 17."/>
  <meta name="citation_reference" content="Ziller, M.J., Gu, H., Müller, F., Donaghey, J., Tsai, L.T.-Y., Kohlbacher, O., De Jager, P.L., Rosen, E.D., Bennett, D.A., Bernstein, B.E., et al. (2013). Charting a dynamic DNA methylation landscape of the human genome. Nature 500, 477–481."/>
  <meta name="twitter:title" content="Copy number-aware deconvolution of tumor-normal DNA methylation profiles"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Aberrant methylation is a hallmark of cancer, but bulk tumor data is confounded by admixed normal cells and copy number changes. Here, we introduce Copy number-Aware Methylation Deconvolution Analysis of Cancers (CAMDAC; <https://github.com/VanLoo-lab/CAMDAC>), which outputs tumor purity, allele-specific copy number and deconvolved methylation estimates. We apply CAMDAC to 122 multi-region samples from 38 TRACERx non-small cell lung cancers profiled by reduced representation bisulfite sequencing. CAMDAC copy number profiles parallel those derived from genome sequencing and highlight widespread chromosomal instability. Deconvolved polymorphism-independent methylation rates enable unbiased tumor-normal and tumor-tumor differential methylation calling. Read-phasing validates CAMDAC methylation rates and directly links genotype and epitype. We show increased epigenetic instability in adenocarcinoma vs. squamous cell carcinoma, frequent hypermethylation at sites carrying somatic mutations, and parallel copy number losses and methylation changes at imprinted loci. Unlike bulk methylomes, CAMDAC profiles recapitulate tumor phylogenies and evidence distinct patterns of epigenetic heterogeneity in lung cancer.

### Competing Interest Statement

E.L.C. and G.A.W. are currently working for and hold shares of Achilles Therapeutics. C.S. acknowledges grant support from Pfizer, AstraZeneca, Bristol Myers Squibb, Roche-Ventana, Boehringer-Ingelheim, Archer Dx Inc (collaboration in minimal residual disease sequencing technologies) and Ono Pharmaceutical, is an AstraZeneca Advisory Board member and Chief Investigator for the MeRmaiD1 clinical trial, has consulted for Pfizer, Novartis, GlaxoSmithKline, MSD, Bristol Myers Squibb, Celgene, AstraZeneca, Illumina, Genentech, Roche-Ventana, GRAIL, Medicxi, Bicycle Therapeutics, and the Sarah Cannon Research Institute, has stock options in Apogen Biotechnologies, Epic Bioscience, GRAIL, and is co-founder and has shares of Achilles Therapeutics. M.J.-H. has consulted, and is a member of the Scientific Advisory Board and Steering Committee, for Achilles Therapeutics, has received speaker honoraria from Astex Pharmaceuticals, Oslo Cancer Cluster, and holds a patent PCT/US2017/028013 relating to methods for lung cancer detection. All other authors declare no competing interests."/>
  <meta name="og-title" property="og:title" content="Copy number-aware deconvolution of tumor-normal DNA methylation profiles"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.11.03.366252v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Aberrant methylation is a hallmark of cancer, but bulk tumor data is confounded by admixed normal cells and copy number changes. Here, we introduce Copy number-Aware Methylation Deconvolution Analysis of Cancers (CAMDAC; <https://github.com/VanLoo-lab/CAMDAC>), which outputs tumor purity, allele-specific copy number and deconvolved methylation estimates. We apply CAMDAC to 122 multi-region samples from 38 TRACERx non-small cell lung cancers profiled by reduced representation bisulfite sequencing. CAMDAC copy number profiles parallel those derived from genome sequencing and highlight widespread chromosomal instability. Deconvolved polymorphism-independent methylation rates enable unbiased tumor-normal and tumor-tumor differential methylation calling. Read-phasing validates CAMDAC methylation rates and directly links genotype and epitype. We show increased epigenetic instability in adenocarcinoma vs. squamous cell carcinoma, frequent hypermethylation at sites carrying somatic mutations, and parallel copy number losses and methylation changes at imprinted loci. Unlike bulk methylomes, CAMDAC profiles recapitulate tumor phylogenies and evidence distinct patterns of epigenetic heterogeneity in lung cancer.

### Competing Interest Statement

E.L.C. and G.A.W. are currently working for and hold shares of Achilles Therapeutics. C.S. acknowledges grant support from Pfizer, AstraZeneca, Bristol Myers Squibb, Roche-Ventana, Boehringer-Ingelheim, Archer Dx Inc (collaboration in minimal residual disease sequencing technologies) and Ono Pharmaceutical, is an AstraZeneca Advisory Board member and Chief Investigator for the MeRmaiD1 clinical trial, has consulted for Pfizer, Novartis, GlaxoSmithKline, MSD, Bristol Myers Squibb, Celgene, AstraZeneca, Illumina, Genentech, Roche-Ventana, GRAIL, Medicxi, Bicycle Therapeutics, and the Sarah Cannon Research Institute, has stock options in Apogen Biotechnologies, Epic Bioscience, GRAIL, and is co-founder and has shares of Achilles Therapeutics. M.J.-H. has consulted, and is a member of the Scientific Advisory Board and Steering Committee, for Achilles Therapeutics, has received speaker honoraria from Astex Pharmaceuticals, Oslo Cancer Cluster, and holds a patent PCT/US2017/028013 relating to methods for lung cancer detection. All other authors declare no competing interests."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-04-24"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.11.03.366252v2"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2510908"/>
  <title>
   Copy number-aware deconvolution of tumor-normal DNA methylation profiles | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"jZFZpaiBE_8RbVsVDhZ86lq6_P9gMuCE0tYGI1FuxsI"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.11.03.366252v2"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.11.03.366252v2"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.11.03.366252v2"}],"nid":"2510908","apath":"\/biorxiv\/early\/2022\/04\/24\/2020.11.03.366252.atom","pisa":"biorxiv;2020.11.03.366252v2","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.11.03.366252v2","title":"Copy number-aware deconvolution of tumor-normal DNA methylation profiles","identifier":"node\/2510908"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AEuAltRty1pUTsVz\\\\\\\/d6M3VAdfQaG1u\\\\\\\/dlN+yHdSYfieHM8GS+eq1W8z\\\\\\\/6WcB837V8NNFgoLriTTN6linXJuVunNVWijz1D7DzIf7xkQtIDgcKK2YPH7APXtQZhOV3B6Js\\\\\\\/ENHEJ\\\\\\\/kFC4b0A7VVgZ+AdVXOfAbPd1pTDtPQHQrZbOegUgUlWhf6j2ZwUyMoja7cyqz3JJ5qgI+vrW2Nwn9E5ZJdrfNoJY9gGfMXGH1OAThTIktWwt0\\\\\\\/voCv+CVglKMZGYoLBEe4z7\\\\\\\/cWWLfisJnE5tUDdtq68aG+AqBxhYxLeElK5Y\\\\\\\/7mdwhqXhs+HnPwSCrBC15KjTJxfiPJprkXaxqi0T8A2uK\\\\\\\/sWp57jaQsa1L5hW+kRU7Yi+m0xOEpIBNI8pYyl1nt2OW+QCXxBgt31ujjqQ+94zRS+w2QBOcGQGxfc3vczuTi+yH6dg1FaBOvPFgYeNnLdcLNRI3fyzcvkUDQjErQpfcgSf8NrPVD6ZD60n2BMqCOpL4WYyMGRs3XUQxhlZO2hN0GwKvyyRRKbsguag+zPV8VQPDpE\\\\\\\/ntse2bfuk6HFK9iBYsQ2ADbKvWJZPG4wC+iGjS77fTzVz5C4RJtOIcQ4LA\\\\\\\/GpcnS1mRjhL8XSMTR+ELMwM0rPesQ\\\\\\\/erqP5VS5K2MNBkiOcA7aJ1WsLqC8HbGV8bSHxV7I1x7Rxdatf2uF3+BrdzLUgBImEZksCqJm9wzWZOWcFxu2\\\\\\\/zfNgtXNpx2AiTplwrkzFvBtzVszRYfr6rfWboHuNE3e\\\\\\\/174MiEOr8Jcy80UZ3W3nTkmfClwsUFzIaxsE47HPcIKZtvFMX67JZFgzcUon39iD2CvwLCBeLkEU1rxGTA+o24Vm5hT4hQa5hQA82QRaztsUm4jInMmpjTj10XefsqNGt3CgAqFXmrfVcmwIG+6iarsoYW44\\\\\\\/hV3J0rsttKUEcNsn\\\\\\\/B\\\\\\\/laNG2XlCYYOYbRQVJ6+qKzYo+N9CfmV3sEass2\\\\\\\/UDe0YSH4vDlGat9XPJCdDatOOwAdakMUL9Tuh1PcGJTp16vRC388l6NvHSTRRfBAp8lOZHVUdgqKzEdtWYQZvrChQZi5HodW04wNU7zraAkLW2SlHeGsMxB4Sgmdc70PkC8gN4etNyIRHzSl+l9DliUwztv6x245SORmPn8\\\\\\\/K5c3abNXiaENAbdMpDLYzArOl2+Xrid8XLOdURLNX+5MXB6X+TG+ZXZmFKBelkdHGEWQLIoFC+\\\\\\\/5hx6+m8qI2hoMNdbF0MdL9MLtOhvpr5PEz70PfGtSlntBNFVolCWLMER9hzOQC8LSxOjsUWgLNm7OLZiyx8cl796n7h2Uf1rCMrk78iflL5G0zbRPM8srOQHbiWplwKYBKsBk9btLsk+HOPKLFB5BtxjPHLwEO2g=\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/ucwbHqeDEL2Vzt62g10OQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00225cf2fedbe75e93a0d4ed2c98914515f6bf8fc4dece161c88f1614d33287528df\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.11.03.366252v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2510908 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.11.03.366252v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_580517252">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_580517252" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1172226989" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-4JCoAvhR8End5lffiFNy8uryDhA29_fJVP8Ea37SsG0"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2510908" data-node-nid="2510908" id="node-2510908--21589367939" data-pisa="biorxiv;2020.11.03.366252v2" data-pisa-master="biorxiv;2020.11.03.366252" data-apath="/biorxiv/early/2022/04/24/2020.11.03.366252.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Copy number-aware deconvolution of tumor-normal DNA methylation profiles
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0002-9320-8040" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Elizabeth Larose
                      </span>
                      <span class="nlm-surname">
                       Cadieux
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0003-0832-2816" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Nana E.
                      </span>
                      <span class="nlm-surname">
                       Mensah
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2">
                      <a href="http://orcid.org/0000-0001-8689-9659" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Carla
                      </span>
                      <span class="nlm-surname">
                       Castignani
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3">
                      <a href="http://orcid.org/0000-0003-3102-776X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Miljana
                      </span>
                      <span class="nlm-surname">
                       Tanić
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4">
                      <a href="http://orcid.org/0000-0002-5703-9710" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Gareth A.
                      </span>
                      <span class="nlm-surname">
                       Wilson
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="5">
                      <span class="nlm-given-names">
                       Michelle
                      </span>
                      <span class="nlm-surname">
                       Dietzen
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="6">
                      <span class="nlm-given-names">
                       Pawan
                      </span>
                      <span class="nlm-surname">
                       Dhami
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="7">
                      <span class="nlm-given-names">
                       Heli
                      </span>
                      <span class="nlm-surname">
                       Vaikkinen
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="8">
                      <span class="nlm-given-names">
                       Annelien
                      </span>
                      <span class="nlm-surname">
                       Verfaillie
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="9">
                      <span class="nlm-given-names">
                       Cristina Cotobal
                      </span>
                      <span class="nlm-surname">
                       Martin
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="10">
                      <span class="nlm-given-names">
                       Toby
                      </span>
                      <span class="nlm-surname">
                       Baker
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="11">
                      <span class="nlm-given-names">
                       Thomas B. K.
                      </span>
                      <span class="nlm-surname">
                       Watkins
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12">
                      <a href="http://orcid.org/0000-0002-1217-0387" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Selvaraju
                      </span>
                      <span class="nlm-surname">
                       Veeriah
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="13">
                      <span class="nlm-given-names">
                       Mariam
                      </span>
                      <span class="nlm-surname">
                       Jamal-Hanjani
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="14">
                      <a href="http://orcid.org/0000-0001-7232-1952" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Nnennaya
                      </span>
                      <span class="nlm-surname">
                       Kanu
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="15">
                      <a href="http://orcid.org/0000-0001-9537-4045" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Nicholas
                      </span>
                      <span class="nlm-surname">
                       McGranahan
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="16">
                      <a href="http://orcid.org/0000-0001-5282-0498" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Andrew
                      </span>
                      <span class="nlm-surname">
                       Feber
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="17">
                      <span class="nlm-collab" data-hwp-id="collab-1">
                       TRACERx Consortium
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="18">
                      <a href="http://orcid.org/0000-0002-4299-3018" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Charles
                      </span>
                      <span class="nlm-surname">
                       Swanton
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="19">
                      <a href="http://orcid.org/0000-0001-5290-2151" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Stephan
                      </span>
                      <span class="nlm-surname">
                       Beck
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="20">
                      <a href="http://orcid.org/0000-0002-2660-2478" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Jonas
                      </span>
                      <span class="nlm-surname">
                       Demeulemeester
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="21">
                      <a href="http://orcid.org/0000-0003-0292-1949" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Peter
                      </span>
                      <span class="nlm-surname">
                       Van Loo
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.11.03.366252
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2510908--21589367939" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Elizabeth Larose Cadieux
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        The Francis Crick Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        University College London Cancer Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Elizabeth%2BLarose%2BCadieux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Cadieux%20EL&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AElizabeth%2BLarose%2BCadieux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-9320-8040" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Elizabeth Larose Cadieux
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Nana E. Mensah
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        The Francis Crick Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        University College London Cancer Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Nana%2BE.%2BMensah%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Mensah%20NE&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ANana%2BE.%2BMensah%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-0832-2816" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Nana E. Mensah
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Carla Castignani
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        The Francis Crick Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        University College London Cancer Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Carla%2BCastignani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Castignani%20C&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ACarla%2BCastignani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-8689-9659" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Carla Castignani
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Miljana Tanić
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        University College London Cancer Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Institute for Oncology and Radiology of Serbia
                       </span>
                       , Belgrade,
                       <span class="nlm-country">
                        Serbia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Miljana%2BTani%C4%87%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Tani%C4%87%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AMiljana%2BTani%25C4%2587%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-3102-776X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Miljana Tanić
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Gareth A. Wilson
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        The Francis Crick Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        University College London Cancer Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Gareth%2BA.%2BWilson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Wilson%20GA&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AGareth%2BA.%2BWilson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-5703-9710" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Gareth A. Wilson
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Michelle Dietzen
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        The Francis Crick Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        University College London Cancer Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Cancer Research UK Lung Cancer Centre of Excellence, University College London Cancer Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Michelle%2BDietzen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Dietzen%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AMichelle%2BDietzen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">
                     Pawan Dhami
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        University College London Cancer Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        NIHR Biomedical Research Centre Guy’s and St Thomas’ NHS Foundation Trust
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Pawan%2BDhami%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Dhami%20P&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3APawan%2BDhami%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-7">
                    <div class="author-tooltip-name">
                     Heli Vaikkinen
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        University College London Cancer Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        NIHR Biomedical Research Centre Guy’s and St Thomas’ NHS Foundation Trust
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Heli%2BVaikkinen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Vaikkinen%20H&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AHeli%2BVaikkinen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-8">
                    <div class="author-tooltip-name">
                     Annelien Verfaillie
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        The Francis Crick Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Annelien%2BVerfaillie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Verfaillie%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAnnelien%2BVerfaillie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-9">
                    <div class="author-tooltip-name">
                     Cristina Cotobal Martin
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        The Francis Crick Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Cristina%2BCotobal%2BMartin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Martin%20CC&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ACristina%2BCotobal%2BMartin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-10">
                    <div class="author-tooltip-name">
                     Toby Baker
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        The Francis Crick Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Toby%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Baker%20T&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AToby%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-11">
                    <div class="author-tooltip-name">
                     Thomas B. K. Watkins
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        The Francis Crick Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Thomas%2BB.%2BK.%2BWatkins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Watkins%20TB&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AThomas%2BB.%2BK.%2BWatkins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-12">
                    <div class="author-tooltip-name">
                     Selvaraju Veeriah
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        University College London Cancer Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Selvaraju%2BVeeriah%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Veeriah%20S&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ASelvaraju%2BVeeriah%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-1217-0387" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Selvaraju Veeriah
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-13">
                    <div class="author-tooltip-name">
                     Mariam Jamal-Hanjani
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Cancer Research UK Lung Cancer Centre of Excellence, University College London Cancer Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        Cancer Metastasis Laboratory, University College London Cancer Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        7
                       </span>
                       <span class="nlm-institution">
                        Department of Medical Oncology, University College London Hospitals
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Mariam%2BJamal-Hanjani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Jamal-Hanjani%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AMariam%2BJamal-Hanjani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-14">
                    <div class="author-tooltip-name">
                     Nnennaya Kanu
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        University College London Cancer Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Nnennaya%2BKanu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Kanu%20N&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ANnennaya%2BKanu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-7232-1952" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Nnennaya Kanu
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-15">
                    <div class="author-tooltip-name">
                     Nicholas McGranahan
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        University College London Cancer Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Cancer Research UK Lung Cancer Centre of Excellence, University College London Cancer Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Nicholas%2BMcGranahan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=McGranahan%20N&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ANicholas%2BMcGranahan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-9537-4045" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Nicholas McGranahan
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-16">
                    <div class="author-tooltip-name">
                     Andrew Feber
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        8
                       </span>
                       <span class="nlm-institution">
                        Translational Epigenetics, Division of Molecular Pathology, Institute for Cancer Research
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        9
                       </span>
                       <span class="nlm-institution">
                        Translational Research Laboratory, Centre for Molecular Pathology, Royal Marsden NHS Trust
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Andrew%2BFeber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Feber%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAndrew%2BFeber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-5282-0498" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Andrew Feber
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-18">
                    <div class="author-tooltip-name">
                     Charles Swanton
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        The Francis Crick Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Cancer Research UK Lung Cancer Centre of Excellence, University College London Cancer Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Charles%2BSwanton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Swanton%20C&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ACharles%2BSwanton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-4299-3018" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Charles Swanton
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-19">
                    <div class="author-tooltip-name">
                     Stephan Beck
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        University College London Cancer Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Stephan%2BBeck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Beck%20S&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AStephan%2BBeck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-5290-2151" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Stephan Beck
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-20">
                    <div class="author-tooltip-name">
                     Jonas Demeulemeester
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        The Francis Crick Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        10
                       </span>
                       <span class="nlm-institution">
                        Department of Human Genetics, University of Leuven
                       </span>
                       , Leuven,
                       <span class="nlm-country">
                        Belgium
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Jonas%2BDemeulemeester%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Demeulemeester%20J&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AJonas%2BDemeulemeester%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-2660-2478" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Jonas Demeulemeester
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-21">
                    <div class="author-tooltip-name">
                     Peter Van Loo
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        The Francis Crick Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        11
                       </span>
                       <span class="nlm-institution">
                        Department of Genetics, The University of Texas MD Anderson Cancer Center
                       </span>
                       , Houston, Texas,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        12
                       </span>
                       <span class="nlm-institution">
                        Department of Genomic Medicine, The University of Texas MD Anderson Cancer Center
                       </span>
                       , Houston, Texas,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Peter%2BVan%2BLoo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Van%20Loo%20P&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3APeter%2BVan%2BLoo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0003-0292-1949" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Peter Van Loo
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:Peter.VanLoo@crick.ac.uk" class="" data-icon-position="" data-hide-link-title="0">
                        Peter.VanLoo@crick.ac.uk
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.11.03.366252v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2510908" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2510908/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.03.366252v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2510908" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2510908/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.03.366252v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2510908" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2510908/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.03.366252v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2510908" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2510908/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.03.366252v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2510908" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2510908/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.11.03.366252v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2510908" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2510908/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             SUMMARY
                            </h2>
                            <p id="p-5">
                             Aberrant methylation is a hallmark of cancer, but bulk tumor data is confounded by admixed normal cells and copy number changes. Here, we introduce Copy number-Aware Methylation Deconvolution Analysis of Cancers (CAMDAC;
                             <a href="https://github.com/VanLoo-lab/CAMDAC">
                              https://github.com/VanLoo-lab/CAMDAC
                             </a>
                             ), which outputs tumor purity, allele-specific copy number and deconvolved methylation estimates. We apply CAMDAC to 122 multi-region samples from 38 TRACERx non-small cell lung cancers profiled by reduced representation bisulfite sequencing. CAMDAC copy number profiles parallel those derived from genome sequencing and highlight widespread chromosomal instability. Deconvolved polymorphism-independent methylation rates enable unbiased tumor-normal and tumor-tumor differential methylation calling. Read-phasing validates CAMDAC methylation rates and directly links genotype and epitype. We show increased epigenetic instability in adenocarcinoma vs. squamous cell carcinoma, frequent hypermethylation at sites carrying somatic mutations, and parallel copy number losses and methylation changes at imprinted loci. Unlike bulk methylomes, CAMDAC profiles recapitulate tumor phylogenies and evidence distinct patterns of epigenetic heterogeneity in lung cancer.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             INTRODUCTION
                            </h2>
                            <p id="p-7">
                             Like DNA mutations, epigenetic alterations, such as disruption of DNA methylation and chromatin architecture, are now acknowledged as a universal feature of tumorigenesis (
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              Baylin and Jones, 2016
                             </a>
                             ;
                             <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                              Feinberg et al., 2016
                             </a>
                             ;
                             <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">
                              Shen and Laird, 2013
                             </a>
                             ). These changes accumulate in normal cells throughout their lifetime (
                             <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                              Hannum et al., 2013
                             </a>
                             ;
                             <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                              Hao et al., 2016
                             </a>
                             ;
                             <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                              Horvath, 2013
                             </a>
                             ) and while most have no effect, a small subset may provide a selective advantage for the cell (
                             <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                              Feinberg et al., 2006
                             </a>
                             ;
                             <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                              Flavahan et al., 2017
                             </a>
                             ). Through successive gains of hallmark cancer cellular capabilities (
                             <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                              Hanahan and Weinberg, 2011
                             </a>
                             ), cells may ultimately acquire a fully malignant phenotype, while continuing to evolve in response to environmental pressures (
                             <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">
                              Quail and Joyce, 2013
                             </a>
                             ).
                            </p>
                            <p id="p-8">
                             DNA methylation is a mitotically heritable covalent DNA modification. In vertebrates, cytosines (C) can be methylated to form 5-methylcytosines (5mC), mostly in a CpG context (
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              Bird, 2002
                             </a>
                             ;
                             <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">
                              Smith and Meissner, 2013
                             </a>
                             ). This methylation occurs throughout the entire genome where it may aid to suppress cryptic transcription (
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              Costello et al., 2010
                             </a>
                             ) and keep mobile genetic elements in check (
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              Deniz et al., 2019
                             </a>
                             ). Active regulatory regions often contain unmethylated CpG-islands, irrespective of the expression level of associated genes (
                             <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                              Deaton and Bird, 2011
                             </a>
                             ;
                             <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                              Eckhardt et al., 2006
                             </a>
                             ;
                             <a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">
                              Ziller et al., 2013
                             </a>
                             ), although methylation of these CpG-rich features is usually anti-correlated with transcription factor binding. Genomic alterations removing binding sites enable DNA methyltransferase activity to spread and deplete gene expression (
                             <a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">
                              Smith and Meissner, 2013
                             </a>
                             ); vice versa, aberrations creating new binding sites can stimulate transcription (
                             <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                              Huang et al., 2013
                             </a>
                             ;
                             <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">
                              Mansour et al., 2014
                             </a>
                             ).
                            </p>
                            <p id="p-9">
                             The cancer methylome displays characteristics from its cell of origin in addition to somatic DNA methylation changes (
                             <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                              Greger et al., 1989
                             </a>
                             ;
                             <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">
                              Wen et al., 2009
                             </a>
                             ). As methylation profiling provides insight into (disease) cell states and other aspects of biology, with a readout that is not reliant on live cells nor large amounts of high quality material, it may yield powerful biomarkers (
                             <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                              Feber et al., 2017
                             </a>
                             ;
                             <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                              Heyn and Esteller, 2012
                             </a>
                             ;
                             <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                              Koch et al., 2018
                             </a>
                             ).
                            </p>
                            <p id="p-10">
                             Massively parallel sequencing efforts by the Cancer Genome Atlas (TCGA) and the International Cancer Genome Consortium (ICGC) have revealed somatic alterations across thousands of cancer genomes (
                             <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                              ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium, 2020
                             </a>
                             ;
                             <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                              Hutter and Zenklusen, 2018
                             </a>
                             ). Although whole-genome (WGBS,
                             <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">
                              Lister et al., 2009
                             </a>
                             ) and reduced-representation bisulfite sequencing (RRBS,
                             <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">
                              Meissner, 2005
                             </a>
                             ) are gaining in popularity (
                             <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                              Hansen et al., 2011
                             </a>
                             ;
                             <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                              Hu et al., 2021
                             </a>
                             ;
                             <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">
                              Pfister et al., 2014
                             </a>
                             ;
                             <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">
                              Sun et al., 2015
                             </a>
                             ;
                             <a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">
                              Zhao et al., 2020
                             </a>
                             ;
                             <a id="xref-ref-97-2" class="xref-bibr" href="#ref-97">
                              Ziller et al., 2013
                             </a>
                             ), the cancer methylome is considerably less well charted. Intra-tumor DNA methylation heterogeneity has been widely reported (
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              Brocks et al., 2014
                             </a>
                             ;
                             <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                              Hua et al., 2020
                             </a>
                             ;
                             <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                              Klughammer et al., 2018
                             </a>
                             ;
                             <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">
                              Mazor et al., 2015
                             </a>
                             ), however the interpretation of bulk tumor methylation data is hampered by the admixture of normal cells (
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              Chakravarthy et al., 2018
                             </a>
                             ) and the occurrence of somatic copy number alterations (CNAs) in most cancers (
                             <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">
                              Martin-Trujillo et al., 2017
                             </a>
                             ). While a number of methods have been developed to assess normal cell contributions (
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              Barrett et al., 2017
                             </a>
                             ;
                             <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                              Guo et al., 2017
                             </a>
                             ;
                             <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">
                              Teschendorff et al., 2017
                             </a>
                             ;
                             <a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">
                              Zheng et al., 2014
                             </a>
                             ), research focus has generally been restricted to high purity samples and CNA-quiet cancer types such as Ewing sarcoma (
                             <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">
                              Sheffield et al., 2017
                             </a>
                             ) and malignancies of the central nervous system (
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              Capper et al., 2018
                             </a>
                             ) or lymphoid origins (
                             <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">
                              Kulis et al., 2012
                             </a>
                             ;
                             <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">
                              Landau et al., 2014
                             </a>
                             ;
                             <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">
                              Nordlund et al., 2013
                             </a>
                             ;
                             <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">
                              Oakes et al., 2016
                             </a>
                             ). Similarly, accurate estimates of purity and copy number are critical when querying intra-tumor heterogeneity and reconstructing tumor evolutionary histories (
                             <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">
                              Tarabichi et al., 2021
                             </a>
                             ). Current metrics to quantify tumor heterogeneity from bisulfite sequencing data also assume high tumor content and/or the absence of CNAs (
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              Chen et al., 2021
                             </a>
                             ;
                             <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">
                              Landan et al., 2012
                             </a>
                             ;
                             <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">
                              Landau et al., 2014
                             </a>
                             ;
                             <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">
                              Li et al., 2016
                             </a>
                             ;
                             <a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">
                              Sheffield et al., 2017
                             </a>
                             ).
                            </p>
                            <p id="p-11">
                             To address these issues, we developed a tool for Copy number-Aware Methylation Deconvolution Analysis of Cancers (CAMDAC). CAMDAC provides accurate allele-specific copy number and purity estimates from tumor RRBS data. Formalizing the relationship between methylation rates, copy number and tumor purity, CAMDAC extracts purified tumor methylomes from bulk tumor and tissue-matched normal bisulfite sequencing data. The corrected tumor methylation rates allow for accurate quantification of differential methylation, both between tumor and normal cells and between different tumors or sampled regions. Phasing to sequence variants, we assess epigenetic instability and the interplay between DNA methylation and somatic copy number and single-nucleotide variants. CAMDAC deconvolved (allele-specific) tumor methylation profiles reveal intra-tumor subclonal relationships and allow to quantify allele-specific methylation signals.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             RESULTS
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Bulk tumor methylation rates are confounded by tumor purity and copy number
                             </h3>
                             <p id="p-12">
                              To study epigenetic inter- and intra-tumor heterogeneity in non-small cell lung cancer, we obtained surgically resected primary tumor samples from 24 lung adenocarcinoma (LUAD) and 14 squamous cell lung carcinoma (LUSC) patients (
                              <strong>
                               Table S1
                              </strong>
                              ) from the TRACERx 100 cohort, and performed multi-region RRBS of 122 tumor regions in total (2-7 per patient,
                              <strong>
                               Table S2
                              </strong>
                              ). As a normal reference, we included samples of adjacent normal tissue for 37/38 patients.
                             </p>
                             <p id="p-13">
                              We hypothesize that normal cell admixture (tumor purity) and somatic copy number alterations affect the methylation rate of bulk tumor samples (
                              <strong>
                               <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                                Figure 1A
                               </a>
                              </strong>
                              ). We set out to assess this effect knowing that previous work using whole-exome sequencing (WES) of the same samples revealed considerable variability in purity and copy number (
                              <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                               Jamal-Hanjani et al., 2017
                              </a>
                              ). We selected CpG loci that are confidently unmethylated in the adjacent normal sample (posterior 99% highest density interval on methylation rate HDI
                              <sup>
                               99
                              </sup>
                              ⊆ [0, 0.2],
                              <strong>
                               Methods
                              </strong>
                              ), stratified them by allele-specific copy number state in three tumor regions with different purity, and evaluated their bulk methylation rates (
                              <strong>
                               <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                                Figure 1B
                               </a>
                              </strong>
                              ). The majority of these loci exhibit bulk methylation rates close to 0 (88% with HDI
                              <sup>
                               99
                              </sup>
                              ⊆ [0, 0.2]), suggesting that most sites are not differentially methylated in the tumor. A second population of CpG sites is visible across all samples, the modal methylation rate of which shifts with tumor purity and copy number. We observed a similar effect evaluating CpG loci that are confidently methylated in the adjacent normal sample (
                              <strong>
                               <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                                Figure S1
                               </a>
                              </strong>
                              ). These observations confirm that normal cell admixture and copy number changes confound tumor methylation signals. We therefore set out to infer tumor purity and copy number from bulk tumor RRBS data, and extract purified tumor methylation rates, unpolluted with signals from non-tumor cells.
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F1.large.jpg?width=800&height=600&carousel=1" title="Tumor purity and copy number affect methylation rates. (A) Multi-region RRBS was performed on surgically resected non-small cell lung cancers and adjacent normal lung tissue. Sampled regions may differ both in tumor purity and copy number. The observed methylation rate of a differentially methylated CpG is expected to be influenced by purity, copy number and the methylation rate of the normal contaminating cells. (B) Bulk methylation rate histograms for tumor regions 1-3 of patient CRUK0062, for CpGs which are confidently unmethylated in the adjacent normal sample. CpGs are stratified by copy number. A dashed line indicates the expected mode of the methylation rate peak corresponding to clonal differentially methylated CpGs on all copies (mt = 1)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Tumor purity and copy number affect methylation rates.</span> (A) Multi-region RRBS was performed on surgically resected non-small cell lung cancers and adjacent normal lung tissue. Sampled regions may differ both in tumor purity and copy number. The observed methylation rate of a differentially methylated CpG is expected to be influenced by purity, copy number and the methylation rate of the normal contaminating cells. (B) Bulk methylation rate histograms for tumor regions 1-3 of patient CRUK0062, for CpGs which are confidently unmethylated in the adjacent normal sample. CpGs are stratified by copy number. A dashed line indicates the expected mode of the methylation rate peak corresponding to clonal differentially methylated CpGs on all copies (<strong><em>m<sub>t</sub></em> = 1</strong>).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F1.medium.gif" width="434" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F1.medium.gif" width="434" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1.
                               </span>
                               <span class="caption-title">
                                Tumor purity and copy number affect methylation rates.
                               </span>
                               <p id="p-14" class="first-child">
                                (A) Multi-region RRBS was performed on surgically resected non-small cell lung cancers and adjacent normal lung tissue. Sampled regions may differ both in tumor purity and copy number. The observed methylation rate of a differentially methylated CpG is expected to be influenced by purity, copy number and the methylation rate of the normal contaminating cells. (B) Bulk methylation rate histograms for tumor regions 1-3 of patient CRUK0062, for CpGs which are confidently unmethylated in the adjacent normal sample. CpGs are stratified by copy number. A dashed line indicates the expected mode of the methylation rate peak corresponding to clonal differentially methylated CpGs on all copies (
                                <strong>
                                 m
                                 <sub>
                                  t
                                 </sub>
                                 = 1
                                </strong>
                                ).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Allele-specific copy number analysis from tumor bisulfite sequencing data
                             </h3>
                             <p id="p-15">
                              We modeled our approach of purity and copy number inference on existing methods that simultaneously obtain allele-specific copy number and purity estimates from array or sequencing data (
                              <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                               Carter et al., 2012
                              </a>
                              ;
                              <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">
                               Van Loo et al., 2010
                              </a>
                              ;
                              <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">
                               Nik-Zainal et al., 2012
                              </a>
                              ). These approaches rely on measures of coverage (LogR) and allelic imbalance (B-allele frequency, BAF) at single nucleotide polymorphisms (SNPs). As CpG islands captured by RRBS are enriched for SNPs (
                              <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">
                               Neininger et al., 2019
                              </a>
                              ), we anticipated the data to be highly amenable to allele-specific copy number analysis.
                             </p>
                             <p id="p-16">
                              The LogR at SNP loci is readily computed from the normalized read coverage of matched tumor and normal RRBS data. Where patient-matched normal data were not available, the median SNP coverage across all sex-matched normal lung samples in this cohort was used instead. This metric of total copy number suffers from a number of biases in RRBS (
                              <strong>
                               <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                                Figure S2
                               </a>
                              </strong>
                              ). (i) MspI digestion used during library preparation results in a heterogeneous insert size distribution, with fragments ranging from just a few base pairs to hundreds of base pairs in length, depending on the distance between two CCGG recognition sequences (
                              <a id="xref-ref-86-2" class="xref-bibr" href="#ref-86">
                               Sun et al., 2015
                              </a>
                              ). (ii) Bisulfite conversion alters the GC content of sequences, potentially breaking standard GC-correction for the biases introduced during PCR amplification and Illumina sequencing (
                              <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                               Benjamini and Speed, 2012
                              </a>
                              ). (iii) Replication timing differs across the genome and between cell types (
                              <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">
                               Ryba et al., 2010
                              </a>
                              ). Sequences that replicate early during S-phase tend to have higher coverage than those that replicate later. We correct our LogR estimates for each of these three biases (
                              <strong>
                               Methods
                              </strong>
                              ,
                              <strong>
                               <a id="xref-fig-10-2" class="xref-fig" href="#F10">
                                Figure S2
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-17">
                              Due to the bisulfite conversion, compiling reference and alternate allele read counts at SNP loci to obtain BAF values is challenging. Unmethylated cytosines are converted to thymine, yielding four possible bisulfite DNA strands: (complementary to) original top and (complementary to) original bottom (
                              <strong>
                               <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                                Figure 2A
                               </a>
                              </strong>
                              ). However, most current single-end bisulfite sequencing protocols are directional, yielding only reads from the original top and bottom strands. In addition, as ∼50% of the SNPs captured by RRBS are found at CpG dinucleotides, methylation further complicates BAF calculation.
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F2.large.jpg?width=800&height=600&carousel=1" title="Allele-Specific copy number profiling of RRBS data. (A) Transformation of (un)methylated reference CpG, and alternate ApG and TpG allele dinucleotides during bisulfite sequencing. (B-C) Derivation of BAF rules from strand specific base counts C>A (B) and C>T (C) SNP loci. (D) BAF formulae for all SNP types. (E) Direct comparison of allele-specific copy number estimates derived by ASCAT.m from RRBS and WGS for sample CRUK0069-R1. (F-G) ASCAT.m RRBS-derived ploidy (F) and purity (G) estimates compared with matched whole-exome (WES; blue squares) and, where available, whole-genome sequencing data (WGS; purple diamonds). (H) Percentage overlap between RRBS- and WES- or WGS-derived allele-specific copy number segments. Exact overlaps (orange) and approximate overlaps (±1 copy on a single allele, green) are both displayed. (I) Tumor ploidy and the fraction of the genome with loss of heterozygosity define whole-genome doubling status. (J) Frequency of gains (red) and losses (blue) across the genome of lung adenocarcinoma (LUAD) and squamous cell lung cancer (LUSC). The black line represents gains and losses across 297 LUAD and 444 LUSC samples in TCGA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Allele-Specific copy number profiling of RRBS data.</span> (A) Transformation of (un)methylated reference CpG, and alternate ApG and TpG allele dinucleotides during bisulfite sequencing. (B-C<strong>)</strong> Derivation of BAF rules from strand specific base counts C>A (B) and C>T (C) SNP loci. (D) BAF formulae for all SNP types. (E) Direct comparison of allele-specific copy number estimates derived by ASCAT.m from RRBS and WGS for sample CRUK0069-R1. (F-G) ASCAT.m RRBS-derived ploidy (F) and purity (G) estimates compared with matched whole-exome (WES; blue squares) and, where available, whole-genome sequencing data (WGS; purple diamonds). (H) Percentage overlap between RRBS- and WES- or WGS-derived allele-specific copy number segments. Exact overlaps (orange) and approximate overlaps (±1 copy on a single allele, green) are both displayed. (I) Tumor ploidy and the fraction of the genome with loss of heterozygosity define whole-genome doubling status. (J) Frequency of gains (red) and losses (blue) across the genome of lung adenocarcinoma (LUAD) and squamous cell lung cancer (LUSC). The black line represents gains and losses across 297 LUAD and 444 LUSC samples in TCGA.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F2.medium.gif" width="327" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F2.medium.gif" width="327" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2.
                               </span>
                               <span class="caption-title">
                                Allele-Specific copy number profiling of RRBS data.
                               </span>
                               <p id="p-18" class="first-child">
                                (A) Transformation of (un)methylated reference CpG, and alternate ApG and TpG allele dinucleotides during bisulfite sequencing. (B-C
                                <strong>
                                 )
                                </strong>
                                Derivation of BAF rules from strand specific base counts C>A (B) and C>T (C) SNP loci. (D) BAF formulae for all SNP types. (E) Direct comparison of allele-specific copy number estimates derived by ASCAT.m from RRBS and WGS for sample CRUK0069-R1. (F-G) ASCAT.m RRBS-derived ploidy (F) and purity (G) estimates compared with matched whole-exome (WES; blue squares) and, where available, whole-genome sequencing data (WGS; purple diamonds). (H) Percentage overlap between RRBS- and WES- or WGS-derived allele-specific copy number segments. Exact overlaps (orange) and approximate overlaps (±1 copy on a single allele, green) are both displayed. (I) Tumor ploidy and the fraction of the genome with loss of heterozygosity define whole-genome doubling status. (J) Frequency of gains (red) and losses (blue) across the genome of lung adenocarcinoma (LUAD) and squamous cell lung cancer (LUSC). The black line represents gains and losses across 297 LUAD and 444 LUSC samples in TCGA.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-19">
                              Addressing these issues, we propose a set of allele counting and BAF calculation rules for all SNP types, following similar reasoning as Liu et al. (
                              <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">
                               Liu et al., 2012
                              </a>
                              ). We illustrate our derivation for C>A and C>T SNPs. Readout bases are reported as original top (+) or complementary to original bottom strand (−) (
                              <strong>
                               <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                                Figure 2A-C
                               </a>
                              </strong>
                              ). For a C>A SNP at a CpG, all reads reporting A on the + or − strand, i.e. A(+) and A(−), can be uniquely attributed to the alternate allele. Likewise, C(−) derives from the reference allele, together with C(+) and T(+) reads from the methylated and unmethylated cytosine, respectively. As a result, BAF
                              <sub>
                               C>A
                              </sub>
                              can be calculated as
                              <span class="inline-formula" id="inline-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (
                              <strong>
                               <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                                Figure 2B
                               </a>
                              </strong>
                              ). The situation is more complex for C>T SNPs, since the + strand cannot be used to distinguish between the alternate allele and the bisulfite converted unmethylated reference, as both yield T(+) (
                              <strong>
                               <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                                Figure 2C
                               </a>
                              </strong>
                              ). However, reads from the − strand do distinguish the (un)methylated reference, C(−), from the alternate allele, T(−). Therefore, it is still possible to quantify BAF
                              <sub>
                               C>T
                              </sub>
                              as
                              <span class="inline-formula" id="inline-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Through similar reasoning, we propose a set of BAF calculation rules for all types of SNPs (
                              <strong>
                               <a id="xref-fig-2-5" class="xref-fig" href="#F2">
                                Figure 2D
                               </a>
                              </strong>
                              ). The same principles are generalizable to non-CpG cytosine methylation, known to occur at low percentages in the mammalian genome (
                              <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">
                               Schultz et al., 2015
                              </a>
                              ). Therefore, our BAF calculation rules are robust to both CpG and non-CpG cytosine methylation. Note that, since only one strand is informative with regard to the allelic imbalance at C>T, T>C, A>G and G>A SNPs, BAF estimates at these sites are based on lower effective sequencing depth.
                             </p>
                             <p id="p-20">
                              We validated our approach by comparing genotype calls and BAF estimates at SNP loci on a subset of adjacent normal samples subjected to both RRBS and whole-genome sequencing (WGS,
                              <strong>
                               Methods
                              </strong>
                              ). We calculated false positive rates (FPR) and false negative rates (FNR) for SNP calling on RRBS data using the WGS-derived genotypes as ground truth. The average FPR across all SNP types and samples was 0.3% whilst the mean FNR was 25% (
                              <strong>
                               <a id="xref-fig-11-1" class="xref-fig" href="#F11">
                                Figure S3A,B
                               </a>
                              </strong>
                              ). Polymorphic CCGGs perturbing or creating an MspI recognition motif, resulting in allele-specific fragments during RRBS library preparation, and thereby skewing allelic coverage, were the main cause of false negatives (49%) (
                              <strong>
                               <a id="xref-fig-11-2" class="xref-fig" href="#F11">
                                Figure S3C,D
                               </a>
                              </strong>
                              ). Within the false negative calls, we saw a bias towards SNPs erroneously called homozygous reference (72%) compared to homozygous alternate (28%) (
                              <strong>
                               <a id="xref-fig-11-3" class="xref-fig" href="#F11">
                                Figure S3B
                               </a>
                              </strong>
                              ), which points towards an alignment bias, likely due to the limited mappability of short MspI fragments with alternate alleles.
                             </p>
                             <p id="p-21">
                              Leveraging the BAF and LogR values, we infer tumor purity and allele-specific copy number profiles. We call this CAMDAC module ASCAT.m, as our approach relies on ASCAT (
                              <a id="xref-ref-90-2" class="xref-bibr" href="#ref-90">
                               Van Loo et al., 2010
                              </a>
                              ), but alternative methods such as Battenberg (
                              <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">
                               Nik-Zainal et al., 2012
                              </a>
                              ) or ABSOLUTE (
                              <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">
                               Carter et al., 2012
                              </a>
                              ) may equally be used. Evaluating ASCAT.m copy number calling by comparing segmented BAF and LogR, and final allele-specific copy number calls to those obtained through WGS of the same tumor samples, confirms the accuracy and robustness of the ASCAT.m estimates (
                              <strong>
                               <a id="xref-fig-2-6" class="xref-fig" href="#F2">
                                Figure 2E
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-22">
                              Comparison of the ASCAT.m RRBS-derived (mean coverage 97x) tumor purity (ρ) and ploidy (ψ) values with those inferred from WGS (mean coverage 67x) and whole-exome sequencing (WES, mean coverage 464x) performed on the same samples (
                              <strong>
                               Table S3
                              </strong>
                              ,
                              <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">
                               Jamal-Hanjani et al., 2017
                              </a>
                              ) shows excellent agreement (WGS: cor
                              <sub>
                               ρ
                              </sub>
                              = 0.996, cor
                              <sub>
                               ψ
                              </sub>
                              = 0.991, WES: cor
                              <sub>
                               ρ
                              </sub>
                              = 0.984, cor
                              <sub>
                               ψ
                              </sub>
                              = 0.978) (
                              <strong>
                               <a id="xref-fig-2-7" class="xref-fig" href="#F2">
                                Figures 2F-G
                               </a>
                              </strong>
                              ). On average, 89.1% of RRBS-derived allele-specific copy number estimates exactly matched those obtained from WES- and WGS data (
                              <strong>
                               <a id="xref-fig-2-8" class="xref-fig" href="#F2">
                                Figure 2H
                               </a>
                              </strong>
                              ). This percentage increased to 98.4% when allowing a single allele to differ by one copy. Profiling the fraction of the genome with loss of heterozygosity (LOH) and ploidy confirms that whole-genome doubling was a frequent event across the cohort (LUAD: 72%, LUSC: 89%,
                              <strong>
                               <a id="xref-fig-2-9" class="xref-fig" href="#F2">
                                Figure 2I
                               </a>
                              </strong>
                              ,
                              <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                               Dentro et al., 2021
                              </a>
                              ;
                              <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">
                               Jamal-Hanjani et al., 2017
                              </a>
                              ). In addition, despite low sample sizes, cohort-wide copy-number profiles closely resembled those derived from TCGA LUAD and LUSC cases (
                              <strong>
                               <a id="xref-fig-2-10" class="xref-fig" href="#F2">
                                Figure 2J
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-23">
                              In conclusion, CAMDAC’s ASCAT.m module allows accurate allele-specific copy number profiling and tumor purity estimation from methylation data, obviating the need to perform separate copy number profiling experiments.
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              SNP-independent methylation rate estimation
                             </h3>
                             <p id="p-24">
                              Similar to methylation at CpG SNPs affecting BAF calculations, polymorphisms confound methylation rate estimation (
                              <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">
                               Liu et al., 2012
                              </a>
                              ). Polymorphisms at CpGs account for 49.9% of SNPs, 83.3% of which are CpG>TpG polymorphisms, and methylation rates at these positions using standard approaches show markedly different distributions (
                              <strong>
                               <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                                Figure 3A-C
                               </a>
                              </strong>
                              ). In tumor samples, this effect shows copy number dependence (
                              <strong>
                               <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                                Figure 3D-F
                               </a>
                              </strong>
                              ).
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F3.large.jpg?width=800&height=600&carousel=1" title="Calculating polymorphism-independent methylation rates with CAMDAC. (A-C) Naïve versus CAMDAC normal methylation rate estimates at (A) all CpGs, (B) polymorphic CpGs and (C) CpG>TpG SNPs, selected from a random sample of 3,000,000 CpGs from this cohort’s 37 normal lung samples. (D-F) Naïve versus CAMDAC bulk tumor methylation rate estimates for CpG>TpG SNPs in segments with total copy number (D) 1, (E) 2 and (F) 3, pooling the data from all 3 sampled regions from patient CRUK0084 of near-equal and high tumor purity (range 0.85-0.87). The data points highlighted by the orange, green and yellow circles indicate heterozygous C>T SNPs with CpG allele copy number 1 and the pink circle CpGs with copy number 2. (G) Derivation of methylation rate estimates at non-polymorphic CpGs. (H) Derivation of the CpG-forming allele-specific methylation rate at a CpG>TpG SNP. (I) Methylation rate formulae for all possible polymorphic CpGs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Calculating polymorphism-independent methylation rates with CAMDAC.</span> (A-C) Naïve versus CAMDAC normal methylation rate estimates at (A) all CpGs, (B) polymorphic CpGs and (C) CpG>TpG SNPs, selected from a random sample of 3,000,000 CpGs from this cohort’s 37 normal lung samples. (D-F) Naïve versus CAMDAC bulk tumor methylation rate estimates for CpG>TpG SNPs in segments with total copy number (D) 1, (E) 2 and (F) 3, pooling the data from all 3 sampled regions from patient CRUK0084 of near-equal and high tumor purity (range 0.85-0.87). The data points highlighted by the orange, green and yellow circles indicate heterozygous C>T SNPs with CpG allele copy number 1 and the pink circle CpGs with copy number 2. (G) Derivation of methylation rate estimates at non-polymorphic CpGs. (H) Derivation of the CpG-forming allele-specific methylation rate at a CpG>TpG SNP. (I) Methylation rate formulae for all possible polymorphic CpGs.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F3.medium.gif" width="440" height="415"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F3.medium.gif" width="440" height="415"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3.
                               </span>
                               <span class="caption-title">
                                Calculating polymorphism-independent methylation rates with CAMDAC.
                               </span>
                               <p id="p-25" class="first-child">
                                (A-C) Naïve versus CAMDAC normal methylation rate estimates at (A) all CpGs, (B) polymorphic CpGs and (C) CpG>TpG SNPs, selected from a random sample of 3,000,000 CpGs from this cohort’s 37 normal lung samples. (D-F) Naïve versus CAMDAC bulk tumor methylation rate estimates for CpG>TpG SNPs in segments with total copy number (D) 1, (E) 2 and (F) 3, pooling the data from all 3 sampled regions from patient CRUK0084 of near-equal and high tumor purity (range 0.85-0.87). The data points highlighted by the orange, green and yellow circles indicate heterozygous C>T SNPs with CpG allele copy number 1 and the pink circle CpGs with copy number 2. (G) Derivation of methylation rate estimates at non-polymorphic CpGs. (H) Derivation of the CpG-forming allele-specific methylation rate at a CpG>TpG SNP. (I) Methylation rate formulae for all possible polymorphic CpGs.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-26">
                              To address these biases, we developed an approach to obtain methylation rates independent of SNP genotype (
                              <strong>
                               <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                                Figure 3G-I
                               </a>
                              </strong>
                              ,
                              <strong>
                               Methods
                              </strong>
                              ). A number of considerations need to be made for this. (i) The original top and original bottom strand in directional protocols encode methylation information for the first and second position of a CpG, respectively. As a result, in case of a CpG>TpG SNP, the unmethylated reference and alternate alleles are distinguishable only on the bottom strand. Likewise, at CpG>CpA SNPs, only the top strand may be used. (ii) Computing the methylation rate at the non-polymorphic position in the CpG, only dinucleotides allow to separate the respective contributions from the unmethylated CpG allele and the alternative allele. (iii) To ensure copy number independence of methylation rate estimates, we define them as the average methylation rate per CpG allele. This enables the methylation rate at a heterozygous CpG to vary between 0 and 1, rather than between 0 and 0.5 in a diploid sample, and ensures further independence between methylation rate and copy number estimates. Methylation rates per CpG are thus computed by aggregating strand-specific dinucleotides as follows:
                              <span class="inline-formula" id="inline-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , except at C>T and G>A SNPs, where only reads from the bottom strand and top strand, respectively, contribute to the estimates (
                              <strong>
                               <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                                Figure 3G-I
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-27">
                              When accounting for these confounders, the observed biases disappear (
                              <strong>
                               <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                                Figures 3A-F
                               </a>
                              </strong>
                              ,
                              <strong>
                               <a id="xref-fig-12-1" class="xref-fig" href="#F12">
                                S4
                               </a>
                              </strong>
                              ), confirming that CAMDAC CpG methylation rate calculation is robust to the presence of polymorphisms.
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Deconvolving tumor methylation profiles
                             </h3>
                             <p id="p-28">
                              Building upon the allele-specific copy number, tumor purity and methylation rate obtained with CAMDAC, we aim to separate tumor and normal methylation signals. When a tumor clone is methylated at a CpG site that is unmethylated in the admixed normal cells (
                              <strong>
                               <a id="xref-fig-9-2" class="xref-fig" href="#F9">
                                Figures S1A-G
                               </a>
                              </strong>
                              ), the fraction of methylated reads at that locus observed from bulk bisulfite sequencing increases along with tumor purity and copy number (
                              <strong>
                               <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                                Figure 1B
                               </a>
                              </strong>
                              ). Vice versa, if the tumor clone lost methylation compared to the normal, the observed methylation rate will decrease with increasing purity and copy number (
                              <strong>
                               <a id="xref-fig-9-3" class="xref-fig" href="#F9">
                                Figure S1H
                               </a>
                              </strong>
                              ). We formally modeled this effect, realizing that the bulk tumor methylation rate (m
                              <sub>
                               b
                              </sub>
                              ) is the sum of a tumor component (with methylation rate m
                              <sub>
                               %
                              </sub>
                              ) and a normal component (with methylation rate m
                              <sub>
                               t
                              </sub>
                              ), weighted by their relative DNA proportions, which can be calculated as the product of purity and copy number (ρ and 1 − ρ, and n
                              <sub>
                               t
                              </sub>
                              and n
                              <sub>
                               n
                              </sub>
                              for tumor and normal, respectively):
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Applying this equation to our non-small cell lung cancer RRBS data, using the adjacent patient-matched normal samples (or a pooled unmatched normal lung reference for sample CRUK0047,
                              <strong>
                               Methods
                              </strong>
                              ) as a proxy for the admixed normal cells, enables interpretation of the observed bulk methylation signals. Indeed, the positions of peaks of clonal bi-allelic tumor-normal differential methylation align closely with those predicted under this model, for different values of tumor purity and copy number (
                              <strong>
                               <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                                Figures 1B
                               </a>
                               ,
                               <a id="xref-fig-9-4" class="xref-fig" href="#F9">
                                S1H
                               </a>
                              </strong>
                              ), and this relationship holds across all tumor samples (
                              <strong>
                               <a id="xref-fig-13-1" class="xref-fig" href="#F13">
                                Figure S5A,B
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-29">
                              While these data show that our CAMDAC model can account for copy number and tumor purity, not all CpG methylation values appear at the expected peaks. Allele-specific methylation is known to be widespread, not only on the inactive X chromosome in females and at imprinted genes, but also at polymorphic regulatory sequences (
                              <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                               Kerkel et al., 2008
                              </a>
                              ;
                              <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">
                               Reik and Walter, 2001
                              </a>
                              ), and this signal is also apparent in the overall methylation rates (
                              <strong>
                               <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                                Figures 1B
                               </a>
                               ,
                               <a id="xref-fig-9-5" class="xref-fig" href="#F9">
                                S1G-H
                               </a>
                              </strong>
                              ). In addition, intra-tumor heterogeneity is expected to contribute intermediate methylation rate signals (
                              <a id="xref-ref-82-3" class="xref-bibr" href="#ref-82">
                               Sheffield et al., 2017
                              </a>
                              ).
                             </p>
                             <p id="p-30">
                              Considering the above, (i) we have formalized the relationship between the bulk and purified tumor methylation rates; (ii) we can directly estimate tumor purity and copy number for each CpG from the bisulfite converted data; and (iii) an adjacent normal sample can be used as a proxy for the methylation rate of the admixed normal cells. We therefore have the necessary information to solve
                              <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">
                               Eq(1)
                              </a>
                              for the purified tumor methylation rate:
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Taken together, bulk tumor methylation rates are confounded by sample purity and local tumor copy number in a way that can be modeled. Formalizing these effects, and inferring purity and copy number from bisulfite sequencing data, CAMDAC corrects bulk methylation rates to yield deconvolved tumor methylation rate estimates.
                             </p>
                             <p id="p-31">
                              Post-deconvolution, we observe CpGs that have become methylated on all chromosome copies in all tumor cells to have purified tumor methylation rates approaching the expected maximum m
                              <sub>
                               t
                              </sub>
                              = 1. Vice versa, tumor-normal hypomethylated loci approach m
                              <sub>
                               t
                              </sub>
                              = 0 (
                              <strong>
                               <a id="xref-fig-13-2" class="xref-fig" href="#F13">
                                Figure S5C
                               </a>
                              </strong>
                              ). Note that the noise on these estimates is proportional to tumor purity and hence statistical power (
                              <strong>
                               <a id="xref-fig-13-3" class="xref-fig" href="#F13">
                                Figures S5D-E
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-32">
                              To evaluate the use of an adjacent normal sample as a proxy for the tumor-admixed normal cell, we used Fluorescent-Activated Cell Sorting (FACS) by DNA content to experimentally separate diploid “normal” cell populations from five bulk tumor samples with clonal whole-genome doubling (average ploidy = 3.53) and performed RRBS on each sorted population (
                              <strong>
                               Methods
                              </strong>
                              ). The FACS-purified and adjacent normal methylomes were highly correlated overall (
                              <strong>
                               <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                                Figures 4A
                               </a>
                               ,
                               <a id="xref-fig-14-1" class="xref-fig" href="#F14">
                                S6A
                               </a>
                              </strong>
                              ). We next classified CpGs into three groups: (i) confidently unmethylated (UM, HDI
                              <sup>
                               99
                              </sup>
                              ⊆ [0, 0.2]), (ii) intermediate (IM, HDI
                              <sup>
                               99
                              </sup>
                              ⊆ [0.2, 0.8]) and (iii) fully methylated (FM, HDI
                              <sup>
                               99
                              </sup>
                              ⊆ [0.8, 1]) and evaluated correspondence across patient-matched FACS-purified and adjacent normal sample pairs (
                              <strong>
                               <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                                Figures 4B
                               </a>
                               ,
                               <a id="xref-fig-14-2" class="xref-fig" href="#F14">
                                S6B,C
                               </a>
                              </strong>
                              ). We observed minimal differential methylation and a gene set enrichment analysis did not reveal any clear gene signatures, suggesting comparatively small differences in cellular composition. These results confirm that in non-small cell lung cancer and at the scale of our biopsies, an adjacent normal sample is an appropriate proxy for the tumor admixed normal.
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F4.large.jpg?width=800&height=600&carousel=1" title="CAMDAC purified methylation profiles. (A) CpG-wise comparison of the adjacent (mn) and FACS-purified (mf,n) normal methylation rates with Pearson correlation annotates. (B) CpG classification based on mn and mf,n. (C) Correlation between bulk tumor, CAMDAC deconvolved tumor, and adjacent normal methylation profiles. Lollipops represent distances between patient-matched adjacent and FACS-purified normal pairs (green) and between CAMDAC- and FACS-purified tumor methylomes of the same samples (blue). (D) Phasing CpGs to SNVs in regions of LOH enables separation of tumor and normal reads. SNV-purified tumor methylation rates are readily obtained from mutant allele read counts methylated and unmethylated. Remaining reads may be assigned to the wild type allele. (E) Comparison of variant allele frequencies (VAF) of single-nucleotide variants (SNV) derived from RRBS and whole genome/exome sequencing in regions of LOH. (F-G) Validation of CAMDAC methylation rates (F) through phasing of SNVs in regions of LOH and comparison to bulk tumor methylation rates (G)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">CAMDAC purified methylation profiles.</span> (A) CpG-wise comparison of the adjacent (<strong><em>m<sub>n</sub></em></strong>) and FACS-purified (<strong><em>m<sub>f,n</sub></em></strong>) normal methylation rates with Pearson correlation annotates. (B) CpG classification based on <strong><em>m<sub>n</sub></em></strong> and <strong><em>m<sub>f,n</sub></em></strong>. (C) Correlation between bulk tumor, CAMDAC deconvolved tumor, and adjacent normal methylation profiles. Lollipops represent distances between patient-matched adjacent and FACS-purified normal pairs (green) and between CAMDAC- and FACS-purified tumor methylomes of the same samples (blue). (D) Phasing CpGs to SNVs in regions of LOH enables separation of tumor and normal reads. SNV-purified tumor methylation rates are readily obtained from mutant allele read counts methylated and unmethylated. Remaining reads may be assigned to the wild type allele. (E) Comparison of variant allele frequencies (VAF) of single-nucleotide variants (SNV) derived from RRBS and whole genome/exome sequencing in regions of LOH. (F-G) Validation of CAMDAC methylation rates (F) through phasing of SNVs in regions of LOH and comparison to bulk tumor methylation rates (G).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F4.medium.gif" width="440" height="317"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F4.medium.gif" width="440" height="317"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4.
                               </span>
                               <span class="caption-title">
                                CAMDAC purified methylation profiles.
                               </span>
                               <p id="p-33" class="first-child">
                                (A) CpG-wise comparison of the adjacent (
                                <strong>
                                 m
                                 <sub>
                                  n
                                 </sub>
                                </strong>
                                ) and FACS-purified (
                                <strong>
                                 m
                                 <sub>
                                  f,n
                                 </sub>
                                </strong>
                                ) normal methylation rates with Pearson correlation annotates. (B) CpG classification based on
                                <strong>
                                 m
                                 <sub>
                                  n
                                 </sub>
                                </strong>
                                and
                                <strong>
                                 m
                                 <sub>
                                  f,n
                                 </sub>
                                </strong>
                                . (C) Correlation between bulk tumor, CAMDAC deconvolved tumor, and adjacent normal methylation profiles. Lollipops represent distances between patient-matched adjacent and FACS-purified normal pairs (green) and between CAMDAC- and FACS-purified tumor methylomes of the same samples (blue). (D) Phasing CpGs to SNVs in regions of LOH enables separation of tumor and normal reads. SNV-purified tumor methylation rates are readily obtained from mutant allele read counts methylated and unmethylated. Remaining reads may be assigned to the wild type allele. (E) Comparison of variant allele frequencies (VAF) of single-nucleotide variants (SNV) derived from RRBS and whole genome/exome sequencing in regions of LOH. (F-G) Validation of CAMDAC methylation rates (F) through phasing of SNVs in regions of LOH and comparison to bulk tumor methylation rates (G).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-34">
                              We evaluated the purified tumor methylation profiles by comparing Pearson distances between pairs of methylation profiles. Compared to bulk signals, CAMDAC deconvolved tumor methylomes show increased distances in inter-patient and tumor-adjacent normal comparisons, while different tumor regions of the same patient remain similar (
                              <strong>
                               <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                                Figure 4C
                               </a>
                              </strong>
                              ). Distances observed between purified tumor profiles computed with CAMDAC and those obtained experimentally by FACS were low and comparable to those observed between samples taken from the same patient. These observations are in line with admixed normal signals being removed from the bulk tumor and tumor-specific signals being retained in CAMDAC-deconvolved methylation profiles.
                             </p>
                             <p id="p-35">
                              Leveraging single nucleotide variant (SNV) calls from additional WGS and previously published WES (
                              <a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">
                               Jamal-Hanjani et al., 2017
                              </a>
                              ), we validate CAMDAC purified tumor methylation rates by phasing CpG methylation to clonal SNVs present on all chromosome copies in regions with loss of heterozygosity (
                              <strong>
                               <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                                Figure 4D
                               </a>
                              </strong>
                              ). At these sites, any read reporting the variant allele can directly be assigned to the tumor cells, and methylation rates obtained from this subset of reads should be an unbiased estimate of the purified tumor methylation rate. Overall, RRBS-derived variant allele frequency (VAF) estimates of somatic SNVs (computed analogously to BAF values of germline SNPs,
                              <strong>
                               <a id="xref-fig-2-11" class="xref-fig" href="#F2">
                                Figure 2D
                               </a>
                              </strong>
                              ) were highly correlated with matched WGS/WES data (Pearson correlation = 0.86,
                              <strong>
                               <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                                Figure 4E
                               </a>
                              </strong>
                              ). In total, we obtained phased methylation estimates at 4,485 CpG loci across our dataset and observed strong correlation between these SNV-purified m
                              <sub>
                               %
                              </sub>
                              values and CAMDAC estimates (Pearson correlation = 0.97,
                              <strong>
                               <a id="xref-fig-4-6" class="xref-fig" href="#F4">
                                Figure 4F
                               </a>
                              </strong>
                              ), but not bulk tumor methylation rates (
                              <strong>
                               <a id="xref-fig-4-7" class="xref-fig" href="#F4">
                                Figure 4G
                               </a>
                              </strong>
                              ). These results confirm that CAMDAC can accurately deconvolute tumor methylation rates from bulk RRBS data.
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Inferring differential methylation from CAMDAC methylation rates
                             </h3>
                             <p id="p-36">
                              We next set out to formally identify differentially methylated CpGs. For tumor-normal comparisons, we directly compute the probabilities P(m
                              <sub>
                               t
                              </sub>
                              > m
                              <sub>
                               n
                              </sub>
                              ) and P(m
                              <sub>
                               t
                              </sub>
                              < m
                              <sub>
                               n
                              </sub>
                              ) of hyper- and hypomethylation, respectively. Substituting m
                              <sub>
                               t
                              </sub>
                              by
                              <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">
                               Eq(2)
                              </a>
                              and modelling the observed bulk and normal methylated read counts in a Bayesian fashion using beta-binomial distributions, the resulting probability density is a scaled difference of two beta posteriors (m
                              <sub>
                               n
                              </sub>
                              and m
                              <sub>
                               b
                              </sub>
                              ∼ Beta(#reads
                              <sub>
                               meth
                              </sub>
                              , #reads
                              <sub>
                               unmeth
                              </sub>
                              )), which we can compute exactly (
                              <strong>
                               Methods
                              </strong>
                              ):
                              <span class="disp-formula" id="disp-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">
                               Equations 2
                              </a>
                              and
                              <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">
                               3
                              </a>
                              reveal how coverage, copy number and tumor purity affect the power to infer tumor–normal differentially methylated positions (DMPs). Directly, the variance of m
                              <sub>
                               t
                              </sub>
                              and m
                              <sub>
                               b
                              </sub>
                              decreases with increasing normal and tumor coverage, respectively. Indirectly, increasing tumor copy number results in higher local coverage but also, together with increasing purity, shifts m
                              <sub>
                               b
                              </sub>
                              away from m
                              <sub>
                               n
                              </sub>
                              at DMPs. To identify tumor–tumor DMPs, we similarly compute P(m
                              <sub>
                               t1
                              </sub>
                              > m
                              <sub>
                               %,
                              </sub>
                              ) and P(m
                              <sub>
                               t1
                              </sub>
                              < m
                              <sub>
                               %,
                              </sub>
                              ) by resampling m
                              <sub>
                               t
                              </sub>
                              from the posterior distributions of m
                              <sub>
                               n
                              </sub>
                              and m
                              <sub>
                               b
                              </sub>
                              in
                              <a id="xref-disp-formula-2-3" class="xref-disp-formula" href="#disp-formula-2">
                               Eq(2)
                              </a>
                              (
                              <strong>
                               Methods
                              </strong>
                              ).
                             </p>
                             <p id="p-37">
                              For differential methylation analysis, it is customary to set a minimal effect size threshold, which we set to Δm > 0.2. In contrast to m
                              <sub>
                               b
                              </sub>
                              , setting this threshold using m
                              <sub>
                               t
                              </sub>
                              removes dependence on purity and copy number. Simulations based on our observed data (
                              <strong>
                               <a id="xref-fig-15-1" class="xref-fig" href="#F15">
                                Figure S7
                               </a>
                               , Methods
                              </strong>
                              ) reveal that true positive bi-allelic and mono-allelic DMPs (in balanced regions) indeed show absolute m
                              <sub>
                               t
                              </sub>
                              − m
                              <sub>
                               n
                              </sub>
                              methylation rate differences near 1 and 0.5 respectively (
                              <strong>
                               <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                                Figures 5A
                               </a>
                               and
                               <a id="xref-fig-15-2" class="xref-fig" href="#F15">
                                S7D
                               </a>
                               , Methods
                              </strong>
                              ). Thresholding on m
                              <sub>
                               t
                              </sub>
                              reduces the number of false negative tumor–normal DMP calls in impure tumors and at loci with lower copy number states, while retaining a low false positive rate (
                              <strong>
                               <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                                Figures 5B-C
                               </a>
                               and
                               <a id="xref-fig-15-3" class="xref-fig" href="#F15">
                                S7E-F
                               </a>
                              </strong>
                              ). Similarly, failure to adjust m
                              <sub>
                               b
                              </sub>
                              for purity and copy number can readily inundate tumor-tumor differential methylation results with false positive DMPs. Use of CAMDAC m
                              <sub>
                               t
                              </sub>
                              values reduces false positives in tumor-tumor comparisons while retaining a similarly low rate of false negatives (
                              <strong>
                               <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                                Figures 5D
                               </a>
                               and
                               <a id="xref-fig-15-4" class="xref-fig" href="#F15">
                                S7G
                               </a>
                              </strong>
                              ).
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F5.large.jpg?width=800&height=600&carousel=1" title="Differential methylation calling using CAMDAC. (A-C) Tumor-normal DMP simulation results. Average methylation difference (A), false negative (B) and false positive rates (C), comparing low versus high simulated tumor purities. (D) Tumor-tumor DMP simulation results. False negative and false positive rates as a function of the copy number state, averaged across simulated sample pairs of low (left panel), low versus high (middle panel) and high (right panel) tumor purities with CAMDAC deconvolved (top row) and bulk (bottom row) tumor methylation rates. (E) Venn diagrams of the overlap between real CAMDAC- and FACS-purified tumor-normal DMPs. The value shown is the percentage of FACS-purified tumor-normal DMPs also called by CAMDAC. Samples are ordered by decreasing power. (F) Comparing observed tumor-tumor differential methylation calls between CRUK0062 regions from bulk (grey) and CAMDAC (blue) approaches. Samples are ordered by fraction tumor DNA content. (G) Venn diagrams showing the overlap of tumor-tumor DMPs between bulk and CAMDAC calls for intra- and inter-tumor DMPs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Differential methylation calling using CAMDAC.</span> (A-C) Tumor-normal DMP simulation results. Average methylation difference (A), false negative (B) and false positive rates (C), comparing low versus high simulated tumor purities. (D) Tumor-tumor DMP simulation results. False negative and false positive rates as a function of the copy number state, averaged across simulated sample pairs of low (left panel), low versus high (middle panel) and high (right panel) tumor purities with CAMDAC deconvolved (top row) and bulk (bottom row) tumor methylation rates. (E) Venn diagrams of the overlap between real CAMDAC- and FACS-purified tumor-normal DMPs. The value shown is the percentage of FACS-purified tumor-normal DMPs also called by CAMDAC. Samples are ordered by decreasing power. (F) Comparing observed tumor-tumor differential methylation calls between CRUK0062 regions from bulk (grey) and CAMDAC (blue) approaches. Samples are ordered by fraction tumor DNA content. (G) Venn diagrams showing the overlap of tumor-tumor DMPs between bulk and CAMDAC calls for intra- and inter-tumor DMPs.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F5.medium.gif" width="399" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F5.medium.gif" width="399" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5.
                               </span>
                               <span class="caption-title">
                                Differential methylation calling using CAMDAC.
                               </span>
                               <p id="p-38" class="first-child">
                                (A-C) Tumor-normal DMP simulation results. Average methylation difference (A), false negative (B) and false positive rates (C), comparing low versus high simulated tumor purities. (D) Tumor-tumor DMP simulation results. False negative and false positive rates as a function of the copy number state, averaged across simulated sample pairs of low (left panel), low versus high (middle panel) and high (right panel) tumor purities with CAMDAC deconvolved (top row) and bulk (bottom row) tumor methylation rates. (E) Venn diagrams of the overlap between real CAMDAC- and FACS-purified tumor-normal DMPs. The value shown is the percentage of FACS-purified tumor-normal DMPs also called by CAMDAC. Samples are ordered by decreasing power. (F) Comparing observed tumor-tumor differential methylation calls between CRUK0062 regions from bulk (grey) and CAMDAC (blue) approaches. Samples are ordered by fraction tumor DNA content. (G) Venn diagrams showing the overlap of tumor-tumor DMPs between bulk and CAMDAC calls for intra- and inter-tumor DMPs.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-39">
                              To gauge performance on real data, we measured the overlap between tumor-normal DMP calls based on CAMDAC m
                              <sub>
                               t
                              </sub>
                              and those obtained from FACS-purified tumor RRBS data of the same tumor regions (m
                              <sub>
                               f,t
                              </sub>
                              ,
                              <strong>
                               Methods
                              </strong>
                              ). While samples were processed from different tissue cuts of the same tumor regions and sequenced to different depths, DMP calls showed 77.3% agreement (
                              <strong>
                               <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                                Figure 5E
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-40">
                              We next compared intra-tumor DMPs called using m
                              <sub>
                               b
                              </sub>
                              or m
                              <sub>
                               t
                              </sub>
                              for patient CRUK0062, selected for its large number of samples with varying tumor purity (
                              <strong>
                               <a id="xref-fig-5-5" class="xref-fig" href="#F5">
                                Figure 5F
                               </a>
                              </strong>
                              ). In this setting, 80.9% of CAMDAC m
                              <sub>
                               t
                              </sub>
                              -based calls are also identified using m
                              <sub>
                               b
                              </sub>
                              and the effect of tumor purity on power can readily be seen as an increase in the number of DMPs identified with sample purity. Using m
                              <sub>
                               t
                              </sub>
                              , more DMPs are called when both samples are high purity, i.e. statistical power is the highest. In contrast, when using m
                              <sub>
                               b
                              </sub>
                              , more DMPs are called when two samples differ more in purity, suggesting the majority are false positives. These findings are in line with our simulation results and suggest that also in real data, controlling methylation rates for tumor purity and copy number greatly reduces the number of false positive DMP calls, while maintaining low false negative rates.
                             </p>
                             <p id="p-41">
                              To get a global overview of the performance of CAMDAC m
                              <sub>
                               t
                              </sub>
                              and bulk m
                              <sub>
                               b
                              </sub>
                              for tumor-tumor differential methylation analysis, we randomly selected CpG loci from samples with a low or high tumor purity both within and between patients and obtained DMP calls (
                              <strong>
                               Methods
                              </strong>
                              ). As expected, results showed a greater number of DMPs for inter-patient comparisons than between samples of shared clonal origins (
                              <strong>
                               <a id="xref-fig-5-6" class="xref-fig" href="#F5">
                                Figure 5G
                               </a>
                              </strong>
                              ). Furthermore, DMP calls unique to the bulk tumor were frequent between samples of differing purities taken from the same patient, in line with the high false-positive rate of DMP calling without deconvolution.
                             </p>
                             <p id="p-42">
                              When investigating disease-related DNA methylation changes, researchers commonly look for differentially methylated regions (DMRs) as opposed to individual CpGs (
                              <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">
                               Robinson et al., 2014
                              </a>
                              ). Building on DMP calls, CAMDAC can identify DMRs by binning CpGs into neighborhoods and identifying DMP hotspots within these clusters (
                              <strong>
                               Methods
                              </strong>
                              ).
                             </p>
                             <p id="p-43">
                              Taken together, analyses of both simulated and observed data show that CAMDAC enables accurate calling of both tumor-normal and tumor-tumor differential methylation from RRBS data.
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              Quantifying allele-specific methylation
                             </h3>
                             <p id="p-44">
                              While most CpGs are either fully methylated or fully unmethylated in the tumor, others show intermediate m
                              <sub>
                               t
                              </sub>
                              values (
                              <strong>
                               <a id="xref-fig-1-6" class="xref-fig" href="#F1">
                                Figures 1B
                               </a>
                               ,
                               <a id="xref-fig-4-8" class="xref-fig" href="#F4">
                                4G
                               </a>
                               ,
                               <a id="xref-fig-9-6" class="xref-fig" href="#F9">
                                S1G
                               </a>
                              </strong>
                              ). We hypothesize that, in addition to intra-tumor epigenetic heterogeneity, part of this is due to allele-specific differential methylation, and we set out to quantify this phenomenon. Note that, by construction, allele-specific methylation at heterozygous CpGs does not contribute to this intermediate m
                              <sub>
                               t
                              </sub>
                              signal (
                              <strong>
                               <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                                Figure 3H-I
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-45">
                              To assess CAMDAC sensitivity for intermediate methylation signals, we first evaluated methylation rates at the imprinted IGF2/H19 locus (
                              <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                               Frevel et al., 1999
                              </a>
                              ). As expected, the normal methylation rate around the imprinting control region is confidently around m
                              <sub>
                               t
                              </sub>
                              = 0.5, whilst the CpG island shore is fully methylated (e.g. CRUK0073,
                              <strong>
                               <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                                Figure 6A
                               </a>
                              </strong>
                              ). Since imprinting results in allele-specific methylation, we aimed to validate this across known imprinted loci (Geneimprint database,
                              <strong>
                               Methods
                              </strong>
                              ) by phasing CpGs to nearby germline heterozygous SNPs. In total, we detected allele-specific methylation at 100/106 (94%) loci in normal samples with phased methylation rates and leveraged the CAMDAC equations to deconvolve allele-specific tumor methylation rates (
                              <strong>
                               Methods
                              </strong>
                              ).
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F6.large.jpg?width=800&height=600&carousel=1" title="Quantifying allele-specific methylation. (A) CRUK0073 normal (CRUK0073-N, top) and CAMDAC purified tumor (CRUK0073-R2, middle) CpG methylation rate along the H19 locus with methylation rate point estimates shown as black dots, HDI99 displayed as a grey ribbon and genomic annotations showing the transcript and promoter region as well as neighboring CpG islands, enhancers and the imprinting control region (ICR, bottom panel). Phased methylation rates to both parental alleles are shown (purple and yellow). (B) Overview of somatic changes at known imprinted loci with phased methylation information across tumor samples. LOI: loss of imprinting; LOH: loss of heterozygosity. (C) Allele-specific methylation signals on chromosome X in adjacent normal lung samples across female and male patients. (D, E) Comparison of (D) sequencing depth and (E) mean BAF at heterozygous SNPs on chromosome X to that on autosomes in males and females. (F) Deconvolved pure tumor methylation rates from CRUK0084-R1, including only phaseable CpGs which are confidently unmethylated in the matched adjacent normal sample and located on segments with allele-specific copy number 1+1. (G) Scatter plot of the allele-specific pure tumor methylation rates of the CpGs loci in (F). (H-I) Comparison of (H) pure tumor methylation rate differences and (I) average pure tumor methylation rates between alleles across samples pairs for the same patient reveals subclonal relationships. (J) SNV-based phylogenetic clone tree for CRUK0084 taken from Jamal-Hanjani et al. (Jamal-Hanjani et al., 2017). (K) UMAP of allele-specific pure tumor methylation rates including CpGs that are hypermethylated in at least 1 sample. Hierarchical clustering and analysis of cluster methylation rates enables epimutation assignments, taking copy number into account. (L) Total number of unique epimutations per class (top panel) and their ratio (bottom panel) per patient, colored by clonality and allele-specificity. (M) Allele-specific (left) and clonal (middle) and clonal bi-allelic only (right) DMP odds ratio given clinical features including histological subtypes (LUAD and LUSC), stage (1a and 1b against later stages) and genomic annotations, including whole genome doubling status and smoking signature SBS4 exposure (high > 0.35, low otherwise) and number of samples sequenced (high > 3, low otherwise)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Quantifying allele-specific methylation.</span> (A) CRUK0073 normal (CRUK0073-N, top) and CAMDAC purified tumor (CRUK0073-R2, middle) CpG methylation rate along the H19 locus with methylation rate point estimates shown as black dots, HDI<sub>99</sub> displayed as a grey ribbon and genomic annotations showing the transcript and promoter region as well as neighboring CpG islands, enhancers and the imprinting control region (ICR, bottom panel). Phased methylation rates to both parental alleles are shown (purple and yellow). (B) Overview of somatic changes at known imprinted loci with phased methylation information across tumor samples. LOI: loss of imprinting; LOH: loss of heterozygosity. (C) Allele-specific methylation signals on chromosome X in adjacent normal lung samples across female and male patients. (D, E) Comparison of (D) sequencing depth and (E) mean BAF at heterozygous SNPs on chromosome X to that on autosomes in males and females. (F) Deconvolved pure tumor methylation rates from CRUK0084-R1, including only phaseable CpGs which are confidently unmethylated in the matched adjacent normal sample and located on segments with allele-specific copy number 1+1. (G) Scatter plot of the allele-specific pure tumor methylation rates of the CpGs loci in (F). (H-I) Comparison of (H) pure tumor methylation rate differences and (I) average pure tumor methylation rates between alleles across samples pairs for the same patient reveals subclonal relationships. (J) SNV-based phylogenetic clone tree for CRUK0084 taken from Jamal-Hanjani et <em>al</em>. (Jamal-Hanjani et al., 2017). (K) UMAP of allele-specific pure tumor methylation rates including CpGs that are hypermethylated in at least 1 sample. Hierarchical clustering and analysis of cluster methylation rates enables epimutation assignments, taking copy number into account. (L) Total number of unique epimutations per class (top panel) and their ratio (bottom panel) per patient, colored by clonality and allele-specificity. (M) Allele-specific (left) and clonal (middle) and clonal bi-allelic only (right) DMP odds ratio given clinical features including histological subtypes (LUAD and LUSC), stage (1a and 1b against later stages) and genomic annotations, including whole genome doubling status and smoking signature SBS4 exposure (high > 0.35, low otherwise) and number of samples sequenced (high > 3, low otherwise).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F6.medium.gif" width="382" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F6.medium.gif" width="382" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6.
                               </span>
                               <span class="caption-title">
                                Quantifying allele-specific methylation.
                               </span>
                               <p id="p-46" class="first-child">
                                (A) CRUK0073 normal (CRUK0073-N, top) and CAMDAC purified tumor (CRUK0073-R2, middle) CpG methylation rate along the H19 locus with methylation rate point estimates shown as black dots, HDI
                                <sub>
                                 99
                                </sub>
                                displayed as a grey ribbon and genomic annotations showing the transcript and promoter region as well as neighboring CpG islands, enhancers and the imprinting control region (ICR, bottom panel). Phased methylation rates to both parental alleles are shown (purple and yellow). (B) Overview of somatic changes at known imprinted loci with phased methylation information across tumor samples. LOI: loss of imprinting; LOH: loss of heterozygosity. (C) Allele-specific methylation signals on chromosome X in adjacent normal lung samples across female and male patients. (D, E) Comparison of (D) sequencing depth and (E) mean BAF at heterozygous SNPs on chromosome X to that on autosomes in males and females. (F) Deconvolved pure tumor methylation rates from CRUK0084-R1, including only phaseable CpGs which are confidently unmethylated in the matched adjacent normal sample and located on segments with allele-specific copy number 1+1. (G) Scatter plot of the allele-specific pure tumor methylation rates of the CpGs loci in (F). (H-I) Comparison of (H) pure tumor methylation rate differences and (I) average pure tumor methylation rates between alleles across samples pairs for the same patient reveals subclonal relationships. (J) SNV-based phylogenetic clone tree for CRUK0084 taken from Jamal-Hanjani et al. (
                                <a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">
                                 Jamal-Hanjani et al., 2017
                                </a>
                                ). (K) UMAP of allele-specific pure tumor methylation rates including CpGs that are hypermethylated in at least 1 sample. Hierarchical clustering and analysis of cluster methylation rates enables epimutation assignments, taking copy number into account. (L) Total number of unique epimutations per class (top panel) and their ratio (bottom panel) per patient, colored by clonality and allele-specificity. (M) Allele-specific (left) and clonal (middle) and clonal bi-allelic only (right) DMP odds ratio given clinical features including histological subtypes (LUAD and LUSC), stage (1a and 1b against later stages) and genomic annotations, including whole genome doubling status and smoking signature SBS4 exposure (high > 0.35, low otherwise) and number of samples sequenced (high > 3, low otherwise).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-47">
                              In line with previous reports (
                              <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                               Cui et al., 2002
                              </a>
                              ;
                              <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">
                               Ogawa et al., 1993
                              </a>
                              ), we observed frequent loss of imprinting in tumors and report cases with or without loss of heterozygosity, accounting for 74/276 (26.8%) and 102/276 (37.0%) loci, respectively (
                              <strong>
                               <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                                Figure 6A,B
                               </a>
                              </strong>
                              ). Loss of imprinting without loss of heterozygosity showed a preference for both parental alleles losing (IGF2/H19, MIMT1/ZIM2/PEG3, SRNPN) or gaining methylation (GNAS/GNASAS, BLCAP/NNAT,
                              <strong>
                               <a id="xref-fig-6-3" class="xref-fig" href="#F6">
                                Figure 6B
                               </a>
                              </strong>
                              ). Overall, somatic alterations at imprinted loci were frequently ubiquitous among samples from the same patient (39/60, 65%,
                              <strong>
                               <a id="xref-fig-16-1" class="xref-fig" href="#F16">
                                Figure S8A
                               </a>
                              </strong>
                              ). Interestingly, we report 7 cases with distinct subclonal alterations affecting the same locus. We map these epimutations to phylogenies of the same samples published in Jamal-Hanjani et al. (
                              <a id="xref-ref-45-6" class="xref-bibr" href="#ref-45">
                               Jamal-Hanjani et al., 2017
                              </a>
                              ), and find evidence of 4 parallel and 2 sequential evolutionary events (
                              <strong>
                               <a id="xref-fig-16-2" class="xref-fig" href="#F16">
                                Figure S8B
                               </a>
                              </strong>
                              ). We therefore posit that deregulation of imprinted loci may play a role in shaping a subset of non-small cell lung cancers.
                             </p>
                             <p id="p-48">
                              Next, we evaluated allele-specific methylation signals in adjacent normal samples from females. Dosage compensation in females by X-chromosome inactivation involves extensive DNA methylation of promoter-associated CpGs on the inactive copy (
                              <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">
                               Lyon, 1962
                              </a>
                              ). Unexpectedly, intermediate methylation values on X were consistently below 0.5 (median m
                              <sub>
                               t
                              </sub>
                              = 0.384) for the 13 normal female samples in this cohort (
                              <strong>
                               <a id="xref-fig-6-4" class="xref-fig" href="#F6">
                                Figure 6C
                               </a>
                              </strong>
                              ). We observed decreased sequencing coverage on X compared to the rest of the genome in females (
                              <strong>
                               <a id="xref-fig-6-5" class="xref-fig" href="#F6">
                                Figure 6D
                               </a>
                              </strong>
                              ) while BAF of heterozygous SNP loci did not show any deviation from 0.5 (
                              <strong>
                               <a id="xref-fig-6-6" class="xref-fig" href="#F6">
                                Figure 6E
                               </a>
                              </strong>
                              ). As this effect was absent from WES data (data not shown), this suggests that the RRBS library preparation may be biased against the methylated Barr body. Due to the mosaic nature of X chromosome inactivation in females, at the scale of our adjacent normal bulk lung samples, both parental X chromosomes are inactivated in an approximately equal number of cells, and no enrichment of allele-specific methylation on chromosome X was observed compared with autosomes (t-test, p = 6.36 × 10
                              <sup>
                               −2
                              </sup>
                              ).
                             </p>
                             <p id="p-49">
                              Leveraging allele-specific pure tumor methylation rates at CpG loci that were confidently unmethylated in the matched normal (HDI
                              <sub>
                               99
                              </sub>
                              [0, 0.2],
                              <strong>
                               Methods
                              </strong>
                              ), we demonstrate that allele-specific CpG hypermethylation contributes to intermediate m
                              <sub>
                               t
                              </sub>
                              values and confirm the presence of bi-allelic alterations (
                              <strong>
                               <a id="xref-fig-6-7" class="xref-fig" href="#F6">
                                Figure 6F,G
                               </a>
                              </strong>
                              ). In addition, direct comparison of average and allele-specific methylation rates at the same CpGs between sample pairs allows visualization of (sub)clonal allele-specific methylation clusters (
                              <strong>
                               <a id="xref-fig-6-8" class="xref-fig" href="#F6">
                                Figures 6H
                               </a>
                              </strong>
                              ). These clusters recapitulate the previously reported SNV-derived phylogeny of the same tumor, but also enable detection of further subclones (
                              <strong>
                               <a id="xref-fig-6-9" class="xref-fig" href="#F6">
                                Figure 6I,J
                               </a>
                              </strong>
                              ). Through uniform manifold approximation and projection (UMAP) analysis of allele-specific and average pure tumor methylation rates across samples from the same patient, followed by clustering, we classified epimutations as (sub)clonal bi-allelic and mono-allelic (
                              <strong>
                               Methods
                              </strong>
                              ,
                              <strong>
                               <a id="xref-fig-6-10" class="xref-fig" href="#F6">
                                Figure 6K
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-50">
                              Interestingly, while varying widely for individual tumors, we observed that the extent of allele-specific vs. bi-allelic and subclonal vs. clonal methylation changes differed between LUSC and LUAD and stage I vs. stage II/III tumors (
                              <strong>
                               <a id="xref-fig-6-11" class="xref-fig" href="#F6">
                                Figure 6L,M
                               </a>
                              </strong>
                              ). This suggests distinct roles and mechanisms underlying DNA methylation heterogeneity and evolution in these tumor types. The fraction of clonal bi-allelic DMPs was the strongest indicator of relapse and could present a unique opportunity for early patient stratification.
                             </p>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              Interplay of somatic mutations and methylation changes
                             </h3>
                             <p id="p-51">
                              To gain insight into the relationship between somatic mutations and epimutations, we next performed DMR calling on the SNV-purified tumor methylation rate estimates. In total, 4,406 methylation bins comprised of 3 or more CpGs could be phased to SNVs, of which 610 were differentially methylated (
                              <strong>
                               <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                                Figure 7A
                               </a>
                              </strong>
                              ,
                              <strong>
                               Table S4
                              </strong>
                              ). These SNV-phased DMRs were more frequently hyper-than hypomethylated, potentially resulting from the disruption of transcription factor binding sites and thus decreased enhancer binding affinity (
                              <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">
                               Morova et al., 2020
                              </a>
                              ;
                              <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">
                               Zhao et al., 2017
                              </a>
                              ).
                             </p>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F7.large.jpg?width=800&height=600&carousel=1" title="The interplay of somatic mutations and methylation changes. (A) Overview of phasing results of 4,406 methylation bins to somatic SNVs. DMRs are annotated as either in-trans or in-cis to the SNV, on both alleles, or showing loss of the wild type (WT) allele. Each DMR category is further sub-divided into hyper- and hypomethylated subgroups. (B-D) Examples of SNV-phased, CAMDAC deconvolved tumor and matched normal methylation rates across genes harboring SNVs and DMRs in-cis (B), in-trans (C), and with loss of the WT allele (D)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">The interplay of somatic mutations and methylation changes.</span> (A) Overview of phasing results of 4,406 methylation bins to somatic SNVs. DMRs are annotated as either in-<em>trans</em> or in-<em>cis</em> to the SNV, on both alleles, or showing loss of the wild type (WT) allele. Each DMR category is further sub-divided into hyper- and hypomethylated subgroups. (B-D) Examples of SNV-phased, CAMDAC deconvolved tumor and matched normal methylation rates across genes harboring SNVs and DMRs in-<em>cis</em> (B), in-<em>trans</em> (C), and with loss of the WT allele (D).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F7.medium.gif" width="440" height="307"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F7.medium.gif" width="440" height="307"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 7.
                               </span>
                               <span class="caption-title">
                                The interplay of somatic mutations and methylation changes.
                               </span>
                               <p id="p-52" class="first-child">
                                (A) Overview of phasing results of 4,406 methylation bins to somatic SNVs. DMRs are annotated as either in-trans or in-cis to the SNV, on both alleles, or showing loss of the wild type (WT) allele. Each DMR category is further sub-divided into hyper- and hypomethylated subgroups. (B-D) Examples of SNV-phased, CAMDAC deconvolved tumor and matched normal methylation rates across genes harboring SNVs and DMRs in-cis (B), in-trans (C), and with loss of the WT allele (D).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-53">
                              Supporting the functional consequences of mutations and/or increased mutability of methylated cytosines, 114 vs. 36 hypermethylated SNV-phased DMRs without loss of the wild type allele were specific to the mutant (in-cis) and the wild-type (in-trans) allele, respectively (p-val = 1.22 × 10
                              <sup>
                               −10
                              </sup>
                              , two-sided Binomial test). For example, patient CRUK0082 harbors a clonal CDKN2A promoter mutation, where, in regions with phased methylation information, only the mutant allele is hypermethylated (
                              <strong>
                               <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                                Figure 7B
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-54">
                              In-trans hypermethylation events may occasionally constitute inactivation of genes due to deleterious mutation of one allele and hypermethylation of the other. In one potential example, enhancer GH06F037648, which is intragenic to MDGA1, was found to be clonally hypermethylated on one copy of the wild type allele in all regions from patient CRUK0071 (
                              <strong>
                               <a id="xref-fig-7-3" class="xref-fig" href="#F7">
                                Figure 7C
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-55">
                              In regions where the wild type allele is lost, it is not possible to determine whether in-cis DMRs were originally restricted to the mutant allele and thus potentially linked to the genetic alterations, or on all copies, suggesting the two events were independent. As an example of this, we observe clonal loss of the wild type allele, combined with a missense mutation and hypomethylation of surrounding CpGs at the CDH10 locus in patient CRUK0072 (
                              <strong>
                               <a id="xref-fig-7-4" class="xref-fig" href="#F7">
                                Figure 7D
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-56">
                              Taken together, CAMDAC deconvolution and phasing enables deeper understanding of the interplay between aberrant DNA methylation and genetic mutations. In our non-small cell lung cancer cohort, we observed frequent phasing of the hypermethylated allele to the mutant SNV allele, potentially through ablation of adjacent transcription factor binding sites.
                             </p>
                            </div>
                            <div id="sec-10" class="subsection">
                             <h3>
                              Relationships between purified methylation profiles
                             </h3>
                             <p id="p-57">
                              To gain insight into similarities and differences between samples, we clustered genome-wide methylation profiles. We focused on tumor-normal promoter DMRs, selected for being enriched in gene regulatory driven differential methylation, and performed UMAP of deconvolved tumor and normal methylation rate profiles (
                              <strong>
                               <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                                Figures 8A,B
                               </a>
                              </strong>
                              ). We identified four main clusters: two clusters of cancer samples and two clusters of adjacent normal samples. Normal lung epithelium samples clustered by sex (
                              <strong>
                               <a id="xref-fig-8-2" class="xref-fig" href="#F8">
                                Figure 8A
                               </a>
                              </strong>
                              ), indicating that differences are dominated by female-to-male differential methylation. These sex-based differences were less dominant in the tumor samples, likely in part because many female samples showed complete (34%) or at least partial (52%) LOH of chromosome X. In contrast, non-small cell lung cancer samples mostly separated by histology (
                              <strong>
                               <a id="xref-fig-8-3" class="xref-fig" href="#F8">
                                Figure 8B
                               </a>
                              </strong>
                              ), pointing towards large methylation differences between adenocarcinoma and squamous cell carcinoma, possibly reflecting a different cell of origin (
                              <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">
                               Sutherland and Berns, 2010
                              </a>
                              ). Different tumor samples from the same patient consistently clustered together (
                              <strong>
                               <a id="xref-fig-8-4" class="xref-fig" href="#F8">
                                Figure 8C,D
                               </a>
                              </strong>
                              ).
                             </p>
                             <div id="F8" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F8.large.jpg?width=800&height=600&carousel=1" title="CAMDAC deconvolved tumor methylation rates inform clustering and intra-tumor heterogeneity analyses. (A-B) UMAP of the mean purified tumor methylation rates at CpGs in tumor-normal promoter DMRs, (A-B) annotated with sex (A) and sample histology (B). (C-D) Zoom in of UMAP clusters 3 (C) and 4 (D), annotated with patient IDs. (E) Correlations between bulk (upper triangle) and purified methylation rates (lower triangle) and matched normal (upmost and right-most columns) using all tumor-normal DMPs in one or more of the seven tumor samples from CRUK0062. Samples are ordered by tumor purity. (F) Neighbor-joining phylogenetic sample tree for CRUK0062 based on binarized bulk tumor and CAMDAC deconvolved tumor methylation estimates at tumor-normal DMPs or using binarized CCF at SNVs. (G) SNV-based phylogenetic clone tree for CRUK0062 from Jamal-Hanjani et al. (Jamal-Hanjani et al., 2017). (H) Empirical p-value obtained when fitting (hyper)methylation data to the matched SNV tree for each patient. Patients are grouped by the number of sampled tumor regions with RRBS data (excluding the adjacent normal). (I) Difference in the trunk to longest branch length ratio between DMP and matched SNV sample trees. Left: CAMDAC pure tumor DMP vs. SNV sample trees; Right: bulk tumor DMP vs. SNV sample trees." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">CAMDAC deconvolved tumor methylation rates inform clustering and intra-tumor heterogeneity analyses.</span> (A-B) UMAP of the mean purified tumor methylation rates at CpGs in tumor-normal promoter DMRs, (A-B) annotated with sex (A) and sample histology (B). (C-D) Zoom in of UMAP clusters 3 (C) and 4 (D), annotated with patient IDs. (E) Correlations between bulk (upper triangle) and purified methylation rates (lower triangle) and matched normal (upmost and right-most columns) using all tumor-normal DMPs in one or more of the seven tumor samples from CRUK0062. Samples are ordered by tumor purity. (F) Neighbor-joining phylogenetic sample tree for CRUK0062 based on binarized bulk tumor and CAMDAC deconvolved tumor methylation estimates at tumor-normal DMPs or using binarized CCF at SNVs. (G) SNV-based phylogenetic clone tree for CRUK0062 from Jamal-Hanjani et <em>al</em>. (Jamal-Hanjani et al., 2017). (H) Empirical p-value obtained when fitting (hyper)methylation data to the matched SNV tree for each patient. Patients are grouped by the number of sampled tumor regions with RRBS data (excluding the adjacent normal). (I) Difference in the trunk to longest branch length ratio between DMP and matched SNV sample trees. Left: CAMDAC pure tumor DMP <em>vs.</em> SNV sample trees; Right: bulk tumor DMP <em>vs.</em> SNV sample trees.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F8.medium.gif" width="440" height="393"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 8." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F8.medium.gif" width="440" height="393"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 8.
                               </span>
                               <span class="caption-title">
                                CAMDAC deconvolved tumor methylation rates inform clustering and intra-tumor heterogeneity analyses.
                               </span>
                               <p id="p-58" class="first-child">
                                (A-B) UMAP of the mean purified tumor methylation rates at CpGs in tumor-normal promoter DMRs, (A-B) annotated with sex (A) and sample histology (B). (C-D) Zoom in of UMAP clusters 3 (C) and 4 (D), annotated with patient IDs. (E) Correlations between bulk (upper triangle) and purified methylation rates (lower triangle) and matched normal (upmost and right-most columns) using all tumor-normal DMPs in one or more of the seven tumor samples from CRUK0062. Samples are ordered by tumor purity. (F) Neighbor-joining phylogenetic sample tree for CRUK0062 based on binarized bulk tumor and CAMDAC deconvolved tumor methylation estimates at tumor-normal DMPs or using binarized CCF at SNVs. (G) SNV-based phylogenetic clone tree for CRUK0062 from Jamal-Hanjani et al. (
                                <a id="xref-ref-45-7" class="xref-bibr" href="#ref-45">
                                 Jamal-Hanjani et al., 2017
                                </a>
                                ). (H) Empirical p-value obtained when fitting (hyper)methylation data to the matched SNV tree for each patient. Patients are grouped by the number of sampled tumor regions with RRBS data (excluding the adjacent normal). (I) Difference in the trunk to longest branch length ratio between DMP and matched SNV sample trees. Left: CAMDAC pure tumor DMP vs. SNV sample trees; Right: bulk tumor DMP vs. SNV sample trees.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-11" class="subsection">
                             <h3>
                              Purified methylation profiles reveal phylogenetic relationships
                             </h3>
                             <p id="p-59">
                              We next focused more closely on the relationship between methylation profiles of different samples from the same patient. For each patient, we select CpG sites that are differentially methylated compared to the normal in at least one sample. After deconvolution, tumor signals show reduced correlation with adjacent normal methylation rates, and tumor purity is no longer the main driver of inter-sample correlations (
                              <strong>
                               <a id="xref-fig-8-5" class="xref-fig" href="#F8">
                                Figure 8E
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-60">
                              Instead, CAMDAC pure tumor methylation profiles may encode evolutionary relationships between subclones. Indeed, sample phylogenetic trees constructed from methylation rates at (hypermethylated) tumor-normal DMPs recapitulate clonal phylogenetic relationships inferred from matched WES (
                              <a id="xref-ref-45-8" class="xref-bibr" href="#ref-45">
                               Jamal-Hanjani et al., 2017
                              </a>
                              ,
                              <strong>
                               Methods
                              </strong>
                              ,
                              <strong>
                               <a id="xref-fig-8-6" class="xref-fig" href="#F8">
                                Figures 8F,G
                               </a>
                               and
                               <a id="xref-fig-17-1" class="xref-fig" href="#F17">
                                S9
                               </a>
                              </strong>
                              ). In contrast, bulk methylation profiles were unable to reproduce these relationships, and generally clustered by tumor purity. Overall, methylation trees based on CAMDAC m
                              <sub>
                               t
                              </sub>
                              outperformed m
                              <sub>
                               b
                              </sub>
                              in reproducing phylogenetic relationships (combined empirical p-values = 1.61 × 10
                              <sup>
                               −5
                              </sup>
                              and 0.109, respectively, Fisher’s method,
                              <strong>
                               <a id="xref-fig-8-7" class="xref-fig" href="#F8">
                                Figures 8H
                               </a>
                               and
                               <a id="xref-fig-18-1" class="xref-fig" href="#F18">
                                S10
                               </a>
                              </strong>
                              ,
                              <strong>
                               Methods
                              </strong>
                              ).
                             </p>
                             <p id="p-61">
                              Epigenetic alterations typically outnumber somatic mutations, and may accumulate early in tumor evolution (
                              <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">
                               Baylin and Jones, 2016
                              </a>
                              ), creating a permissive state for tumor initiation. As such, we expected longer trunk lengths (relative to subclonal branches) in DMP-than SNV-based patient trees. We calculated the relative trunk length difference between DMP and SNV tree pairs and saw longer trunks in CAMDAC m
                              <sub>
                               t
                              </sub>
                              -derived DMP trees than in SNV trees, but only in patients with low SBS4 smoking signature exposure (Pearson correlation = −0.64, p-value = 6.72 × 10
                              <sup>
                               −5
                              </sup>
                              ,
                              <strong>
                               Methods
                              </strong>
                              ,
                              <strong>
                               <a id="xref-fig-8-8" class="xref-fig" href="#F8">
                                Figure 8I
                               </a>
                              </strong>
                              ). This trend was lost in the bulk tumor data, where virtually all DMP trees had shorter trunks compared to SNV trees due to signal from normal contaminating cells.
                             </p>
                             <p id="p-62">
                              We therefore conclude that CAMDAC provides unique opportunities to study intra-tumor heterogeneity in solid tumors, unconfounded by signals from admixed normal cells.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-12">
                            <h2 class="">
                             DISCUSSION
                            </h2>
                            <p id="p-63">
                             Bulk tumor methylation signals are confounded by copy number aberrations and normal cell admixture. CAMDAC determines tumor purity and allele-specific copy number from bulk tumor-normal matched RRBS data and uses this to reconstruct the true tumor (allele-specific) methylation rate from the bulk tumor and matched normal rates. Use of these purified methylomes increases the accuracy of tumor-normal and tumor-tumor differential methylation calling and can reveal the phylogenetic relationships between subclones. In addition, they provide insight into tumor biology and the taxonomy of cancer.
                            </p>
                            <p id="p-64">
                             While a few studies have considered either the effect of sample purity (
                             <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">
                              Hua et al., 2020
                             </a>
                             ;
                             <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">
                              Zheng et al., 2017
                             </a>
                             ) or copy number (
                             <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">
                              Martin-Trujillo et al., 2017
                             </a>
                             ) on bulk tumor methylation rates, CAMDAC is – to our knowledge – the first approach to estimate and leverage both confounders to reconstruct the tumor methylation signal. We demonstrate that tumor-normal differential methylation produces predictable intermediate methylation values given local copy number states. This explains, at least in part, why bulk cancer methylomes exhibit an increased proportion of intermediately methylated regions compared to normal tissues (
                             <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">
                              Lister et al., 2009
                             </a>
                             ).
                            </p>
                            <p id="p-65">
                             After purification, most differentially methylated CpGs are either fully methylated or unmethylated. Nevertheless, a significant subset remains intermediately methylated. Allele-specific CAMDAC-purified tumor methylation rates allow us to decipher this signal and classify epimutations as (sub)clonal bi- or mono-allelic. From these data, we suggest two potential classes of DNA methylation changes: (i) gene regulatory differential methylation and (ii) allele- (and copy-) specific stochastic methylation changes. Regulatory changes typically affect all copies indiscriminately, with possible exceptions at heterozygous loci, while stochastic methylation changes arise due to errors in the DNA methylation maintenance machinery, aberrant TET activity, or spontaneous deamination of methylated cytosines (
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              Coulondre et al., 1978
                             </a>
                             ;
                             <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                              Goyal, 2006
                             </a>
                             ;
                             <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                              Kohli and Zhang, 2013
                             </a>
                             ). Assuming faithful replication thereafter, these changes become discernable after clonal expansion, generating intermediate methylation signals.
                            </p>
                            <p id="p-66">
                             CAMDAC methylation profiles recapitulate the phylogenetic relationships between subclones and unlock a wealth of information on intra-tumor heterogeneity. CAMDAC also recovers the clonal DMP signals that are lost in bulk methylation data due to normal cell contamination. Additionally, CAMDAC DMP trees of both LUSC and LUAD cases had longer relative trunk lengths than SNV trees in the absence of highly mutagenic processes such as smoking, which suggests that DNA methylation may have a role in early tumor evolution. We envisage further development will enable reconstruction of subclones within single samples and accurate clone trees (as opposed to sample trees,
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              Alves et al., 2017
                             </a>
                             ) from bisulfite sequencing data, akin to current methods leveraging single-nucleotide variants (
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              Dentro et al., 2017
                             </a>
                             ;
                             <a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">
                              Nik-Zainal et al., 2012
                             </a>
                             ;
                             <a id="xref-ref-88-2" class="xref-bibr" href="#ref-88">
                              Tarabichi et al., 2021
                             </a>
                             ). As such, application of CAMDAC will help elucidate the role of epimutations in the evolutionary histories of different cancer types.
                            </p>
                            <p id="p-67">
                             As in previous studies (
                             <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                              Gaiti et al., 2019
                             </a>
                             ;
                             <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">
                              Hua et al., 2020
                             </a>
                             ), we infer differential methylation compared to adjacent normal tissue and use this as a proxy for the admixed normal cells in the tumor sample. Where matched normal tissue samples are not available, a tissue-matched normal reference may be constructed for use with CAMDAC. Outside our cohort, the DNA methylation data flowing from large-scale initiatives such as the International Human Epigenome Consortium (IHEC,
                             <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">
                              Stunnenberg et al., 2016
                             </a>
                             ) and the ENCODE (
                             <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                              Feingold et al., 2004
                             </a>
                             ) and BLUEPRINT (
                             <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                              Fernández et al., 2016
                             </a>
                             ) projects, enable construction of reference profiles for a wide range of tissues.
                            </p>
                            <p id="p-68">
                             In low purity samples and in regions with lower tumor copy number, fewer reads report on the methylation state of the tumor. CAMDAC incorporates this greater uncertainty in its estimates for the tumor methylation rates in such cases, greatly reducing the number of false positive hits when inferring differential methylation. Bulk tumor methylomes of low purity samples are dominated by signals from the admixed normal cells, as evidenced by the high false negative rate at simulated tumor-normal epimutations based on bulk profiles. In tumor-tumor differential methylation analyses based on bulk tumor methylomes, more DMPs are identified when two samples differ more in purity, suggesting a high false positive rate. In contrast, when using CAMDAC pure tumor methylation rate increases, the number of DMPs increases with statistical power.
                            </p>
                            <p id="p-69">
                             DNA methylation heterogeneity has been linked to clonal progression in several tumor types (
                             <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">
                              Hao et al., 2016
                             </a>
                             ;
                             <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">
                              Mazor et al., 2015
                             </a>
                             ). Various metrics have been conceived to quantify intra-tumor methylation heterogeneity from bulk data and some have shown prognostic value in leukemia and Ewing sarcoma (
                             <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">
                              Landau et al., 2014
                             </a>
                             ;
                             <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">
                              Li et al., 2016
                             </a>
                             ;
                             <a id="xref-ref-82-4" class="xref-bibr" href="#ref-82">
                              Sheffield et al., 2017
                             </a>
                             ), but not in glioblastoma (
                             <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">
                              Klughammer et al., 2018
                             </a>
                             ). While heterogeneity estimates from bulk tumor methylomes are likely accurate in tumor types for which high purity samples are readily obtained, we hypothesize these metrics are similarly confounded by normal cell admixture and copy number aberrations in most other cases. Applying CAMDAC principles to adjust these metrics accordingly may yield powerful prognosticators and shed further light on the role of DNA methylation heterogeneity in solid tumors.
                            </p>
                            <p id="p-70">
                             In summary, CAMDAC enables us to mine a wealth of information contained in bulk tumor bisulfite sequencing data, combining purity and copy number profiling with accurate differential methylation analysis based on the purified tumor methylation rate estimates. CAMDAC uniquely unlocks insights into intra-tumor heterogeneity and the biology and taxonomy of cancer. We expect CAMDAC will further our understanding of the interplay between genetic and epigenetic mutations and their roles in tumor evolution.
                            </p>
                           </div>
                           <div class="section" id="sec-14">
                            <h2 class="">
                             AUTHOR CONTRIBUTIONS
                            </h2>
                            <p id="p-72">
                             E.L.C. developed ASCAT.m and CAMDAC, under the supervision of J.D. and P.V.L, with important inputs from S.B.; J.D. conceptualized CAMDAC. C.S. and S.B. conceptualized the TRACERx methylation study and acquired financial support, with input from G.A.W., A.F., and M.T.; M.T., G.A.W., H.V., A.F. and P.D. set up and processed the bulk RRBS. A.V. and C.C.M. performed FACS analysis followed by RRBS, with input from N.K. and assistance from the cell sorting and sequencing facilities at the Francis Crick Institute. E.L.C. performed analyses with contributions from N.E.M. and C.C., and input from T.B.; E.L.C. visualized data and designed figures, with additions from N.E.M. and C.C.; T.B.K.M. and M.D. performed WGS analyses under the supervision of N.M. and C.S.; S.V., M.J.H. and C.S. were critical in the funding and acquisition of clinical samples.; J.D. and P.V.L. supervised the study. E.L.C., J.D. and P.V.L. wrote the manuscript, with significant contributions from N.E.M. and C.C., and further input from S.B., M.T. and C.S. All authors read and approved the final manuscript.
                            </p>
                           </div>
                           <div class="section" id="sec-15">
                            <h2 class="">
                             DECLARATION OF INTERESTS
                            </h2>
                            <p id="p-73">
                             E.L.C. and G.A.W. are currently working for and hold shares of Achilles Therapeutics. C.S. acknowledges grant support from Pfizer, AstraZeneca, Bristol Myers Squibb, Roche-Ventana, Boehringer-Ingelheim, Archer Dx Inc (collaboration in minimal residual disease sequencing technologies) and Ono Pharmaceutical, is an AstraZeneca Advisory Board member and Chief Investigator for the MeRmaiD1 clinical trial, has consulted for Pfizer, Novartis, GlaxoSmithKline, MSD, Bristol Myers Squibb, Celgene, AstraZeneca, Illumina, Genentech, Roche-Ventana, GRAIL, Medicxi, Bicycle Therapeutics, and the Sarah Cannon Research Institute, has stock options in Apogen Biotechnologies, Epic Bioscience, GRAIL, and is co-founder and has shares of Achilles Therapeutics. M.J.-H. has consulted, and is a member of the Scientific Advisory Board and Steering Committee, for Achilles Therapeutics, has received speaker honoraria from Astex Pharmaceuticals, Oslo Cancer Cluster, and holds a patent PCT/US2017/028013 relating to methods for lung cancer detection. All other authors declare no competing interests.
                            </p>
                           </div>
                           <div class="section" id="sec-16">
                            <h2 class="">
                             SUPPLEMENTARY FIGURES
                            </h2>
                            <div id="F9" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F9.large.jpg?width=800&height=600&carousel=1" title="Tumor purity and copy number affect methylation rates. (A) Normal and bulk tumor methylation rate density for sample CRUK0062-R3 and adjacent patient-matched normal showing most CpGs are either fully methylated or unmethylated. An increased number of CpGs with intermediate methylation levels is found in the bulk tumor. (B) Simulated 99% Highest Density Interval (HDI99) at example unmethylated CpGs with variable number of methylated and unmethylated read counts and total CpG coverage. (C-F) Normal and bulk tumor methylation rate (mt and mb, respectively) density plot for CpGs labelled as confidently unmethylated in the normal (HDI99 ⊆ [0,0.2]). The majority of these loci are also unmethylated in the bulk tumor. (D-F) Zoom-in, highlighting CpGs with gained methylation in the tumor cells across all copy numbers (D), and in regions of copy number states 2+0 (E) and 2+2 (F). (G) Bulk tumor methylation rate histograms of CRUK0084-R3, having an estimated tumor purity (ρ) of 87%, for CpGs in diploid regions confidently unmethylated (left) and methylated (right) in the patient-matched adjacent normal. Hyper- and hypomethylated CpGs can be found in peaks that vary with the mutation copy number. (H) Bulk methylation rate histograms of CRUK0062 tumor regions 1-3, for CpGs which are confidently methylated in the adjacent normal sample, stratified by allele-specific copy number state. A dashed line indicates the expected mode of the methylation rate peak corresponding to clonal differentially methylated CpGs on all copies." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Tumor purity and copy number affect methylation rates.</span> (A) Normal and bulk tumor methylation rate density for sample CRUK0062-R3 and adjacent patient-matched normal showing most CpGs are either fully methylated or unmethylated. An increased number of CpGs with intermediate methylation levels is found in the bulk tumor. (B) Simulated 99% Highest Density Interval (HDI<sub>99</sub>) at example unmethylated CpGs with variable number of methylated and unmethylated read counts and total CpG coverage. (C-F) Normal and bulk tumor methylation rate (<em>m<sub>t</sub></em> and <em>m<sub>b</sub></em>, respectively) density plot for CpGs labelled as confidently unmethylated in the normal (HDI<sub>99</sub> ⊆ [0,0.2]). The majority of these loci are also unmethylated in the bulk tumor. (D-F) Zoom-in, highlighting CpGs with gained methylation in the tumor cells across all copy numbers (D), and in regions of copy number states 2+0 (E) and 2+2 (F). (G) Bulk tumor methylation rate histograms of CRUK0084-R3, having an estimated tumor purity (<em>ρ</em>) of 87%, for CpGs in diploid regions confidently unmethylated (left) and methylated (right) in the patient-matched adjacent normal. Hyper- and hypomethylated CpGs can be found in peaks that vary with the mutation copy number. (H) Bulk methylation rate histograms of CRUK0062 tumor regions 1-3, for CpGs which are confidently methylated in the adjacent normal sample, stratified by allele-specific copy number state. A dashed line indicates the expected mode of the methylation rate peak corresponding to clonal differentially methylated CpGs on all copies.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F9.medium.gif" width="304" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S1." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F9.medium.gif" width="304" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S1.
                              </span>
                              <span class="caption-title">
                               Tumor purity and copy number affect methylation rates.
                              </span>
                              <p id="p-74" class="first-child">
                               (A) Normal and bulk tumor methylation rate density for sample CRUK0062-R3 and adjacent patient-matched normal showing most CpGs are either fully methylated or unmethylated. An increased number of CpGs with intermediate methylation levels is found in the bulk tumor. (B) Simulated 99% Highest Density Interval (HDI
                               <sub>
                                99
                               </sub>
                               ) at example unmethylated CpGs with variable number of methylated and unmethylated read counts and total CpG coverage. (C-F) Normal and bulk tumor methylation rate (m
                               <sub>
                                t
                               </sub>
                               and m
                               <sub>
                                b
                               </sub>
                               , respectively) density plot for CpGs labelled as confidently unmethylated in the normal (HDI
                               <sub>
                                99
                               </sub>
                               ⊆ [0,0.2]). The majority of these loci are also unmethylated in the bulk tumor. (D-F) Zoom-in, highlighting CpGs with gained methylation in the tumor cells across all copy numbers (D), and in regions of copy number states 2+0 (E) and 2+2 (F). (G) Bulk tumor methylation rate histograms of CRUK0084-R3, having an estimated tumor purity (ρ) of 87%, for CpGs in diploid regions confidently unmethylated (left) and methylated (right) in the patient-matched adjacent normal. Hyper- and hypomethylated CpGs can be found in peaks that vary with the mutation copy number. (H) Bulk methylation rate histograms of CRUK0062 tumor regions 1-3, for CpGs which are confidently methylated in the adjacent normal sample, stratified by allele-specific copy number state. A dashed line indicates the expected mode of the methylation rate peak corresponding to clonal differentially methylated CpGs on all copies.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F10" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F10.large.jpg?width=800&height=600&carousel=1" title="RRBS-derived tumor LogR biases and correction. (A-C) MspI fragment length (A), GC content (B) and replication timing (C) affect tumor LogR. (D) A linear combination of three natural splines, modelling the effect of each of the three biases described in (A-C) is used to correct the raw LogR." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">RRBS-derived tumor LogR biases and correction.</span> (A-C) <em>MspI</em> fragment length (A), GC content (B) and replication timing (C) affect tumor LogR. (D) A linear combination of three natural splines, modelling the effect of each of the three biases described in (A-C) is used to correct the raw LogR.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F10.medium.gif" width="440" height="275"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S2." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F10.medium.gif" width="440" height="275"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S2.
                              </span>
                              <span class="caption-title">
                               RRBS-derived tumor LogR biases and correction.
                              </span>
                              <p id="p-75" class="first-child">
                               (A-C) MspI fragment length (A), GC content (B) and replication timing (C) affect tumor LogR. (D) A linear combination of three natural splines, modelling the effect of each of the three biases described in (A-C) is used to correct the raw LogR.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F11" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F11.large.jpg?width=800&height=600&carousel=1" title="CAMDAC genotyping validation and evaluation of sources of bias affecting heterozygous SNP calling from RRBS data. (A-B) False positive (A) and false negative (B) heterozygous call numbers (scaled radii) and rates (%) stratified by SNP type and sequence context (top) and histograms showing the distribution of CAMDAC BAF estimate errors and noise (bottom). SNPs are considered heterozygous when 0.1 ≤ BAF ≤ 0.9. (C) Performance of heterozygous SNP identification from normal tissue RRBS data at different sequencing coverages and BAF heterozygosity boundaries. (D) Fractions of true positive and false negative heterozygous SNP calls stratified by their location (i) at/or within one read length of a heterozygous CCGG, (ii) covered by only short reads (< 67bp) and (iii) by reads with low reference read mapping quality. χ2 tests show enrichment of false positives at each of these three types of regions. The data shown is from the adjacent normal sample of CRUK0062." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">CAMDAC genotyping validation and evaluation of sources of bias affecting heterozygous SNP calling from RRBS data.</span> (A-B) False positive (A) and false negative (B) heterozygous call numbers (scaled radii) and rates (%) stratified by SNP type and sequence context (top) and histograms showing the distribution of CAMDAC BAF estimate errors and noise (bottom). SNPs are considered heterozygous when 0.1 ≤ BAF ≤ 0.9. (C) Performance of heterozygous SNP identification from normal tissue RRBS data at different sequencing coverages and BAF heterozygosity boundaries. (D) Fractions of true positive and false negative heterozygous SNP calls stratified by their location (i) at/or within one read length of a heterozygous CCGG, (ii) covered by only short reads (< 67bp) and (iii) by reads with low reference read mapping quality. <strong><em>χ</em><sup>2</sup></strong> tests show enrichment of false positives at each of these three types of regions. The data shown is from the adjacent normal sample of CRUK0062.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F11.medium.gif" width="440" height="382"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S3." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F11.medium.gif" width="440" height="382"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S3." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S3.
                              </span>
                              <span class="caption-title">
                               CAMDAC genotyping validation and evaluation of sources of bias affecting heterozygous SNP calling from RRBS data.
                              </span>
                              <p id="p-76" class="first-child">
                               (A-B) False positive (A) and false negative (B) heterozygous call numbers (scaled radii) and rates (%) stratified by SNP type and sequence context (top) and histograms showing the distribution of CAMDAC BAF estimate errors and noise (bottom). SNPs are considered heterozygous when 0.1 ≤ BAF ≤ 0.9. (C) Performance of heterozygous SNP identification from normal tissue RRBS data at different sequencing coverages and BAF heterozygosity boundaries. (D) Fractions of true positive and false negative heterozygous SNP calls stratified by their location (i) at/or within one read length of a heterozygous CCGG, (ii) covered by only short reads (< 67bp) and (iii) by reads with low reference read mapping quality.
                               <strong>
                                χ
                                <sup>
                                 2
                                </sup>
                               </strong>
                               tests show enrichment of false positives at each of these three types of regions. The data shown is from the adjacent normal sample of CRUK0062.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F12" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F12.large.jpg?width=800&height=600&carousel=1" title="Comparison of CAMDAC and naïve bulk CpG methylation rate. (A-B) Difference between CAMDAC and naïve bulk methylation rate estimates at 50,000 random (A) heterozygous CpGs and (B) non-polymorphic CpGs in a normal sample. (C) Distribution of SNP types at CpGs. (D-E) Difference between CAMDAC and naïve bulk methylation rate values at (D) homozygous vs. (E) heterozygous CpG, stratified by type as in (C). All data was taken from the adjacent normal sample of patient CRUK0069, which constitutes a representative example of normal lung methylation in this cohort." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Comparison of CAMDAC and naïve bulk CpG methylation rate.</span> (A-B) Difference between CAMDAC and naïve bulk methylation rate estimates at 50,000 random (A) heterozygous CpGs and (B) non-polymorphic CpGs in a normal sample. (C) Distribution of SNP types at CpGs. (D-E) Difference between CAMDAC and naïve bulk methylation rate values at (D) homozygous <em>vs.</em> (E) heterozygous CpG, stratified by type as in (C). All data was taken from the adjacent normal sample of patient CRUK0069, which constitutes a representative example of normal lung methylation in this cohort.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F12.medium.gif" width="440" height="321"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S4." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F12.medium.gif" width="440" height="321"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S4.
                              </span>
                              <span class="caption-title">
                               Comparison of CAMDAC and naïve bulk CpG methylation rate.
                              </span>
                              <p id="p-77" class="first-child">
                               (A-B) Difference between CAMDAC and naïve bulk methylation rate estimates at 50,000 random (A) heterozygous CpGs and (B) non-polymorphic CpGs in a normal sample. (C) Distribution of SNP types at CpGs. (D-E) Difference between CAMDAC and naïve bulk methylation rate values at (D) homozygous vs. (E) heterozygous CpG, stratified by type as in (C). All data was taken from the adjacent normal sample of patient CRUK0069, which constitutes a representative example of normal lung methylation in this cohort.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F13" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F13.large.jpg?width=800&height=600&carousel=1" title="Validation of CAMDAC equations as a function of purity and copy number. (A) Comparison of the predicted methylation rate under the CAMDAC equations as a function of tumor purity and copy number state, and the observed peak position for clonal bi-allelic tumor-normal hypermethylated CpGs (Methods). (B) As in A, but for unmethylated CpGs that are methylated in the adjacent normal sample. (C) Observed minus expected CAMDAC pure tumor methylation rate at clonal hypo- (blue) and hypermethylated (red) DMPs across samples (purity) and tumor copy numbers. (D-E) Observed minus expected CAMDAC pure tumor methylation rates at clonal hyper- (D) and hypomethylated (E) CpGs across copy number states and tumor samples." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Validation of CAMDAC equations as a function of purity and copy number.</span> (A) Comparison of the predicted methylation rate under the CAMDAC equations as a function of tumor purity and copy number state, and the observed peak position for clonal bi-allelic tumor-normal hypermethylated CpGs (<strong>Methods</strong>). (B) As in A, but for unmethylated CpGs that are methylated in the adjacent normal sample. (C) Observed minus expected CAMDAC pure tumor methylation rate at clonal hypo- (blue) and hypermethylated (red) DMPs across samples (purity) and tumor copy numbers. (D-E) Observed minus expected CAMDAC pure tumor methylation rates at clonal hyper- (D) and hypomethylated (E) CpGs across copy number states and tumor samples.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F13.medium.gif" width="440" height="357"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S5." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F13.medium.gif" width="440" height="357"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S5." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S5.
                              </span>
                              <span class="caption-title">
                               Validation of CAMDAC equations as a function of purity and copy number.
                              </span>
                              <p id="p-78" class="first-child">
                               (A) Comparison of the predicted methylation rate under the CAMDAC equations as a function of tumor purity and copy number state, and the observed peak position for clonal bi-allelic tumor-normal hypermethylated CpGs (
                               <strong>
                                Methods
                               </strong>
                               ). (B) As in A, but for unmethylated CpGs that are methylated in the adjacent normal sample. (C) Observed minus expected CAMDAC pure tumor methylation rate at clonal hypo- (blue) and hypermethylated (red) DMPs across samples (purity) and tumor copy numbers. (D-E) Observed minus expected CAMDAC pure tumor methylation rates at clonal hyper- (D) and hypomethylated (E) CpGs across copy number states and tumor samples.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F14" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F14.large.jpg?width=800&height=600&carousel=1" title="Comparing FACS-purified and adjacent normal methylomes. (A) CpG-wise comparison of the adjacent (mt) and FACS-purified (mf,n) normal methylation rates with Pearson correlation. (B) Methylation rate difference histogram for loci classified as confidently unmethylated (UM, HDI99 ⊆ [0, 0.2], blue), intermediate (IM, HDI99 ⊆ [0.2, 0.8], turquoise) and fully methylated (FM, HDI99 ⊆ [0.8, 1], yellow) in at least one sample for a given patient-matched normal pair (first column). Zooming in on the CpGs classed as IM (second column). (C) CpG classification based on mt and mf,n." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Comparing FACS-purified and adjacent normal methylomes.</span> (A) CpG-wise comparison of the adjacent (<em>m<sub>t</sub></em>) and FACS-purified (<em>m<sub>f,n</sub></em>) normal methylation rates with Pearson correlation. (B) Methylation rate difference histogram for loci classified as confidently unmethylated (UM, HDI<sup>99</sup> ⊆ [0, 0.2], blue), intermediate (IM, HDI<sup>99</sup> ⊆ [0.2, 0.8], turquoise) and fully methylated (FM, HDI<sup>99</sup> ⊆ [0.8, 1], yellow) in at least one sample for a given patient-matched normal pair (first column). Zooming in on the CpGs classed as IM (second column). (C) CpG classification based on <em>m<sub>t</sub></em> and <em>m<sub>f,n</sub></em>.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F14.medium.gif" width="427" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S6." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F14.medium.gif" width="427" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S6." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S6.
                              </span>
                              <span class="caption-title">
                               Comparing FACS-purified and adjacent normal methylomes.
                              </span>
                              <p id="p-79" class="first-child">
                               (A) CpG-wise comparison of the adjacent (m
                               <sub>
                                t
                               </sub>
                               ) and FACS-purified (m
                               <sub>
                                f,n
                               </sub>
                               ) normal methylation rates with Pearson correlation. (B) Methylation rate difference histogram for loci classified as confidently unmethylated (UM, HDI
                               <sup>
                                99
                               </sup>
                               ⊆ [0, 0.2], blue), intermediate (IM, HDI
                               <sup>
                                99
                               </sup>
                               ⊆ [0.2, 0.8], turquoise) and fully methylated (FM, HDI
                               <sup>
                                99
                               </sup>
                               ⊆ [0.8, 1], yellow) in at least one sample for a given patient-matched normal pair (first column). Zooming in on the CpGs classed as IM (second column). (C) CpG classification based on m
                               <sub>
                                t
                               </sub>
                               and m
                               <sub>
                                f,n
                               </sub>
                               .
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F15" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F15.large.jpg?width=800&height=600&carousel=1" title="DMP simulation framework and results of tumor-normal and tumor-tumor mono-allelic DMP simulations. (A) Tumors are ranked by purity and the top 20 (high purity; ρs ≥ 0.58) and bottom 20 (low purity; ρs ≤ 0.3) cases are considered for simulations. Sample s with purity ρs is randomly chosen from either group, for example, ρs = 0.6. Autosomal CpGs are separated into groups based on total copy number nt,s,i = 1, 2, 3, 4, 5 or ≥6. We then sample the bulk tumor coverage, covb,s,i, and (allele-specific) tumor copy numbers of CpG i with total, major and minor allele copy number, nt,s,i, nA,t,s,i and nB,t,s,i, respectively. The matched normal copy number is nn,s,i = 2. We obtain the pure tumor coverages for alleles A and B, covX,t,s,i, where X is either A or B, by sampling from a binomial with the number of trials set to covb,s,i and the likelihood of obtaining a tumor read equal to the tumor fraction, fX,t,s,i, estimated as a function of copy numbers and tumor purity. The matched normal coverage, covn,s,i, is taken as the difference between the bulk and pure tumor coverages, covn,s,i = covb,s,i − covt,s,i. (B) We take confidently unmethylated and methylated loci and collate their methylation rate values into vectors vunmeth and vmeth, respectively. We sample a normal methylation rate prior, pn,s,i from vunmeth or vmeth. Leveraging the inferred normal coverage (see A), we sample the counts methylated from a binomial, Mn,s,i. The unmethylated reads counts, UMn,s,i, is the difference between the normal coverage and the number of methylated reads, UMn,s,i = covn,s,i − Mn,s,i. We then repeat these steps for tumor allele A and B. (C) Possible combinations of normal and allele-specific tumor methylation priors and associated ground truth tumor-normal DMP calls to be compared with CAMDAC DMP calls. (D-F) Results of tumor-normal DMP simulations. (D) Mean methylation difference across false negatives (FN), false positives (FP), true negatives (TN) and true positives (TP) at mono-allelic differentially methylated and non-differentially methylated CpGs. (E-F) False negative rates as a function of tumor copy number (E) and coverage (F) for the purified (solid line) and bulk tumor (dashed line) for low vs. high tumor purity samples (pale and dark orange lines respectively) at mono- (left panel) and bi-allelic (right panel) epimutations. (G) Results of tumor-tumor DMP simulations. False negative and false positives rates as a function of tumor copy number for low (left panel), low versus high (middle panel) and high purity (right panel) sample pairs at mono-allelic epimutations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">DMP simulation framework and results of tumor-normal and tumor-tumor mono-allelic DMP simulations.</span> (A) Tumors are ranked by purity and the top 20 (high purity; <em>ρ<sub>s</sub></em> ≥ 0.58) and bottom 20 (low purity; <em>ρ<sub>s</sub></em> ≤ 0.3) cases are considered for simulations. Sample <em>s</em> with purity <em>ρ<sub>s</sub></em> is randomly chosen from either group, for example, <em>ρ<sub>s</sub></em> = 0.6. Autosomal CpGs are separated into groups based on total copy number <em>n</em><sub><em>t,s,i</em></sub> = 1, 2, 3, 4, 5 or ≥6. We then sample the bulk tumor coverage, <em>cov</em><sub><em>b,s,i</em></sub>, and (allele-specific) tumor copy numbers of CpG <em>i</em> with total, major and minor allele copy number, <em>n</em><sub><em>t,s,i</em></sub>, <em>n<sub>A,t,s,i</sub></em> and <em>n<sub>B,t,s,i</sub></em>, respectively. The matched normal copy number is <em>n</em><sub><em>n,s,i</em></sub> = 2. We obtain the pure tumor coverages for alleles A and B, <em>cov<sub>X,t,s,i</sub></em>, where X is either A or B, by sampling from a binomial with the number of trials set to <em>cov</em><sub><em>b,s,i</em></sub> and the likelihood of obtaining a tumor read equal to the tumor fraction, <em>f<sub>X,t,s,i</sub></em>, estimated as a function of copy numbers and tumor purity. The matched normal coverage, <em>cov</em><sub><em>n,s,i</em></sub>, is taken as the difference between the bulk and pure tumor coverages, <em>cov</em><sub><em>n,s,i</em></sub> = <em>cov</em><sub><em>b,s,i</em></sub> − <em>cov</em><sub><em>t,s,i</em></sub>. (B) We take confidently unmethylated and methylated loci and collate their methylation rate values into vectors <em>v</em><sub><em>unmeth</em></sub> and <em>v</em><sub><em>meth</em></sub>, respectively. We sample a normal methylation rate prior, <em>p</em><sub><em>n,s,i</em></sub> from <em>v</em><sub><em>unmeth</em></sub> or <em>v<sub>meth</sub></em>. Leveraging the inferred normal coverage (see A), we sample the counts methylated from a binomial, <em>M</em><sub><em>n,s,i</em></sub>. The unmethylated reads counts, <em>UM</em><sub><em>n,s,i</em></sub>, is the difference between the normal coverage and the number of methylated reads, <em>UM</em><sub><em>n,s,i</em></sub> = <em>cov</em><sub><em>n,s,i</em></sub> − <em>M</em><sub><em>n,s,i</em></sub>. We then repeat these steps for tumor allele A and B. (C) Possible combinations of normal and allele-specific tumor methylation priors and associated ground truth tumor-normal DMP calls to be compared with CAMDAC DMP calls. (D-F) Results of tumor<strong>-</strong>normal DMP simulations. (D) Mean methylation difference across false negatives (FN), false positives (FP), true negatives (TN) and true positives (TP) at mono-allelic differentially methylated and non-differentially methylated CpGs. (E-F) False negative rates as a function of tumor copy number (E) and coverage (F) for the purified (solid line) and bulk tumor (dashed line) for low <em>vs.</em> high tumor purity samples (pale and dark orange lines respectively) at mono- (left panel) and bi-allelic (right panel) epimutations. (G) Results of tumor-tumor DMP simulations. False negative and false positives rates as a function of tumor copy number for low (left panel), low versus high (middle panel) and high purity (right panel) sample pairs at mono-allelic epimutations.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F15.medium.gif" width="335" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S7." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F15.medium.gif" width="335" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S7." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S7.
                              </span>
                              <span class="caption-title">
                               DMP simulation framework and results of tumor-normal and tumor-tumor mono-allelic DMP simulations.
                              </span>
                              <p id="p-80" class="first-child">
                               (A) Tumors are ranked by purity and the top 20 (high purity; ρ
                               <sub>
                                s
                               </sub>
                               ≥ 0.58) and bottom 20 (low purity; ρ
                               <sub>
                                s
                               </sub>
                               ≤ 0.3) cases are considered for simulations. Sample s with purity ρ
                               <sub>
                                s
                               </sub>
                               is randomly chosen from either group, for example, ρ
                               <sub>
                                s
                               </sub>
                               = 0.6. Autosomal CpGs are separated into groups based on total copy number n
                               <sub>
                                t,s,i
                               </sub>
                               = 1, 2, 3, 4, 5 or ≥6. We then sample the bulk tumor coverage, cov
                               <sub>
                                b,s,i
                               </sub>
                               , and (allele-specific) tumor copy numbers of CpG i with total, major and minor allele copy number, n
                               <sub>
                                t,s,i
                               </sub>
                               , n
                               <sub>
                                A,t,s,i
                               </sub>
                               and n
                               <sub>
                                B,t,s,i
                               </sub>
                               , respectively. The matched normal copy number is n
                               <sub>
                                n,s,i
                               </sub>
                               = 2. We obtain the pure tumor coverages for alleles A and B, cov
                               <sub>
                                X,t,s,i
                               </sub>
                               , where X is either A or B, by sampling from a binomial with the number of trials set to cov
                               <sub>
                                b,s,i
                               </sub>
                               and the likelihood of obtaining a tumor read equal to the tumor fraction, f
                               <sub>
                                X,t,s,i
                               </sub>
                               , estimated as a function of copy numbers and tumor purity. The matched normal coverage, cov
                               <sub>
                                n,s,i
                               </sub>
                               , is taken as the difference between the bulk and pure tumor coverages, cov
                               <sub>
                                n,s,i
                               </sub>
                               = cov
                               <sub>
                                b,s,i
                               </sub>
                               − cov
                               <sub>
                                t,s,i
                               </sub>
                               . (B) We take confidently unmethylated and methylated loci and collate their methylation rate values into vectors v
                               <sub>
                                unmeth
                               </sub>
                               and v
                               <sub>
                                meth
                               </sub>
                               , respectively. We sample a normal methylation rate prior, p
                               <sub>
                                n,s,i
                               </sub>
                               from v
                               <sub>
                                unmeth
                               </sub>
                               or v
                               <sub>
                                meth
                               </sub>
                               . Leveraging the inferred normal coverage (see A), we sample the counts methylated from a binomial, M
                               <sub>
                                n,s,i
                               </sub>
                               . The unmethylated reads counts, UM
                               <sub>
                                n,s,i
                               </sub>
                               , is the difference between the normal coverage and the number of methylated reads, UM
                               <sub>
                                n,s,i
                               </sub>
                               = cov
                               <sub>
                                n,s,i
                               </sub>
                               − M
                               <sub>
                                n,s,i
                               </sub>
                               . We then repeat these steps for tumor allele A and B. (C) Possible combinations of normal and allele-specific tumor methylation priors and associated ground truth tumor-normal DMP calls to be compared with CAMDAC DMP calls. (D-F) Results of tumor
                               <strong>
                                -
                               </strong>
                               normal DMP simulations. (D) Mean methylation difference across false negatives (FN), false positives (FP), true negatives (TN) and true positives (TP) at mono-allelic differentially methylated and non-differentially methylated CpGs. (E-F) False negative rates as a function of tumor copy number (E) and coverage (F) for the purified (solid line) and bulk tumor (dashed line) for low vs. high tumor purity samples (pale and dark orange lines respectively) at mono- (left panel) and bi-allelic (right panel) epimutations. (G) Results of tumor-tumor DMP simulations. False negative and false positives rates as a function of tumor copy number for low (left panel), low versus high (middle panel) and high purity (right panel) sample pairs at mono-allelic epimutations.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F16" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F16.large.jpg?width=800&height=600&carousel=1" title="Loss of imprinting ubiquity analysis reveals instances of parallel evolution. (A) Intra-tumor heterogeneity overview of somatic changes at known imprinted loci. (B) Mapping of heterogeneous loss of imprinting events to patient phylogenies." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Loss of imprinting ubiquity analysis reveals instances of parallel evolution.</span> (A) Intra-tumor heterogeneity overview of somatic changes at known imprinted loci. (B) Mapping of heterogeneous loss of imprinting events to patient phylogenies.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F16.medium.gif" width="440" height="386"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S8." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F16.medium.gif" width="440" height="386"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S8." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S8.
                              </span>
                              <span class="caption-title">
                               Loss of imprinting ubiquity analysis reveals instances of parallel evolution.
                              </span>
                              <p id="p-81" class="first-child">
                               (A) Intra-tumor heterogeneity overview of somatic changes at known imprinted loci. (B) Mapping of heterogeneous loss of imprinting events to patient phylogenies.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F17" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F17.large.jpg?width=800&height=600&carousel=1" title="DMP- and SNV-derived sample trees. Phylogenetic sample trees created by calculating the pairwise hamming distance between samples using SNVs from WES or (hypermethylated) tumor-normal DMPs from RRBS, followed by neighbor-joining (Methods)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">DMP- and SNV-derived sample trees.</span> Phylogenetic sample trees created by calculating the pairwise hamming distance between samples using SNVs from WES or (hypermethylated) tumor-normal DMPs from RRBS, followed by neighbor-joining (<strong>Methods</strong>).</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F17.medium.gif" width="301" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S9." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F17.medium.gif" width="301" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S9." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S9.
                              </span>
                              <span class="caption-title">
                               DMP- and SNV-derived sample trees.
                              </span>
                              <p id="p-82" class="first-child">
                               Phylogenetic sample trees created by calculating the pairwise hamming distance between samples using SNVs from WES or (hypermethylated) tumor-normal DMPs from RRBS, followed by neighbor-joining (
                               <strong>
                                Methods
                               </strong>
                               ).
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F18" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F18.large.jpg?width=800&height=600&carousel=1" title="Methylation-SNV tree fit scores. (A) Scoring concordance between phylogenies derived from aberrant methylation and somatic sequence alterations. We leveraged binarized (hypermethylated) tumor-normal DMP data and, based on the ancestral states given by either the SNV () or DMP () multi-sample tree topologies, counted the number of state changes per DMP locus from the normal root to all leaves (methylated to unmethylated or vice versa). Counts were combined into a single score by taking the total absolute difference in the number of ancestral DMP state changes based on the SNV (si) and DMP tree topologies (di, ) as a fraction of all events given by the DMP tree (). Here, a score of 0 indicates a perfect topology match, with the score increasing with the number of inconsistent per-DMP evolutionary trajectories between the two trees. Hence, the score indicates how closely the methylation events support sample relationships inferred from somatic sequence alterations. (B) A schematic of how ancestral state changes are calculated given the SNV tree by counting edges that switch from methylated to unmethylated (bold). Internal node states are inferred using ancestral reconstruction and, for all calculations, nodes with intermediate methylation are replaced with the state of the first ancestor without intermediate methylation. (C) Methylation-SNV fit scores calculated for DMPs using mb and CAMDAC mt. Boxplots show the distribution of scores for permutated topologies in place of the SNV tree, as used for empirical p-value calculations. A color bar indicates the number of tumor regions per patient (excluding the patient-matched normal)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Methylation-SNV tree fit scores.</span> (A) Scoring concordance between phylogenies derived from aberrant methylation and somatic sequence alterations. We leveraged binarized (hypermethylated) tumor-normal DMP data and, based on the ancestral states given by either the SNV () or DMP () multi-sample tree topologies, counted the number of state changes per DMP locus from the normal root to all leaves (methylated to unmethylated or <em>vice versa</em>). Counts were combined into a single score by taking the total absolute difference in the number of ancestral DMP state changes based on the SNV (<em>s<sub>i</sub></em>) and DMP tree topologies (<em>d<sub>i</sub></em>, ) as a fraction of all events given by the DMP tree (). Here, a score of 0 indicates a perfect topology match, with the score increasing with the number of inconsistent per-DMP evolutionary trajectories between the two trees. Hence, the score indicates how closely the methylation events support sample relationships inferred from somatic sequence alterations. (B) A schematic of how ancestral state changes are calculated given the SNV tree by counting edges that switch from methylated to unmethylated (bold). Internal node states are inferred using ancestral reconstruction and, for all calculations, nodes with intermediate methylation are replaced with the state of the first ancestor without intermediate methylation. (C) Methylation-SNV fit scores calculated for DMPs using <em>m<sub>b</sub></em> and CAMDAC <em>m<sub>t</sub></em>. Boxplots show the distribution of scores for permutated topologies in place of the SNV tree, as used for empirical p-value calculations. A color bar indicates the number of tumor regions per patient (excluding the patient-matched normal).</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S10." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F18.medium.gif" width="440" height="255"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S10." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F18.medium.gif" width="440" height="255"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F18.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S10." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F18.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S10.
                              </span>
                              <span class="caption-title">
                               Methylation-SNV tree fit scores.
                              </span>
                              <p id="p-83" class="first-child">
                               (A) Scoring concordance between phylogenies derived from aberrant methylation and somatic sequence alterations. We leveraged binarized (hypermethylated) tumor-normal DMP data and, based on the ancestral states given by either the SNV (
                               <span class="inline-formula" id="inline-formula-4">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/F18/embed/inline-graphic-4.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/F18/embed/inline-graphic-4.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               ) or DMP (
                               <span class="inline-formula" id="inline-formula-5">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/F18/embed/inline-graphic-5.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/F18/embed/inline-graphic-5.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               ) multi-sample tree topologies, counted the number of state changes per DMP locus from the normal root to all leaves (methylated to unmethylated or vice versa). Counts were combined into a single score by taking the total absolute difference in the number of ancestral DMP state changes based on the SNV (s
                               <sub>
                                i
                               </sub>
                               ) and DMP tree topologies (d
                               <sub>
                                i
                               </sub>
                               ,
                               <span class="inline-formula" id="inline-formula-6">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/F18/embed/inline-graphic-6.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/F18/embed/inline-graphic-6.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               ) as a fraction of all events given by the DMP tree (
                               <span class="inline-formula" id="inline-formula-7">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/F18/embed/inline-graphic-7.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/F18/embed/inline-graphic-7.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               ). Here, a score of 0 indicates a perfect topology match, with the score increasing with the number of inconsistent per-DMP evolutionary trajectories between the two trees. Hence, the score indicates how closely the methylation events support sample relationships inferred from somatic sequence alterations. (B) A schematic of how ancestral state changes are calculated given the SNV tree by counting edges that switch from methylated to unmethylated (bold). Internal node states are inferred using ancestral reconstruction and, for all calculations, nodes with intermediate methylation are replaced with the state of the first ancestor without intermediate methylation. (C) Methylation-SNV fit scores calculated for DMPs using m
                               <sub>
                                b
                               </sub>
                               and CAMDAC m
                               <sub>
                                t
                               </sub>
                               . Boxplots show the distribution of scores for permutated topologies in place of the SNV tree, as used for empirical p-value calculations. A color bar indicates the number of tumor regions per patient (excluding the patient-matched normal).
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-17">
                            <h2 class="">
                             METHODS
                            </h2>
                            <div class="section" id="sec-18">
                             <h3>
                              TRACERx methylation study
                             </h3>
                             <div id="F19" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F19.large.jpg?width=800&height=600&carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F19.medium.gif" width="440" height="141"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure" src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F19.medium.gif" width="440" height="141"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F19.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure19" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F19.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <p id="p-84">
                              Samples from the first 100 patients of the TRACERx lung cancer study were selected for reduced representation bisulfite sequencing (RRBS). Patients with sufficient material remaining from 2 or more tumor regions and the adjacent matched normal were considered for bisulfite sequencing. Samples with purity below 15% were discarded with the exception of CRUK0062-R6 included for comparison with the other 6 tumor regions sampled for this patient. Patients with tumor samples of high purity were prioritized as well as those with matched RNA-Seq data (
                              <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">
                               Rosenthal et al., 2019
                              </a>
                              ). We also chose samples from both lung adenocarcinoma (LUAD) and squamous cell carcinoma (LUSC) subtypes, genders and smoking status.
                             </p>
                             <p id="p-85">
                              The TRACERx WES, WGS and RRBS data generated, used or analyzed in this study are not publicly available and restrictions apply to the availability of these data. These data are available through the Cancer Research UK & University College London Cancer Trials Centre (
                              <span class="em-link">
                               <span class="em-addr">
                                ctc.tracerx{at}ucl.ac.uk
                               </span>
                              </span>
                              ) for academic non-commercial research purposes upon reasonable request, and subject to review of a project proposal that will be evaluated by a TRACERx data access committee, entering into an appropriate data access agreement and any applicable ethical approvals.
                             </p>
                            </div>
                            <div class="section" id="sec-19">
                             <h3>
                              Reduced representation bisulfite sequencing
                             </h3>
                             <p id="p-86">
                              RRBS was obtained for roughly a third of the NSCLC patients from the TRACERx 100 cohort (122/327 tumor regions from 38/100 patients, 37 with matched normal). The NuGEN Ovation RRBS Methyl-Seq System, adapted by the manufacturer for automation on Agilent Bravo liquid handling robot, was used to generate sequencing libraries by enzymatically digesting 100 ng of gDNA using MspI, which recognizes 5′-CCGG-3′ sequences and cleaves the phosphodiester bonds upstream of CpG di-nucleotide, leaving a 2bp overhang suitable for adaptor ligation and then a final end repair step. Generated libraries were bisulfite converted using Qiagen’s EpiTect Fast DNA Bisulfite Kit. Converted libraries were amplified by PCR using 12 cycles and purified using Agencourt® RNAClean® XP magnetic beads. Purified libraries were quantified by Qubit dsDNA HS Assay (Invitrogen) and quality was evaluated using Agilent Bioanalyzer High Sensitivity DNA Assay (Agilent Technologies). Eight samples were multiplexed per flow cell and sequenced on HiSeq2500 system using HiSeq SBS Kit v4 in paired-end 100bp runs for CRUK0062 and single end 100 bp runs for the others yielding an average of 150M raw sequencing reads per sample.
                             </p>
                             <p id="p-87">
                              Sequencing outputs were converted to FASTQ files using Illumina’s bcl2fastq Conversion Software, quality checked with FastQC v0.11.2 (Babraham Institute,
                              <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">
                               https://www.bioinformatics.babraham.ac.uk/projects/fastqc/
                              </a>
                              ) and adapter sequences and diversity bases were trimmed with Trim Galore! version v0.3.7 (Babraham Institute,
                              <a href="https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/">
                               https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/
                              </a>
                              ), a wrapper around Cutadapt, (
                              <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">
                               Martin, 2011
                              </a>
                              ) and NuGEN’s trimRRBSdiversityAdaptCustomers.py v1.0 (
                              <a href="https://github.com/nugen">
                               https://github.com/nugen
                              </a>
                              technologies/NuMetRRBS). Reads are aligned to the UCSC hg19 reference assembly using Bismark v0.14.4 (
                              <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                               Krueger and Andrews, 2011
                              </a>
                              ) and deduplication was carried out using NuDup, leveraging NuGEN’s molecular tagging technology (
                              <a href="https://github.com/nugentechno">
                               https://github.com/nugentechno
                              </a>
                              logies/nudup). Binary alignment map (BAM) files were then sorted and indexed using SAMtools v1.2 (
                              <a href="https://github.com/">
                               https://github.com/
                              </a>
                              samtools/samtools/releases/tag/1.2).
                             </p>
                             <p id="p-88">
                              On average 1.88 × 10
                              <sup>
                               8
                              </sup>
                              reads per sample remained post-processing and alignment (
                              <strong>
                               Table S2
                              </strong>
                              ), resulting in an average of 4.5 million CpGs being supported by at least one read in any one sample.
                             </p>
                            </div>
                            <div class="section" id="sec-20">
                             <h3>
                              Whole genome sequencing
                             </h3>
                             <p id="p-89">
                              Whole genome sequencing was performed on 8 samples from 3 patients included in the TRACERx100 cohort. Simultaneous extraction of DNA and RNA was performed using the AllPrep DNA/RNA Mini Kit (Qiagen, UK). Frozen samples were transferred onto cold petri dishes on dry ice and were manually dissected into 20-30mg pieces. Immediately prior to extraction, the freshly dissected tissue was transferred directly into homogenization tubes containing RLT plus lysis buffer. Homogenization of tissues was carried out using a TissueRuptor II probe or using a bead method and by passing the lysate through a QIAshredder column (Qiagen, UK). The extracted DNA was eluted with 200 µl of EB (no EDTA) buffer and RNA was eluted with 200µl of nuclease-free water and stored immediately at −80°C. The DNA and RNA samples were quantified using a Qubit™ 3.0 Fluorometer (Lifetechnology, UK) and TapeStation system (Agilent, UK) respectively. The integrity of the DNA/RNA was assessed using the Agilent TapeStation system.
                             </p>
                             <p id="p-90">
                              Whole genome sequencing experiments were conducted by Edinburgh Genomics. Samples were sequencing on Illumina HiSeq X in paired-end 100bp runs. FASTQ outputs sequencing reads were quality checked with FastQC v0.11.5 (Babraham Institute,
                              <a href="https://www.babraham.ac.uk/">
                               https://www.babraham.ac.uk/
                              </a>
                              ) and then aligned to the UCSC hg19 reference assembly (including unknown contigs) obtained from GATK bundle 2.8 using bwa-mem v0.7.15 (
                              <a href="http://bio-bwa.sourceforge.net">
                               http://bio-bwa.sourceforge.net
                              </a>
                              ). Picard tools v1.107 was used to clean, sort, de-duplicate and merge files from the same patient region and to remove duplicate reads (
                              <a href="http://broadinstitute.github.io/picard">
                               http://broadinstitute.github.io/picard
                              </a>
                              ). Binary alignment map (.bam) files were sorted and indexed with SAMtools v1.3.1 (
                              <a href="http://github.com/samtools/">
                               http://github.com/samtools/
                              </a>
                              ) setting both the minimum base and mapping quality to 20.
                             </p>
                            </div>
                            <div class="section" id="sec-21">
                             <h3>
                              Copy-number aware methylation deconvolution analysis of cancers (CAMDAC)
                             </h3>
                             <p id="p-91">
                              As depicted below, CAMDAC requires RRBS data prepared from bulk tumor and matched adjacent normal samples. CAMDAC is only compatible with human (directional) RRBS data. The input must be quality and adapter trimmed with PCR duplicates removed and subsequently aligned to hg19 (hg38, GRCH37 and GRHCH38 formats also compatible). The tumor and matched normal data must be provided in .bam file format. The files must be sorted and indexed using SAMtools (
                              <a href="http://github.com/samtools/">
                               http://github.com/samtools/
                              </a>
                              ). The input tumor and matched normal sequencing data is used to compute tumor purity and allele-specific copy numbers as well as bulk tumor and normal methylation rates. From this, we extract purified tumor methylation rates and carry out tumor-normal and tumor-tumor differential methylation analysis.
                             </p>
                             <div id="F20" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F20.large.jpg?width=800&height=600&carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-55887369" data-figure-caption='<div class="highwire-markup"></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F20.medium.gif" width="440" height="168"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure" src="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F20.medium.gif" width="440" height="168"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F20.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure20" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/24/2020.11.03.366252/F20.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <p id="p-92">
                              We created a reference RRBS genome listing all genomic regions supported by at least 5 reads in at least one of the 37 germlines from this study. Genomic regions which fail to reach this criterion are unlikely to achieve minimum depth thresholds required for copy number and differential methylation analysis and can be excluded, ultimately to speed up computation.
                             </p>
                            </div>
                            <div class="section" id="sec-22">
                             <h3>
                              Tumor copy number profiling from RRBS data
                             </h3>
                             <p id="p-93">
                              First, we used CAMDAC get_allele_counts() command (
                              <a href="https://github.com/VanLoo-lab/CAMDAC">
                               https://github.com/VanLoo-lab/CAMDAC
                              </a>
                              , build=“hg19”, mq=0, n_cores=12) to compile base counts for SNP loci described by the 1000 Genomes project (1000g) (
                              <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                               Auton et al., 2015
                              </a>
                              ) which overlap with the above-mentioned list of genomic regions. By default, no mapping quality threshold is set to avoid creating a bias against the alternate allele at SNPs. Allele counts are obtained for each of these SNP to calculate the BAF and LogR.
                             </p>
                             <p id="p-94">
                              The LogR value at SNP i represents the base-2 logarithm of the ratio of the coverage in the tumor (cov
                              <sub>
                               t,i
                              </sub>
                              ) to that of the normal at that SNP (cov
                              <sub>
                               n,i
                              </sub>
                              ), divided by the average ratio. This is easily computed from RRBS data, as shown in
                              <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">
                               Eq(4)
                              </a>
                              .
                              <span class="disp-formula" id="disp-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-24.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-24.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Normalization with the adjacent normal SNP coverage, removes germline copy number variants. Next, we correct these germline normalized LogR values for fragment length, GC content and replication timing biases (
                              <strong>
                               <a id="xref-fig-10-3" class="xref-fig" href="#F10">
                                Figure S2
                               </a>
                              </strong>
                              ). This is a necessary process post-normalization as technical and biological biases affecting the sequencing output can vary between the normal and bulk tumor data. In the first instance, we reconstruct the underlying MspI fragment distribution from the sequencing output for each patient. Assuming, and confirming, complete enzymatic digestion, MspI fragment ends can be found at CCGG motifs, taking into account SNP forming/destroying cleavage sites. A CCGG is confirmed when supported with a 5’ read starting with one of the following trinucleotides: CGG(+), TGG(+), CCG(−) or CCA(−). Having identified the MspI fragments, we can readily annotate the fragment length. Then, we compute the replication timing estimate at every 1000g SNPs leveraging Repli-seq data from 15 cell lines produced by the ENCODE project (Dataset GEO Accession: GSE34399), generating 15 reference profiles. The profile which best fits the LogR data in a given tumor sample is then used to calculate the replication timing for each MspI fragment in that sample. To compute the GC content of all fragments, we leverage information from the bulk tumor reads, adjusting the reference GC content for methylation rate and bisulfite conversion. Where the fragments are longer than 2x read length, we assume that all reference cytosines are converted to thymines unless they are at CpG where we assume a probability equal to the mean of all informative CpGs loci in the fragment. Finally, we fit the observed LogR to a linear combination of the natural splines (df = 5) of MspI fragment length, replication timing and GC content. The model residuals provide corrected LogR values.
                             </p>
                             <p id="p-95">
                              The BAF value at the i
                              <sup>
                               th
                              </sup>
                              1000g SNP is the ratio of the alternate allele counts to the total counts at the ith SNP locus,
                              <span class="inline-formula" id="inline-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . The strand-specific BAF rules for every SNP type are shown in
                              <strong>
                               <a id="xref-fig-2-12" class="xref-fig" href="#F2">
                                Figure 2D
                               </a>
                              </strong>
                              . Here, we make the reasonable assumption that all reference cytosines are converted into thymines (under-conversion rate <0.5%). Note that where a variant makes up more than 5% of the total allele counts at a given SNP, it is excluded from downstream analyses in order to ensure that SNVs or mis-alignments do not affect BAF estimates and subsequent copy number profiling.
                             </p>
                             <p id="p-96">
                              Where allelic imbalance is such that two distinct band are generated, CAMDAC can assign heterozygous SNPs (0.3 ≤ BAF ≤ 0.7) to either the gained or lost alleles. SNP phasing information can then be shared between tumor samples from the same patient. This can be used to identify mirrored subclonal allelic imbalance (
                              <a id="xref-ref-45-9" class="xref-bibr" href="#ref-45">
                               Jamal-Hanjani et al., 2017
                              </a>
                              ), but most importantly, it allows us to rescue signal in tumors where there was no clear separation before haplotyping.
                             </p>
                             <p id="p-97">
                              With partially phased BAF and corrected logR estimates in hand, CAMDAC then uses ASCAT (
                              <a id="xref-ref-90-3" class="xref-bibr" href="#ref-90">
                               Van Loo et al., 2010
                              </a>
                              ) piece-wise constant segmentation (penalty = 200) leveraging germline heterozygous SNPs (0.3 ≤ BAF ≤ 0.7) and copy number fitting functions (gamma = 1) to obtain allele-specific copy number profiles and purity estimates for each tumor region. Note that we set the minimum germline coverage for SNP inclusion to 10 whilst one read is deemed sufficient in the tumor. To remove outliers and further reduce noise without introducing a bias against homozygous deletions, we remove singletons with coverage below 10 in the tumor if and only if their nearest neighbors within a 10kb moving window all pass the above threshold.
                             </p>
                             <p id="p-98">
                              There were 6 patients which showed large intra-tumor differences in their CAMDAC ploidy estimates. In each of those cases, we looked for evidence supporting an alternative ploidy solution. If such a solution existed and was better suited the overall tumor ploidy profile, the ASCAT copy number fitting step was re-run forcing the solution to the ploidy towards a diploid or tetraploid solution (see MINPLOIDY and MAXPLOIDY variables, from the runASCAT() function in ascat.R script; available from
                              <a href="https://github.com/VanLoo-lab/ascat">
                               https://github.com/VanLoo-lab/ascat
                              </a>
                              ). We refit 7 tumor copy number profiles in total. This is an improvement from the 17 samples which were manually curated and re-run in the matched exome sequencing data (
                              <a id="xref-ref-45-10" class="xref-bibr" href="#ref-45">
                               Jamal-Hanjani et al., 2017
                              </a>
                              ).
                             </p>
                            </div>
                            <div class="section" id="sec-23">
                             <h3>
                              Copy number gains and losses summary
                             </h3>
                             <p id="p-99">
                              We calculated the fraction of gains and losses in LUAD and LUSC samples using copy number alterations called with CAMDAC. The genome was partitioned into 10MB bins, each of which were classified per sample: a “gain” if the major allele is above the baseline ploidy and a “loss” if the minor allele falls below the baseline ploidy and the total tumor copy number is lower than 5. A given genomic bin can therefore be classified as both a gain and loss. The baseline ploidy is defined as diploid or tetraploid dependent on ASCAT solution. Fractions of each class were calculated across the cohort, with 99.99% (530 / 536) of bins with coverage in all samples. For TCGA samples, copy number alterations were obtained from SNP array data with ASCAT. TCGA gains and losses were obtained as above and compared with those of the TRACERx cohort by taking the average fractions at the corresponding 10MB bins. Where TCGA SNP array data did not cover RRBS regions (bins on chromosome 1 and 9), the value of the previous segment is taken as an estimate.
                             </p>
                            </div>
                            <div class="section" id="sec-24">
                             <h3>
                              Comparing RRBS- and WGS-derived genotypes
                             </h3>
                             <p id="p-100">
                              First, we compiled allele counts at all 1000g SNPs with coverage in the matched RRBS data using alleleCount v4.0.0 (
                              <a href="http://cancerit.github.io/alleleCount/">
                               http://cancerit.github.io/alleleCount/
                              </a>
                              ). We compared CAMDAC RRBS-derived genotypes with the output from standard alleleCount v4.0.0 pipeline on WGS data for the 3 germline samples of patients CRUK0031, CRUK0062 and CRUK0069. Positive calls are heterozygous SNPs (0.1 ≤ BAF ≤ 0.9) and WGS is considered to provide ground truth with a minimum SNP read depth of 10 required on both platforms for inclusion. We compute the average false positive rate (FPR) as the sum of all false positives over the sum of false positives and true negatives across samples. Similarly, the mean false negative rate (FNR) is calculated as the sum of all false negatives divided by the sum of false negatives and true positives. The FPR allows us to evaluate allele-specific biases introduced by the approach, while the FNR assesses the influence of methylation status on BAF calculation, as well as biases introduced during the RRBS protocol. For all SNP types, we consider three different contexts: (i) SNPs at CCGG, the recognition motif of the MspI restriction enzyme used during library preparation, (ii) SNPs at CpGs (excluding CCGG motifs), and (iii) all other. A chi-square analysis is performed to test whether or not the FNR is context dependent. Next, we recompute FNR and FPR aggregating FP, TP, FN and TN across all samples for a range of heterozygosity boundaries and minimum coverage thresholds (
                              <strong>
                               <a id="xref-fig-12-2" class="xref-fig" href="#F12">
                                Figure S4A
                               </a>
                              </strong>
                              ). Next, we measure the number of false negatives due to (i) allele-specific Msp1 fragments, (ii) additional reference mismatches from neighboring SNPs, and (iii) short fragments yielding invalid alignments given 2 or less reference mismatches (
                              <strong>
                               <a id="xref-fig-12-3" class="xref-fig" href="#F12">
                                Figure S4B
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-101">
                              The mean ASCAT.m FPR across all SNPs and samples was 0.3% and was always below 2%. The average FNR was 25% and was highly context dependent (χ
                              <sup>
                               2
                              </sup>
                              test, p-val < 2.2 × 10
                              <sup>
                               −16
                              </sup>
                              , FNR
                              <sub>
                               CCGG
                              </sub>
                              = 83%, FNR
                              <sub>
                               CG
                              </sub>
                              = 20%, FNR
                              <sub>
                               other
                              </sub>
                              = 15%). Higher sequencing coverage results in only modest improvements in FNR: by requiring a minimum SNP coverage of 30 instead of 10 reads, the average FNR dropped to 20% (
                              <strong>
                               <a id="xref-fig-12-4" class="xref-fig" href="#F12">
                                Figure S4A
                               </a>
                              </strong>
                              ). As expected, the majority of heterozygous SNPs perturbing or creating an MspI recognition motif are missed (49% of all false negatives,
                              <strong>
                               <a id="xref-fig-12-5" class="xref-fig" href="#F12">
                                Figure S4B
                               </a>
                              </strong>
                              ). These polymorphisms result in allele-specific fragments during RRBS library preparation that skew allelic coverage. Heterozygous SNPs on the same MspI fragment as a heterozygous CCGG are similarly affected. Finally, within the false negative calls, we see a bias towards SNPs erroneously called homozygous reference (72%) compared to homozygous alternate (28%), and this effect is present in all three contexts, although less pronounced at CCGGs. This points towards an alignment bias, possibly explained by limited mappability of short MspI fragments with alternate alleles. Supporting this theory, SNPs with lower reference allele read mapping quality scores were more likely to yield homozygous reference false negatives (Wilcoxon test, p < 2.2 × 10
                              <sup>
                               −16
                              </sup>
                              ), especially outside CCGG context.
                             </p>
                            </div>
                            <div class="section" id="sec-25">
                             <h3>
                              Tumor copy-number profiling from whole genome sequencing data
                             </h3>
                             <p id="p-102">
                              First, we compiled allele counts at all 1000g SNPs with coverage in the matched RRBS data using alleleCount v4.0.0 (
                              <a href="http://cancerit.github.io/alleleCount/">
                               http://cancerit.github.io/alleleCount/
                              </a>
                              ). We then we ran ASCAT GC and replication timing LogR correction steps, fed the output into ASCAT piece-wise constant segmentation (penalty = 200) and obtained copy number and purity (gamma = 1). Comparing WGS and RRBS-derived profiles, even with the higher FNR on SNP calling in RRBS data, CAMDAC BAF estimates enable ASCAT to correctly identify separation of the BAF bands on several occasions where the WGS BAF does not seem to provide sufficient power. Likewise, comparison of the LogR tracks indicates that RRBS protocol-induced biases in the coverage are adequately modelled and removed by CAMDAC. Unsurprisingly, final allele-specific copy number segments are virtually indistinguishable between the two data types.
                             </p>
                            </div>
                            <div class="section" id="sec-26">
                             <h3>
                              Methylation rate calculation from RRBS data
                             </h3>
                             <p id="p-103">
                              The bulk tumor and matched normal methylation rate is readily computed by taking the ratio of methylated CpG read counts to the sum of methylated and unmethylated read counts,
                              <span class="inline-formula" id="inline-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . CAMDAC uses strand-specific dinucleotide counts to distinguish between methylated and unmethylated CpGs,
                              <span class="inline-formula" id="inline-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-10.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-10.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . At CpG>TpG (TpG>CpG) and CpG> CpA (CpA> CpG) SNPs, only reads from the bottom strand and top strand, respectively, contribute to CAMDAC methylation rate estimates. In addition, only the CG-forming allele contributes to the methylation rate at polymorphic CpGs. This enables the methylation rate at a heterozygous CpG to vary between 0 and 1, rather than between 0 and 0.5 in a diploid sample and ensures further independence between methylation rate and copy number estimates.
                             </p>
                             <p id="p-104">
                              We compiled bulk tumor and normal methylation rates for all CpGs which fell within the above-mentioned reference RRBS genomic regions list. For each patient, we discarded all CpGs that failed to reach a minimum coverage of 10 in the matched normal RRBS data. CpGs that had less than 3 reads in a given tumor sample were also filtered out from that sample.
                             </p>
                            </div>
                            <div class="section" id="sec-27">
                             <h3>
                              CAMDAC purified tumor methylation rates from RRBS data
                             </h3>
                             <p id="p-105">
                              Bulk tumor methylation rate (m
                              <sub>
                               b
                              </sub>
                              ) could be expressed as a function of the methylation rate in the tumor cells (m
                              <sub>
                               t
                              </sub>
                              ) and contaminating normal cells (m
                              <sub>
                               t
                              </sub>
                              ), scaled for the purity of the sample (ρ) and the local copy number state (i.e. n
                              <sub>
                               n
                              </sub>
                              = 2 in the normal and n
                              <sub>
                               t
                              </sub>
                              in the tumor):
                              <span class="disp-formula" id="disp-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-25.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-25.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Taking estimates of the total copy number at each CpG and tumor purity from the RRBS copy number profile and assuming the matched normal is a reasonable proxy for the contaminating normal, we can calculate the deconvolved tumor methylation rate m
                              <sub>
                               t
                              </sub>
                              :
                              <span class="disp-formula" id="disp-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-26.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-26.gif"/>
                                </noscript>
                               </span>
                              </span>
                              An important distinction is made at heterozygous polymorphic CpGs, where only one copy in the diploid normal contains methylation information (i.e. n
                              <sub>
                               &
                              </sub>
                              = 1). Similarly, if the SNP is CG-destroying and its BAF is smaller than 0.5 or likewise if the SNP is CG-forming and BAF is above 0.5 the major copy informs the methylation (i.e. n
                              <sub>
                               t
                              </sub>
                              = n
                              <sub>
                               major
                              </sub>
                              ). Vice versa, if the tumor BAF is larger than 0.5 and the SNP is CG-destroying or the BAF lower than 0.5 and the SNP is CG-forming, the minor allele only informs the methylation rate (i.e. n
                              <sub>
                               t
                              </sub>
                              = n
                              <sub>
                               minor
                              </sub>
                              ).
                             </p>
                             <p id="p-106">
                              Finally, m
                              <sub>
                               t
                              </sub>
                              must be between 0 and 1. However, due to both technical and biological noise, values can fall outside of these boundaries. For downstream analyses, we set negative methylation rates up to 0 and values sitting above the upper boundary are rounded down to 1. In virtually all cases, the m
                              <sub>
                               t
                              </sub>
                              99% HDI overlaps with allowed values.
                             </p>
                            </div>
                            <div class="section" id="sec-28">
                             <h3>
                              Nuclei extraction and FACS analysis
                             </h3>
                             <p id="p-107">
                              We followed the ‘Frankenstein’ protocol (
                              <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">
                               Martelotto, 2020
                              </a>
                              ) to extract nuclei from seven fresh frozen tissue samples taken from five different patients: CRUK0050-R4, CRUK0062-R5, CRUK0070-R2 and R4, CRUK0079-R1 and R4 and CRUK0090-R1. Samples were taken from different tissue cuts of the same sampled tumor region as the matched WES and RRBS samples. The bulk tumor tissue samples were minced and put into a lysis buffer. The lysate was homogenized with a pestle and the mixture was then passed through a 70μm filtered. The isolated nuclei mixture was washed and suspended into a buffer containing propidium iodide (PI) stain (70 μg/mL PI, 1% BSA, 1×PBS) and passed through a smaller 35μm filter.
                             </p>
                             <p id="p-108">
                              The stained nuclei were then passed through the FACS machine. Nuclei were separated according to the side scatter, a measure of cell morphology and the PI chromophore intensity, representing the quantity of chromatin in each nucleus. We first sorted a small number of nuclei to adjust the sorting parameters to ensure that debris was removed. We also discarded superfluous ploidy populations, including nuclei having originated from replicating diploid or aneuploid cells. We then collected nuclei from both the major aneuploid population, popA, and the diploid population, popD, into separate tubes each with 200 μL PBS and 2% FCS. We counted between 100-300 thousand events per population. DNA extraction for each sample and nuclei subpopulation was performed using the Zymo Research Quick DNA-microprep plus kit (D4074) following the indications from manufacturer. The DNA samples were finally eluted in 12μL of elution buffer.
                             </p>
                            </div>
                            <div class="section" id="sec-29">
                             <h3>
                              RRBS of FACS-purified samples
                             </h3>
                             <p id="p-109">
                              The NuGEN Ovation RRBS Methyl-Seq System protocol was followed for library preparation and for subsequent bisulfite sequencing, as above-described for the bulk tumor RRBS data, but without any automation.
                             </p>
                             <p id="p-110">
                              Libraries were prepared by enzymatically digesting ∼100ng of gDNA with MspI. Qiagen’s EpiTect Fast DNA Bisulfite Kit was used for bisulfite conversion of the resulting DNA fragments. Bisulfite converted libraries were then amplified by PCR using 12 cycles and purified using Agencourt® RNAClean® XP magnetic beads. Library quantification was performed by Qubit dsDNA HS Assay (Invitrogen) and quality control was carried out using Agilent Bioanalyzer High Sensitivity DNA Assay (Agilent Technologies). In cases with two samples from the same patient, both popA and only one popA were made into libraries and sequenced to save on costs. We therefore sequenced 7 tumor aneuploid and 5 (presumably) normal diploid populations.
                             </p>
                             <p id="p-111">
                              RRBS was performed by at the Francis Crick Institute sequencing facility. The 12 samples were multiplexed across 4 lanes on HiSeq 4000 using the HiSeq® 3000/4000 SBS Kit. As for the bulk tumor samples, 100bp SE and 10bp reads were generated for the NuGEN RRBS library insert and unique molecular identifiers, respectively. We aimed to sequence 120,000,000 reads per sample.
                             </p>
                             <p id="p-112">
                              Sequencing reads were QC’ed, adapter trimmed, aligned to hg19, PCR-deduplicated and output binary alignment map (BAM) files were sorted and indexed following the same procedure as described for the TRACERx bulk tumor RRBS dataset (Methods, section 2.4.2.1).
                             </p>
                             <p id="p-113">
                              On average, 109,864,415 raw sequencing reads were obtained per samples. Mapping efficiency averaged around 70.5%, as expected for bisulfite sequencing data aligned with Bismark. However, samples had very high duplication rates (average 57.18%), leaving only 31,533,497 reads per sample post-processing. This is potentially due to lower DNA inputs and combined bisulfite- and FACS-driven DNA degradation. This effect was not observed on the previous cohort generated using the same protocol, but without FACS.
                             </p>
                            </div>
                            <div class="section" id="sec-30">
                             <h3>
                              SNV calling from whole-exome and -genome sequencing data
                             </h3>
                             <p id="p-114">
                              SNV calls from TRACERx 100 patients are readily obtained Jamal-Hanjani et al. (
                              <a id="xref-ref-45-11" class="xref-bibr" href="#ref-45">
                               Jamal-Hanjani et al., 2017
                              </a>
                              ). The same protocol is followed to identify SNVs from newly generated WGS data and is detailed below. In short, SAMtools mpileup (v1.3.1) was used to locate alternate alleles in tumor and germline samples (minimum base quality ≥ 20 and mapping quality ≥ 20). The output was processed with VarScan2 v2.3.6 processSomatic and used to identify somatic variants (VAF ≥ 0.01, purity = 0.5), filtering out germline SNPs using the matched germline samples. The resulting single nucleotide variant (SNV) calls were filtered for false positives using Varscan2’s associated fpfilter.pl script, initially with default settings then repeated again with min-var-frac = 0.02, having first run the data through bam-readcount v0.5.1 (
                              <a href="https://github.com/genome/bam-readcount">
                               https://github.com/genome/bam-readcount
                              </a>
                              ). SNVs were also identified using MuTect v1.1.4, filtered for passing SNVs (“PASS”) and annotated using the GATK bundle 2.8. SNVs were deemed true positives if (A) VAF ≥ 2% and the SNV was called by both VarScan2, with a somatic p-value ≥ 0.01, and MuTect, (B) VAF ≥ 5% and the mutation was only called in VarScan2, again with a somatic p-value ≤ 0.01. A minimum total and mutant allele coverage of 30 and 5 reads was required for inclusion. The number of reads supporting the variant in the germline data must to be < 5 and the VAF ≤ 1%. SNVs were also removed if they were found to match a germline polymorphisms with population frequency > 1% across the TRACERx 100 cohort. SNVs overlapping with a blacklist of genomic regions including reports by the ENCODE project (both DAC and Duke list), simple repeats, segmental duplications and microsatellite regions were obtained from the UCSC Genome Table Browser.
                             </p>
                             <p id="p-115">
                              By sharing mutations calls between samples from the same patient, and reassessing mutant reads at each position, we can reduce the possibility of over-representing the mutational heterogeneity. Where a somatic variant was not called ubiquitously across tumor regions, but was called in one or more sample, reads were re-extracted from the original alignment file using bam-readcount v0.5.1 (
                              <a href="https://github.com/genome/bam-readcount">
                               https://github.com/genome/bam-readcount
                              </a>
                              ). In these instances, minimum VAF requirements were decreased to VAF ≥ 1%, thereby rescuing low frequency variants that would otherwise have been missed.
                             </p>
                            </div>
                            <div class="section" id="sec-31">
                             <h3>
                              SNV-phased methylation rate estimates
                             </h3>
                             <p id="p-116">
                              Leveraging SNV calls from newly obtained WGS data and previously published WES (
                              <a id="xref-ref-45-12" class="xref-bibr" href="#ref-45">
                               Jamal-Hanjani et al., 2017
                              </a>
                              ), we phased CpG methylation to all SNVs, excluding loci with VAF ≤ 0.1 in a tumor sample or VAF > 0 in the patient-matched adjacent normal tissue. Allele-specific methylation counts were compiled for all reads that could be phased to exactly one SNV. Phased methylation rates were obtained for 32,874 CpGs and 6,529 SNVs across samples (14,514 and 2,984 unique CpGs and SNVs respectively across patients). The VAF was derived from the mutant (mut) and wild type (WT) reads counts:
                              <span class="inline-formula" id="inline-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-11.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-11.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . RRBS-derived VAF estimates were compared with those obtained from the WES/WGS (Pearson correlation = 0.86).
                             </p>
                             <p id="p-117">
                              The mutation copy number, n
                              <sub>
                               mut
                              </sub>
                              , was then computed as a function of the variant allele frequency, tumor purity and copy number:
                              <span class="inline-formula" id="inline-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-12.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-12.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . The wild type allele copy number, n
                              <sub>
                               WT
                              </sub>
                              , is obtained by subtracting n
                              <sub>
                               mut
                              </sub>
                              from n
                              <sub>
                               t
                              </sub>
                              : n
                              <sub>
                               WT
                              </sub>
                              = n
                              <sub>
                               t
                              </sub>
                              − n
                              <sub>
                               mut
                              </sub>
                              . The mutant allele methylation rate, m
                              <sub>
                               mut
                              </sub>
                              , is extracted by taking the counts methylated (m
                              <sub>
                               mut
                              </sub>
                              ) and unmethylated (UM
                              <sub>
                               mut
                              </sub>
                              ) divided by all counts phased to the variant allele: m
                              <sub>
                               mut
                              </sub>
                              = m
                              <sub>
                               mut
                              </sub>
                              /(m
                              <sub>
                               mut
                              </sub>
                              + UM
                              <sub>
                               mut
                              </sub>
                              ). The wild type allele methylation rate, m
                              <sub>
                               WT
                              </sub>
                              , is confounded by signal from normal contaminating cells and must be deconvolved. For this, we use a modified version of the CAMDAC
                              <a id="xref-disp-formula-2-4" class="xref-disp-formula" href="#disp-formula-2">
                               equations 2
                              </a>
                              and
                              <a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">
                               3
                              </a>
                              , where the tumor methylation rate and copy number are expressed in terms of the mutant and wild type alleles.
                              <span class="disp-formula" id="disp-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-27.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-27.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <span class="disp-formula" id="disp-formula-8">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-28.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-28.gif"/>
                               </noscript>
                              </span>
                             </span>
                             <p id="p-118">
                              We validate CAMDAC m
                              <sub>
                               t
                              </sub>
                              by comparison with methylation estimates phased to clonal SNVs present on all copies in regions with loss of heterozygosity across our cohort (n
                              <sub>
                               mut
                              </sub>
                              = n
                              <sub>
                               %
                              </sub>
                              ). At these sites, all reads reporting the variant allele can directly be assigned to the tumor cells, and methylation rates obtained from this subset of reads should be an unbiased estimate of the purified tumor methylation rate (i.e. n
                              <sub>
                               mut
                              </sub>
                              = 0). Overall, 4,485 CpG loci met these criteria. A high correlation was observed between these SNV deconvoluted m
                              <sub>
                               t
                              </sub>
                              values and CAMDAC estimates (Pearson correlation = 0.97).
                             </p>
                            </div>
                            <div class="section" id="sec-32">
                             <h3>
                              Tumor-normal differential methylation analysis
                             </h3>
                             <p id="p-119">
                              We designed a statistical test to robustly identify DMPs between tumor and normal. The number of methylated reads at a CpG is often described as a Beta-Binomial distribution where the mean of the probability distribution, the methylation rate, follows a Beta distribution. We can therefore model the observed bulk tumor and matched normal methylation rate at CpG locus i as follows:
                              <span class="disp-formula" id="disp-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-29.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-29.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where: m
                              <sub>
                               x,i
                              </sub>
                              is the methylation rate at the ith CpG locus
                             </p>
                             <p id="p-120">
                              α
                              <sub>
                               x,i
                              </sub>
                              is the counts methylated at the ith CpG locus
                             </p>
                             <p id="p-121">
                              β
                              <sub>
                               x,i
                              </sub>
                              is the counts unmethylated at the ith CpG locus
                             </p>
                             <p id="p-122">
                              x is b for the bulk tumor, n for the normal and t for the purified tumor
                             </p>
                             <p id="p-123">
                              Our statistical design aims to test whether or not the tumor methylation rate at the ith CpG locus, m
                              <sub>
                               t,i
                              </sub>
                              , is different from the normal methylation rate, m
                              <sub>
                               n,i
                              </sub>
                              . This is depicted in equations
                              <a id="xref-disp-formula-10-1" class="xref-disp-formula" href="#disp-formula-10">
                               8
                              </a>
                              and
                              <a id="xref-disp-formula-11-1" class="xref-disp-formula" href="#disp-formula-11">
                               9
                              </a>
                              .
                              <span class="disp-formula" id="disp-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-30.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-30.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <span class="disp-formula" id="disp-formula-11">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-31.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-31.gif"/>
                               </noscript>
                              </span>
                             </span>
                             <p id="p-124">
                              We recall from
                              <a id="xref-disp-formula-2-5" class="xref-disp-formula" href="#disp-formula-2">
                               Eq(2)
                              </a>
                              that the tumor methylation rate, m
                              <sub>
                               t
                              </sub>
                              , can be expressed as a function of m
                              <sub>
                               b
                              </sub>
                              and m
                              <sub>
                               t
                              </sub>
                              . By substituting
                              <a id="xref-disp-formula-2-6" class="xref-disp-formula" href="#disp-formula-2">
                               Eq(2)
                              </a>
                              into
                              <a id="xref-disp-formula-11-2" class="xref-disp-formula" href="#disp-formula-11">
                               Eq(9)
                              </a>
                              , we obtain:
                              <span class="disp-formula" id="disp-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-32.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-32.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Which can be simplified to:
                              <span class="disp-formula" id="disp-formula-13">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-33.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-33.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Expression (11) demonstrates that our statistical model is independent of copy number and tumor purity, which are constant for the i
                              <sup>
                               th
                              </sup>
                              CpG site. However, the power to call DMPs will intrinsically depend on purity, local copy number and CpG coverage. The former two will affect the magnitude of the difference between m
                              <sub>
                               b
                              </sub>
                              and m
                              <sub>
                               t
                              </sub>
                              whilst the latter will affect the width of each beta distribution. Since there is a closed-form solution for testing P(m
                              <sub>
                               b,i
                              </sub>
                              > m
                              <sub>
                               n,i
                              </sub>
                              ), but not for P(m
                              <sub>
                               b,i
                              </sub>
                              = m
                              <sub>
                               n,i
                              </sub>
                              ) (
                              <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">
                               Robinson, 2017
                              </a>
                              ), the null and alternate hypotheses are written as follows:
                             </p>
                             <p id="p-125">
                              H
                              <sub>
                               0
                              </sub>
                              : m
                              <sub>
                               b,i
                              </sub>
                              = m
                              <sub>
                               n,i
                              </sub>
                              The methylation rate of the i
                              <sup>
                               th
                              </sup>
                              CpG is identical in normal and tumor
                             </p>
                             <p id="p-126">
                              H
                              <sub>
                               1
                              </sub>
                              : m
                              <sub>
                               b,i
                              </sub>
                              > m
                              <sub>
                               n,i
                              </sub>
                              The tumor is hypermethylated at this locus
                             </p>
                             <p id="p-127">
                              H
                              <sub>
                               2
                              </sub>
                              : m
                              <sub>
                               b,i
                              </sub>
                              < m
                              <sub>
                               n,i
                              </sub>
                              The tumor is hypomethylated at this locus
                              <span class="disp-formula" id="disp-formula-14">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-34.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-34.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <span class="disp-formula" id="disp-formula-15">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-35.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-35.gif"/>
                               </noscript>
                              </span>
                             </span>
                             <p id="p-128">
                              <a id="xref-disp-formula-14-1" class="xref-disp-formula" href="#disp-formula-14">
                               Eq(12)
                              </a>
                              can be rewritten as follows, incorporating our Beta(0.5, 0.5) prior to each methylation count variable. The prior informs on the underlying methylation rate distribution and ensures finite logarithms:
                              <span class="disp-formula" id="disp-formula-16">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-36.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-36.gif"/>
                                </noscript>
                               </span>
                              </span>
                              We compute the probability that a CpG site is hypo- or hypermethylated and for easier interpretation, we express these probabilities as their complement (C = 1-P), which is a measure the overlap between m
                              <sub>
                               b,i
                              </sub>
                              and m
                              <sub>
                               n,i
                              </sub>
                              . If C(m
                              <sub>
                               b,i
                              </sub>
                              > m
                              <sub>
                               n,i
                              </sub>
                              ) or C(m
                              <sub>
                               b,i
                              </sub>
                              < m
                              <sub>
                               n,i
                              </sub>
                              ) ≤ 0.01 we accept H
                              <sub>
                               1
                              </sub>
                              or H
                              <sub>
                               2
                              </sub>
                              respectively.
                             </p>
                             <p id="p-129">
                              Differential methylation analyses incorporating a beta distribution model reportedly show higher true positive and lower false discovery rate are obtained compared with Fisher’s and z-score methods (
                              <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">
                               Raineri et al., 2014
                              </a>
                              ). Nevertheless, given high enough coverage, even a small difference in methylation can become statistically significant. In order to focus our analysis of biologically significant methylation changes, we require a minimum effect size of 0.2 between the purified tumor methylation rate, m
                              <sub>
                               t,i
                              </sub>
                              , and the matched normal, m
                              <sub>
                               n,i
                              </sub>
                              , for DMP calling. In theory, this allows for subclonal or allele specific changes to be picked up whilst removing spurious signal. We obtained a second set of DMPs by applying the minimum effect size threshold on the difference between the bulk tumor methylation rate, m
                              <sub>
                               b,i
                              </sub>
                              , and the normal methylation rate, m
                              <sub>
                               n,i
                              </sub>
                              as is customary (
                              <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                               Dolzhenko and Smith, 2014
                              </a>
                              ;
                              <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                               Hansen et al., 2012
                              </a>
                              ;
                              <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                               Klein and Hebestreit, 2016
                              </a>
                              ;
                              <a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">
                               Robinson et al., 2014
                              </a>
                              ;
                              <a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">
                               Wu et al., 2015
                              </a>
                              ), and compared the output with CAMDAC calls. The threshold of 0.2 was deemed sufficiently low to capture most mono-allelic aberrations whilst be sufficiently high to remove false positives and filter noise from the heterogeneous normal contaminating cells.
                             </p>
                            </div>
                            <div class="section" id="sec-33">
                             <h3>
                              Validation of CAMDAC purified tumor methylation rates at tumor-normal DMPs
                             </h3>
                             <p id="p-130">
                              For a given tumor sample, we validate our model by comparing observed and expected clonal bi-allelic DMPs modal methylation peak position. Beta regression was used to estimate the mode of the peak generated by hyper- and hypomethylated bi-allelic DMP populations in the bulk tumor (m
                              <sub>
                               b
                              </sub>
                              ) and CAMDAC pure tumor methylation rates distribution, stratified by tumor purity, copy number and matched normal methylation status. Tumor-normal hyper- and hypomethylated CpGs were included if confidently unmethylated (99% HDI ⊆ [0, 0.2]) and methylated (99% HDI ⊆ [0.8, 1]) in the adjacent patient-matched normal, respectively. Note that allele-specific copy number states with ≤ 10,000 loci in a given sample were ignored (inter-quartile range of the number of CpGs per allele-specific copy number state is [180 125, 807 774]). CpGs on sex chromosomes were also excluded due to coverage biases against the inactive chromosome X copy shifting methylation estimates. CpGs meeting these criteria across samples had total tumor copy number ranging from 1 to 8.
                             </p>
                             <p id="p-131">
                              The expected values of hypo- (m
                              <sub>
                               0
                              </sub>
                              ) and hypermethylated (m
                              <sub>
                               +
                              </sub>
                              ) clonal bi-allelic DMPs were derived from the modal methylation rate of the peaks at 0 and 1 in the patient-matched normal as estimated by beta regression as opposed to using exactly 0 and 1. The predicted CAMDAC m
                              <sub>
                               t
                              </sub>
                              are set to exactly m
                              <sub>
                               0
                              </sub>
                              and m
                              <sub>
                               1
                              </sub>
                              for CpGs having lost and gained methylation, respectively. The expected bulk tumor modal methylation rates at hypermethylated loci was computed by feeding sample purity, tumor copy number, n
                              <sub>
                               n
                              </sub>
                              = 2, m
                              <sub>
                               n
                              </sub>
                              = m
                              <sub>
                               0
                              </sub>
                              and m
                              <sub>
                               t
                              </sub>
                              = m
                              <sub>
                               1
                              </sub>
                              into
                              <a id="xref-disp-formula-2-7" class="xref-disp-formula" href="#disp-formula-2">
                               Eq(2)
                              </a>
                              . Vice versa, m
                              <sub>
                               n
                              </sub>
                              and m
                              <sub>
                               t
                              </sub>
                              were substituted with m
                              <sub>
                               1
                              </sub>
                              and m
                              <sub>
                               0
                              </sub>
                              , respectively, to calculate the expectation at hypomethylated loci.
                             </p>
                            </div>
                            <div class="section" id="sec-34">
                             <h3>
                              Tumor-tumor differential methylation analysis
                             </h3>
                             <p id="p-132">
                              The purified tumor methylation rate is not a beta distribution but rather the difference between two betas, the bulk tumor and normal methylation rate, scaled for tumor purity and copy number. As such, there is no exact solution to compute the highest density interval for m
                              <sub>
                               t,i
                              </sub>
                              . To address this, we simulate a credible 99% HDI for m
                              <sub>
                               t,i
                              </sub>
                              at every CpG. We use the tumor purity and CpG copy number and simulate 2000 data points for the bulk tumor (m
                              <sub>
                               b,i
                              </sub>
                              ) and matched normal methylation rate (m
                              <sub>
                               n,i
                              </sub>
                              ), given m
                              <sub>
                               n,i
                              </sub>
                              ∼ B(α
                              <sub>
                               x,i
                              </sub>
                              , β
                              <sub>
                               x,i
                              </sub>
                              ). Substituting these into
                              <a id="xref-disp-formula-2-8" class="xref-disp-formula" href="#disp-formula-2">
                               Eq(2)
                              </a>
                              , we obtain a vector of values for m
                              <sub>
                               t,i
                              </sub>
                              and readily extract the 99% HDI. If the purified tumor methylation rate HDI does not overlap between any two tumor regions at a given CpG and the minimum effect size, 0.2, is reached, a tumor-tumor DMP is identified.
                             </p>
                            </div>
                            <div class="section" id="sec-35">
                             <h3>
                              Tumor-normal and tumor-tumor differential methylation rate simulation framework
                             </h3>
                             <p id="p-133">
                              To appreciate the effect of tumor purity on both tumor-normal and tumor-tumor differential methylation analysis, we extracted the 20 lowest and highest purity samples in our cohort, ρ < 0.3 and ρ > 0.58, respectively. We combined the methylation information at all overlapping autosomal CpGs from these samples including the patient-matched normal of selected samples. The normal methylation rate at confidently unmethylated and methylated CpGs was extracted. Confidently unmethylated and methylated CpGs are respectively defined as having their methylation rate 99% highest density interval (HDI) boundaries in the 0.0-0.2 and 0.8-1.0 intervals (
                              <strong>
                               <a id="xref-fig-9-7" class="xref-fig" href="#F9">
                                Figure S1A-B
                               </a>
                              </strong>
                              ). This vector of values incorporates information for both distributions such as mean methylation rate and deviation as well as their respective contribution to the overall bimodal normal methylation rate profile. Sampling from this vector will yield our simulation priors.
                             </p>
                             <p id="p-134">
                              Next, we randomly selected intra- and inter-tumor CpG pairs from samples within or across purity categories and of equal or differing total copy numbers (1, 2, 3, 4, 5 or ≥6). For each 168 possible combination of these simulation parameters, we sampled 10,000 loci from each tumor samples randomly assigned as x and y as well as their matched normal.
                             </p>
                             <p id="p-135">
                              For each locus, we begin by obtaining the coverage information from i
                              <sup>
                               th
                              </sup>
                              selected CpGs in sample s = x or y. We know the bulk tumor coverage (cov
                              <sub>
                               b,s,i
                              </sub>
                              ), the tumor copy number (n
                              <sub>
                               t,s,i
                              </sub>
                              ), the normal copy number (n
                              <sub>
                               n,s,i
                              </sub>
                              = 2) and the global tumor purity (ρ
                              <sub>
                               s
                              </sub>
                              ) and so the tumor DNA fraction (f
                              <sub>
                               t,s,i
                              </sub>
                              ) is
                              <span class="inline-formula" id="inline-formula-13">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-13.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-13.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . We can work out the purified tumor coverage as cov
                              <sub>
                               t,s,i
                              </sub>
                              = cov
                              <sub>
                               b,s,i
                              </sub>
                              × f
                              <sub>
                               t,s,i
                              </sub>
                              where cov
                              <sub>
                               t,s,i
                              </sub>
                              ∼ Binom(cov
                              <sub>
                               b,s,i
                              </sub>
                              , f
                              <sub>
                               t,s,i
                              </sub>
                              ). The matched normal coverage (cov
                              <sub>
                               n,s,i
                              </sub>
                              ) is therefore cov
                              <sub>
                               n,s,i
                              </sub>
                              = cov
                              <sub>
                               b,s,i
                              </sub>
                              − cov
                              <sub>
                               t,s,i
                              </sub>
                              .
                             </p>
                             <p id="p-136">
                              We then sample a normal methylation rate prior (p
                              <sub>
                               &,6,7
                              </sub>
                              ) from the confidently unmethylated (v
                              <sub>
                               *&’(%)
                              </sub>
                              ) and methylated CpGs (v
                              <sub>
                               meth
                              </sub>
                              ) from the matched normal data which is used to simulate the normal methylation rates of normal contaminating cells from both tumor samples x and y. For each sample, we obtain the counts methylated (M
                              <sub>
                               n,s,i
                              </sub>
                              ) and unmethylated (UM
                              <sub>
                               n,s,i
                              </sub>
                              ): M
                              <sub>
                               n,s,i
                              </sub>
                              ∼ Binom(cov
                              <sub>
                               t,s,i
                              </sub>
                              , p
                              <sub>
                               t,s,i
                              </sub>
                              ) and UM
                              <sub>
                               n,s,i
                              </sub>
                              = cov
                              <sub>
                               t,s,i
                              </sub>
                              − M
                              <sub>
                               n,s,i
                              </sub>
                              .
                             </p>
                             <p id="p-137">
                              The purified tumor methylation rate is obtained by randomly selecting from the same vector as the matched normal ({p
                              <sub>
                               n,s,i
                              </sub>
                              , p
                              <sub>
                               t,s,i
                              </sub>
                              } ∈ v
                              <sub>
                               unmeth
                              </sub>
                              or {p
                              <sub>
                               n,s,i
                              </sub>
                              , p
                              <sub>
                               t,s,i
                              </sub>
                              } ∈ v
                              <sub>
                               meth
                              </sub>
                              ) or from opposite vector states (p
                              <sub>
                               t,s,i
                              </sub>
                              ∈ v
                              <sub>
                               unmeth
                              </sub>
                              and p
                              <sub>
                               n,i
                              </sub>
                              ∈ v
                              <sub>
                               meth
                              </sub>
                              or p
                              <sub>
                               t,s,i
                              </sub>
                              ∈ v
                              <sub>
                               meth
                              </sub>
                              and p
                              <sub>
                               n,i
                              </sub>
                              ∈ v
                              <sub>
                               unmeth
                              </sub>
                              ). We obtain the counts methylated (M
                              <sub>
                               t,s,i
                              </sub>
                              ) and unmethylated (UM
                              <sub>
                               t,s,i
                              </sub>
                              ) as M
                              <sub>
                               t,s,i
                              </sub>
                              ∼ Binom(cov
                              <sub>
                               t,s,i
                              </sub>
                              , p
                              <sub>
                               n,s,i
                              </sub>
                              ) and UM
                              <sub>
                               t,s,i
                              </sub>
                              = cov
                              <sub>
                               t,s,i
                              </sub>
                              − M
                              <sub>
                               t,s,i
                              </sub>
                              . The bulk methylation counts for both samples are easily calculated by adding the tumor and normal counts: M
                              <sub>
                               bs,i
                              </sub>
                              = M
                              <sub>
                               n,s,i
                              </sub>
                              + M
                              <sub>
                               t,s,i
                              </sub>
                              and UM
                              <sub>
                               b,s,i
                              </sub>
                              = UM
                              <sub>
                               n,s,i
                              </sub>
                              + UM
                              <sub>
                               t,s,i
                              </sub>
                              . In balanced copy number regions, we also simulate allele-specific DMPs whereby one allele is in the normal ground state and the other is differentially methylated. We obtain the counts methylated from the minor allele, allele A, and for the major allele, allele B, and combine them to obtain total counts methylated and unmethylated.
                             </p>
                             <p id="p-138">
                              The expected absolute tumor-normal methylation difference at simulated bi-allelic DMPs is |m
                              <sub>
                               t
                              </sub>
                              − m
                              <sub>
                               t
                              </sub>
                              | ≈ 0.95 (
                              <strong>
                               <a id="xref-fig-5-7" class="xref-fig" href="#F5">
                                Figure 5A
                               </a>
                              </strong>
                              ). In the bulk, the magnitude of the difference depends on sample tumor purity and CpG copy number. The purified methylation rate at mono-allelic DMPs usually depends on the copy number of the mutated allele, however, in balanced copy number regions, where n
                              <sub>
                               A
                              </sub>
                              = n
                              <sub>
                               B
                              </sub>
                              and given that one copy is clonally differentially methylated and the other is in ground,
                              <span class="inline-formula" id="inline-formula-14">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-14.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-14.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . The expected tumor-normal difference at simulated mono-allelic DMPs is thus |m
                              <sub>
                               t
                              </sub>
                              − m
                              <sub>
                               t
                              </sub>
                              | ≈ 0.5 (
                              <strong>
                               <a id="xref-fig-18-2" class="xref-fig" href="#F18">
                                Figure S10A
                               </a>
                              </strong>
                              ).
                             </p>
                             <p id="p-139">
                              Tumor-normal and tumor-tumor differential methylation calls were made using CAMDAC differential methylation analysis and the output compared with the ground truth. False negative and positive rates are obtained for the bulk and deconvolved tumor simulated data (
                              <strong>
                               <a id="xref-fig-5-8" class="xref-fig" href="#F5">
                                Figures 5B-D
                               </a>
                               and
                               <a id="xref-fig-15-5" class="xref-fig" href="#F15">
                                S7B-D
                               </a>
                              </strong>
                              ).
                             </p>
                            </div>
                            <div class="section" id="sec-36">
                             <h3>
                              Tumor-normal and tumor-tumor differential methylation analysis on real data
                             </h3>
                             <p id="p-140">
                              We compare tumor-normal DMP calls based on CAMDAC- and FACS-purified tumor methylation rates and the bulk tumor-adjacent normal. We compute the overlap between both DMP calls, taking the ratio of DMPs called by both approaches divided by total number of FACS-purified DMPs. The overlap was 77.3% across combining DMPs across samples and the per sample overlap is displayed in (
                              <strong>
                               <a id="xref-fig-5-9" class="xref-fig" href="#F5">
                                Figure 5E
                               </a>
                              </strong>
                              ). Note that for tumor-normal DMP calling of FACS-purified data, we replace bulk tumor methylation counts, M
                              <sub>
                               b
                              </sub>
                              and UM
                              <sub>
                               b
                              </sub>
                              for M
                              <sub>
                               f,t
                              </sub>
                              and UM
                              <sub>
                               f,t
                              </sub>
                              in
                              <a id="xref-disp-formula-16-1" class="xref-disp-formula" href="#disp-formula-16">
                               Eq(14)
                              </a>
                              and apply the minimum effect size on |m
                              <sub>
                               f,t
                              </sub>
                              − m
                              <sub>
                               t
                              </sub>
                              |. DMP calling power is measured as the number of tumor reads per chromosome copy (nrpcc,
                              <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">
                               Dentro et al., 2021
                              </a>
                              ) in the bulk RRBS data,
                              <span class="inline-formula" id="inline-formula-15">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-15.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-15.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <p id="p-141">
                              To evaluate the impact of CAMDAC bulk tumor methylation deconvolution, we compared deconvolved with bulk tumor-tumor DMP calls between all 21 sample pairs from CRUK0062, taken for having the most tumor regions. Both DMP call sets were obtained by calculating the exact overlap between the 99% HDI of the methylation rate at the i
                              <sup>
                               th
                              </sup>
                              CpG (m
                              <sub>
                               b,i
                              </sub>
                              ) between sample pairs and applying a minimum effect size of 0.2.
                             </p>
                             <p id="p-142">
                              To appreciate the effect of tumor purity on both inter- and intra-tumor-tumor differential methylation, we extracted the bulk and CAMDAC deconvolved tumor methylation profiles from the 20 lowest and highest purity samples in our cohort, ρ < 0.3 and ρ > 0.58, respectively. We randomly selected 1 million CpG loci from samples with the same or different purity categorical assignments both within and between patients and obtained tumor-tumor DMP calls from m
                              <sub>
                               b
                              </sub>
                              and CAMDAC m
                              <sub>
                               t
                              </sub>
                              .
                             </p>
                            </div>
                            <div class="section" id="sec-37">
                             <h3>
                              Quantifying allele-specific methylation
                             </h3>
                             <p id="p-143">
                              For all 37 normal samples in our cohort, we obtained allele-specific normal methylation rates where possible by phasing to nearby heterozygous SNPs. The normal methylation rate for the reference allele, m
                              <sub>
                               n,ref
                              </sub>
                              , is computed from the methylated (M
                              <sub>
                               n,ref
                              </sub>
                              ) and unmethylated (UM
                              <sub>
                               n,ref
                              </sub>
                              ) reads phased to the reference allele: m
                              <sub>
                               n,ref
                              </sub>
                              = M
                              <sub>
                               n,ref
                              </sub>
                              /(M
                              <sub>
                               n,ref
                              </sub>
                              + UM
                              <sub>
                               n,ref
                              </sub>
                              ). Vice versa, the alternate allele normal methylation rate, m
                              <sub>
                               n,alt
                              </sub>
                              , is calculated as: m
                              <sub>
                               n,alt
                              </sub>
                              = M
                              <sub>
                               n,alt
                              </sub>
                              /(M
                              <sub>
                               n,alt
                              </sub>
                              + UM
                              <sub>
                               n,alt
                              </sub>
                              ). We then compute the exact beta posterior HDI
                              <sup>
                               99
                              </sup>
                              for each allele.
                             </p>
                             <p id="p-144">
                              Next, we assess allele-specific methylation in tumor samples. As above, we calculate bulk tumor methylation rates at the reference and alternate alleles, m
                              <sub>
                               b,ref
                              </sub>
                              and m
                              <sub>
                               b,alt
                              </sub>
                              , respectively, directly from phased reads counts (un)methylated. We assign ASCAT.m clonal allele-specific copy numbers to each allele based on the BAF of the heterozygous locus used for phasing. Where multi-samples BAF phasing information is available, the segmented BAF value of the corresponding haplotype is used. If the BAF < 0.5, then major allele, n
                              <sub>
                               A
                              </sub>
                              , is the reference allele copy number, n
                              <sub>
                               ref
                              </sub>
                              = n
                              <sub>
                               A
                              </sub>
                              and the minor allele, n
                              <sub>
                               B
                              </sub>
                              , is the alternate allele, n
                              <sub>
                               alt
                              </sub>
                              = n
                              <sub>
                               B
                              </sub>
                              . If BAF > 0.5, n
                              <sub>
                               alt
                              </sub>
                              = n
                              <sub>
                               A
                              </sub>
                              and n
                              <sub>
                               ref
                              </sub>
                              = n
                              <sub>
                               B
                              </sub>
                              . Both the reference and alternate allele methylation rates are confounded by signal from normal contaminating cells and must be deconvolved. We modified CAMDAC
                              <a id="xref-disp-formula-2-9" class="xref-disp-formula" href="#disp-formula-2">
                               equations 2
                              </a>
                              and
                              <a id="xref-disp-formula-3-3" class="xref-disp-formula" href="#disp-formula-3">
                               3
                              </a>
                              for this purpose, where x is either the reference or alternate allele:
                              <span class="disp-formula" id="disp-formula-17">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-37.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-37.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <span class="disp-formula" id="disp-formula-18">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-38.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/graphic-38.gif"/>
                               </noscript>
                              </span>
                             </span>
                            </div>
                            <div class="section" id="sec-38">
                             <h3>
                              Genomic imprinting in normal samples
                             </h3>
                             <p id="p-145">
                              We collated allele-specific methylation rate estimates at confirmed human imprinting loci from the Geneimprint database (Status=“Imprinted”,
                              <a href="https://www.geneimprint.com">
                               https://www.geneimprint.com
                              </a>
                              ). In total, 106 imprinted gene loci (2.9 genes per patient, 11 unique genes) had at least 3 non-polymorphic CpGs phased to one or more heterozygous SNPs. On average, 8.4 CpGs were close enough to be phased. We tested these CpGs for allele-specific methylation signal, requiring at least 2 consecutive CpGs with non-overlapping phased methylation rate HDI
                              <sup>
                               99
                              </sup>
                              (i.e.
                              <span class="inline-formula" id="inline-formula-16">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-16.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-16.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ) and a minimum methylation rate difference between alleles greater than 0.5 (i.e. |m
                              <sub>
                               n,ref
                              </sub>
                              − m
                              <sub>
                               n,alt
                              </sub>
                              | > 0.5). Overall, we detect allele-specific methylation at 100 out of the 106 loci with phased methylation information (94.3%).
                             </p>
                            </div>
                            <div class="section" id="sec-39">
                             <h3>
                              Somatic alterations at known imprinted loci
                             </h3>
                             <p id="p-146">
                              We then evaluate allele-specific pure tumor methylation at those 100 normal imprinted loci, requiring phased tumor methylation information for 3 or more non-polymorphic CpGs for inclusion.
                             </p>
                             <p id="p-147">
                              As above, we classify tumor samples has having retained normal imprinting when at least 2 consecutive CpGs had |m
                              <sub>
                               t,ref
                              </sub>
                              − m
                              <sub>
                               t,alt
                              </sub>
                              | > 0.5. Tumor samples exhibiting loss of imprinting were divided into cases with and without loss of heterozygosity based on ASCAT.m allele-specific copy numbers and then further divided based on the methylation rate of the remaining allele(s). Methylated genes have mean methylation rates such that
                              <span class="inline-formula" id="inline-formula-17">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-17.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-17.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where the average phased tumor methylation rate, B
                              <sub>
                               t,i
                              </sub>
                              , is set to B
                              <sub>
                               t,i
                              </sub>
                              = m
                              <sub>
                               t,x,i
                              </sub>
                              where x is the remaining allele in cases with loss of heterozygosity and
                              <span class="inline-formula" id="inline-formula-18">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-18.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-18.gif"/>
                                </noscript>
                               </span>
                              </span>
                              in cases without. The average allele-specific normal methylation rate at imprinted loci, B
                              <sub>
                               n,i
                              </sub>
                              , is set to
                              <span class="inline-formula" id="inline-formula-19">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-19.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-19.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Vice versa, loci with loss of methylation have
                              <span class="inline-formula" id="inline-formula-20">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-20.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-20.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <p id="p-148">
                              Where distinct spatially separated genomic bins had phasing information for the same imprinted gene across tumor samples from the same patient, the most altered bin across tumor samples from the same patient was selected. For five gene loci, all but one tumor sample had phased methylation information at ≥ 3 CpGs. The imprinting status of the latter was annotated using at least one phaseable CpG and assignments were manually verified. Assignments were obtained for a total of 276 loci across genes and samples (85 genes across patients, 2.4 genes per patient, 10 unique genes).
                             </p>
                             <p id="p-149">
                              Overall, 100/276 loci retained allele-specific methylation in tumor samples (43 loci across patients, 25/43 clonal). Loss of heterozygosity resulted in gain of methylation at 43 loci (14 loci across patients, 9/14 clonal) and demethylation at 31 loci (13 loci across patients, 8/13 clonal). Loss of imprinting without loss of heterozygosity was observed at 56 methylated loci (21 loci across patients, 10/21 clonal) and 46 unmethylated loci (20 loci across patients, 10/20 clonal). These data are summarized in a gene-wise manner for 5/10 normal imprinted genes, chosen for having phased methylation information in at least 10 tumor samples.
                             </p>
                            </div>
                            <div class="section" id="sec-40">
                             <h3>
                              Allele-specific differentially methylated CpG
                             </h3>
                             <p id="p-150">
                              Next, we obtain allele-specific pure tumor methylation rates for all CpG loci with reads overlapping with a heterozygous SNP, subset to CpGs that are confidently unmethylated in the matched normal. We compute 99% methylation rate HDI for each the reference and alternate alleles. We also evaluate the power to call an allele-specific hypermethylation on either allele for each locus given coverage, copy number and an unmethylated normal state (sampled from confidently unmethylated m
                              <sub>
                               t
                              </sub>
                              values,
                              <span class="inline-formula" id="inline-formula-21">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-21.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-21.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ). CpGs were deemed allele-specifically methylated when (1) there was sufficient power at the loci, (2) the methylation rate 99% HDIs for each allele did not overlap and (3) the between allele methylation rate difference was greater than 0.25.
                             </p>
                             <p id="p-151">
                              For a given patient, we selected loci that were hypermethylated based on m
                              <sub>
                               t
                              </sub>
                              and/or allele-specifically methylated based on m
                              <sub>
                               t,x
                              </sub>
                              in one or more sample and powered in at least one other sample. Copy number states with less than 100 CpGs were filtered out to reduce noise. We then performed UMAP based on a matrix of the allele-specific methylation rates at these loci (R library,
                              <a href="https://github.com/tkonopka/umap">
                               https://github.com/tkonopka/umap
                              </a>
                              ). We required at least two tumor sample per patient and a minimum of 40 CpGs for clustering (32 / 38 patients pass this criteria). The UMAP output was fed through hierarchical clustering to identify epimutation clusters (n=20), and then clusters were merged based on mean m
                              <sub>
                               t
                              </sub>
                              and m
                              <sub>
                               t,x
                              </sub>
                              values. Clusters with mean between allele difference m
                              <sub>
                               t,1
                              </sub>
                              − m
                              <sub>
                               t,1
                              </sub>
                              < 0.125 were classed as bi-allelic. Of these, groups with m
                              <sub>
                               t
                              </sub>
                              > 0.85 ± 0.125 were classed as clonal bi-allelic. We refer to the mean methylation rate of clonal bi-allelic alterations as
                              <span class="inline-formula" id="inline-formula-22">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-22.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-22.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Of the remaining, only those where one allele was fully unmethylated (m
                              <sub>
                               t,i
                              </sub>
                              < 0.125) and the other was methylated on at least one copy (
                              <span class="inline-formula" id="inline-formula-23">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-23.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-23.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , where n
                              <sub>
                               j
                              </sub>
                              is the allele specific copy number of allele j) were classed as clonally allele-specifically methylated. Those where less than one copy was methylated were classed as subclonal allele-specific epimutations (
                              <span class="inline-formula" id="inline-formula-24">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-24.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/04/24/2020.11.03.366252/embed/inline-graphic-24.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ). Epimutations where both alleles were partially and differentially methylated were unclassified and warrant further investigation. These may represent dynamic loci with variable methylation.
                             </p>
                             <p id="p-152">
                              Excluding unclassified clusters, we readily obtain the relative number of (sub)clonal mono- and bi-allelic epimutations. The allele-specificity ratio is defined as the fraction of alterations that are mono-allelic and the clonality or intra-tumor heterogeneity score is the fraction of DMPs that are clonal. We fit a binomial generalized linear model (glm v4.0.5) to each of these 3 dependent variables: the fractions of allele-specific, clonal and clonal bi-allelic mutations based on histological subtype, tumor stage (IA and IB = “early”, “late” for all later stages), recurrence, SBS4 exposure (exposure > 0.35 = “high”, “low” otherwise), WGD status (majority vote) and the number of sequenced tumor samples (n ≤ 3 samples : “high”, low otherwise). We get the odds ratio for each covariate and the 95% confidence interval.
                             </p>
                            </div>
                            <div class="section" id="sec-41">
                             <h3>
                              Identifying tumor-normal differential methylated regions
                             </h3>
                             <p id="p-153">
                              First, CpGs are grouped into bins. CpGs which fall within 100bp of one another are grouped together. For each bin, we compute the number of consecutive DMPs with effect size 0.2 and p < 0.01 and the total number of DMPs. Genomic bins with 4 or more consecutive DMPs and at least 5 DMPs in total are deemed differentially methylated. Each bin is annotated for downstream analyses. Annotated gene features include CpG Islands, shores and shelves, exons, introns, 5’UTR, 3’UTR, promoters and enhancers. CpG islands, intragenic features and enhancers genomic coordinates are pulled from Ensembl via biomaRt. CpG island shores and shelves are respectively defined as the regions 0-2kb and 2kb-5kb either side of a CpG island. Intragenic annotations are simplified for each gene to the GENCODE basic transcript set. Each gene transcript promoter is defined as starting 2.5kb upstream and ending 250bp downstream of the transcription start site. For this work, we used CAMDAC with hg19 annotation set, but hg38 is also available. Note that any given CpG cluster can be associated to several features.
                             </p>
                             <p id="p-154">
                              To validate our DMR calls, we leverage SNV purified tumor methylation rates. First, we perform tumor-normal DMP calling following the same logic as in Eq(14), but this time computing the probability that P(m
                              <sub>
                               mut
                              </sub>
                              > m
                              <sub>
                               t
                              </sub>
                              ), P(m
                              <sub>
                               WT
                              </sub>
                              > m
                              <sub>
                               t
                              </sub>
                              ) and their complements. Due to the reduced power but increased accuracy of phased methylation estimates, we decreased the threshold for DMR calling to at least 2 consecutive DMPs and obtained tumor-normal DMP calls for each allele. To confirm that DMRs that were only detected on one allele were indeed allele-specific and not due to lack of power on one allele, we identified allele-specific differentially methylated regions (AS-DMRs). CpGs were deemed allele-specifically methylated when (1) there was sufficient power at the loci, (2) the methylation rate 99% HDIs for each allele did not overlap and (3) the between allele methylation rate difference was greater than 0.2. Genomic regions with 2 or more consecutive non-polymorphic CpGs with allele-specific methylation were deemed allele-specific. Combining per allele tumor-normal DMR and AS-DMR calls, we then classified DMRs. Bins which were not AS-DMR and which where differentially methylated on both the mutant and wild type alleles were confirmed to be present on both alleles where both alleles, only the mutant allele (in-cis) or only the wild type allele (in-trans). DMRs in regions with loss of the wild type allele (n
                              <sub>
                               EF
                              </sub>
                              < 0.5) are categorized separately as it is not possible to determine whether the DMR was in-cis, in-trans or on both alleles prior to the loss.
                             </p>
                            </div>
                            <div class="section" id="sec-42">
                             <h3>
                              Dimensional reduction of CAMDAC pure tumor methylation profiles
                             </h3>
                             <p id="p-155">
                              We compiled a list of all CpGs which fell within a promoter-associated tumor-normal DMR in at least one sample based on CAMDAC purified methylomes (totaling 8,570 gene promoters) and obtained mean methylation rates across CpGs for each of those genomic regions for each sample in this cohort (122 tumor and 37 normal lung samples). We applied UMAP dimensionality reduction with the R package umap (
                              <a href="https://github.com/tkonopka/">
                               https://github.com/tkonopka/
                              </a>
                              umap) to the resulting promoter methylation profile of each sample. We map histologies (LUAD, LUSC, normal), sex and patient IDs to UMAP coordinates. We repeated this analysis selecting promoter regions based on the bulk tumor-normal DMR calls (totaling 8,387 gene promoters) feeding the mean bulk tumor methylation rates into the UMAP.
                             </p>
                            </div>
                            <div class="section" id="sec-43">
                             <h3>
                              Phylogenetic analysis
                             </h3>
                             <p id="p-156">
                              To construct phylogenetic trees for each patient, we first selected CpGs hypermethylated in at least one sampled tumor region and binned methylation rates: “0” for rate ≤ 0.3 to indicate unmethylated, “1” for rate ≥ 0.6 to indicate hypermethylated and the ambiguous character “-” in all other cases to represent intermediate methylation. Additionally, we applied a CCF threshold to previously published SNV data for each patient (
                              <a id="xref-ref-45-13" class="xref-bibr" href="#ref-45">
                               Jamal-Hanjani et al. 2017
                              </a>
                              ): “0” for CCF ≤ 0.2, “1” for CCF > 0.2. Neighbor-joining trees were constructed based on binarized bulk and CAMDAC pure tumor methylation rates at DMPs and for SNVs using the pairwise hamming distance between samples with ape v5.4.1 (
                              <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">
                               Paradis et al., 2004
                              </a>
                              ). All trees were outgroup-rooted on the normal sample.
                             </p>
                             <p id="p-157">
                              We quantified the concordance between DMP and SNV trees using a score based on ancestral methylation states
                              <strong>
                               (Figure S12A)
                              </strong>
                              . Unlike common topology measures such as the Robinson-Foulds distance, this score incorporates (epi)mutational events. A perfect topology match between the SNV and DMP trees grants a score of 0, whereas scores increase above 0 with greater inconsistencies between ancestral methylation states given by the SNV tree and DMP tree topologies
                              <strong>
                               (Figure S12B)
                              </strong>
                              . To determine the significance of our observed DMP-SNV tree fit, we computed p-values by scoring each DMP tree using all possible branched rooted topologies in place of the patient’s SNV tree. For CRUK0062, our permutation is a random sample of up to n=10,395 possible topologies for computational feasibility. The resulting empirical p-value is interpreted as the probability of observing a DMP tree score given all possible SNV tree topologies – except for CRUK0062 where of subset of possible topologies is sampled – under the null hypothesis that there is no relationship between DMP and SNV tree topologies. This enables comparison between scores calculated using m
                              <sub>
                               b
                              </sub>
                              and CAMDAC m
                              <sub>
                               t
                              </sub>
                              .
                             </p>
                             <p id="p-158">
                              To compare DMP and SNV tree trunks, we calculated the relative trunk lengths for each pair taking the trunk length (from normal to the most recent common ancestor) divided by the total length from the normal to the furthest leaf. The mutational signature (
                              <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                               Alexandrov et al., 2013
                              </a>
                              ) deconvolution software deconstructSigs (
                              <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">
                               Rosenthal et al., 2016
                              </a>
                              ) was used to extract exposures for unique mutations given SBS1, SBS2, SBS4, SBS5 and SBS13 signature profiles (COSMIC v2). A minimum of 50 mutations was required for sample inclusion.
                             </p>
                            </div>
                            <div class="section" id="sec-44">
                             <h3>
                              CAMDAC with tissue-matched normal
                             </h3>
                             <p id="p-159">
                              We were unable to sequence a patient-matched tumor-adjacent normal sample for CRUK0047. Leveraging this cohort’s 37 normal lung samples, we built a healthy lung tissue reference RRBS profile using the median normal SNP coverage and methylation counts which we fed into CAMDAC alongside tumor sample CRUK0047-R2. The ploidy and purity values (ψ
                              <sub>
                               RRBS
                              </sub>
                              = 2.15, ρ
                              <sub>
                               RRBS
                              </sub>
                              = 0.40) obtained were in agreement with the matched exome data (ψ
                              <sub>
                               WES
                              </sub>
                              = 2.29, ρ
                              <sub>
                               WES
                              </sub>
                              = 0.31). UMAP performed on CAMDAC purified tumor methylomes found CRUK0047-R2 in the same cluster as other LUAD samples. We conclude that genome-wide RRBS relative SNP coverage and methylation levels in normal lung are consistent between different patients and that tissue-matched normal RRBS data is a suitable alternative reference for CAMDAC when patient-matched data is unavailable.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-45">
                            <h2 class="">
                             TRACERX CONSORTIUM
                            </h2>
                            <p id="p-160">
                             Charles Swanton
                             <sup>
                              1,2
                             </sup>
                             , Mariam Jamal-Hanjani
                             <sup>
                              1,7
                             </sup>
                             , Nicholas McGranahan
                             <sup>
                              1,9
                             </sup>
                             , Allan Hackshaw
                             <sup>
                              10
                             </sup>
                             , Yin Wu
                             <sup>
                              1,2,3,4
                             </sup>
                             , Dhruva Biswas
                             <sup>
                              1,2,5
                             </sup>
                             , Mihaela Angelova
                             <sup>
                              2
                             </sup>
                             , Stefan Boeing
                             <sup>
                              6
                             </sup>
                             , Takahiro Karasaki
                             <sup>
                              1,2
                             </sup>
                             , Selvaraju Veeriah
                             <sup>
                              1
                             </sup>
                             , Sonya Hessey
                             <sup>
                              1,7
                             </sup>
                             , James Reading
                             <sup>
                              1,8
                             </sup>
                             , Maise Al-Bakir
                             <sup>
                              2
                             </sup>
                             , Sergio A. Quezada
                             <sup>
                              1,8
                             </sup>
                             , Nicolai J Birkbak
                             <sup>
                              11
                             </sup>
                             Gillian Price
                             <sup>
                              12
                             </sup>
                             , Mohammed Khalil
                             <sup>
                              12
                             </sup>
                             , Keith Kerr
                             <sup>
                              12
                             </sup>
                             , Shirley Richardson
                             <sup>
                              12
                             </sup>
                             , Heather Cheyne
                             <sup>
                              12
                             </sup>
                             , Tracey Cruickshank
                             <sup>
                              12
                             </sup>
                             , Gareth A Wilson
                             <sup>
                              13
                             </sup>
                             , Rachel Rosenthal
                             <sup>
                              13
                             </sup>
                             , Hugo Aerts
                             <sup>
                              14, 15
                             </sup>
                             , Madeleine Hewish
                             <sup>
                              16
                             </sup>
                             , Girija Anand
                             <sup>
                              17
                             </sup>
                             , Sajid Khan
                             <sup>
                              17
                             </sup>
                             , Kelvin Lau
                             <sup>
                              18
                             </sup>
                             , Michael Sheaff
                             <sup>
                              18
                             </sup>
                             , Peter Schmid
                             <sup>
                              18
                             </sup>
                             , Louise Lim, John Conibear
                             <sup>
                              18
                             </sup>
                             , Roland Schwarz
                             <sup>
                              19
                             </sup>
                             , Tom L Kaufmann
                             <sup>
                              19
                             </sup>
                             , Matthew Huska
                             <sup>
                              19
                             </sup>
                             , Jacqui Shaw
                             <sup>
                              20
                             </sup>
                             , Joan Riley
                             <sup>
                              20
                             </sup>
                             , Lindsay Primrose
                             <sup>
                              20
                             </sup>
                             , Dean Fennell
                             <sup>
                              20,21
                             </sup>
                             , Allan Hackshaw
                             <sup>
                              22
                             </sup>
                             , Yenting Ngai
                             <sup>
                              22
                             </sup>
                             , Abigail Sharp
                             <sup>
                              22
                             </sup>
                             , Oliver Pressey
                             <sup>
                              22
                             </sup>
                             , Sean Smith
                             <sup>
                              22
                             </sup>
                             , Nicole Gower
                             <sup>
                              22
                             </sup>
                             , Harjot Kaur Dhanda
                             <sup>
                              22
                             </sup>
                             , Kitty Chan
                             <sup>
                              22
                             </sup>
                             , Sonal Chakraborty
                             <sup>
                              22
                             </sup>
                             , Kevin Litchfield
                             <sup>
                              1
                             </sup>
                             , Krupa Thakkar
                             <sup>
                              1
                             </sup>
                             , Jonathan Tugwood
                             <sup>
                              23
                             </sup>
                             , Alexandra Clipson
                             <sup>
                              23
                             </sup>
                             , Caroline Dive
                             <sup>
                              23,24
                             </sup>
                             , Dominic Rothwell
                             <sup>
                              23,24
                             </sup>
                             , Alastair Kerr
                             <sup>
                              23,24
                             </sup>
                             , Elaine Kilgour
                             <sup>
                              23,24
                             </sup>
                             , Fiona Morgan
                             <sup>
                              25
                             </sup>
                             , Malgorzata Kornaszewska
                             <sup>
                              25
                             </sup>
                             , Richard Attanoos
                             <sup>
                              25
                             </sup>
                             , Helen Davies
                             <sup>
                              25
                             </sup>
                             , Katie Baker
                             <sup>
                              26
                             </sup>
                             , Mathew Carter
                             <sup>
                              26
                             </sup>
                             , Colin R Lindsay
                             <sup>
                              26
                             </sup>
                             , Fabio Gomes
                             <sup>
                              26
                             </sup>
                             , Fiona Blackhall
                             <sup>
                              24,26
                             </sup>
                             , Lynsey Priest
                             <sup>
                              24,26
                             </sup>
                             , Matthew G Krebs
                             <sup>
                              24,26
                             </sup>
                             , Anshuman Chaturvedi
                             <sup>
                              24,26
                             </sup>
                             , Pedro Oliveira
                             <sup>
                              24,26
                             </sup>
                             , Zoltan Szallasi
                             <sup>
                              27
                             </sup>
                             , Gary Royle
                             <sup>
                              28
                             </sup>
                             , Catarina Veiga
                             <sup>
                              28
                             </sup>
                             , Marcin Skrzypski
                             <sup>
                              29
                             </sup>
                             , Roberto Salgado
                             <sup>
                              30
                             </sup>
                             , Miklos Diossy
                             <sup>
                              31
                             </sup>
                             , Alan Kirk
                             <sup>
                              32
                             </sup>
                             , Mo Asif
                             <sup>
                              32
                             </sup>
                             , John Butler
                             <sup>
                              32
                             </sup>
                             , Rocco Bilancia
                             <sup>
                              32
                             </sup>
                             , Nikos Kostoulas
                             <sup>
                              32
                             </sup>
                             , Mathew Thomas
                             <sup>
                              32
                             </sup>
                             , Mairead MacKenzie
                             <sup>
                              33
                             </sup>
                             , Maggie Wilcox
                             <sup>
                              33
                             </sup>
                             , Apostolos Nakas
                             <sup>
                              21
                             </sup>
                             , Sridhar Rathinam
                             <sup>
                              21
                             </sup>
                             , Rebecca Boyles
                             <sup>
                              21
                             </sup>
                             , Mohamad Tufail
                             <sup>
                              21
                             </sup>
                             , Amrita Bajaj
                             <sup>
                              21
                             </sup>
                             , Keng Ang
                             <sup>
                              21
                             </sup>
                             , Mohammed Fiyaz Chowdhry
                             <sup>
                              21
                             </sup>
                             , Michael Shackcloth
                             <sup>
                              34
                             </sup>
                             , Julius Asante-Siaw
                             <sup>
                              34
                             </sup>
                             , Angela Leek
                             <sup>
                              35
                             </sup>
                             , Nicola Totten
                             <sup>
                              35
                             </sup>
                             , Jack Davies Hodgkinson
                             <sup>
                              35
                             </sup>
                             , Peter Van Loo
                             <sup>
                              36
                             </sup>
                             , William Monteiro
                             <sup>
                              37
                             </sup>
                             , Hilary Marshal
                             <sup>
                              37
                             </sup>
                             , Kevin G Blyth
                             <sup>
                              38
                             </sup>
                             , Craig Dick
                             <sup>
                              38
                             </sup>
                             , Charles Fekete
                             <sup>
                              15
                             </sup>
                             , Eric Lim
                             <sup>
                              39
                             </sup>
                             , Paulo De Sousa
                             <sup>
                              39
                             </sup>
                             , Simon Jordan
                             <sup>
                              39
                             </sup>
                             , Alexandra Rice
                             <sup>
                              39
                             </sup>
                             , Hilgardt Raubenheimer
                             <sup>
                              39
                             </sup>
                             , Harshil Bhayani
                             <sup>
                              39
                             </sup>
                             , Morag Hamilton
                             <sup>
                              39
                             </sup>
                             , Lyn Ambrose
                             <sup>
                              39
                             </sup>
                             , Anand Devaraj
                             <sup>
                              39
                             </sup>
                             , Hemangi Chavan
                             <sup>
                              39
                             </sup>
                             , Sofina Begum
                             <sup>
                              39
                             </sup>
                             , Silviu I Buderi
                             <sup>
                              39
                             </sup>
                             , Daniel Kaniu
                             <sup>
                              39
                             </sup>
                             , Mpho Malima
                             <sup>
                              39
                             </sup>
                             , Sarah Booth
                             <sup>
                              39
                             </sup>
                             , Andrew G Nicholson
                             <sup>
                              39
                             </sup>
                             , Nadia Fernandes
                             <sup>
                              39
                             </sup>
                             , Pratibha Shah
                             <sup>
                              39
                             </sup>
                             , Chiara Proli
                             <sup>
                              39
                             </sup>
                             , John Gosney
                             <sup>
                              40
                             </sup>
                             , Sarah Danson
                             <sup>
                              41
                             </sup>
                             , Jonathan Bury
                             <sup>
                              41
                             </sup>
                             , John Edwards
                             <sup>
                              41
                             </sup>
                             , Jennifer Hill
                             <sup>
                              41
                             </sup>
                             , Sue Matthews
                             <sup>
                              41
                             </sup>
                             , Yota Kitsanta
                             <sup>
                              41
                             </sup>
                             , Jagan Rao
                             <sup>
                              41
                             </sup>
                             , Sara Tenconi
                             <sup>
                              41
                             </sup>
                             , Laura Socci
                             <sup>
                              41
                             </sup>
                             , Kim Suvarna
                             <sup>
                              41
                             </sup>
                             , Faith Kibutu
                             <sup>
                              41
                             </sup>
                             , Patricia Fisher
                             <sup>
                              41
                             </sup>
                             , Robin Young
                             <sup>
                              41
                             </sup>
                             , Joann Barker
                             <sup>
                              41
                             </sup>
                             , Fiona Taylor
                             <sup>
                              41
                             </sup>
                             , Kirsty Lloyd
                             <sup>
                              41
                             </sup>
                             , Jason Lester
                             <sup>
                              42
                             </sup>
                             , Mickael Escudero
                             <sup>
                              43
                             </sup>
                             , Aengus Stewart
                             <sup>
                              43
                             </sup>
                             , Andrew Rowan
                             <sup>
                              43
                             </sup>
                             , Jacki Goldman
                             <sup>
                              43
                             </sup>
                             , Richard Kevin Stone
                             <sup>
                              43
                             </sup>
                             , Tamara Denner
                             <sup>
                              43
                             </sup>
                             , Emma Nye
                             <sup>
                              43
                             </sup>
                             , Maria Greco
                             <sup>
                              43
                             </sup>
                             , Jerome Nicod
                             <sup>
                              43
                             </sup>
                             , Clare Puttick
                             <sup>
                              43
                             </sup>
                             , Katey Enfield
                             <sup>
                              43
                             </sup>
                             , Emma Colliver
                             <sup>
                              43
                             </sup>
                             , Alastair Magness
                             <sup>
                              43
                             </sup>
                             , Chris Bailey
                             <sup>
                              43
                             </sup>
                             , Krijn Dijkstra
                             <sup>
                              43
                             </sup>
                             , Vittorio Barbè
                             <sup>
                              43
                             </sup>
                             , Roberto Vendramin
                             <sup>
                              43
                             </sup>
                             , Judit Kisistok
                             <sup>
                              43
                             </sup>
                             , Mateo Sokac
                             <sup>
                              43
                             </sup>
                             , Jonas Demeulemeester
                             <sup>
                              43
                             </sup>
                             , Elizabeth Larose Cadieux
                             <sup>
                              43
                             </sup>
                             , Carla Castignani
                             <sup>
                              43
                             </sup>
                             , Hongchang Fu
                             <sup>
                              43
                             </sup>
                             , Kristiana Grigoriadis
                             <sup>
                              43
                             </sup>
                             , Claudia Lee
                             <sup>
                              43,44
                             </sup>
                             , Foteini Athanasopoulou
                             <sup>
                              1,43
                             </sup>
                             , Crispin Hiley
                             <sup>
                              1,43
                             </sup>
                             , Lily Robinson
                             <sup>
                              45
                             </sup>
                             , Tracey Horey
                             <sup>
                              45
                             </sup>
                             , Peter Russell
                             <sup>
                              45
                             </sup>
                             , Dionysis Papadatos-Pastos
                             <sup>
                              45
                             </sup>
                             , Sara Lock
                             <sup>
                              46
                             </sup>
                             , Kayleigh Gilbert
                             <sup>
                              46
                             </sup>
                             , Kayalvizhi Selvaraju
                             <sup>
                              47
                             </sup>
                             , Paul Ashford
                             <sup>
                              47
                             </sup>
                             , Oriol Pich
                             <sup>
                              47
                             </sup>
                             , Thomas B K Watkins
                             <sup>
                              47
                             </sup>
                             , Sophia Ward
                             <sup>
                              47
                             </sup>
                             , Emilia Lim
                             <sup>
                              47
                             </sup>
                             , Alexander M Frankell
                             <sup>
                              47
                             </sup>
                             , Christopher Abbosh
                             <sup>
                              47
                             </sup>
                             , Robert E Hynds
                             <sup>
                              47
                             </sup>
                             , Mariana Werner Sunderland
                             <sup>
                              47
                             </sup>
                             , Karl Peggs
                             <sup>
                              47
                             </sup>
                             , Teresa Marafioti
                             <sup>
                              47
                             </sup>
                             , John A Hartley
                             <sup>
                              47
                             </sup>
                             , Helen Lowe
                             <sup>
                              47
                             </sup>
                             , Leah Ensell
                             <sup>
                              47
                             </sup>
                             , Victoria Spanswick
                             <sup>
                              47
                             </sup>
                             , Angeliki Karamani
                             <sup>
                              47
                             </sup>
                             , David Moore
                             <sup>
                              47
                             </sup>
                             , Stephan Beck
                             <sup>
                              47
                             </sup>
                             , Olga Chervova
                             <sup>
                              47
                             </sup>
                             , Miljana Tanic
                             <sup>
                              47
                             </sup>
                             , Ariana Huebner
                             <sup>
                              47
                             </sup>
                             , Michelle Dietzen
                             <sup>
                              47
                             </sup>
                             , James RM Black
                             <sup>
                              47
                             </sup>
                             , Carlos Martinez Ruiz
                             <sup>
                              47
                             </sup>
                             , Robert Bentham
                             <sup>
                              47
                             </sup>
                             , Cristina Naceur-Lombardelli
                             <sup>
                              47
                             </sup>
                             , Haoran Zhai
                             <sup>
                              47
                             </sup>
                             , Nnennaya Kanu
                             <sup>
                              47
                             </sup>
                             , Francisco Gimeno-Valiente
                             <sup>
                              47
                             </sup>
                             , Supreet Kaur Bola
                             <sup>
                              47
                             </sup>
                             , Ignacio Garcia Matos
                             <sup>
                              47
                             </sup>
                             , Mansi Shah
                             <sup>
                              47
                             </sup>
                             , Felip Galvez Cancino
                             <sup>
                              47
                             </sup>
                             , Despoina Karagianni
                             <sup>
                              47
                             </sup>
                             , Maryam Razaq
                             <sup>
                              47
                             </sup>
                             , Mita Akther
                             <sup>
                              47
                             </sup>
                             , Diana Johnson
                             <sup>
                              47
                             </sup>
                             , Joanne Laycock
                             <sup>
                              47
                             </sup>
                             , Elena Hoxha
                             <sup>
                              47
                             </sup>
                             , Benny Chain
                             <sup>
                              47
                             </sup>
                             , David R Pearce
                             <sup>
                              47
                             </sup>
                             , Kezhong Chen
                             <sup>
                              47
                             </sup>
                             , Javier Herrero
                             <sup>
                              47
                             </sup>
                             , Fleur Monk
                             <sup>
                              47
                             </sup>
                             , Simone Zaccaria
                             <sup>
                              47
                             </sup>
                             , Neil Magno
                             <sup>
                              47
                             </sup>
                             , Paulina Prymas
                             <sup>
                              47
                             </sup>
                             , Antonia Toncheva
                             <sup>
                              47
                             </sup>
                             , Monica Sivakumar
                             <sup>
                              47
                             </sup>
                             , Olivia Lucas
                             <sup>
                              47,48
                             </sup>
                             , Mark S. Hill
                             <sup>
                              43,47
                             </sup>
                             , Othman Al-Sawaf
                             <sup>
                              47
                             </sup>
                             , Seng Ung (Anakin)
                             <sup>
                              48
                             </sup>
                             , Sam Gamble
                             <sup>
                              48
                             </sup>
                             , Sophia Wong
                             <sup>
                              48
                             </sup>
                             , David Lawrence
                             <sup>
                              48
                             </sup>
                             , Martin Hayward
                             <sup>
                              48
                             </sup>
                             , Nikolaos Panagiotopoulos
                             <sup>
                              48
                             </sup>
                             , Robert George
                             <sup>
                              48
                             </sup>
                             , Davide Patrini
                             <sup>
                              48
                             </sup>
                             , Mary Falzon
                             <sup>
                              48
                             </sup>
                             , Elaine Borg
                             <sup>
                              48
                             </sup>
                             , Reena Khiroya
                             <sup>
                              48
                             </sup>
                             , Asia Ahmed
                             <sup>
                              48
                             </sup>
                             , Magali Taylor
                             <sup>
                              48
                             </sup>
                             , Junaid Choudhary
                             <sup>
                              48
                             </sup>
                             , Sam M Janes
                             <sup>
                              48
                             </sup>
                             , Martin Forster
                             <sup>
                              48
                             </sup>
                             , Tanya Ahmad
                             <sup>
                              48
                             </sup>
                             , Siow Ming Lee
                             <sup>
                              48
                             </sup>
                             , Neal Navani
                             <sup>
                              48
                             </sup>
                             , Marco Scarci
                             <sup>
                              48
                             </sup>
                             , Pat Gorman
                             <sup>
                              48
                             </sup>
                             , Elisa Bertoja
                             <sup>
                              48
                             </sup>
                             , Robert CM Stephens
                             <sup>
                              48
                             </sup>
                             , Emilie Martinoni Hoogenboom
                             <sup>
                              48
                             </sup>
                             , James W Holding
                             <sup>
                              48
                             </sup>
                             , Steve Bandula
                             <sup>
                              48
                             </sup>
                             , Ricky Thakrar
                             <sup>
                              48
                             </sup>
                             , James Wilson
                             <sup>
                              48
                             </sup>
                             , Mansi Shah
                             <sup>
                              48
                             </sup>
                             , Marcos
                             <sup>
                              48
                             </sup>
                             , Vasquez Duran
                             <sup>
                              48
                             </sup>
                             , Maria Litovchenko
                             <sup>
                              48
                             </sup>
                             , Sharon Vanloo
                             <sup>
                              48
                             </sup>
                             , Piotr Pawlik
                             <sup>
                              48
                             </sup>
                             , Kerstin Thol
                             <sup>
                              48
                             </sup>
                             , Babu Naidu
                             <sup>
                              49
                             </sup>
                             , Gerald Langman
                             <sup>
                              49
                             </sup>
                             , Hollie Bancroft
                             <sup>
                              49
                             </sup>
                             , Salma Kadiri
                             <sup>
                              49
                             </sup>
                             , Gary Middleton
                             <sup>
                              49
                             </sup>
                             , Madava Djearaman
                             <sup>
                              49
                             </sup>
                             , Aya Osman
                             <sup>
                              49
                             </sup>
                             , Helen Shackleford
                             <sup>
                              49
                             </sup>
                             , Akshay Patel
                             <sup>
                              49
                             </sup>
                             , Christian Ottensmeier
                             <sup>
                              50
                             </sup>
                             , Serena Chee
                             <sup>
                              50
                             </sup>
                             , Aiman Alzetani
                             <sup>
                              50
                             </sup>
                             , Judith Cave
                             <sup>
                              50
                             </sup>
                             , Lydia Scarlett
                             <sup>
                              50
                             </sup>
                             , Jennifer Richards
                             <sup>
                              50
                             </sup>
                             , Papawadee Ingram
                             <sup>
                              50
                             </sup>
                             , Emily Shaw
                             <sup>
                              50
                             </sup>
                             , John Le Quesne
                             <sup>
                              51
                             </sup>
                             , Alan Dawson
                             <sup>
                              52
                             </sup>
                             , Domenic Marrone
                             <sup>
                              52
                             </sup>
                             , Sean Dulloo
                             <sup>
                              52
                             </sup>
                             , Claire Wilson
                             <sup>
                              52
                             </sup>
                             , Yvonne Summers
                             <sup>
                              53
                             </sup>
                             , Raffaele Califano
                             <sup>
                              53
                             </sup>
                             , Rajesh Shah
                             <sup>
                              53
                             </sup>
                             , Piotr Krysiak
                             <sup>
                              53
                             </sup>
                             , Kendadai Rammohan
                             <sup>
                              53
                             </sup>
                             , Eustace Fontaine
                             <sup>
                              53
                             </sup>
                             , Richard Booton
                             <sup>
                              53
                             </sup>
                             , Matthew Evison
                             <sup>
                              53
                             </sup>
                             , Stuart Moss
                             <sup>
                              53
                             </sup>
                             , Juliette Novasio
                             <sup>
                              53
                             </sup>
                             , Leena Joseph
                             <sup>
                              53
                             </sup>
                             , Paul Bishop
                             <sup>
                              53
                             </sup>
                             , Helen Doran
                             <sup>
                              53
                             </sup>
                             , Felice Granato
                             <sup>
                              53
                             </sup>
                             , Vijay Joshi
                             <sup>
                              53
                             </sup>
                             , Elaine Smith
                             <sup>
                              53
                             </sup>
                             , Angeles Montero
                             <sup>
                              53
                             </sup>
                             , Phil Crosbie
                             <sup>
                              53,54
                             </sup>
                            </p>
                            <p id="p-161">
                             <sup>
                              1
                             </sup>
                             Cancer Research UK Lung Cancer Centre of Excellence, University College London Cancer Institute, London, UK
                            </p>
                            <p id="p-162">
                             <sup>
                              2
                             </sup>
                             Cancer Evolution and Genome Instability Laboratory, The Francis Crick Institute, London, UK
                            </p>
                            <p id="p-163">
                             <sup>
                              3
                             </sup>
                             Peter Gorer Department of Immunobiology, School of Immunology & Microbial Sciences, King’s College London, London, UK
                            </p>
                            <p id="p-164">
                             <sup>
                              4
                             </sup>
                             Immunosurveillance Laboratory, The Francis Crick Institute, London, UK
                            </p>
                            <p id="p-165">
                             <sup>
                              5
                             </sup>
                             Bill Lyons Informatics Centre, University College London Cancer Institute, London, UK
                            </p>
                            <p id="p-166">
                             <sup>
                              6
                             </sup>
                             Bioinformatics & Biostatistics and Software Development & Machine Learning Team, The Francis Crick Institute, London, UK
                            </p>
                            <p id="p-167">
                             <sup>
                              7
                             </sup>
                             Cancer Metastasis Lab, University College London Cancer Institute, London, UK
                            </p>
                            <p id="p-168">
                             <sup>
                              8
                             </sup>
                             Cancer Immunology Unit, Research Department of Haematology, University College London Cancer Institute, London, UK
                            </p>
                            <p id="p-169">
                             <sup>
                              9
                             </sup>
                             Cancer Genome Evolution Research Group, University College London Cancer Institute, London, UK
                            </p>
                            <p id="p-170">
                             <sup>
                              10
                             </sup>
                             Cancer Research UK & University College London Cancer Trials Centre, University College London, London, UK
                            </p>
                            <p id="p-171">
                             <sup>
                              11
                             </sup>
                             Aarhus University, Denmark
                            </p>
                            <p id="p-172">
                             <sup>
                              12
                             </sup>
                             Aberdeen Royal Infirmary, Aberdeen, UK
                            </p>
                            <p id="p-173">
                             <sup>
                              13
                             </sup>
                             Achilles Therapeutics UK Limited, London UK
                            </p>
                            <p id="p-174">
                             <sup>
                              14
                             </sup>
                             Artificial Intelligence in Medicine (AIM) Program, Harvard Medical School, Boston, USA
                            </p>
                            <p id="p-175">
                             <sup>
                              15
                             </sup>
                             Radiology and Nuclear Medicine, CARIM & GROW, Maastricht University, Maastricht, The Netherlands.
                            </p>
                            <p id="p-176">
                             <sup>
                              16
                             </sup>
                             Ashford and St. Peter’s Hospitals NHS Foundation Trust, Chertsey, UK.
                            </p>
                            <p id="p-177">
                             <sup>
                              17
                             </sup>
                             Barnet & Chase Farm Hospitals, Barnet, UK
                            </p>
                            <p id="p-178">
                             <sup>
                              18
                             </sup>
                             Barts Health NHS Trust, London, UK
                            </p>
                            <p id="p-179">
                             <sup>
                              19
                             </sup>
                             Berlin Institute for Medical Systems Biology, Max Delbrück Center for Molecular Medicine in the Helmholtz Association (MDC), Berlin, Germany
                            </p>
                            <p id="p-180">
                             <sup>
                              20
                             </sup>
                             Cancer Research Centre, University of Leicester, Leicester, UK
                            </p>
                            <p id="p-181">
                             <sup>
                              21
                             </sup>
                             Leicester University Hospitals, Leicester, UK
                            </p>
                            <p id="p-182">
                             <sup>
                              22
                             </sup>
                             Cancer Research UK & UCL Cancer Trials Centre, London, UK
                            </p>
                            <p id="p-183">
                             <sup>
                              23
                             </sup>
                             Cancer Research UK Manchester Institute, University of Manchester, Manchester, UK
                            </p>
                            <p id="p-184">
                             <sup>
                              24
                             </sup>
                             Cancer Research UK Lung Cancer Centre of Excellence, University of Manchester, Manchester, UK
                            </p>
                            <p id="p-185">
                             <sup>
                              25
                             </sup>
                             Cardiff & Vale University Health Board, Cardiff, UK
                            </p>
                            <p id="p-186">
                             <sup>
                              26
                             </sup>
                             Christie NHS Foundation Trust, Manchester, UK
                            </p>
                            <p id="p-187">
                             <sup>
                              27
                             </sup>
                             Danish Cancer Society Research Centre, Denmark
                            </p>
                            <p id="p-188">
                             <sup>
                              28
                             </sup>
                             Department of Medical Physics and Bioengineering, University College London Cancer Institute, London, UK
                            </p>
                            <p id="p-189">
                             <sup>
                              29
                             </sup>
                             Department of Oncology and Radiotherapy, Medical University of Gdańsk, Gdańsk, Poland
                            </p>
                            <p id="p-190">
                             <sup>
                              30
                             </sup>
                             Department of Pathology, GZA-ZNA Antwerp, Belgium
                            </p>
                            <p id="p-191">
                             <sup>
                              31
                             </sup>
                             Department of Physics of Complex Systems, ELTE Eötvös Loránd University, Budapest, Hungary
                            </p>
                            <p id="p-192">
                             <sup>
                              32
                             </sup>
                             Golden Jubilee National Hospital, Clydebank, UK
                            </p>
                            <p id="p-193">
                             <sup>
                              33
                             </sup>
                             Independent Cancer Patients Voice, London, UK
                            </p>
                            <p id="p-194">
                             <sup>
                              34
                             </sup>
                             Liverpool Heart and Chest Hospital NHS Foundation Trust, Liverpool, UK
                            </p>
                            <p id="p-195">
                             <sup>
                              35
                             </sup>
                             Manchester Cancer Research Centre Biobank, Manchester, UK
                            </p>
                            <p id="p-196">
                             <sup>
                              36
                             </sup>
                             MD Anderson Cancer Center, Houston, USA
                            </p>
                            <p id="p-197">
                             <sup>
                              37
                             </sup>
                             National Institute for Health Research Leicester Respiratory Biomedical Research Unit, Leicester, UK
                            </p>
                            <p id="p-198">
                             <sup>
                              38
                             </sup>
                             NHS Greater Glasgow and Clyde, Glasgow, UK
                            </p>
                            <p id="p-199">
                             <sup>
                              39
                             </sup>
                             Royal Brompton and Harefield NHS Foundation Trust, London, UK
                            </p>
                            <p id="p-200">
                             <sup>
                              40
                             </sup>
                             Royal Liverpool University Hospital, Liverpool, UK
                            </p>
                            <p id="p-201">
                             <sup>
                              41
                             </sup>
                             Sheffield Teaching Hospitals NHS Foundation Trust, Sheffield, London, UK
                            </p>
                            <p id="p-202">
                             <sup>
                              42
                             </sup>
                             Swansea Bay University Health Board, Swansea, UK
                            </p>
                            <p id="p-203">
                             <sup>
                              43
                             </sup>
                             The Francis Crick Institute, London, UK
                            </p>
                            <p id="p-204">
                             <sup>
                              44
                             </sup>
                             University College London Medical School, London, UK
                            </p>
                            <p id="p-205">
                             <sup>
                              45
                             </sup>
                             The Princess Alexandra Hospital NHS Trust, Harlow, UK
                            </p>
                            <p id="p-206">
                             <sup>
                              46
                             </sup>
                             The Whittington Hospital NHS Trust, London, UK
                            </p>
                            <p id="p-207">
                             <sup>
                              47
                             </sup>
                             University College London Cancer Institute, London, UK
                            </p>
                            <p id="p-208">
                             <sup>
                              48
                             </sup>
                             University College London Hospitals, London, UK
                            </p>
                            <p id="p-209">
                             <sup>
                              49
                             </sup>
                             University Hospital Birmingham NHS Foundation Trust, Birmingham, UK
                            </p>
                            <p id="p-210">
                             <sup>
                              50
                             </sup>
                             University Hospital Southampton NHS Foundation Trust, Southampton, UK
                            </p>
                            <p id="p-211">
                             <sup>
                              51
                             </sup>
                             University of Glasgow, Institute of Cancer Sciences, Glasgow, UK
                            </p>
                            <p id="p-212">
                             <sup>
                              52
                             </sup>
                             University of Leicester, Leicester, UK
                            </p>
                            <p id="p-213">
                             <sup>
                              53
                             </sup>
                             Wythenshawe Hospital, Manchester University NHS Foundation Trust, Manchester, UK
                            </p>
                            <p id="p-214">
                             <sup>
                              54
                             </sup>
                             Division of Infection, Immunity and Respiratory Medicine, University of Manchester, Manchester, UK
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             ACKNOWLEDGMENTS
                            </h2>
                            <p id="p-71">
                             This work was supported by the Francis Crick Institute, which receives its core funding from Cancer Research UK (FC001202), the UK Medical Research Council (FC001202), and the Wellcome Trust (FC001202). For the purpose of open access, the authors have applied a CC BY public copyright licence to any author accepted manuscript version arising from this submission. We acknowledge technical support from the CRUK–UCL Centre-funded Genomics and Genome Engineering Core Facility of the UCL Cancer Institute and grant support from the NIHR BRC (BRC275/CN/SB/101330) and the Wellcome Trust (218274/Z/19/Z). J.D. is a postdoctoral fellow supported by the European Union’s Horizon 2020 research and innovation program (Marie Skłodowska-Curie Grant Agreement No. 703594-DECODE) and the Research Foundation – Flanders (FWO; 12J6921N). M.T. was supported by the People Programme, Marie Skłodowska-Curie Actions (FP7/2007-2013/WHRI-ACADEMY-608765) and the Danish Council for Strategic Research (1309-00006B). A.F. received funding from Prostate Cancer UK (ma-tr15-009), Biotechnology and Biological Sciences Research Council (BB/R009295/1) and the Medical Research Council (MR/M025411/1). C.S. is Royal Society Napier Research Professor (RP150154). His work is supported by the Francis Crick Institute, which receives its core funding from Cancer Research UK (FC001169), the UK Medical Research Council (FC001169), and the Wellcome Trust (FC001169). C.S. is funded by Cancer Research UK (TRACERx, C11496/A17786; Cancer Research UK Lung Cancer Centre of Excellence C11496/A30025), the Rosetrees Trust, Butterfield and Stoneygate Trusts, NovoNordisk Foundation (ID16584), a Royal Society Research Professorship Enhancement Award (RP/EA/180007), the National Institute for Health Research (NIHR) Biomedical Research Centre at University College London Hospitals, the Breast Cancer Research Foundation (BCRF 20-157), the CRUK-UCL Centre, Experimental Cancer Medicine Centre. His research is supported by a Stand Up To Cancer-LUNGevity-American Lung Association Lung Cancer Interception Dream Team Translational Research Grant (SU2C-AACR-DT23-17). C.S. also receives funding from the European Research Council (FP7-THESEUS-617844, FP7-PloidyNet 607722, PROTEUS 835297 and Chromavision 665233). M.J.-H. is a CRUK Career Establishment Awardee and has received funding from CRUK, IASLC International Lung Cancer Foundation, Lung Cancer Research Foundation, Rosetrees Trust, UKI NETs, NIHR UCLH Biomedical Research Centre. P.V.L. is a Winton Group Leader in recognition of the Winton Charitable Foundation’s support towards the establishment of The Francis Crick Institute. P.V.L. is a CPRIT Scholar in Cancer Research and acknowledges CPRIT grant support (RR210006).
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-others" id="fn-1">
                              <p id="p-1">
                               <a class="rev-xref" href="#xref-fn-1-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                13
                               </span>
                               A list of members of the TRACERx Consortium can be found at the end of the manuscript.
                              </p>
                             </li>
                             <li class="fn-others" id="fn-3">
                              <p id="p-3">
                               <a class="rev-xref" href="#xref-fn-3-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                15
                               </span>
                               These authors jointly supervised the study.
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             REFERENCES
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.1" data-doi="10.1038/nature12477">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alexandrov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nik-Zainal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wedge
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Aparicio
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. a J.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Behjati
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Biankin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bignell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bolli
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Borg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Børresen-Dale
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.-L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Signatures of mutational processes in human cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  500
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  415
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  421
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D500%26rft.spage%253D415%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12477%26rft_id%253Dinfo%253Apmid%252F23945592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature12477&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23945592&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000323316100026&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alves
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Prieto
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Posada
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Multiregional Tumor Trees Are Not Phylogenies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Trends in Cancer
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  546
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  550
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BCancer%26rft.volume%253D3%26rft.spage%253D546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.3" data-doi="10.1038/nature15393">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Auton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Abecasis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Altshuler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Durbin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bentley
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chakravarti
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Clark
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Donnelly
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eichler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Flicek
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A global reference for human genetic variation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  526
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  68
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  74
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D526%26rft.spage%253D68%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature15393%26rft_id%253Dinfo%253Apmid%252F26432245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature15393&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26432245&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barrett
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Feber
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Herrero
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tanic
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wilson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Swanton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Beck
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Quantification of tumour evolution and heterogeneity via Bayesian epiallele detection
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  354
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D18%26rft.spage%253D354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.5" data-doi="10.1101/cshperspect.a019505">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Baylin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.B.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jones
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.A
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Epigenetic Determinants of Cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cold Spring Harb. Perspect. Biol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  a019505
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a019505%26rft_id%253Dinfo%253Apmid%252F27194046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjExOiI4LzkvYTAxOTUwNSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA0LzI0LzIwMjAuMTEuMDMuMzY2MjUyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.6" data-doi="10.1093/nar/gks001">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Benjamini
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Speed
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.P
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Summarizing and correcting the GC content bias in high-throughput sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  40
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e72
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  e72
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks001%26rft_id%253Dinfo%253Apmid%252F22323520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gks001&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22323520&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.7" data-doi="10.1101/gad.947102">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bird
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  DNA methylation patterns and epigenetic memory
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genes Dev
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  6
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  21
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DBird%26rft.auinit1%253DA.%26rft.volume%253D16%26rft.issue%253D1%26rft.spage%253D6%26rft.epage%253D21%26rft.atitle%253DDNA%2Bmethylation%2Bpatterns%2Band%2Bepigenetic%2Bmemory%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.947102%26rft_id%253Dinfo%253Apmid%252F11782440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NjoiMTYvMS82IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDQvMjQvMjAyMC4xMS4wMy4zNjYyNTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.8" data-doi="10.1016/j.celrep.2014.06.053">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brocks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Assenov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Minner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bogatyrova
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Simon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koop
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Oakes
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zucknick
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lipka
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weischenfeldt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Intratumor DNA Methylation Heterogeneity Reflects Clonal Evolution in Aggressive Prostate Cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell Rep
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  798
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  806
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%2BRep%26rft.volume%253D8%26rft.spage%253D798%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2014.06.053%26rft_id%253Dinfo%253Apmid%252F25066126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.celrep.2014.06.053&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25066126&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.9" data-doi="10.1038/nature26000">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Capper
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jones
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.T.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sill
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hovestadt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schrimpf
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sturm
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koelsche
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sahm
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chavez
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reuss
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  DNA methylation-based classification of central nervous system tumours
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  555
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  469
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  474
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D555%26rft.spage%253D469%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature26000%26rft_id%253Dinfo%253Apmid%252F29539639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature26000&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29539639&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.10" data-doi="10.1038/nbt.2203">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cibulskis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Helman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McKenna
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zack
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Laird
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Onofrio
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Winckler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weir
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Absolute quantification of somatic DNA alterations in human cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Biotechnol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  413
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  421
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DCarter%26rft.auinit1%253DS.%2BL.%26rft.volume%253D30%26rft.issue%253D5%26rft.spage%253D413%26rft.epage%253D421%26rft.atitle%253DAbsolute%2Bquantification%2Bof%2Bsomatic%2BDNA%2Balterations%2Bin%2Bhuman%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2203%26rft_id%253Dinfo%253Apmid%252F22544022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nbt.2203&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22544022&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chakravarthy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Furness
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Joshi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ghorani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ford
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ward
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   King
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lechner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marafioti
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Quezada
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Pan-cancer deconvolution of tumour composition using DNA methylation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Commun
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3220
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D9%26rft.spage%253D3220%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ashoor
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Musich
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  epihet for intra-tumoral epigenetic heterogeneity analysis and visualization
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci. Rep
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  376
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSci.%2BRep%26rft.volume%253D11%26rft.spage%253D376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.13" data-doi="10.1038/nature09165">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Costello
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shchors
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fiore
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bilenky
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hirst
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maunakea
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ballinger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jones
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.J.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nagarajan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Conserved role of intragenic DNA methylation in regulating alternative promoters
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  466
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  253
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  257
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMaunakea%26rft.auinit1%253DA.%2BK.%26rft.volume%253D466%26rft.issue%253D7303%26rft.spage%253D253%26rft.epage%253D257%26rft.atitle%253DConserved%2Brole%2Bof%2Bintragenic%2BDNA%2Bmethylation%2Bin%2Bregulating%2Balternative%2Bpromoters.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09165%26rft_id%253Dinfo%253Apmid%252F20613842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature09165&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20613842&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000279580800040&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.14" data-doi="10.1038/274775a0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Coulondre
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Miller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Farabaugh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.J.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gilbert
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  1978
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Molecular basis of base substitution hotspots in Escherichia coli
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  274
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  775
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  780
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCoulondre%26rft.auinit1%253DC.%26rft.volume%253D274%26rft.issue%253D5673%26rft.spage%253D775%26rft.epage%253D780%26rft.atitle%253DMolecular%2Bbasis%2Bof%2Bbase%2Bsubstitution%2Bhotspots%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F274775a0%26rft_id%253Dinfo%253Apmid%252F355893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/274775a0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=355893&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1978FL81400026&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.15">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cui
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Onyango
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brandenburg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hsieh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.L.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Feinberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.P
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Loss of imprinting in colorectal cancer linked to hypomethylation of H19 and IGF2
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cancer Res
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  62
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  6442
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6446
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DCui%26rft.auinit1%253DH.%26rft.volume%253D62%26rft.issue%253D22%26rft.spage%253D6442%26rft.epage%253D6446%26rft.atitle%253DLoss%2Bof%2BImprinting%2Bin%2BColorectal%2BCancer%2BLinked%2Bto%2BHypomethylation%2Bof%2BH19%2Band%2BIGF2%26rft_id%253Dinfo%253Apmid%252F12438232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjEwOiI2Mi8yMi82NDQyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDQvMjQvMjAyMC4xMS4wMy4zNjYyNTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.16" data-doi="10.1101/gad.2037511">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Deaton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.M.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bird
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  CGIs and the regulation of transcription
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genes Dev
                                 </abbr>
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1010
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1022
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DDeaton%26rft.auinit1%253DA.%2BM.%26rft.volume%253D25%26rft.issue%253D10%26rft.spage%253D1010%26rft.epage%253D1022%26rft.atitle%253DCpG%2Bislands%2Band%2Bthe%2Bregulation%2Bof%2Btranscription%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.2037511%26rft_id%253Dinfo%253Apmid%252F21576262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI1LzEwLzEwMTAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNC8yNC8yMDIwLjExLjAzLjM2NjI1Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.17" data-doi="10.1038/s41576-019-0106-6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Deniz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Ö.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Frost
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.M.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Branco
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.R
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Regulation of transposable elements by DNA modifications
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Rev. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  417
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  431
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet%26rft.volume%253D20%26rft.spage%253D417%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41576-019-0106-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41576-019-0106-6&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.18" data-doi="10.1101/cshperspect.a026625">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dentro
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wedge
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.C.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Van Loo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Principles of Reconstructing the Subclonal Architecture of Cancers
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cold Spring Harb. Perspect. Med
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  a026625
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a026625%26rft_id%253Dinfo%253Apmid%252F28270531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTQ6ImNzaHBlcnNwZWN0bWVkIjtzOjU6InJlc2lkIjtzOjExOiI3LzgvYTAyNjYyNSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA0LzI0LzIwMjAuMTEuMDMuMzY2MjUyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.19" data-doi="10.1016/j.cell.2021.03.009">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dentro
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leshchiner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Haase
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tarabichi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wintersinger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Deshwar
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rubanova
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Macintyre
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Demeulemeester
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Characterizing genetic intra-tumor heterogeneity across 2,658 human cancer genomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  184
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2239
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2254.e39
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D184%26rft.spage%253D2239%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2021.03.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2021.03.009&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.20" data-doi="10.1186/1471-2105-15-215">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dolzhenko
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.D
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Using beta-binomial regression for high-precision differential methylation analysis in multifactor whole-genome bisulfite sequencing experiments
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  215
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D215%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-215%26rft_id%253Dinfo%253Apmid%252F24962134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-215&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24962134&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.21" data-doi="10.1038/ng1909">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eckhardt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lewin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cortese
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rakyan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Attwood
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cox
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Davies
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Down
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  DNA methylation profiling of human chromosomes 6, 20 and 22
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Genet.
                                 </abbr>
                                 <span class="cit-vol">
                                  38
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1378
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1385
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DEckhardt%26rft.auinit1%253DF.%26rft.volume%253D38%26rft.issue%253D12%26rft.spage%253D1378%26rft.epage%253D1385%26rft.atitle%253DDNA%2Bmethylation%2Bprofiling%2Bof%2Bhuman%2Bchromosomes%2B6%252C%2B20%2Band%2B22.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1909%26rft_id%253Dinfo%253Apmid%252F17072317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ng1909&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17072317&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000242404200014&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Feber
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dhami
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   de Winter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Martínez-Fernández
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Paul
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hynes-Allen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rezaee
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gurung
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  UroMark—a urinary biomarker assay for the detection of bladder cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Clin. Epigenetics
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  8
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DClin.%2BEpigenetics%26rft.volume%253D9%26rft.spage%253D8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.23" data-doi="10.1038/nrg1748">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Feinberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ohlsson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Henikoff
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The epigenetic progenitor origin of human cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Rev. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  21
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  33
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DFeinberg%26rft.auinit1%253DA.%2BP.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D33%26rft.atitle%253DThe%2Bepigenetic%2Bprogenitor%2Borigin%2Bof%2Bhuman%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1748%26rft_id%253Dinfo%253Apmid%252F16369569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrg1748&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16369569&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000234139800013&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.24" data-doi="10.1038/nrg.2016.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Feinberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koldobskiy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.A.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Göndör
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Epigenetic modulators, modifiers and mediators in cancer aetiology and progression
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Rev. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  284
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  299
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet%26rft.volume%253D17%26rft.spage%253D284%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg.2016.13%26rft_id%253Dinfo%253Apmid%252F26972587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrg.2016.13&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26972587&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.25" data-doi="10.1126/science.1105136">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Feingold
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Good
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guyer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kamholz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liefer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wetterstrand
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Collins
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gingeras
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kampa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sekinger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The ENCODE (ENCyclopedia of DNA Elements) Project
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  306
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  636
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  640
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DThe%2BENCODE%2BProject%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D306%26rft.issue%253D5696%26rft.spage%253D636%26rft.epage%253D640%26rft.atitle%253DThe%2BENCODE%2B%2528ENCyclopedia%2BOf%2BDNA%2BElements%2529%2BProject%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1105136%26rft_id%253Dinfo%253Apmid%252F15499007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDYvNTY5Ni82MzYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNC8yNC8yMDIwLjExLjAzLjM2NjI1Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fernández
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   de la Torre
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Richardson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Royo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Puiggròs
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moncunill
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fragkogianni
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Clarke
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Flicek
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rico
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The BLUEPRINT Data Analysis Portal
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell Syst
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.27" data-doi="10.1126/science.aal2380">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Flavahan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gaskell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bernstein
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.E
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Epigenetic plasticity and the hallmarks of cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  357
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  eaal2380
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFlavahan%26rft.auinit1%253DW.%2BA.%26rft.volume%253D357%26rft.issue%253D6348%26rft.spage%253Deaal2380%26rft.epage%253Deaal2380%26rft.atitle%253DEpigenetic%2Bplasticity%2Band%2Bthe%2Bhallmarks%2Bof%2Bcancer%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aal2380%26rft_id%253Dinfo%253Apmid%252F28729483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE3OiIzNTcvNjM0OC9lYWFsMjM4MCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA0LzI0LzIwMjAuMTEuMDMuMzY2MjUyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.28" data-doi="10.1074/jbc.274.41.29331">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Frevel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.A.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sowerby
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Petersen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.B.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reeve
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.E
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  1999
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Methylation Sequencing Analysis Refines the Region of H19 Epimutation in Wilms Tumor
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J. Biol. Chem
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  274
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  29331
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  29340
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DFrevel%26rft.auinit1%253DM.%2BA.%2BE.%26rft.volume%253D274%26rft.issue%253D41%26rft.spage%253D29331%26rft.epage%253D29340%26rft.atitle%253DMethylation%2BSequencing%2BAnalysis%2BRefines%2Bthe%2BRegion%2BofH19%2BEpimutation%2Bin%2BWilms%2BTumor%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.274.41.29331%26rft_id%253Dinfo%253Apmid%252F10506193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvNDEvMjkzMzEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNC8yNC8yMDIwLjExLjAzLjM2NjI1Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gaiti
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chaligne
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brand
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kothen-Hill
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schulman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grigorev
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Risso
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.-T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pastore
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Epigenetic evolution and lineage histories of chronic lymphocytic leukaemia
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  569
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  576
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  580
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D569%26rft.spage%253D576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.30" data-doi="10.1093/nar/gkl002">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goyal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Accuracy of DNA methylation pattern preservation by the Dnmt1 methyltransferase
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1182
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1188
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGoyal%26rft.auinit1%253DR.%26rft.volume%253D34%26rft.issue%253D4%26rft.spage%253D1182%26rft.epage%253D1188%26rft.atitle%253DAccuracy%2Bof%2BDNA%2Bmethylation%2Bpattern%2Bpreservation%2Bby%2Bthe%2BDnmt1%2Bmethyltransferase%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl002%26rft_id%253Dinfo%253Apmid%252F16500889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkl002&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16500889&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000235923500019&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.31" data-doi="10.1007/BF00286709">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Greger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Passarge
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Höpping
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Messmer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Horsthemke
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  1989
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Epigenetic changes may contribute to the formation and spontaneous regression of retinoblastoma
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Hum. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  83
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  155
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  158
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DHuman%2Bgenetics%26rft.stitle%253DHum%2BGenet%26rft.aulast%253DGreger%26rft.auinit1%253DV.%26rft.volume%253D83%26rft.issue%253D2%26rft.spage%253D155%26rft.epage%253D158%26rft.atitle%253DEpigenetic%2Bchanges%2Bmay%2Bcontribute%2Bto%2Bthe%2Bformation%2Band%2Bspontaneous%2Bregression%2Bof%2Bretinoblastoma.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00286709%26rft_id%253Dinfo%253Apmid%252F2550354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/BF00286709&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=2550354&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1989AQ15600011&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.32" data-doi="10.1038/ng.3805">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Diep
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Plongthongkum
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fung
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.-L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Identification of methylation haplotype blocks aids in deconvolution of heterogeneous tissue samples and tumor tissue-of-origin mapping from plasma DNA
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  49
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  635
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  642
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet%26rft.volume%253D49%26rft.spage%253D635%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3805%26rft_id%253Dinfo%253Apmid%252F28263317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ng.3805&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28263317&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.33" data-doi="10.1016/j.cell.2011.02.013">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hanahan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weinberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.A
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Hallmarks of Cancer: The Next Generation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  144
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  646
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  674
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHanahan%26rft.auinit1%253DD.%26rft.volume%253D144%26rft.issue%253D5%26rft.spage%253D646%26rft.epage%253D674%26rft.atitle%253DHallmarks%2Bof%2Bcancer%253A%2Bthe%2Bnext%2Bgeneration.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.02.013%26rft_id%253Dinfo%253Apmid%252F21376230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.02.013&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21376230&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000288007100007&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.34" data-doi="10.1016/j.molcel.2012.10.016">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hannum
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guinney
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hughes
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sadda
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Klotzle
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bibikova
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.-B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Genome-wide Methylation Profiles Reveal Quantitative Views of Human Aging Rates
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol. Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  49
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  359
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  367
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D49%26rft.spage%253D359%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2012.10.016%26rft_id%253Dinfo%253Apmid%252F23177740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.molcel.2012.10.016&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23177740&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000314379400018&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.35" data-doi="10.1038/ng.865">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hansen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Timp
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bravo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sabunciyan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Langmead
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McDonald
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Diep
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Increased methylation variation in epigenetic domains across cancer types
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  43
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  768
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  775
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHansen%26rft.auinit1%253DK.%2BD.%26rft.volume%253D43%26rft.issue%253D8%26rft.spage%253D768%26rft.epage%253D775%26rft.atitle%253DIncreased%2Bmethylation%2Bvariation%2Bin%2Bepigenetic%2Bdomains%2Bacross%2Bcancer%2Btypes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.865%26rft_id%253Dinfo%253Apmid%252F21706001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ng.865&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21706001&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.36" data-doi="10.1186/gb-2012-13-10-r83">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hansen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Langmead
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Irizarry
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.A
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  BSmooth: from whole genome bisulfite sequencing reads to differentially methylated regions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  R83
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHansen%26rft.auinit1%253DK.%2BD.%26rft.volume%253D13%26rft.issue%253D10%26rft.spage%253DR83%26rft.epage%253DR83%26rft.atitle%253DBSmooth%253A%2Bfrom%2Bwhole%2Bgenome%2Bbisulfite%2Bsequencing%2Breads%2Bto%2Bdifferentially%2Bmethylated%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-10-r83%26rft_id%253Dinfo%253Apmid%252F23034175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-10-r83&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23034175&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.37">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.-J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.-C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dinh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.Q.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mayakonda
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jiang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.-Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jiang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.-C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.-Z.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Spatial intratumoral heterogeneity and temporal clonal evolution in esophageal squamous cell carcinoma
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  48
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1500
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1507
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet%26rft.volume%253D48%26rft.spage%253D1500%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.38" data-doi="10.1038/nrg3270">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Heyn
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Esteller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  DNA methylation profiling in the clinic: applications and challenges
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Rev. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  679
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  692
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHeyn%26rft.auinit1%253DH.%26rft.volume%253D13%26rft.issue%253D10%26rft.spage%253D679%26rft.epage%253D692%26rft.atitle%253DDNA%2Bmethylation%2Bprofiling%2Bin%2Bthe%2Bclinic%253A%2Bapplications%2Band%2Bchallenges.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3270%26rft_id%253Dinfo%253Apmid%252F22945394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrg3270&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22945394&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.39" data-doi="10.1186/gb-2013-14-10-r115">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Horvath
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  DNA methylation age of human tissues and cell types
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  R115
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D14%26rft.spage%253DR115%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-10-r115%26rft_id%253Dinfo%253Apmid%252F24138928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-10-r115&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24138928&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.40">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Estecio
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reuben
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fujimoto
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carrot-Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McGranahan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ying
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fukuoka
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Evolution of DNA methylome from precancerous lesions to invasive lung adenocarcinomas
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Commun
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  687
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D12%26rft.spage%253D687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.41">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hua
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pesatori
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Consonni
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Caporaso
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jones
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hicks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Genetic and epigenetic intratumor heterogeneity impacts prognosis of lung adenocarcinoma
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Commun
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2459
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D11%26rft.spage%253D2459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.42" data-doi="10.1126/science.1229259">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Huang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hodis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kryukov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Garraway
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.A
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Highly Recurrent TERT Promoter Mutations in Human Melanoma
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  339
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  957
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  959
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1229259%26rft_id%253Dinfo%253Apmid%252F23348506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzkvNjEyMi85NTciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNC8yNC8yMDIwLjExLjAzLjM2NjI1Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.43" data-doi="10.1016/j.cell.2018.03.042">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hutter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zenklusen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.C
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The Cancer Genome Atlas: Creating Lasting Value beyond Its Data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  173
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  283
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  285
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D173%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.03.042%26rft_id%253Dinfo%253Apmid%252F29625045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.03.042&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29625045&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.44" data-doi="10.1038/s41586-020-1969-6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Pan-cancer analysis of whole genomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  578
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  82
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  93
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D578%26rft.spage%253D82%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-1969-6%26rft_id%253Dinfo%253Apmid%252F32025007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-020-1969-6&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=32025007&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.45" data-doi="10.1056/NEJMoa1616288">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jamal-Hanjani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wilson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McGranahan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Birkbak
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Watkins
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.B.K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Veeriah
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shafi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Johnson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mitter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rosenthal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Tracking the Evolution of Non–Small-Cell Lung Cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  N. Engl. J. Med
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  376
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2109
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2121
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed%26rft.volume%253D376%26rft.spage%253D2109%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1616288%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1056/NEJMoa1616288&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.46" data-doi="10.1038/ng.174">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kerkel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Spadola
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yuan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kosek
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jiang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hod
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Murty
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V. V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schupf
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vilain
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Genomic surveys by methylation-sensitive SNP analysis identify sequence-dependent allele-specific DNA methylation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  40
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  904
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  908
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKerkel%26rft.auinit1%253DK.%26rft.volume%253D40%26rft.issue%253D7%26rft.spage%253D904%26rft.epage%253D908%26rft.atitle%253DGenomic%2Bsurveys%2Bby%2Bmethylation-sensitive%2BSNP%2Banalysis%2Bidentify%2Bsequence-dependent%2Ballele-specific%2BDNA%2Bmethylation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.174%26rft_id%253Dinfo%253Apmid%252F18568024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ng.174&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18568024&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000257166500027&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.47" data-doi="10.1093/bib/bbv095">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Klein
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.U.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hebestreit
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  An evaluation of methods to test predefined genomic regions for differential methylation in bisulfite sequencing data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Brief. Bioinform
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  796
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  807
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbv095%26rft_id%253Dinfo%253Apmid%252F26515532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bib/bbv095&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26515532&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.48" data-doi="10.1038/s41591-018-0156-x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Klughammer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kiesel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Roetzer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fortelny
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nemc
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nenning
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.-H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Furtner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sheffield
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Datlinger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Peter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The DNA methylation landscape of glioblastoma disease progression shows extensive heterogeneity in time and space
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Med
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  24
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1611
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1624
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BMed%26rft.volume%253D24%26rft.spage%253D1611%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41591-018-0156-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41591-018-0156-x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.49" data-doi="10.1038/s41571-018-0004-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koch
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Joosten
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Feng
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   de Ruijter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Draht
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Melotte
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smits
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Veeck
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Herman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Van Neste
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Analysis of DNA methylation in cancer: location revisited
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Rev. Clin. Oncol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  459
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  466
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BClin.%2BOncol%26rft.volume%253D15%26rft.spage%253D459%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41571-018-0004-4%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41571-018-0004-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.50" data-doi="10.1038/nature12750">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kohli
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.M.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  TET enzymes, TDG and the dynamics of DNA demethylation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  502
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  472
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  479
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D502%26rft.spage%253D472%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12750%26rft_id%253Dinfo%253Apmid%252F24153300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature12750&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24153300&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000325988400041&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.51" data-doi="10.1093/bioinformatics/btr167">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Krueger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Andrews
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.R
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Bismark: A flexible aligner and methylation caller for Bisulfite-Seq applications
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1571
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1572
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr167%26rft_id%253Dinfo%253Apmid%252F21493656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr167&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21493656&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000291062400018&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.52" data-doi="10.1038/ng.2443">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kulis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Heath
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bibikova
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Queirós
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Navarro
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Clot
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Martínez-Trillos
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Castellano
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brun-Heath
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pinyol
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Epigenomic analysis detects widespread gene-body DNA hypomethylation in chronic lymphocytic leukemia
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  44
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1236
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1242
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKulis%26rft.auinit1%253DM.%26rft.volume%253D44%26rft.issue%253D11%26rft.spage%253D1236%26rft.epage%253D1242%26rft.atitle%253DEpigenomic%2Banalysis%2Bdetects%2Bwidespread%2Bgene-body%2BDNA%2Bhypomethylation%2Bin%2Bchronic%2Blymphocytic%2Bleukemia.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2443%26rft_id%253Dinfo%253Apmid%252F23064414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ng.2443&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23064414&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.53" data-doi="10.1038/ng.2442">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Landan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cohen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mukamel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bar
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Molchadsky
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brosh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Horn-Saban
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zalcenstein
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goldfinger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zundelevich
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Epigenetic polymorphism and the stochastic formation of differentially methylated regions in normal and cancerous tissues
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  44
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1207
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1214
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLandan%26rft.auinit1%253DG.%26rft.volume%253D44%26rft.issue%253D11%26rft.spage%253D1207%26rft.epage%253D1214%26rft.atitle%253DEpigenetic%2Bpolymorphism%2Band%2Bthe%2Bstochastic%2Bformation%2Bof%2Bdifferentially%2Bmethylated%2Bregions%2Bin%2Bnormal%2Band%2Bcancerous%2Btissues.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2442%26rft_id%253Dinfo%253Apmid%252F23064413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ng.2442&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23064413&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.54" data-doi="10.1016/j.ccell.2014.10.012">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Landau
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Clement
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ziller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Boyle
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stevenson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sougnez
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Locally Disordered Methylation Forms the Basis of Intratumor Methylome Variation in Chronic Lymphocytic Leukemia
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cancer Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  813
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  825
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCancer%2BCell%26rft.volume%253D26%26rft.spage%253D813%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ccell.2014.10.012%26rft_id%253Dinfo%253Apmid%252F25490447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.ccell.2014.10.012&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25490447&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.55" data-doi="10.1038/nm.4125">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Garrett-Bakelman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chung
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sanders
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hricik
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rapaport
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Patel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dillon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vijay
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brown
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Distinct evolution and dynamics of epigenetic and genetic heterogeneity in acute myeloid leukemia
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Med
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  792
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  799
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BMed%26rft.volume%253D22%26rft.spage%253D792%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm.4125%26rft_id%253Dinfo%253Apmid%252F27322744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nm.4125&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27322744&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.56" data-doi="10.1038/nature08514">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lister
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pelizzola
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dowen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hawkins
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tonti-Filippini
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nery
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ye
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ngo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Q.-M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Human DNA methylomes at base resolution show widespread epigenomic differences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  462
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  315
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  322
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLister%26rft.auinit1%253DR.%26rft.volume%253D462%26rft.issue%253D7271%26rft.spage%253D315%26rft.epage%253D322%26rft.atitle%253DHuman%2BDNA%2Bmethylomes%2Bat%2Bbase%2Bresolution%2Bshow%2Bwidespread%2Bepigenomic%2Bdifferences.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08514%26rft_id%253Dinfo%253Apmid%252F19829295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature08514&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19829295&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000271899300037&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.57" data-doi="10.1186/gb-2012-13-7-r61">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Siegmund
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Laird
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.W.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Berman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.P
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Bis-SNP: Combined DNA methylation and SNP calling for Bisulfite-seq data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  R61
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLiu%26rft.auinit1%253DY.%26rft.volume%253D13%26rft.issue%253D7%26rft.spage%253DR61%26rft.epage%253DR61%26rft.atitle%253DBis-SNP%253A%2Bcombined%2BDNA%2Bmethylation%2Band%2BSNP%2Bcalling%2Bfor%2BBisulfite-seq%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-7-r61%26rft_id%253Dinfo%253Apmid%252F22784381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-7-r61&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22784381&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.58">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lyon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.F
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  1962
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Sex chromatin and gene action in the mammalian X-chromosome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Am. J. Hum. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  135
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  148
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DLyon%26rft.auinit1%253DM.%2BF.%26rft.volume%253D14%26rft.spage%253D135%26rft.epage%253D148%26rft.atitle%253DSex%2Bchromatin%2Band%2Bgene%2Baction%2Bin%2Bthe%2Bmammalian%2BX-chromosome.%26rft_id%253Dinfo%253Apmid%252F14467629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=14467629&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A19622963A00006&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.59" data-doi="10.1126/science.1259037">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mansour
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Abraham
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Anders
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Berezovskaya
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gutierrez
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Durbin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Etchin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lawton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sallan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Silverman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  An oncogenic super-enhancer formed through somatic mutation of a noncoding intergenic element
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  346
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1373
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1377
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1259037%26rft_id%253Dinfo%253Apmid%252F25394790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDYvNjIxNS8xMzczIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDQvMjQvMjAyMC4xMS4wMy4zNjYyNTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.60">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Martelotto
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.G.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  ‘Frankenstein’ protocol for nuclei isolation from fresh and frozen tissue for snRNAseq
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.61">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Martin-Trujillo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vidal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Monteagudo-Sánchez
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sanchez-Delgado
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moran
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hernandez Mora
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Heyn
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guitart
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Esteller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Monk
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Copy number rather than epigenetic alterations are the major dictator of imprinted methylation in tumors
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Commun
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  467
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D8%26rft.spage%253D467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.62" data-doi="10.14806/EJ.17.1.200">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Martin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Cutadapt removes adapter sequences from high-throughput sequencing reads
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  EMBnet.Journal
                                 </abbr>
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  10
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DEMBnet.Journal%26rft.volume%253D17%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.14806%252FEJ.17.1.200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.14806/EJ.17.1.200&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.63" data-doi="10.1016/j.ccell.2015.07.012">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mazor
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pankov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Johnson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hamilton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.J.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smirnov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I. V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Phillips
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barnes
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  DNA Methylation and Somatic Mutations Converge on the Cell Cycle and Define Similar Evolutionary Histories in Brain Tumors
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cancer Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  307
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  317
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCancer%2BCell%26rft.volume%253D28%26rft.spage%253D307%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ccell.2015.07.012%26rft_id%253Dinfo%253Apmid%252F26373278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.ccell.2015.07.012&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26373278&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.64" data-doi="10.1093/nar/gki901">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Meissner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Reduced representation bisulfite sequencing for comparative high-resolution DNA methylation analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  33
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  5868
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  5877
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMeissner%26rft.auinit1%253DA.%26rft.volume%253D33%26rft.issue%253D18%26rft.spage%253D5868%26rft.epage%253D5877%26rft.atitle%253DReduced%2Brepresentation%2Bbisulfite%2Bsequencing%2Bfor%2Bcomparative%2Bhigh-resolution%2BDNA%2Bmethylation%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki901%26rft_id%253Dinfo%253Apmid%252F16224102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gki901&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16224102&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000233046100029&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.65">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Morova
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McNeill
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lallous
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gönen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dalal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wilson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gürsoy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Keskin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Ö.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lack
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.A
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Androgen receptor-binding sites are highly mutated in prostate cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Commun
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  832
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D11%26rft.spage%253D832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.66" data-doi="10.1371/journal.pone.0214816">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Neininger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marschall
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Helms
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  SNP and indel frequencies at transcription start sites and at canonical and alternative translation initiation sites in the human genome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS One
                                 </abbr>
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e0214816
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D14%26rft.spage%253De0214816%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0214816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0214816&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.67" data-doi="10.1016/j.cell.2012.04.023">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nik-Zainal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Van Loo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wedge
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alexandrov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Greenman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lau
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Raine
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jones
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marshall
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ramakrishna
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The life history of 21 breast cancers
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  149
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  994
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1007
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DNik-Zainal%26rft.auinit1%253DS.%26rft.volume%253D149%26rft.issue%253D5%26rft.spage%253D994%26rft.epage%253D1007%26rft.atitle%253DThe%2Blife%2Bhistory%2Bof%2B21%2Bbreast%2Bcancers.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.04.023%26rft_id%253Dinfo%253Apmid%252F22608083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.04.023&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22608083&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.68" data-doi="10.1186/gb-2013-14-9-r105">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nordlund
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bäcklin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wahlberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Busche
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Berglund
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eloranta
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.-L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Flaegstad
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Forestier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Frost
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.-M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harila-Saari
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Genome-wide signatures of differential DNA methylation in pediatric acute lymphoblastic leukemia
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  r105
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D14%26rft.spage%253Dr105%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-9-r105%26rft_id%253Dinfo%253Apmid%252F24063430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-9-r105&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24063430&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.69" data-doi="10.1038/ng.3488">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Oakes
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Seifert
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Assenov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Przekopowitz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ruppert
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Q.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Imbusch
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Serva
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koser
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  DNA methylation dynamics during B cell maturation underlie a continuum of disease phenotypes in chronic lymphocytic leukemia
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  48
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  253
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  264
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet%26rft.volume%253D48%26rft.spage%253D253%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3488%26rft_id%253Dinfo%253Apmid%252F26780610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ng.3488&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26780610&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.70" data-doi="10.1038/362749a0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ogawa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eccles
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Szeto
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McNoe
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yun
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maw
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.J.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reeve
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.E
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  1993
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Relaxation of insulin-like growth factor II gene imprinting implicated in Wilms’ tumour
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  362
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  749
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  751
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DOgawa%26rft.auinit1%253DO.%26rft.volume%253D362%26rft.issue%253D6422%26rft.spage%253D749%26rft.epage%253D751%26rft.atitle%253DRelaxation%2Bof%2Binsulin-like%2Bgrowth%2Bfactor%2BII%2Bgene%2Bimprinting%2Bimplicated%2Bin%2BWilms%2527%2Btumour.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F362749a0%26rft_id%253Dinfo%253Apmid%252F8097018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/362749a0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=8097018&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1993KY45000056&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.71" data-doi="10.1093/bioinformatics/btg412">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Paradis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Claude
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Strimmer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  APE: Analyses of Phylogenetics and Evolution in R language
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  289
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  290
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DParadis%26rft.auinit1%253DE.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D289%26rft.epage%253D290%26rft.atitle%253DAPE%253A%2BAnalyses%2Bof%2BPhylogenetics%2Band%2BEvolution%2Bin%2BR%2Blanguage%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg412%26rft_id%253Dinfo%253Apmid%252F14734327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg412&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=14734327&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000188389700026&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.72" data-doi="10.1038/nature13268">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pfister
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eils
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Taylor
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yaspo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.-L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pietsch
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bunt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stachurski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Korshunov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Warnatz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.-J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koster
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Decoding the regulatory landscape of medulloblastoma using DNA methylation sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  510
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  537
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  541
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D510%26rft.spage%253D537%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13268%26rft_id%253Dinfo%253Apmid%252F24847876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature13268&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24847876&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.73" data-doi="10.1038/nm.3394">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Quail
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.F.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Joyce
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.A
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Microenvironmental regulation of tumor progression and metastasis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Med
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1423
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1437
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BMed%26rft.volume%253D19%26rft.spage%253D1423%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm.3394%26rft_id%253Dinfo%253Apmid%252F24202395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nm.3394&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24202395&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.74">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Raineri
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dabad
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Heath
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A Note on Exact Differences between Beta Distributions in Genomic (Methylation) Studies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS One
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e97349
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De97349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.75" data-doi="10.1038/35047554">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reik
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Walter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Genomic imprinting: parental influence on the genome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Rev. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  21
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  32
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DReik%26rft.auinit1%253DW.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D32%26rft.atitle%253DGenomic%2Bimprinting%253A%2Bparental%2Binfluence%2Bon%2Bthe%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35047554%26rft_id%253Dinfo%253Apmid%252F11253064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/35047554&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=11253064&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000166056000014&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.76">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Robinson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Introduction to Empirical Bayes: Examples from Baseball Statistics
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.77" data-doi="10.3389/fgene.2014.00001">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Robinson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kahraman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Law
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lindsay
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nowicka
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weber
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.M.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhou
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Statistical methods for detecting differentially methylated loci and regions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Front. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  7
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet%26rft.volume%253D5%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2014.00001%26rft_id%253Dinfo%253Apmid%252F24567736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.3389/fgene.2014.00001&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24567736&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.78" data-doi="10.1186/s13059-016-0893-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rosenthal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McGranahan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Herrero
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Taylor
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.S.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Swanton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  deconstructSigs: delineating mutational processes in single tumors distinguishes DNA repair deficiencies and patterns of carcinoma evolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  31
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D17%26rft.spage%253D31%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-0893-4%26rft_id%253Dinfo%253Apmid%252F26899170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-016-0893-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26899170&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.79">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rosenthal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Larose Cadieux
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Salgado
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bakir
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. Al
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moore
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hiley
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lund
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tanić
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reading
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Joshi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Neoantigen-directed immune escape in lung cancer evolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  567
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  479
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  485
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D567%26rft.spage%253D479%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.80" data-doi="10.1101/gr.099655.109">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ryba
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hiratani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Itoh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kulik
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schulz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Robins
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dalton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gilbert
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.M
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Evolutionarily conserved replication timing profiles predict long-range chromatin interactions and distinguish closely related cell types
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Res
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  761
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  770
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.099655.109%26rft_id%253Dinfo%253Apmid%252F20430782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzYvNzYxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDQvMjQvMjAyMC4xMS4wMy4zNjYyNTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.81" data-doi="10.1038/nature14465">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schultz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   He
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Whitaker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hariharan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mukamel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leung
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rajagopal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nery
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Urich
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Human body epigenome maps reveal noncanonical DNA methylation variation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  523
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  212
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  216
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D523%26rft.spage%253D212%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14465%26rft_id%253Dinfo%253Apmid%252F26030523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature14465&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26030523&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.82" data-doi="10.1038/nm.4273">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sheffield
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pierron
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Klughammer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Datlinger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schönegger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schuster
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hadler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Surdez
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guillemot
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lapouble
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  DNA methylation heterogeneity defines a disease spectrum in Ewing sarcoma
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Med
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  23
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  386
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  395
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BMed%26rft.volume%253D23%26rft.spage%253D386%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm.4273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nm.4273&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.83" data-doi="10.1016/j.cell.2013.03.008">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Laird
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.W
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Interplay between the Cancer Genome and Epigenome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  153
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  38
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  55
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D153%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.03.008%26rft_id%253Dinfo%253Apmid%252F23540689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.03.008&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23540689&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000316853700005&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.84" data-doi="10.1038/nrg3354">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.D.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Meissner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  DNA methylation: Roles in mammalian development
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Rev. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  204
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  220
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet%26rft.volume%253D14%26rft.spage%253D204%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3354%26rft_id%253Dinfo%253Apmid%252F23400093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrg3354&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23400093&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.85" data-doi="10.1016/j.cell.2016.11.007">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stunnenberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hirst
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Abrignani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Adams
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   de Almeida
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Altucci
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Amin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Amit
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Antonarakis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Aparicio
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The International Human Epigenome Consortium: A Blueprint for Scientific Collaboration and Discovery
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  167
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1145
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1149
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D167%26rft.spage%253D1145%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.11.007%26rft_id%253Dinfo%253Apmid%252F27863232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.11.007&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27863232&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.86" data-doi="10.2217/epi.15.21">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sun
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cunningham
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Slager
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kocher
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.-P
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Base resolution methylome profiling: considerations in platform selection, data preprocessing and analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Epigenomics
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  813
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  828
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DEpigenomics%26rft.volume%253D7%26rft.spage%253D813%26rft_id%253Dinfo%253Adoi%252F10.2217%252Fepi.15.21%26rft_id%253Dinfo%253Apmid%252F26366945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.2217/epi.15.21&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26366945&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.87" data-doi="10.1016/j.molonc.2010.05.002">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sutherland
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.D.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Berns
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Cell of origin of lung cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol. Oncol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  397
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  403
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DMol%2BOncol%26rft.aulast%253DSutherland%26rft.auinit1%253DK.%2BD.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253D397%26rft.epage%253D403%26rft.atitle%253DCell%2Bof%2Borigin%2Bof%2Blung%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molonc.2010.05.002%26rft_id%253Dinfo%253Apmid%252F20594926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.molonc.2010.05.002&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20594926&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.88" data-doi="10.1038/s41592-020-01013-2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tarabichi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Salcedo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Deshwar
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ni Leathlobhair
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wintersinger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wedge
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Van Loo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Morris
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Q.D.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Boutros
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A practical guide to cancer subclonal reconstruction from DNA sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Methods
                                 </abbr>
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  144
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  155
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D18%26rft.spage%253D144%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41592-020-01013-2%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41592-020-01013-2&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.89" data-doi="10.1186/s12859-017-1511-5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Teschendorff
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Breeze
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zheng
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.C.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Beck
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A comparison of reference-based algorithms for correcting cell-type heterogeneity in Epigenome-Wide Association Studies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  105
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D18%26rft.spage%253D105%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-017-1511-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s12859-017-1511-5&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.90" data-doi="10.1073/pnas.1009843107">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Van Loo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nordgard
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lingjaerde
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Russnes
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rye
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sun
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weigman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marynen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zetterberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Naume
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Allele-specific copy number analysis of tumors
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  107
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  16910
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  16915
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1009843107%26rft_id%253Dinfo%253Apmid%252F20837533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzM5LzE2OTEwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDQvMjQvMjAyMC4xMS4wMy4zNjYyNTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.91" data-doi="10.1038/ng.298">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Montano
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gabo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Potash
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ji
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sabunciyan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rongione
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Webster
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Irizarry
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  41
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  178
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  186
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DIrizarry%26rft.auinit1%253DR.%2BA.%26rft.volume%253D41%26rft.issue%253D2%26rft.spage%253D178%26rft.epage%253D186%26rft.atitle%253DThe%2Bhuman%2Bcolon%2Bcancer%2Bmethylome%2Bshows%2Bsimilar%2Bhypo-%2Band%2Bhypermethylation%2Bat%2Bconserved%2Btissue-specific%2BCpG%2Bisland%2Bshores.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.298%26rft_id%253Dinfo%253Apmid%252F19151715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ng.298&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19151715&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000263091300017&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.92" data-doi="10.1093/nar/gku1138">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Feng
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Qin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Conneely
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.N
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Detection of differentially methylated regions from whole-genome bisulfite sequencing data without replicates
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  43
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  141
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D43%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gku1138&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.93">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Seo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Allen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.S.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gordân
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Quantifying the Impact of Non-coding Variants on Transcription Factor-DNA Binding
                                 </span>
                                 . pp.
                                 <span class="cit-fpage">
                                  336
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  352
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.94">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Foye
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sjöström
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Aggarwal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Playdle
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alumkal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The DNA methylation landscape of advanced prostate cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  52
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  778
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  789
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet%26rft.volume%253D52%26rft.spage%253D778%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.95" data-doi="10.1186/s13059-014-0419-x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zheng
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Q.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.-J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Meyer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Qin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Q.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jiang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  MethylPurify: tumor purity deconvolution and differential methylation detection from single tumor DNA methylomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  419
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D15%26rft.spage%253D419%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0419-x%26rft_id%253Dinfo%253Apmid%252F25103624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-014-0419-x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25103624&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.96" data-doi="10.1186/s13059-016-1143-5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zheng
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.-J.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Estimating and accounting for tumor purity in the analysis of DNA methylation data from cancer studies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  17
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D18%26rft.spage%253D17%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-1143-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-016-1143-5&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.03.366252v2.97" data-doi="10.1038/nature12433">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ziller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Müller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Donaghey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tsai
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.T.-Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kohlbacher
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   De Jager
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rosen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bennett
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bernstein
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Charting a dynamic DNA methylation landscape of the human genome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  500
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  477
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  481
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D500%26rft.spage%253D477%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12433%26rft_id%253Dinfo%253Apmid%252F23925113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature12433&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23925113&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F24%2F2020.11.03.366252.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000323316100039&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.11.03.366252v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.01.13.475824v2" title="Functional resilience of mutually repressing motifs embedded in larger networks" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2021.12.22.473938v3" title="An analytical solution simulating growth of Lewy bodies" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted April 24, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.11.03.366252v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.11.03.366252v2.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.11.03.366252v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.11.03.366252v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   Copy number-aware deconvolution of tumor-normal DNA methylation profiles
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2510908"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-kddgZB3cMfUuAHtsbYt_iKxHO8aKT7PbKijAhhDG0PQ"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484484140"/>
                                   <input type="hidden" name="captcha_token" value="a58a012569775bcc1ae0e508e770bf73"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2510908--3" data-node-nid="2510908" id="node-2510908--41461477410" data-pisa="biorxiv;2020.11.03.366252v2" data-pisa-master="biorxiv;2020.11.03.366252" data-seqnum="2510908" data-apath="/biorxiv/early/2022/04/24/2020.11.03.366252.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Copy number-aware deconvolution of tumor-normal DNA methylation profiles
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Elizabeth Larose
                                  </span>
                                  <span class="nlm-surname">
                                   Cadieux
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Nana E.
                                  </span>
                                  <span class="nlm-surname">
                                   Mensah
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Carla
                                  </span>
                                  <span class="nlm-surname">
                                   Castignani
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Miljana
                                  </span>
                                  <span class="nlm-surname">
                                   Tanić
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Gareth A.
                                  </span>
                                  <span class="nlm-surname">
                                   Wilson
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Michelle
                                  </span>
                                  <span class="nlm-surname">
                                   Dietzen
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">
                                   Pawan
                                  </span>
                                  <span class="nlm-surname">
                                   Dhami
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="7">
                                  <span class="nlm-given-names">
                                   Heli
                                  </span>
                                  <span class="nlm-surname">
                                   Vaikkinen
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="8">
                                  <span class="nlm-given-names">
                                   Annelien
                                  </span>
                                  <span class="nlm-surname">
                                   Verfaillie
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="9">
                                  <span class="nlm-given-names">
                                   Cristina Cotobal
                                  </span>
                                  <span class="nlm-surname">
                                   Martin
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="10">
                                  <span class="nlm-given-names">
                                   Toby
                                  </span>
                                  <span class="nlm-surname">
                                   Baker
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="11">
                                  <span class="nlm-given-names">
                                   Thomas B. K.
                                  </span>
                                  <span class="nlm-surname">
                                   Watkins
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="12">
                                  <span class="nlm-given-names">
                                   Selvaraju
                                  </span>
                                  <span class="nlm-surname">
                                   Veeriah
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="13">
                                  <span class="nlm-given-names">
                                   Mariam
                                  </span>
                                  <span class="nlm-surname">
                                   Jamal-Hanjani
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="14">
                                  <span class="nlm-given-names">
                                   Nnennaya
                                  </span>
                                  <span class="nlm-surname">
                                   Kanu
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="15">
                                  <span class="nlm-given-names">
                                   Nicholas
                                  </span>
                                  <span class="nlm-surname">
                                   McGranahan
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="16">
                                  <span class="nlm-given-names">
                                   Andrew
                                  </span>
                                  <span class="nlm-surname">
                                   Feber
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="17">
                                  <span class="nlm-collab" data-hwp-id="collab-1">
                                   TRACERx Consortium
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="18">
                                  <span class="nlm-given-names">
                                   Charles
                                  </span>
                                  <span class="nlm-surname">
                                   Swanton
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="19">
                                  <span class="nlm-given-names">
                                   Stephan
                                  </span>
                                  <span class="nlm-surname">
                                   Beck
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="20">
                                  <span class="nlm-given-names">
                                   Jonas
                                  </span>
                                  <span class="nlm-surname">
                                   Demeulemeester
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="21">
                                  <span class="nlm-given-names">
                                   Peter
                                  </span>
                                  <span class="nlm-surname">
                                   Van Loo
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.11.03.366252;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.11.03.366252
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.11.03.366252v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.11.03.366252v2&title=Copy%20number-aware%20deconvolution%20of%20tumor-normal%20DNA%20methylation%20profiles" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.03.366252v2&title=Copy%20number-aware%20deconvolution%20of%20tumor-normal%20DNA%20methylation%20profiles" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.03.366252v2&text=Copy%20number-aware%20deconvolution%20of%20tumor-normal%20DNA%20methylation%20profiles" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.11.03.366252v2&t=Copy%20number-aware%20deconvolution%20of%20tumor-normal%20DNA%20methylation%20profiles" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.11.03.366252v2&title=Copy%20number-aware%20deconvolution%20of%20tumor-normal%20DNA%20methylation%20profiles" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.11.03.366252v2&title=Copy%20number-aware%20deconvolution%20of%20tumor-normal%20DNA%20methylation%20profiles&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.03.366252v2&title=Copy%20number-aware%20deconvolution%20of%20tumor-normal%20DNA%20methylation%20profiles" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2510908--5" data-node-nid="2510908" id="citation-node-2510908--6682992997" data-pisa="biorxiv;2020.11.03.366252v2" data-pisa-master="biorxiv;2020.11.03.366252" data-seqnum="2510908" data-apath="/biorxiv/early/2022/04/24/2020.11.03.366252.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Copy number-aware deconvolution of tumor-normal DNA methylation profiles
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Elizabeth Larose
                                      </span>
                                      <span class="nlm-surname">
                                       Cadieux
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Nana E.
                                      </span>
                                      <span class="nlm-surname">
                                       Mensah
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Carla
                                      </span>
                                      <span class="nlm-surname">
                                       Castignani
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Miljana
                                      </span>
                                      <span class="nlm-surname">
                                       Tanić
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Gareth A.
                                      </span>
                                      <span class="nlm-surname">
                                       Wilson
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Michelle
                                      </span>
                                      <span class="nlm-surname">
                                       Dietzen
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">
                                       Pawan
                                      </span>
                                      <span class="nlm-surname">
                                       Dhami
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="7">
                                      <span class="nlm-given-names">
                                       Heli
                                      </span>
                                      <span class="nlm-surname">
                                       Vaikkinen
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="8">
                                      <span class="nlm-given-names">
                                       Annelien
                                      </span>
                                      <span class="nlm-surname">
                                       Verfaillie
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="9">
                                      <span class="nlm-given-names">
                                       Cristina Cotobal
                                      </span>
                                      <span class="nlm-surname">
                                       Martin
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="10">
                                      <span class="nlm-given-names">
                                       Toby
                                      </span>
                                      <span class="nlm-surname">
                                       Baker
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="11">
                                      <span class="nlm-given-names">
                                       Thomas B. K.
                                      </span>
                                      <span class="nlm-surname">
                                       Watkins
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="12">
                                      <span class="nlm-given-names">
                                       Selvaraju
                                      </span>
                                      <span class="nlm-surname">
                                       Veeriah
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="13">
                                      <span class="nlm-given-names">
                                       Mariam
                                      </span>
                                      <span class="nlm-surname">
                                       Jamal-Hanjani
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="14">
                                      <span class="nlm-given-names">
                                       Nnennaya
                                      </span>
                                      <span class="nlm-surname">
                                       Kanu
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="15">
                                      <span class="nlm-given-names">
                                       Nicholas
                                      </span>
                                      <span class="nlm-surname">
                                       McGranahan
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="16">
                                      <span class="nlm-given-names">
                                       Andrew
                                      </span>
                                      <span class="nlm-surname">
                                       Feber
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="17">
                                      <span class="nlm-collab" data-hwp-id="collab-1">
                                       TRACERx Consortium
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="18">
                                      <span class="nlm-given-names">
                                       Charles
                                      </span>
                                      <span class="nlm-surname">
                                       Swanton
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="19">
                                      <span class="nlm-given-names">
                                       Stephan
                                      </span>
                                      <span class="nlm-surname">
                                       Beck
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="20">
                                      <span class="nlm-given-names">
                                       Jonas
                                      </span>
                                      <span class="nlm-surname">
                                       Demeulemeester
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="21">
                                      <span class="nlm-given-names">
                                       Peter
                                      </span>
                                      <span class="nlm-surname">
                                       Van Loo
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.11.03.366252;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.11.03.366252
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Copy%20number-aware%20deconvolution%20of%20tumor-normal%20DNA%20methylation%20profiles&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Cadieux%2C+Elizabeth+Larose&rft.au=Mensah%2C+Nana+E.&rft.au=Castignani%2C+Carla&rft.au=Tani%C4%87%2C+Miljana&rft.au=Wilson%2C+Gareth+A.&rft.au=Dietzen%2C+Michelle&rft.au=Dhami%2C+Pawan&rft.au=Vaikkinen%2C+Heli&rft.au=Verfaillie%2C+Annelien&rft.au=Martin%2C+Cristina+Cotobal&rft.au=Baker%2C+Toby&rft.au=Watkins%2C+Thomas+B.+K.&rft.au=Veeriah%2C+Selvaraju&rft.au=Jamal-Hanjani%2C+Mariam&rft.au=Kanu%2C+Nnennaya&rft.au=McGranahan%2C+Nicholas&rft.au=Feber%2C+Andrew&rft.au=%2C+&rft.au=Swanton%2C+Charles&rft.au=Beck%2C+Stephan&rft.au=Demeulemeester%2C+Jonas&rft.au=Van+Loo%2C+Peter">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2510908/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2510908/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2510908/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2510908/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2510908/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2510908/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2510908/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2510908/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2510908/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2510908/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2510908/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2510908/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.03.366252v2&count=horizontal&via=&text=Copy%20number-aware%20deconvolution%20of%20tumor-normal%20DNA%20methylation%20profiles&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.11.03.366252v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.11.03.366252v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.11.03.366252v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Genomics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23507)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11824)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11440)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15189)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47703)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
